{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [
    {
     "ename": "ImportError",
     "evalue": "cannot import name 'GProfiler' from 'gprofiler' (unknown location)",
     "output_type": "error",
     "traceback": [
      "\u001b[31m---------------------------------------------------------------------------\u001b[39m",
      "\u001b[31mImportError\u001b[39m                               Traceback (most recent call last)",
      "\u001b[36mCell\u001b[39m\u001b[36m \u001b[39m\u001b[32mIn[2]\u001b[39m\u001b[32m, line 3\u001b[39m\n\u001b[32m      1\u001b[39m \u001b[38;5;28;01mimport\u001b[39;00m\u001b[38;5;250m \u001b[39m\u001b[34;01mpandas\u001b[39;00m\u001b[38;5;250m \u001b[39m\u001b[38;5;28;01mas\u001b[39;00m\u001b[38;5;250m \u001b[39m\u001b[34;01mpd\u001b[39;00m\n\u001b[32m      2\u001b[39m \u001b[38;5;28;01mimport\u001b[39;00m\u001b[38;5;250m \u001b[39m\u001b[34;01mmygene\u001b[39;00m\n\u001b[32m----> \u001b[39m\u001b[32m3\u001b[39m \u001b[38;5;28;01mfrom\u001b[39;00m\u001b[38;5;250m \u001b[39m\u001b[34;01mgprofiler\u001b[39;00m\u001b[38;5;250m \u001b[39m\u001b[38;5;28;01mimport\u001b[39;00m GProfiler\n",
      "\u001b[31mImportError\u001b[39m: cannot import name 'GProfiler' from 'gprofiler' (unknown location)"
     ]
    }
   ],
   "source": [
    "import pandas as pd\n",
    "import mygene\n",
    "from gprofiler import GProfiler\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Input sequence provided is already in string format. No operation performed\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Genes with Label 0 but predicted as 1:\n",
      "            Gene  Label  Pred     Score\n",
      "2           NOS1    0.0     1  0.851384\n",
      "4          ABCC9    0.0     1  1.000000\n",
      "30          FNTB    0.0     1  0.974066\n",
      "75          CDK1    0.0     1  0.999710\n",
      "76         CCNB1    0.0     1  0.994031\n",
      "...          ...    ...   ...       ...\n",
      "13564     ABCC11    0.0     1  0.999998\n",
      "13583     ZNF582    0.0     1  0.999664\n",
      "13595       FAAH    0.0     1  0.999997\n",
      "13598   HLA-DRB1    0.0     1  1.000000\n",
      "13612  KIAA1549L    0.0     1  1.000000\n",
      "\n",
      "[954 rows x 4 columns]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Biological information for each false positive gene:\n",
      "\n",
      "Gene: NOS1\n",
      "Symbol: NOS1\n",
      "Name: nitric oxide synthase 1\n",
      "Summary: The protein encoded by this gene belongs to the family of nitric oxide synthases, which synthesize nitric oxide from L-arginine. Nitric oxide is a reactive free radical, which acts as a biologic mediator in several processes, including neurotransmission, and antimicrobial and antitumoral activities. In the brain and peripheral nervous system, nitric oxide displays many properties of a neurotransmitter, and has been implicated in neurotoxicity associated with stroke and neurodegenerative diseases, neural regulation of smooth muscle, including peristalsis, and penile erection. This protein is ubiquitously expressed, with high level of expression in skeletal muscle. Multiple transcript variants that differ in the 5' UTR have been described for this gene but the full-length nature of these transcripts is not known. Additionally, alternatively spliced transcript variants encoding different isoforms (some testis-specific) have been found for this gene.[provided by RefSeq, Feb 2011].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - response to hypoxia\n",
      "    - regulation of sodium ion transport\n",
      "    - arginine catabolic process\n",
      "    - arginine catabolic process\n",
      "    - nitric oxide biosynthetic process\n",
      "    - nitric oxide biosynthetic process\n",
      "    - striated muscle contraction\n",
      "    - nitric oxide mediated signal transduction\n",
      "    - myoblast fusion\n",
      "    - response to heat\n",
      "    - response to hormone\n",
      "    - negative regulation of calcium ion transport into cytosol\n",
      "    - regulation of cardiac muscle contraction by calcium ion signaling\n",
      "    - response to lipopolysaccharide\n",
      "    - multicellular organismal response to stress\n",
      "    - xenobiotic catabolic process\n",
      "    - vasodilation\n",
      "    - vasodilation\n",
      "    - negative regulation of potassium ion transport\n",
      "    - positive regulation of neuron apoptotic process\n",
      "    - cell redox homeostasis\n",
      "    - negative regulation of blood pressure\n",
      "    - positive regulation of DNA-templated transcription\n",
      "    - positive regulation of transcription by RNA polymerase II\n",
      "    - negative regulation of serotonin uptake\n",
      "    - negative regulation of calcium ion transport\n",
      "    - regulation of cardiac muscle contraction\n",
      "    - cellular response to growth factor stimulus\n",
      "    - positive regulation of the force of heart contraction\n",
      "    - positive regulation of adenylate cyclase-activating G protein-coupled receptor signaling pathway\n",
      "    - positive regulation of sodium ion transmembrane transport\n",
      "    - regulation of calcium ion transmembrane transport via high voltage-gated calcium channel\n",
      "    - positive regulation of membrane repolarization during ventricular cardiac muscle cell action potential\n",
      "  CC:\n",
      "    - photoreceptor inner segment\n",
      "    - nucleus\n",
      "    - nucleoplasm\n",
      "    - cytoplasm\n",
      "    - cytoplasm\n",
      "    - mitochondrion\n",
      "    - cytosol\n",
      "    - cytosol\n",
      "    - cytoskeleton\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - postsynaptic density\n",
      "    - sarcoplasmic reticulum\n",
      "    - protein-containing complex\n",
      "    - sarcolemma\n",
      "    - dendritic spine\n",
      "    - membrane raft\n",
      "    - synapse\n",
      "    - perinuclear region of cytoplasm\n",
      "    - cell periphery\n",
      "  MF:\n",
      "    - nitric-oxide synthase activity\n",
      "    - nitric-oxide synthase activity\n",
      "    - nitric-oxide synthase activity\n",
      "    - nitric-oxide synthase activity\n",
      "    - calcium channel regulator activity\n",
      "    - protein binding\n",
      "    - calmodulin binding\n",
      "    - FMN binding\n",
      "    - FMN binding\n",
      "    - sodium channel regulator activity\n",
      "    - heme binding\n",
      "    - tetrahydrobiopterin binding\n",
      "    - arginine binding\n",
      "    - peptidyl-cysteine S-nitrosylase activity\n",
      "    - transmembrane transporter binding\n",
      "    - transmembrane transporter binding\n",
      "    - cadmium ion binding\n",
      "    - calcium-dependent protein binding\n",
      "    - flavin adenine dinucleotide binding\n",
      "    - flavin adenine dinucleotide binding\n",
      "    - NADP binding\n",
      "    - scaffold protein binding\n",
      "UniProt: {'Swiss-Prot': 'P29475', 'TrEMBL': ['B4DG68', 'B3VK56', 'A0PJJ7']}\n",
      "\n",
      "Gene: ABCC9\n",
      "Symbol: ABCC9\n",
      "Name: ATP binding cassette subfamily C member 9\n",
      "Summary: The protein encoded by this gene is a member of the superfamily of ATP-binding cassette (ABC) transporters. ABC proteins transport various molecules across extra- and intra-cellular membranes. ABC genes are divided into seven distinct subfamilies (ABC1, MDR/TAP, MRP, ALD, OABP, GCN20, White). This protein is a member of the MRP subfamily which is involved in multi-drug resistance. This protein is thought to form ATP-sensitive potassium channels in cardiac, skeletal, and vascular and non-vascular smooth muscle. Protein structure suggests a role as the drug-binding channel-modulating subunit of the extra-pancreatic ATP-sensitive potassium channels. Mutations in this gene are associated with cardiomyopathy dilated type 1O. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Apr 2011].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - MAPK cascade\n",
      "    - action potential\n",
      "    - response to hypoxia\n",
      "    - heart morphogenesis\n",
      "    - regulation of transcription by RNA polymerase II\n",
      "    - mitochondrion organization\n",
      "    - skeletal muscle tissue development\n",
      "    - gene expression\n",
      "    - response to activity\n",
      "    - fatty acid oxidation\n",
      "    - response to ATP\n",
      "    - response to ATP\n",
      "    - cellular response to potassium ion\n",
      "    - vasodilation\n",
      "    - response to hydrogen peroxide\n",
      "    - negative regulation of apoptotic process\n",
      "    - response to estrogen\n",
      "    - cellular respiration\n",
      "    - negative regulation of blood pressure\n",
      "    - ATP metabolic process\n",
      "    - fibroblast proliferation\n",
      "    - defense response to virus\n",
      "    - transmembrane transport\n",
      "    - transmembrane transport\n",
      "    - coronary vasculature development\n",
      "    - cardiac conduction\n",
      "    - cellular response to chemical stress\n",
      "    - cellular response to calcium ion\n",
      "    - cellular response to ATP\n",
      "    - cellular response to xenobiotic stimulus\n",
      "    - potassium ion transmembrane transport\n",
      "    - potassium ion transmembrane transport\n",
      "    - oxygen metabolic process\n",
      "    - cardiac muscle cell contraction\n",
      "    - monoatomic cation transmembrane transport\n",
      "    - monoatomic anion transmembrane transport\n",
      "    - inorganic cation transmembrane transport\n",
      "    - transport across blood-brain barrier\n",
      "    - regulation of potassium ion transmembrane transport\n",
      "    - response to peptide\n",
      "    - reactive oxygen species biosynthetic process\n",
      "    - response to hydrogen sulfide\n",
      "    - potassium ion import across plasma membrane\n",
      "    - potassium ion import across plasma membrane\n",
      "  CC:\n",
      "    - mitochondrion\n",
      "    - plasma membrane\n",
      "    - inward rectifying potassium channel\n",
      "    - inward rectifying potassium channel\n",
      "    - inward rectifying potassium channel\n",
      "    - membrane\n",
      "    - sarcomere\n",
      "    - potassium ion-transporting ATPase complex\n",
      "    - sarcolemma\n",
      "  MF:\n",
      "    - potassium channel activity\n",
      "    - ATP binding\n",
      "    - sulfonylurea receptor activity\n",
      "    - ATP-activated inward rectifier potassium channel activity\n",
      "    - potassium channel regulator activity\n",
      "    - ATP hydrolysis activity\n",
      "    - ATPase-coupled monoatomic cation transmembrane transporter activity\n",
      "    - ATPase-coupled transmembrane transporter activity\n",
      "    - ATPase-coupled transmembrane transporter activity\n",
      "    - ATPase-coupled inorganic anion transmembrane transporter activity\n",
      "    - transmembrane transporter binding\n",
      "    - protein-containing complex binding\n",
      "    - potassium channel activator activity\n",
      "    - ABC-type transporter activity\n",
      "UniProt: {'Swiss-Prot': 'O60706', 'TrEMBL': ['Q8N4N7', 'G3V1N6', 'A0A804HK56', 'A0A804HHX2', 'H0YFV4', 'A0A804HKB7', 'A0A804HL22', 'A0A804HL58', 'A0A804HIU2']}\n",
      "\n",
      "Gene: FNTB\n",
      "Symbol: FNTB\n",
      "Name: farnesyltransferase, CAAX box, subunit beta\n",
      "Summary: Enables zinc ion binding activity. Contributes to protein farnesyltransferase activity. Involved in protein farnesylation. Part of microtubule associated complex and protein farnesyltransferase complex. [provided by Alliance of Genome Resources, Apr 2022]\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - lipid metabolic process\n",
      "    - protein farnesylation\n",
      "    - regulation of microtubule-based movement\n",
      "  CC:\n",
      "    - cytosol\n",
      "    - microtubule associated complex\n",
      "    - protein farnesyltransferase complex\n",
      "    - protein farnesyltransferase complex\n",
      "  MF:\n",
      "    - protein farnesyltransferase activity\n",
      "    - protein farnesyltransferase activity\n",
      "    - protein farnesyltransferase activity\n",
      "    - protein binding\n",
      "    - zinc ion binding\n",
      "    - acetyltransferase activator activity\n",
      "    - enzyme binding\n",
      "UniProt: {'Swiss-Prot': 'P49356', 'TrEMBL': 'A0A384MEJ5'}\n",
      "\n",
      "Gene: CDK1\n",
      "Symbol: CDK1\n",
      "Name: cyclin dependent kinase 1\n",
      "Summary: The protein encoded by this gene is a member of the Ser/Thr protein kinase family. This protein is a catalytic subunit of the highly conserved protein kinase complex known as M-phase promoting factor (MPF), which is essential for G2/M phase transitions of eukaryotic cell cycle. Mitotic cyclins stably associate with this protein and function as regulatory subunits. The kinase activity of this protein is controlled by cyclin accumulation and destruction through the cell cycle. The phosphorylation and dephosphorylation of this protein also play important regulatory roles in cell cycle control. Alternatively spliced transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, May 2023].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - G1/S transition of mitotic cell cycle\n",
      "    - G2/M transition of mitotic cell cycle\n",
      "    - G2/M transition of mitotic cell cycle\n",
      "    - G2/M transition of mitotic cell cycle\n",
      "    - G2/M transition of mitotic cell cycle\n",
      "    - microtubule cytoskeleton organization\n",
      "    - DNA replication\n",
      "    - DNA repair\n",
      "    - chromatin remodeling\n",
      "    - regulation of transcription by RNA polymerase II\n",
      "    - apoptotic process\n",
      "    - DNA damage response\n",
      "    - mitotic nuclear membrane disassembly\n",
      "    - mitotic G2 DNA damage checkpoint signaling\n",
      "    - centrosome cycle\n",
      "    - pronuclear fusion\n",
      "    - response to xenobiotic stimulus\n",
      "    - response to toxic substance\n",
      "    - positive regulation of gene expression\n",
      "    - negative regulation of gene expression\n",
      "    - positive regulation of G2/M transition of mitotic cell cycle\n",
      "    - regulation of Schwann cell differentiation\n",
      "    - response to amine\n",
      "    - response to activity\n",
      "    - cell migration\n",
      "    - protein deubiquitination\n",
      "    - peptidyl-serine phosphorylation\n",
      "    - peptidyl-threonine phosphorylation\n",
      "    - peptidyl-threonine phosphorylation\n",
      "    - chromosome condensation\n",
      "    - epithelial cell differentiation\n",
      "    - protein localization to kinetochore\n",
      "    - protein localization to kinetochore\n",
      "    - positive regulation of protein import into nucleus\n",
      "    - regulation of circadian rhythm\n",
      "    - negative regulation of apoptotic process\n",
      "    - response to ethanol\n",
      "    - positive regulation of DNA replication\n",
      "    - regulation of embryonic development\n",
      "    - response to cadmium ion\n",
      "    - response to copper ion\n",
      "    - symbiont entry into host cell\n",
      "    - fibroblast proliferation\n",
      "    - rhythmic process\n",
      "    - response to axon injury\n",
      "    - cell division\n",
      "    - ventricular cardiac muscle cell development\n",
      "    - positive regulation of cardiac muscle cell proliferation\n",
      "    - positive regulation of mitotic sister chromatid segregation\n",
      "    - protein-containing complex assembly\n",
      "    - cellular response to hydrogen peroxide\n",
      "    - ERK1 and ERK2 cascade\n",
      "    - Golgi disassembly\n",
      "    - positive regulation of protein localization to nucleus\n",
      "    - regulation of attachment of mitotic spindle microtubules to kinetochore\n",
      "    - microtubule cytoskeleton organization involved in mitosis\n",
      "    - positive regulation of mitochondrial ATP synthesis coupled electron transport\n",
      "  CC:\n",
      "    - cyclin-dependent protein kinase holoenzyme complex\n",
      "    - chromosome, telomeric region\n",
      "    - nucleus\n",
      "    - nucleus\n",
      "    - nucleoplasm\n",
      "    - cytoplasm\n",
      "    - mitochondrion\n",
      "    - mitochondrial matrix\n",
      "    - endoplasmic reticulum membrane\n",
      "    - centrosome\n",
      "    - cytosol\n",
      "    - spindle microtubule\n",
      "    - membrane\n",
      "    - midbody\n",
      "    - extracellular exosome\n",
      "    - mitotic spindle\n",
      "    - cyclin A1-CDK1 complex\n",
      "    - cyclin A2-CDK1 complex\n",
      "    - cyclin B1-CDK1 complex\n",
      "    - cyclin B1-CDK1 complex\n",
      "  MF:\n",
      "    - virus receptor activity\n",
      "    - chromatin binding\n",
      "    - protein kinase activity\n",
      "    - protein serine/threonine kinase activity\n",
      "    - protein serine/threonine kinase activity\n",
      "    - cyclin-dependent protein serine/threonine kinase activity\n",
      "    - cyclin-dependent protein serine/threonine kinase activity\n",
      "    - cyclin-dependent protein serine/threonine kinase activity\n",
      "    - cyclin-dependent protein serine/threonine kinase activity\n",
      "    - protein binding\n",
      "    - ATP binding\n",
      "    - RNA polymerase II CTD heptapeptide repeat kinase activity\n",
      "    - kinase activity\n",
      "    - cyclin binding\n",
      "    - cyclin binding\n",
      "    - Hsp70 protein binding\n",
      "    - histone kinase activity\n",
      "    - cyclin-dependent protein kinase activity\n",
      "    - protein serine kinase activity\n",
      "    - RNA polymerase II CTD heptapeptide repeat Y1 kinase activity\n",
      "    - RNA polymerase II CTD heptapeptide repeat S2 kinase activity\n",
      "    - RNA polymerase II CTD heptapeptide repeat T4 kinase activity\n",
      "    - RNA polymerase II CTD heptapeptide repeat S5 kinase activity\n",
      "    - RNA polymerase II CTD heptapeptide repeat S7 kinase activity\n",
      "UniProt: {'Swiss-Prot': 'P06493', 'TrEMBL': ['I6L9I5', 'B7Z3D6', 'A0A024QZP7', 'E5RIU6']}\n",
      "\n",
      "Gene: CCNB1\n",
      "Symbol: CCNB1\n",
      "Name: cyclin B1\n",
      "Summary: The protein encoded by this gene is a regulatory protein involved in mitosis. The gene product complexes with p34(cdc2) to form the maturation-promoting factor (MPF). The encoded protein is necessary for proper control of the G2/M transition phase of the cell cycle. [provided by RefSeq, Aug 2017].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - G1/S transition of mitotic cell cycle\n",
      "    - G2/M transition of mitotic cell cycle\n",
      "    - in utero embryonic development\n",
      "    - mitotic spindle organization\n",
      "    - mitotic metaphase chromosome alignment\n",
      "    - mitotic metaphase chromosome alignment\n",
      "    - positive regulation of G2/M transition of mitotic cell cycle\n",
      "    - positive regulation of mitotic cell cycle\n",
      "    - positive regulation of fibroblast proliferation\n",
      "    - cell division\n",
      "    - positive regulation of attachment of spindle microtubules to kinetochore\n",
      "    - regulation of mitotic cell cycle spindle assembly checkpoint\n",
      "    - positive regulation of mitochondrial ATP synthesis coupled electron transport\n",
      "  CC:\n",
      "    - spindle pole\n",
      "    - outer kinetochore\n",
      "    - nucleus\n",
      "    - nucleus\n",
      "    - nucleoplasm\n",
      "    - cytoplasm\n",
      "    - cytoplasm\n",
      "    - mitochondrial matrix\n",
      "    - mitochondrial matrix\n",
      "    - centrosome\n",
      "    - microtubule organizing center\n",
      "    - cytosol\n",
      "    - cytosol\n",
      "    - membrane\n",
      "    - cyclin B1-CDK1 complex\n",
      "    - cyclin B1-CDK1 complex\n",
      "    - cyclin B1-CDK1 complex\n",
      "  MF:\n",
      "    - patched binding\n",
      "    - protein binding\n",
      "    - cyclin-dependent protein serine/threonine kinase regulator activity\n",
      "    - protein kinase binding\n",
      "    - ubiquitin-like protein ligase binding\n",
      "    - cyclin-dependent protein serine/threonine kinase activator activity\n",
      "UniProt: {'Swiss-Prot': 'P14635', 'TrEMBL': ['H0YA62', 'E9PC90', 'D6RHI0', 'H0Y9U8']}\n",
      "\n",
      "Gene: PTEN\n",
      "Symbol: PTEN\n",
      "Name: phosphatase and tensin homolog\n",
      "Summary: This gene was identified as a tumor suppressor that is mutated in a large number of cancers at high frequency. The protein encoded by this gene is a phosphatidylinositol-3,4,5-trisphosphate 3-phosphatase. It contains a tensin like domain as well as a catalytic domain similar to that of the dual specificity protein tyrosine phosphatases. Unlike most of the protein tyrosine phosphatases, this protein preferentially dephosphorylates phosphoinositide substrates. It negatively regulates intracellular levels of phosphatidylinositol-3,4,5-trisphosphate in cells and functions as a tumor suppressor by negatively regulating AKT/PKB signaling pathway. The use of a non-canonical (CUG) upstream initiation site produces a longer isoform that initiates translation with a leucine, and is thought to be preferentially associated with the mitochondrial inner membrane. This longer isoform may help regulate energy metabolism in the mitochondria. A pseudogene of this gene is found on chromosome 9. Alternative splicing and the use of multiple translation start codons results in multiple transcript variants encoding different isoforms. [provided by RefSeq, Feb 2015].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - chromatin remodeling\n",
      "    - protein dephosphorylation\n",
      "    - protein dephosphorylation\n",
      "    - phosphatidylinositol biosynthetic process\n",
      "    - apoptotic process\n",
      "    - spindle assembly involved in female meiosis\n",
      "    - neuron-neuron synaptic transmission\n",
      "    - synapse assembly\n",
      "    - central nervous system development\n",
      "    - heart development\n",
      "    - learning or memory\n",
      "    - locomotory behavior\n",
      "    - positive regulation of cell population proliferation\n",
      "    - negative regulation of cell population proliferation\n",
      "    - negative regulation of cell population proliferation\n",
      "    - negative regulation of epithelial to mesenchymal transition\n",
      "    - regulation of neuron projection development\n",
      "    - negative regulation of neuron projection development\n",
      "    - cell migration\n",
      "    - dentate gyrus development\n",
      "    - central nervous system neuron axonogenesis\n",
      "    - negative regulation of cell migration\n",
      "    - adult behavior\n",
      "    - regulation of protein stability\n",
      "    - central nervous system myelin maintenance\n",
      "    - negative regulation of peptidyl-serine phosphorylation\n",
      "    - multicellular organismal response to stress\n",
      "    - social behavior\n",
      "    - maternal behavior\n",
      "    - phosphatidylinositol 3-kinase/protein kinase B signal transduction\n",
      "    - locomotor rhythm\n",
      "    - negative regulation of osteoblast differentiation\n",
      "    - negative regulation of cyclin-dependent protein serine/threonine kinase activity\n",
      "    - negative regulation of cell size\n",
      "    - positive regulation of transcription by RNA polymerase II\n",
      "    - negative regulation of organ growth\n",
      "    - phosphatidylinositol dephosphorylation\n",
      "    - phosphatidylinositol dephosphorylation\n",
      "    - phosphatidylinositol dephosphorylation\n",
      "    - forebrain morphogenesis\n",
      "    - cell motility\n",
      "    - negative regulation of axonogenesis\n",
      "    - protein stabilization\n",
      "    - negative regulation of keratinocyte migration\n",
      "    - negative regulation of focal adhesion assembly\n",
      "    - regulation of phosphatidylinositol 3-kinase/protein kinase B signal transduction\n",
      "    - negative regulation of phosphatidylinositol 3-kinase/protein kinase B signal transduction\n",
      "    - negative regulation of phosphatidylinositol 3-kinase/protein kinase B signal transduction\n",
      "    - negative regulation of phosphatidylinositol 3-kinase/protein kinase B signal transduction\n",
      "    - negative regulation of phosphatidylinositol 3-kinase/protein kinase B signal transduction\n",
      "    - negative regulation of phosphatidylinositol 3-kinase/protein kinase B signal transduction\n",
      "    - negative regulation of phosphatidylinositol 3-kinase/protein kinase B signal transduction\n",
      "    - rhythmic synaptic transmission\n",
      "    - synapse maturation\n",
      "    - prepulse inhibition\n",
      "    - dendritic spine morphogenesis\n",
      "    - cellular response to electrical stimulus\n",
      "    - negative regulation of excitatory postsynaptic potential\n",
      "    - presynaptic membrane assembly\n",
      "    - postsynaptic density assembly\n",
      "    - positive regulation of intracellular signal transduction\n",
      "    - negative regulation of cell cycle G1/S phase transition\n",
      "    - negative regulation of wound healing, spreading of epidermal cells\n",
      "    - positive regulation of ubiquitin protein ligase activity\n",
      "    - negative regulation of vascular associated smooth muscle cell proliferation\n",
      "    - positive regulation of ubiquitin-dependent protein catabolic process\n",
      "    - negative regulation of G1/S transition of mitotic cell cycle\n",
      "    - positive regulation of excitatory postsynaptic potential\n",
      "    - negative regulation of cellular senescence\n",
      "    - negative regulation of synaptic vesicle clustering\n",
      "  CC:\n",
      "    - extracellular region\n",
      "    - nucleus\n",
      "    - nucleus\n",
      "    - nucleoplasm\n",
      "    - nucleoplasm\n",
      "    - cytoplasm\n",
      "    - cytoplasm\n",
      "    - cytosol\n",
      "    - cytosol\n",
      "    - cytosol\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - cytoplasmic side of plasma membrane\n",
      "    - postsynaptic density\n",
      "    - apical plasma membrane\n",
      "    - PML body\n",
      "    - myelin sheath adaxonal region\n",
      "    - cell projection\n",
      "    - cell projection\n",
      "    - neuron projection\n",
      "    - dendritic spine\n",
      "    - Schmidt-Lanterman incisure\n",
      "  MF:\n",
      "    - phosphatidylinositol-3-phosphate phosphatase activity\n",
      "    - phosphoprotein phosphatase activity\n",
      "    - protein serine/threonine phosphatase activity\n",
      "    - protein tyrosine phosphatase activity\n",
      "    - protein tyrosine phosphatase activity\n",
      "    - non-membrane spanning protein tyrosine phosphatase activity\n",
      "    - protein binding\n",
      "    - beta-catenin binding\n",
      "    - lipid binding\n",
      "    - anaphase-promoting complex binding\n",
      "    - phosphatidylinositol-3,4,5-trisphosphate 3-phosphatase activity\n",
      "    - phosphatidylinositol-3,4,5-trisphosphate 3-phosphatase activity\n",
      "    - phosphatidylinositol-3,4,5-trisphosphate 3-phosphatase activity\n",
      "    - myosin phosphatase activity\n",
      "    - enzyme binding\n",
      "    - PDZ domain binding\n",
      "    - inositol-1,3,4,5,6-pentakisphosphate 3-phosphatase activity\n",
      "    - protein tyrosine phosphatase activity, metal-dependent\n",
      "    - calmodulin-dependent protein phosphatase activity\n",
      "    - identical protein binding\n",
      "    - inositol-1,3,4,5-tetrakisphosphate 3-phosphatase activity\n",
      "    - phosphatidylinositol-3,4-bisphosphate 3-phosphatase activity\n",
      "    - phosphatidylinositol-3,4-bisphosphate 3-phosphatase activity\n",
      "    - phosphatidylinositol phosphate phosphatase activity\n",
      "    - molecular function inhibitor activity\n",
      "    - histone H2AXS140 phosphatase activity\n",
      "    - histone H2AXY142 phosphatase activity\n",
      "    - RNA polymerase II CTD heptapeptide repeat Y1 phosphatase activity\n",
      "    - RNA polymerase II CTD heptapeptide repeat T4 phosphatase activity\n",
      "    - RNA polymerase II CTD heptapeptide repeat S2 phosphatase activity\n",
      "    - RNA polymerase II CTD heptapeptide repeat S5 phosphatase activity\n",
      "    - RNA polymerase II CTD heptapeptide repeat S7 phosphatase activity\n",
      "    - ubiquitin-specific protease binding\n",
      "    - MAP kinase serine/threonine phosphatase activity\n",
      "    - ubiquitin ligase activator activity\n",
      "UniProt: {'Swiss-Prot': 'P60484', 'TrEMBL': ['A0A8V8TQX2', 'F6KD01', 'A0A087X033', 'A0A8V8TP77', 'A0A8V8TPK6', 'A0A8I5KR36', 'A0A8I5KVQ8', 'A0AAQ5BGY7', 'A0AA34QVZ8', 'A0A9L9PYJ2', 'A0AAQ5BH13', 'A0A8I5KSF9']}\n",
      "\n",
      "Gene: TP53\n",
      "Symbol: TP53\n",
      "Name: tumor protein p53\n",
      "Summary: This gene encodes a tumor suppressor protein containing transcriptional activation, DNA binding, and oligomerization domains. The encoded protein responds to diverse cellular stresses to regulate expression of target genes, thereby inducing cell cycle arrest, apoptosis, senescence, DNA repair, or changes in metabolism. Mutations in this gene are associated with a variety of human cancers, including hereditary cancers such as Li-Fraumeni syndrome. Alternative splicing of this gene and the use of alternate promoters result in multiple transcript variants and isoforms. Additional isoforms have also been shown to result from the use of alternate translation initiation codons from identical transcript variants (PMIDs: 12032546, 20937277). [provided by RefSeq, Dec 2016].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - negative regulation of transcription by RNA polymerase II\n",
      "    - negative regulation of transcription by RNA polymerase II\n",
      "    - mitophagy\n",
      "    - in utero embryonic development\n",
      "    - somitogenesis\n",
      "    - release of cytochrome c from mitochondria\n",
      "    - hematopoietic progenitor cell differentiation\n",
      "    - T cell proliferation involved in immune response\n",
      "    - B cell lineage commitment\n",
      "    - T cell lineage commitment\n",
      "    - response to ischemia\n",
      "    - nucleotide-excision repair\n",
      "    - double-strand break repair\n",
      "    - regulation of DNA-templated transcription\n",
      "    - regulation of DNA-templated transcription\n",
      "    - regulation of transcription by RNA polymerase II\n",
      "    - regulation of transcription by RNA polymerase II\n",
      "    - protein import into nucleus\n",
      "    - autophagy\n",
      "    - DNA damage response\n",
      "    - DNA damage response\n",
      "    - DNA damage response\n",
      "    - ER overload response\n",
      "    - transforming growth factor beta receptor signaling pathway\n",
      "    - Ras protein signal transduction\n",
      "    - gastrulation\n",
      "    - neuroblast proliferation\n",
      "    - negative regulation of neuroblast proliferation\n",
      "    - protein localization\n",
      "    - negative regulation of DNA replication\n",
      "    - negative regulation of cell population proliferation\n",
      "    - negative regulation of cell population proliferation\n",
      "    - negative regulation of cell population proliferation\n",
      "    - negative regulation of cell population proliferation\n",
      "    - determination of adult lifespan\n",
      "    - mRNA transcription\n",
      "    - rRNA transcription\n",
      "    - response to salt stress\n",
      "    - response to X-ray\n",
      "    - response to gamma radiation\n",
      "    - positive regulation of gene expression\n",
      "    - positive regulation of gene expression\n",
      "    - positive regulation of gene expression\n",
      "    - cardiac muscle cell apoptotic process\n",
      "    - positive regulation of cardiac muscle cell apoptotic process\n",
      "    - glial cell proliferation\n",
      "    - viral process\n",
      "    - glucose catabolic process to lactate via pyruvate\n",
      "    - cerebellum development\n",
      "    - negative regulation of cell growth\n",
      "    - DNA damage response, signal transduction by p53 class mediator\n",
      "    - DNA damage response, signal transduction by p53 class mediator\n",
      "    - negative regulation of transforming growth factor beta receptor signaling pathway\n",
      "    - mitotic G1 DNA damage checkpoint signaling\n",
      "    - negative regulation of telomere maintenance via telomerase\n",
      "    - T cell differentiation in thymus\n",
      "    - tumor necrosis factor-mediated signaling pathway\n",
      "    - regulation of tissue remodeling\n",
      "    - cellular response to UV\n",
      "    - multicellular organism growth\n",
      "    - positive regulation of mitochondrial membrane permeability\n",
      "    - cellular response to glucose starvation\n",
      "    - cellular response to glucose starvation\n",
      "    - intrinsic apoptotic signaling pathway in response to DNA damage by p53 class mediator\n",
      "    - intrinsic apoptotic signaling pathway in response to DNA damage by p53 class mediator\n",
      "    - intrinsic apoptotic signaling pathway in response to DNA damage by p53 class mediator\n",
      "    - regulation of apoptotic process\n",
      "    - positive regulation of apoptotic process\n",
      "    - negative regulation of apoptotic process\n",
      "    - negative regulation of apoptotic process\n",
      "    - negative regulation of apoptotic process\n",
      "    - entrainment of circadian clock by photoperiod\n",
      "    - mitochondrial DNA repair\n",
      "    - regulation of DNA damage response, signal transduction by p53 class mediator\n",
      "    - positive regulation of neuron apoptotic process\n",
      "    - transcription initiation-coupled chromatin remodeling\n",
      "    - negative regulation of proteolysis\n",
      "    - negative regulation of DNA-templated transcription\n",
      "    - negative regulation of DNA-templated transcription\n",
      "    - negative regulation of DNA-templated transcription\n",
      "    - positive regulation of DNA-templated transcription\n",
      "    - positive regulation of DNA-templated transcription\n",
      "    - positive regulation of RNA polymerase II transcription preinitiation complex assembly\n",
      "    - positive regulation of transcription by RNA polymerase II\n",
      "    - positive regulation of transcription by RNA polymerase II\n",
      "    - positive regulation of transcription by RNA polymerase II\n",
      "    - positive regulation of transcription by RNA polymerase II\n",
      "    - positive regulation of transcription by RNA polymerase II\n",
      "    - response to antibiotic\n",
      "    - fibroblast proliferation\n",
      "    - negative regulation of fibroblast proliferation\n",
      "    - circadian behavior\n",
      "    - bone marrow development\n",
      "    - embryonic organ development\n",
      "    - protein stabilization\n",
      "    - negative regulation of helicase activity\n",
      "    - protein tetramerization\n",
      "    - chromosome organization\n",
      "    - neuron apoptotic process\n",
      "    - regulation of cell cycle\n",
      "    - regulation of cell cycle\n",
      "    - regulation of cell cycle\n",
      "    - regulation of cell cycle\n",
      "    - hematopoietic stem cell differentiation\n",
      "    - negative regulation of glial cell proliferation\n",
      "    - type II interferon-mediated signaling pathway\n",
      "    - cardiac septum morphogenesis\n",
      "    - positive regulation of programmed necrotic cell death\n",
      "    - protein-containing complex assembly\n",
      "    - intrinsic apoptotic signaling pathway in response to endoplasmic reticulum stress\n",
      "    - thymocyte apoptotic process\n",
      "    - positive regulation of thymocyte apoptotic process\n",
      "    - necroptotic process\n",
      "    - cellular response to hypoxia\n",
      "    - cellular response to xenobiotic stimulus\n",
      "    - cellular response to ionizing radiation\n",
      "    - cellular response to gamma radiation\n",
      "    - cellular response to UV-C\n",
      "    - stem cell proliferation\n",
      "    - signal transduction by p53 class mediator\n",
      "    - intrinsic apoptotic signaling pathway by p53 class mediator\n",
      "    - reactive oxygen species metabolic process\n",
      "    - cellular response to actinomycin D\n",
      "    - positive regulation of release of cytochrome c from mitochondria\n",
      "    - cellular senescence\n",
      "    - replicative senescence\n",
      "    - oxidative stress-induced premature senescence\n",
      "    - intrinsic apoptotic signaling pathway\n",
      "    - oligodendrocyte apoptotic process\n",
      "    - positive regulation of execution phase of apoptosis\n",
      "    - negative regulation of mitophagy\n",
      "    - regulation of mitochondrial membrane permeability involved in apoptotic process\n",
      "    - regulation of intrinsic apoptotic signaling pathway by p53 class mediator\n",
      "    - regulation of cell cycle G2/M phase transition\n",
      "    - positive regulation of miRNA transcription\n",
      "    - positive regulation of miRNA transcription\n",
      "    - positive regulation of miRNA transcription\n",
      "    - positive regulation of miRNA transcription\n",
      "    - negative regulation of G1 to G0 transition\n",
      "    - negative regulation of G1 to G0 transition\n",
      "    - negative regulation of miRNA processing\n",
      "    - negative regulation of glucose catabolic process to lactate via pyruvate\n",
      "    - negative regulation of pentose-phosphate shunt\n",
      "    - intrinsic apoptotic signaling pathway in response to hypoxia\n",
      "    - regulation of fibroblast apoptotic process\n",
      "    - negative regulation of reactive oxygen species metabolic process\n",
      "    - positive regulation of reactive oxygen species metabolic process\n",
      "    - negative regulation of stem cell proliferation\n",
      "    - positive regulation of cellular senescence\n",
      "    - positive regulation of cellular senescence\n",
      "    - positive regulation of intrinsic apoptotic signaling pathway\n",
      "    - positive regulation of intrinsic apoptotic signaling pathway\n",
      "  CC:\n",
      "    - chromatin\n",
      "    - chromatin\n",
      "    - chromatin\n",
      "    - nucleus\n",
      "    - nucleus\n",
      "    - nucleus\n",
      "    - nucleoplasm\n",
      "    - nucleoplasm\n",
      "    - replication fork\n",
      "    - transcription regulator complex\n",
      "    - nucleolus\n",
      "    - cytoplasm\n",
      "    - cytoplasm\n",
      "    - mitochondrion\n",
      "    - mitochondrial matrix\n",
      "    - endoplasmic reticulum\n",
      "    - centrosome\n",
      "    - cytosol\n",
      "    - cytosol\n",
      "    - nuclear matrix\n",
      "    - nuclear body\n",
      "    - PML body\n",
      "    - transcription repressor complex\n",
      "    - protein-containing complex\n",
      "    - protein-containing complex\n",
      "    - site of double-strand break\n",
      "    - germ cell nucleus\n",
      "  MF:\n",
      "    - transcription cis-regulatory region binding\n",
      "    - RNA polymerase II cis-regulatory region sequence-specific DNA binding\n",
      "    - RNA polymerase II cis-regulatory region sequence-specific DNA binding\n",
      "    - RNA polymerase II cis-regulatory region sequence-specific DNA binding\n",
      "    - DNA-binding transcription factor activity, RNA polymerase II-specific\n",
      "    - DNA-binding transcription factor activity, RNA polymerase II-specific\n",
      "    - DNA-binding transcription factor activity, RNA polymerase II-specific\n",
      "    - cis-regulatory region sequence-specific DNA binding\n",
      "    - core promoter sequence-specific DNA binding\n",
      "    - TFIID-class transcription factor complex binding\n",
      "    - DNA-binding transcription repressor activity, RNA polymerase II-specific\n",
      "    - DNA-binding transcription activator activity, RNA polymerase II-specific\n",
      "    - protease binding\n",
      "    - p53 binding\n",
      "    - DNA binding\n",
      "    - DNA binding\n",
      "    - chromatin binding\n",
      "    - DNA-binding transcription factor activity\n",
      "    - DNA-binding transcription factor activity\n",
      "    - mRNA 3'-UTR binding\n",
      "    - copper ion binding\n",
      "    - protein binding\n",
      "    - zinc ion binding\n",
      "    - enzyme binding\n",
      "    - receptor tyrosine kinase binding\n",
      "    - ubiquitin protein ligase binding\n",
      "    - ubiquitin protein ligase binding\n",
      "    - histone deacetylase regulator activity\n",
      "    - ATP-dependent DNA/DNA annealing activity\n",
      "    - identical protein binding\n",
      "    - histone deacetylase binding\n",
      "    - protein heterodimerization activity\n",
      "    - protein-folding chaperone binding\n",
      "    - protein phosphatase 2A binding\n",
      "    - RNA polymerase II-specific DNA-binding transcription factor binding\n",
      "    - 14-3-3 protein binding\n",
      "    - MDM2/MDM4 family protein binding\n",
      "    - disordered domain specific binding\n",
      "    - general transcription initiation factor binding\n",
      "    - molecular function activator activity\n",
      "    - molecular function activator activity\n",
      "    - molecular condensate scaffold activity\n",
      "    - promoter-specific chromatin binding\n",
      "UniProt: {'Swiss-Prot': 'P04637', 'TrEMBL': ['Q53GA5', 'K7PPA8', 'A0A087WT22', 'A0A087X1Q1', 'A0A087WXZ1', 'H2EHT1', 'A0A386NBZ1', 'A0AAQ5BHX5', 'A0AAQ5BHY1', 'S4R334', 'E7EQX7', 'I3L0W9', 'J3KP33', 'E9PFT5', 'A0A0U1RQC9', 'A0AAQ5BHZ9']}\n",
      "\n",
      "Gene: TSC1\n",
      "Symbol: TSC1\n",
      "Name: TSC complex subunit 1\n",
      "Summary: This gene is a tumor suppressor gene that encodes the growth inhibitory protein hamartin. The encoded protein interacts with and stabilizes the GTPase activating protein tuberin. This hamartin-tuberin complex negatively regulates mammalian target of rapamycin complex 1 (mTORC1) signaling which is a major regulator of anabolic cell growth. This protein also functions as a co-chaperone for Hsp90 that inhibits its ATPase activity. This protein functions as a facilitator of Hsp90-mediated folding of kinase and non-kinase clients, including TSC2 and thereby preventing their ubiquitination and proteasomal degradation. Mutations in this gene have been associated with tuberous sclerosis and lymphangioleiomyomatosis. [provided by RefSeq, May 2022].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - kidney development\n",
      "    - neural tube closure\n",
      "    - regulation of cell-matrix adhesion\n",
      "    - adaptive immune response\n",
      "    - protein folding\n",
      "    - potassium ion transport\n",
      "    - cell-matrix adhesion\n",
      "    - cell population proliferation\n",
      "    - negative regulation of cell population proliferation\n",
      "    - negative regulation of cell population proliferation\n",
      "    - associative learning\n",
      "    - adult locomotory behavior\n",
      "    - cellular response to starvation\n",
      "    - negative regulation of macroautophagy\n",
      "    - hippocampus development\n",
      "    - cerebral cortex development\n",
      "    - cell projection organization\n",
      "    - negative regulation of TOR signaling\n",
      "    - negative regulation of TOR signaling\n",
      "    - negative regulation of TOR signaling\n",
      "    - negative regulation of TOR signaling\n",
      "    - negative regulation of ATP-dependent activity\n",
      "    - response to insulin\n",
      "    - cellular response to decreased oxygen levels\n",
      "    - myelination\n",
      "    - memory T cell differentiation\n",
      "    - negative regulation of cell size\n",
      "    - D-glucose import\n",
      "    - synapse organization\n",
      "    - protein stabilization\n",
      "    - protein stabilization\n",
      "    - regulation of stress fiber assembly\n",
      "    - regulation of cell cycle\n",
      "    - positive regulation of focal adhesion assembly\n",
      "    - cardiac muscle cell differentiation\n",
      "    - activation of GTPase activity\n",
      "    - negative regulation of TORC1 signaling\n",
      "  CC:\n",
      "    - nucleus\n",
      "    - cytoplasm\n",
      "    - lysosomal membrane\n",
      "    - lipid droplet\n",
      "    - cytosol\n",
      "    - cytosol\n",
      "    - actin filament\n",
      "    - plasma membrane\n",
      "    - cell cortex\n",
      "    - postsynaptic density\n",
      "    - membrane\n",
      "    - lamellipodium\n",
      "    - protein-containing complex\n",
      "    - TSC1-TSC2 complex\n",
      "    - TSC1-TSC2 complex\n",
      "    - TSC1-TSC2 complex\n",
      "    - TSC1-TSC2 complex\n",
      "    - ciliary basal body\n",
      "    - perinuclear region of cytoplasm\n",
      "    - protein folding chaperone complex\n",
      "  MF:\n",
      "    - protein binding\n",
      "    - Hsp70 protein binding\n",
      "    - ATPase inhibitor activity\n",
      "    - protein folding chaperone\n",
      "    - protein-folding chaperone binding\n",
      "    - Hsp90 protein binding\n",
      "UniProt: {'Swiss-Prot': 'Q92574', 'TrEMBL': ['A0A2R8Y5S3', 'Q86WV8', 'X5D9D2', 'Q32NF0', 'A0AAQ5BI64', 'A0A2R8Y7E9', 'A0AAQ5BI63', 'A0A2R8Y419', 'Q59IT9', 'A0A2R8Y6W1', 'A0A2R8Y6T5', 'A0A2R8YEH9', 'A0A2R8YFV7', 'A0A2R8Y6S1', 'A0A2R8Y5N2', 'A0A2R8Y6N1', 'A0AAQ5BHZ5', 'A0A2R8Y5M3', 'A0AAQ5BI42', 'A0A2R8YGX7', 'A0A2R8YD74', 'A0AAQ5BI43', 'A0A2R8Y5J1', 'A0A2R8Y5Q4', 'A0AAQ5BI60', 'A0A2R8Y5F8', 'A0AAQ5BHZ3', 'A0A2R8Y5U8']}\n",
      "\n",
      "Gene: AIFM2\n",
      "Symbol: AIFM2\n",
      "Name: AIF family member 2\n",
      "Summary: This gene encodes a flavoprotein oxidoreductase that binds single stranded DNA and is thought to contribute to apoptosis in the presence of bacterial and viral DNA. The expression of this gene is also found to be induced by tumor suppressor protein p53 in colon cancer cells. [provided by RefSeq, Nov 2010].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - ubiquinone metabolic process\n",
      "    - apoptotic mitochondrial changes\n",
      "    - apoptotic mitochondrial changes\n",
      "    - respiratory electron transport chain\n",
      "    - vitamin K metabolic process\n",
      "    - positive regulation of apoptotic process\n",
      "    - positive regulation of apoptotic process\n",
      "    - negative regulation of ferroptosis\n",
      "    - negative regulation of ferroptosis\n",
      "    - regulation of cellular response to oxidative stress\n",
      "    - cellular detoxification\n",
      "  CC:\n",
      "    - extracellular space\n",
      "    - nucleus\n",
      "    - cytoplasm\n",
      "    - cytoplasm\n",
      "    - mitochondrion\n",
      "    - mitochondrion\n",
      "    - mitochondrial outer membrane\n",
      "    - mitochondrial outer membrane\n",
      "    - lipid droplet\n",
      "    - cytosol\n",
      "    - plasma membrane\n",
      "  MF:\n",
      "    - DNA binding\n",
      "    - electron-transferring-flavoprotein dehydrogenase activity\n",
      "    - electron-transferring-flavoprotein dehydrogenase activity\n",
      "    - protein binding\n",
      "    - oxidoreductase activity, acting on NAD(P)H, quinone or similar compound as acceptor\n",
      "    - flavin adenine dinucleotide binding\n",
      "    - flavin adenine dinucleotide binding\n",
      "UniProt: {'Swiss-Prot': 'Q9BRQ8'}\n",
      "\n",
      "Gene: AURKA\n",
      "Symbol: AURKA\n",
      "Name: aurora kinase A\n",
      "Summary: The protein encoded by this gene is a cell cycle-regulated kinase that appears to be involved in microtubule formation and/or stabilization at the spindle pole during chromosome segregation. The encoded protein is found at the centrosome in interphase cells and at the spindle poles in mitosis. This gene may play a role in tumor development and progression. A processed pseudogene of this gene has been found on chromosome 1, and an unprocessed pseudogene has been found on chromosome 10. Multiple transcript variants encoding the same protein have been found for this gene. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - G2/M transition of mitotic cell cycle\n",
      "    - mitotic cell cycle\n",
      "    - chromatin remodeling\n",
      "    - protein phosphorylation\n",
      "    - protein phosphorylation\n",
      "    - apoptotic process\n",
      "    - spindle organization\n",
      "    - mitotic spindle organization\n",
      "    - spindle assembly involved in female meiosis I\n",
      "    - mitotic centrosome separation\n",
      "    - response to wounding\n",
      "    - anterior/posterior axis specification\n",
      "    - regulation of G2/M transition of mitotic cell cycle\n",
      "    - negative regulation of gene expression\n",
      "    - peptidyl-serine phosphorylation\n",
      "    - regulation of protein stability\n",
      "    - negative regulation of protein binding\n",
      "    - positive regulation of proteasomal ubiquitin-dependent protein catabolic process\n",
      "    - regulation of cytokinesis\n",
      "    - negative regulation of apoptotic process\n",
      "    - proteasome-mediated ubiquitin-dependent protein catabolic process\n",
      "    - positive regulation of mitotic nuclear division\n",
      "    - positive regulation of mitotic cell cycle\n",
      "    - regulation of centrosome cycle\n",
      "    - protein autophosphorylation\n",
      "    - cell division\n",
      "    - centrosome localization\n",
      "    - cilium disassembly\n",
      "    - protein localization to centrosome\n",
      "    - positive regulation of mitochondrial fission\n",
      "    - liver regeneration\n",
      "    - positive regulation of oocyte maturation\n",
      "    - regulation of signal transduction by p53 class mediator\n",
      "    - neuron projection extension\n",
      "  CC:\n",
      "    - kinetochore\n",
      "    - spindle pole\n",
      "    - nucleus\n",
      "    - nucleus\n",
      "    - nucleoplasm\n",
      "    - nucleoplasm\n",
      "    - centrosome\n",
      "    - centrosome\n",
      "    - centrosome\n",
      "    - centriole\n",
      "    - spindle\n",
      "    - cytosol\n",
      "    - cytosol\n",
      "    - spindle microtubule\n",
      "    - postsynaptic density\n",
      "    - microtubule cytoskeleton\n",
      "    - basolateral plasma membrane\n",
      "    - midbody\n",
      "    - spindle pole centrosome\n",
      "    - chromosome passenger complex\n",
      "    - ciliary basal body\n",
      "    - germinal vesicle\n",
      "    - axon hillock\n",
      "    - pronucleus\n",
      "    - perinuclear region of cytoplasm\n",
      "    - spindle midzone\n",
      "    - mitotic spindle\n",
      "    - meiotic spindle\n",
      "    - mitotic spindle pole\n",
      "    - glutamatergic synapse\n",
      "  MF:\n",
      "    - protein kinase activity\n",
      "    - protein serine/threonine kinase activity\n",
      "    - protein serine/threonine kinase activity\n",
      "    - protein serine/threonine kinase activity\n",
      "    - 3-phosphoinositide-dependent protein kinase activity\n",
      "    - DNA-dependent protein kinase activity\n",
      "    - AMP-activated protein kinase activity\n",
      "    - eukaryotic translation initiation factor 2alpha kinase activity\n",
      "    - ribosomal protein S6 kinase activity\n",
      "    - protein serine/threonine/tyrosine kinase activity\n",
      "    - protein binding\n",
      "    - ATP binding\n",
      "    - protein kinase binding\n",
      "    - ubiquitin protein ligase binding\n",
      "    - histone H3S10 kinase activity\n",
      "    - histone H3T11 kinase activity\n",
      "    - histone H3T6 kinase activity\n",
      "    - histone H2AXS139 kinase activity\n",
      "    - histone H3S28 kinase activity\n",
      "    - histone H4S1 kinase activity\n",
      "    - histone H2AS1 kinase activity\n",
      "    - histone H2BS14 kinase activity\n",
      "    - protein heterodimerization activity\n",
      "    - histone H3T3 kinase activity\n",
      "    - histone H2AS121 kinase activity\n",
      "    - Rho-dependent protein serine/threonine kinase activity\n",
      "    - protein serine kinase activity\n",
      "    - molecular function activator activity\n",
      "    - histone H2BS36 kinase activity\n",
      "    - histone H3S57 kinase activity\n",
      "    - histone H3T45 kinase activity\n",
      "    - histone H2AT120 kinase activity\n",
      "UniProt: {'Swiss-Prot': 'O14965', 'TrEMBL': ['A3KFJ0', 'A3KFJ1', 'Q5QPD4', 'Q5QPD2', 'Q5QPD1', 'A3KFJ2']}\n",
      "\n",
      "Gene: HIPK2\n",
      "Symbol: HIPK2\n",
      "Name: homeodomain interacting protein kinase 2\n",
      "Summary: This gene encodes a conserved serine/threonine kinase that is a member of the homeodomain-interacting protein kinase family. The encoded protein interacts with homeodomain transcription factors and many other transcription factors such as p53, and can function as both a corepressor and a coactivator depending on the transcription factor and its subcellular localization. Multiple transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Nov 2011].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - negative regulation of transcription by RNA polymerase II\n",
      "    - eye development\n",
      "    - respiratory system process\n",
      "    - protein phosphorylation\n",
      "    - transforming growth factor beta receptor signaling pathway\n",
      "    - smoothened signaling pathway\n",
      "    - adult walking behavior\n",
      "    - cell population proliferation\n",
      "    - positive regulation of cell population proliferation\n",
      "    - anterior/posterior pattern specification\n",
      "    - gene expression\n",
      "    - retina layer formation\n",
      "    - peptidyl-serine phosphorylation\n",
      "    - peptidyl-serine phosphorylation\n",
      "    - peptidyl-threonine phosphorylation\n",
      "    - neuron differentiation\n",
      "    - erythrocyte differentiation\n",
      "    - DNA damage response, signal transduction by p53 class mediator\n",
      "    - positive regulation of transforming growth factor beta receptor signaling pathway\n",
      "    - negative regulation of BMP signaling pathway\n",
      "    - PML body organization\n",
      "    - thyroid gland development\n",
      "    - positive regulation of protein binding\n",
      "    - epigenetic regulation of gene expression\n",
      "    - intrinsic apoptotic signaling pathway in response to DNA damage by p53 class mediator\n",
      "    - intrinsic apoptotic signaling pathway in response to DNA damage by p53 class mediator\n",
      "    - negative regulation of neuron apoptotic process\n",
      "    - positive regulation of angiogenesis\n",
      "    - positive regulation of transcription by RNA polymerase II\n",
      "    - positive regulation of transcription by RNA polymerase II\n",
      "    - positive regulation of transcription by RNA polymerase II\n",
      "    - positive regulation of JNK cascade\n",
      "    - embryonic camera-type eye morphogenesis\n",
      "    - voluntary musculoskeletal movement\n",
      "    - positive regulation of DNA-binding transcription factor activity\n",
      "    - neuron apoptotic process\n",
      "    - regulation of cell cycle\n",
      "    - embryonic retina morphogenesis in camera-type eye\n",
      "    - lens induction in camera-type eye\n",
      "    - SMAD protein signal transduction\n",
      "    - lung morphogenesis\n",
      "    - iris morphogenesis\n",
      "    - cellular response to hypoxia\n",
      "    - intrinsic apoptotic signaling pathway\n",
      "    - regulation of signal transduction by p53 class mediator\n",
      "    - negative regulation of ubiquitin-dependent protein catabolic process\n",
      "  CC:\n",
      "    - nucleus\n",
      "    - nucleus\n",
      "    - nucleoplasm\n",
      "    - nucleoplasm\n",
      "    - cytoplasm\n",
      "    - cytoplasm\n",
      "    - cytoplasm\n",
      "    - cytoplasmic stress granule\n",
      "    - nuclear body\n",
      "    - nuclear body\n",
      "    - PML body\n",
      "    - PML body\n",
      "    - RNA polymerase II transcription regulator complex\n",
      "  MF:\n",
      "    - transcription coactivator activity\n",
      "    - transcription coactivator activity\n",
      "    - transcription corepressor activity\n",
      "    - transcription corepressor activity\n",
      "    - protein kinase activity\n",
      "    - protein serine/threonine kinase activity\n",
      "    - protein serine/threonine kinase activity\n",
      "    - protein serine/threonine kinase activity\n",
      "    - protein serine/threonine kinase activity\n",
      "    - 3-phosphoinositide-dependent protein kinase activity\n",
      "    - DNA-dependent protein kinase activity\n",
      "    - AMP-activated protein kinase activity\n",
      "    - eukaryotic translation initiation factor 2alpha kinase activity\n",
      "    - ribosomal protein S6 kinase activity\n",
      "    - protein tyrosine kinase activity\n",
      "    - protein binding\n",
      "    - ATP binding\n",
      "    - histone H3S10 kinase activity\n",
      "    - histone H3T11 kinase activity\n",
      "    - histone H3T6 kinase activity\n",
      "    - histone H2AXS139 kinase activity\n",
      "    - histone H3S28 kinase activity\n",
      "    - histone H4S1 kinase activity\n",
      "    - histone H2AS1 kinase activity\n",
      "    - histone H2BS14 kinase activity\n",
      "    - SMAD binding\n",
      "    - SMAD binding\n",
      "    - virion binding\n",
      "    - RNA polymerase II-specific DNA-binding transcription factor binding\n",
      "    - RNA polymerase II-specific DNA-binding transcription factor binding\n",
      "    - histone H3T3 kinase activity\n",
      "    - histone H2AS121 kinase activity\n",
      "    - Rho-dependent protein serine/threonine kinase activity\n",
      "    - protein serine kinase activity\n",
      "    - histone H2BS36 kinase activity\n",
      "    - histone H3S57 kinase activity\n",
      "    - histone H3T45 kinase activity\n",
      "    - histone H2AT120 kinase activity\n",
      "UniProt: {'Swiss-Prot': 'Q9H2X6', 'TrEMBL': 'H7BXX9'}\n",
      "\n",
      "Gene: HIPK1\n",
      "Symbol: HIPK1\n",
      "Name: homeodomain interacting protein kinase 1\n",
      "Summary: The protein encoded by this gene belongs to the Ser/Thr family of protein kinases and HIPK subfamily. It phosphorylates homeodomain transcription factors and may also function as a co-repressor for homeodomain transcription factors. Alternative splicing results in four transcript variants encoding four distinct isoforms. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - eye development\n",
      "    - chromatin remodeling\n",
      "    - protein phosphorylation\n",
      "    - smoothened signaling pathway\n",
      "    - cell population proliferation\n",
      "    - positive regulation of cell population proliferation\n",
      "    - anterior/posterior pattern specification\n",
      "    - regulation of tumor necrosis factor-mediated signaling pathway\n",
      "    - retina layer formation\n",
      "    - neuron differentiation\n",
      "    - adherens junction assembly\n",
      "    - intrinsic apoptotic signaling pathway in response to DNA damage by p53 class mediator\n",
      "    - positive regulation of angiogenesis\n",
      "    - embryonic camera-type eye morphogenesis\n",
      "    - embryonic retina morphogenesis in camera-type eye\n",
      "    - definitive hemopoiesis\n",
      "    - lens induction in camera-type eye\n",
      "    - iris morphogenesis\n",
      "    - endothelial cell apoptotic process\n",
      "    - extrinsic apoptotic signaling pathway\n",
      "  CC:\n",
      "    - nucleus\n",
      "    - nucleus\n",
      "    - nucleoplasm\n",
      "    - nucleoplasm\n",
      "    - cytoplasm\n",
      "    - cytoplasm\n",
      "    - centrosome\n",
      "    - cytosol\n",
      "    - PML body\n",
      "    - nuclear speck\n",
      "    - ciliary basal body\n",
      "  MF:\n",
      "    - protein serine/threonine kinase activity\n",
      "    - 3-phosphoinositide-dependent protein kinase activity\n",
      "    - DNA-dependent protein kinase activity\n",
      "    - AMP-activated protein kinase activity\n",
      "    - eukaryotic translation initiation factor 2alpha kinase activity\n",
      "    - ribosomal protein S6 kinase activity\n",
      "    - protein tyrosine kinase activity\n",
      "    - protein binding\n",
      "    - ATP binding\n",
      "    - histone H3S10 kinase activity\n",
      "    - histone H3T11 kinase activity\n",
      "    - histone H3T6 kinase activity\n",
      "    - histone H2AXS139 kinase activity\n",
      "    - histone H3S28 kinase activity\n",
      "    - histone H4S1 kinase activity\n",
      "    - histone H2AS1 kinase activity\n",
      "    - histone H2BS14 kinase activity\n",
      "    - histone H3T3 kinase activity\n",
      "    - histone H2AS121 kinase activity\n",
      "    - Rho-dependent protein serine/threonine kinase activity\n",
      "    - protein serine kinase activity\n",
      "    - histone H2BS36 kinase activity\n",
      "    - histone H3S57 kinase activity\n",
      "    - histone H3T45 kinase activity\n",
      "    - histone H2AT120 kinase activity\n",
      "UniProt: {'Swiss-Prot': 'Q86Z02', 'TrEMBL': ['Q5SQL3', 'B4DZ33', 'D6RF28', 'H3BLW0', 'J3KP92', 'D6RC95']}\n",
      "\n",
      "Gene: L3MBTL1\n",
      "Symbol: L3MBTL1\n",
      "Name: L3MBTL histone methyl-lysine binding protein 1\n",
      "Summary: This gene represents a polycomb group gene. The encoded protein functions to regulate gene activity, likely via chromatin modification. The encoded protein may also be necessary for mitosis. Alternatively spliced transcript variants encoding different isoforms have been identified.[provided by RefSeq, Sep 2010].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - chromatin organization\n",
      "    - regulation of mitotic nuclear division\n",
      "    - hemopoiesis\n",
      "    - heterochromatin formation\n",
      "    - regulation of megakaryocyte differentiation\n",
      "    - negative regulation of DNA-templated transcription\n",
      "    - negative regulation of DNA-templated transcription\n",
      "    - negative regulation of DNA-templated transcription\n",
      "    - regulation of cell cycle\n",
      "  CC:\n",
      "    - chromatin\n",
      "    - condensed chromosome\n",
      "    - nucleus\n",
      "    - nucleus\n",
      "    - nucleoplasm\n",
      "    - nucleoplasm\n",
      "    - nucleolus\n",
      "    - chromatin lock complex\n",
      "  MF:\n",
      "    - chromatin binding\n",
      "    - chromatin binding\n",
      "    - protein binding\n",
      "    - zinc ion binding\n",
      "    - nucleosome binding\n",
      "    - SAM domain binding\n",
      "    - methylated histone binding\n",
      "    - methylated histone binding\n",
      "    - histone binding\n",
      "    - histone binding\n",
      "    - identical protein binding\n",
      "UniProt: {'Swiss-Prot': 'Q9Y468', 'TrEMBL': ['A0A3B3ITG5', 'A0A3B3ISV7', 'A0A3B3IUA3', 'B0QYN4', 'A0A3B3IT75', 'H0Y7I2', 'A0A3B3ISP5', 'A0A3B3ISK5', 'A0A3B3ISS0', 'A0A994J4Z1', 'A0A3B3ISB7', 'B0QYN5', 'A0A3F2YNZ1', 'A0A3B3IS65', 'A0A0A0MRR4']}\n",
      "\n",
      "Gene: ESR1\n",
      "Symbol: ESR1\n",
      "Name: estrogen receptor 1\n",
      "Summary: This gene encodes an estrogen receptor and ligand-activated transcription factor. The canonical protein contains an N-terminal ligand-independent transactivation domain, a central DNA binding domain, a hinge domain, and a C-terminal ligand-dependent transactivation domain. The protein localizes to the nucleus where it may form either a homodimer or a heterodimer with estrogen receptor 2. The protein encoded by this gene regulates the transcription of many estrogen-inducible genes that play a role in growth, metabolism, sexual development, gestation, and other reproductive functions and is expressed in many non-reproductive tissues. The receptor encoded by this gene plays a key role in breast cancer, endometrial cancer, and osteoporosis. This gene is reported to have dozens of transcript variants due to the use of alternate promoters and alternative splicing, however, the full-length nature of many of these variants remain uncertain. [provided by RefSeq, Jul 2020].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - negative regulation of transcription by RNA polymerase II\n",
      "    - antral ovarian follicle growth\n",
      "    - epithelial cell development\n",
      "    - chromatin remodeling\n",
      "    - regulation of DNA-templated transcription\n",
      "    - regulation of transcription by RNA polymerase II\n",
      "    - signal transduction\n",
      "    - G protein-coupled receptor signaling pathway\n",
      "    - phospholipase C-activating G protein-coupled receptor signaling pathway\n",
      "    - positive regulation of cytosolic calcium ion concentration\n",
      "    - androgen metabolic process\n",
      "    - male gonad development\n",
      "    - negative regulation of gene expression\n",
      "    - nuclear receptor-mediated steroid hormone signaling pathway\n",
      "    - estrogen receptor signaling pathway\n",
      "    - estrogen receptor signaling pathway\n",
      "    - estrogen receptor signaling pathway\n",
      "    - response to estradiol\n",
      "    - regulation of toll-like receptor signaling pathway\n",
      "    - negative regulation of smooth muscle cell apoptotic process\n",
      "    - negative regulation of canonical NF-kappaB signal transduction\n",
      "    - steroid hormone receptor signaling pathway\n",
      "    - steroid hormone receptor signaling pathway\n",
      "    - negative regulation of DNA-binding transcription factor activity\n",
      "    - response to estrogen\n",
      "    - positive regulation of nitric oxide biosynthetic process\n",
      "    - positive regulation of DNA-templated transcription\n",
      "    - positive regulation of transcription by RNA polymerase II\n",
      "    - positive regulation of transcription by RNA polymerase II\n",
      "    - fibroblast proliferation\n",
      "    - positive regulation of fibroblast proliferation\n",
      "    - stem cell differentiation\n",
      "    - regulation of inflammatory response\n",
      "    - positive regulation of nitric-oxide synthase activity\n",
      "    - positive regulation of DNA-binding transcription factor activity\n",
      "    - RNA polymerase II preinitiation complex assembly\n",
      "    - uterus development\n",
      "    - vagina development\n",
      "    - prostate epithelial cord elongation\n",
      "    - prostate epithelial cord arborization involved in prostate glandular acinus morphogenesis\n",
      "    - regulation of branching involved in prostate gland morphogenesis\n",
      "    - mammary gland branching involved in pregnancy\n",
      "    - mammary gland alveolus development\n",
      "    - epithelial cell proliferation involved in mammary gland duct elongation\n",
      "    - protein localization to chromatin\n",
      "    - cellular response to estrogen stimulus\n",
      "    - cellular response to estradiol stimulus\n",
      "    - cellular response to estradiol stimulus\n",
      "    - negative regulation of miRNA transcription\n",
      "    - regulation of epithelial cell apoptotic process\n",
      "  CC:\n",
      "    - chromatin\n",
      "    - chromatin\n",
      "    - chromatin\n",
      "    - euchromatin\n",
      "    - nucleus\n",
      "    - nucleus\n",
      "    - nucleus\n",
      "    - nucleoplasm\n",
      "    - transcription regulator complex\n",
      "    - cytoplasm\n",
      "    - Golgi apparatus\n",
      "    - cytosol\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - membrane\n",
      "    - protein-containing complex\n",
      "    - protein-containing complex\n",
      "  MF:\n",
      "    - RNA polymerase II cis-regulatory region sequence-specific DNA binding\n",
      "    - DNA-binding transcription factor activity, RNA polymerase II-specific\n",
      "    - TFIIB-class transcription factor binding\n",
      "    - transcription coregulator binding\n",
      "    - transcription corepressor binding\n",
      "    - transcription coactivator binding\n",
      "    - DNA-binding transcription activator activity, RNA polymerase II-specific\n",
      "    - chromatin binding\n",
      "    - DNA-binding transcription factor activity\n",
      "    - nuclear receptor activity\n",
      "    - nuclear receptor activity\n",
      "    - nuclear receptor activity\n",
      "    - steroid binding\n",
      "    - protein binding\n",
      "    - calmodulin binding\n",
      "    - beta-catenin binding\n",
      "    - zinc ion binding\n",
      "    - TBP-class protein binding\n",
      "    - enzyme binding\n",
      "    - protein kinase binding\n",
      "    - nitric-oxide synthase regulator activity\n",
      "    - nuclear estrogen receptor activity\n",
      "    - nuclear estrogen receptor activity\n",
      "    - nuclear estrogen receptor activity\n",
      "    - nuclear estrogen receptor binding\n",
      "    - estrogen response element binding\n",
      "    - estrogen response element binding\n",
      "    - G protein-coupled estrogen receptor activity\n",
      "    - G protein-coupled estrogen receptor activity\n",
      "    - identical protein binding\n",
      "    - ATPase binding\n",
      "    - 14-3-3 protein binding\n",
      "    - sequence-specific double-stranded DNA binding\n",
      "UniProt: {'Swiss-Prot': 'P03372', 'TrEMBL': ['H0Y4W6', 'Q9UBT1', 'G4XH65', 'A8KAF4', 'A0A125SXW3', 'Q9H2M2', 'Q9H2M1', 'Q9UE35', 'B0QYW7', 'Q5T5H8']}\n",
      "\n",
      "Gene: VEGFA\n",
      "Symbol: VEGFA\n",
      "Name: vascular endothelial growth factor A\n",
      "Summary: This gene is a member of the PDGF/VEGF growth factor family. It encodes a heparin-binding protein, which exists as a disulfide-linked homodimer. This growth factor induces proliferation and migration of vascular endothelial cells, and is essential for both physiological and pathological angiogenesis. Disruption of this gene in mice resulted in abnormal embryonic blood vessel formation. This gene is upregulated in many known tumors and its expression is correlated with tumor stage and progression. Elevated levels of this protein are found in patients with POEMS syndrome, also known as Crow-Fukase syndrome. Allelic variants of this gene have been associated with microvascular complications of diabetes 1 (MVCD1) and atherosclerosis. Alternatively spliced transcript variants encoding different isoforms have been described. There is also evidence for alternative translation initiation from upstream non-AUG (CUG) codons resulting in additional isoforms. A recent study showed that a C-terminally extended isoform is produced by use of an alternative in-frame translation termination codon via a stop codon readthrough mechanism, and that this isoform is antiangiogenic. Expression of some isoforms derived from the AUG start codon is regulated by a small upstream open reading frame, which is located within an internal ribosome entry site. The levels of VEGF are increased during infection with severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), thus promoting inflammation by facilitating recruitment of inflammatory cells, and by increasing the level of angiopoietin II (Ang II), one of two products of the SARS-CoV-2 binding target, angiotensin-converting enzyme 2 (ACE2). In turn, Ang II facilitates the elevation of VEGF, thus forming a vicious cycle in the release of inflammatory cytokines. [provided by RefSeq, Jun 2020].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - negative regulation of transcription by RNA polymerase II\n",
      "    - angiogenesis\n",
      "    - ovarian follicle development\n",
      "    - branching involved in blood vessel morphogenesis\n",
      "    - branching involved in blood vessel morphogenesis\n",
      "    - vasculogenesis\n",
      "    - response to hypoxia\n",
      "    - response to hypoxia\n",
      "    - in utero embryonic development\n",
      "    - kidney development\n",
      "    - positive regulation of protein phosphorylation\n",
      "    - endothelial cell proliferation\n",
      "    - positive regulation of endothelial cell proliferation\n",
      "    - positive regulation of endothelial cell proliferation\n",
      "    - positive regulation of endothelial cell proliferation\n",
      "    - lymphangiogenesis\n",
      "    - sprouting angiogenesis\n",
      "    - cell migration involved in sprouting angiogenesis\n",
      "    - positive regulation of neuroblast proliferation\n",
      "    - epithelial cell maturation\n",
      "    - positive regulation of receptor internalization\n",
      "    - basophil chemotaxis\n",
      "    - positive regulation of leukocyte migration\n",
      "    - heart morphogenesis\n",
      "    - outflow tract morphogenesis\n",
      "    - coronary vein morphogenesis\n",
      "    - regulation of transcription by RNA polymerase II\n",
      "    - apoptotic process\n",
      "    - phospholipase C-activating G protein-coupled receptor signaling pathway\n",
      "    - nervous system development\n",
      "    - neuroblast proliferation\n",
      "    - mesoderm development\n",
      "    - lactation\n",
      "    - positive regulation of cell population proliferation\n",
      "    - positive regulation of cell population proliferation\n",
      "    - regulation of cell shape\n",
      "    - positive regulation of endothelial cell migration\n",
      "    - positive regulation of gene expression\n",
      "    - negative regulation of gene expression\n",
      "    - negative regulation of gene expression\n",
      "    - negative regulation of epithelial to mesenchymal transition\n",
      "    - regulation of nitric oxide mediated signal transduction\n",
      "    - monocyte differentiation\n",
      "    - macrophage differentiation\n",
      "    - lung development\n",
      "    - positive regulation of cell migration\n",
      "    - epithelial cell differentiation\n",
      "    - post-embryonic camera-type eye development\n",
      "    - retinal ganglion cell axon guidance\n",
      "    - positive regulation of protein-containing complex assembly\n",
      "    - positive regulation of protein autophosphorylation\n",
      "    - tube formation\n",
      "    - endothelial cell chemotaxis\n",
      "    - cellular response to vascular endothelial growth factor stimulus\n",
      "    - lymph vessel morphogenesis\n",
      "    - vascular endothelial growth factor receptor-2 signaling pathway\n",
      "    - positive regulation of endothelial cell chemotaxis by VEGF-activated vascular endothelial growth factor receptor signaling pathway\n",
      "    - vascular endothelial growth factor signaling pathway\n",
      "    - vascular endothelial growth factor signaling pathway\n",
      "    - vascular endothelial growth factor signaling pathway\n",
      "    - vascular endothelial growth factor signaling pathway\n",
      "    - positive regulation of cell proliferation by VEGF-activated platelet derived growth factor receptor signaling pathway\n",
      "    - VEGF-activated neuropilin signaling pathway\n",
      "    - VEGF-activated neuropilin signaling pathway\n",
      "    - vasodilation\n",
      "    - eye photoreceptor cell development\n",
      "    - negative regulation of apoptotic process\n",
      "    - positive regulation of vascular permeability\n",
      "    - positive regulation of canonical NF-kappaB signal transduction\n",
      "    - surfactant homeostasis\n",
      "    - positive regulation of MAPK cascade\n",
      "    - positive regulation of blood vessel endothelial cell migration\n",
      "    - positive regulation of blood vessel endothelial cell migration\n",
      "    - negative regulation of fat cell differentiation\n",
      "    - positive regulation of osteoblast differentiation\n",
      "    - positive regulation of angiogenesis\n",
      "    - positive regulation of angiogenesis\n",
      "    - positive regulation of cell adhesion\n",
      "    - positive regulation of transcription by RNA polymerase II\n",
      "    - positive regulation of transcription by RNA polymerase II\n",
      "    - vascular endothelial growth factor receptor signaling pathway\n",
      "    - vascular endothelial growth factor receptor signaling pathway\n",
      "    - vascular endothelial growth factor receptor signaling pathway\n",
      "    - cell maturation\n",
      "    - camera-type eye morphogenesis\n",
      "    - positive regulation of axon extension involved in axon guidance\n",
      "    - artery morphogenesis\n",
      "    - homeostasis of number of cells within a tissue\n",
      "    - positive regulation of epithelial cell proliferation\n",
      "    - positive regulation of peptidyl-tyrosine phosphorylation\n",
      "    - positive chemotaxis\n",
      "    - positive chemotaxis\n",
      "    - positive chemotaxis\n",
      "    - positive regulation of positive chemotaxis\n",
      "    - induction of positive chemotaxis\n",
      "    - induction of positive chemotaxis\n",
      "    - induction of positive chemotaxis\n",
      "    - positive regulation of cell division\n",
      "    - positive regulation of focal adhesion assembly\n",
      "    - positive regulation of phosphatidylinositol 3-kinase/protein kinase B signal transduction\n",
      "    - positive regulation of phosphatidylinositol 3-kinase/protein kinase B signal transduction\n",
      "    - positive regulation of phosphatidylinositol 3-kinase/protein kinase B signal transduction\n",
      "    - cardiac muscle cell development\n",
      "    - primitive erythrocyte differentiation\n",
      "    - bone trabecula formation\n",
      "    - lung vasculature development\n",
      "    - mammary gland alveolus development\n",
      "    - positive regulation of mast cell chemotaxis\n",
      "    - positive regulation of mast cell chemotaxis\n",
      "    - cardiac vascular smooth muscle cell development\n",
      "    - coronary artery morphogenesis\n",
      "    - vascular wound healing\n",
      "    - positive regulation of ERK1 and ERK2 cascade\n",
      "    - positive regulation of ERK1 and ERK2 cascade\n",
      "    - cellular response to hypoxia\n",
      "    - cellular response to hypoxia\n",
      "    - dopaminergic neuron differentiation\n",
      "    - commissural neuron axon guidance\n",
      "    - positive regulation of cell migration involved in sprouting angiogenesis\n",
      "    - positive regulation of cell migration involved in sprouting angiogenesis\n",
      "    - positive regulation of cell migration involved in sprouting angiogenesis\n",
      "    - positive regulation of branching involved in ureteric bud morphogenesis\n",
      "    - motor neuron migration\n",
      "    - cellular stress response to acid chemical\n",
      "    - cell-cell adhesion\n",
      "    - positive regulation of cold-induced thermogenesis\n",
      "    - positive regulation of vascular endothelial growth factor signaling pathway\n",
      "    - positive regulation of trophoblast cell migration\n",
      "    - positive regulation of lymphangiogenesis\n",
      "    - regulation of hematopoietic progenitor cell differentiation\n",
      "    - negative regulation of miRNA transcription\n",
      "    - positive regulation of protein localization to early endosome\n",
      "    - negative regulation of establishment of endothelial barrier\n",
      "    - negative regulation of adherens junction organization\n",
      "    - positive regulation of blood vessel endothelial cell proliferation involved in sprouting angiogenesis\n",
      "    - positive regulation of blood vessel endothelial cell proliferation involved in sprouting angiogenesis\n",
      "    - positive regulation of sprouting angiogenesis\n",
      "    - positive regulation of sprouting angiogenesis\n",
      "    - positive regulation of epithelial tube formation\n",
      "    - positive regulation of blood vessel branching\n",
      "    - negative regulation of blood-brain barrier permeability\n",
      "    - negative regulation of cell-cell adhesion mediated by cadherin\n",
      "    - positive regulation of DNA biosynthetic process\n",
      "    - positive regulation of DNA biosynthetic process\n",
      "    - positive regulation of endothelial cell chemotaxis\n",
      "  CC:\n",
      "    - extracellular region\n",
      "    - extracellular space\n",
      "    - extracellular space\n",
      "    - extracellular space\n",
      "    - nucleus\n",
      "    - cytoplasm\n",
      "    - endoplasmic reticulum\n",
      "    - Golgi apparatus\n",
      "    - adherens junction\n",
      "    - cell surface\n",
      "    - membrane\n",
      "    - secretory granule\n",
      "    - extracellular matrix\n",
      "    - platelet alpha granule lumen\n",
      "    - VEGF-A complex\n",
      "  MF:\n",
      "    - fibronectin binding\n",
      "    - cytokine activity\n",
      "    - cytokine activity\n",
      "    - platelet-derived growth factor receptor binding\n",
      "    - vascular endothelial growth factor receptor binding\n",
      "    - vascular endothelial growth factor receptor binding\n",
      "    - protein binding\n",
      "    - growth factor activity\n",
      "    - growth factor activity\n",
      "    - growth factor activity\n",
      "    - heparin binding\n",
      "    - heparin binding\n",
      "    - transmembrane receptor protein tyrosine kinase activator activity\n",
      "    - neuropilin binding\n",
      "    - chemoattractant activity\n",
      "    - chemoattractant activity\n",
      "    - chemoattractant activity\n",
      "    - identical protein binding\n",
      "    - protein homodimerization activity\n",
      "    - vascular endothelial growth factor receptor 1 binding\n",
      "    - vascular endothelial growth factor receptor 2 binding\n",
      "    - receptor ligand activity\n",
      "    - receptor ligand activity\n",
      "    - extracellular matrix binding\n",
      "UniProt: {'Swiss-Prot': 'P15692', 'TrEMBL': ['A2A2V4', 'A0A0Y0IMM4', 'H0YBZ0', 'A0A0A0MSH5', 'H0YBI8']}\n",
      "\n",
      "Gene: TGFA\n",
      "Symbol: TGFA\n",
      "Name: transforming growth factor alpha\n",
      "Summary: This gene encodes a growth factor that is a ligand for the epidermal growth factor receptor, which activates a signaling pathway for cell proliferation, differentiation and development. This protein may act as either a transmembrane-bound ligand or a soluble ligand. This gene has been associated with many types of cancers, and it may also be involved in some cases of cleft lip/palate. Alternatively spliced transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Sep 2011].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - angiogenesis\n",
      "    - cell surface receptor signaling pathway\n",
      "    - epidermal growth factor receptor signaling pathway\n",
      "    - epidermal growth factor receptor signaling pathway\n",
      "    - positive regulation of cell population proliferation\n",
      "    - intracellular signal transduction\n",
      "    - ERBB2-EGFR signaling pathway\n",
      "    - positive regulation of MAPK cascade\n",
      "    - positive regulation of mitotic nuclear division\n",
      "    - positive regulation of epithelial cell proliferation\n",
      "    - positive regulation of cell division\n",
      "    - mammary gland alveolus development\n",
      "    - hepatocyte proliferation\n",
      "  CC:\n",
      "    - extracellular region\n",
      "    - extracellular space\n",
      "    - extracellular space\n",
      "    - extracellular space\n",
      "    - endoplasmic reticulum membrane\n",
      "    - plasma membrane\n",
      "    - cell surface\n",
      "    - ER to Golgi transport vesicle membrane\n",
      "    - basolateral plasma membrane\n",
      "    - clathrin-coated endocytic vesicle membrane\n",
      "    - cytoplasmic vesicle\n",
      "    - endoplasmic reticulum-Golgi intermediate compartment membrane\n",
      "    - perinuclear region of cytoplasm\n",
      "  MF:\n",
      "    - epidermal growth factor receptor binding\n",
      "    - epidermal growth factor receptor binding\n",
      "    - protein binding\n",
      "    - growth factor activity\n",
      "    - growth factor activity\n",
      "    - transmembrane receptor protein tyrosine kinase activator activity\n",
      "    - receptor ligand activity\n",
      "UniProt: {'Swiss-Prot': 'P01135', 'TrEMBL': ['E7EPT6', 'F8VNR3', 'F8WA74', 'H0Y6S5']}\n",
      "\n",
      "Gene: RUNX1\n",
      "Symbol: RUNX1\n",
      "Name: RUNX family transcription factor 1\n",
      "Summary: Core binding factor (CBF) is a heterodimeric transcription factor that binds to the core element of many enhancers and promoters. The protein encoded by this gene represents the alpha subunit of CBF and is thought to be involved in the development of normal hematopoiesis. Chromosomal translocations involving this gene are well-documented and have been associated with several types of leukemia. Three transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - negative regulation of transcription by RNA polymerase II\n",
      "    - ossification\n",
      "    - chondrocyte differentiation\n",
      "    - myeloid leukocyte differentiation\n",
      "    - regulation of transcription by RNA polymerase II\n",
      "    - regulation of plasminogen activation\n",
      "    - hemopoiesis\n",
      "    - hemopoiesis\n",
      "    - hemopoiesis\n",
      "    - myeloid cell differentiation\n",
      "    - neuron differentiation\n",
      "    - negative regulation of granulocyte differentiation\n",
      "    - positive regulation of granulocyte differentiation\n",
      "    - positive regulation of interleukin-2 production\n",
      "    - positive regulation of collagen biosynthetic process\n",
      "    - negative regulation of CD4-positive, alpha-beta T cell differentiation\n",
      "    - positive regulation of CD8-positive, alpha-beta T cell differentiation\n",
      "    - regulation of cell differentiation\n",
      "    - positive regulation of angiogenesis\n",
      "    - positive regulation of DNA-templated transcription\n",
      "    - positive regulation of transcription by RNA polymerase II\n",
      "    - positive regulation of transcription by RNA polymerase II\n",
      "    - peripheral nervous system neuron development\n",
      "    - regulation of cardiac muscle cell proliferation\n",
      "    - cardiac muscle tissue regeneration\n",
      "    - hematopoietic stem cell proliferation\n",
      "    - positive regulation of extracellular matrix organization\n",
      "    - regulation of connective tissue replacement\n",
      "  CC:\n",
      "    - chromatin\n",
      "    - nucleus\n",
      "    - nucleus\n",
      "    - nucleoplasm\n",
      "    - nucleoplasm\n",
      "    - core-binding factor complex\n",
      "    - intracellular membrane-bounded organelle\n",
      "  MF:\n",
      "    - transcription cis-regulatory region binding\n",
      "    - RNA polymerase II transcription regulatory region sequence-specific DNA binding\n",
      "    - RNA polymerase II transcription regulatory region sequence-specific DNA binding\n",
      "    - RNA polymerase II cis-regulatory region sequence-specific DNA binding\n",
      "    - RNA polymerase II cis-regulatory region sequence-specific DNA binding\n",
      "    - DNA-binding transcription factor activity, RNA polymerase II-specific\n",
      "    - DNA-binding transcription factor activity, RNA polymerase II-specific\n",
      "    - transcription corepressor binding\n",
      "    - transcription coactivator binding\n",
      "    - DNA-binding transcription activator activity, RNA polymerase II-specific\n",
      "    - DNA-binding transcription activator activity, RNA polymerase II-specific\n",
      "    - DNA binding\n",
      "    - DNA-binding transcription factor activity\n",
      "    - calcium ion binding\n",
      "    - protein binding\n",
      "    - ATP binding\n",
      "    - protein homodimerization activity\n",
      "    - protein heterodimerization activity\n",
      "    - DNA-binding transcription factor binding\n",
      "UniProt: {'Swiss-Prot': 'Q01196', 'TrEMBL': ['A0A0C4DG58', 'H9KVB1', 'V9GYT3', 'C9JWM1', 'A8MZI9', 'V9GYT5']}\n",
      "\n",
      "Gene: CTSK\n",
      "Symbol: CTSK\n",
      "Name: cathepsin K\n",
      "Summary: The protein encoded by this gene is a lysosomal cysteine proteinase involved in bone remodeling and resorption. This protein, which is a member of the peptidase C1 protein family, is predominantly expressed in osteoclasts. However, the encoded protein is also expressed in a significant fraction of human breast cancers, where it could contribute to tumor invasiveness. Mutations in this gene are the cause of pycnodysostosis, an autosomal recessive disease characterized by osteosclerosis and short stature. [provided by RefSeq, Apr 2013].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - mitophagy\n",
      "    - intramembranous ossification\n",
      "    - proteolysis\n",
      "    - thyroid hormone generation\n",
      "    - extracellular matrix disassembly\n",
      "    - collagen catabolic process\n",
      "    - collagen catabolic process\n",
      "    - response to insulin\n",
      "    - cellular response to zinc ion starvation\n",
      "    - bone resorption\n",
      "    - response to ethanol\n",
      "    - proteolysis involved in protein catabolic process\n",
      "    - proteolysis involved in protein catabolic process\n",
      "    - negative regulation of cartilage development\n",
      "    - cellular response to tumor necrosis factor\n",
      "    - cellular response to transforming growth factor beta stimulus\n",
      "    - mononuclear cell differentiation\n",
      "  CC:\n",
      "    - extracellular region\n",
      "    - extracellular space\n",
      "    - extracellular space\n",
      "    - nucleoplasm\n",
      "    - lysosome\n",
      "    - lysosome\n",
      "    - external side of plasma membrane\n",
      "    - apical plasma membrane\n",
      "    - endolysosome lumen\n",
      "    - lysosomal lumen\n",
      "    - intracellular membrane-bounded organelle\n",
      "  MF:\n",
      "    - fibronectin binding\n",
      "    - cysteine-type endopeptidase activity\n",
      "    - cysteine-type endopeptidase activity\n",
      "    - serine-type endopeptidase activity\n",
      "    - protein binding\n",
      "    - collagen binding\n",
      "    - cysteine-type peptidase activity\n",
      "    - proteoglycan binding\n",
      "UniProt: {'Swiss-Prot': 'P43235', 'TrEMBL': ['A0A7I2V354', 'A0A7I2YQX0', 'A0A7I2V617', 'A0A7I2V5Y6', 'Q5QP40', 'A0A7I2V5Y9', 'A0A7I2V4B6', 'A0A7I2V4B1', 'A0A7I2V2M6']}\n",
      "\n",
      "Gene: CCND1\n",
      "Symbol: CCND1\n",
      "Name: cyclin D1\n",
      "Summary: The protein encoded by this gene belongs to the highly conserved cyclin family, whose members are characterized by a dramatic periodicity in protein abundance throughout the cell cycle. Cyclins function as regulators of CDK kinases. Different cyclins exhibit distinct expression and degradation patterns which contribute to the temporal coordination of each mitotic event. This cyclin forms a complex with and functions as a regulatory subunit of CDK4 or CDK6, whose activity is required for cell cycle G1/S transition. This protein has been shown to interact with tumor suppressor protein Rb and the expression of this gene is regulated positively by Rb. Mutations, amplification and overexpression of this gene, which alters cell cycle progression, are observed frequently in a variety of human cancers. [provided by RefSeq, Dec 2019].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - G1/S transition of mitotic cell cycle\n",
      "    - G1/S transition of mitotic cell cycle\n",
      "    - G1/S transition of mitotic cell cycle\n",
      "    - G1/S transition of mitotic cell cycle\n",
      "    - negative regulation of transcription by RNA polymerase II\n",
      "    - re-entry into mitotic cell cycle\n",
      "    - DNA damage response\n",
      "    - lactation\n",
      "    - response to xenobiotic stimulus\n",
      "    - positive regulation of G2/M transition of mitotic cell cycle\n",
      "    - Wnt signaling pathway\n",
      "    - neuron differentiation\n",
      "    - negative regulation of epithelial cell differentiation\n",
      "    - endoplasmic reticulum unfolded protein response\n",
      "    - mitotic G1 DNA damage checkpoint signaling\n",
      "    - mammary gland epithelial cell proliferation\n",
      "    - positive regulation of mammary gland epithelial cell proliferation\n",
      "    - negative regulation of neuron apoptotic process\n",
      "    - response to leptin\n",
      "    - fat cell differentiation\n",
      "    - cell division\n",
      "    - mammary gland alveolus development\n",
      "    - response to UV-A\n",
      "    - liver regeneration\n",
      "    - positive regulation of G1/S transition of mitotic cell cycle\n",
      "  CC:\n",
      "    - cyclin-dependent protein kinase holoenzyme complex\n",
      "    - cyclin-dependent protein kinase holoenzyme complex\n",
      "    - nucleus\n",
      "    - nucleus\n",
      "    - nucleoplasm\n",
      "    - nucleoplasm\n",
      "    - cytoplasm\n",
      "    - microtubule organizing center\n",
      "    - cytosol\n",
      "    - bicellular tight junction\n",
      "    - transcription repressor complex\n",
      "    - nuclear membrane\n",
      "    - cyclin D1-CDK4 complex\n",
      "    - cyclin D1-CDK6 complex\n",
      "  MF:\n",
      "    - transcription corepressor activity\n",
      "    - protein kinase activity\n",
      "    - protein binding\n",
      "    - cyclin-dependent protein serine/threonine kinase regulator activity\n",
      "    - enzyme binding\n",
      "    - protein kinase binding\n",
      "    - histone deacetylase binding\n",
      "    - protein serine/threonine kinase activator activity\n",
      "    - cyclin-dependent protein serine/threonine kinase activator activity\n",
      "    - cyclin-dependent protein serine/threonine kinase activator activity\n",
      "    - proline-rich region binding\n",
      "UniProt: {'Swiss-Prot': 'P24385', 'TrEMBL': ['F5H437', 'Q6FI00']}\n",
      "\n",
      "Gene: SMAD4\n",
      "Symbol: SMAD4\n",
      "Name: SMAD family member 4\n",
      "Summary: This gene encodes a member of the Smad family of signal transduction proteins. Smad proteins are phosphorylated and activated by transmembrane serine-threonine receptor kinases in response to transforming growth factor (TGF)-beta signaling. The product of this gene forms homomeric complexes and heteromeric complexes with other activated Smad proteins, which then accumulate in the nucleus and regulate the transcription of target genes. This protein binds to DNA and recognizes an 8-bp palindromic sequence (GTCTAGAC) called the Smad-binding element (SBE). The protein acts as a tumor suppressor and inhibits epithelial cell proliferation. It may also have an inhibitory effect on tumors by reducing angiogenesis and increasing blood vessel hyperpermeability. The encoded protein is a crucial component of the bone morphogenetic protein signaling pathway. The Smad proteins are subject to complex regulation by post-translational modifications. Mutations or deletions in this gene have been shown to result in pancreatic cancer, juvenile polyposis syndrome, and hereditary hemorrhagic telangiectasia syndrome. [provided by RefSeq, May 2022].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - negative regulation of transcription by RNA polymerase II\n",
      "    - ovarian follicle development\n",
      "    - osteoblast differentiation\n",
      "    - branching involved in ureteric bud morphogenesis\n",
      "    - response to hypoxia\n",
      "    - in utero embryonic development\n",
      "    - gastrulation with mouth forming second\n",
      "    - epithelial to mesenchymal transition\n",
      "    - epithelial to mesenchymal transition\n",
      "    - outflow tract septum morphogenesis\n",
      "    - cardiac conduction system development\n",
      "    - atrioventricular valve formation\n",
      "    - epithelial to mesenchymal transition involved in endocardial cushion formation\n",
      "    - left ventricular cardiac muscle tissue morphogenesis\n",
      "    - positive regulation of cell proliferation involved in heart valve morphogenesis\n",
      "    - brainstem development\n",
      "    - DNA-templated transcription\n",
      "    - DNA-templated transcription\n",
      "    - regulation of DNA-templated transcription\n",
      "    - regulation of transcription by RNA polymerase II\n",
      "    - transcription by RNA polymerase II\n",
      "    - intracellular iron ion homeostasis\n",
      "    - transforming growth factor beta receptor signaling pathway\n",
      "    - transforming growth factor beta receptor signaling pathway\n",
      "    - transforming growth factor beta receptor signaling pathway\n",
      "    - transforming growth factor beta receptor signaling pathway\n",
      "    - transforming growth factor beta receptor signaling pathway\n",
      "    - transforming growth factor beta receptor signaling pathway\n",
      "    - spermatogenesis\n",
      "    - single fertilization\n",
      "    - axon guidance\n",
      "    - cell population proliferation\n",
      "    - negative regulation of cell population proliferation\n",
      "    - anatomical structure morphogenesis\n",
      "    - negative regulation of cardiac muscle hypertrophy\n",
      "    - positive regulation of gene expression\n",
      "    - epithelial cell migration\n",
      "    - positive regulation of cardiac muscle cell apoptotic process\n",
      "    - positive regulation of epithelial to mesenchymal transition\n",
      "    - neural crest cell differentiation\n",
      "    - cardiac muscle hypertrophy in response to stress\n",
      "    - cell differentiation\n",
      "    - negative regulation of cell growth\n",
      "    - adrenal gland development\n",
      "    - BMP signaling pathway\n",
      "    - BMP signaling pathway\n",
      "    - BMP signaling pathway\n",
      "    - BMP signaling pathway\n",
      "    - BMP signaling pathway\n",
      "    - positive regulation of transforming growth factor beta receptor signaling pathway\n",
      "    - somite rostral/caudal axis specification\n",
      "    - regulation of transforming growth factor beta2 production\n",
      "    - activin receptor signaling pathway\n",
      "    - positive regulation of luteinizing hormone secretion\n",
      "    - intracellular signal transduction\n",
      "    - atrioventricular canal development\n",
      "    - endothelial cell activation\n",
      "    - negative regulation of protein catabolic process\n",
      "    - embryonic digit morphogenesis\n",
      "    - negative regulation of DNA-templated transcription\n",
      "    - positive regulation of DNA-templated transcription\n",
      "    - positive regulation of transcription by RNA polymerase II\n",
      "    - positive regulation of transcription by RNA polymerase II\n",
      "    - positive regulation of transcription by RNA polymerase II\n",
      "    - positive regulation of follicle-stimulating hormone secretion\n",
      "    - mesendoderm development\n",
      "    - developmental growth\n",
      "    - neuron fate specification\n",
      "    - sebaceous gland development\n",
      "    - formation of anatomical boundary\n",
      "    - regulation of hair follicle development\n",
      "    - uterus development\n",
      "    - positive regulation of SMAD protein signal transduction\n",
      "    - SMAD protein signal transduction\n",
      "    - SMAD protein signal transduction\n",
      "    - SMAD protein signal transduction\n",
      "    - SMAD protein signal transduction\n",
      "    - ventricular septum morphogenesis\n",
      "    - endocardial cell differentiation\n",
      "    - female gonad morphogenesis\n",
      "    - secondary palate development\n",
      "    - interleukin-6-mediated signaling pathway\n",
      "    - ERK1 and ERK2 cascade\n",
      "    - negative regulation of ERK1 and ERK2 cascade\n",
      "    - cellular response to glucose stimulus\n",
      "    - response to transforming growth factor beta\n",
      "    - cellular response to transforming growth factor beta stimulus\n",
      "    - cellular response to BMP stimulus\n",
      "    - metanephric mesenchyme morphogenesis\n",
      "    - nephrogenic mesenchyme morphogenesis\n",
      "    - seminiferous tubule development\n",
      "    - negative regulation of canonical Wnt signaling pathway\n",
      "    - extrinsic apoptotic signaling pathway\n",
      "    - positive regulation of extracellular matrix assembly\n",
      "    - positive regulation of miRNA transcription\n",
      "    - negative regulation of cardiac myofibril assembly\n",
      "  CC:\n",
      "    - chromatin\n",
      "    - chromatin\n",
      "    - nucleus\n",
      "    - nucleoplasm\n",
      "    - nucleoplasm\n",
      "    - transcription regulator complex\n",
      "    - transcription regulator complex\n",
      "    - cytoplasm\n",
      "    - centrosome\n",
      "    - cytosol\n",
      "    - cytosol\n",
      "    - activin responsive factor complex\n",
      "    - ciliary basal body\n",
      "    - SMAD protein complex\n",
      "    - SMAD protein complex\n",
      "    - heteromeric SMAD protein complex\n",
      "    - heteromeric SMAD protein complex\n",
      "    - heteromeric SMAD protein complex\n",
      "  MF:\n",
      "    - transcription cis-regulatory region binding\n",
      "    - RNA polymerase II cis-regulatory region sequence-specific DNA binding\n",
      "    - RNA polymerase II cis-regulatory region sequence-specific DNA binding\n",
      "    - DNA-binding transcription factor activity, RNA polymerase II-specific\n",
      "    - DNA-binding transcription factor activity, RNA polymerase II-specific\n",
      "    - transcription corepressor binding\n",
      "    - transcription coactivator binding\n",
      "    - DNA-binding transcription activator activity, RNA polymerase II-specific\n",
      "    - DNA binding\n",
      "    - chromatin binding\n",
      "    - DNA-binding transcription factor activity\n",
      "    - DNA-binding transcription factor activity\n",
      "    - protein binding\n",
      "    - collagen binding\n",
      "    - filamin binding\n",
      "    - identical protein binding\n",
      "    - protein homodimerization activity\n",
      "    - sulfate binding\n",
      "    - sequence-specific DNA binding\n",
      "    - metal ion binding\n",
      "    - RNA polymerase II-specific DNA-binding transcription factor binding\n",
      "    - I-SMAD binding\n",
      "    - I-SMAD binding\n",
      "    - R-SMAD binding\n",
      "UniProt: {'Swiss-Prot': 'Q13485', 'TrEMBL': ['A0A087WUF3', 'A0A024R274', 'A0AAQ5BHU2', 'A0AAQ5BHU1', 'A0AAQ5BHT7', 'A0AAQ5BHV5', 'A0AAQ5BHZ6', 'A0AAQ5BHV1', 'A0AAQ5BHY6', 'A0AAQ5BHW2', 'A0AAQ5BHQ2', 'A0AAQ5BHU0', 'A0AAQ5BHV2', 'A0AAQ5BHQ0', 'K7EIU8', 'A0AAQ5BHU9', 'K7ES96', 'K7ELK2', 'A0AAQ5BHS5', 'A0AAQ5BHS9']}\n",
      "\n",
      "Gene: TNFRSF10A\n",
      "Symbol: TNFRSF10A\n",
      "Name: TNF receptor superfamily member 10a\n",
      "Summary: The protein encoded by this gene is a member of the TNF-receptor superfamily. This receptor is activated by tumor necrosis factor-related apoptosis inducing ligand (TNFSF10/TRAIL), and thus transduces cell death signal and induces cell apoptosis. Studies with FADD-deficient mice suggested that FADD, a death domain containing adaptor protein, is required for the apoptosis mediated by this protein. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - apoptotic process\n",
      "    - signal transduction\n",
      "    - activation of NF-kappaB-inducing kinase activity\n",
      "    - extrinsic apoptotic signaling pathway via death domain receptors\n",
      "    - TRAIL-activated apoptotic signaling pathway\n",
      "    - TRAIL-activated apoptotic signaling pathway\n",
      "    - TRAIL-activated apoptotic signaling pathway\n",
      "    - positive regulation of apoptotic process\n",
      "    - cellular response to mechanical stimulus\n",
      "    - extrinsic apoptotic signaling pathway\n",
      "  CC:\n",
      "    - Golgi apparatus\n",
      "    - cytosol\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - cell surface\n",
      "    - cell surface\n",
      "    - plasma membrane raft\n",
      "    - membrane raft\n",
      "  MF:\n",
      "    - protease binding\n",
      "    - death receptor activity\n",
      "    - protein binding\n",
      "    - signaling receptor activity\n",
      "    - identical protein binding\n",
      "    - TRAIL binding\n",
      "UniProt: {'Swiss-Prot': 'O00220', 'TrEMBL': ['E5RFH1', 'F8U8C0']}\n",
      "\n",
      "Gene: TNFRSF10B\n",
      "Symbol: TNFRSF10B\n",
      "Name: TNF receptor superfamily member 10b\n",
      "Summary: The protein encoded by this gene is a member of the TNF-receptor superfamily, and contains an intracellular death domain. This receptor can be activated by tumor necrosis factor-related apoptosis inducing ligand (TNFSF10/TRAIL/APO-2L), and transduces an apoptosis signal. Studies with FADD-deficient mice suggested that FADD, a death domain containing adaptor protein, is required for the apoptosis mediated by this protein. Two transcript variants encoding different isoforms and one non-coding transcript have been found for this gene. [provided by RefSeq, Mar 2009].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - defense response to tumor cell\n",
      "    - apoptotic process\n",
      "    - cell surface receptor signaling pathway\n",
      "    - activation of NF-kappaB-inducing kinase activity\n",
      "    - extrinsic apoptotic signaling pathway via death domain receptors\n",
      "    - response to endoplasmic reticulum stress\n",
      "    - TRAIL-activated apoptotic signaling pathway\n",
      "    - TRAIL-activated apoptotic signaling pathway\n",
      "    - regulation of apoptotic process\n",
      "    - positive regulation of apoptotic process\n",
      "    - positive regulation of canonical NF-kappaB signal transduction\n",
      "    - intrinsic apoptotic signaling pathway in response to endoplasmic reticulum stress\n",
      "    - intrinsic apoptotic signaling pathway in response to endoplasmic reticulum stress\n",
      "    - cellular response to mechanical stimulus\n",
      "  CC:\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - cell surface\n",
      "    - cell surface\n",
      "  MF:\n",
      "    - protein binding\n",
      "    - TRAIL receptor activity\n",
      "    - signaling receptor activity\n",
      "    - TRAIL binding\n",
      "UniProt: {'Swiss-Prot': 'O14763', 'TrEMBL': ['E9PBT3', 'Q7Z2I8']}\n",
      "\n",
      "Gene: MYD88\n",
      "Symbol: MYD88\n",
      "Name: MYD88 innate immune signal transduction adaptor\n",
      "Summary: This gene encodes a cytosolic adapter protein that plays a central role in the innate and adaptive immune response. This protein functions as an essential signal transducer in the interleukin-1 and Toll-like receptor signaling pathways. These pathways regulate that activation of numerous proinflammatory genes. The encoded protein consists of an N-terminal death domain and a C-terminal Toll-interleukin1 receptor domain. Patients with defects in this gene have an increased susceptibility to pyogenic bacterial infections. Alternate splicing results in multiple transcript variants. [provided by RefSeq, Feb 2010].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - response to molecule of fungal origin\n",
      "    - leukocyte activation involved in inflammatory response\n",
      "    - neutrophil activation involved in immune response\n",
      "    - MyD88-dependent toll-like receptor signaling pathway\n",
      "    - chromatin remodeling\n",
      "    - phagocytosis\n",
      "    - apoptotic process\n",
      "    - signal transduction\n",
      "    - cell surface receptor signaling pathway\n",
      "    - JNK cascade\n",
      "    - Toll signaling pathway\n",
      "    - induced systemic resistance\n",
      "    - gene expression\n",
      "    - positive regulation of gene expression\n",
      "    - microglia differentiation\n",
      "    - response to amine\n",
      "    - immunoglobulin mediated immune response\n",
      "    - lipopolysaccharide-mediated signaling pathway\n",
      "    - positive regulation of type I interferon production\n",
      "    - response to peptidoglycan\n",
      "    - positive regulation of chemokine production\n",
      "    - positive regulation of interleukin-1 beta production\n",
      "    - positive regulation of interleukin-17 production\n",
      "    - positive regulation of interleukin-23 production\n",
      "    - positive regulation of interleukin-6 production\n",
      "    - positive regulation of interleukin-8 production\n",
      "    - positive regulation of tumor necrosis factor production\n",
      "    - toll-like receptor 4 signaling pathway\n",
      "    - toll-like receptor 4 signaling pathway\n",
      "    - toll-like receptor 5 signaling pathway\n",
      "    - toll-like receptor 8 signaling pathway\n",
      "    - toll-like receptor TLR6:TLR2 signaling pathway\n",
      "    - interleukin-33-mediated signaling pathway\n",
      "    - defense response to bacterium\n",
      "    - defense response to protozoan\n",
      "    - positive regulation of canonical NF-kappaB signal transduction\n",
      "    - response to amino acid\n",
      "    - skin development\n",
      "    - innate immune response\n",
      "    - response to ethanol\n",
      "    - positive regulation of transcription by RNA polymerase II\n",
      "    - positive regulation of JNK cascade\n",
      "    - positive regulation of smooth muscle cell proliferation\n",
      "    - positive regulation of lymphocyte proliferation\n",
      "    - regulation of inflammatory response\n",
      "    - defense response to Gram-positive bacterium\n",
      "    - defense response to Gram-positive bacterium\n",
      "    - positive regulation of NF-kappaB transcription factor activity\n",
      "    - defense response to virus\n",
      "    - type I interferon-mediated signaling pathway\n",
      "    - positive regulation of macrophage cytokine production\n",
      "    - interleukin-1-mediated signaling pathway\n",
      "    - response to interleukin-1\n",
      "    - 3'-UTR-mediated mRNA stabilization\n",
      "    - neutrophil-mediated killing of bacterium\n",
      "    - cellular response to lipopolysaccharide\n",
      "    - cellular response to mechanical stimulus\n",
      "    - establishment of endothelial intestinal barrier\n",
      "    - cellular response to oxidised low-density lipoprotein particle stimulus\n",
      "    - positive regulation of cytokine production involved in inflammatory response\n",
      "    - positive regulation of cytokine production involved in inflammatory response\n",
      "    - positive regulation of NLRP3 inflammasome complex assembly\n",
      "    - regulation of neutrophil migration\n",
      "    - regulation of chemokine (C-X-C motif) ligand 1 production\n",
      "    - regulation of chemokine (C-X-C motif) ligand 2 production\n",
      "  CC:\n",
      "    - nucleus\n",
      "    - cytoplasm\n",
      "    - cytoplasm\n",
      "    - cytosol\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - cell surface\n",
      "    - endosome membrane\n",
      "    - extrinsic component of plasma membrane\n",
      "    - extrinsic component of cytoplasmic side of plasma membrane\n",
      "    - protein-containing complex\n",
      "  MF:\n",
      "    - Toll binding\n",
      "    - death receptor binding\n",
      "    - interleukin-1 receptor binding\n",
      "    - protein binding\n",
      "    - Toll-like receptor binding\n",
      "    - signaling adaptor activity\n",
      "    - identical protein binding\n",
      "    - molecular adaptor activity\n",
      "    - TIR domain binding\n",
      "    - ATP-dependent histone chaperone activity\n",
      "UniProt: {'Swiss-Prot': 'Q99836', 'TrEMBL': ['A0A3F2YP85', 'A0A0A0MSI9', 'A0A3B3ITY3', 'A0A8V8TP38', 'A0A494C0J8']}\n",
      "\n",
      "Gene: NFKB1\n",
      "Symbol: NFKB1\n",
      "Name: nuclear factor kappa B subunit 1\n",
      "Summary: This gene encodes a 105 kD protein which can undergo cotranslational processing by the 26S proteasome to produce a 50 kD protein. The 105 kD protein is a Rel protein-specific transcription inhibitor and the 50 kD protein is a DNA binding subunit of the NF-kappa-B (NFKB) protein complex. NFKB is a transcription regulator that is activated by various intra- and extra-cellular stimuli such as cytokines, oxidant-free radicals, ultraviolet irradiation, and bacterial or viral products. Activated NFKB translocates into the nucleus and stimulates the expression of genes involved in a wide variety of biological functions. Inappropriate activation of NFKB has been associated with a number of inflammatory diseases while persistent inhibition of NFKB leads to inappropriate immune cell development or delayed cell growth. NFKB is a critical regulator of the immediate-early response to viral infection. Alternative splicing results in multiple transcript variants encoding different isoforms, at least one of which is proteolytically processed. [provided by RefSeq, Aug 2020].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - negative regulation of transcription by RNA polymerase II\n",
      "    - negative regulation of transcription by RNA polymerase II\n",
      "    - regulation of transcription by RNA polymerase II\n",
      "    - transcription by RNA polymerase II\n",
      "    - apoptotic process\n",
      "    - inflammatory response\n",
      "    - canonical NF-kappaB signal transduction\n",
      "    - JNK cascade\n",
      "    - negative regulation of gene expression\n",
      "    - positive regulation of macrophage derived foam cell differentiation\n",
      "    - positive regulation of cholesterol efflux\n",
      "    - positive regulation of lipid storage\n",
      "    - negative regulation of calcidiol 1-monooxygenase activity\n",
      "    - negative regulation of vitamin D biosynthetic process\n",
      "    - signal transduction involved in regulation of gene expression\n",
      "    - negative regulation of interleukin-12 production\n",
      "    - response to muscle stretch\n",
      "    - non-canonical NF-kappaB signal transduction\n",
      "    - negative regulation of apoptotic process\n",
      "    - positive regulation of DNA-templated transcription\n",
      "    - positive regulation of DNA-templated transcription\n",
      "    - positive regulation of DNA-templated transcription\n",
      "    - positive regulation of transcription by RNA polymerase II\n",
      "    - positive regulation of transcription by RNA polymerase II\n",
      "    - positive regulation of transcription by RNA polymerase II\n",
      "    - negative regulation of inflammatory response\n",
      "    - B cell receptor signaling pathway\n",
      "    - mammary gland involution\n",
      "    - positive regulation of transcription initiation by RNA polymerase II\n",
      "    - cellular response to lipopolysaccharide\n",
      "    - cellular response to mechanical stimulus\n",
      "    - cellular response to nicotine\n",
      "    - cellular response to interleukin-1\n",
      "    - cellular response to interleukin-6\n",
      "    - cellular response to tumor necrosis factor\n",
      "    - cellular response to dsRNA\n",
      "    - positive regulation of canonical Wnt signaling pathway\n",
      "    - cellular response to interleukin-17\n",
      "    - cellular response to virus\n",
      "    - antibacterial innate immune response\n",
      "    - negative regulation of cytokine production involved in inflammatory response\n",
      "    - positive regulation of hyaluronan biosynthetic process\n",
      "    - cellular response to angiotensin\n",
      "    - positive regulation of miRNA metabolic process\n",
      "  CC:\n",
      "    - chromatin\n",
      "    - chromatin\n",
      "    - chromatin\n",
      "    - extracellular region\n",
      "    - nucleus\n",
      "    - nucleus\n",
      "    - nucleoplasm\n",
      "    - nucleoplasm\n",
      "    - transcription regulator complex\n",
      "    - cytoplasm\n",
      "    - cytoplasm\n",
      "    - cytoplasm\n",
      "    - mitochondrion\n",
      "    - cytosol\n",
      "    - cytosol\n",
      "    - I-kappaB/NF-kappaB complex\n",
      "    - secretory granule lumen\n",
      "    - NF-kappaB p50/p65 complex\n",
      "    - specific granule lumen\n",
      "  MF:\n",
      "    - transcription cis-regulatory region binding\n",
      "    - RNA polymerase II transcription regulatory region sequence-specific DNA binding\n",
      "    - RNA polymerase II cis-regulatory region sequence-specific DNA binding\n",
      "    - RNA polymerase II cis-regulatory region sequence-specific DNA binding\n",
      "    - RNA polymerase II cis-regulatory region sequence-specific DNA binding\n",
      "    - DNA-binding transcription factor activity, RNA polymerase II-specific\n",
      "    - DNA-binding transcription factor activity, RNA polymerase II-specific\n",
      "    - DNA-binding transcription factor activity, RNA polymerase II-specific\n",
      "    - DNA-binding transcription factor activity, RNA polymerase II-specific\n",
      "    - transcription coactivator binding\n",
      "    - DNA-binding transcription repressor activity, RNA polymerase II-specific\n",
      "    - DNA-binding transcription activator activity, RNA polymerase II-specific\n",
      "    - DNA-binding transcription activator activity, RNA polymerase II-specific\n",
      "    - chromatin binding\n",
      "    - DNA-binding transcription factor activity\n",
      "    - transcription coregulator activity\n",
      "    - protein binding\n",
      "    - identical protein binding\n",
      "    - actinin binding\n",
      "    - protein sequestering activity\n",
      "UniProt: {'Swiss-Prot': 'P19838', 'TrEMBL': ['D6RH30', 'A0A8V8TLB2', 'A0A494C1E9', 'A0A494C157']}\n",
      "\n",
      "Gene: GANAB\n",
      "Symbol: GANAB\n",
      "Name: glucosidase II alpha subunit\n",
      "Summary: This gene encodes the alpha subunit of glucosidase II and a member of the glycosyl hydrolase 31 family of proteins. The heterodimeric enzyme glucosidase II plays a role in protein folding and quality control by cleaving glucose residues from immature glycoproteins in the endoplasmic reticulum. Expression of the encoded protein is elevated in lung tumor tissue and in response to UV irradiation. Mutations in this gene cause autosomal-dominant polycystic kidney and liver disease. [provided by RefSeq, Jul 2016].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - carbohydrate metabolic process\n",
      "    - N-glycan processing\n",
      "    - N-glycan processing\n",
      "  CC:\n",
      "    - endoplasmic reticulum\n",
      "    - endoplasmic reticulum lumen\n",
      "    - endoplasmic reticulum lumen\n",
      "    - Golgi apparatus\n",
      "    - membrane\n",
      "    - glucosidase II complex\n",
      "    - glucosidase II complex\n",
      "    - glucosidase II complex\n",
      "    - melanosome\n",
      "    - intracellular membrane-bounded organelle\n",
      "    - extracellular exosome\n",
      "  MF:\n",
      "    - RNA binding\n",
      "    - protein binding\n",
      "    - carbohydrate binding\n",
      "    - glucan 1,3-alpha-glucosidase activity\n",
      "    - glucan 1,3-alpha-glucosidase activity\n",
      "    - alpha-glucosidase activity\n",
      "    - Glc2Man9GlcNAc2 oligosaccharide glucosidase activity\n",
      "UniProt: {'Swiss-Prot': 'Q14697', 'TrEMBL': ['E9PKU7', 'F5H6X6', 'B4DSM6', 'V9HWJ0', 'B4DZ53', 'B4DIW2', 'E9PQ69', 'E9PNH1']}\n",
      "\n",
      "Gene: NME1\n",
      "Symbol: NME1\n",
      "Name: NME/NM23 nucleoside diphosphate kinase 1\n",
      "Summary: This gene (NME1) was identified because of its reduced mRNA transcript levels in highly metastatic cells. Nucleoside diphosphate kinase (NDK) exists as a hexamer composed of 'A' (encoded by this gene) and 'B' (encoded by NME2) isoforms. Mutations in this gene have been identified in aggressive neuroblastomas. Two transcript variants encoding different isoforms have been found for this gene. Co-transcription of this gene and the neighboring downstream gene (NME2) generates naturally-occurring transcripts (NME1-NME2), which encodes a fusion protein comprised of sequence sharing identity with each individual gene product. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - GTP biosynthetic process\n",
      "    - UTP biosynthetic process\n",
      "    - CTP biosynthetic process\n",
      "    - DNA metabolic process\n",
      "    - endocytosis\n",
      "    - nervous system development\n",
      "    - lactation\n",
      "    - negative regulation of cell population proliferation\n",
      "    - cell differentiation\n",
      "    - regulation of apoptotic process\n",
      "    - positive regulation of DNA binding\n",
      "    - positive regulation of epithelial cell proliferation\n",
      "  CC:\n",
      "    - nucleus\n",
      "    - nucleus\n",
      "    - nucleus\n",
      "    - cytoplasm\n",
      "    - early endosome\n",
      "    - cytosol\n",
      "    - cytosol\n",
      "    - membrane\n",
      "    - ruffle membrane\n",
      "    - extracellular exosome\n",
      "  MF:\n",
      "    - magnesium ion binding\n",
      "    - RNA binding\n",
      "    - DNA nuclease activity\n",
      "    - nucleoside diphosphate kinase activity\n",
      "    - nucleoside diphosphate kinase activity\n",
      "    - protein binding\n",
      "    - ATP binding\n",
      "    - GTP binding\n",
      "    - 3'-5' exonuclease activity\n",
      "    - identical protein binding\n",
      "    - ribosomal small subunit binding\n",
      "UniProt: {'Swiss-Prot': 'P15531', 'TrEMBL': ['E5RHP0', 'A0A384MTW7', 'C9K028', 'E7ERL0']}\n",
      "\n",
      "Gene: XIAP\n",
      "Symbol: XIAP\n",
      "Name: X-linked inhibitor of apoptosis\n",
      "Summary: This gene encodes a protein that belongs to a family of apoptotic suppressor proteins. Members of this family share a conserved motif termed, baculovirus IAP repeat, which is necessary for their anti-apoptotic function. This protein functions through binding to tumor necrosis factor receptor-associated factors TRAF1 and TRAF2 and inhibits apoptosis induced by menadione, a potent inducer of free radicals, and interleukin 1-beta converting enzyme. This protein also inhibits at least two members of the caspase family of cell-death proteases, caspase-3 and caspase-7. Mutations in this gene are the cause of X-linked lymphoproliferative syndrome. Alternate splicing results in multiple transcript variants. Pseudogenes of this gene are found on chromosomes 2 and 11.[provided by RefSeq, Feb 2011].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - DNA damage response\n",
      "    - negative regulation of tumor necrosis factor-mediated signaling pathway\n",
      "    - Wnt signaling pathway\n",
      "    - regulation of BMP signaling pathway\n",
      "    - positive regulation of protein ubiquitination\n",
      "    - positive regulation of protein ubiquitination\n",
      "    - positive regulation of type I interferon production\n",
      "    - response to lipopolysaccharide\n",
      "    - defense response to bacterium\n",
      "    - regulation of apoptotic process\n",
      "    - negative regulation of apoptotic process\n",
      "    - negative regulation of apoptotic process\n",
      "    - negative regulation of apoptotic process\n",
      "    - positive regulation of canonical NF-kappaB signal transduction\n",
      "    - regulation of innate immune response\n",
      "    - positive regulation of JNK cascade\n",
      "    - regulation of inflammatory response\n",
      "    - neuron apoptotic process\n",
      "    - regulation of cell cycle\n",
      "    - copper ion homeostasis\n",
      "    - regulation of apoptosis involved in tissue homeostasis\n",
      "    - regulation of nucleotide-binding domain, leucine rich repeat containing receptor signaling pathway\n",
      "    - nucleotide-binding oligomerization domain containing 1 signaling pathway\n",
      "    - nucleotide-binding oligomerization domain containing 2 signaling pathway\n",
      "    - protein K63-linked ubiquitination\n",
      "    - positive regulation of canonical Wnt signaling pathway\n",
      "    - positive regulation of canonical Wnt signaling pathway\n",
      "    - positive regulation of protein linear polyubiquitination\n",
      "  CC:\n",
      "    - nucleus\n",
      "    - nucleus\n",
      "    - nucleoplasm\n",
      "    - cytoplasm\n",
      "    - cytoplasm\n",
      "    - cytoplasm\n",
      "    - cytosol\n",
      "  MF:\n",
      "    - ubiquitin-protein transferase activity\n",
      "    - ubiquitin-protein transferase activity\n",
      "    - cysteine-type endopeptidase inhibitor activity\n",
      "    - protein binding\n",
      "    - zinc ion binding\n",
      "    - identical protein binding\n",
      "    - cysteine-type endopeptidase inhibitor activity involved in apoptotic process\n",
      "    - ubiquitin protein ligase activity\n",
      "    - ubiquitin protein ligase activity\n",
      "    - ubiquitin protein ligase activity\n",
      "    - ubiquitin protein ligase activity\n",
      "    - protein serine/threonine kinase binding\n",
      "UniProt: {'Swiss-Prot': 'P98170', 'TrEMBL': ['B1AKU2', 'B2R9R2']}\n",
      "\n",
      "Gene: HBEGF\n",
      "Symbol: HBEGF\n",
      "Name: heparin binding EGF like growth factor\n",
      "Summary: Enables growth factor activity and heparin binding activity. Involved in several processes, including epidermal growth factor receptor signaling pathway; positive regulation of protein kinase B signaling; and positive regulation of wound healing. Located in cell surface and extracellular space. Implicated in glomerulosclerosis and perinatal necrotizing enterocolitis. [provided by Alliance of Genome Resources, Apr 2022]\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - signal transduction\n",
      "    - epidermal growth factor receptor signaling pathway\n",
      "    - epidermal growth factor receptor signaling pathway\n",
      "    - epidermal growth factor receptor signaling pathway\n",
      "    - muscle organ development\n",
      "    - regulation of heart contraction\n",
      "    - positive regulation of cell population proliferation\n",
      "    - positive regulation of cell growth\n",
      "    - positive regulation of cell migration\n",
      "    - wound healing, spreading of epidermal cells\n",
      "    - ERBB2-EGFR signaling pathway\n",
      "    - ERBB2-ERBB4 signaling pathway\n",
      "    - positive regulation of smooth muscle cell proliferation\n",
      "    - negative regulation of elastin biosynthetic process\n",
      "    - positive regulation of keratinocyte migration\n",
      "    - positive regulation of phosphatidylinositol 3-kinase/protein kinase B signal transduction\n",
      "    - cell chemotaxis\n",
      "    - positive regulation of wound healing\n",
      "  CC:\n",
      "    - extracellular region\n",
      "    - extracellular space\n",
      "    - extracellular space\n",
      "    - extracellular space\n",
      "    - extracellular space\n",
      "    - extracellular space\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - cell surface\n",
      "    - cell surface\n",
      "    - endocytic vesicle membrane\n",
      "    - clathrin-coated endocytic vesicle membrane\n",
      "  MF:\n",
      "    - epidermal growth factor receptor binding\n",
      "    - protein binding\n",
      "    - growth factor activity\n",
      "    - growth factor activity\n",
      "    - heparin binding\n",
      "    - heparin binding\n",
      "    - transmembrane receptor protein tyrosine kinase activator activity\n",
      "    - receptor ligand activity\n",
      "UniProt: {'Swiss-Prot': 'Q99075'}\n",
      "\n",
      "Gene: DNMT3A\n",
      "Symbol: DNMT3A\n",
      "Name: DNA methyltransferase 3 alpha\n",
      "Summary: CpG methylation is an epigenetic modification that is important for embryonic development, imprinting, and X-chromosome inactivation. Studies in mice have demonstrated that DNA methylation is required for mammalian development. This gene encodes a DNA methyltransferase that is thought to function in de novo methylation, rather than maintenance methylation. The protein localizes to the cytoplasm and nucleus and its expression is developmentally regulated. [provided by RefSeq, Mar 2016].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - negative regulation of transcription by RNA polymerase II\n",
      "    - negative regulation of transcription by RNA polymerase II\n",
      "    - DNA methylation-dependent constitutive heterochromatin formation\n",
      "    - spermatogenesis\n",
      "    - response to xenobiotic stimulus\n",
      "    - response to toxic substance\n",
      "    - post-embryonic development\n",
      "    - response to ionizing radiation\n",
      "    - response to lead ion\n",
      "    - neuron differentiation\n",
      "    - regulatory ncRNA-mediated heterochromatin formation\n",
      "    - methylation\n",
      "    - response to estradiol\n",
      "    - response to vitamin A\n",
      "    - response to cocaine\n",
      "    - negative regulation of gene expression via chromosomal CpG island methylation\n",
      "    - negative regulation of DNA-templated transcription\n",
      "    - cellular response to amino acid stimulus\n",
      "    - cellular response to ethanol\n",
      "    - cellular response to hypoxia\n",
      "    - hepatocyte apoptotic process\n",
      "    - autosome genomic imprinting\n",
      "    - transposable element silencing by piRNA-mediated DNA methylation\n",
      "    - positive regulation of cellular response to hypoxia\n",
      "    - cellular response to bisphenol A\n",
      "  CC:\n",
      "    - chromosome, centromeric region\n",
      "    - euchromatin\n",
      "    - heterochromatin\n",
      "    - XY body\n",
      "    - nucleus\n",
      "    - nucleus\n",
      "    - nucleus\n",
      "    - nucleoplasm\n",
      "    - nucleoplasm\n",
      "    - cytoplasm\n",
      "    - cytoplasm\n",
      "    - nuclear matrix\n",
      "    - catalytic complex\n",
      "  MF:\n",
      "    - RNA polymerase II cis-regulatory region sequence-specific DNA binding\n",
      "    - DNA binding\n",
      "    - chromatin binding\n",
      "    - transcription corepressor activity\n",
      "    - DNA (cytosine-5-)-methyltransferase activity\n",
      "    - DNA (cytosine-5-)-methyltransferase activity\n",
      "    - protein binding\n",
      "    - zinc ion binding\n",
      "    - identical protein binding\n",
      "    - unmethylated CpG binding\n",
      "    - DNA (cytosine-5-)-methyltransferase activity, acting on CpN substrates\n",
      "    - DNA (cytosine-5-)-methyltransferase activity, acting on CpNpG substrates\n",
      "    - RNA polymerase II-specific DNA-binding transcription factor binding\n",
      "    - lncRNA binding\n",
      "    - protein-cysteine methyltransferase activity\n",
      "UniProt: {'Swiss-Prot': 'Q9Y6K1', 'TrEMBL': ['F8WE91', 'A0A0C4DG02', 'Q59HC6', 'A0A8I5QKW3', 'A0A8I5KTF6', 'A0A8I5KPS0', 'A0A8I5KNQ0', 'A0A8I5KUD4', 'A0A8I5KXG9']}\n",
      "\n",
      "Gene: CCNB2\n",
      "Symbol: CCNB2\n",
      "Name: cyclin B2\n",
      "Summary: Cyclin B2 is a member of the cyclin family, specifically the B-type cyclins. The B-type cyclins, B1 and B2, associate with p34cdc2 and are essential components of the cell cycle regulatory machinery. B1 and B2 differ in their subcellular localization. Cyclin B1 co-localizes with microtubules, whereas cyclin B2 is primarily associated with the Golgi region. Cyclin B2 also binds to transforming growth factor beta RII and thus cyclin B2/cdc2 may play a key role in transforming growth factor beta-mediated cell cycle control. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - G1/S transition of mitotic cell cycle\n",
      "    - in utero embryonic development\n",
      "    - spindle assembly involved in female meiosis I\n",
      "    - G2/MI transition of meiotic cell cycle\n",
      "    - regulation of growth\n",
      "    - T cell homeostasis\n",
      "    - thymus development\n",
      "    - cell division\n",
      "  CC:\n",
      "    - cyclin-dependent protein kinase holoenzyme complex\n",
      "    - nucleus\n",
      "    - cytoplasm\n",
      "    - centrosome\n",
      "    - microtubule organizing center\n",
      "    - cytosol\n",
      "    - cytosol\n",
      "    - microtubule cytoskeleton\n",
      "    - membrane\n",
      "  MF:\n",
      "    - protein binding\n",
      "    - cyclin-dependent protein serine/threonine kinase regulator activity\n",
      "    - cadherin binding\n",
      "UniProt: {'Swiss-Prot': 'O95067', 'TrEMBL': ['H0YMP3', 'H1UBN3']}\n",
      "\n",
      "Gene: PIK3CD\n",
      "Symbol: PIK3CD\n",
      "Name: phosphatidylinositol-4,5-bisphosphate 3-kinase catalytic subunit delta\n",
      "Summary: Phosphoinositide 3-kinases (PI3Ks) phosphorylate inositol lipids and are involved in the immune response. The protein encoded by this gene is a class I PI3K found primarily in leukocytes. Like other class I PI3Ks (p110-alpha p110-beta, and p110-gamma), the encoded protein binds p85 adapter proteins and GTP-bound RAS. However, unlike the other class I PI3Ks, this protein phosphorylates itself, not p85 protein.[provided by RefSeq, Jul 2010].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - natural killer cell differentiation\n",
      "    - positive regulation of cytokine production\n",
      "    - positive regulation of endothelial cell proliferation\n",
      "    - adaptive immune response\n",
      "    - mast cell chemotaxis\n",
      "    - respiratory burst involved in defense response\n",
      "    - protein phosphorylation\n",
      "    - inflammatory response\n",
      "    - immune response\n",
      "    - signal transduction\n",
      "    - positive regulation of endothelial cell migration\n",
      "    - positive regulation of gene expression\n",
      "    - T cell chemotaxis\n",
      "    - cell migration\n",
      "    - natural killer cell activation\n",
      "    - B cell differentiation\n",
      "    - T cell differentiation\n",
      "    - T cell differentiation\n",
      "    - positive regulation of cell migration\n",
      "    - neutrophil chemotaxis\n",
      "    - positive regulation of neutrophil apoptotic process\n",
      "    - natural killer cell chemotaxis\n",
      "    - B cell chemotaxis\n",
      "    - phosphatidylinositol-3-phosphate biosynthetic process\n",
      "    - vascular endothelial growth factor signaling pathway\n",
      "    - T cell activation\n",
      "    - B cell activation\n",
      "    - mast cell degranulation\n",
      "    - phosphatidylinositol 3-kinase/protein kinase B signal transduction\n",
      "    - phosphatidylinositol 3-kinase/protein kinase B signal transduction\n",
      "    - phosphatidylinositol 3-kinase/protein kinase B signal transduction\n",
      "    - phosphatidylinositol 3-kinase/protein kinase B signal transduction\n",
      "    - innate immune response\n",
      "    - positive regulation of angiogenesis\n",
      "    - phosphatidylinositol-mediated signaling\n",
      "    - T cell receptor signaling pathway\n",
      "    - B cell receptor signaling pathway\n",
      "    - mast cell differentiation\n",
      "    - neutrophil extravasation\n",
      "    - positive regulation of epithelial tube formation\n",
      "  CC:\n",
      "    - cytoplasm\n",
      "    - cytosol\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - phosphatidylinositol 3-kinase complex\n",
      "    - phosphatidylinositol 3-kinase complex\n",
      "    - phosphatidylinositol 3-kinase complex, class IA\n",
      "    - phosphatidylinositol 3-kinase complex, class IA\n",
      "  MF:\n",
      "    - protein binding\n",
      "    - ATP binding\n",
      "    - 1-phosphatidylinositol-3-kinase activity\n",
      "    - 1-phosphatidylinositol-3-kinase activity\n",
      "    - 1-phosphatidylinositol-3-kinase activity\n",
      "    - 1-phosphatidylinositol-4-phosphate 3-kinase activity\n",
      "    - 1-phosphatidylinositol-4,5-bisphosphate 3-kinase activity\n",
      "UniProt: {'Swiss-Prot': 'O00329', 'TrEMBL': ['B7ZM44', 'A0A2K8FKV1', 'F8W9P4', 'A0A8V8TMM0', 'A0A8V8TM62', 'A0A8V8TNW9', 'A0A8V8TNP1', 'A0A8V8TML5']}\n",
      "\n",
      "Gene: IL6R\n",
      "Symbol: IL6R\n",
      "Name: interleukin 6 receptor\n",
      "Summary: This gene encodes a subunit of the interleukin 6 (IL6) receptor complex. Interleukin 6 is a potent pleiotropic cytokine that regulates cell growth and differentiation and plays an important role in the immune response. The IL6 receptor is a protein complex consisting of this protein and interleukin 6 signal transducer (IL6ST/GP130/IL6-beta), a receptor subunit also shared by many other cytokines. Dysregulated production of IL6 and this receptor are implicated in the pathogenesis of many diseases, such as multiple myeloma, autoimmune diseases and prostate cancer. Alternatively spliced transcript variants encoding distinct isoforms have been identified in this gene. A pseudogene of this gene is found on chromosome 9. [provided by RefSeq, Aug 2020].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - hepatic immune response\n",
      "    - neutrophil mediated immunity\n",
      "    - monocyte chemotaxis\n",
      "    - positive regulation of leukocyte chemotaxis\n",
      "    - acute-phase response\n",
      "    - positive regulation of cell population proliferation\n",
      "    - positive regulation of cell population proliferation\n",
      "    - positive regulation of cell population proliferation\n",
      "    - vascular endothelial growth factor production\n",
      "    - cytokine-mediated signaling pathway\n",
      "    - endocrine pancreas development\n",
      "    - endocrine pancreas development\n",
      "    - negative regulation of interleukin-8 production\n",
      "    - positive regulation of chemokine production\n",
      "    - positive regulation of interleukin-6 production\n",
      "    - negative regulation of collagen biosynthetic process\n",
      "    - response to cytokine\n",
      "    - interleukin-11-mediated signaling pathway\n",
      "    - positive regulation of MAPK cascade\n",
      "    - positive regulation of osteoblast differentiation\n",
      "    - positive regulation of smooth muscle cell proliferation\n",
      "    - defense response to Gram-negative bacterium\n",
      "    - defense response to Gram-positive bacterium\n",
      "    - interleukin-6-mediated signaling pathway\n",
      "    - interleukin-6-mediated signaling pathway\n",
      "    - interleukin-6-mediated signaling pathway\n",
      "    - interleukin-6-mediated signaling pathway\n",
      "    - ciliary neurotrophic factor-mediated signaling pathway\n",
      "    - positive regulation of glomerular mesangial cell proliferation\n",
      "    - T-helper 17 cell lineage commitment\n",
      "    - T-helper 17 cell lineage commitment\n",
      "    - extrinsic apoptotic signaling pathway\n",
      "    - cell surface receptor signaling pathway via STAT\n",
      "    - positive regulation of platelet aggregation\n",
      "  CC:\n",
      "    - extracellular region\n",
      "    - extracellular region\n",
      "    - extracellular space\n",
      "    - extracellular space\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - interleukin-6 receptor complex\n",
      "    - external side of plasma membrane\n",
      "    - apical plasma membrane\n",
      "    - receptor complex\n",
      "    - ciliary neurotrophic factor receptor complex\n",
      "  MF:\n",
      "    - cytokine receptor activity\n",
      "    - ciliary neurotrophic factor receptor activity\n",
      "    - interleukin-6 receptor activity\n",
      "    - interleukin-11 receptor activity\n",
      "    - interleukin-6 receptor binding\n",
      "    - protein binding\n",
      "    - enzyme binding\n",
      "    - interleukin-11 binding\n",
      "    - interleukin-6 binding\n",
      "    - interleukin-6 binding\n",
      "    - protein homodimerization activity\n",
      "    - ciliary neurotrophic factor binding\n",
      "UniProt: {'Swiss-Prot': 'P08887', 'TrEMBL': ['A0A087WTB5', 'A0N0L5', 'H0Y949', 'H0Y9B1', 'D6R9R8']}\n",
      "\n",
      "Gene: IL6\n",
      "Symbol: IL6\n",
      "Name: interleukin 6\n",
      "Summary: This gene encodes a cytokine that functions in inflammation and the maturation of B cells. In addition, the encoded protein has been shown to be an endogenous pyrogen capable of inducing fever in people with autoimmune diseases or infections. The protein is primarily produced at sites of acute and chronic inflammation, where it is secreted into the serum and induces a transcriptional inflammatory response through interleukin 6 receptor, alpha. The functioning of this gene is implicated in a wide variety of inflammation-associated disease states, including suspectibility to diabetes mellitus and systemic juvenile rheumatoid arthritis. Elevated levels of the encoded protein have been found in virus infections, including COVID-19 (disease caused by SARS-CoV-2). [provided by RefSeq, Aug 2020].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - neutrophil apoptotic process\n",
      "    - germinal center B cell differentiation\n",
      "    - hepatic immune response\n",
      "    - hepatic immune response\n",
      "    - neutrophil mediated immunity\n",
      "    - monocyte chemotaxis\n",
      "    - positive regulation of immunoglobulin production\n",
      "    - positive regulation of immunoglobulin production\n",
      "    - positive regulation of acute inflammatory response\n",
      "    - positive regulation of leukocyte chemotaxis\n",
      "    - acute-phase response\n",
      "    - inflammatory response\n",
      "    - inflammatory response\n",
      "    - humoral immune response\n",
      "    - cell surface receptor signaling pathway via JAK-STAT\n",
      "    - positive regulation of cell population proliferation\n",
      "    - positive regulation of cell population proliferation\n",
      "    - positive regulation of cell population proliferation\n",
      "    - positive regulation of cell population proliferation\n",
      "    - negative regulation of cell population proliferation\n",
      "    - vascular endothelial growth factor production\n",
      "    - regulation of vascular endothelial growth factor production\n",
      "    - positive regulation of vascular endothelial growth factor production\n",
      "    - positive regulation of gene expression\n",
      "    - positive regulation of gene expression\n",
      "    - positive regulation of epithelial to mesenchymal transition\n",
      "    - negative regulation of lipid storage\n",
      "    - response to activity\n",
      "    - cytokine-mediated signaling pathway\n",
      "    - platelet activation\n",
      "    - endocrine pancreas development\n",
      "    - neuron projection development\n",
      "    - response to peptidoglycan\n",
      "    - negative regulation of chemokine production\n",
      "    - positive regulation of chemokine production\n",
      "    - positive regulation of chemokine production\n",
      "    - positive regulation of interleukin-1 beta production\n",
      "    - positive regulation of interleukin-10 production\n",
      "    - positive regulation of interleukin-17 production\n",
      "    - positive regulation of interleukin-21 production\n",
      "    - positive regulation of interleukin-6 production\n",
      "    - positive regulation of interleukin-8 production\n",
      "    - positive regulation of tumor necrosis factor production\n",
      "    - negative regulation of collagen biosynthetic process\n",
      "    - positive regulation of peptidyl-serine phosphorylation\n",
      "    - maintenance of blood-brain barrier\n",
      "    - positive regulation of T cell proliferation\n",
      "    - glucose homeostasis\n",
      "    - positive regulation of apoptotic process\n",
      "    - negative regulation of apoptotic process\n",
      "    - positive regulation of MAPK cascade\n",
      "    - positive regulation of MAPK cascade\n",
      "    - negative regulation of fat cell differentiation\n",
      "    - positive regulation of osteoblast differentiation\n",
      "    - positive regulation of translation\n",
      "    - regulation of angiogenesis\n",
      "    - negative regulation of bone resorption\n",
      "    - positive regulation of DNA-templated transcription\n",
      "    - positive regulation of DNA-templated transcription\n",
      "    - positive regulation of transcription by RNA polymerase II\n",
      "    - positive regulation of transcription by RNA polymerase II\n",
      "    - positive regulation of transcription by RNA polymerase II\n",
      "    - positive regulation of receptor signaling pathway via JAK-STAT\n",
      "    - positive regulation of receptor signaling pathway via JAK-STAT\n",
      "    - positive regulation of receptor signaling pathway via JAK-STAT\n",
      "    - positive regulation of smooth muscle cell proliferation\n",
      "    - positive regulation of peptidyl-tyrosine phosphorylation\n",
      "    - negative regulation of neurogenesis\n",
      "    - regulation of insulin secretion\n",
      "    - defense response to Gram-negative bacterium\n",
      "    - defense response to Gram-positive bacterium\n",
      "    - positive regulation of B cell activation\n",
      "    - positive regulation of DNA-binding transcription factor activity\n",
      "    - response to glucocorticoid\n",
      "    - defense response to virus\n",
      "    - positive regulation of glial cell proliferation\n",
      "    - T follicular helper cell differentiation\n",
      "    - regulation of astrocyte activation\n",
      "    - neuron cellular homeostasis\n",
      "    - glucagon secretion\n",
      "    - regulation of glucagon secretion\n",
      "    - interleukin-6-mediated signaling pathway\n",
      "    - interleukin-6-mediated signaling pathway\n",
      "    - interleukin-6-mediated signaling pathway\n",
      "    - cellular response to hydrogen peroxide\n",
      "    - cellular response to lipopolysaccharide\n",
      "    - T-helper 17 cell lineage commitment\n",
      "    - hepatocyte proliferation\n",
      "    - positive regulation of extracellular matrix disassembly\n",
      "    - inflammatory response to wounding\n",
      "    - liver regeneration\n",
      "    - cell surface receptor signaling pathway via STAT\n",
      "    - cell surface receptor signaling pathway via STAT\n",
      "    - cellular response to virus\n",
      "    - regulation of neuroinflammatory response\n",
      "    - positive regulation of neuroinflammatory response\n",
      "    - positive regulation of cytokine production involved in inflammatory response\n",
      "    - positive regulation of platelet aggregation\n",
      "    - positive regulation of apoptotic DNA fragmentation\n",
      "    - positive regulation of miRNA transcription\n",
      "    - regulation of microglial cell activation\n",
      "    - positive regulation of receptor signaling pathway via STAT\n",
      "    - positive regulation of leukocyte adhesion to vascular endothelial cell\n",
      "    - positive regulation of T-helper 2 cell cytokine production\n",
      "    - negative regulation of primary miRNA processing\n",
      "    - negative regulation of interleukin-1-mediated signaling pathway\n",
      "    - positive regulation of type B pancreatic cell apoptotic process\n",
      "  CC:\n",
      "    - extracellular region\n",
      "    - extracellular space\n",
      "    - extracellular space\n",
      "    - extracellular space\n",
      "    - endoplasmic reticulum lumen\n",
      "    - interleukin-6 receptor complex\n",
      "    - interleukin-6 receptor complex\n",
      "  MF:\n",
      "    - cytokine activity\n",
      "    - cytokine activity\n",
      "    - cytokine activity\n",
      "    - cytokine activity\n",
      "    - cytokine activity\n",
      "    - interleukin-6 receptor binding\n",
      "    - interleukin-6 receptor binding\n",
      "    - interleukin-6 receptor binding\n",
      "    - protein binding\n",
      "    - growth factor activity\n",
      "    - growth factor activity\n",
      "    - identical protein binding\n",
      "UniProt: {'Swiss-Prot': 'P05231', 'TrEMBL': ['B4DVM1', 'Q75MH2', 'B4DNQ5', 'B5MC21', 'B4DNV3', 'B5MC14', 'A0A8Q3SJL1', 'B5MCZ3', 'C9J5B0']}\n",
      "\n",
      "Gene: ATP7A\n",
      "Symbol: ATP7A\n",
      "Name: ATPase copper transporting alpha\n",
      "Summary: This gene encodes a transmembrane protein that functions in copper transport across membranes. This protein is localized to the trans Golgi network, where it is predicted to supply copper to copper-dependent enzymes in the secretory pathway. It relocalizes to the plasma membrane under conditions of elevated extracellular copper, and functions in the efflux of copper from cells. Mutations in this gene are associated with Menkes disease, X-linked distal spinal muscular atrophy, and occipital horn syndrome. Alternatively-spliced transcript variants have been observed. [provided by RefSeq, Aug 2013].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - blood vessel development\n",
      "    - in utero embryonic development\n",
      "    - liver development\n",
      "    - blood vessel remodeling\n",
      "    - regulation of oxidative phosphorylation\n",
      "    - L-tryptophan metabolic process\n",
      "    - catecholamine metabolic process\n",
      "    - copper ion transport\n",
      "    - intracellular copper ion homeostasis\n",
      "    - intracellular copper ion homeostasis\n",
      "    - mitochondrion organization\n",
      "    - female pregnancy\n",
      "    - lactation\n",
      "    - locomotory behavior\n",
      "    - response to iron(III) ion\n",
      "    - response to manganese ion\n",
      "    - response to zinc ion\n",
      "    - detoxification of copper ion\n",
      "    - positive regulation of lamellipodium assembly\n",
      "    - copper ion import\n",
      "    - copper ion import\n",
      "    - peptidyl-lysine modification\n",
      "    - removal of superoxide radicals\n",
      "    - cerebellar Purkinje cell differentiation\n",
      "    - pyramidal neuron development\n",
      "    - central nervous system neuron development\n",
      "    - extracellular matrix organization\n",
      "    - collagen fibril organization\n",
      "    - hair follicle morphogenesis\n",
      "    - positive regulation of tyrosinase activity\n",
      "    - negative regulation of iron ion transmembrane transport\n",
      "    - cellular response to platelet-derived growth factor stimulus\n",
      "    - T-helper cell differentiation\n",
      "    - epinephrine metabolic process\n",
      "    - norepinephrine metabolic process\n",
      "    - dopamine metabolic process\n",
      "    - serotonin metabolic process\n",
      "    - positive regulation of catalytic activity\n",
      "    - pigmentation\n",
      "    - skin development\n",
      "    - positive regulation of cell size\n",
      "    - positive regulation of melanin biosynthetic process\n",
      "    - elastic fiber assembly\n",
      "    - lung alveolus development\n",
      "    - neuron projection morphogenesis\n",
      "    - positive regulation of epithelial cell proliferation\n",
      "    - cartilage development\n",
      "    - elastin biosynthetic process\n",
      "    - copper ion export\n",
      "    - copper ion export\n",
      "    - copper ion export\n",
      "    - cellular response to amino acid stimulus\n",
      "    - cellular response to antibiotic\n",
      "    - cellular response to cadmium ion\n",
      "    - cellular response to cobalt ion\n",
      "    - cellular response to copper ion\n",
      "    - cellular response to iron ion\n",
      "    - cellular response to lead ion\n",
      "    - cellular response to hypoxia\n",
      "    - positive regulation of response to wounding\n",
      "    - positive regulation of vascular associated smooth muscle cell migration\n",
      "  CC:\n",
      "    - late endosome\n",
      "    - endoplasmic reticulum\n",
      "    - endoplasmic reticulum\n",
      "    - Golgi apparatus\n",
      "    - trans-Golgi network\n",
      "    - trans-Golgi network\n",
      "    - cytosol\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - microvillus\n",
      "    - postsynaptic density\n",
      "    - membrane\n",
      "    - basolateral plasma membrane\n",
      "    - apical plasma membrane\n",
      "    - trans-Golgi network transport vesicle\n",
      "    - secretory granule\n",
      "    - axon\n",
      "    - dendrite\n",
      "    - phagocytic vesicle membrane\n",
      "    - cell leading edge\n",
      "    - brush border membrane\n",
      "    - early endosome membrane\n",
      "    - trans-Golgi network membrane\n",
      "    - melanosome membrane\n",
      "    - neuron projection\n",
      "    - neuronal cell body\n",
      "    - perikaryon\n",
      "    - perinuclear region of cytoplasm\n",
      "  MF:\n",
      "    - copper ion transmembrane transporter activity\n",
      "    - copper ion binding\n",
      "    - copper ion binding\n",
      "    - protein binding\n",
      "    - ATP binding\n",
      "    - ATP binding\n",
      "    - superoxide dismutase copper chaperone activity\n",
      "    - ATP hydrolysis activity\n",
      "    - small GTPase binding\n",
      "    - copper-dependent protein binding\n",
      "    - copper-dependent protein binding\n",
      "    - P-type divalent copper transporter activity\n",
      "    - P-type divalent copper transporter activity\n",
      "    - protein-folding chaperone binding\n",
      "    - P-type monovalent copper transporter activity\n",
      "    - cuprous ion binding\n",
      "    - cuprous ion binding\n",
      "UniProt: {'Swiss-Prot': 'Q04656', 'TrEMBL': ['B4DRW0', 'Q762B6', 'A0A8I5KWH1', 'A0A8I5KZ63', 'A0A8I5KU46', 'A0A2R8YD60', 'A0A8I5KQ07', 'A0A8I5KST1', 'A0A8I5QJP0', 'A0A8I5KVB6', 'A0A8I5KQ62', 'A0A8I5KY05', 'A0A8I5KPN6', 'A0A8I5KWA8', 'A0A8I5KXV0', 'A0A8J9FM07', 'A0A8I5KVT9', 'A0A8I5KR00', 'A0A8I5QKR3']}\n",
      "\n",
      "Gene: PSTPIP1\n",
      "Symbol: PSTPIP1\n",
      "Name: proline-serine-threonine phosphatase interacting protein 1\n",
      "Summary: This gene encodes a cytoskeletal protein that is highly expressed in hemopoietic tissues. This protein functions via its interaction with several different proteins involved in cytoskeletal organization and inflammatory processes. It binds to the cytoplasmic tail of CD2, an effector of T cell activation and adhesion, downregulating CD2-triggered adhesion. It binds PEST-type protein tyrosine phosphatases (PTP) and directs them to c-Abl kinase to mediate c-Abl dephosphorylation, thereby, regulating c-Abl activity. It also interacts with pyrin, which is found in association with the cytoskeleton in myeloid/monocytic cells and modulates immunoregulatory functions. Mutations in this gene are associated with PAPA (pyogenic sterile arthritis, pyoderma gangrenosum, and acne) syndrome. It is hypothesized that the disease-causing mutations compromise physiologic signaling necessary for the maintenance of a proper inflammatory response. [provided by RefSeq, Mar 2016].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - endocytosis\n",
      "    - inflammatory response\n",
      "    - cell adhesion\n",
      "    - signal transduction\n",
      "    - innate immune response\n",
      "  CC:\n",
      "    - uropod\n",
      "    - cytoplasm\n",
      "    - cytosol\n",
      "    - cytosol\n",
      "    - cytoskeleton\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - membrane\n",
      "    - lamellipodium\n",
      "    - cleavage furrow\n",
      "    - perinuclear region of cytoplasm\n",
      "  MF:\n",
      "    - protein binding\n",
      "    - identical protein binding\n",
      "UniProt: {'Swiss-Prot': 'O43586', 'TrEMBL': ['B4E1Z9', 'A0A0S2Z5P3', 'H0YNR2', 'J3KPG6', 'H0YNS1', 'H0YLX8', 'H0YLI2', 'H0YLW8', 'H0YKY3', 'H0YLM9', 'H0YKF1']}\n",
      "\n",
      "Gene: GLP2R\n",
      "Symbol: GLP2R\n",
      "Name: glucagon like peptide 2 receptor\n",
      "Summary: This gene encodes a G protein-coupled receptor that is closely related to the glucagon receptor and binds to glucagon-like peptide-2 (GLP2). Signalling through GLP2 stimulates intestinal growth and increases villus height in the small intestine, concomitant with increased crypt cell proliferation and decreased enterocyte apoptosis. [provided by RefSeq, Dec 2014].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - cell surface receptor signaling pathway\n",
      "    - adenylate cyclase-modulating G protein-coupled receptor signaling pathway\n",
      "    - positive regulation of cell population proliferation\n",
      "    - cellular response to glucagon stimulus\n",
      "  CC:\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - membrane\n",
      "  MF:\n",
      "    - G protein-coupled receptor activity\n",
      "    - glucagon receptor activity\n",
      "    - peptide hormone binding\n",
      "UniProt: {'Swiss-Prot': 'O95838', 'TrEMBL': ['B2RAN4', 'A0A384MTS7', 'E9PC86', 'I3L0P5', 'H0Y6C2']}\n",
      "\n",
      "Gene: PTCH1\n",
      "Symbol: PTCH1\n",
      "Name: patched 1\n",
      "Summary: This gene encodes a member of the patched family of proteins and a component of the hedgehog signaling pathway. Hedgehog signaling is important in embryonic development and tumorigenesis. The encoded protein is the receptor for the secreted hedgehog ligands, which include sonic hedgehog, indian hedgehog and desert hedgehog. Following binding by one of the hedgehog ligands, the encoded protein is trafficked away from the primary cilium, relieving inhibition of the G-protein-coupled receptor smoothened, which results in activation of downstream signaling. Mutations of this gene have been associated with basal cell nevus syndrome and holoprosencephaly. [provided by RefSeq, Aug 2017].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - negative regulation of transcription by RNA polymerase II\n",
      "    - branching involved in ureteric bud morphogenesis\n",
      "    - in utero embryonic development\n",
      "    - cell fate determination\n",
      "    - neural tube closure\n",
      "    - heart morphogenesis\n",
      "    - smoothened signaling pathway\n",
      "    - spermatid development\n",
      "    - regulation of mitotic cell cycle\n",
      "    - brain development\n",
      "    - regulation of smoothened signaling pathway\n",
      "    - response to xenobiotic stimulus\n",
      "    - response to mechanical stimulus\n",
      "    - animal organ morphogenesis\n",
      "    - dorsal/ventral pattern formation\n",
      "    - epidermal cell fate specification\n",
      "    - response to chlorate\n",
      "    - negative regulation of keratinocyte proliferation\n",
      "    - positive regulation of cholesterol efflux\n",
      "    - protein processing\n",
      "    - spinal cord motor neuron differentiation\n",
      "    - neural tube patterning\n",
      "    - dorsal/ventral neural tube patterning\n",
      "    - neural plate axis specification\n",
      "    - embryonic limb morphogenesis\n",
      "    - prostate gland development\n",
      "    - response to estradiol\n",
      "    - response to retinoic acid\n",
      "    - regulation of protein localization\n",
      "    - limb morphogenesis\n",
      "    - hindlimb morphogenesis\n",
      "    - negative regulation of multicellular organism growth\n",
      "    - glucose homeostasis\n",
      "    - negative regulation of DNA-binding transcription factor activity\n",
      "    - keratinocyte proliferation\n",
      "    - positive regulation of epidermal cell differentiation\n",
      "    - negative regulation of osteoblast differentiation\n",
      "    - negative regulation of smoothened signaling pathway\n",
      "    - negative regulation of smoothened signaling pathway\n",
      "    - negative regulation of smoothened signaling pathway\n",
      "    - positive regulation of DNA-templated transcription\n",
      "    - embryonic organ development\n",
      "    - smooth muscle tissue development\n",
      "    - negative regulation of cell division\n",
      "    - pharyngeal system development\n",
      "    - mammary gland duct morphogenesis\n",
      "    - mammary gland epithelial cell differentiation\n",
      "    - cell differentiation involved in kidney development\n",
      "    - somite development\n",
      "    - cellular response to cholesterol\n",
      "    - cellular response to cholesterol\n",
      "    - commissural neuron axon guidance\n",
      "    - stem cell proliferation\n",
      "    - cell proliferation involved in metanephros development\n",
      "    - metanephric collecting duct development\n",
      "    - protein localization to plasma membrane\n",
      "    - liver regeneration\n",
      "    - negative regulation of stem cell proliferation\n",
      "  CC:\n",
      "    - Golgi apparatus\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - caveola\n",
      "    - midbody\n",
      "    - endocytic vesicle membrane\n",
      "    - intracellular membrane-bounded organelle\n",
      "    - dendritic growth cone\n",
      "    - axonal growth cone\n",
      "    - apical part of cell\n",
      "    - postsynaptic membrane\n",
      "    - perinuclear region of cytoplasm\n",
      "    - ciliary membrane\n",
      "  MF:\n",
      "    - patched binding\n",
      "    - smoothened binding\n",
      "    - smoothened binding\n",
      "    - protein binding\n",
      "    - hedgehog receptor activity\n",
      "    - heparin binding\n",
      "    - cholesterol binding\n",
      "    - cyclin binding\n",
      "    - protein-containing complex binding\n",
      "    - hedgehog family protein binding\n",
      "    - hedgehog family protein binding\n",
      "UniProt: {'Swiss-Prot': 'Q13635', 'TrEMBL': ['H0Y3B8', 'A0A0C4DGI4', 'F8VXL8', 'A0A0C4DGJ5', 'H3BLX7', 'F8VQS6', 'H0YHK0', 'A0A8I5KPP5', 'F8VPA3']}\n",
      "\n",
      "Gene: KCNJ16\n",
      "Symbol: KCNJ16\n",
      "Name: potassium inwardly rectifying channel subfamily J member 16\n",
      "Summary: Potassium channels are present in most mammalian cells, where they participate in a wide range of physiologic responses. The protein encoded by this gene is an integral membrane protein and inward-rectifier type potassium channel. The encoded protein, which tends to allow potassium to flow into rather than out of a cell, can form heterodimers with two other inward-rectifier type potassium channels. It may function in fluid and pH balance regulation. Alternatively spliced transcript variants have been found for this gene. [provided by RefSeq, Apr 2014].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - potassium ion transport\n",
      "    - regulation of monoatomic ion transmembrane transport\n",
      "    - potassium ion transmembrane transport\n",
      "    - potassium ion import across plasma membrane\n",
      "  CC:\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - voltage-gated potassium channel complex\n",
      "    - basolateral plasma membrane\n",
      "  MF:\n",
      "    - inward rectifier potassium channel activity\n",
      "    - inward rectifier potassium channel activity\n",
      "    - inward rectifier potassium channel activity\n",
      "UniProt: {'Swiss-Prot': 'Q9NPI9', 'TrEMBL': ['A8K434', 'Q8N538', 'K7EKJ4', 'K7ELL5']}\n",
      "\n",
      "Gene: KCNJ15\n",
      "Symbol: KCNJ15\n",
      "Name: potassium inwardly rectifying channel subfamily J member 15\n",
      "Summary: Potassium channels are present in most mammalian cells, where they participate in a wide range of physiologic responses. The protein encoded by this gene is an integral membrane protein and inward-rectifier type potassium channel. The encoded protein has a greater tendency to allow potassium to flow into a cell rather than out of a cell. Eight transcript variants encoding the same protein have been found for this gene. [provided by RefSeq, Feb 2013].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - potassium ion transport\n",
      "    - regulation of monoatomic ion transmembrane transport\n",
      "    - potassium ion import across plasma membrane\n",
      "  CC:\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - monoatomic ion channel complex\n",
      "  MF:\n",
      "    - inward rectifier potassium channel activity\n",
      "    - protein binding\n",
      "UniProt: {'Swiss-Prot': 'Q99712', 'TrEMBL': ['F8VX74', 'A8MVK1', 'E7EPG3', 'C9JTX1', 'E7EWV2', 'A8MTZ5', 'C9J0W5', 'F8VW46']}\n",
      "\n",
      "Gene: C5\n",
      "Symbol: C5\n",
      "Name: complement C5\n",
      "Summary: This gene encodes a component of the complement system, a part of the innate immune system that plays an important role in inflammation, host homeostasis, and host defense against pathogens. The encoded preproprotein is proteolytically processed to generate multiple protein products, including the C5 alpha chain, C5 beta chain, C5a anaphylatoxin and C5b. The C5 protein is comprised of the C5 alpha and beta chains, which are linked by a disulfide bridge. Cleavage of the alpha chain by a convertase enzyme results in the formation of the C5a anaphylatoxin, which possesses potent spasmogenic and chemotactic activity, and the C5b macromolecular cleavage product, a subunit of the membrane attack complex (MAC). Mutations in this gene cause complement component 5 deficiency, a disease characterized by recurrent bacterial infections. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Nov 2015].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - chemotaxis\n",
      "    - inflammatory response\n",
      "    - complement activation, alternative pathway\n",
      "    - complement activation, classical pathway\n",
      "    - cell surface receptor signaling pathway\n",
      "    - G protein-coupled receptor signaling pathway\n",
      "    - positive regulation of vascular endothelial growth factor production\n",
      "    - negative regulation of macrophage chemotaxis\n",
      "    - killing of cells of another organism\n",
      "    - positive regulation of chemokine production\n",
      "    - cell chemotaxis\n",
      "  CC:\n",
      "    - extracellular region\n",
      "    - membrane attack complex\n",
      "    - extracellular space\n",
      "    - extracellular exosome\n",
      "  MF:\n",
      "    - endopeptidase inhibitor activity\n",
      "    - signaling receptor binding\n",
      "    - protein binding\n",
      "    - chemokine activity\n",
      "UniProt: {'Swiss-Prot': 'P01031', 'TrEMBL': ['A0A8Q3SID6', 'Q59GS8', 'A0A8V8TMU5', 'A0A8V8TN26', 'A0A8Q3SIH6']}\n",
      "\n",
      "Gene: GABBR2\n",
      "Symbol: GABBR2\n",
      "Name: gamma-aminobutyric acid type B receptor subunit 2\n",
      "Summary: The multi-pass membrane protein encoded by this gene belongs to the G-protein coupled receptor 3 family and GABA-B receptor subfamily. The GABA-B receptors inhibit neuronal activity through G protein-coupled second-messenger systems, which regulate the release of neurotransmitters, and the activity of ion channels and adenylyl cyclase. This receptor subunit forms an active heterodimeric complex with GABA-B receptor subunit 1, neither of which is effective on its own. Allelic variants of this gene have been associated with nicotine dependence.[provided by RefSeq, Jan 2010].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - G protein-coupled receptor signaling pathway\n",
      "    - adenylate cyclase-inhibiting G protein-coupled receptor signaling pathway\n",
      "    - negative regulation of adenylate cyclase activity\n",
      "    - gamma-aminobutyric acid signaling pathway\n",
      "    - gamma-aminobutyric acid signaling pathway\n",
      "    - chemical synaptic transmission\n",
      "    - synaptic transmission, GABAergic\n",
      "    - neuron-glial cell signaling\n",
      "  CC:\n",
      "    - cytoplasm\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - G protein-coupled receptor heterodimeric complex\n",
      "    - G protein-coupled receptor heterodimeric complex\n",
      "    - neuron projection\n",
      "    - postsynaptic membrane\n",
      "    - GABA receptor complex\n",
      "    - G protein-coupled GABA receptor complex\n",
      "  MF:\n",
      "    - transmembrane signaling receptor activity\n",
      "    - G protein-coupled GABA receptor activity\n",
      "    - protein binding\n",
      "    - protein heterodimerization activity\n",
      "UniProt: {'Swiss-Prot': 'O75899', 'TrEMBL': ['A0A0U1RR59', 'H9NIL8', 'A0A1B0GW60']}\n",
      "\n",
      "Gene: CHRNA4\n",
      "Symbol: CHRNA4\n",
      "Name: cholinergic receptor nicotinic alpha 4 subunit\n",
      "Summary: This gene encodes a nicotinic acetylcholine receptor, which belongs to a superfamily of ligand-gated ion channels that play a role in fast signal transmission at synapses. These pentameric receptors can bind acetylcholine, which causes an extensive change in conformation that leads to the opening of an ion-conducting channel across the plasma membrane. This protein is an integral membrane receptor subunit that can interact with either nAChR beta-2 or nAChR beta-4 to form a functional receptor. Mutations in this gene cause nocturnal frontal lobe epilepsy type 1. Polymorphisms in this gene that provide protection against nicotine addiction have been described. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Feb 2012].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - action potential\n",
      "    - response to hypoxia\n",
      "    - DNA repair\n",
      "    - monoatomic ion transport\n",
      "    - calcium ion transport\n",
      "    - response to oxidative stress\n",
      "    - signal transduction\n",
      "    - chemical synaptic transmission\n",
      "    - synaptic transmission, cholinergic\n",
      "    - synaptic transmission, cholinergic\n",
      "    - synaptic transmission, cholinergic\n",
      "    - neuromuscular synaptic transmission\n",
      "    - regulation of dopamine secretion\n",
      "    - sensory perception of pain\n",
      "    - monoatomic ion transmembrane transport\n",
      "    - response to nicotine\n",
      "    - response to nicotine\n",
      "    - behavioral response to nicotine\n",
      "    - B cell activation\n",
      "    - regulation of membrane potential\n",
      "    - nervous system process\n",
      "    - cognition\n",
      "    - membrane depolarization\n",
      "    - membrane depolarization\n",
      "    - excitatory postsynaptic potential\n",
      "    - inhibitory postsynaptic potential\n",
      "    - acetylcholine receptor signaling pathway\n",
      "    - acetylcholine receptor signaling pathway\n",
      "  CC:\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - acetylcholine-gated channel complex\n",
      "    - acetylcholine-gated channel complex\n",
      "    - external side of plasma membrane\n",
      "    - membrane\n",
      "    - membrane\n",
      "    - dendrite\n",
      "    - cation channel complex\n",
      "    - presynaptic membrane\n",
      "    - neuron projection\n",
      "    - neuronal cell body\n",
      "    - synapse\n",
      "    - postsynaptic membrane\n",
      "    - neurotransmitter receptor complex\n",
      "  MF:\n",
      "    - protein binding\n",
      "    - ligand-gated monoatomic ion channel activity\n",
      "    - acetylcholine receptor activity\n",
      "    - acetylcholine-gated monoatomic cation-selective channel activity\n",
      "    - acetylcholine-gated monoatomic cation-selective channel activity\n",
      "    - acetylcholine-gated monoatomic cation-selective channel activity\n",
      "    - acetylcholine binding\n",
      "UniProt: {'Swiss-Prot': 'P43681', 'TrEMBL': ['Q59FV0', 'Q4VAQ3', 'B4DK78', 'A0A0D9SFU6', 'H0YBH1', 'A0A6Q8PG41']}\n",
      "\n",
      "Gene: CHRNA3\n",
      "Symbol: CHRNA3\n",
      "Name: cholinergic receptor nicotinic alpha 3 subunit\n",
      "Summary: This locus encodes a member of the nicotinic acetylcholine receptor family of proteins. Members of this family of proteins form pentameric complexes comprised of both alpha and beta subunits. This locus encodes an alpha-type subunit, as it contains characteristic adjacent cysteine residues. The encoded protein is a ligand-gated ion channel that likely plays a role in neurotransmission. Polymorphisms in this gene have been associated with an increased risk of smoking initiation and an increased susceptibility to lung cancer. Alternatively spliced transcript variants have been described. [provided by RefSeq, Nov 2009].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - monoatomic ion transport\n",
      "    - regulation of smooth muscle contraction\n",
      "    - signal transduction\n",
      "    - activation of transmembrane receptor protein tyrosine kinase activity\n",
      "    - synaptic transmission, cholinergic\n",
      "    - synaptic transmission, cholinergic\n",
      "    - neuromuscular synaptic transmission\n",
      "    - nervous system development\n",
      "    - locomotory behavior\n",
      "    - regulation of acetylcholine secretion, neurotransmission\n",
      "    - monoatomic ion transmembrane transport\n",
      "    - response to nicotine\n",
      "    - behavioral response to nicotine\n",
      "    - regulation of membrane potential\n",
      "    - regulation of dendrite morphogenesis\n",
      "    - membrane depolarization\n",
      "    - excitatory postsynaptic potential\n",
      "    - synaptic transmission involved in micturition\n",
      "    - acetylcholine receptor signaling pathway\n",
      "    - acetylcholine receptor signaling pathway\n",
      "    - response to acetylcholine\n",
      "  CC:\n",
      "    - nucleolus\n",
      "    - endoplasmic reticulum\n",
      "    - Golgi apparatus\n",
      "    - cytosol\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - acetylcholine-gated channel complex\n",
      "    - acetylcholine-gated channel complex\n",
      "    - postsynaptic density\n",
      "    - membrane\n",
      "    - nuclear speck\n",
      "    - dendrite\n",
      "    - ciliary basal body\n",
      "    - neuron projection\n",
      "    - neuronal cell body\n",
      "    - intracellular membrane-bounded organelle\n",
      "    - plasma membrane raft\n",
      "    - synapse\n",
      "    - postsynaptic membrane\n",
      "  MF:\n",
      "    - protein binding\n",
      "    - ligand-gated monoatomic ion channel activity\n",
      "    - acetylcholine receptor activity\n",
      "    - acetylcholine-gated monoatomic cation-selective channel activity\n",
      "    - acetylcholine-gated monoatomic cation-selective channel activity\n",
      "    - acetylcholine-gated monoatomic cation-selective channel activity\n",
      "    - acetylcholine-gated monoatomic cation-selective channel activity\n",
      "    - acetylcholine binding\n",
      "UniProt: {'Swiss-Prot': 'P32297', 'TrEMBL': 'H0YNF9'}\n",
      "\n",
      "Gene: CHRNB4\n",
      "Symbol: CHRNB4\n",
      "Name: cholinergic receptor nicotinic beta 4 subunit\n",
      "Summary: This gene is found within a conserved gene cluster and encodes one of the beta subunits of the nicotinic acetylcholine receptor (nAChRs) superfamily which form ligand-gated ion channels with a central pore that forms a cation channel. Neuronal nAChRs are pentameric structures that can be either homomeric or heteromeric, with heteromeric structures containing both alpha and beta subunits. Each subunit contains an extracellular amino terminus and four transmembrane domains. Nicotine is one of the agonists that binds to the receptor. Variants in this gene have been associated with nicotine dependence and lung cancer. Alternative splicing results in multiple transcript variants encoding different isoforms. [provided by RefSeq, Sep 2017].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - monoatomic ion transport\n",
      "    - smooth muscle contraction\n",
      "    - regulation of smooth muscle contraction\n",
      "    - signal transduction\n",
      "    - chemical synaptic transmission\n",
      "    - synaptic transmission, cholinergic\n",
      "    - locomotory behavior\n",
      "    - neuronal action potential\n",
      "    - monoatomic ion transmembrane transport\n",
      "    - behavioral response to nicotine\n",
      "    - regulation of neurotransmitter secretion\n",
      "    - membrane depolarization\n",
      "    - positive regulation of transmission of nerve impulse\n",
      "    - excitatory postsynaptic potential\n",
      "    - synaptic transmission involved in micturition\n",
      "    - acetylcholine receptor signaling pathway\n",
      "  CC:\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - acetylcholine-gated channel complex\n",
      "    - acetylcholine-gated channel complex\n",
      "    - acetylcholine-gated channel complex\n",
      "    - membrane\n",
      "    - specific granule membrane\n",
      "    - neuron projection\n",
      "    - synapse\n",
      "    - tertiary granule membrane\n",
      "    - cholinergic synapse\n",
      "    - postsynaptic specialization membrane\n",
      "  MF:\n",
      "    - protein binding\n",
      "    - ligand-gated monoatomic ion channel activity\n",
      "    - acetylcholine receptor activity\n",
      "    - acetylcholine receptor activity\n",
      "    - acetylcholine-gated monoatomic cation-selective channel activity\n",
      "    - acetylcholine-gated monoatomic cation-selective channel activity\n",
      "    - acetylcholine-gated monoatomic cation-selective channel activity\n",
      "UniProt: {'Swiss-Prot': 'P30926', 'TrEMBL': 'H3BU02'}\n",
      "\n",
      "Gene: PLK1\n",
      "Symbol: PLK1\n",
      "Name: polo like kinase 1\n",
      "Summary: The Ser/Thr protein kinase encoded by this gene belongs to the CDC5/Polo subfamily. It is highly expressed during mitosis and elevated levels are found in many different types of cancer. Depletion of this protein in cancer cells dramatically inhibited cell proliferation and induced apoptosis; hence, it is a target for cancer therapy. [provided by RefSeq, Sep 2015].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - mitotic sister chromatid segregation\n",
      "    - mitotic sister chromatid segregation\n",
      "    - G2/M transition of mitotic cell cycle\n",
      "    - G2/M transition of mitotic cell cycle\n",
      "    - negative regulation of transcription by RNA polymerase II\n",
      "    - establishment of mitotic spindle orientation\n",
      "    - mitotic cell cycle\n",
      "    - mitotic cell cycle\n",
      "    - mitotic cytokinesis\n",
      "    - mitotic cytokinesis\n",
      "    - microtubule bundle formation\n",
      "    - double-strand break repair\n",
      "    - protein phosphorylation\n",
      "    - protein phosphorylation\n",
      "    - mitotic spindle organization\n",
      "    - mitotic spindle organization\n",
      "    - sister chromatid cohesion\n",
      "    - mitotic chromosome condensation\n",
      "    - mitotic nuclear membrane disassembly\n",
      "    - metaphase/anaphase transition of mitotic cell cycle\n",
      "    - mitotic spindle assembly checkpoint signaling\n",
      "    - mitotic G2 DNA damage checkpoint signaling\n",
      "    - centrosome cycle\n",
      "    - regulation of mitotic cell cycle\n",
      "    - female meiosis chromosome segregation\n",
      "    - protein ubiquitination\n",
      "    - peptidyl-serine phosphorylation\n",
      "    - regulation of mitotic metaphase/anaphase transition\n",
      "    - regulation of mitotic metaphase/anaphase transition\n",
      "    - protein destabilization\n",
      "    - positive regulation of proteasomal ubiquitin-dependent protein catabolic process\n",
      "    - regulation of cytokinesis\n",
      "    - regulation of cytokinesis\n",
      "    - negative regulation of apoptotic process\n",
      "    - homologous chromosome segregation\n",
      "    - establishment of protein localization\n",
      "    - positive regulation of proteolysis\n",
      "    - Golgi inheritance\n",
      "    - nuclear membrane disassembly\n",
      "    - positive regulation of ubiquitin-protein transferase activity\n",
      "    - regulation of cell cycle\n",
      "    - synaptonemal complex disassembly\n",
      "    - protein localization to chromatin\n",
      "    - protein localization to nuclear envelope\n",
      "    - double-strand break repair via alternative nonhomologous end joining\n",
      "    - positive regulation of protein localization to nucleus\n",
      "    - regulation of mitotic spindle assembly\n",
      "    - regulation of mitotic cell cycle phase transition\n",
      "    - positive regulation of ubiquitin protein ligase activity\n",
      "    - regulation of protein localization to cell cortex\n",
      "    - regulation of anaphase-promoting complex-dependent catabolic process\n",
      "    - negative regulation of double-strand break repair via homologous recombination\n",
      "  CC:\n",
      "    - kinetochore\n",
      "    - kinetochore\n",
      "    - chromatin\n",
      "    - synaptonemal complex\n",
      "    - spindle pole\n",
      "    - spindle pole\n",
      "    - outer kinetochore\n",
      "    - nucleus\n",
      "    - nucleus\n",
      "    - nucleoplasm\n",
      "    - cytoplasm\n",
      "    - centrosome\n",
      "    - centrosome\n",
      "    - centriole\n",
      "    - spindle\n",
      "    - cytosol\n",
      "    - spindle microtubule\n",
      "    - microtubule cytoskeleton\n",
      "    - midbody\n",
      "    - centriolar satellite\n",
      "    - spindle midzone\n",
      "    - mitotic spindle pole\n",
      "  MF:\n",
      "    - magnesium ion binding\n",
      "    - protein kinase activity\n",
      "    - protein kinase activity\n",
      "    - protein serine/threonine kinase activity\n",
      "    - protein serine/threonine kinase activity\n",
      "    - protein serine/threonine kinase activity\n",
      "    - protein serine/threonine kinase activity\n",
      "    - protein serine/threonine kinase activity\n",
      "    - protein binding\n",
      "    - ATP binding\n",
      "    - microtubule binding\n",
      "    - anaphase-promoting complex binding\n",
      "    - protein kinase binding\n",
      "    - identical protein binding\n",
      "    - protein serine kinase activity\n",
      "UniProt: {'Swiss-Prot': 'P53350', 'TrEMBL': ['I3L387', 'I3L2H5', 'I3L309']}\n",
      "\n",
      "Gene: CTNNB1\n",
      "Symbol: CTNNB1\n",
      "Name: catenin beta 1\n",
      "Summary: The protein encoded by this gene is part of a complex of proteins that constitute adherens junctions (AJs). AJs are necessary for the creation and maintenance of epithelial cell layers by regulating cell growth and adhesion between cells. The encoded protein also anchors the actin cytoskeleton and may be responsible for transmitting the contact inhibition signal that causes cells to stop dividing once the epithelial sheet is complete. Finally, this protein binds to the product of the APC gene, which is mutated in adenomatous polyposis of the colon. Mutations in this gene are a cause of colorectal cancer (CRC), pilomatrixoma (PTR), medulloblastoma (MDB), and ovarian cancer. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Aug 2016].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - negative regulation of transcription by RNA polymerase II\n",
      "    - MAPK cascade\n",
      "    - protein polyubiquitination\n",
      "    - embryonic axis specification\n",
      "    - branching involved in blood vessel morphogenesis\n",
      "    - osteoblast differentiation\n",
      "    - branching involved in ureteric bud morphogenesis\n",
      "    - in utero embryonic development\n",
      "    - gastrulation with mouth forming second\n",
      "    - cell fate specification\n",
      "    - endodermal cell fate commitment\n",
      "    - neuron migration\n",
      "    - epithelial to mesenchymal transition\n",
      "    - neural plate development\n",
      "    - positive regulation of neuroblast proliferation\n",
      "    - positive regulation of mesenchymal cell proliferation\n",
      "    - chondrocyte differentiation\n",
      "    - lens morphogenesis in camera-type eye\n",
      "    - outflow tract morphogenesis\n",
      "    - regulation of secondary heart field cardioblast proliferation\n",
      "    - metanephros morphogenesis\n",
      "    - negative regulation of mesenchymal to epithelial transition involved in metanephros morphogenesis\n",
      "    - transcription by RNA polymerase II\n",
      "    - cell adhesion\n",
      "    - cell-matrix adhesion\n",
      "    - epidermal growth factor receptor signaling pathway\n",
      "    - chemical synaptic transmission\n",
      "    - ectoderm development\n",
      "    - glial cell fate determination\n",
      "    - neuroblast proliferation\n",
      "    - positive regulation of cell population proliferation\n",
      "    - negative regulation of cell population proliferation\n",
      "    - fibroblast growth factor receptor signaling pathway\n",
      "    - response to xenobiotic stimulus\n",
      "    - anterior/posterior axis specification\n",
      "    - dorsal/ventral axis specification\n",
      "    - proximal/distal pattern formation\n",
      "    - positive regulation of gene expression\n",
      "    - negative regulation of gene expression\n",
      "    - regulation of epithelial to mesenchymal transition\n",
      "    - positive regulation of epithelial to mesenchymal transition\n",
      "    - positive regulation of epithelial to mesenchymal transition\n",
      "    - positive regulation of heparan sulfate proteoglycan biosynthetic process\n",
      "    - negative regulation of angiogenesis\n",
      "    - stem cell population maintenance\n",
      "    - layer formation in cerebral cortex\n",
      "    - hypothalamus development\n",
      "    - central nervous system vasculogenesis\n",
      "    - osteoclast differentiation\n",
      "    - male genitalia development\n",
      "    - hindbrain development\n",
      "    - regulation of centriole-centriole cohesion\n",
      "    - pancreas development\n",
      "    - hair follicle morphogenesis\n",
      "    - regulation of protein ubiquitination\n",
      "    - regulation of myelination\n",
      "    - positive regulation of telomere maintenance via telomerase\n",
      "    - negative regulation of chondrocyte differentiation\n",
      "    - response to estradiol\n",
      "    - positive regulation of transcription elongation by RNA polymerase II\n",
      "    - T cell differentiation in thymus\n",
      "    - negative regulation of protein sumoylation\n",
      "    - positive regulation of homotypic cell-cell adhesion\n",
      "    - adherens junction assembly\n",
      "    - protein localization to cell surface\n",
      "    - embryonic heart tube development\n",
      "    - genitalia morphogenesis\n",
      "    - embryonic forelimb morphogenesis\n",
      "    - embryonic hindlimb morphogenesis\n",
      "    - hair cell differentiation\n",
      "    - detection of muscle stretch\n",
      "    - embryonic skeletal limb joint morphogenesis\n",
      "    - astrocyte-dopaminergic neuron signaling\n",
      "    - regulation of T cell proliferation\n",
      "    - odontogenesis of dentin-containing tooth\n",
      "    - embryonic digit morphogenesis\n",
      "    - positive regulation of apoptotic process\n",
      "    - negative regulation of apoptotic process\n",
      "    - negative regulation of apoptotic process\n",
      "    - proteasome-mediated ubiquitin-dependent protein catabolic process\n",
      "    - positive regulation of MAPK cascade\n",
      "    - positive regulation of neuron apoptotic process\n",
      "    - cell-cell adhesion mediated by cadherin\n",
      "    - canonical Wnt signaling pathway involved in mesenchymal stem cell differentiation\n",
      "    - bone resorption\n",
      "    - positive regulation of cell differentiation\n",
      "    - positive regulation of endothelial cell differentiation\n",
      "    - positive regulation of osteoblast differentiation\n",
      "    - negative regulation of osteoclast differentiation\n",
      "    - positive regulation of fibroblast growth factor receptor signaling pathway\n",
      "    - regulation of angiogenesis\n",
      "    - negative regulation of DNA-templated transcription\n",
      "    - positive regulation of DNA-templated transcription\n",
      "    - positive regulation of DNA-templated transcription\n",
      "    - positive regulation of transcription by RNA polymerase II\n",
      "    - positive regulation of transcription by RNA polymerase II\n",
      "    - positive regulation of transcription by RNA polymerase II\n",
      "    - positive regulation of transcription by RNA polymerase II\n",
      "    - positive regulation of transcription by RNA polymerase II\n",
      "    - negative regulation of mitotic cell cycle, embryonic\n",
      "    - regulation of fibroblast proliferation\n",
      "    - cell maturation\n",
      "    - synaptic vesicle transport\n",
      "    - thymus development\n",
      "    - oocyte development\n",
      "    - embryonic foregut morphogenesis\n",
      "    - positive regulation of skeletal muscle tissue development\n",
      "    - regulation of smooth muscle cell proliferation\n",
      "    - neuron fate determination\n",
      "    - oligodendrocyte differentiation\n",
      "    - negative regulation of oligodendrocyte differentiation\n",
      "    - regulation of neurogenesis\n",
      "    - regulation of neurogenesis\n",
      "    - synapse organization\n",
      "    - smooth muscle cell differentiation\n",
      "    - myoblast proliferation\n",
      "    - regulation of timing of anagen\n",
      "    - regulation of synapse assembly\n",
      "    - oviduct development\n",
      "    - canonical Wnt signaling pathway\n",
      "    - canonical Wnt signaling pathway\n",
      "    - canonical Wnt signaling pathway\n",
      "    - canonical Wnt signaling pathway\n",
      "    - canonical Wnt signaling pathway\n",
      "    - canonical Wnt signaling pathway\n",
      "    - mesenchymal to epithelial transition\n",
      "    - trachea formation\n",
      "    - epithelial tube branching involved in lung morphogenesis\n",
      "    - lung-associated mesenchyme development\n",
      "    - lung epithelial cell differentiation\n",
      "    - lung induction\n",
      "    - epithelial cell differentiation involved in prostate gland development\n",
      "    - epithelial cell proliferation involved in prostate gland development\n",
      "    - positive regulation of epithelial cell proliferation involved in prostate gland development\n",
      "    - hair follicle placode formation\n",
      "    - establishment of blood-brain barrier\n",
      "    - mesenchymal cell proliferation involved in lung development\n",
      "    - positive regulation of branching involved in lung morphogenesis\n",
      "    - endothelial tube morphogenesis\n",
      "    - fungiform papilla formation\n",
      "    - sympathetic ganglion development\n",
      "    - cranial ganglion development\n",
      "    - regulation of centromeric sister chromatid cohesion\n",
      "    - cellular response to growth factor stimulus\n",
      "    - cellular response to indole-3-methanol\n",
      "    - renal vesicle formation\n",
      "    - renal inner medulla development\n",
      "    - renal outer medulla development\n",
      "    - nephron tubule formation\n",
      "    - stem cell proliferation\n",
      "    - regulation of nephron tubule epithelial cell differentiation\n",
      "    - mesenchymal stem cell differentiation\n",
      "    - negative regulation of canonical Wnt signaling pathway\n",
      "    - regulation of calcium ion import\n",
      "    - acinar cell differentiation\n",
      "    - synaptic vesicle clustering\n",
      "    - apoptotic signaling pathway\n",
      "    - cell-cell adhesion\n",
      "    - positive regulation of odontoblast differentiation\n",
      "    - negative regulation of oxidative stress-induced neuron intrinsic apoptotic signaling pathway\n",
      "    - cranial skeletal system development\n",
      "    - midbrain dopaminergic neuron differentiation\n",
      "    - positive regulation of blood vessel branching\n",
      "    - neuron projection extension\n",
      "    - embryonic brain development\n",
      "    - dorsal root ganglion development\n",
      "    - establishment of blood-retinal barrier\n",
      "    - regulation of protein localization to cell surface\n",
      "    - positive regulation of determination of dorsal identity\n",
      "    - positive regulation of myoblast proliferation\n",
      "    - positive regulation of stem cell proliferation\n",
      "  CC:\n",
      "    - euchromatin\n",
      "    - spindle pole\n",
      "    - nucleus\n",
      "    - nucleus\n",
      "    - nucleoplasm\n",
      "    - transcription regulator complex\n",
      "    - cytoplasm\n",
      "    - cytoplasm\n",
      "    - centrosome\n",
      "    - cytosol\n",
      "    - cytosol\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - cell-cell junction\n",
      "    - adherens junction\n",
      "    - adherens junction\n",
      "    - adherens junction\n",
      "    - fascia adherens\n",
      "    - bicellular tight junction\n",
      "    - focal adhesion\n",
      "    - cilium\n",
      "    - cell cortex\n",
      "    - membrane\n",
      "    - basolateral plasma membrane\n",
      "    - apicolateral plasma membrane\n",
      "    - lateral plasma membrane\n",
      "    - catenin complex\n",
      "    - catenin complex\n",
      "    - flotillin complex\n",
      "    - Z disc\n",
      "    - lamellipodium\n",
      "    - cell junction\n",
      "    - cell junction\n",
      "    - beta-catenin destruction complex\n",
      "    - microvillus membrane\n",
      "    - protein-containing complex\n",
      "    - protein-DNA complex\n",
      "    - Scrib-APC-beta-catenin complex\n",
      "    - ciliary basal body\n",
      "    - presynaptic membrane\n",
      "    - intracellular membrane-bounded organelle\n",
      "    - apical part of cell\n",
      "    - synapse\n",
      "    - postsynaptic membrane\n",
      "    - perinuclear region of cytoplasm\n",
      "    - extracellular exosome\n",
      "    - beta-catenin-TCF7L2 complex\n",
      "    - cell periphery\n",
      "    - Schaffer collateral - CA1 synapse\n",
      "    - presynaptic active zone cytoplasmic component\n",
      "    - glutamatergic synapse\n",
      "    - postsynaptic density, intracellular component\n",
      "    - beta-catenin-ICAT complex\n",
      "    - beta-catenin-TCF complex\n",
      "    - beta-catenin-TCF complex\n",
      "    - Wnt signalosome\n",
      "  MF:\n",
      "    - transcription coregulator binding\n",
      "    - transcription corepressor binding\n",
      "    - chromatin binding\n",
      "    - transcription coactivator activity\n",
      "    - transcription coactivator activity\n",
      "    - transcription coactivator activity\n",
      "    - signaling receptor binding\n",
      "    - protein binding\n",
      "    - nuclear receptor binding\n",
      "    - nuclear receptor binding\n",
      "    - nuclear receptor binding\n",
      "    - enzyme binding\n",
      "    - kinase binding\n",
      "    - protein kinase binding\n",
      "    - phosphatase binding\n",
      "    - protein phosphatase binding\n",
      "    - nuclear estrogen receptor binding\n",
      "    - ubiquitin protein ligase binding\n",
      "    - transmembrane transporter binding\n",
      "    - alpha-catenin binding\n",
      "    - alpha-catenin binding\n",
      "    - cadherin binding\n",
      "    - cadherin binding\n",
      "    - cadherin binding\n",
      "    - SMAD binding\n",
      "    - RNA polymerase II-specific DNA-binding transcription factor binding\n",
      "    - RNA polymerase II-specific DNA-binding transcription factor binding\n",
      "    - I-SMAD binding\n",
      "    - disordered domain specific binding\n",
      "    - DNA-binding transcription factor binding\n",
      "    - DNA-binding transcription factor binding\n",
      "    - histone methyltransferase binding\n",
      "UniProt: {'Swiss-Prot': 'P35222', 'TrEMBL': ['A0AAQ5BIC3', 'B4DGU4', 'A0A2R8Y5A3', 'A0A2R8Y5Z1', 'A0AAQ5BIA4', 'A0A2R8Y750', 'A0A2R8Y543', 'A0A2R8Y815', 'A0A2R8Y6G0', 'A0A2R8Y7Z0', 'A0AAQ5BIB8', 'A0A2R8Y804', 'A0A2R8YCH5', 'A0AAQ5BIC2', 'A0A2R8Y5C3', 'A0AAQ5BIB9', 'A0A2R8YG06', 'A0AAQ5BIC6']}\n",
      "\n",
      "Gene: CD4\n",
      "Symbol: CD4\n",
      "Name: CD4 molecule\n",
      "Summary: This gene encodes the CD4 membrane glycoprotein of T lymphocytes. The CD4 antigen acts as a coreceptor with the T-cell receptor on the T lymphocyte to recognize antigens displayed by an antigen presenting cell in the context of class II MHC molecules. The CD4 antigen is also a primary receptor for entry of the human immunodeficiency virus through interactions with the HIV Env gp120 subunit. This gene is expressed not only in T lymphocytes, but also in B cells, macrophages, granulocytes, as well as in various regions of the brain. The protein functions to initiate or augment the early phase of T-cell activation, and may function as an important mediator of indirect neuronal damage in infectious and immune-mediated diseases of the central nervous system. Multiple alternatively spliced transcript variants encoding different isoforms have been identified in this gene. [provided by RefSeq, May 2020].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - positive regulation of protein phosphorylation\n",
      "    - adaptive immune response\n",
      "    - immune response\n",
      "    - cell adhesion\n",
      "    - signal transduction\n",
      "    - cell surface receptor signaling pathway\n",
      "    - enzyme-linked receptor protein signaling pathway\n",
      "    - cell surface receptor protein tyrosine kinase signaling pathway\n",
      "    - calcium-mediated signaling\n",
      "    - T cell differentiation\n",
      "    - macrophage differentiation\n",
      "    - maintenance of protein location in cell\n",
      "    - positive regulation of interleukin-2 production\n",
      "    - positive regulation of kinase activity\n",
      "    - helper T cell enhancement of adaptive immune response\n",
      "    - interleukin-15-mediated signaling pathway\n",
      "    - positive regulation of canonical NF-kappaB signal transduction\n",
      "    - positive regulation of MAPK cascade\n",
      "    - T cell selection\n",
      "    - positive regulation of monocyte differentiation\n",
      "    - positive regulation of protein kinase activity\n",
      "    - positive regulation of DNA-templated transcription\n",
      "    - positive regulation of viral entry into host cell\n",
      "    - symbiont entry into host cell\n",
      "    - defense response to Gram-negative bacterium\n",
      "    - positive regulation of calcium-mediated signaling\n",
      "    - regulation of T cell activation\n",
      "    - positive regulation of T cell activation\n",
      "    - regulation of calcium ion transport\n",
      "    - positive regulation of ERK1 and ERK2 cascade\n",
      "    - cellular response to granulocyte macrophage colony-stimulating factor stimulus\n",
      "  CC:\n",
      "    - early endosome\n",
      "    - endoplasmic reticulum lumen\n",
      "    - endoplasmic reticulum membrane\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - external side of plasma membrane\n",
      "    - clathrin-coated endocytic vesicle membrane\n",
      "    - T cell receptor complex\n",
      "    - membrane raft\n",
      "  MF:\n",
      "    - virus receptor activity\n",
      "    - transmembrane signaling receptor activity\n",
      "    - extracellular matrix structural constituent\n",
      "    - protein binding\n",
      "    - zinc ion binding\n",
      "    - lipid binding\n",
      "    - coreceptor activity\n",
      "    - enzyme binding\n",
      "    - protein kinase binding\n",
      "    - MHC class II protein complex binding\n",
      "    - signaling receptor activity\n",
      "    - interleukin-16 binding\n",
      "    - interleukin-16 receptor activity\n",
      "    - MHC class II protein binding\n",
      "    - MHC class II protein binding\n",
      "    - MHC class II protein binding\n",
      "    - identical protein binding\n",
      "    - protein homodimerization activity\n",
      "    - protein tyrosine kinase binding\n",
      "UniProt: {'Swiss-Prot': 'P01730', 'TrEMBL': ['B0AZV7', 'B4DT49', 'A0A4Y5UGE4', 'F5H480', 'F5GYA9']}\n",
      "\n",
      "Gene: IL2RB\n",
      "Symbol: IL2RB\n",
      "Name: interleukin 2 receptor subunit beta\n",
      "Summary: The interleukin 2 receptor, which is involved in T cell-mediated immune responses, is present in 3 forms with respect to ability to bind interleukin 2. The low affinity form is a monomer of the alpha subunit and is not involved in signal transduction. The intermediate affinity form consists of an alpha/beta subunit heterodimer, while the high affinity form consists of an alpha/beta/gamma subunit heterotrimer. Both the intermediate and high affinity forms of the receptor are involved in receptor-mediated endocytosis and transduction of mitogenic signals from interleukin 2. The protein encoded by this gene represents the beta subunit and is a type I membrane protein. The use of alternative promoters results in multiple transcript variants encoding the same protein. The protein is primarily expressed in the hematopoietic system. The use by some variants of an alternate promoter in an upstream long terminal repeat (LTR) results in placenta-specific expression. [provided by RefSeq, Sep 2016].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - signal transduction\n",
      "    - immunoglobulin mediated immune response\n",
      "    - cytokine-mediated signaling pathway\n",
      "    - cytokine-mediated signaling pathway\n",
      "    - interleukin-15-mediated signaling pathway\n",
      "    - interleukin-2-mediated signaling pathway\n",
      "    - interleukin-2-mediated signaling pathway\n",
      "    - interleukin-2-mediated signaling pathway\n",
      "    - negative regulation of apoptotic process\n",
      "    - positive regulation of phagocytosis\n",
      "    - protein-containing complex assembly\n",
      "  CC:\n",
      "    - endosome\n",
      "    - cytosol\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - interleukin-2 receptor complex\n",
      "    - external side of plasma membrane\n",
      "    - external side of plasma membrane\n",
      "    - cell surface\n",
      "    - cell surface\n",
      "    - membrane\n",
      "  MF:\n",
      "    - cytokine receptor activity\n",
      "    - interleukin-2 receptor activity\n",
      "    - interleukin-2 receptor activity\n",
      "    - interleukin-2 receptor activity\n",
      "    - protein binding\n",
      "    - coreceptor activity\n",
      "    - interleukin-2 binding\n",
      "    - interleukin-2 binding\n",
      "    - interleukin-2 binding\n",
      "    - interleukin-15 receptor activity\n",
      "UniProt: {'Swiss-Prot': 'P14784', 'TrEMBL': ['A0A8V8TNV4', 'A0A8V8TMF6', 'A0A8V8TMD8', 'A0A8V8TP62', 'A0A8V8TNU9', 'A0A8W6ANL8', 'A0A8V8TMD3', 'A0A8V8TP57', 'H7C1H9', 'A0A8V8TMF2']}\n",
      "\n",
      "Gene: JAK3\n",
      "Symbol: JAK3\n",
      "Name: Janus kinase 3\n",
      "Summary: The protein encoded by this gene is a member of the Janus kinase (JAK) family of tyrosine kinases involved in cytokine receptor-mediated intracellular signal transduction. It is predominantly expressed in immune cells and transduces a signal in response to its activation via tyrosine phosphorylation by interleukin receptors. Mutations in this gene are associated with autosomal SCID (severe combined immunodeficiency disease). [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - adaptive immune response\n",
      "    - negative regulation of dendritic cell cytokine production\n",
      "    - enzyme-linked receptor protein signaling pathway\n",
      "    - cell surface receptor signaling pathway via JAK-STAT\n",
      "    - cell surface receptor signaling pathway via JAK-STAT\n",
      "    - tyrosine phosphorylation of STAT protein\n",
      "    - cytokine-mediated signaling pathway\n",
      "    - cell differentiation\n",
      "    - B cell differentiation\n",
      "    - negative regulation of interleukin-10 production\n",
      "    - negative regulation of interleukin-12 production\n",
      "    - intracellular signal transduction\n",
      "    - intracellular signal transduction\n",
      "    - interleukin-15-mediated signaling pathway\n",
      "    - interleukin-4-mediated signaling pathway\n",
      "    - interleukin-4-mediated signaling pathway\n",
      "    - interleukin-2-mediated signaling pathway\n",
      "    - interleukin-2-mediated signaling pathway\n",
      "    - interleukin-7-mediated signaling pathway\n",
      "    - interleukin-9-mediated signaling pathway\n",
      "    - regulation of apoptotic process\n",
      "    - T cell homeostasis\n",
      "    - T cell homeostasis\n",
      "    - innate immune response\n",
      "    - negative regulation of FasL production\n",
      "    - negative regulation of T-helper 1 cell differentiation\n",
      "    - regulation of receptor signaling pathway via JAK-STAT\n",
      "    - negative regulation of T cell activation\n",
      "    - growth hormone receptor signaling pathway via JAK-STAT\n",
      "    - growth hormone receptor signaling pathway via JAK-STAT\n",
      "    - regulation of T cell apoptotic process\n",
      "    - negative regulation of thymocyte apoptotic process\n",
      "    - response to interleukin-2\n",
      "    - response to interleukin-4\n",
      "    - response to interleukin-15\n",
      "    - response to interleukin-9\n",
      "    - negative regulation of T-helper 17 cell lineage commitment\n",
      "  CC:\n",
      "    - endosome\n",
      "    - cytosol\n",
      "    - cytosol\n",
      "    - cytoskeleton\n",
      "    - plasma membrane\n",
      "    - cytoplasmic side of plasma membrane\n",
      "    - extrinsic component of plasma membrane\n",
      "    - extrinsic component of cytoplasmic side of plasma membrane\n",
      "    - extrinsic component of cytoplasmic side of plasma membrane\n",
      "  MF:\n",
      "    - protein tyrosine kinase activity\n",
      "    - protein tyrosine kinase activity\n",
      "    - protein tyrosine kinase activity\n",
      "    - protein tyrosine kinase activity\n",
      "    - non-membrane spanning protein tyrosine kinase activity\n",
      "    - growth hormone receptor binding\n",
      "    - protein binding\n",
      "    - ATP binding\n",
      "    - protein phosphatase binding\n",
      "UniProt: {'Swiss-Prot': 'P52333', 'TrEMBL': 'A0A0S2Z4R7'}\n",
      "\n",
      "Gene: IL12B\n",
      "Symbol: IL12B\n",
      "Name: interleukin 12B\n",
      "Summary: This gene encodes a subunit of interleukin 12, a cytokine that acts on T and natural killer cells, and has a broad array of biological activities. Interleukin 12 is a disulfide-linked heterodimer composed of the 40 kD cytokine receptor like subunit encoded by this gene, and a 35 kD subunit encoded by IL12A. This cytokine is expressed by activated macrophages that serve as an essential inducer of Th1 cells development. This cytokine has been found to be important for sustaining a sufficient number of memory/effector Th1 cells to mediate long-term protection to an intracellular pathogen. Overexpression of this gene was observed in the central nervous system of patients with multiple sclerosis (MS), suggesting a role of this cytokine in the pathogenesis of the disease. The promoter polymorphism of this gene has been reported to be associated with the severity of atopic and non-atopic asthma in children. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - regulation of cytokine production\n",
      "    - positive regulation of T cell mediated cytotoxicity\n",
      "    - positive regulation of defense response to virus by host\n",
      "    - positive regulation of defense response to virus by host\n",
      "    - natural killer cell activation involved in immune response\n",
      "    - positive regulation of T-helper 1 type immune response\n",
      "    - positive regulation of T-helper 1 type immune response\n",
      "    - positive regulation of natural killer cell mediated cytotoxicity directed against tumor cell target\n",
      "    - negative regulation of inflammatory response to antigenic stimulus\n",
      "    - cell surface receptor signaling pathway via JAK-STAT\n",
      "    - response to UV-B\n",
      "    - cell migration\n",
      "    - sexual reproduction\n",
      "    - natural killer cell activation\n",
      "    - negative regulation of interleukin-10 production\n",
      "    - negative regulation of interleukin-17 production\n",
      "    - positive regulation of granulocyte macrophage colony-stimulating factor production\n",
      "    - positive regulation of type II interferon production\n",
      "    - positive regulation of type II interferon production\n",
      "    - positive regulation of interleukin-10 production\n",
      "    - positive regulation of interleukin-10 production\n",
      "    - positive regulation of interleukin-12 production\n",
      "    - positive regulation of interleukin-17 production\n",
      "    - positive regulation of tumor necrosis factor production\n",
      "    - positive regulation of natural killer cell activation\n",
      "    - positive regulation of natural killer cell activation\n",
      "    - positive regulation of natural killer cell proliferation\n",
      "    - positive regulation of mononuclear cell proliferation\n",
      "    - positive regulation of tissue remodeling\n",
      "    - positive regulation of smooth muscle cell apoptotic process\n",
      "    - interleukin-12-mediated signaling pathway\n",
      "    - interleukin-12-mediated signaling pathway\n",
      "    - T-helper 1 type immune response\n",
      "    - T-helper cell differentiation\n",
      "    - T cell proliferation\n",
      "    - positive regulation of T cell proliferation\n",
      "    - positive regulation of activated T cell proliferation\n",
      "    - defense response to protozoan\n",
      "    - positive regulation of memory T cell differentiation\n",
      "    - positive regulation of osteoclast differentiation\n",
      "    - positive regulation of cell adhesion\n",
      "    - negative regulation of smooth muscle cell proliferation\n",
      "    - positive regulation of lymphocyte proliferation\n",
      "    - negative regulation of protein secretion\n",
      "    - positive regulation of inflammatory response\n",
      "    - defense response to Gram-negative bacterium\n",
      "    - positive regulation of NK T cell activation\n",
      "    - positive regulation of NK T cell activation\n",
      "    - positive regulation of NK T cell proliferation\n",
      "    - defense response to virus\n",
      "    - cellular response to lipopolysaccharide\n",
      "    - cellular response to type II interferon\n",
      "    - negative regulation of vascular endothelial growth factor signaling pathway\n",
      "    - positive regulation of non-canonical NF-kappaB signal transduction\n",
      "    - negative regulation of blood vessel endothelial cell proliferation involved in sprouting angiogenesis\n",
      "    - positive regulation of T-helper 17 type immune response\n",
      "    - positive regulation of T-helper 17 cell lineage commitment\n",
      "  CC:\n",
      "    - extracellular region\n",
      "    - extracellular space\n",
      "    - extracellular space\n",
      "    - endoplasmic reticulum lumen\n",
      "    - cytosol\n",
      "    - cell surface\n",
      "    - membrane\n",
      "    - late endosome lumen\n",
      "    - interleukin-12 complex\n",
      "    - interleukin-12 complex\n",
      "    - interleukin-23 complex\n",
      "  MF:\n",
      "    - cytokine receptor activity\n",
      "    - cytokine activity\n",
      "    - cytokine activity\n",
      "    - cytokine activity\n",
      "    - interleukin-12 receptor binding\n",
      "    - interleukin-12 receptor binding\n",
      "    - interleukin-12 receptor binding\n",
      "    - protein binding\n",
      "    - growth factor activity\n",
      "    - interleukin-12 alpha subunit binding\n",
      "    - interleukin-12 alpha subunit binding\n",
      "    - identical protein binding\n",
      "    - protein-containing complex binding\n",
      "    - interleukin-23 receptor binding\n",
      "    - protein heterodimerization activity\n",
      "UniProt: {'Swiss-Prot': 'P29460', 'TrEMBL': ['A0A8Q3SJ12', 'A0A8Q3WML5']}\n",
      "\n",
      "Gene: IL12RB1\n",
      "Symbol: IL12RB1\n",
      "Name: interleukin 12 receptor subunit beta 1\n",
      "Summary: The protein encoded by this gene is a type I transmembrane protein that belongs to the hemopoietin receptor superfamily. This protein binds to interleukine 12 (IL12) with a low affinity, and is thought to be a part of IL12 receptor complex. This protein forms a disulfide-linked oligomer, which is required for its IL12 binding activity. The coexpression of this and IL12RB2 proteins was shown to lead to the formation of high-affinity IL12 binding sites and reconstitution of IL12 dependent signaling. Mutations in this gene impair the development of interleukin-17-producing T lymphocytes and result in increased susceptibility to mycobacterial and Salmonella infections. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Feb 2014].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - positive regulation of T cell mediated cytotoxicity\n",
      "    - positive regulation of defense response to virus by host\n",
      "    - positive regulation of defense response to virus by host\n",
      "    - positive regulation of T-helper 1 type immune response\n",
      "    - positive regulation of T-helper 1 type immune response\n",
      "    - signal transduction\n",
      "    - positive regulation of cell population proliferation\n",
      "    - cytokine-mediated signaling pathway\n",
      "    - cytokine-mediated signaling pathway\n",
      "    - positive regulation of type II interferon production\n",
      "    - positive regulation of type II interferon production\n",
      "    - interleukin-12-mediated signaling pathway\n",
      "    - interleukin-23-mediated signaling pathway\n",
      "    - positive regulation of activated T cell proliferation\n",
      "    - positive regulation of memory T cell differentiation\n",
      "    - cellular response to type II interferon\n",
      "    - positive regulation of T-helper 17 type immune response\n",
      "    - positive regulation of T-helper 17 cell lineage commitment\n",
      "  CC:\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - external side of plasma membrane\n",
      "    - interleukin-12 receptor complex\n",
      "    - receptor complex\n",
      "    - interleukin-23 receptor complex\n",
      "  MF:\n",
      "    - cytokine receptor activity\n",
      "    - cytokine receptor activity\n",
      "    - interleukin-12 receptor binding\n",
      "    - coreceptor activity\n",
      "    - interleukin-12 receptor activity\n",
      "    - cytokine binding\n",
      "    - interleukin-23 binding\n",
      "    - interleukin-23 receptor activity\n",
      "UniProt: {'Swiss-Prot': 'P42701', 'TrEMBL': ['X6RGM1', 'M0QX06', 'M0R382']}\n",
      "\n",
      "Gene: NCAM1\n",
      "Symbol: NCAM1\n",
      "Name: neural cell adhesion molecule 1\n",
      "Summary: This gene encodes a cell adhesion protein which is a member of the immunoglobulin superfamily. The encoded protein is involved in cell-to-cell interactions as well as cell-matrix interactions during development and differentiation. The encoded protein plays a role in the development of the nervous system by regulating neurogenesis, neurite outgrowth, and cell migration. This protein is also involved in the expansion of T lymphocytes, B lymphocytes and natural killer (NK) cells which play an important role in immune surveillance. This protein plays a role in signal transduction by interacting with fibroblast growth factor receptors, N-cadherin and other components of the extracellular matrix and by triggering signalling cascades involving FYN-focal adhesion kinase (FAK), mitogen-activated protein kinase (MAPK), and phosphatidylinositol 3-kinase (PI3K). One prominent isoform of this gene, cell surface molecule CD56, plays a role in several myeloproliferative disorders such as acute myeloid leukemia and differential expression of this gene is associated with differential disease progression. For example, increased expression of CD56 is correlated with lower survival in acute myeloid leukemia patients whereas increased severity of COVID-19 is correlated with decreased abundance of CD56-expressing NK cells in peripheral blood. Alternative splicing results in multiple transcript variants encoding distinct protein isoforms. [provided by RefSeq, Aug 2020].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - epithelial to mesenchymal transition\n",
      "    - cell adhesion\n",
      "    - symbiont entry into host cell\n",
      "    - commissural neuron axon guidance\n",
      "    - regulation of semaphorin-plexin signaling pathway\n",
      "  CC:\n",
      "    - Golgi membrane\n",
      "    - extracellular region\n",
      "    - cytosol\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - external side of plasma membrane\n",
      "    - cell surface\n",
      "    - membrane\n",
      "    - neuron projection\n",
      "    - collagen-containing extracellular matrix\n",
      "  MF:\n",
      "    - virus receptor activity\n",
      "    - protein binding\n",
      "UniProt: {'Swiss-Prot': 'P13591', 'TrEMBL': ['A0A0C4DGS4', 'A0A087WWD4', 'A0A087WTR3', 'A0A087WWJ5', 'A0A087WVU1', 'A0A087WVD0', 'A0A087WZS4', 'A0A0D9SF98', 'A0A0D9SF30', 'A0A087X1V2', 'H7BYX6']}\n",
      "\n",
      "Gene: GRAP2\n",
      "Symbol: GRAP2\n",
      "Name: GRB2 related adaptor protein 2\n",
      "Summary: This gene encodes a member of the GRB2/Sem5/Drk family. This member is an adaptor-like protein involved in leukocyte-specific protein-tyrosine kinase signaling. Like its related family member, GRB2-related adaptor protein (GRAP), this protein contains an SH2 domain flanked by two SH3 domains. This protein interacts with other proteins, such as GRB2-associated binding protein 1 (GAB1) and the SLP-76 leukocyte protein (LCP2), through its SH3 domains. Multiple alternatively spliced transcript variants encoding distinct isoforms have been found for this gene. [provided by RefSeq, Apr 2014].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - signal transduction\n",
      "    - Ras protein signal transduction\n",
      "    - cell-cell signaling\n",
      "    - regulation of MAPK cascade\n",
      "  CC:\n",
      "    - nucleus\n",
      "    - nucleoplasm\n",
      "    - nucleoplasm\n",
      "    - cytoplasm\n",
      "    - cytoplasm\n",
      "    - endosome\n",
      "    - cytosol\n",
      "    - cytosol\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "  MF:\n",
      "    - phosphotyrosine residue binding\n",
      "    - protein binding\n",
      "UniProt: {'Swiss-Prot': 'O75791', 'TrEMBL': ['B7Z8E3', 'Q6FI14', 'B1AH86']}\n",
      "\n",
      "Gene: AGFG1\n",
      "Symbol: AGFG1\n",
      "Name: ArfGAP with FG repeats 1\n",
      "Summary: The protein encoded by this gene is related to nucleoporins, a class of proteins that mediate nucleocytoplasmic transport. The encoded protein binds the activation domain of the human immunodeficiency virus Rev protein when Rev is assembled onto its RNA target, and is required for the nuclear export of Rev-directed RNAs. Multiple transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Sep 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - acrosome assembly\n",
      "    - mRNA export from nucleus\n",
      "    - spermatid nucleus differentiation\n",
      "    - intermediate filament organization\n",
      "  CC:\n",
      "    - nuclear pore\n",
      "    - cytoplasm\n",
      "    - cytosol\n",
      "    - cytoplasmic vesicle\n",
      "    - intracellular membrane-bounded organelle\n",
      "  MF:\n",
      "    - DNA binding\n",
      "    - RNA binding\n",
      "    - GTPase activator activity\n",
      "    - protein binding\n",
      "    - zinc ion binding\n",
      "UniProt: {'Swiss-Prot': 'P52594', 'TrEMBL': ['C9J2I0', 'A0A0S2Z444', 'H7C0S7', 'B8ZZY2']}\n",
      "\n",
      "Gene: NOTCH2\n",
      "Symbol: NOTCH2\n",
      "Name: notch receptor 2\n",
      "Summary: This gene encodes a member of the Notch family. Members of this Type 1 transmembrane protein family share structural characteristics including an extracellular domain consisting of multiple epidermal growth factor-like (EGF) repeats, and an intracellular domain consisting of multiple, different domain types. Notch family members play a role in a variety of developmental processes by controlling cell fate decisions. The Notch signaling network is an evolutionarily conserved intercellular signaling pathway which regulates interactions between physically adjacent cells. In Drosophilia, notch interaction with its cell-bound ligands (delta, serrate) establishes an intercellular signaling pathway that plays a key role in development. Homologues of the notch-ligands have also been identified in human, but precise interactions between these ligands and the human notch homologues remain to be determined. This protein is cleaved in the trans-Golgi network, and presented on the cell surface as a heterodimer. This protein functions as a receptor for membrane bound ligands, and may play a role in vascular, renal and hepatic development. Two transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Jan 2011].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - negative regulation of transcription by RNA polymerase II\n",
      "    - in utero embryonic development\n",
      "    - cell fate determination\n",
      "    - heart looping\n",
      "    - morphogenesis of an epithelial sheet\n",
      "    - marginal zone B cell differentiation\n",
      "    - inflammatory response to antigenic stimulus\n",
      "    - atrioventricular node development\n",
      "    - pulmonary valve morphogenesis\n",
      "    - apoptotic process\n",
      "    - humoral immune response\n",
      "    - Notch signaling pathway\n",
      "    - Notch signaling pathway\n",
      "    - Notch signaling pathway\n",
      "    - nervous system development\n",
      "    - axon guidance\n",
      "    - animal organ morphogenesis\n",
      "    - negative regulation of gene expression\n",
      "    - positive regulation of keratinocyte proliferation\n",
      "    - hemopoiesis\n",
      "    - embryonic limb morphogenesis\n",
      "    - BMP signaling pathway\n",
      "    - positive regulation of BMP signaling pathway\n",
      "    - multicellular organism growth\n",
      "    - intracellular signal transduction\n",
      "    - intrahepatic bile duct development\n",
      "    - wound healing\n",
      "    - defense response to bacterium\n",
      "    - myeloid dendritic cell differentiation\n",
      "    - positive regulation of apoptotic process\n",
      "    - negative regulation of apoptotic process\n",
      "    - positive regulation of osteoclast differentiation\n",
      "    - positive regulation of transcription by RNA polymerase II\n",
      "    - positive regulation of Ras protein signal transduction\n",
      "    - bone remodeling\n",
      "    - positive regulation of smooth muscle cell differentiation\n",
      "    - atrial septum morphogenesis\n",
      "    - placenta blood vessel development\n",
      "    - ciliary body morphogenesis\n",
      "    - positive regulation of ERK1 and ERK2 cascade\n",
      "    - left/right axis specification\n",
      "    - cellular response to tumor cell\n",
      "    - proximal tubule development\n",
      "    - podocyte development\n",
      "    - glomerular capillary formation\n",
      "    - hepatocyte proliferation\n",
      "    - positive regulation of miRNA transcription\n",
      "    - cholangiocyte proliferation\n",
      "    - regulation of osteoclast development\n",
      "  CC:\n",
      "    - Golgi membrane\n",
      "    - extracellular region\n",
      "    - nucleus\n",
      "    - nucleoplasm\n",
      "    - nucleoplasm\n",
      "    - endoplasmic reticulum membrane\n",
      "    - Golgi apparatus\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - cilium\n",
      "    - cell surface\n",
      "    - cell surface\n",
      "    - membrane\n",
      "    - receptor complex\n",
      "    - receptor complex\n",
      "  MF:\n",
      "    - cis-regulatory region sequence-specific DNA binding\n",
      "    - DNA-binding transcription activator activity, RNA polymerase II-specific\n",
      "    - calcium ion binding\n",
      "    - protein binding\n",
      "    - enzyme binding\n",
      "    - signaling receptor activity\n",
      "    - NF-kappaB binding\n",
      "UniProt: {'Swiss-Prot': 'Q04721', 'TrEMBL': ['Q9UFD5', 'Q6IQ50', 'A0A494C1U9', 'A0A1W2PQQ5', 'A0A494C1F1', 'A0A494C1H8', 'A0A494C049']}\n",
      "\n",
      "Gene: IL18\n",
      "Symbol: IL18\n",
      "Name: interleukin 18\n",
      "Summary: The protein encoded by this gene is a proinflammatory cytokine of the IL-1 family that is constitutively found as a precursor within the cytoplasm of a variety of cells including macrophages and keratinocytes. The inactive IL-18 precursor is processed to its active form by caspase-1, and is capable of stimulating interferon gamma production, and of regulating both T helper (Th) 1 and Th2 responses. This cytokine has been implicated in the injury of different organs, and in potentially fatal conditions characterized by a cytokine storm. In humans, IL-18 gene is located on chromosome 11. Alternatively spliced transcript variants encoding different isoforms have been found for this gene.[provided by RefSeq, Aug 2020].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - angiogenesis\n",
      "    - inflammatory response\n",
      "    - immune response\n",
      "    - cell-cell signaling\n",
      "    - cell population proliferation\n",
      "    - positive regulation of macrophage derived foam cell differentiation\n",
      "    - cytokine-mediated signaling pathway\n",
      "    - natural killer cell activation\n",
      "    - regulation of cell adhesion\n",
      "    - sleep\n",
      "    - positive regulation of chemokine production\n",
      "    - positive regulation of granulocyte macrophage colony-stimulating factor production\n",
      "    - positive regulation of type II interferon production\n",
      "    - positive regulation of type II interferon production\n",
      "    - positive regulation of interleukin-13 production\n",
      "    - positive regulation of interleukin-17 production\n",
      "    - positive regulation of natural killer cell proliferation\n",
      "    - positive regulation of tissue remodeling\n",
      "    - interleukin-18-mediated signaling pathway\n",
      "    - T-helper 1 type immune response\n",
      "    - type 2 immune response\n",
      "    - positive regulation of activated T cell proliferation\n",
      "    - neutrophil activation\n",
      "    - natural killer cell mediated cytotoxicity\n",
      "    - positive regulation of tyrosine phosphorylation of STAT protein\n",
      "    - cholesterol homeostasis\n",
      "    - positive regulation of T-helper 2 cell differentiation\n",
      "    - negative regulation of myoblast differentiation\n",
      "    - positive regulation of transcription by RNA polymerase II\n",
      "    - positive regulation of smooth muscle cell proliferation\n",
      "    - positive regulation of inflammatory response\n",
      "    - positive regulation of inflammatory response\n",
      "    - defense response to Gram-positive bacterium\n",
      "    - positive regulation of NF-kappaB transcription factor activity\n",
      "    - positive regulation of NK T cell proliferation\n",
      "    - positive regulation of phosphatidylinositol 3-kinase/protein kinase B signal transduction\n",
      "    - establishment of skin barrier\n",
      "    - triglyceride homeostasis\n",
      "    - cellular response to lipopolysaccharide\n",
      "    - positive regulation of cold-induced thermogenesis\n",
      "    - positive regulation of neuroinflammatory response\n",
      "    - positive regulation of non-canonical NF-kappaB signal transduction\n",
      "    - positive regulation of T-helper 1 cell cytokine production\n",
      "  CC:\n",
      "    - extracellular region\n",
      "    - extracellular space\n",
      "    - extracellular space\n",
      "    - extracellular space\n",
      "    - cytosol\n",
      "  MF:\n",
      "    - cytokine activity\n",
      "    - cytokine activity\n",
      "    - cytokine activity\n",
      "    - cytokine activity\n",
      "    - protein binding\n",
      "    - interleukin-18 receptor binding\n",
      "UniProt: {'Swiss-Prot': 'Q14116'}\n",
      "\n",
      "Gene: IL15\n",
      "Symbol: IL15\n",
      "Name: interleukin 15\n",
      "Summary: The protein encoded by this gene is a cytokine that regulates T and natural killer cell activation and proliferation. This cytokine and interleukine 2 share many biological activities. They are found to bind common hematopoietin receptor subunits, and may compete for the same receptor, and thus negatively regulate each other's activity. The number of CD8+ memory cells is shown to be controlled by a balance between this cytokine and IL2. This cytokine induces the activation of JAK kinases, as well as the phosphorylation and activation of transcription activators STAT3, STAT5, and STAT6. Studies of the mouse counterpart suggested that this cytokine may increase the expression of apoptosis inhibitor BCL2L1/BCL-x(L), possibly through the transcription activation activity of STAT6, and thus prevent apoptosis. Alternatively spliced transcript variants of this gene have been reported. [provided by RefSeq, Feb 2011].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - natural killer cell differentiation\n",
      "    - natural killer cell proliferation\n",
      "    - positive regulation of cytokine production\n",
      "    - NK T cell proliferation\n",
      "    - immune response\n",
      "    - signal transduction\n",
      "    - tyrosine phosphorylation of STAT protein\n",
      "    - cell-cell signaling\n",
      "    - positive regulation of cell population proliferation\n",
      "    - macrophage differentiation\n",
      "    - positive regulation of interleukin-17 production\n",
      "    - positive regulation of natural killer cell proliferation\n",
      "    - positive regulation of natural killer cell differentiation\n",
      "    - positive regulation of tissue remodeling\n",
      "    - interleukin-15-mediated signaling pathway\n",
      "    - positive regulation of T cell proliferation\n",
      "    - neutrophil activation\n",
      "    - neutrophil activation\n",
      "    - extrathymic T cell selection\n",
      "    - regulation of T cell differentiation\n",
      "    - cell maturation\n",
      "    - lymph node development\n",
      "    - regulation of defense response to virus by host\n",
      "    - positive regulation of inflammatory response\n",
      "    - positive regulation of peptidyl-tyrosine phosphorylation\n",
      "    - positive regulation of phagocytosis\n",
      "    - positive regulation of immune response\n",
      "    - negative regulation of cold-induced thermogenesis\n",
      "  CC:\n",
      "    - extracellular region\n",
      "    - extracellular region\n",
      "    - extracellular space\n",
      "    - extracellular space\n",
      "    - nucleoplasm\n",
      "    - cytoplasm\n",
      "    - endosome\n",
      "    - Golgi apparatus\n",
      "    - cytosol\n",
      "    - nuclear speck\n",
      "  MF:\n",
      "    - cytokine activity\n",
      "    - cytokine activity\n",
      "    - cytokine receptor binding\n",
      "    - protein binding\n",
      "UniProt: {'Swiss-Prot': 'P40933'}\n",
      "\n",
      "Gene: IL23A\n",
      "Symbol: IL23A\n",
      "Name: interleukin 23 subunit alpha\n",
      "Summary: This gene encodes a subunit of the heterodimeric cytokine interleukin 23 (IL23). IL23 is composed of this protein and the p40 subunit of interleukin 12 (IL12B). The receptor of IL23 is formed by the beta 1 subunit of IL12 (IL12RB1) and an IL23 specific subunit, IL23R. Both IL23 and IL12 can activate the transcription activator STAT4, and stimulate the production of interferon-gamma (IFNG). In contrast to IL12, which acts mainly on naive CD4(+) T cells, IL23 preferentially acts on memory CD4(+) T cells. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - positive regulation of T cell mediated cytotoxicity\n",
      "    - positive regulation of defense response to virus by host\n",
      "    - positive regulation of defense response to virus by host\n",
      "    - positive regulation of defense response to virus by host\n",
      "    - positive regulation of T-helper 1 type immune response\n",
      "    - positive regulation of T-helper 1 type immune response\n",
      "    - positive regulation of T-helper 1 type immune response\n",
      "    - positive regulation of T-helper 1 type immune response\n",
      "    - positive regulation of T-helper 1 type immune response\n",
      "    - inflammatory response\n",
      "    - cell surface receptor signaling pathway via JAK-STAT\n",
      "    - negative regulation of interleukin-10 production\n",
      "    - positive regulation of granulocyte macrophage colony-stimulating factor production\n",
      "    - positive regulation of type II interferon production\n",
      "    - positive regulation of type II interferon production\n",
      "    - positive regulation of interleukin-10 production\n",
      "    - positive regulation of interleukin-10 production\n",
      "    - positive regulation of interleukin-12 production\n",
      "    - positive regulation of interleukin-17 production\n",
      "    - positive regulation of tumor necrosis factor production\n",
      "    - positive regulation of natural killer cell activation\n",
      "    - positive regulation of natural killer cell proliferation\n",
      "    - positive regulation of tissue remodeling\n",
      "    - T cell proliferation\n",
      "    - positive regulation of T cell proliferation\n",
      "    - positive regulation of T cell proliferation\n",
      "    - positive regulation of activated T cell proliferation\n",
      "    - positive regulation of memory T cell differentiation\n",
      "    - innate immune response\n",
      "    - positive regulation of osteoclast differentiation\n",
      "    - positive regulation of transcription by RNA polymerase II\n",
      "    - tissue remodeling\n",
      "    - positive regulation of inflammatory response\n",
      "    - defense response to Gram-negative bacterium\n",
      "    - positive regulation of NK T cell activation\n",
      "    - positive regulation of NK T cell activation\n",
      "    - positive regulation of NK T cell proliferation\n",
      "    - defense response to virus\n",
      "    - positive regulation of neutrophil chemotaxis\n",
      "    - cell surface receptor signaling pathway via STAT\n",
      "    - positive regulation of non-canonical NF-kappaB signal transduction\n",
      "    - positive regulation of T-helper 17 type immune response\n",
      "    - positive regulation of T-helper 17 cell lineage commitment\n",
      "    - positive regulation of T-helper 17 cell lineage commitment\n",
      "  CC:\n",
      "    - extracellular region\n",
      "    - endoplasmic reticulum lumen\n",
      "    - interleukin-23 complex\n",
      "    - interleukin-23 complex\n",
      "  MF:\n",
      "    - cytokine activity\n",
      "    - protein binding\n",
      "    - interleukin-23 receptor binding\n",
      "UniProt: {'Swiss-Prot': 'Q9NPF7'}\n",
      "\n",
      "Gene: F8\n",
      "Symbol: F8\n",
      "Name: coagulation factor VIII\n",
      "Summary: This gene encodes coagulation factor VIII, which participates in the intrinsic pathway of blood coagulation; factor VIII is a cofactor for factor IXa which, in the presence of Ca+2 and phospholipids, converts factor X to the activated form Xa. This gene produces two alternatively spliced transcripts. Transcript variant 1 encodes a large glycoprotein, isoform a, which circulates in plasma and associates with von Willebrand factor in a noncovalent complex. This protein undergoes multiple cleavage events. Transcript variant 2 encodes a putative small protein, isoform b, which consists primarily of the phospholipid binding domain of factor VIIIc. This binding domain is essential for coagulant activity. Defects in this gene results in hemophilia A, a common recessive X-linked coagulation disorder. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - acute-phase response\n",
      "    - blood coagulation\n",
      "    - blood coagulation, intrinsic pathway\n",
      "  CC:\n",
      "    - extracellular region\n",
      "    - extracellular region\n",
      "    - extracellular space\n",
      "    - endoplasmic reticulum lumen\n",
      "    - Golgi lumen\n",
      "    - plasma membrane\n",
      "    - COPII-coated ER to Golgi transport vesicle\n",
      "    - platelet alpha granule lumen\n",
      "    - endoplasmic reticulum-Golgi intermediate compartment membrane\n",
      "  MF:\n",
      "    - copper ion binding\n",
      "    - protein binding\n",
      "    - oxidoreductase activity\n",
      "UniProt: {'Swiss-Prot': 'P00451', 'TrEMBL': ['A0A2R8Y7J7', 'B1B0G9', 'B1B0G8', 'A0A2R8Y707']}\n",
      "\n",
      "Gene: VWF\n",
      "Symbol: VWF\n",
      "Name: von Willebrand factor\n",
      "Summary: This gene encodes a glycoprotein involved in hemostasis. The encoded preproprotein is proteolytically processed following assembly into large multimeric complexes. These complexes function in the adhesion of platelets to sites of vascular injury and the transport of various proteins in the blood. Mutations in this gene result in von Willebrand disease, an inherited bleeding disorder. An unprocessed pseudogene has been found on chromosome 22. [provided by RefSeq, Oct 2015].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - cell adhesion\n",
      "    - blood coagulation\n",
      "    - blood coagulation\n",
      "    - hemostasis\n",
      "    - response to wounding\n",
      "    - platelet activation\n",
      "    - platelet activation\n",
      "    - cell-substrate adhesion\n",
      "    - cell-substrate adhesion\n",
      "    - positive regulation of intracellular signal transduction\n",
      "  CC:\n",
      "    - extracellular region\n",
      "    - extracellular region\n",
      "    - extracellular region\n",
      "    - extracellular region\n",
      "    - extracellular space\n",
      "    - endoplasmic reticulum\n",
      "    - extracellular matrix\n",
      "    - platelet alpha granule\n",
      "    - platelet alpha granule lumen\n",
      "    - Weibel-Palade body\n",
      "    - collagen-containing extracellular matrix\n",
      "    - collagen-containing extracellular matrix\n",
      "    - extracellular exosome\n",
      "  MF:\n",
      "    - protease binding\n",
      "    - protease binding\n",
      "    - integrin binding\n",
      "    - extracellular matrix structural constituent\n",
      "    - protein binding\n",
      "    - collagen binding\n",
      "    - immunoglobulin binding\n",
      "    - identical protein binding\n",
      "    - protein-folding chaperone binding\n",
      "UniProt: {'Swiss-Prot': 'P04275', 'TrEMBL': 'I3L4K4'}\n",
      "\n",
      "Gene: GSTP1\n",
      "Symbol: GSTP1\n",
      "Name: glutathione S-transferase pi 1\n",
      "Summary: Glutathione S-transferases (GSTs) are a family of enzymes that play an important role in detoxification by catalyzing the conjugation of many hydrophobic and electrophilic compounds with reduced glutathione. Based on their biochemical, immunologic, and structural properties, the soluble GSTs are categorized into 4 main classes: alpha, mu, pi, and theta. This GST family member is a polymorphic gene encoding active, functionally different GSTP1 variant proteins that are thought to function in xenobiotic metabolism and play a role in susceptibility to cancer, and other diseases. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - response to reactive oxygen species\n",
      "    - negative regulation of acute inflammatory response\n",
      "    - negative regulation of protein kinase activity\n",
      "    - prostaglandin metabolic process\n",
      "    - glutathione metabolic process\n",
      "    - glutathione metabolic process\n",
      "    - xenobiotic metabolic process\n",
      "    - central nervous system development\n",
      "    - negative regulation of tumor necrosis factor-mediated signaling pathway\n",
      "    - negative regulation of interleukin-1 beta production\n",
      "    - negative regulation of tumor necrosis factor production\n",
      "    - regulation of stress-activated MAPK cascade\n",
      "    - negative regulation of stress-activated MAPK cascade\n",
      "    - positive regulation of superoxide anion generation\n",
      "    - common myeloid progenitor cell proliferation\n",
      "    - nitric oxide storage\n",
      "    - negative regulation of apoptotic process\n",
      "    - negative regulation of canonical NF-kappaB signal transduction\n",
      "    - negative regulation of MAP kinase activity\n",
      "    - negative regulation of MAPK cascade\n",
      "    - negative regulation of JUN kinase activity\n",
      "    - linoleic acid metabolic process\n",
      "    - negative regulation of fibroblast proliferation\n",
      "    - hepoxilin biosynthetic process\n",
      "    - regulation of ERK1 and ERK2 cascade\n",
      "    - negative regulation of ERK1 and ERK2 cascade\n",
      "    - negative regulation of leukocyte proliferation\n",
      "    - cellular response to lipopolysaccharide\n",
      "    - negative regulation of monocyte chemotactic protein-1 production\n",
      "    - cellular oxidant detoxification\n",
      "    - glutathione derivative biosynthetic process\n",
      "    - negative regulation of extrinsic apoptotic signaling pathway\n",
      "  CC:\n",
      "    - extracellular region\n",
      "    - extracellular space\n",
      "    - nucleus\n",
      "    - cytoplasm\n",
      "    - mitochondrion\n",
      "    - cytosol\n",
      "    - cytosol\n",
      "    - cytosol\n",
      "    - vesicle\n",
      "    - secretory granule lumen\n",
      "    - extracellular exosome\n",
      "    - TRAF2-GSTP1 complex\n",
      "    - ficolin-1-rich granule lumen\n",
      "  MF:\n",
      "    - glutathione transferase activity\n",
      "    - glutathione transferase activity\n",
      "    - glutathione peroxidase activity\n",
      "    - fatty acid binding\n",
      "    - protein binding\n",
      "    - JUN kinase binding\n",
      "    - kinase regulator activity\n",
      "    - S-nitrosoglutathione binding\n",
      "    - dinitrosyl-iron complex binding\n",
      "    - nitric oxide binding\n",
      "UniProt: {'Swiss-Prot': 'P09211', 'TrEMBL': ['A0A087X2E9', 'V9HWE9', 'A8MX94', 'A0A2R8Y5E5', 'A0A087X243']}\n",
      "\n",
      "Gene: SIRT1\n",
      "Symbol: SIRT1\n",
      "Name: sirtuin 1\n",
      "Summary: This gene encodes a member of the sirtuin family of proteins, homologs to the yeast Sir2 protein. Members of the sirtuin family are characterized by a sirtuin core domain and grouped into four classes. The functions of human sirtuins have not yet been determined; however, yeast sirtuin proteins are known to regulate epigenetic gene silencing and suppress recombination of rDNA. Studies suggest that the human sirtuins may function as intracellular regulatory proteins with mono-ADP-ribosyltransferase activity. The protein encoded by this gene is included in class I of the sirtuin family. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Dec 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - single strand break repair\n",
      "    - negative regulation of transcription by RNA polymerase II\n",
      "    - negative regulation of transcription by RNA polymerase II\n",
      "    - rDNA heterochromatin formation\n",
      "    - rDNA heterochromatin formation\n",
      "    - rDNA heterochromatin formation\n",
      "    - pyrimidine dimer repair by nucleotide-excision repair\n",
      "    - DNA synthesis involved in DNA repair\n",
      "    - angiogenesis\n",
      "    - ovulation from ovarian follicle\n",
      "    - intracellular glucose homeostasis\n",
      "    - positive regulation of protein phosphorylation\n",
      "    - positive regulation of endothelial cell proliferation\n",
      "    - positive regulation of adaptive immune response\n",
      "    - chromatin organization\n",
      "    - DNA methylation-dependent constitutive heterochromatin formation\n",
      "    - protein deacetylation\n",
      "    - protein deacetylation\n",
      "    - triglyceride mobilization\n",
      "    - DNA damage response\n",
      "    - response to oxidative stress\n",
      "    - transforming growth factor beta receptor signaling pathway\n",
      "    - spermatogenesis\n",
      "    - regulation of mitotic cell cycle\n",
      "    - muscle organ development\n",
      "    - positive regulation of cell population proliferation\n",
      "    - cellular response to starvation\n",
      "    - negative regulation of gene expression\n",
      "    - regulation of centrosome duplication\n",
      "    - negative regulation of triglyceride biosynthetic process\n",
      "    - positive regulation of cholesterol efflux\n",
      "    - regulation of lipid storage\n",
      "    - regulation of glucose metabolic process\n",
      "    - positive regulation of macroautophagy\n",
      "    - protein ubiquitination\n",
      "    - peptidyl-lysine acetylation\n",
      "    - macrophage differentiation\n",
      "    - negative regulation of transforming growth factor beta receptor signaling pathway\n",
      "    - negative regulation of prostaglandin biosynthetic process\n",
      "    - heterochromatin formation\n",
      "    - heterochromatin formation\n",
      "    - protein destabilization\n",
      "    - negative regulation of TOR signaling\n",
      "    - regulation of endodeoxyribonuclease activity\n",
      "    - negative regulation of NF-kappaB transcription factor activity\n",
      "    - positive regulation of proteasomal ubiquitin-dependent protein catabolic process\n",
      "    - response to insulin\n",
      "    - circadian regulation of gene expression\n",
      "    - circadian regulation of gene expression\n",
      "    - leptin-mediated signaling pathway\n",
      "    - regulation of smooth muscle cell apoptotic process\n",
      "    - negative regulation of hippo signaling\n",
      "    - intracellular triglyceride homeostasis\n",
      "    - regulation of peroxisome proliferator activated receptor signaling pathway\n",
      "    - regulation of cell population proliferation\n",
      "    - cellular response to glucose starvation\n",
      "    - negative regulation of phosphorylation\n",
      "    - response to hydrogen peroxide\n",
      "    - behavioral response to starvation\n",
      "    - cholesterol homeostasis\n",
      "    - intrinsic apoptotic signaling pathway in response to DNA damage by p53 class mediator\n",
      "    - regulation of apoptotic process\n",
      "    - positive regulation of apoptotic process\n",
      "    - positive regulation of apoptotic process\n",
      "    - negative regulation of apoptotic process\n",
      "    - negative regulation of canonical NF-kappaB signal transduction\n",
      "    - proteasome-mediated ubiquitin-dependent protein catabolic process\n",
      "    - negative regulation of DNA-binding transcription factor activity\n",
      "    - negative regulation of DNA damage response, signal transduction by p53 class mediator\n",
      "    - negative regulation of neuron apoptotic process\n",
      "    - positive regulation of blood vessel endothelial cell migration\n",
      "    - response to leptin\n",
      "    - positive regulation of MHC class II biosynthetic process\n",
      "    - negative regulation of fat cell differentiation\n",
      "    - positive regulation of gluconeogenesis\n",
      "    - positive regulation of DNA repair\n",
      "    - positive regulation of angiogenesis\n",
      "    - positive regulation of angiogenesis\n",
      "    - negative regulation of cell cycle\n",
      "    - negative regulation of DNA-templated transcription\n",
      "    - negative regulation of DNA-templated transcription\n",
      "    - negative regulation of DNA-templated transcription\n",
      "    - positive regulation of transcription by RNA polymerase II\n",
      "    - regulation of transcription by glucose\n",
      "    - positive regulation of insulin receptor signaling pathway\n",
      "    - white fat cell differentiation\n",
      "    - positive regulation of smooth muscle cell differentiation\n",
      "    - maintenance of nucleus location\n",
      "    - positive regulation of phosphatidylinositol 3-kinase/protein kinase B signal transduction\n",
      "    - negative regulation of phosphatidylinositol 3-kinase/protein kinase B signal transduction\n",
      "    - fatty acid homeostasis\n",
      "    - negative regulation of androgen receptor signaling pathway\n",
      "    - positive regulation of macrophage cytokine production\n",
      "    - cellular response to hydrogen peroxide\n",
      "    - regulation of bile acid biosynthetic process\n",
      "    - UV-damage excision repair\n",
      "    - cellular response to tumor necrosis factor\n",
      "    - cellular response to hypoxia\n",
      "    - cellular response to ionizing radiation\n",
      "    - regulation of brown fat cell differentiation\n",
      "    - stress-induced premature senescence\n",
      "    - energy homeostasis\n",
      "    - protein depropionylation\n",
      "    - DNA repair-dependent chromatin remodeling\n",
      "    - regulation of cellular response to heat\n",
      "    - negative regulation of signal transduction by p53 class mediator\n",
      "    - negative regulation of protein acetylation\n",
      "    - negative regulation of intrinsic apoptotic signaling pathway in response to DNA damage by p53 class mediator\n",
      "    - negative regulation of oxidative stress-induced intrinsic apoptotic signaling pathway\n",
      "    - positive regulation of endoplasmic reticulum stress-induced intrinsic apoptotic signaling pathway\n",
      "    - negative regulation of attachment of mitotic spindle microtubules to kinetochore\n",
      "    - positive regulation of adipose tissue development\n",
      "    - cellular response to leukemia inhibitory factor\n",
      "    - positive regulation of macrophage apoptotic process\n",
      "    - positive regulation of cAMP-dependent protein kinase activity\n",
      "    - negative regulation of cellular response to testosterone stimulus\n",
      "    - negative regulation of peptidyl-lysine acetylation\n",
      "    - negative regulation of cellular senescence\n",
      "    - negative regulation of cellular senescence\n",
      "    - positive regulation of cellular senescence\n",
      "    - positive regulation of double-strand break repair\n",
      "  CC:\n",
      "    - chromatin\n",
      "    - euchromatin\n",
      "    - heterochromatin\n",
      "    - fibrillar center\n",
      "    - nucleus\n",
      "    - nucleus\n",
      "    - nuclear envelope\n",
      "    - nuclear inner membrane\n",
      "    - nuclear inner membrane\n",
      "    - nucleoplasm\n",
      "    - nucleoplasm\n",
      "    - nucleoplasm\n",
      "    - chromatin silencing complex\n",
      "    - nucleolus\n",
      "    - nucleolus\n",
      "    - cytoplasm\n",
      "    - cytoplasm\n",
      "    - mitochondrion\n",
      "    - cytosol\n",
      "    - cytosol\n",
      "    - PML body\n",
      "    - rDNA heterochromatin\n",
      "    - rDNA heterochromatin\n",
      "    - ESC/E(Z) complex\n",
      "    - eNoSc complex\n",
      "  MF:\n",
      "    - RNA polymerase II cis-regulatory region sequence-specific DNA binding\n",
      "    - p53 binding\n",
      "    - p53 binding\n",
      "    - transcription coactivator activity\n",
      "    - transcription corepressor activity\n",
      "    - transcription corepressor activity\n",
      "    - transcription corepressor activity\n",
      "    - transcription corepressor activity\n",
      "    - NAD+ poly-ADP-ribosyltransferase activity\n",
      "    - histone deacetylase activity\n",
      "    - protein binding\n",
      "    - enzyme activator activity\n",
      "    - nuclear receptor binding\n",
      "    - histone deacetylase activity, NAD-dependent\n",
      "    - histone deacetylase activity, NAD-dependent\n",
      "    - deacetylase activity\n",
      "    - enzyme binding\n",
      "    - histone H3K14 deacetylase activity, NAD-dependent\n",
      "    - protein lysine deacetylase activity\n",
      "    - protein lysine deacetylase activity\n",
      "    - protein lysine deacetylase activity\n",
      "    - NAD-dependent protein lysine deacetylase activity\n",
      "    - NAD-dependent protein lysine deacetylase activity\n",
      "    - NAD-dependent protein lysine deacetylase activity\n",
      "    - histone binding\n",
      "    - identical protein binding\n",
      "    - HLH domain binding\n",
      "    - bHLH transcription factor binding\n",
      "    - metal ion binding\n",
      "    - histone H3K9 deacetylase activity, NAD-dependent\n",
      "    - histone H3K9 deacetylase activity, NAD-dependent\n",
      "    - histone H4K16 deacetylase activity, NAD-dependent\n",
      "    - mitogen-activated protein kinase binding\n",
      "    - NAD+ binding\n",
      "    - histone H3K18 deacetylase activity, NAD-dependent\n",
      "    - NAD-dependent protein-lysine depropionylase activity\n",
      "    - DNA-binding transcription factor binding\n",
      "    - histone H3K56 deacetylase activity, NAD-dependent\n",
      "    - histone H4K12 deacetylase activity, hydrolytic mechanism\n",
      "    - histone H3K deacetylase activity\n",
      "    - NAD-dependent protein lysine delactylase activity\n",
      "    - histone H3K4 deacetylase activity, NAD-dependent\n",
      "    - NAD-dependent histone decrotonylase activity\n",
      "    - keratin filament binding\n",
      "    - NAD+-protein mono-ADP-ribosyltransferase activity\n",
      "    - promoter-specific chromatin binding\n",
      "UniProt: {'Swiss-Prot': 'Q96EB6', 'TrEMBL': ['E9PC49', 'B0QZ35', 'A8K128']}\n",
      "\n",
      "Gene: MAP3K5\n",
      "Symbol: MAP3K5\n",
      "Name: mitogen-activated protein kinase kinase kinase 5\n",
      "Summary: Mitogen-activated protein kinase (MAPK) signaling cascades include MAPK or extracellular signal-regulated kinase (ERK), MAPK kinase (MKK or MEK), and MAPK kinase kinase (MAPKKK or MEKK). MAPKK kinase/MEKK phosphorylates and activates its downstream protein kinase, MAPK kinase/MEK, which in turn activates MAPK. The kinases of these signaling cascades are highly conserved, and homologs exist in yeast, Drosophila, and mammalian cells. MAPKKK5 contains 1,374 amino acids with all 11 kinase subdomains. Northern blot analysis shows that MAPKKK5 transcript is abundantly expressed in human heart and pancreas. The MAPKKK5 protein phosphorylates and activates MKK4 (aliases SERK1, MAPKK4) in vitro, and activates c-Jun N-terminal kinase (JNK)/stress-activated protein kinase (SAPK) during transient expression in COS and 293 cells; MAPKKK5 does not activate MAPK/ERK. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - MAPK cascade\n",
      "    - response to ischemia\n",
      "    - JNK cascade\n",
      "    - JNK cascade\n",
      "    - JNK cascade\n",
      "    - intrinsic apoptotic signaling pathway in response to oxidative stress\n",
      "    - intrinsic apoptotic signaling pathway in response to oxidative stress\n",
      "    - positive regulation of cardiac muscle cell apoptotic process\n",
      "    - cellular response to amino acid starvation\n",
      "    - response to endoplasmic reticulum stress\n",
      "    - neuron intrinsic apoptotic signaling pathway in response to oxidative stress\n",
      "    - p38MAPK cascade\n",
      "    - positive regulation of apoptotic process\n",
      "    - positive regulation of JUN kinase activity\n",
      "    - innate immune response\n",
      "    - positive regulation of myoblast differentiation\n",
      "    - positive regulation of protein kinase activity\n",
      "    - positive regulation of DNA-templated transcription\n",
      "    - positive regulation of JNK cascade\n",
      "    - neuron apoptotic process\n",
      "    - neuron apoptotic process\n",
      "    - stress-activated MAPK cascade\n",
      "    - intrinsic apoptotic signaling pathway in response to endoplasmic reticulum stress\n",
      "    - intrinsic apoptotic signaling pathway in response to endoplasmic reticulum stress\n",
      "    - cellular response to hydrogen peroxide\n",
      "    - cellular response to tumor necrosis factor\n",
      "    - endothelial cell apoptotic process\n",
      "    - cellular senescence\n",
      "    - apoptotic signaling pathway\n",
      "    - programmed necrotic cell death\n",
      "    - positive regulation of p38MAPK cascade\n",
      "    - cellular response to reactive nitrogen species\n",
      "    - positive regulation of vascular associated smooth muscle cell proliferation\n",
      "  CC:\n",
      "    - cytoplasm\n",
      "    - cytosol\n",
      "    - external side of plasma membrane\n",
      "    - protein-containing complex\n",
      "    - protein kinase complex\n",
      "    - IRE1-TRAF2-ASK1 complex\n",
      "  MF:\n",
      "    - magnesium ion binding\n",
      "    - protein kinase activity\n",
      "    - protein serine/threonine kinase activity\n",
      "    - JUN kinase kinase kinase activity\n",
      "    - MAP kinase kinase kinase activity\n",
      "    - MAP kinase kinase kinase activity\n",
      "    - protein binding\n",
      "    - ATP binding\n",
      "    - protein kinase binding\n",
      "    - protein phosphatase binding\n",
      "    - protein domain specific binding\n",
      "    - identical protein binding\n",
      "    - protein homodimerization activity\n",
      "    - protein serine kinase activity\n",
      "UniProt: {'Swiss-Prot': 'Q99683', 'TrEMBL': 'A0A8V8TMH5'}\n",
      "\n",
      "Gene: CDKN2B\n",
      "Symbol: CDKN2B\n",
      "Name: cyclin dependent kinase inhibitor 2B\n",
      "Summary: This gene lies adjacent to the tumor suppressor gene CDKN2A in a region that is frequently mutated and deleted in a wide variety of tumors. This gene encodes a cyclin-dependent kinase inhibitor, which forms a complex with CDK4 or CDK6, and prevents the activation of the CDK kinases, thus the encoded protein functions as a cell growth regulator that controls cell cycle G1 progression. The expression of this gene was found to be dramatically induced by TGF beta, which suggested its role in the TGF beta induced growth inhibition. Two alternatively spliced transcript variants of this gene, which encode distinct proteins, have been reported. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - regulation of cyclin-dependent protein serine/threonine kinase activity\n",
      "    - liver development\n",
      "    - negative regulation of cell population proliferation\n",
      "    - negative regulation of cell population proliferation\n",
      "    - megakaryocyte differentiation\n",
      "    - positive regulation of transforming growth factor beta receptor signaling pathway\n",
      "    - positive regulation of epithelial cell differentiation\n",
      "    - cellular response to nutrient\n",
      "    - response to cytokine\n",
      "    - negative regulation of phosphorylation\n",
      "    - spleen development\n",
      "    - negative regulation of epithelial cell proliferation\n",
      "    - negative regulation of glial cell proliferation\n",
      "    - regulation of G0 to G1 transition\n",
      "    - cellular response to cell-matrix adhesion\n",
      "    - cellular response to transforming growth factor beta stimulus\n",
      "    - cellular senescence\n",
      "    - regulation of G1/S transition of mitotic cell cycle\n",
      "    - negative regulation of G1/S transition of mitotic cell cycle\n",
      "  CC:\n",
      "    - nucleus\n",
      "    - nucleus\n",
      "    - cytoplasm\n",
      "    - cytoplasm\n",
      "    - cytosol\n",
      "  MF:\n",
      "    - cyclin-dependent protein serine/threonine kinase inhibitor activity\n",
      "    - cyclin-dependent protein serine/threonine kinase inhibitor activity\n",
      "    - cyclin-dependent protein serine/threonine kinase inhibitor activity\n",
      "    - protein binding\n",
      "    - protein kinase binding\n",
      "UniProt: {'Swiss-Prot': 'P42772', 'TrEMBL': 'K7PPU3'}\n",
      "\n",
      "Gene: GTSE1\n",
      "Symbol: GTSE1\n",
      "Name: G2 and S-phase expressed 1\n",
      "Summary: The protein encoded by this gene is only expressed in the S and G2 phases of the cell cycle, where it colocalizes with cytoplasmic tubulin and microtubules. In response to DNA damage, the encoded protein accumulates in the nucleus and binds the tumor suppressor protein p53, shuttling it out of the nucleus and repressing its ability to induce apoptosis. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - microtubule-based process\n",
      "    - DNA damage response, signal transduction by p53 class mediator\n",
      "  CC:\n",
      "    - nucleoplasm\n",
      "    - cytosol\n",
      "    - cytoplasmic microtubule\n",
      "    - cytoplasmic microtubule\n",
      "    - microtubule cytoskeleton\n",
      "    - membrane\n",
      "  MF:\n",
      "    - protein binding\n",
      "    - microtubule binding\n",
      "UniProt: {'Swiss-Prot': 'Q9NYZ3'}\n",
      "\n",
      "Gene: CCNE1\n",
      "Symbol: CCNE1\n",
      "Name: cyclin E1\n",
      "Summary: The protein encoded by this gene belongs to the highly conserved cyclin family, whose members are characterized by a dramatic periodicity in protein abundance through the cell cycle. Cyclins function as regulators of CDK kinases. Different cyclins exhibit distinct expression and degradation patterns which contribute to the temporal coordination of each mitotic event. This cyclin forms a complex with and functions as a regulatory subunit of CDK2, whose activity is required for cell cycle G1/S transition. This protein accumulates at the G1-S phase boundary and is degraded as cells progress through S phase. Overexpression of this gene has been observed in many tumors, which results in chromosome instability, and thus may contribute to tumorigenesis. This protein was found to associate with, and be involved in, the phosphorylation of NPAT protein (nuclear protein mapped to the ATM locus), which participates in cell-cycle regulated histone gene expression and plays a critical role in promoting cell-cycle progression in the absence of pRB. [provided by RefSeq, Apr 2016].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - G1/S transition of mitotic cell cycle\n",
      "    - G1/S transition of mitotic cell cycle\n",
      "    - G1/S transition of mitotic cell cycle\n",
      "    - negative regulation of transcription by RNA polymerase II\n",
      "    - telomere maintenance\n",
      "    - DNA replication initiation\n",
      "    - protein phosphorylation\n",
      "    - homologous chromosome pairing at meiosis\n",
      "    - Wnt signaling pathway\n",
      "    - regulation of protein localization\n",
      "    - cell division\n",
      "    - positive regulation of G1/S transition of mitotic cell cycle\n",
      "    - positive regulation of mesenchymal stem cell proliferation\n",
      "  CC:\n",
      "    - nucleus\n",
      "    - nucleus\n",
      "    - nucleoplasm\n",
      "    - nucleoplasm\n",
      "    - cytoplasm\n",
      "    - microtubule organizing center\n",
      "    - cytosol\n",
      "    - cyclin E1-CDK2 complex\n",
      "    - cyclin E1-CDK2 complex\n",
      "  MF:\n",
      "    - protein binding\n",
      "    - kinase activity\n",
      "    - cyclin-dependent protein serine/threonine kinase regulator activity\n",
      "    - protein kinase binding\n",
      "UniProt: {'Swiss-Prot': 'P24864', 'TrEMBL': ['A0A0G3DHS8', 'V5W5X2', 'C9J2U0', 'I3L1Q9', 'I3L413']}\n",
      "\n",
      "Gene: CCNA1\n",
      "Symbol: CCNA1\n",
      "Name: cyclin A1\n",
      "Summary: The protein encoded by this gene belongs to the highly conserved cyclin family, whose members are characterized by a dramatic periodicity in protein abundance through the cell cycle. Cyclins function as regulators of CDK kinases. Different cyclins exhibit distinct expression and degradation patterns which contribute to the temporal coordination of each mitotic event. The cyclin encoded by this gene was shown to be expressed in testis and brain, as well as in several leukemic cell lines, and is thought to primarily function in the control of the germline meiotic cell cycle. This cyclin binds both CDK2 and CDC2 kinases, which give two distinct kinase activities, one appearing in S phase, the other in G2, and thus regulate separate functions in cell cycle. This cyclin was found to bind to important cell cycle regulators, such as Rb family proteins, transcription factor E2F-1, and the p21 family proteins. Multiple transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - G1/S transition of mitotic cell cycle\n",
      "    - male meiosis I\n",
      "    - spermatogenesis\n",
      "    - cell division\n",
      "  CC:\n",
      "    - nucleus\n",
      "    - nucleoplasm\n",
      "    - cytoplasm\n",
      "    - microtubule organizing center\n",
      "    - cytosol\n",
      "    - microtubule cytoskeleton\n",
      "    - cyclin A1-CDK1 complex\n",
      "    - cyclin A1-CDK2 complex\n",
      "    - cyclin A2-CDK2 complex\n",
      "  MF:\n",
      "    - protein binding\n",
      "    - cyclin-dependent protein serine/threonine kinase regulator activity\n",
      "UniProt: {'Swiss-Prot': 'P78396', 'TrEMBL': 'A0A140VJG0'}\n",
      "\n",
      "Gene: CCNA2\n",
      "Symbol: CCNA2\n",
      "Name: cyclin A2\n",
      "Summary: The protein encoded by this gene belongs to the highly conserved cyclin family, whose members function as regulators of the cell cycle. This protein binds and activates cyclin-dependent kinase 2 and thus promotes transition through G1/S and G2/M. [provided by RefSeq, Aug 2016].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - G1/S transition of mitotic cell cycle\n",
      "    - G1/S transition of mitotic cell cycle\n",
      "    - G2/M transition of mitotic cell cycle\n",
      "    - G2/M transition of mitotic cell cycle\n",
      "    - regulation of DNA replication\n",
      "    - DNA-templated transcription\n",
      "    - Ras protein signal transduction\n",
      "    - animal organ regeneration\n",
      "    - response to glucagon\n",
      "    - cellular response to platelet-derived growth factor stimulus\n",
      "    - post-translational protein modification\n",
      "    - cellular response to leptin stimulus\n",
      "    - cell cycle G1/S phase transition\n",
      "    - positive regulation of DNA-templated transcription\n",
      "    - positive regulation of fibroblast proliferation\n",
      "    - cell division\n",
      "    - cellular response to cocaine\n",
      "    - cellular response to luteinizing hormone stimulus\n",
      "    - cellular response to estradiol stimulus\n",
      "    - cellular response to hypoxia\n",
      "    - cellular response to nitric oxide\n",
      "    - cochlea development\n",
      "    - cellular response to insulin-like growth factor stimulus\n",
      "    - positive regulation of DNA biosynthetic process\n",
      "  CC:\n",
      "    - cyclin-dependent protein kinase holoenzyme complex\n",
      "    - female pronucleus\n",
      "    - male pronucleus\n",
      "    - nucleus\n",
      "    - nucleus\n",
      "    - nucleoplasm\n",
      "    - nucleoplasm\n",
      "    - cytoplasm\n",
      "    - cytoplasm\n",
      "    - microtubule organizing center\n",
      "    - cytosol\n",
      "    - cyclin A2-CDK1 complex\n",
      "    - cyclin A2-CDK2 complex\n",
      "    - cyclin A2-CDK2 complex\n",
      "    - cyclin A2-CDK2 complex\n",
      "  MF:\n",
      "    - protein binding\n",
      "    - cyclin-dependent protein serine/threonine kinase regulator activity\n",
      "    - cyclin-dependent protein serine/threonine kinase regulator activity\n",
      "    - protein kinase binding\n",
      "    - protein domain specific binding\n",
      "UniProt: {'Swiss-Prot': 'P20248'}\n",
      "\n",
      "Gene: CCND3\n",
      "Symbol: CCND3\n",
      "Name: cyclin D3\n",
      "Summary: The protein encoded by this gene belongs to the highly conserved cyclin family, whose members are characterized by a dramatic periodicity in protein abundance through the cell cycle. Cyclins function as regulators of CDK kinases. Different cyclins exhibit distinct expression and degradation patterns which contribute to the temporal coordination of each mitotic event. This cyclin forms a complex with and functions as a regulatory subunit of CDK4 or CDK6, whose activtiy is required for cell cycle G1/S transition. This protein has been shown to interact with and be involved in the phosphorylation of tumor suppressor protein Rb. The CDK4 activity associated with this cyclin was reported to be necessary for cell cycle progression through G2 phase into mitosis after UV radiation. Several transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Oct 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - G1/S transition of mitotic cell cycle\n",
      "    - G1/S transition of mitotic cell cycle\n",
      "    - G1/S transition of mitotic cell cycle\n",
      "    - G1/S transition of mitotic cell cycle\n",
      "    - negative regulation of transcription by RNA polymerase II\n",
      "    - signal transduction\n",
      "    - T cell proliferation\n",
      "    - regulation of cell population proliferation\n",
      "    - cell division\n",
      "    - positive regulation of G1/S transition of mitotic cell cycle\n",
      "  CC:\n",
      "    - cyclin-dependent protein kinase holoenzyme complex\n",
      "    - cyclin-dependent protein kinase holoenzyme complex\n",
      "    - nucleus\n",
      "    - nucleoplasm\n",
      "    - nucleoplasm\n",
      "    - cytoplasm\n",
      "    - microtubule organizing center\n",
      "    - cytosol\n",
      "    - cyclin D3-CDK4 complex\n",
      "    - cyclin D3-CDK6 complex\n",
      "  MF:\n",
      "    - cyclin-dependent protein serine/threonine kinase activity\n",
      "    - protein binding\n",
      "    - cyclin-dependent protein serine/threonine kinase regulator activity\n",
      "    - protein kinase binding\n",
      "    - protein serine/threonine kinase activator activity\n",
      "    - cyclin-dependent protein serine/threonine kinase activator activity\n",
      "    - cyclin-dependent protein serine/threonine kinase activator activity\n",
      "UniProt: {'Swiss-Prot': 'P30281', 'TrEMBL': ['Q5T8J1', 'B3KP19', 'A0A1D5RMS0', 'H0Y8M4', 'D6RI00', 'D6RDF8', 'D6RIX2', 'D6RDL3']}\n",
      "\n",
      "Gene: PCSK9\n",
      "Symbol: PCSK9\n",
      "Name: proprotein convertase subtilisin/kexin type 9\n",
      "Summary: This gene encodes a member of the subtilisin-like proprotein convertase family, which includes proteases that process protein and peptide precursors trafficking through regulated or constitutive branches of the secretory pathway. The encoded protein undergoes an autocatalytic processing event with its prosegment in the ER and is constitutively secreted as an inactive protease into the extracellular matrix and trans-Golgi network. It is expressed in liver, intestine and kidney tissues and escorts specific receptors for lysosomal degradation. It plays a role in cholesterol and fatty acid metabolism. Mutations in this gene have been associated with autosomal dominant familial hypercholesterolemia. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Feb 2014].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - kidney development\n",
      "    - liver development\n",
      "    - negative regulation of receptor recycling\n",
      "    - negative regulation of receptor internalization\n",
      "    - positive regulation of receptor internalization\n",
      "    - triglyceride metabolic process\n",
      "    - phospholipid metabolic process\n",
      "    - apoptotic process\n",
      "    - lysosomal transport\n",
      "    - cholesterol metabolic process\n",
      "    - cellular response to starvation\n",
      "    - regulation of signaling receptor activity\n",
      "    - negative regulation of low-density lipoprotein particle clearance\n",
      "    - protein autoprocessing\n",
      "    - neurogenesis\n",
      "    - neuron differentiation\n",
      "    - low-density lipoprotein particle receptor catabolic process\n",
      "    - positive regulation of low-density lipoprotein particle receptor catabolic process\n",
      "    - cellular response to insulin stimulus\n",
      "    - lipoprotein metabolic process\n",
      "    - cholesterol homeostasis\n",
      "    - regulation of neuron apoptotic process\n",
      "    - positive regulation of neuron apoptotic process\n",
      "    - negative regulation of low-density lipoprotein particle receptor binding\n",
      "    - negative regulation of low-density lipoprotein receptor activity\n",
      "    - negative regulation of receptor-mediated endocytosis involved in cholesterol transport\n",
      "    - negative regulation of sodium ion transmembrane transporter activity\n",
      "  CC:\n",
      "    - extracellular region\n",
      "    - extracellular space\n",
      "    - extracellular space\n",
      "    - cytoplasm\n",
      "    - lysosome\n",
      "    - lysosomal membrane\n",
      "    - early endosome\n",
      "    - late endosome\n",
      "    - endoplasmic reticulum\n",
      "    - endoplasmic reticulum lumen\n",
      "    - Golgi apparatus\n",
      "    - plasma membrane\n",
      "    - cell surface\n",
      "    - COPII-coated ER to Golgi transport vesicle\n",
      "    - extrinsic component of external side of plasma membrane\n",
      "    - endolysosome membrane\n",
      "    - perinuclear region of cytoplasm\n",
      "    - PCSK9-LDLR complex\n",
      "    - PCSK9-AnxA2 complex\n",
      "    - PCSK9-AnxA2 complex\n",
      "  MF:\n",
      "    - RNA binding\n",
      "    - serine-type endopeptidase activity\n",
      "    - serine-type endopeptidase activity\n",
      "    - protein binding\n",
      "    - sodium channel inhibitor activity\n",
      "    - low-density lipoprotein particle binding\n",
      "    - signaling receptor inhibitor activity\n",
      "    - apolipoprotein binding\n",
      "    - very-low-density lipoprotein particle binding\n",
      "    - apolipoprotein receptor binding\n",
      "    - low-density lipoprotein particle receptor binding\n",
      "    - low-density lipoprotein particle receptor binding\n",
      "    - very-low-density lipoprotein particle receptor binding\n",
      "UniProt: {'Swiss-Prot': 'Q8NBP7', 'TrEMBL': ['A0AAQ5BGX4', 'A0A669KBG0', 'A0A669KAY4', 'A0AAQ5BGZ8', 'A0AA34QVH0', 'A0A669KB81', 'A0AAQ5BGU8']}\n",
      "\n",
      "Gene: TBK1\n",
      "Symbol: TBK1\n",
      "Name: TANK binding kinase 1\n",
      "Summary: The NF-kappa-B (NFKB) complex of proteins is inhibited by I-kappa-B (IKB) proteins, which inactivate NFKB by trapping it in the cytoplasm. Phosphorylation of serine residues on the IKB proteins by IKB kinases marks them for destruction via the ubiquitination pathway, thereby allowing activation and nuclear translocation of the NFKB complex. The protein encoded by this gene is similar to IKB kinases and can mediate NFKB activation in response to certain growth factors. The protein is also an important kinase for antiviral innate immunity response. [provided by RefSeq, Sep 2021].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - activation of innate immune response\n",
      "    - activation of innate immune response\n",
      "    - cytoplasmic pattern recognition receptor signaling pathway\n",
      "    - chromatin remodeling\n",
      "    - protein phosphorylation\n",
      "    - inflammatory response\n",
      "    - canonical NF-kappaB signal transduction\n",
      "    - response to virus\n",
      "    - positive regulation of autophagy\n",
      "    - negative regulation of gene expression\n",
      "    - macroautophagy\n",
      "    - positive regulation of macroautophagy\n",
      "    - peptidyl-serine phosphorylation\n",
      "    - peptidyl-threonine phosphorylation\n",
      "    - regulation of type I interferon production\n",
      "    - positive regulation of type I interferon production\n",
      "    - positive regulation of type I interferon production\n",
      "    - positive regulation of type I interferon production\n",
      "    - positive regulation of interferon-alpha production\n",
      "    - positive regulation of interferon-beta production\n",
      "    - toll-like receptor 4 signaling pathway\n",
      "    - positive regulation of canonical NF-kappaB signal transduction\n",
      "    - dendritic cell proliferation\n",
      "    - innate immune response\n",
      "    - innate immune response\n",
      "    - positive regulation of transcription by RNA polymerase II\n",
      "    - defense response to Gram-positive bacterium\n",
      "    - defense response to virus\n",
      "    - type I interferon-mediated signaling pathway\n",
      "    - positive regulation of type I interferon-mediated signaling pathway\n",
      "    - T follicular helper cell differentiation\n",
      "    - antiviral innate immune response\n",
      "    - cGAS/STING signaling pathway\n",
      "    - negative regulation of TORC1 signaling\n",
      "    - positive regulation of TORC1 signaling\n",
      "    - positive regulation of xenophagy\n",
      "    - positive regulation of TORC2 signaling\n",
      "  CC:\n",
      "    - nucleoplasm\n",
      "    - cytoplasm\n",
      "    - cytoplasm\n",
      "    - cytoplasm\n",
      "    - cytosol\n",
      "    - cytosol\n",
      "    - cytosol\n",
      "    - intracellular membrane-bounded organelle\n",
      "    - serine/threonine protein kinase complex\n",
      "  MF:\n",
      "    - nucleic acid binding\n",
      "    - protein kinase activity\n",
      "    - protein serine/threonine kinase activity\n",
      "    - protein serine/threonine kinase activity\n",
      "    - protein serine/threonine kinase activity\n",
      "    - protein serine/threonine kinase activity\n",
      "    - protein serine/threonine kinase activity\n",
      "    - 3-phosphoinositide-dependent protein kinase activity\n",
      "    - DNA-dependent protein kinase activity\n",
      "    - AMP-activated protein kinase activity\n",
      "    - eukaryotic translation initiation factor 2alpha kinase activity\n",
      "    - ribosomal protein S6 kinase activity\n",
      "    - protein binding\n",
      "    - ATP binding\n",
      "    - protein phosphatase binding\n",
      "    - histone H3S10 kinase activity\n",
      "    - histone H3T11 kinase activity\n",
      "    - histone H3T6 kinase activity\n",
      "    - histone H2AXS139 kinase activity\n",
      "    - identical protein binding\n",
      "    - histone H3S28 kinase activity\n",
      "    - histone H4S1 kinase activity\n",
      "    - histone H2AS1 kinase activity\n",
      "    - histone H2BS14 kinase activity\n",
      "    - phosphoprotein binding\n",
      "    - RNA polymerase II-specific DNA-binding transcription factor binding\n",
      "    - histone H3T3 kinase activity\n",
      "    - histone H2AS121 kinase activity\n",
      "    - Rho-dependent protein serine/threonine kinase activity\n",
      "    - protein serine kinase activity\n",
      "    - histone H2BS36 kinase activity\n",
      "    - histone H3S57 kinase activity\n",
      "    - histone H3T45 kinase activity\n",
      "    - histone H2AT120 kinase activity\n",
      "UniProt: {'Swiss-Prot': 'Q9UHD2', 'TrEMBL': ['A0A7I2V615', 'A0A494C167', 'A0A494C1M6', 'A0A7I2V463', 'A0A7I2V4Y1', 'A0A7I2V3Y6', 'A0A494BZY3', 'F5H1A3', 'A0A7I2YQG1', 'A0A7I2V636', 'A0A494C0R4', 'A0A7I2V3E0', 'F5H206', 'A0A7I2V5Y4', 'A0A494C079', 'A0A7I2V2V9', 'A0A7I2V3B7', 'A0A494C148', 'A0A494C045', 'A0A7I2V643', 'A0A7I2V646', 'A0A494C1I2', 'A0A7I2V3S9', 'A0A7I2V4M2', 'A0A494C0A8', 'A0A7I2V5Z0', 'A0A7I2V2N9', 'A0A7I2V4Z2', 'A0A7I2YQG4', 'A0A7I2V3P0', 'A0A7I2V2F4', 'A0A7I2V5W0', 'A0A7I2YQM3', 'A0A7I2V4S0', 'A0A7I2V322', 'F5GZI4', 'A0A7I2V2N0', 'A0A7I2V387', 'A0A7I2V4M8', 'A0A7I2V4C0', 'A0A7I2V573', 'A0A494C0X2', 'A0A7I2V4W4', 'A0A7I2V476', 'A0A7I2V502']}\n",
      "\n",
      "Gene: TNFRSF18\n",
      "Symbol: TNFRSF18\n",
      "Name: TNF receptor superfamily member 18\n",
      "Summary: This gene encodes a member of the TNF-receptor superfamily. The encoded receptor has been shown to have increased expression upon T-cell activation, and it is thought to play a key role in dominant immunological self-tolerance maintained by CD25(+)CD4(+) regulatory T cells. Knockout studies in mice also suggest the role of this receptor is in the regulation of CD3-driven T-cell activation and programmed cell death. Three alternatively spliced transcript variants of this gene encoding distinct isoforms have been reported. [provided by RefSeq, Feb 2011].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - positive regulation of leukocyte migration\n",
      "    - apoptotic process\n",
      "    - signal transduction\n",
      "    - tumor necrosis factor-mediated signaling pathway\n",
      "    - positive regulation of tyrosine phosphorylation of STAT protein\n",
      "    - negative regulation of apoptotic process\n",
      "    - positive regulation of cell adhesion\n",
      "    - positive regulation of cell adhesion\n",
      "  CC:\n",
      "    - extracellular region\n",
      "    - plasma membrane\n",
      "    - external side of plasma membrane\n",
      "  MF:\n",
      "    - tumor necrosis factor receptor activity\n",
      "    - protein binding\n",
      "UniProt: {'Swiss-Prot': 'Q9Y5U5', 'TrEMBL': ['J3KT02', 'A0A0R7FDM1']}\n",
      "\n",
      "Gene: TNFRSF4\n",
      "Symbol: TNFRSF4\n",
      "Name: TNF receptor superfamily member 4\n",
      "Summary: The protein encoded by this gene is a member of the TNF-receptor superfamily. This receptor has been shown to activate NF-kappaB through its interaction with adaptor proteins TRAF2 and TRAF5. Knockout studies in mice suggested that this receptor promotes the expression of apoptosis inhibitors BCL2 and BCL2lL1/BCL2-XL, and thus suppresses apoptosis. The knockout studies also suggested the roles of this receptor in CD4+ T cell response, as well as in T cell-dependent B cell proliferation and differentiation. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - positive regulation of immunoglobulin production\n",
      "    - inflammatory response\n",
      "    - immune response\n",
      "    - positive regulation of B cell proliferation\n",
      "    - tumor necrosis factor-mediated signaling pathway\n",
      "    - T cell proliferation\n",
      "    - negative regulation of DNA-binding transcription factor activity\n",
      "    - negative regulation of DNA-templated transcription\n",
      "    - symbiont entry into host cell\n",
      "  CC:\n",
      "    - plasma membrane\n",
      "    - external side of plasma membrane\n",
      "    - cell surface\n",
      "  MF:\n",
      "    - virus receptor activity\n",
      "    - tumor necrosis factor receptor activity\n",
      "    - protein binding\n",
      "UniProt: {'Swiss-Prot': 'P43489', 'TrEMBL': ['A0A8V8TPH3', 'A0A8V8TQV3', 'A0A8V8TQH5', 'A0A8V8TQV7', 'A0A8V8TP56', 'A0A8V8TP52', 'A0A8V8TPN6', 'A0A8V8TPH7', 'A0A8V8TPP1', 'A0A8V8TQH9']}\n",
      "\n",
      "Gene: TNFSF4\n",
      "Symbol: TNFSF4\n",
      "Name: TNF superfamily member 4\n",
      "Summary: This gene encodes a cytokine of the tumor necrosis factor (TNF) ligand family. The encoded protein functions in T cell antigen-presenting cell (APC) interactions and mediates adhesion of activated T cells to endothelial cells. Polymorphisms in this gene have been associated with Sjogren's syndrome and systemic lupus erythematosus. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Jul 2014].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - positive regulation of cytokine production\n",
      "    - defense response to nematode\n",
      "    - acute inflammatory response\n",
      "    - positive regulation of immunoglobulin production\n",
      "    - positive regulation of T cell cytokine production\n",
      "    - regulation of adaptive immune response\n",
      "    - positive regulation of type 2 immune response\n",
      "    - positive regulation of immunoglobulin mediated immune response\n",
      "    - inflammatory response\n",
      "    - immune response\n",
      "    - signal transduction\n",
      "    - positive regulation of cell population proliferation\n",
      "    - response to virus\n",
      "    - negative regulation of type II interferon production\n",
      "    - negative regulation of interleukin-17 production\n",
      "    - positive regulation of chemokine production\n",
      "    - positive regulation of type II interferon production\n",
      "    - positive regulation of interleukin-10 production\n",
      "    - positive regulation of interleukin-12 production\n",
      "    - positive regulation of interleukin-13 production\n",
      "    - positive regulation of interleukin-4 production\n",
      "    - positive regulation of interleukin-6 production\n",
      "    - memory T cell activation\n",
      "    - T-helper 2 cell activation\n",
      "    - response to nitrogen dioxide\n",
      "    - positive regulation of T cell proliferation\n",
      "    - positive regulation of CD4-positive, alpha-beta T cell differentiation\n",
      "    - positive regulation of memory T cell differentiation\n",
      "    - negative regulation of DNA-binding transcription factor activity\n",
      "    - negative regulation of regulatory T cell differentiation\n",
      "    - negative regulation of T-helper 1 cell differentiation\n",
      "    - positive regulation of T-helper 2 cell differentiation\n",
      "    - negative regulation of DNA-templated transcription\n",
      "    - positive regulation of alpha-beta T cell proliferation\n",
      "    - regulation of inflammatory response\n",
      "    - positive regulation of inflammatory response\n",
      "    - positive regulation of B cell activation\n",
      "    - cellular response to lipopolysaccharide\n",
      "    - cellular response to prostaglandin E stimulus\n",
      "    - positive regulation of CD4-positive, alpha-beta T cell costimulation\n",
      "    - positive regulation of interleukin-4-dependent isotype switching to IgE isotypes\n",
      "  CC:\n",
      "    - extracellular space\n",
      "    - extracellular space\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - cell surface\n",
      "  MF:\n",
      "    - signaling receptor binding\n",
      "    - cytokine activity\n",
      "    - tumor necrosis factor receptor binding\n",
      "    - protein binding\n",
      "    - tumor necrosis factor receptor superfamily binding\n",
      "    - tumor necrosis factor receptor superfamily binding\n",
      "UniProt: {'Swiss-Prot': 'P23510', 'TrEMBL': ['A0AAQ5BI34', 'A0A024R937']}\n",
      "\n",
      "Gene: TNFRSF8\n",
      "Symbol: TNFRSF8\n",
      "Name: TNF receptor superfamily member 8\n",
      "Summary: The protein encoded by this gene is a member of the TNF-receptor superfamily. This receptor is expressed by activated, but not by resting, T and B cells. TRAF2 and TRAF5 can interact with this receptor, and mediate the signal transduction that leads to the activation of NF-kappaB. This receptor is a positive regulator of apoptosis, and also has been shown to limit the proliferative potential of autoreactive CD8 effector T cells and protect the body against autoimmunity. Two alternatively spliced transcript variants of this gene encoding distinct isoforms have been reported. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - signal transduction\n",
      "    - negative regulation of cell population proliferation\n",
      "    - positive regulation of TRAIL production\n",
      "    - positive regulation of tumor necrosis factor production\n",
      "    - positive regulation of apoptotic process\n",
      "    - cellular response to mechanical stimulus\n",
      "  CC:\n",
      "    - nucleoplasm\n",
      "    - cytoplasm\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - extracellular exosome\n",
      "  MF:\n",
      "    - protease binding\n",
      "    - transmembrane signaling receptor activity\n",
      "UniProt: {'Swiss-Prot': 'P28908', 'TrEMBL': ['D6RAG8', 'A5D8T4']}\n",
      "\n",
      "Gene: TNFRSF9\n",
      "Symbol: TNFRSF9\n",
      "Name: TNF receptor superfamily member 9\n",
      "Summary: The protein encoded by this gene is a member of the TNF-receptor superfamily. This receptor contributes to the clonal expansion, survival, and development of T cells. It can also induce proliferation in peripheral monocytes, enhance T cell apoptosis induced by TCR/CD3 triggered activation, and regulate CD28 co-stimulation to promote Th1 cell responses. The expression of this receptor is induced by lymphocyte activation. TRAF adaptor proteins have been shown to bind to this receptor and transduce the signals leading to activation of NF-kappaB. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - apoptotic process\n",
      "    - negative regulation of cell population proliferation\n",
      "    - regulation of immature T cell proliferation in thymus\n",
      "    - regulation of cell population proliferation\n",
      "  CC:\n",
      "    - plasma membrane\n",
      "    - external side of plasma membrane\n",
      "  MF:\n",
      "    - protein binding\n",
      "    - signaling receptor activity\n",
      "UniProt: {'Swiss-Prot': 'Q07011', 'TrEMBL': ['K7EJ11', 'A0A6I8PRR4', 'K7EJQ2']}\n",
      "\n",
      "Gene: AKT1\n",
      "Symbol: AKT1\n",
      "Name: AKT serine/threonine kinase 1\n",
      "Summary: This gene encodes one of the three members of the human AKT serine-threonine protein kinase family which are often referred to as protein kinase B alpha, beta, and gamma. These highly similar AKT proteins all have an N-terminal pleckstrin homology domain, a serine/threonine-specific kinase domain and a C-terminal regulatory domain. These proteins are phosphorylated by phosphoinositide 3-kinase (PI3K). AKT/PI3K forms a key component of many signalling pathways that involve the binding of membrane-bound ligands such as receptor tyrosine kinases, G-protein coupled receptors, and integrin-linked kinase. These AKT proteins therefore regulate a wide variety of cellular functions including cell proliferation, survival, metabolism, and angiogenesis in both normal and malignant cells. AKT proteins are recruited to the cell membrane by phosphatidylinositol 3,4,5-trisphosphate (PIP3) after phosphorylation of phosphatidylinositol 4,5-bisphosphate (PIP2) by PI3K. Subsequent phosphorylation of both threonine residue 308 and serine residue 473 is required for full activation of the AKT1 protein encoded by this gene. Phosphorylation of additional residues also occurs, for example, in response to insulin growth factor-1 and epidermal growth factor. Protein phosphatases act as negative regulators of AKT proteins by dephosphorylating AKT or PIP3. The PI3K/AKT signalling pathway is crucial for tumor cell survival. Survival factors can suppress apoptosis in a transcription-independent manner by activating AKT1 which then phosphorylates and inactivates components of the apoptotic machinery. AKT proteins also participate in the mammalian target of rapamycin (mTOR) signalling pathway which controls the assembly of the eukaryotic translation initiation factor 4F (eIF4E) complex and this pathway, in addition to responding to extracellular signals from growth factors and cytokines, is disregulated in many cancers. Mutations in this gene are associated with multiple types of cancer and excessive tissue growth including Proteus syndrome and Cowden syndrome 6, and breast, colorectal, and ovarian cancers. Multiple alternatively spliced transcript variants have been found for this gene. [provided by RefSeq, Jul 2020].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - osteoblast differentiation\n",
      "    - maternal placenta development\n",
      "    - positive regulation of endothelial cell proliferation\n",
      "    - cell migration involved in sprouting angiogenesis\n",
      "    - sphingosine-1-phosphate receptor signaling pathway\n",
      "    - glycogen biosynthetic process\n",
      "    - regulation of glycogen biosynthetic process\n",
      "    - glucose metabolic process\n",
      "    - chromatin remodeling\n",
      "    - regulation of translation\n",
      "    - protein phosphorylation\n",
      "    - protein phosphorylation\n",
      "    - protein import into nucleus\n",
      "    - nitric oxide biosynthetic process\n",
      "    - activation-induced cell death of T cells\n",
      "    - inflammatory response\n",
      "    - response to oxidative stress\n",
      "    - signal transduction\n",
      "    - epidermal growth factor receptor signaling pathway\n",
      "    - epidermal growth factor receptor signaling pathway\n",
      "    - G protein-coupled receptor signaling pathway\n",
      "    - cell population proliferation\n",
      "    - insulin receptor signaling pathway\n",
      "    - insulin receptor signaling pathway\n",
      "    - apoptotic mitochondrial changes\n",
      "    - response to heat\n",
      "    - gene expression\n",
      "    - negative regulation of autophagy\n",
      "    - positive regulation of endothelial cell migration\n",
      "    - positive regulation of gene expression\n",
      "    - positive regulation of gene expression\n",
      "    - negative regulation of long-chain fatty acid import across plasma membrane\n",
      "    - fibroblast migration\n",
      "    - positive regulation of fibroblast migration\n",
      "    - positive regulation of sodium ion transport\n",
      "    - positive regulation of glucose metabolic process\n",
      "    - regulation of neuron projection development\n",
      "    - negative regulation of macroautophagy\n",
      "    - phosphorylation\n",
      "    - protein ubiquitination\n",
      "    - peptidyl-serine phosphorylation\n",
      "    - peptidyl-threonine phosphorylation\n",
      "    - cytokine-mediated signaling pathway\n",
      "    - mammalian oogenesis stage\n",
      "    - cell differentiation\n",
      "    - positive regulation of cell growth\n",
      "    - regulation of cell migration\n",
      "    - regulation of cell migration\n",
      "    - positive regulation of cell migration\n",
      "    - positive regulation of cell migration\n",
      "    - T cell costimulation\n",
      "    - negative regulation of protein ubiquitination\n",
      "    - regulation of myelination\n",
      "    - lipopolysaccharide-mediated signaling pathway\n",
      "    - TOR signaling\n",
      "    - negative regulation of fatty acid beta-oxidation\n",
      "    - positive regulation of endodeoxyribonuclease activity\n",
      "    - response to food\n",
      "    - peripheral nervous system myelin maintenance\n",
      "    - positive regulation of proteasomal ubiquitin-dependent protein catabolic process\n",
      "    - cellular response to insulin stimulus\n",
      "    - cellular response to insulin stimulus\n",
      "    - cellular response to insulin stimulus\n",
      "    - positive regulation of peptidyl-serine phosphorylation\n",
      "    - cellular response to stress\n",
      "    - response to fluid shear stress\n",
      "    - intracellular signal transduction\n",
      "    - intracellular signal transduction\n",
      "    - interleukin-18-mediated signaling pathway\n",
      "    - cellular response to vascular endothelial growth factor stimulus\n",
      "    - cellular response to decreased oxygen levels\n",
      "    - glucose homeostasis\n",
      "    - regulation of apoptotic process\n",
      "    - negative regulation of apoptotic process\n",
      "    - proteasome-mediated ubiquitin-dependent protein catabolic process\n",
      "    - anoikis\n",
      "    - regulation of mRNA stability\n",
      "    - phosphatidylinositol 3-kinase/protein kinase B signal transduction\n",
      "    - phosphatidylinositol 3-kinase/protein kinase B signal transduction\n",
      "    - phosphatidylinositol 3-kinase/protein kinase B signal transduction\n",
      "    - positive regulation of blood vessel endothelial cell migration\n",
      "    - positive regulation of nitric oxide biosynthetic process\n",
      "    - positive regulation of fat cell differentiation\n",
      "    - positive regulation of glycogen biosynthetic process\n",
      "    - positive regulation of glycogen biosynthetic process\n",
      "    - negative regulation of Notch signaling pathway\n",
      "    - negative regulation of proteolysis\n",
      "    - positive regulation of transcription by RNA polymerase II\n",
      "    - nitric oxide metabolic process\n",
      "    - positive regulation of D-glucose import\n",
      "    - positive regulation of organ growth\n",
      "    - protein autophosphorylation\n",
      "    - positive regulation of lipid biosynthetic process\n",
      "    - positive regulation of lipid biosynthetic process\n",
      "    - insulin-like growth factor receptor signaling pathway\n",
      "    - behavioral response to pain\n",
      "    - positive regulation of smooth muscle cell proliferation\n",
      "    - striated muscle cell differentiation\n",
      "    - positive regulation of protein metabolic process\n",
      "    - excitatory postsynaptic potential\n",
      "    - response to growth hormone\n",
      "    - mammary gland epithelial cell differentiation\n",
      "    - labyrinthine layer blood vessel development\n",
      "    - response to UV-A\n",
      "    - response to growth factor\n",
      "    - cellular response to tumor necrosis factor\n",
      "    - cellular response to epidermal growth factor stimulus\n",
      "    - cellular response to prostaglandin E stimulus\n",
      "    - establishment of protein localization to mitochondrion\n",
      "    - maintenance of protein location in mitochondrion\n",
      "    - cellular response to rapamycin\n",
      "    - negative regulation of release of cytochrome c from mitochondria\n",
      "    - cellular response to granulocyte macrophage colony-stimulating factor stimulus\n",
      "    - execution phase of apoptosis\n",
      "    - vascular endothelial cell response to laminar fluid shear stress\n",
      "    - regulation of postsynapse organization\n",
      "    - regulation of tRNA methylation\n",
      "    - cellular response to oxidised low-density lipoprotein particle stimulus\n",
      "    - negative regulation of protein localization to lysosome\n",
      "    - negative regulation of cGAS/STING signaling pathway\n",
      "    - positive regulation of G1/S transition of mitotic cell cycle\n",
      "    - positive regulation of protein localization to nucleus\n",
      "    - cellular response to peptide\n",
      "    - regulation of signal transduction by p53 class mediator\n",
      "    - negative regulation of cilium assembly\n",
      "    - negative regulation of oxidative stress-induced intrinsic apoptotic signaling pathway\n",
      "    - negative regulation of leukocyte cell-cell adhesion\n",
      "    - positive regulation of protein localization to plasma membrane\n",
      "    - negative regulation of protein maturation\n",
      "    - negative regulation of hydrogen peroxide-induced neuron intrinsic apoptotic signaling pathway\n",
      "    - negative regulation of PERK-mediated unfolded protein response\n",
      "    - positive regulation of TORC1 signaling\n",
      "    - positive regulation of TORC2 signaling\n",
      "    - positive regulation of protein localization to endoplasmic reticulum\n",
      "    - cellular response to nerve growth factor stimulus\n",
      "    - response to insulin-like growth factor stimulus\n",
      "    - positive regulation of protein localization to cell surface\n",
      "    - regulation of type B pancreatic cell development\n",
      "    - negative regulation of lymphocyte migration\n",
      "    - negative regulation of extrinsic apoptotic signaling pathway in absence of ligand\n",
      "  CC:\n",
      "    - nucleus\n",
      "    - nucleus\n",
      "    - nucleoplasm\n",
      "    - nucleoplasm\n",
      "    - cytoplasm\n",
      "    - cytoplasm\n",
      "    - cytoplasm\n",
      "    - cytoplasm\n",
      "    - mitochondrion\n",
      "    - mitochondrial intermembrane space\n",
      "    - spindle\n",
      "    - cytosol\n",
      "    - cytosol\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - cell-cell junction\n",
      "    - cilium\n",
      "    - cell cortex\n",
      "    - microtubule cytoskeleton\n",
      "    - membrane\n",
      "    - lamellipodium\n",
      "    - vesicle\n",
      "    - protein-containing complex\n",
      "    - ciliary basal body\n",
      "    - postsynapse\n",
      "    - glutamatergic synapse\n",
      "  MF:\n",
      "    - protein kinase activity\n",
      "    - protein serine/threonine kinase activity\n",
      "    - protein serine/threonine kinase activity\n",
      "    - protein serine/threonine kinase activity\n",
      "    - protein serine/threonine kinase activity\n",
      "    - protein serine/threonine kinase activity\n",
      "    - protein serine/threonine kinase activity\n",
      "    - protein serine/threonine kinase activity\n",
      "    - 3-phosphoinositide-dependent protein kinase activity\n",
      "    - DNA-dependent protein kinase activity\n",
      "    - AMP-activated protein kinase activity\n",
      "    - eukaryotic translation initiation factor 2alpha kinase activity\n",
      "    - ribosomal protein S6 kinase activity\n",
      "    - protein serine/threonine/tyrosine kinase activity\n",
      "    - protein binding\n",
      "    - calmodulin binding\n",
      "    - ATP binding\n",
      "    - ATP binding\n",
      "    - phosphatidylinositol-3,4,5-trisphosphate binding\n",
      "    - kinase activity\n",
      "    - enzyme binding\n",
      "    - kinase binding\n",
      "    - protein kinase binding\n",
      "    - nitric-oxide synthase regulator activity\n",
      "    - protein serine/threonine kinase inhibitor activity\n",
      "    - protein serine/threonine kinase inhibitor activity\n",
      "    - histone H3S10 kinase activity\n",
      "    - histone H3T11 kinase activity\n",
      "    - histone H3T6 kinase activity\n",
      "    - histone H2AXS139 kinase activity\n",
      "    - identical protein binding\n",
      "    - protein homodimerization activity\n",
      "    - phosphatidylinositol-3,4-bisphosphate binding\n",
      "    - histone H3S28 kinase activity\n",
      "    - histone H4S1 kinase activity\n",
      "    - histone H2AS1 kinase activity\n",
      "    - histone H2BS14 kinase activity\n",
      "    - 14-3-3 protein binding\n",
      "    - histone H3T3 kinase activity\n",
      "    - histone H2AS121 kinase activity\n",
      "    - Rho-dependent protein serine/threonine kinase activity\n",
      "    - potassium channel activator activity\n",
      "    - protein serine kinase activity\n",
      "    - histone H2BS36 kinase activity\n",
      "    - histone H3S57 kinase activity\n",
      "    - histone H3T45 kinase activity\n",
      "    - TORC2 complex binding\n",
      "    - histone H2AT120 kinase activity\n",
      "UniProt: {'Swiss-Prot': 'P31749', 'TrEMBL': ['B3KVH4', 'B0LPE5', 'G3V3X1', 'A0AAQ5BHK3', 'A0A804HJM6', 'A0A087WY56', 'G3V2I6', 'A0AAQ5BHJ3']}\n",
      "\n",
      "Gene: JAM3\n",
      "Symbol: JAM3\n",
      "Name: junctional adhesion molecule 3\n",
      "Summary: Tight junctions represent one mode of cell-to-cell adhesion in epithelial or endothelial cell sheets, forming continuous seals around cells and serving as a physical barrier to prevent solutes and water from passing freely through the paracellular space. The protein encoded by this immunoglobulin superfamily gene member is localized in the tight junctions between high endothelial cells. Unlike other proteins in this family, the this protein is unable to adhere to leukocyte cell lines and only forms weak homotypic interactions. The encoded protein is a member of the junctional adhesion molecule protein family and acts as a receptor for another member of this family. A mutation in an intron of this gene is associated with hemorrhagic destruction of the brain, subependymal calcification, and congenital cataracts. Alternative splicing results in multiple transcript variants.[provided by RefSeq, Apr 2011].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - angiogenesis\n",
      "    - neutrophil homeostasis\n",
      "    - adaptive immune response\n",
      "    - myeloid progenitor cell differentiation\n",
      "    - leukocyte migration involved in inflammatory response\n",
      "    - cell-matrix adhesion\n",
      "    - spermatid development\n",
      "    - cell migration\n",
      "    - transmission of nerve impulse\n",
      "    - establishment of cell polarity\n",
      "    - negative regulation of integrin activation\n",
      "    - negative regulation of cell adhesion mediated by integrin\n",
      "    - heterotypic cell-cell adhesion\n",
      "    - adherens junction assembly\n",
      "    - protein localization to cell surface\n",
      "    - maintenance of blood-brain barrier\n",
      "    - myelination\n",
      "    - apical protein localization\n",
      "    - regulation of neutrophil chemotaxis\n",
      "    - regulation of actin cytoskeleton organization by cell-cell adhesion\n",
      "    - hematopoietic stem cell migration to bone marrow\n",
      "    - granulocyte migration\n",
      "    - cell-cell adhesion\n",
      "    - cell-cell adhesion\n",
      "    - cell-cell adhesion\n",
      "    - protein localization to cell junction\n",
      "    - positive regulation of membrane permeability\n",
      "    - positive regulation of monocyte extravasation\n",
      "  CC:\n",
      "    - extracellular space\n",
      "    - Golgi apparatus\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - microvillus\n",
      "    - cell-cell junction\n",
      "    - bicellular tight junction\n",
      "    - desmosome\n",
      "    - filamentous actin\n",
      "    - paranodal junction\n",
      "    - Schmidt-Lanterman incisure\n",
      "    - cell-cell contact zone\n",
      "    - cell-cell contact zone\n",
      "    - tight junction\n",
      "    - protein complex involved in cell adhesion\n",
      "    - protein complex involved in cell adhesion\n",
      "  MF:\n",
      "    - integrin binding\n",
      "    - integrin binding\n",
      "    - protein binding\n",
      "    - protein homodimerization activity\n",
      "    - protein heterodimerization activity\n",
      "    - cell-cell adhesion mediator activity\n",
      "    - cell-cell adhesion mediator activity\n",
      "UniProt: {'Swiss-Prot': 'Q9BX67', 'TrEMBL': ['H0YCW9', 'H0YD98']}\n",
      "\n",
      "Gene: CD2\n",
      "Symbol: CD2\n",
      "Name: CD2 molecule\n",
      "Summary: The protein encoded by this gene is a surface antigen found on all peripheral blood T-cells. The encoded protein interacts with LFA3 (CD58) on antigen presenting cells to optimize immune recognition. A locus control region (LCR) has been found in the 3' flanking sequence of this gene. [provided by RefSeq, Jun 2016].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - membrane raft polarization\n",
      "    - apoptotic process\n",
      "    - immune response\n",
      "    - cell surface receptor signaling pathway\n",
      "    - natural killer cell activation\n",
      "    - natural killer cell activation\n",
      "    - natural killer cell activation\n",
      "    - positive regulation of myeloid dendritic cell activation\n",
      "    - positive regulation of type II interferon production\n",
      "    - positive regulation of interleukin-8 production\n",
      "    - positive regulation of tumor necrosis factor production\n",
      "    - heterotypic cell-cell adhesion\n",
      "    - T cell activation\n",
      "    - natural killer cell mediated cytotoxicity\n",
      "    - regulation of T cell differentiation\n",
      "    - cell-cell adhesion\n",
      "    - cell-cell adhesion\n",
      "  CC:\n",
      "    - extracellular region\n",
      "    - nucleoplasm\n",
      "    - Golgi apparatus\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - cell-cell junction\n",
      "    - external side of plasma membrane\n",
      "    - cytoplasmic side of plasma membrane\n",
      "    - cell surface\n",
      "    - protein-containing complex\n",
      "  MF:\n",
      "    - signaling receptor binding\n",
      "    - signaling receptor binding\n",
      "    - protein binding\n",
      "    - receptor tyrosine kinase binding\n",
      "    - signaling receptor activity\n",
      "    - identical protein binding\n",
      "UniProt: {'Swiss-Prot': 'P06729', 'TrEMBL': ['Q5JVN7', 'Q53F96']}\n",
      "\n",
      "Gene: ANGPT1\n",
      "Symbol: ANGPT1\n",
      "Name: angiopoietin 1\n",
      "Summary: This gene encodes a secreted glycoprotein that belongs to the angiopoietin family. Members of this family play important roles in vascular development and angiogenesis. All angiopoietins bind with similar affinity to an endothelial cell-specific tyrosine-protein kinase receptor. The protein encoded by this gene is a secreted glycoprotein that activates the receptor by inducing its tyrosine phosphorylation. It plays a critical role in mediating reciprocal interactions between the endothelium and surrounding matrix and mesenchyme and inhibits endothelial permeability. The protein also contributes to blood vessel maturation and stability, and may be involved in early development of the heart. Mutations in this gene are associated with hereditary angioedema. [provided by RefSeq, Aug 2020].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - angiogenesis\n",
      "    - in utero embryonic development\n",
      "    - regulation of endothelial cell proliferation\n",
      "    - sprouting angiogenesis\n",
      "    - positive regulation of receptor internalization\n",
      "    - negative regulation of cytokine production involved in immune response\n",
      "    - negative regulation of cell adhesion\n",
      "    - activation of transmembrane receptor protein tyrosine kinase activity\n",
      "    - blood coagulation\n",
      "    - positive regulation of endothelial cell migration\n",
      "    - positive regulation of gene expression\n",
      "    - regulation of skeletal muscle satellite cell proliferation\n",
      "    - hemopoiesis\n",
      "    - heparin proteoglycan biosynthetic process\n",
      "    - positive regulation of protein ubiquitination\n",
      "    - cell-substrate adhesion\n",
      "    - regulation of tumor necrosis factor production\n",
      "    - protein localization to cell surface\n",
      "    - negative regulation of protein import into nucleus\n",
      "    - negative regulation of apoptotic process\n",
      "    - negative regulation of vascular permeability\n",
      "    - regulation of canonical NF-kappaB signal transduction\n",
      "    - negative regulation of neuron apoptotic process\n",
      "    - positive regulation of blood vessel endothelial cell migration\n",
      "    - positive regulation of cell adhesion\n",
      "    - Tie signaling pathway\n",
      "    - Tie signaling pathway\n",
      "    - Tie signaling pathway\n",
      "    - positive regulation of peptidyl-tyrosine phosphorylation\n",
      "    - positive regulation of coagulation\n",
      "    - positive chemotaxis\n",
      "    - neuron apoptotic process\n",
      "    - positive regulation of phosphatidylinositol 3-kinase/protein kinase B signal transduction\n",
      "    - positive regulation of phosphatidylinositol 3-kinase/protein kinase B signal transduction\n",
      "    - positive regulation of ERK1 and ERK2 cascade\n",
      "    - positive regulation of ERK1 and ERK2 cascade\n",
      "    - glomerulus vasculature development\n",
      "    - negative regulation of vascular endothelial growth factor signaling pathway\n",
      "    - positive regulation of blood-brain barrier permeability\n",
      "    - negative regulation of endothelial cell apoptotic process\n",
      "    - regulation of macrophage migration inhibitory factor signaling pathway\n",
      "  CC:\n",
      "    - extracellular region\n",
      "    - extracellular space\n",
      "    - extracellular space\n",
      "    - plasma membrane\n",
      "    - microvillus\n",
      "    - membrane raft\n",
      "    - collagen-containing extracellular matrix\n",
      "    - extracellular exosome\n",
      "  MF:\n",
      "    - protein binding\n",
      "    - receptor tyrosine kinase binding\n",
      "    - receptor tyrosine kinase binding\n",
      "    - identical protein binding\n",
      "    - receptor ligand activity\n",
      "UniProt: {'Swiss-Prot': 'Q15389', 'TrEMBL': ['E5RFF4', 'B4DTQ9', 'E7ERK4']}\n",
      "\n",
      "Gene: ANGPT2\n",
      "Symbol: ANGPT2\n",
      "Name: angiopoietin 2\n",
      "Summary: This gene belongs to the angiopoietin family of growth factors. The protein encoded by this gene is an antagonist of angiopoietin 1, and both angiopoietin 1 and angiopoietin 2 are ligands for the endothelial TEK receptor tyrosine kinase. Angiopoietin 2 is upregulated in multiple inflammatory diseases and is implicated in the direct control of inflammation-related signaling pathways. The encoded protein affects angiogenesis during embryogenesis and tumorigenesis, disrupts the vascular remodeling ability of angiopoietin 1, and may induce endothelial cell apoptosis. This gene serves a prognostic biomarker for acute respiratory distress syndrome. [provided by RefSeq, Aug 2020].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - angiogenesis\n",
      "    - response to hypoxia\n",
      "    - signal transduction\n",
      "    - germ cell development\n",
      "    - blood coagulation\n",
      "    - response to mechanical stimulus\n",
      "    - response to glucose\n",
      "    - gene expression\n",
      "    - negative regulation of cell-substrate adhesion\n",
      "    - response to activity\n",
      "    - negative regulation of angiogenesis\n",
      "    - animal organ regeneration\n",
      "    - negative regulation of blood vessel endothelial cell migration\n",
      "    - positive regulation of angiogenesis\n",
      "    - Tie signaling pathway\n",
      "    - Tie signaling pathway\n",
      "    - positive regulation of coagulation\n",
      "    - negative regulation of positive chemotaxis\n",
      "    - maternal process involved in female pregnancy\n",
      "    - cellular response to growth factor stimulus\n",
      "    - glomerulus vasculature development\n",
      "  CC:\n",
      "    - extracellular region\n",
      "    - extracellular space\n",
      "    - extracellular space\n",
      "    - extracellular space\n",
      "    - cell projection\n",
      "    - collagen-containing extracellular matrix\n",
      "  MF:\n",
      "    - signaling receptor binding\n",
      "    - protein binding\n",
      "    - receptor tyrosine kinase binding\n",
      "    - receptor tyrosine kinase binding\n",
      "    - metal ion binding\n",
      "    - receptor ligand activity\n",
      "UniProt: {'Swiss-Prot': 'O15123', 'TrEMBL': 'E7EVQ3'}\n",
      "\n",
      "Gene: SLC7A7\n",
      "Symbol: SLC7A7\n",
      "Name: solute carrier family 7 member 7\n",
      "Summary: The protein encoded by this gene is the light subunit of a cationic amino acid transporter. This sodium-independent transporter is formed when the light subunit encoded by this gene dimerizes with the heavy subunit transporter protein SLC3A2. This transporter is found in epithelial cell membranes where it transfers cationic and large neutral amino acids from the cell to the extracellular space. Defects in this gene are a cause of lysinuric protein intolerance (LPI). Alternative splicing results in multiple transcript variants. [provided by RefSeq, Jul 2011].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - regulation of arginine metabolic process\n",
      "    - amino acid transmembrane transport\n",
      "    - L-leucine transport\n",
      "    - L-arginine transmembrane transport\n",
      "    - basic amino acid transmembrane transport\n",
      "  CC:\n",
      "    - plasma membrane\n",
      "    - basolateral plasma membrane\n",
      "  MF:\n",
      "    - protein binding\n",
      "    - basic amino acid transmembrane transporter activity\n",
      "    - L-amino acid transmembrane transporter activity\n",
      "    - L-arginine transmembrane transporter activity\n",
      "UniProt: {'Swiss-Prot': 'Q9UM01', 'TrEMBL': ['H0YJ95', 'A0A0S2Z502', 'G3V5A1', 'A0A8V8TP95', 'G3V362', 'G3V4U1', 'D6RFE5', 'G3V4Z6', 'G3V2H8', 'G3V2L0', 'G3V2K0', 'G3V273']}\n",
      "\n",
      "Gene: SLC7A8\n",
      "Symbol: SLC7A8\n",
      "Name: solute carrier family 7 member 8\n",
      "Summary: Enables several functions, including neutral amino acid transmembrane transporter activity; thyroid hormone transmembrane transporter activity; and toxin transmembrane transporter activity. Involved in L-alanine import across plasma membrane; L-leucine import across plasma membrane; and thyroid hormone transport. Located in plasma membrane. Part of basolateral plasma membrane and microvillus membrane. [provided by Alliance of Genome Resources, Apr 2022]\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - amino acid transmembrane transport\n",
      "    - amino acid transport\n",
      "    - amino acid transport\n",
      "    - neutral amino acid transport\n",
      "    - neutral amino acid transport\n",
      "    - neutral amino acid transport\n",
      "    - glycine transport\n",
      "    - L-leucine transport\n",
      "    - tryptophan transport\n",
      "    - valine transport\n",
      "    - proline transmembrane transport\n",
      "    - thyroid hormone transport\n",
      "    - thyroid hormone transport\n",
      "    - amino acid import across plasma membrane\n",
      "    - transport across blood-brain barrier\n",
      "    - L-leucine import across plasma membrane\n",
      "    - L-leucine import across plasma membrane\n",
      "    - L-leucine import across plasma membrane\n",
      "    - L-alanine import across plasma membrane\n",
      "  CC:\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - basal plasma membrane\n",
      "    - basolateral plasma membrane\n",
      "    - apical plasma membrane\n",
      "    - microvillus membrane\n",
      "  MF:\n",
      "    - protein binding\n",
      "    - organic cation transmembrane transporter activity\n",
      "    - amino acid transmembrane transporter activity\n",
      "    - amino acid transmembrane transporter activity\n",
      "    - amino acid transmembrane transporter activity\n",
      "    - neutral L-amino acid transmembrane transporter activity\n",
      "    - neutral L-amino acid transmembrane transporter activity\n",
      "    - neutral L-amino acid transmembrane transporter activity\n",
      "    - neutral L-amino acid transmembrane transporter activity\n",
      "    - L-amino acid transmembrane transporter activity\n",
      "    - L-alanine transmembrane transporter activity\n",
      "    - glycine transmembrane transporter activity\n",
      "    - L-leucine transmembrane transporter activity\n",
      "    - L-leucine transmembrane transporter activity\n",
      "    - antiporter activity\n",
      "    - thyroid hormone transmembrane transporter activity\n",
      "    - thyroid hormone transmembrane transporter activity\n",
      "    - toxin transmembrane transporter activity\n",
      "    - peptide antigen binding\n",
      "    - protein heterodimerization activity\n",
      "UniProt: {'Swiss-Prot': 'Q9UHI5', 'TrEMBL': ['E9PS92', 'Q53EM9', 'E9PQT4', 'H0Y2X7', 'E9PIC3', 'E9PLV9']}\n",
      "\n",
      "Gene: SLC7A5\n",
      "Symbol: SLC7A5\n",
      "Name: solute carrier family 7 member 5\n",
      "Summary: Enables L-leucine transmembrane transporter activity; L-tryptophan transmembrane transporter activity; and thyroid hormone transmembrane transporter activity. Involved in carboxylic acid transport; thyroid hormone transport; and xenobiotic transport. Located in cytosol; intracellular membrane-bounded organelle; and plasma membrane. Is integral component of membrane. Part of amino acid transport complex; apical plasma membrane; and microvillus membrane. [provided by Alliance of Genome Resources, Apr 2022]\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - positive regulation of cytokine production involved in immune response\n",
      "    - amino acid transmembrane transport\n",
      "    - negative regulation of autophagy\n",
      "    - negative regulation of gene expression\n",
      "    - response to muscle activity\n",
      "    - neutral amino acid transport\n",
      "    - neutral amino acid transport\n",
      "    - neutral amino acid transport\n",
      "    - isoleucine transport\n",
      "    - L-leucine transport\n",
      "    - methionine transport\n",
      "    - phenylalanine transport\n",
      "    - phenylalanine transport\n",
      "    - proline transport\n",
      "    - tryptophan transport\n",
      "    - tyrosine transport\n",
      "    - valine transport\n",
      "    - alanine transport\n",
      "    - positive regulation of type II interferon production\n",
      "    - positive regulation of interleukin-17 production\n",
      "    - positive regulation of interleukin-4 production\n",
      "    - cellular response to glucose starvation\n",
      "    - xenobiotic transport\n",
      "    - response to hyperoxia\n",
      "    - positive regulation of glial cell proliferation\n",
      "    - thyroid hormone transport\n",
      "    - cellular response to lipopolysaccharide\n",
      "    - amino acid import across plasma membrane\n",
      "    - amino acid import across plasma membrane\n",
      "    - amino acid import across plasma membrane\n",
      "    - liver regeneration\n",
      "    - transport across blood-brain barrier\n",
      "    - L-histidine transport\n",
      "    - cellular response to L-arginine\n",
      "    - L-leucine import across plasma membrane\n",
      "    - L-leucine import across plasma membrane\n",
      "    - L-tryptophan transmembrane transport\n",
      "    - negative regulation of vascular associated smooth muscle cell apoptotic process\n",
      "    - positive regulation of L-leucine import across plasma membrane\n",
      "  CC:\n",
      "    - lysosomal membrane\n",
      "    - cytosol\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - basal plasma membrane\n",
      "    - membrane\n",
      "    - membrane\n",
      "    - basolateral plasma membrane\n",
      "    - apical plasma membrane\n",
      "    - apical plasma membrane\n",
      "    - microvillus membrane\n",
      "    - intracellular membrane-bounded organelle\n",
      "    - extracellular exosome\n",
      "    - external side of apical plasma membrane\n",
      "    - amino acid transport complex\n",
      "  MF:\n",
      "    - protein binding\n",
      "    - amino acid transmembrane transporter activity\n",
      "    - amino acid transmembrane transporter activity\n",
      "    - amino acid transmembrane transporter activity\n",
      "    - aromatic amino acid transmembrane transporter activity\n",
      "    - neutral L-amino acid transmembrane transporter activity\n",
      "    - neutral L-amino acid transmembrane transporter activity\n",
      "    - neutral L-amino acid transmembrane transporter activity\n",
      "    - L-amino acid transmembrane transporter activity\n",
      "    - L-leucine transmembrane transporter activity\n",
      "    - L-leucine transmembrane transporter activity\n",
      "    - L-tryptophan transmembrane transporter activity\n",
      "    - antiporter activity\n",
      "    - thyroid hormone transmembrane transporter activity\n",
      "    - peptide antigen binding\n",
      "UniProt: {'Swiss-Prot': 'Q01650'}\n",
      "\n",
      "Gene: HTR3D\n",
      "Symbol: HTR3D\n",
      "Name: 5-hydroxytryptamine receptor 3D\n",
      "Summary: The protein encoded this gene belongs to the ligand-gated ion channel receptor superfamily. This gene encodes subunit D of the type 3 receptor for 5-hydroxytryptamine (serotonin), a biogenic hormone that functions as a neurotransmitter, a mitogen and a hormone. This hormone has been linked to neuropsychiatric disorders, including anxiety, depression, and migraine. Serotonin receptors causes fast and depolarizing responses in neurons following activation. The genes encoding subunits C, D and E of this type 3 receptor form a cluster on chromosome 3. Alternatively spliced transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Jul 2009].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - serotonin receptor signaling pathway\n",
      "    - chemical synaptic transmission\n",
      "    - monoatomic ion transmembrane transport\n",
      "    - regulation of membrane potential\n",
      "    - excitatory postsynaptic potential\n",
      "    - inorganic cation transmembrane transport\n",
      "    - serotonin-gated cation-selective signaling pathway\n",
      "  CC:\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - neuron projection\n",
      "    - synapse\n",
      "    - postsynaptic membrane\n",
      "    - transmembrane transporter complex\n",
      "    - serotonin-activated cation-selective channel complex\n",
      "  MF:\n",
      "    - excitatory extracellular ligand-gated monoatomic ion channel activity\n",
      "    - protein binding\n",
      "    - serotonin-gated monoatomic cation channel activity\n",
      "    - serotonin-gated monoatomic cation channel activity\n",
      "    - transmitter-gated monoatomic ion channel activity involved in regulation of postsynaptic membrane potential\n",
      "UniProt: {'Swiss-Prot': 'Q70Z44', 'TrEMBL': 'F6WC43'}\n",
      "\n",
      "Gene: HTR3A\n",
      "Symbol: HTR3A\n",
      "Name: 5-hydroxytryptamine receptor 3A\n",
      "Summary: The product of this gene belongs to the ligand-gated ion channel receptor superfamily. This gene encodes subunit A of the type 3 receptor for 5-hydroxytryptamine (serotonin), a biogenic hormone that functions as a neurotransmitter, a hormone, and a mitogen. This receptor causes fast, depolarizing responses in neurons after activation. It appears that the heteromeric combination of A and B subunits is necessary to provide the full functional features of this receptor, since either subunit alone results in receptors with very low conductance and response amplitude. Alternatively spliced transcript variants encoding different isoforms have been identified. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - serotonin receptor signaling pathway\n",
      "    - chemical synaptic transmission\n",
      "    - monoatomic ion transmembrane transport\n",
      "    - regulation of membrane potential\n",
      "    - excitatory postsynaptic potential\n",
      "    - inorganic cation transmembrane transport\n",
      "    - regulation of presynaptic membrane potential\n",
      "    - serotonin-gated cation-selective signaling pathway\n",
      "  CC:\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - cleavage furrow\n",
      "    - neuron projection\n",
      "    - synapse\n",
      "    - postsynaptic membrane\n",
      "    - transmembrane transporter complex\n",
      "    - serotonin-activated cation-selective channel complex\n",
      "    - serotonin-activated cation-selective channel complex\n",
      "  MF:\n",
      "    - excitatory extracellular ligand-gated monoatomic ion channel activity\n",
      "    - protein binding\n",
      "    - serotonin-gated monoatomic cation channel activity\n",
      "    - serotonin-gated monoatomic cation channel activity\n",
      "    - identical protein binding\n",
      "    - serotonin binding\n",
      "    - ligand-gated monoatomic ion channel activity involved in regulation of presynaptic membrane potential\n",
      "    - transmitter-gated monoatomic ion channel activity involved in regulation of postsynaptic membrane potential\n",
      "UniProt: {'Swiss-Prot': 'P46098', 'TrEMBL': ['A0A0B4J205', 'B4E398']}\n",
      "\n",
      "Gene: HTR3E\n",
      "Symbol: HTR3E\n",
      "Name: 5-hydroxytryptamine receptor 3E\n",
      "Summary: This locus encodes a 5-hydroxytryptamine (serotonin) receptor subunit. The encoded protein, subunit E, may play a role in neurotransmission in myenteric neurons. Genes encoding subunits C, D and E form a cluster on chromosome 3. Alternatively spliced transcript variants encoding distinct isoforms have been described. [provided by RefSeq, Feb 2012].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - serotonin receptor signaling pathway\n",
      "    - chemical synaptic transmission\n",
      "    - monoatomic ion transmembrane transport\n",
      "    - regulation of membrane potential\n",
      "    - excitatory postsynaptic potential\n",
      "    - inorganic cation transmembrane transport\n",
      "    - serotonin-gated cation-selective signaling pathway\n",
      "  CC:\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - neuron projection\n",
      "    - synapse\n",
      "    - postsynaptic membrane\n",
      "    - transmembrane transporter complex\n",
      "    - serotonin-activated cation-selective channel complex\n",
      "  MF:\n",
      "    - excitatory extracellular ligand-gated monoatomic ion channel activity\n",
      "    - protein binding\n",
      "    - serotonin-gated monoatomic cation channel activity\n",
      "    - serotonin-gated monoatomic cation channel activity\n",
      "    - transmitter-gated monoatomic ion channel activity involved in regulation of postsynaptic membrane potential\n",
      "UniProt: {'Swiss-Prot': 'A5X5Y0', 'TrEMBL': 'C9J420'}\n",
      "\n",
      "Gene: HTR3B\n",
      "Symbol: HTR3B\n",
      "Name: 5-hydroxytryptamine receptor 3B\n",
      "Summary: The product of this gene belongs to the ligand-gated ion channel receptor superfamily. This gene encodes subunit B of the type 3 receptor for 5-hydroxytryptamine (serotonin), a biogenic hormone that functions as a neurotransmitter, a hormone, and a mitogen. This receptor causes fast, depolarizing responses in neurons after activation. It is not functional as a homomeric complex, but a pentaheteromeric complex with subunit A (HTR3A) displays the full functional features of this receptor. [provided by RefSeq, Aug 2011].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - serotonin receptor signaling pathway\n",
      "    - chemical synaptic transmission\n",
      "    - monoatomic ion transmembrane transport\n",
      "    - regulation of membrane potential\n",
      "    - excitatory postsynaptic potential\n",
      "    - inorganic cation transmembrane transport\n",
      "    - serotonin-gated cation-selective signaling pathway\n",
      "  CC:\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - cell surface\n",
      "    - neuron projection\n",
      "    - synapse\n",
      "    - postsynaptic membrane\n",
      "    - transmembrane transporter complex\n",
      "    - serotonin-activated cation-selective channel complex\n",
      "    - serotonin-activated cation-selective channel complex\n",
      "  MF:\n",
      "    - excitatory extracellular ligand-gated monoatomic ion channel activity\n",
      "    - serotonin-gated monoatomic cation channel activity\n",
      "    - serotonin-gated monoatomic cation channel activity\n",
      "    - serotonin-gated monoatomic cation channel activity\n",
      "    - transmitter-gated monoatomic ion channel activity involved in regulation of postsynaptic membrane potential\n",
      "UniProt: {'Swiss-Prot': 'O95264', 'TrEMBL': 'H0YFX8'}\n",
      "\n",
      "Gene: HTR3C\n",
      "Symbol: HTR3C\n",
      "Name: 5-hydroxytryptamine receptor 3C\n",
      "Summary: The product of this gene belongs to the ligand-gated ion channel receptor superfamily. This gene encodes subunit C of the type 3 receptor for 5-hydroxytryptamine (serotonin), a biogenic hormone that functions as a neurotransmitter, a hormone, and a mitogen. This receptor causes fast, depolarizing responses in neurons after activation. Genes encoding subunits C, D and E form a cluster on chromosome 3. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - serotonin receptor signaling pathway\n",
      "    - chemical synaptic transmission\n",
      "    - monoatomic ion transmembrane transport\n",
      "    - regulation of membrane potential\n",
      "    - excitatory postsynaptic potential\n",
      "    - inorganic cation transmembrane transport\n",
      "    - serotonin-gated cation-selective signaling pathway\n",
      "  CC:\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - neuron projection\n",
      "    - synapse\n",
      "    - postsynaptic membrane\n",
      "    - transmembrane transporter complex\n",
      "    - serotonin-activated cation-selective channel complex\n",
      "  MF:\n",
      "    - excitatory extracellular ligand-gated monoatomic ion channel activity\n",
      "    - protein binding\n",
      "    - serotonin-gated monoatomic cation channel activity\n",
      "    - serotonin-gated monoatomic cation channel activity\n",
      "    - transmitter-gated monoatomic ion channel activity involved in regulation of postsynaptic membrane potential\n",
      "UniProt: {'Swiss-Prot': 'Q8WXA8'}\n",
      "\n",
      "Gene: SMAD2\n",
      "Symbol: SMAD2\n",
      "Name: SMAD family member 2\n",
      "Summary: The protein encoded by this gene belongs to the SMAD, a family of proteins similar to the gene products of the Drosophila gene 'mothers against decapentaplegic' (Mad) and the C. elegans gene Sma. SMAD proteins are signal transducers and transcriptional modulators that mediate multiple signaling pathways. This protein mediates the signal of the transforming growth factor (TGF)-beta, and thus regulates multiple cellular processes, such as cell proliferation, apoptosis, and differentiation. This protein is recruited to the TGF-beta receptors through its interaction with the SMAD anchor for receptor activation (SARA) protein. In response to TGF-beta signal, this protein is phosphorylated by the TGF-beta receptors. The phosphorylation induces the dissociation of this protein with SARA and the association with the family member SMAD4. The association with SMAD4 is important for the translocation of this protein into the nucleus, where it binds to target promoters and forms a transcription repressor complex with other cofactors. This protein can also be phosphorylated by activin type 1 receptor kinase, and mediates the signal from the activin. Alternatively spliced transcript variants have been observed for this gene. [provided by RefSeq, May 2012].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - ureteric bud development\n",
      "    - in utero embryonic development\n",
      "    - endoderm formation\n",
      "    - mesoderm formation\n",
      "    - determination of left/right asymmetry in lateral mesoderm\n",
      "    - aortic valve morphogenesis\n",
      "    - pulmonary valve morphogenesis\n",
      "    - endocardial cushion morphogenesis\n",
      "    - DNA-templated transcription\n",
      "    - regulation of DNA-templated transcription\n",
      "    - transforming growth factor beta receptor signaling pathway\n",
      "    - transforming growth factor beta receptor signaling pathway\n",
      "    - transforming growth factor beta receptor signaling pathway\n",
      "    - transforming growth factor beta receptor signaling pathway\n",
      "    - zygotic specification of dorsal/ventral axis\n",
      "    - gastrulation\n",
      "    - cell population proliferation\n",
      "    - negative regulation of cell population proliferation\n",
      "    - anatomical structure morphogenesis\n",
      "    - post-embryonic development\n",
      "    - anterior/posterior pattern specification\n",
      "    - positive regulation of gene expression\n",
      "    - positive regulation of epithelial to mesenchymal transition\n",
      "    - regulation of transforming growth factor beta receptor signaling pathway\n",
      "    - signal transduction involved in regulation of gene expression\n",
      "    - insulin secretion\n",
      "    - cell differentiation\n",
      "    - negative regulation of ossification\n",
      "    - lung development\n",
      "    - adrenal gland development\n",
      "    - positive regulation of BMP signaling pathway\n",
      "    - pancreas development\n",
      "    - primary miRNA processing\n",
      "    - activin receptor signaling pathway\n",
      "    - activin receptor signaling pathway\n",
      "    - activin receptor signaling pathway\n",
      "    - organ growth\n",
      "    - intracellular signal transduction\n",
      "    - nodal signaling pathway\n",
      "    - cell fate commitment\n",
      "    - negative regulation of cell differentiation\n",
      "    - negative regulation of DNA-templated transcription\n",
      "    - positive regulation of DNA-templated transcription\n",
      "    - positive regulation of DNA-templated transcription\n",
      "    - positive regulation of DNA-templated transcription\n",
      "    - positive regulation of transcription by RNA polymerase II\n",
      "    - positive regulation of transcription by RNA polymerase II\n",
      "    - positive regulation of transcription by RNA polymerase II\n",
      "    - paraxial mesoderm morphogenesis\n",
      "    - embryonic foregut morphogenesis\n",
      "    - embryonic cranial skeleton morphogenesis\n",
      "    - regulation of binding\n",
      "    - pericardium development\n",
      "    - SMAD protein signal transduction\n",
      "    - SMAD protein signal transduction\n",
      "    - SMAD protein signal transduction\n",
      "    - SMAD protein signal transduction\n",
      "    - trophoblast cell migration\n",
      "    - secondary palate development\n",
      "    - response to cholesterol\n",
      "    - cellular response to glucose stimulus\n",
      "    - odontoblast differentiation\n",
      "  CC:\n",
      "    - chromatin\n",
      "    - chromatin\n",
      "    - nucleus\n",
      "    - nucleoplasm\n",
      "    - nucleoplasm\n",
      "    - transcription regulator complex\n",
      "    - cytoplasm\n",
      "    - cytosol\n",
      "    - activin responsive factor complex\n",
      "    - protein-containing complex\n",
      "    - intracellular membrane-bounded organelle\n",
      "    - SMAD protein complex\n",
      "    - homomeric SMAD protein complex\n",
      "    - heteromeric SMAD protein complex\n",
      "    - heteromeric SMAD protein complex\n",
      "    - heteromeric SMAD protein complex\n",
      "    - heteromeric SMAD protein complex\n",
      "  MF:\n",
      "    - RNA polymerase II cis-regulatory region sequence-specific DNA binding\n",
      "    - RNA polymerase II cis-regulatory region sequence-specific DNA binding\n",
      "    - DNA-binding transcription factor activity, RNA polymerase II-specific\n",
      "    - DNA-binding transcription factor activity, RNA polymerase II-specific\n",
      "    - cis-regulatory region sequence-specific DNA binding\n",
      "    - DNA-binding transcription activator activity, RNA polymerase II-specific\n",
      "    - DNA-binding transcription activator activity, RNA polymerase II-specific\n",
      "    - DNA binding\n",
      "    - chromatin binding\n",
      "    - double-stranded DNA binding\n",
      "    - DNA-binding transcription factor activity\n",
      "    - DNA-binding transcription factor activity\n",
      "    - transforming growth factor beta receptor binding\n",
      "    - protein binding\n",
      "    - phosphatase binding\n",
      "    - ubiquitin protein ligase binding\n",
      "    - type I transforming growth factor beta receptor binding\n",
      "    - identical protein binding\n",
      "    - SMAD binding\n",
      "    - metal ion binding\n",
      "    - tau protein binding\n",
      "    - RNA polymerase II-specific DNA-binding transcription factor binding\n",
      "    - co-SMAD binding\n",
      "    - I-SMAD binding\n",
      "    - I-SMAD binding\n",
      "    - R-SMAD binding\n",
      "    - disordered domain specific binding\n",
      "    - DNA-binding transcription factor binding\n",
      "UniProt: {'Swiss-Prot': 'Q15796', 'TrEMBL': ['B7Z5N5', 'Q53XR6', 'K7EJX0', 'K7ESI8', 'K7EL92', 'K7ERC7']}\n",
      "\n",
      "Gene: CSF2RA\n",
      "Symbol: CSF2RA\n",
      "Name: colony stimulating factor 2 receptor subunit alpha\n",
      "Summary: The protein encoded by this gene is the alpha subunit of the heterodimeric receptor for colony stimulating factor 2, a cytokine which controls the production, differentiation, and function of granulocytes and macrophages. The encoded protein is a member of the cytokine family of receptors. This gene is found in the pseudoautosomal region (PAR) of the X and Y chromosomes. Multiple transcript variants encoding different isoforms have been found for this gene, with some of the isoforms being membrane-bound and others being soluble. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - cell surface receptor signaling pathway via JAK-STAT\n",
      "    - positive regulation of cell population proliferation\n",
      "    - cytokine-mediated signaling pathway\n",
      "    - granulocyte-macrophage colony-stimulating factor signaling pathway\n",
      "    - positive regulation of receptor signaling pathway via JAK-STAT\n",
      "    - growth hormone receptor signaling pathway\n",
      "    - positive regulation of leukocyte proliferation\n",
      "  CC:\n",
      "    - extracellular region\n",
      "    - cytosol\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - external side of plasma membrane\n",
      "    - granulocyte macrophage colony-stimulating factor receptor complex\n",
      "    - growth hormone receptor complex\n",
      "  MF:\n",
      "    - growth hormone receptor activity\n",
      "    - protein binding\n",
      "    - peptide hormone binding\n",
      "    - cytokine binding\n",
      "    - signaling receptor activity\n",
      "UniProt: {'Swiss-Prot': 'P15509'}\n",
      "\n",
      "Gene: CSF2\n",
      "Symbol: CSF2\n",
      "Name: colony stimulating factor 2\n",
      "Summary: The protein encoded by this gene is a cytokine that controls the production, differentiation, and function of granulocytes and macrophages. The active form of the protein is found extracellularly as a homodimer. This gene has been localized to a cluster of related genes at chromosome region 5q31, which is known to be associated with interstitial deletions in the 5q- syndrome and acute myelogenous leukemia. Other genes in the cluster include those encoding interleukins 4, 5, and 13. This gene plays a role in promoting tissue inflammation. Elevated levels of cytokines, including the one produced by this gene, have been detected in SARS-CoV-2 infected patients that develop acute respiratory distress syndrome. Mice deficient in this gene or its receptor develop pulmonary alveolar proteinosis. [provided by RefSeq, Aug 2020].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - histamine secretion\n",
      "    - embryonic placenta development\n",
      "    - immune response\n",
      "    - cell surface receptor signaling pathway via JAK-STAT\n",
      "    - cell population proliferation\n",
      "    - positive regulation of cell population proliferation\n",
      "    - positive regulation of gene expression\n",
      "    - positive regulation of macrophage derived foam cell differentiation\n",
      "    - myeloid cell differentiation\n",
      "    - neutrophil differentiation\n",
      "    - monocyte differentiation\n",
      "    - macrophage differentiation\n",
      "    - positive regulation of cell migration\n",
      "    - positive regulation of interleukin-23 production\n",
      "    - response to silicon dioxide\n",
      "    - response to fluid shear stress\n",
      "    - granulocyte-macrophage colony-stimulating factor signaling pathway\n",
      "    - epithelial fluid transport\n",
      "    - myeloid dendritic cell differentiation\n",
      "    - regulation of circadian sleep/wake cycle, sleep\n",
      "    - negative regulation of DNA-templated transcription\n",
      "    - positive regulation of leukocyte proliferation\n",
      "    - cellular response to lipopolysaccharide\n",
      "    - positive regulation of podosome assembly\n",
      "    - cellular response to granulocyte macrophage colony-stimulating factor stimulus\n",
      "    - dendritic cell differentiation\n",
      "    - cell surface receptor signaling pathway via STAT\n",
      "    - negative regulation of extrinsic apoptotic signaling pathway in absence of ligand\n",
      "  CC:\n",
      "    - extracellular region\n",
      "    - extracellular space\n",
      "    - extracellular space\n",
      "    - plasma membrane\n",
      "    - granulocyte macrophage colony-stimulating factor receptor complex\n",
      "    - intracellular membrane-bounded organelle\n",
      "  MF:\n",
      "    - cytokine activity\n",
      "    - cytokine activity\n",
      "    - granulocyte macrophage colony-stimulating factor receptor binding\n",
      "    - protein binding\n",
      "    - growth factor activity\n",
      "UniProt: {'Swiss-Prot': 'P04141'}\n",
      "\n",
      "Gene: IL5RA\n",
      "Symbol: IL5RA\n",
      "Name: interleukin 5 receptor subunit alpha\n",
      "Summary: The protein encoded by this gene is an interleukin 5 specific subunit of a heterodimeric cytokine receptor. The receptor is comprised of a ligand specific alpha subunit and a signal transducing beta subunit shared by the receptors for interleukin 3 (IL3), colony stimulating factor 2 (CSF2/GM-CSF), and interleukin 5 (IL5). The binding of this protein to IL5 depends on the beta subunit. The beta subunit is activated by the ligand binding, and is required for the biological activities of IL5. This protein has been found to interact with syndecan binding protein (syntenin), which is required for IL5 mediated activation of the transcription factor SOX4. Several alternatively spliced transcript variants encoding four distinct isoforms have been reported. [provided by RefSeq, Jul 2011].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - inflammatory response to antigenic stimulus\n",
      "    - signal transduction\n",
      "    - positive regulation of cell population proliferation\n",
      "    - cytokine-mediated signaling pathway\n",
      "    - regulation of interleukin-5 production\n",
      "    - interleukin-5-mediated signaling pathway\n",
      "    - positive regulation of leukocyte proliferation\n",
      "  CC:\n",
      "    - extracellular space\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - external side of plasma membrane\n",
      "    - membrane\n",
      "    - receptor complex\n",
      "  MF:\n",
      "    - cytokine receptor activity\n",
      "    - interleukin-5 receptor activity\n",
      "    - protein binding\n",
      "    - cytokine binding\n",
      "UniProt: {'Swiss-Prot': 'Q01344', 'TrEMBL': ['E7ERY4', 'Q8NHV7', 'C9J6C4', 'C9J3K8']}\n",
      "\n",
      "Gene: IL5\n",
      "Symbol: IL5\n",
      "Name: interleukin 5\n",
      "Summary: This gene encodes a cytokine that acts as a growth and differentiation factor for both B cells and eosinophils. The encoded cytokine plays a major role in the regulation of eosinophil formation, maturation, recruitment and survival. The increased production of this cytokine may be related to pathogenesis of eosinophil-dependent inflammatory diseases. This cytokine functions by binding to its receptor, which is a heterodimer, whose beta subunit is shared with the receptors for interleukine 3 (IL3) and colony stimulating factor 2 (CSF2/GM-CSF). This gene is located on chromosome 5 within a cytokine gene cluster which includes interleukin 4 (IL4), interleukin 13 (IL13), and CSF2 . This gene, IL4, and IL13 may be regulated coordinately by long-range regulatory elements spread over 120 kilobases on chromosome 5q31. [provided by RefSeq, Jul 2013].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - positive regulation of immunoglobulin production\n",
      "    - inflammatory response\n",
      "    - immune response\n",
      "    - positive regulation of B cell proliferation\n",
      "    - interleukin-5-mediated signaling pathway\n",
      "    - interleukin-5-mediated signaling pathway\n",
      "    - positive regulation of eosinophil differentiation\n",
      "    - positive regulation of DNA-templated transcription\n",
      "    - positive regulation of receptor signaling pathway via JAK-STAT\n",
      "    - positive regulation of peptidyl-tyrosine phosphorylation\n",
      "    - positive regulation of podosome assembly\n",
      "  CC:\n",
      "    - extracellular region\n",
      "    - extracellular region\n",
      "    - extracellular space\n",
      "    - extracellular space\n",
      "    - extracellular space\n",
      "  MF:\n",
      "    - cytokine activity\n",
      "    - cytokine activity\n",
      "    - interleukin-5 receptor binding\n",
      "    - protein binding\n",
      "    - growth factor activity\n",
      "UniProt: {'Swiss-Prot': 'P05113', 'TrEMBL': 'C9JQP9'}\n",
      "\n",
      "Gene: IL4R\n",
      "Symbol: IL4R\n",
      "Name: interleukin 4 receptor\n",
      "Summary: This gene encodes the alpha chain of the interleukin-4 receptor, a type I transmembrane protein that can bind interleukin 4 and interleukin 13 to regulate IgE production. The encoded protein also can bind interleukin 4 to promote differentiation of Th2 cells. A soluble form of the encoded protein can be produced by proteolysis of the membrane-bound protein, and this soluble form can inhibit IL4-mediated cell proliferation and IL5 upregulation by T-cells. Allelic variations in this gene have been associated with atopy, a condition that can manifest itself as allergic rhinitis, sinusitus, asthma, or eczema. Polymorphisms in this gene are also associated with resistance to human immunodeficiency virus type-1 infection. Alternate splicing results in multiple transcript variants. [provided by RefSeq, Apr 2012].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - production of molecular mediator involved in inflammatory response\n",
      "    - positive regulation of immunoglobulin production\n",
      "    - immune response\n",
      "    - signal transduction\n",
      "    - immunoglobulin mediated immune response\n",
      "    - cytokine-mediated signaling pathway\n",
      "    - positive regulation of chemokine production\n",
      "    - interleukin-4-mediated signaling pathway\n",
      "    - defense response to protozoan\n",
      "    - positive regulation of macrophage activation\n",
      "    - positive regulation of mast cell degranulation\n",
      "    - T-helper 1 cell differentiation\n",
      "    - T-helper 2 cell differentiation\n",
      "    - negative regulation of T-helper 1 cell differentiation\n",
      "    - positive regulation of T-helper 2 cell differentiation\n",
      "    - positive regulation of cold-induced thermogenesis\n",
      "    - positive regulation of myoblast fusion\n",
      "  CC:\n",
      "    - extracellular region\n",
      "    - nucleoplasm\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - external side of plasma membrane\n",
      "    - centriolar satellite\n",
      "    - receptor complex\n",
      "  MF:\n",
      "    - cytokine receptor activity\n",
      "    - interleukin-4 receptor activity\n",
      "    - protein binding\n",
      "UniProt: {'Swiss-Prot': 'P24394', 'TrEMBL': ['J9JII2', 'B2RBY1', 'Q5FC08', 'I3L4H7', 'H3BTD9', 'A0A0G2JLA1', 'H3BRH7', 'I3L4W9', 'H3BSE7']}\n",
      "\n",
      "Gene: IL13\n",
      "Symbol: IL13\n",
      "Name: interleukin 13\n",
      "Summary: This gene encodes an immunoregulatory cytokine produced primarily by activated Th2 cells. This cytokine is involved in several stages of B-cell maturation and differentiation. It up-regulates CD23 and MHC class II expression, and promotes IgE isotype switching of B cells. This cytokine down-regulates macrophage activity, thereby inhibits the production of pro-inflammatory cytokines and chemokines. This cytokine is found to be critical to the pathogenesis of allergen-induced asthma but operates through mechanisms independent of IgE and eosinophils. This gene, IL3, IL5, IL4, and CSF2 form a cytokine gene cluster on chromosome 5q, with this gene particularly close to IL4. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - microglial cell activation\n",
      "    - positive regulation of immunoglobulin production\n",
      "    - inflammatory response\n",
      "    - immune response\n",
      "    - response to nematode\n",
      "    - regulation of proton transport\n",
      "    - response to selenium ion\n",
      "    - positive regulation of gene expression\n",
      "    - positive regulation of B cell proliferation\n",
      "    - response to lipopolysaccharide\n",
      "    - positive regulation of interleukin-10 production\n",
      "    - response to nicotine\n",
      "    - interleukin-13-mediated signaling pathway\n",
      "    - macrophage activation\n",
      "    - positive regulation of macrophage activation\n",
      "    - positive regulation of mast cell degranulation\n",
      "    - response to ethanol\n",
      "    - positive regulation of transcription by RNA polymerase II\n",
      "    - positive regulation of smooth muscle cell proliferation\n",
      "    - positive regulation of protein secretion\n",
      "    - negative regulation of inflammatory response\n",
      "    - positive regulation of release of sequestered calcium ion into cytosol\n",
      "    - cellular response to mechanical stimulus\n",
      "    - negative regulation of transforming growth factor beta production\n",
      "    - positive regulation of cold-induced thermogenesis\n",
      "    - negative regulation of lung ciliated cell differentiation\n",
      "    - positive regulation of lung goblet cell differentiation\n",
      "    - negative regulation of complement-dependent cytotoxicity\n",
      "    - positive regulation of pancreatic stellate cell proliferation\n",
      "    - negative regulation of endothelial cell apoptotic process\n",
      "  CC:\n",
      "    - extracellular region\n",
      "    - extracellular region\n",
      "    - extracellular space\n",
      "    - extracellular space\n",
      "    - cytoplasm\n",
      "    - external side of plasma membrane\n",
      "  MF:\n",
      "    - cytokine activity\n",
      "    - interleukin-13 receptor binding\n",
      "    - protein binding\n",
      "UniProt: {'Swiss-Prot': 'P35225'}\n",
      "\n",
      "Gene: SMARCA4\n",
      "Symbol: SMARCA4\n",
      "Name: SWI/SNF related BAF chromatin remodeling complex subunit ATPase 4\n",
      "Summary: The protein encoded by this gene is a member of the SWI/SNF family of proteins and is similar to the brahma protein of Drosophila. Members of this family have helicase and ATPase activities and are thought to regulate transcription of certain genes by altering the chromatin structure around those genes. The encoded protein is part of the large ATP-dependent chromatin remodeling complex SNF/SWI, which is required for transcriptional activation of genes normally repressed by chromatin. In addition, this protein can bind BRCA1, as well as regulate the expression of the tumorigenic protein CD44. Mutations in this gene cause rhabdoid tumor predisposition syndrome type 2. Multiple transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, May 2012].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - negative regulation of transcription by RNA polymerase II\n",
      "    - RNA polymerase I preinitiation complex assembly\n",
      "    - neural retina development\n",
      "    - nucleosome disassembly\n",
      "    - chromatin remodeling\n",
      "    - chromatin remodeling\n",
      "    - chromatin remodeling\n",
      "    - chromatin remodeling\n",
      "    - chromatin remodeling\n",
      "    - regulation of transcription by RNA polymerase II\n",
      "    - nervous system development\n",
      "    - positive regulation of cell population proliferation\n",
      "    - regulation of mitotic metaphase/anaphase transition\n",
      "    - positive regulation of Wnt signaling pathway\n",
      "    - negative regulation of cell growth\n",
      "    - heterochromatin formation\n",
      "    - positive regulation by host of viral transcription\n",
      "    - positive regulation of T cell differentiation\n",
      "    - negative regulation of cell differentiation\n",
      "    - positive regulation of cell differentiation\n",
      "    - positive regulation of myoblast differentiation\n",
      "    - transcription initiation-coupled chromatin remodeling\n",
      "    - negative regulation of DNA-templated transcription\n",
      "    - negative regulation of DNA-templated transcription\n",
      "    - positive regulation of DNA-templated transcription\n",
      "    - positive regulation of transcription by RNA polymerase II\n",
      "    - positive regulation of transcription by RNA polymerase II\n",
      "    - positive regulation of transcription by RNA polymerase II\n",
      "    - positive regulation of DNA-binding transcription factor activity\n",
      "    - negative regulation of androgen receptor signaling pathway\n",
      "    - regulation of G0 to G1 transition\n",
      "    - positive regulation of cold-induced thermogenesis\n",
      "    - positive regulation of transcription of nucleolar large rRNA by RNA polymerase I\n",
      "    - positive regulation of stem cell population maintenance\n",
      "    - positive regulation of glucose mediated signaling pathway\n",
      "    - positive regulation of miRNA transcription\n",
      "    - regulation of G1/S transition of mitotic cell cycle\n",
      "    - positive regulation of double-strand break repair\n",
      "    - regulation of nucleotide-excision repair\n",
      "  CC:\n",
      "    - kinetochore\n",
      "    - chromatin\n",
      "    - chromatin\n",
      "    - chromatin\n",
      "    - chromatin\n",
      "    - fibrillar center\n",
      "    - extracellular space\n",
      "    - nucleus\n",
      "    - nucleus\n",
      "    - nucleus\n",
      "    - nucleoplasm\n",
      "    - nucleoplasm\n",
      "    - nucleolus\n",
      "    - membrane\n",
      "    - nuclear matrix\n",
      "    - SWI/SNF complex\n",
      "    - SWI/SNF complex\n",
      "    - RSC-type complex\n",
      "    - protein-containing complex\n",
      "    - npBAF complex\n",
      "    - npBAF complex\n",
      "    - npBAF complex\n",
      "    - nBAF complex\n",
      "    - nBAF complex\n",
      "    - bBAF complex\n",
      "    - GBAF complex\n",
      "  MF:\n",
      "    - RNA polymerase I core promoter sequence-specific DNA binding\n",
      "    - transcription coregulator binding\n",
      "    - p53 binding\n",
      "    - DNA binding\n",
      "    - chromatin binding\n",
      "    - transcription coactivator activity\n",
      "    - transcription coactivator activity\n",
      "    - transcription coactivator activity\n",
      "    - transcription corepressor activity\n",
      "    - RNA binding\n",
      "    - helicase activity\n",
      "    - protein binding\n",
      "    - ATP binding\n",
      "    - ATP-dependent activity, acting on DNA\n",
      "    - hydrolase activity\n",
      "    - Tat protein binding\n",
      "    - nucleosomal DNA binding\n",
      "    - histone binding\n",
      "    - nuclear androgen receptor binding\n",
      "    - DNA polymerase binding\n",
      "    - ATP-dependent chromatin remodeler activity\n",
      "    - nucleosome array spacer activity\n",
      "UniProt: {'Swiss-Prot': 'P51532', 'TrEMBL': ['A7E2E1', 'Q9HBD4', 'B3KNW7', 'A0A2R8Y7S2', 'A0A2R8Y523', 'A0A2R8Y6N0', 'A0A2R8Y7R0', 'A0A2R8Y4R6', 'K7EP28', 'A0A2R8YF38', 'A0A2R8YFK5', 'A0A2R8Y866', 'A0A2R8Y526', 'K7EQF0', 'A0A2R8YFV8', 'A0A2R8YF58', 'A0A2R8YF80', 'A0A2R8YDA1', 'A0A2R8Y7F3', 'A0A2R8Y5K3', 'A0A994J6Y9', 'A0A2R8YGG3', 'A0A2R8Y583', 'A0A2R8YG32', 'A0A2R8Y4P4', 'A0A2R8Y4C5', 'A0A994J4Z2', 'A0A2R8Y6V2', 'A0A2R8YCY3', 'A0A2R8Y440', 'A0A2R8YGP5', 'A0A2R8Y7Y7']}\n",
      "\n",
      "Gene: IL22\n",
      "Symbol: IL22\n",
      "Name: interleukin 22\n",
      "Summary: This gene is a member of the IL10 family of cytokines that mediate cellular inflammatory responses. The encoded protein functions in antimicrobial defense at mucosal surfaces and in tissue repair. This protein also has pro-inflammatory properties and plays a role in in the pathogenesis of several intestinal diseases. The encoded protein is a crucial cytokine that regulates host immunity in infectious diseases, including COVID-19 (disease caused by SARS-CoV-2). [provided by RefSeq, Dec 2021].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - acute-phase response\n",
      "    - inflammatory response\n",
      "    - signal transduction\n",
      "    - negative regulation of inflammatory response\n",
      "    - negative regulation of inflammatory response\n",
      "    - response to glucocorticoid\n",
      "  CC:\n",
      "    - extracellular region\n",
      "    - extracellular space\n",
      "    - extracellular space\n",
      "  MF:\n",
      "    - cytokine activity\n",
      "    - cytokine activity\n",
      "    - protein binding\n",
      "    - interleukin-22 receptor binding\n",
      "UniProt: {'Swiss-Prot': 'Q9GZX6', 'TrEMBL': 'A0A7R8C389'}\n",
      "\n",
      "Gene: CSF3R\n",
      "Symbol: CSF3R\n",
      "Name: colony stimulating factor 3 receptor\n",
      "Summary: The protein encoded by this gene is the receptor for colony stimulating factor 3, a cytokine that controls the production, differentiation, and function of granulocytes. The encoded protein, which is a member of the family of cytokine receptors, may also function in some cell surface adhesion or recognition processes. Alternatively spliced transcript variants have been described. Mutations in this gene are a cause of Kostmann syndrome, also known as severe congenital neutropenia. [provided by RefSeq, Aug 2010].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - defense response\n",
      "    - cell adhesion\n",
      "    - signal transduction\n",
      "    - positive regulation of cell population proliferation\n",
      "    - cytokine-mediated signaling pathway\n",
      "    - neutrophil chemotaxis\n",
      "    - regulation of myeloid cell differentiation\n",
      "    - amelogenesis\n",
      "  CC:\n",
      "    - extracellular region\n",
      "    - lysosomal membrane\n",
      "    - plasma membrane\n",
      "    - external side of plasma membrane\n",
      "    - endocytic vesicle membrane\n",
      "    - receptor complex\n",
      "  MF:\n",
      "    - cytokine receptor activity\n",
      "    - protein binding\n",
      "    - cytokine binding\n",
      "    - signaling receptor activity\n",
      "    - granulocyte colony-stimulating factor binding\n",
      "UniProt: {'Swiss-Prot': 'Q99062', 'TrEMBL': ['E9PK56', 'H0YE86', 'A0A8V8TP76', 'A0A8V8TNA4']}\n",
      "\n",
      "Gene: GHR\n",
      "Symbol: GHR\n",
      "Name: growth hormone receptor\n",
      "Summary: This gene encodes a member of the type I cytokine receptor family, which is a transmembrane receptor for growth hormone. Binding of growth hormone to the receptor leads to receptor dimerization and the activation of an intra- and intercellular signal transduction pathway leading to growth. Mutations in this gene have been associated with Laron syndrome, also known as the growth hormone insensitivity syndrome (GHIS), a disorder characterized by short stature. In humans and rabbits, but not rodents, growth hormone binding protein (GHBP) is generated by proteolytic cleavage of the extracellular ligand-binding domain from the mature growth hormone receptor protein. Multiple alternatively spliced transcript variants have been found for this gene.[provided by RefSeq, Jun 2011].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - endocytosis\n",
      "    - cell surface receptor signaling pathway via JAK-STAT\n",
      "    - positive regulation of cell population proliferation\n",
      "    - response to gravity\n",
      "    - hormone-mediated signaling pathway\n",
      "    - cytokine-mediated signaling pathway\n",
      "    - taurine metabolic process\n",
      "    - receptor internalization\n",
      "    - receptor internalization\n",
      "    - response to food\n",
      "    - regulation of response to nutrient levels\n",
      "    - response to estradiol\n",
      "    - cellular response to insulin stimulus\n",
      "    - cellular response to hormone stimulus\n",
      "    - regulation of multicellular organism growth\n",
      "    - positive regulation of multicellular organism growth\n",
      "    - hormone metabolic process\n",
      "    - positive regulation of MAP kinase activity\n",
      "    - positive regulation of cell differentiation\n",
      "    - positive regulation of receptor signaling pathway via JAK-STAT\n",
      "    - response to cycloheximide\n",
      "    - response to cycloheximide\n",
      "    - insulin-like growth factor receptor signaling pathway\n",
      "    - response to glucocorticoid\n",
      "    - cartilage development involved in endochondral bone morphogenesis\n",
      "    - growth hormone receptor signaling pathway\n",
      "    - growth hormone receptor signaling pathway\n",
      "    - growth hormone receptor signaling pathway\n",
      "    - response to interleukin-1\n",
      "  CC:\n",
      "    - extracellular region\n",
      "    - extracellular space\n",
      "    - extracellular space\n",
      "    - cytosol\n",
      "    - cytosol\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - external side of plasma membrane\n",
      "    - cell surface\n",
      "    - membrane\n",
      "    - cytoplasmic ribonucleoprotein granule\n",
      "    - neuronal cell body\n",
      "    - receptor complex\n",
      "    - growth hormone receptor complex\n",
      "    - growth hormone receptor complex\n",
      "  MF:\n",
      "    - growth hormone receptor activity\n",
      "    - growth hormone receptor activity\n",
      "    - protein binding\n",
      "    - lipid binding\n",
      "    - peptide hormone binding\n",
      "    - peptide hormone binding\n",
      "    - growth factor binding\n",
      "    - protein kinase binding\n",
      "    - protein phosphatase binding\n",
      "    - cytokine binding\n",
      "    - SH2 domain binding\n",
      "    - identical protein binding\n",
      "    - protein homodimerization activity\n",
      "    - proline-rich region binding\n",
      "UniProt: {'Swiss-Prot': 'P10912', 'TrEMBL': ['A0A087X0H5', 'A0A087X162', 'E9PCN7', 'E7ES05']}\n",
      "\n",
      "Gene: PRLR\n",
      "Symbol: PRLR\n",
      "Name: prolactin receptor\n",
      "Summary: This gene encodes a receptor for the anterior pituitary hormone, prolactin, and belongs to the type I cytokine receptor family. Prolactin-dependent signaling occurs as the result of ligand-induced dimerization of the prolactin receptor. Several alternatively spliced transcript variants encoding different membrane-bound and soluble isoforms have been described for this gene, which may function to modulate the endocrine and autocrine effects of prolactin in normal tissue and cancer. [provided by RefSeq, Feb 2011].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - steroid biosynthetic process\n",
      "    - activation of transmembrane receptor protein tyrosine kinase activity\n",
      "    - cell surface receptor signaling pathway via JAK-STAT\n",
      "    - embryo implantation\n",
      "    - lactation\n",
      "    - positive regulation of cell population proliferation\n",
      "    - response to bacterium\n",
      "    - cytokine-mediated signaling pathway\n",
      "    - regulation of cell adhesion\n",
      "    - regulation of epithelial cell differentiation\n",
      "    - positive regulation of B cell proliferation\n",
      "    - prolactin signaling pathway\n",
      "    - activation of Janus kinase activity\n",
      "    - negative regulation of apoptotic process\n",
      "    - mammary gland epithelial cell differentiation\n",
      "    - prostate gland growth\n",
      "    - mammary gland alveolus development\n",
      "    - cellular response to granulocyte macrophage colony-stimulating factor stimulus\n",
      "    - positive regulation of cold-induced thermogenesis\n",
      "  CC:\n",
      "    - extracellular region\n",
      "    - plasma membrane\n",
      "    - external side of plasma membrane\n",
      "    - cell surface\n",
      "    - endosome lumen\n",
      "    - receptor complex\n",
      "  MF:\n",
      "    - prolactin receptor activity\n",
      "    - prolactin receptor activity\n",
      "    - protein binding\n",
      "    - lipid binding\n",
      "    - lipid binding\n",
      "    - peptide hormone binding\n",
      "    - peptide hormone binding\n",
      "    - protein kinase binding\n",
      "    - cytokine binding\n",
      "    - metal ion binding\n",
      "UniProt: {'Swiss-Prot': 'P16471', 'TrEMBL': ['D6R9E1', 'D6R9P5', 'D6RC67', 'D6RAN9', 'D6RJC8', 'D6R9V7', 'D6RD41']}\n",
      "\n",
      "Gene: CD40LG\n",
      "Symbol: CD40LG\n",
      "Name: CD40 ligand\n",
      "Summary: The protein encoded by this gene is expressed on the surface of T cells. It regulates B cell function by engaging CD40 on the B cell surface. A defect in this gene results in an inability to undergo immunoglobulin class switch and is associated with hyper-IgM syndrome. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - regulation of immunoglobulin production\n",
      "    - inflammatory response\n",
      "    - leukocyte cell-cell adhesion\n",
      "    - cell surface receptor signaling pathway\n",
      "    - integrin-mediated signaling pathway\n",
      "    - CD40 signaling pathway\n",
      "    - CD40 signaling pathway\n",
      "    - platelet activation\n",
      "    - B cell differentiation\n",
      "    - T cell costimulation\n",
      "    - positive regulation of interleukin-10 production\n",
      "    - positive regulation of interleukin-12 production\n",
      "    - positive regulation of interleukin-4 production\n",
      "    - B cell proliferation\n",
      "    - positive regulation of T cell proliferation\n",
      "    - negative regulation of apoptotic process\n",
      "    - positive regulation of canonical NF-kappaB signal transduction\n",
      "    - isotype switching\n",
      "    - positive regulation of NF-kappaB transcription factor activity\n",
      "    - positive regulation of endothelial cell apoptotic process\n",
      "    - positive regulation of extrinsic apoptotic signaling pathway\n",
      "  CC:\n",
      "    - extracellular space\n",
      "    - extracellular space\n",
      "    - Golgi apparatus\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - external side of plasma membrane\n",
      "    - cell surface\n",
      "    - membrane\n",
      "  MF:\n",
      "    - cytokine activity\n",
      "    - cytokine activity\n",
      "    - tumor necrosis factor receptor binding\n",
      "    - CD40 receptor binding\n",
      "    - CD40 receptor binding\n",
      "    - integrin binding\n",
      "    - protein binding\n",
      "    - protein serine/threonine kinase activator activity\n",
      "UniProt: {'Swiss-Prot': 'P29965', 'TrEMBL': ['A0A8Q3WL88', 'Q3L8U2', 'A0A8Q3WKP2']}\n",
      "\n",
      "Gene: CD40\n",
      "Symbol: CD40\n",
      "Name: CD40 molecule\n",
      "Summary: This gene is a member of the TNF-receptor superfamily. The encoded protein is a receptor on antigen-presenting cells of the immune system and is essential for mediating a broad variety of immune and inflammatory responses including T cell-dependent immunoglobulin class switching, memory B cell development, and germinal center formation. AT-hook transcription factor AKNA is reported to coordinately regulate the expression of this receptor and its ligand, which may be important for homotypic cell interactions. Adaptor protein TNFR2 interacts with this receptor and serves as a mediator of the signal transduction. The interaction of this receptor and its ligand is found to be necessary for amyloid-beta-induced microglial activation, and thus is thought to be an early event in Alzheimer disease pathogenesis. Mutations affecting this gene are the cause of autosomal recessive hyper-IgM immunodeficiency type 3 (HIGM3). Multiple alternatively spliced transcript variants of this gene encoding distinct isoforms have been reported. [provided by RefSeq, Nov 2014].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - immune response-regulating cell surface receptor signaling pathway\n",
      "    - intracellular calcium ion homeostasis\n",
      "    - inflammatory response\n",
      "    - cell surface receptor signaling pathway via JAK-STAT\n",
      "    - B cell mediated immunity\n",
      "    - CD40 signaling pathway\n",
      "    - CD40 signaling pathway\n",
      "    - platelet activation\n",
      "    - positive regulation of B cell proliferation\n",
      "    - positive regulation of interleukin-12 production\n",
      "    - response to cobalamin\n",
      "    - response to type II interferon\n",
      "    - cellular response to erythropoietin\n",
      "    - B cell proliferation\n",
      "    - B cell activation\n",
      "    - defense response to protozoan\n",
      "    - positive regulation of canonical NF-kappaB signal transduction\n",
      "    - positive regulation of canonical NF-kappaB signal transduction\n",
      "    - positive regulation of MAPK cascade\n",
      "    - phosphatidylinositol 3-kinase/protein kinase B signal transduction\n",
      "    - positive regulation of blood vessel endothelial cell migration\n",
      "    - positive regulation of angiogenesis\n",
      "    - positive regulation of transcription by RNA polymerase II\n",
      "    - positive regulation of transcription by RNA polymerase II\n",
      "    - positive regulation of isotype switching to IgG isotypes\n",
      "    - defense response to virus\n",
      "    - protein-containing complex assembly\n",
      "    - cellular response to lipopolysaccharide\n",
      "    - cellular response to mechanical stimulus\n",
      "    - cellular response to interleukin-1\n",
      "    - cellular response to tumor necrosis factor\n",
      "    - positive regulation of protein kinase C signaling\n",
      "    - positive regulation of interleukin-4-mediated signaling pathway\n",
      "    - positive regulation of endothelial cell apoptotic process\n",
      "  CC:\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - external side of plasma membrane\n",
      "    - cell surface\n",
      "    - CD40 receptor complex\n",
      "    - CD40 receptor complex\n",
      "    - neuronal cell body\n",
      "    - varicosity\n",
      "    - intracellular membrane-bounded organelle\n",
      "    - extracellular exosome\n",
      "  MF:\n",
      "    - antigen binding\n",
      "    - protein binding\n",
      "    - enzyme binding\n",
      "    - protein domain specific binding\n",
      "    - ubiquitin protein ligase binding\n",
      "    - signaling receptor activity\n",
      "UniProt: {'Swiss-Prot': 'P25942', 'TrEMBL': ['A0A0S2Z3C7', 'A0A8Q3SI60', 'Q6P2H9', 'A0A0S2Z349', 'A0A8Q3WKP3', 'A0A087X1D0', 'H0YE23', 'A0A8Q3WKP7']}\n",
      "\n",
      "Gene: TNFRSF13C\n",
      "Symbol: TNFRSF13C\n",
      "Name: TNF receptor superfamily member 13C\n",
      "Summary: B cell-activating factor (BAFF) enhances B-cell survival in vitro and is a regulator of the peripheral B-cell population. Overexpression of Baff in mice results in mature B-cell hyperplasia and symptoms of systemic lupus erythematosus (SLE). Also, some SLE patients have increased levels of BAFF in serum. Therefore, it has been proposed that abnormally high levels of BAFF may contribute to the pathogenesis of autoimmune diseases by enhancing the survival of autoreactive B cells. The protein encoded by this gene is a receptor for BAFF and is a type III transmembrane protein containing a single extracellular cysteine-rich domain. It is thought that this receptor is the principal receptor required for BAFF-mediated mature B-cell survival. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - adaptive immune response\n",
      "    - positive regulation of B cell proliferation\n",
      "    - T cell costimulation\n",
      "    - B cell costimulation\n",
      "    - tumor necrosis factor-mediated signaling pathway\n",
      "    - positive regulation of T cell proliferation\n",
      "  CC:\n",
      "    - plasma membrane\n",
      "    - external side of plasma membrane\n",
      "  MF:\n",
      "    - signaling receptor activity\n",
      "UniProt: {'Swiss-Prot': 'Q96RJ3', 'TrEMBL': 'Q5H8V1'}\n",
      "\n",
      "Gene: TNFSF13B\n",
      "Symbol: TNFSF13B\n",
      "Name: TNF superfamily member 13b\n",
      "Summary: The protein encoded by this gene is a cytokine that belongs to the tumor necrosis factor (TNF) ligand family. This cytokine is a ligand for receptors TNFRSF13B/TACI, TNFRSF17/BCMA, and TNFRSF13C/BAFFR. This cytokine is expressed in B cell lineage cells, and acts as a potent B cell activator. It has been also shown to play an important role in the proliferation and differentiation of B cells. Alternatively spliced transcript variants encoding distinct isoforms have been identified. [provided by RefSeq, Mar 2011].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - B cell homeostasis\n",
      "    - transitional one stage B cell differentiation\n",
      "    - germinal center formation\n",
      "    - positive regulation of germinal center formation\n",
      "    - signal transduction\n",
      "    - positive regulation of B cell proliferation\n",
      "    - positive regulation of B cell proliferation\n",
      "    - positive regulation of B cell proliferation\n",
      "    - T cell costimulation\n",
      "    - B cell costimulation\n",
      "    - tumor necrosis factor-mediated signaling pathway\n",
      "    - T cell proliferation\n",
      "    - B cell proliferation\n",
      "    - positive regulation of T cell proliferation\n",
      "    - skin development\n",
      "  CC:\n",
      "    - extracellular region\n",
      "    - extracellular space\n",
      "    - cytoplasm\n",
      "    - plasma membrane\n",
      "    - focal adhesion\n",
      "    - membrane\n",
      "    - intracellular membrane-bounded organelle\n",
      "    - perinuclear region of cytoplasm\n",
      "  MF:\n",
      "    - signaling receptor binding\n",
      "    - cytokine activity\n",
      "    - tumor necrosis factor receptor binding\n",
      "    - protein binding\n",
      "    - receptor ligand activity\n",
      "UniProt: {'Swiss-Prot': 'Q9Y275', 'TrEMBL': 'A0A0U5J7Q1'}\n",
      "\n",
      "Gene: CSK\n",
      "Symbol: CSK\n",
      "Name: C-terminal Src kinase\n",
      "Summary: The protein encoded by this gene is involved in multiple pathways, including the regulation of Src family kinases. It plays an important role in T-cell activation through its association with the protein encoded by the protein tyrosine phosphatase, non-receptor type 22 (PTPN22) gene. This protein also phosphorylates C-terminal tyrosine residues on multiple substrates, including the protein encoded by the SRC proto-oncogene, non-receptor tyrosine kinase gene. Phosphorylation suppresses the kinase activity of the Src family tyrosine kinases. An intronic polymorphism (rs34933034) in this gene has been found to affect B-cell activation and is associated with systemic lupus erythematosus (SLE). Alternative splicing results in multiple transcript variants. [provided by RefSeq, Aug 2017].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - adaptive immune response\n",
      "    - protein phosphorylation\n",
      "    - negative regulation of cell population proliferation\n",
      "    - negative regulation of low-density lipoprotein particle clearance\n",
      "    - T cell costimulation\n",
      "    - negative regulation of interleukin-6 production\n",
      "    - adherens junction organization\n",
      "    - negative regulation of Golgi to plasma membrane protein transport\n",
      "    - negative regulation of bone resorption\n",
      "    - oligodendrocyte differentiation\n",
      "    - negative regulation of phagocytosis\n",
      "    - T cell receptor signaling pathway\n",
      "    - regulation of Fc receptor mediated stimulatory signaling pathway\n",
      "    - negative regulation of ERK1 and ERK2 cascade\n",
      "    - cellular response to peptide hormone stimulus\n",
      "  CC:\n",
      "    - cytoplasm\n",
      "    - cytoplasm\n",
      "    - cytosol\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - cell-cell junction\n",
      "    - extracellular exosome\n",
      "  MF:\n",
      "    - protein tyrosine kinase activity\n",
      "    - non-membrane spanning protein tyrosine kinase activity\n",
      "    - protein binding\n",
      "    - ATP binding\n",
      "    - protein phosphatase binding\n",
      "    - protein kinase A catalytic subunit binding\n",
      "    - identical protein binding\n",
      "    - metal ion binding\n",
      "    - proline-rich region binding\n",
      "    - protein tyrosine kinase binding\n",
      "UniProt: {'Swiss-Prot': 'P41240', 'TrEMBL': ['A8K3B6', 'B2R6Q4', 'H3BU69', 'H3BN15', 'H3BUM9']}\n",
      "\n",
      "Gene: MAP3K7\n",
      "Symbol: MAP3K7\n",
      "Name: mitogen-activated protein kinase kinase kinase 7\n",
      "Summary: The protein encoded by this gene is a member of the serine/threonine protein kinase family. This kinase mediates the signaling transduction induced by TGF beta and morphogenetic protein (BMP), and controls a variety of cell functions including transcription regulation and apoptosis. In response to IL-1, this protein forms a kinase complex including TRAF6, MAP3K7P1/TAB1 and MAP3K7P2/TAB2; this complex is required for the activation of nuclear factor kappa B. This kinase can also activate MAPK8/JNK, MAP2K4/MKK4, and thus plays a role in the cell response to environmental stresses. Four alternatively spliced transcript variants encoding distinct isoforms have been reported. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - MAPK cascade\n",
      "    - stimulatory C-type lectin receptor signaling pathway\n",
      "    - positive regulation of T cell cytokine production\n",
      "    - cytoplasmic pattern recognition receptor signaling pathway\n",
      "    - MyD88-dependent toll-like receptor signaling pathway\n",
      "    - chromatin remodeling\n",
      "    - inflammatory response\n",
      "    - immune response\n",
      "    - transforming growth factor beta receptor signaling pathway\n",
      "    - canonical NF-kappaB signal transduction\n",
      "    - canonical NF-kappaB signal transduction\n",
      "    - I-kappaB phosphorylation\n",
      "    - JNK cascade\n",
      "    - JNK cascade\n",
      "    - JNK cascade\n",
      "    - negative regulation of gene expression\n",
      "    - positive regulation of macroautophagy\n",
      "    - positive regulation of interleukin-2 production\n",
      "    - toll-like receptor 3 signaling pathway\n",
      "    - toll-like receptor 4 signaling pathway\n",
      "    - TRIF-dependent toll-like receptor signaling pathway\n",
      "    - nucleotide-binding domain, leucine rich repeat containing receptor signaling pathway\n",
      "    - p38MAPK cascade\n",
      "    - p38MAPK cascade\n",
      "    - Fc-epsilon receptor signaling pathway\n",
      "    - interleukin-33-mediated signaling pathway\n",
      "    - interleukin-17A-mediated signaling pathway\n",
      "    - defense response to bacterium\n",
      "    - positive regulation of canonical NF-kappaB signal transduction\n",
      "    - positive regulation of canonical NF-kappaB signal transduction\n",
      "    - positive regulation of canonical NF-kappaB signal transduction\n",
      "    - anoikis\n",
      "    - positive regulation of JUN kinase activity\n",
      "    - positive regulation of cell cycle\n",
      "    - positive regulation of cell size\n",
      "    - T cell receptor signaling pathway\n",
      "    - T cell receptor signaling pathway\n",
      "    - stress-activated MAPK cascade\n",
      "    - interleukin-1-mediated signaling pathway\n",
      "    - interleukin-1-mediated signaling pathway\n",
      "    - cellular response to tumor necrosis factor\n",
      "    - cellular response to hypoxia\n",
      "    - positive regulation of cGAS/STING signaling pathway\n",
      "    - positive regulation of non-canonical NF-kappaB signal transduction\n",
      "    - cellular response to angiotensin\n",
      "    - positive regulation of vascular associated smooth muscle cell proliferation\n",
      "    - positive regulation of vascular associated smooth muscle cell migration\n",
      "  CC:\n",
      "    - nucleus\n",
      "    - cytoplasm\n",
      "    - cytoplasm\n",
      "    - endoplasmic reticulum membrane\n",
      "    - cytosol\n",
      "    - cytosol\n",
      "    - cytosol\n",
      "    - plasma membrane\n",
      "    - endosome membrane\n",
      "    - ATAC complex\n",
      "  MF:\n",
      "    - magnesium ion binding\n",
      "    - transcription coactivator binding\n",
      "    - protein serine/threonine kinase activity\n",
      "    - protein serine/threonine kinase activity\n",
      "    - protein serine/threonine kinase activity\n",
      "    - JUN kinase kinase kinase activity\n",
      "    - MAP kinase activity\n",
      "    - MAP kinase kinase kinase activity\n",
      "    - MAP kinase kinase kinase activity\n",
      "    - MAP kinase kinase kinase activity\n",
      "    - type II transforming growth factor beta receptor binding\n",
      "    - protein binding\n",
      "    - ATP binding\n",
      "    - MAP kinase kinase kinase kinase activity\n",
      "    - receptor tyrosine kinase binding\n",
      "    - ubiquitin protein ligase binding\n",
      "    - histone kinase activity\n",
      "    - identical protein binding\n",
      "    - scaffold protein binding\n",
      "    - protein serine kinase activity\n",
      "    - protein serine/threonine kinase binding\n",
      "    - DNA-binding transcription factor binding\n",
      "    - linear polyubiquitin binding\n",
      "UniProt: {'Swiss-Prot': 'O43318', 'TrEMBL': ['A0A8V8TQF8', 'A0A8V8TQF4', 'A0A8V8TPW3', 'A0A8V8TRC0', 'A0A8V8TQ28', 'A0A8V8TR14', 'A0A8V8TQE8', 'A0A8V8TRC2', 'A0A8V8TQF2', 'A0A8V8TRC6', 'A0A8V8TR17', 'A0A8V8TPV8', 'A0A8V8TQL7']}\n",
      "\n",
      "Gene: ICOSLG\n",
      "Symbol: ICOSLG\n",
      "Name: inducible T cell costimulator ligand\n",
      "Summary: Enables identical protein binding activity. Predicted to be involved in T cell receptor signaling pathway and positive regulation of interleukin-4 production. Located in cytoplasmic ribonucleoprotein granule and plasma membrane. [provided by Alliance of Genome Resources, Apr 2022]\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - regulation of cytokine production\n",
      "    - adaptive immune response\n",
      "    - defense response\n",
      "    - hyperosmotic response\n",
      "    - signal transduction\n",
      "    - positive regulation of activated T cell proliferation\n",
      "    - T cell activation\n",
      "    - B cell activation\n",
      "    - T cell receptor signaling pathway\n",
      "    - T follicular helper cell differentiation\n",
      "  CC:\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - external side of plasma membrane\n",
      "    - membrane\n",
      "    - intracellular membrane-bounded organelle\n",
      "    - extracellular exosome\n",
      "  MF:\n",
      "    - signaling receptor binding\n",
      "    - signaling receptor binding\n",
      "    - protein binding\n",
      "    - identical protein binding\n",
      "    - receptor ligand activity\n",
      "UniProt: {'Swiss-Prot': 'O75144', 'TrEMBL': ['A0N0L8', 'B7Z1W8', 'A0A2R8Y5W7', 'K4DIA0', 'A0A8V8TR62', 'A0A096LPE5', 'A0A8V8TQW2', 'A0A8V8TQ73', 'A0A8V8TQ42']}\n",
      "\n",
      "Gene: BTLA\n",
      "Symbol: BTLA\n",
      "Name: B and T lymphocyte associated\n",
      "Summary: This gene encodes a member of the immunoglobulin superfamily. The encoded protein contains a single immunoglobulin (Ig) domain and is a receptor that relays inhibitory signals to suppress the immune response. Alternative splicing results in multiple transcript variants. Polymorphisms in this gene have been associated with an increased risk of rheumatoid arthritis. [provided by RefSeq, Aug 2011].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - adaptive immune response\n",
      "    - immune response-regulating cell surface receptor signaling pathway\n",
      "    - negative regulation of T cell proliferation\n",
      "  CC:\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "  MF:\n",
      "    - protein binding\n",
      "    - signaling receptor activity\n",
      "    - signaling receptor activity\n",
      "UniProt: {'Swiss-Prot': 'Q7Z6A9'}\n",
      "\n",
      "Gene: CD80\n",
      "Symbol: CD80\n",
      "Name: CD80 molecule\n",
      "Summary: The protein encoded by this gene is a membrane receptor that is activated by the binding of CD28 or CTLA-4. The activated protein induces T-cell proliferation and cytokine production. This protein can act as a receptor for adenovirus subgroup B and may play a role in lupus neuropathy. [provided by RefSeq, Aug 2011].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - negative regulation of T cell mediated immunity\n",
      "    - immune response\n",
      "    - cell surface receptor signaling pathway\n",
      "    - positive regulation of signal transduction\n",
      "    - T cell costimulation\n",
      "    - positive regulation of granulocyte macrophage colony-stimulating factor production\n",
      "    - positive regulation of interleukin-2 production\n",
      "    - intracellular signal transduction\n",
      "    - positive regulation of T cell proliferation\n",
      "    - T cell activation\n",
      "    - negative regulation of T cell proliferation\n",
      "    - positive regulation of T-helper 1 cell differentiation\n",
      "    - positive regulation of DNA-templated transcription\n",
      "    - symbiont entry into host cell\n",
      "    - positive regulation of peptidyl-tyrosine phosphorylation\n",
      "    - cellular response to lipopolysaccharide\n",
      "  CC:\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - external side of plasma membrane\n",
      "    - cell surface\n",
      "    - protein complex involved in cell adhesion\n",
      "  MF:\n",
      "    - virus receptor activity\n",
      "    - protein binding\n",
      "    - coreceptor activity\n",
      "    - receptor ligand activity\n",
      "UniProt: {'Swiss-Prot': 'P33681', 'TrEMBL': 'A0N0P2'}\n",
      "\n",
      "Gene: CD86\n",
      "Symbol: CD86\n",
      "Name: CD86 molecule\n",
      "Summary: This gene encodes a type I membrane protein that is a member of the immunoglobulin superfamily. This protein is expressed by antigen-presenting cells, and it is the ligand for two proteins at the cell surface of T cells, CD28 antigen and cytotoxic T-lymphocyte-associated protein 4. Binding of this protein with CD28 antigen is a costimulatory signal for activation of the T-cell. Binding of this protein with cytotoxic T-lymphocyte-associated protein 4 negatively regulates T-cell activation and diminishes the immune response. Alternative splicing results in several transcript variants encoding different isoforms.[provided by RefSeq, May 2011].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - adaptive immune response\n",
      "    - positive regulation of immunoglobulin production\n",
      "    - immune response\n",
      "    - cell surface receptor signaling pathway\n",
      "    - positive regulation of cell population proliferation\n",
      "    - T cell costimulation\n",
      "    - positive regulation of interleukin-2 production\n",
      "    - positive regulation of interleukin-4 production\n",
      "    - positive regulation of lymphotoxin A production\n",
      "    - positive regulation of T cell proliferation\n",
      "    - T cell activation\n",
      "    - B cell activation\n",
      "    - negative regulation of T cell proliferation\n",
      "    - positive regulation of T-helper 2 cell differentiation\n",
      "    - positive regulation of DNA-templated transcription\n",
      "    - symbiont entry into host cell\n",
      "    - cellular response to lipopolysaccharide\n",
      "    - positive regulation of non-canonical NF-kappaB signal transduction\n",
      "  CC:\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - external side of plasma membrane\n",
      "    - cell surface\n",
      "    - centriolar satellite\n",
      "    - extracellular exosome\n",
      "    - extracellular exosome\n",
      "  MF:\n",
      "    - virus receptor activity\n",
      "    - protein binding\n",
      "    - coreceptor activity\n",
      "    - signaling receptor activity\n",
      "    - receptor ligand activity\n",
      "UniProt: {'Swiss-Prot': 'P42081', 'TrEMBL': ['A8K632', 'A0A0X9R4E0', 'C9JXS1', 'H7C4F8']}\n",
      "\n",
      "Gene: NFKBIA\n",
      "Symbol: NFKBIA\n",
      "Name: NFKB inhibitor alpha\n",
      "Summary: This gene encodes a member of the NF-kappa-B inhibitor family, which contain multiple ankrin repeat domains. The encoded protein interacts with REL dimers to inhibit NF-kappa-B/REL complexes which are involved in inflammatory responses. The encoded protein moves between the cytoplasm and the nucleus via a nuclear localization signal and CRM1-mediated nuclear export. Mutations in this gene have been found in ectodermal dysplasia anhidrotic with T-cell immunodeficiency autosomal dominant disease. [provided by RefSeq, Aug 2011].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - negative regulation of transcription by RNA polymerase II\n",
      "    - protein import into nucleus\n",
      "    - Notch signaling pathway\n",
      "    - canonical NF-kappaB signal transduction\n",
      "    - negative regulation of macrophage derived foam cell differentiation\n",
      "    - negative regulation of lipid storage\n",
      "    - signal transduction involved in regulation of gene expression\n",
      "    - lipopolysaccharide-mediated signaling pathway\n",
      "    - negative regulation of NF-kappaB transcription factor activity\n",
      "    - negative regulation of cholesterol transport\n",
      "    - response to muramyl dipeptide\n",
      "    - tumor necrosis factor-mediated signaling pathway\n",
      "    - toll-like receptor 4 signaling pathway\n",
      "    - response to muscle stretch\n",
      "    - non-canonical NF-kappaB signal transduction\n",
      "    - regulation of cell population proliferation\n",
      "    - negative regulation of protein import into nucleus\n",
      "    - negative regulation of canonical NF-kappaB signal transduction\n",
      "    - negative regulation of canonical NF-kappaB signal transduction\n",
      "    - response to exogenous dsRNA\n",
      "    - negative regulation of myeloid cell differentiation\n",
      "    - negative regulation of Notch signaling pathway\n",
      "    - positive regulation of transcription by RNA polymerase II\n",
      "    - positive regulation of inflammatory response\n",
      "    - B cell receptor signaling pathway\n",
      "    - positive regulation of transcription initiation by RNA polymerase II\n",
      "    - cellular response to cold\n",
      "    - nucleotide-binding oligomerization domain containing 1 signaling pathway\n",
      "    - nucleotide-binding oligomerization domain containing 2 signaling pathway\n",
      "    - interleukin-1-mediated signaling pathway\n",
      "    - negative regulation of cytokine production involved in inflammatory response\n",
      "  CC:\n",
      "    - nucleus\n",
      "    - nucleoplasm\n",
      "    - cytoplasm\n",
      "    - cytoplasm\n",
      "    - cytosol\n",
      "    - cytosol\n",
      "    - cytosol\n",
      "    - plasma membrane\n",
      "    - I-kappaB/NF-kappaB complex\n",
      "  MF:\n",
      "    - protein binding\n",
      "    - nuclear localization sequence binding\n",
      "    - enzyme binding\n",
      "    - ubiquitin protein ligase binding\n",
      "    - identical protein binding\n",
      "    - NF-kappaB binding\n",
      "    - NF-kappaB binding\n",
      "    - NF-kappaB binding\n",
      "    - protein sequestering activity\n",
      "    - protein sequestering activity\n",
      "    - protein sequestering activity\n",
      "    - transcription regulator inhibitor activity\n",
      "UniProt: {'Swiss-Prot': 'P25963', 'TrEMBL': ['G3V3K1', 'A0A8V8TLE7', 'A0A8V8TLC3', 'G3V3I4', 'G3V2A2', 'G3V286']}\n",
      "\n",
      "Gene: KLRB1\n",
      "Symbol: KLRB1\n",
      "Name: killer cell lectin like receptor B1\n",
      "Summary: Natural killer (NK) cells are lymphocytes that mediate cytotoxicity and secrete cytokines after immune stimulation. Several genes of the C-type lectin superfamily, including the rodent NKRP1 family of glycoproteins, are expressed by NK cells and may be involved in the regulation of NK cell function. The KLRB1 protein contains an extracellular domain with several motifs characteristic of C-type lectins, a transmembrane domain, and a cytoplasmic domain. The KLRB1 protein is classified as a type II membrane protein because it has an external C terminus. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - cell surface receptor signaling pathway\n",
      "    - regulation of natural killer cell mediated cytotoxicity\n",
      "  CC:\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - cell surface\n",
      "  MF:\n",
      "    - transmembrane signaling receptor activity\n",
      "    - protein binding\n",
      "    - carbohydrate binding\n",
      "    - signaling receptor activity\n",
      "UniProt: {'Swiss-Prot': 'Q12918'}\n",
      "\n",
      "Gene: MAP2K2\n",
      "Symbol: MAP2K2\n",
      "Name: mitogen-activated protein kinase kinase 2\n",
      "Summary: The protein encoded by this gene is a dual specificity protein kinase that belongs to the MAP kinase kinase family. This kinase is known to play a critical role in mitogen growth factor signal transduction. It phosphorylates and thus activates MAPK1/ERK2 and MAPK2/ERK3. The activation of this kinase itself is dependent on the Ser/Thr phosphorylation by MAP kinase kinase kinases. Mutations in this gene cause cardiofaciocutaneous syndrome (CFC syndrome), a disease characterized by heart defects, cognitive disability, and distinctive facial features similar to those found in Noonan syndrome. The inhibition or degradation of this kinase is also found to be involved in the pathogenesis of Yersinia and anthrax. A pseudogene, which is located on chromosome 7, has been identified for this gene. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - MAPK cascade\n",
      "    - JNK cascade\n",
      "    - heart development\n",
      "    - positive regulation of gene expression\n",
      "    - Schwann cell development\n",
      "    - thyroid gland development\n",
      "    - regulation of stress-activated MAPK cascade\n",
      "    - peptidyl-serine autophosphorylation\n",
      "    - ERBB2-ERBB3 signaling pathway\n",
      "    - myelination\n",
      "    - positive regulation of DNA-templated transcription\n",
      "    - insulin-like growth factor receptor signaling pathway\n",
      "    - thymus development\n",
      "    - regulation of axon regeneration\n",
      "    - positive regulation of axonogenesis\n",
      "    - face development\n",
      "    - trachea formation\n",
      "    - epithelial cell proliferation involved in lung morphogenesis\n",
      "    - ERK1 and ERK2 cascade\n",
      "    - ERK1 and ERK2 cascade\n",
      "    - positive regulation of protein serine/threonine kinase activity\n",
      "    - regulation of Golgi inheritance\n",
      "    - positive regulation of cell motility\n",
      "    - regulation of early endosome to late endosome transport\n",
      "  CC:\n",
      "    - extracellular region\n",
      "    - nucleus\n",
      "    - mitochondrion\n",
      "    - early endosome\n",
      "    - late endosome\n",
      "    - peroxisomal membrane\n",
      "    - endoplasmic reticulum\n",
      "    - Golgi apparatus\n",
      "    - Golgi apparatus\n",
      "    - cytosol\n",
      "    - cytosol\n",
      "    - microtubule\n",
      "    - cell-cell junction\n",
      "    - focal adhesion\n",
      "    - cytoplasmic side of plasma membrane\n",
      "    - perinuclear region of cytoplasm\n",
      "  MF:\n",
      "    - protein serine/threonine kinase activity\n",
      "    - MAP kinase kinase activity\n",
      "    - MAP kinase kinase activity\n",
      "    - protein serine/threonine/tyrosine kinase activity\n",
      "    - protein tyrosine kinase activity\n",
      "    - MAP-kinase scaffold activity\n",
      "    - protein binding\n",
      "    - ATP binding\n",
      "    - JUN kinase kinase activity\n",
      "    - PDZ domain binding\n",
      "    - protein serine/threonine kinase activator activity\n",
      "    - metal ion binding\n",
      "    - scaffold protein binding\n",
      "    - protein serine kinase activity\n",
      "UniProt: {'Swiss-Prot': 'P36507', 'TrEMBL': 'M0R1B6'}\n",
      "\n",
      "Gene: MAP2K1\n",
      "Symbol: MAP2K1\n",
      "Name: mitogen-activated protein kinase kinase 1\n",
      "Summary: The protein encoded by this gene is a member of the dual specificity protein kinase family, which acts as a mitogen-activated protein (MAP) kinase kinase. MAP kinases, also known as extracellular signal-regulated kinases (ERKs), act as an integration point for multiple biochemical signals. This protein kinase lies upstream of MAP kinases and stimulates the enzymatic activity of MAP kinases upon wide variety of extra- and intracellular signals. As an essential component of MAP kinase signal transduction pathway, this kinase is involved in many cellular processes such as proliferation, differentiation, transcription regulation and development. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - MAPK cascade\n",
      "    - chemotaxis\n",
      "    - signal transduction\n",
      "    - JNK cascade\n",
      "    - heart development\n",
      "    - negative regulation of cell population proliferation\n",
      "    - positive regulation of gene expression\n",
      "    - Schwann cell development\n",
      "    - cerebellar cortex formation\n",
      "    - neuron differentiation\n",
      "    - keratinocyte differentiation\n",
      "    - thyroid gland development\n",
      "    - regulation of stress-activated MAPK cascade\n",
      "    - endodermal cell differentiation\n",
      "    - ERBB2-ERBB3 signaling pathway\n",
      "    - myelination\n",
      "    - type B pancreatic cell proliferation\n",
      "    - positive regulation of DNA-templated transcription\n",
      "    - insulin-like growth factor receptor signaling pathway\n",
      "    - thymus development\n",
      "    - regulation of axon regeneration\n",
      "    - cell motility\n",
      "    - positive regulation of axonogenesis\n",
      "    - Bergmann glial cell differentiation\n",
      "    - face development\n",
      "    - trachea formation\n",
      "    - epithelial cell proliferation involved in lung morphogenesis\n",
      "    - placenta blood vessel development\n",
      "    - labyrinthine layer development\n",
      "    - ERK1 and ERK2 cascade\n",
      "    - positive regulation of ERK1 and ERK2 cascade\n",
      "    - positive regulation of protein serine/threonine kinase activity\n",
      "    - regulation of Golgi inheritance\n",
      "    - cellular senescence\n",
      "    - positive regulation of endodermal cell differentiation\n",
      "    - regulation of early endosome to late endosome transport\n",
      "  CC:\n",
      "    - nucleus\n",
      "    - mitochondrion\n",
      "    - early endosome\n",
      "    - late endosome\n",
      "    - endoplasmic reticulum\n",
      "    - Golgi apparatus\n",
      "    - centrosome\n",
      "    - cytosol\n",
      "    - cytosol\n",
      "    - plasma membrane\n",
      "    - focal adhesion\n",
      "    - ciliary basal body\n",
      "    - intracellular membrane-bounded organelle\n",
      "  MF:\n",
      "    - protein kinase activity\n",
      "    - protein serine/threonine kinase activity\n",
      "    - MAP kinase kinase activity\n",
      "    - MAP kinase kinase activity\n",
      "    - protein serine/threonine/tyrosine kinase activity\n",
      "    - protein tyrosine kinase activity\n",
      "    - MAP-kinase scaffold activity\n",
      "    - protein binding\n",
      "    - protein binding\n",
      "    - ATP binding\n",
      "    - JUN kinase kinase activity\n",
      "    - protein kinase activator activity\n",
      "    - protein serine/threonine kinase activator activity\n",
      "    - scaffold protein binding\n",
      "    - protein serine kinase activity\n",
      "UniProt: {'Swiss-Prot': 'Q02750', 'TrEMBL': ['B4DFY5', 'A4QPA9', 'A0A8I5QJL9', 'A0A8I5QJC1', 'A0A8I5KRX5', 'A0A8I5QJ96', 'A0A8I5KXN4', 'A0A8I5KVF6', 'A0A8I5KYS7', 'H3BRW9', 'A0A8I5QJP8', 'A0A8I5KYB4']}\n",
      "\n",
      "Gene: CDK9\n",
      "Symbol: CDK9\n",
      "Name: cyclin dependent kinase 9\n",
      "Summary: The protein encoded by this gene is a member of the cyclin-dependent protein kinase (CDK) family. CDK family members are highly similar to the gene products of S. cerevisiae cdc28, and S. pombe cdc2, and known as important cell cycle regulators. This kinase was found to be a component of the multiprotein complex TAK/P-TEFb, which is an elongation factor for RNA polymerase II-directed transcription and functions by phosphorylating the C-terminal domain of the largest subunit of RNA polymerase II. This protein forms a complex with and is regulated by its regulatory subunit cyclin T or cyclin K. HIV-1 Tat protein was found to interact with this protein and cyclin T, which suggested a possible involvement of this protein in AIDS. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - DNA repair\n",
      "    - regulation of DNA repair\n",
      "    - transcription by RNA polymerase II\n",
      "    - transcription initiation at RNA polymerase II promoter\n",
      "    - transcription elongation by RNA polymerase II\n",
      "    - transcription elongation by RNA polymerase II\n",
      "    - protein phosphorylation\n",
      "    - regulation of mitotic cell cycle\n",
      "    - cell population proliferation\n",
      "    - replication fork processing\n",
      "    - regulation of mRNA 3'-end processing\n",
      "    - positive regulation of transcription elongation by RNA polymerase II\n",
      "    - positive regulation of transcription elongation by RNA polymerase II\n",
      "    - positive regulation by host of viral transcription\n",
      "    - positive regulation by host of viral transcription\n",
      "    - positive regulation of transcription by RNA polymerase II\n",
      "    - regulation of muscle cell differentiation\n",
      "    - nucleus localization\n",
      "    - regulation of cell cycle\n",
      "    - cellular response to cytokine stimulus\n",
      "    - negative regulation of protein localization to chromatin\n",
      "    - positive regulation of protein localization to chromatin\n",
      "    - transcription elongation-coupled chromatin remodeling\n",
      "  CC:\n",
      "    - nucleus\n",
      "    - nucleus\n",
      "    - nucleus\n",
      "    - nucleoplasm\n",
      "    - nucleoplasm\n",
      "    - transcription elongation factor complex\n",
      "    - cyclin/CDK positive transcription elongation factor complex\n",
      "    - cyclin/CDK positive transcription elongation factor complex\n",
      "    - membrane\n",
      "    - PML body\n",
      "    - cytoplasmic ribonucleoprotein granule\n",
      "    - P-TEFb complex\n",
      "  MF:\n",
      "    - RNA polymerase II cis-regulatory region sequence-specific DNA binding\n",
      "    - transcription coactivator binding\n",
      "    - DNA binding\n",
      "    - chromatin binding\n",
      "    - transcription elongation factor activity\n",
      "    - protein kinase activity\n",
      "    - protein serine/threonine kinase activity\n",
      "    - protein serine/threonine kinase activity\n",
      "    - cyclin-dependent protein serine/threonine kinase activity\n",
      "    - cyclin-dependent protein serine/threonine kinase activity\n",
      "    - protein binding\n",
      "    - ATP binding\n",
      "    - RNA polymerase II CTD heptapeptide repeat kinase activity\n",
      "    - RNA polymerase II CTD heptapeptide repeat kinase activity\n",
      "    - kinase activity\n",
      "    - protein kinase binding\n",
      "    - 7SK snRNA binding\n",
      "    - protein serine kinase activity\n",
      "    - RNA polymerase II CTD heptapeptide repeat Y1 kinase activity\n",
      "    - RNA polymerase II CTD heptapeptide repeat S2 kinase activity\n",
      "    - RNA polymerase II CTD heptapeptide repeat T4 kinase activity\n",
      "    - RNA polymerase II CTD heptapeptide repeat S5 kinase activity\n",
      "    - RNA polymerase II CTD heptapeptide repeat S7 kinase activity\n",
      "UniProt: {'Swiss-Prot': 'P50750', 'TrEMBL': 'X6RE90'}\n",
      "\n",
      "Gene: CCNT1\n",
      "Symbol: CCNT1\n",
      "Name: cyclin T1\n",
      "Summary: This gene encodes a member of the highly conserved cyclin C subfamily. The encoded protein tightly associates with cyclin-dependent kinase 9, and is a major subunit of positive transcription elongation factor b (p-TEFb). In humans, there are multiple forms of positive transcription elongation factor b, which may include one of several different cyclins along with cyclin-dependent kinase 9. The complex containing the encoded cyclin and cyclin-dependent kinase 9 acts as a cofactor of human immunodeficiency virus type 1 (HIV-1) Tat protein, and is both necessary and sufficient for full activation of viral transcription. This cyclin and its kinase partner are also involved in triggering transcript elongation through phosphorylation of the carboxy-terminal domain of the largest RNA polymerase II subunit. Overexpression of this gene is implicated in tumor growth. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Apr 2013].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - regulation of cyclin-dependent protein serine/threonine kinase activity\n",
      "    - transcription by RNA polymerase II\n",
      "    - response to xenobiotic stimulus\n",
      "    - positive regulation of DNA-templated transcription, elongation\n",
      "    - positive regulation of transcription elongation by RNA polymerase II\n",
      "    - positive regulation by host of viral transcription\n",
      "    - positive regulation by host of viral transcription\n",
      "    - positive regulation of transcription by RNA polymerase II\n",
      "    - positive regulation of transcription by RNA polymerase II\n",
      "    - cell division\n",
      "  CC:\n",
      "    - nucleus\n",
      "    - nucleus\n",
      "    - nucleus\n",
      "    - nucleoplasm\n",
      "    - nucleoplasm\n",
      "    - nucleolus\n",
      "    - cytosol\n",
      "    - cyclin/CDK positive transcription elongation factor complex\n",
      "    - cyclin/CDK positive transcription elongation factor complex\n",
      "    - cyclin/CDK positive transcription elongation factor complex\n",
      "    - P-TEFb complex\n",
      "  MF:\n",
      "    - transcription cis-regulatory region binding\n",
      "    - DNA binding\n",
      "    - chromatin binding\n",
      "    - protein binding\n",
      "    - protein kinase binding\n",
      "    - cyclin-dependent protein serine/threonine kinase activator activity\n",
      "    - cyclin-dependent protein serine/threonine kinase activator activity\n",
      "    - RNA polymerase binding\n",
      "    - 7SK snRNA binding\n",
      "    - DNA-binding transcription factor binding\n",
      "    - molecular condensate scaffold activity\n",
      "UniProt: {'Swiss-Prot': 'O60563', 'TrEMBL': 'A8K4M5'}\n",
      "\n",
      "Gene: CD19\n",
      "Symbol: CD19\n",
      "Name: CD19 molecule\n",
      "Summary: This gene encodes a member of the immunoglobulin gene superfamily. Expression of this cell surface protein is restricted to B cell lymphocytes. This protein is a reliable marker for pre-B cells but its expression diminishes during terminal B cell differentiation in antibody secreting plasma cells. The protein has two N-terminal extracellular Ig-like domains separated by a non-Ig-like domain, a hydrophobic transmembrane domain, and a large C-terminal cytoplasmic domain. This protein forms a complex with several membrane proteins including complement receptor type 2 (CD21) and tetraspanin (CD81) and this complex reduces the threshold for antigen-initiated B cell activation. Activation of this B-cell antigen receptor complex activates the phosphatidylinositol 3-kinase signalling pathway and the subsequent release of intracellular stores of calcium ions. This protein is a target of chimeric antigen receptor (CAR) T-cells used in the treatment of lymphoblastic leukemia. Mutations in this gene are associated with the disease common variable immunodeficiency 3 (CVID3) which results in a failure of B-cell differentiation and impaired secretion of immunoglobulins. CVID3 is characterized by hypogammaglobulinemia, an inability to mount an antibody response to antigen, and recurrent bacterial infections. Alternative splicing results in multiple transcript variants encoding distinct isoforms. [provided by RefSeq, Jul 2020].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - B-1 B cell differentiation\n",
      "    - B cell proliferation involved in immune response\n",
      "    - B cell proliferation involved in immune response\n",
      "    - immunoglobulin mediated immune response\n",
      "    - antigen receptor-mediated signaling pathway\n",
      "    - B cell receptor signaling pathway\n",
      "    - B cell receptor signaling pathway\n",
      "    - regulation of B cell receptor signaling pathway\n",
      "    - regulation of B cell activation\n",
      "    - positive regulation of release of sequestered calcium ion into cytosol\n",
      "    - positive regulation of release of sequestered calcium ion into cytosol\n",
      "    - positive regulation of phosphatidylinositol 3-kinase/protein kinase B signal transduction\n",
      "  CC:\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - external side of plasma membrane\n",
      "    - external side of plasma membrane\n",
      "    - protein-containing complex\n",
      "    - membrane raft\n",
      "    - extracellular exosome\n",
      "UniProt: {'Swiss-Prot': 'P15391'}\n",
      "\n",
      "Gene: MAPK3\n",
      "Symbol: MAPK3\n",
      "Name: mitogen-activated protein kinase 3\n",
      "Summary: The protein encoded by this gene is a member of the MAP kinase family. MAP kinases, also known as extracellular signal-regulated kinases (ERKs), act in a signaling cascade that regulates various cellular processes such as proliferation, differentiation, and cell cycle progression in response to a variety of extracellular signals. This kinase is activated by upstream kinases, resulting in its translocation to the nucleus where it phosphorylates nuclear targets. Alternatively spliced transcript variants encoding different protein isoforms have been described. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - MAPK cascade\n",
      "    - MAPK cascade\n",
      "    - DNA-templated transcription\n",
      "    - protein phosphorylation\n",
      "    - apoptotic process\n",
      "    - cell surface receptor signaling pathway\n",
      "    - epidermal growth factor receptor signaling pathway\n",
      "    - JNK cascade\n",
      "    - insulin receptor signaling pathway\n",
      "    - positive regulation of macrophage chemotaxis\n",
      "    - Schwann cell development\n",
      "    - phosphorylation\n",
      "    - sensory perception of pain\n",
      "    - regulation of ossification\n",
      "    - BMP signaling pathway\n",
      "    - regulation of cellular pH\n",
      "    - thyroid gland development\n",
      "    - positive regulation of cyclase activity\n",
      "    - lipopolysaccharide-mediated signaling pathway\n",
      "    - positive regulation of telomere maintenance\n",
      "    - regulation of stress-activated MAPK cascade\n",
      "    - cellular response to amino acid starvation\n",
      "    - intracellular signal transduction\n",
      "    - peptidyl-tyrosine autophosphorylation\n",
      "    - ERBB2-ERBB3 signaling pathway\n",
      "    - outer ear morphogenesis\n",
      "    - myelination\n",
      "    - signal transduction in response to DNA damage\n",
      "    - response to exogenous dsRNA\n",
      "    - positive regulation of transcription by RNA polymerase II\n",
      "    - insulin-like growth factor receptor signaling pathway\n",
      "    - thymus development\n",
      "    - modulation of chemical synaptic transmission\n",
      "    - cartilage development\n",
      "    - stress-activated MAPK cascade\n",
      "    - regulation of cytoskeleton organization\n",
      "    - Bergmann glial cell differentiation\n",
      "    - face development\n",
      "    - lung morphogenesis\n",
      "    - trachea formation\n",
      "    - cardiac neural crest cell development involved in heart development\n",
      "    - ERK1 and ERK2 cascade\n",
      "    - positive regulation of ERK1 and ERK2 cascade\n",
      "    - interleukin-1-mediated signaling pathway\n",
      "    - response to epidermal growth factor\n",
      "    - cellular response to mechanical stimulus\n",
      "    - cellular response to tumor necrosis factor\n",
      "    - caveolin-mediated endocytosis\n",
      "    - regulation of Golgi inheritance\n",
      "    - xenophagy\n",
      "    - positive regulation of macrophage proliferation\n",
      "    - negative regulation of TORC1 signaling\n",
      "    - positive regulation of xenophagy\n",
      "    - regulation of early endosome to late endosome transport\n",
      "  CC:\n",
      "    - nucleus\n",
      "    - nucleus\n",
      "    - nuclear envelope\n",
      "    - nucleoplasm\n",
      "    - nucleoplasm\n",
      "    - cytoplasm\n",
      "    - cytoplasm\n",
      "    - mitochondrion\n",
      "    - early endosome\n",
      "    - late endosome\n",
      "    - endoplasmic reticulum lumen\n",
      "    - Golgi apparatus\n",
      "    - cytosol\n",
      "    - cytoskeleton\n",
      "    - plasma membrane\n",
      "    - caveola\n",
      "    - caveola\n",
      "    - focal adhesion\n",
      "    - cilium\n",
      "    - microtubule cytoskeleton\n",
      "    - pseudopodium\n",
      "    - ciliary basal body\n",
      "    - glutamatergic synapse\n",
      "  MF:\n",
      "    - phosphotyrosine residue binding\n",
      "    - protein serine/threonine kinase activity\n",
      "    - protein serine/threonine kinase activity\n",
      "    - protein serine/threonine kinase activity\n",
      "    - JUN kinase activity\n",
      "    - MAP kinase activity\n",
      "    - MAP kinase activity\n",
      "    - protein binding\n",
      "    - ATP binding\n",
      "    - phosphatase binding\n",
      "    - identical protein binding\n",
      "    - protein serine kinase activity\n",
      "    - DNA-binding transcription factor binding\n",
      "UniProt: {'Swiss-Prot': 'P27361', 'TrEMBL': ['Q9BWJ1', 'L7RXH5', 'A1QJE5', 'H0YEX6', 'E9PQW4', 'H0YDH9', 'E9PRH7', 'E9PJF0', 'E9PBK7']}\n",
      "\n",
      "Gene: HBB\n",
      "Symbol: HBB\n",
      "Name: hemoglobin subunit beta\n",
      "Summary: The alpha (HBA) and beta (HBB) loci determine the structure of the 2 types of polypeptide chains in adult hemoglobin, Hb A. The normal adult hemoglobin tetramer consists of two alpha chains and two beta chains. Mutant beta globin causes sickle cell anemia. Absence of beta chain causes beta-zero-thalassemia. Reduced amounts of detectable beta globin causes beta-plus-thalassemia. The order of the genes in the beta-globin cluster is 5'-epsilon -- gamma-G -- gamma-A -- delta -- beta--3'. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - regulation of blood pressure\n",
      "    - carbon dioxide transport\n",
      "    - oxygen transport\n",
      "    - oxygen transport\n",
      "    - oxygen transport\n",
      "    - oxygen transport\n",
      "    - nitric oxide transport\n",
      "    - nitric oxide transport\n",
      "    - response to hydrogen peroxide\n",
      "    - hydrogen peroxide catabolic process\n",
      "    - hydrogen peroxide catabolic process\n",
      "    - positive regulation of nitric oxide biosynthetic process\n",
      "    - renal absorption\n",
      "    - platelet aggregation\n",
      "    - blood vessel diameter maintenance\n",
      "    - cellular oxidant detoxification\n",
      "  CC:\n",
      "    - extracellular region\n",
      "    - extracellular space\n",
      "    - cytosol\n",
      "    - hemoglobin complex\n",
      "    - hemoglobin complex\n",
      "    - hemoglobin complex\n",
      "    - hemoglobin complex\n",
      "    - hemoglobin complex\n",
      "    - haptoglobin-hemoglobin complex\n",
      "    - haptoglobin-hemoglobin complex\n",
      "    - extracellular exosome\n",
      "    - endocytic vesicle lumen\n",
      "    - blood microparticle\n",
      "    - tertiary granule lumen\n",
      "    - ficolin-1-rich granule lumen\n",
      "  MF:\n",
      "    - peroxidase activity\n",
      "    - peroxidase activity\n",
      "    - oxygen carrier activity\n",
      "    - oxygen carrier activity\n",
      "    - protein binding\n",
      "    - oxygen binding\n",
      "    - oxygen binding\n",
      "    - heme binding\n",
      "    - hemoglobin binding\n",
      "    - haptoglobin binding\n",
      "    - haptoglobin binding\n",
      "    - hemoglobin alpha binding\n",
      "    - metal ion binding\n",
      "UniProt: {'Swiss-Prot': 'P68871', 'TrEMBL': ['A0A2R8Y7R2', 'D9YZU5', 'A0A0J9YWK4', 'F8W6P5']}\n",
      "\n",
      "Gene: FRK\n",
      "Symbol: FRK\n",
      "Name: fyn related Src family tyrosine kinase\n",
      "Summary: The protein encoded by this gene belongs to the TYR family of protein kinases. This tyrosine kinase is a nuclear protein and may function during G1 and S phase of the cell cycle and suppress growth. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - negative regulation of transcription by RNA polymerase II\n",
      "    - cell surface receptor protein tyrosine kinase signaling pathway\n",
      "    - cell differentiation\n",
      "  CC:\n",
      "    - extracellular region\n",
      "    - nucleus\n",
      "    - nucleoplasm\n",
      "    - cytoplasm\n",
      "    - cytosol\n",
      "    - cytosol\n",
      "    - plasma membrane\n",
      "    - azurophil granule lumen\n",
      "    - specific granule lumen\n",
      "    - extracellular exosome\n",
      "  MF:\n",
      "    - protein tyrosine kinase activity\n",
      "    - protein tyrosine kinase activity\n",
      "    - non-membrane spanning protein tyrosine kinase activity\n",
      "    - signaling receptor binding\n",
      "    - protein binding\n",
      "    - ATP binding\n",
      "UniProt: {'Swiss-Prot': 'P42685'}\n",
      "\n",
      "Gene: MMP14\n",
      "Symbol: MMP14\n",
      "Name: matrix metallopeptidase 14\n",
      "Summary: Proteins of the matrix metalloproteinase (MMP) family are involved in the breakdown of extracellular matrix in normal physiological processes, such as embryonic development, reproduction, and tissue remodeling, as well as in disease processes, such as arthritis and metastasis. Most MMP's are secreted as inactive proproteins which are activated when cleaved by extracellular proteinases. However, the protein encoded by this gene is a member of the membrane-type MMP (MT-MMP) subfamily; each member of this subfamily contains a potential transmembrane domain suggesting that these proteins are expressed at the cell surface rather than secreted. This protein activates MMP2 protein, and this activity may be involved in tumor invasion. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - skeletal system development\n",
      "    - angiogenesis\n",
      "    - ovarian follicle development\n",
      "    - response to hypoxia\n",
      "    - endothelial cell proliferation\n",
      "    - endochondral ossification\n",
      "    - proteolysis\n",
      "    - proteolysis\n",
      "    - response to oxidative stress\n",
      "    - male gonad development\n",
      "    - response to mechanical stimulus\n",
      "    - positive regulation of myotube differentiation\n",
      "    - positive regulation of protein processing\n",
      "    - protein processing\n",
      "    - extracellular matrix disassembly\n",
      "    - protein catabolic process\n",
      "    - extracellular matrix organization\n",
      "    - positive regulation of cell growth\n",
      "    - lung development\n",
      "    - positive regulation of cell migration\n",
      "    - collagen catabolic process\n",
      "    - collagen catabolic process\n",
      "    - zymogen activation\n",
      "    - zymogen activation\n",
      "    - endodermal cell differentiation\n",
      "    - chondrocyte proliferation\n",
      "    - astrocyte cell migration\n",
      "    - response to estrogen\n",
      "    - positive regulation of B cell differentiation\n",
      "    - negative regulation of Notch signaling pathway\n",
      "    - embryonic cranial skeleton morphogenesis\n",
      "    - branching morphogenesis of an epithelial tube\n",
      "    - tissue remodeling\n",
      "    - cell motility\n",
      "    - negative regulation of focal adhesion assembly\n",
      "    - head development\n",
      "    - craniofacial suture morphogenesis\n",
      "    - negative regulation of GDF15-GFRAL signaling pathway\n",
      "    - regulation of protein localization to plasma membrane\n",
      "    - positive regulation of macrophage migration\n",
      "    - response to odorant\n",
      "  CC:\n",
      "    - extracellular space\n",
      "    - extracellular space\n",
      "    - nucleus\n",
      "    - cytoplasm\n",
      "    - Golgi lumen\n",
      "    - cytosol\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - focal adhesion\n",
      "    - extracellular matrix\n",
      "    - cytoplasmic vesicle\n",
      "    - melanosome\n",
      "    - macropinosome\n",
      "    - intermediate filament cytoskeleton\n",
      "  MF:\n",
      "    - endopeptidase activity\n",
      "    - metalloendopeptidase activity\n",
      "    - metalloendopeptidase activity\n",
      "    - metalloendopeptidase activity\n",
      "    - metalloendopeptidase activity\n",
      "    - serine-type endopeptidase activity\n",
      "    - integrin binding\n",
      "    - protein binding\n",
      "    - zinc ion binding\n",
      "    - metalloaminopeptidase activity\n",
      "    - metalloaminopeptidase activity\n",
      "UniProt: {'Swiss-Prot': 'P50281', 'TrEMBL': ['F8VP90', 'A0A7P0TBK8', 'A0A7P0TAG0', 'A0A7P0TAV6']}\n",
      "\n",
      "Gene: APC\n",
      "Symbol: APC\n",
      "Name: APC regulator of WNT signaling pathway\n",
      "Summary: This gene encodes a tumor suppressor protein that acts as an antagonist of the Wnt signaling pathway. It is also involved in other processes including cell migration and adhesion, transcriptional activation, and apoptosis. Defects in this gene cause familial adenomatous polyposis (FAP), an autosomal dominant pre-malignant disease that usually progresses to malignancy. Mutations in the APC gene have been found to occur in most colorectal cancers, where disease-associated mutations tend to be clustered in a small region designated the mutation cluster region (MCR) and result in a truncated protein product. [provided by RefSeq, Jun 2022].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - mitotic cytokinesis\n",
      "    - cell fate specification\n",
      "    - heart valve development\n",
      "    - endocardial cushion morphogenesis\n",
      "    - DNA damage response\n",
      "    - negative regulation of microtubule depolymerization\n",
      "    - negative regulation of microtubule depolymerization\n",
      "    - negative regulation of microtubule depolymerization\n",
      "    - mitotic spindle assembly checkpoint signaling\n",
      "    - cell adhesion\n",
      "    - pattern specification process\n",
      "    - nervous system development\n",
      "    - negative regulation of cell population proliferation\n",
      "    - insulin receptor signaling pathway\n",
      "    - Wnt signaling pathway\n",
      "    - cell migration\n",
      "    - cell migration\n",
      "    - positive regulation of cell migration\n",
      "    - positive regulation of pseudopodium assembly\n",
      "    - regulation of microtubule-based process\n",
      "    - positive regulation of apoptotic process\n",
      "    - proteasome-mediated ubiquitin-dependent protein catabolic process\n",
      "    - positive regulation of protein catabolic process\n",
      "    - positive regulation of protein catabolic process\n",
      "    - negative regulation of cyclin-dependent protein serine/threonine kinase activity\n",
      "    - regulation of attachment of spindle microtubules to kinetochore\n",
      "    - regulation of attachment of spindle microtubules to kinetochore\n",
      "    - regulation of microtubule-based movement\n",
      "    - protein-containing complex assembly\n",
      "    - bicellular tight junction assembly\n",
      "    - negative regulation of canonical Wnt signaling pathway\n",
      "    - negative regulation of canonical Wnt signaling pathway\n",
      "    - negative regulation of canonical Wnt signaling pathway\n",
      "    - negative regulation of canonical Wnt signaling pathway\n",
      "    - negative regulation of canonical Wnt signaling pathway\n",
      "    - negative regulation of canonical Wnt signaling pathway\n",
      "    - positive regulation of cold-induced thermogenesis\n",
      "    - negative regulation of cell cycle G1/S phase transition\n",
      "    - positive regulation of protein localization to centrosome\n",
      "    - negative regulation of G1/S transition of mitotic cell cycle\n",
      "  CC:\n",
      "    - kinetochore\n",
      "    - nucleus\n",
      "    - nucleoplasm\n",
      "    - cytoplasm\n",
      "    - Golgi apparatus\n",
      "    - centrosome\n",
      "    - cytosol\n",
      "    - microtubule\n",
      "    - cytoplasmic microtubule\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - adherens junction\n",
      "    - bicellular tight junction\n",
      "    - lateral plasma membrane\n",
      "    - catenin complex\n",
      "    - catenin complex\n",
      "    - lamellipodium\n",
      "    - beta-catenin destruction complex\n",
      "    - beta-catenin destruction complex\n",
      "    - beta-catenin destruction complex\n",
      "    - ruffle membrane\n",
      "    - perinuclear region of cytoplasm\n",
      "    - Wnt signalosome\n",
      "  MF:\n",
      "    - protein binding\n",
      "    - beta-catenin binding\n",
      "    - beta-catenin binding\n",
      "    - beta-catenin binding\n",
      "    - beta-catenin binding\n",
      "    - microtubule binding\n",
      "    - microtubule binding\n",
      "    - protein kinase regulator activity\n",
      "    - protein kinase binding\n",
      "    - protein kinase binding\n",
      "    - ubiquitin protein ligase binding\n",
      "    - gamma-catenin binding\n",
      "    - gamma-catenin binding\n",
      "    - cadherin binding\n",
      "    - microtubule plus-end binding\n",
      "    - dynein complex binding\n",
      "UniProt: {'Swiss-Prot': 'P25054', 'TrEMBL': ['A0A087WYF3', 'Q4LE70', 'A0AAQ5BGK6', 'A0AAQ5BGI9', 'E7EMH9', 'A0A087X2F3', 'R4GMU6', 'A0AAQ5BGJ9', 'A0AAQ5BGM3']}\n",
      "\n",
      "Gene: INHBA\n",
      "Symbol: INHBA\n",
      "Name: inhibin subunit beta A\n",
      "Summary: This gene encodes a member of the TGF-beta (transforming growth factor-beta) superfamily of proteins. The encoded preproprotein is proteolytically processed to generate a subunit of the dimeric activin and inhibin protein complexes. These complexes activate and inhibit, respectively, follicle stimulating hormone secretion from the pituitary gland. The encoded protein also plays a role in eye, tooth and testis development. Elevated expression of this gene may be associated with cancer cachexia in human patients. [provided by RefSeq, Aug 2016].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - ovarian follicle development\n",
      "    - ovarian follicle development\n",
      "    - hair follicle development\n",
      "    - hematopoietic progenitor cell differentiation\n",
      "    - regulation of transcription by RNA polymerase II\n",
      "    - transcription by RNA polymerase II\n",
      "    - autophagy\n",
      "    - defense response\n",
      "    - cell surface receptor signaling pathway\n",
      "    - cell-cell signaling\n",
      "    - nervous system development\n",
      "    - androgen metabolic process\n",
      "    - negative regulation of cell population proliferation\n",
      "    - male gonad development\n",
      "    - response to wounding\n",
      "    - positive regulation of gene expression\n",
      "    - cytokine-mediated signaling pathway\n",
      "    - striatal medium spiny neuron differentiation\n",
      "    - cell differentiation\n",
      "    - erythrocyte differentiation\n",
      "    - negative regulation of cell growth\n",
      "    - negative regulation of type II interferon production\n",
      "    - activin receptor signaling pathway\n",
      "    - positive regulation of collagen biosynthetic process\n",
      "    - endodermal cell differentiation\n",
      "    - negative regulation of phosphorylation\n",
      "    - odontogenesis\n",
      "    - hemoglobin biosynthetic process\n",
      "    - progesterone secretion\n",
      "    - negative regulation of B cell differentiation\n",
      "    - positive regulation of erythrocyte differentiation\n",
      "    - negative regulation of macrophage differentiation\n",
      "    - positive regulation of DNA-templated transcription\n",
      "    - positive regulation of transcription by RNA polymerase II\n",
      "    - positive regulation of transcription by RNA polymerase III\n",
      "    - regulation of follicle-stimulating hormone secretion\n",
      "    - positive regulation of follicle-stimulating hormone secretion\n",
      "    - negative regulation of follicle-stimulating hormone secretion\n",
      "    - mesodermal cell differentiation\n",
      "    - positive regulation of protein metabolic process\n",
      "    - Sertoli cell differentiation\n",
      "    - roof of mouth development\n",
      "    - positive regulation of ovulation\n",
      "    - positive regulation of SMAD protein signal transduction\n",
      "    - SMAD protein signal transduction\n",
      "    - cardiac fibroblast cell development\n",
      "    - eyelid development in camera-type eye\n",
      "    - testosterone biosynthetic process\n",
      "    - positive regulation of ERK1 and ERK2 cascade\n",
      "    - cellular response to follicle-stimulating hormone stimulus\n",
      "    - cellular response to cholesterol\n",
      "    - cellular response to hypoxia\n",
      "    - cellular response to oxygen-glucose deprivation\n",
      "    - GABAergic neuron differentiation\n",
      "    - extrinsic apoptotic signaling pathway\n",
      "    - androst-4-ene-3,17-dione biosynthetic process\n",
      "    - response to aldosterone\n",
      "    - cellular response to angiotensin\n",
      "    - negative regulation of G1/S transition of mitotic cell cycle\n",
      "    - positive regulation of extrinsic apoptotic signaling pathway in absence of ligand\n",
      "  CC:\n",
      "    - extracellular region\n",
      "    - extracellular region\n",
      "    - extracellular space\n",
      "    - activin A complex\n",
      "    - inhibin A complex\n",
      "    - perinuclear region of cytoplasm\n",
      "    - enzyme activator complex\n",
      "  MF:\n",
      "    - cytokine activity\n",
      "    - cytokine activity\n",
      "    - hormone activity\n",
      "    - protein binding\n",
      "    - growth factor activity\n",
      "    - peptide hormone binding\n",
      "    - identical protein binding\n",
      "    - protein-containing complex binding\n",
      "    - type II activin receptor binding\n",
      "UniProt: {'Swiss-Prot': 'P08476', 'TrEMBL': 'A4D1W7'}\n",
      "\n",
      "Gene: PARP1\n",
      "Symbol: PARP1\n",
      "Name: poly(ADP-ribose) polymerase 1\n",
      "Summary: This gene encodes a chromatin-associated enzyme, poly(ADP-ribosyl)transferase, which modifies various nuclear proteins by poly(ADP-ribosyl)ation. The modification is dependent on DNA and is involved in the regulation of various important cellular processes such as differentiation, proliferation, and tumor transformation and also in the regulation of the molecular events involved in the recovery of cell from DNA damage. In addition, this enzyme may be the site of mutation in Fanconi anemia, and may participate in the pathophysiology of type I diabetes. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - negative regulation of transcription by RNA polymerase II\n",
      "    - negative regulation of transcription by RNA polymerase II\n",
      "    - telomere maintenance\n",
      "    - DNA repair\n",
      "    - DNA repair\n",
      "    - double-strand break repair\n",
      "    - double-strand break repair\n",
      "    - double-strand break repair\n",
      "    - transcription by RNA polymerase II\n",
      "    - apoptotic process\n",
      "    - DNA damage response\n",
      "    - DNA damage response\n",
      "    - DNA damage response\n",
      "    - mitochondrion organization\n",
      "    - transforming growth factor beta receptor signaling pathway\n",
      "    - response to gamma radiation\n",
      "    - positive regulation of cardiac muscle hypertrophy\n",
      "    - carbohydrate biosynthetic process\n",
      "    - protein autoprocessing\n",
      "    - signal transduction involved in regulation of gene expression\n",
      "    - macrophage differentiation\n",
      "    - DNA ADP-ribosylation\n",
      "    - mitochondrial DNA metabolic process\n",
      "    - positive regulation of DNA-templated transcription, elongation\n",
      "    - cellular response to insulin stimulus\n",
      "    - regulation of protein localization\n",
      "    - positive regulation of intracellular estrogen receptor signaling pathway\n",
      "    - negative regulation of transcription elongation by RNA polymerase II\n",
      "    - cellular response to oxidative stress\n",
      "    - cellular response to UV\n",
      "    - protein modification process\n",
      "    - positive regulation of canonical NF-kappaB signal transduction\n",
      "    - mitochondrial DNA repair\n",
      "    - innate immune response\n",
      "    - regulation of circadian sleep/wake cycle, non-REM sleep\n",
      "    - negative regulation of innate immune response\n",
      "    - negative regulation of innate immune response\n",
      "    - negative regulation of DNA-templated transcription\n",
      "    - positive regulation of transcription by RNA polymerase II\n",
      "    - decidualization\n",
      "    - positive regulation of mitochondrial depolarization\n",
      "    - positive regulation of SMAD protein signal transduction\n",
      "    - positive regulation of necroptotic process\n",
      "    - protein poly-ADP-ribosylation\n",
      "    - protein auto-ADP-ribosylation\n",
      "    - protein localization to chromatin\n",
      "    - cellular response to zinc ion\n",
      "    - replication fork reversal\n",
      "    - negative regulation of cGAS/STING signaling pathway\n",
      "    - positive regulation of protein localization to nucleus\n",
      "    - regulation of oxidative stress-induced neuron intrinsic apoptotic signaling pathway\n",
      "    - positive regulation of single strand break repair\n",
      "    - response to aldosterone\n",
      "    - negative regulation of adipose tissue development\n",
      "    - negative regulation of telomere maintenance via telomere lengthening\n",
      "    - cellular response to amyloid-beta\n",
      "    - positive regulation of myofibroblast differentiation\n",
      "    - regulation of base-excision repair\n",
      "    - positive regulation of double-strand break repair via homologous recombination\n",
      "    - cellular response to nerve growth factor stimulus\n",
      "    - ATP generation from poly-ADP-D-ribose\n",
      "    - negative regulation of ATP biosynthetic process\n",
      "  CC:\n",
      "    - chromosome, telomeric region\n",
      "    - chromatin\n",
      "    - chromatin\n",
      "    - nucleus\n",
      "    - nucleus\n",
      "    - nuclear envelope\n",
      "    - nucleoplasm\n",
      "    - nucleoplasm\n",
      "    - transcription regulator complex\n",
      "    - nucleolus\n",
      "    - nucleolus\n",
      "    - mitochondrion\n",
      "    - cytosol\n",
      "    - membrane\n",
      "    - nuclear body\n",
      "    - protein-containing complex\n",
      "    - protein-DNA complex\n",
      "    - site of double-strand break\n",
      "    - site of double-strand break\n",
      "    - nuclear replication fork\n",
      "    - site of DNA damage\n",
      "    - site of DNA damage\n",
      "  MF:\n",
      "    - DNA binding\n",
      "    - chromatin binding\n",
      "    - damaged DNA binding\n",
      "    - RNA binding\n",
      "    - NAD+ poly-ADP-ribosyltransferase activity\n",
      "    - NAD+ poly-ADP-ribosyltransferase activity\n",
      "    - NAD+ poly-ADP-ribosyltransferase activity\n",
      "    - NAD+ poly-ADP-ribosyltransferase activity\n",
      "    - protein binding\n",
      "    - enzyme activator activity\n",
      "    - zinc ion binding\n",
      "    - nucleotidyltransferase activity\n",
      "    - enzyme binding\n",
      "    - protein kinase binding\n",
      "    - nuclear estrogen receptor binding\n",
      "    - nucleosome binding\n",
      "    - ubiquitin protein ligase binding\n",
      "    - identical protein binding\n",
      "    - protein homodimerization activity\n",
      "    - histone deacetylase binding\n",
      "    - NAD binding\n",
      "    - RNA polymerase II-specific DNA-binding transcription factor binding\n",
      "    - R-SMAD binding\n",
      "    - NAD DNA ADP-ribosyltransferase activity\n",
      "    - transcription regulator activator activity\n",
      "    - NAD+-protein-serine ADP-ribosyltransferase activity\n",
      "    - NAD+-protein-aspartate ADP-ribosyltransferase activity\n",
      "    - NAD+-protein-glutamate ADP-ribosyltransferase activity\n",
      "    - NAD+-protein-tyrosine ADP-ribosyltransferase activity\n",
      "    - NAD+-protein-histidine ADP-ribosyltransferase activity\n",
      "    - NAD+-histone H2BS6 serine ADP-ribosyltransferase activity\n",
      "    - NAD+-histone H3S10 serine ADP-ribosyltransferase activity\n",
      "    - NAD+-histone H2BE35 glutamate ADP-ribosyltransferase activity\n",
      "    - NAD+-protein mono-ADP-ribosyltransferase activity\n",
      "UniProt: {'Swiss-Prot': 'P09874', 'TrEMBL': ['A0A7I2V5E9', 'A0A7I2V625', 'A0A7I2V384', 'A0A7I2V3E1']}\n",
      "\n",
      "Gene: HDAC1\n",
      "Symbol: HDAC1\n",
      "Name: histone deacetylase 1\n",
      "Summary: Histone acetylation and deacetylation, catalyzed by multisubunit complexes, play a key role in the regulation of eukaryotic gene expression. The protein encoded by this gene belongs to the histone deacetylase/acuc/apha family and is a component of the histone deacetylase complex. It also interacts with retinoblastoma tumor-suppressor protein and this complex is a key element in the control of cell proliferation and differentiation. Together with metastasis-associated protein-2, it deacetylates p53 and modulates its effect on cell growth and apoptosis. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - negative regulation of transcription by RNA polymerase II\n",
      "    - negative regulation of transcription by RNA polymerase II\n",
      "    - negative regulation of transcription by RNA polymerase II\n",
      "    - negative regulation of transcription by RNA polymerase II\n",
      "    - chromatin organization\n",
      "    - chromatin remodeling\n",
      "    - chromatin remodeling\n",
      "    - chromatin remodeling\n",
      "    - DNA methylation-dependent constitutive heterochromatin formation\n",
      "    - regulation of transcription by RNA polymerase II\n",
      "    - protein deacetylation\n",
      "    - endoderm development\n",
      "    - positive regulation of cell population proliferation\n",
      "    - epidermal cell differentiation\n",
      "    - positive regulation of gene expression\n",
      "    - negative regulation of gene expression\n",
      "    - negative regulation of myotube differentiation\n",
      "    - hippocampus development\n",
      "    - neuron differentiation\n",
      "    - negative regulation of cell migration\n",
      "    - negative regulation of transforming growth factor beta receptor signaling pathway\n",
      "    - heterochromatin formation\n",
      "    - circadian regulation of gene expression\n",
      "    - positive regulation of intracellular estrogen receptor signaling pathway\n",
      "    - cellular response to platelet-derived growth factor stimulus\n",
      "    - odontogenesis of dentin-containing tooth\n",
      "    - regulation of cell fate specification\n",
      "    - embryonic digit morphogenesis\n",
      "    - negative regulation of apoptotic process\n",
      "    - negative regulation of apoptotic process\n",
      "    - negative regulation of canonical NF-kappaB signal transduction\n",
      "    - negative regulation by host of viral transcription\n",
      "    - negative regulation of gene expression, epigenetic\n",
      "    - negative regulation of DNA-templated transcription\n",
      "    - negative regulation of DNA-templated transcription\n",
      "    - negative regulation of DNA-templated transcription\n",
      "    - positive regulation of DNA-templated transcription\n",
      "    - positive regulation of DNA-templated transcription\n",
      "    - positive regulation of transcription by RNA polymerase II\n",
      "    - positive regulation of smooth muscle cell proliferation\n",
      "    - oligodendrocyte differentiation\n",
      "    - positive regulation of oligodendrocyte differentiation\n",
      "    - negative regulation of androgen receptor signaling pathway\n",
      "    - hair follicle placode formation\n",
      "    - eyelid development in camera-type eye\n",
      "    - fungiform papilla formation\n",
      "    - negative regulation of canonical Wnt signaling pathway\n",
      "    - negative regulation of stem cell population maintenance\n",
      "    - positive regulation of stem cell population maintenance\n",
      "    - regulation of stem cell differentiation\n",
      "    - negative regulation of intrinsic apoptotic signaling pathway\n",
      "  CC:\n",
      "    - histone deacetylase complex\n",
      "    - histone deacetylase complex\n",
      "    - chromatin\n",
      "    - chromatin\n",
      "    - heterochromatin\n",
      "    - nucleus\n",
      "    - nucleus\n",
      "    - nucleoplasm\n",
      "    - nucleoplasm\n",
      "    - cytoplasm\n",
      "    - cytosol\n",
      "    - NuRD complex\n",
      "    - NuRD complex\n",
      "    - NuRD complex\n",
      "    - transcription repressor complex\n",
      "    - protein-containing complex\n",
      "    - protein-containing complex\n",
      "    - neuronal cell body\n",
      "    - Sin3-type complex\n",
      "    - Sin3-type complex\n",
      "  MF:\n",
      "    - RNA polymerase II cis-regulatory region sequence-specific DNA binding\n",
      "    - RNA polymerase II core promoter sequence-specific DNA binding\n",
      "    - core promoter sequence-specific DNA binding\n",
      "    - transcription corepressor binding\n",
      "    - p53 binding\n",
      "    - transcription corepressor activity\n",
      "    - histone deacetylase activity\n",
      "    - histone deacetylase activity\n",
      "    - histone deacetylase activity\n",
      "    - histone deacetylase activity\n",
      "    - protein binding\n",
      "    - protein binding\n",
      "    - enzyme binding\n",
      "    - histone H3K14 deacetylase activity, hydrolytic mechanism\n",
      "    - nucleosomal DNA binding\n",
      "    - histone H3K9 deacetylase activity, hydrolytic mechanism\n",
      "    - protein lysine deacetylase activity\n",
      "    - protein lysine deacetylase activity\n",
      "    - protein lysine deacetylase activity\n",
      "    - histone H4K16 deacetylase activity, hydrolytic mechanism\n",
      "    - Krueppel-associated box domain binding\n",
      "    - histone deacetylase binding\n",
      "    - metal ion binding\n",
      "    - NF-kappaB binding\n",
      "    - RNA polymerase II-specific DNA-binding transcription factor binding\n",
      "    - E-box binding\n",
      "    - DNA-binding transcription factor binding\n",
      "    - DNA-binding transcription factor binding\n",
      "    - histone H4K12 deacetylase activity, hydrolytic mechanism\n",
      "    - histone decrotonylase activity\n",
      "    - protein lysine delactylase activity\n",
      "    - histone H4K5 deacetylase activity, hydrolytic mechanism\n",
      "    - histone H4K8 deacetylase activity, hydrolytic mechanism\n",
      "    - histone H3K4 deacetylase activity, hydrolytic mechanism\n",
      "    - promoter-specific chromatin binding\n",
      "UniProt: {'Swiss-Prot': 'Q13547', 'TrEMBL': 'Q6IT96'}\n",
      "\n",
      "Gene: HDAC2\n",
      "Symbol: HDAC2\n",
      "Name: histone deacetylase 2\n",
      "Summary: This gene product belongs to the histone deacetylase family. Histone deacetylases act via the formation of large multiprotein complexes, and are responsible for the deacetylation of lysine residues at the N-terminal regions of core histones (H2A, H2B, H3 and H4). This protein forms transcriptional repressor complexes by associating with many different proteins, including YY1, a mammalian zinc-finger transcription factor. Thus, it plays an important role in transcriptional regulation, cell cycle progression and developmental events. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Apr 2010].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - negative regulation of transcription by RNA polymerase II\n",
      "    - negative regulation of transcription by RNA polymerase II\n",
      "    - response to amphetamine\n",
      "    - cardiac muscle hypertrophy\n",
      "    - chromatin remodeling\n",
      "    - chromatin remodeling\n",
      "    - chromatin remodeling\n",
      "    - positive regulation of cell population proliferation\n",
      "    - response to xenobiotic stimulus\n",
      "    - epidermal cell differentiation\n",
      "    - positive regulation of epithelial to mesenchymal transition\n",
      "    - negative regulation of transcription by competitive promoter binding\n",
      "    - negative regulation of neuron projection development\n",
      "    - dendrite development\n",
      "    - negative regulation of cell migration\n",
      "    - negative regulation of transforming growth factor beta receptor signaling pathway\n",
      "    - response to caffeine\n",
      "    - heterochromatin formation\n",
      "    - heterochromatin formation\n",
      "    - response to lipopolysaccharide\n",
      "    - positive regulation of interleukin-1 production\n",
      "    - positive regulation of tumor necrosis factor production\n",
      "    - circadian regulation of gene expression\n",
      "    - positive regulation of collagen biosynthetic process\n",
      "    - positive regulation of intracellular estrogen receptor signaling pathway\n",
      "    - cellular response to heat\n",
      "    - response to nicotine\n",
      "    - protein modification process\n",
      "    - response to cocaine\n",
      "    - odontogenesis of dentin-containing tooth\n",
      "    - regulation of cell fate specification\n",
      "    - embryonic digit morphogenesis\n",
      "    - negative regulation of apoptotic process\n",
      "    - negative regulation of DNA-binding transcription factor activity\n",
      "    - negative regulation of MHC class II biosynthetic process\n",
      "    - positive regulation of proteolysis\n",
      "    - negative regulation of DNA-templated transcription\n",
      "    - negative regulation of DNA-templated transcription\n",
      "    - positive regulation of DNA-templated transcription\n",
      "    - positive regulation of DNA-templated transcription\n",
      "    - positive regulation of transcription by RNA polymerase II\n",
      "    - behavioral response to ethanol\n",
      "    - positive regulation of oligodendrocyte differentiation\n",
      "    - progesterone receptor signaling pathway\n",
      "    - response to hyperoxia\n",
      "    - hair follicle placode formation\n",
      "    - negative regulation of dendritic spine development\n",
      "    - eyelid development in camera-type eye\n",
      "    - fungiform papilla formation\n",
      "    - cellular response to hydrogen peroxide\n",
      "    - cellular response to retinoic acid\n",
      "    - cellular response to transforming growth factor beta stimulus\n",
      "    - positive regulation of male mating behavior\n",
      "    - negative regulation of stem cell population maintenance\n",
      "    - positive regulation of stem cell population maintenance\n",
      "    - cellular response to dopamine\n",
      "    - response to amyloid-beta\n",
      "    - regulation of stem cell differentiation\n",
      "  CC:\n",
      "    - histone deacetylase complex\n",
      "    - chromosome, telomeric region\n",
      "    - chromatin\n",
      "    - nucleus\n",
      "    - nucleus\n",
      "    - nucleoplasm\n",
      "    - nucleoplasm\n",
      "    - cytoplasm\n",
      "    - cytoplasm\n",
      "    - NuRD complex\n",
      "    - NuRD complex\n",
      "    - NuRD complex\n",
      "    - protein-containing complex\n",
      "    - protein-containing complex\n",
      "    - ESC/E(Z) complex\n",
      "    - Sin3-type complex\n",
      "    - Sin3-type complex\n",
      "  MF:\n",
      "    - chromatin binding\n",
      "    - RNA binding\n",
      "    - histone deacetylase activity\n",
      "    - histone deacetylase activity\n",
      "    - histone deacetylase activity\n",
      "    - protein binding\n",
      "    - enzyme binding\n",
      "    - heat shock protein binding\n",
      "    - histone H3K14 deacetylase activity, hydrolytic mechanism\n",
      "    - nucleosomal DNA binding\n",
      "    - histone H3K9 deacetylase activity, hydrolytic mechanism\n",
      "    - protein lysine deacetylase activity\n",
      "    - protein lysine deacetylase activity\n",
      "    - histone H4K16 deacetylase activity, hydrolytic mechanism\n",
      "    - histone binding\n",
      "    - histone deacetylase binding\n",
      "    - NF-kappaB binding\n",
      "    - RNA polymerase II-specific DNA-binding transcription factor binding\n",
      "    - histone H4K12 deacetylase activity, hydrolytic mechanism\n",
      "    - histone decrotonylase activity\n",
      "    - protein de-2-hydroxyisobutyrylase activity\n",
      "    - protein lysine delactylase activity\n",
      "    - histone H4K5 deacetylase activity, hydrolytic mechanism\n",
      "    - histone H4K8 deacetylase activity, hydrolytic mechanism\n",
      "    - histone H3K4 deacetylase activity, hydrolytic mechanism\n",
      "    - promoter-specific chromatin binding\n",
      "UniProt: {'Swiss-Prot': 'Q92769', 'TrEMBL': ['E5RGV4', 'E5RK19', 'E5RG37', 'E5RH52', 'H3BM24', 'E5RFI6', 'E5RHE7', 'E5RFP9', 'E5RJ04']}\n",
      "\n",
      "Gene: TOP2A\n",
      "Symbol: TOP2A\n",
      "Name: DNA topoisomerase II alpha\n",
      "Summary: This gene encodes a DNA topoisomerase, an enzyme that controls and alters the topologic states of DNA during transcription. This nuclear enzyme is involved in processes such as chromosome condensation, chromatid separation, and the relief of torsional stress that occurs during DNA transcription and replication. It catalyzes the transient breaking and rejoining of two strands of duplex DNA which allows the strands to pass through one another, thus altering the topology of DNA. Two forms of this enzyme exist as likely products of a gene duplication event. The gene encoding this form, alpha, is localized to chromosome 17 and the beta gene is localized to chromosome 3. The gene encoding this enzyme functions as the target for several anticancer agents and a variety of mutations in this gene have been associated with the development of drug resistance. Reduced activity of this enzyme may also play a role in ataxia-telangiectasia. [provided by RefSeq, Jul 2010].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - resolution of meiotic recombination intermediates\n",
      "    - sister chromatid segregation\n",
      "    - hematopoietic progenitor cell differentiation\n",
      "    - DNA topological change\n",
      "    - chromatin organization\n",
      "    - DNA damage response\n",
      "    - chromosome segregation\n",
      "    - female meiotic nuclear division\n",
      "    - apoptotic chromosome condensation\n",
      "    - apoptotic chromosome condensation\n",
      "    - embryonic cleavage\n",
      "    - regulation of circadian rhythm\n",
      "    - positive regulation of apoptotic process\n",
      "    - positive regulation of single stranded viral RNA replication via double stranded DNA intermediate\n",
      "    - positive regulation of transcription by RNA polymerase II\n",
      "    - rhythmic process\n",
      "  CC:\n",
      "    - nuclear chromosome\n",
      "    - chromosome, centromeric region\n",
      "    - condensed chromosome\n",
      "    - male germ cell nucleus\n",
      "    - nucleus\n",
      "    - nucleus\n",
      "    - nucleoplasm\n",
      "    - nucleoplasm\n",
      "    - nucleolus\n",
      "    - nucleolus\n",
      "    - cytoplasm\n",
      "    - centriole\n",
      "    - DNA topoisomerase type II (double strand cut, ATP-hydrolyzing) complex\n",
      "    - protein-containing complex\n",
      "    - ribonucleoprotein complex\n",
      "  MF:\n",
      "    - magnesium ion binding\n",
      "    - DNA binding\n",
      "    - chromatin binding\n",
      "    - RNA binding\n",
      "    - DNA topoisomerase type II (double strand cut, ATP-hydrolyzing) activity\n",
      "    - DNA topoisomerase type II (double strand cut, ATP-hydrolyzing) activity\n",
      "    - protein kinase C binding\n",
      "    - protein binding\n",
      "    - ATP binding\n",
      "    - ATP-dependent activity, acting on DNA\n",
      "    - DNA binding, bending\n",
      "    - DNA negative supercoiling activity\n",
      "    - protein homodimerization activity\n",
      "    - ubiquitin binding\n",
      "    - protein heterodimerization activity\n",
      "UniProt: {'Swiss-Prot': 'P11388', 'TrEMBL': ['J3KTB7', 'J3QR57']}\n",
      "\n",
      "Gene: TOP2B\n",
      "Symbol: TOP2B\n",
      "Name: DNA topoisomerase II beta\n",
      "Summary: This gene encodes a DNA topoisomerase, an enzyme that controls and alters the topologic states of DNA during transcription. This nuclear enzyme is involved in processes such as chromosome condensation, chromatid separation, and the relief of torsional stress that occurs during DNA transcription and replication. It catalyzes the transient breaking and rejoining of two strands of duplex DNA which allows the strands to pass through one another, thus altering the topology of DNA. Two forms of this enzyme exist as likely products of a gene duplication event. The gene encoding this form, beta, is localized to chromosome 3 and the alpha form is localized to chromosome 17. The gene encoding this enzyme functions as the target for several anticancer agents and a variety of mutations in this gene have been associated with the development of drug resistance. Reduced activity of this enzyme may also play a role in ataxia-telangiectasia. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Aug 2016].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - resolution of meiotic recombination intermediates\n",
      "    - sister chromatid segregation\n",
      "    - neuron migration\n",
      "    - DNA topological change\n",
      "    - axonogenesis\n",
      "    - B cell differentiation\n",
      "    - forebrain development\n",
      "    - positive regulation of single stranded viral RNA replication via double stranded DNA intermediate\n",
      "    - cellular response to hydrogen peroxide\n",
      "    - cellular response to ATP\n",
      "    - cellular senescence\n",
      "    - positive regulation of double-strand break repair via nonhomologous end joining\n",
      "  CC:\n",
      "    - heterochromatin\n",
      "    - nucleus\n",
      "    - nucleus\n",
      "    - nucleoplasm\n",
      "    - nucleoplasm\n",
      "    - nucleolus\n",
      "    - nucleolus\n",
      "    - cytosol\n",
      "    - ribonucleoprotein complex\n",
      "  MF:\n",
      "    - DNA binding\n",
      "    - chromatin binding\n",
      "    - DNA topoisomerase type II (double strand cut, ATP-hydrolyzing) activity\n",
      "    - DNA topoisomerase type II (double strand cut, ATP-hydrolyzing) activity\n",
      "    - protein binding\n",
      "    - ATP binding\n",
      "    - DNA negative supercoiling activity\n",
      "    - ribonucleoprotein complex binding\n",
      "    - metal ion binding\n",
      "UniProt: {'Swiss-Prot': 'Q02880', 'TrEMBL': ['H7BZ82', 'Q59H80', 'A0A8V8TP20', 'E9PCY5', 'A0A8V8TPE3', 'A0A8V8TN33']}\n",
      "\n",
      "Gene: RXRG\n",
      "Symbol: RXRG\n",
      "Name: retinoid X receptor gamma\n",
      "Summary: This gene encodes a member of the retinoid X receptor (RXR) family of nuclear receptors which are involved in mediating the antiproliferative effects of retinoic acid (RA). This receptor forms dimers with the retinoic acid, thyroid hormone, and vitamin D receptors, increasing both DNA binding and transcriptional function on their respective response elements. This gene is expressed at significantly lower levels in non-small cell lung cancer cells. Alternatively spliced transcript variants have been described. [provided by RefSeq, Jun 2010].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - nervous system development\n",
      "    - cell differentiation\n",
      "    - nuclear receptor-mediated steroid hormone signaling pathway\n",
      "    - positive regulation of transcription by RNA polymerase II\n",
      "    - retinoic acid receptor signaling pathway\n",
      "  CC:\n",
      "    - chromatin\n",
      "    - nucleoplasm\n",
      "    - cytoplasm\n",
      "    - RNA polymerase II transcription regulator complex\n",
      "  MF:\n",
      "    - DNA-binding transcription factor activity, RNA polymerase II-specific\n",
      "    - nuclear steroid receptor activity\n",
      "    - nuclear receptor activity\n",
      "    - protein binding\n",
      "    - zinc ion binding\n",
      "    - retinoic acid-responsive element binding\n",
      "    - molecular condensate scaffold activity\n",
      "    - sequence-specific double-stranded DNA binding\n",
      "UniProt: {'Swiss-Prot': 'P48443', 'TrEMBL': ['F1T097', 'F1D8Q7', 'A0A087WZ88']}\n",
      "\n",
      "Gene: RARG\n",
      "Symbol: RARG\n",
      "Name: retinoic acid receptor gamma\n",
      "Summary: This gene encodes a retinoic acid receptor that belongs to the nuclear hormone receptor family. Retinoic acid receptors (RARs) act as ligand-dependent transcriptional regulators. When bound to ligands, RARs activate transcription by binding as heterodimers to the retinoic acid response elements (RARE) found in the promoter regions of the target genes. In their unbound form, RARs repress transcription of their target genes. RARs are involved in various biological processes, including limb bud development, skeletal growth, and matrix homeostasis. Alternatively spliced transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Aug 2011].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - negative regulation of transcription by RNA polymerase II\n",
      "    - negative regulation of transcription by RNA polymerase II\n",
      "    - neural tube closure\n",
      "    - glandular epithelial cell development\n",
      "    - growth plate cartilage chondrocyte growth\n",
      "    - apoptotic process\n",
      "    - positive regulation of cell population proliferation\n",
      "    - negative regulation of cell population proliferation\n",
      "    - regulation of cell size\n",
      "    - anterior/posterior pattern specification\n",
      "    - positive regulation of gene expression\n",
      "    - cell differentiation\n",
      "    - embryonic camera-type eye development\n",
      "    - regulation of myelination\n",
      "    - negative regulation of chondrocyte differentiation\n",
      "    - response to retinoic acid\n",
      "    - embryonic hindlimb morphogenesis\n",
      "    - multicellular organism growth\n",
      "    - positive regulation of apoptotic process\n",
      "    - positive regulation of programmed cell death\n",
      "    - regulation of myeloid cell differentiation\n",
      "    - positive regulation of transcription by RNA polymerase II\n",
      "    - positive regulation of transcription by RNA polymerase II\n",
      "    - embryonic eye morphogenesis\n",
      "    - retinoic acid receptor signaling pathway\n",
      "    - retinoic acid receptor signaling pathway\n",
      "    - canonical Wnt signaling pathway\n",
      "    - face development\n",
      "    - trachea cartilage development\n",
      "    - prostate gland epithelium morphogenesis\n",
      "    - Harderian gland development\n",
      "    - cellular response to retinoic acid\n",
      "    - stem cell proliferation\n",
      "    - cellular response to leukemia inhibitory factor\n",
      "    - negative regulation of stem cell proliferation\n",
      "  CC:\n",
      "    - chromatin\n",
      "    - chromatin\n",
      "    - nucleus\n",
      "    - nucleus\n",
      "    - nucleus\n",
      "    - nucleoplasm\n",
      "    - nucleoplasm\n",
      "    - transcription regulator complex\n",
      "    - cytoplasm\n",
      "    - membrane\n",
      "  MF:\n",
      "    - RNA polymerase II cis-regulatory region sequence-specific DNA binding\n",
      "    - DNA-binding transcription factor activity, RNA polymerase II-specific\n",
      "    - DNA binding\n",
      "    - chromatin binding\n",
      "    - DNA-binding transcription factor activity\n",
      "    - nuclear receptor activity\n",
      "    - protein binding\n",
      "    - zinc ion binding\n",
      "    - nuclear retinoid X receptor binding\n",
      "    - sequence-specific double-stranded DNA binding\n",
      "UniProt: {'Swiss-Prot': 'P13631', 'TrEMBL': ['B7Z4B4', 'A8K3H3', 'F1D8P1', 'H3BMK1', 'H3BMH6', 'H3BMY6']}\n",
      "\n",
      "Gene: RXRA\n",
      "Symbol: RXRA\n",
      "Name: retinoid X receptor alpha\n",
      "Summary: Retinoid X receptors (RXRs) and retinoic acid receptors (RARs) are nuclear receptors that mediate the biological effects of retinoids by their involvement in retinoic acid-mediated gene activation. These receptors function as transcription factors by binding as homodimers or heterodimers to specific sequences in the promoters of target genes. The protein encoded by this gene is a member of the steroid and thyroid hormone receptor superfamily of transcriptional regulators. Alternative splicing of this gene results in multiple transcript variants. [provided by RefSeq, May 2014].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - negative regulation of transcription by RNA polymerase II\n",
      "    - positive regulation of thyroid hormone receptor signaling pathway\n",
      "    - nervous system development\n",
      "    - hormone-mediated signaling pathway\n",
      "    - hormone-mediated signaling pathway\n",
      "    - positive regulation of cholesterol efflux\n",
      "    - cell differentiation\n",
      "    - positive regulation of bone mineralization\n",
      "    - nuclear receptor-mediated steroid hormone signaling pathway\n",
      "    - positive regulation of transporter activity\n",
      "    - response to retinoic acid\n",
      "    - peroxisome proliferator activated receptor signaling pathway\n",
      "    - mRNA transcription by RNA polymerase II\n",
      "    - mRNA transcription by RNA polymerase II\n",
      "    - mRNA transcription by RNA polymerase II\n",
      "    - positive regulation of DNA-templated transcription\n",
      "    - positive regulation of transcription by RNA polymerase II\n",
      "    - positive regulation of transcription by RNA polymerase II\n",
      "    - positive regulation of transcription by RNA polymerase II\n",
      "    - positive regulation of transcription by RNA polymerase II\n",
      "    - positive regulation of transcription by RNA polymerase II\n",
      "    - retinoic acid receptor signaling pathway\n",
      "    - retinoic acid receptor signaling pathway\n",
      "    - retinoic acid receptor signaling pathway\n",
      "    - positive regulation of vitamin D receptor signaling pathway\n",
      "  CC:\n",
      "    - chromatin\n",
      "    - chromatin\n",
      "    - chromatin\n",
      "    - nucleus\n",
      "    - nucleoplasm\n",
      "    - transcription regulator complex\n",
      "    - mitochondrion\n",
      "    - cytosol\n",
      "    - receptor complex\n",
      "    - RNA polymerase II transcription regulator complex\n",
      "    - RNA polymerase II transcription regulator complex\n",
      "    - RNA polymerase II transcription regulator complex\n",
      "    - RNA polymerase II transcription regulator complex\n",
      "    - RNA polymerase II transcription regulator complex\n",
      "    - RNA polymerase II transcription regulator complex\n",
      "  MF:\n",
      "    - transcription cis-regulatory region binding\n",
      "    - RNA polymerase II transcription regulatory region sequence-specific DNA binding\n",
      "    - RNA polymerase II cis-regulatory region sequence-specific DNA binding\n",
      "    - DNA-binding transcription factor activity, RNA polymerase II-specific\n",
      "    - transcription coregulator binding\n",
      "    - retinoic acid binding\n",
      "    - double-stranded DNA binding\n",
      "    - DNA-binding transcription factor activity\n",
      "    - nuclear steroid receptor activity\n",
      "    - nuclear receptor activity\n",
      "    - nuclear receptor activity\n",
      "    - protein binding\n",
      "    - zinc ion binding\n",
      "    - enzyme binding\n",
      "    - peptide binding\n",
      "    - identical protein binding\n",
      "    - nuclear vitamin D receptor binding\n",
      "    - sequence-specific DNA binding\n",
      "    - retinoic acid-responsive element binding\n",
      "    - retinoic acid-responsive element binding\n",
      "    - DNA binding domain binding\n",
      "    - LBD domain binding\n",
      "    - vitamin D response element binding\n",
      "    - sequence-specific double-stranded DNA binding\n",
      "UniProt: {'Swiss-Prot': 'P19793', 'TrEMBL': ['F1D8Q5', 'A0A5F9ZHH6', 'Q6P3U7', 'A0A3B3IS44']}\n",
      "\n",
      "Gene: RARA\n",
      "Symbol: RARA\n",
      "Name: retinoic acid receptor alpha\n",
      "Summary: This gene represents a nuclear retinoic acid receptor. The encoded protein, retinoic acid receptor alpha, regulates transcription in a ligand-dependent manner. This gene has been implicated in regulation of development, differentiation, apoptosis, granulopoeisis, and transcription of clock genes. Translocations between this locus and several other loci have been associated with acute promyelocytic leukemia. Alternatively spliced transcript variants have been found for this locus.[provided by RefSeq, Sep 2010].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - negative regulation of transcription by RNA polymerase II\n",
      "    - ureteric bud development\n",
      "    - neural tube closure\n",
      "    - liver development\n",
      "    - glandular epithelial cell development\n",
      "    - outflow tract septum morphogenesis\n",
      "    - growth plate cartilage development\n",
      "    - protein phosphorylation\n",
      "    - germ cell development\n",
      "    - spermatogenesis\n",
      "    - female pregnancy\n",
      "    - positive regulation of cell population proliferation\n",
      "    - negative regulation of cell population proliferation\n",
      "    - negative regulation of translation\n",
      "    - hippocampus development\n",
      "    - cell differentiation\n",
      "    - prostate gland development\n",
      "    - negative regulation of granulocyte differentiation\n",
      "    - embryonic camera-type eye development\n",
      "    - regulation of myelination\n",
      "    - response to estradiol\n",
      "    - response to retinoic acid\n",
      "    - negative regulation of type II interferon production\n",
      "    - negative regulation of tumor necrosis factor production\n",
      "    - positive regulation of interleukin-13 production\n",
      "    - positive regulation of interleukin-4 production\n",
      "    - positive regulation of interleukin-5 production\n",
      "    - response to vitamin A\n",
      "    - response to cytokine\n",
      "    - multicellular organism growth\n",
      "    - mRNA transcription by RNA polymerase II\n",
      "    - mRNA transcription by RNA polymerase II\n",
      "    - negative regulation of apoptotic process\n",
      "    - apoptotic cell clearance\n",
      "    - response to ethanol\n",
      "    - positive regulation of T-helper 2 cell differentiation\n",
      "    - positive regulation of neuron differentiation\n",
      "    - positive regulation of cell cycle\n",
      "    - negative regulation of DNA-templated transcription\n",
      "    - positive regulation of DNA-templated transcription\n",
      "    - positive regulation of DNA-templated transcription\n",
      "    - positive regulation of transcription by RNA polymerase II\n",
      "    - positive regulation of transcription by RNA polymerase II\n",
      "    - positive regulation of transcription by RNA polymerase II\n",
      "    - positive regulation of transcription by RNA polymerase II\n",
      "    - regulation of synaptic plasticity\n",
      "    - retinoic acid receptor signaling pathway\n",
      "    - retinoic acid receptor signaling pathway\n",
      "    - retinoic acid receptor signaling pathway\n",
      "    - positive regulation of binding\n",
      "    - ventricular cardiac muscle cell differentiation\n",
      "    - Sertoli cell fate commitment\n",
      "    - limb development\n",
      "    - face development\n",
      "    - trachea cartilage development\n",
      "    - chondroblast differentiation\n",
      "    - negative regulation of cartilage development\n",
      "    - cellular response to lipopolysaccharide\n",
      "    - cellular response to retinoic acid\n",
      "    - cellular response to estrogen stimulus\n",
      "    - regulation of hematopoietic progenitor cell differentiation\n",
      "    - negative regulation of miRNA transcription\n",
      "  CC:\n",
      "    - chromatin\n",
      "    - chromatin\n",
      "    - chromatin\n",
      "    - nucleus\n",
      "    - nucleus\n",
      "    - nucleoplasm\n",
      "    - nucleoplasm\n",
      "    - transcription regulator complex\n",
      "    - nucleolus\n",
      "    - cytoplasm\n",
      "    - cytosol\n",
      "    - plasma membrane\n",
      "    - actin cytoskeleton\n",
      "    - dendrite\n",
      "    - protein-containing complex\n",
      "    - perinuclear region of cytoplasm\n",
      "    - RNA polymerase II transcription regulator complex\n",
      "    - RNA polymerase II transcription regulator complex\n",
      "  MF:\n",
      "    - mRNA regulatory element binding translation repressor activity\n",
      "    - RNA polymerase II transcription regulatory region sequence-specific DNA binding\n",
      "    - RNA polymerase II cis-regulatory region sequence-specific DNA binding\n",
      "    - RNA polymerase II cis-regulatory region sequence-specific DNA binding\n",
      "    - RNA polymerase II cis-regulatory region sequence-specific DNA binding\n",
      "    - DNA-binding transcription factor activity, RNA polymerase II-specific\n",
      "    - DNA-binding transcription repressor activity\n",
      "    - transcription coactivator binding\n",
      "    - retinoic acid binding\n",
      "    - chromatin binding\n",
      "    - DNA-binding transcription factor activity\n",
      "    - nuclear receptor activity\n",
      "    - nuclear receptor activity\n",
      "    - signaling receptor binding\n",
      "    - protein binding\n",
      "    - zinc ion binding\n",
      "    - enzyme binding\n",
      "    - protein domain specific binding\n",
      "    - chromatin DNA binding\n",
      "    - histone deacetylase binding\n",
      "    - protein kinase B binding\n",
      "    - retinoic acid-responsive element binding\n",
      "    - mRNA 5'-UTR binding\n",
      "    - protein kinase A binding\n",
      "    - alpha-actinin binding\n",
      "    - heterocyclic compound binding\n",
      "    - sequence-specific double-stranded DNA binding\n",
      "UniProt: {'Swiss-Prot': 'P10276', 'TrEMBL': ['Q6I9R7', 'A8MUP8', 'F1D8N9', 'A8K840', 'J3QRM2', 'J3KSJ4']}\n",
      "\n",
      "Gene: RXRB\n",
      "Symbol: RXRB\n",
      "Name: retinoid X receptor beta\n",
      "Summary: This gene encodes a member of the retinoid X receptor (RXR) family of nuclear receptors which are involved in mediating the effects of retinoic acid (RA). The encoded protein forms homodimers with the retinoic acid, thyroid hormone, and vitamin D receptors, increasing both DNA binding and transcriptional function on their respective response elements. This gene lies within the major histocompatibility complex (MHC) class II region on chromosome 6. Alternatively spliced transcript variants encoding multiple isoforms have been observed for this gene. [provided by RefSeq, Jul 2012].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - nervous system development\n",
      "    - hormone-mediated signaling pathway\n",
      "    - hormone-mediated signaling pathway\n",
      "    - cell differentiation\n",
      "    - positive regulation of bone mineralization\n",
      "    - nuclear receptor-mediated steroid hormone signaling pathway\n",
      "    - mRNA transcription by RNA polymerase II\n",
      "    - mRNA transcription by RNA polymerase II\n",
      "    - positive regulation of DNA-templated transcription\n",
      "    - positive regulation of transcription by RNA polymerase II\n",
      "    - positive regulation of transcription by RNA polymerase II\n",
      "    - positive regulation of transcription by RNA polymerase II\n",
      "    - positive regulation of transcription by RNA polymerase II\n",
      "    - retinoic acid receptor signaling pathway\n",
      "    - positive regulation of vitamin D receptor signaling pathway\n",
      "  CC:\n",
      "    - chromatin\n",
      "    - nucleus\n",
      "    - nucleoplasm\n",
      "    - nucleoplasm\n",
      "    - nucleolus\n",
      "    - cytosol\n",
      "    - RNA polymerase II transcription regulator complex\n",
      "    - RNA polymerase II transcription regulator complex\n",
      "    - RNA polymerase II transcription regulator complex\n",
      "  MF:\n",
      "    - RNA polymerase II transcription regulatory region sequence-specific DNA binding\n",
      "    - DNA-binding transcription factor activity, RNA polymerase II-specific\n",
      "    - DNA-binding transcription activator activity, RNA polymerase II-specific\n",
      "    - nuclear steroid receptor activity\n",
      "    - nuclear receptor activity\n",
      "    - nuclear receptor activity\n",
      "    - protein binding\n",
      "    - zinc ion binding\n",
      "    - retinoic acid-responsive element binding\n",
      "    - sequence-specific double-stranded DNA binding\n",
      "UniProt: {'Swiss-Prot': 'P28702', 'TrEMBL': ['A0A0S2Z570', 'Q5STP9', 'B7Z7J5', 'A0A0G2JKR7', 'A0A0G2JHI1', 'E9PK75']}\n",
      "\n",
      "Gene: RARB\n",
      "Symbol: RARB\n",
      "Name: retinoic acid receptor beta\n",
      "Summary: This gene encodes retinoic acid receptor beta, a member of the thyroid-steroid hormone receptor superfamily of nuclear transcriptional regulators. This receptor localizes to the cytoplasm and to subnuclear compartments. It binds retinoic acid, the biologically active form of vitamin A which mediates cellular signalling in embryonic morphogenesis, cell growth and differentiation. It is thought that this protein limits growth of many cell types by regulating gene expression. The gene was first identified in a hepatocellular carcinoma where it flanks a hepatitis B virus integration site. Alternate promoter usage and differential splicing result in multiple transcript variants. [provided by RefSeq, Mar 2014].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - negative regulation of transcription by RNA polymerase II\n",
      "    - ureteric bud development\n",
      "    - glandular epithelial cell development\n",
      "    - outflow tract septum morphogenesis\n",
      "    - growth plate cartilage development\n",
      "    - apoptotic process\n",
      "    - signal transduction\n",
      "    - striatum development\n",
      "    - neurogenesis\n",
      "    - cell differentiation\n",
      "    - regulation of myelination\n",
      "    - negative regulation of chondrocyte differentiation\n",
      "    - embryonic hindlimb morphogenesis\n",
      "    - multicellular organism growth\n",
      "    - positive regulation of apoptotic process\n",
      "    - negative regulation of apoptotic process\n",
      "    - positive regulation of transcription by RNA polymerase II\n",
      "    - embryonic eye morphogenesis\n",
      "    - retinoic acid receptor signaling pathway\n",
      "    - embryonic digestive tract development\n",
      "    - ventricular cardiac muscle cell differentiation\n",
      "    - neural precursor cell proliferation\n",
      "    - stem cell proliferation\n",
      "    - negative regulation of stem cell proliferation\n",
      "  CC:\n",
      "    - chromatin\n",
      "    - nucleus\n",
      "    - nucleus\n",
      "    - nucleoplasm\n",
      "    - nucleoplasm\n",
      "    - cytoplasm\n",
      "  MF:\n",
      "    - RNA polymerase II cis-regulatory region sequence-specific DNA binding\n",
      "    - DNA-binding transcription factor activity, RNA polymerase II-specific\n",
      "    - DNA binding\n",
      "    - nuclear receptor activity\n",
      "    - protein binding\n",
      "    - zinc ion binding\n",
      "    - protein-containing complex binding\n",
      "    - nuclear retinoid X receptor binding\n",
      "    - heterocyclic compound binding\n",
      "    - sequence-specific double-stranded DNA binding\n",
      "UniProt: {'Swiss-Prot': 'P10826', 'TrEMBL': ['Q5QHG3', 'F1D8S6', 'A0A8I5KWP3', 'Q86UC5', 'Q3SB16', 'A0A8I5KVN9', 'A0A8I5KVP8', 'A0A8I5KUH8', 'A0A8I5KXT3', 'A0A8I5KQX3', 'A0A8I5KVJ9', 'A0A8I5KXD0', 'A0A8I5KNZ0', 'A0A8I5KSR9']}\n",
      "\n",
      "Gene: KITLG\n",
      "Symbol: KITLG\n",
      "Name: KIT ligand\n",
      "Summary: This gene encodes the ligand of the tyrosine-kinase receptor encoded by the KIT locus. This ligand is a pleiotropic factor that acts in utero in germ cell and neural cell development, and hematopoiesis, all believed to reflect a role in cell migration. In adults, it functions pleiotropically, while mostly noted for its continued requirement in hematopoiesis. Two transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - ovarian follicle development\n",
      "    - neural crest cell migration\n",
      "    - hematopoietic progenitor cell differentiation\n",
      "    - myeloid leukocyte differentiation\n",
      "    - positive regulation of leukocyte migration\n",
      "    - positive regulation of myeloid leukocyte differentiation\n",
      "    - cell adhesion\n",
      "    - Ras protein signal transduction\n",
      "    - positive regulation of cell population proliferation\n",
      "    - positive regulation of cell population proliferation\n",
      "    - male gonad development\n",
      "    - mast cell apoptotic process\n",
      "    - negative regulation of mast cell apoptotic process\n",
      "    - embryonic hemopoiesis\n",
      "    - ectopic germ cell programmed cell death\n",
      "    - T cell proliferation\n",
      "    - positive regulation of T cell proliferation\n",
      "    - positive regulation of melanocyte differentiation\n",
      "    - positive regulation of Ras protein signal transduction\n",
      "    - mast cell proliferation\n",
      "    - positive regulation of mast cell proliferation\n",
      "    - extrinsic apoptotic signaling pathway in absence of ligand\n",
      "    - melanocyte migration\n",
      "    - mast cell migration\n",
      "    - positive regulation of hematopoietic progenitor cell differentiation\n",
      "    - positive regulation of hematopoietic stem cell proliferation\n",
      "  CC:\n",
      "    - extracellular region\n",
      "    - extracellular space\n",
      "    - cytoplasm\n",
      "    - cytoskeleton\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - lamellipodium\n",
      "    - filopodium\n",
      "  MF:\n",
      "    - cytokine activity\n",
      "    - stem cell factor receptor binding\n",
      "    - protein binding\n",
      "    - growth factor activity\n",
      "UniProt: {'Swiss-Prot': 'P21583', 'TrEMBL': ['A0A2R8Y515', 'S4R442', 'S4R384']}\n",
      "\n",
      "Gene: PRKCA\n",
      "Symbol: PRKCA\n",
      "Name: protein kinase C alpha\n",
      "Summary: Protein kinase C (PKC) is a family of serine- and threonine-specific protein kinases that can be activated by calcium and the second messenger diacylglycerol. PKC family members phosphorylate a wide variety of protein targets and are known to be involved in diverse cellular signaling pathways. PKC family members also serve as major receptors for phorbol esters, a class of tumor promoters. Each member of the PKC family has a specific expression profile and is believed to play a distinct role in cells. The protein encoded by this gene is one of the PKC family members. This kinase has been reported to play roles in many different cellular processes, such as cell adhesion, cell transformation, cell cycle checkpoint, and cell volume control. Knockout studies in mice suggest that this kinase may be a fundamental regulator of cardiac contractility and Ca(2+) handling in myocytes. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - angiogenesis\n",
      "    - positive regulation of endothelial cell proliferation\n",
      "    - desmosome assembly\n",
      "    - chromatin remodeling\n",
      "    - protein phosphorylation\n",
      "    - mitotic nuclear membrane disassembly\n",
      "    - cell adhesion\n",
      "    - positive regulation of endothelial cell migration\n",
      "    - positive regulation of cardiac muscle hypertrophy\n",
      "    - peptidyl-threonine phosphorylation\n",
      "    - positive regulation of cell migration\n",
      "    - positive regulation of lipopolysaccharide-mediated signaling pathway\n",
      "    - positive regulation of insulin secretion\n",
      "    - negative regulation of glial cell apoptotic process\n",
      "    - intracellular signal transduction\n",
      "    - regulation of mRNA stability\n",
      "    - positive regulation of blood vessel endothelial cell migration\n",
      "    - positive regulation of macrophage differentiation\n",
      "    - positive regulation of angiogenesis\n",
      "    - positive regulation of bone resorption\n",
      "    - positive regulation of cell adhesion\n",
      "    - positive regulation of mitotic cell cycle\n",
      "    - positive regulation of phosphatidylinositol 3-kinase/protein kinase B signal transduction\n",
      "    - positive regulation of ERK1 and ERK2 cascade\n",
      "    - protein kinase C signaling\n",
      "    - response to interleukin-1\n",
      "    - regulation of platelet aggregation\n",
      "    - apoptotic signaling pathway\n",
      "    - positive regulation of adenylate cyclase-activating G protein-coupled receptor signaling pathway\n",
      "    - positive regulation of angiotensin-activated signaling pathway\n",
      "    - negative regulation of cytokine production involved in inflammatory response\n",
      "    - positive regulation of dense core granule biogenesis\n",
      "  CC:\n",
      "    - nucleoplasm\n",
      "    - cytoplasm\n",
      "    - mitochondrion\n",
      "    - endoplasmic reticulum\n",
      "    - cytosol\n",
      "    - cytosol\n",
      "    - cytosol\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - mitochondrial membrane\n",
      "    - alphav-beta3 integrin-PKCalpha complex\n",
      "    - ciliary basal body\n",
      "    - perinuclear region of cytoplasm\n",
      "    - extracellular exosome\n",
      "  MF:\n",
      "    - protein kinase activity\n",
      "    - protein serine/threonine kinase activity\n",
      "    - diacylglycerol-dependent serine/threonine kinase activity\n",
      "    - diacylglycerol-dependent serine/threonine kinase activity\n",
      "    - calcium,diacylglycerol-dependent serine/threonine kinase activity\n",
      "    - calcium,diacylglycerol-dependent serine/threonine kinase activity\n",
      "    - calcium,diacylglycerol-dependent serine/threonine kinase activity\n",
      "    - diacylglycerol-dependent, calcium-independent serine/threonine kinase activity\n",
      "    - integrin binding\n",
      "    - protein binding\n",
      "    - ATP binding\n",
      "    - zinc ion binding\n",
      "    - enzyme binding\n",
      "    - diacylglycerol binding\n",
      "    - histone H3T6 kinase activity\n",
      "    - protein serine kinase activity\n",
      "UniProt: {'Swiss-Prot': 'P17252', 'TrEMBL': ['L7RSM7', 'Q7Z727', 'J3KN97', 'J3KRN5']}\n",
      "\n",
      "Gene: ERN1\n",
      "Symbol: ERN1\n",
      "Name: endoplasmic reticulum to nucleus signaling 1\n",
      "Summary: This gene encodes the transmembrane protein kinase inositol-requiring enzyme 1. The encoded protein contains two functional catalytic domains, a serine/threonine-protein kinase domain and an endoribonuclease domain. This protein functions as a sensor of unfolded proteins in the endoplasmic reticulum (ER) and triggers an intracellular signaling pathway termed the unfolded protein response (UPR). The UPR is an ER stress response that is conserved from yeast to mammals and activates genes involved in degrading misfolded proteins, regulating protein synthesis and activating molecular chaperones. This protein specifically mediates the splicing and activation of the stress response transcription factor X-box binding protein 1. [provided by RefSeq, Aug 2017].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - endothelial cell proliferation\n",
      "    - chromatin remodeling\n",
      "    - mRNA catabolic process\n",
      "    - protein phosphorylation\n",
      "    - regulation of macroautophagy\n",
      "    - positive regulation of RNA splicing\n",
      "    - cellular response to unfolded protein\n",
      "    - cellular response to unfolded protein\n",
      "    - response to endoplasmic reticulum stress\n",
      "    - cellular response to vascular endothelial growth factor stimulus\n",
      "    - peptidyl-serine autophosphorylation\n",
      "    - IRE1-mediated unfolded protein response\n",
      "    - IRE1-mediated unfolded protein response\n",
      "    - IRE1-mediated unfolded protein response\n",
      "    - positive regulation of JUN kinase activity\n",
      "    - protein autophosphorylation\n",
      "    - mRNA splicing, via endonucleolytic cleavage and ligation\n",
      "    - mRNA splicing, via endonucleolytic cleavage and ligation\n",
      "    - mRNA splicing, via endonucleolytic cleavage and ligation\n",
      "    - intrinsic apoptotic signaling pathway in response to endoplasmic reticulum stress\n",
      "    - intrinsic apoptotic signaling pathway in response to endoplasmic reticulum stress\n",
      "    - cellular response to hydrogen peroxide\n",
      "    - cellular response to glucose stimulus\n",
      "    - positive regulation of endoplasmic reticulum unfolded protein response\n",
      "    - insulin metabolic process\n",
      "    - positive regulation of vascular associated smooth muscle cell proliferation\n",
      "    - peptidyl-serine trans-autophosphorylation\n",
      "  CC:\n",
      "    - nuclear inner membrane\n",
      "    - cytoplasm\n",
      "    - mitochondrion\n",
      "    - endoplasmic reticulum\n",
      "    - endoplasmic reticulum membrane\n",
      "    - endoplasmic reticulum membrane\n",
      "    - Ire1 complex\n",
      "    - AIP1-IRE1 complex\n",
      "    - IRE1-TRAF2-ASK1 complex\n",
      "    - IRE1-TRAF2-ASK1 complex\n",
      "    - IRE1-RACK1-PP2A complex\n",
      "  MF:\n",
      "    - magnesium ion binding\n",
      "    - RNA endonuclease activity\n",
      "    - RNA endonuclease activity\n",
      "    - protein serine/threonine kinase activity\n",
      "    - protein serine/threonine kinase activity\n",
      "    - 3-phosphoinositide-dependent protein kinase activity\n",
      "    - DNA-dependent protein kinase activity\n",
      "    - AMP-activated protein kinase activity\n",
      "    - eukaryotic translation initiation factor 2alpha kinase activity\n",
      "    - ribosomal protein S6 kinase activity\n",
      "    - platelet-derived growth factor receptor binding\n",
      "    - protein binding\n",
      "    - ATP binding\n",
      "    - enzyme binding\n",
      "    - Hsp70 protein binding\n",
      "    - histone H3S10 kinase activity\n",
      "    - histone H3T11 kinase activity\n",
      "    - histone H3T6 kinase activity\n",
      "    - histone H2AXS139 kinase activity\n",
      "    - identical protein binding\n",
      "    - protein homodimerization activity\n",
      "    - protein homodimerization activity\n",
      "    - protein homodimerization activity\n",
      "    - ADP binding\n",
      "    - histone H3S28 kinase activity\n",
      "    - histone H4S1 kinase activity\n",
      "    - histone H2AS1 kinase activity\n",
      "    - histone H2BS14 kinase activity\n",
      "    - unfolded protein binding\n",
      "    - Hsp90 protein binding\n",
      "    - histone H3T3 kinase activity\n",
      "    - histone H2AS121 kinase activity\n",
      "    - Rho-dependent protein serine/threonine kinase activity\n",
      "    - protein serine kinase activity\n",
      "    - histone H2BS36 kinase activity\n",
      "    - histone H3S57 kinase activity\n",
      "    - histone H3T45 kinase activity\n",
      "    - histone H2AT120 kinase activity\n",
      "UniProt: {'Swiss-Prot': 'O75460', 'TrEMBL': ['A0A7P0TAB0', 'A0A7P0TA38']}\n",
      "\n",
      "Gene: NF2\n",
      "Symbol: NF2\n",
      "Name: NF2, moesin-ezrin-radixin like (MERLIN) tumor suppressor\n",
      "Summary: This gene encodes a protein that is similar to some members of the ERM (ezrin, radixin, moesin) family of proteins that link cytoskeletal components with proteins in the cell membrane. The encoded protein is involved in regulation of contact-dependent inhibition of cell proliferation and functions in cell-cell adhesion and transmembrane signaling. The encoded protein has been shown to interact with cell-surface proteins, proteins involved in cytoskeletal dynamics, and proteins involved in regulating ion transport. Disruption of this protein's function has been implicated in tumorigenesis and metastasis. Mutations in this gene are associated with neurofibromatosis type II which is characterized by nervous system and skin tumors and ocular abnormalities. [provided by RefSeq, May 2022].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - MAPK cascade\n",
      "    - mesoderm formation\n",
      "    - negative regulation of cell-matrix adhesion\n",
      "    - ectoderm development\n",
      "    - negative regulation of cell population proliferation\n",
      "    - negative regulation of cell population proliferation\n",
      "    - negative regulation of cell population proliferation\n",
      "    - regulation of cell shape\n",
      "    - negative regulation of Schwann cell proliferation\n",
      "    - Schwann cell proliferation\n",
      "    - regulation of gliogenesis\n",
      "    - hippocampus development\n",
      "    - negative regulation of cell-cell adhesion\n",
      "    - actin cytoskeleton organization\n",
      "    - negative regulation of cell migration\n",
      "    - regulation of protein stability\n",
      "    - osteoblast proliferation\n",
      "    - negative regulation of osteoblast proliferation\n",
      "    - regulation of hippo signaling\n",
      "    - regulation of hippo signaling\n",
      "    - odontogenesis of dentin-containing tooth\n",
      "    - regulation of apoptotic process\n",
      "    - negative regulation of MAPK cascade\n",
      "    - cell-cell junction organization\n",
      "    - positive regulation of cell differentiation\n",
      "    - negative regulation of receptor signaling pathway via JAK-STAT\n",
      "    - positive regulation of stress fiber assembly\n",
      "    - regulation of cell cycle\n",
      "    - lens fiber cell differentiation\n",
      "    - regulation of stem cell proliferation\n",
      "    - regulation of protein localization to nucleus\n",
      "    - regulation of organelle assembly\n",
      "    - positive regulation of protein localization to early endosome\n",
      "    - regulation of neural precursor cell proliferation\n",
      "    - positive regulation of early endosome to late endosome transport\n",
      "  CC:\n",
      "    - nucleus\n",
      "    - nucleolus\n",
      "    - cytoplasm\n",
      "    - early endosome\n",
      "    - cytosol\n",
      "    - cytoskeleton\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - adherens junction\n",
      "    - membrane\n",
      "    - lamellipodium\n",
      "    - filopodium\n",
      "    - cortical actin cytoskeleton\n",
      "    - filopodium membrane\n",
      "    - cleavage furrow\n",
      "    - ruffle membrane\n",
      "    - neuron projection\n",
      "    - cell body\n",
      "    - apical part of cell\n",
      "    - perinuclear region of cytoplasm\n",
      "  MF:\n",
      "    - actin binding\n",
      "    - integrin binding\n",
      "    - protein binding\n",
      "UniProt: {'Swiss-Prot': 'P35240', 'TrEMBL': ['A0A5F9ZHD8', 'A0A5F9ZHA0', 'A0A5K1VW66']}\n",
      "\n",
      "Gene: CDKN1B\n",
      "Symbol: CDKN1B\n",
      "Name: cyclin dependent kinase inhibitor 1B\n",
      "Summary: This gene encodes a cyclin-dependent kinase inhibitor, which shares a limited similarity with CDK inhibitor CDKN1A/p21. The encoded protein binds to and prevents the activation of cyclin E-CDK2 or cyclin D-CDK4 complexes, and thus controls the cell cycle progression at G1. The degradation of this protein, which is triggered by its CDK dependent phosphorylation and subsequent ubiquitination by SCF complexes, is required for the cellular transition from quiescence to the proliferative state. Mutations in this gene are associated with multiple endocrine neoplasia type IV (MEN4). [provided by RefSeq, Apr 2014].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - regulation of cyclin-dependent protein serine/threonine kinase activity\n",
      "    - G1/S transition of mitotic cell cycle\n",
      "    - G1/S transition of mitotic cell cycle\n",
      "    - placenta development\n",
      "    - potassium ion transport\n",
      "    - regulation of exit from mitosis\n",
      "    - Notch signaling pathway\n",
      "    - heart development\n",
      "    - sensory perception of sound\n",
      "    - positive regulation of cell population proliferation\n",
      "    - negative regulation of cell population proliferation\n",
      "    - response to xenobiotic stimulus\n",
      "    - response to glucose\n",
      "    - negative regulation of cell growth\n",
      "    - regulation of cell migration\n",
      "    - positive regulation of microtubule polymerization\n",
      "    - response to estradiol\n",
      "    - negative regulation of kinase activity\n",
      "    - negative regulation of phosphorylation\n",
      "    - response to amino acid\n",
      "    - response to peptide hormone\n",
      "    - positive regulation of protein catabolic process\n",
      "    - negative regulation of cyclin-dependent protein serine/threonine kinase activity\n",
      "    - positive regulation of DNA replication\n",
      "    - negative regulation of DNA-templated transcription\n",
      "    - negative regulation of mitotic cell cycle\n",
      "    - response to cadmium ion\n",
      "    - autophagic cell death\n",
      "    - inner ear development\n",
      "    - negative regulation of epithelial cell proliferation\n",
      "    - nuclear export\n",
      "    - regulation of cell cycle\n",
      "    - regulation of cell cycle\n",
      "    - epithelial cell proliferation involved in prostate gland development\n",
      "    - negative regulation of epithelial cell proliferation involved in prostate gland development\n",
      "    - cellular response to antibiotic\n",
      "    - cellular response to lithium ion\n",
      "    - cellular response to hypoxia\n",
      "    - cellular senescence\n",
      "    - regulation of lens fiber cell differentiation\n",
      "    - regulation of cell cycle G1/S phase transition\n",
      "    - epithelial cell apoptotic process\n",
      "    - negative regulation of cyclin-dependent protein kinase activity\n",
      "    - negative regulation of epithelial cell apoptotic process\n",
      "    - negative regulation of vascular associated smooth muscle cell proliferation\n",
      "    - negative regulation of cardiac muscle tissue regeneration\n",
      "    - regulation of G1/S transition of mitotic cell cycle\n",
      "  CC:\n",
      "    - nucleus\n",
      "    - nucleus\n",
      "    - nucleoplasm\n",
      "    - nucleoplasm\n",
      "    - cytoplasm\n",
      "    - cytoplasm\n",
      "    - endosome\n",
      "    - centrosome\n",
      "    - cytosol\n",
      "    - cytosol\n",
      "    - cilium\n",
      "    - Cul4A-RING E3 ubiquitin ligase complex\n",
      "    - ciliary basal body\n",
      "  MF:\n",
      "    - protein kinase inhibitor activity\n",
      "    - cyclin-dependent protein serine/threonine kinase inhibitor activity\n",
      "    - cyclin-dependent protein serine/threonine kinase inhibitor activity\n",
      "    - cyclin-dependent protein serine/threonine kinase inhibitor activity\n",
      "    - protein binding\n",
      "    - protein kinase binding\n",
      "    - protein phosphatase binding\n",
      "    - cyclin-dependent protein kinase activating kinase regulator activity\n",
      "    - cyclin binding\n",
      "    - cyclin binding\n",
      "    - Hsp70 protein binding\n",
      "    - ubiquitin protein ligase binding\n",
      "    - protein-containing complex binding\n",
      "    - protein-folding chaperone binding\n",
      "    - molecular adaptor activity\n",
      "    - molecular function inhibitor activity\n",
      "    - ubiquitin ligase activator activity\n",
      "UniProt: {'Swiss-Prot': 'P46527', 'TrEMBL': ['E7ES52', 'Q6I9V6', 'H7C2T1']}\n",
      "\n",
      "Gene: CFTR\n",
      "Symbol: CFTR\n",
      "Name: CF transmembrane conductance regulator\n",
      "Summary: This gene encodes a member of the ATP-binding cassette (ABC) transporter superfamily. The encoded protein functions as a chloride channel, making it unique among members of this protein family, and controls ion and water secretion and absorption in epithelial tissues. Channel activation is mediated by cycles of regulatory domain phosphorylation, ATP-binding by the nucleotide-binding domains, and ATP hydrolysis. Mutations in this gene cause cystic fibrosis, the most common lethal genetic disorder in populations of Northern European descent. The most frequently occurring mutation in cystic fibrosis, DeltaF508, results in impaired folding and trafficking of the encoded protein. Multiple pseudogenes have been identified in the human genome. [provided by RefSeq, Aug 2017].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - cholesterol biosynthetic process\n",
      "    - vesicle docking involved in exocytosis\n",
      "    - bicarbonate transport\n",
      "    - bicarbonate transport\n",
      "    - cholesterol transport\n",
      "    - response to endoplasmic reticulum stress\n",
      "    - transepithelial water transport\n",
      "    - positive regulation of insulin secretion involved in cellular response to glucose stimulus\n",
      "    - positive regulation of insulin secretion involved in cellular response to glucose stimulus\n",
      "    - positive regulation of exocytosis\n",
      "    - sperm capacitation\n",
      "    - multicellular organismal-level water homeostasis\n",
      "    - intracellular pH elevation\n",
      "    - establishment of localization in cell\n",
      "    - transmembrane transport\n",
      "    - membrane hyperpolarization\n",
      "    - positive regulation of enamel mineralization\n",
      "    - cellular response to cAMP\n",
      "    - amelogenesis\n",
      "    - positive regulation of cyclic nucleotide-gated ion channel activity\n",
      "    - chloride transmembrane transport\n",
      "    - chloride transmembrane transport\n",
      "    - chloride transmembrane transport\n",
      "    - chloride transmembrane transport\n",
      "    - positive regulation of voltage-gated chloride channel activity\n",
      "    - cellular response to forskolin\n",
      "  CC:\n",
      "    - nucleus\n",
      "    - cytoplasm\n",
      "    - lysosomal membrane\n",
      "    - early endosome\n",
      "    - endoplasmic reticulum membrane\n",
      "    - endoplasmic reticulum membrane\n",
      "    - cytosol\n",
      "    - cytosol\n",
      "    - cytosol\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - cell surface\n",
      "    - endosome membrane\n",
      "    - membrane\n",
      "    - apical plasma membrane\n",
      "    - apical plasma membrane\n",
      "    - apical plasma membrane\n",
      "    - Golgi-associated vesicle membrane\n",
      "    - clathrin-coated endocytic vesicle membrane\n",
      "    - early endosome membrane\n",
      "    - protein-containing complex\n",
      "    - chloride channel complex\n",
      "    - recycling endosome\n",
      "    - recycling endosome membrane\n",
      "  MF:\n",
      "    - chloride channel activity\n",
      "    - chloride channel activity\n",
      "    - intracellularly ATP-gated chloride channel activity\n",
      "    - intracellularly ATP-gated chloride channel activity\n",
      "    - protein binding\n",
      "    - ATP binding\n",
      "    - bicarbonate transmembrane transporter activity\n",
      "    - bicarbonate transmembrane transporter activity\n",
      "    - chloride transmembrane transporter activity\n",
      "    - chloride transmembrane transporter activity\n",
      "    - isomerase activity\n",
      "    - ATP hydrolysis activity\n",
      "    - chloride channel regulator activity\n",
      "    - chloride channel inhibitor activity\n",
      "    - enzyme binding\n",
      "    - PDZ domain binding\n",
      "    - ATPase-coupled transmembrane transporter activity\n",
      "    - ATPase-coupled inorganic anion transmembrane transporter activity\n",
      "    - protein-folding chaperone binding\n",
      "    - 14-3-3 protein binding\n",
      "    - 14-3-3 protein binding\n",
      "    - 14-3-3 protein binding\n",
      "    - Sec61 translocon complex binding\n",
      "    - ABC-type transporter activity\n",
      "UniProt: {'Swiss-Prot': 'P13569', 'TrEMBL': ['A0A3B3ITE0', 'A0A3B3IT97', 'A0A8I5KVL1', 'A0A8I5KVV2', 'A0A8V8TNG7', 'A0A3B3ITW0', 'A0A8V8TNN0', 'A0A8V8TNH2', 'A0A8V8TQ94', 'A0A669KBE8', 'A0A8V8TPV6', 'A0A3B3ITW5', 'M0QYZ3', 'A0A8V8TQ89', 'A0A8V8TNN7', 'E7EPB6', 'A0A8I5KXQ9', 'H0Y8A9', 'C9J6L5']}\n",
      "\n",
      "Gene: PSEN1\n",
      "Symbol: PSEN1\n",
      "Name: presenilin 1\n",
      "Summary: Alzheimer's disease (AD) patients with an inherited form of the disease carry mutations in the presenilin proteins (PSEN1; PSEN2) or in the amyloid precursor protein (APP). These disease-linked mutations result in increased production of the longer form of amyloid-beta (main component of amyloid deposits found in AD brains). Presenilins are postulated to regulate APP processing through their effects on gamma-secretase, an enzyme that cleaves APP. Also, it is thought that the presenilins are involved in the cleavage of the Notch receptor, such that they either directly regulate gamma-secretase activity or themselves are protease enzymes. Several alternatively spliced transcript variants encoding different isoforms have been identified for this gene, the full-length nature of only some have been determined. [provided by RefSeq, Aug 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - autophagosome assembly\n",
      "    - negative regulation of transcription by RNA polymerase II\n",
      "    - blood vessel development\n",
      "    - cell fate specification\n",
      "    - somitogenesis\n",
      "    - neuron migration\n",
      "    - positive regulation of receptor recycling\n",
      "    - heart looping\n",
      "    - positive regulation of L-glutamate import across plasma membrane\n",
      "    - hematopoietic progenitor cell differentiation\n",
      "    - astrocyte activation involved in immune response\n",
      "    - T cell activation involved in immune response\n",
      "    - neural retina development\n",
      "    - protein glycosylation\n",
      "    - membrane protein ectodomain proteolysis\n",
      "    - membrane protein ectodomain proteolysis\n",
      "    - mitochondrial transport\n",
      "    - DNA damage response\n",
      "    - response to oxidative stress\n",
      "    - Notch signaling pathway\n",
      "    - Notch receptor processing\n",
      "    - Notch receptor processing\n",
      "    - learning or memory\n",
      "    - memory\n",
      "    - post-embryonic development\n",
      "    - regulation of gene expression\n",
      "    - positive regulation of gene expression\n",
      "    - negative regulation of gene expression\n",
      "    - regulation of neuron projection development\n",
      "    - protein transport\n",
      "    - choline transport\n",
      "    - synaptic vesicle targeting\n",
      "    - protein processing\n",
      "    - protein processing\n",
      "    - protein processing\n",
      "    - protein processing\n",
      "    - cerebellum development\n",
      "    - cerebral cortex cell migration\n",
      "    - Cajal-Retzius cell differentiation\n",
      "    - dorsal/ventral neural tube patterning\n",
      "    - embryonic limb morphogenesis\n",
      "    - positive regulation of proteasomal ubiquitin-dependent protein catabolic process\n",
      "    - endoplasmic reticulum calcium ion homeostasis\n",
      "    - endoplasmic reticulum calcium ion homeostasis\n",
      "    - endoplasmic reticulum calcium ion homeostasis\n",
      "    - positive regulation of tumor necrosis factor production\n",
      "    - amyloid-beta formation\n",
      "    - amyloid-beta formation\n",
      "    - amyloid-beta formation\n",
      "    - amyloid-beta formation\n",
      "    - intracellular signal transduction\n",
      "    - locomotion\n",
      "    - positive regulation of protein import into nucleus\n",
      "    - regulation of phosphorylation\n",
      "    - amyloid precursor protein metabolic process\n",
      "    - amyloid precursor protein catabolic process\n",
      "    - amyloid precursor protein catabolic process\n",
      "    - amyloid precursor protein catabolic process\n",
      "    - amyloid precursor protein catabolic process\n",
      "    - myeloid dendritic cell differentiation\n",
      "    - positive regulation of apoptotic process\n",
      "    - negative regulation of apoptotic process\n",
      "    - negative regulation of apoptotic process\n",
      "    - negative regulation of neuron apoptotic process\n",
      "    - skin morphogenesis\n",
      "    - positive regulation of glycolytic process\n",
      "    - positive regulation of DNA-templated transcription\n",
      "    - astrocyte activation\n",
      "    - regulation of synaptic plasticity\n",
      "    - thymus development\n",
      "    - neuron development\n",
      "    - skeletal system morphogenesis\n",
      "    - brain morphogenesis\n",
      "    - epithelial cell proliferation\n",
      "    - negative regulation of axonogenesis\n",
      "    - synapse organization\n",
      "    - positive regulation of coagulation\n",
      "    - T cell receptor signaling pathway\n",
      "    - sequestering of calcium ion\n",
      "    - neuron apoptotic process\n",
      "    - smooth endoplasmic reticulum calcium ion homeostasis\n",
      "    - regulation of synaptic transmission, glutamatergic\n",
      "    - calcium ion homeostasis\n",
      "    - regulation of resting membrane potential\n",
      "    - regulation of canonical Wnt signaling pathway\n",
      "    - positive regulation of dendritic spine development\n",
      "    - neuron cellular homeostasis\n",
      "    - calcium ion transmembrane transport\n",
      "    - apoptotic signaling pathway\n",
      "    - cell-cell adhesion\n",
      "    - regulation of synaptic vesicle cycle\n",
      "    - L-glutamate import across plasma membrane\n",
      "    - regulation of postsynapse organization\n",
      "    - protein catabolic process at postsynapse\n",
      "    - cellular response to amyloid-beta\n",
      "    - negative regulation of core promoter binding\n",
      "    - positive regulation of amyloid fibril formation\n",
      "    - neuron projection maintenance\n",
      "    - negative regulation of ubiquitin-dependent protein catabolic process\n",
      "    - negative regulation of apoptotic signaling pathway\n",
      "  CC:\n",
      "    - Golgi membrane\n",
      "    - kinetochore\n",
      "    - nucleus\n",
      "    - nuclear outer membrane\n",
      "    - nucleoplasm\n",
      "    - mitochondrion\n",
      "    - mitochondrial inner membrane\n",
      "    - endoplasmic reticulum\n",
      "    - endoplasmic reticulum membrane\n",
      "    - smooth endoplasmic reticulum\n",
      "    - smooth endoplasmic reticulum\n",
      "    - rough endoplasmic reticulum\n",
      "    - Golgi apparatus\n",
      "    - centrosome\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - cell cortex\n",
      "    - synaptic vesicle\n",
      "    - cell surface\n",
      "    - membrane\n",
      "    - membrane\n",
      "    - aggresome\n",
      "    - cell junction\n",
      "    - dendrite\n",
      "    - growth cone\n",
      "    - neuromuscular junction\n",
      "    - early endosome membrane\n",
      "    - nuclear membrane\n",
      "    - protein-containing complex\n",
      "    - ciliary rootlet\n",
      "    - azurophil granule membrane\n",
      "    - sarcolemma\n",
      "    - presynaptic membrane\n",
      "    - neuron projection\n",
      "    - neuronal cell body\n",
      "    - dendritic shaft\n",
      "    - membrane raft\n",
      "    - gamma-secretase complex\n",
      "    - gamma-secretase complex\n",
      "    - gamma-secretase complex\n",
      "    - postsynapse\n",
      "    - glutamatergic synapse\n",
      "  MF:\n",
      "    - endopeptidase activity\n",
      "    - endopeptidase activity\n",
      "    - endopeptidase activity\n",
      "    - aspartic-type endopeptidase activity\n",
      "    - calcium channel activity\n",
      "    - protein binding\n",
      "    - beta-catenin binding\n",
      "    - PDZ domain binding\n",
      "    - aspartic endopeptidase activity, intramembrane cleaving\n",
      "    - aspartic endopeptidase activity, intramembrane cleaving\n",
      "    - aspartic endopeptidase activity, intramembrane cleaving\n",
      "    - cadherin binding\n",
      "    - ATPase binding\n",
      "    - growth factor receptor binding\n",
      "UniProt: {'Swiss-Prot': 'P49768', 'TrEMBL': ['A0A024R6A3', 'A0A0S2Z4D2', 'A0A8V8TPL0', 'H0YNU4', 'A0A8V8TR52', 'A0A8V8TQU9', 'G3V490', 'A0A8V8TQ93', 'H0YM52', 'A0A8V8TQV2', 'A0A8V8TQ53', 'G3V3P0', 'A0A8V8TQU7', 'A0A8V8TR55', 'A0A8V8TQ58', 'G3V519', 'G3V449', 'A0A8V8TPL5', 'A0A8V8TQ36', 'A0A8V8TQ30', 'A0A8V8TLT5', 'A0A8V8TQT3', 'A0A8V8TQ90', 'A0A8V8TPK5', 'A0A8V8TR50', 'G3V2G7', 'A0A8V8TQ54', 'G3V3Z0', 'G3V4M0']}\n",
      "\n",
      "Gene: NCSTN\n",
      "Symbol: NCSTN\n",
      "Name: nicastrin\n",
      "Summary: This gene encodes a type I transmembrane glycoprotein that is an integral component of the multimeric gamma-secretase complex. The encoded protein cleaves integral membrane proteins, including Notch receptors and beta-amyloid precursor protein, and may be a stabilizing cofactor required for gamma-secretase complex assembly. The cleavage of beta-amyloid precursor protein yields amyloid beta peptide, the main component of the neuritic plaque and the hallmark lesion in the brains of patients with Alzheimer's disease; however, the nature of the encoded protein's role in Alzheimer's disease is not known for certain. Mutations in this gene are associated with familial acne inversa. A pseudogene of this gene is present on chromosome 21. Alternatively spliced transcript variants of this gene have been described, but the full-length nature of some of these variants has not been determined. [provided by RefSeq, Feb 2014].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - myeloid cell homeostasis\n",
      "    - proteolysis\n",
      "    - membrane protein ectodomain proteolysis\n",
      "    - G protein-coupled dopamine receptor signaling pathway\n",
      "    - glutamate receptor signaling pathway\n",
      "    - Notch signaling pathway\n",
      "    - Notch receptor processing\n",
      "    - Notch receptor processing\n",
      "    - Notch receptor processing\n",
      "    - Notch receptor processing\n",
      "    - Notch receptor processing\n",
      "    - learning or memory\n",
      "    - protein processing\n",
      "    - protein processing\n",
      "    - protein processing\n",
      "    - protein processing\n",
      "    - cerebellum development\n",
      "    - central nervous system myelination\n",
      "    - adult behavior\n",
      "    - membrane protein intracellular domain proteolysis\n",
      "    - membrane protein intracellular domain proteolysis\n",
      "    - amyloid-beta formation\n",
      "    - amyloid-beta formation\n",
      "    - amyloid-beta formation\n",
      "    - T cell proliferation\n",
      "    - amyloid precursor protein metabolic process\n",
      "    - amyloid precursor protein biosynthetic process\n",
      "    - positive regulation of amyloid precursor protein biosynthetic process\n",
      "    - amyloid precursor protein catabolic process\n",
      "    - amyloid precursor protein catabolic process\n",
      "    - amyloid precursor protein catabolic process\n",
      "    - amyloid precursor protein catabolic process\n",
      "    - epithelial cell proliferation\n",
      "    - neuron apoptotic process\n",
      "    - cellular response to calcium ion\n",
      "    - regulation of long-term synaptic potentiation\n",
      "    - short-term synaptic potentiation\n",
      "  CC:\n",
      "    - Golgi membrane\n",
      "    - lysosomal membrane\n",
      "    - early endosome\n",
      "    - endoplasmic reticulum\n",
      "    - endoplasmic reticulum membrane\n",
      "    - Golgi apparatus\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - focal adhesion\n",
      "    - synaptic vesicle\n",
      "    - endosome membrane\n",
      "    - membrane\n",
      "    - membrane\n",
      "    - membrane\n",
      "    - azurophil granule membrane\n",
      "    - sarcolemma\n",
      "    - melanosome\n",
      "    - presynaptic membrane\n",
      "    - extracellular exosome\n",
      "    - gamma-secretase complex\n",
      "    - gamma-secretase complex\n",
      "    - gamma-secretase complex\n",
      "    - gamma-secretase complex\n",
      "    - gamma-secretase complex\n",
      "  MF:\n",
      "    - protein binding\n",
      "    - protein-macromolecule adaptor activity\n",
      "    - protein-macromolecule adaptor activity\n",
      "    - aspartic endopeptidase activity, intramembrane cleaving\n",
      "    - ATPase binding\n",
      "    - endopeptidase activator activity\n",
      "    - endopeptidase activator activity\n",
      "    - growth factor receptor binding\n",
      "UniProt: {'Swiss-Prot': 'Q92542', 'TrEMBL': ['B4DR82', 'A0A8V8TPQ8', 'A0A8V8TPR8', 'A0A2R8YCK1', 'A0A8V8TNU8', 'H0Y6T7', 'A0A8V8TNC8', 'A0A2U3TZL9', 'Q5T209', 'A0A8V8TNV3', 'A0A8V8TQ51', 'H0Y3Z4', 'Q5T210', 'Q5T211', 'A0A2R8YE20', 'Q5T205', 'A0A8V8TNI3', 'A0A8V8TNI9', 'A0A8V8TNV7', 'A0A8V8TQ61']}\n",
      "\n",
      "Gene: CD3E\n",
      "Symbol: CD3E\n",
      "Name: CD3 epsilon subunit of T-cell receptor complex\n",
      "Summary: The protein encoded by this gene is the CD3-epsilon polypeptide, which together with CD3-gamma, -delta and -zeta, and the T-cell receptor alpha/beta and gamma/delta heterodimers, forms the T-cell receptor-CD3 complex. This complex plays an important role in coupling antigen recognition to several intracellular signal-transduction pathways. The genes encoding the epsilon, gamma and delta polypeptides are located in the same cluster on chromosome 11. The epsilon polypeptide plays an essential role in T-cell development. Defects in this gene cause immunodeficiency. This gene has also been linked to a susceptibility to type I diabetes in women. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - positive regulation of cell-matrix adhesion\n",
      "    - adaptive immune response\n",
      "    - positive regulation of T cell anergy\n",
      "    - T cell anergy\n",
      "    - cell surface receptor signaling pathway\n",
      "    - cell surface receptor signaling pathway\n",
      "    - cell surface receptor signaling pathway\n",
      "    - cell surface receptor protein tyrosine kinase signaling pathway\n",
      "    - signal complex assembly\n",
      "    - G protein-coupled receptor signaling pathway\n",
      "    - smoothened signaling pathway\n",
      "    - positive regulation of gene expression\n",
      "    - negative regulation of gene expression\n",
      "    - dendrite development\n",
      "    - calcium-mediated signaling\n",
      "    - cerebellum development\n",
      "    - T cell costimulation\n",
      "    - positive regulation of type II interferon production\n",
      "    - positive regulation of interleukin-2 production\n",
      "    - positive regulation of interleukin-4 production\n",
      "    - positive regulation of cell-cell adhesion mediated by integrin\n",
      "    - CD4-positive, alpha-beta T cell proliferation\n",
      "    - positive regulation of T cell proliferation\n",
      "    - T cell activation\n",
      "    - regulation of apoptotic process\n",
      "    - positive thymic T cell selection\n",
      "    - negative thymic T cell selection\n",
      "    - negative regulation of smoothened signaling pathway\n",
      "    - gamma-delta T cell activation\n",
      "    - alpha-beta T cell activation\n",
      "    - positive regulation of calcium-mediated signaling\n",
      "    - T cell receptor signaling pathway\n",
      "    - T cell receptor signaling pathway\n",
      "    - protein-containing complex assembly\n",
      "    - apoptotic signaling pathway\n",
      "    - positive regulation of CD4-positive, alpha-beta T cell proliferation\n",
      "  CC:\n",
      "    - immunological synapse\n",
      "    - endoplasmic reticulum\n",
      "    - Golgi apparatus\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - cell-cell junction\n",
      "    - external side of plasma membrane\n",
      "    - external side of plasma membrane\n",
      "    - T cell receptor complex\n",
      "    - alpha-beta T cell receptor complex\n",
      "    - alpha-beta T cell receptor complex\n",
      "    - alpha-beta T cell receptor complex\n",
      "    - alpha-beta T cell receptor complex\n",
      "    - gamma-delta T cell receptor complex\n",
      "    - dendritic spine\n",
      "    - cell body\n",
      "  MF:\n",
      "    - transmembrane signaling receptor activity\n",
      "    - transmembrane signaling receptor activity\n",
      "    - transmembrane signaling receptor activity\n",
      "    - protein binding\n",
      "    - SH3 domain binding\n",
      "    - protein kinase binding\n",
      "    - signaling receptor complex adaptor activity\n",
      "    - T cell receptor binding\n",
      "    - identical protein binding\n",
      "UniProt: {'Swiss-Prot': 'P07766', 'TrEMBL': 'E9PSH8'}\n",
      "\n",
      "Gene: PTK2\n",
      "Symbol: PTK2\n",
      "Name: protein tyrosine kinase 2\n",
      "Summary: This gene encodes a cytoplasmic protein tyrosine kinase which is found concentrated in the focal adhesions that form between cells growing in the presence of extracellular matrix constituents. The encoded protein is a member of the FAK subfamily of protein tyrosine kinases but lacks significant sequence similarity to kinases from other subfamilies. Activation of this gene may be an important early step in cell growth and intracellular signal transduction pathways triggered in response to certain neural peptides or to cell interactions with the extracellular matrix. Several transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Jun 2017].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - angiogenesis\n",
      "    - placenta development\n",
      "    - regulation of protein phosphorylation\n",
      "    - positive regulation of protein phosphorylation\n",
      "    - heart morphogenesis\n",
      "    - signal complex assembly\n",
      "    - epidermal growth factor receptor signaling pathway\n",
      "    - transforming growth factor beta receptor signaling pathway\n",
      "    - integrin-mediated signaling pathway\n",
      "    - integrin-mediated signaling pathway\n",
      "    - integrin-mediated signaling pathway\n",
      "    - axon guidance\n",
      "    - positive regulation of cell population proliferation\n",
      "    - regulation of cell shape\n",
      "    - regulation of endothelial cell migration\n",
      "    - regulation of epithelial cell migration\n",
      "    - positive regulation of epithelial cell migration\n",
      "    - positive regulation of epithelial to mesenchymal transition\n",
      "    - positive regulation of macrophage chemotaxis\n",
      "    - positive regulation of fibroblast migration\n",
      "    - cell migration\n",
      "    - peptidyl-tyrosine phosphorylation\n",
      "    - negative regulation of cell-cell adhesion\n",
      "    - establishment of cell polarity\n",
      "    - regulation of cell adhesion\n",
      "    - positive regulation of cell migration\n",
      "    - positive regulation of cell migration\n",
      "    - regulation of cell adhesion mediated by integrin\n",
      "    - detection of muscle stretch\n",
      "    - netrin-activated signaling pathway\n",
      "    - Fc-gamma receptor signaling pathway involved in phagocytosis\n",
      "    - regulation of cell population proliferation\n",
      "    - negative regulation of apoptotic process\n",
      "    - regulation of GTPase activity\n",
      "    - regulation of osteoblast differentiation\n",
      "    - positive regulation of protein kinase activity\n",
      "    - protein autophosphorylation\n",
      "    - vascular endothelial growth factor receptor signaling pathway\n",
      "    - ephrin receptor signaling pathway\n",
      "    - cell motility\n",
      "    - regulation of cytoskeleton organization\n",
      "    - regulation of focal adhesion assembly\n",
      "    - regulation of focal adhesion assembly\n",
      "    - positive regulation of phosphatidylinositol 3-kinase/protein kinase B signal transduction\n",
      "    - positive regulation of phosphatidylinositol 3-kinase/protein kinase B signal transduction\n",
      "    - growth hormone receptor signaling pathway\n",
      "    - positive regulation of wound healing\n",
      "    - vascular endothelial cell response to oscillatory fluid shear stress\n",
      "    - positive regulation of macrophage proliferation\n",
      "    - regulation of substrate adhesion-dependent cell spreading\n",
      "    - positive regulation of ubiquitin-dependent protein catabolic process\n",
      "    - negative regulation of anoikis\n",
      "  CC:\n",
      "    - stress fiber\n",
      "    - nucleus\n",
      "    - centrosome\n",
      "    - cytosol\n",
      "    - cytosol\n",
      "    - cytoskeleton\n",
      "    - plasma membrane\n",
      "    - focal adhesion\n",
      "    - focal adhesion\n",
      "    - cilium\n",
      "    - cell cortex\n",
      "    - ciliary basal body\n",
      "    - intracellular membrane-bounded organelle\n",
      "    - perinuclear region of cytoplasm\n",
      "  MF:\n",
      "    - actin binding\n",
      "    - protein tyrosine kinase activity\n",
      "    - protein tyrosine kinase activity\n",
      "    - protein tyrosine kinase activity\n",
      "    - non-membrane spanning protein tyrosine kinase activity\n",
      "    - non-membrane spanning protein tyrosine kinase activity\n",
      "    - protein tyrosine phosphatase activity\n",
      "    - integrin binding\n",
      "    - protein binding\n",
      "    - ATP binding\n",
      "    - JUN kinase binding\n",
      "    - protein kinase binding\n",
      "    - protein phosphatase binding\n",
      "    - SH2 domain binding\n",
      "    - molecular function activator activity\n",
      "UniProt: {'Swiss-Prot': 'Q05397', 'TrEMBL': ['Q59GN8', 'Q658W2', 'Q59GM6', 'B4DWJ1', 'E7ESA6', 'E5RG54', 'E5RFW9', 'E5RH01', 'E5RJI4', 'H0YB99', 'E5RHD8', 'E5RIR5', 'A0A8Q3WLM4', 'H0YBP1', 'E5RJP0', 'H0YB33', 'E5RI29', 'E5RII9', 'E5RHK7', 'E5RH08', 'H0YB16', 'A0A1D5RMT1', 'E5RH48', 'E5RG80', 'E5RI72', 'E9PEI4', 'E5RG66', 'H0YBZ1', 'H0YAS0', 'E5RIK4', 'E5RJQ2', 'E5RG86', 'E5RI03', 'E5RGA6', 'E5RGP1', 'E5RJN1']}\n",
      "\n",
      "Gene: VAMP2\n",
      "Symbol: VAMP2\n",
      "Name: vesicle associated membrane protein 2\n",
      "Summary: The protein encoded by this gene is a member of the vesicle-associated membrane protein (VAMP)/synaptobrevin family. Synaptobrevins/VAMPs, syntaxins, and the 25-kD synaptosomal-associated protein SNAP25 are the main components of a protein complex involved in the docking and/or fusion of synaptic vesicles with the presynaptic membrane. This gene is thought to participate in neurotransmitter release at a step between docking and fusion. The protein forms a stable complex with syntaxin, synaptosomal-associated protein, 25 kD, and synaptotagmin. It also forms a distinct complex with synaptophysin. It is a likely candidate gene for familial infantile myasthenia (FIMG) because of its map location and because it encodes a synaptic vesicle protein of the type that has been implicated in the pathogenesis of FIMG. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - exocytosis\n",
      "    - vesicle fusion\n",
      "    - vesicle fusion\n",
      "    - response to glucose\n",
      "    - protein transport\n",
      "    - synaptic vesicle exocytosis\n",
      "    - synaptic vesicle exocytosis\n",
      "    - vesicle-mediated transport\n",
      "    - calcium-ion regulated exocytosis\n",
      "    - regulation of exocytosis\n",
      "    - cellular response to insulin stimulus\n",
      "    - SNARE complex assembly\n",
      "    - Golgi to plasma membrane protein transport\n",
      "    - eosinophil degranulation\n",
      "    - neutrophil degranulation\n",
      "    - natural killer cell degranulation\n",
      "    - synaptic vesicle endocytosis\n",
      "    - long-term synaptic potentiation\n",
      "    - regulation of vesicle-mediated transport\n",
      "    - membrane fusion\n",
      "    - protein-containing complex assembly\n",
      "    - mucus secretion\n",
      "    - positive regulation of intracellular protein transport\n",
      "    - regulation of delayed rectifier potassium channel activity\n",
      "    - regulation of histamine secretion by mast cell\n",
      "  CC:\n",
      "    - trans-Golgi network\n",
      "    - cytosol\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - synaptic vesicle\n",
      "    - synaptic vesicle\n",
      "    - voltage-gated potassium channel complex\n",
      "    - membrane\n",
      "    - clathrin-coated vesicle\n",
      "    - secretory granule\n",
      "    - secretory granule membrane\n",
      "    - secretory granule membrane\n",
      "    - clathrin-coated endocytic vesicle membrane\n",
      "    - synaptic vesicle membrane\n",
      "    - synaptic vesicle membrane\n",
      "    - SNARE complex\n",
      "    - SNARE complex\n",
      "    - SNARE complex\n",
      "    - cytoplasmic vesicle\n",
      "    - vesicle\n",
      "    - zymogen granule membrane\n",
      "    - neuron projection\n",
      "    - intracellular membrane-bounded organelle\n",
      "    - neuron projection terminus\n",
      "    - synapse\n",
      "    - perinuclear region of cytoplasm\n",
      "    - clathrin-sculpted glutamate transport vesicle membrane\n",
      "    - clathrin-sculpted gamma-aminobutyric acid transport vesicle membrane\n",
      "    - synaptobrevin 2-SNAP-25-syntaxin-1a-complexin I complex\n",
      "    - synaptobrevin 2-SNAP-25-syntaxin-1a-complexin II complex\n",
      "    - synaptobrevin 2-SNAP-25-syntaxin-1a complex\n",
      "    - clathrin-sculpted monoamine transport vesicle membrane\n",
      "  MF:\n",
      "    - SNARE binding\n",
      "    - SNAP receptor activity\n",
      "    - protein binding\n",
      "    - calmodulin binding\n",
      "    - phospholipid binding\n",
      "    - syntaxin-1 binding\n",
      "    - syntaxin-1 binding\n",
      "    - syntaxin binding\n",
      "    - syntaxin binding\n",
      "    - calcium-dependent protein binding\n",
      "UniProt: {'Swiss-Prot': 'P63027', 'TrEMBL': ['K7ENK9', 'F8WCA0', 'J3QRU4', 'A0AA34QW14']}\n",
      "\n",
      "Gene: IL17A\n",
      "Symbol: IL17A\n",
      "Name: interleukin 17A\n",
      "Summary: This gene is a member of the IL-17 receptor family which includes five members (IL-17RA-E) and the encoded protein is a proinflammatory cytokine produced by activated T cells. IL-17A-mediated downstream pathways induce the production of inflammatory molecules, chemokines, antimicrobial peptides, and remodeling proteins. The encoded protein elicits crucial impacts on host defense, cell trafficking, immune modulation, and tissue repair, with a key role in the induction of innate immune defenses. This cytokine stimulates non-hematopoietic cells and promotes chemokine production thereby attracting myeloid cells to inflammatory sites. This cytokine also regulates the activities of NF-kappaB and mitogen-activated protein kinases and can stimulate the expression of IL6 and cyclooxygenase-2 (PTGS2/COX-2), as well as enhance the production of nitric oxide (NO). IL-17A plays a pivotal role in various infectious diseases, inflammatory and autoimmune disorders, and cancer. High levels of this cytokine are associated with several chronic inflammatory diseases including rheumatoid arthritis, psoriasis and multiple sclerosis. The lung damage induced by the severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) is to a large extent, a result of the inflammatory response promoted by cytokines such as IL17A. [provided by RefSeq, Sep 2020].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - positive regulation of antimicrobial peptide production\n",
      "    - adaptive immune response\n",
      "    - apoptotic process\n",
      "    - inflammatory response\n",
      "    - immune response\n",
      "    - Notch signaling pathway\n",
      "    - cell-cell signaling\n",
      "    - cell death\n",
      "    - response to wounding\n",
      "    - gene expression\n",
      "    - keratinocyte differentiation\n",
      "    - positive regulation of interleukin-1 beta production\n",
      "    - positive regulation of interleukin-12 production\n",
      "    - positive regulation of interleukin-16 production\n",
      "    - positive regulation of interleukin-23 production\n",
      "    - positive regulation of interleukin-6 production\n",
      "    - positive regulation of tumor necrosis factor production\n",
      "    - interleukin-17A-mediated signaling pathway\n",
      "    - keratinocyte proliferation\n",
      "    - innate immune response\n",
      "    - positive regulation of osteoclast differentiation\n",
      "    - positive regulation of transcription by RNA polymerase II\n",
      "    - defense response to Gram-negative bacterium\n",
      "    - defense response to Gram-positive bacterium\n",
      "    - defense response to fungus\n",
      "    - intestinal epithelial structure maintenance\n",
      "    - cellular response to interleukin-1\n",
      "    - fibroblast activation\n",
      "    - interleukin-17-mediated signaling pathway\n",
      "    - granulocyte migration\n",
      "    - negative regulation of inflammatory response to wounding\n",
      "    - positive regulation of cytokine production involved in inflammatory response\n",
      "    - positive regulation of bicellular tight junction assembly\n",
      "    - positive regulation of chemokine (C-X-C motif) ligand 1 production\n",
      "  CC:\n",
      "    - extracellular region\n",
      "    - extracellular space\n",
      "    - external side of plasma membrane\n",
      "  MF:\n",
      "    - cytokine activity\n",
      "    - protein binding\n",
      "    - protein homodimerization activity\n",
      "    - protein heterodimerization activity\n",
      "UniProt: {'Swiss-Prot': 'Q16552'}\n",
      "\n",
      "Gene: IL17F\n",
      "Symbol: IL17F\n",
      "Name: interleukin 17F\n",
      "Summary: The protein encoded by this gene is a cytokine that shares sequence similarity with IL17. This cytokine is expressed by activated T cells, and has been shown to stimulate the production of several other cytokines, including IL6, IL8, and CSF2/GM_CSF. This cytokine is also found to inhibit the angiogenesis of endothelial cells and induce endothelial cells to produce IL2, TGFB1/TGFB, and monocyte chemoattractant protein-1. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - positive regulation of cytokine production\n",
      "    - positive regulation of antimicrobial peptide production\n",
      "    - adaptive immune response\n",
      "    - inflammatory response\n",
      "    - negative regulation of angiogenesis\n",
      "    - regulation of transforming growth factor beta receptor signaling pathway\n",
      "    - regulation of granulocyte macrophage colony-stimulating factor production\n",
      "    - regulation of interleukin-2 production\n",
      "    - regulation of interleukin-6 production\n",
      "    - regulation of interleukin-8 production\n",
      "    - positive regulation of interleukin-6 production\n",
      "    - positive regulation of lymphotoxin A production\n",
      "    - innate immune response\n",
      "    - positive regulation of transcription by RNA polymerase II\n",
      "    - defense response to Gram-negative bacterium\n",
      "    - defense response to Gram-positive bacterium\n",
      "    - cartilage development\n",
      "    - interleukin-17-mediated signaling pathway\n",
      "    - positive regulation of cytokine production involved in inflammatory response\n",
      "    - positive regulation of chemokine (C-X-C motif) ligand 1 production\n",
      "  CC:\n",
      "    - extracellular region\n",
      "    - extracellular space\n",
      "  MF:\n",
      "    - cytokine activity\n",
      "    - cytokine receptor binding\n",
      "    - protein binding\n",
      "    - cytokine binding\n",
      "    - protein homodimerization activity\n",
      "    - protein heterodimerization activity\n",
      "UniProt: {'Swiss-Prot': 'Q96PD4'}\n",
      "\n",
      "Gene: DLG3\n",
      "Symbol: DLG3\n",
      "Name: discs large MAGUK scaffold protein 3\n",
      "Summary: This gene encodes a member of the membrane-associated guanylate kinase protein family. The encoded protein may play a role in clustering of NMDA receptors at excitatory synapses. It may also negatively regulate cell proliferation through interaction with the C-terminal region of the adenomatosis polyposis coli tumor suppressor protein. Mutations in this gene have been associated with X-linked cognitive disability. Alternatively spliced transcript variants have been described. [provided by RefSeq, Oct 2009].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - establishment of planar polarity\n",
      "    - chemical synaptic transmission\n",
      "    - nervous system development\n",
      "    - negative regulation of cell population proliferation\n",
      "    - protein localization to synapse\n",
      "    - receptor clustering\n",
      "    - establishment or maintenance of epithelial cell apical/basal polarity\n",
      "    - establishment or maintenance of epithelial cell apical/basal polarity\n",
      "    - receptor localization to synapse\n",
      "    - cell-cell adhesion\n",
      "    - regulation of postsynaptic membrane neurotransmitter receptor levels\n",
      "  CC:\n",
      "    - extracellular space\n",
      "    - cytosol\n",
      "    - plasma membrane\n",
      "    - adherens junction\n",
      "    - bicellular tight junction\n",
      "    - basolateral plasma membrane\n",
      "    - neuromuscular junction\n",
      "    - AMPA glutamate receptor complex\n",
      "    - neuron projection\n",
      "    - postsynaptic density membrane\n",
      "    - glutamatergic synapse\n",
      "  MF:\n",
      "    - protein binding\n",
      "    - kinase binding\n",
      "    - protein kinase binding\n",
      "    - phosphatase binding\n",
      "    - ubiquitin protein ligase binding\n",
      "    - ionotropic glutamate receptor binding\n",
      "UniProt: {'Swiss-Prot': 'Q92796', 'TrEMBL': ['Q5JUW8', 'Q59FY1']}\n",
      "\n",
      "Gene: HSD17B10\n",
      "Symbol: HSD17B10\n",
      "Name: hydroxysteroid 17-beta dehydrogenase 10\n",
      "Summary: This gene encodes 3-hydroxyacyl-CoA dehydrogenase type II, a member of the short-chain dehydrogenase/reductase superfamily. The gene product is a mitochondrial protein that catalyzes the oxidation of a wide variety of fatty acids and steroids, and is a subunit of mitochondrial ribonuclease P, which is involved in tRNA maturation. The protein has been implicated in the development of Alzheimer disease, and mutations in the gene are the cause of 17beta-hydroxysteroid dehydrogenase type 10 (HSD10) deficiency. Several alternatively spliced transcript variants have been identified, but the full-length nature of only two transcript variants has been determined. [provided by RefSeq, Aug 2014].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - isoleucine catabolic process\n",
      "    - isoleucine catabolic process\n",
      "    - lipid metabolic process\n",
      "    - fatty acid metabolic process\n",
      "    - fatty acid beta-oxidation\n",
      "    - fatty acid beta-oxidation\n",
      "    - bile acid biosynthetic process\n",
      "    - bile acid biosynthetic process\n",
      "    - mitochondrion organization\n",
      "    - C21-steroid hormone metabolic process\n",
      "    - androgen metabolic process\n",
      "    - androgen metabolic process\n",
      "    - estrogen metabolic process\n",
      "    - estrogen metabolic process\n",
      "    - protein homotetramerization\n",
      "    - brexanolone metabolic process\n",
      "    - mitochondrial tRNA methylation\n",
      "    - mitochondrial tRNA 5'-end processing\n",
      "    - mitochondrial tRNA 3'-end processing\n",
      "  CC:\n",
      "    - cytoplasm\n",
      "    - mitochondrion\n",
      "    - mitochondrion\n",
      "    - mitochondrion\n",
      "    - mitochondrion\n",
      "    - mitochondrial matrix\n",
      "    - plasma membrane\n",
      "    - mitochondrial ribonuclease P complex\n",
      "    - mitochondrial ribonuclease P complex\n",
      "    - mitochondrial ribonuclease P complex\n",
      "    - mitochondrial nucleoid\n",
      "    - tRNA methyltransferase complex\n",
      "  MF:\n",
      "    - tRNA binding\n",
      "    - RNA binding\n",
      "    - 3-hydroxyacyl-CoA dehydrogenase activity\n",
      "    - estradiol 17-beta-dehydrogenase [NAD(P)+] activity\n",
      "    - protein binding\n",
      "    - cholate 7-alpha-dehydrogenase (NAD+) activity\n",
      "    - testosterone dehydrogenase [NAD(P)+] activity\n",
      "    - 17-beta-hydroxysteroid dehydrogenase (NAD+) activity\n",
      "    - 3-hydroxy-2-methylbutyryl-CoA dehydrogenase activity\n",
      "    - testosterone dehydrogenase (NAD+) activity\n",
      "    - androstan-3-alpha,17-beta-diol dehydrogenase (NAD+) activity\n",
      "    - chenodeoxycholate 7-alpha-dehydrogenase (NAD+) activity\n",
      "    - isoursodeoxycholate 7-beta-dehydrogenase (NAD+) activity\n",
      "    - ursodeoxycholate 7-beta-dehydrogenase (NAD+) activity\n",
      "UniProt: {'Swiss-Prot': 'Q99714', 'TrEMBL': ['Q5H928', 'A0A0S2Z410', 'A0A804HHW5']}\n",
      "\n",
      "Gene: DCTN3\n",
      "Symbol: DCTN3\n",
      "Name: dynactin subunit 3\n",
      "Summary: This gene encodes the smallest subunit of dynactin, a macromolecular complex consisting of 10 subunits ranging in size from 22 to 150 kD. Dynactin binds to both microtubules and cytoplasmic dynein. It is involved in a diverse array of cellular functions, including ER-to-Golgi transport, the centripetal movement of lysosomes and endosomes, spindle formation, cytokinesis, chromosome movement, nuclear positioning, and axonogenesis. This subunit, like most other dynactin subunits, exists only as a part of the dynactin complex. It is primarily an alpha-helical protein with very little coiled coil, and binds directly to the largest subunit (p150) of dynactin. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Jul 2013].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - mitotic cell cycle\n",
      "    - microtubule-based process\n",
      "    - cytoskeleton-dependent cytokinesis\n",
      "    - cytoskeleton-dependent cytokinesis\n",
      "  CC:\n",
      "    - kinetochore\n",
      "    - nucleolus\n",
      "    - centrosome\n",
      "    - spindle\n",
      "    - cytosol\n",
      "    - cytosol\n",
      "    - dynactin complex\n",
      "    - dynactin complex\n",
      "    - midbody\n",
      "    - cleavage furrow\n",
      "    - perinuclear region of cytoplasm\n",
      "  MF:\n",
      "    - structural molecule activity\n",
      "    - protein binding\n",
      "UniProt: {'Swiss-Prot': 'O75935', 'TrEMBL': ['X6RCK5', 'X6RLR1', 'X6RA56', 'A0A0A0MRV8']}\n",
      "\n",
      "Gene: CASP1\n",
      "Symbol: CASP1\n",
      "Name: caspase 1\n",
      "Summary: This gene encodes a protein which is a member of the cysteine-aspartic acid protease (caspase) family. Sequential activation of caspases plays a central role in the execution-phase of cell apoptosis. Caspases exist as inactive proenzymes which undergo proteolytic processing at conserved aspartic residues to produce 2 subunits, large and small, that dimerize to form the active enzyme. This gene was identified by its ability to proteolytically cleave and activate the inactive precursor of interleukin-1, a cytokine involved in the processes such as inflammation, septic shock, and wound healing. This gene has been shown to induce cell apoptosis and may function in various developmental stages. Studies of a similar gene in mouse suggest a role in the pathogenesis of Huntington disease. Alternative splicing results in transcript variants encoding distinct isoforms. [provided by RefSeq, Mar 2012].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - pattern recognition receptor signaling pathway\n",
      "    - proteolysis\n",
      "    - apoptotic process\n",
      "    - signal transduction\n",
      "    - osmosensory signaling pathway\n",
      "    - protein autoprocessing\n",
      "    - positive regulation of interleukin-1 beta production\n",
      "    - positive regulation of interleukin-1 beta production\n",
      "    - positive regulation of interleukin-1 beta production\n",
      "    - positive regulation of interleukin-1 beta production\n",
      "    - positive regulation of interleukin-18 production\n",
      "    - defense response to bacterium\n",
      "    - regulation of apoptotic process\n",
      "    - positive regulation of canonical NF-kappaB signal transduction\n",
      "    - positive regulation of canonical NF-kappaB signal transduction\n",
      "    - icosanoid biosynthetic process\n",
      "    - regulation of inflammatory response\n",
      "    - regulation of inflammatory response\n",
      "    - positive regulation of inflammatory response\n",
      "    - positive regulation of inflammatory response\n",
      "    - positive regulation of inflammatory response\n",
      "    - protein maturation\n",
      "    - protein maturation\n",
      "    - defense response to virus\n",
      "    - pyroptotic inflammatory response\n",
      "    - pyroptotic inflammatory response\n",
      "    - cellular response to lipopolysaccharide\n",
      "    - cellular response to lipopolysaccharide\n",
      "    - cellular response to mechanical stimulus\n",
      "    - cellular response to type II interferon\n",
      "    - cytokine precursor processing\n",
      "    - signaling receptor ligand precursor processing\n",
      "    - AIM2 inflammasome complex assembly\n",
      "    - positive regulation of tumor necrosis factor-mediated signaling pathway\n",
      "  CC:\n",
      "    - nucleolus\n",
      "    - cytoplasm\n",
      "    - cytoplasm\n",
      "    - cytoplasm\n",
      "    - cytosol\n",
      "    - cytosol\n",
      "    - cytosol\n",
      "    - microtubule\n",
      "    - plasma membrane\n",
      "    - protein-containing complex\n",
      "    - protein-containing complex\n",
      "    - canonical inflammasome complex\n",
      "    - canonical inflammasome complex\n",
      "    - IPAF inflammasome complex\n",
      "    - IPAF inflammasome complex\n",
      "    - NLRP1 inflammasome complex\n",
      "    - NLRP1 inflammasome complex\n",
      "    - NLRP3 inflammasome complex\n",
      "    - NLRP3 inflammasome complex\n",
      "    - NLRP3 inflammasome complex\n",
      "    - AIM2 inflammasome complex\n",
      "    - AIM2 inflammasome complex\n",
      "    - AIM2 inflammasome complex\n",
      "    - protease inhibitor complex\n",
      "  MF:\n",
      "    - endopeptidase activity\n",
      "    - cysteine-type endopeptidase activity\n",
      "    - cysteine-type endopeptidase activity\n",
      "    - protein binding\n",
      "    - cysteine-type endopeptidase activator activity involved in apoptotic process\n",
      "    - kinase binding\n",
      "    - cytokine binding\n",
      "    - identical protein binding\n",
      "    - identical protein binding\n",
      "    - CARD domain binding\n",
      "UniProt: {'Swiss-Prot': 'P29466', 'TrEMBL': ['A8K249', 'A8K257', 'B4DVD8', 'A0A8Q3SIY2', 'G3V169', 'B4DKN4', 'Q5FBZ0', 'A0A8Q3SI67', 'A0A8Q3SI00', 'H0YEC7']}\n",
      "\n",
      "Gene: GRIN1\n",
      "Symbol: GRIN1\n",
      "Name: glutamate ionotropic receptor NMDA type subunit 1\n",
      "Summary: The protein encoded by this gene is a critical subunit of N-methyl-D-aspartate receptors, members of the glutamate receptor channel superfamily which are heteromeric protein complexes with multiple subunits arranged to form a ligand-gated ion channel. These subunits play a key role in the plasticity of synapses, which is believed to underlie memory and learning. Cell-specific factors are thought to control expression of different isoforms, possibly contributing to the functional diversity of the subunits. Alternatively spliced transcript variants have been described. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - monoatomic cation transport\n",
      "    - chemical synaptic transmission\n",
      "    - brain development\n",
      "    - visual learning\n",
      "    - positive regulation of calcium ion transport into cytosol\n",
      "    - propylene metabolic process\n",
      "    - calcium-mediated signaling\n",
      "    - ionotropic glutamate receptor signaling pathway\n",
      "    - ionotropic glutamate receptor signaling pathway\n",
      "    - ionotropic glutamate receptor signaling pathway\n",
      "    - ionotropic glutamate receptor signaling pathway\n",
      "    - ionotropic glutamate receptor signaling pathway\n",
      "    - sodium ion transmembrane transport\n",
      "    - regulation of membrane potential\n",
      "    - regulation of membrane potential\n",
      "    - response to ethanol\n",
      "    - positive regulation of transcription by RNA polymerase II\n",
      "    - regulation of synaptic plasticity\n",
      "    - regulation of synaptic plasticity\n",
      "    - regulation of neuronal synaptic plasticity\n",
      "    - protein heterotetramerization\n",
      "    - positive regulation of synaptic transmission, glutamatergic\n",
      "    - positive regulation of synaptic transmission, glutamatergic\n",
      "    - positive regulation of synaptic transmission, glutamatergic\n",
      "    - calcium ion homeostasis\n",
      "    - excitatory postsynaptic potential\n",
      "    - calcium ion transmembrane transport\n",
      "    - calcium ion transmembrane import into cytosol\n",
      "    - monoatomic cation transmembrane transport\n",
      "    - monoatomic cation transmembrane transport\n",
      "    - monoatomic cation transmembrane transport\n",
      "    - monoatomic cation transmembrane transport\n",
      "    - excitatory chemical synaptic transmission\n",
      "    - positive regulation of reactive oxygen species biosynthetic process\n",
      "    - regulation of monoatomic cation transmembrane transport\n",
      "    - regulation of monoatomic cation transmembrane transport\n",
      "    - regulation of monoatomic cation transmembrane transport\n",
      "    - cellular response to amyloid-beta\n",
      "    - response to glycine\n",
      "    - positive regulation of excitatory postsynaptic potential\n",
      "    - positive regulation of excitatory postsynaptic potential\n",
      "    - positive regulation of excitatory postsynaptic potential\n",
      "  CC:\n",
      "    - cytoplasm\n",
      "    - endoplasmic reticulum membrane\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - synaptic vesicle\n",
      "    - cell surface\n",
      "    - postsynaptic density\n",
      "    - NMDA selective glutamate receptor complex\n",
      "    - NMDA selective glutamate receptor complex\n",
      "    - NMDA selective glutamate receptor complex\n",
      "    - NMDA selective glutamate receptor complex\n",
      "    - NMDA selective glutamate receptor complex\n",
      "    - NMDA selective glutamate receptor complex\n",
      "    - dendrite\n",
      "    - monoatomic ion channel complex\n",
      "    - monoatomic ion channel complex\n",
      "    - neuron projection\n",
      "    - neuron projection\n",
      "    - synaptic cleft\n",
      "    - terminal bouton\n",
      "    - dendritic spine\n",
      "    - synapse\n",
      "    - synapse\n",
      "    - postsynaptic membrane\n",
      "    - postsynaptic membrane\n",
      "    - excitatory synapse\n",
      "    - synaptic membrane\n",
      "    - postsynaptic density membrane\n",
      "    - neurotransmitter receptor complex\n",
      "    - neurotransmitter receptor complex\n",
      "  MF:\n",
      "    - amyloid-beta binding\n",
      "    - NMDA glutamate receptor activity\n",
      "    - NMDA glutamate receptor activity\n",
      "    - NMDA glutamate receptor activity\n",
      "    - NMDA glutamate receptor activity\n",
      "    - NMDA glutamate receptor activity\n",
      "    - NMDA glutamate receptor activity\n",
      "    - calcium channel activity\n",
      "    - calcium ion binding\n",
      "    - protein binding\n",
      "    - calmodulin binding\n",
      "    - ligand-gated monoatomic ion channel activity\n",
      "    - ligand-gated sodium channel activity\n",
      "    - glycine binding\n",
      "    - glycine binding\n",
      "    - glutamate binding\n",
      "    - glutamate-gated calcium ion channel activity\n",
      "    - signaling receptor activity\n",
      "    - protein-containing complex binding\n",
      "    - glycine-gated cation channel activity\n",
      "UniProt: {'Swiss-Prot': 'Q05586', 'TrEMBL': ['Q59GW0', 'Q5VSF9', 'A2AVK2']}\n",
      "\n",
      "Gene: GRIN2B\n",
      "Symbol: GRIN2B\n",
      "Name: glutamate ionotropic receptor NMDA type subunit 2B\n",
      "Summary: This gene encodes a member of the N-methyl-D-aspartate (NMDA) receptor family within the ionotropic glutamate receptor superfamily. The encoded protein is a subunit of the NMDA receptor ion channel which acts as an agonist binding site for glutamate. The NMDA receptors mediate a slow calcium-permeable component of excitatory synaptic transmission in the central nervous system. The NMDA receptors are heterotetramers of seven genetically encoded, differentially expressed subunits including NR1 (GRIN1), NR2 (GRIN2A, GRIN2B, GRIN2C, or GRIN2D) and NR3 (GRIN3A or GRIN3B). The early expression of this gene in development suggests a role in brain development, circuit formation, synaptic plasticity, and cellular migration and differentiation. Naturally occurring mutations within this gene are associated with neurodevelopmental disorders including autism spectrum disorder, attention deficit hyperactivity disorder, epilepsy, and schizophrenia. [provided by RefSeq, Aug 2017].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - glutamate receptor signaling pathway\n",
      "    - chemical synaptic transmission\n",
      "    - brain development\n",
      "    - learning or memory\n",
      "    - calcium-mediated signaling\n",
      "    - ionotropic glutamate receptor signaling pathway\n",
      "    - ionotropic glutamate receptor signaling pathway\n",
      "    - synaptic transmission, glutamatergic\n",
      "    - response to ethanol\n",
      "    - regulation of synaptic plasticity\n",
      "    - regulation of synaptic plasticity\n",
      "    - regulation of neuronal synaptic plasticity\n",
      "    - protein heterotetramerization\n",
      "    - positive regulation of synaptic transmission, glutamatergic\n",
      "    - positive regulation of synaptic transmission, glutamatergic\n",
      "    - excitatory postsynaptic potential\n",
      "    - long-term synaptic potentiation\n",
      "    - calcium ion transmembrane import into cytosol\n",
      "    - calcium ion transmembrane import into cytosol\n",
      "    - monoatomic cation transmembrane transport\n",
      "    - monoatomic cation transmembrane transport\n",
      "    - monoatomic cation transmembrane transport\n",
      "    - excitatory chemical synaptic transmission\n",
      "    - regulation of presynaptic membrane potential\n",
      "    - negative regulation of dendritic spine maintenance\n",
      "    - regulation of monoatomic cation transmembrane transport\n",
      "    - regulation of monoatomic cation transmembrane transport\n",
      "    - positive regulation of excitatory postsynaptic potential\n",
      "    - positive regulation of excitatory postsynaptic potential\n",
      "  CC:\n",
      "    - cytoplasm\n",
      "    - lysosome\n",
      "    - late endosome\n",
      "    - endoplasmic reticulum membrane\n",
      "    - cytoskeleton\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - cell surface\n",
      "    - postsynaptic density\n",
      "    - NMDA selective glutamate receptor complex\n",
      "    - NMDA selective glutamate receptor complex\n",
      "    - NMDA selective glutamate receptor complex\n",
      "    - NMDA selective glutamate receptor complex\n",
      "    - NMDA selective glutamate receptor complex\n",
      "    - dendrite\n",
      "    - neuron projection\n",
      "    - postsynaptic membrane\n",
      "    - postsynaptic membrane\n",
      "    - postsynaptic membrane\n",
      "    - synaptic membrane\n",
      "    - postsynaptic density membrane\n",
      "  MF:\n",
      "    - amyloid-beta binding\n",
      "    - NMDA glutamate receptor activity\n",
      "    - NMDA glutamate receptor activity\n",
      "    - NMDA glutamate receptor activity\n",
      "    - NMDA glutamate receptor activity\n",
      "    - protein binding\n",
      "    - zinc ion binding\n",
      "    - glycine binding\n",
      "    - glutamate binding\n",
      "    - glutamate-gated calcium ion channel activity\n",
      "    - glutamate-gated calcium ion channel activity\n",
      "    - ligand-gated monoatomic ion channel activity involved in regulation of presynaptic membrane potential\n",
      "    - ligand-gated monoatomic ion channel activity involved in regulation of presynaptic membrane potential\n",
      "    - ligand-gated monoatomic ion channel activity involved in regulation of presynaptic membrane potential\n",
      "    - transmitter-gated monoatomic ion channel activity involved in regulation of postsynaptic membrane potential\n",
      "UniProt: {'Swiss-Prot': 'Q13224', 'TrEMBL': ['A0A0D9SFA0', 'A0A8D9PHB2', 'A0AAQ5BH89', 'A0A0D9SFK0', 'A0A1B0GU78']}\n",
      "\n",
      "Gene: CYP1A2\n",
      "Symbol: CYP1A2\n",
      "Name: cytochrome P450 family 1 subfamily A member 2\n",
      "Summary: This gene encodes a member of the cytochrome P450 superfamily of enzymes. The cytochrome P450 proteins are monooxygenases which catalyze many reactions involved in drug metabolism and synthesis of cholesterol, steroids and other lipids. The protein encoded by this gene localizes to the endoplasmic reticulum and its expression is induced by some polycyclic aromatic hydrocarbons (PAHs), some of which are found in cigarette smoke. The enzyme's endogenous substrate is unknown; however, it is able to metabolize some PAHs to carcinogenic intermediates. Other xenobiotic substrates for this enzyme include caffeine, aflatoxin B1, and acetaminophen. The transcript from this gene contains four Alu sequences flanked by direct repeats in the 3' untranslated region. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - steroid catabolic process\n",
      "    - porphyrin-containing compound metabolic process\n",
      "    - xenobiotic metabolic process\n",
      "    - xenobiotic metabolic process\n",
      "    - cholesterol metabolic process\n",
      "    - estrogen metabolic process\n",
      "    - toxin biosynthetic process\n",
      "    - post-embryonic development\n",
      "    - alkaloid metabolic process\n",
      "    - regulation of gene expression\n",
      "    - monoterpenoid metabolic process\n",
      "    - dibenzo-p-dioxin metabolic process\n",
      "    - epoxygenase P450 pathway\n",
      "    - lung development\n",
      "    - methylation\n",
      "    - monocarboxylic acid metabolic process\n",
      "    - xenobiotic catabolic process\n",
      "    - xenobiotic catabolic process\n",
      "    - retinol metabolic process\n",
      "    - retinol metabolic process\n",
      "    - long-chain fatty acid biosynthetic process\n",
      "    - cellular respiration\n",
      "    - aflatoxin metabolic process\n",
      "    - hydrogen peroxide biosynthetic process\n",
      "    - oxidative demethylation\n",
      "    - cellular response to cadmium ion\n",
      "    - omega-hydroxylase P450 pathway\n",
      "  CC:\n",
      "    - endoplasmic reticulum membrane\n",
      "    - intracellular membrane-bounded organelle\n",
      "    - intracellular membrane-bounded organelle\n",
      "  MF:\n",
      "    - monooxygenase activity\n",
      "    - monooxygenase activity\n",
      "    - monooxygenase activity\n",
      "    - iron ion binding\n",
      "    - protein binding\n",
      "    - electron transfer activity\n",
      "    - oxidoreductase activity\n",
      "    - oxidoreductase activity, acting on paired donors, with incorporation or reduction of molecular oxygen, reduced flavin or flavoprotein as one donor, and incorporation of one atom of oxygen\n",
      "    - enzyme binding\n",
      "    - heme binding\n",
      "    - demethylase activity\n",
      "    - caffeine oxidase activity\n",
      "    - estrogen 16-alpha-hydroxylase activity\n",
      "    - estrogen 2-hydroxylase activity\n",
      "    - hydroperoxy icosatetraenoate dehydratase activity\n",
      "UniProt: {'Swiss-Prot': 'P05177'}\n",
      "\n",
      "Gene: POR\n",
      "Symbol: POR\n",
      "Name: cytochrome p450 oxidoreductase\n",
      "Summary: This gene encodes an endoplasmic reticulum membrane oxidoreductase that is essential for multiple metabolic processes, including reactions catalyzed by cytochrome P450 proteins for metabolism of steroid hormones, drugs and xenobiotics. The encoded protein has a flavin adenine dinucleotide (FAD)-binding domain and a flavodoxin-like domain which bind two cofactors, FAD and FMN, that allow it to donate electrons directly from NADPH to all microsomal P450 enzymes. Mutations in this gene cause a complex set of disorders, including apparent combined P450C17 and P450C21 deficiency, amenorrhea and disordered steroidogenesis, congenital adrenal hyperplasia and Antley-Bixler syndrome, that resemble those caused by defects in steroid metabolizing enzymes such as aromatase, 21-hydroxylase, and 17 alpha-hydroxylase. [provided by RefSeq, Aug 2020].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - xenobiotic metabolic process\n",
      "    - nitric oxide biosynthetic process\n",
      "    - response to hormone\n",
      "    - electron transport chain\n",
      "    - positive regulation of monooxygenase activity\n",
      "    - organofluorine metabolic process\n",
      "  CC:\n",
      "    - endoplasmic reticulum membrane\n",
      "    - cytosol\n",
      "    - membrane\n",
      "    - intracellular membrane-bounded organelle\n",
      "  MF:\n",
      "    - NADPH-hemoprotein reductase activity\n",
      "    - NADPH-hemoprotein reductase activity\n",
      "    - NADPH-hemoprotein reductase activity\n",
      "    - protein binding\n",
      "    - FMN binding\n",
      "    - flavin adenine dinucleotide binding\n",
      "    - NADP binding\n",
      "UniProt: {'Swiss-Prot': 'P16435', 'TrEMBL': ['A0A8V8NVY2', 'A0A9L9PY49', 'C9JU80', 'A0A9L9PXL8', 'A0A8V8NVX2', 'A0A8V8PBF2', 'A0A8V8NE24', 'A0A8V8NRY5', 'A0A8V8P672', 'C9JQ78', 'E7EMD0', 'A0A8V8NKH1', 'C9JDE2', 'A0A8V8P1G2', 'A0A8V8NLF0', 'H0Y4R2', 'A0A9L9PXN4', 'E7EWU0']}\n",
      "\n",
      "Gene: CYP2C19\n",
      "Symbol: CYP2C19\n",
      "Name: cytochrome P450 family 2 subfamily C member 19\n",
      "Summary: This gene encodes a member of the cytochrome P450 superfamily of enzymes. The cytochrome P450 proteins are monooxygenases which catalyze many reactions involved in drug metabolism and synthesis of cholesterol, steroids and other lipids. This protein localizes to the endoplasmic reticulum and is known to metabolize many xenobiotics, including the anticonvulsive drug mephenytoin, omeprazole, diazepam and some barbiturates. Polymorphism within this gene is associated with variable ability to metabolize mephenytoin, known as the poor metabolizer and extensive metabolizer phenotypes. The gene is located within a cluster of cytochrome P450 genes on chromosome 10q24. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - long-chain fatty acid metabolic process\n",
      "    - organic acid metabolic process\n",
      "    - xenobiotic metabolic process\n",
      "    - xenobiotic metabolic process\n",
      "    - xenobiotic metabolic process\n",
      "    - steroid metabolic process\n",
      "    - monoterpenoid metabolic process\n",
      "    - epoxygenase P450 pathway\n",
      "    - xenobiotic catabolic process\n",
      "    - omega-hydroxylase P450 pathway\n",
      "  CC:\n",
      "    - cytoplasm\n",
      "    - endoplasmic reticulum membrane\n",
      "    - plasma membrane\n",
      "    - intracellular membrane-bounded organelle\n",
      "    - intracellular membrane-bounded organelle\n",
      "  MF:\n",
      "    - monooxygenase activity\n",
      "    - monooxygenase activity\n",
      "    - iron ion binding\n",
      "    - steroid hydroxylase activity\n",
      "    - oxidoreductase activity\n",
      "    - oxidoreductase activity, acting on paired donors, with incorporation or reduction of molecular oxygen, reduced flavin or flavoprotein as one donor, and incorporation of one atom of oxygen\n",
      "    - (S)-limonene 6-monooxygenase activity\n",
      "    - (S)-limonene 7-monooxygenase activity\n",
      "    - oxygen binding\n",
      "    - enzyme binding\n",
      "    - heme binding\n",
      "    - heme binding\n",
      "    - (R)-limonene 6-monooxygenase activity\n",
      "    - long-chain fatty acid omega-1 hydroxylase activity\n",
      "UniProt: {'Swiss-Prot': 'P33261', 'TrEMBL': 'A0A087X125'}\n",
      "\n",
      "Gene: ATP4A\n",
      "Symbol: ATP4A\n",
      "Name: ATPase H+/K+ transporting subunit alpha\n",
      "Summary: The protein encoded by this gene belongs to a family of P-type cation-transporting ATPases. The gastric H+, K+-ATPase is a heterodimer consisting of a high molecular weight catalytic alpha subunit and a smaller but heavily glycosylated beta subunit. This enzyme is a proton pump that catalyzes the hydrolysis of ATP coupled with the exchange of H(+) and K(+) ions across the plasma membrane. It is also responsible for gastric acid secretion. This gene encodes a catalytic alpha subunit of the gastric H+, K+-ATPase. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - intracellular sodium ion homeostasis\n",
      "    - response to xenobiotic stimulus\n",
      "    - regulation of proton transport\n",
      "    - establishment or maintenance of transmembrane electrochemical gradient\n",
      "    - intracellular potassium ion homeostasis\n",
      "    - monoatomic ion transmembrane transport\n",
      "    - sodium ion export across plasma membrane\n",
      "    - pH reduction\n",
      "    - potassium ion transmembrane transport\n",
      "    - proton transmembrane transport\n",
      "    - potassium ion import across plasma membrane\n",
      "  CC:\n",
      "    - extracellular space\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - potassium:proton exchanging ATPase complex\n",
      "    - apical plasma membrane\n",
      "  MF:\n",
      "    - magnesium ion binding\n",
      "    - P-type sodium:potassium-exchanging transporter activity\n",
      "    - ATP binding\n",
      "    - P-type potassium:proton transporter activity\n",
      "    - P-type potassium:proton transporter activity\n",
      "    - ATP hydrolysis activity\n",
      "    - potassium ion binding\n",
      "UniProt: {'Swiss-Prot': 'P20648', 'TrEMBL': ['A0A384MR29', 'Q658V6', 'M0R249']}\n",
      "\n",
      "Gene: ATP4B\n",
      "Symbol: ATP4B\n",
      "Name: ATPase H+/K+ transporting subunit beta\n",
      "Summary: The protein encoded by this gene belongs to a family of P-type cation-transporting ATPases. The gastric H+, K+-ATPase is a heterodimer consisting of a high molecular weight catalytic alpha subunit and a smaller but heavily glycosylated beta subunit. This enzyme is a proton pump that catalyzes the hydrolysis of ATP coupled with the exchange of H(+) and K(+) ions across the plasma membrane. It is also responsible for gastric acid secretion. This gene encodes the beta subunit of the gastric H+, K+-ATPase. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - intracellular sodium ion homeostasis\n",
      "    - cell adhesion\n",
      "    - intracellular potassium ion homeostasis\n",
      "    - response to lipopolysaccharide\n",
      "    - sodium ion export across plasma membrane\n",
      "    - pH reduction\n",
      "    - potassium ion transmembrane transport\n",
      "    - proton transmembrane transport\n",
      "    - potassium ion import across plasma membrane\n",
      "  CC:\n",
      "    - plasma membrane\n",
      "    - potassium:proton exchanging ATPase complex\n",
      "    - sodium:potassium-exchanging ATPase complex\n",
      "    - apical plasma membrane\n",
      "  MF:\n",
      "    - ATPase activator activity\n",
      "    - protein binding\n",
      "    - P-type potassium:proton transporter activity\n",
      "    - heterocyclic compound binding\n",
      "UniProt: {'Swiss-Prot': 'P51164'}\n",
      "\n",
      "Gene: CXCL10\n",
      "Symbol: CXCL10\n",
      "Name: C-X-C motif chemokine ligand 10\n",
      "Summary: This antimicrobial gene encodes a chemokine of the CXC subfamily and ligand for the receptor CXCR3. Binding of this protein to CXCR3 results in pleiotropic effects, including stimulation of monocytes, natural killer and T-cell migration, and modulation of adhesion molecule expression. This gene may also be a key regulator of the 'cytokine storm' immune response to SARS-CoV-2 infection. [provided by RefSeq, Sep 2020].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - chemotaxis\n",
      "    - inflammatory response\n",
      "    - signal transduction\n",
      "    - cell surface receptor signaling pathway\n",
      "    - G protein-coupled receptor signaling pathway\n",
      "    - adenylate cyclase-activating G protein-coupled receptor signaling pathway\n",
      "    - cell-cell signaling\n",
      "    - muscle organ development\n",
      "    - blood circulation\n",
      "    - positive regulation of cell population proliferation\n",
      "    - response to gamma radiation\n",
      "    - T cell chemotaxis\n",
      "    - regulation of T cell chemotaxis\n",
      "    - response to auditory stimulus\n",
      "    - negative regulation of angiogenesis\n",
      "    - neutrophil chemotaxis\n",
      "    - killing of cells of another organism\n",
      "    - response to vitamin D\n",
      "    - cellular response to heat\n",
      "    - endothelial cell activation\n",
      "    - regulation of cell population proliferation\n",
      "    - regulation of apoptotic process\n",
      "    - negative regulation of myoblast differentiation\n",
      "    - positive regulation of transcription by RNA polymerase II\n",
      "    - positive chemotaxis\n",
      "    - positive regulation of release of sequestered calcium ion into cytosol\n",
      "    - antimicrobial humoral immune response mediated by antimicrobial peptide\n",
      "    - chemokine-mediated signaling pathway\n",
      "    - chemokine-mediated signaling pathway\n",
      "    - cellular response to lipopolysaccharide\n",
      "    - cellular response to lipopolysaccharide\n",
      "    - positive regulation of monocyte chemotaxis\n",
      "    - cellular response to interleukin-17\n",
      "    - cellular response to virus\n",
      "    - antiviral innate immune response\n",
      "    - regulation of endothelial tube morphogenesis\n",
      "    - negative regulation of myoblast fusion\n",
      "    - positive regulation of T cell migration\n",
      "  CC:\n",
      "    - extracellular region\n",
      "    - extracellular region\n",
      "    - extracellular space\n",
      "    - external side of plasma membrane\n",
      "  MF:\n",
      "    - signaling receptor binding\n",
      "    - protein binding\n",
      "    - chemokine activity\n",
      "    - chemokine activity\n",
      "    - heparin binding\n",
      "    - cAMP-dependent protein kinase regulator activity\n",
      "    - chemoattractant activity\n",
      "    - CXCR chemokine receptor binding\n",
      "    - CXCR3 chemokine receptor binding\n",
      "UniProt: {'Swiss-Prot': 'P02778'}\n",
      "\n",
      "Gene: EPHA8\n",
      "Symbol: EPHA8\n",
      "Name: EPH receptor A8\n",
      "Summary: This gene encodes a member of the ephrin receptor subfamily of the protein-tyrosine kinase family. EPH and EPH-related receptors have been implicated in mediating developmental events, particularly in the nervous system. Receptors in the EPH subfamily typically have a single kinase domain and an extracellular region containing a Cys-rich domain and 2 fibronectin type III repeats. The ephrin receptors are divided into 2 groups based on the similarity of their extracellular domain sequences and their affinities for binding ephrin-A and ephrin-B ligands. The protein encoded by this gene functions as a receptor for ephrin A2, A3 and A5 and plays a role in short-range contact-mediated axonal guidance during development of the mammalian nervous system. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - substrate-dependent cell migration\n",
      "    - cell adhesion\n",
      "    - epidermal growth factor receptor signaling pathway\n",
      "    - axon guidance\n",
      "    - axon guidance\n",
      "    - insulin receptor signaling pathway\n",
      "    - fibroblast growth factor receptor signaling pathway\n",
      "    - neuron remodeling\n",
      "    - regulation of cell adhesion\n",
      "    - neuron projection development\n",
      "    - brain-derived neurotrophic factor receptor signaling pathway\n",
      "    - regulation of cell adhesion mediated by integrin\n",
      "    - platelet-derived growth factor receptor-alpha signaling pathway\n",
      "    - platelet-derived growth factor receptor-beta signaling pathway\n",
      "    - vascular endothelial growth factor receptor-1 signaling pathway\n",
      "    - collagen-activated tyrosine kinase receptor signaling pathway\n",
      "    - vascular endothelial growth factor signaling pathway\n",
      "    - Kit signaling pathway\n",
      "    - macrophage colony-stimulating factor signaling pathway\n",
      "    - positive regulation of MAPK cascade\n",
      "    - insulin-like growth factor receptor signaling pathway\n",
      "    - hepatocyte growth factor receptor signaling pathway\n",
      "    - ephrin receptor signaling pathway\n",
      "    - ephrin receptor signaling pathway\n",
      "    - positive regulation of phosphatidylinositol 3-kinase/protein kinase B signal transduction\n",
      "    - cellular response to follicle-stimulating hormone stimulus\n",
      "  CC:\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - dendrite\n",
      "    - early endosome membrane\n",
      "    - neuron projection\n",
      "  MF:\n",
      "    - ephrin receptor activity\n",
      "    - GPI-linked ephrin receptor activity\n",
      "    - transmembrane-ephrin receptor activity\n",
      "    - epidermal growth factor receptor activity\n",
      "    - fibroblast growth factor receptor activity\n",
      "    - hepatocyte growth factor receptor activity\n",
      "    - insulin receptor activity\n",
      "    - insulin-like growth factor receptor activity\n",
      "    - macrophage colony-stimulating factor receptor activity\n",
      "    - platelet-derived growth factor alpha-receptor activity\n",
      "    - platelet-derived growth factor beta-receptor activity\n",
      "    - stem cell factor receptor activity\n",
      "    - vascular endothelial growth factor receptor activity\n",
      "    - ATP binding\n",
      "    - boss receptor activity\n",
      "    - growth factor binding\n",
      "    - placental growth factor receptor activity\n",
      "    - protein tyrosine kinase collagen receptor activity\n",
      "    - brain-derived neurotrophic factor receptor activity\n",
      "UniProt: {'Swiss-Prot': 'P29322'}\n",
      "\n",
      "Gene: NR3C1\n",
      "Symbol: NR3C1\n",
      "Name: nuclear receptor subfamily 3 group C member 1\n",
      "Summary: This gene encodes glucocorticoid receptor, which can function both as a transcription factor that binds to glucocorticoid response elements in the promoters of glucocorticoid responsive genes to activate their transcription, and as a regulator of other transcription factors. This receptor is typically found in the cytoplasm, but upon ligand binding, is transported into the nucleus. It is involved in inflammatory responses, cellular proliferation, and differentiation in target tissues. Mutations in this gene are associated with generalized glucocorticoid resistance. Alternative splicing of this gene results in transcript variants encoding either the same or different isoforms. Additional isoforms resulting from the use of alternate in-frame translation initiation sites have also been described, and shown to be functional, displaying diverse cytoplasm-to-nucleus trafficking patterns and distinct transcriptional activities (PMID:15866175). [provided by RefSeq, Feb 2011].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - negative regulation of transcription by RNA polymerase II\n",
      "    - negative regulation of transcription by RNA polymerase II\n",
      "    - regulation of gluconeogenesis\n",
      "    - chromatin organization\n",
      "    - regulation of DNA-templated transcription\n",
      "    - regulation of transcription by RNA polymerase II\n",
      "    - apoptotic process\n",
      "    - chromosome segregation\n",
      "    - signal transduction\n",
      "    - glucocorticoid metabolic process\n",
      "    - response to wounding\n",
      "    - gene expression\n",
      "    - microglia differentiation\n",
      "    - adrenal gland development\n",
      "    - nuclear receptor-mediated steroid hormone signaling pathway\n",
      "    - regulation of glucocorticoid biosynthetic process\n",
      "    - synaptic transmission, glutamatergic\n",
      "    - maternal behavior\n",
      "    - nuclear receptor-mediated glucocorticoid signaling pathway\n",
      "    - positive regulation of neuron apoptotic process\n",
      "    - negative regulation of DNA-templated transcription\n",
      "    - positive regulation of transcription by RNA polymerase II\n",
      "    - positive regulation of transcription by RNA polymerase II\n",
      "    - astrocyte differentiation\n",
      "    - cell division\n",
      "    - response to cortisol\n",
      "    - mammary gland duct morphogenesis\n",
      "    - motor behavior\n",
      "    - cellular response to steroid hormone stimulus\n",
      "    - cellular response to glucocorticoid stimulus\n",
      "    - cellular response to dexamethasone stimulus\n",
      "    - cellular response to transforming growth factor beta stimulus\n",
      "    - neuroinflammatory response\n",
      "    - positive regulation of miRNA transcription\n",
      "  CC:\n",
      "    - chromatin\n",
      "    - chromatin\n",
      "    - nucleus\n",
      "    - nucleus\n",
      "    - nucleus\n",
      "    - nucleus\n",
      "    - nucleoplasm\n",
      "    - nucleoplasm\n",
      "    - cytoplasm\n",
      "    - mitochondrial matrix\n",
      "    - centrosome\n",
      "    - spindle\n",
      "    - cytosol\n",
      "    - cytosol\n",
      "    - membrane\n",
      "    - nuclear speck\n",
      "    - protein-containing complex\n",
      "    - synapse\n",
      "  MF:\n",
      "    - RNA polymerase II transcription regulatory region sequence-specific DNA binding\n",
      "    - RNA polymerase II cis-regulatory region sequence-specific DNA binding\n",
      "    - DNA-binding transcription factor activity, RNA polymerase II-specific\n",
      "    - core promoter sequence-specific DNA binding\n",
      "    - DNA-binding transcription repressor activity, RNA polymerase II-specific\n",
      "    - DNA-binding transcription activator activity, RNA polymerase II-specific\n",
      "    - DNA-binding transcription factor activity\n",
      "    - RNA binding\n",
      "    - nuclear receptor activity\n",
      "    - nuclear receptor activity\n",
      "    - nuclear glucocorticoid receptor activity\n",
      "    - steroid binding\n",
      "    - protein binding\n",
      "    - zinc ion binding\n",
      "    - TBP-class protein binding\n",
      "    - protein kinase binding\n",
      "    - estrogen response element binding\n",
      "    - identical protein binding\n",
      "    - Hsp90 protein binding\n",
      "    - steroid hormone binding\n",
      "    - sequence-specific double-stranded DNA binding\n",
      "    - promoter-specific chromatin binding\n",
      "UniProt: {'Swiss-Prot': 'P04150', 'TrEMBL': ['E5KQF6', 'E5KQF5', 'F1D8N4', 'Q3MSN4', 'A0A494C0P1', 'D6RDA9']}\n",
      "\n",
      "Gene: HDAC3\n",
      "Symbol: HDAC3\n",
      "Name: histone deacetylase 3\n",
      "Summary: Histones play a critical role in transcriptional regulation, cell cycle progression, and developmental events. Histone acetylation/deacetylation alters chromosome structure and affects transcription factor access to DNA. The protein encoded by this gene belongs to the histone deacetylase/acuc/apha family. It has histone deacetylase activity and represses transcription when tethered to a promoter. It may participate in the regulation of transcription through its binding with the zinc-finger transcription factor YY1. This protein can also down-regulate p53 function and thus modulate cell growth and apoptosis. This gene is regarded as a potential tumor suppressor gene. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - negative regulation of transcription by RNA polymerase II\n",
      "    - negative regulation of transcription by RNA polymerase II\n",
      "    - establishment of mitotic spindle orientation\n",
      "    - in utero embryonic development\n",
      "    - positive regulation of protein phosphorylation\n",
      "    - chromatin organization\n",
      "    - transcription by RNA polymerase II\n",
      "    - protein deacetylation\n",
      "    - regulation of mitotic cell cycle\n",
      "    - negative regulation of myotube differentiation\n",
      "    - positive regulation of protein ubiquitination\n",
      "    - regulation of protein stability\n",
      "    - positive regulation of TOR signaling\n",
      "    - circadian regulation of gene expression\n",
      "    - regulation of multicellular organism growth\n",
      "    - epigenetic regulation of gene expression\n",
      "    - positive regulation of protein import into nucleus\n",
      "    - regulation of circadian rhythm\n",
      "    - negative regulation of apoptotic process\n",
      "    - negative regulation of DNA-templated transcription\n",
      "    - negative regulation of DNA-templated transcription\n",
      "    - positive regulation of transcription by RNA polymerase II\n",
      "    - negative regulation of JNK cascade\n",
      "    - spindle assembly\n",
      "    - random inactivation of X chromosome\n",
      "    - neural precursor cell proliferation\n",
      "    - establishment of skin barrier\n",
      "    - cellular response to fluid shear stress\n",
      "    - positive regulation of cold-induced thermogenesis\n",
      "    - DNA repair-dependent chromatin remodeling\n",
      "    - cornified envelope assembly\n",
      "    - negative regulation of cardiac muscle cell differentiation\n",
      "  CC:\n",
      "    - histone deacetylase complex\n",
      "    - histone deacetylase complex\n",
      "    - chromatin\n",
      "    - nucleus\n",
      "    - nucleus\n",
      "    - nucleoplasm\n",
      "    - nucleoplasm\n",
      "    - cytoplasm\n",
      "    - cytoplasm\n",
      "    - Golgi apparatus\n",
      "    - cytosol\n",
      "    - plasma membrane\n",
      "    - transcription repressor complex\n",
      "    - mitotic spindle\n",
      "  MF:\n",
      "    - transcription corepressor binding\n",
      "    - chromatin binding\n",
      "    - transcription corepressor activity\n",
      "    - transcription corepressor activity\n",
      "    - transcription corepressor activity\n",
      "    - histone deacetylase activity\n",
      "    - histone deacetylase activity\n",
      "    - histone deacetylase activity\n",
      "    - histone deacetylase activity\n",
      "    - protein binding\n",
      "    - enzyme binding\n",
      "    - cyclin binding\n",
      "    - histone H3K14 deacetylase activity, hydrolytic mechanism\n",
      "    - chromatin DNA binding\n",
      "    - histone H3K9 deacetylase activity, hydrolytic mechanism\n",
      "    - protein lysine deacetylase activity\n",
      "    - protein lysine deacetylase activity\n",
      "    - histone H4K16 deacetylase activity, hydrolytic mechanism\n",
      "    - histone deacetylase binding\n",
      "    - NF-kappaB binding\n",
      "    - DNA-binding transcription factor binding\n",
      "    - histone H4K12 deacetylase activity, hydrolytic mechanism\n",
      "    - protein decrotonylase activity\n",
      "    - histone decrotonylase activity\n",
      "    - protein de-2-hydroxyisobutyrylase activity\n",
      "    - protein lysine delactylase activity\n",
      "    - histone H4K5 deacetylase activity, hydrolytic mechanism\n",
      "    - histone H4K8 deacetylase activity, hydrolytic mechanism\n",
      "    - histone H3K4 deacetylase activity, hydrolytic mechanism\n",
      "UniProt: {'Swiss-Prot': 'O15379', 'TrEMBL': ['E7EW19', 'E7ESJ6']}\n",
      "\n",
      "Gene: CEBPA\n",
      "Symbol: CEBPA\n",
      "Name: CCAAT enhancer binding protein alpha\n",
      "Summary: This intronless gene encodes a transcription factor that contains a basic leucine zipper (bZIP) domain and recognizes the CCAAT motif in the promoters of target genes. The encoded protein functions in homodimers and also heterodimers with CCAAT/enhancer-binding proteins beta and gamma. Activity of this protein can modulate the expression of genes involved in cell cycle regulation as well as in body weight homeostasis. Mutation of this gene is associated with acute myeloid leukemia. The use of alternative in-frame non-AUG (GUG) and AUG start codons results in protein isoforms with different lengths. Differential translation initiation is mediated by an out-of-frame, upstream open reading frame which is located between the GUG and the first AUG start codons. [provided by RefSeq, Dec 2013].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - urea cycle\n",
      "    - negative regulation of transcription by RNA polymerase II\n",
      "    - liver development\n",
      "    - embryonic placenta development\n",
      "    - epithelial cell maturation\n",
      "    - generation of precursor metabolites and energy\n",
      "    - DNA-templated transcription\n",
      "    - regulation of DNA-templated transcription\n",
      "    - regulation of transcription by RNA polymerase II\n",
      "    - regulation of transcription by RNA polymerase II\n",
      "    - transcription by RNA polymerase I\n",
      "    - mitochondrion organization\n",
      "    - Notch signaling pathway\n",
      "    - cholesterol metabolic process\n",
      "    - negative regulation of cell population proliferation\n",
      "    - negative regulation of cell population proliferation\n",
      "    - positive regulation of gene expression\n",
      "    - cytokine-mediated signaling pathway\n",
      "    - myeloid cell differentiation\n",
      "    - myeloid cell differentiation\n",
      "    - macrophage differentiation\n",
      "    - lung development\n",
      "    - granulocyte differentiation\n",
      "    - positive regulation of proteasomal ubiquitin-dependent protein catabolic process\n",
      "    - glucose homeostasis\n",
      "    - positive regulation of macrophage activation\n",
      "    - fat cell differentiation\n",
      "    - positive regulation of fat cell differentiation\n",
      "    - positive regulation of osteoblast differentiation\n",
      "    - negative regulation of cell cycle\n",
      "    - negative regulation of DNA-templated transcription\n",
      "    - positive regulation of transcription by RNA polymerase II\n",
      "    - positive regulation of transcription by RNA polymerase II\n",
      "    - positive regulation of transcription by RNA polymerase II\n",
      "    - positive regulation of transcription by RNA polymerase II\n",
      "    - inner ear development\n",
      "    - positive regulation of inflammatory response\n",
      "    - white fat cell differentiation\n",
      "    - brown fat cell differentiation\n",
      "    - regulation of cell cycle\n",
      "    - lipid homeostasis\n",
      "    - interleukin-6-mediated signaling pathway\n",
      "    - cellular response to lithium ion\n",
      "    - cellular response to tumor necrosis factor\n",
      "    - hematopoietic stem cell proliferation\n",
      "    - energy homeostasis\n",
      "    - integrated stress response signaling\n",
      "    - negative regulation of hematopoietic stem cell proliferation\n",
      "    - positive regulation of DNA-templated transcription initiation\n",
      "  CC:\n",
      "    - chromatin\n",
      "    - nucleus\n",
      "    - nucleus\n",
      "    - nucleus\n",
      "    - nucleoplasm\n",
      "    - nucleoplasm\n",
      "    - transcription regulator complex\n",
      "    - nucleolus\n",
      "    - nucleolus\n",
      "    - CHOP-C/EBP complex\n",
      "    - intracellular membrane-bounded organelle\n",
      "    - RNA polymerase II transcription regulator complex\n",
      "    - RNA polymerase II transcription regulator complex\n",
      "    - C/EBP complex\n",
      "  MF:\n",
      "    - transcription cis-regulatory region binding\n",
      "    - RNA polymerase II cis-regulatory region sequence-specific DNA binding\n",
      "    - DNA-binding transcription factor activity, RNA polymerase II-specific\n",
      "    - DNA-binding transcription factor activity, RNA polymerase II-specific\n",
      "    - RNA polymerase I transcription regulatory region sequence-specific DNA binding\n",
      "    - DNA-binding transcription activator activity, RNA polymerase II-specific\n",
      "    - DNA-binding transcription activator activity, RNA polymerase II-specific\n",
      "    - DNA binding\n",
      "    - DNA-binding transcription factor activity\n",
      "    - protein binding\n",
      "    - kinase binding\n",
      "    - chromatin DNA binding\n",
      "    - identical protein binding\n",
      "    - protein homodimerization activity\n",
      "    - RNA polymerase II-specific DNA-binding transcription factor binding\n",
      "    - STAT family protein binding\n",
      "UniProt: {'Swiss-Prot': 'P49715'}\n",
      "\n",
      "Gene: BTC\n",
      "Symbol: BTC\n",
      "Name: betacellulin\n",
      "Summary: This gene encodes a member of the epidermal growth factor (EGF) family of proteins. Alternative splicing results in multiple transcript variants, at least one of which encodes a preproprotein that is proteolytically processed to generate the secreted growth factor. A secreted form and a membrane-anchored form of this protein bind to multiple different EGF receptors. This protein promotes pancreatic cell proliferation and insulin secretion, as well as retinal vascular permeability. Mutations in this gene may be associated with type 2 diabetes in human patients. [provided by RefSeq, Nov 2015].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - epidermal growth factor receptor signaling pathway\n",
      "    - cell population proliferation\n",
      "    - positive regulation of cell population proliferation\n",
      "    - positive regulation of urine volume\n",
      "    - ERBB2-EGFR signaling pathway\n",
      "    - ERBB4-ERBB4 signaling pathway\n",
      "    - positive regulation of cell differentiation\n",
      "    - positive regulation of mitotic nuclear division\n",
      "    - positive regulation of fibroblast proliferation\n",
      "    - positive regulation of cell division\n",
      "    - epithelial cell apoptotic process\n",
      "    - negative regulation of epithelial cell apoptotic process\n",
      "  CC:\n",
      "    - extracellular region\n",
      "    - extracellular space\n",
      "    - extracellular space\n",
      "    - plasma membrane\n",
      "    - clathrin-coated endocytic vesicle membrane\n",
      "  MF:\n",
      "    - epidermal growth factor receptor binding\n",
      "    - protein binding\n",
      "    - growth factor activity\n",
      "    - transmembrane receptor protein tyrosine kinase activator activity\n",
      "    - receptor ligand activity\n",
      "UniProt: {'Swiss-Prot': 'P35070', 'TrEMBL': ['A0A0S2Z3I5', 'A0A0S2Z437', 'H0Y8Q5']}\n",
      "\n",
      "Gene: IL1R1\n",
      "Symbol: IL1R1\n",
      "Name: interleukin 1 receptor type 1\n",
      "Summary: This gene encodes a cytokine receptor that belongs to the interleukin-1 receptor family. The encoded protein is a receptor for interleukin-1 alpha, interleukin-1 beta, and interleukin-1 receptor antagonist. It is an important mediator involved in many cytokine-induced immune and inflammatory responses. This gene is located in a cluster of related cytokine receptor genes on chromosome 2q12. [provided by RefSeq, Dec 2013].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - inflammatory response\n",
      "    - immune response\n",
      "    - cell surface receptor signaling pathway\n",
      "    - positive regulation of platelet-derived growth factor receptor signaling pathway\n",
      "    - smooth muscle adaptation\n",
      "    - positive regulation of type II interferon production\n",
      "    - positive regulation of canonical NF-kappaB signal transduction\n",
      "    - regulation of inflammatory response\n",
      "    - positive regulation of phosphatidylinositol 3-kinase/protein kinase B signal transduction\n",
      "    - interleukin-1-mediated signaling pathway\n",
      "    - interleukin-1-mediated signaling pathway\n",
      "    - response to interleukin-1\n",
      "    - positive regulation of neutrophil extravasation\n",
      "    - positive regulation of T-helper 1 cell cytokine production\n",
      "    - positive regulation of interleukin-1-mediated signaling pathway\n",
      "  CC:\n",
      "    - extracellular region\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - external side of plasma membrane\n",
      "    - cell surface\n",
      "    - membrane\n",
      "  MF:\n",
      "    - protease binding\n",
      "    - transmembrane signaling receptor activity\n",
      "    - interleukin-1 receptor activity\n",
      "    - interleukin-1 receptor activity\n",
      "    - interleukin-1, type I, activating receptor activity\n",
      "    - platelet-derived growth factor receptor binding\n",
      "    - protein binding\n",
      "    - interleukin-1 binding\n",
      "    - NAD+ nucleosidase activity, cyclic ADP-ribose generating\n",
      "UniProt: {'Swiss-Prot': 'P14778', 'TrEMBL': ['B8ZZ73', 'B8ZZW4', 'B9A040', 'C9JWB2', 'H7BZM5', 'C9JQ36', 'C9JW84', 'C9J3W8', 'C9J686', 'F8WF92']}\n",
      "\n",
      "Gene: EPOR\n",
      "Symbol: EPOR\n",
      "Name: erythropoietin receptor\n",
      "Summary: This gene encodes the erythropoietin receptor which is a member of the cytokine receptor family. Upon erythropoietin binding, this receptor activates Jak2 tyrosine kinase which activates different intracellular pathways including: Ras/MAP kinase, phosphatidylinositol 3-kinase and STAT transcription factors. The stimulated erythropoietin receptor appears to have a role in erythroid cell survival. Defects in the erythropoietin receptor may produce erythroleukemia and familial erythrocytosis. Dysregulation of this gene may affect the growth of certain tumors. Alternate splicing results in multiple transcript variants.[provided by RefSeq, May 2010].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - signal transduction\n",
      "    - brain development\n",
      "    - heart development\n",
      "    - positive regulation of cell population proliferation\n",
      "    - cytokine-mediated signaling pathway\n",
      "    - erythropoietin-mediated signaling pathway\n",
      "    - decidualization\n",
      "  CC:\n",
      "    - extracellular region\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - external side of plasma membrane\n",
      "    - nuclear speck\n",
      "  MF:\n",
      "    - erythropoietin receptor activity\n",
      "    - erythropoietin receptor activity\n",
      "    - protein binding\n",
      "    - identical protein binding\n",
      "UniProt: {'Swiss-Prot': 'P19235', 'TrEMBL': ['K7EN47', 'K7ERB5', 'K7EP50']}\n",
      "\n",
      "Gene: NTRK1\n",
      "Symbol: NTRK1\n",
      "Name: neurotrophic receptor tyrosine kinase 1\n",
      "Summary: This gene encodes a member of the neurotrophic tyrosine kinase receptor (NTKR) family. This kinase is a membrane-bound receptor that, upon neurotrophin binding, phosphorylates itself and members of the MAPK pathway. The presence of this kinase leads to cell differentiation and may play a role in specifying sensory neuron subtypes. Mutations in this gene have been associated with congenital insensitivity to pain, anhidrosis, self-mutilating behavior, cognitive disability and cancer. Alternate transcriptional splice variants of this gene have been found, but only three have been characterized to date. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - protein phosphorylation\n",
      "    - cell surface receptor protein tyrosine kinase signaling pathway\n",
      "    - epidermal growth factor receptor signaling pathway\n",
      "    - spermatogenesis\n",
      "    - axon guidance\n",
      "    - learning or memory\n",
      "    - circadian rhythm\n",
      "    - negative regulation of cell population proliferation\n",
      "    - negative regulation of cell population proliferation\n",
      "    - insulin receptor signaling pathway\n",
      "    - fibroblast growth factor receptor signaling pathway\n",
      "    - response to xenobiotic stimulus\n",
      "    - programmed cell death involved in cell development\n",
      "    - positive regulation of neuron projection development\n",
      "    - positive regulation of neuron projection development\n",
      "    - peptidyl-tyrosine phosphorylation\n",
      "    - olfactory nerve development\n",
      "    - B cell differentiation\n",
      "    - neuron projection development\n",
      "    - brain-derived neurotrophic factor receptor signaling pathway\n",
      "    - response to nutrient levels\n",
      "    - platelet-derived growth factor receptor-alpha signaling pathway\n",
      "    - platelet-derived growth factor receptor-beta signaling pathway\n",
      "    - vascular endothelial growth factor receptor-1 signaling pathway\n",
      "    - collagen-activated tyrosine kinase receptor signaling pathway\n",
      "    - peptidyl-tyrosine autophosphorylation\n",
      "    - peptidyl-tyrosine autophosphorylation\n",
      "    - vascular endothelial growth factor signaling pathway\n",
      "    - Kit signaling pathway\n",
      "    - macrophage colony-stimulating factor signaling pathway\n",
      "    - nerve growth factor signaling pathway\n",
      "    - nerve growth factor signaling pathway\n",
      "    - mechanoreceptor differentiation\n",
      "    - negative regulation of apoptotic process\n",
      "    - positive regulation of programmed cell death\n",
      "    - negative regulation of neuron apoptotic process\n",
      "    - positive regulation of GTPase activity\n",
      "    - positive regulation of GTPase activity\n",
      "    - positive regulation of angiogenesis\n",
      "    - positive regulation of Ras protein signal transduction\n",
      "    - positive regulation of Ras protein signal transduction\n",
      "    - protein autophosphorylation\n",
      "    - insulin-like growth factor receptor signaling pathway\n",
      "    - neurotrophin TRK receptor signaling pathway\n",
      "    - hepatocyte growth factor receptor signaling pathway\n",
      "    - ephrin receptor signaling pathway\n",
      "    - sympathetic nervous system development\n",
      "    - neuron development\n",
      "    - response to axon injury\n",
      "    - detection of temperature stimulus involved in sensory perception of pain\n",
      "    - detection of mechanical stimulus involved in sensory perception of pain\n",
      "    - positive regulation of NF-kappaB transcription factor activity\n",
      "    - neuron apoptotic process\n",
      "    - response to hydrostatic pressure\n",
      "    - response to electrical stimulus\n",
      "    - positive regulation of phosphatidylinositol 3-kinase/protein kinase B signal transduction\n",
      "    - positive regulation of phosphatidylinositol 3-kinase/protein kinase B signal transduction\n",
      "    - positive regulation of synapse assembly\n",
      "    - positive regulation of synaptic transmission, glutamatergic\n",
      "    - Sertoli cell development\n",
      "    - axonogenesis involved in innervation\n",
      "    - behavioral response to formalin induced pain\n",
      "    - positive regulation of ERK1 and ERK2 cascade\n",
      "    - positive regulation of ERK1 and ERK2 cascade\n",
      "    - cellular response to nicotine\n",
      "    - cellular response to amyloid-beta\n",
      "    - cellular response to nerve growth factor stimulus\n",
      "    - cellular response to nerve growth factor stimulus\n",
      "    - cellular response to nerve growth factor stimulus\n",
      "  CC:\n",
      "    - mitochondrion\n",
      "    - early endosome\n",
      "    - late endosome\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - cell surface\n",
      "    - endosome membrane\n",
      "    - axon\n",
      "    - dendrite\n",
      "    - early endosome membrane\n",
      "    - late endosome membrane\n",
      "    - protein-containing complex\n",
      "    - neuronal cell body\n",
      "    - receptor complex\n",
      "    - receptor complex\n",
      "    - recycling endosome membrane\n",
      "  MF:\n",
      "    - protein tyrosine kinase activity\n",
      "    - transmembrane receptor protein tyrosine kinase activity\n",
      "    - transmembrane receptor protein tyrosine kinase activity\n",
      "    - transmembrane receptor protein tyrosine kinase activity\n",
      "    - GPI-linked ephrin receptor activity\n",
      "    - transmembrane-ephrin receptor activity\n",
      "    - epidermal growth factor receptor activity\n",
      "    - fibroblast growth factor receptor activity\n",
      "    - hepatocyte growth factor receptor activity\n",
      "    - insulin receptor activity\n",
      "    - insulin-like growth factor receptor activity\n",
      "    - macrophage colony-stimulating factor receptor activity\n",
      "    - platelet-derived growth factor alpha-receptor activity\n",
      "    - platelet-derived growth factor beta-receptor activity\n",
      "    - stem cell factor receptor activity\n",
      "    - vascular endothelial growth factor receptor activity\n",
      "    - neurotrophin receptor activity\n",
      "    - neurotrophin p75 receptor binding\n",
      "    - protein binding\n",
      "    - ATP binding\n",
      "    - boss receptor activity\n",
      "    - nerve growth factor receptor activity\n",
      "    - nerve growth factor receptor activity\n",
      "    - kinase binding\n",
      "    - placental growth factor receptor activity\n",
      "    - protein tyrosine kinase collagen receptor activity\n",
      "    - identical protein binding\n",
      "    - protein homodimerization activity\n",
      "    - neurotrophin binding\n",
      "    - nerve growth factor binding\n",
      "    - nerve growth factor binding\n",
      "    - brain-derived neurotrophic factor receptor activity\n",
      "UniProt: {'Swiss-Prot': 'P04629', 'TrEMBL': ['A0A6Q8PHG5', 'X5DR71', 'A0A6Q8PGU5', 'A0A6Q8PF65', 'J3KP20', 'E9PQG0']}\n",
      "\n",
      "Gene: EPHA2\n",
      "Symbol: EPHA2\n",
      "Name: EPH receptor A2\n",
      "Summary: This gene belongs to the ephrin receptor subfamily of the protein-tyrosine kinase family. EPH and EPH-related receptors have been implicated in mediating developmental events, particularly in the nervous system. Receptors in the EPH subfamily typically have a single kinase domain and an extracellular region containing a Cys-rich domain and 2 fibronectin type III repeats. The ephrin receptors are divided into 2 groups based on the similarity of their extracellular domain sequences and their affinities for binding ephrin-A and ephrin-B ligands. This gene encodes a protein that binds ephrin-A ligands. Mutations in this gene are the cause of certain genetically-related cataract disorders.[provided by RefSeq, May 2010].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - skeletal system development\n",
      "    - angiogenesis\n",
      "    - vasculogenesis\n",
      "    - osteoblast differentiation\n",
      "    - blood vessel endothelial cell proliferation involved in sprouting angiogenesis\n",
      "    - inflammatory response\n",
      "    - cell adhesion\n",
      "    - cell surface receptor protein tyrosine kinase signaling pathway\n",
      "    - epidermal growth factor receptor signaling pathway\n",
      "    - insulin receptor signaling pathway\n",
      "    - fibroblast growth factor receptor signaling pathway\n",
      "    - intrinsic apoptotic signaling pathway in response to DNA damage\n",
      "    - regulation of lamellipodium assembly\n",
      "    - notochord formation\n",
      "    - cell migration\n",
      "    - negative regulation of angiogenesis\n",
      "    - neural tube development\n",
      "    - neuron differentiation\n",
      "    - keratinocyte differentiation\n",
      "    - osteoclast differentiation\n",
      "    - positive regulation of cell migration\n",
      "    - brain-derived neurotrophic factor receptor signaling pathway\n",
      "    - negative regulation of chemokine production\n",
      "    - mammary gland epithelial cell proliferation\n",
      "    - regulation of cell adhesion mediated by integrin\n",
      "    - platelet-derived growth factor receptor-alpha signaling pathway\n",
      "    - platelet-derived growth factor receptor-beta signaling pathway\n",
      "    - vascular endothelial growth factor receptor-1 signaling pathway\n",
      "    - post-anal tail morphogenesis\n",
      "    - collagen-activated tyrosine kinase receptor signaling pathway\n",
      "    - vascular endothelial growth factor signaling pathway\n",
      "    - Kit signaling pathway\n",
      "    - macrophage colony-stimulating factor signaling pathway\n",
      "    - regulation of blood vessel endothelial cell migration\n",
      "    - regulation of angiogenesis\n",
      "    - cAMP metabolic process\n",
      "    - symbiont entry into host cell\n",
      "    - bone remodeling\n",
      "    - insulin-like growth factor receptor signaling pathway\n",
      "    - hepatocyte growth factor receptor signaling pathway\n",
      "    - ephrin receptor signaling pathway\n",
      "    - axial mesoderm formation\n",
      "    - cell motility\n",
      "    - defense response to Gram-positive bacterium\n",
      "    - notochord cell development\n",
      "    - cell chemotaxis\n",
      "    - branching involved in mammary gland duct morphogenesis\n",
      "    - lens fiber cell morphogenesis\n",
      "    - regulation of ERK1 and ERK2 cascade\n",
      "    - response to growth factor\n",
      "    - protein localization to plasma membrane\n",
      "    - activation of GTPase activity\n",
      "    - negative regulation of lymphangiogenesis\n",
      "    - positive regulation of protein localization to plasma membrane\n",
      "    - positive regulation of bicellular tight junction assembly\n",
      "    - pericyte cell differentiation\n",
      "  CC:\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - focal adhesion\n",
      "    - focal adhesion\n",
      "    - cell surface\n",
      "    - cell surface\n",
      "    - lamellipodium\n",
      "    - leading edge membrane\n",
      "    - lamellipodium membrane\n",
      "    - ruffle membrane\n",
      "    - receptor complex\n",
      "    - tight junction\n",
      "  MF:\n",
      "    - virus receptor activity\n",
      "    - transmembrane receptor protein tyrosine kinase activity\n",
      "    - transmembrane receptor protein tyrosine kinase activity\n",
      "    - GPI-linked ephrin receptor activity\n",
      "    - transmembrane-ephrin receptor activity\n",
      "    - epidermal growth factor receptor activity\n",
      "    - fibroblast growth factor receptor activity\n",
      "    - hepatocyte growth factor receptor activity\n",
      "    - insulin receptor activity\n",
      "    - insulin-like growth factor receptor activity\n",
      "    - macrophage colony-stimulating factor receptor activity\n",
      "    - platelet-derived growth factor alpha-receptor activity\n",
      "    - platelet-derived growth factor beta-receptor activity\n",
      "    - stem cell factor receptor activity\n",
      "    - vascular endothelial growth factor receptor activity\n",
      "    - protein binding\n",
      "    - ATP binding\n",
      "    - boss receptor activity\n",
      "    - growth factor binding\n",
      "    - placental growth factor receptor activity\n",
      "    - protein tyrosine kinase collagen receptor activity\n",
      "    - cadherin binding\n",
      "    - brain-derived neurotrophic factor receptor activity\n",
      "    - molecular function activator activity\n",
      "UniProt: {'Swiss-Prot': 'P29317'}\n",
      "\n",
      "Gene: DLL4\n",
      "Symbol: DLL4\n",
      "Name: delta like canonical Notch ligand 4\n",
      "Summary: This gene is a homolog of the Drosophila delta gene. The delta gene family encodes Notch ligands that are characterized by a DSL domain, EGF repeats, and a transmembrane domain. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - negative regulation of transcription by RNA polymerase II\n",
      "    - angiogenesis\n",
      "    - branching involved in blood vessel morphogenesis\n",
      "    - blood vessel remodeling\n",
      "    - aortic valve morphogenesis\n",
      "    - cardiac ventricle morphogenesis\n",
      "    - cardiac atrium morphogenesis\n",
      "    - ventricular trabecula myocardium morphogenesis\n",
      "    - pericardium morphogenesis\n",
      "    - signal transduction\n",
      "    - Notch signaling pathway\n",
      "    - Notch signaling pathway\n",
      "    - Notch signaling pathway\n",
      "    - visual perception\n",
      "    - negative regulation of cell population proliferation\n",
      "    - negative regulation of endothelial cell migration\n",
      "    - positive regulation of gene expression\n",
      "    - positive regulation of gene expression\n",
      "    - negative regulation of gene expression\n",
      "    - T cell differentiation\n",
      "    - dorsal aorta morphogenesis\n",
      "    - cellular response to vascular endothelial growth factor stimulus\n",
      "    - cellular response to fibroblast growth factor stimulus\n",
      "    - negative regulation of Notch signaling pathway\n",
      "    - negative regulation of Notch signaling pathway\n",
      "    - positive regulation of Notch signaling pathway\n",
      "    - regulation of neurogenesis\n",
      "    - ventral spinal cord interneuron fate commitment\n",
      "    - regulation of neural retina development\n",
      "    - blood vessel lumenization\n",
      "    - negative regulation of cell migration involved in sprouting angiogenesis\n",
      "    - negative regulation of cell migration involved in sprouting angiogenesis\n",
      "    - negative regulation of blood vessel endothelial cell proliferation involved in sprouting angiogenesis\n",
      "    - positive regulation of neural precursor cell proliferation\n",
      "  CC:\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "  MF:\n",
      "    - Notch binding\n",
      "    - Notch binding\n",
      "    - calcium ion binding\n",
      "    - protein binding\n",
      "    - receptor ligand activity\n",
      "UniProt: {'Swiss-Prot': 'Q9NR61'}\n",
      "\n",
      "Gene: SMO\n",
      "Symbol: SMO\n",
      "Name: smoothened, frizzled class receptor\n",
      "Summary: The protein encoded by this gene is a G protein-coupled receptor that interacts with the patched protein, a receptor for hedgehog proteins. The encoded protein tranduces signals to other proteins after activation by a hedgehog protein/patched protein complex. [provided by RefSeq, Jul 2010].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - negative regulation of transcription by RNA polymerase II\n",
      "    - vasculogenesis\n",
      "    - osteoblast differentiation\n",
      "    - in utero embryonic development\n",
      "    - cell fate specification\n",
      "    - neural crest cell migration\n",
      "    - negative regulation of protein phosphorylation\n",
      "    - heart looping\n",
      "    - positive regulation of neuroblast proliferation\n",
      "    - positive regulation of mesenchymal cell proliferation\n",
      "    - determination of left/right asymmetry in lateral mesoderm\n",
      "    - type B pancreatic cell development\n",
      "    - protein import into nucleus\n",
      "    - apoptotic process\n",
      "    - G protein-coupled receptor signaling pathway\n",
      "    - smoothened signaling pathway\n",
      "    - smoothened signaling pathway\n",
      "    - smoothened signaling pathway\n",
      "    - smoothened signaling pathway\n",
      "    - smoothened signaling pathway\n",
      "    - ventral midline determination\n",
      "    - pattern specification process\n",
      "    - neuroblast proliferation\n",
      "    - central nervous system development\n",
      "    - midgut development\n",
      "    - anterior/posterior pattern specification\n",
      "    - gene expression\n",
      "    - positive regulation of gene expression\n",
      "    - positive regulation of gene expression\n",
      "    - negative regulation of gene expression\n",
      "    - spinal cord dorsal/ventral patterning\n",
      "    - dentate gyrus development\n",
      "    - cerebellar cortex morphogenesis\n",
      "    - thalamus development\n",
      "    - dorsal/ventral neural tube patterning\n",
      "    - central nervous system neuron differentiation\n",
      "    - cerebral cortex development\n",
      "    - positive regulation of cell migration\n",
      "    - negative regulation of epithelial cell differentiation\n",
      "    - hair follicle morphogenesis\n",
      "    - multicellular organism growth\n",
      "    - positive regulation of multicellular organism growth\n",
      "    - positive regulation of protein import into nucleus\n",
      "    - odontogenesis of dentin-containing tooth\n",
      "    - negative regulation of apoptotic process\n",
      "    - negative regulation of DNA binding\n",
      "    - positive regulation of smoothened signaling pathway\n",
      "    - positive regulation of transcription by RNA polymerase II\n",
      "    - positive regulation of organ growth\n",
      "    - astrocyte activation\n",
      "    - skeletal muscle fiber development\n",
      "    - smooth muscle tissue development\n",
      "    - forebrain morphogenesis\n",
      "    - homeostasis of number of cells within a tissue\n",
      "    - epithelial cell proliferation\n",
      "    - positive regulation of epithelial cell proliferation\n",
      "    - protein stabilization\n",
      "    - myoblast migration\n",
      "    - negative regulation of hair follicle development\n",
      "    - contact inhibition\n",
      "    - atrial septum morphogenesis\n",
      "    - mammary gland epithelial cell differentiation\n",
      "    - epithelial-mesenchymal cell signaling\n",
      "    - somite development\n",
      "    - pancreas morphogenesis\n",
      "    - left/right axis specification\n",
      "    - cellular response to cholesterol\n",
      "    - dopaminergic neuron differentiation\n",
      "    - commissural neuron axon guidance\n",
      "    - mesenchymal to epithelial transition involved in metanephric renal vesicle formation\n",
      "    - positive regulation of branching involved in ureteric bud morphogenesis\n",
      "    - response to inositol\n",
      "    - regulation of somatic stem cell population maintenance\n",
      "    - regulation of heart morphogenesis\n",
      "  CC:\n",
      "    - late endosome\n",
      "    - endoplasmic reticulum\n",
      "    - endoplasmic reticulum-Golgi intermediate compartment\n",
      "    - Golgi apparatus\n",
      "    - centriole\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - caveola\n",
      "    - cilium\n",
      "    - cilium\n",
      "    - cilium\n",
      "    - cilium\n",
      "    - dendrite\n",
      "    - endocytic vesicle membrane\n",
      "    - intracellular membrane-bounded organelle\n",
      "    - ciliary membrane\n",
      "    - extracellular exosome\n",
      "    - ciliary tip\n",
      "    - 9+0 non-motile cilium\n",
      "  MF:\n",
      "    - cAMP-dependent protein kinase inhibitor activity\n",
      "    - G protein-coupled receptor activity\n",
      "    - patched binding\n",
      "    - patched binding\n",
      "    - protein binding\n",
      "    - oxysterol binding\n",
      "    - protein kinase A catalytic subunit binding\n",
      "    - protein sequestering activity\n",
      "UniProt: {'Swiss-Prot': 'Q99835', 'TrEMBL': ['A0A590UJE7', 'H7C5S9', 'H7C509']}\n",
      "\n",
      "Gene: EPHB4\n",
      "Symbol: EPHB4\n",
      "Name: EPH receptor B4\n",
      "Summary: Ephrin receptors and their ligands, the ephrins, mediate numerous developmental processes, particularly in the nervous system. Based on their structures and sequence relationships, ephrins are divided into the ephrin-A (EFNA) class, which are anchored to the membrane by a glycosylphosphatidylinositol linkage, and the ephrin-B (EFNB) class, which are transmembrane proteins. The Eph family of receptors are divided into 2 groups based on the similarity of their extracellular domain sequences and their affinities for binding ephrin-A and ephrin-B ligands. Ephrin receptors make up the largest subgroup of the receptor tyrosine kinase (RTK) family. The protein encoded by this gene binds to ephrin-B2 and plays an essential role in vascular development. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - angiogenesis\n",
      "    - angiogenesis\n",
      "    - cell migration involved in sprouting angiogenesis\n",
      "    - heart morphogenesis\n",
      "    - cell adhesion\n",
      "    - epidermal growth factor receptor signaling pathway\n",
      "    - insulin receptor signaling pathway\n",
      "    - fibroblast growth factor receptor signaling pathway\n",
      "    - brain-derived neurotrophic factor receptor signaling pathway\n",
      "    - platelet-derived growth factor receptor-alpha signaling pathway\n",
      "    - platelet-derived growth factor receptor-beta signaling pathway\n",
      "    - vascular endothelial growth factor receptor-1 signaling pathway\n",
      "    - collagen-activated tyrosine kinase receptor signaling pathway\n",
      "    - vascular endothelial growth factor signaling pathway\n",
      "    - Kit signaling pathway\n",
      "    - macrophage colony-stimulating factor signaling pathway\n",
      "    - protein autophosphorylation\n",
      "    - insulin-like growth factor receptor signaling pathway\n",
      "    - hepatocyte growth factor receptor signaling pathway\n",
      "    - ephrin receptor signaling pathway\n",
      "    - ephrin receptor signaling pathway\n",
      "  CC:\n",
      "    - extracellular region\n",
      "    - cytosol\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - receptor complex\n",
      "    - extracellular exosome\n",
      "  MF:\n",
      "    - transmembrane receptor protein tyrosine kinase activity\n",
      "    - transmembrane receptor protein tyrosine kinase activity\n",
      "    - ephrin receptor activity\n",
      "    - GPI-linked ephrin receptor activity\n",
      "    - transmembrane-ephrin receptor activity\n",
      "    - epidermal growth factor receptor activity\n",
      "    - fibroblast growth factor receptor activity\n",
      "    - hepatocyte growth factor receptor activity\n",
      "    - insulin receptor activity\n",
      "    - insulin-like growth factor receptor activity\n",
      "    - macrophage colony-stimulating factor receptor activity\n",
      "    - platelet-derived growth factor alpha-receptor activity\n",
      "    - platelet-derived growth factor beta-receptor activity\n",
      "    - stem cell factor receptor activity\n",
      "    - vascular endothelial growth factor receptor activity\n",
      "    - protein binding\n",
      "    - ATP binding\n",
      "    - boss receptor activity\n",
      "    - placental growth factor receptor activity\n",
      "    - protein tyrosine kinase collagen receptor activity\n",
      "    - brain-derived neurotrophic factor receptor activity\n",
      "UniProt: {'Swiss-Prot': 'P54760', 'TrEMBL': ['Q541P7', 'Q96L35']}\n",
      "\n",
      "Gene: TDGF1\n",
      "Symbol: CRIPTO\n",
      "Name: cripto, EGF-CFC family member\n",
      "Summary: This gene encodes an epidermal growth factor-related protein that contains a cripto, FRL-1, and cryptic domain. The encoded protein is an extracellular, membrane-bound signaling protein that plays an essential role in embryonic development and tumor growth. Mutations in this gene are associated with forebrain defects. Pseudogenes of this gene are found on chromosomes 2, 3, 6, 8, 19 and X. Alternate splicing results in multiple transcript variants. [provided by RefSeq, Mar 2010].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - blood vessel development\n",
      "    - morphogenesis of a branching structure\n",
      "    - cell migration involved in sprouting angiogenesis\n",
      "    - epidermal growth factor receptor signaling pathway\n",
      "    - determination of left/right symmetry\n",
      "    - heart development\n",
      "    - heart development\n",
      "    - positive regulation of cell population proliferation\n",
      "    - positive regulation of cell population proliferation\n",
      "    - anterior/posterior axis specification, embryo\n",
      "    - embryo development ending in birth or egg hatching\n",
      "    - anterior/posterior pattern specification\n",
      "    - regulation of signal transduction\n",
      "    - positive regulation of endothelial cell migration\n",
      "    - peptidyl-serine phosphorylation\n",
      "    - cell differentiation\n",
      "    - positive regulation of cell migration\n",
      "    - mammary gland development\n",
      "    - cellular response to hepatocyte growth factor stimulus\n",
      "    - nodal signaling pathway\n",
      "    - negative regulation of apoptotic process\n",
      "    - positive regulation of MAP kinase activity\n",
      "    - positive regulation of MAPK cascade\n",
      "    - cellular response to fibroblast growth factor stimulus\n",
      "    - positive regulation of peptidyl-tyrosine phosphorylation\n",
      "    - cellular response to type II interferon\n",
      "    - cellular response to interleukin-6\n",
      "    - cellular response to tumor necrosis factor\n",
      "    - cellular response to epidermal growth factor stimulus\n",
      "  CC:\n",
      "    - extracellular region\n",
      "    - extracellular space\n",
      "    - plasma membrane\n",
      "    - cell surface\n",
      "    - cell surface\n",
      "    - apical plasma membrane\n",
      "    - extrinsic component of plasma membrane\n",
      "    - membrane raft\n",
      "    - side of membrane\n",
      "  MF:\n",
      "    - signaling receptor binding\n",
      "    - protein binding\n",
      "    - growth factor activity\n",
      "    - nodal binding\n",
      "    - activin receptor binding\n",
      "UniProt: {'Swiss-Prot': 'P13385', 'TrEMBL': 'F5H1T8'}\n",
      "\n",
      "Gene: PGF\n",
      "Symbol: PGF\n",
      "Name: placental growth factor\n",
      "Summary: This gene encodes a growth factor found in placenta which is homologous to vascular endothelial growth factor. Alternatively spliced transcripts encoding different isoforms have been found for this gene.[provided by RefSeq, Jun 2011]\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - response to hypoxia\n",
      "    - sprouting angiogenesis\n",
      "    - signal transduction\n",
      "    - cell-cell signaling\n",
      "    - positive regulation of cell population proliferation\n",
      "    - cell differentiation\n",
      "    - vascular endothelial growth factor signaling pathway\n",
      "    - vascular endothelial growth factor receptor signaling pathway\n",
      "    - positive chemotaxis\n",
      "    - induction of positive chemotaxis\n",
      "    - positive regulation of cell division\n",
      "    - positive regulation of mast cell chemotaxis\n",
      "  CC:\n",
      "    - extracellular region\n",
      "    - extracellular space\n",
      "    - membrane\n",
      "  MF:\n",
      "    - vascular endothelial growth factor receptor binding\n",
      "    - protein binding\n",
      "    - growth factor activity\n",
      "    - growth factor activity\n",
      "    - heparin binding\n",
      "    - chemoattractant activity\n",
      "UniProt: {'Swiss-Prot': 'P49763', 'TrEMBL': ['G3XA84', 'Q53XY6', 'Q86TW6']}\n",
      "\n",
      "Gene: CDH2\n",
      "Symbol: CDH2\n",
      "Name: cadherin 2\n",
      "Summary: This gene encodes a classical cadherin and member of the cadherin superfamily. Alternative splicing results in multiple transcript variants, at least one of which encodes a preproprotein is proteolytically processed to generate a calcium-dependent cell adhesion molecule and glycoprotein. This protein plays a role in the establishment of left-right asymmetry, development of the nervous system and the formation of cartilage and bone. [provided by RefSeq, Nov 2015].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - cell morphogenesis\n",
      "    - type B pancreatic cell development\n",
      "    - cell-cell junction assembly\n",
      "    - cell-cell junction assembly\n",
      "    - cell adhesion\n",
      "    - homophilic cell adhesion via plasma membrane adhesion molecules\n",
      "    - heterophilic cell-cell adhesion via plasma membrane cell adhesion molecules\n",
      "    - synapse assembly\n",
      "    - glial cell differentiation\n",
      "    - neural crest cell development\n",
      "    - calcium-dependent cell-cell adhesion via plasma membrane cell adhesion molecules\n",
      "    - cell migration\n",
      "    - cerebral cortex development\n",
      "    - adherens junction organization\n",
      "    - detection of muscle stretch\n",
      "    - positive regulation of MAPK cascade\n",
      "    - cell-cell adhesion mediated by cadherin\n",
      "    - cell-cell adhesion mediated by cadherin\n",
      "    - blood vessel morphogenesis\n",
      "    - brain morphogenesis\n",
      "    - homeostasis of number of cells\n",
      "    - striated muscle cell differentiation\n",
      "    - radial glial cell differentiation\n",
      "    - neuroepithelial cell differentiation\n",
      "    - regulation of oligodendrocyte progenitor proliferation\n",
      "    - protein localization to plasma membrane\n",
      "    - negative regulation of canonical Wnt signaling pathway\n",
      "    - mesenchymal cell migration\n",
      "    - synaptic vesicle clustering\n",
      "    - neuroligin clustering involved in postsynaptic membrane assembly\n",
      "    - neuronal stem cell population maintenance\n",
      "    - cell-cell adhesion\n",
      "    - cell-cell adhesion\n",
      "    - regulation of postsynaptic density protein 95 clustering\n",
      "    - positive regulation of synaptic vesicle clustering\n",
      "  CC:\n",
      "    - cytoplasm\n",
      "    - cytoplasm\n",
      "    - endoplasmic reticulum lumen\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - cell-cell junction\n",
      "    - adherens junction\n",
      "    - adherens junction\n",
      "    - fascia adherens\n",
      "    - focal adhesion\n",
      "    - cell surface\n",
      "    - intercalated disc\n",
      "    - intercalated disc\n",
      "    - basolateral plasma membrane\n",
      "    - apical plasma membrane\n",
      "    - apicolateral plasma membrane\n",
      "    - catenin complex\n",
      "    - catenin complex\n",
      "    - lamellipodium\n",
      "    - lamellipodium\n",
      "    - cell junction\n",
      "    - desmosome\n",
      "    - cortical actin cytoskeleton\n",
      "    - sarcolemma\n",
      "    - neuron projection\n",
      "    - plasma membrane raft\n",
      "    - apical part of cell\n",
      "    - collagen-containing extracellular matrix\n",
      "    - presynapse\n",
      "  MF:\n",
      "    - RNA binding\n",
      "    - calcium ion binding\n",
      "    - protein binding\n",
      "    - beta-catenin binding\n",
      "    - beta-catenin binding\n",
      "    - beta-catenin binding\n",
      "    - protein kinase binding\n",
      "    - protein phosphatase binding\n",
      "    - identical protein binding\n",
      "    - alpha-catenin binding\n",
      "    - gamma-catenin binding\n",
      "    - cadherin binding\n",
      "UniProt: {'Swiss-Prot': 'P19022', 'TrEMBL': ['A0A6Q8PHJ8', 'C9J126', 'A0A6Q8PG20', 'A0A6Q8PF58', 'C9J8J8', 'C9JMH2']}\n",
      "\n",
      "Gene: DOK2\n",
      "Symbol: DOK2\n",
      "Name: docking protein 2\n",
      "Summary: The protein encoded by this gene is constitutively tyrosine phosphorylated in hematopoietic progenitors isolated from chronic myelogenous leukemia (CML) patients in the chronic phase. It may be a critical substrate for p210(bcr/abl), a chimeric protein whose presence is associated with CML. This encoded protein binds p120 (RasGAP) from CML cells. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - signal transduction\n",
      "    - cell surface receptor signaling pathway\n",
      "    - cell surface receptor protein tyrosine kinase signaling pathway\n",
      "    - Ras protein signal transduction\n",
      "  CC:\n",
      "    - cytoplasm\n",
      "    - cytosol\n",
      "    - plasma membrane\n",
      "  MF:\n",
      "    - transmembrane receptor protein tyrosine kinase adaptor activity\n",
      "    - protein binding\n",
      "UniProt: {'Swiss-Prot': 'O60496', 'TrEMBL': ['G3V114', 'B4DKQ6', 'E5RIG9', 'E5RJ11']}\n",
      "\n",
      "Gene: EWSR1\n",
      "Symbol: EWSR1\n",
      "Name: EWS RNA binding protein 1\n",
      "Summary: This gene encodes a multifunctional protein that is involved in various cellular processes, including gene expression, cell signaling, and RNA processing and transport. The protein includes an N-terminal transcriptional activation domain and a C-terminal RNA-binding domain. Chromosomal translocations between this gene and various genes encoding transcription factors result in the production of chimeric proteins that are involved in tumorigenesis. These chimeric proteins usually consist of the N-terminal transcriptional activation domain of this protein fused to the C-terminal DNA-binding domain of the transcription factor protein. Mutations in this gene, specifically a t(11;22)(q24;q12) translocation, are known to cause Ewing sarcoma as well as neuroectodermal and various other tumors. Alternative splicing of this gene results in multiple transcript variants. Related pseudogenes have been identified on chromosomes 1 and 14. [provided by RefSeq, Jul 2009].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - regulation of DNA-templated transcription\n",
      "  CC:\n",
      "    - nucleus\n",
      "    - nucleoplasm\n",
      "    - nucleolus\n",
      "    - cytoplasm\n",
      "    - plasma membrane\n",
      "  MF:\n",
      "    - transcription coregulator activity\n",
      "    - RNA binding\n",
      "    - RNA binding\n",
      "    - protein binding\n",
      "    - calmodulin binding\n",
      "    - zinc ion binding\n",
      "    - identical protein binding\n",
      "UniProt: {'Swiss-Prot': 'Q01844', 'TrEMBL': ['B0QYJ3', 'C9JGE3', 'B0QYJ5', 'H7BY36', 'F8WC90', 'B0QYJ6', 'B0QYJ4', 'B0QYJ7', 'B0QYK0', 'A0A0D9SFL3']}\n",
      "\n",
      "Gene: CDK6\n",
      "Symbol: CDK6\n",
      "Name: cyclin dependent kinase 6\n",
      "Summary: The protein encoded by this gene is a member of the CMGC family of serine/threonine protein kinases. This kinase is a catalytic subunit of the protein kinase complex that is important for cell cycle G1 phase progression and G1/S transition. The activity of this kinase first appears in mid-G1 phase, which is controlled by the regulatory subunits including D-type cyclins and members of INK4 family of CDK inhibitors. This kinase, as well as CDK4, has been shown to phosphorylate, and thus regulate the activity of, tumor suppressor protein Rb. Altered expression of this gene has been observed in multiple human cancers. A mutation in this gene resulting in reduced cell proliferation, and impaired cell motility and polarity, and has been identified in patients with primary microcephaly. [provided by RefSeq, Aug 2017].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - G1/S transition of mitotic cell cycle\n",
      "    - G1/S transition of mitotic cell cycle\n",
      "    - G1/S transition of mitotic cell cycle\n",
      "    - G1/S transition of mitotic cell cycle\n",
      "    - negative regulation of transcription by RNA polymerase II\n",
      "    - positive regulation of cell-matrix adhesion\n",
      "    - type B pancreatic cell development\n",
      "    - signal transduction\n",
      "    - Notch signaling pathway\n",
      "    - negative regulation of cell population proliferation\n",
      "    - response to virus\n",
      "    - regulation of G2/M transition of mitotic cell cycle\n",
      "    - regulation of gene expression\n",
      "    - regulation of gene expression\n",
      "    - regulation of gene expression\n",
      "    - positive regulation of gene expression\n",
      "    - astrocyte development\n",
      "    - dentate gyrus development\n",
      "    - lateral ventricle development\n",
      "    - T cell differentiation in thymus\n",
      "    - gliogenesis\n",
      "    - cell dedifferentiation\n",
      "    - negative regulation of cell differentiation\n",
      "    - negative regulation of myeloid cell differentiation\n",
      "    - regulation of erythrocyte differentiation\n",
      "    - negative regulation of monocyte differentiation\n",
      "    - negative regulation of osteoblast differentiation\n",
      "    - negative regulation of cell cycle\n",
      "    - positive regulation of fibroblast proliferation\n",
      "    - generation of neurons\n",
      "    - negative regulation of epithelial cell proliferation\n",
      "    - cell division\n",
      "    - regulation of cell cycle\n",
      "    - hematopoietic stem cell differentiation\n",
      "    - regulation of hematopoietic stem cell differentiation\n",
      "    - regulation of cell motility\n",
      "    - negative regulation of cellular senescence\n",
      "  CC:\n",
      "    - cyclin-dependent protein kinase holoenzyme complex\n",
      "    - cyclin-dependent protein kinase holoenzyme complex\n",
      "    - ruffle\n",
      "    - nucleus\n",
      "    - nucleus\n",
      "    - nucleoplasm\n",
      "    - nucleoplasm\n",
      "    - cytoplasm\n",
      "    - cytoplasm\n",
      "    - centrosome\n",
      "    - cytosol\n",
      "    - cytosol\n",
      "    - cyclin D1-CDK6 complex\n",
      "    - cyclin D2-CDK6 complex\n",
      "    - cyclin D3-CDK6 complex\n",
      "  MF:\n",
      "    - cyclin-dependent protein serine/threonine kinase activity\n",
      "    - cyclin-dependent protein serine/threonine kinase activity\n",
      "    - protein binding\n",
      "    - ATP binding\n",
      "    - cyclin binding\n",
      "    - cyclin binding\n",
      "    - FBXO family protein binding\n",
      "    - protein serine kinase activity\n",
      "UniProt: {'Swiss-Prot': 'Q00534'}\n",
      "\n",
      "Gene: BAMBI\n",
      "Symbol: BAMBI\n",
      "Name: BMP and activin membrane bound inhibitor\n",
      "Summary: This gene encodes a transmembrane glycoprotein related to the type I receptors of the transforming growth factor-beta (TGF-beta) family, whose members play important roles in signal transduction in many developmental and pathological processes. The encoded protein however is a pseudoreceptor, lacking an intracellular serine/threonine kinase domain required for signaling. Similar proteins in frog, mouse and zebrafish function as negative regulators of TGF-beta, which has led to the suggestion that the encoded protein may function to limit the signaling range of the TGF-beta family during early embryogenesis. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - transforming growth factor beta receptor signaling pathway\n",
      "    - positive regulation of cell population proliferation\n",
      "    - regulation of cell shape\n",
      "    - positive regulation of epithelial to mesenchymal transition\n",
      "    - cell migration\n",
      "    - negative regulation of transforming growth factor beta receptor signaling pathway\n",
      "    - negative regulation of transforming growth factor beta receptor signaling pathway\n",
      "    - negative regulation of BMP signaling pathway\n",
      "    - positive regulation of protein binding\n",
      "    - negative regulation of osteoblast differentiation\n",
      "    - positive regulation of DNA-templated transcription\n",
      "    - positive regulation of canonical Wnt signaling pathway\n",
      "    - positive regulation of canonical Wnt signaling pathway\n",
      "  CC:\n",
      "    - cytoplasm\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "  MF:\n",
      "    - frizzled binding\n",
      "    - frizzled binding\n",
      "    - type II transforming growth factor beta receptor binding\n",
      "    - protein binding\n",
      "UniProt: {'Swiss-Prot': 'Q13145'}\n",
      "\n",
      "Gene: SLC6A3\n",
      "Symbol: SLC6A3\n",
      "Name: solute carrier family 6 member 3\n",
      "Summary: This gene encodes a dopamine transporter which is a member of the sodium- and chloride-dependent neurotransmitter transporter family. The 3' UTR of this gene contains a 40 bp tandem repeat, referred to as a variable number tandem repeat or VNTR, which can be present in 3 to 11 copies. Variation in the number of repeats is associated with idiopathic epilepsy, attention-deficit hyperactivity disorder, dependence on alcohol and cocaine, susceptibility to Parkinson disease and protection against nicotine dependence.[provided by RefSeq, Nov 2009].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - neurotransmitter transport\n",
      "    - amino acid transport\n",
      "    - lactation\n",
      "    - sensory perception of smell\n",
      "    - locomotory behavior\n",
      "    - response to xenobiotic stimulus\n",
      "    - response to iron ion\n",
      "    - monoamine transport\n",
      "    - dopamine transport\n",
      "    - adenohypophysis development\n",
      "    - response to nicotine\n",
      "    - sodium ion transmembrane transport\n",
      "    - positive regulation of multicellular organism growth\n",
      "    - regulation of dopamine metabolic process\n",
      "    - response to cocaine\n",
      "    - dopamine biosynthetic process\n",
      "    - dopamine catabolic process\n",
      "    - response to ethanol\n",
      "    - cognition\n",
      "    - dopamine uptake involved in synaptic transmission\n",
      "    - response to cAMP\n",
      "    - norepinephrine uptake\n",
      "    - prepulse inhibition\n",
      "    - dopamine uptake\n",
      "    - dopamine uptake\n",
      "    - hyaloid vascular plexus regression\n",
      "  CC:\n",
      "    - cytoplasm\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - cell surface\n",
      "    - membrane\n",
      "    - flotillin complex\n",
      "    - axon\n",
      "    - presynaptic membrane\n",
      "    - neuron projection\n",
      "    - neuronal cell body\n",
      "    - axon terminus\n",
      "    - membrane raft\n",
      "    - postsynaptic membrane\n",
      "    - dopaminergic synapse\n",
      "  MF:\n",
      "    - protease binding\n",
      "    - signaling receptor binding\n",
      "    - neurotransmitter transmembrane transporter activity\n",
      "    - dopamine:sodium symporter activity\n",
      "    - dopamine:sodium symporter activity\n",
      "    - norepinephrine:sodium symporter activity\n",
      "    - protein binding\n",
      "    - monoamine transmembrane transporter activity\n",
      "    - monoamine transmembrane transporter activity\n",
      "    - monoamine transmembrane transporter activity\n",
      "    - dopamine binding\n",
      "    - amine binding\n",
      "    - protein-containing complex binding\n",
      "    - metal ion binding\n",
      "    - protein phosphatase 2A binding\n",
      "    - heterocyclic compound binding\n",
      "UniProt: {'Swiss-Prot': 'Q01959', 'TrEMBL': ['A0AAQ5BGS7', 'A0AAQ5BGN6', 'A0AAQ5BGQ7']}\n",
      "\n",
      "Gene: IL1R2\n",
      "Symbol: IL1R2\n",
      "Name: interleukin 1 receptor type 2\n",
      "Summary: The protein encoded by this gene is a cytokine receptor that belongs to the interleukin 1 receptor family. This protein binds interleukin alpha (IL1A), interleukin beta (IL1B), and interleukin 1 receptor, type I(IL1R1/IL1RA), and acts as a decoy receptor that inhibits the activity of its ligands. Interleukin 4 (IL4) is reported to antagonize the activity of interleukin 1 by inducing the expression and release of this cytokine. This gene and three other genes form a cytokine receptor gene cluster on chromosome 2q12. Alternative splicing results in multiple transcript variants and protein isoforms. Alternative splicing produces both membrane-bound and soluble proteins. A soluble protein is also produced by proteolytic cleavage. [provided by RefSeq, May 2012].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - immune response\n",
      "    - cell surface receptor signaling pathway\n",
      "    - negative regulation of protein processing\n",
      "    - protein processing\n",
      "    - cytokine-mediated signaling pathway\n",
      "    - negative regulation of interleukin-1 alpha production\n",
      "    - negative regulation of cytokine production involved in inflammatory response\n",
      "    - negative regulation of interleukin-1-mediated signaling pathway\n",
      "  CC:\n",
      "    - extracellular region\n",
      "    - cytoplasm\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - cell surface\n",
      "  MF:\n",
      "    - interleukin-1 receptor activity\n",
      "    - interleukin-1, type II, blocking receptor activity\n",
      "    - protein binding\n",
      "    - interleukin-1 binding\n",
      "UniProt: {'Swiss-Prot': 'P27930', 'TrEMBL': 'C9JNR0'}\n",
      "\n",
      "Gene: IL1A\n",
      "Symbol: IL1A\n",
      "Name: interleukin 1 alpha\n",
      "Summary: The protein encoded by this gene is a member of the interleukin 1 cytokine family. This cytokine is a pleiotropic cytokine involved in various immune responses, inflammatory processes, and hematopoiesis. This cytokine is produced by monocytes and macrophages as a proprotein, which is proteolytically processed and released in response to cell injury, and thus induces apoptosis. This gene and eight other interleukin 1 family genes form a cytokine gene cluster on chromosome 2. It has been suggested that the polymorphism of these genes is associated with rheumatoid arthritis and Alzheimer's disease. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - fever generation\n",
      "    - response to hypoxia\n",
      "    - positive regulation of cytokine production\n",
      "    - connective tissue replacement involved in inflammatory response wound healing\n",
      "    - intracellular sodium ion homeostasis\n",
      "    - apoptotic process\n",
      "    - inflammatory response\n",
      "    - immune response\n",
      "    - spermatogenesis\n",
      "    - heart development\n",
      "    - negative regulation of cell population proliferation\n",
      "    - response to ozone\n",
      "    - response to gamma radiation\n",
      "    - positive regulation of vascular endothelial growth factor production\n",
      "    - positive regulation of gene expression\n",
      "    - positive regulation of steroid biosynthetic process\n",
      "    - cytokine-mediated signaling pathway\n",
      "    - cytokine-mediated signaling pathway\n",
      "    - keratinization\n",
      "    - positive regulation of prostaglandin secretion\n",
      "    - positive regulation of interleukin-2 production\n",
      "    - positive regulation of interleukin-2 production\n",
      "    - positive regulation of interleukin-6 production\n",
      "    - positive regulation of tumor necrosis factor production\n",
      "    - positive regulation of stress-activated MAPK cascade\n",
      "    - regulation of actin cytoskeleton organization\n",
      "    - positive regulation of immature T cell proliferation in thymus\n",
      "    - response to L-ascorbic acid\n",
      "    - cellular response to heat\n",
      "    - ectopic germ cell programmed cell death\n",
      "    - positive regulation of canonical NF-kappaB signal transduction\n",
      "    - positive regulation of angiogenesis\n",
      "    - positive regulation of mitotic nuclear division\n",
      "    - positive regulation of transcription by RNA polymerase II\n",
      "    - positive regulation of JNK cascade\n",
      "    - response to copper ion\n",
      "    - positive regulation of protein secretion\n",
      "    - positive regulation of cell division\n",
      "    - regulation of ERK1 and ERK2 cascade\n",
      "    - positive regulation of ERK1 and ERK2 cascade\n",
      "    - cellular response to lipopolysaccharide\n",
      "    - extrinsic apoptotic signaling pathway in absence of ligand\n",
      "    - positive regulation of neutrophil migration\n",
      "    - negative regulation of establishment of Sertoli cell barrier\n",
      "    - negative regulation of extrinsic apoptotic signaling pathway in absence of ligand\n",
      "  CC:\n",
      "    - extracellular region\n",
      "    - extracellular space\n",
      "    - extracellular space\n",
      "    - extracellular space\n",
      "    - nucleus\n",
      "    - cytosol\n",
      "    - cell surface\n",
      "  MF:\n",
      "    - cytokine activity\n",
      "    - cytokine activity\n",
      "    - interleukin-1 receptor binding\n",
      "    - copper ion binding\n",
      "    - protein binding\n",
      "UniProt: {'Swiss-Prot': 'P01583'}\n",
      "\n",
      "Gene: GNAQ\n",
      "Symbol: GNAQ\n",
      "Name: G protein subunit alpha q\n",
      "Summary: This locus encodes a guanine nucleotide-binding protein. The encoded protein, an alpha subunit in the Gq class, couples a seven-transmembrane domain receptor to activation of phospolipase C-beta. Mutations at this locus have been associated with problems in platelet activation and aggregation. A related pseudogene exists on chromosome 2.[provided by RefSeq, Nov 2010].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - negative regulation of protein kinase activity\n",
      "    - adenylate cyclase-activating G protein-coupled receptor signaling pathway\n",
      "    - phospholipase C-activating G protein-coupled receptor signaling pathway\n",
      "    - phospholipase C-activating G protein-coupled glutamate receptor signaling pathway\n",
      "    - phospholipase C-activating serotonin receptor signaling pathway\n",
      "    - G protein-coupled acetylcholine receptor signaling pathway\n",
      "    - glutamate receptor signaling pathway\n",
      "    - neuropeptide signaling pathway\n",
      "    - blood coagulation\n",
      "    - phototransduction, visible light\n",
      "    - entrainment of circadian clock\n",
      "    - regulation of platelet activation\n",
      "    - response to prostaglandin E\n",
      "    - protein stabilization\n",
      "    - regulation of canonical Wnt signaling pathway\n",
      "  CC:\n",
      "    - photoreceptor outer segment\n",
      "    - cytoplasm\n",
      "    - lysosomal membrane\n",
      "    - Golgi apparatus\n",
      "    - heterotrimeric G-protein complex\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - nuclear membrane\n",
      "    - extracellular exosome\n",
      "    - postsynaptic cytosol\n",
      "  MF:\n",
      "    - G protein-coupled receptor binding\n",
      "    - GTPase activity\n",
      "    - GTPase activity\n",
      "    - G protein activity\n",
      "    - GTPase activator activity\n",
      "    - GTPase activator activity\n",
      "    - protein binding\n",
      "    - GTP binding\n",
      "    - enzyme regulator activity\n",
      "    - G-protein beta/gamma-subunit complex binding\n",
      "    - metal ion binding\n",
      "UniProt: {'Swiss-Prot': 'P50148', 'TrEMBL': ['B1AM21', 'A0A024R240']}\n",
      "\n",
      "Gene: PLD1\n",
      "Symbol: PLD1\n",
      "Name: phospholipase D1\n",
      "Summary: This gene encodes a phosphatidylcholine-specific phospholipase which catalyzes the hydrolysis of phosphatidylcholine in order to yield phosphatidic acid and choline. The enzyme may play a role in signal transduction and subcellular trafficking. Alternative splicing results in multiple transcript variants with both catalytic and regulatory properties. [provided by RefSeq, Sep 2011].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - phosphatidic acid biosynthetic process\n",
      "    - chemotaxis\n",
      "    - small GTPase-mediated signal transduction\n",
      "    - Ras protein signal transduction\n",
      "    - phospholipid catabolic process\n",
      "    - cellular response to nutrient\n",
      "    - regulation of microvillus assembly\n",
      "    - regulation of microvillus assembly\n",
      "    - positive regulation of translation\n",
      "    - regulation of vesicle-mediated transport\n",
      "    - regulation of synaptic vesicle cycle\n",
      "    - regulation of synaptic vesicle cycle\n",
      "  CC:\n",
      "    - Golgi membrane\n",
      "    - lysosomal membrane\n",
      "    - endosome\n",
      "    - endoplasmic reticulum membrane\n",
      "    - Golgi apparatus\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - membrane\n",
      "    - apical plasma membrane\n",
      "    - endocytic vesicle\n",
      "    - cytoplasmic vesicle\n",
      "    - late endosome membrane\n",
      "    - specific granule membrane\n",
      "    - perinuclear region of cytoplasm\n",
      "    - tertiary granule membrane\n",
      "    - cholinergic synapse\n",
      "    - cholinergic synapse\n",
      "  MF:\n",
      "    - phospholipase D activity\n",
      "    - phospholipase D activity\n",
      "    - phospholipase D activity\n",
      "    - protein binding\n",
      "    - phosphatidylinositol binding\n",
      "UniProt: {'Swiss-Prot': 'Q13393', 'TrEMBL': ['F8WBV7', 'Q59EA4', 'H7C0L3', 'C9IY79']}\n",
      "\n",
      "Gene: WT1\n",
      "Symbol: WT1\n",
      "Name: WT1 transcription factor\n",
      "Summary: This gene encodes a transcription factor that contains four zinc-finger motifs at the C-terminus and a proline/glutamine-rich DNA-binding domain at the N-terminus. It has an essential role in the normal development of the urogenital system, and it is mutated in a small subset of patients with Wilms tumor. This gene exhibits complex tissue-specific and polymorphic imprinting pattern, with biallelic, and monoallelic expression from the maternal and paternal alleles in different tissues. Multiple transcript variants have been described. In several variants, there is evidence for the use of a non-AUG (CUG) translation initiation codon upstream of, and in-frame with the first AUG. Authors of PMID:7926762 also provide evidence that WT1 mRNA undergoes RNA editing in human and rat, and that this process is tissue-restricted and developmentally regulated. [provided by RefSeq, Mar 2015].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - negative regulation of transcription by RNA polymerase II\n",
      "    - negative regulation of transcription by RNA polymerase II\n",
      "    - vasculogenesis\n",
      "    - ureteric bud development\n",
      "    - branching involved in ureteric bud morphogenesis\n",
      "    - kidney development\n",
      "    - regulation of animal organ formation\n",
      "    - regulation of DNA-templated transcription\n",
      "    - regulation of DNA-templated transcription\n",
      "    - regulation of transcription by RNA polymerase II\n",
      "    - regulation of transcription by RNA polymerase II\n",
      "    - germ cell development\n",
      "    - thorax and anterior abdomen determination\n",
      "    - heart development\n",
      "    - sex determination\n",
      "    - negative regulation of cell population proliferation\n",
      "    - negative regulation of cell population proliferation\n",
      "    - RNA splicing\n",
      "    - gonad development\n",
      "    - male gonad development\n",
      "    - tissue development\n",
      "    - positive regulation of gene expression\n",
      "    - negative regulation of translation\n",
      "    - negative regulation of cell growth\n",
      "    - adrenal gland development\n",
      "    - male genitalia development\n",
      "    - epithelial cell differentiation\n",
      "    - glomerulus development\n",
      "    - glomerular basement membrane development\n",
      "    - adrenal cortex formation\n",
      "    - camera-type eye development\n",
      "    - positive regulation of apoptotic process\n",
      "    - negative regulation of apoptotic process\n",
      "    - negative regulation of apoptotic process\n",
      "    - negative regulation of gene expression via chromosomal CpG island methylation\n",
      "    - negative regulation of DNA-templated transcription\n",
      "    - positive regulation of DNA-templated transcription\n",
      "    - positive regulation of DNA-templated transcription\n",
      "    - positive regulation of DNA-templated transcription\n",
      "    - positive regulation of transcription by RNA polymerase II\n",
      "    - mesenchymal to epithelial transition\n",
      "    - positive regulation of heart growth\n",
      "    - diaphragm development\n",
      "    - cardiac muscle cell fate commitment\n",
      "    - visceral serous pericardium development\n",
      "    - cellular response to cAMP\n",
      "    - cellular response to gonadotropin stimulus\n",
      "    - metanephric mesenchyme development\n",
      "    - podocyte differentiation\n",
      "    - posterior mesonephric tubule development\n",
      "    - metanephric epithelium development\n",
      "    - metanephric S-shaped body morphogenesis\n",
      "    - negative regulation of metanephric glomerular mesangial cell proliferation\n",
      "    - positive regulation of miRNA transcription\n",
      "    - positive regulation of male gonad development\n",
      "    - negative regulation of female gonad development\n",
      "    - positive regulation of metanephric ureteric bud development\n",
      "  CC:\n",
      "    - nucleus\n",
      "    - nucleoplasm\n",
      "    - nucleolus\n",
      "    - cytoplasm\n",
      "    - cytosol\n",
      "    - nuclear speck\n",
      "  MF:\n",
      "    - transcription cis-regulatory region binding\n",
      "    - RNA polymerase II cis-regulatory region sequence-specific DNA binding\n",
      "    - RNA polymerase II cis-regulatory region sequence-specific DNA binding\n",
      "    - DNA-binding transcription factor activity, RNA polymerase II-specific\n",
      "    - DNA-binding transcription activator activity, RNA polymerase II-specific\n",
      "    - DNA-binding transcription activator activity, RNA polymerase II-specific\n",
      "    - DNA-binding transcription factor activity\n",
      "    - DNA-binding transcription factor activity\n",
      "    - RNA binding\n",
      "    - protein binding\n",
      "    - zinc ion binding\n",
      "    - double-stranded methylated DNA binding\n",
      "    - sequence-specific DNA binding\n",
      "    - hemi-methylated DNA-binding\n",
      "    - C2H2 zinc finger domain binding\n",
      "UniProt: {'Swiss-Prot': 'P19544', 'TrEMBL': ['A0A494C0I9', 'Q6PI38']}\n",
      "\n",
      "Gene: EREG\n",
      "Symbol: EREG\n",
      "Name: epiregulin\n",
      "Summary: This gene encodes a secreted peptide hormone and member of the epidermal growth factor (EGF) family of proteins. The encoded protein is a ligand of the epidermal growth factor receptor (EGFR) and the structurally related erb-b2 receptor tyrosine kinase 4 (ERBB4). The encoded protein may be involved in a wide range of biological processes including inflammation, wound healing, oocyte maturation, and cell proliferation. Additionally, the encoded protein may promote the progression of cancers of various human tissues. [provided by RefSeq, Jul 2015].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - angiogenesis\n",
      "    - ovarian cumulus expansion\n",
      "    - oocyte maturation\n",
      "    - positive regulation of cytokine production\n",
      "    - positive regulation of cytokine production\n",
      "    - female meiotic nuclear division\n",
      "    - epidermal growth factor receptor signaling pathway\n",
      "    - epidermal growth factor receptor signaling pathway\n",
      "    - epidermal growth factor receptor signaling pathway\n",
      "    - cell-cell signaling\n",
      "    - positive regulation of cell population proliferation\n",
      "    - positive regulation of cell population proliferation\n",
      "    - positive regulation of cell population proliferation\n",
      "    - negative regulation of cell population proliferation\n",
      "    - mRNA transcription\n",
      "    - anatomical structure morphogenesis\n",
      "    - animal organ morphogenesis\n",
      "    - cytokine-mediated signaling pathway\n",
      "    - keratinocyte differentiation\n",
      "    - ovulation\n",
      "    - positive regulation of interleukin-6 production\n",
      "    - ERBB2-EGFR signaling pathway\n",
      "    - ERBB2-ERBB4 signaling pathway\n",
      "    - ERBB4-ERBB4 signaling pathway\n",
      "    - wound healing\n",
      "    - positive regulation of phosphorylation\n",
      "    - luteinizing hormone signaling pathway\n",
      "    - response to peptide hormone\n",
      "    - keratinocyte proliferation\n",
      "    - positive regulation of innate immune response\n",
      "    - positive regulation of DNA replication\n",
      "    - positive regulation of DNA replication\n",
      "    - positive regulation of mitotic nuclear division\n",
      "    - positive regulation of mitotic nuclear division\n",
      "    - positive regulation of protein kinase activity\n",
      "    - negative regulation of DNA-templated transcription\n",
      "    - positive regulation of fibroblast proliferation\n",
      "    - primary follicle stage\n",
      "    - positive regulation of smooth muscle cell proliferation\n",
      "    - negative regulation of epithelial cell proliferation\n",
      "    - negative regulation of smooth muscle cell differentiation\n",
      "    - positive regulation of cell division\n",
      "  CC:\n",
      "    - extracellular region\n",
      "    - extracellular space\n",
      "    - extracellular space\n",
      "    - extracellular space\n",
      "    - plasma membrane\n",
      "    - clathrin-coated endocytic vesicle membrane\n",
      "  MF:\n",
      "    - epidermal growth factor receptor binding\n",
      "    - epidermal growth factor receptor binding\n",
      "    - epidermal growth factor receptor binding\n",
      "    - protein binding\n",
      "    - growth factor activity\n",
      "    - transmembrane receptor protein tyrosine kinase activator activity\n",
      "    - receptor ligand activity\n",
      "UniProt: {'Swiss-Prot': 'O14944'}\n",
      "\n",
      "Gene: SMARCB1\n",
      "Symbol: SMARCB1\n",
      "Name: SWI/SNF related BAF chromatin remodeling complex subunit B1\n",
      "Summary: The protein encoded by this gene is part of a complex that relieves repressive chromatin structures, allowing the transcriptional machinery to access its targets more effectively. The encoded nuclear protein may also bind to and enhance the DNA joining activity of HIV-1 integrase. This gene has been found to be a tumor suppressor, and mutations in it have been associated with malignant rhabdoid tumors. Alternatively spliced transcript variants have been found for this gene. [provided by RefSeq, Dec 2015].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - RNA polymerase I preinitiation complex assembly\n",
      "    - blastocyst hatching\n",
      "    - nucleosome disassembly\n",
      "    - chromatin remodeling\n",
      "    - chromatin remodeling\n",
      "    - chromatin remodeling\n",
      "    - chromatin remodeling\n",
      "    - chromatin remodeling\n",
      "    - regulation of transcription by RNA polymerase II\n",
      "    - regulation of transcription by RNA polymerase II\n",
      "    - nervous system development\n",
      "    - negative regulation of cell population proliferation\n",
      "    - DNA integration\n",
      "    - regulation of mitotic metaphase/anaphase transition\n",
      "    - single stranded viral RNA replication via double stranded DNA intermediate\n",
      "    - positive regulation by host of viral transcription\n",
      "    - positive regulation of T cell differentiation\n",
      "    - positive regulation of cell differentiation\n",
      "    - positive regulation of myoblast differentiation\n",
      "    - transcription initiation-coupled chromatin remodeling\n",
      "    - positive regulation of transcription by RNA polymerase II\n",
      "    - positive regulation of DNA-binding transcription factor activity\n",
      "    - regulation of G0 to G1 transition\n",
      "    - hepatocyte differentiation\n",
      "    - positive regulation of transcription of nucleolar large rRNA by RNA polymerase I\n",
      "    - positive regulation of transcription of nucleolar large rRNA by RNA polymerase I\n",
      "    - positive regulation of stem cell population maintenance\n",
      "    - positive regulation of glucose mediated signaling pathway\n",
      "    - regulation of G1/S transition of mitotic cell cycle\n",
      "    - positive regulation of double-strand break repair\n",
      "    - regulation of nucleotide-excision repair\n",
      "  CC:\n",
      "    - nuclear chromosome\n",
      "    - kinetochore\n",
      "    - chromatin\n",
      "    - chromatin\n",
      "    - fibrillar center\n",
      "    - XY body\n",
      "    - nucleus\n",
      "    - nucleus\n",
      "    - nucleoplasm\n",
      "    - nucleoplasm\n",
      "    - nucleolus\n",
      "    - nuclear matrix\n",
      "    - SWI/SNF complex\n",
      "    - SWI/SNF complex\n",
      "    - RSC-type complex\n",
      "    - protein-containing complex\n",
      "    - brahma complex\n",
      "    - brahma complex\n",
      "    - germ cell nucleus\n",
      "    - intracellular membrane-bounded organelle\n",
      "    - npBAF complex\n",
      "    - npBAF complex\n",
      "    - npBAF complex\n",
      "    - nBAF complex\n",
      "    - nBAF complex\n",
      "    - nBAF complex\n",
      "    - bBAF complex\n",
      "  MF:\n",
      "    - RNA polymerase I core promoter sequence-specific DNA binding\n",
      "    - p53 binding\n",
      "    - DNA binding\n",
      "    - transcription coactivator activity\n",
      "    - transcription coactivator activity\n",
      "    - protein binding\n",
      "    - Tat protein binding\n",
      "    - nucleosomal DNA binding\n",
      "    - identical protein binding\n",
      "UniProt: {'Swiss-Prot': 'Q12824', 'TrEMBL': ['Q9H836', 'G5E975', 'A0A0G2JRV3', 'A0A2R8YDN6', 'C9JTA6', 'A0A2R8Y5N5', 'B5MCL5', 'A0A2R8Y5Z2', 'A0A0U1RRB8', 'A0A0G2JSE9', 'A0A0U1RQQ2', 'A0A2R8YEE8']}\n",
      "\n",
      "Gene: DAAM1\n",
      "Symbol: DAAM1\n",
      "Name: dishevelled associated activator of morphogenesis 1\n",
      "Summary: Cell motility, adhesion, cytokinesis, and other functions of the cell cortex are mediated by reorganization of the actin cytoskeleton and several formin homology (FH) proteins have been associated with these processes. The protein encoded by this gene contains two FH domains and belongs to a novel FH protein subfamily implicated in cell polarity. A key regulator of cytoskeletal architecture, the small GTPase Rho, is activated during development by Wnt/Fz signaling to control cell polarity and movement. The protein encoded by this gene is thought to function as a scaffolding protein for the Wnt-induced assembly of a disheveled (Dvl)-Rho complex. This protein also promotes the nucleation and elongation of new actin filaments and regulates cell growth through the stabilization of microtubules. Alternative splicing results in multiple transcript variants encoding distinct proteins. [provided by RefSeq, Jul 2012].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - Wnt signaling pathway, planar cell polarity pathway\n",
      "    - presynaptic actin cytoskeleton organization\n",
      "  CC:\n",
      "    - stress fiber\n",
      "    - stress fiber\n",
      "    - cytosol\n",
      "    - cytosol\n",
      "    - plasma membrane\n",
      "    - membrane\n",
      "    - motile cilium\n",
      "    - ciliary basal body\n",
      "    - perinuclear region of cytoplasm\n",
      "    - presynapse\n",
      "    - glutamatergic synapse\n",
      "  MF:\n",
      "    - actin binding\n",
      "    - protein binding\n",
      "    - small GTPase binding\n",
      "    - identical protein binding\n",
      "UniProt: {'Swiss-Prot': 'Q9Y4D1', 'TrEMBL': 'G3V275'}\n",
      "\n",
      "Gene: SDC1\n",
      "Symbol: SDC1\n",
      "Name: syndecan 1\n",
      "Summary: The protein encoded by this gene is a transmembrane (type I) heparan sulfate proteoglycan and is a member of the syndecan proteoglycan family. The syndecans mediate cell binding, cell signaling, and cytoskeletal organization and syndecan receptors are required for internalization of the HIV-1 tat protein. The syndecan-1 protein functions as an integral membrane protein and participates in cell proliferation, cell migration and cell-matrix interactions via its receptor for extracellular matrix proteins. Altered syndecan-1 expression has been detected in several different tumor types. While several transcript variants may exist for this gene, the full-length natures of only two have been described to date. These two represent the major variants of this gene and encode the same protein. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - ureteric bud development\n",
      "    - receptor-mediated endocytosis\n",
      "    - response to toxic substance\n",
      "    - cell migration\n",
      "    - response to hydrogen peroxide\n",
      "    - myoblast development\n",
      "    - response to glucocorticoid\n",
      "    - response to cAMP\n",
      "    - response to calcium ion\n",
      "    - striated muscle cell development\n",
      "    - Sertoli cell development\n",
      "    - canonical Wnt signaling pathway\n",
      "    - positive regulation of exosomal secretion\n",
      "    - positive regulation of extracellular exosome assembly\n",
      "  CC:\n",
      "    - Golgi lumen\n",
      "    - plasma membrane\n",
      "    - external side of plasma membrane\n",
      "    - cell surface\n",
      "    - cell surface\n",
      "    - protein-containing complex\n",
      "    - lysosomal lumen\n",
      "    - extracellular exosome\n",
      "  MF:\n",
      "    - protein binding\n",
      "    - cargo receptor activity\n",
      "    - identical protein binding\n",
      "UniProt: {'Swiss-Prot': 'P18827', 'TrEMBL': ['H7C1K4', 'F8WCX9', 'E9PHH3']}\n",
      "\n",
      "Gene: VEGFB\n",
      "Symbol: VEGFB\n",
      "Name: vascular endothelial growth factor B\n",
      "Summary: This gene encodes a member of the PDGF (platelet-derived growth factor)/VEGF (vascular endothelial growth factor) family. The VEGF family members regulate the formation of blood vessels and are involved in endothelial cell physiology. This member is a ligand for VEGFR-1 (vascular endothelial growth factor receptor 1) and NRP-1 (neuropilin-1). Studies in mice showed that this gene was co-expressed with nuclear-encoded mitochondrial genes and the encoded protein specifically controlled endothelial uptake of fatty acids. Alternatively spliced transcript variants encoding distinct isoforms have been identified. [provided by RefSeq, Sep 2011].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - response to hypoxia\n",
      "    - positive regulation of endothelial cell proliferation\n",
      "    - sprouting angiogenesis\n",
      "    - protein O-linked glycosylation\n",
      "    - positive regulation of cell population proliferation\n",
      "    - negative regulation of gene expression\n",
      "    - positive regulation of vascular endothelial growth factor receptor signaling pathway\n",
      "    - positive regulation of vascular wound healing\n",
      "    - vascular endothelial growth factor signaling pathway\n",
      "    - negative regulation of apoptotic process\n",
      "    - positive regulation of vascular permeability\n",
      "    - negative regulation of neuron apoptotic process\n",
      "    - vascular endothelial growth factor receptor signaling pathway\n",
      "    - positive regulation of peptidyl-tyrosine phosphorylation\n",
      "    - positive chemotaxis\n",
      "    - induction of positive chemotaxis\n",
      "    - induction of positive chemotaxis\n",
      "    - positive regulation of cell division\n",
      "    - positive regulation of phosphatidylinositol 3-kinase/protein kinase B signal transduction\n",
      "    - cardiac muscle contraction\n",
      "    - positive regulation of mast cell chemotaxis\n",
      "    - positive regulation of mast cell chemotaxis\n",
      "    - coronary vasculature development\n",
      "    - positive regulation of ERK1 and ERK2 cascade\n",
      "  CC:\n",
      "    - extracellular region\n",
      "    - extracellular space\n",
      "    - extracellular space\n",
      "    - membrane\n",
      "    - platelet alpha granule lumen\n",
      "  MF:\n",
      "    - vascular endothelial growth factor receptor binding\n",
      "    - protein binding\n",
      "    - growth factor activity\n",
      "    - heparin binding\n",
      "    - chemoattractant activity\n",
      "    - chemoattractant activity\n",
      "    - identical protein binding\n",
      "    - vascular endothelial growth factor receptor 1 binding\n",
      "UniProt: {'Swiss-Prot': 'P49765', 'TrEMBL': ['H0YGB6', 'Q7LAP4']}\n",
      "\n",
      "Gene: CSF1\n",
      "Symbol: CSF1\n",
      "Name: colony stimulating factor 1\n",
      "Summary: The protein encoded by this gene is a cytokine that controls the production, differentiation, and function of macrophages. The active form of the protein is found extracellularly as a disulfide-linked homodimer, and is thought to be produced by proteolytic cleavage of membrane-bound precursors. The encoded protein may be involved in development of the placenta. Alternate splicing results in multiple transcript variants. [provided by RefSeq, Sep 2011].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - neutrophil homeostasis\n",
      "    - positive regulation of cell-matrix adhesion\n",
      "    - osteoclast proliferation\n",
      "    - response to ischemia\n",
      "    - developmental process involved in reproduction\n",
      "    - inflammatory response\n",
      "    - cell surface receptor protein tyrosine kinase signaling pathway\n",
      "    - Ras protein signal transduction\n",
      "    - positive regulation of cell population proliferation\n",
      "    - positive regulation of cell population proliferation\n",
      "    - positive regulation of gene expression\n",
      "    - regulation of macrophage derived foam cell differentiation\n",
      "    - positive regulation of macrophage derived foam cell differentiation\n",
      "    - positive regulation of macrophage chemotaxis\n",
      "    - monocyte differentiation\n",
      "    - macrophage differentiation\n",
      "    - regulation of ossification\n",
      "    - osteoclast differentiation\n",
      "    - osteoclast differentiation\n",
      "    - positive regulation of cell migration\n",
      "    - positive regulation of mononuclear cell proliferation\n",
      "    - monocyte homeostasis\n",
      "    - macrophage colony-stimulating factor signaling pathway\n",
      "    - positive regulation of multicellular organism growth\n",
      "    - monocyte activation\n",
      "    - positive regulation of odontogenesis of dentin-containing tooth\n",
      "    - innate immune response\n",
      "    - positive regulation of macrophage differentiation\n",
      "    - positive regulation of macrophage differentiation\n",
      "    - positive regulation of monocyte differentiation\n",
      "    - positive regulation of osteoclast differentiation\n",
      "    - positive regulation of Ras protein signal transduction\n",
      "    - homeostasis of number of cells within a tissue\n",
      "    - positive regulation of protein metabolic process\n",
      "    - branching involved in mammary gland duct morphogenesis\n",
      "    - mammary gland fat development\n",
      "    - mammary duct terminal end bud growth\n",
      "    - microglial cell proliferation\n",
      "    - macrophage homeostasis\n",
      "    - myeloid leukocyte migration\n",
      "    - positive regulation of macrophage colony-stimulating factor signaling pathway\n",
      "    - positive regulation of microglial cell migration\n",
      "    - positive regulation of macrophage migration\n",
      "  CC:\n",
      "    - extracellular region\n",
      "    - extracellular space\n",
      "    - extracellular space\n",
      "    - endoplasmic reticulum lumen\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - membrane\n",
      "    - nuclear body\n",
      "    - perinuclear region of cytoplasm\n",
      "    - CSF1-CSF1R complex\n",
      "  MF:\n",
      "    - cytokine activity\n",
      "    - cytokine activity\n",
      "    - macrophage colony-stimulating factor receptor binding\n",
      "    - macrophage colony-stimulating factor receptor binding\n",
      "    - protein binding\n",
      "    - growth factor activity\n",
      "    - identical protein binding\n",
      "    - protein homodimerization activity\n",
      "UniProt: {'Swiss-Prot': 'P09603', 'TrEMBL': ['E9PQ08', 'E9PKP4', 'E9PJA2', 'H7BY18']}\n",
      "\n",
      "Gene: MAPK9\n",
      "Symbol: MAPK9\n",
      "Name: mitogen-activated protein kinase 9\n",
      "Summary: The protein encoded by this gene is a member of the MAP kinase family. MAP kinases act as an integration point for multiple biochemical signals, and are involved in a wide variety of cellular processes such as proliferation, differentiation, transcription regulation and development. This kinase targets specific transcription factors, and thus mediates immediate-early gene expression in response to various cell stimuli. It is most closely related to MAPK8, both of which are involved in UV radiation induced apoptosis, thought to be related to the cytochrome c-mediated cell death pathway. This gene and MAPK8 are also known as c-Jun N-terminal kinases. This kinase blocks the ubiquitination of tumor suppressor p53, and thus it increases the stability of p53 in nonstressed cells. Studies of this gene's mouse counterpart suggest a key role in T-cell differentiation. Several alternatively spliced transcript variants encoding distinct isoforms have been reported. [provided by RefSeq, Sep 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - protein phosphorylation\n",
      "    - JNK cascade\n",
      "    - JNK cascade\n",
      "    - positive regulation of gene expression\n",
      "    - positive regulation of macrophage derived foam cell differentiation\n",
      "    - positive regulation of protein ubiquitination\n",
      "    - positive regulation of proteasomal ubiquitin-dependent protein catabolic process\n",
      "    - cellular response to reactive oxygen species\n",
      "    - Fc-epsilon receptor signaling pathway\n",
      "    - regulation of circadian rhythm\n",
      "    - response to cadmium ion\n",
      "    - rhythmic process\n",
      "    - modulation of chemical synaptic transmission\n",
      "    - protein localization to tricellular tight junction\n",
      "    - positive regulation of podosome assembly\n",
      "    - cellular senescence\n",
      "    - inflammatory response to wounding\n",
      "    - apoptotic signaling pathway\n",
      "    - positive regulation of cytokine production involved in inflammatory response\n",
      "    - positive regulation of apoptotic signaling pathway\n",
      "  CC:\n",
      "    - nucleus\n",
      "    - nucleoplasm\n",
      "    - cytoplasm\n",
      "    - cytoplasm\n",
      "    - mitochondrion\n",
      "    - cytosol\n",
      "    - cytosol\n",
      "    - plasma membrane\n",
      "    - nuclear speck\n",
      "    - Schaffer collateral - CA1 synapse\n",
      "  MF:\n",
      "    - protein serine/threonine kinase activity\n",
      "    - JUN kinase activity\n",
      "    - JUN kinase activity\n",
      "    - JUN kinase activity\n",
      "    - protein serine/threonine/tyrosine kinase activity\n",
      "    - protein binding\n",
      "    - ATP binding\n",
      "    - protein serine kinase activity\n",
      "UniProt: {'Swiss-Prot': 'P45984', 'TrEMBL': ['D7R528', 'D7R525', 'E5RJ57', 'J3KNK1']}\n",
      "\n",
      "Gene: EPHA4\n",
      "Symbol: EPHA4\n",
      "Name: EPH receptor A4\n",
      "Summary: This gene belongs to the ephrin receptor subfamily of the protein-tyrosine kinase family. EPH and EPH-related receptors have been implicated in mediating developmental events, particularly in the nervous system. Receptors in the EPH subfamily typically have a single kinase domain and an extracellular region containing a Cys-rich domain and 2 fibronectin type III repeats. The ephrin receptors are divided into 2 groups based on the similarity of their extracellular domain sequences and their affinities for binding ephrin-A and ephrin-B ligands. Multiple transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Jan 2015].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - cell adhesion\n",
      "    - negative regulation of cell adhesion\n",
      "    - epidermal growth factor receptor signaling pathway\n",
      "    - axon guidance\n",
      "    - adult walking behavior\n",
      "    - motor neuron axon guidance\n",
      "    - positive regulation of cell population proliferation\n",
      "    - insulin receptor signaling pathway\n",
      "    - glial cell migration\n",
      "    - fibroblast growth factor receptor signaling pathway\n",
      "    - negative regulation of epithelial to mesenchymal transition\n",
      "    - negative regulation of neuron projection development\n",
      "    - negative regulation of translation\n",
      "    - peptidyl-tyrosine phosphorylation\n",
      "    - corticospinal tract morphogenesis\n",
      "    - positive regulation of cell migration\n",
      "    - negative regulation of cell migration\n",
      "    - brain-derived neurotrophic factor receptor signaling pathway\n",
      "    - adherens junction organization\n",
      "    - platelet-derived growth factor receptor-alpha signaling pathway\n",
      "    - platelet-derived growth factor receptor-beta signaling pathway\n",
      "    - vascular endothelial growth factor receptor-1 signaling pathway\n",
      "    - collagen-activated tyrosine kinase receptor signaling pathway\n",
      "    - vascular endothelial growth factor signaling pathway\n",
      "    - Kit signaling pathway\n",
      "    - macrophage colony-stimulating factor signaling pathway\n",
      "    - regulation of GTPase activity\n",
      "    - negative regulation of neuron apoptotic process\n",
      "    - positive regulation of cell adhesion\n",
      "    - protein autophosphorylation\n",
      "    - insulin-like growth factor receptor signaling pathway\n",
      "    - hepatocyte growth factor receptor signaling pathway\n",
      "    - ephrin receptor signaling pathway\n",
      "    - ephrin receptor signaling pathway\n",
      "    - ephrin receptor signaling pathway\n",
      "    - negative regulation of axon regeneration\n",
      "    - regulation of astrocyte differentiation\n",
      "    - regulation of axonogenesis\n",
      "    - positive regulation of dendrite morphogenesis\n",
      "    - protein stabilization\n",
      "    - innervation\n",
      "    - regulation of dendritic spine morphogenesis\n",
      "    - negative regulation of ERK1 and ERK2 cascade\n",
      "    - nephric duct morphogenesis\n",
      "    - cochlea development\n",
      "    - fasciculation of sensory neuron axon\n",
      "    - fasciculation of motor neuron axon\n",
      "    - neuron projection guidance\n",
      "    - synapse pruning\n",
      "    - neuron projection fasciculation\n",
      "    - negative regulation of cellular response to hypoxia\n",
      "    - negative regulation of long-term synaptic potentiation\n",
      "    - positive regulation of amyloid-beta formation\n",
      "    - positive regulation of intracellular signal transduction\n",
      "    - positive regulation of amyloid precursor protein catabolic process\n",
      "    - negative regulation of proteolysis involved in protein catabolic process\n",
      "    - cellular response to amyloid-beta\n",
      "    - regulation of modification of synaptic structure\n",
      "    - regulation of synapse pruning\n",
      "    - positive regulation of Rho guanyl-nucleotide exchange factor activity\n",
      "  CC:\n",
      "    - cytoplasm\n",
      "    - mitochondrial outer membrane\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - adherens junction\n",
      "    - cell surface\n",
      "    - filopodium\n",
      "    - axon\n",
      "    - dendrite\n",
      "    - dendrite\n",
      "    - neuromuscular junction\n",
      "    - early endosome membrane\n",
      "    - presynaptic membrane\n",
      "    - dendritic spine\n",
      "    - dendritic shaft\n",
      "    - perikaryon\n",
      "    - axon terminus\n",
      "    - axonal growth cone\n",
      "    - Schaffer collateral - CA1 synapse\n",
      "    - postsynaptic density membrane\n",
      "    - glutamatergic synapse\n",
      "  MF:\n",
      "    - amyloid-beta binding\n",
      "    - protein kinase activity\n",
      "    - protein tyrosine kinase activity\n",
      "    - GPI-linked ephrin receptor activity\n",
      "    - transmembrane-ephrin receptor activity\n",
      "    - transmembrane-ephrin receptor activity\n",
      "    - epidermal growth factor receptor activity\n",
      "    - fibroblast growth factor receptor activity\n",
      "    - hepatocyte growth factor receptor activity\n",
      "    - insulin receptor activity\n",
      "    - insulin-like growth factor receptor activity\n",
      "    - macrophage colony-stimulating factor receptor activity\n",
      "    - platelet-derived growth factor alpha-receptor activity\n",
      "    - platelet-derived growth factor beta-receptor activity\n",
      "    - stem cell factor receptor activity\n",
      "    - vascular endothelial growth factor receptor activity\n",
      "    - protein binding\n",
      "    - ATP binding\n",
      "    - boss receptor activity\n",
      "    - kinase activity\n",
      "    - placental growth factor receptor activity\n",
      "    - protein tyrosine kinase collagen receptor activity\n",
      "    - PH domain binding\n",
      "    - identical protein binding\n",
      "    - ephrin receptor binding\n",
      "    - brain-derived neurotrophic factor receptor activity\n",
      "    - DH domain binding\n",
      "    - protein tyrosine kinase binding\n",
      "UniProt: {'Swiss-Prot': 'P54764', 'TrEMBL': ['C9JIX8', 'H7C1N8', 'F5GZZ5', 'C9JFX5', 'C9JEM6', 'H7C2W7', 'E9PG71']}\n",
      "\n",
      "Gene: MAPK14\n",
      "Symbol: MAPK14\n",
      "Name: mitogen-activated protein kinase 14\n",
      "Summary: The protein encoded by this gene is a member of the MAP kinase family. MAP kinases act as an integration point for multiple biochemical signals, and are involved in a wide variety of cellular processes such as proliferation, differentiation, transcription regulation and development. This kinase is activated by various environmental stresses and proinflammatory cytokines. The activation requires its phosphorylation by MAP kinase kinases (MKKs), or its autophosphorylation triggered by the interaction of MAP3K7IP1/TAB1 protein with this kinase. The substrates of this kinase include transcription regulator ATF2, MEF2C, and MAX, cell cycle regulator CDC25B, and tumor suppressor p53, which suggest the roles of this kinase in stress related transcription and cell cycle regulation, as well as in genotoxic stress response. Four alternatively spliced transcript variants of this gene encoding distinct isoforms have been reported. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - DNA damage checkpoint signaling\n",
      "    - cell morphogenesis\n",
      "    - cartilage condensation\n",
      "    - angiogenesis\n",
      "    - osteoblast differentiation\n",
      "    - placenta development\n",
      "    - response to dietary excess\n",
      "    - chondrocyte differentiation\n",
      "    - negative regulation of inflammatory response to antigenic stimulus\n",
      "    - glucose metabolic process\n",
      "    - regulation of transcription by RNA polymerase II\n",
      "    - transcription by RNA polymerase II\n",
      "    - apoptotic process\n",
      "    - chemotaxis\n",
      "    - signal transduction\n",
      "    - cell surface receptor signaling pathway\n",
      "    - cell surface receptor protein serine/threonine kinase signaling pathway\n",
      "    - JNK cascade\n",
      "    - skeletal muscle tissue development\n",
      "    - positive regulation of gene expression\n",
      "    - positive regulation of gene expression\n",
      "    - positive regulation of myotube differentiation\n",
      "    - peptidyl-serine phosphorylation\n",
      "    - fatty acid oxidation\n",
      "    - platelet activation\n",
      "    - regulation of ossification\n",
      "    - osteoclast differentiation\n",
      "    - stress-activated protein kinase signaling cascade\n",
      "    - positive regulation of cyclase activity\n",
      "    - lipopolysaccharide-mediated signaling pathway\n",
      "    - response to muramyl dipeptide\n",
      "    - positive regulation of interleukin-12 production\n",
      "    - response to insulin\n",
      "    - negative regulation of hippo signaling\n",
      "    - intracellular signal transduction\n",
      "    - intracellular signal transduction\n",
      "    - cellular response to vascular endothelial growth factor stimulus\n",
      "    - response to muscle stretch\n",
      "    - p38MAPK cascade\n",
      "    - p38MAPK cascade\n",
      "    - p38MAPK cascade\n",
      "    - positive regulation of protein import into nucleus\n",
      "    - signal transduction in response to DNA damage\n",
      "    - positive regulation of blood vessel endothelial cell migration\n",
      "    - positive regulation of erythrocyte differentiation\n",
      "    - positive regulation of myoblast differentiation\n",
      "    - positive regulation of transcription by RNA polymerase II\n",
      "    - D-glucose import\n",
      "    - positive regulation of D-glucose import\n",
      "    - vascular endothelial growth factor receptor signaling pathway\n",
      "    - stem cell differentiation\n",
      "    - striated muscle cell differentiation\n",
      "    - positive regulation of muscle cell differentiation\n",
      "    - stress-activated MAPK cascade\n",
      "    - positive regulation of cardiac muscle cell proliferation\n",
      "    - bone development\n",
      "    - 3'-UTR-mediated mRNA stabilization\n",
      "    - cellular response to lipopolysaccharide\n",
      "    - cellular response to lipoteichoic acid\n",
      "    - cellular response to tumor necrosis factor\n",
      "    - cellular response to ionizing radiation\n",
      "    - cellular response to UV-B\n",
      "    - negative regulation of canonical Wnt signaling pathway\n",
      "    - positive regulation of brown fat cell differentiation\n",
      "    - cellular senescence\n",
      "    - stress-induced premature senescence\n",
      "    - cellular response to virus\n",
      "    - regulation of synaptic membrane adhesion\n",
      "    - regulation of cytokine production involved in inflammatory response\n",
      "    - positive regulation of myoblast fusion\n",
      "    - positive regulation of reactive oxygen species metabolic process\n",
      "  CC:\n",
      "    - spindle pole\n",
      "    - extracellular region\n",
      "    - nucleus\n",
      "    - nucleus\n",
      "    - nucleus\n",
      "    - nucleoplasm\n",
      "    - cytoplasm\n",
      "    - cytoplasm\n",
      "    - mitochondrion\n",
      "    - cytosol\n",
      "    - cytosol\n",
      "    - cytosol\n",
      "    - nuclear speck\n",
      "    - secretory granule lumen\n",
      "    - glutamatergic synapse\n",
      "    - ficolin-1-rich granule lumen\n",
      "  MF:\n",
      "    - protein serine/threonine kinase activity\n",
      "    - protein serine/threonine kinase activity\n",
      "    - protein serine/threonine kinase activity\n",
      "    - protein serine/threonine kinase activity\n",
      "    - JUN kinase activity\n",
      "    - MAP kinase activity\n",
      "    - MAP kinase kinase activity\n",
      "    - protein binding\n",
      "    - ATP binding\n",
      "    - enzyme binding\n",
      "    - protein phosphatase binding\n",
      "    - mitogen-activated protein kinase p38 binding\n",
      "    - NFAT protein binding\n",
      "    - protein serine kinase activity\n",
      "UniProt: {'Swiss-Prot': 'Q16539', 'TrEMBL': ['B4E0K5', 'L7RSM2', 'B5TY33', 'E7EX54', 'H7C4E2']}\n",
      "\n",
      "Gene: AKT2\n",
      "Symbol: AKT2\n",
      "Name: AKT serine/threonine kinase 2\n",
      "Summary: This gene is a putative oncogene encoding a protein belonging to a subfamily of serine/threonine kinases containing SH2-like (Src homology 2-like) domains, which is involved in signaling pathways. The gene serves as an oncogene in the tumorigenesis of cancer cells For example, its overexpression contributes to the malignant phenotype of a subset of human ductal pancreatic cancers. The encoded protein is a general protein kinase capable of phophorylating several known proteins, and has also been implicated in insulin signaling. [provided by RefSeq, Nov 2019].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - glycogen biosynthetic process\n",
      "    - glucose metabolic process\n",
      "    - chromatin remodeling\n",
      "    - signal transduction\n",
      "    - insulin receptor signaling pathway\n",
      "    - insulin receptor signaling pathway\n",
      "    - negative regulation of long-chain fatty acid import across plasma membrane\n",
      "    - positive regulation of glucose metabolic process\n",
      "    - regulation of cell migration\n",
      "    - positive regulation of cell migration\n",
      "    - positive regulation of fatty acid beta-oxidation\n",
      "    - peripheral nervous system myelin maintenance\n",
      "    - cellular response to insulin stimulus\n",
      "    - intracellular signal transduction\n",
      "    - protein modification process\n",
      "    - fat cell differentiation\n",
      "    - positive regulation of glycogen biosynthetic process\n",
      "    - positive regulation of D-glucose import\n",
      "    - protein stabilization\n",
      "    - regulation of cell cycle\n",
      "    - mammary gland epithelial cell differentiation\n",
      "    - cellular response to high light intensity\n",
      "    - protein localization to plasma membrane\n",
      "    - positive regulation of protein targeting to membrane\n",
      "    - retinal rod cell apoptotic process\n",
      "    - positive regulation of cap-dependent translational initiation\n",
      "    - negative regulation of PERK-mediated unfolded protein response\n",
      "    - positive regulation of cell motility\n",
      "  CC:\n",
      "    - nucleus\n",
      "    - nucleus\n",
      "    - nucleoplasm\n",
      "    - nucleoplasm\n",
      "    - cytoplasm\n",
      "    - early endosome\n",
      "    - cytosol\n",
      "    - cytosol\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - cell cortex\n",
      "    - ruffle membrane\n",
      "    - intracellular membrane-bounded organelle\n",
      "  MF:\n",
      "    - protein serine/threonine kinase activity\n",
      "    - protein serine/threonine kinase activity\n",
      "    - protein serine/threonine kinase activity\n",
      "    - 3-phosphoinositide-dependent protein kinase activity\n",
      "    - DNA-dependent protein kinase activity\n",
      "    - AMP-activated protein kinase activity\n",
      "    - eukaryotic translation initiation factor 2alpha kinase activity\n",
      "    - ribosomal protein S6 kinase activity\n",
      "    - protein binding\n",
      "    - ATP binding\n",
      "    - histone H3S10 kinase activity\n",
      "    - histone H3T11 kinase activity\n",
      "    - histone H3T6 kinase activity\n",
      "    - histone H2AXS139 kinase activity\n",
      "    - histone H3S28 kinase activity\n",
      "    - histone H4S1 kinase activity\n",
      "    - histone H2AS1 kinase activity\n",
      "    - histone H2BS14 kinase activity\n",
      "    - metal ion binding\n",
      "    - histone H3T3 kinase activity\n",
      "    - histone H2AS121 kinase activity\n",
      "    - Rho-dependent protein serine/threonine kinase activity\n",
      "    - protein serine kinase activity\n",
      "    - molecular function activator activity\n",
      "    - histone H2BS36 kinase activity\n",
      "    - histone H3S57 kinase activity\n",
      "    - histone H3T45 kinase activity\n",
      "    - histone H2AT120 kinase activity\n",
      "UniProt: {'Swiss-Prot': 'P31751', 'TrEMBL': ['J3KT31', 'B4DG79', 'C9J258', 'M0QZW8', 'J3KTP4', 'C9JHS6', 'J3QKW1', 'J3KTC6', 'M0QZK3', 'J3QLS6', 'J3KRI8', 'C9JIF6', 'M0R283', 'A0A0A0MRF1', 'C9JC83', 'A8MX96', 'M0R0P9', 'M0R275', 'A0A1B0GXA2', 'J3QL45', 'A0A5F9ZHJ8', 'C9JIJ1', 'E7EVP8', 'J3KSY8']}\n",
      "\n",
      "Gene: EPHB2\n",
      "Symbol: EPHB2\n",
      "Name: EPH receptor B2\n",
      "Summary: This gene encodes a member of the Eph receptor family of receptor tyrosine kinase transmembrane glycoproteins. These receptors are composed of an N-terminal glycosylated ligand-binding domain, a transmembrane region and an intracellular kinase domain. They bind ligands called ephrins and are involved in diverse cellular processes including motility, division, and differentiation. A distinguishing characteristic of Eph-ephrin signaling is that both receptors and ligands are competent to transduce a signaling cascade, resulting in bidirectional signaling. This protein belongs to a subgroup of the Eph receptors called EphB. Proteins of this subgroup are distinguished from other members of the family by sequence homology and preferential binding affinity for membrane-bound ephrin-B ligands. Allelic variants are associated with prostate and brain cancer susceptibility. Alternative splicing results in multiple transcript variants. [provided by RefSeq, May 2015].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - angiogenesis\n",
      "    - urogenital system development\n",
      "    - positive regulation of immunoglobulin production\n",
      "    - negative regulation of cell adhesion\n",
      "    - epidermal growth factor receptor signaling pathway\n",
      "    - nervous system development\n",
      "    - axon guidance\n",
      "    - axon guidance\n",
      "    - axonal fasciculation\n",
      "    - learning or memory\n",
      "    - learning\n",
      "    - insulin receptor signaling pathway\n",
      "    - fibroblast growth factor receptor signaling pathway\n",
      "    - positive regulation of gene expression\n",
      "    - phosphorylation\n",
      "    - peptidyl-tyrosine phosphorylation\n",
      "    - optic nerve morphogenesis\n",
      "    - hindbrain tangential cell migration\n",
      "    - central nervous system projection neuron axonogenesis\n",
      "    - corpus callosum development\n",
      "    - regulation of blood coagulation\n",
      "    - positive regulation of cell migration\n",
      "    - positive regulation of B cell proliferation\n",
      "    - retinal ganglion cell axon guidance\n",
      "    - brain-derived neurotrophic factor receptor signaling pathway\n",
      "    - positive regulation of synaptic plasticity\n",
      "    - positive regulation of tumor necrosis factor production\n",
      "    - platelet-derived growth factor receptor-alpha signaling pathway\n",
      "    - platelet-derived growth factor receptor-beta signaling pathway\n",
      "    - vascular endothelial growth factor receptor-1 signaling pathway\n",
      "    - collagen-activated tyrosine kinase receptor signaling pathway\n",
      "    - vascular endothelial growth factor signaling pathway\n",
      "    - Kit signaling pathway\n",
      "    - macrophage colony-stimulating factor signaling pathway\n",
      "    - B cell activation\n",
      "    - inner ear morphogenesis\n",
      "    - regulation of receptor signaling pathway via JAK-STAT\n",
      "    - negative regulation of Ras protein signal transduction\n",
      "    - insulin-like growth factor receptor signaling pathway\n",
      "    - hepatocyte growth factor receptor signaling pathway\n",
      "    - ephrin receptor signaling pathway\n",
      "    - ephrin receptor signaling pathway\n",
      "    - regulation of neuronal synaptic plasticity\n",
      "    - positive regulation of long-term neuronal synaptic plasticity\n",
      "    - camera-type eye morphogenesis\n",
      "    - negative regulation of axonogenesis\n",
      "    - regulation of body fluid levels\n",
      "    - regulation of filopodium assembly\n",
      "    - positive regulation of synapse assembly\n",
      "    - roof of mouth development\n",
      "    - dendritic spine development\n",
      "    - dendritic spine morphogenesis\n",
      "    - positive regulation of dendritic spine morphogenesis\n",
      "    - negative regulation of ERK1 and ERK2 cascade\n",
      "    - cellular response to lipopolysaccharide\n",
      "    - commissural neuron axon guidance\n",
      "    - postsynaptic membrane assembly\n",
      "    - neuron projection retraction\n",
      "    - vesicle-mediated intercellular transport\n",
      "    - tight junction assembly\n",
      "    - negative regulation of cytokine production involved in inflammatory response\n",
      "    - positive regulation of long-term synaptic potentiation\n",
      "    - negative regulation of glutamate receptor signaling pathway\n",
      "    - positive regulation of glutamate receptor signaling pathway\n",
      "    - positive regulation of protein localization to plasma membrane\n",
      "    - cellular response to amyloid-beta\n",
      "    - neuron projection maintenance\n",
      "    - positive regulation of protein localization to cell surface\n",
      "    - regulation of T-helper 17 type immune response\n",
      "    - regulation of autophagosome assembly\n",
      "    - regulation of behavioral fear response\n",
      "  CC:\n",
      "    - extracellular region\n",
      "    - nucleoplasm\n",
      "    - cytosol\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - cell surface\n",
      "    - axon\n",
      "    - dendrite\n",
      "    - dendrite\n",
      "    - presynaptic membrane\n",
      "    - neuronal cell body\n",
      "    - dendritic spine\n",
      "    - postsynaptic membrane\n",
      "    - hippocampal mossy fiber to CA3 synapse\n",
      "    - postsynapse\n",
      "    - glutamatergic synapse\n",
      "  MF:\n",
      "    - amyloid-beta binding\n",
      "    - amyloid-beta binding\n",
      "    - protein tyrosine kinase activity\n",
      "    - GPI-linked ephrin receptor activity\n",
      "    - transmembrane-ephrin receptor activity\n",
      "    - transmembrane-ephrin receptor activity\n",
      "    - transmembrane-ephrin receptor activity\n",
      "    - epidermal growth factor receptor activity\n",
      "    - fibroblast growth factor receptor activity\n",
      "    - hepatocyte growth factor receptor activity\n",
      "    - insulin receptor activity\n",
      "    - insulin-like growth factor receptor activity\n",
      "    - macrophage colony-stimulating factor receptor activity\n",
      "    - platelet-derived growth factor alpha-receptor activity\n",
      "    - platelet-derived growth factor beta-receptor activity\n",
      "    - stem cell factor receptor activity\n",
      "    - vascular endothelial growth factor receptor activity\n",
      "    - signaling receptor binding\n",
      "    - protein binding\n",
      "    - protein binding\n",
      "    - ATP binding\n",
      "    - axon guidance receptor activity\n",
      "    - boss receptor activity\n",
      "    - placental growth factor receptor activity\n",
      "    - protein tyrosine kinase collagen receptor activity\n",
      "    - identical protein binding\n",
      "    - protein-containing complex binding\n",
      "    - brain-derived neurotrophic factor receptor activity\n",
      "UniProt: {'Swiss-Prot': 'P29323', 'TrEMBL': ['Q4LE53', 'Q6NVW1', 'B4DSE0', 'B1AKC9']}\n",
      "\n",
      "Gene: IL1B\n",
      "Symbol: IL1B\n",
      "Name: interleukin 1 beta\n",
      "Summary: The protein encoded by this gene is a member of the interleukin 1 cytokine family. This cytokine is produced by activated macrophages as a proprotein, which is proteolytically processed to its active form by caspase 1 (CASP1/ICE). This cytokine is an important mediator of the inflammatory response, and is involved in a variety of cellular activities, including cell proliferation, differentiation, and apoptosis. The induction of cyclooxygenase-2 (PTGS2/COX2) by this cytokine in the central nervous system (CNS) is found to contribute to inflammatory pain hypersensitivity. Similarly, IL-1B has been implicated in human osteoarthritis pathogenesis. Patients with severe Coronavirus Disease 2019 (COVID-19) present elevated levels of pro-inflammatory cytokines such as IL-1B in bronchial alveolar lavage fluid samples. The lung damage induced by the Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) is to a large extent, a result of the inflammatory response promoted by cytokines such as IL-1B. This gene and eight other interleukin 1 family genes form a cytokine gene cluster on chromosome 2. [provided by RefSeq, Jul 2020].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - fever generation\n",
      "    - positive regulation of T cell mediated immunity\n",
      "    - apoptotic process\n",
      "    - inflammatory response\n",
      "    - inflammatory response\n",
      "    - inflammatory response\n",
      "    - inflammatory response\n",
      "    - immune response\n",
      "    - signal transduction\n",
      "    - JNK cascade\n",
      "    - cell-cell signaling\n",
      "    - embryo implantation\n",
      "    - positive regulation of cell population proliferation\n",
      "    - negative regulation of cell population proliferation\n",
      "    - response to carbohydrate\n",
      "    - vascular endothelial growth factor production\n",
      "    - positive regulation of vascular endothelial growth factor production\n",
      "    - positive regulation of vascular endothelial growth factor production\n",
      "    - positive regulation of gene expression\n",
      "    - positive regulation of gene expression\n",
      "    - positive regulation of gene expression\n",
      "    - positive regulation of platelet-derived growth factor receptor signaling pathway\n",
      "    - positive regulation of epithelial to mesenchymal transition\n",
      "    - positive regulation of macrophage derived foam cell differentiation\n",
      "    - negative regulation of D-glucose transmembrane transport\n",
      "    - smooth muscle adaptation\n",
      "    - cytokine-mediated signaling pathway\n",
      "    - cytokine-mediated signaling pathway\n",
      "    - hyaluronan biosynthetic process\n",
      "    - positive regulation of cell migration\n",
      "    - neutrophil chemotaxis\n",
      "    - positive regulation of vascular endothelial growth factor receptor signaling pathway\n",
      "    - positive regulation of prostaglandin biosynthetic process\n",
      "    - positive regulation of fever generation\n",
      "    - positive regulation of prostaglandin secretion\n",
      "    - response to lipopolysaccharide\n",
      "    - positive regulation of granulocyte macrophage colony-stimulating factor production\n",
      "    - positive regulation of type II interferon production\n",
      "    - positive regulation of interleukin-2 production\n",
      "    - positive regulation of interleukin-6 production\n",
      "    - positive regulation of interleukin-6 production\n",
      "    - positive regulation of interleukin-8 production\n",
      "    - positive regulation of immature T cell proliferation in thymus\n",
      "    - response to ATP\n",
      "    - positive regulation of heterotypic cell-cell adhesion\n",
      "    - positive regulation of heterotypic cell-cell adhesion\n",
      "    - ectopic germ cell programmed cell death\n",
      "    - positive regulation of T cell proliferation\n",
      "    - regulation of canonical NF-kappaB signal transduction\n",
      "    - positive regulation of canonical NF-kappaB signal transduction\n",
      "    - positive regulation of canonical NF-kappaB signal transduction\n",
      "    - positive regulation of canonical NF-kappaB signal transduction\n",
      "    - positive regulation of canonical NF-kappaB signal transduction\n",
      "    - negative regulation of MAP kinase activity\n",
      "    - positive regulation of MAPK cascade\n",
      "    - positive regulation of nitric oxide biosynthetic process\n",
      "    - positive regulation of angiogenesis\n",
      "    - positive regulation of angiogenesis\n",
      "    - negative regulation of lipid metabolic process\n",
      "    - positive regulation of mitotic nuclear division\n",
      "    - positive regulation of DNA-templated transcription\n",
      "    - positive regulation of complement activation\n",
      "    - positive regulation of transcription by RNA polymerase II\n",
      "    - positive regulation of JNK cascade\n",
      "    - negative regulation of insulin receptor signaling pathway\n",
      "    - positive regulation of protein export from nucleus\n",
      "    - astrocyte activation\n",
      "    - regulation of defense response to virus by host\n",
      "    - positive regulation of inflammatory response\n",
      "    - regulation of neurogenesis\n",
      "    - negative regulation of neurogenesis\n",
      "    - regulation of insulin secretion\n",
      "    - negative regulation of synaptic transmission\n",
      "    - defense response to Gram-positive bacterium\n",
      "    - negative regulation of lipid catabolic process\n",
      "    - positive regulation of lipid catabolic process\n",
      "    - regulation of nitric-oxide synthase activity\n",
      "    - positive regulation of membrane protein ectodomain proteolysis\n",
      "    - positive regulation of NF-kappaB transcription factor activity\n",
      "    - positive regulation of NF-kappaB transcription factor activity\n",
      "    - positive regulation of cell division\n",
      "    - positive regulation of phosphatidylinositol 3-kinase/protein kinase B signal transduction\n",
      "    - positive regulation of glial cell proliferation\n",
      "    - positive regulation of cell adhesion molecule production\n",
      "    - positive regulation of calcidiol 1-monooxygenase activity\n",
      "    - negative regulation of adiponectin secretion\n",
      "    - regulation of ERK1 and ERK2 cascade\n",
      "    - regulation of ERK1 and ERK2 cascade\n",
      "    - positive regulation of ERK1 and ERK2 cascade\n",
      "    - monocyte aggregation\n",
      "    - interleukin-1-mediated signaling pathway\n",
      "    - response to interleukin-1\n",
      "    - cellular response to lipopolysaccharide\n",
      "    - cellular response to mechanical stimulus\n",
      "    - cellular response to xenobiotic stimulus\n",
      "    - positive regulation of monocyte chemotactic protein-1 production\n",
      "    - extrinsic apoptotic signaling pathway in absence of ligand\n",
      "    - cellular response to interleukin-17\n",
      "    - positive regulation of neuroinflammatory response\n",
      "    - positive regulation of p38MAPK cascade\n",
      "    - positive regulation of non-canonical NF-kappaB signal transduction\n",
      "    - positive regulation of RNA biosynthetic process\n",
      "    - regulation of establishment of endothelial barrier\n",
      "    - negative regulation of gap junction assembly\n",
      "    - positive regulation of tight junction disassembly\n",
      "    - positive regulation of T-helper 1 cell cytokine production\n",
      "    - negative regulation of extrinsic apoptotic signaling pathway in absence of ligand\n",
      "  CC:\n",
      "    - extracellular region\n",
      "    - extracellular region\n",
      "    - extracellular region\n",
      "    - extracellular space\n",
      "    - extracellular space\n",
      "    - extracellular space\n",
      "    - extracellular space\n",
      "    - lysosome\n",
      "    - cytosol\n",
      "    - secretory granule\n",
      "  MF:\n",
      "    - cytokine activity\n",
      "    - cytokine activity\n",
      "    - cytokine activity\n",
      "    - cytokine activity\n",
      "    - interleukin-1 receptor binding\n",
      "    - integrin binding\n",
      "    - protein binding\n",
      "    - protein domain specific binding\n",
      "UniProt: {'Swiss-Prot': 'P01584', 'TrEMBL': ['C9JVK0', 'C9JWV2', 'C9JSC2']}\n",
      "\n",
      "Gene: CSNK1A1\n",
      "Symbol: CSNK1A1\n",
      "Name: casein kinase 1 alpha 1\n",
      "Summary: Enables protein serine/threonine kinase activity. Involved in several processes, including negative regulation of canonical Wnt signaling pathway; peptidyl-serine phosphorylation; and positive regulation of proteasomal ubiquitin-dependent protein catabolic process. Located in centrosome; cytosol; and nuclear speck. Part of beta-catenin destruction complex. Colocalizes with keratin filament and mRNA cleavage and polyadenylation specificity factor complex. Biomarker of Alzheimer's disease and inclusion body myositis. [provided by Alliance of Genome Resources, Apr 2022]\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - chromatin remodeling\n",
      "    - protein phosphorylation\n",
      "    - Golgi organization\n",
      "    - signal transduction\n",
      "    - cell surface receptor signaling pathway\n",
      "    - Wnt signaling pathway\n",
      "    - viral protein processing\n",
      "    - cellular response to nutrient\n",
      "    - positive regulation of proteasomal ubiquitin-dependent protein catabolic process\n",
      "    - positive regulation of proteasomal ubiquitin-dependent protein catabolic process\n",
      "    - positive regulation of Rho protein signal transduction\n",
      "    - proteasome-mediated ubiquitin-dependent protein catabolic process\n",
      "    - intermediate filament cytoskeleton organization\n",
      "    - cell division\n",
      "    - negative regulation of canonical Wnt signaling pathway\n",
      "    - negative regulation of canonical Wnt signaling pathway\n",
      "    - negative regulation of canonical Wnt signaling pathway\n",
      "    - negative regulation of NLRP3 inflammasome complex assembly\n",
      "    - positive regulation of TORC1 signaling\n",
      "  CC:\n",
      "    - kinetochore\n",
      "    - nucleus\n",
      "    - cytoplasm\n",
      "    - centrosome\n",
      "    - spindle\n",
      "    - cytosol\n",
      "    - mRNA cleavage and polyadenylation specificity factor complex\n",
      "    - cilium\n",
      "    - membrane\n",
      "    - nuclear speck\n",
      "    - beta-catenin destruction complex\n",
      "    - beta-catenin destruction complex\n",
      "    - ciliary basal body\n",
      "    - keratin filament\n",
      "  MF:\n",
      "    - protein kinase activity\n",
      "    - protein kinase activity\n",
      "    - protein serine/threonine kinase activity\n",
      "    - protein serine/threonine kinase activity\n",
      "    - protein serine/threonine kinase activity\n",
      "    - protein serine/threonine kinase activity\n",
      "    - protein serine/threonine kinase activity\n",
      "    - protein serine/threonine kinase activity\n",
      "    - 3-phosphoinositide-dependent protein kinase activity\n",
      "    - DNA-dependent protein kinase activity\n",
      "    - AMP-activated protein kinase activity\n",
      "    - eukaryotic translation initiation factor 2alpha kinase activity\n",
      "    - ribosomal protein S6 kinase activity\n",
      "    - protein binding\n",
      "    - ATP binding\n",
      "    - histone H3S10 kinase activity\n",
      "    - histone H3T11 kinase activity\n",
      "    - histone H3T6 kinase activity\n",
      "    - histone H2AXS139 kinase activity\n",
      "    - histone H3S28 kinase activity\n",
      "    - histone H4S1 kinase activity\n",
      "    - histone H2AS1 kinase activity\n",
      "    - histone H2BS14 kinase activity\n",
      "    - histone H3T3 kinase activity\n",
      "    - histone H2AS121 kinase activity\n",
      "    - Rho-dependent protein serine/threonine kinase activity\n",
      "    - protein serine kinase activity\n",
      "    - histone H2BS36 kinase activity\n",
      "    - histone H3S57 kinase activity\n",
      "    - histone H3T45 kinase activity\n",
      "    - histone H2AT120 kinase activity\n",
      "UniProt: {'Swiss-Prot': 'P48729', 'TrEMBL': ['B4DER9', 'Q6PJ06', 'A0A5F9W493', 'A0A590UJQ7', 'U3KQ83', 'E7ETM0', 'D6REM4', 'U3KQK7', 'E5RG27', 'U3KPX3', 'A0A590UJ43', 'D6RGE5', 'A0A590UK94', 'H0Y9X2']}\n",
      "\n",
      "Gene: MCL1\n",
      "Symbol: MCL1\n",
      "Name: MCL1 apoptosis regulator, BCL2 family member\n",
      "Summary: This gene encodes an anti-apoptotic protein, which is a member of the Bcl-2 family. Alternative splicing results in multiple transcript variants. The longest gene product (isoform 1) enhances cell survival by inhibiting apoptosis while the alternatively spliced shorter gene products (isoform 2 and isoform 3) promote apoptosis and are death-inducing. [provided by RefSeq, Oct 2010].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - cell fate determination\n",
      "    - release of cytochrome c from mitochondria\n",
      "    - DNA damage response\n",
      "    - mitochondrial fusion\n",
      "    - intrinsic apoptotic signaling pathway in response to DNA damage\n",
      "    - negative regulation of autophagy\n",
      "    - cellular homeostasis\n",
      "    - response to cytokine\n",
      "    - regulation of apoptotic process\n",
      "    - positive regulation of apoptotic process\n",
      "    - negative regulation of apoptotic process\n",
      "    - positive regulation of neuron apoptotic process\n",
      "    - protein transmembrane transport\n",
      "    - extrinsic apoptotic signaling pathway in absence of ligand\n",
      "    - extrinsic apoptotic signaling pathway in absence of ligand\n",
      "    - positive regulation of oxidative stress-induced neuron intrinsic apoptotic signaling pathway\n",
      "    - negative regulation of anoikis\n",
      "    - negative regulation of extrinsic apoptotic signaling pathway in absence of ligand\n",
      "  CC:\n",
      "    - nucleus\n",
      "    - nucleoplasm\n",
      "    - cytoplasm\n",
      "    - mitochondrion\n",
      "    - mitochondrion\n",
      "    - mitochondrial outer membrane\n",
      "    - mitochondrial outer membrane\n",
      "    - cytosol\n",
      "    - cytosol\n",
      "    - membrane\n",
      "    - membrane\n",
      "    - Bcl-2 family protein complex\n",
      "    - Bcl-2 family protein complex\n",
      "  MF:\n",
      "    - protein binding\n",
      "    - protein transmembrane transporter activity\n",
      "    - channel activity\n",
      "    - protein heterodimerization activity\n",
      "    - BH3 domain binding\n",
      "UniProt: {'Swiss-Prot': 'Q07820', 'TrEMBL': ['C8YZ26', 'A0A087WT64', 'A0A7I2V2W2']}\n",
      "\n",
      "Gene: ESR2\n",
      "Symbol: ESR2\n",
      "Name: estrogen receptor 2\n",
      "Summary: This gene encodes a member of the family of estrogen receptors and superfamily of nuclear receptor transcription factors. The gene product contains an N-terminal DNA binding domain and C-terminal ligand binding domain and is localized to the nucleus, cytoplasm, and mitochondria. Upon binding to 17beta-estradiol or related ligands, the encoded protein forms homo- or hetero-dimers that interact with specific DNA sequences to activate transcription. Some isoforms dominantly inhibit the activity of other estrogen receptor family members. Several alternatively spliced transcript variants of this gene have been described, but the full-length nature of some of these variants has not been fully characterized. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - negative regulation of transcription by RNA polymerase II\n",
      "    - regulation of DNA-templated transcription\n",
      "    - regulation of transcription by RNA polymerase II\n",
      "    - signal transduction\n",
      "    - cell-cell signaling\n",
      "    - negative regulation of cell growth\n",
      "    - estrogen receptor signaling pathway\n",
      "    - estrogen receptor signaling pathway\n",
      "    - estrogen receptor signaling pathway\n",
      "    - positive regulation of DNA-templated transcription\n",
      "    - positive regulation of transcription by RNA polymerase II\n",
      "    - positive regulation of DNA-binding transcription factor activity\n",
      "    - cellular response to estradiol stimulus\n",
      "    - cellular response to estradiol stimulus\n",
      "  CC:\n",
      "    - chromatin\n",
      "    - chromatin\n",
      "    - nucleus\n",
      "    - nucleus\n",
      "    - nucleus\n",
      "    - nucleoplasm\n",
      "    - nucleoplasm\n",
      "    - mitochondrion\n",
      "    - intracellular membrane-bounded organelle\n",
      "  MF:\n",
      "    - RNA polymerase II cis-regulatory region sequence-specific DNA binding\n",
      "    - DNA-binding transcription factor activity, RNA polymerase II-specific\n",
      "    - DNA binding\n",
      "    - nuclear steroid receptor activity\n",
      "    - nuclear receptor activity\n",
      "    - nuclear receptor activity\n",
      "    - nuclear receptor activity\n",
      "    - steroid binding\n",
      "    - steroid binding\n",
      "    - protein binding\n",
      "    - zinc ion binding\n",
      "    - enzyme binding\n",
      "    - nuclear estrogen receptor activity\n",
      "    - nuclear estrogen receptor activity\n",
      "    - nuclear estrogen receptor activity\n",
      "    - estrogen response element binding\n",
      "    - estrogen response element binding\n",
      "    - receptor antagonist activity\n",
      "UniProt: {'Swiss-Prot': 'Q92731', 'TrEMBL': ['F1D8N3', 'Q7LCB3', 'A0A348FV93', 'Q0PTK2', 'G3V5S2']}\n",
      "\n",
      "Gene: MAP3K2\n",
      "Symbol: MAP3K2\n",
      "Name: mitogen-activated protein kinase kinase kinase 2\n",
      "Summary: The protein encoded by this gene is a member of serine/threonine protein kinase family. This kinase preferentially activates other kinases involved in the MAP kinase signaling pathway. This kinase has been shown to directly phosphorylate and activate Ikappa B kinases, and thus plays a role in NF-kappa B signaling pathway. This kinase has also been found to bind and activate protein kinase C-related kinase 2, which suggests its involvement in a regulated signaling process. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - MAPK cascade\n",
      "    - intracellular signal transduction\n",
      "    - cellular response to mechanical stimulus\n",
      "  CC:\n",
      "    - nucleoplasm\n",
      "    - cytoplasm\n",
      "    - cytosol\n",
      "  MF:\n",
      "    - protein kinase activity\n",
      "    - protein serine/threonine kinase activity\n",
      "    - JUN kinase kinase kinase activity\n",
      "    - protein binding\n",
      "    - ATP binding\n",
      "    - protein kinase binding\n",
      "    - metal ion binding\n",
      "    - protein serine kinase activity\n",
      "UniProt: {'Swiss-Prot': 'Q9Y2U5'}\n",
      "\n",
      "Gene: CCND2\n",
      "Symbol: CCND2\n",
      "Name: cyclin D2\n",
      "Summary: The protein encoded by this gene belongs to the highly conserved cyclin family, whose members are characterized by a dramatic periodicity in protein abundance through the cell cycle. Cyclins function as regulators of CDK kinases. Different cyclins exhibit distinct expression and degradation patterns which contribute to the temporal coordination of each mitotic event. This cyclin forms a complex with CDK4 or CDK6 and functions as a regulatory subunit of the complex, whose activity is required for cell cycle G1/S transition. This protein has been shown to interact with and be involved in the phosphorylation of tumor suppressor protein Rb. Knockout studies of the homologous gene in mouse suggest the essential roles of this gene in ovarian granulosa and germ cell proliferation. High level expression of this gene was observed in ovarian and testicular tumors. Mutations in this gene are associated with megalencephaly-polymicrogyria-polydactyly-hydrocephalus syndrome 3 (MPPH3). [provided by RefSeq, Sep 2014].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - G1/S transition of mitotic cell cycle\n",
      "    - G1/S transition of mitotic cell cycle\n",
      "    - G1/S transition of mitotic cell cycle\n",
      "    - G1/S transition of mitotic cell cycle\n",
      "    - long-term memory\n",
      "    - positive regulation of cell population proliferation\n",
      "    - adult locomotory behavior\n",
      "    - negative regulation of apoptotic process\n",
      "    - cell division\n",
      "    - cellular response to X-ray\n",
      "    - positive regulation of G1/S transition of mitotic cell cycle\n",
      "    - positive regulation of G1/S transition of mitotic cell cycle\n",
      "  CC:\n",
      "    - cyclin-dependent protein kinase holoenzyme complex\n",
      "    - cyclin-dependent protein kinase holoenzyme complex\n",
      "    - chromatin\n",
      "    - nucleus\n",
      "    - nucleus\n",
      "    - nucleoplasm\n",
      "    - nucleoplasm\n",
      "    - nucleolus\n",
      "    - cytoplasm\n",
      "    - microtubule organizing center\n",
      "    - cytosol\n",
      "    - cytosol\n",
      "    - nuclear membrane\n",
      "    - cyclin D2-CDK4 complex\n",
      "  MF:\n",
      "    - protein binding\n",
      "    - cyclin-dependent protein serine/threonine kinase regulator activity\n",
      "    - protein kinase binding\n",
      "    - protein serine/threonine kinase activator activity\n",
      "    - cyclin-dependent protein serine/threonine kinase activator activity\n",
      "    - cyclin-dependent protein serine/threonine kinase activator activity\n",
      "UniProt: {'Swiss-Prot': 'P30279', 'TrEMBL': 'A0A6Q8PGZ3'}\n",
      "\n",
      "Gene: HTT\n",
      "Symbol: HTT\n",
      "Name: huntingtin\n",
      "Summary: Huntingtin is a disease gene linked to Huntington's disease, a neurodegenerative disorder characterized by loss of striatal neurons. This is thought to be caused by an expanded, unstable trinucleotide repeat in the huntingtin gene, which translates as a polyglutamine repeat in the protein product. A fairly broad range of trinucleotide repeats (9-35) has been identified in normal controls, and repeat numbers in excess of 40 have been described as pathological. The huntingtin locus is large, spanning 180 kb and consisting of 67 exons. The huntingtin gene is widely expressed and is required for normal development. It is expressed as 2 alternatively polyadenylated forms displaying different relative abundance in various fetal and adult tissues. The larger transcript is approximately 13.7 kb and is expressed predominantly in adult and fetal brain whereas the smaller transcript of approximately 10.3 kb is more widely expressed. The genetic defect leading to Huntington's disease may not necessarily eliminate transcription, but may confer a new property on the mRNA or alter the function of the protein. One candidate is the huntingtin-associated protein-1, highly expressed in brain, which has increased affinity for huntingtin protein with expanded polyglutamine repeats. This gene contains an upstream open reading frame in the 5' UTR that inhibits expression of the huntingtin gene product through translational repression. [provided by RefSeq, Jul 2016].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - establishment of mitotic spindle orientation\n",
      "    - retrograde vesicle-mediated transport, Golgi to endoplasmic reticulum\n",
      "    - apoptotic process\n",
      "    - Golgi organization\n",
      "    - positive regulation of inositol 1,4,5-trisphosphate-sensitive calcium-release channel activity\n",
      "    - protein destabilization\n",
      "    - vocal learning\n",
      "    - positive regulation of apoptotic process\n",
      "    - positive regulation of cilium assembly\n",
      "    - vesicle transport along microtubule\n",
      "    - microtubule-based transport\n",
      "    - positive regulation of mitophagy\n",
      "    - positive regulation of lipophagy\n",
      "    - regulation of CAMKK-AMPK signaling cascade\n",
      "    - positive regulation of CAMKK-AMPK signaling cascade\n",
      "    - positive regulation of aggrephagy\n",
      "    - negative regulation of extrinsic apoptotic signaling pathway\n",
      "  CC:\n",
      "    - nucleus\n",
      "    - nucleus\n",
      "    - nucleoplasm\n",
      "    - cytoplasm\n",
      "    - cytoplasm\n",
      "    - cytoplasm\n",
      "    - early endosome\n",
      "    - late endosome\n",
      "    - autophagosome\n",
      "    - endoplasmic reticulum\n",
      "    - Golgi apparatus\n",
      "    - centriole\n",
      "    - cytosol\n",
      "    - inclusion body\n",
      "    - axon\n",
      "    - dendrite\n",
      "    - cytoplasmic vesicle membrane\n",
      "    - protein-containing complex\n",
      "    - protein-containing complex\n",
      "    - perinuclear region of cytoplasm\n",
      "    - perinuclear region of cytoplasm\n",
      "    - presynaptic cytosol\n",
      "    - postsynaptic cytosol\n",
      "  MF:\n",
      "    - p53 binding\n",
      "    - phosphoprotein phosphatase activity\n",
      "    - protein binding\n",
      "    - profilin binding\n",
      "    - kinase binding\n",
      "    - heat shock protein binding\n",
      "    - dynactin binding\n",
      "    - identical protein binding\n",
      "    - transmembrane transporter binding\n",
      "    - dynein intermediate chain binding\n",
      "    - beta-tubulin binding\n",
      "UniProt: {'Swiss-Prot': 'P42858', 'TrEMBL': ['A0A7P0TAN5', 'A0A3B3IU25', 'A0A3B3ISR3', 'H0YA07', 'A0A7P0TAC5', 'A0A7P0Z417', 'A0A7P0TA78']}\n",
      "\n",
      "Gene: HSPB1\n",
      "Symbol: HSPB1\n",
      "Name: heat shock protein family B (small) member 1\n",
      "Summary: This gene encodes a member of the small heat shock protein (HSP20) family of proteins. In response to environmental stress, the encoded protein translocates from the cytoplasm to the nucleus and functions as a molecular chaperone that promotes the correct folding of other proteins. This protein plays an important role in the differentiation of a wide variety of cell types. Expression of this gene is correlated with poor clinical outcome in multiple human cancers, and the encoded protein may promote cancer cell proliferation and metastasis, while protecting cancer cells from apoptosis. Mutations in this gene have been identified in human patients with Charcot-Marie-Tooth disease and distal hereditary motor neuropathy. [provided by RefSeq, Aug 2017].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - regulation of protein phosphorylation\n",
      "    - regulation of translational initiation\n",
      "    - negative regulation of protein kinase activity\n",
      "    - response to unfolded protein\n",
      "    - response to heat\n",
      "    - response to virus\n",
      "    - regulation of autophagy\n",
      "    - positive regulation of interleukin-1 beta production\n",
      "    - positive regulation of tumor necrosis factor production\n",
      "    - intracellular signal transduction\n",
      "    - cellular response to vascular endothelial growth factor stimulus\n",
      "    - protein refolding\n",
      "    - negative regulation of apoptotic process\n",
      "    - negative regulation of apoptotic process\n",
      "    - regulation of canonical NF-kappaB signal transduction\n",
      "    - positive regulation of blood vessel endothelial cell migration\n",
      "    - positive regulation of angiogenesis\n",
      "    - vascular endothelial growth factor receptor signaling pathway\n",
      "    - chaperone-mediated protein folding\n",
      "    - platelet aggregation\n",
      "    - anterograde axonal protein transport\n",
      "    - negative regulation of oxidative stress-induced intrinsic apoptotic signaling pathway\n",
      "    - positive regulation of endothelial cell chemotaxis\n",
      "  CC:\n",
      "    - proteasome complex\n",
      "    - cornified envelope\n",
      "    - extracellular space\n",
      "    - nucleus\n",
      "    - nucleus\n",
      "    - cytoplasm\n",
      "    - cytoplasm\n",
      "    - cytoplasm\n",
      "    - spindle\n",
      "    - cytosol\n",
      "    - cytoskeleton\n",
      "    - focal adhesion\n",
      "    - Z disc\n",
      "    - extracellular exosome\n",
      "    - axon cytoplasm\n",
      "  MF:\n",
      "    - RNA binding\n",
      "    - protein kinase C binding\n",
      "    - protein binding\n",
      "    - protein kinase C inhibitor activity\n",
      "    - protein kinase binding\n",
      "    - identical protein binding\n",
      "    - identical protein binding\n",
      "    - protein homodimerization activity\n",
      "    - ubiquitin binding\n",
      "    - protein folding chaperone\n",
      "    - unfolded protein binding\n",
      "    - RNA polymerase II-specific DNA-binding transcription factor binding\n",
      "UniProt: {'Swiss-Prot': 'P04792', 'TrEMBL': 'V9HW43'}\n",
      "\n",
      "Gene: AIP\n",
      "Symbol: AIP\n",
      "Name: AHR interacting HSP90 co-chaperone\n",
      "Summary: The protein encoded by this gene is a receptor for aryl hydrocarbons and a ligand-activated transcription factor. The encoded protein is found in the cytoplasm as part of a multiprotein complex, but upon binding of ligand is transported to the nucleus. This protein can regulate the expression of many xenobiotic metabolizing enzymes. Also, the encoded protein can bind specifically to and inhibit the activity of hepatitis B virus. Three transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Dec 2014].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - protein targeting to mitochondrion\n",
      "    - xenobiotic metabolic process\n",
      "    - protein kinase A signaling\n",
      "    - positive regulation of DNA-templated transcription\n",
      "    - protein maturation\n",
      "  CC:\n",
      "    - nucleoplasm\n",
      "    - cytoplasm\n",
      "    - cytosol\n",
      "    - cytosol\n",
      "    - plasma membrane\n",
      "    - aryl hydrocarbon receptor complex\n",
      "  MF:\n",
      "    - transcription coactivator activity\n",
      "    - peptidyl-prolyl cis-trans isomerase activity\n",
      "    - protein binding\n",
      "    - aryl hydrocarbon receptor binding\n",
      "    - enzyme binding\n",
      "    - GAF domain binding\n",
      "    - unfolded protein binding\n",
      "UniProt: {'Swiss-Prot': 'O00170', 'TrEMBL': ['A0A804HKL7', 'A0A804HJ38', 'A0A804HKH9', 'A0A804HKK3', 'V9GYQ3', 'A0A804HKP8', 'A0A804HIX8', 'A0A804HJ46', 'E9PMH2']}\n",
      "\n",
      "Gene: TNFSF11\n",
      "Symbol: TNFSF11\n",
      "Name: TNF superfamily member 11\n",
      "Summary: This gene encodes a member of the tumor necrosis factor (TNF) cytokine family which is a ligand for osteoprotegerin and functions as a key factor for osteoclast differentiation and activation. This protein was shown to be a dentritic cell survival factor and is involved in the regulation of T cell-dependent immune response. T cell activation was reported to induce expression of this gene and lead to an increase of osteoclastogenesis and bone loss. This protein was shown to activate antiapoptotic kinase AKT/PKB through a signaling complex involving SRC kinase and tumor necrosis factor receptor-associated factor (TRAF) 6, which indicated this protein may have a role in the regulation of cell apoptosis. Targeted disruption of the related gene in mice led to severe osteopetrosis and a lack of osteoclasts. The deficient mice exhibited defects in early differentiation of T and B lymphocytes, and failed to form lobulo-alveolar mammary structures during pregnancy. Two alternatively spliced transcript variants have been found. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - negative regulation of transcription by RNA polymerase II\n",
      "    - ossification\n",
      "    - osteoclast proliferation\n",
      "    - monocyte chemotaxis\n",
      "    - immune response\n",
      "    - cell surface receptor signaling pathway\n",
      "    - JNK cascade\n",
      "    - positive regulation of gene expression\n",
      "    - cytokine-mediated signaling pathway\n",
      "    - calcium-mediated signaling\n",
      "    - osteoclast differentiation\n",
      "    - osteoclast differentiation\n",
      "    - osteoclast differentiation\n",
      "    - tumor necrosis factor-mediated signaling pathway\n",
      "    - tumor necrosis factor-mediated signaling pathway\n",
      "    - mammary gland epithelial cell proliferation\n",
      "    - positive regulation of homotypic cell-cell adhesion\n",
      "    - osteoclast development\n",
      "    - paracrine signaling\n",
      "    - positive regulation of canonical NF-kappaB signal transduction\n",
      "    - positive regulation of MAP kinase activity\n",
      "    - phosphatidylinositol 3-kinase/protein kinase B signal transduction\n",
      "    - tooth eruption\n",
      "    - bone resorption\n",
      "    - positive regulation of osteoclast differentiation\n",
      "    - positive regulation of osteoclast differentiation\n",
      "    - positive regulation of bone resorption\n",
      "    - positive regulation of transcription by RNA polymerase II\n",
      "    - positive regulation of JNK cascade\n",
      "    - positive regulation of T cell activation\n",
      "    - positive regulation of DNA-binding transcription factor activity\n",
      "    - positive regulation of NF-kappaB transcription factor activity\n",
      "    - positive regulation of corticotropin-releasing hormone secretion\n",
      "    - positive regulation of phosphatidylinositol 3-kinase/protein kinase B signal transduction\n",
      "    - calcium ion homeostasis\n",
      "    - mammary gland alveolus development\n",
      "    - ERK1 and ERK2 cascade\n",
      "    - positive regulation of ERK1 and ERK2 cascade\n",
      "    - positive regulation of fever generation by positive regulation of prostaglandin secretion\n",
      "    - positive regulation of intracellular signal transduction\n",
      "    - cellular response to leukemia inhibitory factor\n",
      "    - positive regulation of osteoclast development\n",
      "    - positive regulation of extrinsic apoptotic signaling pathway\n",
      "  CC:\n",
      "    - extracellular region\n",
      "    - extracellular region\n",
      "    - extracellular space\n",
      "    - cytoplasm\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "  MF:\n",
      "    - cytokine activity\n",
      "    - cytokine activity\n",
      "    - cytokine activity\n",
      "    - tumor necrosis factor receptor binding\n",
      "    - protein binding\n",
      "    - tumor necrosis factor receptor superfamily binding\n",
      "    - identical protein binding\n",
      "UniProt: {'Swiss-Prot': 'O14788', 'TrEMBL': ['Q5T9Y4', 'Q54A98']}\n",
      "\n",
      "Gene: CTLA4\n",
      "Symbol: CTLA4\n",
      "Name: cytotoxic T-lymphocyte associated protein 4\n",
      "Summary: This gene is a member of the immunoglobulin superfamily and encodes a protein which transmits an inhibitory signal to T cells. The protein contains a V domain, a transmembrane domain, and a cytoplasmic tail. Alternate transcriptional splice variants, encoding different isoforms, have been characterized. The membrane-bound isoform functions as a homodimer interconnected by a disulfide bond, while the soluble isoform functions as a monomer. Mutations in this gene have been associated with insulin-dependent diabetes mellitus, Graves disease, Hashimoto thyroiditis, celiac disease, systemic lupus erythematosus, thyroid-associated orbitopathy, and other autoimmune diseases. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - adaptive immune response\n",
      "    - immune response\n",
      "    - DNA damage response\n",
      "    - negative regulation of B cell proliferation\n",
      "    - negative regulation of T cell proliferation\n",
      "    - positive regulation of apoptotic process\n",
      "    - negative regulation of regulatory T cell differentiation\n",
      "    - negative regulation of regulatory T cell differentiation\n",
      "    - T cell receptor signaling pathway\n",
      "    - B cell receptor signaling pathway\n",
      "    - B cell receptor signaling pathway\n",
      "  CC:\n",
      "    - Golgi apparatus\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - external side of plasma membrane\n",
      "    - external side of plasma membrane\n",
      "    - clathrin-coated endocytic vesicle\n",
      "    - perinuclear region of cytoplasm\n",
      "    - protein complex involved in cell adhesion\n",
      "  MF:\n",
      "    - protein binding\n",
      "UniProt: {'Swiss-Prot': 'P16410', 'TrEMBL': ['A0A8Q3WKZ2', 'A0A8Q3SIR7']}\n",
      "\n",
      "Gene: MYT1\n",
      "Symbol: MYT1\n",
      "Name: myelin transcription factor 1\n",
      "Summary: The protein encoded by this gene is a member of a family of neural specific, zinc finger-containing DNA-binding proteins. The protein binds to the promoter regions of proteolipid proteins of the central nervous system and plays a role in the developing nervous system. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - regulation of DNA-templated transcription\n",
      "    - regulation of transcription by RNA polymerase II\n",
      "    - nervous system development\n",
      "    - cell differentiation\n",
      "  CC:\n",
      "    - chromatin\n",
      "    - nucleus\n",
      "    - nucleoplasm\n",
      "    - cytosol\n",
      "  MF:\n",
      "    - DNA-binding transcription factor activity, RNA polymerase II-specific\n",
      "    - DNA binding\n",
      "    - DNA-binding transcription factor activity\n",
      "    - zinc ion binding\n",
      "UniProt: {'Swiss-Prot': 'Q01538', 'TrEMBL': ['A0A2R8Y3S5', 'Q6P6D5']}\n",
      "\n",
      "Gene: PPARA\n",
      "Symbol: PPARA\n",
      "Name: peroxisome proliferator activated receptor alpha\n",
      "Summary: Peroxisome proliferators include hypolipidemic drugs, herbicides, leukotriene antagonists, and plasticizers; this term arises because they induce an increase in the size and number of peroxisomes. Peroxisomes are subcellular organelles found in plants and animals that contain enzymes for respiration and for cholesterol and lipid metabolism. The action of peroxisome proliferators is thought to be mediated via specific receptors, called PPARs, which belong to the steroid hormone receptor superfamily. PPARs affect the expression of target genes involved in cell proliferation, cell differentiation and in immune and inflammation responses. Three closely related subtypes (alpha, beta/delta, and gamma) have been identified. This gene encodes the subtype PPAR-alpha, which is a nuclear transcription factor. Multiple alternatively spliced transcript variants have been described for this gene, although the full-length nature of only two has been determined. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - negative regulation of transcription by RNA polymerase II\n",
      "    - negative regulation of transcription by RNA polymerase II\n",
      "    - response to hypoxia\n",
      "    - gluconeogenesis\n",
      "    - fatty acid metabolic process\n",
      "    - heart development\n",
      "    - response to nutrient\n",
      "    - epidermis development\n",
      "    - cellular response to starvation\n",
      "    - hormone-mediated signaling pathway\n",
      "    - gene expression\n",
      "    - regulation of ketone metabolic process\n",
      "    - negative regulation of macrophage derived foam cell differentiation\n",
      "    - negative regulation of cholesterol storage\n",
      "    - negative regulation of cholesterol storage\n",
      "    - regulation of fatty acid metabolic process\n",
      "    - cell differentiation\n",
      "    - negative regulation of transforming growth factor beta receptor signaling pathway\n",
      "    - nuclear receptor-mediated steroid hormone signaling pathway\n",
      "    - intracellular receptor signaling pathway\n",
      "    - positive regulation of fatty acid beta-oxidation\n",
      "    - negative regulation of appetite\n",
      "    - response to insulin\n",
      "    - circadian regulation of gene expression\n",
      "    - behavioral response to nicotine\n",
      "    - peroxisome proliferator activated receptor signaling pathway\n",
      "    - peroxisome proliferator activated receptor signaling pathway\n",
      "    - wound healing\n",
      "    - lipoprotein metabolic process\n",
      "    - regulation of circadian rhythm\n",
      "    - response to ethanol\n",
      "    - positive regulation of gluconeogenesis\n",
      "    - negative regulation of blood pressure\n",
      "    - negative regulation of glycolytic process\n",
      "    - positive regulation of DNA-templated transcription\n",
      "    - positive regulation of fatty acid metabolic process\n",
      "    - positive regulation of transcription by RNA polymerase II\n",
      "    - positive regulation of transcription by RNA polymerase II\n",
      "    - positive regulation of transcription by RNA polymerase II\n",
      "    - nitric oxide metabolic process\n",
      "    - positive regulation of fatty acid oxidation\n",
      "    - positive regulation of lipid biosynthetic process\n",
      "    - negative regulation of inflammatory response\n",
      "    - negative regulation of inflammatory response\n",
      "    - negative regulation of phosphatidylinositol 3-kinase/protein kinase B signal transduction\n",
      "    - negative regulation of cell growth involved in cardiac muscle cell development\n",
      "    - enamel mineralization\n",
      "    - cellular response to fructose stimulus\n",
      "    - negative regulation of cytokine production involved in inflammatory response\n",
      "    - negative regulation of miRNA transcription\n",
      "    - negative regulation of leukocyte cell-cell adhesion\n",
      "    - negative regulation of reactive oxygen species biosynthetic process\n",
      "    - negative regulation of hepatocyte apoptotic process\n",
      "    - positive regulation of transformation of host cell by virus\n",
      "    - regulation of fatty acid transport\n",
      "    - positive regulation of ATP biosynthetic process\n",
      "  CC:\n",
      "    - chromatin\n",
      "    - nucleus\n",
      "    - nucleus\n",
      "    - nucleus\n",
      "    - nucleus\n",
      "    - nucleoplasm\n",
      "  MF:\n",
      "    - RNA polymerase II cis-regulatory region sequence-specific DNA binding\n",
      "    - RNA polymerase II cis-regulatory region sequence-specific DNA binding\n",
      "    - DNA-binding transcription factor activity, RNA polymerase II-specific\n",
      "    - DNA-binding transcription activator activity\n",
      "    - transcription coactivator binding\n",
      "    - DNA-binding transcription repressor activity, RNA polymerase II-specific\n",
      "    - DNA-binding transcription repressor activity, RNA polymerase II-specific\n",
      "    - DNA-binding transcription activator activity, RNA polymerase II-specific\n",
      "    - DNA binding\n",
      "    - DNA-binding transcription factor activity\n",
      "    - DNA-binding transcription factor activity\n",
      "    - nuclear steroid receptor activity\n",
      "    - nuclear receptor activity\n",
      "    - nuclear receptor activity\n",
      "    - protein binding\n",
      "    - zinc ion binding\n",
      "    - lipid binding\n",
      "    - phosphatase binding\n",
      "    - protein domain specific binding\n",
      "    - mitogen-activated protein kinase kinase kinase binding\n",
      "    - ubiquitin conjugating enzyme binding\n",
      "    - sequence-specific DNA binding\n",
      "    - protein-containing complex binding\n",
      "    - NFAT protein binding\n",
      "    - RNA polymerase II-specific DNA-binding transcription factor binding\n",
      "    - MDM2/MDM4 family protein binding\n",
      "UniProt: {'Swiss-Prot': 'Q07869', 'TrEMBL': ['B0QYX2', 'F1D8S4', 'B0QYX1', 'Q86SF0']}\n",
      "\n",
      "Gene: IL3RA\n",
      "Symbol: IL3RA\n",
      "Name: interleukin 3 receptor subunit alpha\n",
      "Summary: The protein encoded by this gene is an interleukin 3 specific subunit of a heterodimeric cytokine receptor. The receptor is comprised of a ligand specific alpha subunit and a signal transducing beta subunit shared by the receptors for interleukin 3 (IL3), colony stimulating factor 2 (CSF2/GM-CSF), and interleukin 5 (IL5). The binding of this protein to IL3 depends on the beta subunit. The beta subunit is activated by the ligand binding, and is required for the biological activities of IL3. This gene and the gene encoding the colony stimulating factor 2 receptor alpha chain (CSF2RA) form a cytokine receptor gene cluster in a X-Y pseudoautosomal region on chromosomes X or Y. Alternatively spliced transcript variants encoding distinct isoforms have been found. [provided by RefSeq, Jun 2012].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - positive regulation of cell population proliferation\n",
      "    - cytokine-mediated signaling pathway\n",
      "    - cellular response to interleukin-3\n",
      "    - interleukin-3-mediated signaling pathway\n",
      "  CC:\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - external side of plasma membrane\n",
      "    - receptor complex\n",
      "  MF:\n",
      "    - cytokine receptor activity\n",
      "    - interleukin-3 receptor activity\n",
      "    - protein binding\n",
      "    - cytokine binding\n",
      "UniProt: {'Swiss-Prot': 'P26951', 'TrEMBL': 'J3JS34'}\n",
      "\n",
      "Gene: IFNAR2\n",
      "Symbol: IFNAR2\n",
      "Name: interferon alpha and beta receptor subunit 2\n",
      "Summary: The protein encoded by this gene is a type I membrane protein that forms one of the two chains of a receptor for interferons alpha and beta. Binding and activation of the receptor stimulates Janus protein kinases, which in turn phosphorylate several proteins, including STAT1 and STAT2. The protein belongs to the type II cytokine receptor family. Mutations in this gene are associated with Immunodeficiency 45. [provided by RefSeq, Jul 2020].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - cell surface receptor signaling pathway\n",
      "    - cell surface receptor signaling pathway via JAK-STAT\n",
      "    - response to virus\n",
      "    - response to interferon-alpha\n",
      "    - response to interferon-alpha\n",
      "    - response to interferon-beta\n",
      "    - response to interferon-beta\n",
      "    - cellular response to interferon-beta\n",
      "    - regulation of receptor signaling pathway via JAK-STAT\n",
      "    - defense response to virus\n",
      "    - type I interferon-mediated signaling pathway\n",
      "    - type I interferon-mediated signaling pathway\n",
      "    - type I interferon-mediated signaling pathway\n",
      "    - type I interferon-mediated signaling pathway\n",
      "    - cellular response to virus\n",
      "  CC:\n",
      "    - extracellular region\n",
      "    - extracellular space\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - receptor complex\n",
      "  MF:\n",
      "    - type I interferon receptor activity\n",
      "    - type I interferon receptor activity\n",
      "    - protein binding\n",
      "    - JAK pathway signal transduction adaptor activity\n",
      "    - protein kinase binding\n",
      "    - cytokine binding\n",
      "    - type I interferon binding\n",
      "UniProt: {'Swiss-Prot': 'P48551', 'TrEMBL': ['H7C176', 'Q9BUA0', 'A0A804HLE8', 'C9JM33', 'C9JCU0', 'F8WDE1', 'C9K067']}\n",
      "\n",
      "Gene: VIM\n",
      "Symbol: VIM\n",
      "Name: vimentin\n",
      "Summary: This gene encodes a type III intermediate filament protein. Intermediate filaments, along with microtubules and actin microfilaments, make up the cytoskeleton. The encoded protein is responsible for maintaining cell shape and integrity of the cytoplasm, and stabilizing cytoskeletal interactions. This protein is involved in neuritogenesis and cholesterol transport and functions as an organizer of a number of other critical proteins involved in cell attachment, migration, and signaling. Bacterial and viral pathogens have been shown to attach to this protein on the host cell surface. Mutations in this gene are associated with congenital cataracts in human patients. [provided by RefSeq, Aug 2017].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - positive regulation of gene expression\n",
      "    - negative regulation of neuron projection development\n",
      "    - astrocyte development\n",
      "    - neuron projection development\n",
      "    - positive regulation of collagen biosynthetic process\n",
      "    - regulation of mRNA stability\n",
      "    - intermediate filament organization\n",
      "    - intermediate filament organization\n",
      "    - Bergmann glial cell differentiation\n",
      "    - lens fiber cell development\n",
      "    - cellular response to lipopolysaccharide\n",
      "    - cellular response to muramyl dipeptide\n",
      "    - cellular response to type II interferon\n",
      "  CC:\n",
      "    - cytoplasm\n",
      "    - cytoplasm\n",
      "    - peroxisome\n",
      "    - microtubule organizing center\n",
      "    - cytosol\n",
      "    - cytosol\n",
      "    - cytoskeleton\n",
      "    - cytoskeleton\n",
      "    - intermediate filament\n",
      "    - intermediate filament\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - focal adhesion\n",
      "    - nuclear matrix\n",
      "    - axon\n",
      "    - cell leading edge\n",
      "    - intermediate filament cytoskeleton\n",
      "    - phagocytic vesicle\n",
      "    - extracellular exosome\n",
      "  MF:\n",
      "    - double-stranded RNA binding\n",
      "    - structural constituent of cytoskeleton\n",
      "    - structural constituent of cytoskeleton\n",
      "    - structural constituent of eye lens\n",
      "    - protein binding\n",
      "    - protein domain specific binding\n",
      "    - identical protein binding\n",
      "    - molecular adaptor activity\n",
      "    - scaffold protein binding\n",
      "    - keratin filament binding\n",
      "UniProt: {'Swiss-Prot': 'P08670', 'TrEMBL': ['B0YJC5', 'V9HWE1', 'B0YJC4', 'A0A1B0GTT5', 'A0A1B0GVG8']}\n",
      "\n",
      "Gene: YES1\n",
      "Symbol: YES1\n",
      "Name: YES proto-oncogene 1, Src family tyrosine kinase\n",
      "Summary: This gene is the cellular homolog of the Yamaguchi sarcoma virus oncogene. The encoded protein has tyrosine kinase activity and belongs to the src family of proteins. This gene lies in close proximity to thymidylate synthase gene on chromosome 18, and a corresponding pseudogene has been found on chromosome 22. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - negative regulation of inflammatory response to antigenic stimulus\n",
      "    - cell surface receptor protein tyrosine kinase signaling pathway\n",
      "    - regulation of D-glucose transmembrane transport\n",
      "    - cell differentiation\n",
      "    - T cell costimulation\n",
      "    - cellular response to platelet-derived growth factor stimulus\n",
      "    - protein modification process\n",
      "    - Fc-gamma receptor signaling pathway involved in phagocytosis\n",
      "    - regulation of vascular permeability\n",
      "    - positive regulation of transcription by RNA polymerase II\n",
      "    - ephrin receptor signaling pathway\n",
      "    - leukocyte migration\n",
      "    - cellular response to retinoic acid\n",
      "    - cellular response to transforming growth factor beta stimulus\n",
      "  CC:\n",
      "    - Golgi apparatus\n",
      "    - centrosome\n",
      "    - cytosol\n",
      "    - cytosol\n",
      "    - actin filament\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - focal adhesion\n",
      "    - extracellular exosome\n",
      "  MF:\n",
      "    - phosphotyrosine residue binding\n",
      "    - protein tyrosine kinase activity\n",
      "    - protein tyrosine kinase activity\n",
      "    - protein tyrosine kinase activity\n",
      "    - non-membrane spanning protein tyrosine kinase activity\n",
      "    - non-membrane spanning protein tyrosine kinase activity\n",
      "    - signaling receptor binding\n",
      "    - protein binding\n",
      "    - ATP binding\n",
      "    - enzyme binding\n",
      "    - transmembrane transporter binding\n",
      "UniProt: {'Swiss-Prot': 'P07947', 'TrEMBL': 'J3QRU1'}\n",
      "\n",
      "Gene: ZAP70\n",
      "Symbol: ZAP70\n",
      "Name: zeta chain of T cell receptor associated protein kinase 70\n",
      "Summary: This gene encodes an enzyme belonging to the protein tyrosine kinase family, and it plays a role in T-cell development and lymphocyte activation. This enzyme, which is phosphorylated on tyrosine residues upon T-cell antigen receptor (TCR) stimulation, functions in the initial step of TCR-mediated signal transduction in combination with the Src family kinases, Lck and Fyn. This enzyme is also essential for thymocyte development. Mutations in this gene cause selective T-cell defect, a severe combined immunodeficiency disease characterized by a selective absence of CD8-positive T-cells. Two transcript variants that encode different isoforms have been found for this gene. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - adaptive immune response\n",
      "    - protein phosphorylation\n",
      "    - protein phosphorylation\n",
      "    - immune response\n",
      "    - peptidyl-tyrosine phosphorylation\n",
      "    - peptidyl-tyrosine phosphorylation\n",
      "    - calcium-mediated signaling\n",
      "    - T cell differentiation\n",
      "    - intracellular signal transduction\n",
      "    - T cell activation\n",
      "    - B cell activation\n",
      "    - beta selection\n",
      "    - positive thymic T cell selection\n",
      "    - negative thymic T cell selection\n",
      "    - positive regulation of T cell differentiation\n",
      "    - positive regulation of alpha-beta T cell differentiation\n",
      "    - positive regulation of alpha-beta T cell proliferation\n",
      "    - positive regulation of calcium-mediated signaling\n",
      "    - T cell receptor signaling pathway\n",
      "    - T cell receptor signaling pathway\n",
      "    - T cell aggregation\n",
      "    - T cell migration\n",
      "  CC:\n",
      "    - immunological synapse\n",
      "    - cytoplasm\n",
      "    - cytoplasm\n",
      "    - cytosol\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - cell-cell junction\n",
      "    - T cell receptor complex\n",
      "    - membrane raft\n",
      "  MF:\n",
      "    - phosphotyrosine residue binding\n",
      "    - protein tyrosine kinase activity\n",
      "    - protein tyrosine kinase activity\n",
      "    - protein tyrosine kinase activity\n",
      "    - protein tyrosine kinase activity\n",
      "    - non-membrane spanning protein tyrosine kinase activity\n",
      "    - non-membrane spanning protein tyrosine kinase activity\n",
      "    - protein binding\n",
      "    - ATP binding\n",
      "UniProt: {'Swiss-Prot': 'P43403'}\n",
      "\n",
      "Gene: CD5\n",
      "Symbol: CD5\n",
      "Name: CD5 molecule\n",
      "Summary: This gene encodes a member of the scavenger receptor cysteine-rich (SRCR) superfamily. Members of this family are secreted or membrane-anchored proteins mainly found in cells associated with the immune system. This protein is a type-I transmembrane glycoprotein found on the surface of thymocytes, T lymphocytes and a subset of B lymphocytes. The encoded protein contains three SRCR domains and may act as a receptor to regulate T-cell proliferation. Alternative splicing results in multiple transcript variants encoding different isoforms. [provided by RefSeq, Oct 2016].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - cell recognition\n",
      "    - T cell costimulation\n",
      "    - apoptotic signaling pathway\n",
      "  CC:\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - external side of plasma membrane\n",
      "  MF:\n",
      "    - protein binding\n",
      "    - signaling receptor activity\n",
      "UniProt: {'Swiss-Prot': 'P06127', 'TrEMBL': 'F5GYK3'}\n",
      "\n",
      "Gene: HRAS\n",
      "Symbol: HRAS\n",
      "Name: HRas proto-oncogene, GTPase\n",
      "Summary: This gene belongs to the Ras oncogene family, whose members are related to the transforming genes of mammalian sarcoma retroviruses. The products encoded by these genes function in signal transduction pathways. These proteins can bind GTP and GDP, and they have intrinsic GTPase activity. This protein undergoes a continuous cycle of de- and re-palmitoylation, which regulates its rapid exchange between the plasma membrane and the Golgi apparatus. Mutations in this gene cause Costello syndrome, a disease characterized by increased growth at the prenatal stage, growth deficiency at the postnatal stage, predisposition to tumor formation, cognitive disability, skin and musculoskeletal abnormalities, distinctive facial appearance and cardiovascular abnormalities. Defects in this gene are implicated in a variety of cancers, including bladder cancer, follicular thyroid cancer, and oral squamous cell carcinoma. Multiple transcript variants, which encode different isoforms, have been identified for this gene. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - MAPK cascade\n",
      "    - regulation of transcription by RNA polymerase II\n",
      "    - endocytosis\n",
      "    - chemotaxis\n",
      "    - signal transduction\n",
      "    - cell surface receptor signaling pathway\n",
      "    - Ras protein signal transduction\n",
      "    - Ras protein signal transduction\n",
      "    - positive regulation of cell population proliferation\n",
      "    - negative regulation of cell population proliferation\n",
      "    - insulin receptor signaling pathway\n",
      "    - animal organ morphogenesis\n",
      "    - negative regulation of gene expression\n",
      "    - Schwann cell development\n",
      "    - positive regulation of cell migration\n",
      "    - positive regulation of type II interferon production\n",
      "    - regulation of actin cytoskeleton organization\n",
      "    - negative regulation of GTPase activity\n",
      "    - T-helper 1 type immune response\n",
      "    - regulation of cell population proliferation\n",
      "    - myelination\n",
      "    - defense response to protozoan\n",
      "    - positive regulation of MAP kinase activity\n",
      "    - positive regulation of MAPK cascade\n",
      "    - negative regulation of neuron apoptotic process\n",
      "    - positive regulation of GTPase activity\n",
      "    - positive regulation of transcription by RNA polymerase II\n",
      "    - positive regulation of JNK cascade\n",
      "    - fibroblast proliferation\n",
      "    - positive regulation of fibroblast proliferation\n",
      "    - regulation of long-term neuronal synaptic plasticity\n",
      "    - positive regulation of epithelial cell proliferation\n",
      "    - T cell receptor signaling pathway\n",
      "    - neuron apoptotic process\n",
      "    - regulation of cell cycle\n",
      "    - adipose tissue development\n",
      "    - positive regulation of ERK1 and ERK2 cascade\n",
      "    - cellular response to gamma radiation\n",
      "    - positive regulation of wound healing\n",
      "    - positive regulation of protein targeting to membrane\n",
      "    - cellular senescence\n",
      "    - cellular senescence\n",
      "    - oncogene-induced cell senescence\n",
      "    - intrinsic apoptotic signaling pathway\n",
      "    - regulation of neurotransmitter receptor localization to postsynaptic specialization membrane\n",
      "    - regulation of neurotransmitter receptor localization to postsynaptic specialization membrane\n",
      "    - positive regulation of ruffle assembly\n",
      "    - positive regulation of miRNA metabolic process\n",
      "  CC:\n",
      "    - Golgi membrane\n",
      "    - nucleoplasm\n",
      "    - cytoplasm\n",
      "    - endoplasmic reticulum membrane\n",
      "    - Golgi apparatus\n",
      "    - cytosol\n",
      "    - cytosol\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - ciliary basal body\n",
      "    - perinuclear region of cytoplasm\n",
      "    - glutamatergic synapse\n",
      "    - glutamatergic synapse\n",
      "    - GTPase complex\n",
      "  MF:\n",
      "    - GTPase activity\n",
      "    - GTPase activity\n",
      "    - GTPase activity\n",
      "    - GTPase activity\n",
      "    - G protein activity\n",
      "    - protein binding\n",
      "    - GTP binding\n",
      "    - GTP binding\n",
      "    - GTP binding\n",
      "    - GDP binding\n",
      "    - GDP binding\n",
      "    - protein-membrane adaptor activity\n",
      "    - phospholipase C activator activity\n",
      "UniProt: {'Swiss-Prot': 'P01112', 'TrEMBL': ['A0A8C8MQR2', 'X5D945', 'A0A804HKM6', 'A0A804HJ06', 'A0A0J9YXG8']}\n",
      "\n",
      "Gene: DRD1\n",
      "Symbol: DRD1\n",
      "Name: dopamine receptor D1\n",
      "Summary: This gene encodes the D1 subtype of the dopamine receptor. The D1 subtype is the most abundant dopamine receptor in the central nervous system. This G-protein coupled receptor stimulates adenylyl cyclase and activates cyclic AMP-dependent protein kinases. D1 receptors regulate neuronal growth and development, mediate some behavioral responses, and modulate dopamine receptor D2-mediated events. Alternate transcription initiation sites result in two transcript variants of this gene. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - temperature homeostasis\n",
      "    - conditioned taste aversion\n",
      "    - behavioral fear response\n",
      "    - synaptic transmission, dopaminergic\n",
      "    - response to amphetamine\n",
      "    - protein import into nucleus\n",
      "    - G protein-coupled receptor signaling pathway, coupled to cyclic nucleotide second messenger\n",
      "    - adenylate cyclase-activating G protein-coupled receptor signaling pathway\n",
      "    - activation of adenylate cyclase activity\n",
      "    - adenylate cyclase-activating dopamine receptor signaling pathway\n",
      "    - adenylate cyclase-activating dopamine receptor signaling pathway\n",
      "    - G protein-coupled dopamine receptor signaling pathway\n",
      "    - synapse assembly\n",
      "    - memory\n",
      "    - mating behavior\n",
      "    - grooming behavior\n",
      "    - adult walking behavior\n",
      "    - visual learning\n",
      "    - response to xenobiotic stimulus\n",
      "    - astrocyte development\n",
      "    - dopamine transport\n",
      "    - transmission of nerve impulse\n",
      "    - neuronal action potential\n",
      "    - dentate gyrus development\n",
      "    - striatum development\n",
      "    - cerebral cortex GABAergic interneuron migration\n",
      "    - positive regulation of cell migration\n",
      "    - peristalsis\n",
      "    - operant conditioning\n",
      "    - synaptic transmission, glutamatergic\n",
      "    - regulation of dopamine metabolic process\n",
      "    - vasodilation\n",
      "    - dopamine metabolic process\n",
      "    - maternal behavior\n",
      "    - positive regulation of potassium ion transport\n",
      "    - positive regulation of MAPK cascade\n",
      "    - D-glucose import\n",
      "    - habituation\n",
      "    - sensitization\n",
      "    - behavioral response to cocaine\n",
      "    - positive regulation of release of sequestered calcium ion into cytosol\n",
      "    - regulation of dopamine uptake involved in synaptic transmission\n",
      "    - positive regulation of synaptic transmission, glutamatergic\n",
      "    - prepulse inhibition\n",
      "    - phospholipase C-activating dopamine receptor signaling pathway\n",
      "    - long-term synaptic potentiation\n",
      "    - long-term synaptic depression\n",
      "    - cellular response to catecholamine stimulus\n",
      "    - adenylate cyclase-activating adrenergic receptor signaling pathway\n",
      "    - modification of postsynaptic structure\n",
      "    - presynaptic modulation of chemical synaptic transmission\n",
      "    - positive regulation of neuron migration\n",
      "  CC:\n",
      "    - nucleus\n",
      "    - endoplasmic reticulum membrane\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - cilium\n",
      "    - presynaptic membrane\n",
      "    - dendritic spine\n",
      "    - postsynaptic membrane\n",
      "    - ciliary membrane\n",
      "    - G protein-coupled receptor complex\n",
      "    - non-motile cilium\n",
      "    - glutamatergic synapse\n",
      "    - GABA-ergic synapse\n",
      "  MF:\n",
      "    - dopamine neurotransmitter receptor activity, coupled via Gs\n",
      "    - dopamine neurotransmitter receptor activity, coupled via Gs\n",
      "    - dopamine neurotransmitter receptor activity, coupled via Gs\n",
      "    - G-protein alpha-subunit binding\n",
      "    - G protein-coupled receptor activity\n",
      "    - dopamine neurotransmitter receptor activity\n",
      "    - protein binding\n",
      "    - heterotrimeric G-protein binding\n",
      "    - dopamine binding\n",
      "    - arrestin family protein binding\n",
      "UniProt: {'Swiss-Prot': 'P21728'}\n",
      "\n",
      "Gene: HCK\n",
      "Symbol: HCK\n",
      "Name: HCK proto-oncogene, Src family tyrosine kinase\n",
      "Summary: The protein encoded by this gene is a member of the Src family of tyrosine kinases. This protein is primarily hemopoietic, particularly in cells of the myeloid and B-lymphoid lineages. It may help couple the Fc receptor to the activation of the respiratory burst. In addition, it may play a role in neutrophil migration and in the degranulation of neutrophils. Multiple isoforms with different subcellular distributions are produced due to both alternative splicing and the use of alternative translation initiation codons, including a non-AUG (CUG) codon. [provided by RefSeq, Feb 2010].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - leukocyte migration involved in immune response\n",
      "    - innate immune response-activating signaling pathway\n",
      "    - negative regulation of inflammatory response to antigenic stimulus\n",
      "    - protein phosphorylation\n",
      "    - inflammatory response\n",
      "    - cell adhesion\n",
      "    - cell surface receptor protein tyrosine kinase signaling pathway\n",
      "    - integrin-mediated signaling pathway\n",
      "    - mesoderm development\n",
      "    - positive regulation of cell population proliferation\n",
      "    - regulation of cell shape\n",
      "    - peptidyl-tyrosine phosphorylation\n",
      "    - cytokine-mediated signaling pathway\n",
      "    - cell differentiation\n",
      "    - positive regulation of actin filament polymerization\n",
      "    - lipopolysaccharide-mediated signaling pathway\n",
      "    - regulation of actin cytoskeleton organization\n",
      "    - intracellular signal transduction\n",
      "    - Fc-gamma receptor signaling pathway involved in phagocytosis\n",
      "    - negative regulation of apoptotic process\n",
      "    - leukocyte degranulation\n",
      "    - respiratory burst after phagocytosis\n",
      "    - protein autophosphorylation\n",
      "    - regulation of inflammatory response\n",
      "    - regulation of phagocytosis\n",
      "    - regulation of DNA-binding transcription factor activity\n",
      "    - type II interferon-mediated signaling pathway\n",
      "    - regulation of podosome assembly\n",
      "  CC:\n",
      "    - nucleus\n",
      "    - lysosome\n",
      "    - Golgi apparatus\n",
      "    - cytosol\n",
      "    - cytosol\n",
      "    - actin filament\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - caveola\n",
      "    - focal adhesion\n",
      "    - cytoplasmic side of plasma membrane\n",
      "    - transport vesicle\n",
      "    - cell projection\n",
      "    - intracellular membrane-bounded organelle\n",
      "  MF:\n",
      "    - phosphotyrosine residue binding\n",
      "    - protein tyrosine kinase activity\n",
      "    - protein tyrosine kinase activity\n",
      "    - protein tyrosine kinase activity\n",
      "    - non-membrane spanning protein tyrosine kinase activity\n",
      "    - signaling receptor binding\n",
      "    - protein binding\n",
      "    - ATP binding\n",
      "    - lipid binding\n",
      "UniProt: {'Swiss-Prot': 'P08631', 'TrEMBL': ['H7BXG4', 'A8K4G3', 'H7C5P9']}\n",
      "\n",
      "Gene: BTK\n",
      "Symbol: BTK\n",
      "Name: Bruton tyrosine kinase\n",
      "Summary: The protein encoded by this gene plays a crucial role in B-cell development. Mutations in this gene cause X-linked agammaglobulinemia type 1, which is an immunodeficiency characterized by the failure to produce mature B lymphocytes, and associated with a failure of Ig heavy chain rearrangement. Alternative splicing results in multiple transcript variants encoding different isoforms. [provided by RefSeq, Dec 2013].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - neutrophil homeostasis\n",
      "    - positive regulation of type III hypersensitivity\n",
      "    - positive regulation of type I hypersensitivity\n",
      "    - adaptive immune response\n",
      "    - adaptive immune response\n",
      "    - B cell affinity maturation\n",
      "    - histamine secretion by mast cell\n",
      "    - positive regulation of immunoglobulin production\n",
      "    - regulation of B cell cytokine production\n",
      "    - MyD88-dependent toll-like receptor signaling pathway\n",
      "    - regulation of B cell apoptotic process\n",
      "    - protein phosphorylation\n",
      "    - mesoderm development\n",
      "    - peptidyl-tyrosine phosphorylation\n",
      "    - peptidyl-tyrosine phosphorylation\n",
      "    - calcium-mediated signaling\n",
      "    - proteoglycan catabolic process\n",
      "    - negative regulation of B cell proliferation\n",
      "    - positive regulation of B cell proliferation\n",
      "    - response to lipopolysaccharide\n",
      "    - negative regulation of interleukin-10 production\n",
      "    - positive regulation of interleukin-6 production\n",
      "    - positive regulation of tumor necrosis factor production\n",
      "    - cellular response to reactive oxygen species\n",
      "    - intracellular signal transduction\n",
      "    - intracellular signal transduction\n",
      "    - intracellular signal transduction\n",
      "    - Fc-epsilon receptor signaling pathway\n",
      "    - B cell activation\n",
      "    - innate immune response\n",
      "    - positive regulation of B cell differentiation\n",
      "    - cell maturation\n",
      "    - positive regulation of phagocytosis\n",
      "    - T cell receptor signaling pathway\n",
      "    - B cell receptor signaling pathway\n",
      "    - B cell receptor signaling pathway\n",
      "    - B cell receptor signaling pathway\n",
      "    - positive regulation of NF-kappaB transcription factor activity\n",
      "    - monocyte proliferation\n",
      "    - cellular response to molecule of fungal origin\n",
      "    - apoptotic signaling pathway\n",
      "    - cellular response to interleukin-7\n",
      "    - positive regulation of cGAS/STING signaling pathway\n",
      "    - positive regulation of interleukin-17A production\n",
      "    - positive regulation of NLRP3 inflammasome complex assembly\n",
      "    - positive regulation of synoviocyte proliferation\n",
      "    - eosinophil homeostasis\n",
      "  CC:\n",
      "    - nucleus\n",
      "    - cytoplasm\n",
      "    - cytosol\n",
      "    - cytosol\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - cytoplasmic vesicle\n",
      "    - membrane raft\n",
      "    - perinuclear region of cytoplasm\n",
      "  MF:\n",
      "    - protein tyrosine kinase activity\n",
      "    - protein tyrosine kinase activity\n",
      "    - protein tyrosine kinase activity\n",
      "    - non-membrane spanning protein tyrosine kinase activity\n",
      "    - non-membrane spanning protein tyrosine kinase activity\n",
      "    - non-membrane spanning protein tyrosine kinase activity\n",
      "    - non-membrane spanning protein tyrosine kinase activity\n",
      "    - protein binding\n",
      "    - ATP binding\n",
      "    - phosphatidylinositol-3,4,5-trisphosphate binding\n",
      "    - zinc ion binding\n",
      "    - phospholipase activator activity\n",
      "    - identical protein binding\n",
      "    - phospholipase binding\n",
      "UniProt: {'Swiss-Prot': 'Q06187', 'TrEMBL': ['Q3MS97', 'Q5JY90', 'Q3MS94', 'Q3MS88', 'A0A8Q3WLR2', 'Q3MS90', 'A0A8Q3SHX9', 'A0A8Q3SHZ6', 'A0A8Q3WKN1', 'A0A8Q3WKM0', 'A0A8Q3WLZ5', 'A0A8Q3WL62', 'A0A8Q3WKS7', 'A0A8Q3SI38', 'A0A8Q3SI35', 'A0A8Q3SI44', 'A0A8Q3WKR7']}\n",
      "\n",
      "Gene: PIP4K2B\n",
      "Symbol: PIP4K2B\n",
      "Name: phosphatidylinositol-5-phosphate 4-kinase type 2 beta\n",
      "Summary: The protein encoded by this gene catalyzes the phosphorylation of phosphatidylinositol-5-phosphate on the fourth hydroxyl of the myo-inositol ring to form phosphatidylinositol-5,4-bisphosphate. This gene is a member of the phosphatidylinositol-5-phosphate 4-kinase family. The encoded protein sequence does not show similarity to other kinases, but the protein does exhibit kinase activity. Additionally, the encoded protein interacts with p55 TNF receptor. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - cell surface receptor signaling pathway\n",
      "    - regulation of autophagy\n",
      "    - regulation of autophagy\n",
      "    - negative regulation of insulin receptor signaling pathway\n",
      "    - phosphatidylinositol phosphate biosynthetic process\n",
      "    - autophagosome-lysosome fusion\n",
      "    - 1-phosphatidyl-1D-myo-inositol 4,5-bisphosphate biosynthetic process\n",
      "    - positive regulation of autophagosome assembly\n",
      "  CC:\n",
      "    - nucleus\n",
      "    - nucleoplasm\n",
      "    - nucleoplasm\n",
      "    - autophagosome\n",
      "    - endoplasmic reticulum membrane\n",
      "    - cytosol\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "  MF:\n",
      "    - protein binding\n",
      "    - ATP binding\n",
      "    - GTP binding\n",
      "    - 1-phosphatidylinositol-4-phosphate 5-kinase activity\n",
      "    - 1-phosphatidylinositol-5-phosphate 4-kinase activity\n",
      "    - protein homodimerization activity\n",
      "UniProt: {'Swiss-Prot': 'P78356', 'TrEMBL': 'A0A087WT35'}\n",
      "\n",
      "Gene: CD22\n",
      "Symbol: CD22\n",
      "Name: CD22 molecule\n",
      "Summary: Predicted to enable CD4 receptor binding activity; protein phosphatase binding activity; and sialic acid binding activity. Involved in B cell activation; negative regulation of B cell receptor signaling pathway; and regulation of endocytosis. Located in early endosome and recycling endosome. [provided by Alliance of Genome Resources, Apr 2022]\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - negative regulation of immunoglobulin production\n",
      "    - cell adhesion\n",
      "    - regulation of endocytosis\n",
      "    - regulation of B cell proliferation\n",
      "    - regulation of B cell proliferation\n",
      "    - B cell activation\n",
      "    - B cell activation\n",
      "    - regulation of immune response\n",
      "    - negative regulation of calcium-mediated signaling\n",
      "    - negative regulation of B cell receptor signaling pathway\n",
      "    - negative regulation of B cell receptor signaling pathway\n",
      "    - negative regulation of B cell receptor signaling pathway\n",
      "    - negative regulation of B cell receptor signaling pathway\n",
      "  CC:\n",
      "    - cytoplasm\n",
      "    - early endosome\n",
      "    - early endosome\n",
      "    - plasma membrane\n",
      "    - external side of plasma membrane\n",
      "    - cell surface\n",
      "    - membrane\n",
      "    - neuronal cell body membrane\n",
      "    - recycling endosome\n",
      "    - recycling endosome\n",
      "    - extracellular exosome\n",
      "  MF:\n",
      "    - IgM binding\n",
      "    - signaling receptor binding\n",
      "    - protein binding\n",
      "    - protein phosphatase binding\n",
      "    - protein phosphatase binding\n",
      "    - protein phosphatase binding\n",
      "    - carbohydrate binding\n",
      "    - sialic acid binding\n",
      "    - sialic acid binding\n",
      "    - sialic acid binding\n",
      "    - CD4 receptor binding\n",
      "    - CD4 receptor binding\n",
      "UniProt: {'Swiss-Prot': 'P20273', 'TrEMBL': ['M0QZ01', 'Q0EAF5', 'M0QYP4', 'M0R2R8', 'M0R3H1', 'M0QZR7', 'M0QZP5', 'M0QY05', 'M0R2M0', 'M0QY14', 'M0R1M2', 'M0R0R6', 'A0A087WZQ4']}\n",
      "\n",
      "Gene: PIP5K1A\n",
      "Symbol: PIP5K1A\n",
      "Name: phosphatidylinositol-4-phosphate 5-kinase type 1 alpha\n",
      "Summary: Enables 1-phosphatidylinositol-4-phosphate 5-kinase activity and kinase binding activity. Involved in several processes, including activation of GTPase activity; focal adhesion assembly; and ruffle assembly. Located in several cellular components, including lamellipodium; nuclear speck; and ruffle membrane. Colocalizes with mRNA cleavage and polyadenylation specificity factor complex. [provided by Alliance of Genome Resources, Apr 2022]\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - glycerophospholipid metabolic process\n",
      "    - phosphatidylinositol biosynthetic process\n",
      "    - phagocytosis\n",
      "    - signal transduction\n",
      "    - phospholipid biosynthetic process\n",
      "    - fibroblast migration\n",
      "    - cell migration\n",
      "    - actin cytoskeleton organization\n",
      "    - keratinocyte differentiation\n",
      "    - phosphatidylinositol phosphate biosynthetic process\n",
      "    - focal adhesion assembly\n",
      "    - regulation of phosphatidylinositol 3-kinase/protein kinase B signal transduction\n",
      "    - cell chemotaxis\n",
      "    - protein localization to plasma membrane\n",
      "    - activation of GTPase activity\n",
      "    - ruffle assembly\n",
      "  CC:\n",
      "    - nucleus\n",
      "    - nucleoplasm\n",
      "    - cytosol\n",
      "    - cytosol\n",
      "    - mRNA cleavage and polyadenylation specificity factor complex\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - focal adhesion\n",
      "    - nuclear speck\n",
      "    - nuclear speck\n",
      "    - lamellipodium\n",
      "    - ruffle membrane\n",
      "  MF:\n",
      "    - 1-phosphatidylinositol-3-phosphate 5-kinase activity\n",
      "    - protein binding\n",
      "    - ATP binding\n",
      "    - 1-phosphatidylinositol-4-phosphate 5-kinase activity\n",
      "    - 1-phosphatidylinositol-4-phosphate 5-kinase activity\n",
      "    - 1-phosphatidylinositol-4-phosphate 5-kinase activity\n",
      "    - kinase binding\n",
      "    - phosphatidylinositol kinase activity\n",
      "    - 1-phosphatidylinositol-5-kinase activity\n",
      "UniProt: {'Swiss-Prot': 'Q99755', 'TrEMBL': ['E9PSF8', 'A6PW57', 'A6PW56', 'A6PW58', 'A2A5X0']}\n",
      "\n",
      "Gene: BLK\n",
      "Symbol: BLK\n",
      "Name: BLK proto-oncogene, Src family tyrosine kinase\n",
      "Summary: This gene encodes a nonreceptor tyrosine-kinase of the src family of proto-oncogenes that are typically involved in cell proliferation and differentiation. The protein has a role in B-cell receptor signaling and B-cell development. The protein also stimulates insulin synthesis and secretion in response to glucose and enhances the expression of several pancreatic beta-cell transcription factors. [provided by RefSeq, Aug 2010].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - cell surface receptor protein tyrosine kinase signaling pathway\n",
      "    - peptidyl-tyrosine phosphorylation\n",
      "    - cell differentiation\n",
      "    - positive regulation of insulin secretion\n",
      "    - positive regulation of protein binding\n",
      "    - intracellular signal transduction\n",
      "    - B cell receptor signaling pathway\n",
      "    - B cell receptor signaling pathway\n",
      "  CC:\n",
      "    - cytosol\n",
      "    - plasma membrane\n",
      "  MF:\n",
      "    - protein tyrosine kinase activity\n",
      "    - non-membrane spanning protein tyrosine kinase activity\n",
      "    - non-membrane spanning protein tyrosine kinase activity\n",
      "    - non-membrane spanning protein tyrosine kinase activity\n",
      "    - signaling receptor binding\n",
      "    - protein binding\n",
      "    - ATP binding\n",
      "UniProt: {'Swiss-Prot': 'P51451', 'TrEMBL': 'Q05D26'}\n",
      "\n",
      "Gene: GNAS\n",
      "Symbol: GNAS\n",
      "Name: GNAS complex locus\n",
      "Summary: This locus has a highly complex imprinted expression pattern. It gives rise to maternally, paternally, and biallelically expressed transcripts that are derived from four alternative promoters and 5' exons. Some transcripts contain a differentially methylated region (DMR) at their 5' exons, and this DMR is commonly found in imprinted genes and correlates with transcript expression. An antisense transcript is produced from an overlapping locus on the opposite strand. One of the transcripts produced from this locus, and the antisense transcript, are paternally expressed noncoding RNAs, and may regulate imprinting in this region. In addition, one of the transcripts contains a second overlapping ORF, which encodes a structurally unrelated protein - Alex. Alternative splicing of downstream exons is also observed, which results in different forms of the stimulatory G-protein alpha subunit, a key element of the classical signal transduction pathway linking receptor-ligand interactions with the activation of adenylyl cyclase and a variety of cellular reponses. Multiple transcript variants encoding different isoforms have been found for this gene. Mutations in this gene result in pseudohypoparathyroidism type 1a, pseudohypoparathyroidism type 1b, Albright hereditary osteodystrophy, pseudopseudohypoparathyroidism, McCune-Albright syndrome, progressive osseus heteroplasia, polyostotic fibrous dysplasia of bone, and some pituitary tumors. [provided by RefSeq, Aug 2012].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - negative regulation of inflammatory response to antigenic stimulus\n",
      "    - renal water homeostasis\n",
      "    - G protein-coupled receptor signaling pathway\n",
      "    - adenylate cyclase-activating G protein-coupled receptor signaling pathway\n",
      "    - adenylate cyclase-activating G protein-coupled receptor signaling pathway\n",
      "    - activation of adenylate cyclase activity\n",
      "    - adenylate cyclase-activating dopamine receptor signaling pathway\n",
      "    - adenylate cyclase-activating dopamine receptor signaling pathway\n",
      "    - adenylate cyclase-activating serotonin receptor signaling pathway\n",
      "    - female pregnancy\n",
      "    - sensory perception of chemical stimulus\n",
      "    - sensory perception of smell\n",
      "    - protein secretion\n",
      "    - regulation of signal transduction\n",
      "    - response to prostaglandin E\n",
      "    - negative regulation of multicellular organism growth\n",
      "    - intracellular transport\n",
      "    - developmental growth\n",
      "    - developmental growth\n",
      "    - regulation of insulin secretion\n",
      "    - cognition\n",
      "    - cognition\n",
      "    - bone development\n",
      "    - bone development\n",
      "    - hair follicle placode formation\n",
      "    - hair follicle placode formation\n",
      "    - platelet aggregation\n",
      "    - platelet aggregation\n",
      "    - response to parathyroid hormone\n",
      "    - cellular response to glucagon stimulus\n",
      "    - cellular response to prostaglandin E stimulus\n",
      "    - cellular response to catecholamine stimulus\n",
      "    - adenylate cyclase-activating adrenergic receptor signaling pathway\n",
      "    - vascular endothelial cell response to laminar fluid shear stress\n",
      "    - positive regulation of cold-induced thermogenesis\n",
      "  CC:\n",
      "    - ruffle\n",
      "    - extracellular region\n",
      "    - nucleus\n",
      "    - cytoplasm\n",
      "    - cytoplasm\n",
      "    - cytoplasm\n",
      "    - cytosol\n",
      "    - cytosol\n",
      "    - heterotrimeric G-protein complex\n",
      "    - heterotrimeric G-protein complex\n",
      "    - heterotrimeric G-protein complex\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - membrane\n",
      "    - membrane\n",
      "    - membrane\n",
      "    - apical plasma membrane\n",
      "    - transport vesicle\n",
      "    - trans-Golgi network membrane\n",
      "    - perinuclear region of cytoplasm\n",
      "    - extracellular exosome\n",
      "  MF:\n",
      "    - GTPase activity\n",
      "    - GTPase activity\n",
      "    - G protein activity\n",
      "    - G protein activity\n",
      "    - insulin-like growth factor receptor binding\n",
      "    - protein binding\n",
      "    - GTP binding\n",
      "    - adenylate cyclase regulator activity\n",
      "    - adenylate cyclase activator activity\n",
      "    - adenylate cyclase activator activity\n",
      "    - G-protein beta/gamma-subunit complex binding\n",
      "    - G-protein beta/gamma-subunit complex binding\n",
      "    - beta-2 adrenergic receptor binding\n",
      "    - D1 dopamine receptor binding\n",
      "    - D1 dopamine receptor binding\n",
      "    - mu-type opioid receptor binding\n",
      "    - ionotropic glutamate receptor binding\n",
      "    - metal ion binding\n",
      "    - corticotropin-releasing hormone receptor 1 binding\n",
      "UniProt: {'Swiss-Prot': ['Q5JWF2', 'P84996', 'P63092', 'O95467'], 'TrEMBL': ['B0AZR9', 'A0A590UJF0', 'A0A0S2Z3S5', 'Q5JWD1', 'Q14455', 'A0A7I2V5R6', 'Q5FWY2', 'A0A0S2Z3H8', 'A0A590UJX3', 'A0A590UJ47', 'A0A590UK28', 'A0A590UJ46', 'A0A590UJQ9', 'A0A8I5F5B5', 'A0A0A0MR13', 'A2A2R6', 'A0A590UJS2', 'A0A590UJX6', 'H0Y7E8', 'A0A590UJC9', 'A0A590UJY2', 'A0A804HIH4', 'Q5JWE9']}\n",
      "\n",
      "Gene: GNA11\n",
      "Symbol: GNA11\n",
      "Name: G protein subunit alpha 11\n",
      "Summary: The protein encoded by this gene belongs to the family of guanine nucleotide-binding proteins (G proteins), which function as modulators or transducers in various transmembrane signaling systems. G proteins are composed of 3 units: alpha, beta and gamma. This gene encodes one of the alpha subunits (subunit alpha-11). Mutations in this gene have been associated with hypocalciuric hypercalcemia type II (HHC2) and hypocalcemia dominant 2 (HYPOC2). Patients with HHC2 and HYPOC2 exhibit decreased or increased sensitivity, respectively, to changes in extracellular calcium concentrations. [provided by RefSeq, Dec 2013].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - skeletal system development\n",
      "    - action potential\n",
      "    - signal transduction\n",
      "    - adenylate cyclase-modulating G protein-coupled receptor signaling pathway\n",
      "    - phospholipase C-activating G protein-coupled acetylcholine receptor signaling pathway\n",
      "    - G protein-coupled acetylcholine receptor signaling pathway\n",
      "    - heart development\n",
      "    - phototransduction, visible light\n",
      "    - regulation of blood pressure\n",
      "    - entrainment of circadian clock\n",
      "    - positive regulation of insulin secretion\n",
      "    - regulation of melanocyte differentiation\n",
      "    - developmental pigmentation\n",
      "    - phospholipase C-activating dopamine receptor signaling pathway\n",
      "    - cellular response to pH\n",
      "    - endothelin receptor signaling pathway\n",
      "    - ion channel modulating, G protein-coupled receptor signaling pathway\n",
      "    - cranial skeletal system development\n",
      "    - ligand-gated ion channel signaling pathway\n",
      "  CC:\n",
      "    - photoreceptor outer segment\n",
      "    - cytoplasm\n",
      "    - cytoplasm\n",
      "    - lysosomal membrane\n",
      "    - heterotrimeric G-protein complex\n",
      "    - plasma membrane\n",
      "    - synapse\n",
      "    - extracellular exosome\n",
      "  MF:\n",
      "    - G protein-coupled receptor binding\n",
      "    - GTPase activity\n",
      "    - GTPase activity\n",
      "    - G protein activity\n",
      "    - protein binding\n",
      "    - GTP binding\n",
      "    - enzyme regulator activity\n",
      "    - G-protein beta/gamma-subunit complex binding\n",
      "    - metal ion binding\n",
      "UniProt: {'Swiss-Prot': 'P29992', 'TrEMBL': ['A0A087WVZ3', 'K7EL62']}\n",
      "\n",
      "Gene: IRAK4\n",
      "Symbol: IRAK4\n",
      "Name: interleukin 1 receptor associated kinase 4\n",
      "Summary: This gene encodes a kinase that activates NF-kappaB in both the Toll-like receptor (TLR) and T-cell receptor (TCR) signaling pathways. The protein is essential for most innate immune responses. Mutations in this gene result in IRAK4 deficiency and recurrent invasive pneumococcal disease. Multiple transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Aug 2011].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - toll-like receptor signaling pathway\n",
      "    - neutrophil mediated immunity\n",
      "    - MyD88-dependent toll-like receptor signaling pathway\n",
      "    - chromatin remodeling\n",
      "    - JNK cascade\n",
      "    - Toll signaling pathway\n",
      "    - cytokine-mediated signaling pathway\n",
      "    - lipopolysaccharide-mediated signaling pathway\n",
      "    - toll-like receptor 4 signaling pathway\n",
      "    - toll-like receptor 9 signaling pathway\n",
      "    - intracellular signal transduction\n",
      "    - interleukin-33-mediated signaling pathway\n",
      "    - positive regulation of canonical NF-kappaB signal transduction\n",
      "    - positive regulation of canonical NF-kappaB signal transduction\n",
      "    - innate immune response\n",
      "    - positive regulation of smooth muscle cell proliferation\n",
      "    - interleukin-1-mediated signaling pathway\n",
      "    - interleukin-1-mediated signaling pathway\n",
      "    - neutrophil migration\n",
      "  CC:\n",
      "    - extracellular space\n",
      "    - nucleus\n",
      "    - cytoplasm\n",
      "    - cytoplasm\n",
      "    - cytosol\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - cell surface\n",
      "    - endosome membrane\n",
      "    - extrinsic component of plasma membrane\n",
      "  MF:\n",
      "    - magnesium ion binding\n",
      "    - protein serine/threonine kinase activity\n",
      "    - protein serine/threonine kinase activity\n",
      "    - protein serine/threonine kinase activity\n",
      "    - 3-phosphoinositide-dependent protein kinase activity\n",
      "    - DNA-dependent protein kinase activity\n",
      "    - AMP-activated protein kinase activity\n",
      "    - eukaryotic translation initiation factor 2alpha kinase activity\n",
      "    - ribosomal protein S6 kinase activity\n",
      "    - interleukin-1 receptor binding\n",
      "    - protein binding\n",
      "    - ATP binding\n",
      "    - kinase activity\n",
      "    - protein kinase binding\n",
      "    - histone H3S10 kinase activity\n",
      "    - histone H3T11 kinase activity\n",
      "    - histone H3T6 kinase activity\n",
      "    - histone H2AXS139 kinase activity\n",
      "    - histone H3S28 kinase activity\n",
      "    - histone H4S1 kinase activity\n",
      "    - histone H2AS1 kinase activity\n",
      "    - histone H2BS14 kinase activity\n",
      "    - histone H3T3 kinase activity\n",
      "    - histone H2AS121 kinase activity\n",
      "    - Rho-dependent protein serine/threonine kinase activity\n",
      "    - protein serine kinase activity\n",
      "    - histone H2BS36 kinase activity\n",
      "    - histone H3S57 kinase activity\n",
      "    - histone H3T45 kinase activity\n",
      "    - histone H2AT120 kinase activity\n",
      "UniProt: {'Swiss-Prot': 'Q9NWZ3', 'TrEMBL': ['B2RAP9', 'Q69FE3', 'B4E359', 'F8VVZ1', 'F8VW24', 'F8VR40', 'A0A8Q3SIY0', 'A0A8Q3SIT8']}\n",
      "\n",
      "Gene: XPO1\n",
      "Symbol: XPO1\n",
      "Name: exportin 1\n",
      "Summary: This cell-cycle-regulated gene encodes a protein that mediates leucine-rich nuclear export signal (NES)-dependent protein transport. The protein specifically inhibits the nuclear export of Rev and U snRNAs. It is involved in the control of several cellular processes by controlling the localization of cyclin B, MPAK, and MAPKAP kinase 2. This protein also regulates NFAT and AP-1. [provided by RefSeq, Jan 2015].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - ribosomal subunit export from nucleus\n",
      "    - ribosomal large subunit export from nucleus\n",
      "    - ribosomal large subunit export from nucleus\n",
      "    - ribosomal small subunit export from nucleus\n",
      "    - ribosomal small subunit export from nucleus\n",
      "    - mRNA export from nucleus\n",
      "    - protein export from nucleus\n",
      "    - protein export from nucleus\n",
      "    - protein export from nucleus\n",
      "    - protein export from nucleus\n",
      "    - protein export from nucleus\n",
      "    - nucleocytoplasmic transport\n",
      "    - regulation of centrosome duplication\n",
      "    - regulation of proteasomal ubiquitin-dependent protein catabolic process\n",
      "    - protein localization to nucleus\n",
      "    - ribosome biogenesis\n",
      "    - regulation of protein export from nucleus\n",
      "  CC:\n",
      "    - kinetochore\n",
      "    - nucleus\n",
      "    - nuclear envelope\n",
      "    - annulate lamellae\n",
      "    - nucleoplasm\n",
      "    - nucleoplasm\n",
      "    - nucleolus\n",
      "    - cytoplasm\n",
      "    - cytoplasm\n",
      "    - cytosol\n",
      "    - cytosol\n",
      "    - Cajal body\n",
      "    - membrane\n",
      "    - nuclear membrane\n",
      "    - protein-containing complex\n",
      "    - intracellular membrane-bounded organelle\n",
      "    - ribonucleoprotein complex\n",
      "  MF:\n",
      "    - RNA binding\n",
      "    - nuclear export signal receptor activity\n",
      "    - nuclear export signal receptor activity\n",
      "    - nuclear export signal receptor activity\n",
      "    - nuclear export signal receptor activity\n",
      "    - nuclear export signal receptor activity\n",
      "    - protein binding\n",
      "    - small GTPase binding\n",
      "UniProt: {'Swiss-Prot': 'O14980', 'TrEMBL': ['A0A7I2V6B9', 'A0A7I2V2Y6', 'B3KWD0', 'A0A7I2V461', 'A0A7I2V3N0', 'A0A7I2YQP1', 'A0A7I2V2C5', 'A0A7I2V3W6', 'A0A7I2V4A3', 'A0A7I2V2S3', 'C9JKM9', 'A0A7I2V531', 'A0A7I2V396', 'A0A7P0Z4B7', 'A0A7I2V488', 'F8WF71', 'A0A7I2V2H0', 'A0A7I2YQX3', 'C9IYM2', 'A0A7I2YQV4', 'C9J673', 'A0A7I2V3P3', 'A0A7I2V3J1']}\n",
      "\n",
      "Gene: ROCK1\n",
      "Symbol: ROCK1\n",
      "Name: Rho associated coiled-coil containing protein kinase 1\n",
      "Summary: This gene encodes a protein serine/threonine kinase that is activated when bound to the GTP-bound form of Rho. The small GTPase Rho regulates formation of focal adhesions and stress fibers of fibroblasts, as well as adhesion and aggregation of platelets and lymphocytes by shuttling between the inactive GDP-bound form and the active GTP-bound form. Rho is also essential in cytokinesis and plays a role in transcriptional activation by serum response factor. This protein, a downstream effector of Rho, phosphorylates and activates LIM kinase, which in turn, phosphorylates cofilin, inhibiting its actin-depolymerizing activity. A pseudogene, related to this gene, is also located on chromosome 18. [provided by RefSeq, Aug 2015].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - mitotic cytokinesis\n",
      "    - epithelial to mesenchymal transition\n",
      "    - aortic valve morphogenesis\n",
      "    - apical constriction\n",
      "    - chromatin remodeling\n",
      "    - smooth muscle contraction\n",
      "    - leukocyte cell-cell adhesion\n",
      "    - signal transduction\n",
      "    - Rho protein signal transduction\n",
      "    - Rho protein signal transduction\n",
      "    - positive regulation of autophagy\n",
      "    - positive regulation of cardiac muscle hypertrophy\n",
      "    - positive regulation of gene expression\n",
      "    - positive regulation of gene expression\n",
      "    - positive regulation of phosphatase activity\n",
      "    - negative regulation of angiogenesis\n",
      "    - peptidyl-serine phosphorylation\n",
      "    - membrane to membrane docking\n",
      "    - actin cytoskeleton organization\n",
      "    - regulation of cell adhesion\n",
      "    - regulation of cell migration\n",
      "    - cortical actin cytoskeleton organization\n",
      "    - actomyosin structure organization\n",
      "    - neuron projection development\n",
      "    - bleb assembly\n",
      "    - negative regulation of protein binding\n",
      "    - regulation of actin cytoskeleton organization\n",
      "    - regulation of actin cytoskeleton organization\n",
      "    - regulation of actin cytoskeleton organization\n",
      "    - regulation of actin cytoskeleton organization\n",
      "    - positive regulation of dephosphorylation\n",
      "    - positive regulation of MAPK cascade\n",
      "    - regulation of keratinocyte differentiation\n",
      "    - regulation of neuron differentiation\n",
      "    - embryonic morphogenesis\n",
      "    - leukocyte migration\n",
      "    - leukocyte tethering or rolling\n",
      "    - negative regulation of membrane protein ectodomain proteolysis\n",
      "    - myoblast migration\n",
      "    - regulation of stress fiber assembly\n",
      "    - regulation of focal adhesion assembly\n",
      "    - positive regulation of focal adhesion assembly\n",
      "    - mRNA destabilization\n",
      "    - negative regulation of biomineral tissue development\n",
      "    - regulation of microtubule cytoskeleton organization\n",
      "    - response to transforming growth factor beta\n",
      "    - protein localization to plasma membrane\n",
      "    - protein localization to plasma membrane\n",
      "    - regulation of synapse maturation\n",
      "    - podocyte cell migration\n",
      "    - motor neuron apoptotic process\n",
      "    - blood vessel diameter maintenance\n",
      "    - regulation of angiotensin-activated signaling pathway\n",
      "    - neuron projection arborization\n",
      "    - positive regulation of amyloid-beta clearance\n",
      "    - regulation of synaptic vesicle endocytosis\n",
      "    - regulation of cell junction assembly\n",
      "    - negative regulation of amyloid-beta formation\n",
      "    - negative regulation of amyloid precursor protein catabolic process\n",
      "    - regulation of establishment of endothelial barrier\n",
      "    - regulation of establishment of endothelial barrier\n",
      "    - negative regulation of bicellular tight junction assembly\n",
      "    - positive regulation of connective tissue replacement\n",
      "    - response to angiotensin\n",
      "    - regulation of establishment of cell polarity\n",
      "    - regulation of cell motility\n",
      "    - negative regulation of motor neuron apoptotic process\n",
      "  CC:\n",
      "    - Golgi membrane\n",
      "    - ruffle\n",
      "    - extracellular region\n",
      "    - cytoplasm\n",
      "    - centriole\n",
      "    - cytosol\n",
      "    - cytoskeleton\n",
      "    - cytoskeleton\n",
      "    - plasma membrane\n",
      "    - cytoplasmic stress granule\n",
      "    - cytoplasmic stress granule\n",
      "    - lamellipodium\n",
      "    - bleb\n",
      "    - secretory granule lumen\n",
      "    - Schaffer collateral - CA1 synapse\n",
      "  MF:\n",
      "    - protein serine/threonine kinase activity\n",
      "    - protein serine/threonine kinase activity\n",
      "    - protein serine/threonine kinase activity\n",
      "    - 3-phosphoinositide-dependent protein kinase activity\n",
      "    - DNA-dependent protein kinase activity\n",
      "    - AMP-activated protein kinase activity\n",
      "    - eukaryotic translation initiation factor 2alpha kinase activity\n",
      "    - ribosomal protein S6 kinase activity\n",
      "    - protein binding\n",
      "    - ATP binding\n",
      "    - zinc ion binding\n",
      "    - small GTPase binding\n",
      "    - histone H3S10 kinase activity\n",
      "    - histone H3T11 kinase activity\n",
      "    - histone H3T6 kinase activity\n",
      "    - histone H2AXS139 kinase activity\n",
      "    - histone H3S28 kinase activity\n",
      "    - histone H4S1 kinase activity\n",
      "    - histone H2AS1 kinase activity\n",
      "    - histone H2BS14 kinase activity\n",
      "    - tau protein binding\n",
      "    - histone H3T3 kinase activity\n",
      "    - histone H2AS121 kinase activity\n",
      "    - Rho-dependent protein serine/threonine kinase activity\n",
      "    - Rho-dependent protein serine/threonine kinase activity\n",
      "    - protein serine kinase activity\n",
      "    - histone H2BS36 kinase activity\n",
      "    - histone H3S57 kinase activity\n",
      "    - histone H3T45 kinase activity\n",
      "    - histone H2AT120 kinase activity\n",
      "UniProt: {'Swiss-Prot': 'Q13464', 'TrEMBL': ['A0A0U1RQV4', 'J3KRH6']}\n",
      "\n",
      "Gene: IL2RA\n",
      "Symbol: IL2RA\n",
      "Name: interleukin 2 receptor subunit alpha\n",
      "Summary: The interleukin 2 (IL2) receptor alpha (IL2RA) and beta (IL2RB) chains, together with the common gamma chain (IL2RG), constitute the high-affinity IL2 receptor. Homodimeric alpha chains (IL2RA) result in low-affinity receptor, while homodimeric beta (IL2RB) chains produce a medium-affinity receptor. Normally an integral-membrane protein, soluble IL2RA has been isolated and determined to result from extracellular proteolyisis. Alternately-spliced IL2RA mRNAs have been isolated, but the significance of each is presently unknown. Mutations in this gene are associated with interleukin 2 receptor alpha deficiency. Patients with severe Coronavirus Disease 2019 (COVID-19), the disease caused by the novel severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), have significantly elevated levels of IL2R in their plasma. Similarly, serum IL-2R levels are found to be elevated in patients with different types of carcinomas. Certain IL2RA and IL2RB gene polymorphisms have been associated with lung cancer risk. [provided by RefSeq, Jul 2020].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - inflammatory response to antigenic stimulus\n",
      "    - regulation of T cell tolerance induction\n",
      "    - apoptotic process\n",
      "    - activation-induced cell death of T cells\n",
      "    - inflammatory response\n",
      "    - immune response\n",
      "    - cell surface receptor signaling pathway\n",
      "    - Notch signaling pathway\n",
      "    - interleukin-2-mediated signaling pathway\n",
      "    - positive regulation of activated T cell proliferation\n",
      "    - negative regulation of T cell proliferation\n",
      "    - positive regulation of T cell differentiation\n",
      "    - regulation of T cell homeostatic proliferation\n",
      "    - negative regulation of inflammatory response\n",
      "    - activated T cell proliferation\n",
      "    - regulation of CD4-positive, alpha-beta T cell proliferation\n",
      "  CC:\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - interleukin-2 receptor complex\n",
      "    - external side of plasma membrane\n",
      "  MF:\n",
      "    - interleukin-2 receptor activity\n",
      "    - interleukin-2 receptor activity\n",
      "    - protein binding\n",
      "    - interleukin-2 binding\n",
      "UniProt: {'Swiss-Prot': 'P01589', 'TrEMBL': ['H0Y5Z0', 'Q5W005', 'Q5W006', 'A0A8V8TMM2']}\n",
      "\n",
      "Gene: IL4\n",
      "Symbol: IL4\n",
      "Name: interleukin 4\n",
      "Summary: The protein encoded by this gene is a pleiotropic cytokine produced by activated T cells. This cytokine is a ligand for interleukin 4 receptor. The interleukin 4 receptor also binds to IL13, which may contribute to many overlapping functions of this cytokine and IL13. STAT6, a signal transducer and activator of transcription, has been shown to play a central role in mediating the immune regulatory signal of this cytokine. This gene, IL3, IL5, IL13, and CSF2 form a cytokine gene cluster on chromosome 5q, with this gene particularly close to IL13. This gene, IL13 and IL5 are found to be regulated coordinately by several long-range regulatory elements in an over 120 kilobase range on the chromosome. IL4 is considered an important cytokine for tissue repair, counterbalancing the effects of proinflammatory type 1 cytokines, however, it also promotes allergic airway inflammation. Moreover, IL-4, a type 2 cytokine, mediates and regulates a variety of human host responses such as allergic, anti-parasitic, wound healing, and acute inflammation. This cytokine has been reported to promote resolution of neutrophil-mediated acute lung injury. In an allergic response, IL-4 has an essential role in the production of allergen-specific immunoglobin (Ig) E. This pro-inflammatory cytokine has been observed to be increased in COVID-19 (Coronavirus disease 2019) patients, but is not necessarily associated with severe COVID-19 pathology. Two alternatively spliced transcript variants of this gene encoding distinct isoforms have been reported. [provided by RefSeq, Aug 2020].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - negative regulation of transcription by RNA polymerase II\n",
      "    - immune response\n",
      "    - cell surface receptor signaling pathway via JAK-STAT\n",
      "    - cholesterol metabolic process\n",
      "    - positive regulation of cell population proliferation\n",
      "    - positive regulation of gene expression\n",
      "    - negative regulation of epithelial cell migration\n",
      "    - positive regulation of macroautophagy\n",
      "    - B cell differentiation\n",
      "    - positive regulation of cell migration\n",
      "    - positive regulation of B cell proliferation\n",
      "    - negative regulation of tumor necrosis factor production\n",
      "    - positive regulation of interleukin-10 production\n",
      "    - positive regulation of interleukin-13 production\n",
      "    - interleukin-4-mediated signaling pathway\n",
      "    - interleukin-4-mediated signaling pathway\n",
      "    - type 2 immune response\n",
      "    - positive regulation of T cell proliferation\n",
      "    - T cell activation\n",
      "    - macrophage activation\n",
      "    - regulation of phosphorylation\n",
      "    - myeloid dendritic cell differentiation\n",
      "    - negative regulation of apoptotic process\n",
      "    - regulation of isotype switching\n",
      "    - positive regulation of MHC class II biosynthetic process\n",
      "    - positive regulation of T cell differentiation\n",
      "    - negative regulation of osteoclast differentiation\n",
      "    - negative regulation of DNA-templated transcription\n",
      "    - positive regulation of DNA-templated transcription\n",
      "    - positive regulation of DNA-templated transcription\n",
      "    - positive regulation of transcription by RNA polymerase II\n",
      "    - positive regulation of transcription by RNA polymerase II\n",
      "    - positive regulation of receptor-mediated endocytosis\n",
      "    - positive regulation of isotype switching to IgE isotypes\n",
      "    - positive regulation of isotype switching to IgG isotypes\n",
      "    - negative regulation of inflammatory response\n",
      "    - negative regulation of inflammatory response\n",
      "    - regulation of immune response\n",
      "    - regulation of immune response\n",
      "    - dendritic cell differentiation\n",
      "    - positive regulation of cold-induced thermogenesis\n",
      "    - neuroinflammatory response\n",
      "    - negative regulation of neuroinflammatory response\n",
      "    - positive regulation of amyloid-beta clearance\n",
      "    - positive regulation of cellular respiration\n",
      "    - negative regulation of complement-dependent cytotoxicity\n",
      "    - negative regulation of cellular response to transforming growth factor beta stimulus\n",
      "    - negative regulation of endothelial cell apoptotic process\n",
      "    - positive regulation of T-helper 2 cell cytokine production\n",
      "    - positive regulation of ATP biosynthetic process\n",
      "  CC:\n",
      "    - extracellular region\n",
      "    - extracellular space\n",
      "    - extracellular space\n",
      "  MF:\n",
      "    - cytokine activity\n",
      "    - interleukin-4 receptor binding\n",
      "    - protein binding\n",
      "    - growth factor activity\n",
      "UniProt: {'Swiss-Prot': 'P05112', 'TrEMBL': ['D4HNR6', 'Q5FC01', 'U3LVN1']}\n",
      "\n",
      "Gene: MAP2K4\n",
      "Symbol: MAP2K4\n",
      "Name: mitogen-activated protein kinase kinase 4\n",
      "Summary: This gene encodes a member of the mitogen-activated protein kinase (MAPK) family. Members of this family act as an integration point for multiple biochemical signals and are involved in a wide variety of cellular processes such as proliferation, differentiation, transcription regulation, and development. They form a three-tiered signaling module composed of MAPKKKs, MAPKKs, and MAPKs. This protein is phosphorylated at serine and threonine residues by MAPKKKs and subsequently phosphorylates downstream MAPK targets at threonine and tyrosine residues. A similar protein in mouse has been reported to play a role in liver organogenesis. A pseudogene of this gene is located on the long arm of chromosome X. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Jul 2013].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - MAPK cascade\n",
      "    - signal transduction\n",
      "    - JNK cascade\n",
      "    - response to wounding\n",
      "    - smooth muscle cell apoptotic process\n",
      "    - intrinsic apoptotic signaling pathway in response to hydrogen peroxide\n",
      "    - Fc-epsilon receptor signaling pathway\n",
      "    - positive regulation of neuron apoptotic process\n",
      "    - positive regulation of DNA replication\n",
      "    - positive regulation of JNK cascade\n",
      "    - cell growth involved in cardiac muscle cell development\n",
      "    - cellular response to mechanical stimulus\n",
      "    - cellular response to sorbitol\n",
      "    - cellular senescence\n",
      "    - negative regulation of motor neuron apoptotic process\n",
      "  CC:\n",
      "    - nucleus\n",
      "    - cytosol\n",
      "    - axon\n",
      "    - dendrite cytoplasm\n",
      "    - perikaryon\n",
      "  MF:\n",
      "    - protein kinase activity\n",
      "    - protein serine/threonine kinase activity\n",
      "    - MAP kinase kinase activity\n",
      "    - protein tyrosine kinase activity\n",
      "    - protein binding\n",
      "    - ATP binding\n",
      "    - JUN kinase kinase activity\n",
      "    - molecular adaptor activity\n",
      "    - protein serine kinase activity\n",
      "UniProt: {'Swiss-Prot': 'P45985', 'TrEMBL': ['R4GN68', 'J3QLE2', 'K7EP58', 'R4GN37']}\n",
      "\n",
      "Gene: KDM3A\n",
      "Symbol: KDM3A\n",
      "Name: lysine demethylase 3A\n",
      "Summary: This gene encodes a zinc finger protein that contains a jumonji domain and may play a role in hormone-dependent transcriptional activation. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Apr 2009]\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - chromatin remodeling\n",
      "    - regulation of transcription by RNA polymerase II\n",
      "    - spermatid nucleus elongation\n",
      "    - hormone-mediated signaling pathway\n",
      "    - androgen receptor signaling pathway\n",
      "    - positive regulation of DNA-templated transcription\n",
      "    - positive regulation of transcription by RNA polymerase II\n",
      "    - formaldehyde biosynthetic process\n",
      "    - positive regulation of cold-induced thermogenesis\n",
      "    - cellular response to leukemia inhibitory factor\n",
      "    - regulation of stem cell population maintenance\n",
      "    - regulation of stem cell differentiation\n",
      "  CC:\n",
      "    - histone deacetylase complex\n",
      "    - chromatin\n",
      "    - male germ cell nucleus\n",
      "    - nucleus\n",
      "    - nucleoplasm\n",
      "    - nucleoplasm\n",
      "    - cytoplasm\n",
      "    - membrane\n",
      "  MF:\n",
      "    - transcription coregulator activity\n",
      "    - iron ion binding\n",
      "    - protein binding\n",
      "    - zinc ion binding\n",
      "    - chromatin DNA binding\n",
      "    - histone H3K9 demethylase activity\n",
      "    - histone H3K9 demethylase activity\n",
      "    - nuclear androgen receptor binding\n",
      "    - histone H3K9me/H3K9me2 demethylase activity\n",
      "    - FAD-dependent histone H3K9me/H3K9me2 demethylase activity\n",
      "UniProt: {'Swiss-Prot': 'Q9Y4C1', 'TrEMBL': ['F8WE62', 'C9JC73', 'C9J7Q7']}\n",
      "\n",
      "Gene: COX5B\n",
      "Symbol: COX5B\n",
      "Name: cytochrome c oxidase subunit 5B\n",
      "Summary: Cytochrome C oxidase (COX) is the terminal enzyme of the mitochondrial respiratory chain. It is a multi-subunit enzyme complex that couples the transfer of electrons from cytochrome c to molecular oxygen and contributes to a proton electrochemical gradient across the inner mitochondrial membrane. The complex consists of 13 mitochondrial- and nuclear-encoded subunits. The mitochondrially-encoded subunits perform the electron transfer and proton pumping activities. The functions of the nuclear-encoded subunits are unknown but they may play a role in the regulation and assembly of the complex. This gene encodes the nuclear-encoded subunit Vb of the human mitochondrial respiratory chain enzyme. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - mitochondrial electron transport, cytochrome c to oxygen\n",
      "    - mitochondrial electron transport, cytochrome c to oxygen\n",
      "    - respiratory gaseous exchange by respiratory system\n",
      "    - cellular respiration\n",
      "    - proton transmembrane transport\n",
      "  CC:\n",
      "    - mitochondrion\n",
      "    - mitochondrion\n",
      "    - mitochondrial inner membrane\n",
      "    - mitochondrial membrane\n",
      "    - respiratory chain complex IV\n",
      "  MF:\n",
      "    - cytochrome-c oxidase activity\n",
      "    - protein binding\n",
      "    - metal ion binding\n",
      "UniProt: {'Swiss-Prot': 'P10606', 'TrEMBL': 'A0A384NL93'}\n",
      "\n",
      "Gene: LAMTOR5\n",
      "Symbol: LAMTOR5\n",
      "Name: late endosomal/lysosomal adaptor, MAPK and MTOR activator 5\n",
      "Summary: This gene encodes a protein that specifically complexes with the C-terminus of hepatitis B virus X protein (HBx). The function of this protein is to negatively regulate HBx activity and thus to alter the replication life cycle of the virus. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - regulation of cell size\n",
      "    - response to virus\n",
      "    - positive regulation of gene expression\n",
      "    - viral genome replication\n",
      "    - positive regulation of TOR signaling\n",
      "    - positive regulation of interleukin-8 production\n",
      "    - TORC1 signaling\n",
      "    - negative regulation of apoptotic process\n",
      "    - positive regulation of canonical NF-kappaB signal transduction\n",
      "    - protein localization to lysosome\n",
      "    - cellular response to amino acid stimulus\n",
      "    - cellular response to amino acid stimulus\n",
      "    - cellular response to amino acid stimulus\n",
      "    - positive regulation of protein localization to nucleus\n",
      "    - positive regulation of TORC1 signaling\n",
      "    - positive regulation of TORC1 signaling\n",
      "  CC:\n",
      "    - lysosome\n",
      "    - lysosome\n",
      "    - lysosomal membrane\n",
      "    - lysosomal membrane\n",
      "    - lysosomal membrane\n",
      "    - cytosol\n",
      "    - late endosome membrane\n",
      "    - protein-containing complex\n",
      "    - Ragulator complex\n",
      "    - Ragulator complex\n",
      "    - Ragulator complex\n",
      "    - FNIP-folliculin RagC/D GAP\n",
      "  MF:\n",
      "    - guanyl-nucleotide exchange factor activity\n",
      "    - guanyl-nucleotide exchange factor activity\n",
      "    - protein binding\n",
      "    - molecular adaptor activity\n",
      "UniProt: {'Swiss-Prot': 'O43504', 'TrEMBL': ['E9PLX3', 'A0A8Z5A536', 'R4GMU8']}\n",
      "\n",
      "Gene: DDC\n",
      "Symbol: DDC\n",
      "Name: dopa decarboxylase\n",
      "Summary: The encoded protein catalyzes the decarboxylation of L-3,4-dihydroxyphenylalanine (DOPA) to dopamine, L-5-hydroxytryptophan to serotonin and L-tryptophan to tryptamine. Defects in this gene are the cause of aromatic L-amino-acid decarboxylase deficiency (AADCD). AADCD deficiency is an inborn error in neurotransmitter metabolism that leads to combined serotonin and catecholamine deficiency. Multiple alternatively spliced transcript variants encoding different isoforms have been identified for this gene. [provided by RefSeq, Jun 2011].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - kidney development\n",
      "    - amino acid metabolic process\n",
      "    - catecholamine metabolic process\n",
      "    - response to toxic substance\n",
      "    - gene expression\n",
      "    - carboxylic acid metabolic process\n",
      "    - dopamine biosynthetic process\n",
      "    - dopamine biosynthetic process\n",
      "    - serotonin biosynthetic process\n",
      "  CC:\n",
      "    - cytoplasm\n",
      "    - cytosol\n",
      "    - extracellular exosome\n",
      "  MF:\n",
      "    - aromatic-L-amino-acid decarboxylase activity\n",
      "    - aromatic-L-amino-acid decarboxylase activity\n",
      "    - protein binding\n",
      "    - enzyme binding\n",
      "    - pyridoxal phosphate binding\n",
      "    - 5-hydroxy-L-tryptophan decarboxylase activity\n",
      "    - L-dopa decarboxylase activity\n",
      "UniProt: {'Swiss-Prot': 'P20711', 'TrEMBL': ['A0A0S2Z3N4', 'Q53Y41', 'A0A087WV24', 'C9JMP0', 'F8WER1', 'A0A087WU57', 'H7BZF7']}\n",
      "\n",
      "Gene: PDE3A\n",
      "Symbol: PDE3A\n",
      "Name: phosphodiesterase 3A\n",
      "Summary: This gene encodes a member of the cGMP-inhibited cyclic nucleotide phosphodiesterase (cGI-PDE) family. cGI-PDE enzymes hydrolyze both cAMP and cGMP, and play critical roles in many cellular processes by regulating the amplitude and duration of intracellular cyclic nucleotide signals. The encoded protein mediates platelet aggregation and also plays important roles in cardiovascular function by regulating vascular smooth muscle contraction and relaxation. Inhibitors of the encoded protein may be effective in treating congestive heart failure. Alternatively spliced transcript variants encoding multiple isoforms have been observed for this gene. [provided by RefSeq, Sep 2011].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - oocyte maturation\n",
      "    - lipid metabolic process\n",
      "    - G protein-coupled receptor signaling pathway\n",
      "    - response to xenobiotic stimulus\n",
      "    - cAMP-mediated signaling\n",
      "    - cGMP-mediated signaling\n",
      "    - nuclear receptor-mediated steroid hormone signaling pathway\n",
      "    - regulation of meiotic nuclear division\n",
      "    - negative regulation of apoptotic process\n",
      "    - negative regulation of vascular permeability\n",
      "    - positive regulation of vascular permeability\n",
      "    - positive regulation of oocyte development\n",
      "    - regulation of ribonuclease activity\n",
      "    - regulation of ribonuclease activity\n",
      "    - cellular response to cGMP\n",
      "    - cellular response to transforming growth factor beta stimulus\n",
      "    - apoptotic signaling pathway\n",
      "    - apoptotic signaling pathway\n",
      "    - negative regulation of adenylate cyclase-activating G protein-coupled receptor signaling pathway\n",
      "    - negative regulation of cAMP/PKA signal transduction\n",
      "  CC:\n",
      "    - cytosol\n",
      "    - cytosol\n",
      "    - cytosol\n",
      "    - membrane\n",
      "  MF:\n",
      "    - 3',5'-cyclic-nucleotide phosphodiesterase activity\n",
      "    - 3',5'-cyclic-AMP phosphodiesterase activity\n",
      "    - 3',5'-cyclic-AMP phosphodiesterase activity\n",
      "    - 3',5'-cyclic-AMP phosphodiesterase activity\n",
      "    - 3',5'-cGMP-stimulated cyclic-nucleotide phosphodiesterase activity\n",
      "    - 3',5'-cGMP-inhibited cyclic-nucleotide phosphodiesterase activity\n",
      "    - protein binding\n",
      "    - nuclear estrogen receptor activity\n",
      "    - metal ion binding\n",
      "    - 3',5'-cyclic-GMP phosphodiesterase activity\n",
      "    - 3',5'-cyclic-GMP phosphodiesterase activity\n",
      "    - estrogen binding\n",
      "UniProt: {'Swiss-Prot': 'Q14432'}\n",
      "\n",
      "Gene: TNFSF12\n",
      "Symbol: TNFSF12\n",
      "Name: TNF superfamily member 12\n",
      "Summary: The protein encoded by this gene is a cytokine that belongs to the tumor necrosis factor (TNF) ligand family. This protein is a ligand for the FN14/TWEAKR receptor. This cytokine has overlapping signaling functions with TNF, but displays a much wider tissue distribution. This cytokine, which exists in both membrane-bound and secreted forms, can induce apoptosis via multiple pathways of cell death in a cell type-specific manner. This cytokine is also found to promote proliferation and migration of endothelial cells, and thus acts as a regulator of angiogenesis. Alternative splicing results in multiple transcript variants. Some transcripts skip the last exon of this gene and continue into the second exon of the neighboring TNFSF13 gene; such read-through transcripts are contained in GeneID 407977, TNFSF12-TNFSF13. [provided by RefSeq, Oct 2010].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - angiogenesis\n",
      "    - positive regulation of endothelial cell proliferation\n",
      "    - apoptotic process\n",
      "    - immune response\n",
      "    - signal transduction\n",
      "    - cell differentiation\n",
      "    - endothelial cell migration\n",
      "    - positive regulation of protein catabolic process\n",
      "    - positive regulation of angiogenesis\n",
      "    - apoptotic signaling pathway\n",
      "    - extrinsic apoptotic signaling pathway\n",
      "    - positive regulation of extrinsic apoptotic signaling pathway\n",
      "  CC:\n",
      "    - extracellular region\n",
      "    - extracellular space\n",
      "    - extracellular space\n",
      "    - plasma membrane\n",
      "    - perinuclear region of cytoplasm\n",
      "  MF:\n",
      "    - signaling receptor binding\n",
      "    - cytokine activity\n",
      "    - cytokine activity\n",
      "    - tumor necrosis factor receptor binding\n",
      "    - protein binding\n",
      "    - receptor ligand activity\n",
      "UniProt: {'Swiss-Prot': 'O43508', 'TrEMBL': ['A0A0U5J5F5', 'Q4ACW9', 'C0H5Y4', 'A0A8V8TP82', 'A0A8V8TQJ4', 'A0A8V8TQX4', 'A0A8V8TQK2', 'A0A8V8TPL4', 'A0A8V8TPS2']}\n",
      "\n",
      "Gene: TNFRSF12A\n",
      "Symbol: TNFRSF12A\n",
      "Name: TNF receptor superfamily member 12A\n",
      "Summary: Involved in positive regulation of extrinsic apoptotic signaling pathway and regulation of wound healing. Predicted to be located in cell surface and ruffle. Predicted to be active in plasma membrane. [provided by Alliance of Genome Resources, Apr 2022]\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - angiogenesis\n",
      "    - apoptotic process\n",
      "    - cell adhesion\n",
      "    - cell differentiation\n",
      "    - positive regulation of apoptotic process\n",
      "    - regulation of wound healing\n",
      "    - positive regulation of extrinsic apoptotic signaling pathway\n",
      "    - positive regulation of extrinsic apoptotic signaling pathway\n",
      "  CC:\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "  MF:\n",
      "    - protein binding\n",
      "UniProt: {'Swiss-Prot': 'Q9NP84', 'TrEMBL': ['I3L1J9', 'I3L3P4', 'I3L539', 'I3L0S4']}\n",
      "\n",
      "Gene: RPS6KA1\n",
      "Symbol: RPS6KA1\n",
      "Name: ribosomal protein S6 kinase A1\n",
      "Summary: This gene encodes a member of the RSK (ribosomal S6 kinase) family of serine/threonine kinases. This kinase contains 2 nonidentical kinase catalytic domains and phosphorylates various substrates, including members of the mitogen-activated kinase (MAPK) signalling pathway. The activity of this protein has been implicated in controlling cell growth and differentiation. Alternate transcriptional splice variants, encoding different isoforms, have been characterized. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - chromatin remodeling\n",
      "    - protein phosphorylation\n",
      "    - signal transduction\n",
      "    - chemical synaptic transmission\n",
      "    - positive regulation of cell growth\n",
      "    - negative regulation of TOR signaling\n",
      "    - TORC1 signaling\n",
      "    - negative regulation of apoptotic process\n",
      "    - negative regulation of apoptotic process\n",
      "    - regulation of translation in response to stress\n",
      "    - positive regulation of cell differentiation\n",
      "    - positive regulation of DNA-templated transcription\n",
      "    - positive regulation of transcription by RNA polymerase II\n",
      "    - hepatocyte proliferation\n",
      "    - positive regulation of hepatic stellate cell activation\n",
      "  CC:\n",
      "    - nucleoplasm\n",
      "    - nucleoplasm\n",
      "    - nucleoplasm\n",
      "    - cytoplasm\n",
      "    - cytosol\n",
      "    - cytosol\n",
      "    - synapse\n",
      "  MF:\n",
      "    - magnesium ion binding\n",
      "    - protein serine/threonine kinase activity\n",
      "    - protein serine/threonine kinase activity\n",
      "    - 3-phosphoinositide-dependent protein kinase activity\n",
      "    - DNA-dependent protein kinase activity\n",
      "    - AMP-activated protein kinase activity\n",
      "    - eukaryotic translation initiation factor 2alpha kinase activity\n",
      "    - ribosomal protein S6 kinase activity\n",
      "    - protein serine/threonine/tyrosine kinase activity\n",
      "    - protein binding\n",
      "    - ATP binding\n",
      "    - histone H3S10 kinase activity\n",
      "    - histone H3T11 kinase activity\n",
      "    - histone H3T6 kinase activity\n",
      "    - histone H2AXS139 kinase activity\n",
      "    - histone H3S28 kinase activity\n",
      "    - histone H4S1 kinase activity\n",
      "    - histone H2AS1 kinase activity\n",
      "    - histone H2BS14 kinase activity\n",
      "    - histone H3T3 kinase activity\n",
      "    - histone H2AS121 kinase activity\n",
      "    - Rho-dependent protein serine/threonine kinase activity\n",
      "    - protein serine kinase activity\n",
      "    - histone H2BS36 kinase activity\n",
      "    - histone H3S57 kinase activity\n",
      "    - histone H3T45 kinase activity\n",
      "    - histone H2AT120 kinase activity\n",
      "UniProt: {'Swiss-Prot': 'Q15418', 'TrEMBL': ['Q5SVM6', 'E9PPC1', 'A6NND1', 'Q5SVM7', 'E9PPN6', 'E9PMM7', 'E9PGT3', 'E9PAN7']}\n",
      "\n",
      "Gene: RPS6KA2\n",
      "Symbol: RPS6KA2\n",
      "Name: ribosomal protein S6 kinase A2\n",
      "Summary: This gene encodes a member of the RSK (ribosomal S6 kinase) family of serine/threonine kinases. This kinase contains two non-identical kinase catalytic domains and phosphorylates various substrates, including members of the mitogen-activated kinase (MAPK) signalling pathway. The activity of this protein has been implicated in controlling cell growth and differentiation. Alternative splice variants, encoding different isoforms, have been characterized. [provided by RefSeq, Jan 2016].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - chromatin remodeling\n",
      "    - signal transduction\n",
      "    - chemical synaptic transmission\n",
      "    - negative regulation of cell population proliferation\n",
      "    - intracellular signal transduction\n",
      "    - TORC1 signaling\n",
      "    - positive regulation of apoptotic process\n",
      "    - negative regulation of cell cycle\n",
      "  CC:\n",
      "    - nucleus\n",
      "    - nucleoplasm\n",
      "    - nucleoplasm\n",
      "    - nucleoplasm\n",
      "    - cytoplasm\n",
      "    - cytoplasm\n",
      "    - cytosol\n",
      "    - synapse\n",
      "  MF:\n",
      "    - magnesium ion binding\n",
      "    - protein serine/threonine kinase activity\n",
      "    - 3-phosphoinositide-dependent protein kinase activity\n",
      "    - DNA-dependent protein kinase activity\n",
      "    - AMP-activated protein kinase activity\n",
      "    - eukaryotic translation initiation factor 2alpha kinase activity\n",
      "    - ribosomal protein S6 kinase activity\n",
      "    - protein serine/threonine/tyrosine kinase activity\n",
      "    - protein binding\n",
      "    - ATP binding\n",
      "    - histone H3S10 kinase activity\n",
      "    - histone H3T11 kinase activity\n",
      "    - histone H3T6 kinase activity\n",
      "    - histone H2AXS139 kinase activity\n",
      "    - histone H3S28 kinase activity\n",
      "    - histone H4S1 kinase activity\n",
      "    - histone H2AS1 kinase activity\n",
      "    - histone H2BS14 kinase activity\n",
      "    - histone H3T3 kinase activity\n",
      "    - histone H2AS121 kinase activity\n",
      "    - Rho-dependent protein serine/threonine kinase activity\n",
      "    - protein serine kinase activity\n",
      "    - histone H2BS36 kinase activity\n",
      "    - histone H3S57 kinase activity\n",
      "    - histone H3T45 kinase activity\n",
      "    - histone H2AT120 kinase activity\n",
      "UniProt: {'Swiss-Prot': 'Q15349', 'TrEMBL': ['X5D337', 'B7Z3B5', 'A0AAQ5BI00', 'D6R910', 'D6RHW7', 'D6RD75', 'D6RE03', 'D6RA62', 'F2Z2J1']}\n",
      "\n",
      "Gene: KRT18\n",
      "Symbol: KRT18\n",
      "Name: keratin 18\n",
      "Summary: KRT18 encodes the type I intermediate filament chain keratin 18. Keratin 18, together with its filament partner keratin 8, are perhaps the most commonly found members of the intermediate filament gene family. They are expressed in single layer epithelial tissues of the body. Mutations in this gene have been linked to cryptogenic cirrhosis. Two transcript variants encoding the same protein have been found for this gene. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - anatomical structure morphogenesis\n",
      "    - tumor necrosis factor-mediated signaling pathway\n",
      "    - Golgi to plasma membrane protein transport\n",
      "    - negative regulation of apoptotic process\n",
      "    - intermediate filament cytoskeleton organization\n",
      "    - intermediate filament cytoskeleton organization\n",
      "    - extrinsic apoptotic signaling pathway\n",
      "    - hepatocyte apoptotic process\n",
      "    - cell-cell adhesion\n",
      "  CC:\n",
      "    - nucleolus\n",
      "    - cytoplasm\n",
      "    - microtubule organizing center\n",
      "    - cytosol\n",
      "    - cytosol\n",
      "    - cytoskeleton\n",
      "    - intermediate filament\n",
      "    - adherens junction\n",
      "    - nuclear matrix\n",
      "    - centriolar satellite\n",
      "    - keratin filament\n",
      "    - keratin filament\n",
      "    - keratin filament\n",
      "    - perinuclear region of cytoplasm\n",
      "    - extracellular exosome\n",
      "    - cell periphery\n",
      "  MF:\n",
      "    - RNA binding\n",
      "    - structural molecule activity\n",
      "    - protein binding\n",
      "    - scaffold protein binding\n",
      "    - cadherin binding involved in cell-cell adhesion\n",
      "UniProt: {'Swiss-Prot': 'P05783', 'TrEMBL': 'F8VZY9'}\n",
      "\n",
      "Gene: MAP2K3\n",
      "Symbol: MAP2K3\n",
      "Name: mitogen-activated protein kinase kinase 3\n",
      "Summary: The protein encoded by this gene is a dual specificity protein kinase that belongs to the MAP kinase kinase family. This kinase is activated by mitogenic and environmental stress, and participates in the MAP kinase-mediated signaling cascade. It phosphorylates and thus activates MAPK14/p38-MAPK. This kinase can be activated by insulin, and is necessary for the expression of glucose transporter. Expression of RAS oncogene is found to result in the accumulation of the active form of this kinase, which thus leads to the constitutive activation of MAPK14, and confers oncogenic transformation of primary cells. The inhibition of this kinase is involved in the pathogenesis of Yersina pseudotuberculosis. Multiple alternatively spliced transcript variants that encode distinct isoforms have been reported for this gene. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - MAPK cascade\n",
      "    - regulation of cytokine production\n",
      "    - response to ischemia\n",
      "    - inflammatory response\n",
      "    - signal transduction\n",
      "    - JNK cascade\n",
      "    - heart development\n",
      "    - stress-activated protein kinase signaling cascade\n",
      "    - negative regulation of hippo signaling\n",
      "    - cellular response to vascular endothelial growth factor stimulus\n",
      "    - p38MAPK cascade\n",
      "    - p38MAPK cascade\n",
      "    - positive regulation of MAPK cascade\n",
      "    - positive regulation of blood vessel endothelial cell migration\n",
      "    - positive regulation of protein kinase activity\n",
      "    - positive regulation of DNA-templated transcription\n",
      "    - cardiac muscle contraction\n",
      "    - cellular response to lipopolysaccharide\n",
      "    - cellular response to sorbitol\n",
      "    - cellular senescence\n",
      "  CC:\n",
      "    - nucleoplasm\n",
      "    - cytoplasm\n",
      "    - cytosol\n",
      "    - membrane\n",
      "  MF:\n",
      "    - protein serine/threonine kinase activity\n",
      "    - MAP kinase kinase activity\n",
      "    - MAP kinase kinase activity\n",
      "    - MAP kinase kinase activity\n",
      "    - protein tyrosine kinase activity\n",
      "    - protein binding\n",
      "    - ATP binding\n",
      "    - JUN kinase kinase activity\n",
      "    - protein kinase binding\n",
      "    - protein serine kinase activity\n",
      "UniProt: {'Swiss-Prot': 'P46734', 'TrEMBL': ['J3QL77', 'Q6FI23', 'X6RB39', 'J3QR49', 'J3KRV4', 'E9PRZ0', 'E9PMA8']}\n",
      "\n",
      "Gene: KRAS\n",
      "Symbol: KRAS\n",
      "Name: KRAS proto-oncogene, GTPase\n",
      "Summary: This gene, a Kirsten ras oncogene homolog from the mammalian ras gene family, encodes a protein that is a member of the small GTPase superfamily. A single amino acid substitution is responsible for an activating mutation. The transforming protein that results is implicated in various malignancies, including lung adenocarcinoma, mucinous adenoma, ductal carcinoma of the pancreas and colorectal carcinoma. Alternative splicing leads to variants encoding two isoforms that differ in the C-terminal region. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - MAPK cascade\n",
      "    - Ras protein signal transduction\n",
      "    - visual learning\n",
      "    - gene expression\n",
      "    - positive regulation of gene expression\n",
      "    - glial cell proliferation\n",
      "    - Rac protein signal transduction\n",
      "    - forebrain astrocyte development\n",
      "    - actin cytoskeleton organization\n",
      "    - negative regulation of epithelial cell differentiation\n",
      "    - regulation of synaptic transmission, GABAergic\n",
      "    - positive regulation of Rac protein signal transduction\n",
      "    - skeletal muscle cell differentiation\n",
      "    - negative regulation of neuron apoptotic process\n",
      "    - regulation of long-term neuronal synaptic plasticity\n",
      "    - homeostasis of number of cells within a tissue\n",
      "    - striated muscle cell differentiation\n",
      "    - neuron apoptotic process\n",
      "    - positive regulation of glial cell proliferation\n",
      "    - epithelial tube branching involved in lung morphogenesis\n",
      "    - type I pneumocyte differentiation\n",
      "  CC:\n",
      "    - Golgi membrane\n",
      "    - cytoplasm\n",
      "    - mitochondrial outer membrane\n",
      "    - endoplasmic reticulum membrane\n",
      "    - cytosol\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - focal adhesion\n",
      "    - cytoplasmic side of plasma membrane\n",
      "    - membrane\n",
      "  MF:\n",
      "    - GTPase activity\n",
      "    - GTPase activity\n",
      "    - GTPase activity\n",
      "    - G protein activity\n",
      "    - protein binding\n",
      "    - GTP binding\n",
      "    - GDP binding\n",
      "    - identical protein binding\n",
      "    - protein-membrane adaptor activity\n",
      "    - protein-containing complex binding\n",
      "UniProt: {'Swiss-Prot': 'P01116', 'TrEMBL': ['L7RSL8', 'I1SRC5', 'G3V4K2', 'A0A8I5KQ21', 'A0A8I5KQU3', 'A0A8I5KUB5', 'A0A8I5KXN3', 'G3V5T7', 'A0A8I5KR86', 'A0A8I5KYH6']}\n",
      "\n",
      "Gene: KRT8\n",
      "Symbol: KRT8\n",
      "Name: keratin 8\n",
      "Summary: This gene is a member of the type II keratin family clustered on the long arm of chromosome 12. Type I and type II keratins heteropolymerize to form intermediate-sized filaments in the cytoplasm of epithelial cells. The product of this gene typically dimerizes with keratin 18 to form an intermediate filament in simple single-layered epithelial cells. This protein plays a role in maintaining cellular structural integrity and also functions in signal transduction and cellular differentiation. Mutations in this gene cause cryptogenic cirrhosis. Alternatively spliced transcript variants have been found for this gene. [provided by RefSeq, Jan 2012].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - tumor necrosis factor-mediated signaling pathway\n",
      "    - sarcomere organization\n",
      "    - response to hydrostatic pressure\n",
      "    - response to other organism\n",
      "    - cell differentiation involved in embryonic placenta development\n",
      "    - extrinsic apoptotic signaling pathway\n",
      "    - hepatocyte apoptotic process\n",
      "  CC:\n",
      "    - nucleus\n",
      "    - nucleoplasm\n",
      "    - cytoplasm\n",
      "    - cytosol\n",
      "    - intermediate filament\n",
      "    - cell-cell junction\n",
      "    - dystrophin-associated glycoprotein complex\n",
      "    - apicolateral plasma membrane\n",
      "    - nuclear matrix\n",
      "    - Z disc\n",
      "    - sarcolemma\n",
      "    - costamere\n",
      "    - keratin filament\n",
      "    - intermediate filament cytoskeleton\n",
      "    - extracellular exosome\n",
      "  MF:\n",
      "    - protein binding\n",
      "    - protein-containing complex binding\n",
      "    - scaffold protein binding\n",
      "UniProt: {'Swiss-Prot': 'P05787', 'TrEMBL': ['A0A0U1RQU6', 'Q7L4M3', 'F8W1U3', 'F8VQY3', 'H0YIB2', 'F8VP67', 'F8VUG2']}\n",
      "\n",
      "Gene: MAP2K5\n",
      "Symbol: MAP2K5\n",
      "Name: mitogen-activated protein kinase kinase 5\n",
      "Summary: The protein encoded by this gene is a dual specificity protein kinase that belongs to the MAP kinase kinase family. This kinase specifically interacts with and activates MAPK7/ERK5. This kinase itself can be phosphorylated and activated by MAP3K3/MEKK3, as well as by atypical protein kinase C isoforms (aPKCs). The signal cascade mediated by this kinase is involved in growth factor stimulated cell proliferation and muscle cell differentiation. Three alternatively spliced transcript variants of this gene encoding distinct isoforms have been described. [provided by RefSeq, May 2011].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - negative regulation of transcription by RNA polymerase II\n",
      "    - MAPK cascade\n",
      "    - MAPK cascade\n",
      "    - signal transduction\n",
      "    - JNK cascade\n",
      "    - heart development\n",
      "    - positive regulation of cell growth\n",
      "    - negative regulation of interleukin-8 production\n",
      "    - negative regulation of heterotypic cell-cell adhesion\n",
      "    - negative regulation of smooth muscle cell apoptotic process\n",
      "    - negative regulation of canonical NF-kappaB signal transduction\n",
      "    - positive regulation of transcription by RNA polymerase II\n",
      "    - insulin-like growth factor receptor signaling pathway\n",
      "    - positive regulation of epithelial cell proliferation\n",
      "    - positive regulation of protein metabolic process\n",
      "    - negative regulation of response to cytokine stimulus\n",
      "    - ERK5 cascade\n",
      "    - cellular response to growth factor stimulus\n",
      "    - cellular response to laminar fluid shear stress\n",
      "    - negative regulation of cell migration involved in sprouting angiogenesis\n",
      "    - negative regulation of chemokine (C-X-C motif) ligand 2 production\n",
      "    - negative regulation of extrinsic apoptotic signaling pathway in absence of ligand\n",
      "  CC:\n",
      "    - spindle\n",
      "  MF:\n",
      "    - protein kinase activity\n",
      "    - protein serine/threonine kinase activity\n",
      "    - MAP kinase kinase activity\n",
      "    - protein tyrosine kinase activity\n",
      "    - protein binding\n",
      "    - ATP binding\n",
      "    - JUN kinase kinase activity\n",
      "    - metal ion binding\n",
      "    - protein serine kinase activity\n",
      "UniProt: {'Swiss-Prot': 'Q13163', 'TrEMBL': ['H7BZL1', 'A0A087WWX7', 'A6NK28', 'A0A087X0Z8']}\n",
      "\n",
      "Gene: THRB\n",
      "Symbol: THRB\n",
      "Name: thyroid hormone receptor beta\n",
      "Summary: The protein encoded by this gene is a nuclear hormone receptor for triiodothyronine. It is one of the several receptors for thyroid hormone, and has been shown to mediate the biological activities of thyroid hormone. Knockout studies in mice suggest that the different receptors, while having certain extent of redundancy, may mediate different functions of thyroid hormone. Mutations in this gene are known to be a cause of generalized thyroid hormone resistance (GTHR), a syndrome characterized by goiter and high levels of circulating thyroid hormone (T3-T4), with normal or slightly elevated thyroid stimulating hormone (TSH). Several alternatively spliced transcript variants encoding the same protein have been observed for this gene. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - negative regulation of transcription by RNA polymerase II\n",
      "    - thyroid hormone receptor signaling pathway\n",
      "    - positive regulation of thyroid hormone receptor signaling pathway\n",
      "    - DNA-templated transcription\n",
      "    - sensory perception of sound\n",
      "    - negative regulation of female receptivity\n",
      "    - regulation of heart contraction\n",
      "    - female courtship behavior\n",
      "    - cell differentiation\n",
      "    - mRNA transcription by RNA polymerase II\n",
      "    - positive regulation of transcription by RNA polymerase II\n",
      "    - positive regulation of transcription by RNA polymerase II\n",
      "    - retinal cone cell development\n",
      "    - retinoic acid receptor signaling pathway\n",
      "    - type I pneumocyte differentiation\n",
      "    - cellular response to thyroid hormone stimulus\n",
      "    - retinal cone cell apoptotic process\n",
      "  CC:\n",
      "    - chromatin\n",
      "    - nucleus\n",
      "    - nucleoplasm\n",
      "    - nuclear body\n",
      "    - RNA polymerase II transcription regulator complex\n",
      "    - RNA polymerase II transcription regulator complex\n",
      "  MF:\n",
      "    - RNA polymerase II cis-regulatory region sequence-specific DNA binding\n",
      "    - DNA-binding transcription factor activity, RNA polymerase II-specific\n",
      "    - transcription coactivator binding\n",
      "    - DNA binding\n",
      "    - DNA-binding transcription factor activity\n",
      "    - nuclear receptor activity\n",
      "    - nuclear receptor activity\n",
      "    - protein binding\n",
      "    - zinc ion binding\n",
      "    - enzyme binding\n",
      "    - chromatin DNA binding\n",
      "    - thyroid hormone binding\n",
      "    - sequence-specific double-stranded DNA binding\n",
      "UniProt: {'Swiss-Prot': 'P10828', 'TrEMBL': ['C9JHC2', 'A0A024R2I8', 'A0A2R8YF24', 'C9JZS5', 'J3KR21', 'C9JTN1', 'C9JNQ4', 'C9JJM3', 'A0A0C4DG57']}\n",
      "\n",
      "Gene: MATK\n",
      "Symbol: MATK\n",
      "Name: megakaryocyte-associated tyrosine kinase\n",
      "Summary: The protein encoded by this gene has amino acid sequence similarity to Csk tyrosine kinase and has the structural features of the CSK subfamily: SRC homology SH2 and SH3 domains, a catalytic domain, a unique N terminus, lack of myristylation signals, lack of a negative regulatory phosphorylation site, and lack of an autophosphorylation site. This protein is thought to play a significant role in the signal transduction of hematopoietic cells. It is able to phosphorylate and inactivate Src family kinases, and may play an inhibitory role in the control of T-cell proliferation. This protein might be involved in signaling in some cases of breast cancer. Three alternatively spliced transcript variants that encode different isoforms have been described for this gene. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - protein phosphorylation\n",
      "    - positive regulation of cell population proliferation\n",
      "  CC:\n",
      "    - cytosol\n",
      "    - plasma membrane\n",
      "  MF:\n",
      "    - protein tyrosine kinase activity\n",
      "    - non-membrane spanning protein tyrosine kinase activity\n",
      "    - protein binding\n",
      "    - ATP binding\n",
      "UniProt: {'Swiss-Prot': 'P42679', 'TrEMBL': ['K7EQY5', 'F1T0G6', 'K7EQV3', 'K7EPP2', 'K7ENL8', 'K7ERY4', 'K7ES68', 'K7EKS5', 'K7EKC4']}\n",
      "\n",
      "Gene: FGR\n",
      "Symbol: FGR\n",
      "Name: FGR proto-oncogene, Src family tyrosine kinase\n",
      "Summary: This gene is a member of the Src family of protein tyrosine kinases (PTKs). The encoded protein contains N-terminal sites for myristylation and palmitylation, a PTK domain, and SH2 and SH3 domains which are involved in mediating protein-protein interactions with phosphotyrosine-containing and proline-rich motifs, respectively. The protein localizes to plasma membrane ruffles, and functions as a negative regulator of cell migration and adhesion triggered by the beta-2 integrin signal transduction pathway. Infection with Epstein-Barr virus results in the overexpression of this gene. Multiple alternatively spliced variants, encoding the same protein, have been identified. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - positive regulation of cytokine production\n",
      "    - immune response-regulating cell surface receptor signaling pathway\n",
      "    - negative regulation of inflammatory response to antigenic stimulus\n",
      "    - protein phosphorylation\n",
      "    - cell surface receptor protein tyrosine kinase signaling pathway\n",
      "    - integrin-mediated signaling pathway\n",
      "    - integrin-mediated signaling pathway\n",
      "    - integrin-mediated signaling pathway\n",
      "    - regulation of cell shape\n",
      "    - response to virus\n",
      "    - peptidyl-tyrosine phosphorylation\n",
      "    - cell differentiation\n",
      "    - bone mineralization\n",
      "    - positive regulation of cell migration\n",
      "    - negative regulation of natural killer cell activation\n",
      "    - Fc-gamma receptor signaling pathway involved in phagocytosis\n",
      "    - positive regulation of mast cell degranulation\n",
      "    - innate immune response\n",
      "    - regulation of innate immune response\n",
      "    - regulation of protein kinase activity\n",
      "    - protein autophosphorylation\n",
      "    - skeletal system morphogenesis\n",
      "    - regulation of phagocytosis\n",
      "    - defense response to Gram-positive bacterium\n",
      "    - myoblast proliferation\n",
      "    - positive regulation of phosphatidylinositol 3-kinase/protein kinase B signal transduction\n",
      "  CC:\n",
      "    - extracellular region\n",
      "    - mitochondrial inner membrane\n",
      "    - mitochondrial intermembrane space\n",
      "    - cytosol\n",
      "    - cytoskeleton\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - actin cytoskeleton\n",
      "    - aggresome\n",
      "    - ruffle membrane\n",
      "    - secretory granule lumen\n",
      "    - extracellular exosome\n",
      "  MF:\n",
      "    - phosphotyrosine residue binding\n",
      "    - phosphotyrosine residue binding\n",
      "    - protein tyrosine kinase activity\n",
      "    - protein tyrosine kinase activity\n",
      "    - protein tyrosine kinase activity\n",
      "    - non-membrane spanning protein tyrosine kinase activity\n",
      "    - non-membrane spanning protein tyrosine kinase activity\n",
      "    - non-membrane spanning protein tyrosine kinase activity\n",
      "    - signaling receptor binding\n",
      "    - protein binding\n",
      "    - ATP binding\n",
      "    - protein kinase binding\n",
      "    - immunoglobulin receptor binding\n",
      "    - Fc-gamma receptor I complex binding\n",
      "UniProt: {'Swiss-Prot': 'P09769', 'TrEMBL': ['Q5TGY6', 'P78453']}\n",
      "\n",
      "Gene: NFYB\n",
      "Symbol: NFYB\n",
      "Name: nuclear transcription factor Y subunit beta\n",
      "Summary: The protein encoded by this gene is one subunit of a trimeric complex, forming a highly conserved transcription factor that binds with high specificity to CCAAT motifs in the promoter regions in a variety of genes. This gene product, subunit B, forms a tight dimer with the C subunit, a prerequisite for subunit A association. The resulting trimer binds to DNA with high specificity and affinity. Subunits B and C each contain a histone-like motif. Observation of the histone nature of these subunits is supported by two types of evidence; protein sequence alignments and experiments with mutants. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - regulation of DNA-templated transcription\n",
      "    - regulation of transcription by RNA polymerase II\n",
      "    - positive regulation of transcription by RNA polymerase II\n",
      "    - cellular response to leukemia inhibitory factor\n",
      "  CC:\n",
      "    - chromatin\n",
      "    - nucleus\n",
      "    - nucleoplasm\n",
      "    - nucleoplasm\n",
      "    - CCAAT-binding factor complex\n",
      "    - CCAAT-binding factor complex\n",
      "    - CCAAT-binding factor complex\n",
      "    - protein-DNA complex\n",
      "    - RNA polymerase II transcription regulator complex\n",
      "  MF:\n",
      "    - transcription cis-regulatory region binding\n",
      "    - RNA polymerase II cis-regulatory region sequence-specific DNA binding\n",
      "    - RNA polymerase II cis-regulatory region sequence-specific DNA binding\n",
      "    - RNA polymerase II cis-regulatory region sequence-specific DNA binding\n",
      "    - DNA-binding transcription factor activity, RNA polymerase II-specific\n",
      "    - DNA-binding transcription factor activity, RNA polymerase II-specific\n",
      "    - DNA-binding transcription activator activity, RNA polymerase II-specific\n",
      "    - DNA binding\n",
      "    - DNA-binding transcription factor activity\n",
      "    - protein binding\n",
      "    - protein heterodimerization activity\n",
      "    - DNA-binding transcription factor binding\n",
      "UniProt: {'Swiss-Prot': 'P25208', 'TrEMBL': 'F8VSL3'}\n",
      "\n",
      "Gene: MAP2K6\n",
      "Symbol: MAP2K6\n",
      "Name: mitogen-activated protein kinase kinase 6\n",
      "Summary: This gene encodes a member of the dual specificity protein kinase family, which functions as a mitogen-activated protein (MAP) kinase kinase. MAP kinases, also known as extracellular signal-regulated kinases (ERKs), act as an integration point for multiple biochemical signals. This protein phosphorylates and activates p38 MAP kinase in response to inflammatory cytokines or environmental stress. As an essential component of p38 MAP kinase mediated signal transduction pathway, this gene is involved in many cellular processes such as stress induced cell cycle arrest, transcription activation and apoptosis. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - MAPK cascade\n",
      "    - MAPK cascade\n",
      "    - osteoblast differentiation\n",
      "    - response to ischemia\n",
      "    - apoptotic process\n",
      "    - signal transduction\n",
      "    - JNK cascade\n",
      "    - response to xenobiotic stimulus\n",
      "    - ovulation cycle process\n",
      "    - stress-activated protein kinase signaling cascade\n",
      "    - positive regulation of prostaglandin secretion\n",
      "    - nucleotide-binding domain, leucine rich repeat containing receptor signaling pathway\n",
      "    - p38MAPK cascade\n",
      "    - p38MAPK cascade\n",
      "    - signal transduction in response to DNA damage\n",
      "    - positive regulation of apoptotic process\n",
      "    - positive regulation of MAPK cascade\n",
      "    - stress-activated MAPK cascade\n",
      "    - regulation of cell cycle\n",
      "    - cardiac muscle contraction\n",
      "    - bone development\n",
      "    - cellular response to sorbitol\n",
      "    - cellular senescence\n",
      "    - negative regulation of cold-induced thermogenesis\n",
      "    - regulation of signal transduction by p53 class mediator\n",
      "  CC:\n",
      "    - nucleoplasm\n",
      "    - nucleoplasm\n",
      "    - cytoplasm\n",
      "    - cytosol\n",
      "    - cytosol\n",
      "    - cytosol\n",
      "    - cytoskeleton\n",
      "  MF:\n",
      "    - protein serine/threonine kinase activity\n",
      "    - MAP kinase kinase activity\n",
      "    - MAP kinase kinase activity\n",
      "    - MAP kinase kinase activity\n",
      "    - protein tyrosine kinase activity\n",
      "    - protein binding\n",
      "    - ATP binding\n",
      "    - JUN kinase kinase activity\n",
      "    - protein kinase binding\n",
      "    - identical protein binding\n",
      "    - protein serine kinase activity\n",
      "UniProt: {'Swiss-Prot': 'P52564', 'TrEMBL': ['A0A0A0MRF7', 'A8K3Y2', 'K7ELM6', 'K7EIW3']}\n",
      "\n",
      "Gene: TGFB3\n",
      "Symbol: TGFB3\n",
      "Name: transforming growth factor beta 3\n",
      "Summary: This gene encodes a secreted ligand of the TGF-beta (transforming growth factor-beta) superfamily of proteins. Ligands of this family bind various TGF-beta receptors leading to recruitment and activation of SMAD family transcription factors that regulate gene expression. The encoded preproprotein is proteolytically processed to generate a latency-associated peptide (LAP) and a mature peptide, and is found in either a latent form composed of a mature peptide homodimer, a LAP homodimer, and a latent TGF-beta binding protein, or in an active form consisting solely of the mature peptide homodimer. The mature peptide may also form heterodimers with other TGF-beta family members. This protein is involved in embryogenesis and cell differentiation, and may play a role in wound healing. Mutations in this gene are a cause of aortic aneurysms and dissections, as well as familial arrhythmogenic right ventricular dysplasia 1. [provided by RefSeq, Aug 2016].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - response to hypoxia\n",
      "    - in utero embryonic development\n",
      "    - transforming growth factor beta receptor signaling pathway\n",
      "    - transforming growth factor beta receptor signaling pathway\n",
      "    - transforming growth factor beta receptor signaling pathway\n",
      "    - salivary gland morphogenesis\n",
      "    - female pregnancy\n",
      "    - positive regulation of cell population proliferation\n",
      "    - negative regulation of cell population proliferation\n",
      "    - positive regulation of epithelial to mesenchymal transition\n",
      "    - positive regulation of epithelial to mesenchymal transition\n",
      "    - negative regulation of macrophage cytokine production\n",
      "    - negative regulation of transforming growth factor beta receptor signaling pathway\n",
      "    - mammary gland development\n",
      "    - response to progesterone\n",
      "    - positive regulation of collagen biosynthetic process\n",
      "    - response to laminar fluid shear stress\n",
      "    - regulation of cell population proliferation\n",
      "    - odontogenesis\n",
      "    - uterine wall breakdown\n",
      "    - positive regulation of apoptotic process\n",
      "    - positive regulation of MAPK cascade\n",
      "    - negative regulation of neuron apoptotic process\n",
      "    - response to estrogen\n",
      "    - cell-cell junction organization\n",
      "    - positive regulation of DNA-templated transcription\n",
      "    - positive regulation of DNA-templated transcription\n",
      "    - positive regulation of transcription by RNA polymerase II\n",
      "    - lung alveolus development\n",
      "    - digestive tract development\n",
      "    - embryonic neurocranium morphogenesis\n",
      "    - inner ear development\n",
      "    - positive regulation of protein secretion\n",
      "    - positive regulation of filopodium assembly\n",
      "    - positive regulation of stress fiber assembly\n",
      "    - positive regulation of cell division\n",
      "    - face morphogenesis\n",
      "    - frontal suture morphogenesis\n",
      "    - positive regulation of SMAD protein signal transduction\n",
      "    - secondary palate development\n",
      "    - detection of hypoxia\n",
      "    - negative regulation of vascular associated smooth muscle cell proliferation\n",
      "    - regulation of epithelial to mesenchymal transition involved in endocardial cushion formation\n",
      "    - positive regulation of tight junction disassembly\n",
      "  CC:\n",
      "    - extracellular region\n",
      "    - extracellular space\n",
      "    - plasma membrane\n",
      "    - cell surface\n",
      "    - T-tubule\n",
      "    - platelet alpha granule lumen\n",
      "    - neuronal cell body\n",
      "    - intracellular membrane-bounded organelle\n",
      "    - collagen-containing extracellular matrix\n",
      "  MF:\n",
      "    - type II transforming growth factor beta receptor binding\n",
      "    - type II transforming growth factor beta receptor binding\n",
      "    - type II transforming growth factor beta receptor binding\n",
      "    - cytokine activity\n",
      "    - protein binding\n",
      "    - growth factor activity\n",
      "    - type I transforming growth factor beta receptor binding\n",
      "    - type III transforming growth factor beta receptor binding\n",
      "    - identical protein binding\n",
      "    - identical protein binding\n",
      "    - protein-containing complex binding\n",
      "    - transforming growth factor beta binding\n",
      "UniProt: {'Swiss-Prot': 'P10600', 'TrEMBL': ['B3KVH9', 'A5YM40', 'A0A6Q8PGK5']}\n",
      "\n",
      "Gene: HNF4A\n",
      "Symbol: HNF4A\n",
      "Name: hepatocyte nuclear factor 4 alpha\n",
      "Summary: The protein encoded by this gene is a nuclear transcription factor which binds DNA as a homodimer. The encoded protein controls the expression of several genes, including hepatocyte nuclear factor 1 alpha, a transcription factor which regulates the expression of several hepatic genes. This gene may play a role in development of the liver, kidney, and intestines. Mutations in this gene have been associated with monogenic autosomal dominant non-insulin-dependent diabetes mellitus type I. Alternative splicing of this gene results in multiple transcript variants encoding several different isoforms. [provided by RefSeq, Apr 2012].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - regulation of transcription by RNA polymerase II\n",
      "    - regulation of transcription by RNA polymerase II\n",
      "    - transcription by RNA polymerase II\n",
      "    - lipid metabolic process\n",
      "    - xenobiotic metabolic process\n",
      "    - sex differentiation\n",
      "    - blood coagulation\n",
      "    - negative regulation of cell population proliferation\n",
      "    - response to glucose\n",
      "    - regulation of gastrulation\n",
      "    - regulation of lipid metabolic process\n",
      "    - signal transduction involved in regulation of gene expression\n",
      "    - cell differentiation\n",
      "    - negative regulation of cell growth\n",
      "    - intracellular receptor signaling pathway\n",
      "    - glucose homeostasis\n",
      "    - cholesterol homeostasis\n",
      "    - regulation of circadian rhythm\n",
      "    - negative regulation of DNA-templated transcription\n",
      "    - positive regulation of DNA-templated transcription\n",
      "    - positive regulation of transcription by RNA polymerase II\n",
      "    - positive regulation of transcription by RNA polymerase II\n",
      "    - positive regulation of transcription by RNA polymerase II\n",
      "    - rhythmic process\n",
      "    - regulation of insulin secretion\n",
      "    - lipid homeostasis\n",
      "    - phospholipid homeostasis\n",
      "    - regulation of growth hormone receptor signaling pathway\n",
      "    - triglyceride homeostasis\n",
      "    - regulation of ornithine metabolic process\n",
      "  CC:\n",
      "    - chromatin\n",
      "    - nucleus\n",
      "    - nucleoplasm\n",
      "    - nucleoplasm\n",
      "    - cytoplasm\n",
      "  MF:\n",
      "    - transcription cis-regulatory region binding\n",
      "    - RNA polymerase II cis-regulatory region sequence-specific DNA binding\n",
      "    - DNA-binding transcription factor activity, RNA polymerase II-specific\n",
      "    - DNA-binding transcription activator activity, RNA polymerase II-specific\n",
      "    - DNA binding\n",
      "    - chromatin binding\n",
      "    - DNA-binding transcription factor activity\n",
      "    - nuclear receptor activity\n",
      "    - nuclear receptor activity\n",
      "    - signaling receptor binding\n",
      "    - fatty acid binding\n",
      "    - protein binding\n",
      "    - zinc ion binding\n",
      "    - protein homodimerization activity\n",
      "    - RNA polymerase II-specific DNA-binding transcription factor binding\n",
      "    - sequence-specific double-stranded DNA binding\n",
      "UniProt: {'Swiss-Prot': 'P41235', 'TrEMBL': ['B9VVT8', 'F1D8T0', 'F1D8T1', 'F1D8S2', 'A0A087WXV4', 'A0A804HI58', 'F8WBS7', 'A0A804HJ81', 'A0A804HIR4', 'A0A804HJJ6', 'A0A804HJL7', 'A0A804HJ41']}\n",
      "\n",
      "Gene: CD44\n",
      "Symbol: CD44\n",
      "Name: CD44 molecule (IN blood group)\n",
      "Summary: The protein encoded by this gene is a cell-surface glycoprotein involved in cell-cell interactions, cell adhesion and migration. It is a receptor for hyaluronic acid (HA) and can also interact with other ligands, such as osteopontin, collagens, and matrix metalloproteinases (MMPs). This protein participates in a wide variety of cellular functions including lymphocyte activation, recirculation and homing, hematopoiesis, and tumor metastasis. Transcripts for this gene undergo complex alternative splicing that results in many functionally distinct isoforms, however, the full length nature of some of these variants has not been determined. Alternative splicing is the basis for the structural and functional diversity of this protein, and may be related to tumor metastasis. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - inflammatory response\n",
      "    - cell adhesion\n",
      "    - cell adhesion\n",
      "    - cell-matrix adhesion\n",
      "    - cell migration\n",
      "    - cytokine-mediated signaling pathway\n",
      "    - hyaluronan catabolic process\n",
      "    - positive regulation of heterotypic cell-cell adhesion\n",
      "    - T cell activation\n",
      "    - negative regulation of apoptotic process\n",
      "    - negative regulation of DNA damage response, signal transduction by p53 class mediator\n",
      "    - wound healing, spreading of cells\n",
      "    - cellular response to fibroblast growth factor stimulus\n",
      "    - cartilage development\n",
      "    - positive regulation of ERK1 and ERK2 cascade\n",
      "    - positive regulation of ERK1 and ERK2 cascade\n",
      "    - monocyte aggregation\n",
      "    - cell-cell adhesion\n",
      "    - positive regulation of monocyte aggregation\n",
      "    - negative regulation of intrinsic apoptotic signaling pathway in response to DNA damage by p53 class mediator\n",
      "    - regulation of lamellipodium morphogenesis\n",
      "  CC:\n",
      "    - Golgi apparatus\n",
      "    - cytosol\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - microvillus\n",
      "    - focal adhesion\n",
      "    - cell surface\n",
      "    - basolateral plasma membrane\n",
      "    - apical plasma membrane\n",
      "    - secretory granule membrane\n",
      "    - lamellipodium membrane\n",
      "    - macrophage migration inhibitory factor receptor complex\n",
      "    - macrophage migration inhibitory factor receptor complex\n",
      "    - cell projection\n",
      "    - extracellular exosome\n",
      "  MF:\n",
      "    - transmembrane signaling receptor activity\n",
      "    - cytokine receptor activity\n",
      "    - cytokine receptor activity\n",
      "    - protein binding\n",
      "    - collagen binding\n",
      "    - hyaluronic acid binding\n",
      "    - hyaluronic acid binding\n",
      "    - hyaluronic acid binding\n",
      "UniProt: {'Swiss-Prot': 'P16070', 'TrEMBL': ['H0YDW7', 'Q86UZ1', 'H0YF08', 'H0YEV3', 'H0YDV8', 'H0YD13', 'H0Y2P0', 'H0YEU1', 'H0YCV9', 'H0Y5E4', 'H0YES0', 'H0YD17', 'H0YCL4', 'E9PKC6', 'H0YDX6', 'H0YE40', 'H0YEA1', 'H0YD90', 'J3KN83']}\n",
      "\n",
      "Gene: KCNQ1\n",
      "Symbol: KCNQ1\n",
      "Name: potassium voltage-gated channel subfamily Q member 1\n",
      "Summary: This gene encodes a voltage-gated potassium channel required for repolarization phase of the cardiac action potential. This protein can form heteromultimers with two other potassium channel proteins, KCNE1 and KCNE3. Mutations in this gene are associated with hereditary long QT syndrome 1 (also known as Romano-Ward syndrome), Jervell and Lange-Nielsen syndrome, and familial atrial fibrillation. This gene exhibits tissue-specific imprinting, with preferential expression from the maternal allele in some tissues, and biallelic expression in others. This gene is located in a region of chromosome 11 amongst other imprinted genes that are associated with Beckwith-Wiedemann syndrome (BWS), and itself has been shown to be disrupted by chromosomal rearrangements in patients with BWS. Alternatively spliced transcript variants have been found for this gene. [provided by RefSeq, Aug 2011].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - action potential\n",
      "    - gastrin-induced gastric acid secretion\n",
      "    - glucose metabolic process\n",
      "    - heart development\n",
      "    - sensory perception of sound\n",
      "    - rhythmic behavior\n",
      "    - regulation of heart contraction\n",
      "    - regulation of blood pressure\n",
      "    - positive regulation of heart rate\n",
      "    - iodide transport\n",
      "    - erythrocyte differentiation\n",
      "    - intracellular chloride ion homeostasis\n",
      "    - response to insulin\n",
      "    - social behavior\n",
      "    - corticosterone secretion\n",
      "    - inner ear morphogenesis\n",
      "    - inner ear development\n",
      "    - intestinal absorption\n",
      "    - detection of mechanical stimulus involved in sensory perception of sound\n",
      "    - potassium ion homeostasis\n",
      "    - cardiac muscle contraction\n",
      "    - auditory receptor cell development\n",
      "    - regulation of membrane repolarization\n",
      "    - regulation of membrane repolarization\n",
      "    - regulation of ventricular cardiac muscle cell membrane repolarization\n",
      "    - regulation of atrial cardiac muscle cell membrane repolarization\n",
      "    - positive regulation of cardiac muscle contraction\n",
      "    - regulation of gastric acid secretion\n",
      "    - stomach development\n",
      "    - renal absorption\n",
      "    - renal sodium ion absorption\n",
      "    - cellular response to cAMP\n",
      "    - cellular response to cAMP\n",
      "    - cellular response to xenobiotic stimulus\n",
      "    - potassium ion transmembrane transport\n",
      "    - potassium ion transmembrane transport\n",
      "    - cellular response to epinephrine stimulus\n",
      "    - adrenergic receptor signaling pathway\n",
      "    - cardiac muscle cell contraction\n",
      "    - ventricular cardiac muscle cell action potential\n",
      "    - membrane repolarization during action potential\n",
      "    - membrane repolarization during cardiac muscle cell action potential\n",
      "    - membrane repolarization during cardiac muscle cell action potential\n",
      "    - membrane repolarization during cardiac muscle cell action potential\n",
      "    - atrial cardiac muscle cell action potential\n",
      "    - regulation of heart rate by cardiac conduction\n",
      "    - cochlea development\n",
      "    - potassium ion export across plasma membrane\n",
      "    - potassium ion export across plasma membrane\n",
      "    - membrane repolarization during atrial cardiac muscle cell action potential\n",
      "    - membrane repolarization during ventricular cardiac muscle cell action potential\n",
      "    - membrane repolarization during ventricular cardiac muscle cell action potential\n",
      "    - positive regulation of potassium ion transmembrane transport\n",
      "    - negative regulation of delayed rectifier potassium channel activity\n",
      "    - negative regulation of voltage-gated potassium channel activity\n",
      "    - non-motile cilium assembly\n",
      "    - potassium ion import across plasma membrane\n",
      "  CC:\n",
      "    - cytoplasm\n",
      "    - lysosome\n",
      "    - early endosome\n",
      "    - early endosome\n",
      "    - late endosome\n",
      "    - endoplasmic reticulum\n",
      "    - endoplasmic reticulum\n",
      "    - cytosol\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - voltage-gated potassium channel complex\n",
      "    - voltage-gated potassium channel complex\n",
      "    - voltage-gated potassium channel complex\n",
      "    - voltage-gated potassium channel complex\n",
      "    - membrane\n",
      "    - membrane\n",
      "    - basolateral plasma membrane\n",
      "    - apical plasma membrane\n",
      "    - transport vesicle\n",
      "    - cytoplasmic vesicle membrane\n",
      "    - monoatomic ion channel complex\n",
      "    - neuron projection\n",
      "    - neuronal cell body\n",
      "    - membrane raft\n",
      "    - ciliary base\n",
      "    - lumenal side of membrane\n",
      "    - basolateral part of cell\n",
      "  MF:\n",
      "    - voltage-gated potassium channel activity\n",
      "    - voltage-gated potassium channel activity\n",
      "    - delayed rectifier potassium channel activity\n",
      "    - delayed rectifier potassium channel activity\n",
      "    - protein binding\n",
      "    - calmodulin binding\n",
      "    - phosphatidylinositol-4,5-bisphosphate binding\n",
      "    - protein phosphatase 1 binding\n",
      "    - outward rectifier potassium channel activity\n",
      "    - outward rectifier potassium channel activity\n",
      "    - protein kinase A catalytic subunit binding\n",
      "    - protein kinase A regulatory subunit binding\n",
      "    - transmembrane transporter binding\n",
      "    - voltage-gated potassium channel activity involved in cardiac muscle cell action potential repolarization\n",
      "    - voltage-gated potassium channel activity involved in atrial cardiac muscle cell action potential repolarization\n",
      "    - voltage-gated potassium channel activity involved in atrial cardiac muscle cell action potential repolarization\n",
      "    - scaffold protein binding\n",
      "    - voltage-gated potassium channel activity involved in ventricular cardiac muscle cell action potential repolarization\n",
      "    - voltage-gated potassium channel activity involved in ventricular cardiac muscle cell action potential repolarization\n",
      "UniProt: {'Swiss-Prot': 'P51787', 'TrEMBL': ['A0AAQ5BGV1', 'Q96AI9', 'A0AAQ5BGS5', 'A0A2R8YEQ9', 'A0A2R8YDV1', 'E9PPZ0']}\n",
      "\n",
      "Gene: FKBP5\n",
      "Symbol: FKBP5\n",
      "Name: FKBP prolyl isomerase 5\n",
      "Summary: The protein encoded by this gene is a member of the immunophilin protein family, which play a role in immunoregulation and basic cellular processes involving protein folding and trafficking. This encoded protein is a cis-trans prolyl isomerase that binds to the immunosuppressants FK506 and rapamycin. It is thought to mediate calcineurin inhibition. It also interacts functionally with mature hetero-oligomeric progesterone receptor complexes along with the 90 kDa heat shock protein and P23 protein. This gene has been found to have multiple polyadenylation sites. Alternative splicing results in multiple transcript variants.[provided by RefSeq, Mar 2009].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - regulation of transcription by RNA polymerase II\n",
      "    - protein folding\n",
      "    - response to bacterium\n",
      "    - response to cocaine\n",
      "    - chaperone-mediated protein folding\n",
      "    - chaperone-mediated protein folding\n",
      "    - response to alcohol\n",
      "  CC:\n",
      "    - nucleoplasm\n",
      "    - cytoplasm\n",
      "    - cytoplasm\n",
      "    - cytosol\n",
      "    - cytosol\n",
      "    - membrane\n",
      "    - extracellular exosome\n",
      "  MF:\n",
      "    - peptidyl-prolyl cis-trans isomerase activity\n",
      "    - peptidyl-prolyl cis-trans isomerase activity\n",
      "    - protein binding\n",
      "    - FK506 binding\n",
      "    - protein-macromolecule adaptor activity\n",
      "    - heat shock protein binding\n",
      "    - RNA polymerase II CTD heptapeptide repeat P3 isomerase activity\n",
      "    - RNA polymerase II CTD heptapeptide repeat P6 isomerase activity\n",
      "UniProt: {'Swiss-Prot': 'Q13451', 'TrEMBL': 'Q2TA84'}\n",
      "\n",
      "Gene: PSMD6\n",
      "Symbol: PSMD6\n",
      "Name: proteasome 26S subunit, non-ATPase 6\n",
      "Summary: This gene encodes a member of the protease subunit S10 family. The encoded protein is a subunit of the 26S proteasome which colocalizes with DNA damage foci and is involved in the ATP-dependent degradation of ubiquinated proteins. Alternative splicing results in multiple transcript variants [provided by RefSeq, Nov 2012].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - proteasome-mediated ubiquitin-dependent protein catabolic process\n",
      "    - proteasome-mediated ubiquitin-dependent protein catabolic process\n",
      "  CC:\n",
      "    - proteasome complex\n",
      "    - proteasome complex\n",
      "    - extracellular region\n",
      "    - nucleoplasm\n",
      "    - cytosol\n",
      "    - proteasome regulatory particle\n",
      "    - proteasome accessory complex\n",
      "    - secretory granule lumen\n",
      "    - ficolin-1-rich granule lumen\n",
      "  MF:\n",
      "    - protein binding\n",
      "UniProt: {'Swiss-Prot': 'Q15008', 'TrEMBL': ['C9J7B7', 'C9J0E9', 'H7C531']}\n",
      "\n",
      "Gene: ROMO1\n",
      "Symbol: ROMO1\n",
      "Name: reactive oxygen species modulator 1\n",
      "Summary: The protein encoded by this gene is a mitochondrial membrane protein that is responsible for increasing the level of reactive oxygen species (ROS) in cells. The protein also has antimicrobial activity against a variety of bacteria by inducing bacterial membrane breakage. [provided by RefSeq, Nov 2014].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - intracellular protein transport\n",
      "    - protein import into mitochondrial matrix\n",
      "    - killing of cells of another organism\n",
      "    - killing of cells of another organism\n",
      "    - cellular response to reactive oxygen species\n",
      "    - defense response to bacterium\n",
      "    - protein insertion into mitochondrial inner membrane\n",
      "    - protein insertion into mitochondrial inner membrane\n",
      "    - defense response to Gram-negative bacterium\n",
      "    - defense response to Gram-negative bacterium\n",
      "    - defense response to Gram-positive bacterium\n",
      "    - cytolysis by host of symbiont cells\n",
      "    - antimicrobial humoral immune response mediated by antimicrobial peptide\n",
      "    - replicative senescence\n",
      "    - monoatomic cation transmembrane transport\n",
      "    - positive regulation of reactive oxygen species metabolic process\n",
      "  CC:\n",
      "    - mitochondrion\n",
      "    - mitochondrion\n",
      "    - mitochondrial inner membrane\n",
      "    - mitochondrial inner membrane\n",
      "    - TIM23 mitochondrial import inner membrane translocase complex\n",
      "    - TIM23 mitochondrial import inner membrane translocase complex\n",
      "    - TIM23 mitochondrial import inner membrane translocase complex\n",
      "  MF:\n",
      "    - protein binding\n",
      "    - monoatomic cation transmembrane transporter activity\n",
      "UniProt: {'Swiss-Prot': 'P60602'}\n",
      "\n",
      "Gene: MAP4K2\n",
      "Symbol: MAP4K2\n",
      "Name: mitogen-activated protein kinase kinase kinase kinase 2\n",
      "Summary: The protein encoded by this gene is a member of the serine/threonine protein kinase family. Although this kinase is found in many tissues, its expression in lymphoid follicles is restricted to the cells of germinal centre, where it may participate in B-cell differentiation. This kinase can be activated by TNF-alpha, and has been shown to specifically activate MAP kinases. This kinase is also found to interact with TNF receptor-associated factor 2 (TRAF2), which is involved in the activation of MAP3K1/MEKK1. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Apr 2015].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - chromatin remodeling\n",
      "    - protein phosphorylation\n",
      "    - vesicle targeting\n",
      "    - immune response\n",
      "    - JNK cascade\n",
      "    - intracellular signal transduction\n",
      "    - intracellular signal transduction\n",
      "    - positive regulation of JUN kinase activity\n",
      "    - innate immune response\n",
      "    - positive regulation of JNK cascade\n",
      "  CC:\n",
      "    - Golgi membrane\n",
      "    - cytoplasm\n",
      "    - basolateral plasma membrane\n",
      "  MF:\n",
      "    - protein serine/threonine kinase activity\n",
      "    - 3-phosphoinositide-dependent protein kinase activity\n",
      "    - DNA-dependent protein kinase activity\n",
      "    - AMP-activated protein kinase activity\n",
      "    - eukaryotic translation initiation factor 2alpha kinase activity\n",
      "    - ribosomal protein S6 kinase activity\n",
      "    - protein binding\n",
      "    - ATP binding\n",
      "    - MAP kinase kinase kinase kinase activity\n",
      "    - mitogen-activated protein kinase kinase kinase binding\n",
      "    - histone H3S10 kinase activity\n",
      "    - histone H3T11 kinase activity\n",
      "    - histone H3T6 kinase activity\n",
      "    - histone H2AXS139 kinase activity\n",
      "    - histone H3S28 kinase activity\n",
      "    - histone H4S1 kinase activity\n",
      "    - histone H2AS1 kinase activity\n",
      "    - histone H2BS14 kinase activity\n",
      "    - histone H3T3 kinase activity\n",
      "    - histone H2AS121 kinase activity\n",
      "    - Rho-dependent protein serine/threonine kinase activity\n",
      "    - protein serine kinase activity\n",
      "    - histone H2BS36 kinase activity\n",
      "    - histone H3S57 kinase activity\n",
      "    - histone H3T45 kinase activity\n",
      "    - histone H2AT120 kinase activity\n",
      "UniProt: {'Swiss-Prot': 'Q12851', 'TrEMBL': ['F8WCP4', 'F2Z2B3', 'H7C208', 'C9JCU6']}\n",
      "\n",
      "Gene: IGF1\n",
      "Symbol: IGF1\n",
      "Name: insulin like growth factor 1\n",
      "Summary: The protein encoded by this gene is similar to insulin in function and structure and is a member of a family of proteins involved in mediating growth and development. The encoded protein is processed from a precursor, bound by a specific receptor, and secreted. Defects in this gene are a cause of insulin-like growth factor I deficiency. Alternative splicing results in multiple transcript variants encoding different isoforms that may undergo similar processing to generate mature protein. [provided by RefSeq, Sep 2015].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - skeletal system development\n",
      "    - cell activation\n",
      "    - epithelial to mesenchymal transition\n",
      "    - signal transduction\n",
      "    - Ras protein signal transduction\n",
      "    - muscle organ development\n",
      "    - cell population proliferation\n",
      "    - cell population proliferation\n",
      "    - positive regulation of cell population proliferation\n",
      "    - positive regulation of cell population proliferation\n",
      "    - positive regulation of cell population proliferation\n",
      "    - insulin receptor signaling pathway\n",
      "    - response to heat\n",
      "    - glycolate metabolic process\n",
      "    - regulation of gene expression\n",
      "    - positive regulation of glycoprotein biosynthetic process\n",
      "    - positive regulation of cardiac muscle hypertrophy\n",
      "    - positive regulation of cardiac muscle hypertrophy\n",
      "    - positive regulation of gene expression\n",
      "    - negative regulation of gene expression\n",
      "    - skeletal muscle satellite cell maintenance involved in skeletal muscle regeneration\n",
      "    - muscle hypertrophy\n",
      "    - myotube cell development\n",
      "    - positive regulation of smooth muscle cell migration\n",
      "    - proteoglycan biosynthetic process\n",
      "    - proteoglycan biosynthetic process\n",
      "    - positive regulation of cell migration\n",
      "    - activation of protein kinase B activity\n",
      "    - negative regulation of interleukin-1 beta production\n",
      "    - negative regulation of tumor necrosis factor production\n",
      "    - negative regulation of smooth muscle cell apoptotic process\n",
      "    - bone mineralization involved in bone maturation\n",
      "    - wound healing\n",
      "    - positive regulation of activated T cell proliferation\n",
      "    - negative regulation of apoptotic process\n",
      "    - negative regulation of apoptotic process\n",
      "    - positive regulation of DNA binding\n",
      "    - positive regulation of MAPK cascade\n",
      "    - positive regulation of insulin-like growth factor receptor signaling pathway\n",
      "    - myoblast differentiation\n",
      "    - positive regulation of osteoblast differentiation\n",
      "    - positive regulation of glycogen biosynthetic process\n",
      "    - positive regulation of glycolytic process\n",
      "    - positive regulation of mitotic nuclear division\n",
      "    - positive regulation of DNA-templated transcription\n",
      "    - positive regulation of DNA-templated transcription\n",
      "    - positive regulation of transcription by RNA polymerase II\n",
      "    - positive regulation of D-glucose import\n",
      "    - positive regulation of Ras protein signal transduction\n",
      "    - insulin-like growth factor receptor signaling pathway\n",
      "    - insulin-like growth factor receptor signaling pathway\n",
      "    - insulin-like growth factor receptor signaling pathway\n",
      "    - positive regulation of fibroblast proliferation\n",
      "    - positive regulation of smooth muscle cell proliferation\n",
      "    - positive regulation of epithelial cell proliferation\n",
      "    - positive regulation of protein secretion\n",
      "    - protein stabilization\n",
      "    - myoblast proliferation\n",
      "    - positive regulation of phosphatidylinositol 3-kinase/protein kinase B signal transduction\n",
      "    - positive regulation of phosphatidylinositol 3-kinase/protein kinase B signal transduction\n",
      "    - positive regulation of phosphatidylinositol 3-kinase/protein kinase B signal transduction\n",
      "    - positive regulation of phosphatidylinositol 3-kinase/protein kinase B signal transduction\n",
      "    - negative regulation of oocyte development\n",
      "    - growth hormone receptor signaling pathway\n",
      "    - positive regulation of cell growth involved in cardiac muscle cell development\n",
      "    - positive regulation of cell growth involved in cardiac muscle cell development\n",
      "    - positive regulation of ERK1 and ERK2 cascade\n",
      "    - positive regulation of calcineurin-NFAT signaling cascade\n",
      "    - negative regulation of release of cytochrome c from mitochondria\n",
      "    - cell surface receptor signaling pathway via STAT\n",
      "    - negative regulation of neuroinflammatory response\n",
      "    - negative regulation of amyloid-beta formation\n",
      "    - positive regulation of trophectodermal cell proliferation\n",
      "    - cellular response to amyloid-beta\n",
      "    - positive regulation of vascular associated smooth muscle cell proliferation\n",
      "    - negative regulation of vascular associated smooth muscle cell apoptotic process\n",
      "    - positive regulation of transcription regulatory region DNA binding\n",
      "    - negative regulation of extrinsic apoptotic signaling pathway\n",
      "  CC:\n",
      "    - extracellular region\n",
      "    - extracellular region\n",
      "    - extracellular space\n",
      "    - extracellular space\n",
      "    - insulin-like growth factor binding protein complex\n",
      "    - platelet alpha granule lumen\n",
      "    - alphav-beta3 integrin-IGF-1-IGF1R complex\n",
      "    - insulin-like growth factor ternary complex\n",
      "    - exocytic vesicle\n",
      "  MF:\n",
      "    - insulin receptor binding\n",
      "    - insulin-like growth factor receptor binding\n",
      "    - insulin-like growth factor receptor binding\n",
      "    - insulin-like growth factor receptor binding\n",
      "    - insulin-like growth factor receptor binding\n",
      "    - integrin binding\n",
      "    - hormone activity\n",
      "    - hormone activity\n",
      "    - protein binding\n",
      "    - growth factor activity\n",
      "    - transmembrane receptor protein tyrosine kinase activator activity\n",
      "UniProt: {'Swiss-Prot': 'P05019', 'TrEMBL': ['Q59GC5', 'Q5U743', 'Q13429']}\n",
      "\n",
      "Gene: SLC9A1\n",
      "Symbol: SLC9A1\n",
      "Name: solute carrier family 9 member A1\n",
      "Summary: This gene encodes a Na+/H+ antiporter that is a member of the solute carrier family 9. The encoded protein is a plasma membrane transporter that is expressed in the kidney and intestine. This protein plays a central role in regulating pH homeostasis, cell migration and cell volume. This protein may also be involved in tumor growth. [provided by RefSeq, Sep 2011].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - monoatomic ion transport\n",
      "    - intracellular sodium ion homeostasis\n",
      "    - regulation of pH\n",
      "    - regulation of pH\n",
      "    - response to acidic pH\n",
      "    - positive regulation of cardiac muscle hypertrophy\n",
      "    - regulation of cardiac muscle contraction by calcium ion signaling\n",
      "    - cell migration\n",
      "    - maintenance of cell polarity\n",
      "    - positive regulation of cell growth\n",
      "    - cellular response to insulin stimulus\n",
      "    - positive regulation of mitochondrial membrane permeability\n",
      "    - response to muscle stretch\n",
      "    - sodium ion export across plasma membrane\n",
      "    - positive regulation of apoptotic process\n",
      "    - negative regulation of apoptotic process\n",
      "    - positive regulation of action potential\n",
      "    - positive regulation of transcription by RNA polymerase II\n",
      "    - stem cell differentiation\n",
      "    - protein complex oligomerization\n",
      "    - regulation of intracellular pH\n",
      "    - regulation of intracellular pH\n",
      "    - regulation of stress fiber assembly\n",
      "    - regulation of focal adhesion assembly\n",
      "    - cardiac muscle cell differentiation\n",
      "    - cellular response to cold\n",
      "    - positive regulation of calcineurin-NFAT signaling cascade\n",
      "    - cellular response to antibiotic\n",
      "    - cellular response to electrical stimulus\n",
      "    - cellular response to mechanical stimulus\n",
      "    - cellular response to hypoxia\n",
      "    - cellular response to acidic pH\n",
      "    - cellular response to acidic pH\n",
      "    - potassium ion transmembrane transport\n",
      "    - cellular response to epinephrine stimulus\n",
      "    - cardiac muscle cell contraction\n",
      "    - regulation of cardiac muscle cell membrane potential\n",
      "    - regulation of the force of heart contraction by cardiac conduction\n",
      "    - sodium ion import across plasma membrane\n",
      "    - sodium ion import across plasma membrane\n",
      "    - positive regulation of the force of heart contraction\n",
      "    - proton transmembrane transport\n",
      "    - positive regulation of calcium:sodium antiporter activity\n",
      "  CC:\n",
      "    - nucleoplasm\n",
      "    - cytoplasm\n",
      "    - mitochondrion\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - focal adhesion\n",
      "    - cell surface\n",
      "    - intercalated disc\n",
      "    - membrane\n",
      "    - basolateral plasma membrane\n",
      "    - apical plasma membrane\n",
      "    - lamellipodium\n",
      "    - T-tubule\n",
      "    - membrane raft\n",
      "    - perinuclear region of cytoplasm\n",
      "    - extracellular exosome\n",
      "    - cation-transporting ATPase complex\n",
      "  MF:\n",
      "    - protein binding\n",
      "    - calmodulin binding\n",
      "    - phospholipid binding\n",
      "    - phosphatidylinositol-4,5-bisphosphate binding\n",
      "    - sodium:proton antiporter activity\n",
      "    - sodium:proton antiporter activity\n",
      "    - sodium:proton antiporter activity\n",
      "    - sodium:proton antiporter activity\n",
      "    - potassium:proton antiporter activity\n",
      "    - protein phosphatase 2B binding\n",
      "    - protein phosphatase 2B binding\n",
      "    - protein-macromolecule adaptor activity\n",
      "    - identical protein binding\n",
      "    - calcium-dependent protein binding\n",
      "    - molecular adaptor activity\n",
      "UniProt: {'Swiss-Prot': 'P19634', 'TrEMBL': ['B1ALD5', 'B2RAH2']}\n",
      "\n",
      "Gene: NR1H4\n",
      "Symbol: NR1H4\n",
      "Name: nuclear receptor subfamily 1 group H member 4\n",
      "Summary: This gene encodes a ligand-activated transcription factor that shares structural features in common with nuclear hormone receptor family members. This protein functions as a receptor for bile acids, and when bound to bile acids, binds to DNA and regulates the expression of genes involved in bile acid synthesis and transport. Alternatively spliced transcript variants encoding different isoforms have been described. [provided by RefSeq, Feb 2016].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - negative regulation of transcription by RNA polymerase II\n",
      "    - negative regulation of transcription by RNA polymerase II\n",
      "    - intracellular glucose homeostasis\n",
      "    - regulation of DNA-templated transcription\n",
      "    - regulation of transcription by RNA polymerase II\n",
      "    - transcription by RNA polymerase II\n",
      "    - inflammatory response\n",
      "    - cell-cell junction assembly\n",
      "    - Notch signaling pathway\n",
      "    - bile acid metabolic process\n",
      "    - negative regulation of tumor necrosis factor-mediated signaling pathway\n",
      "    - negative regulation of very-low-density lipoprotein particle remodeling\n",
      "    - regulation of low-density lipoprotein particle clearance\n",
      "    - bile acid and bile salt transport\n",
      "    - cell differentiation\n",
      "    - intracellular receptor signaling pathway\n",
      "    - negative regulation of type II interferon production\n",
      "    - negative regulation of interleukin-1 production\n",
      "    - negative regulation of interleukin-2 production\n",
      "    - negative regulation of interleukin-6 production\n",
      "    - negative regulation of tumor necrosis factor production\n",
      "    - positive regulation of interleukin-17 production\n",
      "    - toll-like receptor 9 signaling pathway\n",
      "    - regulation of urea metabolic process\n",
      "    - intracellular triglyceride homeostasis\n",
      "    - positive regulation of insulin secretion involved in cellular response to glucose stimulus\n",
      "    - nuclear receptor-mediated bile acid signaling pathway\n",
      "    - nuclear receptor-mediated bile acid signaling pathway\n",
      "    - cholesterol homeostasis\n",
      "    - defense response to bacterium\n",
      "    - negative regulation of apoptotic process\n",
      "    - negative regulation of canonical NF-kappaB signal transduction\n",
      "    - innate immune response\n",
      "    - positive regulation of DNA-templated transcription\n",
      "    - positive regulation of transcription by RNA polymerase II\n",
      "    - positive regulation of transcription by RNA polymerase II\n",
      "    - positive regulation of transcription by RNA polymerase II\n",
      "    - positive regulation of insulin receptor signaling pathway\n",
      "    - negative regulation of inflammatory response\n",
      "    - fatty acid homeostasis\n",
      "    - regulation of insulin secretion involved in cellular response to glucose stimulus\n",
      "    - regulation of bile acid biosynthetic process\n",
      "    - cellular response to lipopolysaccharide\n",
      "    - cellular response to fatty acid\n",
      "    - negative regulation of monocyte chemotactic protein-1 production\n",
      "    - regulation of cholesterol metabolic process\n",
      "    - cellular response to bile acid\n",
      "    - cellular response to bile acid\n",
      "    - positive regulation of adipose tissue development\n",
      "    - positive regulation of phosphatidic acid biosynthetic process\n",
      "    - positive regulation of glutamate metabolic process\n",
      "    - positive regulation of ammonia assimilation cycle\n",
      "  CC:\n",
      "    - chromatin\n",
      "    - chromatin\n",
      "    - euchromatin\n",
      "    - nucleus\n",
      "    - nucleoplasm\n",
      "    - nucleoplasm\n",
      "    - cytosol\n",
      "    - nuclear speck\n",
      "    - receptor complex\n",
      "    - RNA polymerase II transcription regulator complex\n",
      "  MF:\n",
      "    - transcription cis-regulatory region binding\n",
      "    - RNA polymerase II transcription regulatory region sequence-specific DNA binding\n",
      "    - RNA polymerase II cis-regulatory region sequence-specific DNA binding\n",
      "    - RNA polymerase II cis-regulatory region sequence-specific DNA binding\n",
      "    - DNA-binding transcription factor activity, RNA polymerase II-specific\n",
      "    - DNA-binding transcription factor activity, RNA polymerase II-specific\n",
      "    - transcription coregulator binding\n",
      "    - DNA-binding transcription activator activity, RNA polymerase II-specific\n",
      "    - DNA-binding transcription factor activity\n",
      "    - nuclear receptor activity\n",
      "    - nuclear receptor activity\n",
      "    - protein binding\n",
      "    - zinc ion binding\n",
      "    - nuclear receptor binding\n",
      "    - bile acid binding\n",
      "    - bile acid binding\n",
      "    - bile acid nuclear receptor activity\n",
      "    - bile acid nuclear receptor activity\n",
      "    - sequence-specific DNA binding\n",
      "    - nuclear retinoid X receptor binding\n",
      "    - chenodeoxycholic acid binding\n",
      "UniProt: {'Swiss-Prot': 'Q96RI1', 'TrEMBL': ['F1DAL1', 'B6ZGS9', 'F8W1M1', 'F8W656', 'G8JLB0', 'H0YHD5']}\n",
      "\n",
      "Gene: CTSL\n",
      "Symbol: CTSL\n",
      "Name: cathepsin L\n",
      "Summary: The protein encoded by this gene is a lysosomal cysteine proteinase that plays a major role in intracellular protein catabolism. Its substrates include collagen and elastin, as well as alpha-1 protease inhibitor, a major controlling element of neutrophil elastase activity. The encoded protein has been implicated in several pathologic processes, including myofibril necrosis in myopathies and in myocardial ischemia, and in the renal tubular response to proteinuria. This protein, which is a member of the peptidase C1 family, is a dimer composed of disulfide-linked heavy and light chains, both produced from a single protein precursor. Additionally, this protein cleaves the S1 subunit of the SARS-CoV-2 spike protein, which is necessary for entry of the virus into the cell. [provided by RefSeq, Aug 2020].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - adaptive immune response\n",
      "    - proteolysis\n",
      "    - protein autoprocessing\n",
      "    - fusion of virus membrane with host plasma membrane\n",
      "    - receptor-mediated endocytosis of virus by host cell\n",
      "    - antigen processing and presentation\n",
      "    - antigen processing and presentation of exogenous peptide antigen via MHC class II\n",
      "    - collagen catabolic process\n",
      "    - zymogen activation\n",
      "    - enkephalin processing\n",
      "    - fusion of virus membrane with host endosome membrane\n",
      "    - CD4-positive, alpha-beta T cell lineage commitment\n",
      "    - symbiont entry into host cell\n",
      "    - antigen processing and presentation of peptide antigen\n",
      "    - proteolysis involved in protein catabolic process\n",
      "    - proteolysis involved in protein catabolic process\n",
      "    - elastin catabolic process\n",
      "    - macrophage apoptotic process\n",
      "    - cellular response to thyroid hormone stimulus\n",
      "  CC:\n",
      "    - extracellular region\n",
      "    - extracellular region\n",
      "    - extracellular space\n",
      "    - extracellular space\n",
      "    - nucleus\n",
      "    - lysosome\n",
      "    - lysosome\n",
      "    - multivesicular body\n",
      "    - Golgi apparatus\n",
      "    - plasma membrane\n",
      "    - apical plasma membrane\n",
      "    - endolysosome lumen\n",
      "    - chromaffin granule\n",
      "    - lysosomal lumen\n",
      "    - intracellular membrane-bounded organelle\n",
      "    - collagen-containing extracellular matrix\n",
      "    - extracellular exosome\n",
      "    - endocytic vesicle lumen\n",
      "  MF:\n",
      "    - fibronectin binding\n",
      "    - cysteine-type endopeptidase activity\n",
      "    - cysteine-type endopeptidase activity\n",
      "    - cysteine-type endopeptidase activity\n",
      "    - cysteine-type endopeptidase activity\n",
      "    - protein binding\n",
      "    - collagen binding\n",
      "    - cysteine-type peptidase activity\n",
      "    - histone binding\n",
      "    - proteoglycan binding\n",
      "    - serpin family protein binding\n",
      "UniProt: {'Swiss-Prot': 'P07711', 'TrEMBL': ['A0A7I2V5F7', 'Q9HBQ7', 'A0A7I2YQA2', 'A0A7I2V484', 'Q5T8F0', 'A0A7I2V2H7', 'A0A7I2YQB5', 'A0A7I2V5M3', 'A0A7I2V601']}\n",
      "\n",
      "Gene: DHX58\n",
      "Symbol: DHX58\n",
      "Name: DExH-box helicase 58\n",
      "Summary: Enables double-stranded RNA binding activity; single-stranded RNA binding activity; and zinc ion binding activity. Involved in negative regulation of defense response and negative regulation of type I interferon production. Predicted to be active in cytoplasm. [provided by Alliance of Genome Resources, Apr 2022]\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - cytoplasmic pattern recognition receptor signaling pathway\n",
      "    - cytoplasmic pattern recognition receptor signaling pathway\n",
      "    - response to virus\n",
      "    - response to bacterium\n",
      "    - negative regulation of type I interferon production\n",
      "    - positive regulation of type I interferon production\n",
      "    - negative regulation of MDA-5 signaling pathway\n",
      "    - negative regulation of RIG-I signaling pathway\n",
      "    - negative regulation of RIG-I signaling pathway\n",
      "    - negative regulation of RIG-I signaling pathway\n",
      "    - regulation of innate immune response\n",
      "    - negative regulation of innate immune response\n",
      "    - antiviral innate immune response\n",
      "    - positive regulation of MDA-5 signaling pathway\n",
      "    - positive regulation of RIG-I signaling pathway\n",
      "  CC:\n",
      "    - cytoplasm\n",
      "  MF:\n",
      "    - DNA binding\n",
      "    - RNA helicase activity\n",
      "    - double-stranded RNA binding\n",
      "    - double-stranded RNA binding\n",
      "    - single-stranded RNA binding\n",
      "    - single-stranded RNA binding\n",
      "    - protein binding\n",
      "    - ATP binding\n",
      "    - zinc ion binding\n",
      "    - zinc ion binding\n",
      "    - ATP hydrolysis activity\n",
      "UniProt: {'Swiss-Prot': 'Q96C10', 'TrEMBL': ['C9JE76', 'A0AA51U9C6', 'K7EPP0', 'K7EMZ5', 'C9JG98']}\n",
      "\n",
      "Gene: MAPK10\n",
      "Symbol: MAPK10\n",
      "Name: mitogen-activated protein kinase 10\n",
      "Summary: The protein encoded by this gene is a member of the MAP kinase family. MAP kinases act as integration points for multiple biochemical signals, and thus are involved in a wide variety of cellular processes, such as proliferation, differentiation, transcription regulation and development. This kinase is specifically expressed in a subset of neurons in the nervous system, and is activated by threonine and tyrosine phosphorylation. Targeted deletion of this gene in mice suggests that it may have a role in stress-induced neuronal apoptosis. Alternatively spliced transcript variants encoding different isoforms have been described for this gene. A recent study provided evidence for translational readthrough in this gene, and expression of an additional C-terminally extended isoform via the use of an alternative in-frame translation termination codon. [provided by RefSeq, Dec 2017].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - protein phosphorylation\n",
      "    - protein phosphorylation\n",
      "    - signal transduction\n",
      "    - JNK cascade\n",
      "    - JNK cascade\n",
      "    - response to light stimulus\n",
      "    - Fc-epsilon receptor signaling pathway\n",
      "    - regulation of circadian rhythm\n",
      "    - rhythmic process\n",
      "    - cellular senescence\n",
      "  CC:\n",
      "    - nucleus\n",
      "    - nucleoplasm\n",
      "    - nucleoplasm\n",
      "    - cytoplasm\n",
      "    - cytoplasm\n",
      "    - mitochondrion\n",
      "    - cytosol\n",
      "    - plasma membrane\n",
      "  MF:\n",
      "    - JUN kinase activity\n",
      "    - JUN kinase activity\n",
      "    - JUN kinase activity\n",
      "    - MAP kinase kinase activity\n",
      "    - protein binding\n",
      "    - ATP binding\n",
      "    - protein serine kinase activity\n",
      "UniProt: {'Swiss-Prot': 'P53779', 'TrEMBL': ['Q499Y8', 'A0A286YF97', 'F8W9R5', 'A0A1W2PQX4', 'A0A286YES0', 'A0A286YF02', 'D6RJF9', 'A8MPV1', 'A0A286YEX7', 'A0A286YFB7', 'A0A1W2PPM4', 'A0A286YEQ0', 'A0A1P0B7D2', 'A0A1W2PPW1', 'A0A1W2PRF2', 'A0A286YF04', 'D6RCB1', 'A0A286YF95', 'D6RAU3', 'A0A286YF53', 'A0A286YFH4', 'A0A1W2PRZ2', 'A0A286YES9', 'D6RDG1', 'D6RC26', 'A0A286YFB6', 'A0A286YF19', 'A0A286YES8', 'D6R9C1', 'A0A286YF83', 'A0A286YF35', 'A0A1W2PPZ3', 'A0A286YEV3', 'A0A1W2PRG9', 'A0A286YEQ7', 'A0A286YFD9', 'A0A286YFN2', 'D6RFX8', 'A0A286YEN5', 'A0A286YFK3', 'D6RFZ7', 'A0A286YFE9', 'A0A286YFJ6', 'A0A286YFD7', 'A0A286YEV8', 'A0A286YFI3', 'A0A286YFC0', 'A0A286YFJ4', 'A0A286YF85', 'A0A1W2PQ03', 'A0A286YFJ0', 'A0A286YF62', 'A0A286YFA7', 'A0A286YFE2', 'D6RBH2', 'D6RAJ0', 'A0A286YF80', 'A0A286YEY0', 'A0A286YFM6', 'A0A286YEW4', 'A0A286YFC3', 'A0A286YF43', 'A0A286YEY2', 'A0A286YFA6', 'A0A286YEV5', 'A0A286YEW9', 'A0A286YFA3', 'A0A286YEZ1']}\n",
      "\n",
      "Gene: RHEB\n",
      "Symbol: RHEB\n",
      "Name: Ras homolog, mTORC1 binding\n",
      "Summary: This gene is a member of the small GTPase superfamily and encodes a lipid-anchored, cell membrane protein with five repeats of the RAS-related GTP-binding region. This protein is vital in regulation of growth and cell cycle progression due to its role in the insulin/TOR/S6K signaling pathway. The protein has GTPase activity and shuttles between a GDP-bound form and a GTP-bound form, and farnesylation of the protein is required for this activity. Three pseudogenes have been mapped, two on chromosome 10 and one on chromosome 22. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - signal transduction\n",
      "    - small GTPase-mediated signal transduction\n",
      "    - regulation of macroautophagy\n",
      "    - cellular response to nutrient levels\n",
      "    - positive regulation of TOR signaling\n",
      "    - oligodendrocyte differentiation\n",
      "    - positive regulation of oligodendrocyte differentiation\n",
      "    - regulation of cell cycle\n",
      "    - negative regulation of cold-induced thermogenesis\n",
      "    - positive regulation of TORC1 signaling\n",
      "    - regulation of type B pancreatic cell development\n",
      "  CC:\n",
      "    - Golgi membrane\n",
      "    - spliceosomal complex\n",
      "    - lysosomal membrane\n",
      "    - lysosomal membrane\n",
      "    - endoplasmic reticulum membrane\n",
      "    - cytosol\n",
      "    - plasma membrane\n",
      "    - endomembrane system\n",
      "    - postsynaptic density\n",
      "    - postsynaptic density\n",
      "    - membrane\n",
      "    - extracellular exosome\n",
      "  MF:\n",
      "    - magnesium ion binding\n",
      "    - GTPase activity\n",
      "    - GTPase activity\n",
      "    - GTPase activity\n",
      "    - protein binding\n",
      "    - GTP binding\n",
      "    - GTP binding\n",
      "    - GTP binding\n",
      "    - GDP binding\n",
      "    - GDP binding\n",
      "    - protein kinase binding\n",
      "    - protein kinase activator activity\n",
      "    - protein serine/threonine kinase activator activity\n",
      "UniProt: {'Swiss-Prot': 'Q15382', 'TrEMBL': ['C9J931', 'A0A090N900', 'C9J469', 'F8WBL3']}\n",
      "\n",
      "Gene: RALA\n",
      "Symbol: RALA\n",
      "Name: RAS like proto-oncogene A\n",
      "Summary: The product of this gene belongs to the small GTPase superfamily, Ras family of proteins. GTP-binding proteins mediate the transmembrane signaling initiated by the occupancy of certain cell surface receptors. This gene encodes a low molecular mass ras-like GTP-binding protein that shares about 50% similarity with other ras proteins. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - neural tube closure\n",
      "    - exocytosis\n",
      "    - chemotaxis\n",
      "    - signal transduction\n",
      "    - Ras protein signal transduction\n",
      "    - Ras protein signal transduction\n",
      "    - regulation of exocytosis\n",
      "    - regulation of exocytosis\n",
      "    - receptor internalization\n",
      "    - regulation of actin cytoskeleton organization\n",
      "    - positive regulation of epidermal growth factor receptor signaling pathway\n",
      "    - cell division\n",
      "    - positive regulation of filopodium assembly\n",
      "    - membrane raft localization\n",
      "    - establishment of protein localization to mitochondrion\n",
      "    - positive regulation of mitochondrial fission\n",
      "    - regulation of postsynaptic neurotransmitter receptor internalization\n",
      "  CC:\n",
      "    - mitochondrion\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - focal adhesion\n",
      "    - focal adhesion\n",
      "    - cell surface\n",
      "    - endocytic vesicle\n",
      "    - cytoplasmic vesicle membrane\n",
      "    - cleavage furrow\n",
      "    - extracellular exosome\n",
      "    - Flemming body\n",
      "    - synaptic membrane\n",
      "    - Schaffer collateral - CA1 synapse\n",
      "  MF:\n",
      "    - GTPase activity\n",
      "    - GTPase activity\n",
      "    - G protein activity\n",
      "    - protein binding\n",
      "    - GTP binding\n",
      "    - GTP binding\n",
      "    - myosin binding\n",
      "    - GDP binding\n",
      "    - GDP binding\n",
      "    - ubiquitin protein ligase binding\n",
      "    - Edg-2 lysophosphatidic acid receptor binding\n",
      "    - ATPase binding\n",
      "UniProt: {'Swiss-Prot': 'P11233', 'TrEMBL': ['C9JPE8', 'H7C3P7']}\n",
      "\n",
      "Gene: TYRO3\n",
      "Symbol: TYRO3\n",
      "Name: TYRO3 protein tyrosine kinase\n",
      "Summary: The gene is part of a 3-member transmembrane receptor kinase receptor family with a processed pseudogene distal on chromosome 15. The encoded protein is activated by the products of the growth arrest-specific gene 6 and protein S genes and is involved in controlling cell survival and proliferation, spermatogenesis, immunoregulation and phagocytosis. The encoded protein has also been identified as a cell entry factor for Ebola and Marburg viruses. [provided by RefSeq, May 2010].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - neuron migration\n",
      "    - natural killer cell differentiation\n",
      "    - phagocytosis\n",
      "    - cell adhesion\n",
      "    - signal transduction\n",
      "    - cell surface receptor protein tyrosine kinase signaling pathway\n",
      "    - epidermal growth factor receptor signaling pathway\n",
      "    - neuropeptide signaling pathway\n",
      "    - spermatogenesis\n",
      "    - nervous system development\n",
      "    - insulin receptor signaling pathway\n",
      "    - fibroblast growth factor receptor signaling pathway\n",
      "    - cell migration\n",
      "    - forebrain cell migration\n",
      "    - platelet activation\n",
      "    - brain-derived neurotrophic factor receptor signaling pathway\n",
      "    - secretion by cell\n",
      "    - negative regulation of toll-like receptor signaling pathway\n",
      "    - substrate adhesion-dependent cell spreading\n",
      "    - platelet-derived growth factor receptor-alpha signaling pathway\n",
      "    - platelet-derived growth factor receptor-beta signaling pathway\n",
      "    - vascular endothelial growth factor receptor-1 signaling pathway\n",
      "    - collagen-activated tyrosine kinase receptor signaling pathway\n",
      "    - vascular endothelial growth factor signaling pathway\n",
      "    - Kit signaling pathway\n",
      "    - macrophage colony-stimulating factor signaling pathway\n",
      "    - ovulation cycle\n",
      "    - apoptotic cell clearance\n",
      "    - phosphatidylinositol 3-kinase/protein kinase B signal transduction\n",
      "    - negative regulation of neuron apoptotic process\n",
      "    - negative regulation of innate immune response\n",
      "    - symbiont entry into host cell\n",
      "    - protein autophosphorylation\n",
      "    - insulin-like growth factor receptor signaling pathway\n",
      "    - hepatocyte growth factor receptor signaling pathway\n",
      "    - ephrin receptor signaling pathway\n",
      "    - negative regulation of inflammatory response\n",
      "    - negative regulation of lymphocyte activation\n",
      "    - neuron apoptotic process\n",
      "    - establishment of localization in cell\n",
      "    - positive regulation of phosphatidylinositol 3-kinase/protein kinase B signal transduction\n",
      "    - vagina development\n",
      "    - neuron cellular homeostasis\n",
      "    - platelet aggregation\n",
      "    - platelet aggregation\n",
      "    - positive regulation of viral life cycle\n",
      "  CC:\n",
      "    - nucleus\n",
      "    - nuclear envelope\n",
      "    - endoplasmic reticulum membrane\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - cell surface\n",
      "    - receptor complex\n",
      "  MF:\n",
      "    - virus receptor activity\n",
      "    - protein tyrosine kinase activity\n",
      "    - transmembrane receptor protein tyrosine kinase activity\n",
      "    - transmembrane receptor protein tyrosine kinase activity\n",
      "    - GPI-linked ephrin receptor activity\n",
      "    - transmembrane-ephrin receptor activity\n",
      "    - epidermal growth factor receptor activity\n",
      "    - fibroblast growth factor receptor activity\n",
      "    - hepatocyte growth factor receptor activity\n",
      "    - insulin receptor activity\n",
      "    - insulin-like growth factor receptor activity\n",
      "    - macrophage colony-stimulating factor receptor activity\n",
      "    - platelet-derived growth factor alpha-receptor activity\n",
      "    - platelet-derived growth factor beta-receptor activity\n",
      "    - stem cell factor receptor activity\n",
      "    - vascular endothelial growth factor receptor activity\n",
      "    - protein binding\n",
      "    - ATP binding\n",
      "    - boss receptor activity\n",
      "    - placental growth factor receptor activity\n",
      "    - protein tyrosine kinase collagen receptor activity\n",
      "    - phosphatidylinositol 3-kinase binding\n",
      "    - brain-derived neurotrophic factor receptor activity\n",
      "UniProt: {'Swiss-Prot': 'Q06418', 'TrEMBL': ['H0YKZ2', 'H0YNK6', 'H3BPR7', 'H0YN24']}\n",
      "\n",
      "Gene: MRPL1\n",
      "Symbol: MRPL1\n",
      "Name: mitochondrial ribosomal protein L1\n",
      "Summary: Mammalian mitochondrial ribosomal proteins are encoded by nuclear genes and help in protein synthesis within the mitochondrion. Mitochondrial ribosomes (mitoribosomes) consist of a small 28S subunit and a large 39S subunit. They have an estimated 75% protein to rRNA composition compared to prokaryotic ribosomes, where this ratio is reversed. Another difference between mammalian mitoribosomes and prokaryotic ribosomes is that the latter contain a 5S rRNA. Among different species, the proteins comprising the mitoribosome differ greatly in sequence, and sometimes in biochemical properties, which prevents easy recognition by sequence homology. This gene encodes a 39S subunit protein that belongs to the L1 ribosomal protein family. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - mitochondrial translation\n",
      "  CC:\n",
      "    - mitochondrion\n",
      "    - mitochondrial inner membrane\n",
      "    - mitochondrial inner membrane\n",
      "    - mitochondrial large ribosomal subunit\n",
      "  MF:\n",
      "    - RNA binding\n",
      "    - structural constituent of ribosome\n",
      "    - protein binding\n",
      "UniProt: {'Swiss-Prot': 'Q9BYD6', 'TrEMBL': 'H0Y8N7'}\n",
      "\n",
      "Gene: CD274\n",
      "Symbol: CD274\n",
      "Name: CD274 molecule\n",
      "Summary: This gene encodes an immune inhibitory receptor ligand that is expressed by hematopoietic and non-hematopoietic cells, such as T cells and B cells and various types of tumor cells. The encoded protein is a type I transmembrane protein that has immunoglobulin V-like and C-like domains. Interaction of this ligand with its receptor inhibits T-cell activation and cytokine production. During infection or inflammation of normal tissue, this interaction is important for preventing autoimmunity by maintaining homeostasis of the immune response. In tumor microenvironments, this interaction provides an immune escape for tumor cells through cytotoxic T-cell inactivation. Expression of this gene in tumor cells is considered to be prognostic in many types of human malignancies, including colon cancer and renal cell carcinoma. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Sep 2015].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - adaptive immune response\n",
      "    - negative regulation of T cell mediated immune response to tumor cell\n",
      "    - negative regulation of T cell mediated immune response to tumor cell\n",
      "    - immune response\n",
      "    - immune response\n",
      "    - signal transduction\n",
      "    - cell surface receptor signaling pathway\n",
      "    - cell surface receptor signaling pathway\n",
      "    - positive regulation of cell migration\n",
      "    - T cell costimulation\n",
      "    - T cell costimulation\n",
      "    - negative regulation of type II interferon production\n",
      "    - negative regulation of interleukin-10 production\n",
      "    - positive regulation of interleukin-10 production\n",
      "    - response to cytokine\n",
      "    - TRIF-dependent toll-like receptor signaling pathway\n",
      "    - positive regulation of T cell proliferation\n",
      "    - positive regulation of T cell proliferation\n",
      "    - negative regulation of T cell proliferation\n",
      "    - positive regulation of DNA-templated transcription\n",
      "    - regulation of activated T cell proliferation\n",
      "    - negative regulation of activated T cell proliferation\n",
      "    - regulation of T cell apoptotic process\n",
      "    - cellular response to lipopolysaccharide\n",
      "    - negative regulation of tumor necrosis factor superfamily cytokine production\n",
      "    - regulation of activated CD4-positive, alpha-beta T cell apoptotic process\n",
      "    - positive regulation of activated CD8-positive, alpha-beta T cell apoptotic process\n",
      "    - negative regulation of CD4-positive, alpha-beta T cell proliferation\n",
      "    - negative regulation of CD8-positive, alpha-beta T cell activation\n",
      "    - negative regulation of CD8-positive, alpha-beta T cell activation\n",
      "  CC:\n",
      "    - nucleus\n",
      "    - nucleoplasm\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - external side of plasma membrane\n",
      "    - actin cytoskeleton\n",
      "    - early endosome membrane\n",
      "    - recycling endosome membrane\n",
      "    - extracellular exosome\n",
      "  MF:\n",
      "    - transcription coactivator activity\n",
      "    - protein binding\n",
      "    - receptor ligand activity\n",
      "UniProt: {'Swiss-Prot': 'Q9NZQ7', 'TrEMBL': 'Q0GN75'}\n",
      "\n",
      "Gene: PDCD1\n",
      "Symbol: PDCD1\n",
      "Name: programmed cell death 1\n",
      "Summary: Programmed cell death protein 1 (PDCD1) is an immune-inhibitory receptor expressed in activated T cells; it is involved in the regulation of T-cell functions, including those of effector CD8+ T cells. In addition, this protein can also promote the differentiation of CD4+ T cells into T regulatory cells. PDCD1 is expressed in many types of tumors including melanomas, and has demonstrated to play a role in anti-tumor immunity. Moreover, this protein has been shown to be involved in safeguarding against autoimmunity, however, it can also contribute to the inhibition of effective anti-tumor and anti-microbial immunity. [provided by RefSeq, Aug 2020].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - B cell apoptotic process\n",
      "    - adaptive immune response\n",
      "    - negative regulation of tolerance induction\n",
      "    - negative regulation of T cell mediated immune response to tumor cell\n",
      "    - negative regulation of B cell apoptotic process\n",
      "    - apoptotic process\n",
      "    - humoral immune response\n",
      "    - regulation of immune response\n",
      "    - negative regulation of immune response\n",
      "    - negative regulation of T cell activation\n",
      "    - positive regulation of T cell apoptotic process\n",
      "    - regulatory T cell apoptotic process\n",
      "  CC:\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - external side of plasma membrane\n",
      "  MF:\n",
      "    - protein binding\n",
      "    - signaling receptor activity\n",
      "UniProt: {'Swiss-Prot': 'Q15116', 'TrEMBL': ['H0Y2W6', 'A0A0M3M0G7', 'E7ER21']}\n",
      "\n",
      "Gene: MAPK11\n",
      "Symbol: MAPK11\n",
      "Name: mitogen-activated protein kinase 11\n",
      "Summary: This gene encodes a member of a family of protein kinases that are involved in the integration of biochemical signals for a wide variety of cellular processes, including cell proliferation, differentiation, transcriptional regulation, and development. The encoded protein can be activated by proinflammatory cytokines and environmental stresses through phosphorylation by mitogen activated protein kinase kinases (MKKs). Alternative splicing results in multiple transcript variants. [provided by RefSeq, Mar 2014].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - osteoblast differentiation\n",
      "    - JNK cascade\n",
      "    - positive regulation of gene expression\n",
      "    - stress-activated protein kinase signaling cascade\n",
      "    - positive regulation of interleukin-12 production\n",
      "    - intracellular signal transduction\n",
      "    - p38MAPK cascade\n",
      "    - p38MAPK cascade\n",
      "    - positive regulation of erythrocyte differentiation\n",
      "    - positive regulation of muscle cell differentiation\n",
      "    - stress-activated MAPK cascade\n",
      "    - cardiac muscle cell proliferation\n",
      "    - negative regulation of cardiac muscle cell proliferation\n",
      "    - bone development\n",
      "    - cellular response to interleukin-1\n",
      "    - cellular response to UV-B\n",
      "    - cellular senescence\n",
      "    - cellular response to virus\n",
      "  CC:\n",
      "    - nucleus\n",
      "    - nucleoplasm\n",
      "    - cytoplasm\n",
      "    - cytosol\n",
      "  MF:\n",
      "    - protein serine/threonine kinase activity\n",
      "    - protein serine/threonine kinase activity\n",
      "    - protein serine/threonine kinase activity\n",
      "    - JUN kinase activity\n",
      "    - MAP kinase activity\n",
      "    - protein binding\n",
      "    - ATP binding\n",
      "    - protein serine kinase activity\n",
      "UniProt: {'Swiss-Prot': 'Q15759', 'TrEMBL': 'F8WDP4'}\n",
      "\n",
      "Gene: EZH2\n",
      "Symbol: EZH2\n",
      "Name: enhancer of zeste 2 polycomb repressive complex 2 subunit\n",
      "Summary: This gene encodes a member of the Polycomb-group (PcG) family. PcG family members form multimeric protein complexes, which are involved in maintaining the transcriptional repressive state of genes over successive cell generations. This protein associates with the embryonic ectoderm development protein, the VAV1 oncoprotein, and the X-linked nuclear protein. This protein may play a role in the hematopoietic and central nervous systems. Multiple alternatively splcied transcript variants encoding distinct isoforms have been identified for this gene. [provided by RefSeq, Feb 2011].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - G1/S transition of mitotic cell cycle\n",
      "    - negative regulation of transcription by RNA polymerase II\n",
      "    - chromatin organization\n",
      "    - DNA methylation-dependent constitutive heterochromatin formation\n",
      "    - regulation of DNA-templated transcription\n",
      "    - positive regulation of cell population proliferation\n",
      "    - positive regulation of epithelial to mesenchymal transition\n",
      "    - regulation of gliogenesis\n",
      "    - skeletal muscle satellite cell maintenance involved in skeletal muscle regeneration\n",
      "    - cardiac muscle hypertrophy in response to stress\n",
      "    - cerebellar cortex development\n",
      "    - hippocampus development\n",
      "    - B cell differentiation\n",
      "    - keratinocyte differentiation\n",
      "    - positive regulation of cell migration\n",
      "    - regulatory ncRNA-mediated heterochromatin formation\n",
      "    - heterochromatin formation\n",
      "    - subtelomeric heterochromatin formation\n",
      "    - methylation\n",
      "    - response to estradiol\n",
      "    - negative regulation of transcription elongation by RNA polymerase II\n",
      "    - cellular response to trichostatin A\n",
      "    - protein modification process\n",
      "    - hepatocyte homeostasis\n",
      "    - regulation of circadian rhythm\n",
      "    - positive regulation of MAP kinase activity\n",
      "    - positive regulation of GTPase activity\n",
      "    - negative regulation of keratinocyte differentiation\n",
      "    - negative regulation of gene expression, epigenetic\n",
      "    - negative regulation of gene expression, epigenetic\n",
      "    - negative regulation of gene expression, epigenetic\n",
      "    - negative regulation of DNA-templated transcription\n",
      "    - negative regulation of retinoic acid receptor signaling pathway\n",
      "    - rhythmic process\n",
      "    - stem cell differentiation\n",
      "    - negative regulation of striated muscle cell differentiation\n",
      "    - synaptic transmission, GABAergic\n",
      "    - cellular response to hydrogen peroxide\n",
      "    - G1 to G0 transition\n",
      "    - protein localization to chromatin\n",
      "    - positive regulation of protein serine/threonine kinase activity\n",
      "    - regulation of kidney development\n",
      "    - liver regeneration\n",
      "    - facultative heterochromatin formation\n",
      "    - positive regulation of dendrite development\n",
      "    - negative regulation of cytokine production involved in inflammatory response\n",
      "    - positive regulation of cell cycle G1/S phase transition\n",
      "    - response to tetrachloromethane\n",
      "    - negative regulation of G1/S transition of mitotic cell cycle\n",
      "    - negative regulation of stem cell differentiation\n",
      "  CC:\n",
      "    - chromosome, telomeric region\n",
      "    - chromatin\n",
      "    - nucleus\n",
      "    - nucleus\n",
      "    - nucleoplasm\n",
      "    - nucleoplasm\n",
      "    - chromatin silencing complex\n",
      "    - chromosome\n",
      "    - pericentric heterochromatin\n",
      "    - ESC/E(Z) complex\n",
      "    - ESC/E(Z) complex\n",
      "    - pronucleus\n",
      "    - synapse\n",
      "  MF:\n",
      "    - RNA polymerase II cis-regulatory region sequence-specific DNA binding\n",
      "    - RNA polymerase II core promoter sequence-specific DNA binding\n",
      "    - transcription corepressor binding\n",
      "    - chromatin binding\n",
      "    - chromatin binding\n",
      "    - transcription corepressor activity\n",
      "    - RNA binding\n",
      "    - protein binding\n",
      "    - protein-lysine N-methyltransferase activity\n",
      "    - protein-lysine N-methyltransferase activity\n",
      "    - chromatin DNA binding\n",
      "    - nucleosome binding\n",
      "    - histone methyltransferase activity\n",
      "    - histone binding\n",
      "    - ribonucleoprotein complex binding\n",
      "    - histone H3K27 methyltransferase activity\n",
      "    - histone H3K27 methyltransferase activity\n",
      "    - histone H3K27 methyltransferase activity\n",
      "    - histone H3K27 methyltransferase activity\n",
      "    - primary miRNA binding\n",
      "    - lncRNA binding\n",
      "    - histone H3 methyltransferase activity\n",
      "    - histone H3K27 trimethyltransferase activity\n",
      "    - promoter-specific chromatin binding\n",
      "UniProt: {'Swiss-Prot': 'Q15910', 'TrEMBL': ['A0A090N8E9', 'S4S3R8', 'G3XAL2', 'A0A804HLB8', 'E9PDH6', 'A0A804HJH1']}\n",
      "\n",
      "Gene: MAP3K11\n",
      "Symbol: MAP3K11\n",
      "Name: mitogen-activated protein kinase kinase kinase 11\n",
      "Summary: The protein encoded by this gene is a member of the serine/threonine kinase family. This kinase contains a SH3 domain and a leucine zipper-basic motif. This kinase preferentially activates MAPK8/JNK kinase, and functions as a positive regulator of JNK signaling pathway. This kinase can directly phosphorylate, and activates IkappaB kinase alpha and beta, and is found to be involved in the transcription activity of NF-kappaB mediated by Rho family GTPases and CDC42. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - MAPK cascade\n",
      "    - protein phosphorylation\n",
      "    - microtubule-based process\n",
      "    - microtubule-based process\n",
      "    - JNK cascade\n",
      "    - JNK cascade\n",
      "    - positive regulation of apoptotic process\n",
      "    - positive regulation of JUN kinase activity\n",
      "    - positive regulation of neuron apoptotic process\n",
      "    - cell cycle G1/S phase transition\n",
      "    - positive regulation of JNK cascade\n",
      "    - protein autophosphorylation\n",
      "  CC:\n",
      "    - cytoplasm\n",
      "    - centrosome\n",
      "    - centrosome\n",
      "    - cytosol\n",
      "    - microtubule\n",
      "    - membrane\n",
      "  MF:\n",
      "    - protein kinase activity\n",
      "    - protein serine/threonine kinase activity\n",
      "    - JUN kinase kinase kinase activity\n",
      "    - JUN kinase kinase kinase activity\n",
      "    - JUN kinase kinase kinase activity\n",
      "    - MAP kinase kinase kinase activity\n",
      "    - protein binding\n",
      "    - ATP binding\n",
      "    - small GTPase binding\n",
      "    - mitogen-activated protein kinase kinase binding\n",
      "    - mitogen-activated protein kinase kinase kinase binding\n",
      "    - identical protein binding\n",
      "    - protein homodimerization activity\n",
      "    - protein serine kinase activity\n",
      "UniProt: {'Swiss-Prot': 'Q16584', 'TrEMBL': ['H0YCF5', 'E9PRQ2', 'E9PLB1', 'E9PID4']}\n",
      "\n",
      "Gene: MICALL2\n",
      "Symbol: MICALL2\n",
      "Name: MICAL like 2\n",
      "Summary: Enables filamin binding activity. Involved in positive regulation of protein targeting to mitochondrion. Predicted to be located in several cellular components, including bicellular tight junction; neuron projection; and recycling endosome. Predicted to colocalize with stress fiber. [provided by Alliance of Genome Resources, Apr 2022]\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - actin cytoskeleton organization\n",
      "    - actin filament polymerization\n",
      "    - neuron projection development\n",
      "    - endocytic recycling\n",
      "    - Rab protein signal transduction\n",
      "    - substrate adhesion-dependent cell spreading\n",
      "    - bicellular tight junction assembly\n",
      "    - endosome membrane tubulation\n",
      "    - positive regulation of protein targeting to mitochondrion\n",
      "  CC:\n",
      "    - stress fiber\n",
      "    - plasma membrane\n",
      "    - cell-cell junction\n",
      "    - bicellular tight junction\n",
      "    - actin filament bundle\n",
      "    - neuron projection\n",
      "    - recycling endosome\n",
      "  MF:\n",
      "    - protein binding\n",
      "    - filamin binding\n",
      "    - small GTPase binding\n",
      "    - actinin binding\n",
      "    - metal ion binding\n",
      "    - actin filament binding\n",
      "UniProt: {'Swiss-Prot': 'Q8IY33', 'TrEMBL': 'Q6UWK3'}\n",
      "\n",
      "Gene: LTA\n",
      "Symbol: LTA\n",
      "Name: lymphotoxin alpha\n",
      "Summary: The encoded protein, a member of the tumor necrosis factor family, is a cytokine produced by lymphocytes. The protein is highly inducible, secreted, and forms heterotrimers with lymphotoxin-beta which anchor lymphotoxin-alpha to the cell surface. This protein also mediates a large variety of inflammatory, immunostimulatory, and antiviral responses, is involved in the formation of secondary lymphoid organs during development and plays a role in apoptosis. Genetic variations in this gene are associated with susceptibility to leprosy type 4, myocardial infarction, non-Hodgkin's lymphoma, and psoriatic arthritis. Alternatively spliced transcript variants have been observed for this gene. [provided by RefSeq, Jul 2012].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - response to hypoxia\n",
      "    - positive regulation of chronic inflammatory response to antigenic stimulus\n",
      "    - positive regulation of humoral immune response mediated by circulating immunoglobulin\n",
      "    - apoptotic process\n",
      "    - immune response\n",
      "    - humoral immune response\n",
      "    - signal transduction\n",
      "    - cell surface receptor signaling pathway\n",
      "    - cell-cell signaling\n",
      "    - response to nutrient\n",
      "    - response to xenobiotic stimulus\n",
      "    - response to lipopolysaccharide\n",
      "    - positive regulation of type II interferon production\n",
      "    - positive regulation of canonical NF-kappaB signal transduction\n",
      "    - negative regulation of fibroblast proliferation\n",
      "    - lymph node development\n",
      "    - defense response to Gram-positive bacterium\n",
      "    - positive regulation of glial cell proliferation\n",
      "    - positive regulation of extrinsic apoptotic signaling pathway\n",
      "  CC:\n",
      "    - extracellular space\n",
      "    - plasma membrane\n",
      "  MF:\n",
      "    - signaling receptor binding\n",
      "    - cytokine activity\n",
      "    - tumor necrosis factor receptor binding\n",
      "    - protein binding\n",
      "UniProt: {'Swiss-Prot': 'P01374', 'TrEMBL': 'Q5STV3'}\n",
      "\n",
      "Gene: PLA2G4A\n",
      "Symbol: PLA2G4A\n",
      "Name: phospholipase A2 group IVA\n",
      "Summary: This gene encodes a member of the cytosolic phospholipase A2 group IV family. The enzyme catalyzes the hydrolysis of membrane phospholipids to release arachidonic acid which is subsequently metabolized into eicosanoids. Eicosanoids, including prostaglandins and leukotrienes, are lipid-based cellular hormones that regulate hemodynamics, inflammatory responses, and other intracellular pathways. The hydrolysis reaction also produces lysophospholipids that are converted into platelet-activating factor. The enzyme is activated by increased intracellular Ca(2+) levels and phosphorylation, resulting in its translocation from the cytosol and nucleus to perinuclear membrane vesicles. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Jul 2015].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - prostaglandin biosynthetic process\n",
      "    - prostaglandin biosynthetic process\n",
      "    - prostaglandin biosynthetic process\n",
      "    - positive regulation of T-helper 1 type immune response\n",
      "    - glycerol metabolic process\n",
      "    - monoacylglycerol biosynthetic process\n",
      "    - platelet activating factor biosynthetic process\n",
      "    - icosanoid metabolic process\n",
      "    - positive regulation of platelet activation\n",
      "    - arachidonate metabolic process\n",
      "    - arachidonate metabolic process\n",
      "    - leukotriene biosynthetic process\n",
      "    - positive regulation of prostaglandin secretion\n",
      "    - phosphatidylglycerol catabolic process\n",
      "    - phosphatidylcholine catabolic process\n",
      "    - phosphatidylcholine acyl-chain remodeling\n",
      "    - regulation of cell population proliferation\n",
      "    - positive regulation of macrophage activation\n",
      "    - glycerophospholipid catabolic process\n",
      "    - arachidonate secretion\n",
      "    - establishment of localization in cell\n",
      "    - cellular response to antibiotic\n",
      "  CC:\n",
      "    - Golgi membrane\n",
      "    - nucleus\n",
      "    - nuclear envelope\n",
      "    - cytoplasm\n",
      "    - mitochondrial inner membrane\n",
      "    - endoplasmic reticulum\n",
      "    - endoplasmic reticulum membrane\n",
      "    - Golgi apparatus\n",
      "    - cytosol\n",
      "    - cytosol\n",
      "    - cytosol\n",
      "    - cytosol\n",
      "    - intracellular membrane-bounded organelle\n",
      "  MF:\n",
      "    - lysophospholipase activity\n",
      "    - phospholipase A2 activity\n",
      "    - phospholipase A2 activity\n",
      "    - calcium ion binding\n",
      "    - calcium ion binding\n",
      "    - calcium-dependent phospholipid binding\n",
      "    - calcium-dependent phospholipid binding\n",
      "    - O-acyltransferase activity\n",
      "    - phosphatidylinositol-5-phosphate binding\n",
      "    - phosphatidylinositol-3-phosphate binding\n",
      "    - calcium-dependent phospholipase A2 activity\n",
      "    - calcium-dependent phospholipase A2 activity\n",
      "    - calcium-dependent phospholipase A2 activity\n",
      "    - calcium-dependent phospholipase A2 activity\n",
      "    - calcium-independent phospholipase A2 activity\n",
      "    - phosphatidylinositol-4-phosphate binding\n",
      "    - ceramide 1-phosphate binding\n",
      "UniProt: {'Swiss-Prot': 'P47712', 'TrEMBL': 'B4DZI4'}\n",
      "\n",
      "Gene: PTX3\n",
      "Symbol: PTX3\n",
      "Name: pentraxin 3\n",
      "Summary: This gene encodes a member of the pentraxin protein family. The expression of this protein is induced by inflammatory cytokines in response to inflammatory stimuli in several mesenchymal and epithelial cell types, particularly endothelial cells and mononuclear phagocytes. The protein promotes fibrocyte differentiation and is involved in regulating inflammation and complement activation. It also plays a role in angiogenesis and tissue remodeling. The protein serves as a biomarker for several inflammatory conditions. [provided by RefSeq, Jun 2016].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - ovarian cumulus expansion\n",
      "    - response to yeast\n",
      "    - inflammatory response\n",
      "    - opsonization\n",
      "    - extracellular matrix organization\n",
      "    - negative regulation by host of viral process\n",
      "    - negative regulation by host of viral glycoprotein metabolic process\n",
      "    - innate immune response\n",
      "    - innate immune response\n",
      "    - positive regulation of nitric oxide biosynthetic process\n",
      "    - host-mediated suppression of symbiont invasion\n",
      "    - positive regulation of phagocytosis\n",
      "    - negative regulation of glycoprotein metabolic process\n",
      "  CC:\n",
      "    - extracellular region\n",
      "    - extracellular space\n",
      "    - extracellular space\n",
      "    - extracellular matrix\n",
      "    - specific granule lumen\n",
      "    - tertiary granule lumen\n",
      "  MF:\n",
      "    - complement component C1q complex binding\n",
      "    - complement component C1q complex binding\n",
      "    - (1->3)-beta-D-glucan binding\n",
      "    - protein binding\n",
      "    - identical protein binding\n",
      "    - virion binding\n",
      "UniProt: {'Swiss-Prot': 'P26022'}\n",
      "\n",
      "Gene: MAPK13\n",
      "Symbol: MAPK13\n",
      "Name: mitogen-activated protein kinase 13\n",
      "Summary: This gene encodes a member of the mitogen-activated protein (MAP) kinase family. MAP kinases act as an integration point for multiple biochemical signals, and are involved in a wide variety of cellular processes such as proliferation, differentiation, transcription regulation and development. The encoded protein is a p38 MAP kinase and is activated by proinflammatory cytokines and cellular stress. Substrates of the encoded protein include the transcription factor ATF2 and the microtubule dynamics regulator stathmin. Alternatively spliced transcript variants have been observed for this gene. [provided by RefSeq, Jul 2012].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - response to osmotic stress\n",
      "    - JNK cascade\n",
      "    - peptidyl-serine phosphorylation\n",
      "    - positive regulation of interleukin-6 production\n",
      "    - cellular response to UV\n",
      "    - intracellular signal transduction\n",
      "    - positive regulation of inflammatory response\n",
      "    - stress-activated MAPK cascade\n",
      "    - cellular response to hydrogen peroxide\n",
      "    - cellular response to interleukin-1\n",
      "    - cellular response to sorbitol\n",
      "    - cellular response to anisomycin\n",
      "    - cellular response to sodium arsenite\n",
      "  CC:\n",
      "    - nucleus\n",
      "    - cytoplasm\n",
      "    - cytosol\n",
      "  MF:\n",
      "    - protein serine/threonine kinase activity\n",
      "    - protein serine/threonine kinase activity\n",
      "    - JUN kinase activity\n",
      "    - MAP kinase activity\n",
      "    - protein binding\n",
      "    - ATP binding\n",
      "    - protein serine kinase activity\n",
      "UniProt: {'Swiss-Prot': 'O15264', 'TrEMBL': ['Q5R3E6', 'A0A024RD04']}\n",
      "\n",
      "Gene: CSNK1E\n",
      "Symbol: CSNK1E\n",
      "Name: casein kinase 1 epsilon\n",
      "Summary: The protein encoded by this gene is a serine/threonine protein kinase and a member of the casein kinase I protein family, whose members have been implicated in the control of cytoplasmic and nuclear processes, including DNA replication and repair. The encoded protein is found in the cytoplasm as a monomer and can phosphorylate a variety of proteins, including itself. This protein has been shown to phosphorylate period, a circadian rhythm protein. Two transcript variants encoding the same protein have been found for this gene. [provided by RefSeq, Feb 2014].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - DNA repair\n",
      "    - chromatin remodeling\n",
      "    - protein phosphorylation\n",
      "    - protein phosphorylation\n",
      "    - endocytosis\n",
      "    - signal transduction\n",
      "    - protein localization\n",
      "    - negative regulation of Wnt signaling pathway\n",
      "    - negative regulation of protein binding\n",
      "    - positive regulation of proteasomal ubiquitin-dependent protein catabolic process\n",
      "    - positive regulation of proteasomal ubiquitin-dependent protein catabolic process\n",
      "    - regulation of protein localization\n",
      "    - circadian regulation of gene expression\n",
      "    - regulation of circadian rhythm\n",
      "    - circadian behavior\n",
      "    - canonical Wnt signaling pathway\n",
      "    - canonical Wnt signaling pathway\n",
      "    - canonical Wnt signaling pathway\n",
      "    - positive regulation of canonical Wnt signaling pathway\n",
      "    - positive regulation of amyloid-beta formation\n",
      "    - cellular response to nerve growth factor stimulus\n",
      "    - positive regulation of non-canonical Wnt signaling pathway\n",
      "  CC:\n",
      "    - nucleus\n",
      "    - nucleus\n",
      "    - nucleoplasm\n",
      "    - cytoplasm\n",
      "    - cytoplasm\n",
      "    - cytosol\n",
      "    - growth cone\n",
      "    - neuronal cell body\n",
      "    - ribonucleoprotein complex\n",
      "  MF:\n",
      "    - RNA binding\n",
      "    - protein kinase activity\n",
      "    - protein kinase activity\n",
      "    - protein serine/threonine kinase activity\n",
      "    - protein serine/threonine kinase activity\n",
      "    - protein serine/threonine kinase activity\n",
      "    - 3-phosphoinositide-dependent protein kinase activity\n",
      "    - DNA-dependent protein kinase activity\n",
      "    - AMP-activated protein kinase activity\n",
      "    - eukaryotic translation initiation factor 2alpha kinase activity\n",
      "    - ribosomal protein S6 kinase activity\n",
      "    - protein binding\n",
      "    - ATP binding\n",
      "    - histone H3S10 kinase activity\n",
      "    - histone H3T11 kinase activity\n",
      "    - histone H3T6 kinase activity\n",
      "    - histone H2AXS139 kinase activity\n",
      "    - histone H3S28 kinase activity\n",
      "    - histone H4S1 kinase activity\n",
      "    - histone H2AS1 kinase activity\n",
      "    - histone H2BS14 kinase activity\n",
      "    - histone H3T3 kinase activity\n",
      "    - histone H2AS121 kinase activity\n",
      "    - Rho-dependent protein serine/threonine kinase activity\n",
      "    - protein serine kinase activity\n",
      "    - histone H2BS36 kinase activity\n",
      "    - histone H3S57 kinase activity\n",
      "    - histone H3T45 kinase activity\n",
      "    - histone H2AT120 kinase activity\n",
      "UniProt: {'Swiss-Prot': 'P49674', 'TrEMBL': ['H7C3J2', 'Q5U045', 'H0Y5S9', 'H7C0Y5', 'B0QY34', 'H0Y645', 'B0QY36', 'H0Y682', 'A0A669KBC0', 'B0QY35']}\n",
      "\n",
      "Gene: CD33\n",
      "Symbol: CD33\n",
      "Name: CD33 molecule\n",
      "Summary: Enables protein phosphatase binding activity and sialic acid binding activity. Involved in several processes, including negative regulation of cytokine production; negative regulation of monocyte activation; and positive regulation of protein tyrosine phosphatase activity. Located in several cellular components, including Golgi apparatus; external side of plasma membrane; and peroxisome. [provided by Alliance of Genome Resources, Apr 2022]\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - immune response-inhibiting signal transduction\n",
      "    - cell adhesion\n",
      "    - signal transduction\n",
      "    - cell-cell signaling\n",
      "    - negative regulation of cell population proliferation\n",
      "    - negative regulation of interleukin-1 beta production\n",
      "    - negative regulation of interleukin-8 production\n",
      "    - negative regulation of tumor necrosis factor production\n",
      "    - Fc-gamma receptor signaling pathway\n",
      "    - positive regulation of protein secretion\n",
      "    - cell-cell adhesion\n",
      "    - negative regulation of monocyte activation\n",
      "  CC:\n",
      "    - peroxisome\n",
      "    - peroxisome\n",
      "    - Golgi apparatus\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - external side of plasma membrane\n",
      "    - cell surface\n",
      "    - specific granule membrane\n",
      "    - tertiary granule membrane\n",
      "  MF:\n",
      "    - protein binding\n",
      "    - protein tyrosine phosphatase activator activity\n",
      "    - protein phosphatase binding\n",
      "    - carbohydrate binding\n",
      "    - sialic acid binding\n",
      "    - sialic acid binding\n",
      "    - signaling receptor activity\n",
      "UniProt: {'Swiss-Prot': 'P20138', 'TrEMBL': ['B4DF51', 'Q546G0']}\n",
      "\n",
      "Gene: ERCC2\n",
      "Symbol: ERCC2\n",
      "Name: ERCC excision repair 2, TFIIH core complex helicase subunit\n",
      "Summary: The nucleotide excision repair pathway is a mechanism to repair damage to DNA. The protein encoded by this gene is involved in transcription-coupled nucleotide excision repair and is an integral member of the basal transcription factor BTF2/TFIIH complex. The gene product has ATP-dependent DNA helicase activity and belongs to the RAD3/XPD subfamily of helicases. Defects in this gene can result in three different disorders, the cancer-prone syndrome xeroderma pigmentosum complementation group D, trichothiodystrophy, and Cockayne syndrome. Alternatively spliced transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Aug 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - maturation of SSU-rRNA from tricistronic rRNA transcript (SSU-rRNA, 5.8S rRNA, LSU-rRNA)\n",
      "    - response to hypoxia\n",
      "    - in utero embryonic development\n",
      "    - transcription-coupled nucleotide-excision repair\n",
      "    - transcription-coupled nucleotide-excision repair\n",
      "    - nucleotide-excision repair\n",
      "    - nucleotide-excision repair\n",
      "    - regulation of transcription by RNA polymerase II\n",
      "    - transcription elongation by RNA polymerase I\n",
      "    - transcription by RNA polymerase II\n",
      "    - transcription by RNA polymerase II\n",
      "    - transcription by RNA polymerase II\n",
      "    - transcription initiation at RNA polymerase II promoter\n",
      "    - apoptotic process\n",
      "    - response to oxidative stress\n",
      "    - chromosome segregation\n",
      "    - determination of adult lifespan\n",
      "    - UV protection\n",
      "    - post-embryonic development\n",
      "    - spinal cord development\n",
      "    - extracellular matrix organization\n",
      "    - bone mineralization\n",
      "    - central nervous system myelin formation\n",
      "    - multicellular organism growth\n",
      "    - hair cell differentiation\n",
      "    - embryonic cleavage\n",
      "    - erythrocyte maturation\n",
      "    - positive regulation of mitotic recombination\n",
      "    - insulin-like growth factor receptor signaling pathway\n",
      "    - embryonic organ development\n",
      "    - hair follicle maturation\n",
      "    - hematopoietic stem cell differentiation\n",
      "    - hematopoietic stem cell proliferation\n",
      "    - intrinsic apoptotic signaling pathway by p53 class mediator\n",
      "    - regulation of mitotic cell cycle phase transition\n",
      "  CC:\n",
      "    - transcription factor TFIIH core complex\n",
      "    - nucleus\n",
      "    - nucleus\n",
      "    - nucleoplasm\n",
      "    - nucleoplasm\n",
      "    - transcription factor TFIID complex\n",
      "    - transcription factor TFIIH holo complex\n",
      "    - transcription factor TFIIH holo complex\n",
      "    - cytoplasm\n",
      "    - spindle\n",
      "    - cytosol\n",
      "    - CAK-ERCC2 complex\n",
      "    - MMXD complex\n",
      "  MF:\n",
      "    - DNA helicase activity\n",
      "    - damaged DNA binding\n",
      "    - protein binding\n",
      "    - ATP binding\n",
      "    - four-way junction helicase activity\n",
      "    - isomerase activity\n",
      "    - ATP hydrolysis activity\n",
      "    - protein-macromolecule adaptor activity\n",
      "    - double-stranded DNA helicase activity\n",
      "    - 5'-3' DNA helicase activity\n",
      "    - 5'-3' DNA helicase activity\n",
      "    - metal ion binding\n",
      "    - 4 iron, 4 sulfur cluster binding\n",
      "    - forked DNA-dependent helicase activity\n",
      "    - single-stranded 3'-5' DNA helicase activity\n",
      "UniProt: {'Swiss-Prot': 'P18074', 'TrEMBL': ['K7EKF3', 'B4E0F6', 'K7EIT8', 'A0A804HK75', 'A8MX75', 'A0A804HK53', 'A0A804HL97', 'E7EVE9', 'K7ENL1']}\n",
      "\n",
      "Gene: CCNB3\n",
      "Symbol: CCNB3\n",
      "Name: cyclin B3\n",
      "Summary: The protein encoded by this gene belongs to the highly conserved cyclin family, whose members are characterized by a dramatic periodicity in protein abundance through the cell cycle. Cyclins function as positive regulators of cyclin-dependent kinases (CDKs), and thereby play an essential role in the control of the cell cycle. Different cyclins exhibit distinct expression and degradation patterns, which contribute to the temporal coordination of each mitotic event. Studies of similar genes in chicken and drosophila suggest that this cyclin may associate with CDC2 and CDK2 kinases, and may be required for proper spindle reorganization and restoration of the interphase nucleus. Alternatively spliced transcript variants encoding different isoforms have been described for this gene. [provided by RefSeq, Oct 2011].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - G1/S transition of mitotic cell cycle\n",
      "    - cell division\n",
      "    - meiotic cell cycle\n",
      "  CC:\n",
      "    - cyclin-dependent protein kinase holoenzyme complex\n",
      "    - nucleus\n",
      "    - nucleoplasm\n",
      "    - cytoplasm\n",
      "  MF:\n",
      "    - protein binding\n",
      "    - cyclin-dependent protein serine/threonine kinase regulator activity\n",
      "UniProt: {'Swiss-Prot': 'Q8WWL7'}\n",
      "\n",
      "Gene: ALOX5\n",
      "Symbol: ALOX5\n",
      "Name: arachidonate 5-lipoxygenase\n",
      "Summary: This gene encodes a member of the lipoxygenase gene family and plays a dual role in the synthesis of leukotrienes from arachidonic acid. The encoded protein, which is expressed specifically in bone marrow-derived cells, catalyzes the conversion of arachidonic acid to 5(S)-hydroperoxy-6-trans-8,11,14-cis-eicosatetraenoic acid, and further to the allylic epoxide 5(S)-trans-7,9-trans-11,14-cis-eicosatetrenoic acid (leukotriene A4). Leukotrienes are important mediators of a number of inflammatory and allergic conditions. Mutations in the promoter region of this gene lead to a diminished response to antileukotriene drugs used in the treatment of asthma and may also be associated with atherosclerosis and several cancers. Alternatively spliced transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Jan 2012].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - negative regulation of endothelial cell proliferation\n",
      "    - leukocyte chemotaxis involved in inflammatory response\n",
      "    - leukocyte migration involved in inflammatory response\n",
      "    - leukotriene production involved in inflammatory response\n",
      "    - leukotriene metabolic process\n",
      "    - humoral immune response\n",
      "    - negative regulation of angiogenesis\n",
      "    - arachidonate metabolic process\n",
      "    - leukotriene biosynthetic process\n",
      "    - lipoxygenase pathway\n",
      "    - lipoxygenase pathway\n",
      "    - positive regulation of bone mineralization\n",
      "    - lipid oxidation\n",
      "    - dendritic cell migration\n",
      "    - glucose homeostasis\n",
      "    - long-chain fatty acid biosynthetic process\n",
      "    - regulation of fat cell differentiation\n",
      "    - regulation of inflammatory response\n",
      "    - negative regulation of inflammatory response\n",
      "    - regulation of insulin secretion\n",
      "    - negative regulation of vascular wound healing\n",
      "    - negative regulation of wound healing\n",
      "    - regulation of inflammatory response to wounding\n",
      "    - regulation of cytokine production involved in inflammatory response\n",
      "    - regulation of cellular response to oxidative stress\n",
      "    - leukotriene A4 biosynthetic process\n",
      "    - leukotriene A4 biosynthetic process\n",
      "    - regulation of reactive oxygen species biosynthetic process\n",
      "    - negative regulation of response to endoplasmic reticulum stress\n",
      "    - negative regulation of sprouting angiogenesis\n",
      "    - positive regulation of leukocyte adhesion to arterial endothelial cell\n",
      "    - lipoxin biosynthetic process\n",
      "    - lipoxin biosynthetic process\n",
      "  CC:\n",
      "    - extracellular region\n",
      "    - extracellular space\n",
      "    - nuclear envelope\n",
      "    - nuclear envelope\n",
      "    - nuclear envelope\n",
      "    - nuclear envelope lumen\n",
      "    - nucleoplasm\n",
      "    - cytosol\n",
      "    - cytosol\n",
      "    - nuclear matrix\n",
      "    - nuclear membrane\n",
      "    - secretory granule lumen\n",
      "    - perinuclear region of cytoplasm\n",
      "    - ficolin-1-rich granule lumen\n",
      "  MF:\n",
      "    - arachidonate 5-lipoxygenase activity\n",
      "    - arachidonate 5-lipoxygenase activity\n",
      "    - arachidonate 12(S)-lipoxygenase activity\n",
      "    - iron ion binding\n",
      "    - protein binding\n",
      "    - oxidoreductase activity\n",
      "    - oxidoreductase activity, acting on single donors with incorporation of molecular oxygen\n",
      "    - hydrolase activity\n",
      "    - arachidonate 8(S)-lipoxygenase activity\n",
      "UniProt: {'Swiss-Prot': 'P09917', 'TrEMBL': 'A0A087X109'}\n",
      "\n",
      "Gene: BMX\n",
      "Symbol: BMX\n",
      "Name: BMX non-receptor tyrosine kinase\n",
      "Summary: This gene encodes a non-receptor tyrosine kinase belonging to the Tec kinase family. The protein contains a PH-like domain, which mediates membrane targeting by binding to phosphatidylinositol 3,4,5-triphosphate (PIP3), and a SH2 domain that binds to tyrosine-phosphorylated proteins and functions in signal transduction. The protein is implicated in several signal transduction pathways including the Stat pathway, and regulates differentiation and tumorigenicity of several types of cancer cells. Alternatively spliced transcript variants have been found for this gene. [provided by RefSeq, Mar 2016].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - adaptive immune response\n",
      "    - protein phosphorylation\n",
      "    - phosphatidylinositol biosynthetic process\n",
      "    - apoptotic process\n",
      "    - cell adhesion\n",
      "    - signal transduction\n",
      "    - mesoderm development\n",
      "    - intracellular signal transduction\n",
      "    - protein autophosphorylation\n",
      "    - B cell receptor signaling pathway\n",
      "  CC:\n",
      "    - nucleoplasm\n",
      "    - cytosol\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - ruffle membrane\n",
      "  MF:\n",
      "    - protein tyrosine kinase activity\n",
      "    - non-membrane spanning protein tyrosine kinase activity\n",
      "    - non-membrane spanning protein tyrosine kinase activity\n",
      "    - non-membrane spanning protein tyrosine kinase activity\n",
      "    - protein binding\n",
      "    - ATP binding\n",
      "    - zinc ion binding\n",
      "UniProt: {'Swiss-Prot': 'P51813'}\n",
      "\n",
      "Gene: CSRP1\n",
      "Symbol: CSRP1\n",
      "Name: cysteine and glycine rich protein 1\n",
      "Summary: This gene encodes a member of the cysteine-rich protein (CSRP) family. This gene family includes a group of LIM domain proteins, which may be involved in regulatory processes important for development and cellular differentiation. The LIM/double zinc-finger motif found in this gene product occurs in proteins with critical functions in gene regulation, cell growth, and somatic differentiation. Alternatively spliced transcript variants have been described. [provided by RefSeq, Aug 2010].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - sarcomere organization\n",
      "    - muscle tissue development\n",
      "    - platelet aggregation\n",
      "  CC:\n",
      "    - nucleus\n",
      "    - nucleus\n",
      "    - cytoplasm\n",
      "    - focal adhesion\n",
      "    - Z disc\n",
      "    - extracellular exosome\n",
      "  MF:\n",
      "    - RNA binding\n",
      "    - protein binding\n",
      "    - zinc ion binding\n",
      "    - structural constituent of muscle\n",
      "    - actinin binding\n",
      "UniProt: {'Swiss-Prot': 'P21291', 'TrEMBL': ['A0A384P5K2', 'B4DY28', 'E9PS42', 'E9PP21', 'E9PND2', 'A0A087X134']}\n",
      "\n",
      "Gene: ALDH3A2\n",
      "Symbol: ALDH3A2\n",
      "Name: aldehyde dehydrogenase 3 family member A2\n",
      "Summary: Aldehyde dehydrogenase isozymes are thought to play a major role in the detoxification of aldehydes generated by alcohol metabolism and lipid peroxidation. This gene product catalyzes the oxidation of long-chain aliphatic aldehydes to fatty acid. Mutations in the gene cause Sjogren-Larsson syndrome. Alternatively spliced transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - aldehyde metabolic process\n",
      "    - aldehyde metabolic process\n",
      "    - aldehyde metabolic process\n",
      "    - fatty acid metabolic process\n",
      "    - sesquiterpenoid metabolic process\n",
      "    - central nervous system development\n",
      "    - peripheral nervous system development\n",
      "    - epidermis development\n",
      "    - phytol metabolic process\n",
      "    - hexadecanal metabolic process\n",
      "  CC:\n",
      "    - cytoplasm\n",
      "    - peroxisome\n",
      "    - peroxisomal membrane\n",
      "    - endoplasmic reticulum membrane\n",
      "    - cytosol\n",
      "    - intracellular membrane-bounded organelle\n",
      "  MF:\n",
      "    - 3-chloroallyl aldehyde dehydrogenase activity\n",
      "    - aldehyde dehydrogenase (NAD+) activity\n",
      "    - aldehyde dehydrogenase (NAD+) activity\n",
      "    - aldehyde dehydrogenase (NAD+) activity\n",
      "    - protein binding\n",
      "    - protein homodimerization activity\n",
      "    - long-chain-alcohol oxidase activity\n",
      "    - long-chain fatty aldehyde dehydrogenase (NAD+) activity\n",
      "    - medium-chain fatty aldehyde dehydrogenase (NAD+) activity\n",
      "UniProt: {'Swiss-Prot': 'P51648', 'TrEMBL': ['A0A5F9ZHN9', 'I3L0X1', 'J3QRD1', 'I3L1M4', 'A0A5K1VW72', 'J3QKK9', 'A0A0D9SGC3', 'J3QQV9', 'A0A5F9ZHP6', 'A0A5F9ZHZ9', 'I3L4G6', 'I3L2W1', 'J3QS00', 'A0A5F9ZHI5', 'A0A5F9ZI12', 'J3KTG1', 'K7EN73', 'A0A5F9ZGX4', 'J3KTD9']}\n",
      "\n",
      "Gene: RYR1\n",
      "Symbol: RYR1\n",
      "Name: ryanodine receptor 1\n",
      "Summary: This gene encodes a ryanodine receptor found in skeletal muscle. The encoded protein functions as a calcium release channel in the sarcoplasmic reticulum but also serves to connect the sarcoplasmic reticulum and transverse tubule. Mutations in this gene are associated with malignant hyperthermia susceptibility, central core disease, and minicore myopathy with external ophthalmoplegia. Alternatively spliced transcripts encoding different isoforms have been described. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - response to hypoxia\n",
      "    - outflow tract morphogenesis\n",
      "    - calcium ion transport\n",
      "    - muscle contraction\n",
      "    - striated muscle contraction\n",
      "    - release of sequestered calcium ion into cytosol by sarcoplasmic reticulum\n",
      "    - release of sequestered calcium ion into cytosol by sarcoplasmic reticulum\n",
      "    - calcium-mediated signaling\n",
      "    - response to caffeine\n",
      "    - skin development\n",
      "    - ossification involved in bone maturation\n",
      "    - skeletal muscle fiber development\n",
      "    - release of sequestered calcium ion into cytosol\n",
      "    - release of sequestered calcium ion into cytosol\n",
      "    - release of sequestered calcium ion into cytosol\n",
      "    - protein homotetramerization\n",
      "    - regulation of cytosolic calcium ion concentration\n",
      "    - cellular response to calcium ion\n",
      "    - cellular response to caffeine\n",
      "    - cellular response to caffeine\n",
      "  CC:\n",
      "    - cytoplasm\n",
      "    - smooth endoplasmic reticulum\n",
      "    - plasma membrane\n",
      "    - cell cortex\n",
      "    - junctional sarcoplasmic reticulum membrane\n",
      "    - terminal cisterna\n",
      "    - sarcoplasmic reticulum\n",
      "    - Z disc\n",
      "    - organelle membrane\n",
      "    - I band\n",
      "    - sarcoplasmic reticulum membrane\n",
      "    - sarcoplasmic reticulum membrane\n",
      "    - sarcoplasmic reticulum membrane\n",
      "    - sarcoplasmic reticulum membrane\n",
      "    - calcium channel complex\n",
      "    - sarcolemma\n",
      "    - extracellular exosome\n",
      "    - ryanodine receptor complex\n",
      "  MF:\n",
      "    - ryanodine-sensitive calcium-release channel activity\n",
      "    - ryanodine-sensitive calcium-release channel activity\n",
      "    - ryanodine-sensitive calcium-release channel activity\n",
      "    - ryanodine-sensitive calcium-release channel activity\n",
      "    - ryanodine-sensitive calcium-release channel activity\n",
      "    - voltage-gated calcium channel activity\n",
      "    - calcium channel activity\n",
      "    - calcium ion binding\n",
      "    - calmodulin binding\n",
      "    - ATP binding\n",
      "    - intracellularly gated calcium channel activity\n",
      "    - calcium-induced calcium release activity\n",
      "UniProt: {'Swiss-Prot': 'P21817', 'TrEMBL': ['M0R014', 'A0A8I5KSR1', 'M0R127', 'A0AAQ5BH72', 'A0AAQ5BH76', 'M0R0S0', 'A0A8I5KUY6']}\n",
      "\n",
      "Gene: PTPN7\n",
      "Symbol: PTPN7\n",
      "Name: protein tyrosine phosphatase non-receptor type 7\n",
      "Summary: The protein encoded by this gene is a member of the protein tyrosine phosphatase (PTP) family. PTPs are known to be signaling molecules that regulate a variety of cellular processes including cell growth, differentiation, mitotic cycle, and oncogenic transformation. This gene is preferentially expressed in a variety of hematopoietic cells, and is an early response gene in lymphokine stimulated cells. The non-catalytic N-terminus of this PTP can interact with MAP kinases and suppress the MAP kinase activities. This PTP was shown to be involved in the regulation of T cell antigen receptor (TCR) signaling, which was thought to function through dephosphorylating the molecules related to MAP kinase pathway. Multiple alternatively spliced transcript variants have been found for this gene. [provided by RefSeq, Dec 2010].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - MAPK cascade\n",
      "    - chromatin remodeling\n",
      "    - protein dephosphorylation\n",
      "    - signal transduction\n",
      "  CC:\n",
      "    - nucleoplasm\n",
      "    - cytoplasm\n",
      "    - cytosol\n",
      "    - cytosol\n",
      "    - cytoplasmic side of plasma membrane\n",
      "    - microtubule cytoskeleton\n",
      "    - mitotic spindle\n",
      "  MF:\n",
      "    - protein tyrosine phosphatase activity\n",
      "    - non-membrane spanning protein tyrosine phosphatase activity\n",
      "    - protein binding\n",
      "    - protein tyrosine phosphatase activity, metal-dependent\n",
      "    - histone H2AXY142 phosphatase activity\n",
      "UniProt: {'Swiss-Prot': 'P35236', 'TrEMBL': 'B4DZD9'}\n",
      "\n",
      "Gene: IL17RA\n",
      "Symbol: IL17RA\n",
      "Name: interleukin 17 receptor A\n",
      "Summary: Interleukin 17A (IL17A) is a proinflammatory cytokine secreted by activated T-lymphocytes. It is a potent inducer of the maturation of CD34-positive hematopoietic precursors into neutrophils. The transmembrane protein encoded by this gene (interleukin 17A receptor; IL17RA) is a ubiquitous type I membrane glycoprotein that binds with low affinity to interleukin 17A. Interleukin 17A and its receptor play a pathogenic role in many inflammatory and autoimmune diseases such as rheumatoid arthritis. Like other cytokine receptors, this receptor likely has a multimeric structure. Alternative splicing results in multiple transcript variants encoding different isoforms. [provided by RefSeq, Feb 2014].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - inflammatory response\n",
      "    - cell surface receptor signaling pathway\n",
      "    - response to virus\n",
      "    - protein catabolic process\n",
      "    - positive regulation of interleukin-13 production\n",
      "    - positive regulation of interleukin-23 production\n",
      "    - positive regulation of interleukin-5 production\n",
      "    - positive regulation of interleukin-6 production\n",
      "    - interleukin-17A-mediated signaling pathway\n",
      "    - innate immune response\n",
      "    - positive regulation of inflammatory response\n",
      "    - defense response to fungus\n",
      "    - granulocyte chemotaxis\n",
      "    - fibroblast activation\n",
      "    - T-helper 17 type immune response\n",
      "    - interleukin-17-mediated signaling pathway\n",
      "    - interleukin-17-mediated signaling pathway\n",
      "    - positive regulation of cytokine production involved in inflammatory response\n",
      "    - positive regulation of chemokine (C-X-C motif) ligand 1 production\n",
      "  CC:\n",
      "    - extracellular region\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "  MF:\n",
      "    - signaling receptor binding\n",
      "    - protein binding\n",
      "    - interleukin-17 receptor activity\n",
      "    - interleukin-17 receptor activity\n",
      "    - interleukin-17 receptor activity\n",
      "UniProt: {'Swiss-Prot': 'Q96F46', 'TrEMBL': ['A0A8Q3WK71', 'A0A8Q3WKC6']}\n",
      "\n",
      "Gene: TXNRD1\n",
      "Symbol: TXNRD1\n",
      "Name: thioredoxin reductase 1\n",
      "Summary: The protein encoded by this gene belongs to the pyridine nucleotide-disulfide oxidoreductase family, and is a member of the thioredoxin (Trx) system. Three thioredoxin reductase (TrxR) isozymes are found in mammals. TrxRs are selenocysteine-containing flavoenzymes, which reduce thioredoxins, as well as other substrates, and play a key role in redox homoeostasis. This gene encodes an ubiquitously expressed, cytosolic form of TrxR, which functions as a homodimer containing FAD, and selenocysteine (Sec) at the active site. Sec is encoded by UGA codon that normally signals translation termination. The 3' UTRs of selenoprotein mRNAs contain a conserved stem-loop structure, the Sec insertion sequence (SECIS) element, which is necessary for the recognition of UGA as a Sec codon rather than as a stop signal. Alternative splicing, primarily at the 5' end, results in transcript variants encoding same or different isoforms, including a glutaredoxin-containing isoform that is predominantly expressed in testis. [provided by RefSeq, May 2017].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - mesoderm formation\n",
      "    - signal transduction\n",
      "    - cell population proliferation\n",
      "    - cell redox homeostasis\n",
      "    - cellular oxidant detoxification\n",
      "  CC:\n",
      "    - fibrillar center\n",
      "    - nucleoplasm\n",
      "    - cytoplasm\n",
      "    - mitochondrion\n",
      "    - cytosol\n",
      "    - cytosol\n",
      "    - extracellular exosome\n",
      "  MF:\n",
      "    - thioredoxin-disulfide reductase (NADPH) activity\n",
      "    - thioredoxin-disulfide reductase (NADPH) activity\n",
      "    - protein binding\n",
      "    - identical protein binding\n",
      "    - NADPH peroxidase activity\n",
      "    - FAD binding\n",
      "UniProt: {'Swiss-Prot': 'Q16881', 'TrEMBL': ['E9PQI3', 'B7Z2S5', 'E9PIR7', 'A0A0B4J225', 'E9PKD3', 'E9PLT3', 'E9PRI8', 'E9PKI4', 'E9PIZ5']}\n",
      "\n",
      "Gene: ULK2\n",
      "Symbol: ULK2\n",
      "Name: unc-51 like autophagy activating kinase 2\n",
      "Summary: This gene encodes a protein that is similar to a serine/threonine kinase in C. elegans which is involved in axonal elongation. The structure of this protein is similar to the C. elegans protein in that both proteins have an N-terminal kinase domain, a central proline/serine rich (PS) domain, and a C-terminal (C) domain. The gene is located within the Smith-Magenis syndrome region on chromosome 17. Alternatively spliced transcript variants encoding the same protein have been identified. [provided by RefSeq, Dec 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - autophagosome assembly\n",
      "    - mitophagy\n",
      "    - chromatin remodeling\n",
      "    - autophagy\n",
      "    - autophagy\n",
      "    - signal transduction\n",
      "    - axon guidance\n",
      "    - protein localization\n",
      "    - regulation of autophagy\n",
      "    - piecemeal microautophagy of the nucleus\n",
      "    - response to starvation\n",
      "    - response to starvation\n",
      "    - protein autophosphorylation\n",
      "    - collateral sprouting\n",
      "    - negative regulation of collateral sprouting\n",
      "    - axon extension\n",
      "    - reticulophagy\n",
      "    - somatic sensory system development\n",
      "  CC:\n",
      "    - phagophore assembly site\n",
      "    - cytoplasm\n",
      "    - autophagosome\n",
      "    - cytosol\n",
      "    - cytoplasmic vesicle membrane\n",
      "    - phagophore assembly site membrane\n",
      "    - phagophore assembly site membrane\n",
      "  MF:\n",
      "    - protein serine/threonine kinase activity\n",
      "    - protein serine/threonine kinase activity\n",
      "    - 3-phosphoinositide-dependent protein kinase activity\n",
      "    - DNA-dependent protein kinase activity\n",
      "    - AMP-activated protein kinase activity\n",
      "    - eukaryotic translation initiation factor 2alpha kinase activity\n",
      "    - ribosomal protein S6 kinase activity\n",
      "    - protein binding\n",
      "    - ATP binding\n",
      "    - histone H3S10 kinase activity\n",
      "    - histone H3T11 kinase activity\n",
      "    - histone H3T6 kinase activity\n",
      "    - histone H2AXS139 kinase activity\n",
      "    - histone H3S28 kinase activity\n",
      "    - histone H4S1 kinase activity\n",
      "    - histone H2AS1 kinase activity\n",
      "    - histone H2BS14 kinase activity\n",
      "    - histone H3T3 kinase activity\n",
      "    - histone H2AS121 kinase activity\n",
      "    - Rho-dependent protein serine/threonine kinase activity\n",
      "    - protein serine kinase activity\n",
      "    - histone H2BS36 kinase activity\n",
      "    - histone H3S57 kinase activity\n",
      "    - histone H3T45 kinase activity\n",
      "    - histone H2AT120 kinase activity\n",
      "UniProt: {'Swiss-Prot': 'Q8IYT8', 'TrEMBL': ['K7EIU2', 'K7EP89', 'K7EKK9']}\n",
      "\n",
      "Gene: DAPK1\n",
      "Symbol: DAPK1\n",
      "Name: death associated protein kinase 1\n",
      "Summary: Death-associated protein kinase 1 is a positive mediator of gamma-interferon induced programmed cell death. DAPK1 encodes a structurally unique 160-kD calmodulin dependent serine-threonine kinase that carries 8 ankyrin repeats and 2 putative P-loop consensus sites. It is a tumor suppressor candidate. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Dec 2013].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - defense response to tumor cell\n",
      "    - regulation of response to tumor cell\n",
      "    - chromatin remodeling\n",
      "    - protein phosphorylation\n",
      "    - protein phosphorylation\n",
      "    - apoptotic process\n",
      "    - apoptotic process\n",
      "    - extrinsic apoptotic signaling pathway via death domain receptors\n",
      "    - regulation of autophagy\n",
      "    - positive regulation of autophagy\n",
      "    - negative regulation of translation\n",
      "    - intracellular signal transduction\n",
      "    - intracellular signal transduction\n",
      "    - regulation of apoptotic process\n",
      "    - positive regulation of apoptotic process\n",
      "    - positive regulation of apoptotic process\n",
      "    - positive regulation of apoptotic process\n",
      "    - negative regulation of apoptotic process\n",
      "    - protein autophosphorylation\n",
      "    - protein autophosphorylation\n",
      "    - cellular response to type II interferon\n",
      "    - cellular response to hydroperoxide\n",
      "    - apoptotic signaling pathway\n",
      "    - positive regulation of autophagic cell death\n",
      "    - regulation of NMDA receptor activity\n",
      "  CC:\n",
      "    - nucleus\n",
      "    - cytoplasm\n",
      "    - cytoplasm\n",
      "    - plasma membrane\n",
      "    - postsynaptic density\n",
      "    - actin cytoskeleton\n",
      "    - glutamatergic synapse\n",
      "    - DAPK1-calmodulin complex\n",
      "  MF:\n",
      "    - protein kinase activity\n",
      "    - protein serine/threonine kinase activity\n",
      "    - protein serine/threonine kinase activity\n",
      "    - 3-phosphoinositide-dependent protein kinase activity\n",
      "    - DNA-dependent protein kinase activity\n",
      "    - AMP-activated protein kinase activity\n",
      "    - calcium/calmodulin-dependent protein kinase activity\n",
      "    - eukaryotic translation initiation factor 2alpha kinase activity\n",
      "    - ribosomal protein S6 kinase activity\n",
      "    - protein binding\n",
      "    - calmodulin binding\n",
      "    - calmodulin binding\n",
      "    - calmodulin binding\n",
      "    - ATP binding\n",
      "    - GTP binding\n",
      "    - syntaxin-1 binding\n",
      "    - histone H3S10 kinase activity\n",
      "    - histone H3T11 kinase activity\n",
      "    - histone H3T6 kinase activity\n",
      "    - histone H2AXS139 kinase activity\n",
      "    - identical protein binding\n",
      "    - histone H3S28 kinase activity\n",
      "    - histone H4S1 kinase activity\n",
      "    - histone H2AS1 kinase activity\n",
      "    - histone H2BS14 kinase activity\n",
      "    - histone H3T3 kinase activity\n",
      "    - histone H2AS121 kinase activity\n",
      "    - Rho-dependent protein serine/threonine kinase activity\n",
      "    - protein serine kinase activity\n",
      "    - histone H2BS36 kinase activity\n",
      "    - histone H3S57 kinase activity\n",
      "    - histone H3T45 kinase activity\n",
      "    - histone H2AT120 kinase activity\n",
      "UniProt: {'Swiss-Prot': 'P53355', 'TrEMBL': ['Q59H88', 'B4DHI4', 'F8WCQ3']}\n",
      "\n",
      "Gene: THRA\n",
      "Symbol: THRA\n",
      "Name: thyroid hormone receptor alpha\n",
      "Summary: The protein encoded by this gene is a nuclear hormone receptor for triiodothyronine. It is one of the several receptors for thyroid hormone, and has been shown to mediate the biological activities of thyroid hormone. Knockout studies in mice suggest that the different receptors, while having certain extent of redundancy, may mediate different functions of thyroid hormone. Alternatively spliced transcript variants encoding distinct isoforms have been reported. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - negative regulation of transcription by RNA polymerase II\n",
      "    - cartilage condensation\n",
      "    - ossification\n",
      "    - thyroid hormone receptor signaling pathway\n",
      "    - regulation of thyroid hormone receptor signaling pathway\n",
      "    - positive regulation of thyroid hormone receptor signaling pathway\n",
      "    - regulation of transcription by RNA polymerase II\n",
      "    - transcription by RNA polymerase II\n",
      "    - learning or memory\n",
      "    - regulation of heart contraction\n",
      "    - female courtship behavior\n",
      "    - response to cold\n",
      "    - hormone-mediated signaling pathway\n",
      "    - negative regulation of RNA polymerase II transcription preinitiation complex assembly\n",
      "    - cell differentiation\n",
      "    - erythrocyte differentiation\n",
      "    - thyroid gland development\n",
      "    - regulation of myeloid cell apoptotic process\n",
      "    - mRNA transcription by RNA polymerase II\n",
      "    - negative regulation of DNA-templated transcription\n",
      "    - positive regulation of female receptivity\n",
      "    - positive regulation of transcription by RNA polymerase II\n",
      "    - positive regulation of transcription by RNA polymerase II\n",
      "    - retinoic acid receptor signaling pathway\n",
      "    - regulation of lipid catabolic process\n",
      "    - type I pneumocyte differentiation\n",
      "    - positive regulation of cold-induced thermogenesis\n",
      "    - negative regulation of DNA-templated transcription initiation\n",
      "  CC:\n",
      "    - chromatin\n",
      "    - nucleus\n",
      "    - nucleus\n",
      "    - nucleoplasm\n",
      "    - cytosol\n",
      "    - RNA polymerase II transcription regulator complex\n",
      "    - RNA polymerase II transcription regulator complex\n",
      "  MF:\n",
      "    - transcription cis-regulatory region binding\n",
      "    - RNA polymerase II cis-regulatory region sequence-specific DNA binding\n",
      "    - DNA-binding transcription factor activity, RNA polymerase II-specific\n",
      "    - DNA-binding transcription factor activity\n",
      "    - nuclear receptor activity\n",
      "    - nuclear receptor activity\n",
      "    - protein binding\n",
      "    - zinc ion binding\n",
      "    - TBP-class protein binding\n",
      "    - protein domain specific binding\n",
      "    - chromatin DNA binding\n",
      "    - protein-containing complex binding\n",
      "    - thyroid hormone binding\n",
      "    - thyroid hormone binding\n",
      "    - general transcription initiation factor binding\n",
      "UniProt: {'Swiss-Prot': 'P10827', 'TrEMBL': ['J3QRW5', 'Q6FH41', 'J3KTF3', 'J3QR26', 'J3QRA9']}\n",
      "\n",
      "Gene: CD248\n",
      "Symbol: CD248\n",
      "Name: CD248 molecule\n",
      "Summary: Predicted to enable extracellular matrix binding activity and extracellular matrix protein binding activity. Predicted to be involved in cell migration. Predicted to act upstream of or within several processes, including anatomical structure regression; lymph node development; and positive regulation of endothelial cell apoptotic process. Located in extracellular exosome. [provided by Alliance of Genome Resources, Apr 2022]\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - fibroblast migration\n",
      "    - cell migration\n",
      "    - fibroblast proliferation\n",
      "    - positive regulation of fibroblast proliferation\n",
      "    - lymph node development\n",
      "    - anatomical structure regression\n",
      "    - endothelial cell apoptotic process\n",
      "    - positive regulation of endothelial cell apoptotic process\n",
      "  CC:\n",
      "    - cytoplasm\n",
      "    - external side of plasma membrane\n",
      "    - extracellular matrix\n",
      "    - extracellular exosome\n",
      "  MF:\n",
      "    - calcium ion binding\n",
      "    - protein binding\n",
      "    - carbohydrate binding\n",
      "    - extracellular matrix binding\n",
      "    - extracellular matrix protein binding\n",
      "UniProt: {'Swiss-Prot': 'Q9HCU0'}\n",
      "\n",
      "Gene: EGLN3\n",
      "Symbol: EGLN3\n",
      "Name: egl-9 family hypoxia inducible factor 3\n",
      "Summary: Enables peptidyl-proline 4-dioxygenase activity. Involved in several processes, including activation of cysteine-type endopeptidase activity involved in apoptotic process; peptidyl-proline hydroxylation to 4-hydroxy-L-proline; and response to hypoxia. Located in cytosol and nucleus. Implicated in renal cell carcinoma. Biomarker of clear cell renal cell carcinoma. [provided by Alliance of Genome Resources, Apr 2022]\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - apoptotic process\n",
      "    - DNA damage response\n",
      "    - protein hydroxylation\n",
      "    - regulation of neuron apoptotic process\n",
      "    - regulation of neuron apoptotic process\n",
      "    - cellular response to hypoxia\n",
      "  CC:\n",
      "    - nucleus\n",
      "    - nucleus\n",
      "    - nucleoplasm\n",
      "    - cytoplasm\n",
      "    - cytoplasm\n",
      "    - cytosol\n",
      "    - cytosol\n",
      "  MF:\n",
      "    - protein binding\n",
      "    - ferrous iron binding\n",
      "    - 2-oxoglutarate-dependent dioxygenase activity\n",
      "    - L-ascorbic acid binding\n",
      "    - peptidyl-proline 4-dioxygenase activity\n",
      "    - peptidyl-proline 4-dioxygenase activity\n",
      "    - hypoxia-inducible factor-proline dioxygenase activity\n",
      "UniProt: {'Swiss-Prot': 'Q9H6Z9', 'TrEMBL': ['F8W1G2', 'Q3T1B0', 'B3KVT0', 'G3V3M1', 'F8VR39']}\n",
      "\n",
      "Gene: MAL\n",
      "Symbol: MAL\n",
      "Name: mal, T cell differentiation protein\n",
      "Summary: The protein encoded by this gene is a highly hydrophobic integral membrane protein belonging to the MAL family of proteolipids. The protein has been localized to the endoplasmic reticulum of T-cells and is a candidate linker protein in T-cell signal transduction. In addition, this proteolipid is localized in compact myelin of cells in the nervous system and has been implicated in myelin biogenesis and/or function. The protein plays a role in the formation, stabilization and maintenance of glycosphingolipid-enriched membrane microdomains. Down-regulation of this gene has been associated with a variety of human epithelial malignancies. Alternative splicing produces four transcript variants which vary from each other by the presence or absence of alternatively spliced exons 2 and 3. [provided by RefSeq, May 2012].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - membrane raft polarization\n",
      "    - protein localization to paranode region of axon\n",
      "    - apoptotic process\n",
      "    - central nervous system development\n",
      "    - central nervous system myelination\n",
      "    - cell differentiation\n",
      "    - myelination\n",
      "    - myelination\n",
      "    - myelination\n",
      "    - apical protein localization\n",
      "    - protein insertion into plasma membrane\n",
      "    - positive regulation of extrinsic apoptotic signaling pathway via death domain receptors\n",
      "  CC:\n",
      "    - endoplasmic reticulum\n",
      "    - membrane\n",
      "    - apical plasma membrane\n",
      "    - Schmidt-Lanterman incisure\n",
      "    - plasma membrane raft\n",
      "    - membrane raft\n",
      "    - hinge region between urothelial plaques of apical plasma membrane\n",
      "  MF:\n",
      "    - protein binding\n",
      "    - lipid binding\n",
      "    - peptidase activator activity involved in apoptotic process\n",
      "    - structural constituent of myelin sheath\n",
      "    - structural constituent of myelin sheath\n",
      "UniProt: {'Swiss-Prot': 'P21145'}\n",
      "\n",
      "Gene: MYLK\n",
      "Symbol: MYLK\n",
      "Name: myosin light chain kinase\n",
      "Summary: This gene, a muscle member of the immunoglobulin gene superfamily, encodes myosin light chain kinase which is a calcium/calmodulin dependent enzyme. This kinase phosphorylates myosin regulatory light chains to facilitate myosin interaction with actin filaments to produce contractile activity. This gene encodes both smooth muscle and nonmuscle isoforms. In addition, using a separate promoter in an intron in the 3' region, it encodes telokin, a small protein identical in sequence to the C-terminus of myosin light chain kinase, that is independently expressed in smooth muscle and functions to stabilize unphosphorylated myosin filaments. A pseudogene is located on the p arm of chromosome 3. Four transcript variants that produce four isoforms of the calcium/calmodulin dependent enzyme have been identified as well as two transcripts that produce two isoforms of telokin. Additional variants have been identified but lack full length transcripts. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - protein phosphorylation\n",
      "    - smooth muscle contraction\n",
      "    - tonic smooth muscle contraction\n",
      "    - tonic smooth muscle contraction\n",
      "    - positive regulation of cell migration\n",
      "    - bleb assembly\n",
      "    - positive regulation of calcium ion transport\n",
      "    - aorta smooth muscle tissue morphogenesis\n",
      "    - cellular hypotonic response\n",
      "    - positive regulation of wound healing\n",
      "    - regulation of synaptic vesicle endocytosis\n",
      "  CC:\n",
      "    - stress fiber\n",
      "    - stress fiber\n",
      "    - cytoplasm\n",
      "    - cytoplasm\n",
      "    - cytosol\n",
      "    - plasma membrane\n",
      "    - actin cytoskeleton\n",
      "    - lamellipodium\n",
      "    - lamellipodium\n",
      "    - cleavage furrow\n",
      "    - cleavage furrow\n",
      "    - synapse\n",
      "  MF:\n",
      "    - actin binding\n",
      "    - myosin light chain kinase activity\n",
      "    - myosin light chain kinase activity\n",
      "    - protein binding\n",
      "    - calmodulin binding\n",
      "    - ATP binding\n",
      "    - metal ion binding\n",
      "    - scaffold protein binding\n",
      "UniProt: {'Swiss-Prot': 'Q15746', 'TrEMBL': ['Q05B97', 'A0A8I5KTQ1', 'Q05B98', 'A0A8I5KYA1', 'A0A8I5QKW8', 'A0A8I5KUB2', 'F8WBL7', 'A0A8I5KUP2', 'A0A8I5QKL9', 'A0A8I5QJW9', 'A0A8J9G5A3', 'A0A8I5KUH4', 'A0A8I5KQY6', 'A0A8I5KYB9', 'A0A8I5QJT9', 'A0A8I5KYZ0', 'A0A8I5KZ33', 'A0A8I5KS78', 'A0A8I5KU53', 'A0A8I5KVV3', 'A0A8I5KXG8']}\n",
      "\n",
      "Gene: IRAK3\n",
      "Symbol: IRAK3\n",
      "Name: interleukin 1 receptor associated kinase 3\n",
      "Summary: This gene encodes a member of the interleukin-1 receptor-associated kinase protein family. Members of this family are essential components of the Toll/IL-R immune signal transduction pathways. This protein is primarily expressed in monocytes and macrophages and functions as a negative regulator of Toll-like receptor signaling. Mutations in this gene are associated with a susceptibility to asthma. Alternate splicing results in multiple transcript variants. [provided by RefSeq, May 2010].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - positive regulation of cytokine production\n",
      "    - negative regulation of cytokine-mediated signaling pathway\n",
      "    - MyD88-dependent toll-like receptor signaling pathway\n",
      "    - protein phosphorylation\n",
      "    - Toll signaling pathway\n",
      "    - response to virus\n",
      "    - positive regulation of macrophage tolerance induction\n",
      "    - negative regulation of macrophage cytokine production\n",
      "    - cytokine-mediated signaling pathway\n",
      "    - lipopolysaccharide-mediated signaling pathway\n",
      "    - negative regulation of NF-kappaB transcription factor activity\n",
      "    - response to peptidoglycan\n",
      "    - response to lipopolysaccharide\n",
      "    - negative regulation of interleukin-12 production\n",
      "    - negative regulation of interleukin-12 production\n",
      "    - negative regulation of interleukin-6 production\n",
      "    - negative regulation of interleukin-6 production\n",
      "    - negative regulation of tumor necrosis factor production\n",
      "    - negative regulation of toll-like receptor signaling pathway\n",
      "    - intracellular signal transduction\n",
      "    - negative regulation of protein catabolic process\n",
      "    - positive regulation of canonical NF-kappaB signal transduction\n",
      "    - negative regulation of canonical NF-kappaB signal transduction\n",
      "    - negative regulation of protein-containing complex disassembly\n",
      "    - regulation of protein-containing complex disassembly\n",
      "    - response to exogenous dsRNA\n",
      "    - negative regulation of MAP kinase activity\n",
      "    - negative regulation of innate immune response\n",
      "    - interleukin-1-mediated signaling pathway\n",
      "    - interleukin-1-mediated signaling pathway\n",
      "    - response to interleukin-1\n",
      "  CC:\n",
      "    - nucleus\n",
      "    - nucleus\n",
      "    - cytoplasm\n",
      "    - cytoplasm\n",
      "    - plasma membrane\n",
      "  MF:\n",
      "    - magnesium ion binding\n",
      "    - protein serine/threonine kinase activity\n",
      "    - protein binding\n",
      "    - ATP binding\n",
      "    - protein kinase binding\n",
      "    - protein homodimerization activity\n",
      "    - protein homodimerization activity\n",
      "    - protein heterodimerization activity\n",
      "UniProt: {'Swiss-Prot': 'Q9Y616', 'TrEMBL': 'F5GYN6'}\n",
      "\n",
      "Gene: PAK3\n",
      "Symbol: PAK3\n",
      "Name: p21 (RAC1) activated kinase 3\n",
      "Summary: The protein encoded by this gene is a serine-threonine kinase and forms an activated complex with GTP-bound RAS-like (P21), CDC2 and RAC1. This protein may be necessary for dendritic development and for the rapid cytoskeletal reorganization in dendritic spines associated with synaptic plasticity. Defects in this gene are the cause of a non-syndromic form of X-linked intellectual disability. Alternatively spliced transcript variants encoding different isoforms have been identified. [provided by RefSeq, Jul 2017].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - MAPK cascade\n",
      "    - stimulatory C-type lectin receptor signaling pathway\n",
      "    - chromatin remodeling\n",
      "    - axonogenesis\n",
      "    - cellular response to starvation\n",
      "    - dendrite development\n",
      "    - cell migration\n",
      "    - regulation of actin filament polymerization\n",
      "    - regulation of actin cytoskeleton organization\n",
      "    - intracellular signal transduction\n",
      "    - regulation of MAPK cascade\n",
      "    - ephrin receptor signaling pathway\n",
      "    - regulation of axonogenesis\n",
      "    - synapse organization\n",
      "    - dendritic spine morphogenesis\n",
      "    - dendritic spine morphogenesis\n",
      "    - regulation of postsynapse organization\n",
      "  CC:\n",
      "    - cytoplasm\n",
      "    - cytosol\n",
      "    - plasma membrane\n",
      "    - postsynaptic density\n",
      "    - glutamatergic synapse\n",
      "  MF:\n",
      "    - protein serine/threonine kinase activity\n",
      "    - protein serine/threonine kinase activity\n",
      "    - protein serine/threonine kinase activity\n",
      "    - 3-phosphoinositide-dependent protein kinase activity\n",
      "    - DNA-dependent protein kinase activity\n",
      "    - AMP-activated protein kinase activity\n",
      "    - eukaryotic translation initiation factor 2alpha kinase activity\n",
      "    - MAP kinase kinase activity\n",
      "    - ribosomal protein S6 kinase activity\n",
      "    - protein binding\n",
      "    - ATP binding\n",
      "    - SH3 domain binding\n",
      "    - small GTPase binding\n",
      "    - histone H3S10 kinase activity\n",
      "    - histone H3T11 kinase activity\n",
      "    - histone H3T6 kinase activity\n",
      "    - histone H2AXS139 kinase activity\n",
      "    - histone H3S28 kinase activity\n",
      "    - histone H4S1 kinase activity\n",
      "    - histone H2AS1 kinase activity\n",
      "    - histone H2BS14 kinase activity\n",
      "    - metal ion binding\n",
      "    - histone H3T3 kinase activity\n",
      "    - histone H2AS121 kinase activity\n",
      "    - Rho-dependent protein serine/threonine kinase activity\n",
      "    - protein serine kinase activity\n",
      "    - histone H2BS36 kinase activity\n",
      "    - histone H3S57 kinase activity\n",
      "    - histone H3T45 kinase activity\n",
      "    - histone H2AT120 kinase activity\n",
      "UniProt: {'Swiss-Prot': 'O75914', 'TrEMBL': 'B2RCU6'}\n",
      "\n",
      "Gene: AKT3\n",
      "Symbol: AKT3\n",
      "Name: AKT serine/threonine kinase 3\n",
      "Summary: The protein encoded by this gene is a member of the AKT, also called PKB,  serine/threonine protein kinase family. AKT kinases are known to be regulators of cell signaling in response to insulin and growth factors. They are involved in a wide variety of biological processes including cell proliferation, differentiation, apoptosis, tumorigenesis, as well as glycogen synthesis and glucose uptake. This kinase has been shown to be stimulated by platelet-derived growth factor (PDGF), insulin, and insulin-like growth factor 1 (IGF1). Alternatively splice transcript variants encoding distinct isoforms have been described. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - mitochondrial genome maintenance\n",
      "    - positive regulation of endothelial cell proliferation\n",
      "    - chromatin remodeling\n",
      "    - protein phosphorylation\n",
      "    - signal transduction\n",
      "    - positive regulation of TOR signaling\n",
      "    - intracellular signal transduction\n",
      "    - positive regulation of blood vessel endothelial cell migration\n",
      "    - positive regulation of angiogenesis\n",
      "    - positive regulation of cell size\n",
      "    - brain morphogenesis\n",
      "    - homeostasis of number of cells within a tissue\n",
      "    - positive regulation of cell migration involved in sprouting angiogenesis\n",
      "    - negative regulation of PERK-mediated unfolded protein response\n",
      "    - positive regulation of vascular endothelial cell proliferation\n",
      "    - positive regulation of artery morphogenesis\n",
      "    - negative regulation of cellular senescence\n",
      "  CC:\n",
      "    - nucleoplasm\n",
      "    - nucleoplasm\n",
      "    - Golgi apparatus\n",
      "    - cytosol\n",
      "    - plasma membrane\n",
      "    - cilium\n",
      "    - ciliary basal body\n",
      "  MF:\n",
      "    - protein kinase activity\n",
      "    - protein serine/threonine kinase activity\n",
      "    - protein serine/threonine kinase activity\n",
      "    - protein serine/threonine kinase activity\n",
      "    - 3-phosphoinositide-dependent protein kinase activity\n",
      "    - DNA-dependent protein kinase activity\n",
      "    - AMP-activated protein kinase activity\n",
      "    - eukaryotic translation initiation factor 2alpha kinase activity\n",
      "    - ribosomal protein S6 kinase activity\n",
      "    - protein binding\n",
      "    - ATP binding\n",
      "    - histone H3S10 kinase activity\n",
      "    - histone H3T11 kinase activity\n",
      "    - histone H3T6 kinase activity\n",
      "    - histone H2AXS139 kinase activity\n",
      "    - histone H3S28 kinase activity\n",
      "    - histone H4S1 kinase activity\n",
      "    - histone H2AS1 kinase activity\n",
      "    - histone H2BS14 kinase activity\n",
      "    - histone H3T3 kinase activity\n",
      "    - histone H2AS121 kinase activity\n",
      "    - Rho-dependent protein serine/threonine kinase activity\n",
      "    - protein serine kinase activity\n",
      "    - histone H2BS36 kinase activity\n",
      "    - histone H3S57 kinase activity\n",
      "    - histone H3T45 kinase activity\n",
      "    - histone H2AT120 kinase activity\n",
      "UniProt: {'Swiss-Prot': 'Q9Y243', 'TrEMBL': ['A0A5F9ZGZ4', 'A0A5F9ZHA9', 'A0A7P0TB29', 'A0A5F9ZHU3', 'F8VS91', 'A0A7P0T8T9', 'A0A5K1VW74', 'A0A7P0TBJ6', 'A0A5F9ZGY0', 'A0A7P0T872', 'A0A7P0TB32']}\n",
      "\n",
      "Gene: PDE4D\n",
      "Symbol: PDE4D\n",
      "Name: phosphodiesterase 4D\n",
      "Summary: This gene encodes one of four mammalian counterparts to the fruit fly 'dunce' gene. The encoded protein has 3',5'-cyclic-AMP phosphodiesterase activity and degrades cAMP, which acts as a signal transduction molecule in multiple cell types. This gene uses different promoters to generate multiple alternatively spliced transcript variants that encode functional proteins.[provided by RefSeq, Sep 2009].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - regulation of heart rate\n",
      "    - cAMP catabolic process\n",
      "    - positive regulation of heart rate\n",
      "    - regulation of release of sequestered calcium ion into cytosol by sarcoplasmic reticulum\n",
      "    - cAMP-mediated signaling\n",
      "    - positive regulation of type II interferon production\n",
      "    - positive regulation of interleukin-2 production\n",
      "    - positive regulation of interleukin-5 production\n",
      "    - negative regulation of heart contraction\n",
      "    - T cell receptor signaling pathway\n",
      "    - establishment of endothelial barrier\n",
      "    - cellular response to cAMP\n",
      "    - cellular response to epinephrine stimulus\n",
      "    - adrenergic receptor signaling pathway\n",
      "    - regulation of cardiac muscle cell contraction\n",
      "    - negative regulation of adenylate cyclase-activating G protein-coupled receptor signaling pathway\n",
      "    - negative regulation of adenylate cyclase-activating G protein-coupled receptor signaling pathway\n",
      "    - regulation of cell communication by electrical coupling involved in cardiac conduction\n",
      "    - negative regulation of relaxation of cardiac muscle\n",
      "    - regulation of calcium ion transmembrane transport via high voltage-gated calcium channel\n",
      "  CC:\n",
      "    - nucleoplasm\n",
      "    - centrosome\n",
      "    - cytosol\n",
      "    - cytosol\n",
      "    - plasma membrane\n",
      "    - voltage-gated calcium channel complex\n",
      "    - cilium\n",
      "    - membrane\n",
      "    - membrane\n",
      "    - apical plasma membrane\n",
      "    - nuclear membrane\n",
      "    - calcium channel complex\n",
      "  MF:\n",
      "    - 3',5'-cyclic-nucleotide phosphodiesterase activity\n",
      "    - 3',5'-cyclic-AMP phosphodiesterase activity\n",
      "    - 3',5'-cyclic-AMP phosphodiesterase activity\n",
      "    - 3',5'-cyclic-AMP phosphodiesterase activity\n",
      "    - 3',5'-cyclic-AMP phosphodiesterase activity\n",
      "    - 3',5'-cyclic-AMP phosphodiesterase activity\n",
      "    - calcium channel regulator activity\n",
      "    - protein binding\n",
      "    - enzyme binding\n",
      "    - signaling receptor regulator activity\n",
      "    - cAMP binding\n",
      "    - beta-2 adrenergic receptor binding\n",
      "    - transmembrane transporter binding\n",
      "    - metal ion binding\n",
      "    - 3',5'-cyclic-GMP phosphodiesterase activity\n",
      "    - ATPase binding\n",
      "    - scaffold protein binding\n",
      "    - heterocyclic compound binding\n",
      "UniProt: {'Swiss-Prot': 'Q08499', 'TrEMBL': ['D6RAQ0', 'A0A140VJR0', 'D6RIG1', 'D6RHE0', 'A0A1B0GW84', 'D6RBB2', 'D6R9L4']}\n",
      "\n",
      "Gene: MLST8\n",
      "Symbol: MLST8\n",
      "Name: MTOR associated protein, LST8 homolog\n",
      "Summary: Enables protein serine/threonine kinase activator activity. Involved in TORC1 signaling; positive regulation of TOR signaling; and regulation of actin cytoskeleton organization. Part of TORC1 complex and TORC2 complex. [provided by Alliance of Genome Resources, Apr 2022]\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - DNA damage response\n",
      "    - cytoskeleton organization\n",
      "    - negative regulation of autophagy\n",
      "    - positive regulation of cell growth\n",
      "    - positive regulation of actin filament polymerization\n",
      "    - cellular response to nutrient levels\n",
      "    - TOR signaling\n",
      "    - positive regulation of TOR signaling\n",
      "    - regulation of actin cytoskeleton organization\n",
      "    - regulation of actin cytoskeleton organization\n",
      "    - TORC1 signaling\n",
      "    - TORC1 signaling\n",
      "    - TORC2 signaling\n",
      "    - negative regulation of apoptotic process\n",
      "    - positive regulation of glycolytic process\n",
      "    - positive regulation of lipid biosynthetic process\n",
      "    - cellular response to hypoxia\n",
      "    - cellular response to osmotic stress\n",
      "    - positive regulation of pentose-phosphate shunt\n",
      "  CC:\n",
      "    - nucleoplasm\n",
      "    - cytoplasm\n",
      "    - lysosomal membrane\n",
      "    - cytosol\n",
      "    - TORC1 complex\n",
      "    - TORC1 complex\n",
      "    - TORC1 complex\n",
      "    - TORC2 complex\n",
      "    - TORC2 complex\n",
      "    - TORC2 complex\n",
      "    - serine/threonine protein kinase complex\n",
      "  MF:\n",
      "    - protein binding\n",
      "    - protein-macromolecule adaptor activity\n",
      "    - protein serine/threonine kinase activator activity\n",
      "UniProt: {'Swiss-Prot': 'Q9BVC4', 'TrEMBL': ['H3BM50', 'H3BR38', 'I3L2E7', 'H3BQ74', 'H3BSZ4', 'A0A0A0MR05', 'H3BR25', 'H3BPT1', 'H3BPU5', 'H3BN58']}\n",
      "\n",
      "Gene: CHEK1\n",
      "Symbol: CHEK1\n",
      "Name: checkpoint kinase 1\n",
      "Summary: The protein encoded by this gene belongs to the Ser/Thr protein kinase family. It is required for checkpoint mediated cell cycle arrest in response to DNA damage or the presence of unreplicated DNA. This protein acts to integrate signals from ATM and ATR, two cell cycle proteins involved in DNA damage responses, that also associate with chromatin in meiotic prophase I. Phosphorylation of CDC25A protein phosphatase by this protein is required for cells to delay cell cycle progression in response to double-strand DNA breaks. Several alternatively spliced transcript variants have been found for this gene. [provided by RefSeq, Oct 2011].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - DNA damage checkpoint signaling\n",
      "    - DNA damage checkpoint signaling\n",
      "    - G2/M transition of mitotic cell cycle\n",
      "    - inner cell mass cell proliferation\n",
      "    - DNA replication\n",
      "    - DNA repair\n",
      "    - chromatin remodeling\n",
      "    - protein phosphorylation\n",
      "    - apoptotic process\n",
      "    - DNA damage response\n",
      "    - DNA damage response\n",
      "    - nucleus organization\n",
      "    - mitotic G2 DNA damage checkpoint signaling\n",
      "    - mitotic G2 DNA damage checkpoint signaling\n",
      "    - regulation of double-strand break repair via homologous recombination\n",
      "    - peptidyl-threonine phosphorylation\n",
      "    - regulation of cell population proliferation\n",
      "    - signal transduction in response to DNA damage\n",
      "    - mitotic G2/M transition checkpoint\n",
      "    - mitotic G2/M transition checkpoint\n",
      "    - positive regulation of cell cycle\n",
      "    - negative regulation of gene expression, epigenetic\n",
      "    - negative regulation of mitotic nuclear division\n",
      "    - regulation of mitotic centrosome separation\n",
      "    - negative regulation of G0 to G1 transition\n",
      "    - cellular response to mechanical stimulus\n",
      "    - cellular response to caffeine\n",
      "    - replicative senescence\n",
      "    - regulation of signal transduction by p53 class mediator\n",
      "    - apoptotic process involved in development\n",
      "    - negative regulation of DNA biosynthetic process\n",
      "  CC:\n",
      "    - chromosome, telomeric region\n",
      "    - chromatin\n",
      "    - condensed nuclear chromosome\n",
      "    - extracellular space\n",
      "    - nucleus\n",
      "    - nucleoplasm\n",
      "    - nucleoplasm\n",
      "    - replication fork\n",
      "    - cytoplasm\n",
      "    - centrosome\n",
      "    - cytosol\n",
      "    - protein-containing complex\n",
      "    - intracellular membrane-bounded organelle\n",
      "  MF:\n",
      "    - protein kinase activity\n",
      "    - protein serine/threonine kinase activity\n",
      "    - protein serine/threonine kinase activity\n",
      "    - 3-phosphoinositide-dependent protein kinase activity\n",
      "    - DNA-dependent protein kinase activity\n",
      "    - AMP-activated protein kinase activity\n",
      "    - eukaryotic translation initiation factor 2alpha kinase activity\n",
      "    - ribosomal protein S6 kinase activity\n",
      "    - protein binding\n",
      "    - ATP binding\n",
      "    - protein domain specific binding\n",
      "    - histone H3S10 kinase activity\n",
      "    - histone H3T11 kinase activity\n",
      "    - histone H3T11 kinase activity\n",
      "    - histone H3T6 kinase activity\n",
      "    - histone H2AXS139 kinase activity\n",
      "    - histone H3S28 kinase activity\n",
      "    - histone H4S1 kinase activity\n",
      "    - histone H2AS1 kinase activity\n",
      "    - histone H2BS14 kinase activity\n",
      "    - histone H3T3 kinase activity\n",
      "    - histone H2AS121 kinase activity\n",
      "    - Rho-dependent protein serine/threonine kinase activity\n",
      "    - protein serine kinase activity\n",
      "    - histone H2BS36 kinase activity\n",
      "    - histone H3S57 kinase activity\n",
      "    - histone H3T45 kinase activity\n",
      "    - histone H2AT120 kinase activity\n",
      "UniProt: {'Swiss-Prot': 'O14757', 'TrEMBL': ['B4DT73', 'E7EPP6', 'A0A087WT52', 'E9PKQ3', 'E9PRU7', 'E9PQW7', 'E9PPA5', 'E9PJI4']}\n",
      "\n",
      "Gene: QPCTL\n",
      "Symbol: QPCTL\n",
      "Name: glutaminyl-peptide cyclotransferase like\n",
      "Summary: Enables glutaminyl-peptide cyclotransferase activity and zinc ion binding activity. Acts upstream of or within peptidyl-pyroglutamic acid biosynthetic process, using glutaminyl-peptide cyclotransferase. Located in Golgi apparatus. [provided by Alliance of Genome Resources, Apr 2022]\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - peptidyl-pyroglutamic acid biosynthetic process, using glutaminyl-peptide cyclotransferase\n",
      "    - peptidyl-pyroglutamic acid biosynthetic process, using glutaminyl-peptide cyclotransferase\n",
      "  CC:\n",
      "    - Golgi membrane\n",
      "    - Golgi apparatus\n",
      "    - membrane\n",
      "  MF:\n",
      "    - protein binding\n",
      "    - zinc ion binding\n",
      "    - zinc ion binding\n",
      "    - glutaminyl-peptide cyclotransferase activity\n",
      "    - glutaminyl-peptide cyclotransferase activity\n",
      "UniProt: {'Swiss-Prot': 'Q9NXS2', 'TrEMBL': ['K7EQG1', 'A0A7I2V353', 'A0A7I2V5X1']}\n",
      "\n",
      "Gene: ETV5\n",
      "Symbol: ETV5\n",
      "Name: ETS variant transcription factor 5\n",
      "Summary: Enables DNA-binding transcription activator activity, RNA polymerase II-specific and RNA polymerase II transcription regulatory region sequence-specific DNA binding activity. Involved in cellular response to oxidative stress; negative regulation of transcription by RNA polymerase II; and positive regulation of transcription by RNA polymerase II. Located in nucleoplasm. [provided by Alliance of Genome Resources, Apr 2022]\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - negative regulation of transcription by RNA polymerase II\n",
      "    - regulation of transcription by RNA polymerase II\n",
      "    - cell differentiation\n",
      "    - cellular response to oxidative stress\n",
      "    - positive regulation of neuron differentiation\n",
      "    - positive regulation of transcription by RNA polymerase II\n",
      "    - male germ-line stem cell asymmetric division\n",
      "    - positive regulation of glial cell proliferation\n",
      "  CC:\n",
      "    - chromatin\n",
      "    - nucleus\n",
      "    - nucleus\n",
      "    - nucleoplasm\n",
      "  MF:\n",
      "    - transcription cis-regulatory region binding\n",
      "    - RNA polymerase II transcription regulatory region sequence-specific DNA binding\n",
      "    - DNA-binding transcription factor activity, RNA polymerase II-specific\n",
      "    - DNA-binding transcription factor activity, RNA polymerase II-specific\n",
      "    - DNA-binding transcription activator activity, RNA polymerase II-specific\n",
      "    - DNA binding\n",
      "    - DNA-binding transcription factor activity\n",
      "    - protein binding\n",
      "    - sequence-specific double-stranded DNA binding\n",
      "UniProt: {'Swiss-Prot': 'P41161', 'TrEMBL': ['H7C1D2', 'C9J5A5', 'C9JMF0', 'C9JR66', 'C9J0Z2']}\n",
      "\n",
      "Gene: ERG\n",
      "Symbol: ERG\n",
      "Name: ETS transcription factor ERG\n",
      "Summary: This gene encodes a member of the erythroblast transformation-specific (ETS) family of transcriptions factors. All members of this family are key regulators of embryonic development, cell proliferation, differentiation, angiogenesis, inflammation, and apoptosis. The protein encoded by this gene is mainly expressed in the nucleus. It contains an ETS DNA-binding domain and a PNT (pointed) domain which is implicated in the self-association of chimeric oncoproteins. This protein is required for platelet adhesion to the subendothelium, inducing vascular cell remodeling. It also regulates hematopoesis, and the differentiation and maturation of megakaryocytic cells. This gene is involved in chromosomal translocations, resulting in different fusion gene products, such as TMPSSR2-ERG and NDRG1-ERG in prostate cancer, EWS-ERG in Ewing's sarcoma and FUS-ERG in acute myeloid leukemia. More than two dozens of transcript variants generated from combinatorial usage of three alternative promoters and multiple alternative splicing events have been reported, but the full-length nature of many of these variants has not been determined. [provided by RefSeq, Apr 2014].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - regulation of transcription by RNA polymerase II\n",
      "    - protein phosphorylation\n",
      "    - signal transduction\n",
      "    - cell differentiation\n",
      "    - positive regulation of transcription by RNA polymerase II\n",
      "    - positive regulation of transcription by RNA polymerase II\n",
      "  CC:\n",
      "    - chromatin\n",
      "    - nucleus\n",
      "    - nucleoplasm\n",
      "    - cytosol\n",
      "    - ribonucleoprotein complex\n",
      "  MF:\n",
      "    - RNA polymerase II cis-regulatory region sequence-specific DNA binding\n",
      "    - DNA-binding transcription factor activity, RNA polymerase II-specific\n",
      "    - DNA-binding transcription factor activity, RNA polymerase II-specific\n",
      "    - DNA-binding transcription activator activity, RNA polymerase II-specific\n",
      "    - DNA binding\n",
      "    - chromatin binding\n",
      "    - protein binding\n",
      "    - sequence-specific double-stranded DNA binding\n",
      "UniProt: {'Swiss-Prot': 'P11308', 'TrEMBL': ['A0A0C4DG41', 'B5MDW0', 'B4DN83', 'B4DVX5', 'A8MX39', 'A8MZ24']}\n",
      "\n",
      "Gene: NMU\n",
      "Symbol: NMU\n",
      "Name: neuromedin U\n",
      "Summary: This gene encodes a member of the neuromedin family of neuropeptides. The encoded protein is a precursor that is proteolytically processed to generate a biologically active neuropeptide that plays a role in pain, stress, immune-mediated inflammatory diseases and feeding regulation. Increased expression of this gene was observed in renal, pancreatic and lung cancers. Alternative splicing results in multiple transcript variants encoding different isoforms. Some of these isoforms may undergo similar processing to generate the mature peptide. [provided by RefSeq, Jul 2015].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - temperature homeostasis\n",
      "    - G protein-coupled receptor signaling pathway\n",
      "    - neuropeptide signaling pathway\n",
      "    - neuropeptide signaling pathway\n",
      "    - positive regulation of smooth muscle contraction\n",
      "    - positive regulation of synaptic transmission\n",
      "    - regulation of feeding behavior\n",
      "    - energy homeostasis\n",
      "    - regulation of grooming behavior\n",
      "  CC:\n",
      "    - extracellular region\n",
      "    - terminal bouton\n",
      "  MF:\n",
      "    - signaling receptor binding\n",
      "    - protein binding\n",
      "    - type 1 neuromedin U receptor binding\n",
      "    - type 2 neuromedin U receptor binding\n",
      "    - neuromedin U receptor binding\n",
      "UniProt: {'Swiss-Prot': 'P48645', 'TrEMBL': ['A0A0B4J202', 'A0A250SH36', 'E9PDJ7', 'Q86WB6', 'D6RBC9']}\n",
      "\n",
      "Gene: EGLN1\n",
      "Symbol: EGLN1\n",
      "Name: egl-9 family hypoxia inducible factor 1\n",
      "Summary: The protein encoded by this gene catalyzes the post-translational formation of 4-hydroxyproline in hypoxia-inducible factor (HIF) alpha proteins. HIF is a transcriptional complex that plays a central role in mammalian oxygen homeostasis. This protein functions as a cellular oxygen sensor, and under normal oxygen concentration, modification by prolyl hydroxylation is a key regulatory event that targets HIF subunits for proteasomal destruction via the von Hippel-Lindau ubiquitylation complex. Mutations in this gene are associated with erythrocytosis familial type 3 (ECYT3). [provided by RefSeq, Nov 2009].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - response to hypoxia\n",
      "    - intracellular iron ion homeostasis\n",
      "    - intracellular oxygen homeostasis\n",
      "    - negative regulation of DNA-binding transcription factor activity\n",
      "    - regulation of neuron apoptotic process\n",
      "    - regulation of angiogenesis\n",
      "    - positive regulation of transcription by RNA polymerase II\n",
      "    - negative regulation of cyclic-nucleotide phosphodiesterase activity\n",
      "    - cardiac muscle tissue morphogenesis\n",
      "    - heart trabecula formation\n",
      "    - ventricular septum morphogenesis\n",
      "    - labyrinthine layer development\n",
      "    - cellular response to hypoxia\n",
      "    - response to nitric oxide\n",
      "    - regulation of modification of postsynaptic structure\n",
      "    - regulation protein catabolic process at postsynapse\n",
      "  CC:\n",
      "    - nucleus\n",
      "    - cytoplasm\n",
      "    - cytoplasm\n",
      "    - cytosol\n",
      "    - cytosol\n",
      "    - postsynaptic density\n",
      "    - intracellular membrane-bounded organelle\n",
      "    - glutamatergic synapse\n",
      "  MF:\n",
      "    - protein binding\n",
      "    - ferrous iron binding\n",
      "    - ferrous iron binding\n",
      "    - zinc ion binding\n",
      "    - 2-oxoglutarate-dependent dioxygenase activity\n",
      "    - enzyme binding\n",
      "    - L-ascorbic acid binding\n",
      "    - peptidyl-proline dioxygenase activity\n",
      "    - peptidyl-proline 4-dioxygenase activity\n",
      "    - hypoxia-inducible factor-proline dioxygenase activity\n",
      "UniProt: {'Swiss-Prot': 'Q9GZT9', 'TrEMBL': ['A0A590UJD6', 'R4SCQ0', 'A0A590UJ78', 'A0A590UJT7', 'A0A590UJZ0']}\n",
      "\n",
      "Gene: LAMB1\n",
      "Symbol: LAMB1\n",
      "Name: laminin subunit beta 1\n",
      "Summary: Laminins, a family of extracellular matrix glycoproteins, are the major noncollagenous constituent of basement membranes. They have been implicated in a wide variety of biological processes including cell adhesion, differentiation, migration, signaling, neurite outgrowth and metastasis. Laminins are composed of 3 non identical chains: laminin alpha, beta and gamma (formerly A, B1, and B2, respectively) and they form a cruciform structure consisting of 3 short arms, each formed by a different chain, and a long arm composed of all 3 chains. Each laminin chain is a multidomain protein encoded by a distinct gene. Several isoforms of each chain have been described. Different alpha, beta and gamma chain isomers combine to give rise to different heterotrimeric laminin isoforms which are designated by Arabic numerals in the order of their discovery, i.e. alpha1beta1gamma1 heterotrimer is laminin 1. The biological functions of the different chains and trimer molecules are largely unknown, but some of the chains have been shown to differ with respect to their tissue distribution, presumably reflecting diverse functions in vivo. This gene encodes the beta chain isoform laminin, beta 1. The beta 1 chain has 7 structurally distinct domains which it shares with other beta chain isomers. The C-terminal helical region containing domains I and II are separated by domain alpha, domains III and V contain several EGF-like repeats, and domains IV and VI have a globular conformation. Laminin, beta 1 is expressed in most tissues that produce basement membranes, and is one of the 3 chains constituting laminin 1, the first laminin isolated from Engelbreth-Holm-Swarm (EHS) tumor. A sequence in the beta 1 chain that is involved in cell attachment, chemotaxis, and binding to the laminin receptor was identified and shown to have the capacity to inhibit metastasis. [provided by RefSeq, Aug 2011].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - cell adhesion\n",
      "    - neuronal-glial interaction involved in cerebral cortex radial glia guided migration\n",
      "    - positive regulation of cell migration\n",
      "    - neuron projection development\n",
      "    - substrate adhesion-dependent cell spreading\n",
      "    - endodermal cell differentiation\n",
      "    - odontogenesis\n",
      "    - positive regulation of cell adhesion\n",
      "    - positive regulation of epithelial cell proliferation\n",
      "    - positive regulation of muscle cell differentiation\n",
      "    - regulation of basement membrane organization\n",
      "    - positive regulation of integrin-mediated signaling pathway\n",
      "  CC:\n",
      "    - extracellular region\n",
      "    - basement membrane\n",
      "    - basement membrane\n",
      "    - basement membrane\n",
      "    - laminin-1 complex\n",
      "    - laminin-1 complex\n",
      "    - laminin-1 complex\n",
      "    - laminin-2 complex\n",
      "    - laminin-2 complex\n",
      "    - extracellular space\n",
      "    - endoplasmic reticulum lumen\n",
      "    - laminin-8 complex\n",
      "    - laminin-8 complex\n",
      "    - laminin-10 complex\n",
      "    - perinuclear region of cytoplasm\n",
      "    - collagen-containing extracellular matrix\n",
      "    - collagen-containing extracellular matrix\n",
      "    - extracellular exosome\n",
      "    - protein complex involved in cell-matrix adhesion\n",
      "  MF:\n",
      "    - structural molecule activity\n",
      "    - extracellular matrix structural constituent\n",
      "    - extracellular matrix structural constituent\n",
      "    - extracellular matrix structural constituent\n",
      "    - protein binding\n",
      "UniProt: {'Swiss-Prot': 'P07942', 'TrEMBL': ['A0A7I2V378', 'Q8TAS6', 'A0A7I2V3J7', 'A0A7I2V5T9', 'G3XAI2', 'A0A7I2V682', 'A0A7I2V2J0', 'A0A7I2V4J9', 'A0A7I2V5X5', 'E9PCS6', 'A0A7I2V2R2', 'A0A7I2V2T9', 'C9J296', 'E7EPA6', 'A0A7I2V407', 'A0A7I2V5R0', 'A0A7I2V3I5']}\n",
      "\n",
      "Gene: IST1\n",
      "Symbol: IST1\n",
      "Name: IST1 factor associated with ESCRT-III\n",
      "Summary: This gene encodes a protein with MIT-interacting motifs that interacts with components of endosomal sorting complexes required for transport (ESCRT). ESCRT functions in vesicle budding, such as that which occurs during membrane abscission in cytokinesis. There is a pseudogene for this gene on chromosome 19. Alternatively spliced transcript variants encoding multiple isoforms have been observed for this gene. [provided by RefSeq, Aug 2012].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - protein localization\n",
      "    - protein localization\n",
      "    - protein transport\n",
      "    - multivesicular body assembly\n",
      "    - establishment of protein localization\n",
      "    - positive regulation of proteolysis\n",
      "    - collateral sprouting\n",
      "    - positive regulation of collateral sprouting\n",
      "    - cell division\n",
      "    - cytoskeleton-dependent cytokinesis\n",
      "    - midbody abscission\n",
      "    - midbody abscission\n",
      "    - ESCRT III complex disassembly\n",
      "  CC:\n",
      "    - chromatin\n",
      "    - extracellular region\n",
      "    - nuclear envelope\n",
      "    - endoplasmic reticulum-Golgi intermediate compartment\n",
      "    - centrosome\n",
      "    - cytosol\n",
      "    - cytosol\n",
      "    - midbody\n",
      "    - azurophil granule lumen\n",
      "    - intracellular membrane-bounded organelle\n",
      "    - extracellular exosome\n",
      "    - extracellular exosome\n",
      "    - Flemming body\n",
      "  MF:\n",
      "    - protein binding\n",
      "    - protein domain specific binding\n",
      "    - identical protein binding\n",
      "    - protein-containing complex binding\n",
      "    - cadherin binding\n",
      "    - MIT domain binding\n",
      "UniProt: {'Swiss-Prot': 'P53990', 'TrEMBL': ['H3BUI0', 'B4DXC9', 'E9PF00', 'J3QQP8', 'H3BQF7', 'H3BPP6', 'H3BMU1', 'F5GXM3', 'J3KR23', 'H3BRE2', 'H3BQ38']}\n",
      "\n",
      "Gene: NR3C2\n",
      "Symbol: NR3C2\n",
      "Name: nuclear receptor subfamily 3 group C member 2\n",
      "Summary: This gene encodes the mineralocorticoid receptor, which mediates aldosterone actions on salt and water balance within restricted target cells. The protein functions as a ligand-dependent transcription factor that binds to mineralocorticoid response elements in order to transactivate target genes. Mutations in this gene cause autosomal dominant pseudohypoaldosteronism type I, a disorder characterized by urinary salt wasting. Defects in this gene are also associated with early onset hypertension with severe exacerbation in pregnancy. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Oct 2009].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - regulation of transcription by RNA polymerase II\n",
      "    - signal transduction\n",
      "    - nuclear receptor-mediated steroid hormone signaling pathway\n",
      "    - positive regulation of non-canonical NF-kappaB signal transduction\n",
      "  CC:\n",
      "    - chromatin\n",
      "    - chromatin\n",
      "    - nucleus\n",
      "    - nucleoplasm\n",
      "    - nucleoplasm\n",
      "    - endoplasmic reticulum membrane\n",
      "    - cytosol\n",
      "    - receptor complex\n",
      "  MF:\n",
      "    - DNA-binding transcription factor activity, RNA polymerase II-specific\n",
      "    - DNA-binding transcription factor activity\n",
      "    - nuclear steroid receptor activity\n",
      "    - nuclear receptor activity\n",
      "    - steroid binding\n",
      "    - protein binding\n",
      "    - zinc ion binding\n",
      "    - TBP-class protein binding\n",
      "    - estrogen response element binding\n",
      "    - sequence-specific double-stranded DNA binding\n",
      "UniProt: {'Swiss-Prot': 'P08235', 'TrEMBL': 'B0ZBF6'}\n",
      "\n",
      "Gene: LTBR\n",
      "Symbol: LTBR\n",
      "Name: lymphotoxin beta receptor\n",
      "Summary: This gene encodes a member of the tumor necrosis factor receptor superfamily. The major ligands of this receptor include lymphotoxin alpha/beta and tumor necrosis factor ligand superfamily member 14. The encoded protein plays a role in signalling during the development of lymphoid and other organs, lipid metabolism, immune response, and programmed cell death. Activity of this receptor has also been linked to carcinogenesis. Alternatively spliced transcript variants encoding multiple isoforms have been observed. [provided by RefSeq, Aug 2012].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - apoptotic process\n",
      "    - immune response\n",
      "    - signal transduction\n",
      "    - myeloid dendritic cell differentiation\n",
      "    - positive regulation of canonical NF-kappaB signal transduction\n",
      "    - positive regulation of canonical NF-kappaB signal transduction\n",
      "    - positive regulation of JNK cascade\n",
      "    - hematopoietic or lymphoid organ development\n",
      "    - cellular response to mechanical stimulus\n",
      "    - positive regulation of extrinsic apoptotic signaling pathway\n",
      "  CC:\n",
      "    - Golgi apparatus\n",
      "    - plasma membrane\n",
      "  MF:\n",
      "    - protein binding\n",
      "    - ubiquitin protein ligase binding\n",
      "    - ubiquitin protein ligase binding\n",
      "    - identical protein binding\n",
      "UniProt: {'Swiss-Prot': 'P36941', 'TrEMBL': ['F5H6F8', 'F5H627', 'F5H2E7', 'F5H1S5']}\n",
      "\n",
      "Gene: TLR7\n",
      "Symbol: TLR7\n",
      "Name: toll like receptor 7\n",
      "Summary: The protein encoded by this gene is a member of the Toll-like receptor (TLR) family which plays a fundamental role in pathogen recognition and activation of innate immunity. TLRs are highly conserved from Drosophila to humans and share structural and functional similarities. The human TLR family comprises 11 members. They recognize pathogen-associated molecular patterns (PAMPs) that are expressed on infectious agents, and mediate the production of cytokines necessary for the development of effective immunity. For the recognition of structural components in foreign microorganisms, the various TLRs exhibit different patterns of expression as well; in this way for example, TLR-3, -7, and -8 are essential in the recognition of single-stranded RNA viruses. TLR7 senses single-stranded RNA oligonucleotides containing guanosine- and uridine-rich sequences from RNA viruses, a recognition occuring in the endosomes of plasmacytoid dendritic cells and B cells. This gene is predominantly expressed in lung, placenta, and spleen, and is phylogenetically related and lies in close proximity to another family member, TLR8, on chromosome X. [provided by RefSeq, Aug 2020].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - toll-like receptor signaling pathway\n",
      "    - inflammatory response\n",
      "    - canonical NF-kappaB signal transduction\n",
      "    - canonical NF-kappaB signal transduction\n",
      "    - I-kappaB phosphorylation\n",
      "    - JNK cascade\n",
      "    - positive regulation of chemokine production\n",
      "    - positive regulation of interferon-alpha production\n",
      "    - positive regulation of interferon-beta production\n",
      "    - positive regulation of type II interferon production\n",
      "    - positive regulation of interleukin-6 production\n",
      "    - positive regulation of interleukin-8 production\n",
      "    - positive regulation of interleukin-8 production\n",
      "    - toll-like receptor 7 signaling pathway\n",
      "    - toll-like receptor 7 signaling pathway\n",
      "    - toll-like receptor 8 signaling pathway\n",
      "    - positive regulation of canonical NF-kappaB signal transduction\n",
      "    - innate immune response\n",
      "    - positive regulation of transcription by RNA polymerase II\n",
      "    - positive regulation of inflammatory response\n",
      "    - defense response to virus\n",
      "    - defense response to virus\n",
      "    - defense response to virus\n",
      "    - positive regulation of macrophage cytokine production\n",
      "    - response to cGMP\n",
      "    - cellular response to mechanical stimulus\n",
      "    - cellular response to virus\n",
      "    - positive regulation of non-canonical NF-kappaB signal transduction\n",
      "  CC:\n",
      "    - Golgi membrane\n",
      "    - cytoplasm\n",
      "    - lysosome\n",
      "    - endosome\n",
      "    - endoplasmic reticulum\n",
      "    - endoplasmic reticulum membrane\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - endosome membrane\n",
      "    - early phagosome\n",
      "    - endolysosome membrane\n",
      "    - receptor complex\n",
      "  MF:\n",
      "    - double-stranded RNA binding\n",
      "    - single-stranded RNA binding\n",
      "    - single-stranded RNA binding\n",
      "    - protein binding\n",
      "    - siRNA binding\n",
      "    - pattern recognition receptor activity\n",
      "UniProt: {'Swiss-Prot': 'Q9NYK1', 'TrEMBL': 'B2R9N9'}\n",
      "\n",
      "Gene: SRPK2\n",
      "Symbol: SRPK2\n",
      "Name: SRSF protein kinase 2\n",
      "Summary: Enables ATP binding activity; magnesium ion binding activity; and protein serine/threonine kinase activity. Involved in several processes, including nucleic acid metabolic process; peptidyl-serine phosphorylation; and regulation of viral genome replication. Located in chromatin; cytosol; and nuclear lumen. [provided by Alliance of Genome Resources, Apr 2022]\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - spliceosomal complex assembly\n",
      "    - spliceosomal complex assembly\n",
      "    - angiogenesis\n",
      "    - protein phosphorylation\n",
      "    - positive regulation of cell population proliferation\n",
      "    - RNA splicing\n",
      "    - positive regulation of gene expression\n",
      "    - peptidyl-serine phosphorylation\n",
      "    - cell differentiation\n",
      "    - nuclear speck organization\n",
      "    - intracellular signal transduction\n",
      "    - intracellular signal transduction\n",
      "    - positive regulation of neuron apoptotic process\n",
      "    - positive regulation of viral genome replication\n",
      "    - negative regulation of viral genome replication\n",
      "    - innate immune response\n",
      "    - positive regulation of cell cycle\n",
      "    - regulation of mRNA splicing, via spliceosome\n",
      "    - regulation of mRNA processing\n",
      "    - R-loop processing\n",
      "  CC:\n",
      "    - chromatin\n",
      "    - nucleus\n",
      "    - nucleus\n",
      "    - nucleoplasm\n",
      "    - nucleolus\n",
      "    - cytoplasm\n",
      "    - cytoplasm\n",
      "    - cytosol\n",
      "    - cytosol\n",
      "    - nuclear speck\n",
      "    - presynapse\n",
      "  MF:\n",
      "    - magnesium ion binding\n",
      "    - RNA binding\n",
      "    - protein serine/threonine kinase activity\n",
      "    - protein serine/threonine kinase activity\n",
      "    - protein serine/threonine kinase activity\n",
      "    - 3-phosphoinositide-dependent protein kinase activity\n",
      "    - DNA-dependent protein kinase activity\n",
      "    - AMP-activated protein kinase activity\n",
      "    - eukaryotic translation initiation factor 2alpha kinase activity\n",
      "    - ribosomal protein S6 kinase activity\n",
      "    - protein binding\n",
      "    - ATP binding\n",
      "    - histone H3S10 kinase activity\n",
      "    - histone H3T11 kinase activity\n",
      "    - histone H3T6 kinase activity\n",
      "    - histone H2AXS139 kinase activity\n",
      "    - histone H3S28 kinase activity\n",
      "    - histone H4S1 kinase activity\n",
      "    - histone H2AS1 kinase activity\n",
      "    - histone H2BS14 kinase activity\n",
      "    - 14-3-3 protein binding\n",
      "    - histone H3T3 kinase activity\n",
      "    - histone H2AS121 kinase activity\n",
      "    - Rho-dependent protein serine/threonine kinase activity\n",
      "    - protein serine kinase activity\n",
      "    - histone H2BS36 kinase activity\n",
      "    - histone H3S57 kinase activity\n",
      "    - histone H3T45 kinase activity\n",
      "    - histone H2AT120 kinase activity\n",
      "UniProt: {'Swiss-Prot': 'P78362', 'TrEMBL': ['H7C521', 'C9J2M4', 'F8WAV6', 'F8WBW4', 'A0A994J591', 'H7C5L6', 'C9JWF7']}\n",
      "\n",
      "Gene: KIR2DL4\n",
      "Symbol: KIR2DL4\n",
      "Name: killer cell immunoglobulin like receptor, two Ig domains and long cytoplasmic tail 4\n",
      "Summary: Killer cell immunoglobulin-like receptors (KIRs) are transmembrane glycoproteins expressed by natural killer cells and subsets of T cells. The KIR genes are polymorphic and highly homologous and they are found in a cluster on chromosome 19q13.4 within the 1 Mb leukocyte receptor complex (LRC). The gene content of the KIR gene cluster varies among haplotypes, although several 'framework' genes are found in all haplotypes (KIR3DL3, KIR3DP1, KIR3DL4, KIR3DL2). The KIR proteins are classified by the number of extracellular immunoglobulin domains (2D or 3D) and by whether they have a long (L) or short (S) cytoplasmic domain. KIR proteins with the long cytoplasmic domain transduce inhibitory signals upon ligand binding via an immune tyrosine-based inhibitory motif (ITIM), while KIR proteins with the short cytoplasmic domain lack the ITIM motif and instead associate with the TYRO protein tyrosine kinase binding protein to transduce activating signals. The ligands for several KIR proteins are subsets of HLA class I molecules; thus, KIR proteins are thought to play an important role in regulation of the immune response. This gene is one of the 'framework' loci that is present on all haplotypes. Alternate alleles of this gene are represented on multiple alternate reference loci (ALT_REF_LOCs). Alternative splicing results in multiple transcript variants, some of which may not be annotated on the primary reference assembly. [provided by RefSeq, Jul 2016].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - positive regulation of natural killer cell cytokine production\n",
      "    - immune response-regulating signaling pathway\n",
      "    - cellular defense response\n",
      "    - signal transduction\n",
      "    - negative regulation of natural killer cell mediated cytotoxicity\n",
      "    - positive regulation of cellular senescence\n",
      "  CC:\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - membrane\n",
      "    - early endosome membrane\n",
      "  MF:\n",
      "    - transmembrane signaling receptor activity\n",
      "    - protein binding\n",
      "    - MHC class Ib receptor activity\n",
      "UniProt: {'Swiss-Prot': 'Q99706', 'TrEMBL': ['A0A376A929', 'A0A0B4J1S6', 'A0A0G2JQ13', 'K7RE46', 'A0A0G2JLM4', 'A0A0G2JLN0', 'Q8N736', 'A0A0G2JN69', 'A8MT18', 'E7EST5', 'A0A0G2JNE8', 'A0A1W2PQV0', 'A0A0G2JMQ9', 'A0A0G2JPS9', 'A0A0G2JMD1', 'A0A0G2JNG7', 'A0A0G2JLZ7', 'A0A0G2JQ39', 'A0A0G2JLM0']}\n",
      "\n",
      "Gene: MAPT\n",
      "Symbol: MAPT\n",
      "Name: microtubule associated protein tau\n",
      "Summary: This gene encodes the microtubule-associated protein tau (MAPT) whose transcript undergoes complex, regulated alternative splicing, giving rise to several mRNA species. MAPT transcripts are differentially expressed in the nervous system, depending on stage of neuronal maturation and neuron type. MAPT gene mutations have been associated with several neurodegenerative disorders such as Alzheimer's disease, Pick's disease, frontotemporal dementia, cortico-basal degeneration and progressive supranuclear palsy. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - microtubule cytoskeleton organization\n",
      "    - microtubule cytoskeleton organization\n",
      "    - microtubule cytoskeleton organization\n",
      "    - microglial cell activation\n",
      "    - DNA damage response\n",
      "    - cell-cell signaling\n",
      "    - synapse assembly\n",
      "    - learning or memory\n",
      "    - memory\n",
      "    - response to lead ion\n",
      "    - regulation of autophagy\n",
      "    - negative regulation of gene expression\n",
      "    - negative regulation of mitochondrial membrane potential\n",
      "    - rRNA metabolic process\n",
      "    - axonal transport of mitochondrion\n",
      "    - central nervous system neuron development\n",
      "    - regulation of microtubule polymerization or depolymerization\n",
      "    - regulation of microtubule polymerization\n",
      "    - regulation of microtubule polymerization\n",
      "    - positive regulation of microtubule polymerization\n",
      "    - cytoplasmic microtubule organization\n",
      "    - neuron projection development\n",
      "    - neuron projection development\n",
      "    - positive regulation of superoxide anion generation\n",
      "    - regulation of chromosome organization\n",
      "    - stress granule assembly\n",
      "    - cellular response to heat\n",
      "    - cellular response to reactive oxygen species\n",
      "    - positive regulation of axon extension\n",
      "    - microtubule polymerization\n",
      "    - microtubule polymerization\n",
      "    - astrocyte activation\n",
      "    - regulation of synaptic plasticity\n",
      "    - intracellular distribution of mitochondria\n",
      "    - generation of neurons\n",
      "    - synapse organization\n",
      "    - regulation of calcium-mediated signaling\n",
      "    - protein polymerization\n",
      "    - protein homooligomerization\n",
      "    - axon development\n",
      "    - regulation of microtubule cytoskeleton organization\n",
      "    - regulation of microtubule cytoskeleton organization\n",
      "    - plus-end-directed organelle transport along microtubule\n",
      "    - regulation of mitochondrial fission\n",
      "    - negative regulation of mitochondrial fission\n",
      "    - supramolecular fiber organization\n",
      "    - supramolecular fiber organization\n",
      "    - axonal transport\n",
      "    - axonal transport\n",
      "    - regulation of cellular response to heat\n",
      "    - regulation of long-term synaptic depression\n",
      "    - positive regulation of protein localization to synapse\n",
      "    - neurofibrillary tangle assembly\n",
      "    - negative regulation of establishment of protein localization to mitochondrion\n",
      "    - positive regulation of protein localization\n",
      "    - negative regulation of tubulin deacetylation\n",
      "    - negative regulation of tubulin deacetylation\n",
      "    - amyloid fibril formation\n",
      "    - amyloid fibril formation\n",
      "    - amyloid fibril formation\n",
      "    - amyloid fibril formation\n",
      "    - cellular response to nerve growth factor stimulus\n",
      "    - cellular response to brain-derived neurotrophic factor stimulus\n",
      "  CC:\n",
      "    - extracellular region\n",
      "    - nucleus\n",
      "    - nucleus\n",
      "    - cytoplasm\n",
      "    - cytoplasm\n",
      "    - cytoplasm\n",
      "    - mitochondrion\n",
      "    - cytosol\n",
      "    - cytosol\n",
      "    - cytosol\n",
      "    - microtubule\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - microtubule cytoskeleton\n",
      "    - microtubule cytoskeleton\n",
      "    - membrane\n",
      "    - axon\n",
      "    - axon\n",
      "    - axon\n",
      "    - dendrite\n",
      "    - growth cone\n",
      "    - axolemma\n",
      "    - nuclear periphery\n",
      "    - cytoplasmic ribonucleoprotein granule\n",
      "    - somatodendritic compartment\n",
      "    - somatodendritic compartment\n",
      "    - neuron projection\n",
      "    - neuronal cell body\n",
      "    - dendritic spine\n",
      "    - cell body\n",
      "    - cell body\n",
      "    - main axon\n",
      "    - membrane raft\n",
      "    - tubulin complex\n",
      "    - glial cell projection\n",
      "    - neurofibrillary tangle\n",
      "    - neurofibrillary tangle\n",
      "    - axon cytoplasm\n",
      "  MF:\n",
      "    - DNA binding\n",
      "    - DNA binding\n",
      "    - minor groove of adenine-thymine-rich DNA binding\n",
      "    - double-stranded DNA binding\n",
      "    - single-stranded DNA binding\n",
      "    - RNA binding\n",
      "    - actin binding\n",
      "    - protein binding\n",
      "    - microtubule binding\n",
      "    - microtubule binding\n",
      "    - microtubule binding\n",
      "    - microtubule binding\n",
      "    - microtubule binding\n",
      "    - SH3 domain binding\n",
      "    - enzyme binding\n",
      "    - enzyme binding\n",
      "    - protein kinase binding\n",
      "    - protein kinase binding\n",
      "    - protein-macromolecule adaptor activity\n",
      "    - apolipoprotein binding\n",
      "    - dynactin binding\n",
      "    - phosphatidylinositol binding\n",
      "    - identical protein binding\n",
      "    - identical protein binding\n",
      "    - identical protein binding\n",
      "    - sequence-specific DNA binding\n",
      "    - protein-folding chaperone binding\n",
      "    - protein phosphatase 2A binding\n",
      "    - Hsp90 protein binding\n",
      "    - lipoprotein particle binding\n",
      "    - histone-dependent DNA binding\n",
      "    - microtubule lateral binding\n",
      "    - phosphatidylinositol bisphosphate binding\n",
      "UniProt: {'Swiss-Prot': 'P10636', 'TrEMBL': ['A0A0G2JPD5', 'B3KTM0', 'I3L2Z2', 'A0A0G2JMX7', 'A0A0G2JQJ7', 'I3L170', 'A0A0G2JS76', 'A0A7P0T936', 'A0A7I2PLE3', 'A0A7I2PJZ2']}\n",
      "\n",
      "Gene: KCNH2\n",
      "Symbol: KCNH2\n",
      "Name: potassium voltage-gated channel subfamily H member 2\n",
      "Summary: This gene encodes a component of a voltage-activated potassium channel found in cardiac muscle, nerve cells, and microglia. Four copies of this protein interact with one copy of the KCNE2 protein to form a functional potassium channel. Mutations in this gene can cause long QT syndrome type 2 (LQT2). Transcript variants encoding distinct isoforms have been identified. [provided by RefSeq, May 2022].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - regulation of heart rate by hormone\n",
      "    - potassium ion transport\n",
      "    - regulation of membrane potential\n",
      "    - positive regulation of DNA-templated transcription\n",
      "    - potassium ion homeostasis\n",
      "    - cardiac muscle contraction\n",
      "    - regulation of membrane repolarization\n",
      "    - regulation of ventricular cardiac muscle cell membrane repolarization\n",
      "    - regulation of ventricular cardiac muscle cell membrane repolarization\n",
      "    - cellular response to xenobiotic stimulus\n",
      "    - potassium ion transmembrane transport\n",
      "    - potassium ion transmembrane transport\n",
      "    - potassium ion transmembrane transport\n",
      "    - ventricular cardiac muscle cell action potential\n",
      "    - membrane repolarization\n",
      "    - membrane depolarization during action potential\n",
      "    - membrane repolarization during action potential\n",
      "    - membrane repolarization during cardiac muscle cell action potential\n",
      "    - membrane repolarization during cardiac muscle cell action potential\n",
      "    - regulation of heart rate by cardiac conduction\n",
      "    - regulation of heart rate by cardiac conduction\n",
      "    - regulation of heart rate by cardiac conduction\n",
      "    - potassium ion export across plasma membrane\n",
      "    - potassium ion export across plasma membrane\n",
      "    - membrane repolarization during ventricular cardiac muscle cell action potential\n",
      "    - regulation of potassium ion transmembrane transport\n",
      "    - negative regulation of potassium ion transmembrane transport\n",
      "    - positive regulation of potassium ion transmembrane transport\n",
      "    - negative regulation of potassium ion export across plasma membrane\n",
      "    - potassium ion import across plasma membrane\n",
      "  CC:\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - voltage-gated potassium channel complex\n",
      "    - voltage-gated potassium channel complex\n",
      "    - cell surface\n",
      "    - perinuclear region of cytoplasm\n",
      "    - inward rectifier potassium channel complex\n",
      "  MF:\n",
      "    - transcription cis-regulatory region binding\n",
      "    - inward rectifier potassium channel activity\n",
      "    - inward rectifier potassium channel activity\n",
      "    - inward rectifier potassium channel activity\n",
      "    - inward rectifier potassium channel activity\n",
      "    - voltage-gated potassium channel activity\n",
      "    - delayed rectifier potassium channel activity\n",
      "    - delayed rectifier potassium channel activity\n",
      "    - protein binding\n",
      "    - ubiquitin protein ligase binding\n",
      "    - identical protein binding\n",
      "    - protein homodimerization activity\n",
      "    - scaffold protein binding\n",
      "    - voltage-gated potassium channel activity involved in ventricular cardiac muscle cell action potential repolarization\n",
      "UniProt: {'Swiss-Prot': 'Q12809', 'TrEMBL': ['A0A090N7X5', 'A0A090N7W1', 'Q15BH2', 'A0AAQ5BGQ9', 'A0A090N8Q0', 'A0AAQ5BGR0']}\n",
      "\n",
      "Gene: CD37\n",
      "Symbol: CD37\n",
      "Name: CD37 molecule\n",
      "Summary: The protein encoded by this gene is a member of the transmembrane 4 superfamily, also known as the tetraspanin family. Most of these members are cell-surface proteins that are characterized by the presence of four hydrophobic domains. The proteins mediate signal transduction events that play a role in the regulation of cell development, activation, growth and motility. This encoded protein is a cell surface glycoprotein that is known to complex with integrins and other transmembrane 4 superfamily proteins. It may play a role in T-cell-B-cell interactions. Alternate splicing results in multiple transcript variants encoding different isoforms. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  CC:\n",
      "    - immunological synapse\n",
      "    - plasma membrane\n",
      "    - membrane\n",
      "    - extracellular exosome\n",
      "  MF:\n",
      "    - protein binding\n",
      "UniProt: {'Swiss-Prot': 'P11049', 'TrEMBL': ['B7ZAN3', 'Q96LM7', 'M0R1J1', 'M0QZP6', 'M0R083', 'M0QZ33']}\n",
      "\n",
      "Gene: SULT2B1\n",
      "Symbol: SULT2B1\n",
      "Name: sulfotransferase family 2B member 1\n",
      "Summary: Sulfotransferase enzymes catalyze the sulfate conjugation of many hormones, neurotransmitters, drugs, and xenobiotic compounds. These cytosolic enzymes are different in their tissue distributions and substrate specificities. The gene structure (number and length of exons) is similar among family members. This gene sulfates dehydroepiandrosterone but not 4-nitrophenol, a typical substrate for the phenol and estrogen sulfotransferase subfamilies. Two alternatively spliced variants that encode different isoforms have been described. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - steroid metabolic process\n",
      "    - cholesterol metabolic process\n",
      "    - negative regulation of cell population proliferation\n",
      "    - positive regulation of epidermal cell differentiation\n",
      "    - 3'-phosphoadenosine 5'-phosphosulfate metabolic process\n",
      "    - sulfation\n",
      "  CC:\n",
      "    - nucleus\n",
      "    - cytoplasm\n",
      "    - endoplasmic reticulum\n",
      "    - cytosol\n",
      "    - cytosol\n",
      "    - intracellular membrane-bounded organelle\n",
      "    - extracellular exosome\n",
      "  MF:\n",
      "    - nucleic acid binding\n",
      "    - protein binding\n",
      "    - sulfotransferase activity\n",
      "    - cholesterol binding\n",
      "    - small molecule binding\n",
      "    - steroid sulfotransferase activity\n",
      "    - cholesterol sulfotransferase activity\n",
      "    - steroid hormone binding\n",
      "UniProt: {'Swiss-Prot': 'O00204'}\n",
      "\n",
      "Gene: ABCC4\n",
      "Symbol: ABCC4\n",
      "Name: ATP binding cassette subfamily C member 4 (PEL blood group)\n",
      "Summary: The protein encoded by this gene is a member of the superfamily of ATP-binding cassette (ABC) transporters. ABC proteins transport various molecules across extra- and intra-cellular membranes. ABC genes are divided into seven distinct subfamilies (ABC1, MDR/TAP, MRP, ALD, OABP, GCN20, White). This protein is a member of the MRP subfamily which is involved in multi-drug resistance. This family member plays a role in cellular detoxification as a pump for its substrate, organic anions. It may also function in prostaglandin-mediated cAMP signaling in ciliogenesis. Alternative splicing of this gene results in multiple transcript variants. [provided by RefSeq, Sep 2014].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - platelet degranulation\n",
      "    - xenobiotic metabolic process\n",
      "    - xenobiotic transmembrane transport\n",
      "    - bile acid and bile salt transport\n",
      "    - prostaglandin transport\n",
      "    - urate transport\n",
      "    - prostaglandin secretion\n",
      "    - glutathione transmembrane transport\n",
      "    - transmembrane transport\n",
      "    - transmembrane transport\n",
      "    - cilium assembly\n",
      "    - cAMP transport\n",
      "    - cAMP transport\n",
      "    - leukotriene transport\n",
      "    - monoatomic anion transmembrane transport\n",
      "    - export across plasma membrane\n",
      "    - transport across blood-brain barrier\n",
      "    - guanine nucleotide transmembrane transport\n",
      "  CC:\n",
      "    - nucleolus\n",
      "    - Golgi apparatus\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - membrane\n",
      "    - membrane\n",
      "    - basolateral plasma membrane\n",
      "    - apical plasma membrane\n",
      "    - platelet dense granule membrane\n",
      "    - platelet dense granule membrane\n",
      "    - intracellular membrane-bounded organelle\n",
      "    - external side of apical plasma membrane\n",
      "  MF:\n",
      "    - guanine nucleotide transmembrane transporter activity\n",
      "    - protein binding\n",
      "    - ATP binding\n",
      "    - ABC-type xenobiotic transporter activity\n",
      "    - prostaglandin transmembrane transporter activity\n",
      "    - urate transmembrane transporter activity\n",
      "    - purine nucleotide transmembrane transporter activity\n",
      "    - ABC-type glutathione S-conjugate transporter activity\n",
      "    - ABC-type bile acid transporter activity\n",
      "    - efflux transmembrane transporter activity\n",
      "    - 15-hydroxyprostaglandin dehydrogenase (NAD+) activity\n",
      "    - ATP hydrolysis activity\n",
      "    - glutathione transmembrane transporter activity\n",
      "    - ATPase-coupled transmembrane transporter activity\n",
      "    - ATPase-coupled transmembrane transporter activity\n",
      "    - ATPase-coupled transmembrane transporter activity\n",
      "    - xenobiotic transmembrane transporter activity\n",
      "    - xenobiotic transmembrane transporter activity\n",
      "    - ATPase-coupled inorganic anion transmembrane transporter activity\n",
      "    - ABC-type transporter activity\n",
      "UniProt: {'Swiss-Prot': 'O15439', 'TrEMBL': ['Q8IWU1', 'A8K2Q2', 'A0A2R8YDG7', 'A0A2R8Y5Z3', 'A0A2R8Y6V8', 'A0A2R8YDC1']}\n",
      "\n",
      "Gene: BRAP\n",
      "Symbol: BRAP\n",
      "Name: BRCA1 associated protein\n",
      "Summary: The protein encoded by this gene was identified by its ability to bind to the nuclear localization signal of BRCA1 and other proteins. It is a cytoplasmic protein which may regulate nuclear targeting by retaining proteins with a nuclear localization signal in the cytoplasm. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - MAPK cascade\n",
      "    - MAPK cascade\n",
      "    - Ras protein signal transduction\n",
      "    - Ras protein signal transduction\n",
      "    - negative regulation of signal transduction\n",
      "    - protein ubiquitination\n",
      "    - protein ubiquitination\n",
      "    - protein ubiquitination\n",
      "  CC:\n",
      "    - ubiquitin ligase complex\n",
      "    - nucleoplasm\n",
      "    - cytoplasm\n",
      "    - cytoplasm\n",
      "    - cytosol\n",
      "    - cytosol\n",
      "    - nuclear membrane\n",
      "  MF:\n",
      "    - nucleic acid binding\n",
      "    - ubiquitin-protein transferase activity\n",
      "    - protein binding\n",
      "    - nuclear localization sequence binding\n",
      "    - zinc ion binding\n",
      "    - identical protein binding\n",
      "    - ubiquitin protein ligase activity\n",
      "    - ubiquitin protein ligase activity\n",
      "    - ubiquitin protein ligase activity\n",
      "UniProt: {'Swiss-Prot': 'Q7Z569', 'TrEMBL': ['J3KNN7', 'Q59H81']}\n",
      "\n",
      "Gene: TGFB2\n",
      "Symbol: TGFB2\n",
      "Name: transforming growth factor beta 2\n",
      "Summary: This gene encodes a secreted ligand of the TGF-beta (transforming growth factor-beta) superfamily of proteins. Ligands of this family bind various TGF-beta receptors leading to recruitment and activation of SMAD family transcription factors that regulate gene expression. The encoded preproprotein is proteolytically processed to generate a latency-associated peptide (LAP) and a mature peptide, and is found in either a latent form composed of a mature peptide homodimer, a LAP homodimer, and a latent TGF-beta binding protein, or in an active form consisting solely of the mature peptide homodimer. The mature peptide may also form heterodimers with other TGF-beta family members. Disruption of the TGF-beta/SMAD pathway has been implicated in a variety of human cancers. A chromosomal translocation that includes this gene is associated with Peters' anomaly, a congenital defect of the anterior chamber of the eye. Mutations in this gene may be associated with Loeys-Dietz syndrome. This gene encodes multiple isoforms that may undergo similar proteolytic processing. [provided by RefSeq, Aug 2016].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - cell morphogenesis\n",
      "    - skeletal system development\n",
      "    - eye development\n",
      "    - response to hypoxia\n",
      "    - kidney development\n",
      "    - epithelial to mesenchymal transition\n",
      "    - epithelial to mesenchymal transition\n",
      "    - neural tube closure\n",
      "    - hair follicle development\n",
      "    - hair follicle development\n",
      "    - heart morphogenesis\n",
      "    - outflow tract septum morphogenesis\n",
      "    - membranous septum morphogenesis\n",
      "    - heart valve morphogenesis\n",
      "    - atrioventricular valve morphogenesis\n",
      "    - pulmonary valve morphogenesis\n",
      "    - endocardial cushion morphogenesis\n",
      "    - cardiac right ventricle morphogenesis\n",
      "    - ventricular trabecula myocardium morphogenesis\n",
      "    - endocardial cushion fusion\n",
      "    - atrial septum primum morphogenesis\n",
      "    - neural retina development\n",
      "    - transforming growth factor beta receptor signaling pathway\n",
      "    - transforming growth factor beta receptor signaling pathway\n",
      "    - salivary gland morphogenesis\n",
      "    - heart development\n",
      "    - positive regulation of cell population proliferation\n",
      "    - negative regulation of cell population proliferation\n",
      "    - glial cell migration\n",
      "    - male gonad development\n",
      "    - response to wounding\n",
      "    - embryo development ending in birth or egg hatching\n",
      "    - cardioblast differentiation\n",
      "    - negative regulation of gene expression\n",
      "    - positive regulation of epithelial cell migration\n",
      "    - positive regulation of epithelial to mesenchymal transition\n",
      "    - negative regulation of macrophage cytokine production\n",
      "    - cell migration\n",
      "    - negative regulation of angiogenesis\n",
      "    - signaling\n",
      "    - hemopoiesis\n",
      "    - collagen fibril organization\n",
      "    - positive regulation of cell growth\n",
      "    - negative regulation of cell growth\n",
      "    - embryonic limb morphogenesis\n",
      "    - neutrophil chemotaxis\n",
      "    - epithelial cell differentiation\n",
      "    - hair follicle morphogenesis\n",
      "    - activation of protein kinase activity\n",
      "    - response to progesterone\n",
      "    - positive regulation of stress-activated MAPK cascade\n",
      "    - regulation of transforming growth factor beta2 production\n",
      "    - positive regulation of cell adhesion mediated by integrin\n",
      "    - ascending aorta morphogenesis\n",
      "    - wound healing\n",
      "    - regulation of cell population proliferation\n",
      "    - regulation of cell population proliferation\n",
      "    - dopamine biosynthetic process\n",
      "    - odontogenesis\n",
      "    - uterine wall breakdown\n",
      "    - positive regulation of neuron apoptotic process\n",
      "    - cell-cell junction organization\n",
      "    - positive regulation of integrin biosynthetic process\n",
      "    - positive regulation of Notch signaling pathway\n",
      "    - positive regulation of cell cycle\n",
      "    - positive regulation of heart contraction\n",
      "    - negative regulation of Ras protein signal transduction\n",
      "    - somatic stem cell division\n",
      "    - embryonic digestive tract development\n",
      "    - neuron development\n",
      "    - generation of neurons\n",
      "    - inner ear development\n",
      "    - negative regulation of epithelial cell proliferation\n",
      "    - negative regulation of epithelial cell proliferation\n",
      "    - positive regulation of protein secretion\n",
      "    - positive regulation of immune response\n",
      "    - positive regulation of cell division\n",
      "    - regulation of timing of catagen\n",
      "    - positive regulation of timing of catagen\n",
      "    - positive regulation of cardioblast differentiation\n",
      "    - positive regulation of phosphatidylinositol 3-kinase/protein kinase B signal transduction\n",
      "    - cardiac muscle cell proliferation\n",
      "    - uterus development\n",
      "    - cardiac epithelial to mesenchymal transition\n",
      "    - positive regulation of SMAD protein signal transduction\n",
      "    - ventricular septum morphogenesis\n",
      "    - atrial septum morphogenesis\n",
      "    - pharyngeal arch artery morphogenesis\n",
      "    - secondary palate development\n",
      "    - extrinsic apoptotic signaling pathway\n",
      "    - regulation of apoptotic process involved in outflow tract morphogenesis\n",
      "    - positive regulation of miRNA transcription\n",
      "    - regulation of complement-dependent cytotoxicity\n",
      "    - substantia propria of cornea development\n",
      "    - cranial skeletal system development\n",
      "    - negative regulation of epithelial to mesenchymal transition involved in endocardial cushion formation\n",
      "    - positive regulation of epithelial to mesenchymal transition involved in endocardial cushion formation\n",
      "  CC:\n",
      "    - extracellular region\n",
      "    - extracellular region\n",
      "    - extracellular space\n",
      "    - extracellular space\n",
      "    - axon\n",
      "    - platelet alpha granule lumen\n",
      "    - neuronal cell body\n",
      "    - collagen-containing extracellular matrix\n",
      "  MF:\n",
      "    - amyloid-beta binding\n",
      "    - signaling receptor binding\n",
      "    - type II transforming growth factor beta receptor binding\n",
      "    - type II transforming growth factor beta receptor binding\n",
      "    - type II transforming growth factor beta receptor binding\n",
      "    - cytokine activity\n",
      "    - transforming growth factor beta receptor binding\n",
      "    - protein binding\n",
      "    - growth factor activity\n",
      "    - type III transforming growth factor beta receptor binding\n",
      "    - type III transforming growth factor beta receptor binding\n",
      "    - protein homodimerization activity\n",
      "UniProt: {'Swiss-Prot': 'P61812', 'TrEMBL': 'Q59EG9'}\n",
      "\n",
      "Gene: ADH4\n",
      "Symbol: ADH4\n",
      "Name: alcohol dehydrogenase 4 (class II), pi polypeptide\n",
      "Summary: This gene encodes class II alcohol dehydrogenase 4 pi subunit, which is a member of the alcohol dehydrogenase family. Members of this enzyme family metabolize a wide variety of substrates, including ethanol, retinol, other aliphatic alcohols, hydroxysteroids, and lipid peroxidation products. Class II alcohol dehydrogenase is a homodimer composed of 2 pi subunits. It exhibits a high activity for oxidation of long-chain aliphatic alcohols and aromatic alcohols and is less sensitive to pyrazole. This gene is localized to chromosome 4 in the cluster of alcohol dehydrogenase genes. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - retinoid metabolic process\n",
      "    - alcohol metabolic process\n",
      "    - ethanol metabolic process\n",
      "    - aldehyde metabolic process\n",
      "    - fatty acid omega-oxidation\n",
      "    - retinol metabolic process\n",
      "    - alcohol catabolic process\n",
      "    - formaldehyde catabolic process\n",
      "    - quinone metabolic process\n",
      "  CC:\n",
      "    - nucleoplasm\n",
      "    - cytosol\n",
      "    - cytosol\n",
      "    - cytosol\n",
      "  MF:\n",
      "    - NADPH:quinone reductase activity\n",
      "    - alcohol dehydrogenase (NAD+) activity\n",
      "    - alcohol dehydrogenase (NAD+) activity\n",
      "    - aldose reductase (NADPH) activity\n",
      "    - all-trans-retinol dehydrogenase (NAD+) activity\n",
      "    - all-trans retinal binding\n",
      "    - protein binding\n",
      "    - zinc ion binding\n",
      "    - zinc ion binding\n",
      "    - benzaldehyde dehydrogenase (NAD+) activity\n",
      "    - retinol binding\n",
      "    - NAD binding\n",
      "    - S-(hydroxymethyl)glutathione dehydrogenase [NAD(P)+] activity\n",
      "UniProt: {'Swiss-Prot': 'P08319', 'TrEMBL': ['D6RF43', 'V9HVX7', 'D6R9K8', 'H0Y9N0', 'D6RIB1']}\n",
      "\n",
      "Gene: MAP4K4\n",
      "Symbol: MAP4K4\n",
      "Name: mitogen-activated protein kinase kinase kinase kinase 4\n",
      "Summary: The protein encoded by this gene is a member of the serine/threonine protein kinase family. This kinase has been shown to specifically activate MAPK8/JNK. The activation of MAPK8 by this kinase is found to be inhibited by the dominant-negative mutants of MAP3K7/TAK1, MAP2K4/MKK4, and MAP2K7/MKK7, which suggests that this kinase may function through the MAP3K7-MAP2K4-MAP2K7 kinase cascade, and mediate the TNF-alpha signaling pathway. Alternatively spliced transcript variants encoding different isoforms have been identified. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - MAPK cascade\n",
      "    - negative regulation of cell-matrix adhesion\n",
      "    - chromatin remodeling\n",
      "    - protein phosphorylation\n",
      "    - microvillus assembly\n",
      "    - positive regulation of cell migration\n",
      "    - positive regulation of ARF protein signal transduction\n",
      "    - positive regulation of hippo signaling\n",
      "    - intracellular signal transduction\n",
      "    - negative regulation of apoptotic process\n",
      "    - regulation of MAPK cascade\n",
      "    - positive regulation of GTPase activity\n",
      "    - regulation of JNK cascade\n",
      "    - neuron projection morphogenesis\n",
      "    - positive regulation of keratinocyte migration\n",
      "    - positive regulation of focal adhesion assembly\n",
      "    - positive regulation of focal adhesion disassembly\n",
      "  CC:\n",
      "    - cytoplasm\n",
      "    - cytoplasm\n",
      "    - cytoplasm\n",
      "    - focal adhesion\n",
      "  MF:\n",
      "    - creatine kinase activity\n",
      "    - protein serine/threonine kinase activity\n",
      "    - protein serine/threonine kinase activity\n",
      "    - 3-phosphoinositide-dependent protein kinase activity\n",
      "    - DNA-dependent protein kinase activity\n",
      "    - AMP-activated protein kinase activity\n",
      "    - eukaryotic translation initiation factor 2alpha kinase activity\n",
      "    - ribosomal protein S6 kinase activity\n",
      "    - protein binding\n",
      "    - ATP binding\n",
      "    - microtubule binding\n",
      "    - MAP kinase kinase kinase kinase activity\n",
      "    - histone H3S10 kinase activity\n",
      "    - histone H3T11 kinase activity\n",
      "    - histone H3T6 kinase activity\n",
      "    - histone H2AXS139 kinase activity\n",
      "    - histone H3S28 kinase activity\n",
      "    - histone H4S1 kinase activity\n",
      "    - histone H2AS1 kinase activity\n",
      "    - histone H2BS14 kinase activity\n",
      "    - histone H3T3 kinase activity\n",
      "    - histone H2AS121 kinase activity\n",
      "    - Rho-dependent protein serine/threonine kinase activity\n",
      "    - protein serine kinase activity\n",
      "    - histone H2BS36 kinase activity\n",
      "    - histone H3S57 kinase activity\n",
      "    - histone H3T45 kinase activity\n",
      "    - histone H2AT120 kinase activity\n",
      "UniProt: {'Swiss-Prot': 'O95819', 'TrEMBL': ['V9HWH3', 'E7EN19', 'B7Z3V5', 'H7C360', 'G5E948', 'H7C0P6', 'E7EX83', 'H7C3Z6', 'E7ENQ1', 'E7ETN6']}\n",
      "\n",
      "Gene: DRD4\n",
      "Symbol: DRD4\n",
      "Name: dopamine receptor D4\n",
      "Summary: This gene encodes the D4 subtype of the dopamine receptor. The D4 subtype is a G-protein coupled receptor which inhibits adenylyl cyclase. It is a target for drugs which treat schizophrenia and Parkinson disease. Mutations in this gene have been associated with various behavioral phenotypes, including autonomic nervous system dysfunction, attention deficit/hyperactivity disorder, and the personality trait of novelty seeking. This gene contains a polymorphic number (2-10 copies) of tandem 48 nt repeats; the sequence shown contains four repeats. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - behavioral fear response\n",
      "    - synaptic transmission, dopaminergic\n",
      "    - response to amphetamine\n",
      "    - intracellular calcium ion homeostasis\n",
      "    - G protein-coupled receptor signaling pathway, coupled to cyclic nucleotide second messenger\n",
      "    - adenylate cyclase-modulating G protein-coupled receptor signaling pathway\n",
      "    - adenylate cyclase-inhibiting dopamine receptor signaling pathway\n",
      "    - adenylate cyclase-inhibiting dopamine receptor signaling pathway\n",
      "    - chemical synaptic transmission\n",
      "    - adult locomotory behavior\n",
      "    - response to histamine\n",
      "    - social behavior\n",
      "    - regulation of dopamine metabolic process\n",
      "    - fear response\n",
      "    - regulation of circadian rhythm\n",
      "    - positive regulation of MAP kinase activity\n",
      "    - behavioral response to cocaine\n",
      "    - behavioral response to ethanol\n",
      "    - rhythmic process\n",
      "    - arachidonate secretion\n",
      "    - negative regulation of protein secretion\n",
      "    - positive regulation of dopamine uptake involved in synaptic transmission\n",
      "    - inhibitory postsynaptic potential\n",
      "    - phospholipase C-activating dopamine receptor signaling pathway\n",
      "    - G protein-coupled serotonin receptor signaling pathway\n",
      "    - regulation of postsynaptic neurotransmitter receptor internalization\n",
      "  CC:\n",
      "    - centrosome\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - membrane\n",
      "    - dendrite\n",
      "    - postsynapse\n",
      "    - glutamatergic synapse\n",
      "  MF:\n",
      "    - dopamine neurotransmitter receptor activity, coupled via Gi/Go\n",
      "    - dopamine neurotransmitter receptor activity, coupled via Gi/Go\n",
      "    - dopamine neurotransmitter receptor activity, coupled via Gi/Go\n",
      "    - dopamine neurotransmitter receptor activity\n",
      "    - G protein-coupled serotonin receptor activity\n",
      "    - protein binding\n",
      "    - potassium channel regulator activity\n",
      "    - SH3 domain binding\n",
      "    - neurotransmitter receptor activity\n",
      "    - dopamine binding\n",
      "    - identical protein binding\n",
      "    - metal ion binding\n",
      "    - epinephrine binding\n",
      "    - norepinephrine binding\n",
      "UniProt: {'Swiss-Prot': 'P21917'}\n",
      "\n",
      "Gene: MUC5AC\n",
      "Symbol: MUC5AC\n",
      "Name: mucin 5AC, oligomeric mucus/gel-forming\n",
      "Summary: Predicted to be an extracellular matrix structural constituent. Involved in phosphatidylinositol-mediated signaling. Located in cytoplasm; extracellular space; and mucus layer. Implicated in dry eye syndrome. Biomarker of several diseases, including Sjogren's syndrome; biliary tract disease (multiple); cystic fibrosis; eye disease (multiple); and pancreatic cancer (multiple). [provided by Alliance of Genome Resources, Apr 2022]\n",
      "GO Annotations:\n",
      "  CC:\n",
      "    - extracellular space\n",
      "    - extracellular space\n",
      "    - Golgi lumen\n",
      "    - plasma membrane\n",
      "    - extracellular matrix\n",
      "    - extracellular exosome\n",
      "    - mucus layer\n",
      "  MF:\n",
      "    - extracellular matrix structural constituent\n",
      "    - metal ion binding\n",
      "UniProt: {'Swiss-Prot': 'P98088'}\n",
      "\n",
      "Gene: GRIN2C\n",
      "Symbol: GRIN2C\n",
      "Name: glutamate ionotropic receptor NMDA type subunit 2C\n",
      "Summary: This gene encodes a subunit of the N-methyl-D-aspartate (NMDA) receptor, which is a subtype of ionotropic glutamate receptor. NMDA receptors are found in the central nervous system, are permeable to cations and have an important role in physiological processes such as learning, memory, and synaptic development. The receptor is a tetramer of different subunits (typically heterodimer of subunit 1 with one or more of subunits 2A-D), forming a channel that is permeable to calcium, potassium, and sodium, and whose properties are determined by subunit composition. Alterations in the subunit composition of the receptor are associated with pathophysiological conditions such as Parkinson's disease, Alzheimer's disease, depression, and schizophrenia. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Jun 2013].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - glutamate receptor signaling pathway\n",
      "    - brain development\n",
      "    - response to wounding\n",
      "    - directional locomotion\n",
      "    - ionotropic glutamate receptor signaling pathway\n",
      "    - synaptic transmission, glutamatergic\n",
      "    - negative regulation of protein catabolic process\n",
      "    - regulation of synaptic plasticity\n",
      "    - regulation of synaptic plasticity\n",
      "    - regulation of neuronal synaptic plasticity\n",
      "    - neuromuscular process controlling balance\n",
      "    - positive regulation of synaptic transmission, glutamatergic\n",
      "    - excitatory postsynaptic potential\n",
      "    - long-term synaptic potentiation\n",
      "    - calcium ion transmembrane import into cytosol\n",
      "    - monoatomic cation transmembrane transport\n",
      "    - monoatomic cation transmembrane transport\n",
      "    - excitatory chemical synaptic transmission\n",
      "    - protein localization to postsynaptic membrane\n",
      "    - regulation of monoatomic cation transmembrane transport\n",
      "    - positive regulation of excitatory postsynaptic potential\n",
      "  CC:\n",
      "    - endoplasmic reticulum membrane\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - NMDA selective glutamate receptor complex\n",
      "    - NMDA selective glutamate receptor complex\n",
      "    - NMDA selective glutamate receptor complex\n",
      "    - NMDA selective glutamate receptor complex\n",
      "    - postsynaptic membrane\n",
      "    - postsynaptic membrane\n",
      "    - postsynaptic density membrane\n",
      "    - glutamatergic synapse\n",
      "  MF:\n",
      "    - NMDA glutamate receptor activity\n",
      "    - NMDA glutamate receptor activity\n",
      "    - NMDA glutamate receptor activity\n",
      "    - monoatomic cation channel activity\n",
      "    - protein binding\n",
      "    - transmitter-gated monoatomic ion channel activity involved in regulation of postsynaptic membrane potential\n",
      "UniProt: {'Swiss-Prot': 'Q14957', 'TrEMBL': ['H0Y2V8', 'A0A8D9PH81', 'O15398', 'Q8IW23']}\n",
      "\n",
      "Gene: TRPA1\n",
      "Symbol: TRPA1\n",
      "Name: transient receptor potential cation channel subfamily A member 1\n",
      "Summary: The structure of the protein encoded by this gene is highly related to both the protein ankyrin and transmembrane proteins.  The specific function of this protein has not yet been determined; however, studies indicate the function may involve a role in signal transduction and growth control. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - monoatomic ion transport\n",
      "    - intracellular calcium ion homeostasis\n",
      "    - cell surface receptor signaling pathway\n",
      "    - response to cold\n",
      "    - response to xenobiotic stimulus\n",
      "    - sensory perception of pain\n",
      "    - calcium-mediated signaling\n",
      "    - response to pain\n",
      "    - thermoception\n",
      "    - detection of mechanical stimulus involved in sensory perception of pain\n",
      "    - detection of chemical stimulus involved in sensory perception of pain\n",
      "    - protein homotetramerization\n",
      "    - cellular response to hydrogen peroxide\n",
      "    - calcium ion transmembrane transport\n",
      "    - calcium ion transmembrane transport\n",
      "    - cellular response to food\n",
      "  CC:\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - stereocilium bundle\n",
      "  MF:\n",
      "    - calcium channel activity\n",
      "    - channel activity\n",
      "    - intracellularly gated calcium channel activity\n",
      "    - intracellularly gated calcium channel activity\n",
      "    - identical protein binding\n",
      "    - temperature-gated cation channel activity\n",
      "UniProt: {'Swiss-Prot': 'O75762', 'TrEMBL': 'H0YAW0'}\n",
      "\n",
      "Gene: KRT19\n",
      "Symbol: KRT19\n",
      "Name: keratin 19\n",
      "Summary: The protein encoded by this gene is a member of the keratin family. The keratins are intermediate filament proteins responsible for the structural integrity of epithelial cells and are subdivided into cytokeratins and hair keratins. The type I cytokeratins consist of acidic proteins which are arranged in pairs of heterotypic keratin chains. Unlike its related family members, this smallest known acidic cytokeratin is not paired with a basic cytokeratin in epithelial cells. It is specifically expressed in the periderm, the transiently superficial layer that envelopes the developing epidermis. The type I cytokeratins are clustered in a region of chromosome 17q12-q21. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - Notch signaling pathway\n",
      "    - epithelial cell differentiation\n",
      "    - response to estrogen\n",
      "    - intermediate filament organization\n",
      "    - sarcomere organization\n",
      "    - cell differentiation involved in embryonic placenta development\n",
      "  CC:\n",
      "    - cytosol\n",
      "    - cytoskeleton\n",
      "    - intermediate filament\n",
      "    - plasma membrane\n",
      "    - apicolateral plasma membrane\n",
      "    - Z disc\n",
      "    - sarcolemma\n",
      "    - costamere\n",
      "    - extracellular exosome\n",
      "    - cell periphery\n",
      "    - terminal web\n",
      "  MF:\n",
      "    - structural constituent of cytoskeleton\n",
      "    - protein binding\n",
      "    - structural constituent of muscle\n",
      "UniProt: {'Swiss-Prot': 'P08727', 'TrEMBL': ['K7EMS3', 'C9JM50']}\n",
      "\n",
      "Gene: AKNA\n",
      "Symbol: AKNA\n",
      "Name: AT-hook transcription factor\n",
      "Summary: Predicted to enable DNA binding activity. Involved in positive regulation of transcription by RNA polymerase II. Located in centrosome; cytosol; and nuclear lumen. [provided by Alliance of Genome Resources, Apr 2022]\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - epithelial to mesenchymal transition\n",
      "    - epithelial to mesenchymal transition\n",
      "    - neuroblast division in subventricular zone\n",
      "    - neuroblast division in subventricular zone\n",
      "    - positive regulation of transcription by RNA polymerase II\n",
      "    - regulation of inflammatory response\n",
      "    - delamination\n",
      "    - neuroblast delamination\n",
      "    - neuroblast delamination\n",
      "  CC:\n",
      "    - fibrillar center\n",
      "    - nucleoplasm\n",
      "    - centrosome\n",
      "    - centrosome\n",
      "    - centriole\n",
      "    - cytosol\n",
      "    - microtubule\n",
      "    - membrane\n",
      "    - intracellular membrane-bounded organelle\n",
      "  MF:\n",
      "    - DNA binding\n",
      "    - protein binding\n",
      "UniProt: {'Swiss-Prot': 'Q7Z591', 'TrEMBL': 'Q64FY1'}\n",
      "\n",
      "Gene: ULK1\n",
      "Symbol: ULK1\n",
      "Name: unc-51 like autophagy activating kinase 1\n",
      "Summary: Enables identical protein binding activity; protein serine/threonine kinase activity; and small GTPase binding activity. Involved in several processes, including autophagosome assembly; positive regulation by symbiont of host autophagy; and protein phosphorylation. Located in autophagosome; cytosol; and phagophore assembly site membrane. Is extrinsic component of autophagosome membrane; extrinsic component of omegasome membrane; and extrinsic component of phagophore assembly site membrane. Part of Atg1/ULK1 kinase complex. [provided by Alliance of Genome Resources, Apr 2022]\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - autophagosome assembly\n",
      "    - autophagosome assembly\n",
      "    - mitophagy\n",
      "    - chromatin remodeling\n",
      "    - protein phosphorylation\n",
      "    - autophagy\n",
      "    - autophagy\n",
      "    - autophagy\n",
      "    - autophagy\n",
      "    - signal transduction\n",
      "    - protein localization\n",
      "    - negative regulation of cell population proliferation\n",
      "    - regulation of autophagy\n",
      "    - positive regulation of autophagy\n",
      "    - positive regulation of autophagy\n",
      "    - positive regulation of autophagy\n",
      "    - regulation of tumor necrosis factor-mediated signaling pathway\n",
      "    - macroautophagy\n",
      "    - macroautophagy\n",
      "    - regulation of macroautophagy\n",
      "    - peptidyl-serine phosphorylation\n",
      "    - neuron projection regeneration\n",
      "    - neuron projection development\n",
      "    - negative regulation of protein-containing complex assembly\n",
      "    - cellular response to nutrient levels\n",
      "    - cellular response to stress\n",
      "    - piecemeal microautophagy of the nucleus\n",
      "    - response to starvation\n",
      "    - response to starvation\n",
      "    - protein autophosphorylation\n",
      "    - negative regulation of collateral sprouting\n",
      "    - axon extension\n",
      "    - reticulophagy\n",
      "    - positive regulation of autophagosome assembly\n",
      "  CC:\n",
      "    - phagophore assembly site\n",
      "    - phagophore assembly site\n",
      "    - autophagosome membrane\n",
      "    - cytoplasm\n",
      "    - cytoplasm\n",
      "    - cytoplasm\n",
      "    - cytoplasm\n",
      "    - mitochondrial outer membrane\n",
      "    - autophagosome\n",
      "    - autophagosome\n",
      "    - endoplasmic reticulum membrane\n",
      "    - cytosol\n",
      "    - cytosol\n",
      "    - cytosol\n",
      "    - axon\n",
      "    - phagophore assembly site membrane\n",
      "    - phagophore assembly site membrane\n",
      "    - recycling endosome\n",
      "    - omegasome membrane\n",
      "    - Atg1/ULK1 kinase complex\n",
      "  MF:\n",
      "    - protein serine/threonine kinase activity\n",
      "    - protein serine/threonine kinase activity\n",
      "    - protein serine/threonine kinase activity\n",
      "    - protein serine/threonine kinase activity\n",
      "    - protein serine/threonine kinase activity\n",
      "    - 3-phosphoinositide-dependent protein kinase activity\n",
      "    - DNA-dependent protein kinase activity\n",
      "    - AMP-activated protein kinase activity\n",
      "    - eukaryotic translation initiation factor 2alpha kinase activity\n",
      "    - ribosomal protein S6 kinase activity\n",
      "    - protein binding\n",
      "    - ATP binding\n",
      "    - small GTPase binding\n",
      "    - histone H3S10 kinase activity\n",
      "    - histone H3T11 kinase activity\n",
      "    - histone H3T6 kinase activity\n",
      "    - histone H2AXS139 kinase activity\n",
      "    - identical protein binding\n",
      "    - histone H3S28 kinase activity\n",
      "    - histone H4S1 kinase activity\n",
      "    - histone H2AS1 kinase activity\n",
      "    - histone H2BS14 kinase activity\n",
      "    - protein-containing complex binding\n",
      "    - GTPase binding\n",
      "    - histone H3T3 kinase activity\n",
      "    - histone H2AS121 kinase activity\n",
      "    - Rho-dependent protein serine/threonine kinase activity\n",
      "    - protein serine kinase activity\n",
      "    - histone H2BS36 kinase activity\n",
      "    - histone H3S57 kinase activity\n",
      "    - histone H3T45 kinase activity\n",
      "    - histone H2AT120 kinase activity\n",
      "UniProt: {'Swiss-Prot': 'O75385'}\n",
      "\n",
      "Gene: PON2\n",
      "Symbol: PON2\n",
      "Name: paraoxonase 2\n",
      "Summary: This gene encodes a member of the paraoxonase gene family, which includes three known members located adjacent to each other on the long arm of chromosome 7. The encoded protein is ubiquitously expressed in human tissues, membrane-bound, and may act as a cellular antioxidant, protecting cells from oxidative stress. Hydrolytic activity against acylhomoserine lactones, important bacterial quorum-sensing mediators, suggests the encoded protein may also play a role in defense responses to pathogenic bacteria. Mutations in this gene may be associated with vascular disease and a number of quantitative phenotypes related to diabetes. Alternatively spliced transcript variants encoding different isoforms have been described. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - response to toxic substance\n",
      "    - lactone catabolic process\n",
      "  CC:\n",
      "    - extracellular region\n",
      "    - plasma membrane\n",
      "  MF:\n",
      "    - arylesterase activity\n",
      "    - protein binding\n",
      "    - protein homodimerization activity\n",
      "    - metal ion binding\n",
      "    - acyl-L-homoserine-lactone lactonohydrolase activity\n",
      "UniProt: {'Swiss-Prot': 'Q15165', 'TrEMBL': ['A0A0J9YXM1', 'G3XAK4', 'A0A0J9YXF2', 'A0A0J9YYJ1', 'A0A0J9YVW3', 'A0A0J9YYF0', 'A0A0J9YXU7', 'A0A2Q2SAK4', 'A0A0J9YYF3']}\n",
      "\n",
      "Gene: LTK\n",
      "Symbol: LTK\n",
      "Name: leukocyte receptor tyrosine kinase\n",
      "Summary: The protein encoded by this gene is a member of the ros/insulin receptor family of tyrosine kinases. Tyrosine-specific phosphorylation of proteins is a key to the control of diverse pathways leading to cell growth and differentiation. Multiple transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Oct 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - protein phosphorylation\n",
      "    - signal transduction\n",
      "    - cell surface receptor protein tyrosine kinase signaling pathway\n",
      "    - cell surface receptor protein tyrosine kinase signaling pathway\n",
      "    - cell surface receptor protein tyrosine kinase signaling pathway\n",
      "    - epidermal growth factor receptor signaling pathway\n",
      "    - cell population proliferation\n",
      "    - insulin receptor signaling pathway\n",
      "    - fibroblast growth factor receptor signaling pathway\n",
      "    - positive regulation of cardiac muscle cell apoptotic process\n",
      "    - positive regulation of neuron projection development\n",
      "    - brain-derived neurotrophic factor receptor signaling pathway\n",
      "    - platelet-derived growth factor receptor-alpha signaling pathway\n",
      "    - platelet-derived growth factor receptor-beta signaling pathway\n",
      "    - vascular endothelial growth factor receptor-1 signaling pathway\n",
      "    - collagen-activated tyrosine kinase receptor signaling pathway\n",
      "    - peptidyl-tyrosine autophosphorylation\n",
      "    - vascular endothelial growth factor signaling pathway\n",
      "    - Kit signaling pathway\n",
      "    - macrophage colony-stimulating factor signaling pathway\n",
      "    - regulation of cell population proliferation\n",
      "    - negative regulation of apoptotic process\n",
      "    - regulation of neuron differentiation\n",
      "    - insulin-like growth factor receptor signaling pathway\n",
      "    - hepatocyte growth factor receptor signaling pathway\n",
      "    - ephrin receptor signaling pathway\n",
      "    - positive regulation of phosphatidylinositol 3-kinase/protein kinase B signal transduction\n",
      "    - cellular response to retinoic acid\n",
      "  CC:\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - membrane\n",
      "    - receptor complex\n",
      "  MF:\n",
      "    - protein kinase activity\n",
      "    - protein tyrosine kinase activity\n",
      "    - transmembrane receptor protein tyrosine kinase activity\n",
      "    - transmembrane receptor protein tyrosine kinase activity\n",
      "    - GPI-linked ephrin receptor activity\n",
      "    - transmembrane-ephrin receptor activity\n",
      "    - epidermal growth factor receptor activity\n",
      "    - fibroblast growth factor receptor activity\n",
      "    - hepatocyte growth factor receptor activity\n",
      "    - insulin receptor activity\n",
      "    - insulin-like growth factor receptor activity\n",
      "    - macrophage colony-stimulating factor receptor activity\n",
      "    - platelet-derived growth factor alpha-receptor activity\n",
      "    - platelet-derived growth factor beta-receptor activity\n",
      "    - stem cell factor receptor activity\n",
      "    - vascular endothelial growth factor receptor activity\n",
      "    - protein binding\n",
      "    - ATP binding\n",
      "    - boss receptor activity\n",
      "    - receptor signaling protein tyrosine kinase activator activity\n",
      "    - placental growth factor receptor activity\n",
      "    - protein tyrosine kinase collagen receptor activity\n",
      "    - brain-derived neurotrophic factor receptor activity\n",
      "UniProt: {'Swiss-Prot': 'P29376', 'TrEMBL': 'H3BVG6'}\n",
      "\n",
      "Gene: SOD1\n",
      "Symbol: SOD1\n",
      "Name: superoxide dismutase 1\n",
      "Summary: The protein encoded by this gene binds copper and zinc ions and is one of two isozymes responsible for destroying free superoxide radicals in the body. The encoded isozyme is a soluble cytoplasmic protein, acting as a homodimer to convert naturally-occuring but harmful superoxide radicals to molecular oxygen and hydrogen peroxide. The other isozyme is a mitochondrial protein. In addition, this protein contains an antimicrobial peptide that displays antibacterial, antifungal, and anti-MRSA activity against E. coli, E. faecalis, S. aureus, S. aureus MRSA LPV+, S. agalactiae, and yeast C. krusei. Mutations in this gene have been implicated as causes of familial amyotrophic lateral sclerosis. Rare transcript variants have been reported for this gene. [provided by RefSeq, Jul 2020].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - response to superoxide\n",
      "    - ovarian follicle development\n",
      "    - positive regulation of cytokine production\n",
      "    - placenta development\n",
      "    - retina homeostasis\n",
      "    - myeloid cell homeostasis\n",
      "    - glutathione metabolic process\n",
      "    - superoxide metabolic process\n",
      "    - superoxide metabolic process\n",
      "    - intracellular iron ion homeostasis\n",
      "    - apoptotic process\n",
      "    - spermatogenesis\n",
      "    - embryo implantation\n",
      "    - embryo implantation\n",
      "    - sensory perception of sound\n",
      "    - locomotory behavior\n",
      "    - anterograde axonal transport\n",
      "    - retrograde axonal transport\n",
      "    - regulation of blood pressure\n",
      "    - determination of adult lifespan\n",
      "    - response to heat\n",
      "    - response to xenobiotic stimulus\n",
      "    - gene expression\n",
      "    - transmission of nerve impulse\n",
      "    - neuronal action potential\n",
      "    - removal of superoxide radicals\n",
      "    - removal of superoxide radicals\n",
      "    - removal of superoxide radicals\n",
      "    - peripheral nervous system myelin maintenance\n",
      "    - positive regulation of superoxide anion generation\n",
      "    - regulation of T cell differentiation in thymus\n",
      "    - ectopic germ cell programmed cell death\n",
      "    - regulation of multicellular organism growth\n",
      "    - response to hydrogen peroxide\n",
      "    - superoxide anion generation\n",
      "    - positive regulation of apoptotic process\n",
      "    - regulation of GTPase activity\n",
      "    - positive regulation of MAPK cascade\n",
      "    - negative regulation of neuron apoptotic process\n",
      "    - response to ethanol\n",
      "    - regulation of organ growth\n",
      "    - muscle cell cellular homeostasis\n",
      "    - thymus development\n",
      "    - response to axon injury\n",
      "    - hydrogen peroxide biosynthetic process\n",
      "    - hydrogen peroxide biosynthetic process\n",
      "    - negative regulation of inflammatory response\n",
      "    - positive regulation of phagocytosis\n",
      "    - negative regulation of developmental process\n",
      "    - regulation of mitochondrial membrane potential\n",
      "    - heart contraction\n",
      "    - neurofilament cytoskeleton organization\n",
      "    - relaxation of vascular associated smooth muscle\n",
      "    - auditory receptor cell stereocilium organization\n",
      "    - reactive oxygen species metabolic process\n",
      "    - action potential initiation\n",
      "    - positive regulation of oxidative stress-induced intrinsic apoptotic signaling pathway\n",
      "    - negative regulation of reproductive process\n",
      "  CC:\n",
      "    - extracellular region\n",
      "    - extracellular space\n",
      "    - nucleus\n",
      "    - nucleus\n",
      "    - nucleus\n",
      "    - nucleoplasm\n",
      "    - cytoplasm\n",
      "    - mitochondrion\n",
      "    - mitochondrion\n",
      "    - mitochondrion\n",
      "    - mitochondrial intermembrane space\n",
      "    - mitochondrial matrix\n",
      "    - peroxisome\n",
      "    - peroxisome\n",
      "    - peroxisome\n",
      "    - cytosol\n",
      "    - cytosol\n",
      "    - cytosol\n",
      "    - plasma membrane\n",
      "    - cytoplasmic vesicle\n",
      "    - dendrite cytoplasm\n",
      "    - protein-containing complex\n",
      "    - neuronal cell body\n",
      "    - extracellular exosome\n",
      "    - axon cytoplasm\n",
      "  MF:\n",
      "    - superoxide dismutase activity\n",
      "    - superoxide dismutase activity\n",
      "    - copper ion binding\n",
      "    - copper ion binding\n",
      "    - protein binding\n",
      "    - zinc ion binding\n",
      "    - protein phosphatase 2B binding\n",
      "    - small GTPase binding\n",
      "    - identical protein binding\n",
      "    - protein homodimerization activity\n",
      "    - protein-folding chaperone binding\n",
      "UniProt: {'Swiss-Prot': 'P00441', 'TrEMBL': ['H7BYH4', 'V9HWC9']}\n",
      "\n",
      "Gene: EGLN2\n",
      "Symbol: EGLN2\n",
      "Name: egl-9 family hypoxia inducible factor 2\n",
      "Summary: The hypoxia inducible factor (HIF) is a transcriptional complex that is involved in oxygen homeostasis. At normal oxygen levels, the alpha subunit of HIF is targeted for degration by prolyl hydroxylation. This gene encodes an enzyme responsible for this post-translational modification. Alternative splicing results in multiple transcript variants. Read-through transcription also exists between this gene and the upstream RAB4B (RAB4B, member RAS oncogene family) gene. [provided by RefSeq, Feb 2011].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - regulation of cell growth\n",
      "    - response to hypoxia\n",
      "    - peptidyl-proline hydroxylation to 4-hydroxy-L-proline\n",
      "    - estrogen receptor signaling pathway\n",
      "    - regulation of neuron apoptotic process\n",
      "    - regulation of neuron apoptotic process\n",
      "    - cell redox homeostasis\n",
      "    - positive regulation of protein catabolic process\n",
      "    - cellular response to hypoxia\n",
      "  CC:\n",
      "    - nucleus\n",
      "    - nucleus\n",
      "    - nucleoplasm\n",
      "    - nucleoplasm\n",
      "    - cytoplasm\n",
      "  MF:\n",
      "    - protein binding\n",
      "    - ferrous iron binding\n",
      "    - ferrous iron binding\n",
      "    - ferrous iron binding\n",
      "    - 2-oxoglutarate-dependent dioxygenase activity\n",
      "    - oxygen sensor activity\n",
      "    - L-ascorbic acid binding\n",
      "    - peptidyl-proline 4-dioxygenase activity\n",
      "    - peptidyl-proline 4-dioxygenase activity\n",
      "    - hypoxia-inducible factor-proline dioxygenase activity\n",
      "UniProt: {'Swiss-Prot': 'Q96KS0', 'TrEMBL': ['M0R110', 'M0R0Z6', 'M0R035', 'M0R2X9', 'M0R1W4', 'M0QXR0', 'M0QXM8', 'M0R350', 'A0A0C4DGR4', 'M0R1L0', 'M0R1A3']}\n",
      "\n",
      "Gene: DLAT\n",
      "Symbol: DLAT\n",
      "Name: dihydrolipoamide S-acetyltransferase\n",
      "Summary: This gene encodes component E2 of the multi-enzyme pyruvate dehydrogenase complex (PDC). PDC resides in the inner mitochondrial membrane and catalyzes the conversion of pyruvate to acetyl coenzyme A. The protein product of this gene, dihydrolipoamide acetyltransferase, accepts acetyl groups formed by the oxidative decarboxylation of pyruvate and transfers them to coenzyme A. Dihydrolipoamide acetyltransferase is the antigen for antimitochondrial antibodies. These autoantibodies are present in nearly 95% of patients with the autoimmune liver disease primary biliary cirrhosis (PBC). In PBC, activated T lymphocytes attack and destroy epithelial cells in the bile duct where this protein is abnormally distributed and overexpressed. PBC enventually leads to cirrhosis and liver failure. Mutations in this gene are also a cause of pyruvate dehydrogenase E2 deficiency which causes primary lactic acidosis in infancy and early childhood.[provided by RefSeq, Oct 2009].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - glucose metabolic process\n",
      "    - pyruvate decarboxylation to acetyl-CoA\n",
      "    - pyruvate decarboxylation to acetyl-CoA\n",
      "    - pyruvate decarboxylation to acetyl-CoA\n",
      "    - pyruvate decarboxylation to acetyl-CoA\n",
      "    - tricarboxylic acid cycle\n",
      "    - pyruvate catabolic process\n",
      "  CC:\n",
      "    - mitochondrion\n",
      "    - mitochondrion\n",
      "    - mitochondrion\n",
      "    - mitochondrion\n",
      "    - mitochondrial matrix\n",
      "    - pyruvate dehydrogenase complex\n",
      "    - pyruvate dehydrogenase complex\n",
      "    - pyruvate dehydrogenase complex\n",
      "    - pyruvate dehydrogenase complex\n",
      "    - pyruvate dehydrogenase complex\n",
      "  MF:\n",
      "    - dihydrolipoyllysine-residue acetyltransferase activity\n",
      "    - dihydrolipoyllysine-residue acetyltransferase activity\n",
      "    - dihydrolipoyllysine-residue acetyltransferase activity\n",
      "    - dihydrolipoyllysine-residue acetyltransferase activity\n",
      "    - protein binding\n",
      "    - pyruvate dehydrogenase (NAD+) activity\n",
      "    - pyruvate dehydrogenase (NAD+) activity\n",
      "    - identical protein binding\n",
      "UniProt: {'Swiss-Prot': 'P10515', 'TrEMBL': ['A0A7P0Z423', 'Q86YI5', 'A0A7P0Z4G4', 'A0A7P0T9N8', 'A0A7P0TAG1', 'H0YDD4', 'E9PKC7', 'A0A7P0TA47', 'A0A7P0TBE2', 'E9PEJ4', 'A0A7P0TAX2', 'A0A7P0TBK2', 'A0A7P0T997', 'A0A7P0Z459']}\n",
      "\n",
      "Gene: LPAR2\n",
      "Symbol: LPAR2\n",
      "Name: lysophosphatidic acid receptor 2\n",
      "Summary: This gene encodes a member of family I of the G protein-coupled receptors, as well as the EDG family of proteins. This protein functions as a lysophosphatidic acid (LPA) receptor and contributes to Ca2+ mobilization, a critical cellular response to LPA in cells, through association with Gi and Gq proteins. An alternative splice variant has been described but its full length sequence has not been determined. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - G protein-coupled receptor signaling pathway\n",
      "    - adenylate cyclase-activating G protein-coupled receptor signaling pathway\n",
      "    - positive regulation of cytosolic calcium ion concentration\n",
      "    - regulation of metabolic process\n",
      "    - positive regulation of Rho protein signal transduction\n",
      "    - positive regulation of MAPK cascade\n",
      "  CC:\n",
      "    - cytoplasm\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - cell surface\n",
      "    - endocytic vesicle\n",
      "    - presynaptic active zone membrane\n",
      "    - glutamatergic synapse\n",
      "  MF:\n",
      "    - G protein-coupled receptor activity\n",
      "    - protein binding\n",
      "    - lipid binding\n",
      "    - PDZ domain binding\n",
      "    - lysophosphatidic acid receptor activity\n",
      "\n",
      "Gene: CAMK2D\n",
      "Symbol: CAMK2D\n",
      "Name: calcium/calmodulin dependent protein kinase II delta\n",
      "Summary: The product of this gene belongs to the serine/threonine protein kinase family and to the Ca(2+)/calmodulin-dependent protein kinase subfamily. Calcium signaling is crucial for several aspects of plasticity at glutamatergic synapses. In mammalian cells, the enzyme is composed of four different chains: alpha, beta, gamma, and delta. The product of this gene is a delta chain. Alternative splicing results in multiple transcript variants encoding distinct isoforms. Distinct isoforms of this chain have different expression patterns.[provided by RefSeq, Nov 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - regulation of cell growth\n",
      "    - regulation of the force of heart contraction\n",
      "    - regulation of membrane depolarization\n",
      "    - regulation of transcription by RNA polymerase II\n",
      "    - protein phosphorylation\n",
      "    - regulation of heart contraction\n",
      "    - positive regulation of cardiac muscle hypertrophy\n",
      "    - regulation of cell communication by electrical coupling\n",
      "    - positive regulation of cardiac muscle cell apoptotic process\n",
      "    - regulation of release of sequestered calcium ion into cytosol by sarcoplasmic reticulum\n",
      "    - regulation of cardiac muscle contraction by regulation of the release of sequestered calcium ion\n",
      "    - regulation of cardiac muscle contraction by regulation of the release of sequestered calcium ion\n",
      "    - peptidyl-serine phosphorylation\n",
      "    - peptidyl-threonine phosphorylation\n",
      "    - endoplasmic reticulum calcium ion homeostasis\n",
      "    - protein autophosphorylation\n",
      "    - regulation of neuronal synaptic plasticity\n",
      "    - relaxation of cardiac muscle\n",
      "    - long-term synaptic potentiation\n",
      "    - regulation of ryanodine-sensitive calcium-release channel activity\n",
      "    - regulation of cellular localization\n",
      "    - cellular response to calcium ion\n",
      "    - cardiac muscle cell contraction\n",
      "    - regulation of heart rate by cardiac conduction\n",
      "    - regulation of cardiac muscle cell action potential\n",
      "    - regulation of cardiac muscle cell action potential involved in regulation of contraction\n",
      "    - regulation of cell communication by electrical coupling involved in cardiac conduction\n",
      "    - regulation of relaxation of cardiac muscle\n",
      "    - negative regulation of sodium ion transmembrane transport\n",
      "    - regulation of calcium ion transmembrane transport via high voltage-gated calcium channel\n",
      "    - regulation of protein localization to plasma membrane\n",
      "    - negative regulation of sodium ion transmembrane transporter activity\n",
      "  CC:\n",
      "    - nucleus\n",
      "    - nucleoplasm\n",
      "    - cytoplasm\n",
      "    - cytoplasm\n",
      "    - cytosol\n",
      "    - calcium- and calmodulin-dependent protein kinase complex\n",
      "    - postsynaptic density\n",
      "    - membrane\n",
      "    - endocytic vesicle membrane\n",
      "    - sarcoplasmic reticulum membrane\n",
      "    - sarcolemma\n",
      "    - neuron projection\n",
      "  MF:\n",
      "    - protein serine/threonine kinase activity\n",
      "    - calcium/calmodulin-dependent protein kinase activity\n",
      "    - calcium/calmodulin-dependent protein kinase activity\n",
      "    - protein binding\n",
      "    - calmodulin binding\n",
      "    - calmodulin binding\n",
      "    - ATP binding\n",
      "    - sodium channel inhibitor activity\n",
      "    - titin binding\n",
      "    - identical protein binding\n",
      "    - protein homodimerization activity\n",
      "    - transmembrane transporter binding\n",
      "    - transmembrane transporter binding\n",
      "    - protein serine kinase activity\n",
      "UniProt: {'Swiss-Prot': 'Q13557', 'TrEMBL': ['E9PBG7', 'D6R938', 'A0A8V8TPJ8', 'A0A8V8TN60', 'A0A8V8TNA1', 'A0A8V8TN88', 'A0A994J5B0', 'A0A8V8TPY4', 'A0A8V8TN65', 'A0A8V8TNA7', 'A0A994J516', 'E9PF82', 'H0Y9C2', 'A0A8V8TND4', 'A0A8V8TQ10', 'A0A8V8TPM4', 'A0A8V8TPK3', 'A0A8V8TNN9', 'A0A8V8TNR6', 'H0Y9J2', 'A0A8V8TNN4', 'A0A994J7X8']}\n",
      "\n",
      "Gene: IMPDH2\n",
      "Symbol: IMPDH2\n",
      "Name: inosine monophosphate dehydrogenase 2\n",
      "Summary: This gene encodes the rate-limiting enzyme in the de novo guanine nucleotide biosynthesis. It is thus involved in maintaining cellular guanine deoxy- and ribonucleotide pools needed for DNA and RNA synthesis. The encoded protein catalyzes the NAD-dependent oxidation of inosine-5'-monophosphate into xanthine-5'-monophosphate, which is then converted into guanosine-5'-monophosphate. This gene is up-regulated in some neoplasms, suggesting it may play a role in malignant transformation. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - GMP biosynthetic process\n",
      "    - GTP biosynthetic process\n",
      "    - GTP biosynthetic process\n",
      "    - circadian rhythm\n",
      "    - lymphocyte proliferation\n",
      "    - cellular response to interleukin-4\n",
      "    - 'de novo' XMP biosynthetic process\n",
      "  CC:\n",
      "    - extracellular region\n",
      "    - nucleus\n",
      "    - cytoplasm\n",
      "    - cytoplasm\n",
      "    - peroxisomal membrane\n",
      "    - cytosol\n",
      "    - cytosol\n",
      "    - membrane\n",
      "    - secretory granule lumen\n",
      "    - extracellular exosome\n",
      "    - ficolin-1-rich granule lumen\n",
      "  MF:\n",
      "    - nucleotide binding\n",
      "    - DNA binding\n",
      "    - RNA binding\n",
      "    - IMP dehydrogenase activity\n",
      "    - IMP dehydrogenase activity\n",
      "    - protein binding\n",
      "    - metal ion binding\n",
      "UniProt: {'Swiss-Prot': 'P12268', 'TrEMBL': ['H0Y4R1', 'A0A384N6C2', 'E7ETK5', 'A0A7I2V337', 'A0A7I2YQK5', 'A0A7I2V5N6', 'A0A7I2V2T3']}\n",
      "\n",
      "Gene: SCN4A\n",
      "Symbol: SCN4A\n",
      "Name: sodium voltage-gated channel alpha subunit 4\n",
      "Summary: Voltage-gated sodium channels are transmembrane glycoprotein complexes composed of a large alpha subunit with 24 transmembrane domains and one or more regulatory beta subunits. They are responsible for the generation and propagation of action potentials in neurons and muscle. This gene encodes one member of the sodium channel alpha subunit gene family. It is expressed in skeletal muscle, and mutations in this gene have been linked to several myotonia and periodic paralysis disorders. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - sodium ion transport\n",
      "    - muscle contraction\n",
      "    - sodium ion transmembrane transport\n",
      "    - sodium ion transmembrane transport\n",
      "    - cardiac muscle cell action potential involved in contraction\n",
      "    - regulation of skeletal muscle contraction by action potential\n",
      "  CC:\n",
      "    - voltage-gated sodium channel complex\n",
      "    - voltage-gated sodium channel complex\n",
      "    - plasma membrane\n",
      "  MF:\n",
      "    - voltage-gated sodium channel activity\n",
      "    - voltage-gated sodium channel activity\n",
      "    - protein binding\n",
      "UniProt: {'Swiss-Prot': 'P35499'}\n",
      "\n",
      "Gene: YAF2\n",
      "Symbol: YAF2\n",
      "Name: YY1 associated factor 2\n",
      "Summary: This gene encodes a zinc finger containing protein that functions in the regulation of transcription. This protein was identified as an interacting partner of transcriptional repressor protein Yy1, and also interacts with other transcriptional regulators, including Myc and Polycomb. This protein can promote proteolysis of Yy1. Multiple alternatively spliced transcript variants have been found. [provided by RefSeq, Feb 2016].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - regulation of DNA-templated transcription\n",
      "    - negative regulation of DNA-templated transcription\n",
      "    - positive regulation of DNA-templated transcription\n",
      "  CC:\n",
      "    - nucleus\n",
      "    - nucleus\n",
      "    - nucleoplasm\n",
      "    - nucleoplasm\n",
      "    - cytosol\n",
      "  MF:\n",
      "    - DNA binding\n",
      "    - transcription coregulator activity\n",
      "    - transcription coactivator activity\n",
      "    - transcription corepressor activity\n",
      "    - protein binding\n",
      "    - zinc ion binding\n",
      "UniProt: {'Swiss-Prot': 'Q8IY57', 'TrEMBL': ['B4DJQ9', 'G3V212', 'A0A0B4J255', 'G3V1X5', 'G3V219', 'F8W0B9', 'F8VZR2', 'F8VRE5']}\n",
      "\n",
      "Gene: CACNA1I\n",
      "Symbol: CACNA1I\n",
      "Name: calcium voltage-gated channel subunit alpha1 I\n",
      "Summary: This gene encodes the pore-forming alpha subunit of a voltage gated calcium channel. The encoded protein is a member of a subfamily of calcium channels referred to as is a low voltage-activated, T-type, calcium channel. The channel encoded by this protein is characterized by a slower activation and inactivation compared to other T-type calcium channels. This protein may be involved in calcium signaling in neurons. Alternate splicing results in multiple transcript variants. [provided by RefSeq, Oct 2011].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - signal transduction\n",
      "    - neuronal action potential\n",
      "    - sleep\n",
      "    - calcium ion transmembrane transport\n",
      "    - calcium ion import across plasma membrane\n",
      "  CC:\n",
      "    - plasma membrane\n",
      "    - voltage-gated calcium channel complex\n",
      "  MF:\n",
      "    - voltage-gated calcium channel activity\n",
      "    - voltage-gated calcium channel activity\n",
      "    - protein binding\n",
      "    - high voltage-gated calcium channel activity\n",
      "UniProt: {'Swiss-Prot': 'Q9P0X4'}\n",
      "\n",
      "Gene: VHL\n",
      "Symbol: VHL\n",
      "Name: von Hippel-Lindau tumor suppressor\n",
      "Summary: This gene encodes a component of a ubiquitination complex. The encoded protein is involved in the ubiquitination and degradation of hypoxia-inducible-factor (HIF), which is a transcription factor that plays a central role in the regulation of gene expression by oxygen. In addition to oxygen-related gene expression, this protein plays a role in many other cellular processes including cilia formation, cytokine signaling, regulation of senescence, and formation of the extracellular matrix. Variants of this gene are associated with von Hippel-Lindau syndrome, pheochromocytoma, erythrocytosis, renal cell carcinoma, and cerebellar hemangioblastoma. [provided by RefSeq, Jun 2022].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - negative regulation of transcription by RNA polymerase II\n",
      "    - cell morphogenesis\n",
      "    - regulation of DNA-templated transcription\n",
      "    - proteolysis\n",
      "    - negative regulation of cell population proliferation\n",
      "    - negative regulation of cell population proliferation\n",
      "    - negative regulation of signal transduction\n",
      "    - regulation of gene expression\n",
      "    - regulation of gene expression\n",
      "    - regulation of gene expression\n",
      "    - negative regulation of autophagy\n",
      "    - negative regulation of gene expression\n",
      "    - protein ubiquitination\n",
      "    - protein ubiquitination\n",
      "    - protein ubiquitination\n",
      "    - negative regulation of transcription elongation by RNA polymerase II\n",
      "    - negative regulation of apoptotic process\n",
      "    - proteasome-mediated ubiquitin-dependent protein catabolic process\n",
      "    - positive regulation of cell differentiation\n",
      "    - positive regulation of DNA-templated transcription\n",
      "    - negative regulation of receptor signaling pathway via JAK-STAT\n",
      "    - protein stabilization\n",
      "    - cellular response to hypoxia\n",
      "    - regulation of cellular response to hypoxia\n",
      "    - negative regulation of TORC1 signaling\n",
      "    - amyloid fibril formation\n",
      "    - amyloid fibril formation\n",
      "  CC:\n",
      "    - nucleus\n",
      "    - nucleoplasm\n",
      "    - nucleoplasm\n",
      "    - mitochondrion\n",
      "    - endoplasmic reticulum\n",
      "    - endoplasmic reticulum\n",
      "    - cytosol\n",
      "    - plasma membrane\n",
      "    - cilium\n",
      "    - microtubule cytoskeleton\n",
      "    - intracellular membraneless organelle\n",
      "  MF:\n",
      "    - transcription elongation factor activity\n",
      "    - ubiquitin-protein transferase activity\n",
      "    - protein binding\n",
      "    - enzyme binding\n",
      "    - molecular adaptor activity\n",
      "    - protein serine/threonine kinase binding\n",
      "    - DNA-binding transcription factor binding\n",
      "    - ubiquitin-like ligase-substrate adaptor activity\n",
      "UniProt: {'Swiss-Prot': 'P40337', 'TrEMBL': ['A0A024R2F2', 'A0A0S2Z4K1']}\n",
      "\n",
      "Gene: COMT\n",
      "Symbol: COMT\n",
      "Name: catechol-O-methyltransferase\n",
      "Summary: Catechol-O-methyltransferase catalyzes the transfer of a methyl group from S-adenosylmethionine to catecholamines, including the neurotransmitters dopamine, epinephrine, and norepinephrine. This O-methylation results in one of the major degradative pathways of the catecholamine transmitters. In addition to its role in the metabolism of endogenous substances, COMT is important in the metabolism of catechol drugs used in the treatment of hypertension, asthma, and Parkinson disease. COMT is found in two forms in tissues, a soluble form (S-COMT) and a membrane-bound form (MB-COMT). The differences between S-COMT and MB-COMT reside within the N-termini. Several transcript variants are formed through the use of alternative translation initiation sites and promoters. [provided by RefSeq, Sep 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - behavioral fear response\n",
      "    - response to hypoxia\n",
      "    - synaptic transmission, dopaminergic\n",
      "    - startle response\n",
      "    - response to amphetamine\n",
      "    - renin secretion into blood stream\n",
      "    - glycogen metabolic process\n",
      "    - prostaglandin metabolic process\n",
      "    - response to oxidative stress\n",
      "    - memory\n",
      "    - visual learning\n",
      "    - response to xenobiotic stimulus\n",
      "    - response to wounding\n",
      "    - response to toxic substance\n",
      "    - gene expression\n",
      "    - dopamine secretion\n",
      "    - cellular response to phosphate starvation\n",
      "    - cerebellar cortex morphogenesis\n",
      "    - response to food\n",
      "    - methylation\n",
      "    - methylation\n",
      "    - developmental process\n",
      "    - glomerulus development\n",
      "    - cholesterol efflux\n",
      "    - response to cytokine\n",
      "    - multicellular organism growth\n",
      "    - exploration behavior\n",
      "    - renal sodium excretion\n",
      "    - norepinephrine metabolic process\n",
      "    - dopamine metabolic process\n",
      "    - dopamine catabolic process\n",
      "    - catecholamine catabolic process\n",
      "    - catecholamine catabolic process\n",
      "    - habituation\n",
      "    - norepinephrine secretion\n",
      "    - detection of temperature stimulus involved in sensory perception of pain\n",
      "    - response to corticosterone\n",
      "    - artery development\n",
      "    - cellular response to cocaine\n",
      "    - mastication\n",
      "    - renal albumin absorption\n",
      "    - renal filtration\n",
      "    - response to salt\n",
      "    - response to dopamine\n",
      "    - response to angiotensin\n",
      "  CC:\n",
      "    - cytosol\n",
      "    - cytosol\n",
      "    - plasma membrane\n",
      "    - membrane\n",
      "    - membrane\n",
      "    - axon\n",
      "    - dendrite\n",
      "    - intracellular membrane-bounded organelle\n",
      "    - synapse\n",
      "    - extracellular exosome\n",
      "  MF:\n",
      "    - magnesium ion binding\n",
      "    - protein binding\n",
      "    - methyltransferase activity\n",
      "    - O-methyltransferase activity\n",
      "    - catechol O-methyltransferase activity\n",
      "    - catechol O-methyltransferase activity\n",
      "    - catechol O-methyltransferase activity\n",
      "UniProt: {'Swiss-Prot': 'P21964', 'TrEMBL': ['E7EMS6', 'A0A140VJG8', 'F8WBW9', 'A0A7I2V3G7', 'A0A7I2V370', 'E7EUU8', 'H7BZ45']}\n",
      "\n",
      "Gene: SMN2\n",
      "Symbol: SMN2\n",
      "Name: survival of motor neuron 2, centromeric\n",
      "Summary: This gene is part of a 500 kb inverted duplication on chromosome 5q13. This duplicated region contains at least four genes and repetitive elements which make it prone to rearrangements and deletions. The repetitiveness and complexity of the sequence have also caused difficulty in determining the organization of this genomic region. The telomeric and centromeric copies of this gene are nearly identical and encode the same protein. While mutations in the telomeric copy are associated with spinal muscular atrophy, mutations in this gene, the centromeric copy, do not lead to disease. This gene may be a modifier of disease caused by mutation in the telomeric copy. The critical sequence difference between the two genes is a single nucleotide in exon 7, which is thought to be an exon splice enhancer. Note that the nine exons of both the telomeric and centromeric copies are designated historically as exon 1, 2a, 2b, and 3-8. It is thought that gene conversion events may involve the two genes, leading to varying copy numbers of each gene. The full length protein encoded by this gene localizes to both the cytoplasm and the nucleus. Within the nucleus, the protein localizes to subnuclear bodies called gems which are found near coiled bodies containing high concentrations of small ribonucleoproteins (snRNPs). This protein forms heteromeric complexes with proteins such as SIP1 and GEMIN4, and also interacts with several proteins known to be involved in the biogenesis of snRNPs, such as hnRNP U protein and the small nucleolar RNA binding protein. Four transcript variants encoding distinct isoforms have been described. [provided by RefSeq, Sep 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - spliceosomal complex assembly\n",
      "    - spliceosomal complex assembly\n",
      "    - spliceosomal snRNP assembly\n",
      "    - spliceosomal snRNP assembly\n",
      "    - spliceosomal snRNP assembly\n",
      "    - DNA-templated transcription termination\n",
      "    - nervous system development\n",
      "  CC:\n",
      "    - nucleus\n",
      "    - nucleus\n",
      "    - nucleoplasm\n",
      "    - nucleoplasm\n",
      "    - cytoplasm\n",
      "    - cytosol\n",
      "    - cytosol\n",
      "    - Cajal body\n",
      "    - Cajal body\n",
      "    - nuclear body\n",
      "    - Z disc\n",
      "    - axon\n",
      "    - SMN complex\n",
      "    - SMN complex\n",
      "    - SMN complex\n",
      "    - SMN-Sm protein complex\n",
      "    - cytoplasmic ribonucleoprotein granule\n",
      "    - neuron projection\n",
      "    - perikaryon\n",
      "    - Gemini of Cajal bodies\n",
      "    - Gemini of Cajal bodies\n",
      "  MF:\n",
      "    - RNA binding\n",
      "    - protein binding\n",
      "    - identical protein binding\n",
      "UniProt: {'Swiss-Prot': 'Q16637', 'TrEMBL': ['E7EQZ4', 'B4DP61', 'H0YBZ9', 'U3KPX7', 'A0A0G2JRY6', 'A0A0G2JQN8', 'A0A0G2JRX5']}\n",
      "\n",
      "Gene: SMN1\n",
      "Symbol: SMN1\n",
      "Name: survival of motor neuron 1, telomeric\n",
      "Summary: This gene is part of a 500 kb inverted duplication on chromosome 5q13. This duplicated region contains at least four genes and repetitive elements which make it prone to rearrangements and deletions. The repetitiveness and complexity of the sequence have also caused difficulty in determining the organization of this genomic region. The telomeric and centromeric copies of this gene are nearly identical and encode the same protein. However, mutations in this gene, the telomeric copy, are associated with spinal muscular atrophy; mutations in the centromeric copy do not lead to disease. The centromeric copy may be a modifier of disease caused by mutation in the telomeric copy. The critical sequence difference between the two genes is a single nucleotide in exon 7, which is thought to be an exon splice enhancer. Note that the nine exons of both the telomeric and centromeric copies are designated historically as exon 1, 2a, 2b, and 3-8. It is thought that gene conversion events may involve the two genes, leading to varying copy numbers of each gene. The protein encoded by this gene localizes to both the cytoplasm and the nucleus. Within the nucleus, the protein localizes to subnuclear bodies called gems which are found near coiled bodies containing high concentrations of small ribonucleoproteins (snRNPs). This protein forms heteromeric complexes with proteins such as SIP1 and GEMIN4, and also interacts with several proteins known to be involved in the biogenesis of snRNPs, such as hnRNP U protein and the small nucleolar RNA binding protein. Multiple transcript variants encoding distinct isoforms have been described. [provided by RefSeq, Jul 2014].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - spliceosomal complex assembly\n",
      "    - spliceosomal complex assembly\n",
      "    - spliceosomal snRNP assembly\n",
      "    - spliceosomal snRNP assembly\n",
      "    - spliceosomal snRNP assembly\n",
      "    - DNA-templated transcription termination\n",
      "    - nervous system development\n",
      "  CC:\n",
      "    - nucleus\n",
      "    - nucleus\n",
      "    - nucleoplasm\n",
      "    - nucleoplasm\n",
      "    - cytoplasm\n",
      "    - cytosol\n",
      "    - cytosol\n",
      "    - Cajal body\n",
      "    - Cajal body\n",
      "    - nuclear body\n",
      "    - Z disc\n",
      "    - axon\n",
      "    - SMN complex\n",
      "    - SMN complex\n",
      "    - SMN complex\n",
      "    - SMN-Sm protein complex\n",
      "    - cytoplasmic ribonucleoprotein granule\n",
      "    - neuron projection\n",
      "    - perikaryon\n",
      "    - Gemini of Cajal bodies\n",
      "    - Gemini of Cajal bodies\n",
      "  MF:\n",
      "    - RNA binding\n",
      "    - protein binding\n",
      "    - identical protein binding\n",
      "UniProt: {'Swiss-Prot': 'Q16637', 'TrEMBL': ['E7EQZ4', 'B4DP61', 'H0YBZ9', 'U3KPX7', 'A0A0G2JRY6', 'A0A0G2JQN8', 'A0A0G2JRX5']}\n",
      "\n",
      "Gene: GRK4\n",
      "Symbol: GRK4\n",
      "Name: G protein-coupled receptor kinase 4\n",
      "Summary: This gene encodes a member of the guanine nucleotide-binding protein (G protein)-coupled receptor kinase subfamily of the Ser/Thr protein kinase family. The protein phosphorylates the activated forms of G protein-coupled receptors thus initiating its deactivation. This gene has been linked to both genetic and acquired hypertension. Several transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Nov 2013].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - signal transduction\n",
      "    - regulation of G protein-coupled receptor signaling pathway\n",
      "    - regulation of signal transduction\n",
      "    - regulation of opsin-mediated signaling pathway\n",
      "    - receptor internalization\n",
      "  CC:\n",
      "    - cytoplasm\n",
      "    - cytosol\n",
      "    - plasma membrane\n",
      "    - cell cortex\n",
      "    - photoreceptor disc membrane\n",
      "  MF:\n",
      "    - protein kinase activity\n",
      "    - protein binding\n",
      "    - ATP binding\n",
      "    - rhodopsin kinase activity\n",
      "UniProt: {'Swiss-Prot': 'P32298', 'TrEMBL': 'D6REW5'}\n",
      "\n",
      "Gene: FGA\n",
      "Symbol: FGA\n",
      "Name: fibrinogen alpha chain\n",
      "Summary: This gene encodes the alpha subunit of the coagulation factor fibrinogen, which is a component of the blood clot. Following vascular injury, the encoded preproprotein is proteolytically processed by thrombin during the conversion of fibrinogen to fibrin. Mutations in this gene lead to several disorders, including dysfibrinogenemia, hypofibrinogenemia, afibrinogenemia and renal amyloidosis. Alternative splicing results in multiple transcript variants, at least one of which encodes an isoform that undergoes proteolytic processing. [provided by RefSeq, Jan 2016].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - adaptive immune response\n",
      "    - cell-matrix adhesion\n",
      "    - plasminogen activation\n",
      "    - positive regulation of heterotypic cell-cell adhesion\n",
      "    - positive regulation of heterotypic cell-cell adhesion\n",
      "    - fibrinolysis\n",
      "    - fibrinolysis\n",
      "    - induction of bacterial agglutination\n",
      "    - innate immune response\n",
      "    - positive regulation of vasoconstriction\n",
      "    - positive regulation of exocytosis\n",
      "    - positive regulation of protein secretion\n",
      "    - protein polymerization\n",
      "    - response to calcium ion\n",
      "    - protein-containing complex assembly\n",
      "    - protein-containing complex assembly\n",
      "    - positive regulation of ERK1 and ERK2 cascade\n",
      "    - platelet aggregation\n",
      "    - platelet aggregation\n",
      "    - platelet aggregation\n",
      "    - blood coagulation, common pathway\n",
      "    - blood coagulation, common pathway\n",
      "    - blood coagulation, fibrin clot formation\n",
      "    - positive regulation of peptide hormone secretion\n",
      "    - positive regulation of substrate adhesion-dependent cell spreading\n",
      "    - negative regulation of extrinsic apoptotic signaling pathway via death domain receptors\n",
      "    - negative regulation of blood coagulation, common pathway\n",
      "    - negative regulation of endothelial cell apoptotic process\n",
      "  CC:\n",
      "    - extracellular region\n",
      "    - extracellular region\n",
      "    - fibrinogen complex\n",
      "    - fibrinogen complex\n",
      "    - extracellular space\n",
      "    - endoplasmic reticulum\n",
      "    - endoplasmic reticulum lumen\n",
      "    - plasma membrane\n",
      "    - external side of plasma membrane\n",
      "    - cell surface\n",
      "    - platelet alpha granule\n",
      "    - platelet alpha granule lumen\n",
      "    - collagen-containing extracellular matrix\n",
      "    - extracellular exosome\n",
      "    - blood microparticle\n",
      "    - extracellular vesicle\n",
      "  MF:\n",
      "    - signaling receptor binding\n",
      "    - structural molecule activity\n",
      "    - structural molecule activity\n",
      "    - extracellular matrix structural constituent\n",
      "    - extracellular matrix structural constituent\n",
      "    - protein binding\n",
      "    - protein-macromolecule adaptor activity\n",
      "    - metal ion binding\n",
      "    - cell adhesion molecule binding\n",
      "UniProt: {'Swiss-Prot': 'P02671'}\n",
      "\n",
      "Gene: FASN\n",
      "Symbol: FASN\n",
      "Name: fatty acid synthase\n",
      "Summary: The enzyme encoded by this gene is a multifunctional protein. Its main function is to catalyze the synthesis of palmitate from acetyl-CoA and malonyl-CoA, in the presence of NADPH, into long-chain saturated fatty acids. In some cancer cell lines, this protein has been found to be fused with estrogen receptor-alpha (ER-alpha), in which the N-terminus of FAS is fused in-frame with the C-terminus of ER-alpha. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - osteoblast differentiation\n",
      "    - glandular epithelial cell development\n",
      "    - fatty acid metabolic process\n",
      "    - fatty acid biosynthetic process\n",
      "    - fatty acid biosynthetic process\n",
      "    - inflammatory response\n",
      "    - ether lipid biosynthetic process\n",
      "    - neutrophil differentiation\n",
      "    - monocyte differentiation\n",
      "    - mammary gland development\n",
      "    - modulation by host of viral process\n",
      "    - fatty-acyl-CoA biosynthetic process\n",
      "    - cellular response to interleukin-4\n",
      "    - establishment of endothelial intestinal barrier\n",
      "  CC:\n",
      "    - cytoplasm\n",
      "    - Golgi apparatus\n",
      "    - cytosol\n",
      "    - cytosol\n",
      "    - plasma membrane\n",
      "    - membrane\n",
      "    - melanosome\n",
      "    - glycogen granule\n",
      "    - extracellular exosome\n",
      "  MF:\n",
      "    - RNA binding\n",
      "    - fatty acid synthase activity\n",
      "    - fatty acid synthase activity\n",
      "    - [acyl-carrier-protein] S-acetyltransferase activity\n",
      "    - [acyl-carrier-protein] S-malonyltransferase activity\n",
      "    - 3-oxoacyl-[acyl-carrier-protein] synthase activity\n",
      "    - 3-oxoacyl-[acyl-carrier-protein] reductase (NADPH) activity\n",
      "    - protein binding\n",
      "    - fatty acyl-[ACP] hydrolase activity\n",
      "    - (3R)-hydroxyacyl-[acyl-carrier-protein] dehydratase activity\n",
      "    - phosphopantetheine binding\n",
      "    - cadherin binding\n",
      "    - enoyl-[acyl-carrier-protein] reductase (NADPH) activity\n",
      "UniProt: {'Swiss-Prot': 'P49327', 'TrEMBL': ['A0A1B0GVK4', 'A0A0U1RRG3', 'A0A1B0GWG8', 'J3KTF0', 'A0A0U1RQF0', 'A0A1B0GTR5']}\n",
      "\n",
      "Gene: TNFSF14\n",
      "Symbol: TNFSF14\n",
      "Name: TNF superfamily member 14\n",
      "Summary: The protein encoded by this gene is a member of the tumor necrosis factor (TNF) ligand family. This protein is a ligand for TNFRSF14, which is a member of the tumor necrosis factor receptor superfamily, and which is also known as a herpesvirus entry mediator (HVEM). This protein may function as a costimulatory factor for the activation of lymphoid cells and as a deterrent to infection by herpesvirus. This protein has been shown to stimulate the proliferation of T cells, and trigger apoptosis of various tumor cells. This protein is also reported to prevent tumor necrosis factor alpha mediated apoptosis in primary hepatocyte. Two alternatively spliced transcript variant encoding distinct isoforms have been reported. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - apoptotic process\n",
      "    - immune response\n",
      "    - signal transduction\n",
      "    - cell surface receptor signaling pathway\n",
      "    - T cell chemotaxis\n",
      "    - positive regulation of T cell chemotaxis\n",
      "    - T cell costimulation\n",
      "    - T cell proliferation\n",
      "    - T cell activation\n",
      "    - T cell homeostasis\n",
      "    - positive regulation of canonical NF-kappaB signal transduction\n",
      "    - positive regulation of myoblast differentiation\n",
      "    - cellular response to mechanical stimulus\n",
      "    - positive regulation of non-canonical NF-kappaB signal transduction\n",
      "    - positive regulation of myoblast fusion\n",
      "    - positive regulation of extrinsic apoptotic signaling pathway\n",
      "  CC:\n",
      "    - extracellular space\n",
      "    - cytoplasm\n",
      "    - plasma membrane\n",
      "  MF:\n",
      "    - signaling receptor binding\n",
      "    - cytokine activity\n",
      "    - tumor necrosis factor receptor binding\n",
      "    - protein binding\n",
      "    - identical protein binding\n",
      "    - cysteine-type endopeptidase inhibitor activity involved in apoptotic process\n",
      "UniProt: {'Swiss-Prot': 'O43557'}\n",
      "\n",
      "Gene: ROCK2\n",
      "Symbol: ROCK2\n",
      "Name: Rho associated coiled-coil containing protein kinase 2\n",
      "Summary: The protein encoded by this gene is a serine/threonine kinase that regulates cytokinesis, smooth muscle contraction, the formation of actin stress fibers and focal adhesions, and the activation of the c-fos serum response element. This protein, which is an isozyme of ROCK1 is a target for the small GTPase Rho. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - mitotic cytokinesis\n",
      "    - epithelial to mesenchymal transition\n",
      "    - positive regulation of protein phosphorylation\n",
      "    - response to ischemia\n",
      "    - aortic valve morphogenesis\n",
      "    - chromatin remodeling\n",
      "    - protein phosphorylation\n",
      "    - smooth muscle contraction\n",
      "    - Rho protein signal transduction\n",
      "    - Rho protein signal transduction\n",
      "    - positive regulation of endothelial cell migration\n",
      "    - positive regulation of cardiac muscle hypertrophy\n",
      "    - positive regulation of gene expression\n",
      "    - positive regulation of gene expression\n",
      "    - negative regulation of gene expression\n",
      "    - positive regulation of centrosome duplication\n",
      "    - negative regulation of angiogenesis\n",
      "    - actin cytoskeleton organization\n",
      "    - regulation of cell adhesion\n",
      "    - positive regulation of cell migration\n",
      "    - cortical actin cytoskeleton organization\n",
      "    - actomyosin structure organization\n",
      "    - regulation of nervous system process\n",
      "    - positive regulation of connective tissue growth factor production\n",
      "    - regulation of actin cytoskeleton organization\n",
      "    - regulation of actin cytoskeleton organization\n",
      "    - regulation of circadian rhythm\n",
      "    - positive regulation of MAPK cascade\n",
      "    - modulation by host of viral process\n",
      "    - negative regulation of nitric oxide biosynthetic process\n",
      "    - negative regulation of nitric oxide biosynthetic process\n",
      "    - regulation of keratinocyte differentiation\n",
      "    - rhythmic process\n",
      "    - embryonic morphogenesis\n",
      "    - centrosome duplication\n",
      "    - regulation of stress fiber assembly\n",
      "    - positive regulation of stress fiber assembly\n",
      "    - positive regulation of stress fiber assembly\n",
      "    - regulation of focal adhesion assembly\n",
      "    - mRNA destabilization\n",
      "    - negative regulation of biomineral tissue development\n",
      "    - cellular response to testosterone stimulus\n",
      "    - response to transforming growth factor beta\n",
      "    - protein localization to plasma membrane\n",
      "    - protein localization to plasma membrane\n",
      "    - positive regulation of fibroblast growth factor production\n",
      "    - blood vessel diameter maintenance\n",
      "    - regulation of angiotensin-activated signaling pathway\n",
      "    - negative regulation of protein localization to lysosome\n",
      "    - regulation of cellular response to hypoxia\n",
      "    - regulation of cell junction assembly\n",
      "    - positive regulation of amyloid-beta formation\n",
      "    - positive regulation of protein localization to early endosome\n",
      "    - positive regulation of amyloid precursor protein catabolic process\n",
      "    - regulation of establishment of endothelial barrier\n",
      "    - regulation of establishment of endothelial barrier\n",
      "    - negative regulation of bicellular tight junction assembly\n",
      "    - cellular response to acetylcholine\n",
      "    - positive regulation of connective tissue replacement\n",
      "    - response to angiotensin\n",
      "    - regulation of establishment of cell polarity\n",
      "    - regulation of cell motility\n",
      "  CC:\n",
      "    - nucleus\n",
      "    - cytoplasm\n",
      "    - cytoplasm\n",
      "    - centrosome\n",
      "    - centrosome\n",
      "    - cytosol\n",
      "    - cytosol\n",
      "    - cytoskeleton\n",
      "    - plasma membrane\n",
      "    - cytoplasmic ribonucleoprotein granule\n",
      "  MF:\n",
      "    - protease binding\n",
      "    - RNA binding\n",
      "    - protein serine/threonine kinase activity\n",
      "    - protein serine/threonine kinase activity\n",
      "    - protein serine/threonine kinase activity\n",
      "    - 3-phosphoinositide-dependent protein kinase activity\n",
      "    - DNA-dependent protein kinase activity\n",
      "    - AMP-activated protein kinase activity\n",
      "    - eukaryotic translation initiation factor 2alpha kinase activity\n",
      "    - ribosomal protein S6 kinase activity\n",
      "    - structural molecule activity\n",
      "    - protein binding\n",
      "    - ATP binding\n",
      "    - zinc ion binding\n",
      "    - small GTPase binding\n",
      "    - histone H3S10 kinase activity\n",
      "    - histone H3T11 kinase activity\n",
      "    - histone H3T6 kinase activity\n",
      "    - histone H2AXS139 kinase activity\n",
      "    - histone H3S28 kinase activity\n",
      "    - histone H4S1 kinase activity\n",
      "    - histone H2AS1 kinase activity\n",
      "    - histone H2BS14 kinase activity\n",
      "    - tau protein binding\n",
      "    - endopeptidase activator activity\n",
      "    - histone H3T3 kinase activity\n",
      "    - histone H2AS121 kinase activity\n",
      "    - Rho-dependent protein serine/threonine kinase activity\n",
      "    - Rho-dependent protein serine/threonine kinase activity\n",
      "    - Rho-dependent protein serine/threonine kinase activity\n",
      "    - protein serine kinase activity\n",
      "    - histone H2BS36 kinase activity\n",
      "    - histone H3S57 kinase activity\n",
      "    - histone H3T45 kinase activity\n",
      "    - histone H2AT120 kinase activity\n",
      "UniProt: {'Swiss-Prot': 'O75116', 'TrEMBL': ['A0A2P9DU05', 'Q14DU5', 'A0A8V8TL82', 'A0A8V8TL90', 'C9JFJ0', 'D6REE7', 'E9PF63']}\n",
      "\n",
      "Gene: PUM2\n",
      "Symbol: PUM2\n",
      "Name: pumilio RNA binding family member 2\n",
      "Summary: This gene encodes a protein that belongs to a family of RNA-binding proteins. The encoded protein functions as a translational repressor during embryonic development and cell differentiation. This protein is also thought to be a positive regulator of cell proliferation in adipose-derived stem cells. Alternate splicing results in multiple transcript variants. [provided by RefSeq, Sep 2013].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - skeletal system development\n",
      "    - hair follicle development\n",
      "    - regulation of translation\n",
      "    - mitochondrion organization\n",
      "    - post-transcriptional regulation of gene expression\n",
      "    - post-transcriptional regulation of gene expression\n",
      "    - respiratory electron transport chain\n",
      "    - stress granule assembly\n",
      "    - miRNA processing\n",
      "    - miRNA processing\n",
      "    - regulation of mRNA stability\n",
      "    - regulation of mRNA stability\n",
      "    - positive regulation of sprouting of injured axon\n",
      "    - chromosome organization\n",
      "    - regulation of chromosome segregation\n",
      "    - adipose tissue development\n",
      "    - regulation of miRNA-mediated gene silencing\n",
      "    - 3'-UTR-mediated mRNA destabilization\n",
      "    - regulation of postsynapse assembly\n",
      "    - positive regulation of RIG-I signaling pathway\n",
      "    - regulation of intracellular mRNA localization\n",
      "    - positive regulation of miRNA-mediated gene silencing\n",
      "  CC:\n",
      "    - cytoplasm\n",
      "    - cytosol\n",
      "    - cytosol\n",
      "    - cytoplasmic stress granule\n",
      "    - cytoplasmic stress granule\n",
      "    - nuclear membrane\n",
      "    - neuronal cell body\n",
      "    - perinuclear region of cytoplasm\n",
      "    - postsynapse\n",
      "    - glutamatergic synapse\n",
      "  MF:\n",
      "    - RNA binding\n",
      "    - RNA binding\n",
      "    - mRNA 3'-UTR binding\n",
      "    - mRNA 3'-UTR binding\n",
      "    - mRNA 3'-UTR binding\n",
      "    - protein binding\n",
      "    - miRNA binding\n",
      "    - pumilio-response element binding\n",
      "    - lncRNA binding\n",
      "UniProt: {'Swiss-Prot': 'Q8TB72', 'TrEMBL': ['F2Z366', 'B7ZL34', 'C9JE24', 'F8WD57', 'C9JW01', 'A0A0C4DG68']}\n",
      "\n",
      "Gene: NUDT21\n",
      "Symbol: NUDT21\n",
      "Name: nudix hydrolase 21\n",
      "Summary: The protein encoded by this gene is one subunit of a cleavage factor required for 3' RNA cleavage and polyadenylation processing. The interaction of the protein with the RNA is one of the earliest steps in the assembly of the 3' end processing complex and facilitates the recruitment of other processing factors. This gene encodes the 25kD subunit of the protein complex, which is composed of four polypeptides. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - mRNA processing\n",
      "    - mRNA processing\n",
      "    - post-transcriptional regulation of gene expression\n",
      "    - cell differentiation\n",
      "    - mRNA 3'-end processing\n",
      "    - protein tetramerization\n",
      "    - protein heterotetramerization\n",
      "    - mRNA alternative polyadenylation\n",
      "    - co-transcriptional mRNA 3'-end processing, cleavage and polyadenylation pathway\n",
      "    - positive regulation of stem cell differentiation\n",
      "    - positive regulation of pro-B cell differentiation\n",
      "  CC:\n",
      "    - nucleus\n",
      "    - nucleoplasm\n",
      "    - cytoplasm\n",
      "    - centrosome\n",
      "    - mRNA cleavage and polyadenylation specificity factor complex\n",
      "    - mRNA cleavage factor complex\n",
      "    - mRNA cleavage factor complex\n",
      "    - mRNA cleavage factor complex\n",
      "    - nuclear body\n",
      "    - centriolar satellite\n",
      "    - paraspeckles\n",
      "  MF:\n",
      "    - chromatin binding\n",
      "    - RNA binding\n",
      "    - RNA binding\n",
      "    - mRNA binding\n",
      "    - protein binding\n",
      "    - mRNA 3'-UTR AU-rich region binding\n",
      "    - mRNA 3'-UTR AU-rich region binding\n",
      "    - identical protein binding\n",
      "    - identical protein binding\n",
      "    - protein homodimerization activity\n",
      "    - histone deacetylase binding\n",
      "UniProt: {'Swiss-Prot': 'O43809', 'TrEMBL': ['H3BND3', 'H3BV41']}\n",
      "\n",
      "Gene: CACNA1H\n",
      "Symbol: CACNA1H\n",
      "Name: calcium voltage-gated channel subunit alpha1 H\n",
      "Summary: This gene encodes a T-type member of the alpha-1 subunit family, a protein in the voltage-dependent calcium channel complex. Calcium channels mediate the influx of calcium ions into the cell upon membrane polarization and consist of a complex of alpha-1, alpha-2/delta, beta, and gamma subunits in a 1:1:1:1 ratio. The alpha-1 subunit has 24 transmembrane segments and forms the pore through which ions pass into the cell. There are multiple isoforms of each of the proteins in the complex, either encoded by different genes or the result of alternative splicing of transcripts. Alternate transcriptional splice variants, encoding different isoforms, have been characterized for the gene described here. Studies suggest certain mutations in this gene lead to childhood absence epilepsy (CAE). [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - muscle contraction\n",
      "    - muscle organ development\n",
      "    - myoblast fusion\n",
      "    - regulation of heart contraction\n",
      "    - aldosterone biosynthetic process\n",
      "    - cellular response to hormone stimulus\n",
      "    - cortisol biosynthetic process\n",
      "    - cellular response to potassium ion\n",
      "    - regulation of membrane potential\n",
      "    - calcium ion import\n",
      "    - inorganic cation transmembrane transport\n",
      "    - calcium ion import across plasma membrane\n",
      "    - positive regulation of acrosome reaction\n",
      "  CC:\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - voltage-gated calcium channel complex\n",
      "    - membrane\n",
      "  MF:\n",
      "    - voltage-gated monoatomic ion channel activity\n",
      "    - voltage-gated calcium channel activity\n",
      "    - voltage-gated calcium channel activity\n",
      "    - protein binding\n",
      "    - high voltage-gated calcium channel activity\n",
      "    - low voltage-gated calcium channel activity\n",
      "    - metal ion binding\n",
      "    - scaffold protein binding\n",
      "UniProt: {'Swiss-Prot': 'O95180', 'TrEMBL': ['B3KQH9', 'H3BUA8', 'A0AAA9YHH8', 'H3BNT0', 'H3BMW6', 'A0A1W2PQW2', 'A0AAA9YHX7', 'A0AAA9YHI3', 'A0AAA9YHS9', 'A0AAA9YHT2', 'A0AAA9YHS5', 'A0AAA9YHX9', 'A0A1W2PS38', 'A0AAA9YHG1', 'A0AAA9YHG4', 'A0AAA9YHS1', 'A0AAA9YHY5', 'A0AAA9YHL9', 'A0AAA9YHM0', 'A0AAA9YHG8', 'A0AAA9YHI7', 'A0A1W2PR14', 'A0AAA9YHY2', 'A0AAA9YHM5', 'A0AAA9YHH3', 'A0A1W2PQ19']}\n",
      "\n",
      "Gene: SRPK1\n",
      "Symbol: SRPK1\n",
      "Name: SRSF protein kinase 1\n",
      "Summary: This gene encodes a serine/arginine protein kinase specific for the SR (serine/arginine-rich domain) family of splicing factors. The protein localizes to the nucleus and the cytoplasm. It is thought to play a role in regulation of both constitutive and alternative splicing by regulating intracellular localization of splicing factors. Alternative splicing of this gene results in multiple transcript variants. Additional alternatively spliced transcript variants have been described for this gene, but their full length nature have not been determined.[provided by RefSeq, Jul 2010].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - spliceosomal complex assembly\n",
      "    - chromatin remodeling\n",
      "    - protein phosphorylation\n",
      "    - chromosome segregation\n",
      "    - RNA splicing\n",
      "    - sperm DNA condensation\n",
      "    - intracellular signal transduction\n",
      "    - intracellular signal transduction\n",
      "    - positive regulation of viral genome replication\n",
      "    - negative regulation of viral genome replication\n",
      "    - innate immune response\n",
      "    - regulation of mRNA splicing, via spliceosome\n",
      "    - regulation of mRNA processing\n",
      "    - regulation of mRNA processing\n",
      "  CC:\n",
      "    - chromatin\n",
      "    - nucleus\n",
      "    - nucleus\n",
      "    - nucleoplasm\n",
      "    - cytoplasm\n",
      "    - cytoplasm\n",
      "    - endoplasmic reticulum\n",
      "    - cytosol\n",
      "    - cytosol\n",
      "    - plasma membrane\n",
      "    - nuclear matrix\n",
      "    - nuclear speck\n",
      "  MF:\n",
      "    - magnesium ion binding\n",
      "    - RNA binding\n",
      "    - protein kinase activity\n",
      "    - protein serine/threonine kinase activity\n",
      "    - protein serine/threonine kinase activity\n",
      "    - 3-phosphoinositide-dependent protein kinase activity\n",
      "    - DNA-dependent protein kinase activity\n",
      "    - AMP-activated protein kinase activity\n",
      "    - eukaryotic translation initiation factor 2alpha kinase activity\n",
      "    - ribosomal protein S6 kinase activity\n",
      "    - protein binding\n",
      "    - ATP binding\n",
      "    - histone H3S10 kinase activity\n",
      "    - histone H3T11 kinase activity\n",
      "    - histone H3T6 kinase activity\n",
      "    - histone H2AXS139 kinase activity\n",
      "    - histone H3S28 kinase activity\n",
      "    - histone H4S1 kinase activity\n",
      "    - histone H2AS1 kinase activity\n",
      "    - histone H2BS14 kinase activity\n",
      "    - histone H3T3 kinase activity\n",
      "    - histone H2AS121 kinase activity\n",
      "    - Rho-dependent protein serine/threonine kinase activity\n",
      "    - protein serine kinase activity\n",
      "    - histone H2BS36 kinase activity\n",
      "    - histone H3S57 kinase activity\n",
      "    - histone H3T45 kinase activity\n",
      "    - histone H2AT120 kinase activity\n",
      "UniProt: {'Swiss-Prot': 'Q96SB4', 'TrEMBL': ['D6RB98', 'D6RDZ3', 'H0Y932', 'D6RBF8', 'D6RBM8', 'H3BLV9']}\n",
      "\n",
      "Gene: BMF\n",
      "Symbol: BMF\n",
      "Name: Bcl2 modifying factor\n",
      "Summary: The protein encoded by this gene belongs to the BCL2 protein family. BCL2 family members form hetero- or homodimers and act as anti- or pro-apoptotic regulators that are involved in a wide variety of cellular activities. This protein contains a single BCL2 homology domain 3 (BH3), and has been shown to bind BCL2 proteins and function as an apoptotic activator. This protein is found to be sequestered to myosin V motors by its association with dynein light chain 2, which may be important for sensing intracellular damage and triggering apoptosis. Alternatively spliced transcript variants encoding different isoforms have been identified. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - negative regulation of autophagy\n",
      "    - negative regulation of autophagy\n",
      "    - positive regulation of protein-containing complex assembly\n",
      "    - cellular response to UV\n",
      "    - positive regulation of apoptotic process\n",
      "    - positive regulation of apoptotic process\n",
      "    - anoikis\n",
      "    - positive regulation of release of cytochrome c from mitochondria\n",
      "  CC:\n",
      "    - mitochondrial outer membrane\n",
      "    - cytosol\n",
      "    - plasma membrane\n",
      "    - myosin complex\n",
      "    - myosin complex\n",
      "  MF:\n",
      "    - protein binding\n",
      "UniProt: {'Swiss-Prot': 'Q96LC9', 'TrEMBL': ['H0YNT3', 'H0YK50']}\n",
      "\n",
      "Gene: SLC22A8\n",
      "Symbol: SLC22A8\n",
      "Name: solute carrier family 22 member 8\n",
      "Summary: This gene encodes a protein involved in the sodium-independent transport and excretion of organic anions, some of which are potentially toxic. The encoded protein is an integral membrane protein and appears to be localized to the basolateral membrane of the kidney. Multiple alternatively spliced transcript variants that encode different protein isoforms have been described for this gene. [provided by RefSeq, May 2010].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - monoatomic ion transport\n",
      "    - response to toxic substance\n",
      "    - inorganic anion transport\n",
      "    - organic anion transport\n",
      "    - prostaglandin transport\n",
      "    - xenobiotic transport\n",
      "    - transmembrane transport\n",
      "    - transport across blood-brain barrier\n",
      "  CC:\n",
      "    - plasma membrane\n",
      "    - basolateral plasma membrane\n",
      "    - basolateral plasma membrane\n",
      "    - apical plasma membrane\n",
      "    - apical plasma membrane\n",
      "    - extracellular exosome\n",
      "  MF:\n",
      "    - solute:inorganic anion antiporter activity\n",
      "    - organic anion transmembrane transporter activity\n",
      "    - organic anion transmembrane transporter activity\n",
      "    - prostaglandin transmembrane transporter activity\n",
      "    - xenobiotic transmembrane transporter activity\n",
      "UniProt: {'Swiss-Prot': 'Q8TCC7', 'TrEMBL': ['H7BXN9', 'B2R807']}\n",
      "\n",
      "Gene: CDH3\n",
      "Symbol: CDH3\n",
      "Name: cadherin 3\n",
      "Summary: This gene encodes a classical cadherin of the cadherin superfamily. Alternative splicing results in multiple transcript variants, at least one of which encodes a preproprotein that is proteolytically processed to generate the mature glycoprotein. This calcium-dependent cell-cell adhesion protein is comprised of five extracellular cadherin repeats, a transmembrane region and a highly conserved cytoplasmic tail. This gene is located in a gene cluster in a region on the long arm of chromosome 16 that is involved in loss of heterozygosity events in breast and prostate cancer. In addition, aberrant expression of this protein is observed in cervical adenocarcinomas. Mutations in this gene are associated with hypotrichosis with juvenile macular dystrophy and ectodermal dysplasia, ectrodactyly, and macular dystrophy syndrome (EEMS). [provided by RefSeq, Nov 2015].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - cell morphogenesis\n",
      "    - retina homeostasis\n",
      "    - cell-cell junction assembly\n",
      "    - cell adhesion\n",
      "    - homophilic cell adhesion via plasma membrane adhesion molecules\n",
      "    - visual perception\n",
      "    - response to xenobiotic stimulus\n",
      "    - positive regulation of gene expression\n",
      "    - positive regulation of keratinocyte proliferation\n",
      "    - calcium-dependent cell-cell adhesion via plasma membrane cell adhesion molecules\n",
      "    - cell migration\n",
      "    - hair cycle process\n",
      "    - negative regulation of transforming growth factor beta receptor signaling pathway\n",
      "    - keratinization\n",
      "    - positive regulation of tyrosinase activity\n",
      "    - adherens junction organization\n",
      "    - positive regulation of insulin-like growth factor receptor signaling pathway\n",
      "    - cell-cell adhesion mediated by cadherin\n",
      "    - positive regulation of melanin biosynthetic process\n",
      "    - negative regulation of timing of catagen\n",
      "    - positive regulation of canonical Wnt signaling pathway\n",
      "    - positive regulation of melanosome transport\n",
      "  CC:\n",
      "    - cytoplasm\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - adherens junction\n",
      "    - catenin complex\n",
      "    - cell junction\n",
      "  MF:\n",
      "    - calcium ion binding\n",
      "    - beta-catenin binding\n",
      "    - cadherin binding\n",
      "UniProt: {'Swiss-Prot': 'P22223', 'TrEMBL': ['J3QR60', 'B4DLF0', 'J3QR34', 'J3KRQ1', 'J3QKW6', 'J3QL41', 'J3QL75']}\n",
      "\n",
      "Gene: MKNK1\n",
      "Symbol: MKNK1\n",
      "Name: MAPK interacting serine/threonine kinase 1\n",
      "Summary: This gene encodes a Ser/Thr protein kinase that interacts with, and is activated by ERK1 and p38 mitogen-activated protein kinases, and thus may play a role in the response to environmental stress and cytokines. This kinase may also regulate transcription by phosphorylating eIF4E via interaction with the C-terminal region of eIF4G. Alternatively spliced transcript variants have been noted for this gene. [provided by RefSeq, Jan 2012].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - chromatin remodeling\n",
      "    - regulation of translation\n",
      "    - protein phosphorylation\n",
      "    - peptidyl-serine phosphorylation\n",
      "    - intracellular signal transduction\n",
      "    - intracellular signal transduction\n",
      "  CC:\n",
      "    - nucleus\n",
      "    - nucleoplasm\n",
      "    - cytoplasm\n",
      "    - cytosol\n",
      "  MF:\n",
      "    - protein serine/threonine kinase activity\n",
      "    - protein serine/threonine kinase activity\n",
      "    - 3-phosphoinositide-dependent protein kinase activity\n",
      "    - DNA-dependent protein kinase activity\n",
      "    - AMP-activated protein kinase activity\n",
      "    - calcium/calmodulin-dependent protein kinase activity\n",
      "    - eukaryotic translation initiation factor 2alpha kinase activity\n",
      "    - ribosomal protein S6 kinase activity\n",
      "    - protein binding\n",
      "    - calmodulin binding\n",
      "    - ATP binding\n",
      "    - calcium-dependent protein serine/threonine kinase activity\n",
      "    - histone H3S10 kinase activity\n",
      "    - histone H3T11 kinase activity\n",
      "    - histone H3T6 kinase activity\n",
      "    - histone H2AXS139 kinase activity\n",
      "    - histone H3S28 kinase activity\n",
      "    - histone H4S1 kinase activity\n",
      "    - histone H2AS1 kinase activity\n",
      "    - histone H2BS14 kinase activity\n",
      "    - metal ion binding\n",
      "    - histone H3T3 kinase activity\n",
      "    - histone H2AS121 kinase activity\n",
      "    - Rho-dependent protein serine/threonine kinase activity\n",
      "    - protein serine kinase activity\n",
      "    - histone H2BS36 kinase activity\n",
      "    - histone H3S57 kinase activity\n",
      "    - histone H3T45 kinase activity\n",
      "    - histone H2AT120 kinase activity\n",
      "UniProt: {'Swiss-Prot': 'Q9BUB5', 'TrEMBL': ['E9PN34', 'A8K341', 'Q7Z319', 'E9PLE7', 'A0A499FJS1', 'E9PSC9', 'E9PMM8', 'E9PSE0', 'E9PII5', 'E9PIA0', 'A0A499FJN1', 'H0YE67', 'A0A499FIS5', 'E9PQ26']}\n",
      "\n",
      "Gene: EPHX1\n",
      "Symbol: EPHX1\n",
      "Name: epoxide hydrolase 1\n",
      "Summary: Epoxide hydrolase is a critical biotransformation enzyme that converts epoxides from the degradation of aromatic compounds to trans-dihydrodiols which can be conjugated and excreted from the body. Epoxide hydrolase functions in both the activation and detoxification of epoxides. Mutations in this gene cause preeclampsia, epoxide hydrolase deficiency or increased epoxide hydrolase activity. Alternatively spliced transcript variants encoding the same protein have been found for this gene.[provided by RefSeq, Dec 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - xenobiotic metabolic process\n",
      "    - response to toxic substance\n",
      "    - arachidonate metabolic process\n",
      "    - arachidonate metabolic process\n",
      "    - epoxide metabolic process\n",
      "    - epoxide metabolic process\n",
      "    - hydrocarbon catabolic process\n",
      "    - response to polycyclic arene\n",
      "  CC:\n",
      "    - endoplasmic reticulum membrane\n",
      "  MF:\n",
      "    - epoxide hydrolase activity\n",
      "    - epoxide hydrolase activity\n",
      "    - protein binding\n",
      "    - oxysterol binding\n",
      "    - cis-stilbene-oxide hydrolase activity\n",
      "    - cis-stilbene-oxide hydrolase activity\n",
      "UniProt: {'Swiss-Prot': 'P07099', 'TrEMBL': ['B1AQP8', 'R4SBI6']}\n",
      "\n",
      "Gene: TBXA2R\n",
      "Symbol: TBXA2R\n",
      "Name: thromboxane A2 receptor\n",
      "Summary: This gene encodes a member of the G protein-coupled receptor family. The protein interacts with thromboxane A2 to induce platelet aggregation and regulate hemostasis. A mutation in this gene results in a bleeding disorder. Multiple transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Oct 2009].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - smooth muscle contraction\n",
      "    - inflammatory response\n",
      "    - G protein-coupled receptor signaling pathway\n",
      "    - adenylate cyclase-activating G protein-coupled receptor signaling pathway\n",
      "    - positive regulation of cytosolic calcium ion concentration\n",
      "    - response to nutrient\n",
      "    - response to xenobiotic stimulus\n",
      "    - positive regulation of blood coagulation\n",
      "    - response to testosterone\n",
      "    - thromboxane A2 signaling pathway\n",
      "    - response to ethanol\n",
      "    - positive regulation of angiogenesis\n",
      "    - positive regulation of blood pressure\n",
      "    - positive regulation of vasoconstriction\n",
      "    - positive regulation of smooth muscle contraction\n",
      "    - cellular response to lipopolysaccharide\n",
      "    - negative regulation of cell migration involved in sprouting angiogenesis\n",
      "  CC:\n",
      "    - acrosomal vesicle\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - nuclear speck\n",
      "  MF:\n",
      "    - thromboxane A2 receptor activity\n",
      "    - guanyl-nucleotide exchange factor activity\n",
      "    - protein binding\n",
      "UniProt: {'Swiss-Prot': 'P21731', 'TrEMBL': ['Q0VAB0', 'Q05C92', 'K7ER80']}\n",
      "\n",
      "Gene: LMNB2\n",
      "Symbol: LMNB2\n",
      "Name: lamin B2\n",
      "Summary: This gene encodes a B type nuclear lamin. The nuclear lamina consists of a two-dimensional matrix of proteins located next to the inner nuclear membrane. The lamin family of proteins make up the matrix and are highly conserved in evolution. During mitosis, the lamina matrix is reversibly disassembled as the lamin proteins are phosphorylated. Lamin proteins are thought to be involved in nuclear stability, chromatin structure and gene expression. Vertebrate lamins consist of two types, A and B. Mutations in this gene are associated with acquired partial lipodystrophy. [provided by RefSeq, May 2012].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - nuclear envelope organization\n",
      "    - cytoskeleton organization\n",
      "    - nuclear migration\n",
      "    - heterochromatin formation\n",
      "    - nuclear pore localization\n",
      "    - protein localization to nuclear envelope\n",
      "  CC:\n",
      "    - nuclear envelope\n",
      "    - nuclear lamina\n",
      "    - intermediate filament\n",
      "    - nuclear membrane\n",
      "  MF:\n",
      "    - structural constituent of cytoskeleton\n",
      "    - protein binding\n",
      "    - identical protein binding\n",
      "UniProt: {'Swiss-Prot': 'Q03252'}\n",
      "\n",
      "Gene: NME4\n",
      "Symbol: NME4\n",
      "Name: NME/NM23 nucleoside diphosphate kinase 4\n",
      "Summary: The nucleoside diphosphate (NDP) kinases (EC 2.7.4.6) are ubiquitous enzymes that catalyze transfer of gamma-phosphates, via a phosphohistidine intermediate, between nucleoside and dioxynucleoside tri- and diphosphates. The enzymes are products of the nm23 gene family, which includes NME4 (Milon et al., 1997 [PubMed 9099850]).[supplied by OMIM, May 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - GTP biosynthetic process\n",
      "    - UTP biosynthetic process\n",
      "    - CTP biosynthetic process\n",
      "    - lipid transport\n",
      "    - nucleoside metabolic process\n",
      "  CC:\n",
      "    - mitochondrion\n",
      "    - mitochondrion\n",
      "    - mitochondrial inner membrane\n",
      "    - mitochondrial intermembrane space\n",
      "    - mitochondrial matrix\n",
      "  MF:\n",
      "    - nucleoside diphosphate kinase activity\n",
      "    - nucleoside diphosphate kinase activity\n",
      "    - protein binding\n",
      "    - ATP binding\n",
      "    - metal ion binding\n",
      "    - cardiolipin binding\n",
      "UniProt: {'Swiss-Prot': 'O00746', 'TrEMBL': ['Q4TT34', 'A0A087WVT9', 'A2IDC9', 'H0Y6J0', 'F2Z297', 'F2Z2X0']}\n",
      "\n",
      "Gene: SOD2\n",
      "Symbol: SOD2\n",
      "Name: superoxide dismutase 2\n",
      "Summary: This gene is a member of the iron/manganese superoxide dismutase family. It encodes a mitochondrial protein that forms a homotetramer and binds one manganese ion per subunit. This protein binds to the superoxide byproducts of oxidative phosphorylation and converts them to hydrogen peroxide and diatomic oxygen. Mutations in this gene have been associated with idiopathic cardiomyopathy (IDC), premature aging, sporadic motor neuron disease, and cancer. Alternative splicing of this gene results in multiple transcript variants. A related pseudogene has been identified on chromosome 1. [provided by RefSeq, Apr 2016].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - response to superoxide\n",
      "    - response to hypoxia\n",
      "    - release of cytochrome c from mitochondria\n",
      "    - liver development\n",
      "    - detection of oxygen\n",
      "    - acetylcholine-mediated vasodilation involved in regulation of systemic arterial blood pressure\n",
      "    - regulation of transcription by RNA polymerase II\n",
      "    - glutathione metabolic process\n",
      "    - superoxide metabolic process\n",
      "    - heart development\n",
      "    - locomotory behavior\n",
      "    - regulation of blood pressure\n",
      "    - negative regulation of cell population proliferation\n",
      "    - intrinsic apoptotic signaling pathway in response to DNA damage\n",
      "    - intrinsic apoptotic signaling pathway in response to oxidative stress\n",
      "    - response to xenobiotic stimulus\n",
      "    - post-embryonic development\n",
      "    - response to manganese ion\n",
      "    - response to zinc ion\n",
      "    - response to selenium ion\n",
      "    - response to gamma radiation\n",
      "    - positive regulation of hydrogen peroxide biosynthetic process\n",
      "    - response to activity\n",
      "    - removal of superoxide radicals\n",
      "    - respiratory electron transport chain\n",
      "    - hemopoiesis\n",
      "    - positive regulation of cell migration\n",
      "    - intracellular oxygen homeostasis\n",
      "    - response to lipopolysaccharide\n",
      "    - response to L-ascorbic acid\n",
      "    - response to silicon dioxide\n",
      "    - cellular response to oxidative stress\n",
      "    - response to isolation stress\n",
      "    - response to immobilization stress\n",
      "    - response to hydrogen peroxide\n",
      "    - superoxide anion generation\n",
      "    - negative regulation of neuron apoptotic process\n",
      "    - positive regulation of nitric oxide biosynthetic process\n",
      "    - negative regulation of fat cell differentiation\n",
      "    - response to cadmium ion\n",
      "    - negative regulation of fibroblast proliferation\n",
      "    - neuron development\n",
      "    - response to axon injury\n",
      "    - erythrophore differentiation\n",
      "    - hydrogen peroxide biosynthetic process\n",
      "    - protein homotetramerization\n",
      "    - response to electrical stimulus\n",
      "    - regulation of mitochondrial membrane potential\n",
      "    - response to hyperoxia\n",
      "    - multicellular organismal-level iron ion homeostasis\n",
      "    - response to magnetism\n",
      "    - cellular response to ethanol\n",
      "    - negative regulation of oxidative stress-induced intrinsic apoptotic signaling pathway\n",
      "    - negative regulation of membrane hyperpolarization\n",
      "    - negative regulation of vascular associated smooth muscle cell proliferation\n",
      "    - positive regulation of vascular associated smooth muscle cell apoptotic process\n",
      "    - positive regulation of vascular associated smooth muscle cell differentiation involved in phenotypic switching\n",
      "  CC:\n",
      "    - mitochondrion\n",
      "    - mitochondrion\n",
      "    - mitochondrion\n",
      "    - mitochondrial matrix\n",
      "    - mitochondrial matrix\n",
      "    - mitochondrial nucleoid\n",
      "    - extracellular exosome\n",
      "  MF:\n",
      "    - DNA binding\n",
      "    - superoxide dismutase activity\n",
      "    - superoxide dismutase activity\n",
      "    - protein binding\n",
      "    - oxygen binding\n",
      "    - enzyme binding\n",
      "    - manganese ion binding\n",
      "    - manganese ion binding\n",
      "    - manganese ion binding\n",
      "    - identical protein binding\n",
      "UniProt: {'Swiss-Prot': 'P04179', 'TrEMBL': ['A0A384NL29', 'G5E9P6', 'Q96AM7', 'Q9UG59', 'F5H4R2', 'F5H3C5', 'G8JLJ2', 'F5GYZ5', 'F5GXZ9']}\n",
      "\n",
      "Gene: BATF2\n",
      "Symbol: BATF2\n",
      "Name: basic leucine zipper ATF-like transcription factor 2\n",
      "Summary: Predicted to enable DNA-binding transcription factor activity, RNA polymerase II-specific and RNA polymerase II cis-regulatory region sequence-specific DNA binding activity. Predicted to be involved in defense response to protozoan; myeloid dendritic cell differentiation; and regulation of transcription by RNA polymerase II. Predicted to be part of chromatin. Predicted to be active in nucleus. [provided by Alliance of Genome Resources, Apr 2022]\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - regulation of transcription by RNA polymerase II\n",
      "    - regulation of transcription by RNA polymerase II\n",
      "    - defense response to protozoan\n",
      "    - myeloid dendritic cell differentiation\n",
      "    - integrated stress response signaling\n",
      "  CC:\n",
      "    - chromatin\n",
      "    - nucleus\n",
      "    - nucleus\n",
      "    - RNA polymerase II transcription regulator complex\n",
      "  MF:\n",
      "    - RNA polymerase II cis-regulatory region sequence-specific DNA binding\n",
      "    - DNA-binding transcription factor activity, RNA polymerase II-specific\n",
      "    - DNA-binding transcription factor activity, RNA polymerase II-specific\n",
      "    - protein binding\n",
      "UniProt: {'Swiss-Prot': 'Q8N1L9', 'TrEMBL': ['E9PJ73', 'B4DV37']}\n",
      "\n",
      "Gene: PDE4A\n",
      "Symbol: PDE4A\n",
      "Name: phosphodiesterase 4A\n",
      "Summary: The protein encoded by this gene belongs to the cyclic nucleotide phosphodiesterase (PDE) family, and PDE4 subfamily. This PDE hydrolyzes the second messenger, cAMP, which is a regulator and mediator of a number of cellular responses to extracellular signals. Thus, by regulating the cellular concentration of cAMP, this protein plays a key role in many important physiological processes. Alternatively spliced transcript variants encoding different isoforms have been described for this gene.[provided by RefSeq, Jul 2011].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - cAMP catabolic process\n",
      "    - cAMP catabolic process\n",
      "    - signal transduction\n",
      "    - G protein-coupled receptor signaling pathway\n",
      "    - sensory perception of smell\n",
      "    - cAMP-mediated signaling\n",
      "    - cellular response to xenobiotic stimulus\n",
      "    - regulation of adenylate cyclase-activating G protein-coupled receptor signaling pathway\n",
      "    - regulation of cAMP/PKA signal transduction\n",
      "  CC:\n",
      "    - nucleoplasm\n",
      "    - cytosol\n",
      "    - cytosol\n",
      "    - plasma membrane\n",
      "    - membrane\n",
      "    - extrinsic component of membrane\n",
      "    - ruffle membrane\n",
      "    - perinuclear region of cytoplasm\n",
      "  MF:\n",
      "    - 3',5'-cyclic-AMP phosphodiesterase activity\n",
      "    - 3',5'-cyclic-AMP phosphodiesterase activity\n",
      "    - 3',5'-cyclic-AMP phosphodiesterase activity\n",
      "    - 3',5'-cyclic-AMP phosphodiesterase activity\n",
      "    - protein binding\n",
      "    - cAMP binding\n",
      "    - metal ion binding\n",
      "    - 3',5'-cyclic-GMP phosphodiesterase activity\n",
      "UniProt: {'Swiss-Prot': 'P27815', 'TrEMBL': ['K7EKV1', 'K7ENX7']}\n",
      "\n",
      "Gene: PTGDR\n",
      "Symbol: PTGDR\n",
      "Name: prostaglandin D2 receptor\n",
      "Summary: This gene encodes a member of the guanine nucleotide-binding protein (G protein)-coupled receptor (GPCR) superfamily. The receptors are seven-pass transmembrane proteins that respond to extracellular cues and activate intracellular signal transduction pathways. This protein is reported to be a receptor for prostaglandin D2, which is a mediator of allergic inflammation and allergic airway inflammation in asthma. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Jul 2013].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - inflammatory response\n",
      "    - G protein-coupled receptor signaling pathway\n",
      "    - positive regulation of cytosolic calcium ion concentration\n",
      "    - male sex determination\n",
      "    - sleep\n",
      "    - mast cell degranulation\n",
      "    - adenosine metabolic process\n",
      "    - cellular response to prostaglandin D stimulus\n",
      "  CC:\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - membrane\n",
      "  MF:\n",
      "    - prostaglandin J receptor activity\n",
      "    - prostaglandin D receptor activity\n",
      "    - prostaglandin D receptor activity\n",
      "    - protein binding\n",
      "UniProt: {'Swiss-Prot': 'Q13258'}\n",
      "\n",
      "Gene: MOGS\n",
      "Symbol: MOGS\n",
      "Name: mannosyl-oligosaccharide glucosidase\n",
      "Summary: This gene encodes the first enzyme in the N-linked oligosaccharide processing pathway. The enzyme cleaves the distal alpha-1,2-linked glucose residue from the Glc(3)-Man(9)-GlcNAc(2) oligosaccharide precursor. This protein is located in the lumen of the endoplasmic reticulum. Defects in this gene are a cause of type IIb congenital disorder of glycosylation (CDGIIb). Two transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Mar 2009].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - protein folding\n",
      "    - protein N-linked glycosylation\n",
      "    - oligosaccharide metabolic process\n",
      "    - viral protein processing\n",
      "  CC:\n",
      "    - endoplasmic reticulum\n",
      "    - endoplasmic reticulum membrane\n",
      "    - endoplasmic reticulum membrane\n",
      "    - membrane\n",
      "    - extracellular exosome\n",
      "  MF:\n",
      "    - Glc3Man9GlcNAc2 oligosaccharide glucosidase activity\n",
      "    - Glc3Man9GlcNAc2 oligosaccharide glucosidase activity\n",
      "    - glucosidase activity\n",
      "UniProt: {'Swiss-Prot': 'Q13724', 'TrEMBL': ['A0A384MDR6', 'Q58F09', 'B4E3B8', 'A0A3B3IRW2', 'A0A3B3IRI1', 'A0A8I5KQG9', 'C9JDQ1', 'B8ZZE2', 'A0A3B3IRI7', 'A0A3B3IS52', 'A0A3B3IRK6', 'A0A3B3ITU6', 'A0A8I5KTK5', 'A0A3B3IU81', 'A0A3B3ITC1']}\n",
      "\n",
      "Gene: KIR2DL1\n",
      "Symbol: KIR2DL1\n",
      "Name: killer cell immunoglobulin like receptor, two Ig domains and long cytoplasmic tail 1\n",
      "Summary: Killer cell immunoglobulin-like receptors (KIRs) are transmembrane glycoproteins expressed by natural killer cells and subsets of T cells. The KIR genes are polymorphic and highly homologous and they are found in a cluster on chromosome 19q13.4 within the 1 Mb leukocyte receptor complex (LRC). The gene content of the KIR gene cluster varies among haplotypes, although several 'framework' genes are found in all haplotypes (KIR3DL3, KIR3DP1, KIR3DL4, KIR3DL2). The KIR proteins are classified by the number of extracellular immunoglobulin domains (2D or 3D) and by whether they have a long (L) or short (S) cytoplasmic domain. KIR proteins with the long cytoplasmic domain transduce inhibitory signals upon ligand binding via an immune tyrosine-based inhibitory motif (ITIM), while KIR proteins with the short cytoplasmic domain lack the ITIM motif and instead associate with the TYRO protein tyrosine kinase binding protein to transduce activating signals. The ligands for several KIR proteins are subsets of HLA class I molecules; thus, KIR proteins are thought to play an important role in regulation of the immune response. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - immune response-regulating signaling pathway\n",
      "    - natural killer cell inhibitory signaling pathway\n",
      "    - immune response\n",
      "  CC:\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "  MF:\n",
      "    - protein binding\n",
      "    - signaling receptor activity\n",
      "UniProt: {'Swiss-Prot': 'P43626', 'TrEMBL': ['A0A0G2JQ05', 'Q6H2H2', 'A0A0G2JND8', 'A0A1W2PRZ1']}\n",
      "\n",
      "Gene: MGST1\n",
      "Symbol: MGST1\n",
      "Name: microsomal glutathione S-transferase 1\n",
      "Summary: The MAPEG (Membrane Associated Proteins in Eicosanoid and Glutathione metabolism) family consists of six human proteins, two of which are involved in the production of leukotrienes and prostaglandin E, important mediators of inflammation. Other family members, demonstrating glutathione S-transferase and peroxidase activities, are involved in cellular defense against toxic, carcinogenic, and pharmacologically active electrophilic compounds. This gene encodes a protein that catalyzes the conjugation of glutathione to electrophiles and the reduction of lipid hydroperoxides. This protein is localized to the endoplasmic reticulum and outer mitochondrial membrane where it is thought to protect these membranes from oxidative stress. Several transcript variants, some non-protein coding and some protein coding, have been found for this gene. [provided by RefSeq, May 2012].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - glutathione transport\n",
      "    - cellular response to lipid hydroperoxide\n",
      "    - cellular oxidant detoxification\n",
      "  CC:\n",
      "    - mitochondrion\n",
      "    - mitochondrion\n",
      "    - mitochondrial outer membrane\n",
      "    - peroxisomal membrane\n",
      "    - endoplasmic reticulum\n",
      "    - endoplasmic reticulum membrane\n",
      "    - plasma membrane\n",
      "    - membrane\n",
      "    - azurophil granule membrane\n",
      "  MF:\n",
      "    - glutathione transferase activity\n",
      "    - glutathione transferase activity\n",
      "    - glutathione peroxidase activity\n",
      "    - protein binding\n",
      "UniProt: {'Swiss-Prot': 'P10620', 'TrEMBL': ['F5H7F6', 'F5H760', 'F5H613', 'F5GX73', 'F5H6X2']}\n",
      "\n",
      "Gene: AKAP8L\n",
      "Symbol: AKAP8L\n",
      "Name: A-kinase anchoring protein 8 like\n",
      "Summary: Enables histone deacetylase binding activity and lamin binding activity. Involved in several processes, including mitotic chromosome condensation; regulation of histone modification; and regulation of mRNA export from nucleus. Located in chromatin; cytoplasm; and nuclear lumen. Part of ribonucleoprotein complex. [provided by Alliance of Genome Resources, Apr 2022]\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - mRNA processing\n",
      "    - mitotic chromosome condensation\n",
      "    - RNA splicing\n",
      "    - regulation of mRNA export from nucleus\n",
      "    - cell cycle G2/M phase transition\n",
      "    - positive regulation of transcription by RNA polymerase II\n",
      "    - nuclear membrane disassembly\n",
      "  CC:\n",
      "    - chromatin\n",
      "    - nucleus\n",
      "    - nucleus\n",
      "    - nucleus\n",
      "    - nucleoplasm\n",
      "    - cytoplasm\n",
      "    - nuclear matrix\n",
      "    - nuclear matrix\n",
      "    - PML body\n",
      "    - nuclear speck\n",
      "    - ribonucleoprotein complex\n",
      "  MF:\n",
      "    - DNA binding\n",
      "    - RNA binding\n",
      "    - protein binding\n",
      "    - lamin binding\n",
      "    - zinc ion binding\n",
      "    - DEAD/H-box RNA helicase binding\n",
      "    - protein kinase A regulatory subunit binding\n",
      "    - histone deacetylase binding\n",
      "UniProt: {'Swiss-Prot': 'Q9ULX6', 'TrEMBL': ['V9GZ50', 'A0A7P0T847', 'M0R008', 'A0A7P0T9N5', 'M0R1L5', 'A0A7P0T8N3', 'M0R1Y5', 'A0A7P0TAE3', 'A0A7P0TBP0', 'M0R010', 'A0A7P0TAJ5', 'M0QYT7', 'A0A7P0TAD3']}\n",
      "\n",
      "Gene: FKBP4\n",
      "Symbol: FKBP4\n",
      "Name: FKBP prolyl isomerase 4\n",
      "Summary: The protein encoded by this gene is a member of the immunophilin protein family, which play a role in immunoregulation and basic cellular processes involving protein folding and trafficking. This encoded protein is a cis-trans prolyl isomerase that binds to the immunosuppressants FK506 and rapamycin. It has high structural and functional similarity to FK506-binding protein 1A (FKBP1A), but unlike FKBP1A, this protein does not have immunosuppressant activity when complexed with FK506. It interacts with interferon regulatory factor-4 and plays an important role in immunoregulatory gene expression in B and T lymphocytes. This encoded protein is known to associate with phytanoyl-CoA alpha-hydroxylase. It can also associate with two heat shock proteins (hsp90 and hsp70) and thus may play a role in the intracellular trafficking of hetero-oligomeric forms of the steroid hormone receptors. This protein correlates strongly with adeno-associated virus type 2 vectors (AAV) resulting in a significant increase in AAV-mediated transgene expression in human cell lines. Thus this encoded protein is thought to have important implications for the optimal use of AAV vectors in human gene therapy. The human genome contains several non-transcribed pseudogenes similar to this gene. [provided by RefSeq, Sep 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - regulation of transcription by RNA polymerase II\n",
      "    - protein folding\n",
      "    - steroid hormone receptor complex assembly\n",
      "    - copper ion transport\n",
      "    - embryo implantation\n",
      "    - negative regulation of neuron projection development\n",
      "    - androgen receptor signaling pathway\n",
      "    - prostate gland development\n",
      "    - negative regulation of microtubule polymerization or depolymerization\n",
      "    - negative regulation of microtubule polymerization\n",
      "    - protein-containing complex localization\n",
      "    - male sex differentiation\n",
      "    - chaperone-mediated protein folding\n",
      "    - chaperone-mediated protein folding\n",
      "  CC:\n",
      "    - nucleoplasm\n",
      "    - nucleoplasm\n",
      "    - cytoplasm\n",
      "    - mitochondrion\n",
      "    - cytosol\n",
      "    - cytosol\n",
      "    - cytosol\n",
      "    - microtubule\n",
      "    - protein-containing complex\n",
      "    - neuronal cell body\n",
      "    - axonal growth cone\n",
      "    - perinuclear region of cytoplasm\n",
      "    - extracellular exosome\n",
      "  MF:\n",
      "    - RNA binding\n",
      "    - peptidyl-prolyl cis-trans isomerase activity\n",
      "    - peptidyl-prolyl cis-trans isomerase activity\n",
      "    - protein binding\n",
      "    - ATP binding\n",
      "    - GTP binding\n",
      "    - FK506 binding\n",
      "    - protein-macromolecule adaptor activity\n",
      "    - heat shock protein binding\n",
      "    - copper-dependent protein binding\n",
      "    - nuclear glucocorticoid receptor binding\n",
      "    - tau protein binding\n",
      "    - phosphoprotein binding\n",
      "    - RNA polymerase II CTD heptapeptide repeat P3 isomerase activity\n",
      "    - RNA polymerase II CTD heptapeptide repeat P6 isomerase activity\n",
      "UniProt: {'Swiss-Prot': 'Q02790', 'TrEMBL': ['F5H1U3', 'F5H120', 'H0YFG2', 'H0YG86']}\n",
      "\n",
      "Gene: SULT2A1\n",
      "Symbol: SULT2A1\n",
      "Name: sulfotransferase family 2A member 1\n",
      "Summary: This gene encodes a member of the sulfotransferase family. Sulfotransferases aid in the metabolism of drugs and endogenous compounds by converting these substances into more hydrophilic water-soluble sulfate conjugates that can be easily excreted. This protein catalyzes the sulfation of steroids and bile acids in the liver and adrenal glands, and may have a role in the inherited adrenal androgen excess in women with polycystic ovary syndrome. [provided by RefSeq, Mar 2010].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - ethanol catabolic process\n",
      "    - xenobiotic metabolic process\n",
      "    - steroid metabolic process\n",
      "    - steroid metabolic process\n",
      "    - cholesterol metabolic process\n",
      "    - lipid catabolic process\n",
      "    - bile acid catabolic process\n",
      "    - thyroid hormone metabolic process\n",
      "    - 3'-phosphoadenosine 5'-phosphosulfate metabolic process\n",
      "    - sulfation\n",
      "    - sulfation\n",
      "    - sulfation\n",
      "  CC:\n",
      "    - cytoplasm\n",
      "    - cytosol\n",
      "    - cytosol\n",
      "  MF:\n",
      "    - alcohol sulfotransferase activity\n",
      "    - protein binding\n",
      "    - sulfotransferase activity\n",
      "    - sulfotransferase activity\n",
      "    - bile-salt sulfotransferase activity\n",
      "    - steroid sulfotransferase activity\n",
      "    - steroid sulfotransferase activity\n",
      "    - 3'-phosphoadenosine 5'-phosphosulfate binding\n",
      "UniProt: {'Swiss-Prot': 'Q06520', 'TrEMBL': 'A8K015'}\n",
      "\n",
      "Gene: ADH1B\n",
      "Symbol: ADH1B\n",
      "Name: alcohol dehydrogenase 1B (class I), beta polypeptide\n",
      "Summary: The protein encoded by this gene is a member of the alcohol dehydrogenase family. Members of this enzyme family metabolize a wide variety of substrates, including ethanol, retinol, other aliphatic alcohols, hydroxysteroids, and lipid peroxidation products. This encoded protein, consisting of several homo- and heterodimers of alpha, beta, and gamma subunits, exhibits high activity for ethanol oxidation and plays a major role in ethanol catabolism. Three genes encoding alpha, beta and gamma subunits are tandemly organized in a genomic segment as a gene cluster. Two transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Nov 2013].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - retinoid metabolic process\n",
      "    - retinol metabolic process\n",
      "    - retinoic acid metabolic process\n",
      "  CC:\n",
      "    - nucleoplasm\n",
      "    - cytosol\n",
      "    - cytosol\n",
      "    - cytosol\n",
      "    - plasma membrane\n",
      "    - cilium\n",
      "    - ciliary basal body\n",
      "  MF:\n",
      "    - alcohol dehydrogenase (NAD+) activity\n",
      "    - all-trans-retinol dehydrogenase (NAD+) activity\n",
      "    - all-trans-retinol dehydrogenase (NAD+) activity\n",
      "    - protein binding\n",
      "    - zinc ion binding\n",
      "    - zinc ion binding\n",
      "UniProt: {'Swiss-Prot': 'P00325', 'TrEMBL': ['V9HW50', 'D6RHZ6']}\n",
      "\n",
      "Gene: LCN2\n",
      "Symbol: LCN2\n",
      "Name: lipocalin 2\n",
      "Summary: This gene encodes a protein that belongs to the lipocalin family. Members of this family transport small hydrophobic molecules such as lipids, steroid hormones and retinoids. The protein encoded by this gene is a neutrophil gelatinase-associated lipocalin and plays a role in innate immunity by limiting bacterial growth as a result of sequestering iron-containing siderophores. The presence of this protein in blood and urine is an early biomarker of acute kidney injury. This protein is thought to be be involved in multiple cellular processes, including maintenance of skin homeostasis, and suppression of invasiveness and metastasis. Mice lacking this gene are more susceptible to bacterial infection than wild type mice. [provided by RefSeq, Sep 2015].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - apoptotic process\n",
      "    - siderophore transport\n",
      "    - defense response to bacterium\n",
      "    - innate immune response\n",
      "    - positive regulation of cold-induced thermogenesis\n",
      "  CC:\n",
      "    - extracellular region\n",
      "    - extracellular region\n",
      "    - extracellular region\n",
      "    - extracellular space\n",
      "    - extracellular space\n",
      "    - extracellular space\n",
      "    - specific granule lumen\n",
      "    - extracellular exosome\n",
      "  MF:\n",
      "    - iron ion binding\n",
      "    - iron ion binding\n",
      "    - protein binding\n",
      "    - identical protein binding\n",
      "    - iron ion sequestering activity\n",
      "    - enterobactin binding\n",
      "UniProt: {'Swiss-Prot': 'P80188', 'TrEMBL': 'X6R8F3'}\n",
      "\n",
      "Gene: SCTR\n",
      "Symbol: SCTR\n",
      "Name: secretin receptor\n",
      "Summary: The protein encoded by this gene is a G protein-coupled receptor and belongs to the glucagon-VIP-secretin receptor family. It binds secretin which is the most potent regulator of pancreatic bicarbonate, electrolyte and volume secretion. Secretin and its receptor are suggested to be involved in pancreatic cancer and autism. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - diet induced thermogenesis\n",
      "    - cell surface receptor signaling pathway\n",
      "    - G protein-coupled receptor signaling pathway\n",
      "    - adenylate cyclase-modulating G protein-coupled receptor signaling pathway\n",
      "    - adenylate cyclase-activating G protein-coupled receptor signaling pathway\n",
      "    - adenylate cyclase-activating G protein-coupled receptor signaling pathway\n",
      "    - intracellular water homeostasis\n",
      "    - response to nutrient levels\n",
      "    - regulation of appetite\n",
      "    - regulation of synaptic plasticity\n",
      "  CC:\n",
      "    - cytoplasmic microtubule\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - basolateral plasma membrane\n",
      "  MF:\n",
      "    - G protein-coupled peptide receptor activity\n",
      "    - secretin receptor activity\n",
      "    - secretin receptor activity\n",
      "    - peptide hormone binding\n",
      "    - peptide hormone binding\n",
      "UniProt: {'Swiss-Prot': 'P47872', 'TrEMBL': ['A0A0D9SFY2', 'Q8IV17', 'A0A0D9SFV7']}\n",
      "\n",
      "Gene: CEACAM3\n",
      "Symbol: CEACAM3\n",
      "Name: CEA cell adhesion molecule 3\n",
      "Summary: This gene encodes a member of the family of carcinoembryonic antigen-related cell adhesion molecules (CEACAMs), which are used by several bacterial pathogens to bind and invade host cells. The encoded transmembrane protein directs phagocytosis of several bacterial species that is dependent on the small GTPase Rac. It is thought to serve an important role in controlling human-specific pathogens by the innate immune system. Alternatively spliced transcript variants have been described. [provided by RefSeq, Mar 2013].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - regulation of immune system process\n",
      "    - signal transduction\n",
      "  CC:\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - cell surface\n",
      "    - specific granule membrane\n",
      "  MF:\n",
      "    - protein binding\n",
      "    - protein tyrosine kinase binding\n",
      "UniProt: {'Swiss-Prot': 'P40198', 'TrEMBL': 'M0QXR5'}\n",
      "\n",
      "Gene: OPRD1\n",
      "Symbol: OPRD1\n",
      "Name: opioid receptor delta 1\n",
      "Summary: Enables G protein-coupled enkephalin receptor activity. Involved in several processes, including G protein-coupled opioid receptor signaling pathway; cellular response to hypoxia; and positive regulation of peptidyl-serine phosphorylation. Is intrinsic component of plasma membrane. [provided by Alliance of Genome Resources, Apr 2022]\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - immune response\n",
      "    - G protein-coupled receptor signaling pathway\n",
      "    - G protein-coupled receptor signaling pathway, coupled to cyclic nucleotide second messenger\n",
      "    - adenylate cyclase-inhibiting G protein-coupled receptor signaling pathway\n",
      "    - phospholipase C-activating G protein-coupled receptor signaling pathway\n",
      "    - neuropeptide signaling pathway\n",
      "    - adult locomotory behavior\n",
      "    - negative regulation of gene expression\n",
      "    - negative regulation of protein-containing complex assembly\n",
      "    - positive regulation of CREB transcription factor activity\n",
      "    - positive regulation of peptidyl-serine phosphorylation\n",
      "    - response to nicotine\n",
      "    - G protein-coupled opioid receptor signaling pathway\n",
      "    - G protein-coupled opioid receptor signaling pathway\n",
      "    - eating behavior\n",
      "    - response to ethanol\n",
      "    - regulation of mitochondrial membrane potential\n",
      "    - regulation of calcium ion transport\n",
      "    - cellular response to growth factor stimulus\n",
      "    - cellular response to hypoxia\n",
      "    - cellular response to toxic substance\n",
      "  CC:\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - synaptic vesicle membrane\n",
      "    - dendrite membrane\n",
      "    - presynaptic membrane\n",
      "    - neuron projection\n",
      "    - axon terminus\n",
      "    - spine apparatus\n",
      "    - postsynaptic density membrane\n",
      "    - neuronal dense core vesicle\n",
      "  MF:\n",
      "    - G protein-coupled opioid receptor activity\n",
      "    - protein binding\n",
      "    - receptor serine/threonine kinase binding\n",
      "    - G protein-coupled enkephalin receptor activity\n",
      "    - G protein-coupled enkephalin receptor activity\n",
      "    - neuropeptide binding\n",
      "UniProt: {'Swiss-Prot': 'P41143'}\n",
      "\n",
      "Gene: MAGT1\n",
      "Symbol: MAGT1\n",
      "Name: magnesium transporter 1\n",
      "Summary: This gene encodes a ubiquitously expressed magnesium cation transporter protein that localizes to the cell membrane. This protein also associates with N-oligosaccharyl transferase and therefore may have a role in N-glycosylation. Mutations in this gene cause a form of X-linked intellectual disability (XLID). This gene may have multiple in-frame translation initiation sites, one of which would encode a shorter protein with an N-terminus containing a signal peptide at amino acids 1-29. [provided by RefSeq, Jul 2017].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - protein N-linked glycosylation\n",
      "    - protein N-linked glycosylation\n",
      "    - magnesium ion transport\n",
      "    - protein N-linked glycosylation via asparagine\n",
      "    - protein N-linked glycosylation via asparagine\n",
      "    - cognition\n",
      "    - transmembrane transport\n",
      "    - magnesium ion transmembrane transport\n",
      "  CC:\n",
      "    - endoplasmic reticulum\n",
      "    - endoplasmic reticulum membrane\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - oligosaccharyltransferase complex\n",
      "    - oligosaccharyltransferase complex\n",
      "    - oligosaccharyltransferase complex\n",
      "    - membrane\n",
      "    - azurophil granule membrane\n",
      "    - oligosaccharyltransferase complex B\n",
      "  MF:\n",
      "    - magnesium ion transmembrane transporter activity\n",
      "UniProt: {'Swiss-Prot': 'Q9H0U3', 'TrEMBL': ['A0A8I5KY62', 'A0A087WU53', 'A0A8I5KYH1', 'A0A8I5KUC4', 'A0A8I5QJJ8', 'A0A8I5QKX7', 'A0A8I5QJM4']}\n",
      "\n",
      "Gene: ICAM3\n",
      "Symbol: ICAM3\n",
      "Name: intercellular adhesion molecule 3\n",
      "Summary: The protein encoded by this gene is a member of the intercellular adhesion molecule (ICAM) family. All ICAM proteins are type I transmembrane glycoproteins, contain 2-9 immunoglobulin-like C2-type domains, and bind to the leukocyte adhesion LFA-1 protein. This protein is constitutively and abundantly expressed by all leucocytes and may be the most important ligand for LFA-1 in the initiation of the immune response. It functions not only as an adhesion molecule, but also as a potent signalling molecule. Alternative splicing results in multiple transcript variants encoding different isoforms. [provided by RefSeq, Feb 2016].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - phagocytosis\n",
      "    - cell adhesion\n",
      "    - cell-cell adhesion\n",
      "  CC:\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - extracellular exosome\n",
      "  MF:\n",
      "    - signaling receptor binding\n",
      "    - integrin binding\n",
      "    - integrin binding\n",
      "    - protein binding\n",
      "UniProt: {'Swiss-Prot': 'P32942', 'TrEMBL': ['A0A9L9PXU5', 'A0A9L9PXS1', 'K7EMH0']}\n",
      "\n",
      "Gene: KRTAP11-1\n",
      "Symbol: KRTAP11-1\n",
      "Name: keratin associated protein 11-1\n",
      "Summary: Predicted to be located in cytosol. [provided by Alliance of Genome Resources, Apr 2022]\n",
      "GO Annotations:\n",
      "  CC:\n",
      "    - cytosol\n",
      "    - keratin filament\n",
      "  MF:\n",
      "    - structural molecule activity\n",
      "    - protein binding\n",
      "UniProt: {'Swiss-Prot': 'Q8IUC1'}\n",
      "\n",
      "Gene: SNAP25\n",
      "Symbol: SNAP25\n",
      "Name: synaptosome associated protein 25\n",
      "Summary: Synaptic vesicle membrane docking and fusion is mediated by SNAREs (soluble N-ethylmaleimide-sensitive factor attachment protein receptors) located on the vesicle membrane (v-SNAREs) and the target membrane (t-SNAREs). The assembled v-SNARE/t-SNARE complex consists of a bundle of four helices, one of which is supplied by v-SNARE and the other three by t-SNARE. For t-SNAREs on the plasma membrane, the protein syntaxin supplies one helix and the protein encoded by this gene contributes the other two. Therefore, this gene product is a presynaptic plasma membrane protein involved in the regulation of neurotransmitter release. Two alternative transcript variants encoding different protein isoforms have been described for this gene. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - neurotransmitter uptake\n",
      "    - exocytosis\n",
      "    - chemical synaptic transmission\n",
      "    - axonogenesis\n",
      "    - long-term memory\n",
      "    - locomotory behavior\n",
      "    - associative learning\n",
      "    - regulation of neuron projection development\n",
      "    - synaptic vesicle exocytosis\n",
      "    - synaptic vesicle docking\n",
      "    - synaptic vesicle priming\n",
      "    - endosomal transport\n",
      "    - synaptic vesicle fusion to presynaptic active zone membrane\n",
      "    - positive regulation of insulin secretion\n",
      "    - SNARE complex assembly\n",
      "    - regulation of insulin secretion\n",
      "    - regulation of synapse assembly\n",
      "    - long-term synaptic potentiation\n",
      "    - potassium ion transmembrane transport\n",
      "    - presynaptic dense core vesicle exocytosis\n",
      "  CC:\n",
      "    - photoreceptor inner segment\n",
      "    - cytoplasm\n",
      "    - cytoplasm\n",
      "    - endosome\n",
      "    - trans-Golgi network\n",
      "    - cytosol\n",
      "    - cytoskeleton\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - cell cortex\n",
      "    - synaptic vesicle\n",
      "    - voltage-gated potassium channel complex\n",
      "    - actin cytoskeleton\n",
      "    - membrane\n",
      "    - lamellipodium\n",
      "    - filopodium\n",
      "    - growth cone\n",
      "    - BLOC-1 complex\n",
      "    - SNARE complex\n",
      "    - SNARE complex\n",
      "    - SNARE complex\n",
      "    - vesicle\n",
      "    - specific granule membrane\n",
      "    - presynaptic membrane\n",
      "    - neuron projection\n",
      "    - neuronal cell body\n",
      "    - axonal growth cone\n",
      "    - perinuclear region of cytoplasm\n",
      "    - synaptobrevin 2-SNAP-25-syntaxin-1a-complexin I complex\n",
      "    - synaptobrevin 2-SNAP-25-syntaxin-1a-complexin II complex\n",
      "    - synaptobrevin 2-SNAP-25-syntaxin-1a complex\n",
      "    - tertiary granule membrane\n",
      "    - ribbon synapse\n",
      "    - extrinsic component of presynaptic membrane\n",
      "    - glutamatergic synapse\n",
      "  MF:\n",
      "    - voltage-gated potassium channel activity\n",
      "    - SNAP receptor activity\n",
      "    - protein binding\n",
      "    - lipid binding\n",
      "    - myosin binding\n",
      "    - syntaxin-1 binding\n",
      "    - syntaxin-1 binding\n",
      "    - protein domain specific binding\n",
      "    - transmembrane transporter binding\n",
      "    - calcium-dependent protein binding\n",
      "UniProt: {'Swiss-Prot': 'P60880', 'TrEMBL': 'A0A0A0MSS0'}\n",
      "\n",
      "Gene: BRICD5\n",
      "Symbol: BRICD5\n",
      "Name: BRICHOS domain containing 5\n",
      "Summary: Predicted to be involved in regulation of cell population proliferation. Predicted to be integral component of membrane. Predicted to be active in extracellular space. [provided by Alliance of Genome Resources, Apr 2022]\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - regulation of cell population proliferation\n",
      "  CC:\n",
      "    - extracellular space\n",
      "    - membrane\n",
      "  MF:\n",
      "    - protein binding\n",
      "UniProt: {'Swiss-Prot': 'Q6PL45', 'TrEMBL': 'H3BV65'}\n",
      "\n",
      "Gene: OPRK1\n",
      "Symbol: OPRK1\n",
      "Name: opioid receptor kappa 1\n",
      "Summary: This gene encodes an opioid receptor, which is a member of the 7 transmembrane-spanning G protein-coupled receptor family. It functions as a receptor for endogenous ligands, as well as a receptor for various synthetic opioids. Ligand binding results in inhibition of adenylate cyclase activity and neurotransmitter release. This opioid receptor plays a role in the perception of pain and mediating the hypolocomotor, analgesic and aversive actions of synthetic opioids. Variations in this gene have also been associated with alcohol dependence and opiate addiction. Alternatively spliced transcript variants encoding different isoforms have been found for this gene. A recent study provided evidence for translational readthrough in this gene, and expression of an additional C-terminally extended isoform via the use of an alternative in-frame translation termination codon. [provided by RefSeq, Dec 2017].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - immune response\n",
      "    - adenylate cyclase-inhibiting G protein-coupled receptor signaling pathway\n",
      "    - phospholipase C-activating G protein-coupled receptor signaling pathway\n",
      "    - neuropeptide signaling pathway\n",
      "    - chemical synaptic transmission\n",
      "    - sensory perception\n",
      "    - locomotory behavior\n",
      "    - sensory perception of pain\n",
      "    - sensory perception of pain\n",
      "    - adenylate cyclase-inhibiting opioid receptor signaling pathway\n",
      "    - response to insulin\n",
      "    - positive regulation of dopamine secretion\n",
      "    - negative regulation of luteinizing hormone secretion\n",
      "    - response to nicotine\n",
      "    - G protein-coupled opioid receptor signaling pathway\n",
      "    - G protein-coupled opioid receptor signaling pathway\n",
      "    - maternal behavior\n",
      "    - eating behavior\n",
      "    - response to estrogen\n",
      "    - estrous cycle\n",
      "    - response to ethanol\n",
      "    - regulation of saliva secretion\n",
      "    - behavioral response to cocaine\n",
      "    - sensory perception of temperature stimulus\n",
      "    - defense response to virus\n",
      "    - cellular response to lipopolysaccharide\n",
      "    - cellular response to glucose stimulus\n",
      "    - positive regulation of p38MAPK cascade\n",
      "    - positive regulation of potassium ion transmembrane transport\n",
      "    - response to acrylamide\n",
      "    - positive regulation of eating behavior\n",
      "    - conditioned place preference\n",
      "  CC:\n",
      "    - nucleoplasm\n",
      "    - mitochondrion\n",
      "    - cytosol\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - membrane\n",
      "    - sarcoplasmic reticulum\n",
      "    - T-tubule\n",
      "    - dendrite\n",
      "    - synaptic vesicle membrane\n",
      "    - presynaptic membrane\n",
      "    - neuron projection\n",
      "    - perikaryon\n",
      "    - axon terminus\n",
      "    - postsynaptic membrane\n",
      "  MF:\n",
      "    - G protein-coupled opioid receptor activity\n",
      "    - protein binding\n",
      "    - receptor serine/threonine kinase binding\n",
      "    - dynorphin receptor activity\n",
      "    - dynorphin receptor activity\n",
      "    - dynorphin receptor activity\n",
      "    - neuropeptide binding\n",
      "UniProt: {'Swiss-Prot': 'P41145', 'TrEMBL': ['E5RJI5', 'A0A5F9ZI09']}\n",
      "\n",
      "Gene: GRM2\n",
      "Symbol: GRM2\n",
      "Name: glutamate metabotropic receptor 2\n",
      "Summary: L-glutamate is the major excitatory neurotransmitter in the central nervous system and activates both ionotropic and metabotropic glutamate receptors. Glutamatergic neurotransmission is involved in most aspects of normal brain function and can be perturbed in many neuropathologic conditions. The metabotropic glutamate receptors are a family of G protein-coupled receptors, that have been divided into 3 groups on the basis of sequence homology, putative signal transduction mechanisms, and pharmacologic properties. Group I includes GRM1 and GRM5 and these receptors have been shown to activate phospholipase C. Group II includes GRM2 and GRM3 while Group III includes GRM4, GRM6, GRM7 and GRM8. Group II and III receptors are linked to the inhibition of the cyclic AMP cascade but differ in their agonist selectivities. Several transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Mar 2017].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - negative regulation of adenylate cyclase activity\n",
      "    - adenylate cyclase-inhibiting G protein-coupled glutamate receptor signaling pathway\n",
      "    - G protein-coupled glutamate receptor signaling pathway\n",
      "    - chemical synaptic transmission\n",
      "    - gene expression\n",
      "    - glutamate secretion\n",
      "    - regulation of glutamate secretion\n",
      "    - regulation of dopamine secretion\n",
      "    - cellular response to stress\n",
      "    - behavioral response to nicotine\n",
      "    - response to cocaine\n",
      "    - positive regulation of phosphatidylinositol 3-kinase/protein kinase B signal transduction\n",
      "    - regulation of synaptic transmission, glutamatergic\n",
      "    - long-term synaptic depression\n",
      "    - intracellular glutamate homeostasis\n",
      "    - presynaptic modulation of chemical synaptic transmission\n",
      "    - regulation of response to drug\n",
      "  CC:\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - axon\n",
      "    - dendrite\n",
      "    - presynaptic membrane\n",
      "    - postsynaptic membrane\n",
      "    - astrocyte projection\n",
      "    - glutamatergic synapse\n",
      "  MF:\n",
      "    - group II metabotropic glutamate receptor activity\n",
      "    - G protein-coupled receptor activity\n",
      "    - calcium channel regulator activity\n",
      "    - protein binding\n",
      "    - glutamate receptor activity\n",
      "    - scaffold protein binding\n",
      "UniProt: {'Swiss-Prot': 'Q14416', 'TrEMBL': ['C9JL63', 'H7BXL3', 'C9JD41']}\n",
      "\n",
      "Gene: NRAS\n",
      "Symbol: NRAS\n",
      "Name: NRAS proto-oncogene, GTPase\n",
      "Summary: This is an N-ras oncogene encoding a membrane protein that shuttles between the Golgi apparatus and the plasma membrane. This shuttling is regulated through palmitoylation and depalmitoylation by the ZDHHC9-GOLGA7 complex. The encoded protein, which has intrinsic GTPase activity, is activated by a guanine nucleotide-exchange factor and inactivated by a GTPase activating protein. Mutations in this gene have been associated with somatic rectal cancer, follicular thyroid cancer, autoimmune lymphoproliferative syndrome, Noonan syndrome, and juvenile myelomonocytic leukemia. [provided by RefSeq, Jun 2011].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - MAPK cascade\n",
      "    - positive regulation of endothelial cell proliferation\n",
      "    - Ras protein signal transduction\n",
      "    - Ras protein signal transduction\n",
      "    - myoblast differentiation\n",
      "  CC:\n",
      "    - Golgi membrane\n",
      "    - endoplasmic reticulum membrane\n",
      "    - Golgi apparatus\n",
      "    - cytosol\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - membrane\n",
      "    - extracellular exosome\n",
      "    - tertiary granule membrane\n",
      "  MF:\n",
      "    - GTPase activity\n",
      "    - GTPase activity\n",
      "    - GTPase activity\n",
      "    - G protein activity\n",
      "    - protein binding\n",
      "    - GTP binding\n",
      "    - GDP binding\n",
      "    - protein-containing complex binding\n",
      "UniProt: {'Swiss-Prot': 'P01111', 'TrEMBL': 'Q5U091'}\n",
      "\n",
      "Gene: KRT34\n",
      "Symbol: KRT34\n",
      "Name: keratin 34\n",
      "Summary: The protein encoded by this gene is a member of the keratin gene family. As a type I hair keratin, it is an acidic protein which heterodimerizes with type II keratins to form hair and nails. The type I hair keratins are clustered in a region of chromosome 17q12-q21 and have the same direction of transcription. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - epidermis development\n",
      "    - epithelial cell differentiation\n",
      "    - intermediate filament organization\n",
      "  CC:\n",
      "    - extracellular space\n",
      "    - cytosol\n",
      "    - cytoskeleton\n",
      "    - intermediate filament\n",
      "  MF:\n",
      "    - structural molecule activity\n",
      "    - protein binding\n",
      "\n",
      "Gene: KRTAP5-11\n",
      "Symbol: KRTAP5-11\n",
      "Name: keratin associated protein 5-11\n",
      "Summary: Predicted to be located in cytosol. [provided by Alliance of Genome Resources, Apr 2022]\n",
      "GO Annotations:\n",
      "  CC:\n",
      "    - cytosol\n",
      "    - intermediate filament\n",
      "  MF:\n",
      "    - protein binding\n",
      "UniProt: {'Swiss-Prot': 'Q6L8G4'}\n",
      "\n",
      "Gene: INCA1\n",
      "Symbol: INCA1\n",
      "Name: inhibitor of CDK, cyclin A1 interacting protein 1\n",
      "Summary: Enables cyclin binding activity; cyclin-dependent protein serine/threonine kinase inhibitor activity; and identical protein binding activity. Acts upstream of or within negative regulation of cyclin-dependent protein serine/threonine kinase activity. Located in cytoplasm and nuclear body. [provided by Alliance of Genome Resources, Apr 2022]\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - negative regulation of cell population proliferation\n",
      "    - negative regulation of cyclin-dependent protein serine/threonine kinase activity\n",
      "    - fibroblast proliferation\n",
      "    - negative regulation of fibroblast proliferation\n",
      "    - apoptotic signaling pathway\n",
      "    - positive regulation of apoptotic signaling pathway\n",
      "  CC:\n",
      "    - nucleus\n",
      "    - nucleoplasm\n",
      "    - cytoplasm\n",
      "    - cytoplasm\n",
      "    - nuclear body\n",
      "  MF:\n",
      "    - cyclin-dependent protein serine/threonine kinase inhibitor activity\n",
      "    - cyclin-dependent protein serine/threonine kinase inhibitor activity\n",
      "    - protein binding\n",
      "    - cyclin binding\n",
      "    - cyclin binding\n",
      "    - cyclin binding\n",
      "    - identical protein binding\n",
      "    - protein-containing complex binding\n",
      "UniProt: {'Swiss-Prot': 'Q0VD86'}\n",
      "\n",
      "Gene: PSEN2\n",
      "Symbol: PSEN2\n",
      "Name: presenilin 2\n",
      "Summary: Alzheimer's disease (AD) patients with an inherited form of the disease carry mutations in the presenilin proteins (PSEN1 or PSEN2) or the amyloid precursor protein (APP). These disease-linked mutations result in increased production of the longer form of amyloid-beta (main component of amyloid deposits found in AD brains). Presenilins are postulated to regulate APP processing through their effects on gamma-secretase, an enzyme that cleaves APP. Also, it is thought that the presenilins are involved in the cleavage of the Notch receptor such that, they either directly regulate gamma-secretase activity, or themselves act are protease enzymes. Two alternatively spliced transcript variants encoding different isoforms of PSEN2 have been identified. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - response to hypoxia\n",
      "    - membrane protein ectodomain proteolysis\n",
      "    - membrane protein ectodomain proteolysis\n",
      "    - Notch signaling pathway\n",
      "    - Notch receptor processing\n",
      "    - protein processing\n",
      "    - protein processing\n",
      "    - amyloid-beta formation\n",
      "    - amyloid-beta formation\n",
      "    - intracellular signal transduction\n",
      "    - amyloid precursor protein catabolic process\n",
      "    - amyloid precursor protein catabolic process\n",
      "    - calcium ion homeostasis\n",
      "    - regulation of calcium import into the mitochondrion\n",
      "    - mitochondrion-endoplasmic reticulum membrane tethering\n",
      "  CC:\n",
      "    - Golgi membrane\n",
      "    - kinetochore\n",
      "    - nuclear inner membrane\n",
      "    - early endosome\n",
      "    - endoplasmic reticulum\n",
      "    - endoplasmic reticulum membrane\n",
      "    - Golgi apparatus\n",
      "    - centrosome\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - synaptic vesicle\n",
      "    - membrane\n",
      "    - protein-containing complex\n",
      "    - presynaptic membrane\n",
      "    - gamma-secretase complex\n",
      "    - gamma-secretase complex\n",
      "  MF:\n",
      "    - protein binding\n",
      "    - aspartic endopeptidase activity, intramembrane cleaving\n",
      "UniProt: {'Swiss-Prot': 'P49810', 'TrEMBL': ['A0A7I2V3Y8', 'A0A7I2YQG9', 'A0A7I2V551', 'A0A7I2V4Y6', 'E5RHT1', 'E5RG63', 'A0A7I2V4T4', 'A0A7I2V4D0', 'A0A7I2V355', 'A0A7I2V5Y0', 'A0A7I2V3R9', 'A0A7I2V5Q3', 'A0A7I2V5S6', 'E5RFW4', 'A0A7I2V2W1', 'A0A7I2V5L1']}\n",
      "\n",
      "Gene: MGST3\n",
      "Symbol: MGST3\n",
      "Name: microsomal glutathione S-transferase 3\n",
      "Summary: This gene encodes a member of the MAPEG (Membrane Associated Proteins in Eicosanoid and Glutathione metabolism) protein family. Members of this family are involved in the production of leukotrienes and prostaglandin E, important mediators of inflammation. This gene encodes an enzyme which catalyzes the conjugation of leukotriene A4 and reduced glutathione to produce leukotriene C4. This enzyme also demonstrates glutathione-dependent peroxidase activity towards lipid hydroperoxides.[provided by RefSeq, May 2011].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - lipid metabolic process\n",
      "    - prostanoid metabolic process\n",
      "    - arachidonate metabolic process\n",
      "    - leukotriene biosynthetic process\n",
      "    - cellular oxidant detoxification\n",
      "  CC:\n",
      "    - nuclear envelope\n",
      "    - mitochondrion\n",
      "    - mitochondrial outer membrane\n",
      "    - endoplasmic reticulum\n",
      "    - endoplasmic reticulum membrane\n",
      "    - membrane\n",
      "    - membrane\n",
      "    - intracellular membrane-bounded organelle\n",
      "  MF:\n",
      "    - glutathione transferase activity\n",
      "    - glutathione transferase activity\n",
      "    - glutathione transferase activity\n",
      "    - leukotriene-C4 synthase activity\n",
      "    - glutathione peroxidase activity\n",
      "    - glutathione peroxidase activity\n",
      "    - protein binding\n",
      "    - identical protein binding\n",
      "UniProt: {'Swiss-Prot': 'O14880', 'TrEMBL': ['Q5VV87', 'Q5VV89']}\n",
      "\n",
      "Gene: ZSCAN4\n",
      "Symbol: ZSCAN4\n",
      "Name: zinc finger and SCAN domain containing 4\n",
      "Summary: The ZSCAN4 gene encodes a protein involved in telomere maintenance and with a key role in the critical feature of mouse embryonic stem (ES) cells, namely, defying cellular senescence and maintaining normal karyotype for many cell divisions in culture (Zalzman et al., 2010 [PubMed 20336070]).[supplied by OMIM, May 2010].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - regulation of transcription by RNA polymerase II\n",
      "    - telomere maintenance via telomere lengthening\n",
      "    - negative regulation of mitotic recombination\n",
      "  CC:\n",
      "    - chromosome, telomeric region\n",
      "    - nucleus\n",
      "  MF:\n",
      "    - RNA polymerase II cis-regulatory region sequence-specific DNA binding\n",
      "    - DNA-binding transcription factor activity, RNA polymerase II-specific\n",
      "    - protein binding\n",
      "    - zinc ion binding\n",
      "    - sequence-specific double-stranded DNA binding\n",
      "UniProt: {'Swiss-Prot': 'Q8NAM6'}\n",
      "\n",
      "Gene: OPRM1\n",
      "Symbol: OPRM1\n",
      "Name: opioid receptor mu 1\n",
      "Summary: This gene encodes one of at least three opioid receptors in humans; the mu opioid receptor (MOR). The MOR is the principal target of endogenous opioid peptides and opioid analgesic agents such as beta-endorphin and enkephalins. The MOR also has an important role in dependence to other drugs of abuse, such as nicotine, cocaine, and alcohol via its modulation of the dopamine system. The NM_001008503.2:c.118A>G allele has been associated with opioid and alcohol addiction and variations in pain sensitivity but evidence for it having a causal role is conflicting. Multiple transcript variants encoding different isoforms have been found for this gene. Though the canonical MOR belongs to the superfamily of 7-transmembrane-spanning G-protein-coupled receptors some isoforms of this gene have only 6 transmembrane domains. [provided by RefSeq, Oct 2013].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - G protein-coupled receptor signaling pathway, coupled to cyclic nucleotide second messenger\n",
      "    - adenylate cyclase-inhibiting G protein-coupled acetylcholine receptor signaling pathway\n",
      "    - phospholipase C-activating G protein-coupled receptor signaling pathway\n",
      "    - positive regulation of cytosolic calcium ion concentration\n",
      "    - neuropeptide signaling pathway\n",
      "    - sensory perception\n",
      "    - negative regulation of cell population proliferation\n",
      "    - sensory perception of pain\n",
      "    - sensory perception of pain\n",
      "    - G protein-coupled opioid receptor signaling pathway\n",
      "    - G protein-coupled opioid receptor signaling pathway\n",
      "    - positive regulation of cAMP-mediated signaling\n",
      "    - negative regulation of cAMP-mediated signaling\n",
      "    - negative regulation of nitric oxide biosynthetic process\n",
      "    - positive regulation of nitric oxide biosynthetic process\n",
      "    - behavioral response to ethanol\n",
      "    - positive regulation of neurogenesis\n",
      "    - negative regulation of cytosolic calcium ion concentration\n",
      "    - negative regulation of Wnt protein secretion\n",
      "    - positive regulation of ERK1 and ERK2 cascade\n",
      "    - calcium ion transmembrane transport\n",
      "    - cellular response to morphine\n",
      "    - regulation of cellular response to stress\n",
      "    - regulation of NMDA receptor activity\n",
      "  CC:\n",
      "    - endosome\n",
      "    - endoplasmic reticulum\n",
      "    - Golgi apparatus\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - axon\n",
      "    - dendrite\n",
      "    - neuron projection\n",
      "    - perikaryon\n",
      "    - synapse\n",
      "  MF:\n",
      "    - G-protein alpha-subunit binding\n",
      "    - G-protein alpha-subunit binding\n",
      "    - G protein-coupled receptor activity\n",
      "    - beta-endorphin receptor activity\n",
      "    - beta-endorphin receptor activity\n",
      "    - voltage-gated calcium channel activity\n",
      "    - protein binding\n",
      "    - G-protein beta-subunit binding\n",
      "    - morphine receptor activity\n",
      "    - morphine receptor activity\n",
      "    - neuropeptide binding\n",
      "UniProt: {'Swiss-Prot': 'P35372', 'TrEMBL': ['G8XRH5', 'L0E130', 'B8K2Q5', 'G8XRH4', 'E7EW71']}\n",
      "\n",
      "Gene: DHRS1\n",
      "Symbol: DHRS1\n",
      "Name: dehydrogenase/reductase 1\n",
      "Summary: This gene encodes a member of the short-chain dehydrogenases/reductases (SDR) family. The encoded enzyme contains a conserved catalytic domain and likely functions as an oxidoreductase. Multiple alternatively spliced variants, encoding the same protein, have been identified. [provided by RefSeq, Nov 2008].\n",
      "GO Annotations:\n",
      "  CC:\n",
      "    - endoplasmic reticulum\n",
      "  MF:\n",
      "    - carbonyl reductase (NADPH) activity\n",
      "    - protein binding\n",
      "    - oxidoreductase activity, acting on the CH-OH group of donors, NAD or NADP as acceptor\n",
      "    - testosterone 17-beta-dehydrogenase (NADP+) activity\n",
      "    - estradiol 17-alpha-dehydrogenase [NAD(P)+] activity\n",
      "UniProt: {'Swiss-Prot': 'Q96LJ7', 'TrEMBL': 'H0YNC2'}\n",
      "\n",
      "Gene: TMPRSS2\n",
      "Symbol: TMPRSS2\n",
      "Name: transmembrane serine protease 2\n",
      "Summary: This gene encodes a protein that belongs to the serine protease family. The encoded protein contains a type II transmembrane domain, a receptor class A domain, a scavenger receptor cysteine-rich domain and a protease domain. Serine proteases are known to be involved in many physiological and pathological processes. This gene was demonstrated to be up-regulated by androgenic hormones in prostate cancer cells and down-regulated in androgen-independent prostate cancer tissue. The protease domain of this protein is thought to be cleaved and secreted into cell media after autocleavage. This protein also facilitates entry of viruses into host cells by proteolytically cleaving and activating viral envelope glycoproteins. Viruses found to use this protein for cell entry include Influenza virus and the human coronaviruses HCoV-229E, MERS-CoV, SARS-CoV and SARS-CoV-2 (COVID-19 virus). Alternatively spliced transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Apr 2020].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - proteolysis\n",
      "    - protein autoprocessing\n",
      "    - protein autoprocessing\n",
      "    - viral translation\n",
      "    - positive regulation of viral entry into host cell\n",
      "  CC:\n",
      "    - extracellular region\n",
      "    - nucleoplasm\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - extracellular exosome\n",
      "  MF:\n",
      "    - serine-type endopeptidase activity\n",
      "    - serine-type endopeptidase activity\n",
      "    - protein binding\n",
      "    - serine-type peptidase activity\n",
      "UniProt: {'Swiss-Prot': 'O15393', 'TrEMBL': ['A0A7I2V5F9', 'A0A7I2V5N2', 'C9J5Y1', 'A0A7I2V3D1', 'A0A7I2V509', 'A0A7I2V650', 'A0A7I2YQ98', 'A0A7I2V474', 'C9JB05']}\n",
      "\n",
      "Gene: PMP22\n",
      "Symbol: PMP22\n",
      "Name: peripheral myelin protein 22\n",
      "Summary: This gene encodes an integral membrane protein that is a major component of myelin in the peripheral nervous system. Studies suggest two alternately used promoters drive tissue-specific expression. Various mutations of this gene are causes of Charcot-Marie-Tooth disease Type IA, Dejerine-Sottas syndrome, and hereditary neuropathy with liability to pressure palsies. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Jul 2013].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - apoptotic process\n",
      "    - chemical synaptic transmission\n",
      "    - peripheral nervous system development\n",
      "    - negative regulation of cell population proliferation\n",
      "    - negative regulation of neuron projection development\n",
      "    - cell differentiation\n",
      "    - bleb assembly\n",
      "    - myelin assembly\n",
      "  CC:\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - bicellular tight junction\n",
      "    - compact myelin\n",
      "    - synapse\n",
      "  MF:\n",
      "    - protein binding\n",
      "UniProt: {'Swiss-Prot': 'Q01453', 'TrEMBL': ['A8MU75', 'Q6FH25', 'A0A6Q8PF08', 'J3KT36', 'A0A6Q8PFS0', 'A0A2R8Y5L5', 'A0A6Q8PFD5', 'A0A6Q8PF92', 'J3KQW0']}\n",
      "\n",
      "Gene: CCL11\n",
      "Symbol: CCL11\n",
      "Name: C-C motif chemokine ligand 11\n",
      "Summary: This antimicrobial gene is one of several chemokine genes clustered on the q-arm of chromosome 17. Chemokines form a superfamily of secreted proteins involved in immunoregulatory and inflammatory processes. The superfamily is divided into four subfamilies based on the arrangement of the N-terminal cysteine residues of the mature peptide. This chemokine, a member of the CC subfamily, displays chemotactic activity for eosinophils, but not mononuclear cells or neutrophils. This eosinophil-specific chemokine is thought to be involved in eosinophilic inflammatory diseases such as atopic dermatitis, allergic rhinitis, asthma and parasitic infections. [provided by RefSeq, Sep 2014].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - positive regulation of endothelial cell proliferation\n",
      "    - chronic inflammatory response\n",
      "    - mast cell chemotaxis\n",
      "    - protein phosphorylation\n",
      "    - intracellular calcium ion homeostasis\n",
      "    - chemotaxis\n",
      "    - inflammatory response\n",
      "    - cytoskeleton organization\n",
      "    - actin filament organization\n",
      "    - cell adhesion\n",
      "    - signal transduction\n",
      "    - learning or memory\n",
      "    - regulation of cell shape\n",
      "    - response to radiation\n",
      "    - response to virus\n",
      "    - positive regulation of cell migration\n",
      "    - positive regulation of cell migration\n",
      "    - positive regulation of actin filament polymerization\n",
      "    - killing of cells of another organism\n",
      "    - response to interleukin-13\n",
      "    - positive regulation of GTPase activity\n",
      "    - positive regulation of angiogenesis\n",
      "    - eosinophil chemotaxis\n",
      "    - eosinophil chemotaxis\n",
      "    - negative regulation of neurogenesis\n",
      "    - branching involved in mammary gland duct morphogenesis\n",
      "    - mammary duct terminal end bud growth\n",
      "    - antimicrobial humoral immune response mediated by antimicrobial peptide\n",
      "    - antimicrobial humoral immune response mediated by antimicrobial peptide\n",
      "    - chemokine-mediated signaling pathway\n",
      "    - ERK1 and ERK2 cascade\n",
      "    - response to interleukin-4\n",
      "  CC:\n",
      "    - extracellular region\n",
      "    - extracellular region\n",
      "    - extracellular space\n",
      "  MF:\n",
      "    - protein binding\n",
      "    - chemokine activity\n",
      "    - chemokine activity\n",
      "    - CCR3 chemokine receptor binding\n",
      "    - protein dimerization activity\n",
      "    - receptor ligand activity\n",
      "    - CCR chemokine receptor binding\n",
      "UniProt: {'Swiss-Prot': 'P51671', 'TrEMBL': 'Q6I9T4'}\n",
      "\n",
      "Gene: MTNR1A\n",
      "Symbol: MTNR1A\n",
      "Name: melatonin receptor 1A\n",
      "Summary: This gene encodes one of two high affinity forms of a receptor for melatonin, the primary hormone secreted by the pineal gland. This receptor is a G-protein coupled, 7-transmembrane receptor that is responsible for melatonin effects on mammalian circadian rhythm and reproductive alterations affected by day length. The receptor is an integral membrane protein that is readily detectable and localized to two specific regions of the brain. The hypothalamic suprachiasmatic nucleus appears to be involved in circadian rhythm while the hypophysial pars tuberalis may be responsible for the reproductive effects of melatonin. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - G protein-coupled receptor signaling pathway\n",
      "    - G protein-coupled receptor signaling pathway, coupled to cyclic nucleotide second messenger\n",
      "    - adenylate cyclase-inhibiting G protein-coupled receptor signaling pathway\n",
      "    - mating behavior\n",
      "    - circadian rhythm\n",
      "  CC:\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - receptor complex\n",
      "  MF:\n",
      "    - G protein-coupled receptor activity\n",
      "    - protein binding\n",
      "    - melatonin receptor activity\n",
      "    - hormone binding\n",
      "UniProt: {'Swiss-Prot': 'P48039'}\n",
      "\n",
      "Gene: MAP1LC3B\n",
      "Symbol: MAP1LC3B\n",
      "Name: microtubule associated protein 1 light chain 3 beta\n",
      "Summary: The product of this gene is a subunit of neuronal microtubule-associated MAP1A and MAP1B proteins, which are involved in microtubule assembly and important for neurogenesis. Studies on the rat homolog implicate a role for this gene in autophagy, a process that involves the bulk degradation of cytoplasmic component. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - autophagosome assembly\n",
      "    - autophagosome assembly\n",
      "    - mitophagy\n",
      "    - mitophagy\n",
      "    - mitophagy\n",
      "    - autophagy\n",
      "    - cellular response to nitrogen starvation\n",
      "    - cellular response to starvation\n",
      "    - macroautophagy\n",
      "    - autophagosome maturation\n",
      "    - autophagosome maturation\n",
      "  CC:\n",
      "    - autophagosome membrane\n",
      "    - autophagosome membrane\n",
      "    - autophagosome membrane\n",
      "    - mitochondrion\n",
      "    - autophagosome\n",
      "    - autophagosome\n",
      "    - cytosol\n",
      "    - microtubule\n",
      "    - axoneme\n",
      "    - endomembrane system\n",
      "    - organelle membrane\n",
      "    - cytoplasmic vesicle\n",
      "    - mitochondrial membrane\n",
      "  MF:\n",
      "    - protein binding\n",
      "    - microtubule binding\n",
      "    - phosphatidylethanolamine binding\n",
      "    - phosphatidylethanolamine binding\n",
      "    - ubiquitin protein ligase binding\n",
      "    - ubiquitin protein ligase binding\n",
      "    - ceramide binding\n",
      "UniProt: {'Swiss-Prot': 'Q9GZQ8', 'TrEMBL': ['H3BTL1', 'Q658J6', 'H3BM99', 'H3BTE7']}\n",
      "\n",
      "Gene: CCNE2\n",
      "Symbol: CCNE2\n",
      "Name: cyclin E2\n",
      "Summary: The protein encoded by this gene belongs to the highly conserved cyclin family, whose members are characterized by a dramatic periodicity in protein abundance through the cell cycle. Cyclins function as regulators of CDK kinases. Different cyclins exhibit distinct expression and degradation patterns which contribute to the temporal coordination of each mitotic event. This cyclin forms a complex with and functions as a regulatory subunit of CDK2. This cyclin has been shown to specifically interact with CIP/KIP family of CDK inhibitors, and plays a role in cell cycle G1/S transition. The expression of this gene peaks at the G1-S phase and exhibits a pattern of tissue specificity distinct from that of cyclin E1. A significantly increased expression level of this gene was observed in tumor-derived cells. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - regulation of cyclin-dependent protein serine/threonine kinase activity\n",
      "    - G1/S transition of mitotic cell cycle\n",
      "    - G1/S transition of mitotic cell cycle\n",
      "    - telomere maintenance\n",
      "    - DNA replication initiation\n",
      "    - homologous chromosome pairing at meiosis\n",
      "    - regulation of protein localization\n",
      "    - cell division\n",
      "    - positive regulation of G1/S transition of mitotic cell cycle\n",
      "  CC:\n",
      "    - nucleus\n",
      "    - nucleoplasm\n",
      "    - cytoplasm\n",
      "    - microtubule organizing center\n",
      "    - cytosol\n",
      "    - cyclin E1-CDK2 complex\n",
      "    - cyclin E2-CDK2 complex\n",
      "    - cyclin E2-CDK2 complex\n",
      "  MF:\n",
      "    - protein binding\n",
      "    - cyclin-dependent protein serine/threonine kinase regulator activity\n",
      "    - protein kinase binding\n",
      "UniProt: {'Swiss-Prot': 'O96020', 'TrEMBL': ['H0YBQ2', 'Q8WUE3', 'E5RK20', 'E5RHN2']}\n",
      "\n",
      "Gene: PTGER3\n",
      "Symbol: PTGER3\n",
      "Name: prostaglandin E receptor 3\n",
      "Summary: The protein encoded by this gene is a member of the G-protein coupled receptor family. This protein is one of four receptors identified for prostaglandin E2 (PGE2). This receptor may have many biological functions, which involve digestion, nervous system, kidney reabsorption, and uterine contraction activities. Studies of the mouse counterpart suggest that this receptor may also mediate adrenocorticotropic hormone response as well as fever generation in response to exogenous and endogenous stimuli. Multiple transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Aug 2009].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - inflammatory response\n",
      "    - G protein-coupled receptor signaling pathway\n",
      "    - adenylate cyclase-activating G protein-coupled receptor signaling pathway\n",
      "    - phospholipase C-activating G protein-coupled receptor signaling pathway\n",
      "    - positive regulation of cytosolic calcium ion concentration\n",
      "    - cell death\n",
      "    - intestine smooth muscle contraction\n",
      "    - positive regulation of fever generation\n",
      "    - negative regulation of gastric acid secretion\n",
      "  CC:\n",
      "    - nuclear envelope\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - membrane\n",
      "  MF:\n",
      "    - prostaglandin E receptor activity\n",
      "    - prostaglandin E receptor activity\n",
      "    - protein binding\n",
      "UniProt: {'Swiss-Prot': 'P43115', 'TrEMBL': ['A0A0B4J204', 'O00325']}\n",
      "\n",
      "Gene: KRTAP17-1\n",
      "Symbol: KRTAP17-1\n",
      "Name: keratin associated protein 17-1\n",
      "Summary: This protein is a member of the keratin-associated protein (KAP) family. The KAP proteins form a matrix of keratin intermediate filaments which contribute to the structure of hair fibers. KAP family members appear to have unique, family-specific amino- and carboxyl-terminal regions and are subdivided into three multi-gene families according to amino acid composition: the high sulfur, the ultrahigh sulfur, and the high tyrosine/glycine KAPs. This protein is a member of the ultrahigh sulfur KAP family and the gene is localized to a cluster of KAPs at 17q12-q21. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  CC:\n",
      "    - cytosol\n",
      "    - intermediate filament\n",
      "  MF:\n",
      "    - protein binding\n",
      "UniProt: {'Swiss-Prot': 'Q9BYP8'}\n",
      "\n",
      "Gene: SLC10A1\n",
      "Symbol: SLC10A1\n",
      "Name: solute carrier family 10 member 1\n",
      "Summary: The protein encoded by this gene belongs to the sodium/bile acid cotransporter family, which are integral membrane glycoproteins that participate in the enterohepatic circulation of bile acids. Two homologous transporters are involved in the reabsorption of bile acids; the ileal sodium/bile acid cotransporter with an apical cell localization that absorbs bile acids from the intestinal lumen, bile duct and kidney, and the liver-specific sodium/bile acid cotransporter, represented by this protein, that is found in the basolateral membranes of hepatocytes. Bile acids are the catabolic product of cholesterol metabolism, hence this protein is important for cholesterol homeostasis. [provided by RefSeq, Oct 2011].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - sodium ion transport\n",
      "    - bile acid and bile salt transport\n",
      "    - bile acid and bile salt transport\n",
      "    - response to nutrient levels\n",
      "    - response to estrogen\n",
      "    - response to ethanol\n",
      "    - symbiont entry into host cell\n",
      "    - transmembrane transport\n",
      "    - cellular response to xenobiotic stimulus\n",
      "  CC:\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - basolateral plasma membrane\n",
      "  MF:\n",
      "    - virus receptor activity\n",
      "    - protein binding\n",
      "    - bile acid:sodium symporter activity\n",
      "    - bile acid:sodium symporter activity\n",
      "    - bile acid transmembrane transporter activity\n",
      "UniProt: {'Swiss-Prot': 'Q14973', 'TrEMBL': 'B2RA41'}\n",
      "\n",
      "Gene: GPR182\n",
      "Symbol: GPR182\n",
      "Name: G protein-coupled receptor 182\n",
      "Summary: Adrenomedullin is a potent vasodilator peptide that exerts major effects on cardiovascular function. This gene encodes a seven-transmembrane protein that belongs to the family 1 of G-protein coupled receptors. Studies of the rat counterpart suggest that the encoded protein may function as a receptor for adrenomedullin. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - cell surface receptor signaling pathway\n",
      "    - G protein-coupled receptor signaling pathway\n",
      "  CC:\n",
      "    - plasma membrane\n",
      "    - membrane\n",
      "  MF:\n",
      "    - transmembrane signaling receptor activity\n",
      "    - G protein-coupled receptor activity\n",
      "UniProt: {'Swiss-Prot': 'O15218', 'TrEMBL': ['A0A0D9SG31', 'Q7Z4H2']}\n",
      "\n",
      "Gene: AK4\n",
      "Symbol: AK4\n",
      "Name: adenylate kinase 4\n",
      "Summary: This gene encodes a member of the adenylate kinase family of enzymes. The encoded protein is localized to the mitochondrial matrix. Adenylate kinases regulate the adenine and guanine nucleotide compositions within a cell by catalyzing the reversible transfer of phosphate group among these nucleotides. Five isozymes of adenylate kinase have been identified in vertebrates. Expression of these isozymes is tissue-specific and developmentally regulated. A pseudogene for this gene has been located on chromosome 17. Three transcript variants encoding the same protein have been identified for this gene. Sequence alignment suggests that the gene defined by NM_013410, NM_203464, and NM_001005353 is located on chromosome 1. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - regulation of oxidative phosphorylation\n",
      "    - ADP biosynthetic process\n",
      "    - nucleoside triphosphate biosynthetic process\n",
      "    - ribonucleoside diphosphate biosynthetic process\n",
      "    - nucleobase-containing small molecule interconversion\n",
      "    - AMP metabolic process\n",
      "    - AMP metabolic process\n",
      "    - ATP metabolic process\n",
      "    - GTP metabolic process\n",
      "    - nucleoside monophosphate phosphorylation\n",
      "    - cellular response to hypoxia\n",
      "    - cellular response to hypoxia\n",
      "  CC:\n",
      "    - cytoplasm\n",
      "    - mitochondrion\n",
      "    - mitochondrion\n",
      "    - mitochondrial matrix\n",
      "    - mitochondrial matrix\n",
      "    - mitochondrial matrix\n",
      "  MF:\n",
      "    - adenylate kinase activity\n",
      "    - adenylate kinase activity\n",
      "    - nucleoside diphosphate kinase activity\n",
      "    - nucleoside diphosphate kinase activity\n",
      "    - protein binding\n",
      "    - ATP binding\n",
      "    - GTP binding\n",
      "    - CMP kinase activity\n",
      "    - dCMP kinase activity\n",
      "    - nucleoside triphosphate adenylate kinase activity\n",
      "    - nucleoside triphosphate adenylate kinase activity\n",
      "    - deoxyadenylate kinase activity\n",
      "    - nucleoside monophosphate kinase activity\n",
      "UniProt: {'Swiss-Prot': 'P27144', 'TrEMBL': 'F8VS11'}\n",
      "\n",
      "Gene: PNP\n",
      "Symbol: PNP\n",
      "Name: purine nucleoside phosphorylase\n",
      "Summary: This gene encodes an enzyme which reversibly catalyzes the phosphorolysis of purine nucleosides. The enzyme is trimeric, containing three identical subunits. Mutations which result in nucleoside phosphorylase deficiency result in defective T-cell (cell-mediated) immunity but can also affect B-cell immunity and antibody responses. Neurologic disorders may also be apparent in patients with immune defects. A known polymorphism at aa position 51 that does not affect enzyme activity has been described. A pseudogene has been identified on chromosome 2. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - allantoin metabolic process\n",
      "    - nucleobase-containing compound metabolic process\n",
      "    - inosine catabolic process\n",
      "    - inosine catabolic process\n",
      "    - deoxyinosine catabolic process\n",
      "    - deoxyadenosine catabolic process\n",
      "    - purine ribonucleoside salvage\n",
      "    - IMP catabolic process\n",
      "    - nicotinamide riboside catabolic process\n",
      "    - immune response\n",
      "    - nucleotide biosynthetic process\n",
      "    - response to xenobiotic stimulus\n",
      "    - positive regulation of interleukin-2 production\n",
      "    - urate biosynthetic process\n",
      "    - positive regulation of T cell proliferation\n",
      "    - purine-containing compound salvage\n",
      "    - dAMP catabolic process\n",
      "    - positive regulation of alpha-beta T cell differentiation\n",
      "  CC:\n",
      "    - extracellular region\n",
      "    - cytoplasm\n",
      "    - cytoplasm\n",
      "    - cytosol\n",
      "    - cytosol\n",
      "    - secretory granule lumen\n",
      "    - extracellular exosome\n",
      "    - ficolin-1-rich granule lumen\n",
      "  MF:\n",
      "    - nucleoside binding\n",
      "    - purine nucleobase binding\n",
      "    - purine-nucleoside phosphorylase activity\n",
      "    - purine-nucleoside phosphorylase activity\n",
      "    - protein binding\n",
      "    - phosphate ion binding\n",
      "    - identical protein binding\n",
      "    - guanosine phosphorylase activity\n",
      "UniProt: {'Swiss-Prot': 'P00491', 'TrEMBL': ['A0A8V8TL37', 'V9HWH6', 'G3V393', 'G3V5M2', 'G3V2H3']}\n",
      "\n",
      "Gene: KDM6A\n",
      "Symbol: KDM6A\n",
      "Name: lysine demethylase 6A\n",
      "Summary: This gene is located on the X chromosome and is the corresponding locus to a Y-linked gene which encodes a tetratricopeptide repeat (TPR) protein. The encoded protein of this gene contains a JmjC-domain and catalyzes the demethylation of tri/dimethylated histone H3. Multiple alternatively spliced transcript variants have been found for this gene. [provided by RefSeq, Apr 2014].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - chromatin remodeling\n",
      "    - heart development\n",
      "    - regulation of gene expression\n",
      "  CC:\n",
      "    - nucleus\n",
      "    - nucleoplasm\n",
      "    - histone methyltransferase complex\n",
      "    - MLL3/4 complex\n",
      "    - MLL3/4 complex\n",
      "    - MLL3/4 complex\n",
      "  MF:\n",
      "    - RNA polymerase II cis-regulatory region sequence-specific DNA binding\n",
      "    - protein binding\n",
      "    - chromatin DNA binding\n",
      "    - histone demethylase activity\n",
      "    - metal ion binding\n",
      "    - histone H3K27me2/H3K27me3 demethylase activity\n",
      "UniProt: {'Swiss-Prot': 'O15550', 'TrEMBL': ['A0A087X0R0', 'B7ZKN5', 'B7ZKN1', 'F8W8R6', 'B4E0L8', 'F5H5V6', 'Q86TD1', 'B7ZKN6', 'A0A804HJA2', 'E1U0S6', 'F5H6S1', 'Q59HG3', 'A0A6Q8PGG6', 'A0A6Q8PGN0', 'A0A6Q8PFK0', 'H0Y6V5', 'A0A087WUN6', 'A0A6Q8PG92', 'A0A6Q8PHB6', 'A0A6Q8PH94', 'A0A6Q8PHJ0', 'A0A804HLJ3', 'A0A6Q8PFD0', 'A0A6Q8PG32', 'A0A804HIZ7']}\n",
      "\n",
      "Gene: ABHD16A\n",
      "Symbol: ABHD16A\n",
      "Name: abhydrolase domain containing 16A, phospholipase\n",
      "Summary: A cluster of genes, BAT1-BAT5, has been localized in the vicinity of the genes for tumor necrosis factor alpha and tumor necrosis factor beta. These genes are all within the human major histocompatibility complex class III region. The protein encoded by this gene is thought to be involved in some aspects of immunity. Alternatively spliced transcript variants have been described. [provided by RefSeq, Apr 2010].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - phosphatidylserine catabolic process\n",
      "    - phosphatidylserine catabolic process\n",
      "    - monoacylglycerol catabolic process\n",
      "    - monoacylglycerol catabolic process\n",
      "    - prostaglandin catabolic process\n",
      "  CC:\n",
      "    - membrane\n",
      "  MF:\n",
      "    - phospholipase activity\n",
      "    - phospholipase activity\n",
      "    - lysophospholipase activity\n",
      "    - protein binding\n",
      "    - monoacylglycerol lipase activity\n",
      "    - monoacylglycerol lipase activity\n",
      "UniProt: {'Swiss-Prot': 'O95870', 'TrEMBL': ['A0A1U9X777', 'B3KNX9', 'A0A0G2JI89', 'F2Z3H2', 'F2Z3Q3', 'F2Z3G6', 'A0A0G2JJD3']}\n",
      "\n",
      "Gene: SLC10A2\n",
      "Symbol: SLC10A2\n",
      "Name: solute carrier family 10 member 2\n",
      "Summary: This gene encodes a sodium/bile acid cotransporter. This transporter is the primary mechanism for uptake of intestinal bile acids by apical cells in the distal ileum. Bile acids are the catabolic product of cholesterol metabolism, so this protein is also critical for cholesterol homeostasis. Mutations in this gene cause primary bile acid malabsorption (PBAM); muatations in this gene may also be associated with other diseases of the liver and intestines, such as familial hypertriglyceridemia (FHTG). [provided by RefSeq, Mar 2010].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - sodium ion transport\n",
      "    - response to bacterium\n",
      "    - bile acid and bile salt transport\n",
      "    - bile acid and bile salt transport\n",
      "    - transmembrane transport\n",
      "  CC:\n",
      "    - plasma membrane\n",
      "    - microvillus\n",
      "    - apical plasma membrane\n",
      "  MF:\n",
      "    - protein binding\n",
      "    - bile acid:sodium symporter activity\n",
      "    - bile acid:sodium symporter activity\n",
      "UniProt: {'Swiss-Prot': 'Q12908'}\n",
      "\n",
      "Gene: DPYD\n",
      "Symbol: DPYD\n",
      "Name: dihydropyrimidine dehydrogenase\n",
      "Summary: The protein encoded by this gene is a pyrimidine catabolic enzyme and the initial and rate-limiting factor in the pathway of uracil and thymidine catabolism. Mutations in this gene result in dihydropyrimidine dehydrogenase deficiency, an error in pyrimidine metabolism associated with thymine-uraciluria and an increased risk of toxicity in cancer patients receiving 5-fluorouracil chemotherapy. Two transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, May 2009].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - purine nucleobase catabolic process\n",
      "    - pyrimidine nucleobase catabolic process\n",
      "    - pyrimidine nucleobase catabolic process\n",
      "    - thymine catabolic process\n",
      "    - thymine catabolic process\n",
      "    - uracil catabolic process\n",
      "    - uracil catabolic process\n",
      "    - thymidine catabolic process\n",
      "    - CMP catabolic process\n",
      "    - dCMP catabolic process\n",
      "    - beta-alanine biosynthetic process\n",
      "    - UMP catabolic process\n",
      "    - dUMP catabolic process\n",
      "  CC:\n",
      "    - cytoplasm\n",
      "    - cytosol\n",
      "    - cytosol\n",
      "    - cytosol\n",
      "  MF:\n",
      "    - uracil binding\n",
      "    - protein binding\n",
      "    - dihydropyrimidine dehydrogenase (NADP+) activity\n",
      "    - dihydropyrimidine dehydrogenase (NADP+) activity\n",
      "    - dihydropyrimidine dehydrogenase (NADP+) activity\n",
      "    - dihydropyrimidine dehydrogenase (NADP+) activity\n",
      "    - protein homodimerization activity\n",
      "    - protein homodimerization activity\n",
      "    - metal ion binding\n",
      "    - flavin adenine dinucleotide binding\n",
      "    - NADP binding\n",
      "    - NADP binding\n",
      "    - 4 iron, 4 sulfur cluster binding\n",
      "UniProt: {'Swiss-Prot': 'Q12882'}\n",
      "\n",
      "Gene: GSTZ1\n",
      "Symbol: GSTZ1\n",
      "Name: glutathione S-transferase zeta 1\n",
      "Summary: This gene is a member of the glutathione S-transferase (GSTs) super-family which encodes multifunctional enzymes important in the detoxification of electrophilic molecules, including carcinogens, mutagens, and several therapeutic drugs, by conjugation with glutathione. This enzyme catalyzes the conversion of maleylacetoacetate to fumarylacetoacatate, which is one of the steps in the phenylalanine/tyrosine degradation pathway. Deficiency of a similar gene in mouse causes oxidative stress. Several transcript variants of this gene encode multiple protein isoforms. [provided by RefSeq, Jul 2015].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - L-phenylalanine catabolic process\n",
      "    - L-phenylalanine catabolic process\n",
      "    - tyrosine catabolic process\n",
      "    - glutathione metabolic process\n",
      "    - glutathione metabolic process\n",
      "    - cellular oxidant detoxification\n",
      "    - cellular detoxification\n",
      "  CC:\n",
      "    - mitochondrion\n",
      "    - mitochondrion\n",
      "    - mitochondrion\n",
      "    - mitochondrion\n",
      "    - mitochondrial matrix\n",
      "    - cytosol\n",
      "    - cytosol\n",
      "  MF:\n",
      "    - glutathione transferase activity\n",
      "    - glutathione transferase activity\n",
      "    - glutathione peroxidase activity\n",
      "    - protein binding\n",
      "    - maleylacetoacetate isomerase activity\n",
      "    - maleylacetoacetate isomerase activity\n",
      "    - identical protein binding\n",
      "    - protein homodimerization activity\n",
      "UniProt: {'Swiss-Prot': 'O43708', 'TrEMBL': ['A0A0A0MR33', 'A0A0C4DFM0', 'G3V3B9', 'G3V5U6', 'G3V267', 'H0YJN8', 'G3V4T6', 'G3V5G8', 'G3V5T0']}\n",
      "\n",
      "Gene: PTPRJ\n",
      "Symbol: PTPRJ\n",
      "Name: protein tyrosine phosphatase receptor type J\n",
      "Summary: The protein encoded by this gene is a member of the protein tyrosine phosphatase (PTP) family. PTPs are known to be signaling molecules that regulate a variety of cellular processes, including cell growth, differentiation, mitotic cycle, and oncogenic transformation. This PTP possesses an extracellular region containing five fibronectin type III repeats, a single transmembrane region, and a single intracytoplasmic catalytic domain, and thus represents a receptor-type PTP. This protein is present in all hematopoietic lineages, and was shown to negatively regulate T cell receptor signaling possibly through interfering with the phosphorylation of Phospholipase C Gamma 1 and Linker for Activation of T Cells. This protein can also dephosphorylate the PDGF beta receptor, and may be involved in UV-induced signal transduction. Multiple transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - angiogenesis\n",
      "    - vasculogenesis\n",
      "    - chromatin remodeling\n",
      "    - signal transduction\n",
      "    - axon guidance\n",
      "    - heart development\n",
      "    - blood coagulation\n",
      "    - negative regulation of cell population proliferation\n",
      "    - positive regulation of platelet activation\n",
      "    - negative regulation of platelet-derived growth factor receptor signaling pathway\n",
      "    - positive regulation of macrophage chemotaxis\n",
      "    - cytokine-mediated signaling pathway\n",
      "    - regulation of cell adhesion\n",
      "    - B cell differentiation\n",
      "    - platelet formation\n",
      "    - negative regulation of cell growth\n",
      "    - negative regulation of cell migration\n",
      "    - positive regulation of tumor necrosis factor production\n",
      "    - peptidyl-tyrosine dephosphorylation\n",
      "    - peptidyl-tyrosine dephosphorylation\n",
      "    - negative regulation of epidermal growth factor receptor signaling pathway\n",
      "    - glucose homeostasis\n",
      "    - negative regulation of vascular permeability\n",
      "    - negative regulation of MAP kinase activity\n",
      "    - positive regulation of MAPK cascade\n",
      "    - positive regulation of cell adhesion\n",
      "    - negative regulation of insulin receptor signaling pathway\n",
      "    - platelet-derived growth factor receptor signaling pathway\n",
      "    - oligodendrocyte differentiation\n",
      "    - positive regulation of phagocytosis\n",
      "    - positive regulation of calcium-mediated signaling\n",
      "    - T cell receptor signaling pathway\n",
      "    - negative regulation of T cell receptor signaling pathway\n",
      "    - negative regulation of T cell receptor signaling pathway\n",
      "    - positive chemotaxis\n",
      "    - positive regulation of focal adhesion assembly\n",
      "    - positive regulation of phosphatidylinositol 3-kinase/protein kinase B signal transduction\n",
      "    - negative regulation of phosphatidylinositol 3-kinase/protein kinase B signal transduction\n",
      "    - contact inhibition\n",
      "    - positive regulation of Fc receptor mediated stimulatory signaling pathway\n",
      "  CC:\n",
      "    - immunological synapse\n",
      "    - nucleoplasm\n",
      "    - nucleolus\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - cell-cell junction\n",
      "    - cell surface\n",
      "    - nuclear body\n",
      "    - cell junction\n",
      "    - ruffle membrane\n",
      "    - specific granule membrane\n",
      "    - extracellular exosome\n",
      "  MF:\n",
      "    - protein tyrosine phosphatase activity\n",
      "    - protein tyrosine phosphatase activity\n",
      "    - protein tyrosine phosphatase activity\n",
      "    - protein tyrosine phosphatase activity\n",
      "    - non-membrane spanning protein tyrosine phosphatase activity\n",
      "    - platelet-derived growth factor receptor binding\n",
      "    - protein binding\n",
      "    - beta-catenin binding\n",
      "    - phosphatase activity\n",
      "    - phosphatase activity\n",
      "    - protein kinase binding\n",
      "    - protein tyrosine phosphatase activity, metal-dependent\n",
      "    - gamma-catenin binding\n",
      "    - cadherin binding\n",
      "    - mitogen-activated protein kinase binding\n",
      "    - delta-catenin binding\n",
      "    - histone H2AXY142 phosphatase activity\n",
      "UniProt: {'Swiss-Prot': 'Q12913', 'TrEMBL': ['E9PPH3', 'Q9NPR5', 'E9PJ83', 'A0A8V8TP51']}\n",
      "\n",
      "Gene: PSMB1\n",
      "Symbol: PSMB1\n",
      "Name: proteasome 20S subunit beta 1\n",
      "Summary: The proteasome is a multicatalytic proteinase complex with a highly ordered ring-shaped 20S core structure. The core structure is composed of 4 rings of 28 non-identical subunits; 2 rings are composed of 7 alpha subunits and 2 rings are composed of 7 beta subunits. Proteasomes are distributed throughout eukaryotic cells at a high concentration and cleave peptides in an ATP/ubiquitin-dependent process in a non-lysosomal pathway.  An essential function of a modified proteasome, the immunoproteasome, is the processing of class I MHC peptides. This gene encodes a member of the proteasome B-type family, also known as the T1B family, that is a 20S core beta subunit. This gene is tightly linked to the TBP (TATA-binding protein) gene in human and in mouse, and is transcribed in the opposite orientation in both species. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - proteasome-mediated ubiquitin-dependent protein catabolic process\n",
      "    - proteolysis involved in protein catabolic process\n",
      "  CC:\n",
      "    - proteasome complex\n",
      "    - proteasome complex\n",
      "    - extracellular region\n",
      "    - nucleus\n",
      "    - nucleus\n",
      "    - nucleus\n",
      "    - nucleoplasm\n",
      "    - nucleoplasm\n",
      "    - cytoplasm\n",
      "    - cytoplasm\n",
      "    - cytosol\n",
      "    - proteasome core complex\n",
      "    - proteasome core complex\n",
      "    - proteasome core complex\n",
      "    - proteasome core complex, beta-subunit complex\n",
      "    - secretory granule lumen\n",
      "    - extracellular exosome\n",
      "    - ficolin-1-rich granule lumen\n",
      "  MF:\n",
      "    - protein binding\n",
      "UniProt: {'Swiss-Prot': 'P20618', 'TrEMBL': 'A0A140VK45'}\n",
      "\n",
      "Gene: KLF15\n",
      "Symbol: KLF15\n",
      "Name: KLF transcription factor 15\n",
      "Summary: Enables DNA-binding transcription activator activity, RNA polymerase II-specific and RNA polymerase II cis-regulatory region sequence-specific DNA binding activity. Involved in negative regulation of peptidyl-lysine acetylation and positive regulation of transcription by RNA polymerase II. Located in nuclear speck. [provided by Alliance of Genome Resources, Apr 2022]\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - intracellular glucose homeostasis\n",
      "    - regulation of transcription by RNA polymerase II\n",
      "    - glial cell differentiation\n",
      "    - cardiac muscle hypertrophy in response to stress\n",
      "    - regulation of Wnt signaling pathway\n",
      "    - response to insulin\n",
      "    - positive regulation of transcription by RNA polymerase II\n",
      "    - positive regulation of D-glucose import\n",
      "    - podocyte differentiation\n",
      "    - cellular response to peptide\n",
      "    - negative regulation of peptidyl-lysine acetylation\n",
      "  CC:\n",
      "    - chromatin\n",
      "    - nucleus\n",
      "    - nucleoplasm\n",
      "    - nuclear speck\n",
      "    - intracellular membrane-bounded organelle\n",
      "  MF:\n",
      "    - transcription cis-regulatory region binding\n",
      "    - RNA polymerase II transcription regulatory region sequence-specific DNA binding\n",
      "    - RNA polymerase II cis-regulatory region sequence-specific DNA binding\n",
      "    - RNA polymerase II cis-regulatory region sequence-specific DNA binding\n",
      "    - DNA-binding transcription factor activity, RNA polymerase II-specific\n",
      "    - DNA-binding transcription factor activity, RNA polymerase II-specific\n",
      "    - DNA-binding transcription activator activity, RNA polymerase II-specific\n",
      "    - protein binding\n",
      "    - zinc ion binding\n",
      "    - sequence-specific double-stranded DNA binding\n",
      "UniProt: {'Swiss-Prot': 'Q9UIH9'}\n",
      "\n",
      "Gene: LRRK2\n",
      "Symbol: LRRK2\n",
      "Name: leucine rich repeat kinase 2\n",
      "Summary: This gene is a member of the leucine-rich repeat kinase family and encodes a protein with an ankryin repeat region, a leucine-rich repeat (LRR) domain, a kinase domain, a DFG-like motif, a RAS domain, a GTPase domain, a MLK-like domain, and a WD40 domain. The protein is present largely in the cytoplasm but also associates with the mitochondrial outer membrane. Mutations in this gene have been associated with Parkinson disease-8. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - MAPK cascade\n",
      "    - positive regulation of protein phosphorylation\n",
      "    - chromatin remodeling\n",
      "    - protein phosphorylation\n",
      "    - protein import into nucleus\n",
      "    - endocytosis\n",
      "    - endocytosis\n",
      "    - autophagy\n",
      "    - response to oxidative stress\n",
      "    - mitochondrion organization\n",
      "    - endoplasmic reticulum organization\n",
      "    - endoplasmic reticulum organization\n",
      "    - Golgi organization\n",
      "    - Golgi organization\n",
      "    - lysosome organization\n",
      "    - JNK cascade\n",
      "    - Rho protein signal transduction\n",
      "    - spermatogenesis\n",
      "    - neuromuscular junction development\n",
      "    - protein localization\n",
      "    - determination of adult lifespan\n",
      "    - cellular response to starvation\n",
      "    - regulation of autophagy\n",
      "    - positive regulation of autophagy\n",
      "    - positive regulation of autophagy\n",
      "    - positive regulation of autophagy\n",
      "    - negative regulation of protein processing\n",
      "    - negative regulation of neuron projection development\n",
      "    - regulation of neuron maturation\n",
      "    - negative regulation of macroautophagy\n",
      "    - phosphorylation\n",
      "    - peptidyl-serine phosphorylation\n",
      "    - peptidyl-threonine phosphorylation\n",
      "    - peptidyl-threonine phosphorylation\n",
      "    - calcium-mediated signaling\n",
      "    - striatum development\n",
      "    - olfactory bulb development\n",
      "    - tangential migration from the subventricular zone to the olfactory bulb\n",
      "    - regulation of cell projection organization\n",
      "    - positive regulation of protein ubiquitination\n",
      "    - regulation of protein stability\n",
      "    - negative regulation of protein binding\n",
      "    - positive regulation of protein binding\n",
      "    - positive regulation of proteasomal ubiquitin-dependent protein catabolic process\n",
      "    - positive regulation of tumor necrosis factor production\n",
      "    - negative regulation of GTPase activity\n",
      "    - cellular response to oxidative stress\n",
      "    - cellular response to reactive oxygen species\n",
      "    - intracellular signal transduction\n",
      "    - intracellular signal transduction\n",
      "    - regulation of kidney size\n",
      "    - exploration behavior\n",
      "    - locomotory exploration behavior\n",
      "    - regulation of lysosomal lumen pH\n",
      "    - regulation of locomotion\n",
      "    - regulation of membrane potential\n",
      "    - positive regulation of programmed cell death\n",
      "    - positive regulation of MAP kinase activity\n",
      "    - positive regulation of MAP kinase activity\n",
      "    - negative regulation of Notch signaling pathway\n",
      "    - positive regulation of protein kinase activity\n",
      "    - GTP metabolic process\n",
      "    - protein autophosphorylation\n",
      "    - protein autophosphorylation\n",
      "    - intracellular distribution of mitochondria\n",
      "    - neuron projection morphogenesis\n",
      "    - mitochondrion localization\n",
      "    - regulation of mitochondrial depolarization\n",
      "    - regulation of synaptic transmission, glutamatergic\n",
      "    - canonical Wnt signaling pathway\n",
      "    - excitatory postsynaptic potential\n",
      "    - regulation of dopamine receptor signaling pathway\n",
      "    - regulation of dopamine receptor signaling pathway\n",
      "    - positive regulation of dopamine receptor signaling pathway\n",
      "    - regulation of ER to Golgi vesicle-mediated transport\n",
      "    - regulation of canonical Wnt signaling pathway\n",
      "    - regulation of canonical Wnt signaling pathway\n",
      "    - regulation of dendritic spine morphogenesis\n",
      "    - regulation of dendritic spine morphogenesis\n",
      "    - protein localization to mitochondrion\n",
      "    - protein localization to endoplasmic reticulum exit site\n",
      "    - cellular response to manganese ion\n",
      "    - regulation of mitochondrial fission\n",
      "    - positive regulation of canonical Wnt signaling pathway\n",
      "    - negative regulation of excitatory postsynaptic potential\n",
      "    - neuron projection arborization\n",
      "    - regulation of cAMP/PKA signal transduction\n",
      "    - regulation of synaptic vesicle endocytosis\n",
      "    - positive regulation of synaptic vesicle endocytosis\n",
      "    - negative regulation of endoplasmic reticulum stress-induced intrinsic apoptotic signaling pathway\n",
      "    - positive regulation of protein autoubiquitination\n",
      "    - regulation of neuroblast proliferation\n",
      "    - regulation of synaptic vesicle transport\n",
      "    - regulation of synaptic vesicle transport\n",
      "    - negative regulation of late endosome to lysosome transport\n",
      "    - negative regulation of autophagosome assembly\n",
      "    - negative regulation of thioredoxin peroxidase activity by peptidyl-threonine phosphorylation\n",
      "    - negative regulation of protein targeting to mitochondrion\n",
      "    - negative regulation of protein processing involved in protein targeting to mitochondrion\n",
      "    - cellular response to dopamine\n",
      "    - positive regulation of microglial cell activation\n",
      "    - Wnt signalosome assembly\n",
      "    - Wnt signalosome assembly\n",
      "    - regulation of retrograde transport, endosome to Golgi\n",
      "    - regulation of CAMKK-AMPK signaling cascade\n",
      "    - regulation of branching morphogenesis of a nerve\n",
      "    - regulation of synaptic vesicle exocytosis\n",
      "    - regulation of reactive oxygen species metabolic process\n",
      "  CC:\n",
      "    - Golgi membrane\n",
      "    - extracellular space\n",
      "    - nucleoplasm\n",
      "    - cytoplasm\n",
      "    - cytoplasm\n",
      "    - cytoplasm\n",
      "    - mitochondrion\n",
      "    - mitochondrial outer membrane\n",
      "    - mitochondrial inner membrane\n",
      "    - mitochondrial matrix\n",
      "    - lysosome\n",
      "    - endosome\n",
      "    - endoplasmic reticulum\n",
      "    - endoplasmic reticulum\n",
      "    - endoplasmic reticulum membrane\n",
      "    - Golgi apparatus\n",
      "    - Golgi apparatus\n",
      "    - Golgi-associated vesicle\n",
      "    - trans-Golgi network\n",
      "    - cytosol\n",
      "    - cytosol\n",
      "    - plasma membrane\n",
      "    - microvillus\n",
      "    - axon\n",
      "    - axon\n",
      "    - dendrite\n",
      "    - dendrite\n",
      "    - growth cone\n",
      "    - synaptic vesicle membrane\n",
      "    - cytoplasmic vesicle\n",
      "    - nuclear membrane\n",
      "    - mitochondrial membrane\n",
      "    - cytoplasmic side of mitochondrial outer membrane\n",
      "    - dendrite cytoplasm\n",
      "    - ciliary basal body\n",
      "    - neuron projection\n",
      "    - neuronal cell body\n",
      "    - terminal bouton\n",
      "    - perikaryon\n",
      "    - perikaryon\n",
      "    - intracellular membrane-bounded organelle\n",
      "    - amphisome\n",
      "    - autolysosome\n",
      "    - phagocytic vesicle\n",
      "    - extracellular exosome\n",
      "    - endoplasmic reticulum exit site\n",
      "    - multivesicular body, internal vesicle\n",
      "    - postsynapse\n",
      "    - glutamatergic synapse\n",
      "    - caveola neck\n",
      "    - presynaptic cytosol\n",
      "    - ribonucleoprotein complex\n",
      "    - Wnt signalosome\n",
      "    - Wnt signalosome\n",
      "  MF:\n",
      "    - SNARE binding\n",
      "    - magnesium ion binding\n",
      "    - actin binding\n",
      "    - GTPase activity\n",
      "    - protein kinase activity\n",
      "    - protein serine/threonine kinase activity\n",
      "    - protein serine/threonine kinase activity\n",
      "    - protein serine/threonine kinase activity\n",
      "    - protein serine/threonine kinase activity\n",
      "    - 3-phosphoinositide-dependent protein kinase activity\n",
      "    - DNA-dependent protein kinase activity\n",
      "    - AMP-activated protein kinase activity\n",
      "    - eukaryotic translation initiation factor 2alpha kinase activity\n",
      "    - JUN kinase kinase kinase activity\n",
      "    - JUN kinase kinase kinase activity\n",
      "    - MAP kinase kinase kinase activity\n",
      "    - ribosomal protein S6 kinase activity\n",
      "    - GTPase activator activity\n",
      "    - protein binding\n",
      "    - ATP binding\n",
      "    - GTP binding\n",
      "    - GTP binding\n",
      "    - microtubule binding\n",
      "    - tubulin binding\n",
      "    - kinase activity\n",
      "    - kinase activity\n",
      "    - syntaxin-1 binding\n",
      "    - signaling receptor complex adaptor activity\n",
      "    - clathrin binding\n",
      "    - small GTPase binding\n",
      "    - GTP-dependent protein kinase activity\n",
      "    - GTP-dependent protein kinase activity\n",
      "    - histone H3S10 kinase activity\n",
      "    - histone H3T11 kinase activity\n",
      "    - histone H3T6 kinase activity\n",
      "    - histone H2AXS139 kinase activity\n",
      "    - peroxidase inhibitor activity\n",
      "    - co-receptor binding\n",
      "    - identical protein binding\n",
      "    - protein homodimerization activity\n",
      "    - histone H3S28 kinase activity\n",
      "    - histone H4S1 kinase activity\n",
      "    - histone H2AS1 kinase activity\n",
      "    - histone H2BS14 kinase activity\n",
      "    - transmembrane transporter binding\n",
      "    - protein kinase A binding\n",
      "    - histone H3T3 kinase activity\n",
      "    - histone H2AS121 kinase activity\n",
      "    - Rho-dependent protein serine/threonine kinase activity\n",
      "    - protein serine kinase activity\n",
      "    - histone H2BS36 kinase activity\n",
      "    - histone H3S57 kinase activity\n",
      "    - histone H3T45 kinase activity\n",
      "    - beta-catenin destruction complex binding\n",
      "    - histone H2AT120 kinase activity\n",
      "UniProt: {'Swiss-Prot': 'Q5S007', 'TrEMBL': ['A0A7P0Z468', 'Q17RV3', 'C9JBF0', 'A0A7P0T8T5', 'A0A7P0T9F8', 'A0A2R8Y4F8', 'A0A7P0T8T1', 'A0A7P0TAW6', 'A0A1B0GUQ3', 'A0A7P0TB00', 'A0A7P0T8S1', 'H7C3B6', 'A0A7P0Z4D9', 'A0A7P0TAJ2', 'E9PC85']}\n",
      "\n",
      "Gene: KRTAP4-5\n",
      "Symbol: KRTAP4-5\n",
      "Name: keratin associated protein 4-5\n",
      "Summary: This protein is a member of the keratin-associated protein (KAP) family. The KAP proteins form a matrix of keratin intermediate filaments which contribute to the structure of hair fibers. KAP family members appear to have unique, family-specific amino- and carboxyl-terminal regions and are subdivided into three multi-gene families according to amino acid composition: the high sulfur, the ultrahigh sulfur, and the high tyrosine/glycine KAPs. This protein is a member of the ultrahigh sulfur KAP family and the gene is localized to a cluster of KAPs at 17q12-q21. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - hair cycle\n",
      "  CC:\n",
      "    - cytosol\n",
      "    - keratin filament\n",
      "  MF:\n",
      "    - protein binding\n",
      "UniProt: {'Swiss-Prot': 'Q9BYR2'}\n",
      "\n",
      "Gene: CYP4F2\n",
      "Symbol: CYP4F2\n",
      "Name: cytochrome P450 family 4 subfamily F member 2\n",
      "Summary: This gene encodes a member of the cytochrome P450 superfamily of enzymes. The cytochrome P450 proteins are monooxygenases which catalyze many reactions involved in drug metabolism and synthesis of cholesterol, steroids and other lipids. This protein localizes to the endoplasmic reticulum. The enzyme starts the process of inactivating and degrading leukotriene B4, a potent mediator of inflammation. This gene is part of a cluster of cytochrome P450 genes on chromosome 19. Another member of this family, CYP4F11, is approximately 16 kb away. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - very long-chain fatty acid metabolic process\n",
      "    - long-chain fatty acid metabolic process\n",
      "    - renal water homeostasis\n",
      "    - pressure natriuresis\n",
      "    - icosanoid metabolic process\n",
      "    - leukotriene metabolic process\n",
      "    - xenobiotic metabolic process\n",
      "    - blood coagulation\n",
      "    - regulation of blood pressure\n",
      "    - fatty acid omega-oxidation\n",
      "    - arachidonate metabolic process\n",
      "    - arachidonate metabolic process\n",
      "    - epoxygenase P450 pathway\n",
      "    - negative regulation of icosanoid secretion\n",
      "    - positive regulation of icosanoid secretion\n",
      "    - leukotriene B4 catabolic process\n",
      "    - vitamin E metabolic process\n",
      "    - menaquinone catabolic process\n",
      "    - menaquinone catabolic process\n",
      "    - phylloquinone catabolic process\n",
      "    - phylloquinone catabolic process\n",
      "    - phylloquinone catabolic process\n",
      "    - vitamin K catabolic process\n",
      "    - sodium ion homeostasis\n",
      "    - omega-hydroxylase P450 pathway\n",
      "    - omega-hydroxylase P450 pathway\n",
      "  CC:\n",
      "    - cytoplasm\n",
      "    - endoplasmic reticulum membrane\n",
      "    - apical plasma membrane\n",
      "    - intracellular membrane-bounded organelle\n",
      "  MF:\n",
      "    - monooxygenase activity\n",
      "    - iron ion binding\n",
      "    - protein binding\n",
      "    - arachidonate epoxygenase activity\n",
      "    - oxidoreductase activity, acting on paired donors, with incorporation or reduction of molecular oxygen, NAD(P)H as one donor, and incorporation of one atom of oxygen\n",
      "    - alkane 1-monooxygenase activity\n",
      "    - heme binding\n",
      "    - leukotriene-B4 20-monooxygenase activity\n",
      "    - leukotriene-B4 20-monooxygenase activity\n",
      "    - arachidonate omega-hydroxylase activity\n",
      "    - alpha-tocopherol omega-hydroxylase activity\n",
      "    - 20-hydroxy-leukotriene B4 omega oxidase activity\n",
      "    - 20-aldehyde-leukotriene B4 20-monooxygenase activity\n",
      "    - long-chain fatty acid omega-hydroxylase activity\n",
      "UniProt: {'Swiss-Prot': 'P78329', 'TrEMBL': ['K7EPM0', 'A0A0A0MQR0', 'K7EQI8', 'K7EK90']}\n",
      "\n",
      "Gene: DHRS2\n",
      "Symbol: DHRS2\n",
      "Name: dehydrogenase/reductase 2\n",
      "Summary: This gene encodes a member of the short-chain dehydrogenases/reductases (SDR) family, which has over 46,000 members. Members of this family are enzymes that metabolize many different compounds, such as steroid hormones, prostaglandins, retinoids, lipids and xenobiotics. Alternative promoter use and alternative splicing results in multiple transcript variants. [provided by RefSeq, Feb 2017].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - C21-steroid hormone metabolic process\n",
      "    - negative regulation of cell population proliferation\n",
      "    - response to toxic substance\n",
      "    - electron transport chain\n",
      "    - cellular response to oxidative stress\n",
      "    - myeloid dendritic cell differentiation\n",
      "    - negative regulation of apoptotic process\n",
      "  CC:\n",
      "    - nucleus\n",
      "    - nuclear envelope\n",
      "    - nucleoplasm\n",
      "    - cytoplasm\n",
      "    - mitochondrion\n",
      "    - mitochondrion\n",
      "    - mitochondrial matrix\n",
      "  MF:\n",
      "    - carbonyl reductase (NADPH) activity\n",
      "    - carbonyl reductase (NADPH) activity\n",
      "    - protein binding\n",
      "    - oxidoreductase activity, acting on the CH-OH group of donors, NAD or NADP as acceptor\n",
      "UniProt: {'Swiss-Prot': 'Q13268', 'TrEMBL': ['H0YJG9', 'C9JZP6', 'H0YJP4']}\n",
      "\n",
      "Gene: KRT27\n",
      "Symbol: KRT27\n",
      "Name: keratin 27\n",
      "Summary: This gene encodes a member of the type I (acidic) keratin family, which belongs to the superfamily of intermediate filament (IF) proteins. Keratins are heteropolymeric structural proteins which form the intermediate filament. These filaments, along with actin microfilaments and microtubules, compose the cytoskeleton of epithelial cells. The type I keratin genes are clustered in a region of chromosome 17q12-q21. [provided by RefSeq, Jul 2009].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - epithelial cell differentiation\n",
      "    - hair follicle morphogenesis\n",
      "    - intermediate filament organization\n",
      "  CC:\n",
      "    - cytosol\n",
      "    - cytoskeleton\n",
      "    - intermediate filament\n",
      "    - extracellular exosome\n",
      "  MF:\n",
      "    - structural molecule activity\n",
      "    - protein binding\n",
      "UniProt: {'Swiss-Prot': 'Q7Z3Y8'}\n",
      "\n",
      "Gene: SLC47A1\n",
      "Symbol: SLC47A1\n",
      "Name: solute carrier family 47 member 1\n",
      "Summary: This gene is located within the Smith-Magenis syndrome region on chromosome 17. It encodes a protein of unknown function. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - xenobiotic transmembrane transport\n",
      "    - organic cation transport\n",
      "    - organic cation transport\n",
      "    - putrescine transport\n",
      "    - xenobiotic transport\n",
      "    - transmembrane transport\n",
      "    - thiamine transmembrane transport\n",
      "    - amino acid import across plasma membrane\n",
      "    - L-arginine import across plasma membrane\n",
      "    - L-alpha-amino acid transmembrane transport\n",
      "    - proton transmembrane transport\n",
      "    - L-arginine transmembrane transport\n",
      "    - xenobiotic detoxification by transmembrane export across the plasma membrane\n",
      "  CC:\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - membrane\n",
      "    - basolateral plasma membrane\n",
      "    - apical plasma membrane\n",
      "  MF:\n",
      "    - protein binding\n",
      "    - organic cation transmembrane transporter activity\n",
      "    - L-amino acid transmembrane transporter activity\n",
      "    - thiamine transmembrane transporter activity\n",
      "    - antiporter activity\n",
      "    - putrescine transmembrane transporter activity\n",
      "    - transmembrane transporter activity\n",
      "    - xenobiotic transmembrane transporter activity\n",
      "    - xenobiotic transmembrane transporter activity\n",
      "    - L-arginine transmembrane transporter activity\n",
      "    - polyspecific organic cation:proton antiporter activity\n",
      "UniProt: {'Swiss-Prot': 'Q96FL8', 'TrEMBL': ['I3L345', 'J3KSS8', 'E7EX57', 'J3KRZ3']}\n",
      "\n",
      "Gene: AAGAB\n",
      "Symbol: AAGAB\n",
      "Name: alpha and gamma adaptin binding protein\n",
      "Summary: The protein encoded by this gene interacts with the gamma-adaptin and alpha-adaptin subunits of complexes involved in clathrin-coated vesicle trafficking. Mutations in this gene are associated with type I punctate palmoplantar keratoderma. Alternatively spliced transcript variants have been found for this gene. [provided by RefSeq, Dec 2012].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - protein transport\n",
      "  CC:\n",
      "    - cytoplasm\n",
      "    - cytosol\n",
      "    - nuclear speck\n",
      "  MF:\n",
      "    - protein binding\n",
      "UniProt: {'Swiss-Prot': 'Q6PD74', 'TrEMBL': 'H0YL49'}\n",
      "\n",
      "Gene: GSC2\n",
      "Symbol: GSC2\n",
      "Name: goosecoid homeobox 2\n",
      "Summary: Goosecoidlike (GSCL), a homeodomain-containing gene, resides in the critical region for VCFS/DGS on 22q11. Velocardiofacial syndrome (VCFS) is a developmental disorder characterized by conotruncal heart defects, craniofacial anomalies, and learning disabilities. VCFS is phenotypically related to DiGeorge syndrome (DGS) and both syndromes are associated with hemizygous 22q11 deletions. Because many of the tissues and structures affected in VCFS/DGS derive from the pharyngeal arches of the developing embryo, it is believed that haploinsufficiency of a gene involved in embryonic development may be responsible for its etiology. The gene is expressed in a limited number of adult tissues, as well as in early human development. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - regulation of transcription by RNA polymerase II\n",
      "    - regulation of transcription by RNA polymerase II\n",
      "    - anatomical structure morphogenesis\n",
      "  CC:\n",
      "    - chromatin\n",
      "    - nucleus\n",
      "  MF:\n",
      "    - RNA polymerase II cis-regulatory region sequence-specific DNA binding\n",
      "    - DNA-binding transcription factor activity, RNA polymerase II-specific\n",
      "    - DNA-binding transcription factor activity, RNA polymerase II-specific\n",
      "    - DNA binding\n",
      "    - DNA-binding transcription factor activity\n",
      "    - protein binding\n",
      "    - sequence-specific DNA binding\n",
      "    - sequence-specific double-stranded DNA binding\n",
      "UniProt: {'Swiss-Prot': 'O15499'}\n",
      "\n",
      "Gene: LAP3\n",
      "Symbol: LAP3\n",
      "Name: leucine aminopeptidase 3\n",
      "Summary: Predicted to enable peptidase activity. Predicted to be involved in proteolysis. Located in cytosol; midbody; and nucleoplasm. [provided by Alliance of Genome Resources, Apr 2022]\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - proteolysis\n",
      "    - proteolysis\n",
      "  CC:\n",
      "    - nucleus\n",
      "    - cytoplasm\n",
      "    - mitochondrion\n",
      "    - focal adhesion\n",
      "    - extracellular exosome\n",
      "  MF:\n",
      "    - aminopeptidase activity\n",
      "    - carboxypeptidase activity\n",
      "    - protein binding\n",
      "    - peptidase activity\n",
      "    - metalloexopeptidase activity\n",
      "    - manganese ion binding\n",
      "    - metalloaminopeptidase activity\n",
      "UniProt: {'Swiss-Prot': 'P28838', 'TrEMBL': 'H0Y9Q1'}\n",
      "\n",
      "Gene: CEACAM6\n",
      "Symbol: CEACAM6\n",
      "Name: CEA cell adhesion molecule 6\n",
      "Summary: This gene encodes a protein that belongs to the carcinoembryonic antigen (CEA) family whose members are glycosyl phosphatidyl inositol (GPI) anchored cell surface glycoproteins. Members of this family play a role in cell adhesion and are widely used as tumor markers in serum immunoassay determinations of carcinoma. This gene affects the sensitivity of tumor cells to adenovirus infection. The protein encoded by this gene acts as a receptor for adherent-invasive E. coli adhesion to the surface of ileal epithelial cells in patients with Crohn's disease. This gene is clustered with genes and pseudogenes of the cell adhesion molecules subgroup of the CEA family on chromosome 19. [provided by RefSeq, Apr 2014].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - apoptotic process\n",
      "    - homophilic cell adhesion via plasma membrane adhesion molecules\n",
      "    - heterophilic cell-cell adhesion via plasma membrane cell adhesion molecules\n",
      "    - heterophilic cell-cell adhesion via plasma membrane cell adhesion molecules\n",
      "    - positive regulation of cell population proliferation\n",
      "    - positive regulation of cell migration\n",
      "    - positive regulation of heterotypic cell-cell adhesion\n",
      "    - positive regulation of endothelial cell-matrix adhesion via fibronectin\n",
      "    - negative regulation of anoikis\n",
      "  CC:\n",
      "    - extracellular space\n",
      "    - plasma membrane\n",
      "    - cell surface\n",
      "    - cell surface\n",
      "    - cell surface\n",
      "    - membrane\n",
      "    - apical plasma membrane\n",
      "    - azurophil granule membrane\n",
      "    - side of membrane\n",
      "  MF:\n",
      "    - protein binding\n",
      "    - identical protein binding\n",
      "    - identical protein binding\n",
      "    - protein heterodimerization activity\n",
      "UniProt: {'Swiss-Prot': 'P40199', 'TrEMBL': 'M0QYD3'}\n",
      "\n",
      "Gene: MTNR1B\n",
      "Symbol: MTNR1B\n",
      "Name: melatonin receptor 1B\n",
      "Summary: This gene encodes one of two high affinity forms of a receptor for melatonin, the primary hormone secreted by the pineal gland. This gene product is an integral membrane protein that is a G-protein coupled, 7-transmembrane receptor. It is found primarily in the retina and brain although this detection requires RT-PCR. It is thought to participate in light-dependent functions in the retina and may be involved in the neurobiological effects of melatonin. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - G protein-coupled receptor signaling pathway\n",
      "    - G protein-coupled receptor signaling pathway, coupled to cyclic nucleotide second messenger\n",
      "    - chemical synaptic transmission\n",
      "    - negative regulation of cGMP-mediated signaling\n",
      "    - glucose homeostasis\n",
      "    - camera-type eye development\n",
      "    - negative regulation of neuron apoptotic process\n",
      "    - negative regulation of vasoconstriction\n",
      "    - positive regulation of circadian sleep/wake cycle, non-REM sleep\n",
      "    - negative regulation of insulin secretion\n",
      "    - regulation of insulin secretion\n",
      "    - negative regulation of cytosolic calcium ion concentration\n",
      "    - negative regulation of transmission of nerve impulse\n",
      "    - positive regulation of transmission of nerve impulse\n",
      "    - regulation of neuronal action potential\n",
      "  CC:\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - synapse\n",
      "  MF:\n",
      "    - G protein-coupled receptor activity\n",
      "    - protein binding\n",
      "    - melatonin receptor activity\n",
      "UniProt: {'Swiss-Prot': 'P49286', 'TrEMBL': ['H0YDG4', 'E9PR36']}\n",
      "\n",
      "Gene: KIF3C\n",
      "Symbol: KIF3C\n",
      "Name: kinesin family member 3C\n",
      "Summary: Predicted to enable microtubule binding activity and microtubule motor activity. Predicted to be involved in microtubule-based movement. Predicted to act upstream of or within organelle transport along microtubule. Predicted to be located in microtubule cytoskeleton; neuronal cell body; and neuronal ribonucleoprotein granule. Predicted to be part of kinesin complex. Predicted to be active in microtubule. [provided by Alliance of Genome Resources, Apr 2022]\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - microtubule-based movement\n",
      "  CC:\n",
      "    - cytoplasm\n",
      "    - cytosol\n",
      "    - kinesin complex\n",
      "    - microtubule\n",
      "    - cilium\n",
      "    - ciliary tip\n",
      "  MF:\n",
      "    - cytoskeletal motor activity\n",
      "    - microtubule motor activity\n",
      "    - protein binding\n",
      "    - ATP binding\n",
      "    - microtubule binding\n",
      "    - ATP hydrolysis activity\n",
      "UniProt: {'Swiss-Prot': 'O14782', 'TrEMBL': ['F8WER6', 'A2RU78', 'F8WAR6']}\n",
      "\n",
      "Gene: FER\n",
      "Symbol: FER\n",
      "Name: FER tyrosine kinase\n",
      "Summary: The protein encoded by this gene is a member of the FPS/FES family of non-transmembrane receptor tyrosine kinases. It regulates cell-cell adhesion and mediates signaling from the cell surface to the cytoskeleton via growth factor receptors. Alternative splicing results in multiple transcript variants. A related pseudogene has been identified on chromosome X. [provided by RefSeq, Apr 2015].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - microtubule cytoskeleton organization\n",
      "    - regulation of protein phosphorylation\n",
      "    - protein phosphorylation\n",
      "    - chemotaxis\n",
      "    - cell adhesion\n",
      "    - tyrosine phosphorylation of STAT protein\n",
      "    - germ cell development\n",
      "    - positive regulation of cell population proliferation\n",
      "    - positive regulation of cell population proliferation\n",
      "    - insulin receptor signaling pathway\n",
      "    - regulation of lamellipodium assembly\n",
      "    - regulation of fibroblast migration\n",
      "    - peptidyl-tyrosine phosphorylation\n",
      "    - cytokine-mediated signaling pathway\n",
      "    - actin cytoskeleton organization\n",
      "    - positive regulation of cell migration\n",
      "    - positive regulation of actin filament polymerization\n",
      "    - response to lipopolysaccharide\n",
      "    - negative regulation of mast cell activation involved in immune response\n",
      "    - adherens junction assembly\n",
      "    - substrate adhesion-dependent cell spreading\n",
      "    - cellular response to reactive oxygen species\n",
      "    - extracellular matrix-cell signaling\n",
      "    - intracellular signal transduction\n",
      "    - cellular response to macrophage colony-stimulating factor stimulus\n",
      "    - response to platelet-derived growth factor\n",
      "    - Fc-epsilon receptor signaling pathway\n",
      "    - Fc-epsilon receptor signaling pathway\n",
      "    - Kit signaling pathway\n",
      "    - regulation of epidermal growth factor receptor signaling pathway\n",
      "    - cell-cell adhesion mediated by cadherin\n",
      "    - protein autophosphorylation\n",
      "    - platelet-derived growth factor receptor signaling pathway\n",
      "    - platelet-derived growth factor receptor signaling pathway\n",
      "    - diapedesis\n",
      "    - positive regulation of NF-kappaB transcription factor activity\n",
      "    - positive regulation of phosphatidylinositol 3-kinase/protein kinase B signal transduction\n",
      "    - Sertoli cell development\n",
      "    - interleukin-6-mediated signaling pathway\n",
      "    - seminiferous tubule development\n",
      "    - adherens junction disassembly\n",
      "  CC:\n",
      "    - chromatin\n",
      "    - nucleus\n",
      "    - cytoplasm\n",
      "    - cytosol\n",
      "    - cytosol\n",
      "    - plasma membrane\n",
      "    - adherens junction\n",
      "    - cell cortex\n",
      "    - cytoplasmic side of plasma membrane\n",
      "    - actin cytoskeleton\n",
      "    - microtubule cytoskeleton\n",
      "    - lamellipodium\n",
      "    - cell junction\n",
      "    - protein-containing complex\n",
      "  MF:\n",
      "    - protein tyrosine kinase activity\n",
      "    - non-membrane spanning protein tyrosine kinase activity\n",
      "    - epidermal growth factor receptor binding\n",
      "    - protein binding\n",
      "    - ATP binding\n",
      "    - protein phosphatase 1 binding\n",
      "    - lipid binding\n",
      "UniProt: {'Swiss-Prot': 'P16591', 'TrEMBL': ['W0S0X4', 'W0S1B5', 'Q6PEJ9', 'W0S4B9', 'D6RAF9']}\n",
      "\n",
      "Gene: BEST1\n",
      "Symbol: BEST1\n",
      "Name: bestrophin 1\n",
      "Summary: This gene encodes a member of the bestrophin gene family. This small gene family is characterized by proteins with a highly conserved N-terminus with four to six transmembrane domains. Bestrophins may form chloride ion channels or may regulate voltage-gated L-type calcium-ion channels. Bestrophins are generally believed to form calcium-activated chloride-ion channels in epithelial cells but they have also been shown to be highly permeable to bicarbonate ion transport in retinal tissue. Mutations in this gene are responsible for juvenile-onset vitelliform macular dystrophy (VMD2), also known as Best macular dystrophy, in addition to adult-onset vitelliform macular dystrophy (AVMD) and other retinopathies. Alternative splicing results in multiple variants encoding distinct isoforms.[provided by RefSeq, Nov 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - chloride transport\n",
      "    - visual perception\n",
      "    - glutamate secretion\n",
      "    - bicarbonate transport\n",
      "    - transepithelial chloride transport\n",
      "    - monoatomic ion transmembrane transport\n",
      "    - regulation of synaptic plasticity\n",
      "    - detection of light stimulus involved in visual perception\n",
      "    - protein complex oligomerization\n",
      "    - regulation of calcium ion transport\n",
      "    - gamma-aminobutyric acid secretion, neurotransmission\n",
      "    - chloride transmembrane transport\n",
      "  CC:\n",
      "    - cytosol\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - basal plasma membrane\n",
      "    - membrane\n",
      "    - basolateral plasma membrane\n",
      "    - chloride channel complex\n",
      "    - presynapse\n",
      "    - membrane microdomain\n",
      "  MF:\n",
      "    - intracellularly calcium-gated chloride channel activity\n",
      "    - intracellularly calcium-gated chloride channel activity\n",
      "    - chloride channel activity\n",
      "    - chloride channel activity\n",
      "    - chloride channel activity\n",
      "    - protein binding\n",
      "    - bicarbonate transmembrane transporter activity\n",
      "    - ligand-gated channel activity\n",
      "    - identical protein binding\n",
      "    - bicarbonate channel activity\n",
      "UniProt: {'Swiss-Prot': 'O76090', 'TrEMBL': ['B7Z375', 'A0A0S2Z4M4', 'A0A0S2Z579', 'B7Z1N8', 'A0A0C4DGE9', 'E9PMB5']}\n",
      "\n",
      "Gene: FMO1\n",
      "Symbol: FMO1\n",
      "Name: flavin containing dimethylaniline monoxygenase 1\n",
      "Summary: Metabolic N-oxidation of the diet-derived amino-trimethylamine (TMA) is mediated by flavin-containing monooxygenase and is subject to an inherited FMO3 polymorphism in man resulting in a small subpopulation with reduced TMA N-oxidation capacity resulting in fish odor syndrome Trimethylaminuria. Three forms of the enzyme, FMO1 found in fetal liver, FMO2 found in adult liver, and FMO3 are encoded by genes clustered in the 1q23-q25 region. Flavin-containing monooxygenases are NADPH-dependent flavoenzymes that catalyzes the oxidation of soft nucleophilic heteroatom centers in drugs, pesticides, and xenobiotics. Several transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Sep 2013].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - sulfur amino acid catabolic process\n",
      "    - organic acid metabolic process\n",
      "    - xenobiotic metabolic process\n",
      "    - toxin metabolic process\n",
      "    - response to lipopolysaccharide\n",
      "    - taurine biosynthetic process\n",
      "    - negative regulation of fatty acid oxidation\n",
      "    - NADPH oxidation\n",
      "    - energy homeostasis\n",
      "  CC:\n",
      "    - endoplasmic reticulum lumen\n",
      "    - endoplasmic reticulum membrane\n",
      "  MF:\n",
      "    - monooxygenase activity\n",
      "    - N,N-dimethylaniline monooxygenase activity\n",
      "    - N,N-dimethylaniline monooxygenase activity\n",
      "    - protein binding\n",
      "    - trimethylamine monooxygenase activity\n",
      "    - hypotaurine monooxygenase activity\n",
      "    - hypotaurine monooxygenase activity\n",
      "    - flavin adenine dinucleotide binding\n",
      "    - NADP binding\n",
      "UniProt: {'Swiss-Prot': 'Q01740', 'TrEMBL': ['A0A1W2PNR8', 'B2RCG5']}\n",
      "\n",
      "Gene: CCL2\n",
      "Symbol: CCL2\n",
      "Name: C-C motif chemokine ligand 2\n",
      "Summary: This gene is one of several cytokine genes clustered on the q-arm of chromosome 17. Chemokines are a superfamily of secreted proteins involved in immunoregulatory and inflammatory processes. The superfamily is divided into four subfamilies based on the arrangement of N-terminal cysteine residues of the mature peptide. This chemokine is a member of the CC subfamily which is characterized by two adjacent cysteine residues. This cytokine displays chemotactic activity for monocytes and basophils but not for neutrophils or eosinophils. It has been implicated in the pathogenesis of diseases characterized by monocytic infiltrates, like psoriasis, rheumatoid arthritis and atherosclerosis. It binds to chemokine receptors CCR2 and CCR4. Elevated expression of the encoded protein is associated with severe acute respiratory syndrome coronavirus 2 (SARS&#8208;CoV&#8208;2) infection. [provided by RefSeq, Aug 2020].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - angiogenesis\n",
      "    - monocyte chemotaxis\n",
      "    - protein phosphorylation\n",
      "    - chemotaxis\n",
      "    - inflammatory response\n",
      "    - humoral immune response\n",
      "    - cytoskeleton organization\n",
      "    - cell adhesion\n",
      "    - signal transduction\n",
      "    - cell surface receptor signaling pathway\n",
      "    - G protein-coupled receptor signaling pathway\n",
      "    - G protein-coupled receptor signaling pathway, coupled to cyclic nucleotide second messenger\n",
      "    - cell surface receptor signaling pathway via JAK-STAT\n",
      "    - regulation of cell shape\n",
      "    - response to bacterium\n",
      "    - animal organ morphogenesis\n",
      "    - positive regulation of gene expression\n",
      "    - positive regulation of macrophage chemotaxis\n",
      "    - viral genome replication\n",
      "    - cytokine-mediated signaling pathway\n",
      "    - sensory perception of pain\n",
      "    - cellular homeostasis\n",
      "    - positive regulation of cell migration\n",
      "    - negative regulation of glial cell apoptotic process\n",
      "    - helper T cell extravasation\n",
      "    - chemokine (C-C motif) ligand 2 signaling pathway\n",
      "    - negative regulation of neuron apoptotic process\n",
      "    - astrocyte cell migration\n",
      "    - cellular response to fibroblast growth factor stimulus\n",
      "    - eosinophil chemotaxis\n",
      "    - macrophage chemotaxis\n",
      "    - positive regulation of T cell activation\n",
      "    - positive chemotaxis\n",
      "    - positive regulation of synaptic transmission, glutamatergic\n",
      "    - antimicrobial humoral immune response mediated by antimicrobial peptide\n",
      "    - chemokine-mediated signaling pathway\n",
      "    - cellular response to lipopolysaccharide\n",
      "    - cellular response to type II interferon\n",
      "    - cellular response to type II interferon\n",
      "    - cellular response to interleukin-1\n",
      "    - cellular response to interleukin-1\n",
      "    - cellular response to tumor necrosis factor\n",
      "    - cellular response to tumor necrosis factor\n",
      "    - positive regulation of calcium ion import\n",
      "    - positive regulation of NMDA glutamate receptor activity\n",
      "    - negative regulation of vascular endothelial cell proliferation\n",
      "    - negative regulation of G1/S transition of mitotic cell cycle\n",
      "    - positive regulation of endothelial cell apoptotic process\n",
      "    - positive regulation of apoptotic cell clearance\n",
      "    - negative regulation of natural killer cell chemotaxis\n",
      "  CC:\n",
      "    - extracellular region\n",
      "    - extracellular region\n",
      "    - extracellular space\n",
      "  MF:\n",
      "    - protein kinase activity\n",
      "    - signaling receptor binding\n",
      "    - protein binding\n",
      "    - chemokine activity\n",
      "    - CCR2 chemokine receptor binding\n",
      "    - CCR2 chemokine receptor binding\n",
      "    - chemoattractant activity\n",
      "    - CCR chemokine receptor binding\n",
      "UniProt: {'Swiss-Prot': 'P13500', 'TrEMBL': 'J3KRT7'}\n",
      "\n",
      "Gene: ROS1\n",
      "Symbol: ROS1\n",
      "Name: ROS proto-oncogene 1, receptor tyrosine kinase\n",
      "Summary: This proto-oncogene, highly-expressed in a variety of tumor cell lines, belongs to the sevenless subfamily of tyrosine kinase insulin receptor genes. The protein encoded by this gene is a type I integral membrane protein with tyrosine kinase activity. The protein may function as a growth or differentiation factor receptor. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - regulation of cell growth\n",
      "    - columnar/cuboidal epithelial cell development\n",
      "    - protein phosphorylation\n",
      "    - cell surface receptor protein tyrosine kinase signaling pathway\n",
      "    - epidermal growth factor receptor signaling pathway\n",
      "    - spermatogenesis\n",
      "    - insulin receptor signaling pathway\n",
      "    - fibroblast growth factor receptor signaling pathway\n",
      "    - cell differentiation\n",
      "    - brain-derived neurotrophic factor receptor signaling pathway\n",
      "    - regulation of TOR signaling\n",
      "    - regulation of TOR signaling\n",
      "    - platelet-derived growth factor receptor-alpha signaling pathway\n",
      "    - platelet-derived growth factor receptor-beta signaling pathway\n",
      "    - vascular endothelial growth factor receptor-1 signaling pathway\n",
      "    - collagen-activated tyrosine kinase receptor signaling pathway\n",
      "    - vascular endothelial growth factor signaling pathway\n",
      "    - Kit signaling pathway\n",
      "    - macrophage colony-stimulating factor signaling pathway\n",
      "    - insulin-like growth factor receptor signaling pathway\n",
      "    - hepatocyte growth factor receptor signaling pathway\n",
      "    - ephrin receptor signaling pathway\n",
      "    - regulation of ERK1 and ERK2 cascade\n",
      "  CC:\n",
      "    - plasma membrane\n",
      "    - membrane\n",
      "    - receptor complex\n",
      "  MF:\n",
      "    - protein tyrosine kinase activity\n",
      "    - transmembrane receptor protein tyrosine kinase activity\n",
      "    - GPI-linked ephrin receptor activity\n",
      "    - transmembrane-ephrin receptor activity\n",
      "    - epidermal growth factor receptor activity\n",
      "    - fibroblast growth factor receptor activity\n",
      "    - hepatocyte growth factor receptor activity\n",
      "    - insulin receptor activity\n",
      "    - insulin-like growth factor receptor activity\n",
      "    - macrophage colony-stimulating factor receptor activity\n",
      "    - platelet-derived growth factor alpha-receptor activity\n",
      "    - platelet-derived growth factor beta-receptor activity\n",
      "    - stem cell factor receptor activity\n",
      "    - vascular endothelial growth factor receptor activity\n",
      "    - protein binding\n",
      "    - ATP binding\n",
      "    - boss receptor activity\n",
      "    - protein phosphatase binding\n",
      "    - placental growth factor receptor activity\n",
      "    - protein tyrosine kinase collagen receptor activity\n",
      "    - brain-derived neurotrophic factor receptor activity\n",
      "UniProt: {'Swiss-Prot': 'P08922', 'TrEMBL': ['H0Y3T9', 'Q5H8Y1']}\n",
      "\n",
      "Gene: FKBP1B\n",
      "Symbol: FKBP1B\n",
      "Name: FKBP prolyl isomerase 1B\n",
      "Summary: The protein encoded by this gene is a member of the immunophilin protein family, which play a role in immunoregulation and basic cellular processes involving protein folding and trafficking. This encoded protein is a cis-trans prolyl isomerase that binds the immunosuppressants FK506 and rapamycin. It is highly similar to the FK506-binding protein 1A. Its physiological role is thought to be in excitation-contraction coupling in cardiac muscle. There are two alternatively spliced transcript variants of this gene encoding different isoforms. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - regulation of transcription by RNA polymerase II\n",
      "    - 'de novo' protein folding\n",
      "    - smooth muscle contraction\n",
      "    - positive regulation of cytosolic calcium ion concentration\n",
      "    - negative regulation of heart rate\n",
      "    - regulation of release of sequestered calcium ion into cytosol by sarcoplasmic reticulum\n",
      "    - regulation of cardiac muscle contraction by regulation of the release of sequestered calcium ion\n",
      "    - regulation of cardiac muscle contraction by regulation of the release of sequestered calcium ion\n",
      "    - regulation of cardiac muscle contraction by regulation of the release of sequestered calcium ion\n",
      "    - regulation of cardiac muscle contraction by regulation of the release of sequestered calcium ion\n",
      "    - neuronal action potential propagation\n",
      "    - calcium-mediated signaling\n",
      "    - response to vitamin E\n",
      "    - insulin secretion involved in cellular response to glucose stimulus\n",
      "    - protein refolding\n",
      "    - T cell proliferation\n",
      "    - response to hydrogen peroxide\n",
      "    - positive regulation of axon regeneration\n",
      "    - negative regulation of calcium-mediated signaling\n",
      "    - release of sequestered calcium ion into cytosol\n",
      "    - negative regulation of release of sequestered calcium ion into cytosol\n",
      "    - positive regulation of sequestering of calcium ion\n",
      "    - protein maturation\n",
      "    - response to redox state\n",
      "    - negative regulation of insulin secretion involved in cellular response to glucose stimulus\n",
      "  CC:\n",
      "    - cytoplasm\n",
      "    - cytoplasm\n",
      "    - membrane\n",
      "    - Z disc\n",
      "    - sarcoplasmic reticulum membrane\n",
      "    - sarcoplasmic reticulum membrane\n",
      "    - calcium channel complex\n",
      "  MF:\n",
      "    - peptidyl-prolyl cis-trans isomerase activity\n",
      "    - peptidyl-prolyl cis-trans isomerase activity\n",
      "    - signaling receptor binding\n",
      "    - calcium channel regulator activity\n",
      "    - protein binding\n",
      "    - FK506 binding\n",
      "    - calcium channel inhibitor activity\n",
      "    - cyclic nucleotide binding\n",
      "    - transmembrane transporter binding\n",
      "    - RNA polymerase II CTD heptapeptide repeat P3 isomerase activity\n",
      "    - RNA polymerase II CTD heptapeptide repeat P6 isomerase activity\n",
      "UniProt: {'Swiss-Prot': 'P68106', 'TrEMBL': ['G5E9U6', 'F8W6G9', 'H7C0Y3']}\n",
      "\n",
      "Gene: LCE1B\n",
      "Symbol: LCE1B\n",
      "Name: late cornified envelope 1B\n",
      "Summary: Enables identical protein binding activity. Predicted to be involved in keratinization. [provided by Alliance of Genome Resources, Apr 2022]\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - keratinization\n",
      "  MF:\n",
      "    - protein binding\n",
      "    - identical protein binding\n",
      "UniProt: {'Swiss-Prot': 'Q5T7P3'}\n",
      "\n",
      "Gene: CBR3\n",
      "Symbol: CBR3\n",
      "Name: carbonyl reductase 3\n",
      "Summary: Carbonyl reductase 3 catalyzes the reduction of a large number of biologically and pharmacologically active carbonyl compounds to their corresponding alcohols.  The enzyme is classified as a monomeric NADPH-dependent oxidoreductase.  CBR3 contains three exons spanning 11.2 kilobases and is closely linked to another carbonyl reductase gene - CBR1. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - xenobiotic metabolic process\n",
      "    - phylloquinone catabolic process\n",
      "    - cognition\n",
      "  CC:\n",
      "    - extracellular space\n",
      "    - nucleoplasm\n",
      "    - cytosol\n",
      "    - cytosol\n",
      "  MF:\n",
      "    - 3-beta-hydroxysteroid 3-dehydrogenase (NADP+) activity\n",
      "    - carbonyl reductase (NADPH) activity\n",
      "    - carbonyl reductase (NADPH) activity\n",
      "    - carbonyl reductase (NADPH) activity\n",
      "    - protein binding\n",
      "    - NADPH dehydrogenase (quinone) activity\n",
      "    - NADPH binding\n",
      "UniProt: {'Swiss-Prot': 'O75828', 'TrEMBL': 'V9HW40'}\n",
      "\n",
      "Gene: CPOX\n",
      "Symbol: CPOX\n",
      "Name: coproporphyrinogen oxidase\n",
      "Summary: The protein encoded by this gene is the sixth enzyme of the heme biosynthetic pathway. The encoded enzyme is soluble and found in the intermembrane space of mitochondria. This enzyme catalyzes the stepwise oxidative decarboxylation of coproporphyrinogen III to protoporphyrinogen IX, a precursor of heme. Defects in this gene are a cause of hereditary coproporphyria (HCP).[provided by RefSeq, Oct 2009].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - protoporphyrinogen IX biosynthetic process\n",
      "    - protoporphyrinogen IX biosynthetic process\n",
      "    - heme biosynthetic process\n",
      "    - heme A biosynthetic process\n",
      "    - heme B biosynthetic process\n",
      "    - heme O biosynthetic process\n",
      "  CC:\n",
      "    - cytoplasm\n",
      "    - mitochondrion\n",
      "    - mitochondrion\n",
      "    - mitochondrial intermembrane space\n",
      "    - cytosol\n",
      "    - membrane\n",
      "  MF:\n",
      "    - coproporphyrinogen oxidase activity\n",
      "    - coproporphyrinogen oxidase activity\n",
      "    - structural constituent of eye lens\n",
      "    - protein homodimerization activity\n",
      "UniProt: {'Swiss-Prot': 'P36551', 'TrEMBL': 'D6RER6'}\n",
      "\n",
      "Gene: HRH1\n",
      "Symbol: HRH1\n",
      "Name: histamine receptor H1\n",
      "Summary: Histamine is a ubiquitous messenger molecule released from mast cells, enterochromaffin-like cells, and neurons. Its various actions are mediated by histamine receptors H1, H2, H3 and H4. The protein encoded by this gene is an integral membrane protein and belongs to the G protein-coupled receptor superfamily. It mediates the contraction of smooth muscles, the increase in capillary permeability due to contraction of terminal venules, the release of catecholamine from adrenal medulla, and neurotransmission in the central nervous system. It has been associated with multiple processes, including memory and learning, circadian rhythm, and thermoregulation. It is also known to contribute to the pathophysiology of allergic diseases such as atopic dermatitis, asthma, anaphylaxis and allergic rhinitis. Multiple alternatively spliced variants, encoding the same protein, have been identified. [provided by RefSeq, Jan 2015].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - inflammatory response\n",
      "    - G protein-coupled receptor signaling pathway\n",
      "    - G protein-coupled receptor signaling pathway\n",
      "    - G protein-coupled receptor signaling pathway, coupled to cyclic nucleotide second messenger\n",
      "    - phospholipase C-activating G protein-coupled receptor signaling pathway\n",
      "    - chemical synaptic transmission\n",
      "    - memory\n",
      "    - visual learning\n",
      "    - regulation of vascular permeability\n",
      "    - positive regulation of vasoconstriction\n",
      "    - regulation of synaptic plasticity\n",
      "    - cellular response to histamine\n",
      "    - cellular response to histamine\n",
      "  CC:\n",
      "    - cytosol\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - dendrite\n",
      "    - synapse\n",
      "  MF:\n",
      "    - histamine receptor activity\n",
      "    - histamine receptor activity\n",
      "    - histamine receptor activity\n",
      "UniProt: {'Swiss-Prot': 'P35367', 'TrEMBL': 'C9J2E6'}\n",
      "\n",
      "Gene: KIR2DL3\n",
      "Symbol: KIR2DL3\n",
      "Name: killer cell immunoglobulin like receptor, two Ig domains and long cytoplasmic tail 3\n",
      "Summary: Killer cell immunoglobulin-like receptors (KIRs) are transmembrane glycoproteins expressed by natural killer cells and subsets of T cells. The KIR genes are polymorphic and highly homologous and they are found in a cluster on chromosome 19q13.4 within the 1 Mb leukocyte receptor complex (LRC). The gene content of the KIR gene cluster varies among haplotypes, although several 'framework' genes are found in all haplotypes (KIR3DL3, KIR3DP1, KIR3DL4, KIR3DL2). The KIR proteins are classified by the number of extracellular immunoglobulin domains (2D or 3D) and by whether they have a long (L) or short (S) cytoplasmic domain. KIR proteins with the long cytoplasmic domain transduce inhibitory signals upon ligand binding via an immune tyrosine-based inhibitory motif (ITIM), while KIR proteins with the short cytoplasmic domain lack the ITIM motif and instead associate with the TYRO protein tyrosine kinase binding protein to transduce activating signals. The ligands for several KIR proteins are subsets of HLA class I molecules; thus, KIR proteins are thought to play an important role in regulation of the immune response. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - immune response-regulating signaling pathway\n",
      "    - immune response\n",
      "  CC:\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - membrane\n",
      "  MF:\n",
      "    - antigen binding\n",
      "    - protein binding\n",
      "    - signaling receptor activity\n",
      "    - identical protein binding\n",
      "UniProt: {'Swiss-Prot': 'P43628', 'TrEMBL': 'E3NZD8'}\n",
      "\n",
      "Gene: RORC\n",
      "Symbol: RORC\n",
      "Name: RAR related orphan receptor C\n",
      "Summary: The protein encoded by this gene is a DNA-binding transcription factor and is a member of the NR1 subfamily of nuclear hormone receptors. The specific functions of this protein are not known; however, studies of a similar gene in mice have shown that this gene may be essential for lymphoid organogenesis and may play an important regulatory role in thymopoiesis. In addition, studies in mice suggest that the protein encoded by this gene may inhibit the expression of Fas ligand and IL2. Two transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - negative regulation of transcription by RNA polymerase II\n",
      "    - regulation of transcription by RNA polymerase II\n",
      "    - regulation of transcription by RNA polymerase II\n",
      "    - xenobiotic metabolic process\n",
      "    - regulation of glucose metabolic process\n",
      "    - regulation of steroid metabolic process\n",
      "    - intracellular receptor signaling pathway\n",
      "    - circadian regulation of gene expression\n",
      "    - cellular response to sterol\n",
      "    - T-helper cell differentiation\n",
      "    - positive regulation of circadian rhythm\n",
      "    - regulation of fat cell differentiation\n",
      "    - positive regulation of DNA-templated transcription\n",
      "    - lymph node development\n",
      "    - Peyer's patch development\n",
      "    - adipose tissue development\n",
      "    - negative regulation of thymocyte apoptotic process\n",
      "    - T-helper 17 cell differentiation\n",
      "  CC:\n",
      "    - chromatin\n",
      "    - nucleus\n",
      "    - nucleus\n",
      "    - nucleus\n",
      "    - nucleoplasm\n",
      "    - nuclear body\n",
      "  MF:\n",
      "    - RNA polymerase II cis-regulatory region sequence-specific DNA binding\n",
      "    - RNA polymerase II cis-regulatory region sequence-specific DNA binding\n",
      "    - DNA-binding transcription factor activity, RNA polymerase II-specific\n",
      "    - DNA-binding transcription factor activity, RNA polymerase II-specific\n",
      "    - DNA-binding transcription repressor activity, RNA polymerase II-specific\n",
      "    - DNA-binding transcription factor activity\n",
      "    - nuclear receptor activity\n",
      "    - protein binding\n",
      "    - oxysterol binding\n",
      "    - oxysterol binding\n",
      "    - zinc ion binding\n",
      "    - ligand-modulated transcription factor activity\n",
      "    - sequence-specific double-stranded DNA binding\n",
      "UniProt: {'Swiss-Prot': 'P51449', 'TrEMBL': ['F1D8P6', 'Q6I9R9', 'A0A494C0H2', 'A0A494C174', 'A0A1W2PRM4', 'A0A8V8TL94', 'A0A494C0S6']}\n",
      "\n",
      "Gene: FCGR1A\n",
      "Symbol: FCGR1A\n",
      "Name: Fc gamma receptor Ia\n",
      "Summary: This gene encodes a protein that plays an important role in the immune response. This protein is a high-affinity Fc-gamma receptor. The gene is one of three related gene family members located on chromosome 1. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - antibody-dependent cellular cytotoxicity\n",
      "    - antibody-dependent cellular cytotoxicity\n",
      "    - positive regulation of type IIa hypersensitivity\n",
      "    - positive regulation of type III hypersensitivity\n",
      "    - receptor-mediated endocytosis\n",
      "    - phagocytosis, recognition\n",
      "    - phagocytosis, engulfment\n",
      "    - immune response\n",
      "    - signal transduction\n",
      "    - cell surface receptor signaling pathway\n",
      "    - positive regulation of tumor necrosis factor production\n",
      "    - Fc-gamma receptor signaling pathway\n",
      "    - antigen processing and presentation of exogenous peptide antigen via MHC class I\n",
      "    - defense response to bacterium\n",
      "    - innate immune response\n",
      "    - positive regulation of phagocytosis\n",
      "  CC:\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - external side of plasma membrane\n",
      "    - clathrin-coated endocytic vesicle membrane\n",
      "    - early endosome membrane\n",
      "  MF:\n",
      "    - protein binding\n",
      "    - IgG receptor activity\n",
      "    - high-affinity IgG receptor activity\n",
      "    - IgG binding\n",
      "UniProt: {'Swiss-Prot': 'P12314', 'TrEMBL': 'C9JSN8'}\n",
      "\n",
      "Gene: CLDN18\n",
      "Symbol: CLDN18\n",
      "Name: claudin 18\n",
      "Summary: This gene encodes a member of the claudin family. Claudins are integral membrane proteins and components of tight junction strands. Tight junction strands serve as a physical barrier to prevent solutes and water from passing freely through the paracellular space between epithelial or endothelial cell sheets, and also play critical roles in maintaining cell polarity and signal transductions. This gene is upregulated in patients with ulcerative colitis and highly overexpressed in infiltrating ductal adenocarcinomas. PKC/MAPK/AP-1 (protein kinase C/mitogen-activated protein kinase/activator protein-1) dependent pathway regulates the expression of this gene in gastric cells. Alternatively spliced transcript variants encoding different isoforms have been identified. [provided by RefSeq, Jun 2010].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - cell adhesion\n",
      "    - negative regulation of tumor necrosis factor-mediated signaling pathway\n",
      "    - calcium-independent cell-cell adhesion via plasma membrane cell-adhesion molecules\n",
      "    - protein localization to nucleus\n",
      "    - organ growth\n",
      "    - epithelial fluid transport\n",
      "    - response to ethanol\n",
      "    - negative regulation of bone resorption\n",
      "    - lung alveolus development\n",
      "    - digestive tract development\n",
      "    - epithelial cell proliferation\n",
      "    - bicellular tight junction assembly\n",
      "    - cellular response to estrogen stimulus\n",
      "    - tight junction organization\n",
      "    - negative regulation of protein localization to nucleus\n",
      "    - negative regulation of osteoclast development\n",
      "  CC:\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - cell-cell junction\n",
      "    - bicellular tight junction\n",
      "    - bicellular tight junction\n",
      "    - lateral plasma membrane\n",
      "  MF:\n",
      "    - structural molecule activity\n",
      "    - protein binding\n",
      "    - identical protein binding\n",
      "UniProt: {'Swiss-Prot': 'P56856', 'TrEMBL': 'F8WEY4'}\n",
      "\n",
      "Gene: P4HTM\n",
      "Symbol: P4HTM\n",
      "Name: prolyl 4-hydroxylase, transmembrane\n",
      "Summary: The product of this gene belongs to the family of prolyl 4-hydroxylases. This protein is a prolyl hydroxylase that may be involved in the degradation of hypoxia-inducible transcription factors under normoxia. It plays a role in adaptation to hypoxia and may be related to cellular oxygen sensing. Alternatively spliced variants encoding different isoforms have been identified. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - regulation of erythrocyte differentiation\n",
      "  CC:\n",
      "    - endoplasmic reticulum\n",
      "    - endoplasmic reticulum membrane\n",
      "  MF:\n",
      "    - procollagen-proline 4-dioxygenase activity\n",
      "    - iron ion binding\n",
      "    - calcium ion binding\n",
      "    - 2-oxoglutarate-dependent dioxygenase activity\n",
      "    - L-ascorbic acid binding\n",
      "    - hypoxia-inducible factor-proline dioxygenase activity\n",
      "UniProt: {'Swiss-Prot': 'Q9NXG6', 'TrEMBL': ['H7C153', 'A0A0C4DGS7', 'V9GYK5', 'V9GY20']}\n",
      "\n",
      "Gene: ABCG1\n",
      "Symbol: ABCG1\n",
      "Name: ATP binding cassette subfamily G member 1\n",
      "Summary: The protein encoded by this gene is a member of the superfamily of ATP-binding cassette (ABC) transporters. ABC proteins transport various molecules across extra- and intra-cellular membranes. ABC genes are divided into seven distinct subfamilies (ABC1, MDR/TAP, MRP, ALD, OABP, GCN20, White). This protein is a member of the White subfamily. It is involved in macrophage cholesterol and phospholipids transport, and may regulate cellular lipid homeostasis in other cell types. Six alternative splice variants have been identified. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - cholesterol metabolic process\n",
      "    - negative regulation of macrophage derived foam cell differentiation\n",
      "    - positive regulation of cholesterol efflux\n",
      "    - negative regulation of cholesterol storage\n",
      "    - intracellular cholesterol transport\n",
      "    - cholesterol efflux\n",
      "    - cholesterol efflux\n",
      "    - cholesterol efflux\n",
      "    - phospholipid efflux\n",
      "    - response to lipid\n",
      "    - lipid translocation\n",
      "    - low-density lipoprotein particle remodeling\n",
      "    - high-density lipoprotein particle remodeling\n",
      "    - glycoprotein transport\n",
      "    - cholesterol homeostasis\n",
      "    - cholesterol homeostasis\n",
      "    - cholesterol homeostasis\n",
      "    - amyloid precursor protein catabolic process\n",
      "    - reverse cholesterol transport\n",
      "    - positive regulation of cholesterol biosynthetic process\n",
      "    - positive regulation of protein secretion\n",
      "    - transmembrane transport\n",
      "    - phospholipid homeostasis\n",
      "    - cellular response to high density lipoprotein particle stimulus\n",
      "    - regulation of cholesterol metabolic process\n",
      "    - positive regulation of amyloid-beta formation\n",
      "  CC:\n",
      "    - Golgi membrane\n",
      "    - nucleoplasm\n",
      "    - mitochondrion\n",
      "    - endosome\n",
      "    - endoplasmic reticulum membrane\n",
      "    - Golgi apparatus\n",
      "    - Golgi apparatus\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - external side of plasma membrane\n",
      "    - intracellular membrane-bounded organelle\n",
      "    - recycling endosome\n",
      "  MF:\n",
      "    - protein binding\n",
      "    - ATP binding\n",
      "    - phospholipid binding\n",
      "    - cholesterol binding\n",
      "    - ATP hydrolysis activity\n",
      "    - toxin transmembrane transporter activity\n",
      "    - ABC-type sterol transporter activity\n",
      "    - ABC-type sterol transporter activity\n",
      "    - protein homodimerization activity\n",
      "    - ADP binding\n",
      "    - protein heterodimerization activity\n",
      "    - phosphatidylcholine floppase activity\n",
      "    - cholesterol transfer activity\n",
      "    - floppase activity\n",
      "UniProt: {'Swiss-Prot': 'P45844'}\n",
      "\n",
      "Gene: SULT1E1\n",
      "Symbol: SULT1E1\n",
      "Name: sulfotransferase family 1E member 1\n",
      "Summary: Sulfotransferase enzymes catalyze the sulfate conjugation of many hormones, neurotransmitters, drugs, and xenobiotic compounds. These cytosolic enzymes are different in their tissue distributions and substrate specificities. The gene structure (number and length of exons) is similar among family members. This gene encodes a protein that transfers a sulfo moiety to and from estrone, which may control levels of estrogen receptors. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - ethanol catabolic process\n",
      "    - estrogen catabolic process\n",
      "    - steroid metabolic process\n",
      "    - estrogen metabolic process\n",
      "    - positive regulation of fat cell differentiation\n",
      "    - 3'-phosphoadenosine 5'-phosphosulfate metabolic process\n",
      "    - sulfation\n",
      "    - sulfation\n",
      "    - sulfation\n",
      "  CC:\n",
      "    - cytoplasm\n",
      "    - cytosol\n",
      "    - cytosol\n",
      "    - nuclear membrane\n",
      "  MF:\n",
      "    - aryl sulfotransferase activity\n",
      "    - estrone sulfotransferase activity\n",
      "    - steroid binding\n",
      "    - protein binding\n",
      "    - sulfotransferase activity\n",
      "    - flavonol 3-sulfotransferase activity\n",
      "    - steroid sulfotransferase activity\n",
      "UniProt: {'Swiss-Prot': 'P49888', 'TrEMBL': 'Q53X91'}\n",
      "\n",
      "Gene: NLGN3\n",
      "Symbol: NLGN3\n",
      "Name: neuroligin 3\n",
      "Summary: This gene encodes a member of a family of neuronal cell surface proteins. Members of this family may act as splice site-specific ligands for beta-neurexins and may be involved in the formation and remodeling of central nervous system synapses. Mutations in this gene may be associated with autism and Asperger syndrome. Multiple transcript variants encoding distinct isoforms have been identified for this gene. [provided by RefSeq, Oct 2009].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - regulation of respiratory gaseous exchange by nervous system process\n",
      "    - receptor-mediated endocytosis\n",
      "    - neuron cell-cell adhesion\n",
      "    - neuron cell-cell adhesion\n",
      "    - chemical synaptic transmission\n",
      "    - synapse assembly\n",
      "    - learning\n",
      "    - adult behavior\n",
      "    - social behavior\n",
      "    - social behavior\n",
      "    - synaptic vesicle endocytosis\n",
      "    - axon extension\n",
      "    - modulation of chemical synaptic transmission\n",
      "    - modulation of chemical synaptic transmission\n",
      "    - synapse organization\n",
      "    - positive regulation of synapse assembly\n",
      "    - positive regulation of synaptic transmission, glutamatergic\n",
      "    - rhythmic synaptic transmission\n",
      "    - inhibitory postsynaptic potential\n",
      "    - vocalization behavior\n",
      "    - postsynaptic membrane assembly\n",
      "    - postsynaptic membrane assembly\n",
      "    - presynaptic membrane assembly\n",
      "    - presynaptic membrane assembly\n",
      "    - presynapse assembly\n",
      "    - positive regulation of glutamate receptor signaling pathway\n",
      "    - positive regulation of excitatory postsynaptic potential\n",
      "  CC:\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - cell surface\n",
      "    - cell surface\n",
      "    - membrane\n",
      "    - endocytic vesicle\n",
      "    - synapse\n",
      "    - synapse\n",
      "    - postsynaptic membrane\n",
      "    - excitatory synapse\n",
      "    - presynapse\n",
      "    - symmetric, GABA-ergic, inhibitory synapse\n",
      "    - asymmetric, glutamatergic, excitatory synapse\n",
      "  MF:\n",
      "    - protein binding\n",
      "    - signaling receptor activity\n",
      "    - signaling receptor activity\n",
      "    - neurexin family protein binding\n",
      "    - neurexin family protein binding\n",
      "    - cell adhesion molecule binding\n",
      "    - scaffold protein binding\n",
      "UniProt: {'Swiss-Prot': 'Q9NZ94', 'TrEMBL': ['X5D2P6', 'A0A8I5QJH8', 'A0A8I5KSB7', 'Q6MZK0', 'Q4G160', 'X5D7L6', 'X5DNV3', 'B7Z3P8', 'A0A8I5QL38', 'A0A8I5KSX7', 'A0A8I5KTH4', 'A0A8I5KVP5', 'A0A8I5QJU7', 'A0A8I5KQ26', 'A0A8I5KSW0', 'E7EVK0']}\n",
      "\n",
      "Gene: DERL1\n",
      "Symbol: DERL1\n",
      "Name: derlin 1\n",
      "Summary: The protein encoded by this gene is a member of the derlin family. Members of this family participate in the ER-associated degradation response and retrotranslocate misfolded or unfolded proteins from the ER lumen to the cytosol for proteasomal degradation. This protein recognizes substrate in the ER and works in a complex to retrotranslocate it across the ER membrane into the cytosol. This protein may select cystic fibrosis transmembrane conductance regulator protein (CFTR) for degradation as well as unfolded proteins in Alzheimer's disease. Alternative splicing results in multiple transcript variants that encode different protein isoforms. [provided by RefSeq, Aug 2012].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - response to unfolded protein\n",
      "    - endoplasmic reticulum unfolded protein response\n",
      "    - endoplasmic reticulum unfolded protein response\n",
      "    - retrograde protein transport, ER to cytosol\n",
      "    - retrograde protein transport, ER to cytosol\n",
      "    - retrograde protein transport, ER to cytosol\n",
      "    - positive regulation of protein ubiquitination\n",
      "    - protein destabilization\n",
      "    - ERAD pathway\n",
      "    - ERAD pathway\n",
      "    - ERAD pathway\n",
      "    - ERAD pathway\n",
      "    - proteasome-mediated ubiquitin-dependent protein catabolic process\n",
      "    - establishment of protein localization\n",
      "    - cellular response to misfolded protein\n",
      "  CC:\n",
      "    - early endosome\n",
      "    - late endosome\n",
      "    - endoplasmic reticulum\n",
      "    - endoplasmic reticulum\n",
      "    - endoplasmic reticulum membrane\n",
      "    - endoplasmic reticulum membrane\n",
      "    - endoplasmic reticulum membrane\n",
      "    - membrane\n",
      "    - membrane\n",
      "    - Derlin-1-VIMP complex\n",
      "    - Derlin-1 retrotranslocation complex\n",
      "    - endoplasmic reticulum quality control compartment\n",
      "  MF:\n",
      "    - protease binding\n",
      "    - signal recognition particle binding\n",
      "    - signal recognition particle binding\n",
      "    - protein binding\n",
      "    - ubiquitin protein ligase binding\n",
      "    - signaling receptor activity\n",
      "    - MHC class I protein binding\n",
      "    - identical protein binding\n",
      "    - protein-containing complex binding\n",
      "    - ATPase binding\n",
      "    - ubiquitin-specific protease binding\n",
      "UniProt: {'Swiss-Prot': 'Q9BUN8', 'TrEMBL': ['B4E1G1', 'E5RGY0']}\n",
      "\n",
      "Gene: ZFY\n",
      "Symbol: ZFY\n",
      "Name: zinc finger protein Y-linked\n",
      "Summary: This gene encodes a zinc finger-containing protein that may function as a transcription factor. This gene was once a candidate gene for the testis-determining factor (TDF) and was erroneously referred to as TDF. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - regulation of transcription by RNA polymerase II\n",
      "    - positive regulation of transcription by RNA polymerase II\n",
      "  CC:\n",
      "    - chromatin\n",
      "    - nucleus\n",
      "    - nucleoplasm\n",
      "    - nucleolus\n",
      "  MF:\n",
      "    - RNA polymerase II cis-regulatory region sequence-specific DNA binding\n",
      "    - DNA-binding transcription factor activity, RNA polymerase II-specific\n",
      "    - DNA-binding transcription activator activity, RNA polymerase II-specific\n",
      "    - protein binding\n",
      "    - zinc ion binding\n",
      "    - sequence-specific DNA binding\n",
      "UniProt: {'Swiss-Prot': 'P08048', 'TrEMBL': 'C9JQM1'}\n",
      "\n",
      "Gene: QRSL1\n",
      "Symbol: QRSL1\n",
      "Name: glutaminyl-tRNA amidotransferase subunit QRSL1\n",
      "Summary: Enables glutaminyl-tRNA synthase (glutamine-hydrolyzing) activity. Involved in glutaminyl-tRNAGln biosynthesis via transamidation and mitochondrial translation. Located in mitochondrion. Part of glutamyl-tRNA(Gln) amidotransferase complex. Implicated in combined oxidative phosphorylation deficiency 40. [provided by Alliance of Genome Resources, Apr 2022]\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - regulation of protein stability\n",
      "    - mitochondrial translation\n",
      "    - mitochondrial translation\n",
      "    - glutaminyl-tRNAGln biosynthesis via transamidation\n",
      "    - glutaminyl-tRNAGln biosynthesis via transamidation\n",
      "  CC:\n",
      "    - mitochondrion\n",
      "    - mitochondrion\n",
      "    - mitochondrion\n",
      "    - glutamyl-tRNA(Gln) amidotransferase complex\n",
      "    - glutamyl-tRNA(Gln) amidotransferase complex\n",
      "    - glutamyl-tRNA(Gln) amidotransferase complex\n",
      "  MF:\n",
      "    - protein binding\n",
      "    - ATP binding\n",
      "    - glutaminyl-tRNA synthase (glutamine-hydrolyzing) activity\n",
      "UniProt: {'Swiss-Prot': 'Q9H0R6', 'TrEMBL': 'X6R772'}\n",
      "\n",
      "Gene: UGT1A10\n",
      "Symbol: UGT1A10\n",
      "Name: UDP glucuronosyltransferase family 1 member A10\n",
      "Summary: This gene encodes a UDP-glucuronosyltransferase, an enzyme of the glucuronidation pathway that transforms small lipophilic molecules, such as steroids, bilirubin, hormones, and drugs, into water-soluble, excretable metabolites. This gene is part of a complex locus that encodes several UDP-glucuronosyltransferases. The locus includes thirteen unique alternate first exons followed by four common exons. Four of the alternate first exons are considered pseudogenes. Each of the remaining nine 5' exons may be spliced to the four common exons, resulting in nine proteins with different N-termini and identical C-termini. Each first exon encodes the substrate binding site, and is regulated by its own promoter. The enzyme encoded by this gene has glucuronidase activity on mycophenolic acid, coumarins, and quinolines. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - liver development\n",
      "    - lipid metabolic process\n",
      "    - xenobiotic metabolic process\n",
      "    - xenobiotic metabolic process\n",
      "    - xenobiotic metabolic process\n",
      "    - toxin catabolic process\n",
      "    - xenobiotic catabolic process\n",
      "    - negative regulation of fatty acid metabolic process\n",
      "    - flavone metabolic process\n",
      "  CC:\n",
      "    - endoplasmic reticulum\n",
      "    - endoplasmic reticulum\n",
      "    - endoplasmic reticulum\n",
      "    - endoplasmic reticulum membrane\n",
      "    - endoplasmic reticulum membrane\n",
      "    - endoplasmic reticulum membrane\n",
      "  MF:\n",
      "    - retinoic acid binding\n",
      "    - enzyme inhibitor activity\n",
      "    - protein kinase C binding\n",
      "    - protein binding\n",
      "    - glucuronosyltransferase activity\n",
      "    - glucuronosyltransferase activity\n",
      "    - glucuronosyltransferase activity\n",
      "    - glucuronosyltransferase activity\n",
      "    - enzyme binding\n",
      "    - enzyme binding\n",
      "    - protein homodimerization activity\n",
      "    - protein heterodimerization activity\n",
      "    - protein heterodimerization activity\n",
      "UniProt: {'Swiss-Prot': 'Q9HAW8', 'TrEMBL': 'Q5DT02'}\n",
      "\n",
      "Gene: UGT1A6\n",
      "Symbol: UGT1A6\n",
      "Name: UDP glucuronosyltransferase family 1 member A6\n",
      "Summary: This gene encodes a UDP-glucuronosyltransferase, an enzyme of the glucuronidation pathway that transforms small lipophilic molecules, such as steroids, bilirubin, hormones, and drugs, into water-soluble, excretable metabolites. This gene is part of a complex locus that encodes several UDP-glucuronosyltransferases. The locus includes thirteen unique alternate first exons followed by four common exons. Four of the alternate first exons are considered pseudogenes. Each of the remaining nine 5' exons may be spliced to the four common exons, resulting in nine proteins with different N-termini and identical C-termini. Each first exon encodes the substrate binding site, and is regulated by its own promoter. The enzyme encoded by this gene is active on phenolic and planar compounds. Alternative splicing in the unique 5' end of this gene results in two transcript variants. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - xenobiotic metabolic process\n",
      "    - xenobiotic metabolic process\n",
      "    - xenobiotic metabolic process\n",
      "    - negative regulation of fatty acid metabolic process\n",
      "  CC:\n",
      "    - endoplasmic reticulum\n",
      "    - endoplasmic reticulum\n",
      "    - endoplasmic reticulum\n",
      "    - endoplasmic reticulum membrane\n",
      "    - endoplasmic reticulum membrane\n",
      "    - endoplasmic reticulum membrane\n",
      "    - intracellular membrane-bounded organelle\n",
      "  MF:\n",
      "    - retinoic acid binding\n",
      "    - enzyme inhibitor activity\n",
      "    - glucuronosyltransferase activity\n",
      "    - glucuronosyltransferase activity\n",
      "    - glucuronosyltransferase activity\n",
      "    - glucuronosyltransferase activity\n",
      "    - enzyme binding\n",
      "    - enzyme binding\n",
      "    - protein homodimerization activity\n",
      "    - protein heterodimerization activity\n",
      "    - protein heterodimerization activity\n",
      "UniProt: {'Swiss-Prot': 'P19224', 'TrEMBL': ['Q5DT01', 'Q5DSZ8', 'B5MCT4', 'H7C5F8', 'C9JMY5']}\n",
      "\n",
      "Gene: KIF15\n",
      "Symbol: KIF15\n",
      "Name: kinesin family member 15\n",
      "Summary: Predicted to enable microtubule binding activity and microtubule motor activity. Predicted to be involved in microtubule-based movement. Located in membrane. [provided by Alliance of Genome Resources, Apr 2022]\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - mitotic cell cycle\n",
      "    - microtubule-based movement\n",
      "    - centrosome separation\n",
      "    - mitotic spindle assembly\n",
      "  CC:\n",
      "    - spindle pole\n",
      "    - cytoplasm\n",
      "    - centrosome\n",
      "    - cytosol\n",
      "    - kinesin complex\n",
      "    - plus-end kinesin complex\n",
      "    - microtubule\n",
      "    - membrane\n",
      "  MF:\n",
      "    - cytoskeletal motor activity\n",
      "    - microtubule motor activity\n",
      "    - protein binding\n",
      "    - ATP binding\n",
      "    - microtubule binding\n",
      "    - plus-end-directed microtubule motor activity\n",
      "    - ATP hydrolysis activity\n",
      "UniProt: {'Swiss-Prot': 'Q9NS87', 'TrEMBL': ['F8WC33', 'C9JKA9', 'D6RCT7', 'H7BZT2', 'H7C1K7']}\n",
      "\n",
      "Gene: KIF16B\n",
      "Symbol: KIF16B\n",
      "Name: kinesin family member 16B\n",
      "Summary: The protein encoded by this gene is a kinesin-like protein that may be involved in intracellular trafficking. Three transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Dec 2010].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - formation of primary germ layer\n",
      "    - regulation of receptor recycling\n",
      "    - Golgi to endosome transport\n",
      "    - epidermal growth factor receptor signaling pathway\n",
      "    - endoderm development\n",
      "    - fibroblast growth factor receptor signaling pathway\n",
      "    - receptor catabolic process\n",
      "    - early endosome to late endosome transport\n",
      "    - vesicle transport along microtubule\n",
      "  CC:\n",
      "    - cytoplasm\n",
      "    - cytoplasm\n",
      "    - endosome\n",
      "    - early endosome\n",
      "    - spindle\n",
      "    - cytosol\n",
      "    - kinesin complex\n",
      "    - microtubule\n",
      "    - early endosome membrane\n",
      "  MF:\n",
      "    - protein binding\n",
      "    - ATP binding\n",
      "    - phosphatidylinositol-3,4,5-trisphosphate binding\n",
      "    - microtubule binding\n",
      "    - plus-end-directed microtubule motor activity\n",
      "    - plus-end-directed microtubule motor activity\n",
      "    - ATP hydrolysis activity\n",
      "    - phosphatidylinositol-3-phosphate binding\n",
      "    - phosphatidylinositol-3,4-bisphosphate binding\n",
      "    - phosphatidylinositol-3,5-bisphosphate binding\n",
      "UniProt: {'Swiss-Prot': 'Q96L93', 'TrEMBL': ['A0A1B0GTU3', 'A0A140VK74', 'H0Y534', 'A0A1B0GVS8']}\n",
      "\n",
      "Gene: MICALL1\n",
      "Symbol: MICALL1\n",
      "Name: MICAL like 1\n",
      "Summary: Enables identical protein binding activity; phosphatidic acid binding activity; and small GTPase binding activity. Involved in several processes, including plasma membrane tubulation; protein localization to endosome; and slow endocytic recycling. Located in late endosome and recycling endosome membrane. Is extrinsic component of membrane. [provided by Alliance of Genome Resources, Apr 2022]\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - cell migration involved in sprouting angiogenesis\n",
      "    - protein targeting to membrane\n",
      "    - endocytosis\n",
      "    - receptor-mediated endocytosis\n",
      "    - protein transport\n",
      "    - neuron projection development\n",
      "    - endocytic recycling\n",
      "    - slow endocytic recycling\n",
      "    - protein localization to endosome\n",
      "    - cilium assembly\n",
      "    - protein localization to cilium\n",
      "    - plasma membrane tubulation\n",
      "  CC:\n",
      "    - early endosome\n",
      "    - late endosome\n",
      "    - trans-Golgi network\n",
      "    - centriole\n",
      "    - adherens junction\n",
      "    - cilium\n",
      "    - cytoplasmic side of endosome membrane\n",
      "    - late endosome membrane\n",
      "    - recycling endosome membrane\n",
      "    - recycling endosome membrane\n",
      "    - ciliary membrane\n",
      "  MF:\n",
      "    - protein binding\n",
      "    - cytoskeletal anchor activity\n",
      "    - protein-macromolecule adaptor activity\n",
      "    - small GTPase binding\n",
      "    - identical protein binding\n",
      "    - cadherin binding\n",
      "    - metal ion binding\n",
      "    - phosphatidic acid binding\n",
      "UniProt: {'Swiss-Prot': 'Q8N3F8', 'TrEMBL': ['A0A7P0Z427', 'B0QY91', 'A0A7P0Z4L3', 'H0Y6J8', 'A0A7P0T8A6', 'B0QY86', 'A0A7P0T9P2', 'A0A7P0T7Z5']}\n",
      "\n",
      "Gene: SLC9A3\n",
      "Symbol: SLC9A3\n",
      "Name: solute carrier family 9 member A3\n",
      "Summary: The protein encoded by this gene is an epithelial brush border Na/H exchanger that uses an inward sodium ion gradient to expel acids from the cell. Defects in this gene are a cause of congenital secretory sodium diarrhea. Pseudogenes of this gene exist on chromosomes 10 and 22. [provided by RefSeq, Mar 2016].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - monoatomic ion transport\n",
      "    - regulation of intracellular pH\n",
      "    - regulation of intracellular pH\n",
      "    - potassium ion transmembrane transport\n",
      "    - sodium ion import across plasma membrane\n",
      "    - sodium ion import across plasma membrane\n",
      "    - proton transmembrane transport\n",
      "  CC:\n",
      "    - early endosome\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - brush border\n",
      "    - cell surface\n",
      "    - apical plasma membrane\n",
      "    - apical plasma membrane\n",
      "    - brush border membrane\n",
      "    - brush border membrane\n",
      "    - early endosome membrane\n",
      "    - recycling endosome membrane\n",
      "    - extracellular exosome\n",
      "    - extracellular exosome\n",
      "  MF:\n",
      "    - protein binding\n",
      "    - sodium:proton antiporter activity\n",
      "    - sodium:proton antiporter activity\n",
      "    - sodium:proton antiporter activity\n",
      "    - sodium:proton antiporter activity\n",
      "    - potassium:proton antiporter activity\n",
      "    - PDZ domain binding\n",
      "    - phosphatidylinositol binding\n",
      "    - identical protein binding\n",
      "UniProt: {'Swiss-Prot': 'P48764', 'TrEMBL': 'A0A2R8Y780'}\n",
      "\n",
      "Gene: ARSA\n",
      "Symbol: ARSA\n",
      "Name: arylsulfatase A\n",
      "Summary: The protein encoded by this gene hydrolyzes cerebroside sulfate to cerebroside and sulfate. Defects in this gene lead to metachromatic leucodystrophy (MLD), a progressive demyelination disease which results in a variety of neurological symptoms and ultimately death. Alternatively spliced transcript variants have been described for this gene. [provided by RefSeq, Dec 2010].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - lipid metabolic process\n",
      "  CC:\n",
      "    - extracellular region\n",
      "    - lysosome\n",
      "    - endoplasmic reticulum lumen\n",
      "    - azurophil granule lumen\n",
      "    - lysosomal lumen\n",
      "    - extracellular exosome\n",
      "  MF:\n",
      "    - arylsulfatase activity\n",
      "    - cerebroside-sulfatase activity\n",
      "    - calcium ion binding\n",
      "    - protein binding\n",
      "    - sulfuric ester hydrolase activity\n",
      "UniProt: {'Swiss-Prot': 'P15289', 'TrEMBL': ['B4DVI5', 'A0A0C4DFZ2', 'V9GYR0']}\n",
      "\n",
      "Gene: HCCS\n",
      "Symbol: HCCS\n",
      "Name: holocytochrome c synthase\n",
      "Summary: The protein encoded by this gene is an enzyme that covalently links a heme group to the apoprotein of cytochrome c. Defects in this gene are a cause of microphthalmia syndromic type 7 (MCOPS7). Three transcript variants encoding the same protein have been found for this gene. [provided by RefSeq, Jan 2010].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - animal organ morphogenesis\n",
      "    - cytochrome c-heme linkage\n",
      "    - respiratory electron transport chain\n",
      "  CC:\n",
      "    - mitochondrion\n",
      "    - mitochondrion\n",
      "    - mitochondrion\n",
      "    - mitochondrial inner membrane\n",
      "    - cytosol\n",
      "    - membrane\n",
      "  MF:\n",
      "    - holocytochrome-c synthase activity\n",
      "    - holocytochrome-c synthase activity\n",
      "    - holocytochrome-c synthase activity\n",
      "    - protein binding\n",
      "    - heme binding\n",
      "    - metal ion binding\n",
      "UniProt: {'Swiss-Prot': 'P53701'}\n",
      "\n",
      "Gene: ZBTB39\n",
      "Symbol: ZBTB39\n",
      "Name: zinc finger and BTB domain containing 39\n",
      "Summary: Predicted to enable DNA-binding transcription repressor activity, RNA polymerase II-specific and RNA polymerase II cis-regulatory region sequence-specific DNA binding activity. Predicted to be involved in regulation of transcription by RNA polymerase II. Predicted to be active in nucleoplasm. [provided by Alliance of Genome Resources, Apr 2022]\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - negative regulation of transcription by RNA polymerase II\n",
      "    - regulation of cytokine production\n",
      "    - regulation of immune system process\n",
      "  CC:\n",
      "    - nucleoplasm\n",
      "  MF:\n",
      "    - RNA polymerase II cis-regulatory region sequence-specific DNA binding\n",
      "    - DNA-binding transcription repressor activity, RNA polymerase II-specific\n",
      "    - protein binding\n",
      "    - zinc ion binding\n",
      "UniProt: {'Swiss-Prot': 'O15060'}\n",
      "\n",
      "Gene: CYP4F12\n",
      "Symbol: CYP4F12\n",
      "Name: cytochrome P450 family 4 subfamily F member 12\n",
      "Summary: This gene encodes a member of the cytochrome P450 superfamily of enzymes. The cytochrome P450 proteins are monooxygenases which catalyze many reactions involved in drug metabolism and synthesis of cholesterol, steroids and other lipids. This protein likely localizes to the endoplasmic reticulum. When expressed in yeast the enzyme is capable of oxdizing arachidonic acid. It can also catalyze the epoxidation of 22:6n-3 and 22:5n-3 polyunsaturated long-chain fatty acids. This gene is part of a cluster of cytochrome P450 genes on chromosome 19. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Apr 2014].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - very long-chain fatty acid metabolic process\n",
      "    - long-chain fatty acid metabolic process\n",
      "    - renal water homeostasis\n",
      "    - pressure natriuresis\n",
      "    - fatty acid metabolic process\n",
      "    - xenobiotic metabolic process\n",
      "    - arachidonate metabolic process\n",
      "    - arachidonate metabolic process\n",
      "    - epoxygenase P450 pathway\n",
      "    - leukotriene B4 catabolic process\n",
      "    - vitamin E metabolic process\n",
      "    - menaquinone catabolic process\n",
      "    - phylloquinone catabolic process\n",
      "    - sodium ion homeostasis\n",
      "  CC:\n",
      "    - cytoplasm\n",
      "    - endoplasmic reticulum membrane\n",
      "    - apical plasma membrane\n",
      "    - intracellular membrane-bounded organelle\n",
      "  MF:\n",
      "    - monooxygenase activity\n",
      "    - iron ion binding\n",
      "    - protein binding\n",
      "    - arachidonate epoxygenase activity\n",
      "    - alkane 1-monooxygenase activity\n",
      "    - heme binding\n",
      "    - leukotriene-B4 20-monooxygenase activity\n",
      "UniProt: {'Swiss-Prot': 'Q9HCS2', 'TrEMBL': ['F8VU50', 'F8VZP1', 'F8VR75']}\n",
      "\n",
      "Gene: CYP2C9\n",
      "Symbol: CYP2C9\n",
      "Name: cytochrome P450 family 2 subfamily C member 9\n",
      "Summary: This gene encodes a member of the cytochrome P450 superfamily of enzymes. The cytochrome P450 proteins are monooxygenases which catalyze many reactions involved in drug metabolism and synthesis of cholesterol, steroids and other lipids. This protein localizes to the endoplasmic reticulum and its expression is induced by rifampin. The enzyme is known to metabolize many xenobiotics, including phenytoin, tolbutamide, ibuprofen and S-warfarin. Studies identifying individuals who are poor metabolizers of phenytoin and tolbutamide suggest that this gene is polymorphic. The gene is located within a cluster of cytochrome P450 genes on chromosome 10q24. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - organic acid metabolic process\n",
      "    - xenobiotic metabolic process\n",
      "    - xenobiotic metabolic process\n",
      "    - xenobiotic metabolic process\n",
      "    - xenobiotic metabolic process\n",
      "    - steroid metabolic process\n",
      "    - cholesterol metabolic process\n",
      "    - estrogen metabolic process\n",
      "    - monoterpenoid metabolic process\n",
      "    - epoxygenase P450 pathway\n",
      "    - epoxygenase P450 pathway\n",
      "    - urea metabolic process\n",
      "    - monocarboxylic acid metabolic process\n",
      "    - xenobiotic catabolic process\n",
      "    - xenobiotic catabolic process\n",
      "    - long-chain fatty acid biosynthetic process\n",
      "    - amide metabolic process\n",
      "    - icosanoid biosynthetic process\n",
      "    - oxidative demethylation\n",
      "    - omega-hydroxylase P450 pathway\n",
      "  CC:\n",
      "    - cytoplasm\n",
      "    - endoplasmic reticulum membrane\n",
      "    - plasma membrane\n",
      "    - intracellular membrane-bounded organelle\n",
      "    - intracellular membrane-bounded organelle\n",
      "    - intracellular membrane-bounded organelle\n",
      "  MF:\n",
      "    - monooxygenase activity\n",
      "    - monooxygenase activity\n",
      "    - monooxygenase activity\n",
      "    - iron ion binding\n",
      "    - arachidonate epoxygenase activity\n",
      "    - steroid hydroxylase activity\n",
      "    - arachidonate 14,15-epoxygenase activity\n",
      "    - arachidonate 11,12-epoxygenase activity\n",
      "    - oxidoreductase activity\n",
      "    - oxidoreductase activity, acting on paired donors, with incorporation or reduction of molecular oxygen, reduced flavin or flavoprotein as one donor, and incorporation of one atom of oxygen\n",
      "    - (S)-limonene 6-monooxygenase activity\n",
      "    - (S)-limonene 7-monooxygenase activity\n",
      "    - heme binding\n",
      "    - caffeine oxidase activity\n",
      "    - (R)-limonene 6-monooxygenase activity\n",
      "    - estrogen 2-hydroxylase activity\n",
      "UniProt: {'Swiss-Prot': 'P11712', 'TrEMBL': ['A0A2R8YF67', 'S5RV20']}\n",
      "\n",
      "Gene: NXT1\n",
      "Symbol: NXT1\n",
      "Name: nuclear transport factor 2 like export factor 1\n",
      "Summary: The protein encoded by this gene is located in the nuclear envelope. It has protein similarity to nuclear transport factor 2. This protein functions as a nuclear export factor in both RAN (Ras-related nuclear protein)- and CRM1 (required for chromosome region maintenance)-dependent pathways. It is found to stimulate the export of U1 snRNA in RAN- and CRM1-dependent pathways and the export of tRNA and mRNA in a CRM1-independent pathway. The encoded protein heterodimerizes with Tap protein and may regulate the ability of Tap protein to mediate nuclear mRNA export. The use of alternate polyadenylation sites has been found for this gene. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - mRNA export from nucleus\n",
      "    - mRNA export from nucleus\n",
      "    - protein export from nucleus\n",
      "    - poly(A)+ mRNA export from nucleus\n",
      "  CC:\n",
      "    - nucleus\n",
      "    - nucleus\n",
      "    - nuclear pore\n",
      "    - nucleoplasm\n",
      "    - nucleoplasm\n",
      "    - cytoplasm\n",
      "    - cytosol\n",
      "    - cytosol\n",
      "    - nuclear speck\n",
      "    - nuclear RNA export factor complex\n",
      "    - nuclear RNA export factor complex\n",
      "    - nuclear pore central transport channel\n",
      "  MF:\n",
      "    - protein binding\n",
      "    - small GTPase binding\n",
      "UniProt: {'Swiss-Prot': 'Q9UKK6'}\n",
      "\n",
      "Gene: GSTO2\n",
      "Symbol: GSTO2\n",
      "Name: glutathione S-transferase omega 2\n",
      "Summary: The protein encoded by this gene is an omega class glutathione S-transferase (GST). GSTs are involved in the metabolism of xenobiotics and carcinogens. Four transcript variants encoding different isoforms have been found for this gene.[provided by RefSeq, Jul 2010].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - glutathione metabolic process\n",
      "    - xenobiotic metabolic process\n",
      "    - L-ascorbic acid metabolic process\n",
      "    - L-ascorbic acid metabolic process\n",
      "    - cellular response to arsenic-containing substance\n",
      "    - cellular oxidant detoxification\n",
      "  CC:\n",
      "    - cytoplasm\n",
      "    - cytosol\n",
      "    - extracellular exosome\n",
      "  MF:\n",
      "    - glutathione transferase activity\n",
      "    - protein binding\n",
      "    - oxidoreductase activity\n",
      "    - identical protein binding\n",
      "    - glutathione dehydrogenase (ascorbate) activity\n",
      "    - glutathione dehydrogenase (ascorbate) activity\n",
      "    - methylarsonate reductase activity\n",
      "UniProt: {'Swiss-Prot': 'Q9H4Y5'}\n",
      "\n",
      "Gene: PPIP5K2\n",
      "Symbol: PPIP5K2\n",
      "Name: diphosphoinositol pentakisphosphate kinase 2\n",
      "Summary: This gene encodes a member of the histidine acid phosphatase family of proteins. Despite containing a histidine acid phosphatase domain, the encoded protein functions as an inositol pyrophosphate kinase, and is thought to lack phosphatase activity. This kinase activity is the mechanism by which the encoded protein synthesizes high-energy inositol pyrophosphates, which act as signaling molecules that regulate cellular homeostasis and other processes. This gene may be associated with autism spectrum disorder in human patients. [provided by RefSeq, Sep 2016].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - inositol metabolic process\n",
      "    - inositol metabolic process\n",
      "    - sensory perception of sound\n",
      "    - inositol phosphate biosynthetic process\n",
      "    - inositol phosphate metabolic process\n",
      "  CC:\n",
      "    - cytosol\n",
      "    - cytosol\n",
      "    - cytosol\n",
      "    - cytosol\n",
      "  MF:\n",
      "    - inositol-1,3,4,5,6-pentakisphosphate kinase activity\n",
      "    - inositol hexakisphosphate kinase activity\n",
      "    - inositol hexakisphosphate kinase activity\n",
      "    - inositol hexakisphosphate kinase activity\n",
      "    - diphosphoinositol pentakisphosphate kinase activity\n",
      "    - inositol hexakisphosphate 5-kinase activity\n",
      "    - protein binding\n",
      "    - ATP binding\n",
      "    - 5-diphosphoinositol pentakisphosphate 1-kinase activity\n",
      "    - 5-diphosphoinositol pentakisphosphate 1-kinase activity\n",
      "    - 5-diphosphoinositol pentakisphosphate 1-kinase activity\n",
      "    - inositol hexakisphosphate 1-kinase activity\n",
      "UniProt: {'Swiss-Prot': 'O43314', 'TrEMBL': ['A0A8V8TMN4', 'A0A087WZV0', 'B4DGV1', 'K7ENU7', 'H0Y9S9', 'K7EPT7', 'A0A087WWN8', 'D6RBU4', 'K7EJG8', 'D6RFG4', 'H0Y9M0']}\n",
      "\n",
      "Gene: CST6\n",
      "Symbol: CST6\n",
      "Name: cystatin E/M\n",
      "Summary: The cystatin superfamily encompasses proteins that contain multiple cystatin-like sequences. Some of the members are active cysteine protease inhibitors, while others have lost or perhaps never acquired this inhibitory activity. There are three inhibitory families in the superfamily, including the type 1 cystatins (stefins), type 2 cystatins and the kininogens. The type 2 cystatin proteins are a class of cysteine proteinase inhibitors found in a variety of human fluids and secretions, where they appear to provide protective functions. This gene encodes a cystatin from the type 2 family, which is down-regulated in metastatic breast tumor cells as compared to primary tumor cells. Loss of expression is likely associated with the progression of a primary tumor to a metastatic phenotype. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - epidermis development\n",
      "    - anatomical structure morphogenesis\n",
      "  CC:\n",
      "    - cornified envelope\n",
      "    - extracellular exosome\n",
      "  MF:\n",
      "    - cysteine-type endopeptidase inhibitor activity\n",
      "    - protein binding\n",
      "UniProt: {'Swiss-Prot': 'Q15828'}\n",
      "\n",
      "Gene: FASTKD3\n",
      "Symbol: FASTKD3\n",
      "Name: FAST kinase domains 3\n",
      "Summary: This gene encodes a member of a small family of Fas-activated serine/threonine kinase domain (FASTKD) containing proteins that share an amino terminal mitochondrial targeting domain and multiple carboxy terminal FAST domains as well as a putative RNA-binding RAP domain. The members of this family are ubiquitously expressed and are generally most abundant in mitochondria-enriched tissues such as heart, skeletal muscle and brown-adipose tissue. Some members of this protein family may play a role in apoptosis. The protein encoded by this gene interacts with components of the mitochondrial respiratory and translation networks. A pseudogene of this gene is also present on chromosome 5. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Jan 2013].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - mitochondrial RNA processing\n",
      "    - mitochondrial cytochrome c oxidase assembly\n",
      "    - regulation of mitochondrial mRNA stability\n",
      "    - regulation of mitochondrial mRNA stability\n",
      "    - positive regulation of mitochondrial translation\n",
      "  CC:\n",
      "    - nucleoplasm\n",
      "    - mitochondrion\n",
      "    - mitochondrion\n",
      "    - mitochondrial matrix\n",
      "    - ribonucleoprotein granule\n",
      "  MF:\n",
      "    - RNA binding\n",
      "    - RNA binding\n",
      "    - protein binding\n",
      "UniProt: {'Swiss-Prot': 'Q14CZ7', 'TrEMBL': ['D6RHY4', 'D6RAR6', 'D6RC07', 'D6RB04']}\n",
      "\n",
      "Gene: IMPDH1\n",
      "Symbol: IMPDH1\n",
      "Name: inosine monophosphate dehydrogenase 1\n",
      "Summary: The protein encoded by this gene acts as a homotetramer to regulate cell growth. The encoded protein is an enzyme that catalyzes the synthesis of xanthine monophosphate (XMP) from inosine-5'-monophosphate (IMP). This is the rate-limiting step in the de novo synthesis of guanine nucleotides. Defects in this gene are a cause of retinitis pigmentosa type 10 (RP10). Several transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Dec 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - GMP biosynthetic process\n",
      "    - GTP biosynthetic process\n",
      "  CC:\n",
      "    - extracellular region\n",
      "    - nucleus\n",
      "    - cytoplasm\n",
      "    - cytoplasm\n",
      "    - cytosol\n",
      "    - secretory granule lumen\n",
      "    - azurophil granule lumen\n",
      "    - ficolin-1-rich granule lumen\n",
      "  MF:\n",
      "    - nucleotide binding\n",
      "    - nucleic acid binding\n",
      "    - DNA binding\n",
      "    - RNA binding\n",
      "    - IMP dehydrogenase activity\n",
      "    - protein binding\n",
      "    - identical protein binding\n",
      "    - metal ion binding\n",
      "UniProt: {'Swiss-Prot': 'P20839', 'TrEMBL': ['B3KRZ3', 'Q6ZNB1', 'H7C5T1', 'C9K0R9', 'A0A3B3IRL5', 'F8WDE9', 'C9J029', 'C9J381']}\n",
      "\n",
      "Gene: FAR1\n",
      "Symbol: FAR1\n",
      "Name: fatty acyl-CoA reductase 1\n",
      "Summary: The protein encoded by this gene is required for the reduction of fatty acids to fatty alcohols, a process that is required for the synthesis of monoesters and ether lipids. NADPH is required as a cofactor in this reaction, and 16-18 carbon saturated and unsaturated fatty acids are the preferred substrate. This is a peroxisomal membrane protein, and studies suggest that the N-terminus contains a large catalytic domain located on the outside of the peroxisome, while the C-terminus is exposed to the matrix of the peroxisome. Studies indicate that the regulation of this protein is dependent on plasmalogen levels. Mutations in this gene have been associated with individuals affected by severe intellectual disability, early-onset epilepsy, microcephaly, congenital cataracts, growth retardation, and spasticity (PMID: 25439727). A pseudogene of this gene is located on chromosome 13. [provided by RefSeq, Jan 2015].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - ether lipid biosynthetic process\n",
      "    - ether lipid biosynthetic process\n",
      "    - wax biosynthetic process\n",
      "    - wax biosynthetic process\n",
      "    - long-chain fatty-acyl-CoA metabolic process\n",
      "    - long-chain fatty-acyl-CoA metabolic process\n",
      "    - glycerophospholipid biosynthetic process\n",
      "  CC:\n",
      "    - peroxisome\n",
      "    - peroxisome\n",
      "    - peroxisomal membrane\n",
      "    - peroxisomal membrane\n",
      "    - peroxisomal membrane\n",
      "  MF:\n",
      "    - protein binding\n",
      "    - oxidoreductase activity\n",
      "    - alcohol-forming very long-chain fatty acyl-CoA reductase activity\n",
      "    - alcohol-forming long-chain fatty acyl-CoA reductase activity\n",
      "UniProt: {'Swiss-Prot': 'Q8WVX9', 'TrEMBL': ['A0A8V8TQT9', 'A0AAQ5BHM1', 'A0AAQ5BHJ7', 'A0AAQ5BHN8', 'A0AAQ5BHK0', 'A0AAQ5BHL4', 'A0AAQ5BHL3', 'E9PNW8', 'A0AAQ5BHL0']}\n",
      "\n",
      "Gene: MRGBP\n",
      "Symbol: MRGBP\n",
      "Name: MRG domain binding protein\n",
      "Summary: Predicted to be involved in histone acetylation and regulation of transcription by RNA polymerase II. Located in nucleoplasm. [provided by Alliance of Genome Resources, Apr 2022]\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - chromatin organization\n",
      "    - regulation of transcription by RNA polymerase II\n",
      "    - regulation of apoptotic process\n",
      "    - positive regulation of DNA-templated transcription\n",
      "    - regulation of cell cycle\n",
      "    - positive regulation of double-strand break repair via homologous recombination\n",
      "    - regulation of double-strand break repair\n",
      "  CC:\n",
      "    - nucleosome\n",
      "    - nucleoplasm\n",
      "    - nucleoplasm\n",
      "    - NuA4 histone acetyltransferase complex\n",
      "    - NuA4 histone acetyltransferase complex\n",
      "  MF:\n",
      "    - protein binding\n",
      "UniProt: {'Swiss-Prot': 'Q9NV56'}\n",
      "\n",
      "Gene: KRTAP12-3\n",
      "Symbol: KRTAP12-3\n",
      "Name: keratin associated protein 12-3\n",
      "Summary: Enables identical protein binding activity. Predicted to be located in cytosol. [provided by Alliance of Genome Resources, Apr 2022]\n",
      "GO Annotations:\n",
      "  CC:\n",
      "    - cytosol\n",
      "    - keratin filament\n",
      "  MF:\n",
      "    - protein binding\n",
      "    - identical protein binding\n",
      "UniProt: {'Swiss-Prot': 'P60328'}\n",
      "\n",
      "Gene: PSMB2\n",
      "Symbol: PSMB2\n",
      "Name: proteasome 20S subunit beta 2\n",
      "Summary: The proteasome is a multicatalytic proteinase complex with a highly ordered ring-shaped 20S core structure. The core structure is composed of 4 rings of 28 non-identical subunits; 2 rings are composed of 7 alpha subunits and 2 rings are composed of 7 beta subunits. Proteasomes are distributed throughout eukaryotic cells at a high concentration and cleave peptides in an ATP/ubiquitin-dependent process in a non-lysosomal pathway. An essential function of a modified proteasome, the immunoproteasome, is the processing of class I MHC peptides. This gene encodes a member of the proteasome B-type family, also known as the T1B family, that is a 20S core beta subunit. Multiple alternatively spliced transcript variants encoding distinct isoforms have been found for this gene. [provided by RefSeq, Dec 2010].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - proteasome-mediated ubiquitin-dependent protein catabolic process\n",
      "    - proteasome-mediated ubiquitin-dependent protein catabolic process\n",
      "  CC:\n",
      "    - proteasome complex\n",
      "    - proteasome complex\n",
      "    - nucleus\n",
      "    - nucleus\n",
      "    - nucleus\n",
      "    - nucleoplasm\n",
      "    - nucleoplasm\n",
      "    - cytoplasm\n",
      "    - cytosol\n",
      "    - cytosol\n",
      "    - proteasome core complex\n",
      "    - proteasome core complex\n",
      "    - membrane\n",
      "    - proteasome core complex, beta-subunit complex\n",
      "    - proteasome core complex, beta-subunit complex\n",
      "    - extracellular exosome\n",
      "  MF:\n",
      "    - protein binding\n",
      "UniProt: {'Swiss-Prot': 'P49721', 'TrEMBL': ['A0A087WVV1', 'B7Z478', 'A0A140VJS6']}\n",
      "\n",
      "Gene: MAOB\n",
      "Symbol: MAOB\n",
      "Name: monoamine oxidase B\n",
      "Summary: The protein encoded by this gene belongs to the flavin monoamine oxidase family. It is a enzyme located in the mitochondrial outer membrane. It catalyzes the oxidative deamination of biogenic and xenobiotic amines and plays an important role in the metabolism of neuroactive and vasoactive amines in the central nervous sysytem and peripheral tissues. This protein preferentially degrades benzylamine and phenylethylamine. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - response to xenobiotic stimulus\n",
      "    - response to toxic substance\n",
      "    - response to aluminum ion\n",
      "    - response to selenium ion\n",
      "    - negative regulation of serotonin secretion\n",
      "    - phenylethylamine catabolic process\n",
      "    - substantia nigra development\n",
      "    - response to lipopolysaccharide\n",
      "    - dopamine catabolic process\n",
      "    - response to ethanol\n",
      "    - positive regulation of dopamine metabolic process\n",
      "    - hydrogen peroxide biosynthetic process\n",
      "    - response to corticosterone\n",
      "  CC:\n",
      "    - mitochondrion\n",
      "    - mitochondrion\n",
      "    - mitochondrion\n",
      "    - mitochondrion\n",
      "    - mitochondrial envelope\n",
      "    - mitochondrial outer membrane\n",
      "    - dendrite\n",
      "    - neuronal cell body\n",
      "  MF:\n",
      "    - protein binding\n",
      "    - primary methylamine oxidase activity\n",
      "    - primary methylamine oxidase activity\n",
      "    - primary methylamine oxidase activity\n",
      "    - electron transfer activity\n",
      "    - identical protein binding\n",
      "    - flavin adenine dinucleotide binding\n",
      "    - monoamine oxidase activity\n",
      "UniProt: {'Swiss-Prot': 'P27338', 'TrEMBL': 'B7Z242'}\n",
      "\n",
      "Gene: RPS6KA6\n",
      "Symbol: RPS6KA6\n",
      "Name: ribosomal protein S6 kinase A6\n",
      "Summary: This gene encodes a member of ribosomal S6 kinase family, serine-threonine protein kinases which are regulated by growth factors. The encoded protein may be distinct from other members of this family, however, as studies suggest it is not growth factor dependent and may not participate in the same signaling pathways. [provided by RefSeq, Jan 2010].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - chromatin remodeling\n",
      "    - signal transduction\n",
      "    - central nervous system development\n",
      "    - DNA damage response, signal transduction by p53 class mediator\n",
      "    - DNA damage response, signal transduction by p53 class mediator\n",
      "    - TORC1 signaling\n",
      "    - negative regulation of embryonic development\n",
      "    - negative regulation of ERK1 and ERK2 cascade\n",
      "    - negative regulation of mesoderm development\n",
      "  CC:\n",
      "    - fibrillar center\n",
      "    - nucleoplasm\n",
      "    - nucleoplasm\n",
      "    - nucleolus\n",
      "    - cytoplasm\n",
      "    - mitochondrion\n",
      "    - cytosol\n",
      "  MF:\n",
      "    - magnesium ion binding\n",
      "    - protein kinase activity\n",
      "    - 3-phosphoinositide-dependent protein kinase activity\n",
      "    - DNA-dependent protein kinase activity\n",
      "    - AMP-activated protein kinase activity\n",
      "    - eukaryotic translation initiation factor 2alpha kinase activity\n",
      "    - ribosomal protein S6 kinase activity\n",
      "    - protein binding\n",
      "    - ATP binding\n",
      "    - histone H3S10 kinase activity\n",
      "    - histone H3T11 kinase activity\n",
      "    - histone H3T6 kinase activity\n",
      "    - histone H2AXS139 kinase activity\n",
      "    - histone H3S28 kinase activity\n",
      "    - histone H4S1 kinase activity\n",
      "    - histone H2AS1 kinase activity\n",
      "    - histone H2BS14 kinase activity\n",
      "    - histone H3T3 kinase activity\n",
      "    - histone H2AS121 kinase activity\n",
      "    - Rho-dependent protein serine/threonine kinase activity\n",
      "    - protein serine kinase activity\n",
      "    - histone H2BS36 kinase activity\n",
      "    - histone H3S57 kinase activity\n",
      "    - histone H3T45 kinase activity\n",
      "    - histone H2AT120 kinase activity\n",
      "UniProt: {'Swiss-Prot': 'Q9UK32'}\n",
      "\n",
      "Gene: SLC40A1\n",
      "Symbol: SLC40A1\n",
      "Name: solute carrier family 40 member 1\n",
      "Summary: The protein encoded by this gene is a cell membrane protein that may be involved in iron export from duodenal epithelial cells. Defects in this gene are a cause of hemochromatosis type 4 (HFE4). [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - lymphocyte homeostasis\n",
      "    - endothelium development\n",
      "    - transcription by RNA polymerase II\n",
      "    - intracellular iron ion homeostasis\n",
      "    - apoptotic process\n",
      "    - iron ion transmembrane transport\n",
      "    - iron ion transmembrane transport\n",
      "    - negative regulation of apoptotic process\n",
      "    - positive regulation of transcription by RNA polymerase II\n",
      "    - establishment of localization in cell\n",
      "    - spleen trabecula formation\n",
      "    - multicellular organismal-level iron ion homeostasis\n",
      "    - iron ion export across plasma membrane\n",
      "  CC:\n",
      "    - nucleoplasm\n",
      "    - cytoplasm\n",
      "    - cytosol\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - synaptic vesicle\n",
      "    - membrane\n",
      "    - basolateral plasma membrane\n",
      "    - basolateral plasma membrane\n",
      "    - basolateral plasma membrane\n",
      "    - basolateral plasma membrane\n",
      "  MF:\n",
      "    - iron ion transmembrane transporter activity\n",
      "    - iron ion transmembrane transporter activity\n",
      "    - iron ion transmembrane transporter activity\n",
      "    - protein binding\n",
      "    - ferrous iron transmembrane transporter activity\n",
      "    - ferrous iron transmembrane transporter activity\n",
      "    - peptide hormone binding\n",
      "    - peptide hormone binding\n",
      "    - peptide hormone binding\n",
      "    - identical protein binding\n",
      "    - metal ion binding\n",
      "UniProt: {'Swiss-Prot': 'Q9NP59', 'TrEMBL': ['E7EQF8', 'Q4PNE6', 'E7ES28']}\n",
      "\n",
      "Gene: LAPTM4A\n",
      "Symbol: LAPTM4A\n",
      "Name: lysosomal protein transmembrane 4 alpha\n",
      "Summary: This gene encodes a protein that has four predicted transmembrane domains. The function of this gene has not yet been determined; however, studies in the mouse homolog suggest a role in the transport of small molecules across endosomal and lysosomal membranes. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  CC:\n",
      "    - lysosomal membrane\n",
      "    - lysosomal membrane\n",
      "    - Golgi apparatus\n",
      "    - late endosome membrane\n",
      "  MF:\n",
      "    - protein binding\n",
      "UniProt: {'Swiss-Prot': 'Q15012', 'TrEMBL': 'Q6IBP4'}\n",
      "\n",
      "Gene: NGB\n",
      "Symbol: NGB\n",
      "Name: neuroglobin\n",
      "Summary: This gene encodes an oxygen-binding protein that is distantly related to members of the globin gene family. It is highly conserved among other vertebrates. It is expressed in the central and peripheral nervous system where it may be involved in increasing oxygen availability and providing protection under hypoxic/ischemic conditions. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - response to hypoxia\n",
      "    - oxygen transport\n",
      "    - oxygen transport\n",
      "    - cellular response to hypoxia\n",
      "  CC:\n",
      "    - mitochondrial matrix\n",
      "    - cytosol\n",
      "  MF:\n",
      "    - GDP-dissociation inhibitor activity\n",
      "    - oxygen carrier activity\n",
      "    - oxygen carrier activity\n",
      "    - protein binding\n",
      "    - oxygen binding\n",
      "    - oxygen binding\n",
      "    - heme binding\n",
      "    - metal ion binding\n",
      "    - nitrite reductase activity\n",
      "UniProt: {'Swiss-Prot': 'Q9NPG2', 'TrEMBL': 'A0M8W9'}\n",
      "\n",
      "Gene: DCLK1\n",
      "Symbol: DCLK1\n",
      "Name: doublecortin like kinase 1\n",
      "Summary: This gene encodes a member of the protein kinase superfamily and the doublecortin family. The protein encoded by this gene contains two N-terminal doublecortin domains, which bind microtubules and regulate microtubule polymerization, a C-terminal serine/threonine protein kinase domain, which shows substantial homology to Ca2+/calmodulin-dependent protein kinase, and a serine/proline-rich domain in between the doublecortin and the protein kinase domains, which mediates multiple protein-protein interactions. The microtubule-polymerizing activity of the encoded protein is independent of its protein kinase activity. The encoded protein is involved in several different cellular processes, including neuronal migration, retrograde transport, neuronal apoptosis and neurogenesis. This gene is up-regulated by brain-derived neurotrophic factor and associated with memory and general cognitive abilities. Multiple transcript variants generated by two alternative promoter usage and alternative splicing have been reported, but the full-length nature and biological validity of some variants have not been defined. These variants encode different isoforms, which are differentially expressed and have different kinase activities.[provided by RefSeq, Sep 2010].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - neuron migration\n",
      "    - chromatin remodeling\n",
      "    - protein phosphorylation\n",
      "    - nervous system development\n",
      "    - central nervous system development\n",
      "    - response to virus\n",
      "    - endosomal transport\n",
      "    - central nervous system projection neuron axonogenesis\n",
      "    - forebrain development\n",
      "    - protein localization to nucleus\n",
      "    - intracellular signal transduction\n",
      "    - axon extension\n",
      "    - dendrite morphogenesis\n",
      "    - regulation of postsynapse assembly\n",
      "    - negative regulation of protein localization to nucleus\n",
      "  CC:\n",
      "    - cytoplasm\n",
      "    - plasma membrane\n",
      "    - postsynaptic density\n",
      "    - glutamatergic synapse\n",
      "  MF:\n",
      "    - protein kinase activity\n",
      "    - protein serine/threonine kinase activity\n",
      "    - 3-phosphoinositide-dependent protein kinase activity\n",
      "    - DNA-dependent protein kinase activity\n",
      "    - AMP-activated protein kinase activity\n",
      "    - eukaryotic translation initiation factor 2alpha kinase activity\n",
      "    - ribosomal protein S6 kinase activity\n",
      "    - protein binding\n",
      "    - ATP binding\n",
      "    - histone H3S10 kinase activity\n",
      "    - histone H3T11 kinase activity\n",
      "    - histone H3T6 kinase activity\n",
      "    - histone H2AXS139 kinase activity\n",
      "    - histone H3S28 kinase activity\n",
      "    - histone H4S1 kinase activity\n",
      "    - histone H2AS1 kinase activity\n",
      "    - histone H2BS14 kinase activity\n",
      "    - histone H3T3 kinase activity\n",
      "    - histone H2AS121 kinase activity\n",
      "    - Rho-dependent protein serine/threonine kinase activity\n",
      "    - protein serine kinase activity\n",
      "    - histone H2BS36 kinase activity\n",
      "    - histone H3S57 kinase activity\n",
      "    - histone H3T45 kinase activity\n",
      "    - histone H2AT120 kinase activity\n",
      "UniProt: {'Swiss-Prot': 'O15075', 'TrEMBL': ['B7Z5K4', 'A0A9L9PXT2', 'Q5VZY9']}\n",
      "\n",
      "Gene: IVNS1ABP\n",
      "Symbol: IVNS1ABP\n",
      "Name: influenza virus NS1A binding protein\n",
      "Summary: Involved in RNA splicing; negative regulation of protein ubiquitination; and response to virus. Located in cytosol. Implicated in immunodeficiency 70. [provided by Alliance of Genome Resources, Apr 2022]\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - transcription by RNA polymerase III\n",
      "    - mRNA processing\n",
      "    - RNA splicing\n",
      "    - response to virus\n",
      "    - negative regulation of protein ubiquitination\n",
      "    - proteasome-mediated ubiquitin-dependent protein catabolic process\n",
      "    - intrinsic apoptotic signaling pathway\n",
      "    - negative regulation of intrinsic apoptotic signaling pathway\n",
      "  CC:\n",
      "    - nucleoplasm\n",
      "    - transcription regulator complex\n",
      "    - spliceosomal complex\n",
      "    - cytoplasm\n",
      "    - cytosol\n",
      "    - cytoskeleton\n",
      "    - Cul3-RING ubiquitin ligase complex\n",
      "  MF:\n",
      "    - protein binding\n",
      "    - ubiquitin-like ligase-substrate adaptor activity\n",
      "UniProt: {'Swiss-Prot': 'Q9Y6Y0'}\n",
      "\n",
      "Gene: HSD17B14\n",
      "Symbol: HSD17B14\n",
      "Name: hydroxysteroid 17-beta dehydrogenase 14\n",
      "Summary: 17-beta-hydroxysteroid dehydrogenases, such as HSD17B14, are primarily involved in metabolism of steroids at the C17 position and also of other substrates, such as fatty acids, prostaglandins, and xenobiotics (Lukacik et al., 2007 [PubMed 17067289]).[supplied by OMIM, Jun 2009].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - steroid catabolic process\n",
      "    - L-fucose catabolic process\n",
      "  CC:\n",
      "    - cytosol\n",
      "    - cytosol\n",
      "    - cytosol\n",
      "  MF:\n",
      "    - estradiol 17-beta-dehydrogenase [NAD(P)+] activity\n",
      "    - protein binding\n",
      "    - identical protein binding\n",
      "    - D-threo-aldose 1-dehydrogenase activity\n",
      "UniProt: {'Swiss-Prot': 'Q9BPX1', 'TrEMBL': ['M0R2R3', 'A0A140VJH8', 'M0QY35', 'M0R147']}\n",
      "\n",
      "Gene: CTSB\n",
      "Symbol: CTSB\n",
      "Name: cathepsin B\n",
      "Summary: This gene encodes a member of the C1 family of peptidases. Alternative splicing of this gene results in multiple transcript variants. At least one of these variants encodes a preproprotein that is proteolytically processed to generate multiple protein products. These products include the cathepsin B light and heavy chains, which can dimerize to form the double chain form of the enzyme. This enzyme is a lysosomal cysteine protease with both endopeptidase and exopeptidase activity that may play a role in protein turnover. It is also known as amyloid precursor protein secretase and is involved in the proteolytic processing of amyloid precursor protein (APP). Incomplete proteolytic processing of APP has been suggested to be a causative factor in Alzheimer's disease, the most common cause of dementia. Overexpression of the encoded protein has been associated with esophageal adenocarcinoma and other tumors. Both Cathepsin B and Cathepsin L are involved in the cleavage of the spike protein from the severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) upon its entry to the human host cell. Multiple pseudogenes of this gene have been identified. [provided by RefSeq, Sep 2020].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - proteolysis\n",
      "    - thyroid hormone generation\n",
      "    - collagen catabolic process\n",
      "    - epithelial cell differentiation\n",
      "    - regulation of apoptotic process\n",
      "    - decidualization\n",
      "    - symbiont entry into host cell\n",
      "    - proteolysis involved in protein catabolic process\n",
      "    - proteolysis involved in protein catabolic process\n",
      "    - cellular response to thyroid hormone stimulus\n",
      "  CC:\n",
      "    - extracellular region\n",
      "    - extracellular space\n",
      "    - extracellular space\n",
      "    - extracellular space\n",
      "    - lysosome\n",
      "    - lysosome\n",
      "    - external side of plasma membrane\n",
      "    - apical plasma membrane\n",
      "    - endolysosome lumen\n",
      "    - melanosome\n",
      "    - perinuclear region of cytoplasm\n",
      "    - collagen-containing extracellular matrix\n",
      "    - collagen-containing extracellular matrix\n",
      "    - extracellular exosome\n",
      "    - peptidase inhibitor complex\n",
      "    - ficolin-1-rich granule lumen\n",
      "  MF:\n",
      "    - cysteine-type endopeptidase activity\n",
      "    - cysteine-type endopeptidase activity\n",
      "    - protein binding\n",
      "    - collagen binding\n",
      "    - peptidase activity\n",
      "    - cysteine-type peptidase activity\n",
      "    - proteoglycan binding\n",
      "UniProt: {'Swiss-Prot': 'P07858', 'TrEMBL': ['Q5HYG5', 'A0A024R374', 'A0A7P0NGZ6', 'A0A7I2V668', 'A0A7I2V2J1', 'A0A7I2V367', 'A0A7I2V440', 'E9PCB3', 'E9PR21', 'E9PHZ5', 'A0A7I2V302', 'E9PSG5', 'A0A7I2YQK9', 'A0A7I2V4Z9', 'A0A7I2V4S6']}\n",
      "\n",
      "Gene: GTF2H3\n",
      "Symbol: GTF2H3\n",
      "Name: general transcription factor IIH subunit 3\n",
      "Summary: This gene encodes a member of the TFB4 family. The encoded protein is a subunit of the core-TFIIH basal transcription factor and localizes to the nucleus. The encoded protein is involved in RNA transcription by RNA polymerase II and nucleotide excision repair and associates with the Cdk-activating kinase complex. Alternative splicing results in multiple transcript variants. A related pseudogene has been identified on chromosome 14. [provided by RefSeq, Dec 2012].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - DNA repair\n",
      "    - nucleotide-excision repair\n",
      "    - regulation of DNA-templated transcription\n",
      "    - transcription by RNA polymerase II\n",
      "  CC:\n",
      "    - core TFIIH complex portion of holo TFIIH complex\n",
      "    - transcription factor TFIIH core complex\n",
      "    - nucleus\n",
      "    - nucleoplasm\n",
      "    - nucleoplasm\n",
      "    - transcription factor TFIID complex\n",
      "    - transcription factor TFIIH holo complex\n",
      "    - transcription factor TFIIH holo complex\n",
      "    - transcription preinitiation complex\n",
      "  MF:\n",
      "    - protein binding\n",
      "    - zinc ion binding\n",
      "    - RNA polymerase II general transcription initiation factor activity\n",
      "UniProt: {'Swiss-Prot': 'Q13889', 'TrEMBL': ['F5GXG6', 'A0A087WYD5', 'F5H3G1', 'F5GWD3', 'F5GX35', 'H0YFF7', 'F5H4K7', 'F5H6X0']}\n",
      "\n",
      "Gene: PTGER4\n",
      "Symbol: PTGER4\n",
      "Name: prostaglandin E receptor 4\n",
      "Summary: The protein encoded by this gene is a member of the G-protein coupled receptor family. This protein is one of four receptors identified for prostaglandin E2 (PGE2). This receptor can activate T-cell factor signaling. It has been shown to mediate PGE2 induced expression of early growth response 1 (EGR1), regulate the level and stability of cyclooxygenase-2 mRNA, and lead to the phosphorylation of glycogen synthase kinase-3. Knockout studies in mice suggest that this receptor may be involved in the neonatal adaptation of circulatory system, osteoporosis, as well as initiation of skin immune responses. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - negative regulation of cytokine production\n",
      "    - positive regulation of cytokine production\n",
      "    - inflammatory response\n",
      "    - immune response\n",
      "    - adenylate cyclase-modulating G protein-coupled receptor signaling pathway\n",
      "    - adenylate cyclase-activating G protein-coupled receptor signaling pathway\n",
      "    - adenylate cyclase-activating G protein-coupled receptor signaling pathway\n",
      "    - adenylate cyclase-inhibiting G protein-coupled receptor signaling pathway\n",
      "    - positive regulation of cytosolic calcium ion concentration\n",
      "    - JNK cascade\n",
      "    - response to mechanical stimulus\n",
      "    - response to nematode\n",
      "    - regulation of ossification\n",
      "    - response to lipopolysaccharide\n",
      "    - negative regulation of integrin activation\n",
      "    - response to prostaglandin E\n",
      "    - T-helper cell differentiation\n",
      "    - negative regulation of inflammatory response\n",
      "    - negative regulation of inflammatory response\n",
      "    - positive regulation of inflammatory response\n",
      "    - regulation of stress fiber assembly\n",
      "    - bone development\n",
      "    - ERK1 and ERK2 cascade\n",
      "    - cellular response to mechanical stimulus\n",
      "    - cellular response to prostaglandin E stimulus\n",
      "    - negative regulation of eosinophil extravasation\n",
      "  CC:\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - membrane\n",
      "  MF:\n",
      "    - prostaglandin E receptor activity\n",
      "    - prostaglandin E receptor activity\n",
      "    - protein binding\n",
      "UniProt: {'Swiss-Prot': 'P35408', 'TrEMBL': 'A0PJF5'}\n",
      "\n",
      "Gene: KIF22\n",
      "Symbol: KIF22\n",
      "Name: kinesin family member 22\n",
      "Summary: The protein encoded by this gene is a member of the kinesin-like protein family. The family members are microtubule-dependent molecular motors that transport organelles within cells and move chromosomes during cell division. The C-terminal half of this protein has been shown to bind DNA. Studies with the Xenopus homolog suggests its essential role in metaphase chromosome alignment and maintenance. Alternatively spliced transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Jan 2012].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - mitotic cell cycle\n",
      "    - DNA repair\n",
      "    - microtubule-based movement\n",
      "    - sister chromatid cohesion\n",
      "    - mitotic metaphase chromosome alignment\n",
      "    - metaphase chromosome alignment\n",
      "  CC:\n",
      "    - kinetochore\n",
      "    - chromatin\n",
      "    - nucleus\n",
      "    - cytoplasm\n",
      "    - cytosol\n",
      "    - cytosol\n",
      "    - kinesin complex\n",
      "    - microtubule\n",
      "    - nuclear speck\n",
      "    - mitotic spindle\n",
      "  MF:\n",
      "    - DNA binding\n",
      "    - microtubule motor activity\n",
      "    - protein binding\n",
      "    - ATP binding\n",
      "    - microtubule binding\n",
      "    - ATP hydrolysis activity\n",
      "UniProt: {'Swiss-Prot': 'Q14807', 'TrEMBL': ['A0A8I5QL17', 'B7Z9T5', 'A0A8I5QJC5', 'A0A8I5KRF7', 'H3BQB9', 'H3BTH5', 'H3BRB3', 'A0A8I5KXF2', 'A0A8I5QJF2', 'A0A8I5KSP6', 'A0A8I5KW08', 'A0A8I5QKY3', 'I3L306', 'A0A8I5KXF9', 'A0A8I5KQ45', 'A0A8I5KTF7']}\n",
      "\n",
      "Gene: BTG3\n",
      "Symbol: BTG3\n",
      "Name: BTG anti-proliferation factor 3\n",
      "Summary: The protein encoded by this gene is a member of the BTG/Tob family. This family has structurally related proteins that appear to have antiproliferative properties. This encoded protein might play a role in neurogenesis in the central nervous system. Two transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Aug 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - negative regulation of cell population proliferation\n",
      "    - negative regulation of mitotic cell cycle\n",
      "  CC:\n",
      "    - nucleus\n",
      "    - cytoplasm\n",
      "    - cytoplasm\n",
      "  MF:\n",
      "    - protein binding\n",
      "UniProt: {'Swiss-Prot': 'Q14201', 'TrEMBL': ['C9JLA2', 'Q6IAU3']}\n",
      "\n",
      "Gene: COX6A1\n",
      "Symbol: COX6A1\n",
      "Name: cytochrome c oxidase subunit 6A1\n",
      "Summary: Cytochrome c oxidase (COX), the terminal enzyme of the mitochondrial respiratory chain, catalyzes the electron transfer from reduced cytochrome c to oxygen. It is a heteromeric complex consisting of 3 catalytic subunits encoded by mitochondrial genes and multiple structural subunits encoded by nuclear genes. The mitochondrially-encoded subunits function in the electron transfer and the nuclear-encoded subunits may function in the regulation and assembly of the complex. This nuclear gene encodes polypeptide 1 (liver isoform) of subunit VIa, and polypeptide 1 is found in all non-muscle tissues. Polypeptide 2 (heart/muscle isoform) of subunit VIa is encoded by a different gene, and is present only in striated muscles. These two polypeptides share 66% amino acid sequence identity. It has been reported that there may be several pseudogenes on chromosomes 1, 6, 7q21, 7q31-32 and 12. However, only one pseudogene (COX6A1P) on chromosome 1p31.1 has been documented. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - generation of precursor metabolites and energy\n",
      "    - mitochondrial electron transport, cytochrome c to oxygen\n",
      "    - mitochondrial electron transport, cytochrome c to oxygen\n",
      "    - cellular respiration\n",
      "  CC:\n",
      "    - mitochondrion\n",
      "    - mitochondrion\n",
      "    - mitochondrial inner membrane\n",
      "    - mitochondrial membrane\n",
      "    - respiratory chain complex IV\n",
      "  MF:\n",
      "    - protein binding\n",
      "    - oxidoreductase activity\n",
      "    - enzyme regulator activity\n",
      "UniProt: {'Swiss-Prot': 'P12074', 'TrEMBL': 'H6SG15'}\n",
      "\n",
      "Gene: MLEC\n",
      "Symbol: MLEC\n",
      "Name: malectin\n",
      "Summary: This gene encodes the carbohydrate-binding protein malectin which is a Type I membrane-anchored endoplasmic reticulum protein. This protein has an affinity for Glc2Man9GlcNAc2 (G2M9) N-glycans and is involved in regulating glycosylation in the endoplasmic reticulum. This protein has also been shown to interact with ribophorin I and may be involved in the directing the degradation of misfolded proteins. Alternate splicing results in multiple transcript variants. [provided by RefSeq, Jan 2015].\n",
      "GO Annotations:\n",
      "  CC:\n",
      "    - endoplasmic reticulum\n",
      "    - endoplasmic reticulum membrane\n",
      "    - endoplasmic reticulum membrane\n",
      "    - plasma membrane\n",
      "    - oligosaccharyltransferase complex\n",
      "    - membrane\n",
      "    - membrane\n",
      "    - specific granule membrane\n",
      "  MF:\n",
      "    - protein binding\n",
      "    - enzyme binding\n",
      "    - carbohydrate binding\n",
      "UniProt: {'Swiss-Prot': 'Q14165', 'TrEMBL': ['H0YG07', 'F5GX14', 'F5H1S8']}\n",
      "\n",
      "Gene: UGT1A4\n",
      "Symbol: UGT1A4\n",
      "Name: UDP glucuronosyltransferase family 1 member A4\n",
      "Summary: This gene encodes a UDP-glucuronosyltransferase, an enzyme of the glucuronidation pathway that transforms small lipophilic molecules, such as steroids, bilirubin, hormones, and drugs, into water-soluble, excretable metabolites. This gene is part of a complex locus that encodes several UDP-glucuronosyltransferases. The locus includes thirteen unique alternate first exons followed by four common exons. Four of the alternate first exons are considered pseudogenes. Each of the remaining nine 5' exons may be spliced to the four common exons, resulting in nine proteins with different N-termini and identical C-termini. Each first exon encodes the substrate binding site, and is regulated by its own promoter. This enzyme has some glucuronidase activity towards bilirubin, although is is more active on amines, steroids, and sapogenins. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - bilirubin conjugation\n",
      "    - xenobiotic metabolic process\n",
      "    - heme catabolic process\n",
      "    - negative regulation of fatty acid metabolic process\n",
      "    - vitamin D3 metabolic process\n",
      "  CC:\n",
      "    - endoplasmic reticulum\n",
      "    - endoplasmic reticulum\n",
      "    - endoplasmic reticulum\n",
      "    - endoplasmic reticulum membrane\n",
      "    - endoplasmic reticulum membrane\n",
      "    - endoplasmic reticulum membrane\n",
      "  MF:\n",
      "    - retinoic acid binding\n",
      "    - enzyme inhibitor activity\n",
      "    - glucuronosyltransferase activity\n",
      "    - glucuronosyltransferase activity\n",
      "    - glucuronosyltransferase activity\n",
      "    - glucuronosyltransferase activity\n",
      "    - enzyme binding\n",
      "    - enzyme binding\n",
      "    - protein homodimerization activity\n",
      "    - protein heterodimerization activity\n",
      "    - protein heterodimerization activity\n",
      "UniProt: {'Swiss-Prot': 'P22310'}\n",
      "\n",
      "Gene: UGT1A9\n",
      "Symbol: UGT1A9\n",
      "Name: UDP glucuronosyltransferase family 1 member A9\n",
      "Summary: This gene encodes a UDP-glucuronosyltransferase, an enzyme of the glucuronidation pathway that transforms small lipophilic molecules, such as steroids, bilirubin, hormones, and drugs, into water-soluble, excretable metabolites. This gene is part of a complex locus that encodes several UDP-glucuronosyltransferases. The locus includes thirteen unique alternate first exons followed by four common exons. Four of the alternate first exons are considered pseudogenes. Each of the remaining nine 5' exons may be spliced to the four common exons, resulting in nine proteins with different N-termini and identical C-termini. Each first exon encodes the substrate binding site, and is regulated by its own promoter. The enzyme encoded by this gene is active on phenols. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - liver development\n",
      "    - xenobiotic metabolic process\n",
      "    - xenobiotic metabolic process\n",
      "    - xenobiotic metabolic process\n",
      "    - flavonoid metabolic process\n",
      "    - retinoic acid metabolic process\n",
      "    - negative regulation of fatty acid metabolic process\n",
      "    - flavone metabolic process\n",
      "  CC:\n",
      "    - endoplasmic reticulum\n",
      "    - endoplasmic reticulum\n",
      "    - endoplasmic reticulum\n",
      "    - endoplasmic reticulum membrane\n",
      "    - endoplasmic reticulum membrane\n",
      "    - endoplasmic reticulum membrane\n",
      "  MF:\n",
      "    - retinoic acid binding\n",
      "    - enzyme inhibitor activity\n",
      "    - protein binding\n",
      "    - glucuronosyltransferase activity\n",
      "    - glucuronosyltransferase activity\n",
      "    - glucuronosyltransferase activity\n",
      "    - glucuronosyltransferase activity\n",
      "    - enzyme binding\n",
      "    - enzyme binding\n",
      "    - identical protein binding\n",
      "    - protein homodimerization activity\n",
      "    - protein heterodimerization activity\n",
      "    - protein heterodimerization activity\n",
      "UniProt: {'Swiss-Prot': 'O60656', 'TrEMBL': 'Q5DSZ5'}\n",
      "\n",
      "Gene: ESRRB\n",
      "Symbol: ESRRB\n",
      "Name: estrogen related receptor beta\n",
      "Summary: This gene encodes a protein with similarity to the estrogen receptor. Its function is unknown; however, a similar protein in mouse plays an essential role in placental development. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - embryonic placenta development\n",
      "    - regulation of DNA-templated transcription\n",
      "    - regulation of transcription by RNA polymerase II\n",
      "    - regulation of transcription by RNA polymerase II\n",
      "    - cell population proliferation\n",
      "    - stem cell division\n",
      "    - stem cell population maintenance\n",
      "    - nuclear receptor-mediated steroid hormone signaling pathway\n",
      "    - somatic stem cell population maintenance\n",
      "    - cell dedifferentiation\n",
      "    - photoreceptor cell maintenance\n",
      "    - positive regulation of DNA-templated transcription\n",
      "    - positive regulation of transcription by RNA polymerase II\n",
      "    - inner ear development\n",
      "    - morula formation\n",
      "    - positive regulation of stem cell population maintenance\n",
      "    - regulation of stem cell division\n",
      "    - negative regulation of stem cell differentiation\n",
      "  CC:\n",
      "    - chromatin\n",
      "    - chromatin\n",
      "    - condensed chromosome\n",
      "    - nucleus\n",
      "    - nucleus\n",
      "    - nucleoplasm\n",
      "    - nucleoplasm\n",
      "    - cytoplasm\n",
      "    - cytosol\n",
      "    - integrator complex\n",
      "  MF:\n",
      "    - RNA polymerase II cis-regulatory region sequence-specific DNA binding\n",
      "    - DNA-binding transcription factor activity, RNA polymerase II-specific\n",
      "    - DNA-binding transcription factor activity, RNA polymerase II-specific\n",
      "    - DNA-binding transcription factor activity, RNA polymerase II-specific\n",
      "    - cis-regulatory region sequence-specific DNA binding\n",
      "    - RNA polymerase II complex binding\n",
      "    - DNA-binding transcription activator activity, RNA polymerase II-specific\n",
      "    - DNA-binding transcription factor activity\n",
      "    - nuclear steroid receptor activity\n",
      "    - nuclear receptor activity\n",
      "    - nuclear receptor activity\n",
      "    - steroid binding\n",
      "    - protein binding\n",
      "    - zinc ion binding\n",
      "    - estrogen response element binding\n",
      "    - sequence-specific DNA binding\n",
      "    - RNA polymerase II-specific DNA-binding transcription factor binding\n",
      "    - sequence-specific double-stranded DNA binding\n",
      "UniProt: {'Swiss-Prot': 'O95718', 'TrEMBL': ['A0A2R8Y491', 'E7EWD9', 'A0A2R8Y509']}\n",
      "\n",
      "Gene: KIRREL3\n",
      "Symbol: KIRREL3\n",
      "Name: kirre like nephrin family adhesion molecule 3\n",
      "Summary: The protein encoded by this gene is a member of the nephrin-like protein family. These proteins are expressed in fetal and adult brain, and also in podocytes of kidney glomeruli. The cytoplasmic domains of these proteins interact with the C-terminus of podocin, also expressed in the podocytes, cells involved in ensuring size- and charge-selective ultrafiltration. The protein encoded by this gene is a synaptic cell adhesion molecule with multiple extracellular immunoglobulin-like domains and a cytoplasmic PDZ domain-binding motif. Mutations in this gene are associated with several neurological and cognitive disorders. Alternatively spliced transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Jul 2017].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - neuron migration\n",
      "    - inter-male aggressive behavior\n",
      "    - homophilic cell adhesion via plasma membrane adhesion molecules\n",
      "    - synapse assembly\n",
      "    - synapse assembly\n",
      "    - principal sensory nucleus of trigeminal nerve development\n",
      "    - hippocampus development\n",
      "    - hemopoiesis\n",
      "    - neuron projection morphogenesis\n",
      "    - glomerulus morphogenesis\n",
      "    - cell-cell adhesion\n",
      "  CC:\n",
      "    - extracellular region\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - cell-cell junction\n",
      "    - synaptic vesicle\n",
      "    - membrane\n",
      "    - axon\n",
      "    - dendrite\n",
      "    - dendritic shaft\n",
      "  MF:\n",
      "    - protein binding\n",
      "    - cell adhesion molecule binding\n",
      "UniProt: {'Swiss-Prot': 'Q8IZU9', 'TrEMBL': 'E9PRX9'}\n",
      "\n",
      "Gene: DCP1B\n",
      "Symbol: DCP1B\n",
      "Name: decapping mRNA 1B\n",
      "Summary: This gene encodes a member of a family of proteins that function in removing the 5' cap from mRNAs, which is a step in regulated mRNA decay. This protein localizes to cytoplasmic foci which are the site of mRNA breakdown and turnover. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Feb 2016].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - nuclear-transcribed mRNA catabolic process, nonsense-mediated decay\n",
      "    - deadenylation-dependent decapping of nuclear-transcribed mRNA\n",
      "    - deadenylation-independent decapping of nuclear-transcribed mRNA\n",
      "  CC:\n",
      "    - P-body\n",
      "    - nucleus\n",
      "    - cytosol\n",
      "    - cytosol\n",
      "    - membrane\n",
      "    - intracellular membrane-bounded organelle\n",
      "  MF:\n",
      "    - mRNA binding\n",
      "    - protein binding\n",
      "    - enzyme activator activity\n",
      "    - 5'-(N(7)-methylguanosine 5'-triphospho)-[mRNA] hydrolase activity\n",
      "UniProt: {'Swiss-Prot': 'Q8IZD4', 'TrEMBL': ['F5GZK9', 'B4DVJ6', 'F5H4R4']}\n",
      "\n",
      "Gene: CBR1\n",
      "Symbol: CBR1\n",
      "Name: carbonyl reductase 1\n",
      "Summary: The protein encoded by this gene belongs to the short-chain dehydrogenases/reductases (SDR) family, which function as NADPH-dependent oxidoreductases having wide specificity for carbonyl compounds, such as quinones, prostaglandins, and various xenobiotics. Alternatively spliced transcript variants have been found for this gene. [provided by RefSeq, Nov 2013].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - xenobiotic metabolic process\n",
      "    - glucocorticoid metabolic process\n",
      "    - epithelial cell differentiation\n",
      "    - vitamin K metabolic process\n",
      "    - prostanoid biosynthetic process\n",
      "    - positive regulation of reactive oxygen species metabolic process\n",
      "  CC:\n",
      "    - cytosol\n",
      "    - extracellular exosome\n",
      "    - extracellular vesicle\n",
      "  MF:\n",
      "    - carbonyl reductase (NADPH) activity\n",
      "    - carbonyl reductase (NADPH) activity\n",
      "    - protein binding\n",
      "    - oxidoreductase activity, acting on the CH-OH group of donors, NAD or NADP as acceptor\n",
      "    - oxidoreductase activity, acting on NAD(P)H, quinone or similar compound as acceptor\n",
      "    - 15-hydroxyprostaglandin-D dehydrogenase (NADP+) activity\n",
      "    - 15-hydroxyprostaglandin dehydrogenase (NADP+) activity\n",
      "    - prostaglandin E2 9-reductase activity\n",
      "    - prostaglandin E2 9-reductase activity\n",
      "    - S-nitrosoglutathione reductase (NADPH) activity\n",
      "UniProt: {'Swiss-Prot': 'P16152', 'TrEMBL': ['E9PQ63', 'A0A384NL53', 'A8MTM1']}\n",
      "\n",
      "Gene: AGPAT4\n",
      "Symbol: AGPAT4\n",
      "Name: 1-acylglycerol-3-phosphate O-acyltransferase 4\n",
      "Summary: This gene encodes a member of the 1-acylglycerol-3-phosphate O-acyltransferase family. This integral membrane protein converts lysophosphatidic acid to phosphatidic acid, the second step in de novo phospholipid biosynthesis. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - phosphatidic acid biosynthetic process\n",
      "    - phospholipid biosynthetic process\n",
      "    - CDP-diacylglycerol biosynthetic process\n",
      "  CC:\n",
      "    - mitochondrial outer membrane\n",
      "    - endoplasmic reticulum\n",
      "    - endoplasmic reticulum membrane\n",
      "    - endomembrane system\n",
      "  MF:\n",
      "    - 1-acylglycerol-3-phosphate O-acyltransferase activity\n",
      "    - 1-acylglycerol-3-phosphate O-acyltransferase activity\n",
      "    - 1-acylglycerol-3-phosphate O-acyltransferase activity\n",
      "    - 1-acylglycerol-3-phosphate O-acyltransferase activity\n",
      "    - protein binding\n",
      "UniProt: {'Swiss-Prot': 'Q9NRZ5', 'TrEMBL': ['Q5TEE8', 'G3XAF1', 'Q6AI25', 'H0Y5R3']}\n",
      "\n",
      "Gene: CYP3A5\n",
      "Symbol: CYP3A5\n",
      "Name: cytochrome P450 family 3 subfamily A member 5\n",
      "Summary: This gene encodes a member of the cytochrome P450 superfamily of enzymes. The cytochrome P450 proteins are monooxygenases which catalyze many reactions involved in drug metabolism and synthesis of cholesterol, steroids and other lipids. The encoded protein metabolizes drugs as well as the steroid hormones testosterone and progesterone. This gene is part of a cluster of cytochrome P450 genes on chromosome 7q21.1. Two pseudogenes of this gene have been identified within this cluster on chromosome 7. Expression of this gene is widely variable among populations, and a single nucleotide polymorphism that affects transcript splicing has been associated with susceptibility to hypertensions. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Apr 2014].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - lipid hydroxylation\n",
      "    - xenobiotic metabolic process\n",
      "    - steroid metabolic process\n",
      "    - steroid metabolic process\n",
      "    - estrogen metabolic process\n",
      "    - alkaloid catabolic process\n",
      "    - xenobiotic catabolic process\n",
      "    - retinol metabolic process\n",
      "    - retinoic acid metabolic process\n",
      "    - aflatoxin metabolic process\n",
      "    - oxidative demethylation\n",
      "    - oxidative demethylation\n",
      "  CC:\n",
      "    - endoplasmic reticulum membrane\n",
      "    - intracellular membrane-bounded organelle\n",
      "  MF:\n",
      "    - monooxygenase activity\n",
      "    - monooxygenase activity\n",
      "    - iron ion binding\n",
      "    - protein binding\n",
      "    - retinoic acid 4-hydroxylase activity\n",
      "    - oxidoreductase activity\n",
      "    - oxygen binding\n",
      "    - heme binding\n",
      "    - testosterone 6-beta-hydroxylase activity\n",
      "    - estrogen 16-alpha-hydroxylase activity\n",
      "    - estrogen 16-alpha-hydroxylase activity\n",
      "    - estrogen 2-hydroxylase activity\n",
      "UniProt: {'Swiss-Prot': 'P20815', 'TrEMBL': ['Q7Z3N0', 'B7Z3P6']}\n",
      "\n",
      "Gene: NUBPL\n",
      "Symbol: NUBPL\n",
      "Name: NUBP iron-sulfur cluster assembly factor, mitochondrial\n",
      "Summary: This gene encodes a member of the Mrp/NBP35 ATP-binding proteins family. The encoded protein is required for the assembly of the respiratory chain NADH dehydrogenase (complex I), an oligomeric enzymatic complex located in the inner mitochondrial membrane. Mutations in this gene cause mitochondrial complex I deficiency. Alternative splicing results in multiple transcript variants. [provided by RefSeq, May 2014].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - mitochondrion organization\n",
      "    - iron-sulfur cluster assembly\n",
      "    - mitochondrial respiratory chain complex I assembly\n",
      "    - mitochondrial respiratory chain complex I assembly\n",
      "  CC:\n",
      "    - mitochondrion\n",
      "    - mitochondrion\n",
      "    - mitochondrion\n",
      "    - mitochondrial matrix\n",
      "    - plasma membrane\n",
      "  MF:\n",
      "    - protein binding\n",
      "    - ATP binding\n",
      "    - metal ion binding\n",
      "    - 4 iron, 4 sulfur cluster binding\n",
      "    - 4 iron, 4 sulfur cluster binding\n",
      "    - ATP-dependent FeS chaperone activity\n",
      "UniProt: {'Swiss-Prot': 'Q8TB37', 'TrEMBL': ['B4DWB0', 'X5D2R5', 'B3KSK2', 'F8W0A2', 'H0YHR7', 'F8VP02', 'F8VZR8', 'F8W061']}\n",
      "\n",
      "Gene: SLC22A2\n",
      "Symbol: SLC22A2\n",
      "Name: solute carrier family 22 member 2\n",
      "Summary: Polyspecific organic cation transporters in the liver, kidney, intestine, and other organs are critical for elimination of many endogenous small organic cations as well as a wide array of drugs and environmental toxins. This gene is one of three similar cation transporter genes located in a cluster on chromosome 6. The encoded protein contains twelve putative transmembrane domains and is a plasma integral membrane protein. It is found primarily in the kidney, where it may mediate the first step in cation reabsorption. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - monoatomic cation transport\n",
      "    - neurotransmitter transport\n",
      "    - neurotransmitter transport\n",
      "    - serotonin transport\n",
      "    - body fluid secretion\n",
      "    - organic cation transport\n",
      "    - organic cation transport\n",
      "    - organic cation transport\n",
      "    - organic cation transport\n",
      "    - quaternary ammonium group transport\n",
      "    - prostaglandin transport\n",
      "    - amine transport\n",
      "    - putrescine transport\n",
      "    - spermidine transport\n",
      "    - acetylcholine transport\n",
      "    - choline transport\n",
      "    - dopamine transport\n",
      "    - dopamine transport\n",
      "    - norepinephrine transport\n",
      "    - xenobiotic transport\n",
      "    - xenobiotic transport\n",
      "    - xenobiotic transport\n",
      "    - epinephrine transport\n",
      "    - epinephrine transport\n",
      "    - histamine transport\n",
      "    - serotonin uptake\n",
      "    - histamine uptake\n",
      "    - norepinephrine uptake\n",
      "    - norepinephrine uptake\n",
      "    - thiamine transmembrane transport\n",
      "    - purine-containing compound transmembrane transport\n",
      "    - amino acid import across plasma membrane\n",
      "    - dopamine uptake\n",
      "    - L-arginine import across plasma membrane\n",
      "    - export across plasma membrane\n",
      "    - transport across blood-brain barrier\n",
      "    - L-alpha-amino acid transmembrane transport\n",
      "    - spermidine transmembrane transport\n",
      "    - L-arginine transmembrane transport\n",
      "    - cellular detoxification\n",
      "    - cellular detoxification\n",
      "    - xenobiotic transport across blood-brain barrier\n",
      "  CC:\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - basal plasma membrane\n",
      "    - membrane\n",
      "    - basolateral plasma membrane\n",
      "    - apical plasma membrane\n",
      "    - extracellular exosome\n",
      "    - presynapse\n",
      "  MF:\n",
      "    - amine transmembrane transporter activity\n",
      "    - acetylcholine transmembrane transporter activity\n",
      "    - neurotransmitter transmembrane transporter activity\n",
      "    - neurotransmitter transmembrane transporter activity\n",
      "    - monoamine transmembrane transporter activity\n",
      "    - organic anion transmembrane transporter activity\n",
      "    - organic cation transmembrane transporter activity\n",
      "    - organic cation transmembrane transporter activity\n",
      "    - organic cation transmembrane transporter activity\n",
      "    - organic cation transmembrane transporter activity\n",
      "    - organic cation transmembrane transporter activity\n",
      "    - prostaglandin transmembrane transporter activity\n",
      "    - L-amino acid transmembrane transporter activity\n",
      "    - pyrimidine nucleoside transmembrane transporter activity\n",
      "    - choline transmembrane transporter activity\n",
      "    - thiamine transmembrane transporter activity\n",
      "    - putrescine transmembrane transporter activity\n",
      "    - efflux transmembrane transporter activity\n",
      "    - spermidine transmembrane transporter activity\n",
      "    - quaternary ammonium group transmembrane transporter activity\n",
      "    - toxin transmembrane transporter activity\n",
      "    - toxin transmembrane transporter activity\n",
      "    - toxin transmembrane transporter activity\n",
      "    - xenobiotic transmembrane transporter activity\n",
      "    - xenobiotic transmembrane transporter activity\n",
      "    - L-arginine transmembrane transporter activity\n",
      "UniProt: {'Swiss-Prot': 'O15244', 'TrEMBL': 'Q5T7Q5'}\n",
      "\n",
      "Gene: KRT25\n",
      "Symbol: KRT25\n",
      "Name: keratin 25\n",
      "Summary: This gene encodes a member of the type I (acidic) keratin family, which belongs to the superfamily of intermediate filament (IF) proteins. Keratins are heteropolymeric structural proteins which form the intermediate filament. These filaments, along with actin microfilaments and microtubules, compose the cytoskeleton of epithelial cells. The type I keratin genes are clustered in a region of chromosome 17q12-q21. [provided by RefSeq, Jul 2009].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - cytoskeleton organization\n",
      "    - epithelial cell differentiation\n",
      "    - hair follicle morphogenesis\n",
      "    - hair cycle\n",
      "    - intermediate filament organization\n",
      "  CC:\n",
      "    - cytosol\n",
      "    - cytoskeleton\n",
      "    - keratin filament\n",
      "    - extracellular exosome\n",
      "  MF:\n",
      "    - structural molecule activity\n",
      "    - protein binding\n",
      "    - protein heterodimerization activity\n",
      "    - protein heterodimerization activity\n",
      "UniProt: {'Swiss-Prot': 'Q7Z3Z0', 'TrEMBL': 'Q6ZPD6'}\n",
      "\n",
      "Gene: LCE1E\n",
      "Symbol: LCE1E\n",
      "Name: late cornified envelope 1E\n",
      "Summary: Enables identical protein binding activity. Predicted to be involved in keratinization. [provided by Alliance of Genome Resources, Apr 2022]\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - keratinization\n",
      "  MF:\n",
      "    - protein binding\n",
      "    - identical protein binding\n",
      "UniProt: {'Swiss-Prot': 'Q5T753', 'TrEMBL': 'A0A087WWQ4'}\n",
      "\n",
      "Gene: KRTAP19-3\n",
      "Symbol: KRTAP19-3\n",
      "Name: keratin associated protein 19-3\n",
      "Summary: Predicted to be located in cytosol. [provided by Alliance of Genome Resources, Apr 2022]\n",
      "GO Annotations:\n",
      "  CC:\n",
      "    - cytosol\n",
      "    - intermediate filament\n",
      "  MF:\n",
      "    - protein binding\n",
      "UniProt: {'Swiss-Prot': 'Q7Z4W3'}\n",
      "\n",
      "Gene: MS4A1\n",
      "Symbol: MS4A1\n",
      "Name: membrane spanning 4-domains A1\n",
      "Summary: This gene encodes a member of the membrane-spanning 4A gene family. Members of this nascent protein family are characterized by common structural features and similar intron/exon splice boundaries and display unique expression patterns among hematopoietic cells and nonlymphoid tissues. This gene encodes a B-lymphocyte surface molecule which plays a role in the development and differentiation of B-cells into plasma cells. This family member is localized to 11q12, among a cluster of family members. Alternative splicing of this gene results in two transcript variants which encode the same protein. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - store-operated calcium entry\n",
      "    - humoral immune response\n",
      "    - cell surface receptor signaling pathway\n",
      "    - response to bacterium\n",
      "    - B cell differentiation\n",
      "    - B cell proliferation\n",
      "    - B cell activation\n",
      "    - B cell receptor signaling pathway\n",
      "    - protein tetramerization\n",
      "    - calcium ion import into cytosol\n",
      "    - positive regulation of calcium ion import across plasma membrane\n",
      "  CC:\n",
      "    - extracellular space\n",
      "    - nucleoplasm\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - external side of plasma membrane\n",
      "    - cell surface\n",
      "    - plasma membrane raft\n",
      "    - extracellular exosome\n",
      "  MF:\n",
      "    - epidermal growth factor receptor binding\n",
      "    - protein binding\n",
      "    - immunoglobulin binding\n",
      "    - MHC class II protein complex binding\n",
      "    - identical protein binding\n",
      "UniProt: {'Swiss-Prot': 'P11836', 'TrEMBL': ['E9PPL6', 'E9PKH8']}\n",
      "\n",
      "Gene: PORCN\n",
      "Symbol: PORCN\n",
      "Name: porcupine O-acyltransferase\n",
      "Summary: This gene belongs to the evolutionarily conserved porcupine (Porc) gene family. Genes of the porcupine family encode endoplasmic reticulum proteins with multiple transmembrane domains. Porcupine proteins are involved in the processing of Wnt (wingless and int homologue) proteins. Disruption of this gene is associated with focal dermal hypoplasia, and the encoded protein has been implicated in cancer. Multiple alternatively spliced transcript variants encoding distinct isoforms have been observed. [provided by RefSeq, Aug 2013].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - protein lipidation\n",
      "    - glycoprotein metabolic process\n",
      "    - Wnt signaling pathway\n",
      "    - lipid modification\n",
      "    - protein palmitoleylation\n",
      "    - Wnt protein secretion\n",
      "    - regulation of postsynaptic membrane neurotransmitter receptor levels\n",
      "  CC:\n",
      "    - endoplasmic reticulum\n",
      "    - endoplasmic reticulum\n",
      "    - endoplasmic reticulum membrane\n",
      "    - membrane\n",
      "    - AMPA glutamate receptor complex\n",
      "    - glutamatergic synapse\n",
      "  MF:\n",
      "    - protein binding\n",
      "    - Wnt-protein binding\n",
      "    - palmitoleoyltransferase activity\n",
      "    - palmitoleoyltransferase activity\n",
      "UniProt: {'Swiss-Prot': 'Q9H237', 'TrEMBL': ['C9JWI5', 'F2Z360', 'F8WEW7']}\n",
      "\n",
      "Gene: DNAH6\n",
      "Symbol: DNAH6\n",
      "Name: dynein axonemal heavy chain 6\n",
      "Summary: This gene belongs to the dynein family, whose members encode large proteins that are constituents of the microtubule-associated motor protein complex. This complex is composed of dynein heavy, intermediate and light chains, which can be axonemal or cytoplasmic. This protein is an axonemal dynein heavy chain. It is involved in producing force for ciliary beating by using energy from ATP hydrolysis. Mutations in this gene may cause primary ciliary dyskinesia (PCD) as well as heterotaxy. [provided by RefSeq, Jun 2016].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - cilium movement involved in cell motility\n",
      "  CC:\n",
      "    - microtubule\n",
      "    - axoneme\n",
      "    - dynein complex\n",
      "    - 9+2 motile cilium\n",
      "  MF:\n",
      "    - ATP binding\n",
      "    - minus-end-directed microtubule motor activity\n",
      "    - ATP hydrolysis activity\n",
      "    - dynein intermediate chain binding\n",
      "    - dynein light intermediate chain binding\n",
      "UniProt: {'Swiss-Prot': 'Q9C0G6'}\n",
      "\n",
      "Gene: AICDA\n",
      "Symbol: AICDA\n",
      "Name: activation induced cytidine deaminase\n",
      "Summary: This gene encodes a RNA-editing deaminase that is a member of the cytidine deaminase family. AICDA is specifically expressed and active in germinal center-like B cells. In the germinal center, AICDA is involved in somatic hypermutation, gene conversion, and class-switch recombination of immunoglobulin genes. An epigenetic role in neoplastic transformation and lymphoma progression has been experimentally ascribed to AICDA using mouse models. Defects in this gene are the cause of autosomal recessive hyper-IgM immunodeficiency syndrome type 2 (HIGM2). [provided by RefSeq, Jul 2020].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - mRNA processing\n",
      "    - cytidine deamination\n",
      "    - somatic diversification of immunoglobulins\n",
      "    - somatic hypermutation of immunoglobulin genes\n",
      "    - cytidine to uridine editing\n",
      "    - B cell differentiation\n",
      "    - regulation of nuclear cell cycle DNA replication\n",
      "    - defense response to bacterium\n",
      "    - positive regulation of gene expression via chromosomal CpG island demethylation\n",
      "    - isotype switching\n",
      "    - negative regulation of single stranded viral RNA replication via double stranded DNA intermediate\n",
      "    - defense response to virus\n",
      "    - DNA cytosine deamination\n",
      "    - cellular response to lipopolysaccharide\n",
      "  CC:\n",
      "    - P-body\n",
      "    - nucleus\n",
      "    - nucleus\n",
      "    - cytoplasm\n",
      "    - cytoplasm\n",
      "    - cytosol\n",
      "    - protein-containing complex\n",
      "  MF:\n",
      "    - RNA binding\n",
      "    - cytidine deaminase activity\n",
      "    - cytidine deaminase activity\n",
      "    - cytidine deaminase activity\n",
      "    - protein binding\n",
      "    - zinc ion binding\n",
      "    - ubiquitin protein ligase binding\n",
      "    - identical protein binding\n",
      "UniProt: {'Swiss-Prot': 'Q9GZX7', 'TrEMBL': ['Q7Z599', 'Q546Y9', 'Q6QLN7', 'Q6QJ80', 'A0A8Q3SII6', 'A0A8Q3WL45', 'A0A8Q3SIK8']}\n",
      "\n",
      "Gene: SIK1\n",
      "Symbol: SIK1\n",
      "Name: salt inducible kinase 1\n",
      "Summary: This gene encodes a serine/threonine protein kinase that contains a ubiquitin-associated (UBA) domain. The encoded protein is a member of the adenosine monophosphate-activated kinase (AMPK) subfamily of kinases that play a role in conserved signal transduction pathways. A mutation in this gene is associated with early infantile epileptic encephalopathy 30. [provided by RefSeq, Nov 2016].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - regulation of sodium ion transport\n",
      "    - chromatin remodeling\n",
      "    - protein phosphorylation\n",
      "    - protein phosphorylation\n",
      "    - regulation of mitotic cell cycle\n",
      "    - regulation of myotube differentiation\n",
      "    - negative regulation of triglyceride biosynthetic process\n",
      "    - negative regulation of CREB transcription factor activity\n",
      "    - intracellular signal transduction\n",
      "    - intracellular signal transduction\n",
      "    - intracellular signal transduction\n",
      "    - entrainment of circadian clock by photoperiod\n",
      "    - anoikis\n",
      "    - regulation of cell differentiation\n",
      "    - negative regulation of gluconeogenesis\n",
      "    - protein autophosphorylation\n",
      "    - rhythmic process\n",
      "    - cardiac muscle cell differentiation\n",
      "    - positive regulation of anoikis\n",
      "  CC:\n",
      "    - nucleus\n",
      "    - nucleus\n",
      "    - cytoplasm\n",
      "    - cytoplasm\n",
      "    - cytoplasm\n",
      "  MF:\n",
      "    - magnesium ion binding\n",
      "    - magnesium ion binding\n",
      "    - protein serine/threonine kinase activity\n",
      "    - protein serine/threonine kinase activity\n",
      "    - 3-phosphoinositide-dependent protein kinase activity\n",
      "    - DNA-dependent protein kinase activity\n",
      "    - AMP-activated protein kinase activity\n",
      "    - eukaryotic translation initiation factor 2alpha kinase activity\n",
      "    - ribosomal protein S6 kinase activity\n",
      "    - protein binding\n",
      "    - ATP binding\n",
      "    - ATP binding\n",
      "    - cAMP response element binding protein binding\n",
      "    - protein kinase binding\n",
      "    - histone H3S10 kinase activity\n",
      "    - histone H3T11 kinase activity\n",
      "    - histone H3T6 kinase activity\n",
      "    - histone H2AXS139 kinase activity\n",
      "    - histone deacetylase binding\n",
      "    - histone H3S28 kinase activity\n",
      "    - histone H4S1 kinase activity\n",
      "    - histone H2AS1 kinase activity\n",
      "    - histone H2BS14 kinase activity\n",
      "    - 14-3-3 protein binding\n",
      "    - histone H3T3 kinase activity\n",
      "    - histone H2AS121 kinase activity\n",
      "    - Rho-dependent protein serine/threonine kinase activity\n",
      "    - protein serine kinase activity\n",
      "    - histone H2BS36 kinase activity\n",
      "    - histone H3S57 kinase activity\n",
      "    - histone H3T45 kinase activity\n",
      "    - histone H2AT120 kinase activity\n",
      "UniProt: {'Swiss-Prot': 'P57059', 'TrEMBL': 'A0A2R8Y6E4'}\n",
      "\n",
      "Gene: MGAM\n",
      "Symbol: MGAM\n",
      "Name: maltase-glucoamylase\n",
      "Summary: This gene encodes maltase-glucoamylase, which is a brush border membrane enzyme that plays a role in the final steps of digestion of starch. The protein has two catalytic sites identical to those of sucrase-isomaltase, but the proteins are only 59% homologous. Both are members of glycosyl hydrolase family 31, which has a variety of substrate specificities. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - maltose catabolic process\n",
      "    - starch catabolic process\n",
      "    - dextrin catabolic process\n",
      "  CC:\n",
      "    - plasma membrane\n",
      "    - apical plasma membrane\n",
      "    - extracellular exosome\n",
      "    - tertiary granule membrane\n",
      "    - ficolin-1-rich granule membrane\n",
      "  MF:\n",
      "    - catalytic activity\n",
      "    - glucan 1,4-alpha-glucosidase activity\n",
      "    - alpha-1,4-glucosidase activity\n",
      "    - alpha-1,4-glucosidase activity\n",
      "    - oligo-1,6-glucosidase activity\n",
      "    - protein binding\n",
      "    - amylase activity\n",
      "    - carbohydrate binding\n",
      "UniProt: {'Swiss-Prot': 'O43451', 'TrEMBL': ['E7EW87', 'E9PEQ5']}\n",
      "\n",
      "Gene: LCE1C\n",
      "Symbol: LCE1C\n",
      "Name: late cornified envelope 1C\n",
      "Summary: Predicted to be involved in keratinization. [provided by Alliance of Genome Resources, Apr 2022]\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - keratinization\n",
      "  MF:\n",
      "    - protein binding\n",
      "UniProt: {'Swiss-Prot': 'Q5T751', 'TrEMBL': 'U3KQM4'}\n",
      "\n",
      "Gene: MAST1\n",
      "Symbol: MAST1\n",
      "Name: microtubule associated serine/threonine kinase 1\n",
      "Summary: This gene is a member of the microtubule-associated serine/threonine kinase (MAST) family. The protein encoded by this gene has an N-terminal serine/threonine kinase domain followed by a postsynaptic density protein-95/discs large/zona occludens-1 (PDZ) domain. In mouse and rat, the orthologous protein associates with the cytoskeleton and can bind both beta-2-syntrophin and neuronal nitric oxide synthase (nNOS) through its PDZ domain. In mouse and rat, this protein also co-localizes with dystrophin- and utrophin-associated protein complexes (DAPC/UAPC) in the vascular endothelium of the central nervous system. [provided by RefSeq, May 2017].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - chromatin remodeling\n",
      "    - protein phosphorylation\n",
      "    - cytoskeleton organization\n",
      "    - cytoskeleton organization\n",
      "    - brain development\n",
      "    - brain development\n",
      "    - intracellular signal transduction\n",
      "    - intracellular signal transduction\n",
      "  CC:\n",
      "    - cytoplasm\n",
      "    - cytoskeleton\n",
      "    - plasma membrane\n",
      "    - axon\n",
      "    - dendrite\n",
      "    - neuron projection\n",
      "    - neuronal cell body\n",
      "  MF:\n",
      "    - magnesium ion binding\n",
      "    - protein serine/threonine kinase activity\n",
      "    - protein serine/threonine kinase activity\n",
      "    - 3-phosphoinositide-dependent protein kinase activity\n",
      "    - DNA-dependent protein kinase activity\n",
      "    - AMP-activated protein kinase activity\n",
      "    - eukaryotic translation initiation factor 2alpha kinase activity\n",
      "    - ribosomal protein S6 kinase activity\n",
      "    - protein binding\n",
      "    - ATP binding\n",
      "    - microtubule binding\n",
      "    - histone H3S10 kinase activity\n",
      "    - histone H3T11 kinase activity\n",
      "    - histone H3T6 kinase activity\n",
      "    - histone H2AXS139 kinase activity\n",
      "    - histone H3S28 kinase activity\n",
      "    - histone H4S1 kinase activity\n",
      "    - histone H2AS1 kinase activity\n",
      "    - histone H2BS14 kinase activity\n",
      "    - histone H3T3 kinase activity\n",
      "    - histone H2AS121 kinase activity\n",
      "    - Rho-dependent protein serine/threonine kinase activity\n",
      "    - protein serine kinase activity\n",
      "    - histone H2BS36 kinase activity\n",
      "    - histone H3S57 kinase activity\n",
      "    - histone H3T45 kinase activity\n",
      "    - histone H2AT120 kinase activity\n",
      "UniProt: {'Swiss-Prot': 'Q9Y2H9', 'TrEMBL': ['K7EME4', 'A0A8V8TQG4', 'K7EJI7']}\n",
      "\n",
      "Gene: INCENP\n",
      "Symbol: INCENP\n",
      "Name: inner centromere protein\n",
      "Summary: In mammalian cells, 2 broad groups of centromere-interacting proteins have been described: constitutively binding centromere proteins and 'passenger,' or transiently interacting, proteins (reviewed by Choo, 1997). The constitutive proteins include CENPA (centromere protein A; MIM 117139), CENPB (MIM 117140), CENPC1 (MIM 117141), and CENPD (MIM 117142). The term 'passenger proteins' encompasses a broad collection of proteins that localize to the centromere during specific stages of the cell cycle (Earnshaw and Mackay, 1994 [PubMed 8088460]). These include CENPE (MIM 117143); MCAK (MIM 604538); KID (MIM 603213); cytoplasmic dynein (e.g., MIM 600112); CliPs (e.g., MIM 179838); and CENPF/mitosin (MIM 600236). The inner centromere proteins (INCENPs) (Earnshaw and Cooke, 1991 [PubMed 1860899]), the initial members of the passenger protein group, display a broad localization along chromosomes in the early stages of mitosis but gradually become concentrated at centromeres as the cell cycle progresses into mid-metaphase. During telophase, the proteins are located within the midbody in the intercellular bridge, where they are discarded after cytokinesis (Cutts et al., 1999 [PubMed 10369859]).[supplied by OMIM, Mar 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - mitotic cell cycle\n",
      "    - mitotic cytokinesis\n",
      "    - mitotic cytokinesis\n",
      "    - mitotic cytokinesis\n",
      "    - chromosome segregation\n",
      "    - mitotic spindle midzone assembly\n",
      "    - meiotic spindle midzone assembly\n",
      "    - metaphase chromosome alignment\n",
      "    - positive regulation of mitotic cell cycle spindle assembly checkpoint\n",
      "    - mitotic spindle assembly\n",
      "    - positive regulation of mitotic sister chromatid separation\n",
      "    - positive regulation of attachment of mitotic spindle microtubules to kinetochore\n",
      "    - positive regulation of mitotic cytokinesis\n",
      "  CC:\n",
      "    - chromosome, centromeric region\n",
      "    - kinetochore\n",
      "    - kinetochore\n",
      "    - lateral element\n",
      "    - central element\n",
      "    - nucleus\n",
      "    - nucleoplasm\n",
      "    - nucleoplasm\n",
      "    - pericentric heterochromatin\n",
      "    - spindle\n",
      "    - cytosol\n",
      "    - cytosol\n",
      "    - microtubule\n",
      "    - chromocenter\n",
      "    - microtubule cytoskeleton\n",
      "    - nuclear body\n",
      "    - midbody\n",
      "    - midbody\n",
      "    - chromosome passenger complex\n",
      "    - chromosome passenger complex\n",
      "    - protein-containing complex\n",
      "    - meiotic spindle midzone\n",
      "  MF:\n",
      "    - protein binding\n",
      "    - molecular function activator activity\n",
      "UniProt: {'Swiss-Prot': 'Q9NQS7'}\n",
      "\n",
      "Gene: IMPA1\n",
      "Symbol: IMPA1\n",
      "Name: inositol monophosphatase 1\n",
      "Summary: This gene encodes an enzyme that dephosphorylates myo-inositol monophosphate to generate free myo-inositol, a precursor of phosphatidylinositol, and is therefore an important modulator of intracellular signal transduction via the production of the second messengers myoinositol 1,4,5-trisphosphate and diacylglycerol. This enzyme can also use myo-inositol-1,3-diphosphate, myo-inositol-1,4-diphosphate, scyllo-inositol-phosphate, glucose-1-phosphate, glucose-6-phosphate, fructose-1-phosphate, beta-glycerophosphate, and 2'-AMP as substrates. This enzyme shows magnesium-dependent phosphatase activity and is inhibited by therapeutic concentrations of lithium. Inhibition of inositol monophosphate hydroylosis and subsequent depletion of inositol for phosphatidylinositol synthesis may explain the anti-manic and anti-depressive effects of lithium administered to treat bipolar disorder. Alternative splicing results in multiple transcript variants encoding distinct isoforms. A pseudogene of this gene is also present on chromosome 8q21.13. [provided by RefSeq, Dec 2014].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - inositol metabolic process\n",
      "    - inositol biosynthetic process\n",
      "    - phosphatidylinositol biosynthetic process\n",
      "    - phosphate-containing compound metabolic process\n",
      "    - signal transduction\n",
      "    - signal transduction\n",
      "    - phosphatidylinositol phosphate biosynthetic process\n",
      "  CC:\n",
      "    - cytoplasm\n",
      "    - cytosol\n",
      "  MF:\n",
      "    - magnesium ion binding\n",
      "    - glucose-6-phosphatase activity\n",
      "    - protein binding\n",
      "    - glucose-1-phosphatase activity\n",
      "    - inositol monophosphate 1-phosphatase activity\n",
      "    - inositol monophosphate 1-phosphatase activity\n",
      "    - inositol monophosphate 1-phosphatase activity\n",
      "    - manganese ion binding\n",
      "    - lithium ion binding\n",
      "    - identical protein binding\n",
      "    - protein homodimerization activity\n",
      "    - glycerol-2-phosphatase activity\n",
      "    - inositol monophosphate 3-phosphatase activity\n",
      "    - inositol monophosphate 4-phosphatase activity\n",
      "    - inositol monophosphate phosphatase activity\n",
      "    - fructose-1-phosphatase activity\n",
      "UniProt: {'Swiss-Prot': 'P29218', 'TrEMBL': ['E5RG13', 'A0A140VJL8', 'E5RHE9', 'E5RG94', 'E5RGY4', 'E5RIP7', 'H0YBL1', 'E5RI82', 'E5RIF4']}\n",
      "\n",
      "Gene: MYH7\n",
      "Symbol: MYH7\n",
      "Name: myosin heavy chain 7\n",
      "Summary: Muscle myosin is a hexameric protein containing 2 heavy chain subunits, 2 alkali light chain subunits, and 2 regulatory light chain subunits. This gene encodes the beta (or slow) heavy chain subunit of cardiac myosin. It is expressed predominantly in normal human ventricle. It is also expressed in skeletal muscle tissues rich in slow-twitch type I muscle fibers. Changes in the relative abundance of this protein and the alpha (or fast) heavy subunit of cardiac myosin correlate with the contractile velocity of cardiac muscle. Its expression is also altered during thyroid hormone depletion and hemodynamic overloading. Mutations in this gene are associated with familial hypertrophic cardiomyopathy, myosin storage myopathy, dilated cardiomyopathy, and Laing distal myopathy. [provided by RefSeq, May 2022].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - regulation of the force of heart contraction\n",
      "    - regulation of heart rate\n",
      "    - skeletal muscle contraction\n",
      "    - muscle contraction\n",
      "    - muscle contraction\n",
      "    - striated muscle contraction\n",
      "    - adult heart development\n",
      "    - adult heart development\n",
      "    - regulation of the force of skeletal muscle contraction\n",
      "    - transition between fast and slow fiber\n",
      "    - cardiac muscle hypertrophy in response to stress\n",
      "    - muscle filament sliding\n",
      "    - muscle filament sliding\n",
      "    - regulation of slow-twitch skeletal muscle fiber contraction\n",
      "    - ATP metabolic process\n",
      "    - ventricular cardiac muscle tissue morphogenesis\n",
      "    - cardiac muscle contraction\n",
      "    - cardiac muscle contraction\n",
      "  CC:\n",
      "    - stress fiber\n",
      "    - cytoplasm\n",
      "    - muscle myosin complex\n",
      "    - myosin complex\n",
      "    - myosin II complex\n",
      "    - myofibril\n",
      "    - sarcomere\n",
      "    - sarcomere\n",
      "    - Z disc\n",
      "    - myosin filament\n",
      "    - myosin filament\n",
      "  MF:\n",
      "    - microfilament motor activity\n",
      "    - microfilament motor activity\n",
      "    - microfilament motor activity\n",
      "    - microfilament motor activity\n",
      "    - protein binding\n",
      "    - calmodulin binding\n",
      "    - ATP binding\n",
      "    - actin filament binding\n",
      "UniProt: {'Swiss-Prot': 'P12883', 'TrEMBL': 'A0AAQ5BGU7'}\n",
      "\n",
      "Gene: DSTYK\n",
      "Symbol: DSTYK\n",
      "Name: dual serine/threonine and tyrosine protein kinase\n",
      "Summary: This gene encodes a dual serine/threonine and tyrosine protein kinase which is expressed in multiple tissues. It is thought to function as a regulator of cell death. Multiple transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Dec 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - positive regulation of kinase activity\n",
      "    - negative regulation of apoptotic process\n",
      "    - negative regulation of apoptotic process\n",
      "    - cellular response to fibroblast growth factor stimulus\n",
      "    - cellular response to fibroblast growth factor stimulus\n",
      "    - positive regulation of fibroblast growth factor receptor signaling pathway\n",
      "    - positive regulation of fibroblast growth factor receptor signaling pathway\n",
      "    - positive regulation of ERK1 and ERK2 cascade\n",
      "    - positive regulation of ERK1 and ERK2 cascade\n",
      "  CC:\n",
      "    - cytoplasm\n",
      "    - cytoplasm\n",
      "    - basolateral plasma membrane\n",
      "    - apical plasma membrane\n",
      "    - anchoring junction\n",
      "  MF:\n",
      "    - protein serine/threonine kinase activity\n",
      "    - protein serine/threonine/tyrosine kinase activity\n",
      "    - protein tyrosine kinase activity\n",
      "    - protein binding\n",
      "    - ATP binding\n",
      "    - protein serine kinase activity\n",
      "UniProt: {'Swiss-Prot': 'Q6XUX3'}\n",
      "\n",
      "Gene: DMPK\n",
      "Symbol: DMPK\n",
      "Name: DM1 protein kinase\n",
      "Summary: The protein encoded by this gene is a serine-threonine kinase that is closely related to other kinases that interact with members of the Rho family of small GTPases. Substrates for this enzyme include myogenin, the beta-subunit of the L-type calcium channels, and phospholemman. The 3' untranslated region of this gene contains 5-38 copies of a CTG trinucleotide repeat. Expansion of this unstable motif to 50-5,000 copies causes myotonic dystrophy type I, which increases in severity with increasing repeat element copy number. Repeat expansion is associated with condensation of local chromatin structure that disrupts the expression of genes in this region. Several alternatively spliced transcript variants of this gene have been described, but the full-length nature of some of these variants has not been determined. [provided by RefSeq, Jul 2016].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - regulation of sodium ion transport\n",
      "    - chromatin remodeling\n",
      "    - protein phosphorylation\n",
      "    - intracellular calcium ion homeostasis\n",
      "    - intracellular calcium ion homeostasis\n",
      "    - nuclear envelope organization\n",
      "    - nuclear envelope organization\n",
      "    - regulation of heart contraction\n",
      "    - regulation of heart contraction\n",
      "    - muscle cell apoptotic process\n",
      "    - regulation of myotube differentiation\n",
      "    - regulation of skeletal muscle contraction by calcium ion signaling\n",
      "    - regulation of excitatory postsynaptic membrane potential involved in skeletal muscle contraction\n",
      "    - regulation of synapse structural plasticity\n",
      "  CC:\n",
      "    - nuclear outer membrane\n",
      "    - mitochondrial outer membrane\n",
      "    - endoplasmic reticulum membrane\n",
      "    - cytosol\n",
      "    - plasma membrane\n",
      "    - nuclear membrane\n",
      "    - sarcoplasmic reticulum membrane\n",
      "    - sarcoplasmic reticulum membrane\n",
      "  MF:\n",
      "    - protein serine/threonine kinase activity\n",
      "    - protein serine/threonine kinase activity\n",
      "    - 3-phosphoinositide-dependent protein kinase activity\n",
      "    - DNA-dependent protein kinase activity\n",
      "    - AMP-activated protein kinase activity\n",
      "    - eukaryotic translation initiation factor 2alpha kinase activity\n",
      "    - ribosomal protein S6 kinase activity\n",
      "    - protein binding\n",
      "    - ATP binding\n",
      "    - myosin phosphatase regulator activity\n",
      "    - histone H3S10 kinase activity\n",
      "    - histone H3T11 kinase activity\n",
      "    - histone H3T6 kinase activity\n",
      "    - histone H2AXS139 kinase activity\n",
      "    - histone H3S28 kinase activity\n",
      "    - histone H4S1 kinase activity\n",
      "    - histone H2AS1 kinase activity\n",
      "    - histone H2BS14 kinase activity\n",
      "    - metal ion binding\n",
      "    - histone H3T3 kinase activity\n",
      "    - histone H2AS121 kinase activity\n",
      "    - Rho-dependent protein serine/threonine kinase activity\n",
      "    - protein serine kinase activity\n",
      "    - histone H2BS36 kinase activity\n",
      "    - histone H3S57 kinase activity\n",
      "    - histone H3T45 kinase activity\n",
      "    - histone H2AT120 kinase activity\n",
      "UniProt: {'Swiss-Prot': 'Q09013', 'TrEMBL': ['E5KR07', 'E5KR06', 'E5KR05', 'B4DZE2', 'I6L989', 'B4DM55', 'M0QXJ9', 'M0R1F3', 'K7EQX1']}\n",
      "\n",
      "Gene: RNH1\n",
      "Symbol: RNH1\n",
      "Name: ribonuclease/angiogenin inhibitor 1\n",
      "Summary: Placental ribonuclease inhibitor (PRI) is a member of a family of proteinaceous cytoplasmic RNase inhibitors that occur in many tissues and bind to both intracellular and extracellular RNases (summarized by Lee et al., 1988 [PubMed 3219362]). In addition to control of intracellular RNases, the inhibitor may have a role in the regulation of angiogenin (MIM 105850). Ribonuclease inhibitor, of 50,000 Da, binds to ribonucleases and holds them in a latent form. Since neutral and alkaline ribonucleases probably play a critical role in the turnover of RNA in eukaryotic cells, RNH may be essential for control of mRNA turnover; the interaction of eukaryotic cells with ribonuclease may be reversible in vivo.[supplied by OMIM, Jul 2010].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - mRNA catabolic process\n",
      "    - cell migration\n",
      "    - regulation of Arp2/3 complex-mediated actin nucleation\n",
      "    - tRNA stabilization\n",
      "    - regulation of angiogenesis\n",
      "    - regulation of angiogenesis\n",
      "  CC:\n",
      "    - nucleus\n",
      "    - nucleoplasm\n",
      "    - nucleoplasm\n",
      "    - cytoplasm\n",
      "    - cytosol\n",
      "    - plasma membrane\n",
      "    - lamellipodium\n",
      "    - angiogenin-PRI complex\n",
      "    - angiogenin-PRI complex\n",
      "    - extracellular exosome\n",
      "  MF:\n",
      "    - protein binding\n",
      "    - ribonuclease inhibitor activity\n",
      "    - ribonuclease inhibitor activity\n",
      "UniProt: {'Swiss-Prot': 'P13489', 'TrEMBL': ['E9PMI1', 'A0A140VJT8', 'E9PR82', 'E9PMA9', 'E9PLZ3', 'E9PMJ3', 'E9PIM9', 'A0A0J9YXC4', 'H0YCR7', 'E9PIK5']}\n",
      "\n",
      "Gene: ADORA1\n",
      "Symbol: ADORA1\n",
      "Name: adenosine A1 receptor\n",
      "Summary: The protein encoded by this gene is an adenosine receptor that belongs to the G-protein coupled receptor 1 family. There are 3 types of adenosine receptors, each with a specific pattern of ligand binding and tissue distribution, and together they regulate a diverse set of physiologic functions. The type A1 receptors inhibit adenylyl cyclase, and play a role in the fertilization process. Animal studies also suggest a role for A1 receptors in kidney function and ethanol intoxication. Transcript variants with alternative splicing in the 5' UTR have been found for this gene. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - temperature homeostasis\n",
      "    - response to hypoxia\n",
      "    - G protein-coupled adenosine receptor signaling pathway\n",
      "    - regulation of respiratory gaseous exchange by nervous system process\n",
      "    - negative regulation of acute inflammatory response\n",
      "    - negative regulation of leukocyte migration\n",
      "    - positive regulation of peptide secretion\n",
      "    - positive regulation of systemic arterial blood pressure\n",
      "    - negative regulation of systemic arterial blood pressure\n",
      "    - regulation of glomerular filtration\n",
      "    - protein targeting to membrane\n",
      "    - phagocytosis\n",
      "    - inflammatory response\n",
      "    - signal transduction\n",
      "    - G protein-coupled receptor signaling pathway\n",
      "    - adenylate cyclase-inhibiting G protein-coupled receptor signaling pathway\n",
      "    - cell-cell signaling\n",
      "    - nervous system development\n",
      "    - negative regulation of cell population proliferation\n",
      "    - negative regulation of glutamate secretion\n",
      "    - response to purine-containing compound\n",
      "    - lipid catabolic process\n",
      "    - negative regulation of synaptic transmission, GABAergic\n",
      "    - positive regulation of nucleoside transport\n",
      "    - negative regulation of neurotrophin production\n",
      "    - positive regulation of dephosphorylation\n",
      "    - vasodilation\n",
      "    - negative regulation of circadian sleep/wake cycle, non-REM sleep\n",
      "    - negative regulation of apoptotic process\n",
      "    - positive regulation of potassium ion transport\n",
      "    - positive regulation of MAPK cascade\n",
      "    - negative regulation of hormone secretion\n",
      "    - cognition\n",
      "    - leukocyte migration\n",
      "    - detection of temperature stimulus involved in sensory perception of pain\n",
      "    - negative regulation of lipid catabolic process\n",
      "    - positive regulation of lipid catabolic process\n",
      "    - regulation of sensory perception of pain\n",
      "    - negative regulation of synaptic transmission, glutamatergic\n",
      "    - fatty acid homeostasis\n",
      "    - excitatory postsynaptic potential\n",
      "    - long-term synaptic depression\n",
      "    - mucus secretion\n",
      "    - negative regulation of mucus secretion\n",
      "    - triglyceride homeostasis\n",
      "    - regulation of cardiac muscle cell contraction\n",
      "    - apoptotic signaling pathway\n",
      "    - regulation of presynaptic cytosolic calcium ion concentration\n",
      "    - negative regulation of long-term synaptic potentiation\n",
      "    - negative regulation of long-term synaptic depression\n",
      "  CC:\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - cilium\n",
      "    - basolateral plasma membrane\n",
      "    - dendrite\n",
      "    - axolemma\n",
      "    - asymmetric synapse\n",
      "    - presynaptic membrane\n",
      "    - neuronal cell body\n",
      "    - terminal bouton\n",
      "    - dendritic spine\n",
      "    - calyx of Held\n",
      "    - synapse\n",
      "    - postsynaptic membrane\n",
      "    - presynaptic active zone\n",
      "  MF:\n",
      "    - G protein-coupled adenosine receptor activity\n",
      "    - G protein-coupled receptor binding\n",
      "    - purine nucleoside binding\n",
      "    - protein binding\n",
      "    - heat shock protein binding\n",
      "    - G-protein beta/gamma-subunit complex binding\n",
      "    - heterotrimeric G-protein binding\n",
      "    - protein heterodimerization activity\n",
      "UniProt: {'Swiss-Prot': 'P30542'}\n",
      "\n",
      "Gene: MED31\n",
      "Symbol: MED31\n",
      "Name: mediator complex subunit 31\n",
      "Summary: Predicted to enable transcription coregulator activity and ubiquitin protein ligase activity. Predicted to be involved in regulation of transcription by RNA polymerase II. Predicted to act upstream of or within limb development and negative regulation of fibroblast proliferation. Predicted to be located in nucleoplasm. Predicted to be part of core mediator complex and mediator complex. [provided by Alliance of Genome Resources, Apr 2022]\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - regulation of transcription by RNA polymerase II\n",
      "    - protein ubiquitination\n",
      "    - positive regulation of transcription elongation by RNA polymerase II\n",
      "    - negative regulation of fibroblast proliferation\n",
      "    - RNA polymerase II preinitiation complex assembly\n",
      "    - limb development\n",
      "    - positive regulation of transcription initiation by RNA polymerase II\n",
      "  CC:\n",
      "    - ubiquitin ligase complex\n",
      "    - nucleus\n",
      "    - nucleoplasm\n",
      "    - mediator complex\n",
      "    - core mediator complex\n",
      "    - core mediator complex\n",
      "  MF:\n",
      "    - transcription coregulator activity\n",
      "    - protein binding\n",
      "    - ubiquitin protein ligase activity\n",
      "UniProt: {'Swiss-Prot': 'Q9Y3C7', 'TrEMBL': ['I3L2J1', 'I3L0P8']}\n",
      "\n",
      "Gene: LAMC1\n",
      "Symbol: LAMC1\n",
      "Name: laminin subunit gamma 1\n",
      "Summary: Laminins, a family of extracellular matrix glycoproteins, are the major noncollagenous constituent of basement membranes. They have been implicated in a wide variety of biological processes including cell adhesion, differentiation, migration, signaling, neurite outgrowth and metastasis. Laminins, composed of 3 non identical chains: laminin alpha, beta and gamma (formerly A, B1, and B2, respectively), have a cruciform structure consisting of 3 short arms, each formed by a different chain, and a long arm composed of all 3 chains. Each laminin chain is a multidomain protein encoded by a distinct gene. Several isoforms of each chain have been described. Different alpha, beta and gamma chain isomers combine to give rise to different heterotrimeric laminin isoforms which are designated by Arabic numerals in the order of their discovery, i.e. alpha1beta1gamma1 heterotrimer is laminin 1. The biological functions of the different chains and trimer molecules are largely unknown, but some of the chains have been shown to differ with respect to their tissue distribution, presumably reflecting diverse functions in vivo. This gene encodes the gamma chain isoform laminin, gamma 1. The gamma 1 chain, formerly thought to be a beta chain, contains structural domains similar to beta chains, however, lacks the short alpha region separating domains I and II. The structural organization of this gene also suggested that it had diverged considerably from the beta chain genes. Embryos of transgenic mice in which both alleles of the gamma 1 chain gene were inactivated by homologous recombination, lacked basement membranes, indicating that laminin, gamma 1 chain is necessary for laminin heterotrimer assembly. It has been inferred by analogy with the strikingly similar 3' UTR sequence in mouse laminin gamma 1 cDNA, that multiple polyadenylation sites are utilized in human to generate the 2 different sized mRNAs (5.5 and 7.5 kb) seen on Northern analysis. [provided by RefSeq, Aug 2011].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - cell adhesion\n",
      "    - endoderm development\n",
      "    - cell migration\n",
      "    - extracellular matrix disassembly\n",
      "    - hemidesmosome assembly\n",
      "    - substrate adhesion-dependent cell spreading\n",
      "    - maintenance of blood-brain barrier\n",
      "    - positive regulation of cell adhesion\n",
      "    - positive regulation of epithelial cell proliferation\n",
      "    - positive regulation of muscle cell differentiation\n",
      "    - protein-containing complex assembly\n",
      "    - regulation of basement membrane organization\n",
      "    - positive regulation of integrin-mediated signaling pathway\n",
      "  CC:\n",
      "    - extracellular region\n",
      "    - basement membrane\n",
      "    - basement membrane\n",
      "    - laminin-1 complex\n",
      "    - laminin-1 complex\n",
      "    - extracellular space\n",
      "    - endoplasmic reticulum lumen\n",
      "    - laminin-10 complex\n",
      "    - laminin-11 complex\n",
      "    - collagen-containing extracellular matrix\n",
      "    - collagen-containing extracellular matrix\n",
      "    - collagen-containing extracellular matrix\n",
      "    - extracellular exosome\n",
      "    - protein complex involved in cell-matrix adhesion\n",
      "  MF:\n",
      "    - extracellular matrix structural constituent\n",
      "    - extracellular matrix structural constituent\n",
      "    - extracellular matrix structural constituent\n",
      "    - extracellular matrix constituent conferring elasticity\n",
      "UniProt: {'Swiss-Prot': 'P11047', 'TrEMBL': ['R4GNC7', 'Q6NVY8']}\n",
      "\n",
      "Gene: MRAS\n",
      "Symbol: MRAS\n",
      "Name: muscle RAS oncogene homolog\n",
      "Summary: This gene encodes a member of the Ras family of small GTPases. These membrane-associated proteins function as signal transducers in multiple processes including cell growth and differentiation, and dysregulation of Ras signaling has been associated with many types of cancer. The encoded protein may play a role in the tumor necrosis factor-alpha and MAP kinase signaling pathways. Alternatively spliced transcript variants encoding multiple isoforms have been observed for this gene. [provided by RefSeq, Nov 2011].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - Ras protein signal transduction\n",
      "    - actin cytoskeleton organization\n",
      "    - cellular response to leukemia inhibitory factor\n",
      "  CC:\n",
      "    - protein phosphatase type 1 complex\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "  MF:\n",
      "    - GTPase activity\n",
      "    - GTPase activity\n",
      "    - G protein activity\n",
      "    - protein binding\n",
      "    - GTP binding\n",
      "    - GDP binding\n",
      "    - GTP-dependent protein binding\n",
      "UniProt: {'Swiss-Prot': 'O14807', 'TrEMBL': ['Q8WVM9', 'Q6FGP0', 'C9J601', 'C9J8Q6']}\n",
      "\n",
      "Gene: NOS2\n",
      "Symbol: NOS2\n",
      "Name: nitric oxide synthase 2\n",
      "Summary: Nitric oxide is a reactive free radical which acts as a biologic mediator in several processes, including neurotransmission and antimicrobial and antitumoral activities. This gene encodes a nitric oxide synthase which is expressed in liver and is inducible by a combination of lipopolysaccharide and certain cytokines. Three related pseudogenes are located within the Smith-Magenis syndrome region on chromosome 17. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - response to hypoxia\n",
      "    - positive regulation of leukocyte mediated cytotoxicity\n",
      "    - innate immune response in mucosa\n",
      "    - arginine catabolic process\n",
      "    - arginine catabolic process\n",
      "    - superoxide metabolic process\n",
      "    - nitric oxide biosynthetic process\n",
      "    - nitric oxide biosynthetic process\n",
      "    - inflammatory response\n",
      "    - nitric oxide mediated signal transduction\n",
      "    - circadian rhythm\n",
      "    - response to bacterium\n",
      "    - response to hormone\n",
      "    - negative regulation of gene expression\n",
      "    - peptidyl-cysteine S-nitrosylation\n",
      "    - prostaglandin secretion\n",
      "    - response to lipopolysaccharide\n",
      "    - positive regulation of interleukin-6 production\n",
      "    - positive regulation of interleukin-8 production\n",
      "    - Fc-gamma receptor signaling pathway involved in phagocytosis\n",
      "    - regulation of cell population proliferation\n",
      "    - negative regulation of protein catabolic process\n",
      "    - defense response to bacterium\n",
      "    - defense response to bacterium\n",
      "    - defense response to bacterium\n",
      "    - regulation of cellular respiration\n",
      "    - cell redox homeostasis\n",
      "    - negative regulation of blood pressure\n",
      "    - regulation of insulin secretion\n",
      "    - defense response to Gram-negative bacterium\n",
      "    - cellular response to lipopolysaccharide\n",
      "    - cellular response to type II interferon\n",
      "    - cellular response to xenobiotic stimulus\n",
      "    - regulation of cytokine production involved in inflammatory response\n",
      "  CC:\n",
      "    - nucleus\n",
      "    - nucleus\n",
      "    - nucleoplasm\n",
      "    - cytoplasm\n",
      "    - peroxisome\n",
      "    - peroxisomal matrix\n",
      "    - cytosol\n",
      "    - cytosol\n",
      "    - cytosol\n",
      "    - plasma membrane\n",
      "    - cortical cytoskeleton\n",
      "    - perinuclear region of cytoplasm\n",
      "  MF:\n",
      "    - nitric-oxide synthase activity\n",
      "    - nitric-oxide synthase activity\n",
      "    - nitric-oxide synthase activity\n",
      "    - protein binding\n",
      "    - calmodulin binding\n",
      "    - FMN binding\n",
      "    - FMN binding\n",
      "    - heme binding\n",
      "    - tetrahydrobiopterin binding\n",
      "    - arginine binding\n",
      "    - protein homodimerization activity\n",
      "    - metal ion binding\n",
      "    - flavin adenine dinucleotide binding\n",
      "    - flavin adenine dinucleotide binding\n",
      "    - NADP binding\n",
      "UniProt: {'Swiss-Prot': 'P35228', 'TrEMBL': ['A0A8V8TKZ7', 'A0A2R8YDS4', 'A0A8V8TMI7', 'A0A8V8TLB1', 'A0A8V8TKX9']}\n",
      "\n",
      "Gene: DPM3\n",
      "Symbol: DPM3\n",
      "Name: dolichyl-phosphate mannosyltransferase subunit 3, regulatory\n",
      "Summary: Dolichol-phosphate mannose (Dol-P-Man) serves as a donor of mannosyl residues on the lumenal side of the endoplasmic reticulum (ER). Lack of Dol-P-Man results in defective surface expression of GPI-anchored proteins. Dol-P-Man is synthesized from GDP-mannose and dolichol-phosphate on the cytosolic side of the ER by the enzyme dolichyl-phosphate mannosyltransferase. The protein encoded by this gene is a subunit of dolichyl-phosphate mannosyltransferase and acts as a stabilizer subunit of the dolichyl-phosphate mannosyltransferase complex. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - carbohydrate metabolic process\n",
      "    - GPI anchor biosynthetic process\n",
      "    - GPI anchor biosynthetic process\n",
      "    - GPI anchor biosynthetic process\n",
      "    - protein C-linked glycosylation via 2'-alpha-mannosyl-L-tryptophan\n",
      "    - dolichol metabolic process\n",
      "    - regulation of protein stability\n",
      "    - protein mannosylation\n",
      "    - protein O-linked mannosylation\n",
      "  CC:\n",
      "    - endoplasmic reticulum\n",
      "    - endoplasmic reticulum membrane\n",
      "    - endoplasmic reticulum membrane\n",
      "    - endoplasmic reticulum membrane\n",
      "    - endoplasmic reticulum membrane\n",
      "    - membrane\n",
      "    - mannosyltransferase complex\n",
      "    - dolichol-phosphate-mannose synthase complex\n",
      "    - dolichol-phosphate-mannose synthase complex\n",
      "    - dolichol-phosphate-mannose synthase complex\n",
      "  MF:\n",
      "    - protein binding\n",
      "    - enzyme activator activity\n",
      "UniProt: {'Swiss-Prot': 'Q9P2X0', 'TrEMBL': ['Q86TM7', 'A0A140VJI4']}\n",
      "\n",
      "Gene: UGT1A3\n",
      "Symbol: UGT1A3\n",
      "Name: UDP glucuronosyltransferase family 1 member A3\n",
      "Summary: This gene encodes a UDP-glucuronosyltransferase, an enzyme of the glucuronidation pathway that transforms small lipophilic molecules, such as steroids, bilirubin, hormones, and drugs, into water-soluble, excretable metabolites. This gene is part of a complex locus that encodes several UDP-glucuronosyltransferases. The locus includes thirteen unique alternate first exons followed by four common exons. Four of the alternate first exons are considered pseudogenes. Each of the remaining nine 5' exons may be spliced to the four common exons, resulting in nine proteins with different N-termini and identical C-termini. Each first exon encodes the substrate binding site, and is regulated by its own promoter. Substrates of this enzyme include estrone, 2-hydroxyestrone, and metabolites of benzo alpha-pyrene. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - xenobiotic metabolic process\n",
      "    - xenobiotic metabolic process\n",
      "    - xenobiotic metabolic process\n",
      "    - estrogen metabolic process\n",
      "    - estrogen metabolic process\n",
      "    - flavonoid metabolic process\n",
      "    - bile acid secretion\n",
      "    - retinoic acid metabolic process\n",
      "    - negative regulation of fatty acid metabolic process\n",
      "    - vitamin D3 metabolic process\n",
      "  CC:\n",
      "    - endoplasmic reticulum\n",
      "    - endoplasmic reticulum\n",
      "    - endoplasmic reticulum\n",
      "    - endoplasmic reticulum membrane\n",
      "    - endoplasmic reticulum membrane\n",
      "    - endoplasmic reticulum membrane\n",
      "  MF:\n",
      "    - retinoic acid binding\n",
      "    - enzyme inhibitor activity\n",
      "    - glucuronosyltransferase activity\n",
      "    - glucuronosyltransferase activity\n",
      "    - glucuronosyltransferase activity\n",
      "    - glucuronosyltransferase activity\n",
      "    - enzyme binding\n",
      "    - enzyme binding\n",
      "    - protein homodimerization activity\n",
      "    - protein heterodimerization activity\n",
      "    - protein heterodimerization activity\n",
      "UniProt: {'Swiss-Prot': 'P35503', 'TrEMBL': 'Q5DT01'}\n",
      "\n",
      "Gene: FGB\n",
      "Symbol: FGB\n",
      "Name: fibrinogen beta chain\n",
      "Summary: The protein encoded by this gene is the beta component of fibrinogen, a blood-borne glycoprotein comprised of three pairs of nonidentical polypeptide chains. Following vascular injury, fibrinogen is cleaved by thrombin to form fibrin which is the most abundant component of blood clots. In addition, various cleavage products of fibrinogen and fibrin regulate cell adhesion and spreading, display vasoconstrictor and chemotactic activities, and are mitogens for several cell types. Fibrinogen serves key roles in hemostasis and antimicrobial host defense. Mutations in this gene lead to several disorders, including afibrinogenemia, dysfibrinogenemia, hypodysfibrinogenemia and thrombotic tendency. [provided by RefSeq, Aug 2020].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - adaptive immune response\n",
      "    - cell-matrix adhesion\n",
      "    - cell-matrix adhesion\n",
      "    - plasminogen activation\n",
      "    - positive regulation of heterotypic cell-cell adhesion\n",
      "    - fibrinolysis\n",
      "    - induction of bacterial agglutination\n",
      "    - cellular response to leptin stimulus\n",
      "    - innate immune response\n",
      "    - positive regulation of vasoconstriction\n",
      "    - positive regulation of exocytosis\n",
      "    - positive regulation of protein secretion\n",
      "    - protein polymerization\n",
      "    - response to calcium ion\n",
      "    - protein-containing complex assembly\n",
      "    - positive regulation of ERK1 and ERK2 cascade\n",
      "    - platelet aggregation\n",
      "    - platelet aggregation\n",
      "    - cellular response to interleukin-1\n",
      "    - blood coagulation, fibrin clot formation\n",
      "    - blood coagulation, fibrin clot formation\n",
      "    - positive regulation of peptide hormone secretion\n",
      "    - positive regulation of substrate adhesion-dependent cell spreading\n",
      "    - negative regulation of extrinsic apoptotic signaling pathway via death domain receptors\n",
      "    - negative regulation of endothelial cell apoptotic process\n",
      "  CC:\n",
      "    - extracellular region\n",
      "    - extracellular region\n",
      "    - fibrinogen complex\n",
      "    - extracellular space\n",
      "    - extracellular space\n",
      "    - extracellular space\n",
      "    - endoplasmic reticulum\n",
      "    - plasma membrane\n",
      "    - cell cortex\n",
      "    - external side of plasma membrane\n",
      "    - cell surface\n",
      "    - platelet alpha granule\n",
      "    - platelet alpha granule lumen\n",
      "    - synapse\n",
      "    - collagen-containing extracellular matrix\n",
      "    - collagen-containing extracellular matrix\n",
      "    - extracellular exosome\n",
      "    - blood microparticle\n",
      "    - extracellular vesicle\n",
      "  MF:\n",
      "    - signaling receptor binding\n",
      "    - structural molecule activity\n",
      "    - extracellular matrix structural constituent\n",
      "    - extracellular matrix structural constituent\n",
      "    - protein binding\n",
      "    - cell adhesion molecule binding\n",
      "    - protein-folding chaperone binding\n",
      "UniProt: {'Swiss-Prot': 'P02675', 'TrEMBL': ['F8W6P4', 'V9HVY1', 'D6REL8']}\n",
      "\n",
      "Gene: ZNF726\n",
      "Symbol: ZNF726\n",
      "Name: zinc finger protein 726\n",
      "Summary: Predicted to enable DNA-binding transcription factor activity, RNA polymerase II-specific and RNA polymerase II cis-regulatory region sequence-specific DNA binding activity. Predicted to be involved in regulation of transcription by RNA polymerase II. Predicted to be active in nucleus. [provided by Alliance of Genome Resources, Apr 2022]\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - regulation of DNA-templated transcription\n",
      "  CC:\n",
      "    - nucleus\n",
      "  MF:\n",
      "    - DNA binding\n",
      "    - protein binding\n",
      "    - zinc ion binding\n",
      "UniProt: {'Swiss-Prot': 'A6NNF4', 'TrEMBL': ['E9PLI7', 'F1T0F3', 'A0A8I5GK09', 'J3KNQ9']}\n",
      "\n",
      "Gene: VKORC1\n",
      "Symbol: VKORC1\n",
      "Name: vitamin K epoxide reductase complex subunit 1\n",
      "Summary: This gene encodes the catalytic subunit of the vitamin K epoxide reductase complex, which is responsible for the reduction of inactive vitamin K 2,3-epoxide to active vitamin K in the endoplasmic reticulum membrane. Vitamin K is a required co-factor for carboxylation of glutamic acid residues by vitamin K-dependent gamma-carboxylase in blood-clotting enzymes. Allelic variation in this gene is associated with vitamin k-dependent clotting factors combined deficiency of 2, and increased resistance or sensitivity to warfarin, an inhibitor of vitamin K epoxide reductase. Pseudogenes of this gene are located on chromosomes 1 and X. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Aug 2015].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - xenobiotic metabolic process\n",
      "    - blood coagulation\n",
      "    - blood coagulation\n",
      "    - peptidyl-glutamic acid carboxylation\n",
      "    - regulation of blood coagulation\n",
      "    - vitamin K metabolic process\n",
      "    - vitamin K metabolic process\n",
      "    - positive regulation of coagulation\n",
      "    - bone development\n",
      "  CC:\n",
      "    - endoplasmic reticulum\n",
      "    - endoplasmic reticulum lumen\n",
      "    - endoplasmic reticulum membrane\n",
      "    - endoplasmic reticulum membrane\n",
      "  MF:\n",
      "    - protein binding\n",
      "    - vitamin-K-epoxide reductase (warfarin-sensitive) activity\n",
      "    - vitamin-K-epoxide reductase (warfarin-sensitive) activity\n",
      "    - vitamin-K-epoxide reductase (warfarin-insensitive) activity\n",
      "    - quinone binding\n",
      "UniProt: {'Swiss-Prot': 'Q9BQB6', 'TrEMBL': ['A0A0S2Z6I4', 'A0A0S2Z5X7', 'A8MV79', 'F2Z3Q2', 'H0YF24', 'F8W9H0', 'I3L1P9']}\n",
      "\n",
      "Gene: DAPK2\n",
      "Symbol: DAPK2\n",
      "Name: death associated protein kinase 2\n",
      "Summary: This gene encodes a protein that belongs to the serine/threonine protein kinase family. This protein contains a N-terminal protein kinase domain followed by a conserved calmodulin-binding domain with significant similarity to that of death-associated protein kinase 1 (DAPK1), a positive regulator of programmed cell death. Overexpression of this gene was shown to induce cell apoptosis.  It uses multiple polyadenylation sites. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - chromatin remodeling\n",
      "    - protein phosphorylation\n",
      "    - apoptotic process\n",
      "    - regulation of autophagy\n",
      "    - intracellular signal transduction\n",
      "    - intracellular signal transduction\n",
      "    - regulation of apoptotic process\n",
      "    - positive regulation of apoptotic process\n",
      "    - anoikis\n",
      "    - protein autophosphorylation\n",
      "    - positive regulation of neutrophil chemotaxis\n",
      "    - positive regulation of eosinophil chemotaxis\n",
      "    - regulation of intrinsic apoptotic signaling pathway\n",
      "  CC:\n",
      "    - nucleus\n",
      "    - cytoplasm\n",
      "    - cytoplasm\n",
      "    - Golgi apparatus\n",
      "    - cytoplasmic vesicle\n",
      "    - autophagosome lumen\n",
      "    - intracellular membrane-bounded organelle\n",
      "  MF:\n",
      "    - protein serine/threonine kinase activity\n",
      "    - protein serine/threonine kinase activity\n",
      "    - 3-phosphoinositide-dependent protein kinase activity\n",
      "    - DNA-dependent protein kinase activity\n",
      "    - AMP-activated protein kinase activity\n",
      "    - eukaryotic translation initiation factor 2alpha kinase activity\n",
      "    - ribosomal protein S6 kinase activity\n",
      "    - protein binding\n",
      "    - calmodulin binding\n",
      "    - ATP binding\n",
      "    - histone H3S10 kinase activity\n",
      "    - histone H3T11 kinase activity\n",
      "    - histone H3T6 kinase activity\n",
      "    - histone H2AXS139 kinase activity\n",
      "    - identical protein binding\n",
      "    - histone H3S28 kinase activity\n",
      "    - histone H4S1 kinase activity\n",
      "    - histone H2AS1 kinase activity\n",
      "    - histone H2BS14 kinase activity\n",
      "    - histone H3T3 kinase activity\n",
      "    - histone H2AS121 kinase activity\n",
      "    - Rho-dependent protein serine/threonine kinase activity\n",
      "    - protein serine kinase activity\n",
      "    - histone H2BS36 kinase activity\n",
      "    - histone H3S57 kinase activity\n",
      "    - histone H3T45 kinase activity\n",
      "    - histone H2AT120 kinase activity\n",
      "UniProt: {'Swiss-Prot': 'Q9UIK4', 'TrEMBL': ['H0YLP5', 'H0YL86', 'H0YKW8', 'H0YMK1']}\n",
      "\n",
      "Gene: DLX4\n",
      "Symbol: DLX4\n",
      "Name: distal-less homeobox 4\n",
      "Summary: Many vertebrate homeo box-containing genes have been identified on the basis of their sequence similarity with Drosophila developmental genes. Members of the Dlx gene family contain a homeobox that is related to that of Distal-less (Dll), a gene expressed in the head and limbs of the developing fruit fly. The Distal-less (Dlx) family of genes comprises at least 6 different members, DLX1-DLX6. The DLX proteins are postulated to play a role in forebrain and craniofacial development. Three transcript variants have been described for this gene, however, the full length nature of one variant has not been described. Studies of the two splice variants revealed that one encoded isoform functions as a repressor of the beta-globin gene while the other isoform lacks that function. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - negative regulation of transcription by RNA polymerase II\n",
      "    - regulation of DNA-templated transcription\n",
      "    - regulation of transcription by RNA polymerase II\n",
      "    - cell differentiation\n",
      "    - embryonic skeletal system development\n",
      "  CC:\n",
      "    - chromatin\n",
      "    - nucleus\n",
      "    - nucleoplasm\n",
      "  MF:\n",
      "    - RNA polymerase II cis-regulatory region sequence-specific DNA binding\n",
      "    - RNA polymerase II cis-regulatory region sequence-specific DNA binding\n",
      "    - DNA-binding transcription factor activity, RNA polymerase II-specific\n",
      "    - DNA-binding transcription factor activity, RNA polymerase II-specific\n",
      "    - DNA-binding transcription repressor activity, RNA polymerase II-specific\n",
      "    - DNA-binding transcription factor activity\n",
      "    - protein binding\n",
      "    - sequence-specific DNA binding\n",
      "    - sequence-specific double-stranded DNA binding\n",
      "UniProt: {'Swiss-Prot': 'Q92988'}\n",
      "\n",
      "Gene: KCTD5\n",
      "Symbol: KCTD5\n",
      "Name: potassium channel tetramerization domain containing 5\n",
      "Summary: Enables identical protein binding activity. Predicted to be involved in proteasome-mediated ubiquitin-dependent protein catabolic process. Located in cytosol. [provided by Alliance of Genome Resources, Apr 2022]\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - proteasome-mediated ubiquitin-dependent protein catabolic process\n",
      "    - protein homooligomerization\n",
      "  CC:\n",
      "    - nucleus\n",
      "    - cytoplasm\n",
      "    - cytosol\n",
      "    - Cul3-RING ubiquitin ligase complex\n",
      "  MF:\n",
      "    - protein binding\n",
      "    - identical protein binding\n",
      "    - protein-containing complex binding\n",
      "    - cullin family protein binding\n",
      "UniProt: {'Swiss-Prot': 'Q9NXV2', 'TrEMBL': 'H3BSS5'}\n",
      "\n",
      "Gene: AK3\n",
      "Symbol: AK3\n",
      "Name: adenylate kinase 3\n",
      "Summary: The protein encoded by this gene is a GTP:ATP phosphotransferase that is found in the mitochondrial matrix. Several transcript variants encoding a few different isoforms have been found for this gene. [provided by RefSeq, Dec 2010].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - ADP biosynthetic process\n",
      "    - blood coagulation\n",
      "    - nucleoside triphosphate biosynthetic process\n",
      "    - AMP metabolic process\n",
      "    - AMP metabolic process\n",
      "    - GTP metabolic process\n",
      "    - ITP metabolic process\n",
      "    - UTP metabolic process\n",
      "  CC:\n",
      "    - cytoplasm\n",
      "    - mitochondrion\n",
      "    - mitochondrion\n",
      "    - mitochondrion\n",
      "    - mitochondrial matrix\n",
      "    - mitochondrial matrix\n",
      "    - mitochondrial matrix\n",
      "  MF:\n",
      "    - adenylate kinase activity\n",
      "    - protein binding\n",
      "    - ATP binding\n",
      "    - GTP binding\n",
      "    - ligase activity\n",
      "    - nucleoside triphosphate adenylate kinase activity\n",
      "    - nucleoside triphosphate adenylate kinase activity\n",
      "    - nucleoside triphosphate adenylate kinase activity\n",
      "UniProt: {'Swiss-Prot': 'Q9UIJ7', 'TrEMBL': 'Q7Z4Y4'}\n",
      "\n",
      "Gene: CYTH3\n",
      "Symbol: CYTH3\n",
      "Name: cytohesin 3\n",
      "Summary: This gene encodes a member of the PSCD (pleckstrin homology, Sec7 and coiled-coil domains) family. PSCD family members have identical structural organization that consists of an N-terminal coiled-coil motif, a central Sec7 domain, and a C-terminal pleckstrin homology (PH) domain. The coiled-coil motif is involved in homodimerization, the Sec7 domain contains guanine-nucleotide exchange protein (GEP) activity, and the PH domain interacts with phospholipids and is responsible for association of PSCDs with membranes. Members of this family appear to mediate the regulation of protein sorting and membrane trafficking. This encoded protein is involved in the control of Golgi structure and function, and it may have a physiological role in regulating ADP-ribosylation factor protein 6 (ARF) functions, in addition to acting on ARF1. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - regulation of ARF protein signal transduction\n",
      "    - positive regulation of cell adhesion\n",
      "    - Golgi vesicle transport\n",
      "    - establishment of epithelial cell polarity\n",
      "  CC:\n",
      "    - Golgi membrane\n",
      "    - ruffle\n",
      "    - nucleoplasm\n",
      "    - cytosol\n",
      "    - cytosol\n",
      "    - plasma membrane\n",
      "    - adherens junction\n",
      "    - bicellular tight junction\n",
      "  MF:\n",
      "    - guanyl-nucleotide exchange factor activity\n",
      "    - protein binding\n",
      "    - phosphatidylinositol-3,4,5-trisphosphate binding\n",
      "UniProt: {'Swiss-Prot': 'O43739'}\n",
      "\n",
      "Gene: HSD17B1\n",
      "Symbol: HSD17B1\n",
      "Name: hydroxysteroid 17-beta dehydrogenase 1\n",
      "Summary: This gene encodes a member of the 17beta-hydroxysteroid dehydrogenase family of short-chain dehydrogenases/reductases. It has a dual function in estrogen activation and androgen inactivation and plays a major role in establishing the estrogen E2 concentration gradient between serum and peripheral tissues. The encoded protein catalyzes the last step in estrogen activation, using NADPH to convert estrogens E1 and E2 and androgens like 4-androstenedione, to testosterone. It has an N-terminal short-chain dehydrogenase domain with a cofactor binding site, and a narrow, hydrophobic C-terminal domain with a steroid substrate binding site. This gene is expressed primarily in the placenta and ovarian granulosa cells, and to a lesser extent, in the endometrium, adipose tissue, and prostate. Polymorphisms in this gene have been linked to breast and prostate cancer. A pseudogene of this gene has been identified. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Sep 2016].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - steroid biosynthetic process\n",
      "    - estrogen biosynthetic process\n",
      "    - estrogen biosynthetic process\n",
      "    - estrogen biosynthetic process\n",
      "    - estrogen biosynthetic process\n",
      "    - lysosome organization\n",
      "    - skeletal muscle tissue development\n",
      "    - estrogen metabolic process\n",
      "    - gene expression\n",
      "    - bone development\n",
      "    - adipose tissue development\n",
      "    - testosterone biosynthetic process\n",
      "    - cellular response to metal ion\n",
      "  CC:\n",
      "    - cytoplasm\n",
      "    - cytosol\n",
      "    - cytosol\n",
      "    - cytosol\n",
      "  MF:\n",
      "    - catalytic activity\n",
      "    - estradiol 17-beta-dehydrogenase [NAD(P)+] activity\n",
      "    - estradiol 17-beta-dehydrogenase [NAD(P)+] activity\n",
      "    - steroid binding\n",
      "    - protein binding\n",
      "    - testosterone dehydrogenase [NAD(P)+] activity\n",
      "    - small molecule binding\n",
      "    - protein homodimerization activity\n",
      "    - testosterone dehydrogenase (NAD+) activity\n",
      "    - testosterone 17-beta-dehydrogenase (NADP+) activity\n",
      "    - NADP binding\n",
      "    - NADP+ binding\n",
      "    - 17-beta-hydroxysteroid dehydrogenase (NADP+) activity\n",
      "    - estradiol binding\n",
      "UniProt: {'Swiss-Prot': 'P14061', 'TrEMBL': ['B4DU11', 'A0A0A0MQS7']}\n",
      "\n",
      "Gene: KRTAP23-1\n",
      "Symbol: KRTAP23-1\n",
      "Name: keratin associated protein 23-1\n",
      "Summary: Predicted to be located in cytosol. [provided by Alliance of Genome Resources, Apr 2022]\n",
      "GO Annotations:\n",
      "  CC:\n",
      "    - cytosol\n",
      "    - intermediate filament\n",
      "  MF:\n",
      "    - protein binding\n",
      "UniProt: {'Swiss-Prot': 'A1A580'}\n",
      "\n",
      "Gene: ZFP90\n",
      "Symbol: ZFP90\n",
      "Name: ZFP90 zinc finger protein\n",
      "Summary: This gene encodes a member of the zinc finger protein family that modulates gene expression. The encoded protein derepresses the transcription of certain fetal cardiac genes and may contribute to the genetic reprogramming that occurs during the development of heart failure. Genome wide association studies have identified this gene among ulcerative colitis risk loci. Alternative splicing of this gene results in multiple transcript variants. [provided by RefSeq, Mar 2015].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - negative regulation of transcription by RNA polymerase II\n",
      "    - regulation of transcription by RNA polymerase II\n",
      "    - positive regulation of DNA-templated transcription\n",
      "    - positive regulation of DNA-templated transcription\n",
      "  CC:\n",
      "    - nucleus\n",
      "  MF:\n",
      "    - RNA polymerase II transcription regulatory region sequence-specific DNA binding\n",
      "    - DNA-binding transcription factor activity, RNA polymerase II-specific\n",
      "    - DNA-binding transcription repressor activity, RNA polymerase II-specific\n",
      "    - protein binding\n",
      "    - zinc ion binding\n",
      "UniProt: {'Swiss-Prot': 'Q8TF47', 'TrEMBL': ['H3BMI1', 'J3QKQ8', 'H3BVC6', 'I3L0X7', 'I3L1L4', 'I3L0T2', 'I3L3M0', 'I3L2Q3']}\n",
      "\n",
      "Gene: DNAJB5\n",
      "Symbol: DNAJB5\n",
      "Name: DnaJ heat shock protein family (Hsp40) member B5\n",
      "Summary: This gene encodes a member of the DNAJ heat shock protein 40 family of co-chaperone proteins. The encoded protein contains an N-terminal DNAJ domain and a C-terminal substrate binding domain but lacks the cysteine-rich domain found in other DNAJ family members. In mice, a multi-protein complex containing this protein, thioredoxin 1, and histone deacetylase 4, serves as a master negative regulator of cardiac hypertrophy. [provided by RefSeq, Mar 2017].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - response to unfolded protein\n",
      "    - chaperone cofactor-dependent protein refolding\n",
      "  CC:\n",
      "    - cytosol\n",
      "    - cytosol\n",
      "  MF:\n",
      "    - protein binding\n",
      "    - unfolded protein binding\n",
      "    - protein-folding chaperone binding\n",
      "    - protein-folding chaperone binding\n",
      "UniProt: {'Swiss-Prot': 'O75953', 'TrEMBL': ['A0A7I2RN43', 'H7C5W0', 'H0YFL9', 'H0YH04', 'Q5T657', 'H0Y4S1']}\n",
      "\n",
      "Gene: SCN5A\n",
      "Symbol: SCN5A\n",
      "Name: sodium voltage-gated channel alpha subunit 5\n",
      "Summary: The protein encoded by this gene is an integral membrane protein and tetrodotoxin-resistant voltage-gated sodium channel subunit. This protein is found primarily in cardiac muscle and is responsible for the initial upstroke of the action potential in an electrocardiogram. Defects in this gene have been associated with long QT syndrome type 3 (LQT3), atrial fibrillation, cardiomyopathy, and Brugada syndrome 1, all autosomal dominant cardiac diseases. Alternative splicing results in several transcript variants encoding different isoforms. [provided by RefSeq, May 2022].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - regulation of heart rate\n",
      "    - cardiac conduction system development\n",
      "    - cardiac ventricle development\n",
      "    - brainstem development\n",
      "    - sodium ion transport\n",
      "    - positive regulation of sodium ion transport\n",
      "    - response to denervation involved in regulation of muscle adaptation\n",
      "    - telencephalon development\n",
      "    - cerebellum development\n",
      "    - sodium ion transmembrane transport\n",
      "    - sodium ion transmembrane transport\n",
      "    - sodium ion transmembrane transport\n",
      "    - odontogenesis of dentin-containing tooth\n",
      "    - positive regulation of action potential\n",
      "    - positive regulation of epithelial cell proliferation\n",
      "    - membrane depolarization\n",
      "    - cardiac muscle contraction\n",
      "    - regulation of ventricular cardiac muscle cell membrane repolarization\n",
      "    - regulation of atrial cardiac muscle cell membrane depolarization\n",
      "    - regulation of atrial cardiac muscle cell membrane repolarization\n",
      "    - regulation of ventricular cardiac muscle cell membrane depolarization\n",
      "    - cellular response to calcium ion\n",
      "    - cardiac muscle cell action potential involved in contraction\n",
      "    - cardiac muscle cell action potential involved in contraction\n",
      "    - regulation of cardiac muscle cell contraction\n",
      "    - ventricular cardiac muscle cell action potential\n",
      "    - membrane depolarization during action potential\n",
      "    - membrane depolarization during cardiac muscle cell action potential\n",
      "    - atrial cardiac muscle cell action potential\n",
      "    - SA node cell action potential\n",
      "    - AV node cell action potential\n",
      "    - bundle of His cell action potential\n",
      "    - membrane depolarization during AV node cell action potential\n",
      "    - membrane depolarization during SA node cell action potential\n",
      "    - membrane depolarization during Purkinje myocyte cell action potential\n",
      "    - membrane depolarization during bundle of His cell action potential\n",
      "    - AV node cell to bundle of His cell communication\n",
      "    - regulation of heart rate by cardiac conduction\n",
      "    - membrane depolarization during atrial cardiac muscle cell action potential\n",
      "    - regulation of sodium ion transmembrane transport\n",
      "  CC:\n",
      "    - voltage-gated sodium channel complex\n",
      "    - voltage-gated sodium channel complex\n",
      "    - voltage-gated sodium channel complex\n",
      "    - nucleoplasm\n",
      "    - nucleolus\n",
      "    - endoplasmic reticulum\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - caveola\n",
      "    - caveola\n",
      "    - cell surface\n",
      "    - intercalated disc\n",
      "    - intercalated disc\n",
      "    - membrane\n",
      "    - lateral plasma membrane\n",
      "    - Z disc\n",
      "    - T-tubule\n",
      "    - sarcolemma\n",
      "    - perinuclear region of cytoplasm\n",
      "  MF:\n",
      "    - voltage-gated sodium channel activity\n",
      "    - voltage-gated sodium channel activity\n",
      "    - voltage-gated sodium channel activity\n",
      "    - protein binding\n",
      "    - calmodulin binding\n",
      "    - fibroblast growth factor binding\n",
      "    - enzyme binding\n",
      "    - protein kinase binding\n",
      "    - protein domain specific binding\n",
      "    - ankyrin binding\n",
      "    - ubiquitin protein ligase binding\n",
      "    - transmembrane transporter binding\n",
      "    - nitric-oxide synthase binding\n",
      "    - voltage-gated sodium channel activity involved in cardiac muscle cell action potential\n",
      "    - voltage-gated sodium channel activity involved in cardiac muscle cell action potential\n",
      "    - voltage-gated sodium channel activity involved in AV node cell action potential\n",
      "    - voltage-gated sodium channel activity involved in bundle of His cell action potential\n",
      "    - voltage-gated sodium channel activity involved in Purkinje myocyte action potential\n",
      "    - voltage-gated sodium channel activity involved in SA node cell action potential\n",
      "    - scaffold protein binding\n",
      "UniProt: {'Swiss-Prot': 'Q14524', 'TrEMBL': ['E9PHB6', 'Q86V90', 'K4DIA1', 'E9PG18', 'H9KVD2', 'A0A0A0MT39', 'A0AAQ5BGT0', 'A0AAQ5BGV8']}\n",
      "\n",
      "Gene: PSCA\n",
      "Symbol: PSCA\n",
      "Name: prostate stem cell antigen\n",
      "Summary: This gene encodes a glycosylphosphatidylinositol-anchored cell membrane glycoprotein. In addition to being highly expressed in the prostate it is also expressed in the bladder, placenta, colon, kidney, and stomach. This gene is up-regulated in a large proportion of prostate cancers and is also detected in cancers of the bladder and pancreas. This gene includes a polymorphism that results in an upstream start codon in some individuals; this polymorphism is thought to be associated with a risk for certain gastric and bladder cancers. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Feb 2010].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - negative regulation of ERK1 and ERK2 cascade\n",
      "    - acetylcholine receptor signaling pathway\n",
      "    - regulation of neurotransmitter receptor activity\n",
      "  CC:\n",
      "    - extracellular region\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - synapse\n",
      "    - extracellular exosome\n",
      "    - side of membrane\n",
      "  MF:\n",
      "    - acetylcholine receptor regulator activity\n",
      "    - acetylcholine receptor binding\n",
      "    - acetylcholine receptor binding\n",
      "UniProt: {'Swiss-Prot': 'O43653', 'TrEMBL': 'H0YAA6'}\n",
      "\n",
      "Gene: FIG4\n",
      "Symbol: FIG4\n",
      "Name: FIG4 phosphoinositide 5-phosphatase\n",
      "Summary: The protein encoded by this gene belongs to the SAC domain-containing protein gene family. The SAC domain, approximately 400 amino acids in length and consisting of seven conserved motifs, has been shown to possess phosphoinositide phosphatase activity. The yeast homolog, Sac1p, is involved in the regulation of various phosphoinositides, and affects diverse cellular functions such as actin cytoskeleton organization, Golgi function, and maintenance of vacuole morphology. Membrane-bound phosphoinositides function as signaling molecules and play a key role in vesicle trafficking in eukaryotic cells. Mutations in this gene have been associated with Charcot-Marie-Tooth disease, type 4J. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - phosphatidylinositol biosynthetic process\n",
      "    - vacuole organization\n",
      "    - locomotory behavior\n",
      "    - positive regulation of neuron projection development\n",
      "    - negative regulation of myelination\n",
      "    - myelin assembly\n",
      "    - phosphatidylinositol-3-phosphate biosynthetic process\n",
      "    - pigmentation\n",
      "    - phosphatidylinositol dephosphorylation\n",
      "    - neuron development\n",
      "  CC:\n",
      "    - Golgi membrane\n",
      "    - lipid droplet\n",
      "    - endosome membrane\n",
      "    - early endosome membrane\n",
      "    - late endosome membrane\n",
      "    - intracellular membrane-bounded organelle\n",
      "    - intracellular membrane-bounded organelle\n",
      "    - recycling endosome\n",
      "  MF:\n",
      "    - phosphatidylinositol-3-phosphate phosphatase activity\n",
      "    - phosphatidylinositol-4,5-bisphosphate 5-phosphatase activity\n",
      "    - protein serine/threonine phosphatase activity\n",
      "    - protein binding\n",
      "    - phosphatidylinositol-3,4,5-trisphosphate 5-phosphatase activity\n",
      "    - phosphatidylinositol-4-phosphate phosphatase activity\n",
      "    - phosphatidylinositol-3,5-bisphosphate 5-phosphatase activity\n",
      "    - phosphatidylinositol-3,5-bisphosphate 5-phosphatase activity\n",
      "UniProt: {'Swiss-Prot': 'Q92562', 'TrEMBL': ['A0A6Q8PF86', 'A0A6Q8PG49', 'A0A6Q8PGL0', 'Q5TCS4', 'A0A6Q8PF70', 'A0A6Q8PHP5', 'A0A6Q8PHA1', 'A0A6Q8PFJ3', 'A0A6Q8PGC1', 'A0A6Q8PHH5', 'H0Y6G7', 'A0A6Q8PHM5', 'A0A6Q8PGP4', 'A0A6Q8PGY4', 'A0A6Q8PF62', 'A0A6Q8PHC7', 'A0A6Q8PGM1', 'A0A6Q8PHK3', 'A0A6Q8PHB4', 'A0A6Q8PGW0', 'A0A6Q8PF27', 'A0A6Q8PG61', 'A0A6Q8PFK7', 'A0A6Q8PF04', 'A0A6Q8PH92', 'A0A6Q8PF90', 'A0A6Q8PGB4', 'Q5TCS5', 'A0A6Q8PGY7', 'A0A6Q8PG59', 'A0A6Q8PGW5', 'A0A6Q8PGP1']}\n",
      "\n",
      "Gene: CA3\n",
      "Symbol: CA3\n",
      "Name: carbonic anhydrase 3\n",
      "Summary: Carbonic anhydrase III (CAIII) is a member of a multigene family (at least six separate genes are known) that encodes carbonic anhydrase isozymes. These carbonic anhydrases are a class of metalloenzymes that catalyze the reversible hydration of carbon dioxide and are differentially expressed in a number of cell types. The expression of the CA3 gene is strictly tissue specific and present at high levels in skeletal muscle and much lower levels in cardiac and smooth muscle. A proportion of carriers of Duchenne muscle dystrophy have a higher CA3 level than normal. The gene spans 10.3 kb and contains seven exons and six introns. [provided by RefSeq, Oct 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - response to bacterium\n",
      "  CC:\n",
      "    - cytoplasm\n",
      "    - cytosol\n",
      "    - cytosol\n",
      "    - cytosol\n",
      "  MF:\n",
      "    - carbonate dehydratase activity\n",
      "    - carbonate dehydratase activity\n",
      "    - protein binding\n",
      "    - zinc ion binding\n",
      "    - nickel cation binding\n",
      "UniProt: {'Swiss-Prot': 'P07451', 'TrEMBL': ['E5RHI4', 'V9HWA3']}\n",
      "\n",
      "Gene: ZBTB6\n",
      "Symbol: ZBTB6\n",
      "Name: zinc finger and BTB domain containing 6\n",
      "Summary: Predicted to enable DNA-binding transcription repressor activity, RNA polymerase II-specific and RNA polymerase II cis-regulatory region sequence-specific DNA binding activity. Predicted to be involved in regulation of transcription by RNA polymerase II. Located in mitochondrion and nucleoplasm. [provided by Alliance of Genome Resources, Apr 2022]\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - negative regulation of transcription by RNA polymerase II\n",
      "    - regulation of cytokine production\n",
      "    - regulation of immune system process\n",
      "  CC:\n",
      "    - nucleus\n",
      "    - nucleoplasm\n",
      "    - nucleoplasm\n",
      "    - mitochondrion\n",
      "  MF:\n",
      "    - RNA polymerase II cis-regulatory region sequence-specific DNA binding\n",
      "    - DNA-binding transcription repressor activity, RNA polymerase II-specific\n",
      "    - DNA binding\n",
      "    - protein binding\n",
      "    - zinc ion binding\n",
      "UniProt: {'Swiss-Prot': 'Q15916'}\n",
      "\n",
      "Gene: UGT8\n",
      "Symbol: UGT8\n",
      "Name: UDP glycosyltransferase 8\n",
      "Summary: The protein encoded by this gene belongs to the UDP-glycosyltransferase family. It catalyzes the transfer of galactose to ceramide, a key enzymatic step in the biosynthesis of galactocerebrosides, which are abundant sphingolipids of the myelin membrane of the central and peripheral nervous systems. Alternatively spliced transcript variants have been found for this gene. [provided by RefSeq, Sep 2011].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - protein localization to paranode region of axon\n",
      "    - galactosylceramide biosynthetic process\n",
      "    - galactosylceramide biosynthetic process\n",
      "    - galactosylceramide biosynthetic process\n",
      "    - glycosphingolipid biosynthetic process\n",
      "    - cytoskeleton organization\n",
      "    - central nervous system development\n",
      "    - peripheral nervous system development\n",
      "    - paranodal junction assembly\n",
      "    - response to immobilization stress\n",
      "    - neuron projection morphogenesis\n",
      "  CC:\n",
      "    - endoplasmic reticulum\n",
      "    - endoplasmic reticulum\n",
      "    - endoplasmic reticulum membrane\n",
      "  MF:\n",
      "    - N-acylsphingosine galactosyltransferase activity\n",
      "    - N-acylsphingosine galactosyltransferase activity\n",
      "    - N-acylsphingosine galactosyltransferase activity\n",
      "    - UDP-galactose:glucosylceramide beta-1,4-galactosyltransferase activity\n",
      "UniProt: {'Swiss-Prot': 'Q16880', 'TrEMBL': 'D6RFW2'}\n",
      "\n",
      "Gene: AGPAT5\n",
      "Symbol: AGPAT5\n",
      "Name: 1-acylglycerol-3-phosphate O-acyltransferase 5\n",
      "Summary: This gene encodes a member of the 1-acylglycerol-3-phosphate O-acyltransferase family. This integral membrane protein converts lysophosphatidic acid to phosphatidic acid, the second step in de novo phospholipid biosynthesis. A pseudogene of this gene is present on the Y chromosome. [provided by RefSeq, Aug 2014].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - hematopoietic progenitor cell differentiation\n",
      "    - acylglycerol metabolic process\n",
      "    - phosphatidic acid biosynthetic process\n",
      "    - phospholipid biosynthetic process\n",
      "    - CDP-diacylglycerol biosynthetic process\n",
      "    - phosphatidylinositol acyl-chain remodeling\n",
      "  CC:\n",
      "    - nuclear envelope\n",
      "    - nucleolus\n",
      "    - mitochondrion\n",
      "    - mitochondrion\n",
      "    - mitochondrion\n",
      "    - mitochondrial outer membrane\n",
      "    - endoplasmic reticulum\n",
      "    - endoplasmic reticulum membrane\n",
      "    - endoplasmic reticulum membrane\n",
      "    - endomembrane system\n",
      "  MF:\n",
      "    - 1-acylglycerol-3-phosphate O-acyltransferase activity\n",
      "    - 1-acylglycerol-3-phosphate O-acyltransferase activity\n",
      "    - 1-acylglycerol-3-phosphate O-acyltransferase activity\n",
      "    - protein binding\n",
      "    - acyltransferase activity\n",
      "UniProt: {'Swiss-Prot': 'Q9NUQ2', 'TrEMBL': ['H0YC22', 'A0A024R640', 'E5RH21', 'H0YAW1']}\n",
      "\n",
      "Gene: GYG1\n",
      "Symbol: GYG1\n",
      "Name: glycogenin 1\n",
      "Summary: This gene encodes a member of the glycogenin family. Glycogenin is a glycosyltransferase that catalyzes the formation of a short glucose polymer from uridine diphosphate glucose in an autoglucosylation reaction. This reaction is followed by elongation and branching of the polymer, catalyzed by glycogen synthase and branching enzyme, to form glycogen. This gene is expressed in muscle and other tissues. Mutations in this gene result in glycogen storage disease XV. This gene has pseudogenes on chromosomes 1, 8 and 13 respectively. Alternatively spliced transcript variants encoding different isoforms have been identified.[provided by RefSeq, Sep 2010].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - glycogen biosynthetic process\n",
      "    - glycogen biosynthetic process\n",
      "    - glycogen biosynthetic process\n",
      "    - glycogen biosynthetic process\n",
      "  CC:\n",
      "    - extracellular region\n",
      "    - nucleus\n",
      "    - cytoplasm\n",
      "    - cytoplasm\n",
      "    - cytoplasm\n",
      "    - cytosol\n",
      "    - membrane\n",
      "    - secretory granule lumen\n",
      "    - lysosomal lumen\n",
      "    - ficolin-1-rich granule lumen\n",
      "  MF:\n",
      "    - protein binding\n",
      "    - glycogenin glucosyltransferase activity\n",
      "    - glycogenin glucosyltransferase activity\n",
      "    - glycosyltransferase activity\n",
      "    - manganese ion binding\n",
      "    - protein homodimerization activity\n",
      "UniProt: {'Swiss-Prot': 'P46976', 'TrEMBL': ['C9J7C7', 'G5E9W8', 'C9JQ42', 'C9J8R8']}\n",
      "\n",
      "Gene: MGST2\n",
      "Symbol: MGST2\n",
      "Name: microsomal glutathione S-transferase 2\n",
      "Summary: The MAPEG (Membrane Associated Proteins in Eicosanoid and Glutathione metabolism) family consists of six human proteins, several of which are involved in the production of leukotrienes and prostaglandin E, important mediators of inflammation. This gene encodes a protein which catalyzes the conjugation of leukotriene A4 and reduced glutathione to produce leukotriene C4. Alternatively spliced transcript variants encoding different isoforms have been identified in this gene. [provided by RefSeq, Feb 2011].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - lipid metabolic process\n",
      "    - glutathione biosynthetic process\n",
      "    - leukotriene biosynthetic process\n",
      "    - leukotriene biosynthetic process\n",
      "    - response to lipopolysaccharide\n",
      "    - membrane lipid catabolic process\n",
      "    - positive regulation of inflammatory response\n",
      "    - cellular oxidant detoxification\n",
      "  CC:\n",
      "    - nuclear envelope\n",
      "    - endoplasmic reticulum\n",
      "    - endoplasmic reticulum membrane\n",
      "    - plasma membrane\n",
      "    - membrane\n",
      "    - intracellular membrane-bounded organelle\n",
      "  MF:\n",
      "    - glutathione transferase activity\n",
      "    - glutathione transferase activity\n",
      "    - leukotriene-C4 synthase activity\n",
      "    - leukotriene-C4 synthase activity\n",
      "    - glutathione peroxidase activity\n",
      "    - glutathione peroxidase activity\n",
      "    - protein binding\n",
      "    - enzyme activator activity\n",
      "    - identical protein binding\n",
      "    - identical protein binding\n",
      "    - glutathione binding\n",
      "UniProt: {'Swiss-Prot': 'Q99735'}\n",
      "\n",
      "Gene: CHRNG\n",
      "Symbol: CHRNG\n",
      "Name: cholinergic receptor nicotinic gamma subunit\n",
      "Summary: The mammalian muscle-type acetylcholine receptor is a transmembrane pentameric glycoprotein with two alpha subunits, one beta, one delta, and one epsilon (in adult skeletal muscle) or gamma (in fetal and denervated muscle) subunit. This gene, which encodes the gamma subunit, is expressed prior to the thirty-third week of gestation in humans. The gamma subunit of the acetylcholine receptor plays a role in neuromuscular organogenesis and ligand binding and disruption of gamma subunit expression prevents the correct localization of the receptor in cell membranes. Mutations in this gene cause Escobar syndrome and a lethal form of multiple pterygium syndrome. Muscle-type acetylcholine receptor is the major antigen in the autoimmune disease myasthenia gravis.[provided by RefSeq, Sep 2009].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - muscle contraction\n",
      "    - signal transduction\n",
      "    - chemical synaptic transmission\n",
      "    - synaptic transmission, cholinergic\n",
      "    - monoatomic ion transmembrane transport\n",
      "    - membrane depolarization\n",
      "    - excitatory postsynaptic potential\n",
      "    - acetylcholine receptor signaling pathway\n",
      "  CC:\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - acetylcholine-gated channel complex\n",
      "    - neuron projection\n",
      "    - synapse\n",
      "    - postsynaptic membrane\n",
      "  MF:\n",
      "    - protein binding\n",
      "    - channel activity\n",
      "    - acetylcholine receptor activity\n",
      "    - acetylcholine-gated monoatomic cation-selective channel activity\n",
      "    - transmitter-gated monoatomic ion channel activity involved in regulation of postsynaptic membrane potential\n",
      "UniProt: {'Swiss-Prot': 'P07510', 'TrEMBL': 'A0A6F7YAP6'}\n",
      "\n",
      "Gene: ALDH1A1\n",
      "Symbol: ALDH1A1\n",
      "Name: aldehyde dehydrogenase 1 family member A1\n",
      "Summary: The protein encoded by this gene belongs to the aldehyde dehydrogenase family. Aldehyde dehydrogenase is the next enzyme after alcohol dehydrogenase in the major pathway of alcohol metabolism. There are two major aldehyde dehydrogenase isozymes in the liver, cytosolic and mitochondrial, which are encoded by distinct genes, and can be distinguished by their electrophoretic mobility, kinetic properties, and subcellular localization. This gene encodes the cytosolic isozyme. Studies in mice show that through its role in retinol metabolism, this gene may also be involved in the regulation of the metabolic responses to high-fat diet. [provided by RefSeq, Mar 2011].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - retinoid metabolic process\n",
      "    - aldehyde metabolic process\n",
      "    - gamma-aminobutyric acid biosynthetic process\n",
      "    - fructosamine catabolic process\n",
      "    - maintenance of lens transparency\n",
      "    - retinol metabolic process\n",
      "    - cellular detoxification of aldehyde\n",
      "    - negative regulation of cold-induced thermogenesis\n",
      "  CC:\n",
      "    - cytoplasm\n",
      "    - cytosol\n",
      "    - cytosol\n",
      "    - axon\n",
      "    - synapse\n",
      "    - extracellular exosome\n",
      "  MF:\n",
      "    - retinal dehydrogenase activity\n",
      "    - aldehyde dehydrogenase (NAD+) activity\n",
      "    - aldehyde dehydrogenase (NAD+) activity\n",
      "    - GTPase activator activity\n",
      "    - androgen binding\n",
      "    - protein binding\n",
      "    - benzaldehyde dehydrogenase (NAD+) activity\n",
      "    - aminobutyraldehyde dehydrogenase (NAD+) activity\n",
      "    - NAD binding\n",
      "    - 3-deoxyglucosone dehydrogenase activity\n",
      "UniProt: {'Swiss-Prot': 'P00352', 'TrEMBL': ['Q5SYQ8', 'V9HW83', 'Q5SYQ9', 'Q5SYQ7']}\n",
      "\n",
      "Gene: PTPN5\n",
      "Symbol: PTPN5\n",
      "Name: protein tyrosine phosphatase non-receptor type 5\n",
      "Summary: Enables phosphotyrosine residue binding activity. Predicted to be involved in peptidyl-tyrosine dephosphorylation. Predicted to act upstream of or within protein dephosphorylation. Predicted to be located in nucleoplasm. Predicted to be integral component of membrane. Biomarker of Alzheimer's disease. [provided by Alliance of Genome Resources, Apr 2022]\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - chromatin remodeling\n",
      "    - protein dephosphorylation\n",
      "    - signal transduction\n",
      "  CC:\n",
      "    - nucleoplasm\n",
      "    - endoplasmic reticulum membrane\n",
      "    - cytosol\n",
      "    - plasma membrane\n",
      "    - membrane\n",
      "    - cell junction\n",
      "  MF:\n",
      "    - phosphotyrosine residue binding\n",
      "    - protein tyrosine phosphatase activity\n",
      "    - protein tyrosine phosphatase activity\n",
      "    - non-membrane spanning protein tyrosine phosphatase activity\n",
      "    - protein binding\n",
      "    - protein kinase binding\n",
      "    - protein tyrosine phosphatase activity, metal-dependent\n",
      "    - histone H2AXY142 phosphatase activity\n",
      "UniProt: {'Swiss-Prot': 'P54829', 'TrEMBL': ['B7Z2F8', 'Q86TL3', 'A8MXF1', 'E9PLZ4']}\n",
      "\n",
      "Gene: MAT1A\n",
      "Symbol: MAT1A\n",
      "Name: methionine adenosyltransferase 1A\n",
      "Summary: This gene catalyzes a two-step reaction that involves the transfer of the adenosyl moiety of ATP to methionine to form S-adenosylmethionine and tripolyphosphate, which is subsequently cleaved to PPi and Pi. S-adenosylmethionine is the source of methyl groups for most biological methylations. The encoded protein is found as a homotetramer (MAT I) or a homodimer (MAT III) whereas a third form, MAT II (gamma), is encoded by the MAT2A gene. Mutations in this gene are associated with methionine adenosyltransferase deficiency. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - S-adenosylmethionine biosynthetic process\n",
      "    - S-adenosylmethionine biosynthetic process\n",
      "    - S-adenosylmethionine biosynthetic process\n",
      "    - S-adenosylmethionine biosynthetic process\n",
      "    - S-adenosylmethionine biosynthetic process\n",
      "    - one-carbon metabolic process\n",
      "    - methionine catabolic process\n",
      "    - protein homotetramerization\n",
      "  CC:\n",
      "    - cytosol\n",
      "    - cytosol\n",
      "    - methionine adenosyltransferase complex\n",
      "  MF:\n",
      "    - methionine adenosyltransferase activity\n",
      "    - methionine adenosyltransferase activity\n",
      "    - protein binding\n",
      "    - ATP binding\n",
      "    - identical protein binding\n",
      "    - metal ion binding\n",
      "UniProt: {'Swiss-Prot': 'Q00266', 'TrEMBL': 'B1ANE6'}\n",
      "\n",
      "Gene: GRIK3\n",
      "Symbol: GRIK3\n",
      "Name: glutamate ionotropic receptor kainate type subunit 3\n",
      "Summary: Glutamate receptors are the predominant excitatory neurotransmitter receptors in the mammalian brain and are activated in a variety of normal neurophysiologic processes. This gene product belongs to the kainate family of glutamate receptors, which are composed of four subunits and function as ligand-activated ion channels. It is not certain if the subunit encoded by this gene is subject to RNA editing as the other 2 family members (GRIK1 and GRIK2). A Ser310Ala polymorphism has been associated with schizophrenia, and there are conflicting reports of its association with the pathogenesis of delirium tremens in alcoholics. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - adenylate cyclase-inhibiting G protein-coupled glutamate receptor signaling pathway\n",
      "    - glutamate receptor signaling pathway\n",
      "    - G protein-coupled glutamate receptor signaling pathway\n",
      "    - calcium-mediated signaling\n",
      "    - monoatomic ion transmembrane transport\n",
      "    - ionotropic glutamate receptor signaling pathway\n",
      "    - synaptic transmission, glutamatergic\n",
      "    - regulation of membrane potential\n",
      "    - regulation of membrane potential\n",
      "    - modulation of chemical synaptic transmission\n",
      "    - negative regulation of synaptic transmission, glutamatergic\n",
      "    - regulation of postsynaptic membrane potential\n",
      "    - regulation of presynaptic membrane potential\n",
      "  CC:\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - axon\n",
      "    - dendrite\n",
      "    - dendrite cytoplasm\n",
      "    - kainate selective glutamate receptor complex\n",
      "    - presynaptic membrane\n",
      "    - terminal bouton\n",
      "    - perikaryon\n",
      "    - postsynaptic density membrane\n",
      "    - glutamatergic synapse\n",
      "  MF:\n",
      "    - adenylate cyclase inhibiting G protein-coupled glutamate receptor activity\n",
      "    - glutamate-gated receptor activity\n",
      "    - glutamate-gated receptor activity\n",
      "    - glutamate receptor activity\n",
      "    - kainate selective glutamate receptor activity\n",
      "    - kainate selective glutamate receptor activity\n",
      "    - glutamate-gated calcium ion channel activity\n",
      "    - ligand-gated monoatomic ion channel activity involved in regulation of presynaptic membrane potential\n",
      "    - transmitter-gated monoatomic ion channel activity involved in regulation of postsynaptic membrane potential\n",
      "UniProt: {'Swiss-Prot': 'Q13003'}\n",
      "\n",
      "Gene: GPX1\n",
      "Symbol: GPX1\n",
      "Name: glutathione peroxidase 1\n",
      "Summary: The protein encoded by this gene belongs to the glutathione peroxidase family, members of which catalyze the reduction of organic hydroperoxides and hydrogen peroxide (H2O2) by glutathione, and thereby protect cells against oxidative damage. Other studies indicate that H2O2 is also essential for growth-factor mediated signal transduction, mitochondrial function, and maintenance of thiol redox-balance; therefore, by limiting H2O2 accumulation, glutathione peroxidases are also involved in modulating these processes. Several isozymes of this gene family exist in vertebrates, which vary in cellular location and substrate specificity. This isozyme is the most abundant, is ubiquitously expressed and localized in the cytoplasm, and whose preferred substrate is hydrogen peroxide. It is also a selenoprotein, containing the rare amino acid selenocysteine (Sec) at its active site. Sec is encoded by the UGA codon, which normally signals translation termination. The 3' UTRs of selenoprotein mRNAs contain a conserved stem-loop structure, designated the Sec insertion sequence (SECIS) element, that is necessary for the recognition of UGA as a Sec codon, rather than as a stop signal. This gene contains an in-frame GCG trinucleotide repeat in the coding region, and three alleles with 4, 5 or 6 repeats have been found in the human population. The allele with 4 GCG repeats has been significantly associated with breast cancer risk in premenopausal women. Alternatively spliced transcript variants have been found for this gene. Pseudogenes of this locus have been identified on chromosomes X and 21. [provided by RefSeq, Aug 2017].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - temperature homeostasis\n",
      "    - endothelial cell development\n",
      "    - negative regulation of inflammatory response to antigenic stimulus\n",
      "    - triglyceride metabolic process\n",
      "    - glutathione metabolic process\n",
      "    - glutathione metabolic process\n",
      "    - sensory perception of sound\n",
      "    - intrinsic apoptotic signaling pathway in response to oxidative stress\n",
      "    - response to xenobiotic stimulus\n",
      "    - response to symbiotic bacterium\n",
      "    - UV protection\n",
      "    - response to hormone\n",
      "    - response to selenium ion\n",
      "    - response to selenium ion\n",
      "    - response to gamma radiation\n",
      "    - arachidonate metabolic process\n",
      "    - lipoxygenase pathway\n",
      "    - response to estradiol\n",
      "    - response to lipopolysaccharide\n",
      "    - response to hydroperoxide\n",
      "    - response to vitamin E\n",
      "    - regulation of mammary gland epithelial cell proliferation\n",
      "    - cellular response to oxidative stress\n",
      "    - response to nicotine\n",
      "    - epigenetic regulation of gene expression\n",
      "    - vasodilation\n",
      "    - response to hydrogen peroxide\n",
      "    - response to hydrogen peroxide\n",
      "    - hydrogen peroxide catabolic process\n",
      "    - hydrogen peroxide catabolic process\n",
      "    - skeletal muscle tissue regeneration\n",
      "    - blood vessel endothelial cell migration\n",
      "    - fat cell differentiation\n",
      "    - myoblast differentiation\n",
      "    - cell redox homeostasis\n",
      "    - fibroblast proliferation\n",
      "    - skeletal muscle fiber development\n",
      "    - neuron apoptotic process\n",
      "    - myoblast proliferation\n",
      "    - response to folic acid\n",
      "    - biological process involved in interaction with symbiont\n",
      "    - positive regulation of phosphatidylinositol 3-kinase/protein kinase B signal transduction\n",
      "    - heart contraction\n",
      "    - angiogenesis involved in wound healing\n",
      "    - regulation of proteasomal protein catabolic process\n",
      "    - cellular response to glucose stimulus\n",
      "    - negative regulation of release of cytochrome c from mitochondria\n",
      "    - cellular oxidant detoxification\n",
      "    - negative regulation of extrinsic apoptotic signaling pathway via death domain receptors\n",
      "    - negative regulation of oxidative stress-induced intrinsic apoptotic signaling pathway\n",
      "    - positive regulation of supramolecular fiber organization\n",
      "  CC:\n",
      "    - cytoplasm\n",
      "    - mitochondrion\n",
      "    - mitochondrion\n",
      "    - mitochondrion\n",
      "    - mitochondrial matrix\n",
      "    - cytosol\n",
      "    - cytosol\n",
      "    - cytosol\n",
      "    - Lewy body\n",
      "  MF:\n",
      "    - glutathione peroxidase activity\n",
      "    - glutathione peroxidase activity\n",
      "    - protein binding\n",
      "    - SH3 domain binding\n",
      "    - phospholipid-hydroperoxide glutathione peroxidase activity\n",
      "UniProt: {'Swiss-Prot': 'P07203', 'TrEMBL': ['Q7L4Q3', 'A0A2R8Y6B6', 'A0A994J4E2', 'A0A994J4L4', 'A0A994J4K6', 'A0A994J430', 'A0A994J702', 'A0A7I2YMD7', 'A0A994J707', 'A0A7I2UQ36', 'A0A994J7E8']}\n",
      "\n",
      "Gene: GSTA4\n",
      "Symbol: GSTA4\n",
      "Name: glutathione S-transferase alpha 4\n",
      "Summary: Cytosolic and membrane-bound forms of glutathione S-transferase are encoded by two distinct supergene families. These enzymes are involved in cellular defense against toxic, carcinogenic, and pharmacologically active electrophilic compounds. At present, eight distinct classes of the soluble cytoplasmic mammalian glutathione S-transferases have been identified: alpha, kappa, mu, omega, pi, sigma, theta and zeta. This gene encodes a glutathione S-tranferase belonging to the alpha class. The alpha class genes, which are located in a cluster on chromosome 6, are highly related and encode enzymes with glutathione peroxidase activity that function in the detoxification of lipid peroxidation products. Reactive electrophiles produced by oxidative metabolism have been linked to a number of degenerative diseases including Parkinson's disease, Alzheimer's disease, cataract formation, and atherosclerosis. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - glutathione metabolic process\n",
      "    - glutathione metabolic process\n",
      "    - xenobiotic metabolic process\n",
      "    - xenobiotic metabolic process\n",
      "  CC:\n",
      "    - cytosol\n",
      "    - cytosol\n",
      "  MF:\n",
      "    - glutathione transferase activity\n",
      "    - glutathione transferase activity\n",
      "    - protein binding\n",
      "    - identical protein binding\n",
      "    - protein homodimerization activity\n",
      "UniProt: {'Swiss-Prot': 'O15217', 'TrEMBL': 'Q5JW88'}\n",
      "\n",
      "Gene: PDE3B\n",
      "Symbol: PDE3B\n",
      "Name: phosphodiesterase 3B\n",
      "Summary: Enables 3',5'-cyclic-nucleotide phosphodiesterase activity. Involved in negative regulation of angiogenesis; negative regulation of cell adhesion; and negative regulation of lipid catabolic process. Located in membrane. Part of guanyl-nucleotide exchange factor complex. [provided by Alliance of Genome Resources, Apr 2022]\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - angiogenesis\n",
      "    - negative regulation of cell adhesion\n",
      "    - G protein-coupled receptor signaling pathway\n",
      "    - negative regulation of angiogenesis\n",
      "    - cAMP-mediated signaling\n",
      "    - cellular response to insulin stimulus\n",
      "    - negative regulation of cell adhesion mediated by integrin\n",
      "    - regulation of angiogenesis\n",
      "    - negative regulation of lipid catabolic process\n",
      "    - regulation of phosphatidylinositol 3-kinase/protein kinase B signal transduction\n",
      "    - negative regulation of adenylate cyclase-activating G protein-coupled receptor signaling pathway\n",
      "  CC:\n",
      "    - endoplasmic reticulum\n",
      "    - Golgi apparatus\n",
      "    - cytosol\n",
      "    - membrane\n",
      "    - membrane\n",
      "    - guanyl-nucleotide exchange factor complex\n",
      "  MF:\n",
      "    - 3',5'-cyclic-nucleotide phosphodiesterase activity\n",
      "    - 3',5'-cyclic-nucleotide phosphodiesterase activity\n",
      "    - 3',5'-cyclic-nucleotide phosphodiesterase activity\n",
      "    - 3',5'-cyclic-AMP phosphodiesterase activity\n",
      "    - 3',5'-cyclic-AMP phosphodiesterase activity\n",
      "    - 3',5'-cyclic-AMP phosphodiesterase activity\n",
      "    - 3',5'-cGMP-stimulated cyclic-nucleotide phosphodiesterase activity\n",
      "    - 3',5'-cGMP-inhibited cyclic-nucleotide phosphodiesterase activity\n",
      "    - protein binding\n",
      "    - protein kinase B binding\n",
      "    - metal ion binding\n",
      "    - 3',5'-cyclic-GMP phosphodiesterase activity\n",
      "UniProt: {'Swiss-Prot': 'Q13370', 'TrEMBL': 'A7E2E5'}\n",
      "\n",
      "Gene: SLC22A6\n",
      "Symbol: SLC22A6\n",
      "Name: solute carrier family 22 member 6\n",
      "Summary: The protein encoded by this gene is involved in the sodium-dependent transport and excretion of organic anions, some of which are potentially toxic. The encoded protein is an integral membrane protein and may be localized to the basolateral membrane. Four transcript variants encoding four different isoforms have been found for this gene. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - monoatomic anion transport\n",
      "    - inorganic anion transport\n",
      "    - organic anion transport\n",
      "    - organic anion transport\n",
      "    - organic anion transport\n",
      "    - prostaglandin transport\n",
      "    - alpha-ketoglutarate transport\n",
      "    - xenobiotic transport\n",
      "    - sodium-independent organic anion transport\n",
      "    - sodium-independent organic anion transport\n",
      "    - transmembrane transport\n",
      "    - metanephric proximal tubule development\n",
      "    - renal tubular secretion\n",
      "    - renal tubular secretion\n",
      "  CC:\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - caveola\n",
      "    - basal plasma membrane\n",
      "    - basolateral plasma membrane\n",
      "    - protein-containing complex\n",
      "    - extracellular exosome\n",
      "  MF:\n",
      "    - solute:inorganic anion antiporter activity\n",
      "    - protein binding\n",
      "    - organic anion transmembrane transporter activity\n",
      "    - organic anion transmembrane transporter activity\n",
      "    - organic anion transmembrane transporter activity\n",
      "    - prostaglandin transmembrane transporter activity\n",
      "    - alpha-ketoglutarate transmembrane transporter activity\n",
      "    - antiporter activity\n",
      "    - sodium-independent organic anion transmembrane transporter activity\n",
      "    - sodium-independent organic anion transmembrane transporter activity\n",
      "    - transmembrane transporter activity\n",
      "    - chloride ion binding\n",
      "    - identical protein binding\n",
      "    - xenobiotic transmembrane transporter activity\n",
      "UniProt: {'Swiss-Prot': 'Q4U2R8', 'TrEMBL': 'F5H0T7'}\n",
      "\n",
      "Gene: DCPS\n",
      "Symbol: DCPS\n",
      "Name: decapping enzyme, scavenger\n",
      "Summary: This gene encodes a member of the histidine triad family of pyrophosphatases that removes short mRNA fragments containing the 5' mRNA cap structure, which appear in the 3' -> 5' mRNA decay pathway, following deadenylation and exosome-mediated turnover. This enzyme hydrolyzes the triphosphate linkage of the cap structure (7-methylguanosine nucleoside triphosphate) to yield 7-methylguanosine monophosphate and nucleoside diphosphate. It protects the cell from the potentially toxic accumulation of these short, capped mRNA fragments, and regulates the activity of other cap-binding proteins, which are inhibited by their accumulation. It also acts as a transcript-specific modulator of pre-mRNA splicing and microRNA turnover. [provided by RefSeq, Apr 2017].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - nuclear-transcribed mRNA catabolic process, deadenylation-dependent decay\n",
      "    - deadenylation-dependent decapping of nuclear-transcribed mRNA\n",
      "    - mRNA cis splicing, via spliceosome\n",
      "    - mRNA methylguanosine-cap decapping\n",
      "  CC:\n",
      "    - P-body\n",
      "    - nucleus\n",
      "    - nucleus\n",
      "    - nucleoplasm\n",
      "    - cytoplasm\n",
      "    - mitochondrion\n",
      "    - cytosol\n",
      "    - cytosol\n",
      "  MF:\n",
      "    - RNA 7-methylguanosine cap binding\n",
      "    - RNA 7-methylguanosine cap binding\n",
      "    - RNA exonuclease activity\n",
      "    - protein binding\n",
      "    - identical protein binding\n",
      "    - 5'-(N(7)-methyl 5'-triphosphoguanosine)-[mRNA] diphosphatase activity\n",
      "UniProt: {'Swiss-Prot': 'Q96C86', 'TrEMBL': ['A0A3B3ITF0', 'A0A384MTI8']}\n",
      "\n",
      "Gene: JMY\n",
      "Symbol: JMY\n",
      "Name: junction mediating and regulatory protein, p53 cofactor\n",
      "Summary: Predicted to enable Arp2/3 complex binding activity and transcription coactivator activity. Predicted to be involved in several processes, including actin nucleation; intrinsic apoptotic signaling pathway by p53 class mediator; and regulation of transcription, DNA-templated. Located in cell leading edge. [provided by Alliance of Genome Resources, Apr 2022]\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - DNA repair\n",
      "    - regulation of transcription by RNA polymerase II\n",
      "    - cellular response to starvation\n",
      "    - Arp2/3 complex-mediated actin nucleation\n",
      "    - Arp2/3 complex-mediated actin nucleation\n",
      "    - positive regulation of apoptotic process\n",
      "    - positive regulation of apoptotic process\n",
      "    - positive regulation of DNA-templated transcription\n",
      "    - 'de novo' actin filament nucleation\n",
      "    - 'de novo' actin filament nucleation\n",
      "    - actin polymerization-dependent cell motility\n",
      "    - intrinsic apoptotic signaling pathway by p53 class mediator\n",
      "  CC:\n",
      "    - autophagosome membrane\n",
      "    - nucleus\n",
      "    - nucleus\n",
      "    - nucleoplasm\n",
      "    - cytoplasm\n",
      "    - cytoskeleton\n",
      "    - endomembrane system\n",
      "    - cell leading edge\n",
      "    - cytoplasmic vesicle\n",
      "  MF:\n",
      "    - transcription coactivator activity\n",
      "    - transcription coactivator activity\n",
      "    - actin binding\n",
      "    - protein binding\n",
      "    - microtubule binding\n",
      "    - Arp2/3 complex binding\n",
      "UniProt: {'Swiss-Prot': 'Q8N9B5'}\n",
      "\n",
      "Gene: IAPP\n",
      "Symbol: IAPP\n",
      "Name: islet amyloid polypeptide\n",
      "Summary: This gene encodes a member of the calcitonin family of peptide hormones. This hormone is released from pancreatic beta cells following food intake to regulate blood glucose levels and act as a satiation signal. Human patients with type 1 and advanced type 2 diabetes exhibit reduced levels of the encoded hormone in blood and pancreas. This protein also exhibits a bactericidal, antimicrobial activity. [provided by RefSeq, Jul 2016].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - apoptotic process\n",
      "    - signal transduction\n",
      "    - adenylate cyclase-activating G protein-coupled receptor signaling pathway\n",
      "    - cell-cell signaling\n",
      "    - negative regulation of cell population proliferation\n",
      "    - positive regulation of gene expression\n",
      "    - sensory perception of pain\n",
      "    - osteoclast differentiation\n",
      "    - negative regulation of protein-containing complex assembly\n",
      "    - protein destabilization\n",
      "    - eating behavior\n",
      "    - positive regulation of apoptotic process\n",
      "    - positive regulation of MAPK cascade\n",
      "    - bone resorption\n",
      "    - negative regulation of osteoclast differentiation\n",
      "    - negative regulation of bone resorption\n",
      "    - positive regulation of calcium-mediated signaling\n",
      "    - protein homooligomerization\n",
      "    - positive regulation of phosphatidylinositol 3-kinase/protein kinase B signal transduction\n",
      "    - positive regulation of ERK1 and ERK2 cascade\n",
      "    - amylin receptor signaling pathway\n",
      "    - amylin receptor signaling pathway\n",
      "    - amylin receptor signaling pathway\n",
      "    - positive regulation of cAMP/PKA signal transduction\n",
      "    - amylin receptor 1 signaling pathway\n",
      "    - amylin receptor 2 signaling pathway\n",
      "    - amylin receptor 3 signaling pathway\n",
      "    - negative regulation of amyloid fibril formation\n",
      "    - amyloid fibril formation\n",
      "  CC:\n",
      "    - extracellular region\n",
      "    - extracellular space\n",
      "    - extracellular space\n",
      "    - inclusion body\n",
      "    - neuronal cell body\n",
      "  MF:\n",
      "    - amyloid-beta binding\n",
      "    - signaling receptor binding\n",
      "    - hormone activity\n",
      "    - protein binding\n",
      "    - lipid binding\n",
      "    - identical protein binding\n",
      "    - identical protein binding\n",
      "    - receptor ligand activity\n",
      "    - receptor ligand activity\n",
      "UniProt: {'Swiss-Prot': 'P10997', 'TrEMBL': ['F5H1F0', 'F5H0S1', 'H0YF85']}\n",
      "\n",
      "Gene: SULT1A2\n",
      "Symbol: SULT1A2\n",
      "Name: sulfotransferase family 1A member 2\n",
      "Summary: Sulfotransferase enzymes catalyze the sulfate conjugation of many hormones, neurotransmitters, drugs, and xenobiotic compounds. These cytosolic enzymes are different in their tissue distributions and substrate specificities. The gene structure (number and length of exons) is similar among family members. This gene encodes one of two phenol sulfotransferases with thermostable enzyme activity. Two alternatively spliced variants that encode the same protein have been described. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - ethanol catabolic process\n",
      "    - catecholamine metabolic process\n",
      "    - xenobiotic metabolic process\n",
      "    - steroid metabolic process\n",
      "    - amine biosynthetic process\n",
      "    - phenol-containing compound metabolic process\n",
      "    - 3'-phosphoadenosine 5'-phosphosulfate metabolic process\n",
      "    - sulfation\n",
      "    - sulfation\n",
      "  CC:\n",
      "    - cytoplasm\n",
      "    - cytosol\n",
      "  MF:\n",
      "    - aryl sulfotransferase activity\n",
      "    - aryl sulfotransferase activity\n",
      "    - protein binding\n",
      "    - sulfotransferase activity\n",
      "    - flavonol 3-sulfotransferase activity\n",
      "UniProt: {'Swiss-Prot': 'P50226', 'TrEMBL': ['E9PKW4', 'E9PM76', 'E9PKR8']}\n",
      "\n",
      "Gene: PTGES\n",
      "Symbol: PTGES\n",
      "Name: prostaglandin E synthase\n",
      "Summary: The protein encoded by this gene is a glutathione-dependent prostaglandin E synthase. The expression of this gene has been shown to be induced by proinflammatory cytokine interleukin 1 beta (IL1B). Its expression can also be induced by tumor suppressor protein TP53, and may be involved in TP53 induced apoptosis. Knockout studies in mice suggest that this gene may contribute to the pathogenesis of collagen-induced arthritis and mediate acute pain during inflammatory responses. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - prostaglandin biosynthetic process\n",
      "    - prostaglandin biosynthetic process\n",
      "    - prostaglandin biosynthetic process\n",
      "    - prostaglandin metabolic process\n",
      "    - signal transduction\n",
      "    - cell population proliferation\n",
      "    - negative regulation of cell population proliferation\n",
      "    - sensory perception of pain\n",
      "    - regulation of fever generation\n",
      "    - positive regulation of prostaglandin secretion\n",
      "    - regulation of inflammatory response\n",
      "    - cellular oxidant detoxification\n",
      "  CC:\n",
      "    - nuclear envelope lumen\n",
      "    - endoplasmic reticulum membrane\n",
      "    - membrane\n",
      "    - membrane\n",
      "    - perinuclear region of cytoplasm\n",
      "  MF:\n",
      "    - glutathione transferase activity\n",
      "    - glutathione peroxidase activity\n",
      "    - prostaglandin-D synthase activity\n",
      "    - protein binding\n",
      "    - glutathione binding\n",
      "    - prostaglandin-E synthase activity\n",
      "    - prostaglandin-E synthase activity\n",
      "UniProt: {'Swiss-Prot': 'O14684'}\n",
      "\n",
      "Gene: UGT1A7\n",
      "Symbol: UGT1A7\n",
      "Name: UDP glucuronosyltransferase family 1 member A7\n",
      "Summary: This gene encodes a UDP-glucuronosyltransferase, an enzyme of the glucuronidation pathway that transforms small lipophilic molecules, such as steroids, bilirubin, hormones, and drugs, into water-soluble, excretable metabolites. This gene is part of a complex locus that encodes several UDP-glucuronosyltransferases. The locus includes thirteen unique alternate first exons followed by four common exons. Four of the alternate first exons are considered pseudogenes. Each of the remaining nine 5' exons may be spliced to the four common exons, resulting in nine proteins with different N-termini and identical C-termini. Each first exon encodes the substrate binding site, and is regulated by its own promoter. The enzyme encoded by this gene has moderate glucuronidase activity with phenols. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - liver development\n",
      "    - fatty acid metabolic process\n",
      "    - xenobiotic metabolic process\n",
      "    - xenobiotic metabolic process\n",
      "    - xenobiotic metabolic process\n",
      "    - estrogen metabolic process\n",
      "    - toxin catabolic process\n",
      "    - coumarin metabolic process\n",
      "    - flavonoid metabolic process\n",
      "    - xenobiotic catabolic process\n",
      "    - retinoic acid metabolic process\n",
      "    - negative regulation of fatty acid metabolic process\n",
      "    - flavone metabolic process\n",
      "  CC:\n",
      "    - endoplasmic reticulum\n",
      "    - endoplasmic reticulum\n",
      "    - endoplasmic reticulum\n",
      "    - endoplasmic reticulum membrane\n",
      "    - endoplasmic reticulum membrane\n",
      "    - endoplasmic reticulum membrane\n",
      "  MF:\n",
      "    - retinoic acid binding\n",
      "    - enzyme inhibitor activity\n",
      "    - protein kinase C binding\n",
      "    - protein binding\n",
      "    - glucuronosyltransferase activity\n",
      "    - glucuronosyltransferase activity\n",
      "    - enzyme binding\n",
      "    - enzyme binding\n",
      "    - protein homodimerization activity\n",
      "    - protein heterodimerization activity\n",
      "    - protein heterodimerization activity\n",
      "UniProt: {'Swiss-Prot': 'Q9HAW7', 'TrEMBL': ['H0Y8D7', 'Q5DSZ7']}\n",
      "\n",
      "Gene: CD6\n",
      "Symbol: CD6\n",
      "Name: CD6 molecule\n",
      "Summary: This gene encodes a protein found on the outer membrane of T-lymphocytes as well as some other immune cells. The encoded protein contains three scavenger receptor cysteine-rich (SRCR) domains and a binding site for an activated leukocyte cell adhesion molecule. The gene product is important for continuation of T cell activation. This gene may be associated with susceptibility to multiple sclerosis (PMID: 19525953, 21849685). Multiple transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Dec 2011].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - immunological synapse formation\n",
      "    - adaptive immune response\n",
      "    - acute inflammatory response to antigenic stimulus\n",
      "    - heterophilic cell-cell adhesion via plasma membrane cell adhesion molecules\n",
      "    - lipopolysaccharide-mediated signaling pathway\n",
      "    - response to lipopolysaccharide\n",
      "    - positive regulation of T cell proliferation\n",
      "    - innate immune response\n",
      "    - positive regulation of cytokine production involved in inflammatory response\n",
      "  CC:\n",
      "    - immunological synapse\n",
      "    - extracellular region\n",
      "    - plasma membrane\n",
      "    - cell surface\n",
      "    - T cell receptor complex\n",
      "  MF:\n",
      "    - lipopolysaccharide binding\n",
      "    - protein binding\n",
      "    - protein kinase binding\n",
      "    - identical protein binding\n",
      "    - lipoteichoic acid binding\n",
      "UniProt: {'Swiss-Prot': 'P30203', 'TrEMBL': ['Q8N4Q7', 'Q6AZ88', 'G5E973', 'E7ER04', 'F5H835', 'H0YGL3', 'F6UFK6']}\n",
      "\n",
      "Gene: MPP5\n",
      "Symbol: PALS1\n",
      "Name: protein associated with LIN7 1, MAGUK p55 family member\n",
      "Summary: This gene encodes a member of the p55-like subfamily of the membrane-associated guanylate kinase (MAGUK) gene superfamily. The encoded protein participates in the polarization of differentiating cells, has been shown to regulate myelinating Schwann cells (PMID: 20237282), and is one of the components of the Crumbs complex in the retina. Mice which express lower levels of the orthologous protein have retinal degeneration and impaired vision (PMID: 22114289). Multiple transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Feb 2012].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - morphogenesis of an epithelial sheet\n",
      "    - morphogenesis of an epithelial sheet\n",
      "    - gene expression\n",
      "    - establishment or maintenance of polarity of embryonic epithelium\n",
      "    - regulation of transforming growth factor beta receptor signaling pathway\n",
      "    - central nervous system neuron development\n",
      "    - cerebral cortex development\n",
      "    - peripheral nervous system myelin maintenance\n",
      "    - myelin assembly\n",
      "    - protein localization to myelin sheath abaxonal region\n",
      "    - establishment or maintenance of epithelial cell apical/basal polarity\n",
      "    - establishment or maintenance of epithelial cell apical/basal polarity\n",
      "    - generation of neurons\n",
      "    - protein localization to plasma membrane\n",
      "  CC:\n",
      "    - cytoplasm\n",
      "    - Golgi apparatus\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - adherens junction\n",
      "    - adherens junction\n",
      "    - adherens junction\n",
      "    - bicellular tight junction\n",
      "    - apical plasma membrane\n",
      "    - axon\n",
      "    - protein-containing complex\n",
      "    - endoplasmic reticulum-Golgi intermediate compartment membrane\n",
      "    - myelin sheath adaxonal region\n",
      "    - perikaryon\n",
      "    - lateral loop\n",
      "    - Schmidt-Lanterman incisure\n",
      "    - apical junction complex\n",
      "    - extracellular exosome\n",
      "  MF:\n",
      "    - protein binding\n",
      "    - ATP binding\n",
      "    - protein domain specific binding\n",
      "    - identical protein binding\n",
      "UniProt: {'Swiss-Prot': 'Q8N3R9', 'TrEMBL': ['G3V2H1', 'Q658X5', 'G3V2B0']}\n",
      "\n",
      "Gene: GPA33\n",
      "Symbol: GPA33\n",
      "Name: glycoprotein A33\n",
      "Summary: The glycoprotein encoded by this gene is a cell surface antigen that is expressed in greater than 95% of human colon cancers. The open reading frame encodes a 319-amino acid polypeptide having a putative secretory signal sequence and 3 potential glycosylation sites. The predicted mature protein has a 213-amino acid extracellular region, a single transmembrane domain, and a 62-amino acid intracellular tail. The sequence of the extracellular region contains 2 domains characteristic of the CD2 subgroup of the immunoglobulin (Ig) superfamily. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  CC:\n",
      "    - plasma membrane\n",
      "    - extracellular exosome\n",
      "  MF:\n",
      "    - protein binding\n",
      "    - signaling receptor activity\n",
      "UniProt: {'Swiss-Prot': 'Q99795', 'TrEMBL': ['A0A0J9YXH7', 'E9PMB2']}\n",
      "\n",
      "Gene: TH\n",
      "Symbol: TH\n",
      "Name: tyrosine hydroxylase\n",
      "Summary: The protein encoded by this gene is involved in the conversion of tyrosine to dopamine. It is the rate-limiting enzyme in the synthesis of catecholamines, hence plays a key role in the physiology of adrenergic neurons. Mutations in this gene have been associated with autosomal recessive Segawa syndrome. Alternatively spliced transcript variants encoding different isoforms have been noted for this gene. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - response to hypoxia\n",
      "    - response to hypoxia\n",
      "    - synaptic transmission, dopaminergic\n",
      "    - response to amphetamine\n",
      "    - heart morphogenesis\n",
      "    - dopamine biosynthetic process from tyrosine\n",
      "    - dopamine biosynthetic process from tyrosine\n",
      "    - fatty acid metabolic process\n",
      "    - sphingolipid metabolic process\n",
      "    - heart development\n",
      "    - heart development\n",
      "    - visual perception\n",
      "    - learning\n",
      "    - memory\n",
      "    - mating behavior\n",
      "    - locomotory behavior\n",
      "    - regulation of heart contraction\n",
      "    - response to light stimulus\n",
      "    - response to herbicide\n",
      "    - response to salt stress\n",
      "    - anatomical structure morphogenesis\n",
      "    - animal organ morphogenesis\n",
      "    - response to zinc ion\n",
      "    - response to activity\n",
      "    - aminergic neurotransmitter loading into synaptic vesicle\n",
      "    - glycoside metabolic process\n",
      "    - phthalate metabolic process\n",
      "    - cerebral cortex development\n",
      "    - response to nutrient levels\n",
      "    - response to estradiol\n",
      "    - response to lipopolysaccharide\n",
      "    - isoquinoline alkaloid metabolic process\n",
      "    - social behavior\n",
      "    - response to isolation stress\n",
      "    - response to immobilization stress\n",
      "    - terpene metabolic process\n",
      "    - dopamine biosynthetic process\n",
      "    - epinephrine biosynthetic process\n",
      "    - norepinephrine biosynthetic process\n",
      "    - eye photoreceptor cell development\n",
      "    - circadian sleep/wake cycle\n",
      "    - eating behavior\n",
      "    - response to peptide hormone\n",
      "    - pigmentation\n",
      "    - response to ethanol\n",
      "    - response to ethanol\n",
      "    - response to ether\n",
      "    - response to pyrethroid\n",
      "    - embryonic camera-type eye morphogenesis\n",
      "    - response to corticosterone\n",
      "    - response to electrical stimulus\n",
      "    - phytoalexin metabolic process\n",
      "    - cellular response to manganese ion\n",
      "    - cellular response to alkaloid\n",
      "    - cellular response to nicotine\n",
      "    - cellular response to glucose stimulus\n",
      "    - cellular response to growth factor stimulus\n",
      "    - cellular response to xenobiotic stimulus\n",
      "    - hyaloid vascular plexus regression\n",
      "  CC:\n",
      "    - nucleus\n",
      "    - nucleus\n",
      "    - cytoplasm\n",
      "    - cytoplasm\n",
      "    - mitochondrion\n",
      "    - smooth endoplasmic reticulum\n",
      "    - cytosol\n",
      "    - synaptic vesicle\n",
      "    - cytoplasmic side of plasma membrane\n",
      "    - axon\n",
      "    - dendrite\n",
      "    - cytoplasmic vesicle\n",
      "    - melanosome membrane\n",
      "    - neuron projection\n",
      "    - terminal bouton\n",
      "    - perikaryon\n",
      "    - perikaryon\n",
      "    - perinuclear region of cytoplasm\n",
      "  MF:\n",
      "    - tyrosine 3-monooxygenase activity\n",
      "    - tyrosine 3-monooxygenase activity\n",
      "    - protein binding\n",
      "    - ferrous iron binding\n",
      "    - ferric iron binding\n",
      "    - amino acid binding\n",
      "    - oxygen binding\n",
      "    - enzyme binding\n",
      "    - protein domain specific binding\n",
      "    - tetrahydrobiopterin binding\n",
      "    - dopamine binding\n",
      "    - identical protein binding\n",
      "UniProt: {'Swiss-Prot': 'P07101', 'TrEMBL': ['H0Y670', 'P78428', 'H0Y677', 'F8W8M5', 'E7EQI0']}\n",
      "\n",
      "Gene: MED27\n",
      "Symbol: MED27\n",
      "Name: mediator complex subunit 27\n",
      "Summary: The activation of gene transcription is a multistep process that is triggered by factors that recognize transcriptional enhancer sites in DNA. These factors work with co-activators to direct transcriptional initiation by the RNA polymerase II apparatus. The protein encoded by this gene is a subunit of the CRSP (cofactor required for SP1 activation) complex, which, along with TFIID, is required for efficient activation by SP1. This protein is also a component of other multisubunit complexes e.g. thyroid hormone receptor-(TR-) associated proteins which interact with TR and facilitate TR function on DNA templates in conjunction with initiation factors and cofactors. Alternatively spliced transcript variants encoding multiple isoforms have been observed for this gene, and a pseudogene of this gene is located on the long arm of chromosome 5. [provided by RefSeq, Dec 2011].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - regulation of transcription by RNA polymerase II\n",
      "    - regulation of transcription by RNA polymerase II\n",
      "    - transcription initiation at RNA polymerase II promoter\n",
      "    - protein ubiquitination\n",
      "    - positive regulation of transcription elongation by RNA polymerase II\n",
      "    - somatic stem cell population maintenance\n",
      "    - RNA polymerase II preinitiation complex assembly\n",
      "    - positive regulation of transcription initiation by RNA polymerase II\n",
      "  CC:\n",
      "    - ubiquitin ligase complex\n",
      "    - nucleus\n",
      "    - nucleoplasm\n",
      "    - nucleoplasm\n",
      "    - transcription regulator complex\n",
      "    - nucleolus\n",
      "    - cytosol\n",
      "    - mediator complex\n",
      "    - core mediator complex\n",
      "  MF:\n",
      "    - transcription coactivator activity\n",
      "    - transcription coactivator activity\n",
      "    - protein binding\n",
      "    - ubiquitin protein ligase activity\n",
      "UniProt: {'Swiss-Prot': 'Q6P2C8', 'TrEMBL': ['A0A494C0P0', 'A0A494C0K7']}\n",
      "\n",
      "Gene: RASAL3\n",
      "Symbol: RASAL3\n",
      "Name: RAS protein activator like 3\n",
      "Summary: This gene belongs to the Ras GTPase-activating proteins (RasGAP) family and encodes a protein with pleckstrin homology (PH), C2, and Ras GTPase-activation protein (RasGAP) domains. This protein is localized near or at the plasma membrane when expressed exogenously. Reduced expression of this gene in some cell lines resulted in increased levels of the active form of Ras (Ras-GTP), suggesting that this gene may play a role in negatively regulating the Ras signaling pathway. Alternative splicing results in multiple transcript variants encoding different isoforms. [provided by RefSeq, Jan 2017].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - negative regulation of Ras protein signal transduction\n",
      "    - positive regulation of NK T cell proliferation\n",
      "  CC:\n",
      "    - cytoplasm\n",
      "    - cytosol\n",
      "    - cell cortex\n",
      "    - membrane\n",
      "    - extracellular exosome\n",
      "    - cytoplasmic side of membrane\n",
      "  MF:\n",
      "    - GTPase activator activity\n",
      "    - protein binding\n",
      "    - identical protein binding\n",
      "UniProt: {'Swiss-Prot': 'Q86YV0', 'TrEMBL': ['V9GYD1', 'M0QX12']}\n",
      "\n",
      "Gene: CTSV\n",
      "Symbol: CTSV\n",
      "Name: cathepsin V\n",
      "Summary: The protein encoded by this gene, a member of the peptidase C1 family, is a lysosomal cysteine proteinase that may play an important role in corneal physiology. This gene is expressed in colorectal and breast carcinomas but not in normal colon, mammary gland, or peritumoral tissues, suggesting a possible role for this gene in tumor processes. Alternatively spliced variants, encoding the same protein, have been identified. [provided by RefSeq, Jan 2011].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - antigen processing and presentation of exogenous peptide antigen via MHC class II\n",
      "    - extracellular matrix disassembly\n",
      "    - proteolysis involved in protein catabolic process\n",
      "  CC:\n",
      "    - extracellular region\n",
      "    - extracellular space\n",
      "    - lysosome\n",
      "    - lysosomal lumen\n",
      "  MF:\n",
      "    - cysteine-type endopeptidase activity\n",
      "    - cysteine-type endopeptidase activity\n",
      "    - serine-type endopeptidase activity\n",
      "    - protein binding\n",
      "    - cysteine-type peptidase activity\n",
      "UniProt: {'Swiss-Prot': 'O60911', 'TrEMBL': ['A0A7P0T891', 'B2R717', 'A0A804EQJ3', 'A0A7P0T8X8', 'A0A7P0T983', 'A0A7P0Z415']}\n",
      "\n",
      "Gene: LAMA2\n",
      "Symbol: LAMA2\n",
      "Name: laminin subunit alpha 2\n",
      "Summary: Laminin, an extracellular protein, is a major component of the basement membrane. It is thought to mediate the attachment, migration, and organization of cells into tissues during embryonic development by interacting with other extracellular matrix components. It is composed of three subunits, alpha, beta, and gamma, which are bound to each other by disulfide bonds into a cross-shaped molecule. This gene encodes the alpha 2 chain, which constitutes one of the subunits of laminin 2 (merosin) and laminin 4 (s-merosin). Mutations in this gene have been identified as the cause of congenital merosin-deficient muscular dystrophy. Two transcript variants encoding different proteins have been found for this gene. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - cell adhesion\n",
      "    - axon guidance\n",
      "    - muscle organ development\n",
      "    - Schwann cell differentiation\n",
      "    - regulation of cell migration\n",
      "    - positive regulation of synaptic transmission, cholinergic\n",
      "    - maintenance of blood-brain barrier\n",
      "    - positive regulation of cell adhesion\n",
      "    - regulation of embryonic development\n",
      "    - positive regulation of muscle cell differentiation\n",
      "    - regulation of basement membrane organization\n",
      "    - positive regulation of integrin-mediated signaling pathway\n",
      "  CC:\n",
      "    - extracellular region\n",
      "    - basement membrane\n",
      "    - basement membrane\n",
      "    - neuromuscular junction\n",
      "    - sarcolemma\n",
      "    - synaptic cleft\n",
      "    - dendritic spine\n",
      "    - collagen-containing extracellular matrix\n",
      "    - protein complex involved in cell-matrix adhesion\n",
      "  MF:\n",
      "    - signaling receptor binding\n",
      "    - structural molecule activity\n",
      "    - extracellular matrix structural constituent\n",
      "UniProt: {'Swiss-Prot': 'P24043', 'TrEMBL': ['A0A8I5KQG5', 'Q59H37', 'A0A087WYF1', 'A0A087WX80', 'A0A8I5KYA2']}\n",
      "\n",
      "Gene: P2RX1\n",
      "Symbol: P2RX1\n",
      "Name: purinergic receptor P2X 1\n",
      "Summary: The protein encoded by this gene belongs to the P2X family of G-protein-coupled receptors. These proteins can form homo-and heterotimers and function as ATP-gated ion channels and mediate rapid and selective permeability to cations. This protein is primarily localized to smooth muscle where binds ATP and mediates synaptic transmission between neurons and from neurons to smooth muscle and may being responsible for sympathetic vasoconstriction in small arteries, arterioles and vas deferens. Mouse studies suggest that this receptor is essential for normal male reproductive function. This protein may also be involved in promoting apoptosis. [provided by RefSeq, Jun 2013].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - serotonin secretion by platelet\n",
      "    - regulation of vascular associated smooth muscle contraction\n",
      "    - monoatomic ion transport\n",
      "    - apoptotic process\n",
      "    - signal transduction\n",
      "    - insemination\n",
      "    - regulation of blood pressure\n",
      "    - neuronal action potential\n",
      "    - calcium-mediated signaling\n",
      "    - platelet activation\n",
      "    - response to ATP\n",
      "    - synaptic transmission, glutamatergic\n",
      "    - purinergic nucleotide receptor signaling pathway\n",
      "    - ceramide biosynthetic process\n",
      "    - excitatory postsynaptic potential\n",
      "    - calcium ion transmembrane transport\n",
      "    - regulation of presynaptic cytosolic calcium ion concentration\n",
      "    - positive regulation of calcium ion import across plasma membrane\n",
      "    - regulation of synaptic vesicle exocytosis\n",
      "  CC:\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - external side of plasma membrane\n",
      "    - secretory granule membrane\n",
      "    - protein-containing complex\n",
      "    - specific granule membrane\n",
      "    - membrane raft\n",
      "    - postsynaptic membrane\n",
      "    - presynaptic active zone membrane\n",
      "    - glutamatergic synapse\n",
      "  MF:\n",
      "    - purinergic nucleotide receptor activity\n",
      "    - extracellularly ATP-gated monoatomic cation channel activity\n",
      "    - extracellularly ATP-gated monoatomic cation channel activity\n",
      "    - monoatomic cation channel activity\n",
      "    - protein binding\n",
      "    - ATP binding\n",
      "    - identical protein binding\n",
      "    - suramin binding\n",
      "    - protein-containing complex binding\n",
      "    - ligand-gated calcium channel activity\n",
      "UniProt: {'Swiss-Prot': 'P51575', 'TrEMBL': 'I3L3H3'}\n",
      "\n",
      "Gene: NKD2\n",
      "Symbol: NKD2\n",
      "Name: NKD inhibitor of WNT signaling pathway 2\n",
      "Summary: This gene encodes a member of a family of proteins that function as negative regulators of Wnt receptor signaling through interaction with Dishevelled family members. The encoded protein participates in the delivery of transforming growth factor alpha-containing vesicles to the cell membrane. Alternatively spliced transcript variants encoding multiple isoforms have been observed for this gene. [provided by RefSeq, Aug 2012].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - exocytosis\n",
      "    - positive regulation of protein processing\n",
      "    - Wnt signaling pathway\n",
      "    - negative regulation of Wnt signaling pathway\n",
      "    - positive regulation of proteasomal ubiquitin-dependent protein catabolic process\n",
      "    - Golgi vesicle fusion to target membrane\n",
      "    - protein localization to plasma membrane\n",
      "    - negative regulation of canonical Wnt signaling pathway\n",
      "    - negative regulation of canonical Wnt signaling pathway\n",
      "    - positive regulation of protein localization to plasma membrane\n",
      "  CC:\n",
      "    - cytoplasm\n",
      "    - plasma membrane\n",
      "    - basolateral plasma membrane\n",
      "    - lateral plasma membrane\n",
      "    - cytoplasmic vesicle\n",
      "    - exocytic vesicle\n",
      "    - cell periphery\n",
      "  MF:\n",
      "    - calcium ion binding\n",
      "    - protein binding\n",
      "    - growth factor binding\n",
      "    - ubiquitin protein ligase binding\n",
      "    - myosin heavy chain binding\n",
      "    - ATPase binding\n",
      "    - molecular adaptor activity\n",
      "UniProt: {'Swiss-Prot': 'Q969F2'}\n",
      "\n",
      "Gene: HEPHL1\n",
      "Symbol: HEPHL1\n",
      "Name: hephaestin like 1\n",
      "Summary: Enables ferroxidase activity. Involved in cellular iron ion homeostasis. Predicted to be integral component of membrane. Predicted to be active in plasma membrane. [provided by Alliance of Genome Resources, Apr 2022]\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - copper ion transport\n",
      "    - intracellular iron ion homeostasis\n",
      "  CC:\n",
      "    - plasma membrane\n",
      "  MF:\n",
      "    - ferroxidase activity\n",
      "    - ferroxidase activity\n",
      "    - copper ion binding\n",
      "UniProt: {'Swiss-Prot': 'Q6MZM0'}\n",
      "\n",
      "Gene: KDELR3\n",
      "Symbol: KDELR3\n",
      "Name: KDEL endoplasmic reticulum protein retention receptor 3\n",
      "Summary: This gene encodes a member of the KDEL endoplasmic reticulum protein retention receptor family. Retention of resident soluble proteins in the lumen of the endoplasmic reticulum (ER) is achieved in both yeast and animal cells by their continual retrieval from the cis-Golgi, or a pre-Golgi compartment. Sorting of these proteins is dependent on a C-terminal tetrapeptide signal, usually lys-asp-glu-leu (KDEL) in animal cells, and his-asp-glu-leu (HDEL) in S. cerevisiae. This process is mediated by a receptor that recognizes, and binds the tetrapeptide-containing protein, and returns it to the ER. In yeast, the sorting receptor encoded by a single gene, ERD2, is a seven-transmembrane protein. Unlike yeast, several human homologs of the ERD2 gene, constituting the KDEL receptor gene family, have been described. KDELR3 was the third member of the family to be identified. Alternate splicing results in multiple transcript variants. [provided by RefSeq, Jul 2013].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - protein retention in ER lumen\n",
      "    - retrograde vesicle-mediated transport, Golgi to endoplasmic reticulum\n",
      "    - protein transport\n",
      "  CC:\n",
      "    - Golgi membrane\n",
      "    - Golgi membrane\n",
      "    - endoplasmic reticulum\n",
      "    - endoplasmic reticulum\n",
      "    - endoplasmic reticulum membrane\n",
      "    - endoplasmic reticulum membrane\n",
      "    - cis-Golgi network\n",
      "    - transport vesicle\n",
      "    - COPI-coated vesicle membrane\n",
      "  MF:\n",
      "    - KDEL sequence binding\n",
      "    - protein binding\n",
      "    - ER retention sequence binding\n",
      "UniProt: {'Swiss-Prot': 'O43731'}\n",
      "\n",
      "Gene: ABCC12\n",
      "Symbol: ABCC12\n",
      "Name: ATP binding cassette subfamily C member 12\n",
      "Summary: This gene is a member of the superfamily of ATP-binding cassette (ABC) transporters and the encoded protein contains two ATP-binding domains and 12 transmembrane regions. ABC proteins transport various molecules across extra- and intracellular membranes. ABC genes are divided into seven distinct subfamilies: ABC1, MDR/TAP, MRP, ALD, OABP, GCN20, and White. This gene is a member of the MRP subfamily which is involved in multi-drug resistance. This gene and another subfamily member are arranged head-to-tail on chromosome 16q12.1. Increased expression of this gene is associated with breast cancer. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - transmembrane transport\n",
      "  CC:\n",
      "    - endoplasmic reticulum\n",
      "    - endoplasmic reticulum membrane\n",
      "    - membrane\n",
      "  MF:\n",
      "    - ATP binding\n",
      "    - ATP hydrolysis activity\n",
      "    - ATPase-coupled transmembrane transporter activity\n",
      "    - ABC-type transporter activity\n",
      "UniProt: {'Swiss-Prot': 'Q96J65', 'TrEMBL': ['A0A8I5KPE2', 'B4DYN2', 'A0A3B3ITL4', 'A0A8I5QJT3', 'E9PHY2', 'A0A8J8YUR7']}\n",
      "\n",
      "Gene: KIF2A\n",
      "Symbol: KIF2A\n",
      "Name: kinesin family member 2A\n",
      "Summary: The protein encoded by this gene is a plus end-directed motor required for normal mitotic progression. The encoded protein is required for normal spindle activity during mitosis and is necessary for normal brain development. Several transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Sep 2011].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - microtubule cytoskeleton organization\n",
      "    - microtubule-based movement\n",
      "    - microtubule depolymerization\n",
      "    - mitotic spindle organization\n",
      "    - nervous system development\n",
      "    - cell differentiation\n",
      "    - regulation of cell migration\n",
      "    - cell division\n",
      "    - mitotic spindle assembly\n",
      "  CC:\n",
      "    - spindle pole\n",
      "    - nucleoplasm\n",
      "    - nucleolus\n",
      "    - cytoplasm\n",
      "    - cytoplasm\n",
      "    - centrosome\n",
      "    - centrosome\n",
      "    - centriole\n",
      "    - spindle\n",
      "    - cytosol\n",
      "    - cytosol\n",
      "    - kinesin complex\n",
      "    - microtubule\n",
      "    - spindle microtubule\n",
      "    - membrane\n",
      "    - nuclear body\n",
      "    - ciliary basal body\n",
      "    - mitotic spindle\n",
      "    - centriolar subdistal appendage\n",
      "  MF:\n",
      "    - cytoskeletal motor activity\n",
      "    - microtubule motor activity\n",
      "    - protein binding\n",
      "    - ATP binding\n",
      "    - microtubule binding\n",
      "    - microtubule binding\n",
      "    - ATP hydrolysis activity\n",
      "UniProt: {'Swiss-Prot': 'O00139', 'TrEMBL': ['A0A6Q8PFA6', 'B0AZS5', 'A0A6Q8PGG1', 'A0A6Q8PGH7', 'D6RD93', 'A0A6Q8PH57', 'A0A6Q8PG37', 'D6R9M0', 'H0Y8H2']}\n",
      "\n",
      "Gene: CEACAM5\n",
      "Symbol: CEACAM5\n",
      "Name: CEA cell adhesion molecule 5\n",
      "Summary: This gene encodes a cell surface glycoprotein that represents the founding member of the carcinoembryonic antigen (CEA) family of proteins. The encoded protein is used as a clinical biomarker for gastrointestinal cancers and may promote tumor development through its role as a cell adhesion molecule. Additionally, the encoded protein may regulate differentiation, apoptosis, and cell polarity. This gene is present in a CEA family gene cluster on chromosome 19. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Jul 2015].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - apoptotic process\n",
      "    - homophilic cell adhesion via plasma membrane adhesion molecules\n",
      "    - heterophilic cell-cell adhesion via plasma membrane cell adhesion molecules\n",
      "    - heterophilic cell-cell adhesion via plasma membrane cell adhesion molecules\n",
      "    - negative regulation of myotube differentiation\n",
      "    - homotypic cell-cell adhesion\n",
      "    - negative regulation of apoptotic process\n",
      "    - negative regulation of anoikis\n",
      "  CC:\n",
      "    - extracellular region\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - cell surface\n",
      "    - cell surface\n",
      "    - cell surface\n",
      "    - membrane\n",
      "    - basolateral plasma membrane\n",
      "    - apical plasma membrane\n",
      "    - extracellular exosome\n",
      "    - side of membrane\n",
      "  MF:\n",
      "    - protein binding\n",
      "    - GPI anchor binding\n",
      "    - identical protein binding\n",
      "    - identical protein binding\n",
      "    - protein homodimerization activity\n",
      "UniProt: {'Swiss-Prot': 'P06731', 'TrEMBL': ['Q53G30', 'A0A024R0K5', 'M0R3J1', 'M0QX98', 'A0A087WYX0']}\n",
      "\n",
      "Gene: NVL\n",
      "Symbol: NVL\n",
      "Name: nuclear VCP like\n",
      "Summary: This gene encodes a member of the AAA (ATPases associated with diverse cellular activities) superfamily. Multiple transcript variants encoding different isoforms have been found for this gene. Two encoded proteins, described as major and minor isoforms, have been localized to distinct regions of the nucleus. The largest encoded protein (major isoform) has been localized to the nucleolus and shown to participate in ribosome biosynthesis (PMID: 15469983, 16782053), while the minor isoform has been localized to the nucleoplasmin. [provided by RefSeq, Aug 2011].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - rRNA processing\n",
      "    - positive regulation of protein binding\n",
      "    - positive regulation of telomere maintenance\n",
      "    - ribosome biogenesis\n",
      "    - ribosome biogenesis\n",
      "    - ribosomal large subunit biogenesis\n",
      "    - ribosomal large subunit biogenesis\n",
      "    - regulation of protein localization to nucleolus\n",
      "    - telomerase holoenzyme complex assembly\n",
      "  CC:\n",
      "    - nuclear exosome (RNase complex)\n",
      "    - nucleus\n",
      "    - nucleus\n",
      "    - nucleoplasm\n",
      "    - telomerase holoenzyme complex\n",
      "    - nucleolus\n",
      "    - nucleolus\n",
      "    - membrane\n",
      "  MF:\n",
      "    - RNA binding\n",
      "    - protein binding\n",
      "    - ATP binding\n",
      "    - ATP hydrolysis activity\n",
      "    - preribosome binding\n",
      "    - preribosome binding\n",
      "UniProt: {'Swiss-Prot': 'O15381', 'TrEMBL': ['F8WF01', 'B4DF43', 'C9J6P7', 'E7EWK7', 'E9PGD8', 'E9PH71', 'A0A087WV72', 'C9JP83']}\n",
      "\n",
      "Gene: NR1I2\n",
      "Symbol: NR1I2\n",
      "Name: nuclear receptor subfamily 1 group I member 2\n",
      "Summary: This gene product belongs to the nuclear receptor superfamily, members of which are transcription factors characterized by a ligand-binding domain and a DNA-binding domain. The encoded protein is a transcriptional regulator of the cytochrome P450 gene CYP3A4, binding to the response element of the CYP3A4 promoter as a heterodimer with the 9-cis retinoic acid receptor RXR. It is activated by a range of compounds that induce CYP3A4, including dexamethasone and rifampicin. Several alternatively spliced transcripts encoding different isoforms, some of which use non-AUG (CUG) translation initiation codon, have been described for this gene. Additional transcript variants exist, however, they have not been fully characterized. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - negative regulation of transcription by RNA polymerase II\n",
      "    - regulation of DNA-templated transcription\n",
      "    - xenobiotic metabolic process\n",
      "    - signal transduction\n",
      "    - steroid metabolic process\n",
      "    - positive regulation of gene expression\n",
      "    - cell differentiation\n",
      "    - intracellular receptor signaling pathway\n",
      "    - xenobiotic catabolic process\n",
      "    - xenobiotic transport\n",
      "    - negative regulation of DNA-templated transcription\n",
      "    - positive regulation of DNA-templated transcription\n",
      "    - positive regulation of transcription by RNA polymerase II\n",
      "    - positive regulation of transcription by RNA polymerase II\n",
      "    - positive regulation of transcription by RNA polymerase II\n",
      "    - intestinal epithelial structure maintenance\n",
      "    - cellular response to molecule of bacterial origin\n",
      "  CC:\n",
      "    - chromatin\n",
      "    - nucleus\n",
      "    - nucleus\n",
      "    - nucleoplasm\n",
      "    - nucleoplasm\n",
      "    - transcription regulator complex\n",
      "    - nuclear body\n",
      "    - intermediate filament cytoskeleton\n",
      "  MF:\n",
      "    - RNA polymerase II transcription regulatory region sequence-specific DNA binding\n",
      "    - RNA polymerase II cis-regulatory region sequence-specific DNA binding\n",
      "    - DNA-binding transcription factor activity, RNA polymerase II-specific\n",
      "    - DNA-binding transcription activator activity, RNA polymerase II-specific\n",
      "    - nuclear receptor activity\n",
      "    - nuclear receptor activity\n",
      "    - protein binding\n",
      "    - zinc ion binding\n",
      "    - nuclear receptor binding\n",
      "    - sequence-specific double-stranded DNA binding\n",
      "UniProt: {'Swiss-Prot': 'O75469', 'TrEMBL': 'F1D8P9'}\n",
      "\n",
      "Gene: SULT1B1\n",
      "Symbol: SULT1B1\n",
      "Name: sulfotransferase family 1B member 1\n",
      "Summary: Sulfotransferase enzymes catalyze the sulfate conjugation of many hormones, neurotransmitters, drugs, and xenobiotic compounds. These cytosolic enzymes are different in their tissue distributions and substrate specificities. The gene structure (number and length of exons) is similar among family members. However, the total genomic length of this gene is greater than that of other SULT1 genes. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - ethanol catabolic process\n",
      "    - biogenic amine metabolic process\n",
      "    - xenobiotic metabolic process\n",
      "    - flavonoid metabolic process\n",
      "    - phenol-containing compound metabolic process\n",
      "    - epithelial cell differentiation\n",
      "    - thyroid hormone metabolic process\n",
      "    - 3'-phosphoadenosine 5'-phosphosulfate metabolic process\n",
      "    - sulfation\n",
      "    - sulfation\n",
      "    - sulfation\n",
      "  CC:\n",
      "    - cytoplasm\n",
      "    - cytosol\n",
      "    - cytosol\n",
      "  MF:\n",
      "    - aryl sulfotransferase activity\n",
      "    - aryl sulfotransferase activity\n",
      "    - protein binding\n",
      "    - sulfotransferase activity\n",
      "UniProt: {'Swiss-Prot': 'O43704', 'TrEMBL': ['D6RD70', 'D6RIA8']}\n",
      "\n",
      "Gene: FGG\n",
      "Symbol: FGG\n",
      "Name: fibrinogen gamma chain\n",
      "Summary: The protein encoded by this gene is the gamma component of fibrinogen, a blood-borne glycoprotein comprised of three pairs of nonidentical polypeptide chains. Following vascular injury, fibrinogen is cleaved by thrombin to form fibrin which is the most abundant component of blood clots. In addition, various cleavage products of fibrinogen and fibrin regulate cell adhesion and spreading, display vasoconstrictor and chemotactic activities, and are mitogens for several cell types. Mutations in this gene lead to several disorders, including dysfibrinogenemia, hypofibrinogenemia and thrombophilia. Alternative splicing results in transcript variants encoding different isoforms. [provided by RefSeq, Aug 2015].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - cell-matrix adhesion\n",
      "    - cell-matrix adhesion\n",
      "    - protein secretion\n",
      "    - plasminogen activation\n",
      "    - positive regulation of heterotypic cell-cell adhesion\n",
      "    - platelet maturation\n",
      "    - fibrinolysis\n",
      "    - positive regulation of vasoconstriction\n",
      "    - positive regulation of exocytosis\n",
      "    - positive regulation of protein secretion\n",
      "    - protein polymerization\n",
      "    - response to calcium ion\n",
      "    - protein-containing complex assembly\n",
      "    - positive regulation of ERK1 and ERK2 cascade\n",
      "    - platelet aggregation\n",
      "    - platelet aggregation\n",
      "    - cellular response to interleukin-1\n",
      "    - cellular response to interleukin-6\n",
      "    - blood coagulation, fibrin clot formation\n",
      "    - blood coagulation, fibrin clot formation\n",
      "    - positive regulation of peptide hormone secretion\n",
      "    - positive regulation of substrate adhesion-dependent cell spreading\n",
      "    - negative regulation of extrinsic apoptotic signaling pathway via death domain receptors\n",
      "    - negative regulation of endothelial cell apoptotic process\n",
      "  CC:\n",
      "    - extracellular region\n",
      "    - fibrinogen complex\n",
      "    - extracellular space\n",
      "    - extracellular space\n",
      "    - extracellular space\n",
      "    - endoplasmic reticulum lumen\n",
      "    - plasma membrane\n",
      "    - external side of plasma membrane\n",
      "    - cell surface\n",
      "    - platelet alpha granule\n",
      "    - platelet alpha granule lumen\n",
      "    - collagen-containing extracellular matrix\n",
      "    - collagen-containing extracellular matrix\n",
      "    - extracellular exosome\n",
      "    - blood microparticle\n",
      "  MF:\n",
      "    - signaling receptor binding\n",
      "    - signaling receptor binding\n",
      "    - structural molecule activity\n",
      "    - extracellular matrix structural constituent\n",
      "    - extracellular matrix structural constituent\n",
      "    - protein binding\n",
      "    - identical protein binding\n",
      "    - metal ion binding\n",
      "    - cell adhesion molecule binding\n",
      "UniProt: {'Swiss-Prot': 'P02679', 'TrEMBL': ['C9JU00', 'A0A140VJJ6', 'C9JEU5', 'C9JPQ9', 'C9JC84']}\n",
      "\n",
      "Gene: CYP1B1\n",
      "Symbol: CYP1B1\n",
      "Name: cytochrome P450 family 1 subfamily B member 1\n",
      "Summary: This gene encodes a member of the cytochrome P450 superfamily of enzymes. The cytochrome P450 proteins are monooxygenases which catalyze many reactions involved in drug metabolism and synthesis of cholesterol, steroids and other lipids. The enzyme encoded by this gene localizes to the endoplasmic reticulum and metabolizes procarcinogens such as polycyclic aromatic hydrocarbons and 17beta-estradiol. Mutations in this gene have been associated with primary congenital glaucoma; therefore it is thought that the enzyme also metabolizes a signaling molecule involved in eye development, possibly a steroid. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - angiogenesis\n",
      "    - trabecular meshwork development\n",
      "    - DNA modification\n",
      "    - xenobiotic metabolic process\n",
      "    - nitric oxide biosynthetic process\n",
      "    - cell adhesion\n",
      "    - response to nutrient\n",
      "    - steroid metabolic process\n",
      "    - estrogen metabolic process\n",
      "    - negative regulation of cell population proliferation\n",
      "    - male gonad development\n",
      "    - intrinsic apoptotic signaling pathway in response to oxidative stress\n",
      "    - toxin metabolic process\n",
      "    - positive regulation of vascular endothelial growth factor production\n",
      "    - positive regulation of smooth muscle cell migration\n",
      "    - sterol metabolic process\n",
      "    - arachidonate metabolic process\n",
      "    - epoxygenase P450 pathway\n",
      "    - collagen fibril organization\n",
      "    - adrenal gland development\n",
      "    - negative regulation of cell migration\n",
      "    - negative regulation of NF-kappaB transcription factor activity\n",
      "    - response to follicle-stimulating hormone\n",
      "    - response to estradiol\n",
      "    - negative regulation of cell adhesion mediated by integrin\n",
      "    - benzene-containing compound metabolic process\n",
      "    - retinol metabolic process\n",
      "    - retinol metabolic process\n",
      "    - retinal metabolic process\n",
      "    - positive regulation of apoptotic process\n",
      "    - blood vessel endothelial cell migration\n",
      "    - endothelial cell migration\n",
      "    - estrous cycle\n",
      "    - positive regulation of translation\n",
      "    - positive regulation of angiogenesis\n",
      "    - positive regulation of receptor signaling pathway via JAK-STAT\n",
      "    - membrane lipid catabolic process\n",
      "    - response to arsenic-containing substance\n",
      "    - blood vessel morphogenesis\n",
      "    - retinal blood vessel morphogenesis\n",
      "    - ganglion development\n",
      "    - cellular response to hydrogen peroxide\n",
      "    - cellular response to cAMP\n",
      "    - cellular response to tumor necrosis factor\n",
      "    - cellular response to luteinizing hormone stimulus\n",
      "    - cellular response to cortisol stimulus\n",
      "    - cellular response to progesterone stimulus\n",
      "    - response to dexamethasone\n",
      "    - endothelial cell-cell adhesion\n",
      "    - response to indole-3-methanol\n",
      "    - cellular response to toxic substance\n",
      "    - omega-hydroxylase P450 pathway\n",
      "    - response to 3-methylcholanthrene\n",
      "    - regulation of reactive oxygen species metabolic process\n",
      "    - positive regulation of reactive oxygen species metabolic process\n",
      "    - positive regulation of DNA biosynthetic process\n",
      "  CC:\n",
      "    - mitochondrion\n",
      "    - endoplasmic reticulum membrane\n",
      "    - intracellular membrane-bounded organelle\n",
      "  MF:\n",
      "    - monooxygenase activity\n",
      "    - monooxygenase activity\n",
      "    - monooxygenase activity\n",
      "    - iron ion binding\n",
      "    - protein binding\n",
      "    - oxidoreductase activity, acting on paired donors, with incorporation or reduction of molecular oxygen, NAD(P)H as one donor, and incorporation of one atom of oxygen\n",
      "    - oxidoreductase activity, acting on paired donors, with incorporation or reduction of molecular oxygen, reduced flavin or flavoprotein as one donor, and incorporation of one atom of oxygen\n",
      "    - heme binding\n",
      "    - testosterone 6-beta-hydroxylase activity\n",
      "    - estrogen 16-alpha-hydroxylase activity\n",
      "    - estrogen 2-hydroxylase activity\n",
      "    - hydroperoxy icosatetraenoate dehydratase activity\n",
      "UniProt: {'Swiss-Prot': 'Q16678', 'TrEMBL': ['A0A087WW26', 'Q53TK1']}\n",
      "\n",
      "Gene: DAGLA\n",
      "Symbol: DAGLA\n",
      "Name: diacylglycerol lipase alpha\n",
      "Summary: This gene encodes a diacylglycerol lipase. The encoded enzyme is involved in the biosynthesis of the endocannabinoid 2-arachidonoyl-glycerol.[provided by RefSeq, Nov 2010].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - monoacylglycerol biosynthetic process\n",
      "    - neuroblast proliferation\n",
      "    - arachidonate metabolic process\n",
      "    - arachidonate metabolic process\n",
      "    - arachidonate metabolic process\n",
      "    - diacylglycerol catabolic process\n",
      "    - diacylglycerol catabolic process\n",
      "    - diacylglycerol catabolic process\n",
      "    - retrograde trans-synaptic signaling by endocannabinoid\n",
      "    - retrograde trans-synaptic signaling by endocannabinoid\n",
      "    - retrograde trans-synaptic signaling by endocannabinoid\n",
      "    - regulation of neuroinflammatory response\n",
      "    - cannabinoid biosynthetic process\n",
      "  CC:\n",
      "    - cytoplasm\n",
      "    - plasma membrane\n",
      "    - early endosome membrane\n",
      "    - dendrite membrane\n",
      "    - dendrite membrane\n",
      "    - dendritic spine membrane\n",
      "    - varicosity\n",
      "    - postsynaptic membrane\n",
      "    - postsynaptic density membrane\n",
      "  MF:\n",
      "    - lipoprotein lipase activity\n",
      "    - protein binding\n",
      "    - metal ion binding\n",
      "    - monoacylglycerol lipase activity\n",
      "UniProt: {'Swiss-Prot': 'Q9Y4D2', 'TrEMBL': 'F5GY58'}\n",
      "\n",
      "Gene: KIAA0319\n",
      "Symbol: KIAA0319\n",
      "Name: KIAA0319\n",
      "Summary: This gene encodes a transmembrane protein that contains a large extracellular domain with multiple polycystic kidney disease (PKD) domains. The encoded protein may play a role in the development of the cerebral cortex by regulating neuronal migration and cell adhesion. Single nucleotide polymorphisms in this gene are associated with dyslexia. Alternatively spliced transcript variants encoding multiple isoforms have been observed for this gene. [provided by RefSeq, Nov 2011].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - neuron migration\n",
      "    - neuron migration\n",
      "    - sensory perception of sound\n",
      "    - thalamus development\n",
      "    - central nervous system neuron development\n",
      "    - negative regulation of axon extension\n",
      "    - multicellular organismal response to stress\n",
      "    - vocal learning\n",
      "    - negative regulation of axon extension involved in regeneration\n",
      "    - positive regulation of SMAD protein signal transduction\n",
      "    - positive regulation of SMAD protein signal transduction\n",
      "    - negative regulation of dendrite development\n",
      "  CC:\n",
      "    - early endosome\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - clathrin-coated endocytic vesicle membrane\n",
      "    - cytoplasmic vesicle\n",
      "    - early endosome membrane\n",
      "    - intracellular membrane-bounded organelle\n",
      "  MF:\n",
      "    - protein binding\n",
      "UniProt: {'Swiss-Prot': 'Q5VV43', 'TrEMBL': 'A0A087X0U9'}\n",
      "\n",
      "Gene: SLC22A7\n",
      "Symbol: SLC22A7\n",
      "Name: solute carrier family 22 member 7\n",
      "Summary: The protein encoded by this gene is involved in the sodium-independent transport and excretion of organic anions, some of which are potentially toxic. The encoded protein is an integral membrane protein and appears to be localized to the basolateral membrane of the kidney. Alternatively spliced transcript variants encoding different isoforms have been described. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - xenobiotic metabolic process\n",
      "    - monoatomic ion transport\n",
      "    - organic anion transport\n",
      "    - prostaglandin transport\n",
      "    - alpha-ketoglutarate transport\n",
      "    - transmembrane transport\n",
      "  CC:\n",
      "    - cytosol\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - basal plasma membrane\n",
      "    - membrane\n",
      "    - basolateral plasma membrane\n",
      "    - apical plasma membrane\n",
      "  MF:\n",
      "    - protein binding\n",
      "    - organic anion transmembrane transporter activity\n",
      "    - organic anion transmembrane transporter activity\n",
      "    - organic anion transmembrane transporter activity\n",
      "    - prostaglandin transmembrane transporter activity\n",
      "    - alpha-ketoglutarate transmembrane transporter activity\n",
      "    - sodium-independent organic anion transmembrane transporter activity\n",
      "    - sodium-independent organic anion transmembrane transporter activity\n",
      "    - transmembrane transporter activity\n",
      "UniProt: {'Swiss-Prot': 'Q9Y694', 'TrEMBL': ['Q5T047', 'Q5T051', 'X6RDF7']}\n",
      "\n",
      "Gene: ALDH9A1\n",
      "Symbol: ALDH9A1\n",
      "Name: aldehyde dehydrogenase 9 family member A1\n",
      "Summary: This protein belongs to the aldehyde dehydrogenase family of proteins. It has a high activity for oxidation of gamma-aminobutyraldehyde and other amino aldehydes. The enzyme catalyzes the dehydrogenation of gamma-aminobutyraldehyde to gamma-aminobutyric acid (GABA). This isozyme is a tetramer of identical 54-kD subunits. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - aldehyde metabolic process\n",
      "    - carnitine biosynthetic process\n",
      "    - carnitine biosynthetic process\n",
      "    - protein homotetramerization\n",
      "  CC:\n",
      "    - cytoplasm\n",
      "    - cytoplasm\n",
      "    - mitochondrion\n",
      "    - cytosol\n",
      "    - extracellular exosome\n",
      "  MF:\n",
      "    - aldehyde dehydrogenase (NAD+) activity\n",
      "    - formaldehyde dehydrogenase (NAD+) activity\n",
      "    - aminobutyraldehyde dehydrogenase (NAD+) activity\n",
      "    - aminobutyraldehyde dehydrogenase (NAD+) activity\n",
      "    - small molecule binding\n",
      "    - 4-trimethylammoniobutyraldehyde dehydrogenase activity\n",
      "    - 4-trimethylammoniobutyraldehyde dehydrogenase activity\n",
      "    - 4-trimethylammoniobutyraldehyde dehydrogenase activity\n",
      "UniProt: {'Swiss-Prot': 'P49189'}\n",
      "\n",
      "Gene: CYP11A1\n",
      "Symbol: CYP11A1\n",
      "Name: cytochrome P450 family 11 subfamily A member 1\n",
      "Summary: This gene encodes a member of the cytochrome P450 superfamily of enzymes. The cytochrome P450 proteins are monooxygenases which catalyze many reactions involved in drug metabolism and synthesis of cholesterol, steroids and other lipids. This protein localizes to the mitochondrial inner membrane and catalyzes the conversion of cholesterol to pregnenolone, the first and rate-limiting step in the synthesis of the steroid hormones. Two transcript variants encoding different isoforms have been found for this gene. The cellular location of the smaller isoform is unclear since it lacks the mitochondrial-targeting transit peptide. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - C21-steroid hormone biosynthetic process\n",
      "    - C21-steroid hormone biosynthetic process\n",
      "    - C21-steroid hormone biosynthetic process\n",
      "    - glucocorticoid biosynthetic process\n",
      "    - cholesterol metabolic process\n",
      "    - cholesterol metabolic process\n",
      "    - cholesterol metabolic process\n",
      "    - cholesterol metabolic process\n",
      "    - cholesterol metabolic process\n",
      "    - sterol metabolic process\n",
      "    - cortisol metabolic process\n",
      "    - vitamin D metabolic process\n",
      "    - cellular response to peptide hormone stimulus\n",
      "  CC:\n",
      "    - mitochondrion\n",
      "    - mitochondrion\n",
      "    - mitochondrial inner membrane\n",
      "    - mitochondrial inner membrane\n",
      "    - mitochondrial matrix\n",
      "  MF:\n",
      "    - iron ion binding\n",
      "    - protein binding\n",
      "    - cholesterol monooxygenase (side-chain-cleaving) activity\n",
      "    - cholesterol monooxygenase (side-chain-cleaving) activity\n",
      "    - cholesterol monooxygenase (side-chain-cleaving) activity\n",
      "    - cholesterol monooxygenase (side-chain-cleaving) activity\n",
      "    - heme binding\n",
      "UniProt: {'Swiss-Prot': 'P05108', 'TrEMBL': ['C9JPU9', 'A0A0S2Z3R3', 'H3BS93', 'E7EPP8', 'H3BSZ1', 'C9JXV4']}\n",
      "\n",
      "Gene: DPM1\n",
      "Symbol: DPM1\n",
      "Name: dolichyl-phosphate mannosyltransferase subunit 1, catalytic\n",
      "Summary: Dolichol-phosphate mannose (Dol-P-Man) serves as a donor of mannosyl residues on the lumenal side of the endoplasmic reticulum (ER). Lack of Dol-P-Man results in defective surface expression of GPI-anchored proteins. Dol-P-Man is synthesized from GDP-mannose and dolichol-phosphate on the cytosolic side of the ER by the enzyme dolichyl-phosphate mannosyltransferase. Human DPM1 lacks a carboxy-terminal transmembrane domain and signal sequence and is regulated by DPM2. Mutations in this gene are associated with congenital disorder of glycosylation type Ie. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Nov 2015].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - dolichol-linked oligosaccharide biosynthetic process\n",
      "    - GPI anchor biosynthetic process\n",
      "    - GPI anchor biosynthetic process\n",
      "    - GPI anchor biosynthetic process\n",
      "    - protein N-linked glycosylation via asparagine\n",
      "    - dolichol metabolic process\n",
      "    - dolichol metabolic process\n",
      "    - GDP-mannose metabolic process\n",
      "    - protein mannosylation\n",
      "    - protein O-linked mannosylation\n",
      "    - protein O-linked mannosylation\n",
      "    - response to oxygen levels\n",
      "    - dolichol phosphate mannose biosynthetic process\n",
      "  CC:\n",
      "    - nucleus\n",
      "    - endoplasmic reticulum\n",
      "    - endoplasmic reticulum membrane\n",
      "    - endoplasmic reticulum membrane\n",
      "    - endoplasmic reticulum membrane\n",
      "    - endoplasmic reticulum membrane\n",
      "    - membrane\n",
      "    - membrane\n",
      "    - dolichol-phosphate-mannose synthase complex\n",
      "    - dolichol-phosphate-mannose synthase complex\n",
      "  MF:\n",
      "    - dolichyl-phosphate-mannose-protein mannosyltransferase activity\n",
      "    - dolichyl-phosphate beta-D-mannosyltransferase activity\n",
      "    - dolichyl-phosphate beta-D-mannosyltransferase activity\n",
      "    - dolichyl-phosphate beta-D-mannosyltransferase activity\n",
      "    - protein binding\n",
      "    - D-mannose binding\n",
      "    - alcohol binding\n",
      "    - metal ion binding\n",
      "UniProt: {'Swiss-Prot': 'O60762', 'TrEMBL': ['A0A0S2Z4Y5', 'Q5QPK2', 'H0Y368', 'A0A804HIK9', 'Q5QPJ9', 'A0A804HIB3', 'A0A804HJ93']}\n",
      "\n",
      "Gene: HLA-DRA\n",
      "Symbol: HLA-DRA\n",
      "Name: major histocompatibility complex, class II, DR alpha\n",
      "Summary: HLA-DRA is one of the HLA class II alpha chain paralogues. This class II molecule is a heterodimer consisting of an alpha and a beta chain, both anchored in the membrane. This molecule is expressed on the surface of various antigen presenting cells such as B lymphocytes, dendritic cells, and monocytes/macrophages, and plays a central role in the immune system and response by presenting peptides derived from extracellular proteins, in particular, pathogen-derived peptides to T cells. The alpha chain is approximately 33-35 kDa and its gene contains 5 exons. Exon 1 encodes the leader peptide, exons 2 and 3 encode the two extracellular domains, and exon 4 encodes the transmembrane domain and the cytoplasmic tail. DRA does not have polymorphisms in the peptide binding part and acts as the sole alpha chain for DRB1, DRB3, DRB4 and DRB5. [provided by RefSeq, Aug 2020].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - positive regulation of T cell mediated cytotoxicity\n",
      "    - adaptive immune response\n",
      "    - myeloid dendritic cell antigen processing and presentation\n",
      "    - antigen processing and presentation of endogenous peptide antigen via MHC class II\n",
      "    - peptide antigen assembly with MHC class II protein complex\n",
      "    - peptide antigen assembly with MHC class II protein complex\n",
      "    - antigen processing and presentation of peptide or polysaccharide antigen via MHC class II\n",
      "    - immune response\n",
      "    - antigen processing and presentation of exogenous peptide antigen via MHC class II\n",
      "    - antigen processing and presentation of exogenous peptide antigen via MHC class II\n",
      "    - positive regulation of CD4-positive, CD25-positive, alpha-beta regulatory T cell differentiation\n",
      "    - positive regulation of memory T cell differentiation\n",
      "    - regulation of T-helper cell differentiation\n",
      "    - positive regulation of immune response\n",
      "    - positive regulation of T cell activation\n",
      "    - cognition\n",
      "    - positive regulation of CD4-positive, alpha-beta T cell activation\n",
      "  CC:\n",
      "    - Golgi membrane\n",
      "    - immunological synapse\n",
      "    - lysosome\n",
      "    - lysosomal membrane\n",
      "    - lysosomal membrane\n",
      "    - lysosomal membrane\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - cell surface\n",
      "    - ER to Golgi transport vesicle membrane\n",
      "    - transport vesicle membrane\n",
      "    - endocytic vesicle membrane\n",
      "    - clathrin-coated endocytic vesicle membrane\n",
      "    - early endosome membrane\n",
      "    - late endosome membrane\n",
      "    - late endosome membrane\n",
      "    - trans-Golgi network membrane\n",
      "    - MHC class II protein complex\n",
      "    - MHC class II protein complex\n",
      "    - MHC class II protein complex\n",
      "    - extracellular exosome\n",
      "    - lumenal side of endoplasmic reticulum membrane\n",
      "    - autolysosome membrane\n",
      "  MF:\n",
      "    - protein binding\n",
      "    - MHC class II protein complex binding\n",
      "    - MHC class II protein complex binding\n",
      "    - polysaccharide binding\n",
      "    - MHC class II receptor activity\n",
      "    - peptide antigen binding\n",
      "    - peptide antigen binding\n",
      "    - T cell receptor binding\n",
      "UniProt: {'Swiss-Prot': 'P01903', 'TrEMBL': ['Q30118', 'A0A0G2JMH6', 'A0A0G2JH46']}\n",
      "\n",
      "Gene: HSD3B2\n",
      "Symbol: HSD3B2\n",
      "Name: hydroxy-delta-5-steroid dehydrogenase, 3 beta- and steroid delta-isomerase 2\n",
      "Summary: The protein encoded by this gene is a bifunctional enzyme that catalyzes the oxidative conversion of delta(5)-ene-3-beta-hydroxy steroid, and the oxidative conversion of ketosteroids. It plays a crucial role in the biosynthesis of all classes of hormonal steroids. This gene is predominantly expressed in the adrenals and the gonads. Mutations in this gene are associated with 3-beta-hydroxysteroid dehydrogenase, type II, deficiency. Alternatively spliced transcript variants have been found for this gene. [provided by RefSeq, Oct 2009].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - steroid biosynthetic process\n",
      "    - steroid biosynthetic process\n",
      "    - steroid biosynthetic process\n",
      "    - androgen biosynthetic process\n",
      "    - C21-steroid hormone metabolic process\n",
      "  CC:\n",
      "    - nucleolus\n",
      "    - cytoplasm\n",
      "    - mitochondrial inner membrane\n",
      "    - mitochondrial intermembrane space\n",
      "    - endoplasmic reticulum\n",
      "    - endoplasmic reticulum\n",
      "    - endoplasmic reticulum\n",
      "    - endoplasmic reticulum membrane\n",
      "    - cilium\n",
      "    - microtubule cytoskeleton\n",
      "    - membrane\n",
      "    - smooth endoplasmic reticulum membrane\n",
      "    - mitochondrial membrane\n",
      "    - intracellular membrane-bounded organelle\n",
      "    - intercellular bridge\n",
      "  MF:\n",
      "    - 3-beta-hydroxy-Delta5-steroid dehydrogenase (NAD+) activity\n",
      "    - 3-beta-hydroxy-Delta5-steroid dehydrogenase (NAD+) activity\n",
      "    - steroid Delta-isomerase activity\n",
      "    - steroid Delta-isomerase activity\n",
      "    - protein binding\n",
      "    - oxidoreductase activity, acting on the CH-OH group of donors, NAD or NADP as acceptor\n",
      "UniProt: {'Swiss-Prot': 'P26439', 'TrEMBL': 'Q5QP01'}\n",
      "\n",
      "Gene: DHODH\n",
      "Symbol: DHODH\n",
      "Name: dihydroorotate dehydrogenase (quinone)\n",
      "Summary: The protein encoded by this gene catalyzes the fourth enzymatic step, the ubiquinone-mediated oxidation of dihydroorotate to orotate, in de novo pyrimidine biosynthesis. This protein is a mitochondrial protein located on the outer surface of the inner mitochondrial membrane. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - 'de novo' pyrimidine nucleobase biosynthetic process\n",
      "    - UDP biosynthetic process\n",
      "    - pyrimidine ribonucleotide biosynthetic process\n",
      "    - 'de novo' UMP biosynthetic process\n",
      "  CC:\n",
      "    - nucleoplasm\n",
      "    - mitochondrion\n",
      "    - mitochondrion\n",
      "    - mitochondrial inner membrane\n",
      "    - mitochondrial inner membrane\n",
      "    - cytosol\n",
      "  MF:\n",
      "    - dihydroorotase activity\n",
      "    - dihydroorotate dehydrogenase activity\n",
      "    - protein binding\n",
      "    - dihydroorotate dehydrogenase (quinone) activity\n",
      "UniProt: {'Swiss-Prot': 'Q02127', 'TrEMBL': ['I3NI32', 'J3QRQ3', 'I3L449', 'J3QRJ4']}\n",
      "\n",
      "Gene: IGF2\n",
      "Symbol: IGF2\n",
      "Name: insulin like growth factor 2\n",
      "Summary: This gene encodes a member of the insulin family of polypeptide growth factors, which are involved in development and growth. It is an imprinted gene, expressed only from the paternal allele, and epigenetic changes at this locus are associated with Wilms tumour, Beckwith-Wiedemann syndrome, rhabdomyosarcoma, and Silver-Russell syndrome. A read-through INS-IGF2 gene exists, whose 5' region overlaps the INS gene and the 3' region overlaps this gene. Alternatively spliced transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Oct 2010].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - negative regulation of transcription by RNA polymerase II\n",
      "    - osteoblast differentiation\n",
      "    - in utero embryonic development\n",
      "    - embryonic placenta development\n",
      "    - glucose metabolic process\n",
      "    - regulation of DNA-templated transcription\n",
      "    - positive regulation of cell population proliferation\n",
      "    - positive regulation of cell population proliferation\n",
      "    - insulin receptor signaling pathway\n",
      "    - animal organ morphogenesis\n",
      "    - exocrine pancreas development\n",
      "    - positive regulation of multicellular organism growth\n",
      "    - positive regulation of activated T cell proliferation\n",
      "    - positive regulation of activated T cell proliferation\n",
      "    - positive regulation of MAPK cascade\n",
      "    - positive regulation of MAPK cascade\n",
      "    - positive regulation of glycogen biosynthetic process\n",
      "    - positive regulation of mitotic nuclear division\n",
      "    - positive regulation of transcription by RNA polymerase II\n",
      "    - positive regulation of organ growth\n",
      "    - positive regulation of insulin receptor signaling pathway\n",
      "    - positive regulation of insulin receptor signaling pathway\n",
      "    - insulin-like growth factor receptor signaling pathway\n",
      "    - positive regulation of skeletal muscle tissue growth\n",
      "    - striated muscle cell differentiation\n",
      "    - regulation of muscle cell differentiation\n",
      "    - regulation of muscle cell differentiation\n",
      "    - negative regulation of muscle cell differentiation\n",
      "    - positive regulation of cell division\n",
      "    - positive regulation of phosphatidylinositol 3-kinase/protein kinase B signal transduction\n",
      "    - positive regulation of phosphatidylinositol 3-kinase/protein kinase B signal transduction\n",
      "    - embryonic placenta morphogenesis\n",
      "    - genomic imprinting\n",
      "    - positive regulation of vascular endothelial cell proliferation\n",
      "  CC:\n",
      "    - extracellular region\n",
      "    - extracellular space\n",
      "    - platelet alpha granule lumen\n",
      "  MF:\n",
      "    - insulin receptor binding\n",
      "    - insulin-like growth factor receptor binding\n",
      "    - insulin-like growth factor receptor binding\n",
      "    - integrin binding\n",
      "    - hormone activity\n",
      "    - protein binding\n",
      "    - growth factor activity\n",
      "    - growth factor activity\n",
      "    - transmembrane receptor protein tyrosine kinase activator activity\n",
      "    - protein serine/threonine kinase activator activity\n",
      "    - protein serine/threonine kinase activator activity\n",
      "    - receptor ligand activity\n",
      "UniProt: {'Swiss-Prot': 'P01344', 'TrEMBL': 'E3UN46'}\n",
      "\n",
      "Gene: CPLX4\n",
      "Symbol: CPLX4\n",
      "Name: complexin 4\n",
      "Summary: This gene likely encodes a member of the complexin family. The encoded protein may be involved in synaptic vesicle exocytosis. [provided by RefSeq, Jan 2009].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - visual perception\n",
      "    - synaptic vesicle exocytosis\n",
      "    - regulation of synaptic vesicle fusion to presynaptic active zone membrane\n",
      "    - regulation of neurotransmitter secretion\n",
      "    - modulation of chemical synaptic transmission\n",
      "  CC:\n",
      "    - plasma membrane\n",
      "    - SNARE complex\n",
      "    - terminal bouton\n",
      "  MF:\n",
      "    - SNARE binding\n",
      "    - protein binding\n",
      "    - syntaxin binding\n",
      "UniProt: {'Swiss-Prot': 'Q7Z7G2', 'TrEMBL': 'K7EM04'}\n",
      "\n",
      "Gene: HPD\n",
      "Symbol: HPD\n",
      "Name: 4-hydroxyphenylpyruvate dioxygenase\n",
      "Summary: The protein encoded by this gene is an enzyme in the catabolic pathway of tyrosine. The encoded protein catalyzes the conversion of 4-hydroxyphenylpyruvate to homogentisate. Defects in this gene are a cause of tyrosinemia type 3 (TYRO3) and hawkinsinuria (HAWK). Two transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Jan 2010].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - L-phenylalanine catabolic process\n",
      "    - tyrosine catabolic process\n",
      "    - tyrosine catabolic process\n",
      "    - tyrosine catabolic process\n",
      "  CC:\n",
      "    - Golgi membrane\n",
      "    - endoplasmic reticulum membrane\n",
      "    - cytosol\n",
      "    - extracellular exosome\n",
      "  MF:\n",
      "    - 4-hydroxyphenylpyruvate dioxygenase activity\n",
      "    - 4-hydroxyphenylpyruvate dioxygenase activity\n",
      "    - 4-hydroxyphenylpyruvate dioxygenase activity\n",
      "    - protein homodimerization activity\n",
      "    - metal ion binding\n",
      "UniProt: {'Swiss-Prot': 'P32754'}\n",
      "\n",
      "Gene: FLCN\n",
      "Symbol: FLCN\n",
      "Name: folliculin\n",
      "Summary: This gene is located within the Smith-Magenis syndrome region on chromosome 17. Mutations in this gene are associated with Birt-Hogg-Dube syndrome, which is characterized by fibrofolliculomas, renal tumors, lung cysts, and pneumothorax. Alternative splicing of this gene results in two transcript variants encoding different isoforms. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - negative regulation of transcription by RNA polymerase II\n",
      "    - negative regulation of transcription by RNA polymerase II\n",
      "    - in utero embryonic development\n",
      "    - cell-cell junction assembly\n",
      "    - transforming growth factor beta receptor signaling pathway\n",
      "    - cellular response to starvation\n",
      "    - positive regulation of autophagy\n",
      "    - positive regulation of autophagy\n",
      "    - hemopoiesis\n",
      "    - positive regulation of transforming growth factor beta receptor signaling pathway\n",
      "    - positive regulation of transforming growth factor beta receptor signaling pathway\n",
      "    - TOR signaling\n",
      "    - regulation of TOR signaling\n",
      "    - negative regulation of TOR signaling\n",
      "    - positive regulation of TOR signaling\n",
      "    - positive regulation of TOR signaling\n",
      "    - lysosome localization\n",
      "    - cellular response to amino acid starvation\n",
      "    - negative regulation of Rho protein signal transduction\n",
      "    - intracellular signal transduction\n",
      "    - positive regulation of apoptotic process\n",
      "    - phosphatidylinositol 3-kinase/protein kinase B signal transduction\n",
      "    - negative regulation of glycolytic process\n",
      "    - regulation of Ras protein signal transduction\n",
      "    - epithelial cell proliferation\n",
      "    - negative regulation of epithelial cell proliferation\n",
      "    - negative regulation of phosphatidylinositol 3-kinase/protein kinase B signal transduction\n",
      "    - ERK1 and ERK2 cascade\n",
      "    - negative regulation of ERK1 and ERK2 cascade\n",
      "    - cell proliferation involved in kidney development\n",
      "    - energy homeostasis\n",
      "    - intrinsic apoptotic signaling pathway\n",
      "    - negative regulation of cold-induced thermogenesis\n",
      "    - negative regulation of cell proliferation involved in kidney development\n",
      "    - negative regulation of post-translational protein modification\n",
      "    - negative regulation of brown fat cell differentiation\n",
      "    - positive regulation of TORC1 signaling\n",
      "    - positive regulation of TORC1 signaling\n",
      "    - negative regulation of lysosome organization\n",
      "    - regulation of pro-B cell differentiation\n",
      "    - positive regulation of intrinsic apoptotic signaling pathway\n",
      "  CC:\n",
      "    - nucleus\n",
      "    - cytoplasm\n",
      "    - lysosome\n",
      "    - lysosomal membrane\n",
      "    - centrosome\n",
      "    - cytosol\n",
      "    - cytosol\n",
      "    - plasma membrane\n",
      "    - cilium\n",
      "    - cilium\n",
      "    - midbody\n",
      "    - intracellular membrane-bounded organelle\n",
      "    - cell-cell contact zone\n",
      "    - mitotic spindle\n",
      "    - FNIP-folliculin RagC/D GAP\n",
      "  MF:\n",
      "    - enzyme inhibitor activity\n",
      "    - GTPase activator activity\n",
      "    - protein binding\n",
      "    - enzyme binding\n",
      "    - protein-containing complex binding\n",
      "UniProt: {'Swiss-Prot': 'Q8NFG4', 'TrEMBL': ['J3QQZ7', 'A0A0S2Z5Y7', 'C9J4C4']}\n",
      "\n",
      "Gene: MCPH1\n",
      "Symbol: MCPH1\n",
      "Name: microcephalin 1\n",
      "Summary: This gene encodes a DNA damage response protein. The encoded protein may play a role in G2/M checkpoint arrest via maintenance of inhibitory phosphorylation of cyclin-dependent kinase 1. Mutations in this gene have been associated with primary autosomal recessive microcephaly 1 and premature chromosome condensation syndrome. Alternatively spliced transcript variants have been described. [provided by RefSeq, Feb 2010].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - negative regulation of transcription by RNA polymerase II\n",
      "    - establishment of mitotic spindle orientation\n",
      "    - mitotic cell cycle\n",
      "    - cerebral cortex development\n",
      "    - regulation of centrosome cycle\n",
      "    - regulation of inflammatory response\n",
      "    - bone development\n",
      "    - regulation of chromosome condensation\n",
      "    - protein localization to centrosome\n",
      "    - neuronal stem cell population maintenance\n",
      "  CC:\n",
      "    - nucleoplasm\n",
      "    - cytoplasm\n",
      "    - centrosome\n",
      "  MF:\n",
      "    - protein binding\n",
      "    - identical protein binding\n",
      "UniProt: {'Swiss-Prot': 'Q8NEM0', 'TrEMBL': ['A0A8I5KRS3', 'A0A8I5KYX6', 'A0A8I5KW78', 'A0A8I5QKY7', 'A0A8I5QJK3', 'A0A8I5KSF2', 'A0A8I5KXP9', 'A0A8I5KYD2', 'A0A8I5KRI7', 'A0A8I5KXJ5', 'A0A8I5KZ89', 'A0A8I5KX36', 'A0A8I5KQZ4', 'A0A8I5KR97', 'A0A8I5KTK9', 'A0A8I5KR64', 'A0A8I5QKX9', 'A0A8I5KWR1', 'A0A8I5KPV6', 'A0A8I5KV10', 'A0A8I5KQQ3']}\n",
      "\n",
      "Gene: SLC18A2\n",
      "Symbol: SLC18A2\n",
      "Name: solute carrier family 18 member A2\n",
      "Summary: This gene encodes an transmembrane protein that functions as an ATP-dependent transporter of monoamines, such as dopamine, norepinephrine, serotonin, and histamine. This protein transports amine neurotransmitters into synaptic vesicles. Polymorphisms in this gene may be associated with schizophrenia, bipolar disorder, and other neurological/psychiatric ailments. [provided by RefSeq, Jun 2018].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - response to amphetamine\n",
      "    - serotonin secretion by mast cell\n",
      "    - histamine secretion by mast cell\n",
      "    - neurotransmitter transport\n",
      "    - chemical synaptic transmission\n",
      "    - locomotory behavior\n",
      "    - response to toxic substance\n",
      "    - post-embryonic development\n",
      "    - aminergic neurotransmitter loading into synaptic vesicle\n",
      "    - monoamine transport\n",
      "    - dopamine transport\n",
      "    - serotonin uptake\n",
      "    - histamine uptake\n",
      "    - neurotransmitter loading into synaptic vesicle\n",
      "    - neurotransmitter loading into synaptic vesicle\n",
      "    - somato-dendritic dopamine secretion\n",
      "    - proton transmembrane transport\n",
      "    - negative regulation of reactive oxygen species biosynthetic process\n",
      "  CC:\n",
      "    - centrosome\n",
      "    - plasma membrane\n",
      "    - synaptic vesicle\n",
      "    - membrane\n",
      "    - axon\n",
      "    - dendrite\n",
      "    - secretory granule membrane\n",
      "    - synaptic vesicle membrane\n",
      "    - synaptic vesicle membrane\n",
      "    - terminal bouton\n",
      "    - intracellular membrane-bounded organelle\n",
      "    - clathrin-sculpted monoamine transport vesicle membrane\n",
      "    - dopaminergic synapse\n",
      "  MF:\n",
      "    - serotonin:sodium:chloride symporter activity\n",
      "    - protein binding\n",
      "    - monoamine transmembrane transporter activity\n",
      "    - monoamine:proton antiporter activity\n",
      "UniProt: {'Swiss-Prot': 'Q05940'}\n",
      "\n",
      "Gene: HDHD3\n",
      "Symbol: HDHD3\n",
      "Name: haloacid dehalogenase like hydrolase domain containing 3\n",
      "Summary: Located in nucleolus. [provided by Alliance of Genome Resources, Apr 2022]\n",
      "GO Annotations:\n",
      "  CC:\n",
      "    - nucleus\n",
      "    - nucleolus\n",
      "    - mitochondrion\n",
      "    - intracellular membrane-bounded organelle\n",
      "  MF:\n",
      "    - protein binding\n",
      "UniProt: {'Swiss-Prot': 'Q9BSH5'}\n",
      "\n",
      "Gene: CDKL5\n",
      "Symbol: CDKL5\n",
      "Name: cyclin dependent kinase like 5\n",
      "Summary: This gene is a member of Ser/Thr protein kinase family and encodes a phosphorylated protein with protein kinase activity. Mutations in this gene have been associated with X-linked infantile spasm syndrome (ISSX), also known as X-linked West syndrome, and Rett syndrome (RTT). Alternate transcriptional splice variants have been characterized. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - neuron migration\n",
      "    - positive regulation of Rac protein signal transduction\n",
      "    - positive regulation of axon extension\n",
      "    - positive regulation of axon extension\n",
      "    - regulation of dendrite development\n",
      "    - regulation of dendrite development\n",
      "    - positive regulation of dendrite morphogenesis\n",
      "    - modulation of chemical synaptic transmission\n",
      "    - modulation of chemical synaptic transmission\n",
      "    - regulation of cell cycle\n",
      "    - positive regulation of dendritic spine development\n",
      "    - regulation of postsynapse organization\n",
      "    - regulation of postsynapse organization\n",
      "    - regulation of cilium assembly\n",
      "  CC:\n",
      "    - nucleus\n",
      "    - nucleus\n",
      "    - nucleoplasm\n",
      "    - centrosome\n",
      "    - ruffle membrane\n",
      "    - dendrite cytoplasm\n",
      "    - dendrite cytoplasm\n",
      "    - ciliary basal body\n",
      "    - dendritic growth cone\n",
      "    - perinuclear region of cytoplasm\n",
      "    - ciliary tip\n",
      "    - glutamatergic synapse\n",
      "    - glutamatergic synapse\n",
      "    - postsynaptic density, intracellular component\n",
      "  MF:\n",
      "    - protein kinase activity\n",
      "    - protein kinase activity\n",
      "    - protein serine/threonine kinase activity\n",
      "    - cyclin-dependent protein serine/threonine kinase activity\n",
      "    - protein binding\n",
      "    - ATP binding\n",
      "    - kinase activity\n",
      "    - small GTPase binding\n",
      "    - protein serine kinase activity\n",
      "UniProt: {'Swiss-Prot': 'O76039', 'TrEMBL': ['A0A096LPG3', 'A0A669KBC2', 'A0A1B0GTX4', 'A0A096LNR9', 'A0A1B0GUM4', 'A0A096LPI4', 'A0A096LP32']}\n",
      "\n",
      "Gene: GRM1\n",
      "Symbol: GRM1\n",
      "Name: glutamate metabotropic receptor 1\n",
      "Summary: This gene encodes a metabotropic glutamate receptor that functions by activating phospholipase C. L-glutamate is the major excitatory neurotransmitter in the central nervous system and activates both ionotropic and metabotropic glutamate receptors. Glutamatergic neurotransmission is involved in most aspects of normal brain function and can be perturbed in many neuropathologic conditions. The canonical alpha isoform of the encoded protein is a disulfide-linked homodimer whose activity is mediated by a G-protein-coupled phosphatidylinositol-calcium second messenger system. This gene may be associated with many disease states, including schizophrenia, bipolar disorder, depression, and breast cancer. Alternative splicing results in multiple transcript variants encoding different isoforms. [provided by RefSeq, May 2013].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - G protein-coupled receptor signaling pathway\n",
      "    - adenylate cyclase-inhibiting G protein-coupled glutamate receptor signaling pathway\n",
      "    - phospholipase C-activating G protein-coupled receptor signaling pathway\n",
      "    - phospholipase C-activating G protein-coupled glutamate receptor signaling pathway\n",
      "    - G protein-coupled glutamate receptor signaling pathway\n",
      "    - chemical synaptic transmission\n",
      "    - locomotory behavior\n",
      "    - sensory perception of pain\n",
      "    - positive regulation of MAPK cascade\n",
      "    - regulation of sensory perception of pain\n",
      "    - regulation of synaptic transmission, glutamatergic\n",
      "    - regulation of postsynaptic membrane potential\n",
      "    - cellular response to electrical stimulus\n",
      "    - L-glutamate import across plasma membrane\n",
      "    - synaptic signaling via neuropeptide\n",
      "    - regulation of postsynaptic cytosolic calcium ion concentration\n",
      "  CC:\n",
      "    - nucleus\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - dendrite\n",
      "    - G protein-coupled receptor dimeric complex\n",
      "    - G protein-coupled receptor homodimeric complex\n",
      "    - Schaffer collateral - CA1 synapse\n",
      "    - postsynaptic density membrane\n",
      "    - glutamatergic synapse\n",
      "  MF:\n",
      "    - adenylate cyclase inhibiting G protein-coupled glutamate receptor activity\n",
      "    - G protein-coupled receptor activity\n",
      "    - protein binding\n",
      "    - glutamate receptor activity\n",
      "    - glutamate receptor activity\n",
      "    - G protein-coupled neurotransmitter receptor activity involved in regulation of postsynaptic cytosolic calcium ion concentration\n",
      "    - G protein-coupled receptor activity involved in regulation of postsynaptic membrane potential\n",
      "    - G protein-coupled receptor activity involved in regulation of postsynaptic membrane potential\n",
      "    - neurotransmitter receptor activity involved in regulation of postsynaptic cytosolic calcium ion concentration\n",
      "UniProt: {'Swiss-Prot': 'Q13255', 'TrEMBL': 'Q59HC2'}\n",
      "\n",
      "Gene: MPZL2\n",
      "Symbol: MPZL2\n",
      "Name: myelin protein zero like 2\n",
      "Summary: Thymus development depends on a complex series of interactions between thymocytes and the stromal component of the organ. Epithelial V-like antigen (EVA) is expressed in thymus epithelium and strongly downregulated by thymocyte developmental progression. This gene is expressed in the thymus and in several epithelial structures early in embryogenesis. It is highly homologous to the myelin protein zero and, in thymus-derived epithelial cell lines, is poorly soluble in nonionic detergents, strongly suggesting an association to the cytoskeleton. Its capacity to mediate cell adhesion through a homophilic interaction and its selective regulation by T cell maturation might imply the participation of EVA in the earliest phases of thymus organogenesis. The protein bears a characteristic V-type domain and two potential N-glycosylation sites in the extracellular domain; a putative serine phosphorylation site for casein kinase 2 is also present in the cytoplasmic tail. Two transcript variants encoding the same protein have been found for this gene. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - homophilic cell adhesion via plasma membrane adhesion molecules\n",
      "    - anatomical structure morphogenesis\n",
      "    - cell-cell adhesion\n",
      "  CC:\n",
      "    - cytoskeleton\n",
      "    - plasma membrane\n",
      "  MF:\n",
      "    - protein binding\n",
      "UniProt: {'Swiss-Prot': 'O60487'}\n",
      "\n",
      "Gene: NUDT3\n",
      "Symbol: NUDT3\n",
      "Name: nudix hydrolase 3\n",
      "Summary: NUDT3 belongs to the MutT, or Nudix, protein family. Nudix proteins act as homeostatic checkpoints at important stages in nucleoside phosphate metabolic pathways, guarding against elevated levels of potentially dangerous intermediates, like 8-oxo-dGTP, which promotes AT-to-CG transversions (Safrany et al., 1998 [PubMed 9822604]).[supplied by OMIM, Feb 2011].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - cell-cell signaling\n",
      "    - diadenosine polyphosphate catabolic process\n",
      "    - diphosphoinositol polyphosphate metabolic process\n",
      "    - diphosphoinositol polyphosphate catabolic process\n",
      "    - RNA decapping\n",
      "    - RNA decapping\n",
      "    - diadenosine pentaphosphate catabolic process\n",
      "    - diadenosine hexaphosphate catabolic process\n",
      "    - diadenosine hexaphosphate catabolic process\n",
      "    - adenosine 5'-(hexahydrogen pentaphosphate) catabolic process\n",
      "  CC:\n",
      "    - nucleus\n",
      "    - nucleus\n",
      "    - cytoplasm\n",
      "    - cytoplasm\n",
      "    - cytosol\n",
      "    - cytosol\n",
      "    - glutamatergic synapse\n",
      "  MF:\n",
      "    - magnesium ion binding\n",
      "    - endopolyphosphatase activity\n",
      "    - endopolyphosphatase activity\n",
      "    - protein binding\n",
      "    - zinc ion binding\n",
      "    - diphosphoinositol-polyphosphate diphosphatase activity\n",
      "    - diphosphoinositol-polyphosphate diphosphatase activity\n",
      "    - manganese ion binding\n",
      "    - bis(5'-adenosyl)-hexaphosphatase activity\n",
      "    - bis(5'-adenosyl)-pentaphosphatase activity\n",
      "    - inositol diphosphate tetrakisphosphate diphosphatase activity\n",
      "    - inositol diphosphate tetrakisphosphate diphosphatase activity\n",
      "    - inositol diphosphate pentakisphosphate diphosphatase activity\n",
      "    - inositol diphosphate pentakisphosphate diphosphatase activity\n",
      "    - inositol-5-diphosphate-1,2,3,4,6-pentakisphosphate diphosphatase activity\n",
      "    - inositol-3,5-bisdiphosphate-2,3,4,6-tetrakisphosphate 5-diphosphatase activity\n",
      "    - 5'-(N(7)-methyl 5'-triphosphoguanosine)-[mRNA] diphosphatase activity\n",
      "    - 5'-(N(7)-methylguanosine 5'-triphospho)-[mRNA] hydrolase activity\n",
      "UniProt: {'Swiss-Prot': 'O95989'}\n",
      "\n",
      "Gene: GRM5\n",
      "Symbol: GRM5\n",
      "Name: glutamate metabotropic receptor 5\n",
      "Summary: This gene encodes a member of the G-protein coupled receptor 3 protein family. The encoded protein is a metabatropic glutamate receptor, whose signaling activates a phosphatidylinositol-calcium second messenger system. This protein may be involved in the regulation of neural network activity and synaptic plasticity. Glutamatergic neurotransmission is involved in most aspects of normal brain function and can be perturbed in many neuropathologic conditions. A pseudogene of this gene has been defined on chromosome 11. Alternative splicing results in multiple transcript variants. [provided by RefSeq, May 2014].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - desensitization of G protein-coupled receptor signaling pathway\n",
      "    - regulation of DNA-templated transcription\n",
      "    - adenylate cyclase-inhibiting G protein-coupled glutamate receptor signaling pathway\n",
      "    - phospholipase C-activating G protein-coupled glutamate receptor signaling pathway\n",
      "    - G protein-coupled glutamate receptor signaling pathway\n",
      "    - G protein-coupled glutamate receptor signaling pathway\n",
      "    - chemical synaptic transmission\n",
      "    - learning or memory\n",
      "    - learning\n",
      "    - locomotory behavior\n",
      "    - positive regulation of MAPK cascade\n",
      "    - positive regulation of long-term neuronal synaptic plasticity\n",
      "    - synapse organization\n",
      "    - positive regulation of calcium-mediated signaling\n",
      "    - cognition\n",
      "    - regulation of synaptic transmission, glutamatergic\n",
      "    - regulation of postsynaptic membrane potential\n",
      "    - regulation of postsynaptic cytosolic calcium ion concentration\n",
      "    - cellular response to amyloid-beta\n",
      "    - cellular response to amyloid-beta\n",
      "  CC:\n",
      "    - cytoplasm\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - dendrite\n",
      "    - neuronal cell body\n",
      "    - dendritic spine\n",
      "    - dendritic shaft\n",
      "    - astrocyte projection\n",
      "    - Schaffer collateral - CA1 synapse\n",
      "    - postsynaptic density membrane\n",
      "    - glutamatergic synapse\n",
      "  MF:\n",
      "    - adenylate cyclase inhibiting G protein-coupled glutamate receptor activity\n",
      "    - G protein-coupled receptor activity\n",
      "    - protein binding\n",
      "    - glutamate receptor activity\n",
      "    - protein tyrosine kinase activator activity\n",
      "    - A2A adenosine receptor binding\n",
      "    - identical protein binding\n",
      "    - G protein-coupled receptor activity involved in regulation of postsynaptic membrane potential\n",
      "    - neurotransmitter receptor activity involved in regulation of postsynaptic cytosolic calcium ion concentration\n",
      "    - protein tyrosine kinase binding\n",
      "UniProt: {'Swiss-Prot': 'P41594', 'TrEMBL': ['H7C223', 'A8K5P7', 'A8MT20']}\n",
      "\n",
      "Gene: LPAR6\n",
      "Symbol: LPAR6\n",
      "Name: lysophosphatidic acid receptor 6\n",
      "Summary: The protein encoded by this gene belongs to the family of G-protein coupled receptors, that are preferentially activated by adenosine and uridine nucleotides. This gene aligns with an internal intron of the retinoblastoma susceptibility gene in the reverse orientation. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Jun 2009].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - blastocyst hatching\n",
      "    - G protein-coupled receptor signaling pathway\n",
      "  CC:\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - intracellular membrane-bounded organelle\n",
      "  MF:\n",
      "    - protein binding\n",
      "    - lysophosphatidic acid receptor activity\n",
      "UniProt: {'Swiss-Prot': 'P43657', 'TrEMBL': 'B3KVQ5'}\n",
      "\n",
      "Gene: NR1I3\n",
      "Symbol: NR1I3\n",
      "Name: nuclear receptor subfamily 1 group I member 3\n",
      "Summary: This gene encodes a member of the nuclear receptor superfamily, and is a key regulator of xenobiotic and endobiotic metabolism. The protein binds to DNA as a monomer or a heterodimer with the retinoid X receptor and regulates the transcription of target genes involved in drug metabolism and bilirubin clearance, such as cytochrome P450 family members. Unlike most nuclear receptors, this transcriptional regulator is constitutively active in the absence of ligand but is regulated by both agonists and inverse agonists. Ligand binding results in translocation of this protein to the nucleus, where it activates or represses target gene transcription. These ligands include bilirubin, a variety of foreign compounds, steroid hormones, and prescription drugs. In addition to drug metabolism, the CAR protein is also reported to regulate genes involved in glucose metabolism, lipid metabolism, cell proliferation, and circadian clock regulation. Multiple transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Jul 2020].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - negative regulation of transcription by RNA polymerase II\n",
      "    - osteoblast differentiation\n",
      "    - signal transduction\n",
      "    - cell differentiation\n",
      "    - intracellular receptor signaling pathway\n",
      "    - positive regulation of transcription by RNA polymerase II\n",
      "    - positive regulation of transcription by RNA polymerase II\n",
      "  CC:\n",
      "    - chromatin\n",
      "    - nucleus\n",
      "    - nucleus\n",
      "    - nucleoplasm\n",
      "    - nucleoplasm\n",
      "    - cytoplasm\n",
      "    - cytosol\n",
      "    - cytoskeleton\n",
      "  MF:\n",
      "    - RNA polymerase II transcription regulatory region sequence-specific DNA binding\n",
      "    - RNA polymerase II cis-regulatory region sequence-specific DNA binding\n",
      "    - RNA polymerase II cis-regulatory region sequence-specific DNA binding\n",
      "    - DNA-binding transcription factor activity, RNA polymerase II-specific\n",
      "    - DNA-binding transcription activator activity, RNA polymerase II-specific\n",
      "    - DNA-binding transcription factor activity\n",
      "    - nuclear receptor activity\n",
      "    - nuclear receptor activity\n",
      "    - protein binding\n",
      "    - zinc ion binding\n",
      "    - sequence-specific double-stranded DNA binding\n",
      "UniProt: {'Swiss-Prot': 'Q14994', 'TrEMBL': ['B7Z8R7', 'F1DAL4', 'Q6GZ72', 'C0H5Y2', 'Q6GZ70', 'Q6GZ81', 'Q6GZ69', 'D6REZ7', 'E9PCF2']}\n",
      "\n",
      "Gene: ONECUT3\n",
      "Symbol: ONECUT3\n",
      "Name: one cut homeobox 3\n",
      "Summary: Enables sequence-specific double-stranded DNA binding activity. Predicted to be involved in regulation of transcription by RNA polymerase II. Predicted to be part of chromatin. Predicted to be active in nucleus. [provided by Alliance of Genome Resources, Apr 2022]\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - regulation of transcription by RNA polymerase II\n",
      "    - positive regulation of transcription by RNA polymerase II\n",
      "  CC:\n",
      "    - chromatin\n",
      "    - nucleus\n",
      "    - transcription regulator complex\n",
      "  MF:\n",
      "    - RNA polymerase II cis-regulatory region sequence-specific DNA binding\n",
      "    - DNA-binding transcription factor activity, RNA polymerase II-specific\n",
      "    - DNA-binding transcription factor activity, RNA polymerase II-specific\n",
      "    - DNA-binding transcription activator activity, RNA polymerase II-specific\n",
      "    - protein binding\n",
      "    - sequence-specific double-stranded DNA binding\n",
      "UniProt: {'Swiss-Prot': 'O60422'}\n",
      "\n",
      "Gene: ABCB6\n",
      "Symbol: ABCB6\n",
      "Name: ATP binding cassette subfamily B member 6 (LAN blood group)\n",
      "Summary: This gene encodes a member of the ATP-binding cassette (ABC) transporter superfamily. ABC proteins transport various molecules across extra- and intra-cellular membranes. This protein is a member of the heavy metal importer subfamily and plays a role in porphyrin transport. This gene is the molecular basis of the Langereis (Lan) blood group antigen and mutations in this gene underlie familial pseudohyperkalemia and dyschromatosis universalis hereditaria. [provided by RefSeq, Mar 2017].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - porphyrin-containing compound metabolic process\n",
      "    - porphyrin-containing compound metabolic process\n",
      "    - porphyrin-containing compound biosynthetic process\n",
      "    - heme B biosynthetic process\n",
      "    - intracellular copper ion homeostasis\n",
      "    - intracellular iron ion homeostasis\n",
      "    - brain development\n",
      "    - heme transport\n",
      "    - heme transport\n",
      "    - heme transport\n",
      "    - tetrapyrrole metabolic process\n",
      "    - heme transmembrane transport\n",
      "    - heme metabolic process\n",
      "    - skin development\n",
      "    - transmembrane transport\n",
      "    - cellular detoxification of cadmium ion\n",
      "    - melanosome assembly\n",
      "  CC:\n",
      "    - Golgi membrane\n",
      "    - nucleoplasm\n",
      "    - mitochondrion\n",
      "    - mitochondrion\n",
      "    - mitochondrion\n",
      "    - mitochondrion\n",
      "    - mitochondrial envelope\n",
      "    - mitochondrial outer membrane\n",
      "    - mitochondrial outer membrane\n",
      "    - lysosomal membrane\n",
      "    - endosome\n",
      "    - vacuolar membrane\n",
      "    - endoplasmic reticulum\n",
      "    - endoplasmic reticulum membrane\n",
      "    - Golgi apparatus\n",
      "    - cytosol\n",
      "    - plasma membrane\n",
      "    - early endosome membrane\n",
      "    - multivesicular body membrane\n",
      "    - melanosome membrane\n",
      "    - endolysosome membrane\n",
      "    - ATP-binding cassette (ABC) transporter complex\n",
      "    - extracellular exosome\n",
      "    - extracellular exosome\n",
      "  MF:\n",
      "    - ATP binding\n",
      "    - ABC-type heme transporter activity\n",
      "    - ABC-type heme transporter activity\n",
      "    - ABC-type heme transporter activity\n",
      "    - efflux transmembrane transporter activity\n",
      "    - ATP hydrolysis activity\n",
      "    - heme binding\n",
      "    - heme binding\n",
      "    - tetrapyrrole binding\n",
      "    - ABC-type transporter activity\n",
      "    - ABC-type transporter activity\n",
      "    - ABC-type transporter activity\n",
      "UniProt: {'Swiss-Prot': 'Q9NP58', 'TrEMBL': ['H7C3Z0', 'H7C245', 'H7C049', 'H7C1R6']}\n",
      "\n",
      "Gene: CHST13\n",
      "Symbol: CHST13\n",
      "Name: carbohydrate sulfotransferase 13\n",
      "Summary: The protein encoded by this gene belongs to the sulfotransferase 2 family. It is localized to the golgi membrane, and catalyzes the transfer of sulfate to the C4 hydroxyl of beta-1,4-linked N-acetylgalactosamine (GalNAc) flanked by glucuronic acid residue in chondroitin. Chondroitin sulfate constitutes the predominant proteoglycan present in cartilage and is distributed on the surfaces of many cells and extracellular matrices. [provided by RefSeq, Aug 2011].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - carbohydrate biosynthetic process\n",
      "    - proteoglycan biosynthetic process\n",
      "    - chondroitin sulfate biosynthetic process\n",
      "    - chondroitin sulfate biosynthetic process\n",
      "  CC:\n",
      "    - Golgi membrane\n",
      "    - membrane\n",
      "  MF:\n",
      "    - dermatan 4-sulfotransferase activity\n",
      "    - protein binding\n",
      "    - sulfotransferase activity\n",
      "    - chondroitin 4-sulfotransferase activity\n",
      "    - chondroitin 4-sulfotransferase activity\n",
      "UniProt: {'Swiss-Prot': 'Q8NET6'}\n",
      "\n",
      "Gene: KIF5A\n",
      "Symbol: KIF5A\n",
      "Name: kinesin family member 5A\n",
      "Summary: This gene encodes a member of the kinesin family of proteins. Members of this family are part of a multisubunit complex that functions as a microtubule motor in intracellular organelle transport. Mutations in this gene cause autosomal dominant spastic paraplegia 10. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - microtubule-based movement\n",
      "    - chemical synaptic transmission\n",
      "    - axon guidance\n",
      "    - vesicle-mediated transport\n",
      "    - synaptic vesicle transport\n",
      "    - anterograde dendritic transport of neurotransmitter receptor complex\n",
      "    - anterograde axonal protein transport\n",
      "    - retrograde neuronal dense core vesicle transport\n",
      "  CC:\n",
      "    - cytoplasm\n",
      "    - cytosol\n",
      "    - kinesin complex\n",
      "    - microtubule\n",
      "    - membrane\n",
      "    - dendrite cytoplasm\n",
      "    - ciliary rootlet\n",
      "    - perikaryon\n",
      "    - perinuclear region of cytoplasm\n",
      "    - postsynaptic cytosol\n",
      "    - axon cytoplasm\n",
      "  MF:\n",
      "    - cytoskeletal motor activity\n",
      "    - microtubule motor activity\n",
      "    - protein binding\n",
      "    - ATP binding\n",
      "    - microtubule binding\n",
      "    - plus-end-directed microtubule motor activity\n",
      "    - ATP hydrolysis activity\n",
      "    - kinesin binding\n",
      "UniProt: {'Swiss-Prot': 'Q12840', 'TrEMBL': ['J3KNA1', 'A0A6Q8PFR6', 'A0A6Q8PGT2', 'A0A6Q8PGG7', 'A0A6Q8PEZ8', 'A0A6Q8PH74', 'A0A6Q8PGP0', 'A0A6Q8PH72', 'A0A6Q8PFB8', 'A0A6Q8PFG5', 'A0A6Q8PH22', 'A0A6Q8PGP8', 'A0A6Q8PGN1', 'A0A6Q8PGJ3', 'A0A6Q8PFV6', 'A0A6Q8PFD6', 'A0A6Q8PHA9', 'A0A6Q8PFQ3']}\n",
      "\n",
      "Gene: GJC3\n",
      "Symbol: GJC3\n",
      "Name: gap junction protein gamma 3\n",
      "Summary: This gene encodes a gap junction protein. The encoded protein, also known as a connexin, plays a role in formation of gap junctions, which provide direct connections between neighboring cells. Mutations in this gene have been reported to be associated with nonsyndromic hearing loss.[provided by RefSeq, Feb 2010].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - cell-cell signaling\n",
      "    - sensory perception of sound\n",
      "    - myelination\n",
      "    - transmembrane transport\n",
      "    - AV node cell to bundle of His cell communication by electrical coupling\n",
      "  CC:\n",
      "    - connexin complex\n",
      "    - myelin sheath\n",
      "  MF:\n",
      "    - gap junction channel activity\n",
      "    - protein binding\n",
      "    - identical protein binding\n",
      "    - gap junction channel activity involved in AV node cell-bundle of His cell electrical coupling\n",
      "UniProt: {'Swiss-Prot': 'Q8NFK1'}\n",
      "\n",
      "Gene: DLX2\n",
      "Symbol: DLX2\n",
      "Name: distal-less homeobox 2\n",
      "Summary: Many vertebrate homeo box-containing genes have been identified on the basis of their sequence similarity with Drosophila developmental genes. Members of the Dlx gene family contain a homeobox that is related to that of Distal-less (Dll), a gene expressed in the head and limbs of the developing fruit fly. The Distal-less (Dlx) family of genes comprises at least 6 different members, DLX1-DLX6. The DLX proteins are postulated to play a role in forebrain and craniofacial development. This gene is located in a tail-to-tail configuration with another member of the gene family on the long arm of chromosome 2. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - negative regulation of transcription by RNA polymerase II\n",
      "    - regulation of transcription by RNA polymerase II\n",
      "    - Notch signaling pathway\n",
      "    - brain development\n",
      "    - proximal/distal pattern formation\n",
      "    - neuroblast differentiation\n",
      "    - subpallium development\n",
      "    - hippocampus development\n",
      "    - olfactory bulb development\n",
      "    - forebrain neuron differentiation\n",
      "    - cerebral cortex GABAergic interneuron fate commitment\n",
      "    - cell differentiation\n",
      "    - odontogenesis of dentin-containing tooth\n",
      "    - positive regulation of cell differentiation\n",
      "    - negative regulation of Notch signaling pathway\n",
      "    - positive regulation of transcription by RNA polymerase II\n",
      "    - negative regulation of photoreceptor cell differentiation\n",
      "    - embryonic cranial skeleton morphogenesis\n",
      "    - embryonic skeletal system development\n",
      "    - oligodendrocyte differentiation\n",
      "    - negative regulation of oligodendrocyte differentiation\n",
      "    - branching morphogenesis of a nerve\n",
      "    - cartilage development\n",
      "    - positive regulation of amacrine cell differentiation\n",
      "  CC:\n",
      "    - chromatin\n",
      "    - nucleus\n",
      "  MF:\n",
      "    - transcription cis-regulatory region binding\n",
      "    - RNA polymerase II cis-regulatory region sequence-specific DNA binding\n",
      "    - DNA-binding transcription factor activity, RNA polymerase II-specific\n",
      "    - DNA-binding transcription factor activity, RNA polymerase II-specific\n",
      "    - DNA-binding transcription activator activity, RNA polymerase II-specific\n",
      "    - chromatin binding\n",
      "    - DNA-binding transcription factor activity\n",
      "    - single-stranded RNA binding\n",
      "    - protein binding\n",
      "    - sequence-specific double-stranded DNA binding\n",
      "UniProt: {'Swiss-Prot': 'Q07687', 'TrEMBL': ['X5D7D8', 'Q53QU7']}\n",
      "\n",
      "Gene: EPHX2\n",
      "Symbol: EPHX2\n",
      "Name: epoxide hydrolase 2\n",
      "Summary: This gene encodes a member of the epoxide hydrolase family. The protein, found in both the cytosol and peroxisomes, binds to specific epoxides and converts them to the corresponding dihydrodiols. Mutations in this gene have been associated with familial hypercholesterolemia. Alternatively spliced transcript variants have been described. [provided by RefSeq, Feb 2012].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - regulation of cell growth\n",
      "    - response to toxic substance\n",
      "    - positive regulation of gene expression\n",
      "    - dephosphorylation\n",
      "    - cholesterol homeostasis\n",
      "    - cholesterol homeostasis\n",
      "    - stilbene catabolic process\n",
      "    - phospholipid dephosphorylation\n",
      "    - phospholipid dephosphorylation\n",
      "    - regulation of cholesterol metabolic process\n",
      "    - epoxide metabolic process\n",
      "  CC:\n",
      "    - peroxisome\n",
      "    - peroxisome\n",
      "    - peroxisomal matrix\n",
      "    - cytosol\n",
      "    - cytosol\n",
      "    - extracellular exosome\n",
      "  MF:\n",
      "    - magnesium ion binding\n",
      "    - magnesium ion binding\n",
      "    - epoxide hydrolase activity\n",
      "    - toxic substance binding\n",
      "    - phosphatase activity\n",
      "    - 10-hydroxy-9-(phosphonooxy)octadecanoate phosphatase activity\n",
      "    - lipid phosphatase activity\n",
      "    - lipid phosphatase activity\n",
      "    - protein homodimerization activity\n",
      "    - lysophosphatidic acid phosphatase activity\n",
      "UniProt: {'Swiss-Prot': 'P34913', 'TrEMBL': ['E5RI53', 'H0YAW7', 'E5RFU2', 'E5RFH6']}\n",
      "\n",
      "Gene: SRD5A2\n",
      "Symbol: SRD5A2\n",
      "Name: steroid 5 alpha-reductase 2\n",
      "Summary: This gene encodes a microsomal protein expressed at high levels in androgen-sensitive tissues such as the prostate. The encoded protein is active at acidic pH and is sensitive to the 4-azasteroid inhibitor finasteride. Deficiencies in this gene can result in male pseudohermaphroditism, specifically pseudovaginal perineoscrotal hypospadias (PPSH). [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - steroid biosynthetic process\n",
      "    - androgen biosynthetic process\n",
      "    - steroid catabolic process\n",
      "    - cell-cell signaling\n",
      "    - androgen metabolic process\n",
      "    - male gonad development\n",
      "    - response to xenobiotic stimulus\n",
      "    - biphenyl metabolic process\n",
      "    - dibenzo-p-dioxin metabolic process\n",
      "    - phthalate metabolic process\n",
      "    - hippocampus development\n",
      "    - hypothalamus development\n",
      "    - cell differentiation\n",
      "    - male genitalia development\n",
      "    - female genitalia development\n",
      "    - response to nutrient levels\n",
      "    - response to follicle-stimulating hormone\n",
      "    - response to testosterone\n",
      "    - response to peptide hormone\n",
      "    - response to steroid hormone\n",
      "    - bone development\n",
      "    - testosterone biosynthetic process\n",
      "    - response to biphenyl\n",
      "  CC:\n",
      "    - endoplasmic reticulum membrane\n",
      "    - neuronal cell body\n",
      "    - cell body fiber\n",
      "  MF:\n",
      "    - 3-oxo-5-alpha-steroid 4-dehydrogenase activity\n",
      "    - 3-oxo-5-alpha-steroid 4-dehydrogenase activity\n",
      "    - protein binding\n",
      "    - testosterone dehydrogenase [NAD(P)+] activity\n",
      "    - amide binding\n",
      "    - 3-oxo-5-alpha-steroid 4-dehydrogenase (NADP+) activity\n",
      "UniProt: {'Swiss-Prot': 'P31213'}\n",
      "\n",
      "Gene: GM2A\n",
      "Symbol: GM2A\n",
      "Name: ganglioside GM2 activator\n",
      "Summary: This gene encodes a small glycolipid transport protein which acts as a substrate specific co-factor for the lysosomal enzyme beta-hexosaminidase A. Beta-hexosaminidase A, together with GM2 ganglioside activator, catalyzes the degradation of the ganglioside GM2, and other molecules containing terminal N-acetyl hexosamines. Mutations in this gene result in GM2-gangliosidosis type AB or the AB variant of Tay-Sachs disease. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Nov 2009].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - ganglioside catabolic process\n",
      "    - lipid transport\n",
      "    - learning or memory\n",
      "    - oligosaccharide catabolic process\n",
      "    - lipid storage\n",
      "    - glycosphingolipid catabolic process\n",
      "    - neuromuscular process controlling balance\n",
      "    - maintenance of location in cell\n",
      "  CC:\n",
      "    - extracellular region\n",
      "    - cytosol\n",
      "    - cytoplasmic side of plasma membrane\n",
      "    - basolateral plasma membrane\n",
      "    - apical plasma membrane\n",
      "    - azurophil granule lumen\n",
      "    - lysosomal lumen\n",
      "    - intracellular membrane-bounded organelle\n",
      "    - extracellular exosome\n",
      "  MF:\n",
      "    - lipid transporter activity\n",
      "    - protein binding\n",
      "    - phospholipase activator activity\n",
      "    - sphingolipid activator protein activity\n",
      "    - beta-N-acetylgalactosaminidase activity\n",
      "UniProt: {'Swiss-Prot': 'P17900', 'TrEMBL': ['H0YBY3', 'E5RJD0']}\n",
      "\n",
      "Gene: MED18\n",
      "Symbol: MED18\n",
      "Name: mediator complex subunit 18\n",
      "Summary: MED18 is a component of the Mediator complex, which is a coactivator for DNA-binding factors that activate transcription via RNA polymerase II (Sato et al., 2003 [PubMed 12584197]).[supplied by OMIM, Oct 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - protein ubiquitination\n",
      "    - positive regulation of transcription elongation by RNA polymerase II\n",
      "    - RNA polymerase II preinitiation complex assembly\n",
      "    - positive regulation of transcription initiation by RNA polymerase II\n",
      "    - positive regulation of transcription initiation by RNA polymerase II\n",
      "  CC:\n",
      "    - ubiquitin ligase complex\n",
      "    - nucleus\n",
      "    - nucleoplasm\n",
      "    - mediator complex\n",
      "    - mediator complex\n",
      "    - core mediator complex\n",
      "    - core mediator complex\n",
      "  MF:\n",
      "    - transcription coregulator activity\n",
      "    - protein binding\n",
      "    - ubiquitin protein ligase activity\n",
      "UniProt: {'Swiss-Prot': 'Q9BUE0'}\n",
      "\n",
      "Gene: PARP2\n",
      "Symbol: PARP2\n",
      "Name: poly(ADP-ribose) polymerase 2\n",
      "Summary: This gene encodes poly(ADP-ribosyl)transferase-like 2 protein, which contains a catalytic domain and is capable of catalyzing a poly(ADP-ribosyl)ation reaction. This protein has a catalytic domain which is homologous to that of poly (ADP-ribosyl) transferase, but lacks an N-terminal DNA binding domain which activates the C-terminal catalytic domain of poly (ADP-ribosyl) transferase. The basic residues within the N-terminal region of this protein may bear potential DNA-binding properties, and may be involved in the nuclear and/or nucleolar targeting of the protein. Two alternatively spliced transcript variants encoding distinct isoforms have been found. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - DNA repair\n",
      "    - base-excision repair\n",
      "    - double-strand break repair\n",
      "    - DNA damage response\n",
      "    - DNA ADP-ribosylation\n",
      "    - decidualization\n",
      "    - positive regulation of cell growth involved in cardiac muscle cell development\n",
      "    - protein poly-ADP-ribosylation\n",
      "    - protein auto-ADP-ribosylation\n",
      "    - response to oxygen-glucose deprivation\n",
      "    - extrinsic apoptotic signaling pathway\n",
      "    - hippocampal neuron apoptotic process\n",
      "    - DNA repair-dependent chromatin remodeling\n",
      "  CC:\n",
      "    - nucleus\n",
      "    - nucleoplasm\n",
      "    - nucleolus\n",
      "    - nucleolus\n",
      "    - cytosol\n",
      "    - site of DNA damage\n",
      "    - site of DNA damage\n",
      "  MF:\n",
      "    - chromatin binding\n",
      "    - damaged DNA binding\n",
      "    - NAD+ poly-ADP-ribosyltransferase activity\n",
      "    - NAD+ poly-ADP-ribosyltransferase activity\n",
      "    - protein binding\n",
      "    - nucleotidyltransferase activity\n",
      "    - nucleosome binding\n",
      "    - poly-ADP-D-ribose binding\n",
      "    - NAD DNA ADP-ribosyltransferase activity\n",
      "    - NAD+-protein-serine ADP-ribosyltransferase activity\n",
      "    - NAD+-protein-aspartate ADP-ribosyltransferase activity\n",
      "    - NAD+-protein-glutamate ADP-ribosyltransferase activity\n",
      "    - poly-ADP-D-ribose modification-dependent protein binding\n",
      "    - NAD+-protein mono-ADP-ribosyltransferase activity\n",
      "UniProt: {'Swiss-Prot': 'Q9UGN5', 'TrEMBL': ['H0YH02', 'G3V167', 'E9PJ27']}\n",
      "\n",
      "Gene: STEAP1\n",
      "Symbol: STEAP1\n",
      "Name: STEAP family member 1\n",
      "Summary: This gene is predominantly expressed in prostate tissue, and is found to be upregulated in multiple cancer cell lines. The gene product is predicted to be a six-transmembrane protein, and was shown to be a cell surface antigen significantly expressed at cell-cell junctions. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  CC:\n",
      "    - endosome\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - cell-cell junction\n",
      "    - endosome membrane\n",
      "    - membrane\n",
      "  MF:\n",
      "    - protein binding\n",
      "    - heme binding\n",
      "    - metal ion binding\n",
      "    - ferric-chelate reductase (NADPH) activity\n",
      "UniProt: {'Swiss-Prot': 'Q9UHE8', 'TrEMBL': 'H7C0D8'}\n",
      "\n",
      "Gene: MSTN\n",
      "Symbol: MSTN\n",
      "Name: myostatin\n",
      "Summary: This gene encodes a secreted ligand of the TGF-beta (transforming growth factor-beta) superfamily of proteins. Ligands of this family bind various TGF-beta receptors leading to recruitment and activation of SMAD family transcription factors that regulate gene expression. The encoded preproprotein is proteolytically processed to generate each subunit of the disulfide-linked homodimer. This protein negatively regulates skeletal muscle cell proliferation and differentiation. Mutations in this gene are associated with increased skeletal muscle mass in humans and other mammals. [provided by RefSeq, Jul 2016].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - transforming growth factor beta receptor signaling pathway\n",
      "    - muscle organ development\n",
      "    - response to gravity\n",
      "    - positive regulation of lamellipodium assembly\n",
      "    - positive regulation of macrophage chemotaxis\n",
      "    - skeletal muscle atrophy\n",
      "    - negative regulation of muscle hypertrophy\n",
      "    - skeletal muscle satellite cell differentiation\n",
      "    - myoblast migration involved in skeletal muscle regeneration\n",
      "    - response to muscle activity\n",
      "    - ovulation cycle process\n",
      "    - response to testosterone\n",
      "    - response to estrogen\n",
      "    - response to ethanol\n",
      "    - negative regulation of myoblast differentiation\n",
      "    - positive regulation of DNA-templated transcription\n",
      "    - negative regulation of insulin receptor signaling pathway\n",
      "    - muscle cell cellular homeostasis\n",
      "    - negative regulation of skeletal muscle tissue growth\n",
      "    - response to electrical stimulus\n",
      "    - negative regulation of phosphatidylinositol 3-kinase/protein kinase B signal transduction\n",
      "    - trophoblast cell migration\n",
      "    - cellular response to hypoxia\n",
      "    - cellular response to dexamethasone stimulus\n",
      "    - negative regulation of skeletal muscle satellite cell proliferation\n",
      "    - negative regulation of satellite cell differentiation\n",
      "    - negative regulation of myoblast proliferation\n",
      "  CC:\n",
      "    - extracellular space\n",
      "    - extracellular space\n",
      "  MF:\n",
      "    - signaling receptor binding\n",
      "    - cytokine activity\n",
      "    - cytokine activity\n",
      "    - protein binding\n",
      "    - growth factor activity\n",
      "    - heparin binding\n",
      "    - identical protein binding\n",
      "    - identical protein binding\n",
      "    - protein homodimerization activity\n",
      "    - protein serine/threonine kinase activator activity\n",
      "UniProt: {'Swiss-Prot': 'O14793', 'TrEMBL': 'Q53S46'}\n",
      "\n",
      "Gene: ATP7B\n",
      "Symbol: ATP7B\n",
      "Name: ATPase copper transporting beta\n",
      "Summary: This gene is a member of the P-type cation transport ATPase family and encodes a protein with several membrane-spanning domains, an ATPase consensus sequence, a hinge domain, a phosphorylation site, and at least 2 putative copper-binding sites. This protein is a monomer, and functions as a copper-transporting ATPase which exports copper out of the cells, such as the efflux of hepatic copper into the bile. Alternate transcriptional splice variants, encoding different isoforms with distinct cellular localizations, have been characterized. Mutations in this gene have been associated with Wilson disease which is characterized by copper accumulation. [provided by RefSeq, Dec 2019].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - copper ion transport\n",
      "    - copper ion transport\n",
      "    - copper ion transport\n",
      "    - intracellular copper ion homeostasis\n",
      "    - intracellular copper ion homeostasis\n",
      "    - intracellular zinc ion homeostasis\n",
      "    - lactation\n",
      "    - copper ion import\n",
      "    - copper ion import\n",
      "    - monoatomic ion transmembrane transport\n",
      "    - response to copper ion\n",
      "    - sequestering of calcium ion\n",
      "    - protein maturation\n",
      "    - establishment of localization in cell\n",
      "    - copper ion export\n",
      "    - viral translational frameshifting\n",
      "    - xenobiotic detoxification by transmembrane export across the plasma membrane\n",
      "  CC:\n",
      "    - Golgi membrane\n",
      "    - mitochondrion\n",
      "    - late endosome\n",
      "    - Golgi apparatus\n",
      "    - trans-Golgi network\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - membrane\n",
      "    - basolateral plasma membrane\n",
      "    - cytoplasmic vesicle\n",
      "    - trans-Golgi network membrane\n",
      "    - perinuclear region of cytoplasm\n",
      "  MF:\n",
      "    - copper ion transmembrane transporter activity\n",
      "    - copper ion binding\n",
      "    - copper ion binding\n",
      "    - protein binding\n",
      "    - ATP binding\n",
      "    - ATP hydrolysis activity\n",
      "    - P-type divalent copper transporter activity\n",
      "    - P-type divalent copper transporter activity\n",
      "    - P-type divalent copper transporter activity\n",
      "    - P-type divalent copper transporter activity\n",
      "    - P-type monovalent copper transporter activity\n",
      "UniProt: {'Swiss-Prot': 'P35670', 'TrEMBL': ['A0A669KB88', 'B7ZLR4', 'B7ZLR3', 'E7ET55', 'F5H748', 'A0AAQ5BGP2', 'A0AAQ5BGL6', 'A0A669KB21', 'F5H562', 'A0A0U1RQY8', 'A0A0U1RRG1']}\n",
      "\n",
      "Gene: SULT4A1\n",
      "Symbol: SULT4A1\n",
      "Name: sulfotransferase family 4A member 1\n",
      "Summary: This gene encodes a member of the sulfotransferase family. The encoded protein is a brain-specific sulfotransferase believed to be involved in the metabolism of neurotransmitters. Polymorphisms in this gene may be associated with susceptibility to schizophrenia. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - steroid metabolic process\n",
      "    - sulfation\n",
      "    - dendrite arborization\n",
      "  CC:\n",
      "    - cytoplasm\n",
      "    - mitochondrion\n",
      "    - cytosol\n",
      "  MF:\n",
      "    - protein binding\n",
      "    - sulfotransferase activity\n",
      "    - sulfotransferase activity\n",
      "    - identical protein binding\n",
      "    - identical protein binding\n",
      "UniProt: {'Swiss-Prot': 'Q9BR01', 'TrEMBL': ['F8WE22', 'B7Z2E1']}\n",
      "\n",
      "Gene: UGT1A8\n",
      "Symbol: UGT1A8\n",
      "Name: UDP glucuronosyltransferase family 1 member A8\n",
      "Summary: This gene encodes a UDP-glucuronosyltransferase, an enzyme of the glucuronidation pathway that transforms small lipophilic molecules, such as steroids, bilirubin, hormones, and drugs, into water-soluble, excretable metabolites. This gene is part of a complex locus that encodes several UDP-glucuronosyltransferases. The locus includes thirteen unique alternate first exons followed by four common exons. Four of the alternate first exons are considered pseudogenes. Each of the remaining nine 5' exons may be spliced to the four common exons, resulting in nine proteins with different N-termini and identical C-termini. Each first exon encodes the substrate binding site, and is regulated by its own promoter. The enzyme encoded by this gene has glucuronidase activity with many substrates including coumarins, phenols, anthraquinones, flavones, and some opioids. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - liver development\n",
      "    - fatty acid metabolic process\n",
      "    - fatty acid metabolic process\n",
      "    - xenobiotic metabolic process\n",
      "    - xenobiotic metabolic process\n",
      "    - steroid metabolic process\n",
      "    - steroid metabolic process\n",
      "    - coumarin metabolic process\n",
      "    - flavonoid metabolic process\n",
      "    - retinoic acid metabolic process\n",
      "    - negative regulation of fatty acid metabolic process\n",
      "    - negative regulation of fatty acid metabolic process\n",
      "    - negative regulation of steroid metabolic process\n",
      "    - flavone metabolic process\n",
      "  CC:\n",
      "    - endoplasmic reticulum\n",
      "    - endoplasmic reticulum\n",
      "    - endoplasmic reticulum\n",
      "    - endoplasmic reticulum membrane\n",
      "    - endoplasmic reticulum membrane\n",
      "    - endoplasmic reticulum membrane\n",
      "  MF:\n",
      "    - retinoic acid binding\n",
      "    - enzyme inhibitor activity\n",
      "    - steroid binding\n",
      "    - fatty acid binding\n",
      "    - protein binding\n",
      "    - glucuronosyltransferase activity\n",
      "    - glucuronosyltransferase activity\n",
      "    - enzyme binding\n",
      "    - enzyme binding\n",
      "    - protein homodimerization activity\n",
      "    - protein heterodimerization activity\n",
      "    - protein heterodimerization activity\n",
      "UniProt: {'Swiss-Prot': 'Q9HAW9', 'TrEMBL': 'Q5DSZ6'}\n",
      "\n",
      "Gene: UGT1A1\n",
      "Symbol: UGT1A1\n",
      "Name: UDP glucuronosyltransferase family 1 member A1\n",
      "Summary: This gene encodes a UDP-glucuronosyltransferase, an enzyme of the glucuronidation pathway that transforms small lipophilic molecules, such as steroids, bilirubin, hormones, and drugs, into water-soluble, excretable metabolites. This gene is part of a complex locus that encodes several UDP-glucuronosyltransferases. The locus includes thirteen unique alternate first exons followed by four common exons. Four of the alternate first exons are considered pseudogenes. Each of the remaining nine 5' exons may be spliced to the four common exons, resulting in nine proteins with different N-termini and identical C-termini. Each first exon encodes the substrate binding site, and is regulated by its own promoter. The preferred substrate of this enzyme is bilirubin, although it also has moderate activity with simple phenols, flavones, and C18 steroids. Mutations in this gene result in Crigler-Najjar syndromes types I and II and in Gilbert syndrome. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - liver development\n",
      "    - bilirubin conjugation\n",
      "    - xenobiotic metabolic process\n",
      "    - xenobiotic metabolic process\n",
      "    - xenobiotic metabolic process\n",
      "    - xenobiotic metabolic process\n",
      "    - acute-phase response\n",
      "    - response to nutrient\n",
      "    - steroid metabolic process\n",
      "    - estrogen metabolic process\n",
      "    - flavonoid metabolic process\n",
      "    - animal organ regeneration\n",
      "    - response to lipopolysaccharide\n",
      "    - retinoic acid metabolic process\n",
      "    - response to starvation\n",
      "    - negative regulation of fatty acid metabolic process\n",
      "    - negative regulation of steroid metabolic process\n",
      "    - flavone metabolic process\n",
      "    - biphenyl catabolic process\n",
      "    - cellular response to ethanol\n",
      "    - cellular response to glucocorticoid stimulus\n",
      "    - cellular response to estradiol stimulus\n",
      "  CC:\n",
      "    - endoplasmic reticulum\n",
      "    - endoplasmic reticulum\n",
      "    - endoplasmic reticulum\n",
      "    - endoplasmic reticulum membrane\n",
      "    - endoplasmic reticulum membrane\n",
      "    - endoplasmic reticulum membrane\n",
      "    - plasma membrane\n",
      "    - endoplasmic reticulum chaperone complex\n",
      "    - perinuclear region of cytoplasm\n",
      "    - cytochrome complex\n",
      "  MF:\n",
      "    - retinoic acid binding\n",
      "    - enzyme inhibitor activity\n",
      "    - enzyme inhibitor activity\n",
      "    - steroid binding\n",
      "    - protein binding\n",
      "    - glucuronosyltransferase activity\n",
      "    - glucuronosyltransferase activity\n",
      "    - glucuronosyltransferase activity\n",
      "    - glucuronosyltransferase activity\n",
      "    - enzyme binding\n",
      "    - enzyme binding\n",
      "    - protein homodimerization activity\n",
      "    - protein heterodimerization activity\n",
      "    - protein heterodimerization activity\n",
      "UniProt: {'Swiss-Prot': 'P22309', 'TrEMBL': 'Q5DT03'}\n",
      "\n",
      "Gene: GAA\n",
      "Symbol: GAA\n",
      "Name: alpha glucosidase\n",
      "Summary: This gene encodes lysosomal alpha-glucosidase, which is essential for the degradation of glycogen to glucose in lysosomes. The encoded preproprotein is proteolytically processed to generate multiple intermediate forms and the mature form of the enzyme. Defects in this gene are the cause of glycogen storage disease II, also known as Pompe's disease, which is an autosomal recessive disorder with a broad clinical spectrum. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Jan 2016].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - maltose metabolic process\n",
      "    - regulation of the force of heart contraction\n",
      "    - diaphragm contraction\n",
      "    - heart morphogenesis\n",
      "    - glycogen catabolic process\n",
      "    - glycogen catabolic process\n",
      "    - glycogen catabolic process\n",
      "    - sucrose metabolic process\n",
      "    - glucose metabolic process\n",
      "    - lysosome organization\n",
      "    - lysosome organization\n",
      "    - locomotory behavior\n",
      "    - tissue development\n",
      "    - aorta development\n",
      "    - vacuolar sequestering\n",
      "    - muscle cell cellular homeostasis\n",
      "    - neuromuscular process controlling posture\n",
      "    - neuromuscular process controlling balance\n",
      "    - cardiac muscle contraction\n",
      "    - glycophagy\n",
      "  CC:\n",
      "    - lysosome\n",
      "    - lysosomal membrane\n",
      "    - plasma membrane\n",
      "    - membrane\n",
      "    - azurophil granule membrane\n",
      "    - lysosomal lumen\n",
      "    - intracellular membrane-bounded organelle\n",
      "    - extracellular exosome\n",
      "    - tertiary granule membrane\n",
      "    - ficolin-1-rich granule membrane\n",
      "    - autolysosome lumen\n",
      "  MF:\n",
      "    - alpha-1,4-glucosidase activity\n",
      "    - alpha-1,4-glucosidase activity\n",
      "    - carbohydrate binding\n",
      "    - alpha-glucosidase activity\n",
      "    - alpha-glucosidase activity\n",
      "UniProt: {'Swiss-Prot': 'P10253', 'TrEMBL': ['I3L0S5', 'I3L3L3', 'I3L2V9', 'A0AAQ5BHD3', 'A0AAQ5BHG9', 'A0AAQ5BHE2', 'A0AAG2UWL5', 'A0AAQ5BHE1', 'A0AAQ5BHE5']}\n",
      "\n",
      "Gene: ISOC1\n",
      "Symbol: ISOC1\n",
      "Name: isochorismatase domain containing 1\n",
      "Summary: Predicted to be located in peroxisome. Predicted to be active in cytoplasm. [provided by Alliance of Genome Resources, Apr 2022]\n",
      "GO Annotations:\n",
      "  CC:\n",
      "    - cytoplasm\n",
      "    - peroxisome\n",
      "  MF:\n",
      "    - protein binding\n",
      "UniProt: {'Swiss-Prot': 'Q96CN7', 'TrEMBL': 'D6RGE2'}\n",
      "\n",
      "Gene: PTGS2\n",
      "Symbol: PTGS2\n",
      "Name: prostaglandin-endoperoxide synthase 2\n",
      "Summary: Prostaglandin-endoperoxide synthase (PTGS), also known as cyclooxygenase, is the key enzyme in prostaglandin biosynthesis, and acts both as a dioxygenase and as a peroxidase. There are two isozymes of PTGS: a constitutive PTGS1 and an inducible PTGS2, which differ in their regulation of expression and tissue distribution. This gene encodes the inducible isozyme. It is regulated by specific stimulatory events, suggesting that it is responsible for the prostanoid biosynthesis involved in inflammation and mitogenesis. [provided by RefSeq, Feb 2009].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - prostaglandin biosynthetic process\n",
      "    - prostaglandin biosynthetic process\n",
      "    - prostaglandin biosynthetic process\n",
      "    - angiogenesis\n",
      "    - response to oxidative stress\n",
      "    - embryo implantation\n",
      "    - learning\n",
      "    - memory\n",
      "    - regulation of blood pressure\n",
      "    - negative regulation of cell population proliferation\n",
      "    - response to xenobiotic stimulus\n",
      "    - response to nematode\n",
      "    - response to fructose\n",
      "    - response to manganese ion\n",
      "    - response to selenium ion\n",
      "    - positive regulation of vascular endothelial growth factor production\n",
      "    - cyclooxygenase pathway\n",
      "    - cyclooxygenase pathway\n",
      "    - lipoxygenase pathway\n",
      "    - bone mineralization\n",
      "    - positive regulation of prostaglandin biosynthetic process\n",
      "    - positive regulation of fever generation\n",
      "    - positive regulation of synaptic plasticity\n",
      "    - negative regulation of synaptic transmission, dopaminergic\n",
      "    - prostaglandin secretion\n",
      "    - response to estradiol\n",
      "    - response to vitamin D\n",
      "    - cellular response to heat\n",
      "    - response to tumor necrosis factor\n",
      "    - maintenance of blood-brain barrier\n",
      "    - positive regulation of protein import into nucleus\n",
      "    - hair cycle\n",
      "    - long-chain fatty acid biosynthetic process\n",
      "    - positive regulation of apoptotic process\n",
      "    - positive regulation of nitric oxide biosynthetic process\n",
      "    - negative regulation of cell cycle\n",
      "    - positive regulation of vasoconstriction\n",
      "    - negative regulation of smooth muscle contraction\n",
      "    - positive regulation of smooth muscle contraction\n",
      "    - decidualization\n",
      "    - positive regulation of smooth muscle cell proliferation\n",
      "    - regulation of inflammatory response\n",
      "    - brown fat cell differentiation\n",
      "    - response to glucocorticoid\n",
      "    - negative regulation of calcium ion transport\n",
      "    - positive regulation of synaptic transmission, glutamatergic\n",
      "    - response to fatty acid\n",
      "    - cellular response to lipopolysaccharide\n",
      "    - cellular response to mechanical stimulus\n",
      "    - cellular response to lead ion\n",
      "    - cellular response to ATP\n",
      "    - cellular response to hypoxia\n",
      "    - cellular response to non-ionic osmotic stress\n",
      "    - cellular response to fluid shear stress\n",
      "    - positive regulation of transforming growth factor beta production\n",
      "    - positive regulation of cell migration involved in sprouting angiogenesis\n",
      "    - positive regulation of fibroblast growth factor production\n",
      "    - positive regulation of brown fat cell differentiation\n",
      "    - positive regulation of platelet-derived growth factor production\n",
      "    - cellular oxidant detoxification\n",
      "    - regulation of neuroinflammatory response\n",
      "    - negative regulation of intrinsic apoptotic signaling pathway in response to osmotic stress\n",
      "    - cellular response to homocysteine\n",
      "    - response to angiotensin\n",
      "  CC:\n",
      "    - nuclear inner membrane\n",
      "    - nuclear outer membrane\n",
      "    - cytoplasm\n",
      "    - cytoplasm\n",
      "    - cytoplasm\n",
      "    - endoplasmic reticulum\n",
      "    - endoplasmic reticulum lumen\n",
      "    - endoplasmic reticulum membrane\n",
      "    - cytosol\n",
      "    - caveola\n",
      "    - protein-containing complex\n",
      "    - neuron projection\n",
      "    - neuron projection\n",
      "    - intracellular membrane-bounded organelle\n",
      "  MF:\n",
      "    - peroxidase activity\n",
      "    - prostaglandin-endoperoxide synthase activity\n",
      "    - prostaglandin-endoperoxide synthase activity\n",
      "    - protein binding\n",
      "    - oxidoreductase activity, acting on single donors with incorporation of molecular oxygen\n",
      "    - oxidoreductase activity, acting on single donors with incorporation of molecular oxygen\n",
      "    - oxidoreductase activity, acting on single donors with incorporation of molecular oxygen, incorporation of two atoms of oxygen\n",
      "    - enzyme binding\n",
      "    - heme binding\n",
      "    - protein homodimerization activity\n",
      "    - metal ion binding\n",
      "UniProt: {'Swiss-Prot': 'P35354', 'TrEMBL': ['A0A7P0T828', 'Q6ZYK7']}\n",
      "\n",
      "Gene: SULT1A1\n",
      "Symbol: SULT1A1\n",
      "Name: sulfotransferase family 1A member 1\n",
      "Summary: Sulfotransferase enzymes catalyze the sulfate conjugation of many hormones, neurotransmitters, drugs, and xenobiotic compounds. These cytosolic enzymes are different in their tissue distributions and substrate specificities. The gene structure (number and length of exons) is similar among family members. This gene encodes one of two phenol sulfotransferases with thermostable enzyme activity.  Multiple alternatively spliced variants that encode two isoforms have been identified for this gene. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - ethanol catabolic process\n",
      "    - xenobiotic metabolic process\n",
      "    - estrogen metabolic process\n",
      "    - amine metabolic process\n",
      "    - flavonoid metabolic process\n",
      "    - dopamine catabolic process\n",
      "    - 3'-phosphoadenosine 5'-phosphosulfate metabolic process\n",
      "    - sulfation\n",
      "    - sulfation\n",
      "    - sulfation\n",
      "  CC:\n",
      "    - cytoplasm\n",
      "    - cytosol\n",
      "    - cytosol\n",
      "  MF:\n",
      "    - aryl sulfotransferase activity\n",
      "    - aryl sulfotransferase activity\n",
      "    - protein binding\n",
      "    - sulfotransferase activity\n",
      "    - flavonol 3-sulfotransferase activity\n",
      "    - steroid sulfotransferase activity\n",
      "    - 3'-phosphoadenosine 5'-phosphosulfate binding\n",
      "UniProt: {'Swiss-Prot': 'P50225', 'TrEMBL': ['H3BRY5', 'H3BQX5']}\n",
      "\n",
      "Gene: NPC1\n",
      "Symbol: NPC1\n",
      "Name: NPC intracellular cholesterol transporter 1\n",
      "Summary: This gene encodes a large protein that resides in the limiting membrane of endosomes and lysosomes and mediates intracellular cholesterol trafficking via binding of cholesterol to its N-terminal domain. It is predicted to have a cytoplasmic C-terminus, 13 transmembrane domains, and 3 large loops in the lumen of the endosome - the last loop being at the N-terminus. This protein transports low-density lipoproteins to late endosomal/lysosomal compartments where they are hydrolized and released as free cholesterol. Defects in this gene cause Niemann-Pick type C disease, a rare autosomal recessive neurodegenerative disorder characterized by over accumulation of cholesterol and glycosphingolipids in late endosomal/lysosomal compartments.[provided by RefSeq, Aug 2009].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - liver development\n",
      "    - protein glycosylation\n",
      "    - endocytosis\n",
      "    - autophagy\n",
      "    - lysosomal transport\n",
      "    - adult walking behavior\n",
      "    - cholesterol metabolic process\n",
      "    - bile acid metabolic process\n",
      "    - response to xenobiotic stimulus\n",
      "    - gene expression\n",
      "    - cholesterol storage\n",
      "    - programmed cell death\n",
      "    - sterol transport\n",
      "    - macroautophagy\n",
      "    - negative regulation of macroautophagy\n",
      "    - neurogenesis\n",
      "    - intestinal cholesterol absorption\n",
      "    - cholesterol transport\n",
      "    - membrane raft organization\n",
      "    - intracellular lipid transport\n",
      "    - intracellular cholesterol transport\n",
      "    - cholesterol efflux\n",
      "    - cholesterol homeostasis\n",
      "    - cholesterol homeostasis\n",
      "    - cholesterol homeostasis\n",
      "    - cholesterol homeostasis\n",
      "    - response to cadmium ion\n",
      "    - symbiont entry into host cell\n",
      "    - cellular response to steroid hormone stimulus\n",
      "    - cellular response to low-density lipoprotein particle stimulus\n",
      "    - establishment of protein localization to membrane\n",
      "    - negative regulation of epithelial cell apoptotic process\n",
      "    - negative regulation of TORC1 signaling\n",
      "    - cyclodextrin metabolic process\n",
      "  CC:\n",
      "    - extracellular region\n",
      "    - nuclear envelope\n",
      "    - lysosome\n",
      "    - lysosomal membrane\n",
      "    - lysosomal membrane\n",
      "    - lysosomal membrane\n",
      "    - lysosomal membrane\n",
      "    - endoplasmic reticulum\n",
      "    - Golgi apparatus\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - membrane\n",
      "    - late endosome membrane\n",
      "    - membrane raft\n",
      "    - perinuclear region of cytoplasm\n",
      "    - extracellular exosome\n",
      "  MF:\n",
      "    - virus receptor activity\n",
      "    - transmembrane signaling receptor activity\n",
      "    - protein binding\n",
      "    - cholesterol binding\n",
      "    - cholesterol binding\n",
      "    - signaling receptor activity\n",
      "    - cholesterol transfer activity\n",
      "UniProt: {'Swiss-Prot': 'O15118', 'TrEMBL': ['K7ERW2', 'K7ENZ0', 'K7ERM4', 'K7EQ23', 'K7EIH7']}\n",
      "\n",
      "Gene: FMO3\n",
      "Symbol: FMO3\n",
      "Name: flavin containing dimethylaniline monoxygenase 3\n",
      "Summary: Flavin-containing monooxygenases (FMO) are an important class of drug-metabolizing enzymes that catalyze the NADPH-dependent oxygenation of various nitrogen-,sulfur-, and phosphorous-containing xenobiotics such as therapeutic drugs, dietary compounds, pesticides, and other foreign compounds. The human FMO gene family is composed of 5 genes and multiple pseudogenes. FMO members have distinct developmental- and tissue-specific expression patterns. The expression of this FMO3 gene, the major FMO expressed in adult liver, can vary up to 20-fold between individuals. This inter-individual variation in FMO3 expression levels is likely to have significant effects on the rate at which xenobiotics are metabolised and, therefore, is of considerable interest to the pharmaceutical industry. This transmembrane protein localizes to the endoplasmic reticulum of many tissues. Alternative splicing of this gene results in multiple transcript variants encoding different isoforms. Mutations in this gene cause the disorder trimethylaminuria (TMAu) which is characterized by the accumulation and excretion of unmetabolized trimethylamine and a distinctive body odor. In healthy individuals, trimethylamine is primarily converted to the non odorous trimethylamine N-oxide.[provided by RefSeq, Jan 2016].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - taurine biosynthetic process\n",
      "  CC:\n",
      "    - endoplasmic reticulum membrane\n",
      "    - intracellular membrane-bounded organelle\n",
      "  MF:\n",
      "    - N,N-dimethylaniline monooxygenase activity\n",
      "    - protein binding\n",
      "    - trimethylamine monooxygenase activity\n",
      "    - albendazole monooxygenase activity\n",
      "    - hypotaurine monooxygenase activity\n",
      "    - flavin adenine dinucleotide binding\n",
      "    - NADP binding\n",
      "UniProt: {'Swiss-Prot': 'P31513', 'TrEMBL': ['B7Z543', 'A0A024R8Z4', 'Q53FW5', 'B7Z3M2', 'V9GYP3', 'V9GZ60']}\n",
      "\n",
      "Gene: CHST1\n",
      "Symbol: CHST1\n",
      "Name: carbohydrate sulfotransferase 1\n",
      "Summary: This locus encodes a member of the keratin sulfotransferase family of proteins. The encoded enzyme catalyzes the sulfation of the proteoglycan keratin. [provided by RefSeq, Aug 2011].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - polysaccharide metabolic process\n",
      "    - galactose metabolic process\n",
      "    - N-acetylglucosamine metabolic process\n",
      "    - sulfur compound metabolic process\n",
      "    - inflammatory response\n",
      "    - keratan sulfate proteoglycan biosynthetic process\n",
      "    - keratan sulfate proteoglycan biosynthetic process\n",
      "    - keratan sulfate proteoglycan metabolic process\n",
      "    - keratan sulfate proteoglycan metabolic process\n",
      "  CC:\n",
      "    - Golgi membrane\n",
      "    - membrane\n",
      "  MF:\n",
      "    - N-acetylglucosamine 6-O-sulfotransferase activity\n",
      "    - protein binding\n",
      "    - sulfotransferase activity\n",
      "    - keratan sulfotransferase activity\n",
      "    - keratan sulfotransferase activity\n",
      "    - keratan sulfotransferase activity\n",
      "UniProt: {'Swiss-Prot': 'O43916'}\n",
      "\n",
      "Gene: ZFP62\n",
      "Symbol: ZFP62\n",
      "Name: ZFP62 zinc finger protein\n",
      "Summary: Predicted to enable DNA-binding transcription activator activity, RNA polymerase II-specific and RNA polymerase II cis-regulatory region sequence-specific DNA binding activity. Predicted to be involved in regulation of transcription by RNA polymerase II. Predicted to be active in nucleus. [provided by Alliance of Genome Resources, Apr 2022]\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - regulation of transcription by RNA polymerase II\n",
      "  CC:\n",
      "    - nucleus\n",
      "  MF:\n",
      "    - zinc ion binding\n",
      "UniProt: {'Swiss-Prot': 'Q8NB50', 'TrEMBL': ['D6RCF6', 'D6R9C0', 'D6RBG3']}\n",
      "\n",
      "Gene: KIF13A\n",
      "Symbol: KIF13A\n",
      "Name: kinesin family member 13A\n",
      "Summary: This gene encodes a member of the kinesin family of microtubule-based motor proteins that function in the positioning of endosomes. This family member can direct mannose-6-phosphate receptor-containing vesicles from the trans-Golgi network to the plasma membrane, and it is necessary for the steady-state distribution of late endosomes/lysosomes. It is also required for the translocation of FYVE-CENT and TTC19 from the centrosome to the midbody during cytokinesis, and it plays a role in melanosome maturation. Alternative splicing of this gene results in multiple transcript variants. [provided by RefSeq, Aug 2011].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - intracellular protein transport\n",
      "    - intracellular protein transport\n",
      "    - microtubule-based movement\n",
      "    - endosome to lysosome transport\n",
      "    - melanosome organization\n",
      "    - regulation of cytokinesis\n",
      "    - vesicle cargo loading\n",
      "    - Golgi to plasma membrane protein transport\n",
      "    - cell division\n",
      "    - plus-end-directed vesicle transport along microtubule\n",
      "  CC:\n",
      "    - Golgi membrane\n",
      "    - cytoplasm\n",
      "    - centrosome\n",
      "    - kinesin complex\n",
      "    - microtubule\n",
      "    - endosome membrane\n",
      "    - midbody\n",
      "    - trans-Golgi network membrane\n",
      "  MF:\n",
      "    - microtubule motor activity\n",
      "    - microtubule motor activity\n",
      "    - protein binding\n",
      "    - ATP binding\n",
      "    - microtubule binding\n",
      "    - ATP hydrolysis activity\n",
      "UniProt: {'Swiss-Prot': 'Q9H1H9', 'TrEMBL': ['H0Y925', 'H0Y307', 'A0A1B0GUA8', 'H0Y9Z7']}\n",
      "\n",
      "Gene: GLUL\n",
      "Symbol: GLUL\n",
      "Name: glutamate-ammonia ligase\n",
      "Summary: The protein encoded by this gene belongs to the glutamine synthetase family. It catalyzes the synthesis of glutamine from glutamate and ammonia in an ATP-dependent reaction. This protein plays a role in ammonia and glutamate detoxification, acid-base homeostasis, cell signaling, and cell proliferation. Glutamine is an abundant amino acid, and is important to the biosynthesis of several amino acids, pyrimidines, and purines. Mutations in this gene are associated with congenital glutamine deficiency, and overexpression of this gene was observed in some primary liver cancer samples. There are six pseudogenes of this gene found on chromosomes 2, 5, 9, 11, and 12. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Dec 2014].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - angiogenesis\n",
      "    - glutamate catabolic process\n",
      "    - glutamine biosynthetic process\n",
      "    - glutamine biosynthetic process\n",
      "    - cell population proliferation\n",
      "    - cellular response to starvation\n",
      "    - response to glucose\n",
      "    - regulation of endothelial cell migration\n",
      "    - protein palmitoylation\n",
      "    - ribosome biogenesis\n",
      "    - positive regulation of erythrocyte differentiation\n",
      "    - intracellular ammonium homeostasis\n",
      "    - regulation of sprouting angiogenesis\n",
      "    - regulation of protein localization to nucleolus\n",
      "  CC:\n",
      "    - nucleus\n",
      "    - cytoplasm\n",
      "    - mitochondrion\n",
      "    - endoplasmic reticulum\n",
      "    - cytosol\n",
      "    - cytosol\n",
      "    - plasma membrane\n",
      "    - cell body\n",
      "    - extracellular exosome\n",
      "    - glial cell projection\n",
      "  MF:\n",
      "    - glutamine synthetase activity\n",
      "    - glutamine synthetase activity\n",
      "    - protein binding\n",
      "    - ATP binding\n",
      "    - protein-cysteine S-palmitoyltransferase activity\n",
      "    - identical protein binding\n",
      "    - metal ion binding\n",
      "UniProt: {'Swiss-Prot': 'P15104', 'TrEMBL': 'A8YXX4'}\n",
      "\n",
      "Gene: SLC5A1\n",
      "Symbol: SLC5A1\n",
      "Name: solute carrier family 5 member 1\n",
      "Summary: This gene encodes a member of the sodium-dependent glucose transporter (SGLT) family. The encoded integral membrane protein is the primary mediator of dietary glucose and galactose uptake from the intestinal lumen. Mutations in this gene have been associated with glucose-galactose malabsorption. Multiple transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Jan 2012].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - alpha-glucoside transport\n",
      "    - alpha-glucoside transport\n",
      "    - alpha-glucoside transport\n",
      "    - intestinal D-glucose absorption\n",
      "    - sodium ion transport\n",
      "    - pentose transmembrane transport\n",
      "    - fucose transmembrane transport\n",
      "    - galactose transmembrane transport\n",
      "    - myo-inositol transport\n",
      "    - transepithelial water transport\n",
      "    - renal D-glucose absorption\n",
      "    - D-glucose import across plasma membrane\n",
      "    - D-glucose import across plasma membrane\n",
      "    - sodium ion import across plasma membrane\n",
      "    - intestinal hexose absorption\n",
      "    - transport across blood-brain barrier\n",
      "    - transport across blood-brain barrier\n",
      "    - chloride transmembrane transport\n",
      "    - D-glucose transmembrane transport\n",
      "    - D-glucose transmembrane transport\n",
      "  CC:\n",
      "    - early endosome\n",
      "    - Golgi apparatus\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - apical plasma membrane\n",
      "    - apical plasma membrane\n",
      "    - brush border membrane\n",
      "    - nuclear membrane\n",
      "    - intracellular organelle\n",
      "    - perinuclear region of cytoplasm\n",
      "    - extracellular exosome\n",
      "    - intracellular vesicle\n",
      "  MF:\n",
      "    - galactose transmembrane transporter activity\n",
      "    - myo-inositol:sodium symporter activity\n",
      "    - water transmembrane transporter activity\n",
      "    - D-glucose:sodium symporter activity\n",
      "    - D-glucose:sodium symporter activity\n",
      "    - D-glucose:sodium symporter activity\n",
      "    - protein binding\n",
      "    - pentose transmembrane transporter activity\n",
      "    - fucose transmembrane transporter activity\n",
      "    - alpha-glucoside transmembrane transporter activity\n",
      "    - alpha-glucoside transmembrane transporter activity\n",
      "    - alpha-glucoside transmembrane transporter activity\n",
      "    - galactose:sodium symporter activity\n",
      "    - D-glucose transmembrane transporter activity\n",
      "UniProt: {'Swiss-Prot': 'P13866'}\n",
      "\n",
      "Gene: UGT2B11\n",
      "Symbol: UGT2B11\n",
      "Name: UDP glucuronosyltransferase family 2 member B11\n",
      "Summary: Enables glucuronosyltransferase activity. Involved in estrogen metabolic process and xenobiotic glucuronidation. Predicted to be located in endoplasmic reticulum membrane. Predicted to be integral component of membrane. Predicted to be active in intracellular membrane-bounded organelle. [provided by Alliance of Genome Resources, Apr 2022]\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - xenobiotic metabolic process\n",
      "    - estrogen metabolic process\n",
      "    - estrogen metabolic process\n",
      "  CC:\n",
      "    - endoplasmic reticulum membrane\n",
      "  MF:\n",
      "    - protein binding\n",
      "    - glucuronosyltransferase activity\n",
      "    - glucuronosyltransferase activity\n",
      "UniProt: {'Swiss-Prot': 'O75310'}\n",
      "\n",
      "Gene: CACNA1C\n",
      "Symbol: CACNA1C\n",
      "Name: calcium voltage-gated channel subunit alpha1 C\n",
      "Summary: This gene encodes an alpha-1 subunit of a voltage-dependent calcium channel. Calcium channels mediate the influx of calcium ions into the cell upon membrane polarization. The alpha-1 subunit consists of 24 transmembrane segments and forms the pore through which ions pass into the cell. The calcium channel consists of a complex of alpha-1, alpha-2/delta, beta, and gamma subunits in a 1:1:1:1 ratio. There are multiple isoforms of each of these proteins, either encoded by different genes or the result of alternative splicing of transcripts. The protein encoded by this gene binds to and is inhibited by dihydropyridine. Alternative splicing results in many transcript variants encoding different proteins. Some of the predicted proteins may not produce functional ion channel subunits. [provided by RefSeq, Oct 2012].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - immune system development\n",
      "    - positive regulation of cytosolic calcium ion concentration\n",
      "    - heart development\n",
      "    - regulation of cardiac muscle contraction by regulation of the release of sequestered calcium ion\n",
      "    - regulation of cardiac muscle contraction by regulation of the release of sequestered calcium ion\n",
      "    - embryonic forelimb morphogenesis\n",
      "    - camera-type eye development\n",
      "    - positive regulation of adenylate cyclase activity\n",
      "    - positive regulation of muscle contraction\n",
      "    - calcium ion transport into cytosol\n",
      "    - calcium ion transport into cytosol\n",
      "    - cardiac conduction\n",
      "    - calcium ion transmembrane transport via high voltage-gated calcium channel\n",
      "    - calcium ion transmembrane transport via high voltage-gated calcium channel\n",
      "    - calcium ion transmembrane transport\n",
      "    - calcium ion transmembrane transport\n",
      "    - calcium ion transmembrane transport\n",
      "    - calcium ion transmembrane transport\n",
      "    - cardiac muscle cell action potential involved in contraction\n",
      "    - membrane depolarization during cardiac muscle cell action potential\n",
      "    - membrane depolarization during AV node cell action potential\n",
      "    - cell communication by electrical coupling involved in cardiac conduction\n",
      "    - regulation of heart rate by cardiac conduction\n",
      "    - calcium ion import across plasma membrane\n",
      "    - regulation of ventricular cardiac muscle cell action potential\n",
      "    - membrane depolarization during atrial cardiac muscle cell action potential\n",
      "  CC:\n",
      "    - nucleoplasm\n",
      "    - cytoplasm\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - voltage-gated calcium channel complex\n",
      "    - voltage-gated calcium channel complex\n",
      "    - cilium\n",
      "    - postsynaptic density\n",
      "    - membrane\n",
      "    - Z disc\n",
      "    - dendrite\n",
      "    - perikaryon\n",
      "    - postsynaptic density membrane\n",
      "    - L-type voltage-gated calcium channel complex\n",
      "    - L-type voltage-gated calcium channel complex\n",
      "  MF:\n",
      "    - voltage-gated calcium channel activity\n",
      "    - voltage-gated calcium channel activity\n",
      "    - protein binding\n",
      "    - calmodulin binding\n",
      "    - high voltage-gated calcium channel activity\n",
      "    - high voltage-gated calcium channel activity\n",
      "    - high voltage-gated calcium channel activity\n",
      "    - metal ion binding\n",
      "    - alpha-actinin binding\n",
      "    - voltage-gated calcium channel activity involved in cardiac muscle cell action potential\n",
      "    - voltage-gated calcium channel activity involved in cardiac muscle cell action potential\n",
      "    - voltage-gated calcium channel activity involved in AV node cell action potential\n",
      "UniProt: {'Swiss-Prot': 'Q13936', 'TrEMBL': ['A0A0A0MR67', 'A0A804HIJ8', 'F5H522', 'A0A804HKC4', 'A0A804HJB6', 'A0A0A0MSA1', 'A0A804HI37', 'E9PDI6', 'Q59GU3', 'Q5V9X9', 'A0A804HIZ0', 'A0A804HKE9', 'A0A804HJR1', 'A0A804HIR0', 'F5GY28', 'F5H638', 'A0AA34QVT1', 'A0A087WUX4', 'A0A5F9ZHD9', 'A0A804HHT8', 'Q5V9X8']}\n",
      "\n",
      "Gene: AKR1C3\n",
      "Symbol: AKR1C3\n",
      "Name: aldo-keto reductase family 1 member C3\n",
      "Summary: This gene encodes a member of the aldo/keto reductase superfamily, which consists of more than 40 known enzymes and proteins. These enzymes catalyze the conversion of aldehydes and ketones to their corresponding alcohols by utilizing NADH and/or NADPH as cofactors. The enzymes display overlapping but distinct substrate specificity. This enzyme catalyzes the reduction of prostaglandin (PG) D2, PGH2 and phenanthrenequinone (PQ), and the oxidation of 9alpha,11beta-PGF2 to PGD2. It may play an important role in the pathogenesis of allergic diseases such as asthma, and may also have a role in controlling cell growth and/or differentiation. This gene shares high sequence identity with three other gene members and is clustered with those three genes at chromosome 10p15-p14. Three transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Dec 2011].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - retinoid metabolic process\n",
      "    - prostaglandin metabolic process\n",
      "    - prostaglandin metabolic process\n",
      "    - prostaglandin metabolic process\n",
      "    - G protein-coupled receptor signaling pathway\n",
      "    - response to nutrient\n",
      "    - steroid metabolic process\n",
      "    - positive regulation of cell population proliferation\n",
      "    - positive regulation of cell population proliferation\n",
      "    - male gonad development\n",
      "    - cellular response to starvation\n",
      "    - farnesol catabolic process\n",
      "    - keratinocyte differentiation\n",
      "    - progesterone metabolic process\n",
      "    - progesterone metabolic process\n",
      "    - retinol metabolic process\n",
      "    - retinal metabolic process\n",
      "    - macromolecule metabolic process\n",
      "    - daunorubicin metabolic process\n",
      "    - daunorubicin metabolic process\n",
      "    - doxorubicin metabolic process\n",
      "    - doxorubicin metabolic process\n",
      "    - prostanoid biosynthetic process\n",
      "    - regulation of retinoic acid receptor signaling pathway\n",
      "    - positive regulation of phosphatidylinositol 3-kinase/protein kinase B signal transduction\n",
      "    - testosterone biosynthetic process\n",
      "    - renal absorption\n",
      "    - cellular response to calcium ion\n",
      "    - cellular response to prostaglandin stimulus\n",
      "    - cellular response to corticosteroid stimulus\n",
      "    - cellular response to jasmonic acid stimulus\n",
      "    - cellular response to prostaglandin D stimulus\n",
      "    - negative regulation of retinoic acid biosynthetic process\n",
      "    - regulation of testosterone biosynthetic process\n",
      "    - positive regulation of endothelial cell apoptotic process\n",
      "    - positive regulation of reactive oxygen species metabolic process\n",
      "  CC:\n",
      "    - nucleus\n",
      "    - cytoplasm\n",
      "    - cytosol\n",
      "    - cytosol\n",
      "    - extracellular exosome\n",
      "  MF:\n",
      "    - retinal dehydrogenase activity\n",
      "    - aldose reductase (NADPH) activity\n",
      "    - aldose reductase (NADPH) activity\n",
      "    - aldo-keto reductase (NADPH) activity\n",
      "    - aldo-keto reductase (NADPH) activity\n",
      "    - estradiol 17-beta-dehydrogenase [NAD(P)+] activity\n",
      "    - all-trans-retinol dehydrogenase (NAD+) activity\n",
      "    - protein binding\n",
      "    - oxidoreductase activity, acting on NAD(P)H, quinone or similar compound as acceptor\n",
      "    - testosterone dehydrogenase [NAD(P)+] activity\n",
      "    - bile acid binding\n",
      "    - prostaglandin H2 endoperoxidase reductase activity\n",
      "    - prostaglandin D2 11-ketoreductase activity\n",
      "    - geranylgeranyl reductase activity\n",
      "    - ketoreductase activity\n",
      "    - prostaglandin F synthase activity\n",
      "    - 15-hydroxyprostaglandin-D dehydrogenase (NADP+) activity\n",
      "    - androsterone dehydrogenase [NAD(P)+] activity\n",
      "    - androsterone dehydrogenase [NAD(P)+] activity\n",
      "    - 5-alpha-androstane-3-beta,17-beta-diol dehydrogenase (NADP+) activity\n",
      "    - testosterone dehydrogenase (NAD+) activity\n",
      "    - androstan-3-alpha,17-beta-diol dehydrogenase (NAD+) activity\n",
      "    - testosterone 17-beta-dehydrogenase (NADP+) activity\n",
      "    - ketosteroid monooxygenase activity\n",
      "    - ketosteroid monooxygenase activity\n",
      "    - Delta4-3-oxosteroid 5beta-reductase activity\n",
      "    - all-trans-retinol dehydrogenase (NADP+) activity\n",
      "UniProt: {'Swiss-Prot': 'P42330', 'TrEMBL': ['B4DKT3', 'A0A0A0MSS8', 'S4R3Z2', 'S4R3D5']}\n",
      "\n",
      "Gene: CYP2S1\n",
      "Symbol: CYP2S1\n",
      "Name: cytochrome P450 family 2 subfamily S member 1\n",
      "Summary: This gene encodes a member of the cytochrome P450 superfamily of enzymes. The cytochrome P450 proteins are monooxygenases which catalyze many reactions involved in drug metabolism and synthesis of cholesterol, steroids and other lipids. This protein localizes to the endoplasmic reticulum. In rodents, the homologous protein has been shown to metabolize certain carcinogens; however, the specific function of the human protein has not been determined. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - icosanoid metabolic process\n",
      "    - prostaglandin metabolic process\n",
      "    - xenobiotic metabolic process\n",
      "    - epoxygenase P450 pathway\n",
      "    - retinoic acid metabolic process\n",
      "    - cytochrome metabolic process\n",
      "  CC:\n",
      "    - cytoplasm\n",
      "    - endoplasmic reticulum\n",
      "    - endoplasmic reticulum membrane\n",
      "    - intracellular membrane-bounded organelle\n",
      "  MF:\n",
      "    - monooxygenase activity\n",
      "    - thromboxane-A synthase activity\n",
      "    - iron ion binding\n",
      "    - protein binding\n",
      "    - arachidonate epoxygenase activity\n",
      "    - retinoic acid 4-hydroxylase activity\n",
      "    - oxidoreductase activity, acting on paired donors, with incorporation or reduction of molecular oxygen, reduced flavin or flavoprotein as one donor, and incorporation of one atom of oxygen\n",
      "    - hydro-lyase activity\n",
      "    - heme binding\n",
      "    - 12-hydroxyheptadecatrienoic acid synthase activity\n",
      "    - hydroperoxy icosatetraenoate dehydratase activity\n",
      "UniProt: {'Swiss-Prot': 'Q96SQ9', 'TrEMBL': ['M0R2G8', 'M0R057', 'M0QYI2', 'M0R152', 'M0QY51']}\n",
      "\n",
      "Gene: RYR3\n",
      "Symbol: RYR3\n",
      "Name: ryanodine receptor 3\n",
      "Summary: The protein encoded by this gene is a ryanodine receptor, which functions to release calcium from intracellular storage for use in many cellular processes. For example, the encoded protein is involved in skeletal muscle contraction by releasing calcium from the sarcoplasmic reticulum followed by depolarization of T-tubules. Two transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Sep 2011].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - calcium ion transport\n",
      "    - intracellular calcium ion homeostasis\n",
      "    - striated muscle contraction\n",
      "    - release of sequestered calcium ion into cytosol by sarcoplasmic reticulum\n",
      "    - calcium-mediated signaling\n",
      "    - release of sequestered calcium ion into cytosol\n",
      "    - protein homotetramerization\n",
      "    - calcium ion transmembrane transport\n",
      "    - cellular response to calcium ion\n",
      "    - cellular response to magnesium ion\n",
      "    - cellular response to caffeine\n",
      "    - cellular response to ATP\n",
      "  CC:\n",
      "    - smooth endoplasmic reticulum\n",
      "    - membrane\n",
      "    - Z disc\n",
      "    - sarcoplasmic reticulum membrane\n",
      "    - sarcoplasmic reticulum membrane\n",
      "    - sarcoplasmic reticulum membrane\n",
      "    - sarcoplasmic reticulum membrane\n",
      "    - calcium channel complex\n",
      "    - sarcolemma\n",
      "  MF:\n",
      "    - ryanodine-sensitive calcium-release channel activity\n",
      "    - ryanodine-sensitive calcium-release channel activity\n",
      "    - ryanodine-sensitive calcium-release channel activity\n",
      "    - calcium ion binding\n",
      "    - calmodulin binding\n",
      "    - intracellularly gated calcium channel activity\n",
      "    - calcium-induced calcium release activity\n",
      "UniProt: {'Swiss-Prot': 'Q15413', 'TrEMBL': ['A0A1B0GTT7', 'A0A0U1RRH1', 'A0A1B0GU27', 'A0A0U1RR54', 'A0A1B0GTF2', 'A0A0X1KG73', 'A0A1B0GVA9', 'A0A1B0GUB0', 'A0A0U1RR21', 'A0A1B0GVS2']}\n",
      "\n",
      "Gene: F5\n",
      "Symbol: F5\n",
      "Name: coagulation factor V\n",
      "Summary: This gene encodes an essential cofactor of the blood coagulation cascade. This factor circulates in plasma, and is converted to the active form by the release of the activation peptide by thrombin during coagulation. This generates a heavy chain and a light chain which are held together by calcium ions. The activated protein is a cofactor that participates with activated coagulation factor X to activate prothrombin to thrombin. Defects in this gene result in either an autosomal recessive hemorrhagic diathesis or an autosomal dominant form of thrombophilia, which is known as activated protein C resistance. [provided by RefSeq, Oct 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - blood coagulation\n",
      "    - blood circulation\n",
      "    - response to vitamin K\n",
      "  CC:\n",
      "    - extracellular region\n",
      "    - extracellular region\n",
      "    - extracellular space\n",
      "    - endoplasmic reticulum lumen\n",
      "    - plasma membrane\n",
      "    - membrane\n",
      "    - COPII-coated ER to Golgi transport vesicle\n",
      "    - platelet alpha granule\n",
      "    - platelet alpha granule lumen\n",
      "    - endoplasmic reticulum-Golgi intermediate compartment membrane\n",
      "    - extracellular vesicle\n",
      "  MF:\n",
      "    - copper ion binding\n",
      "    - protein binding\n",
      "UniProt: {'Swiss-Prot': 'P12259', 'TrEMBL': 'A0A0A0MRJ7'}\n",
      "\n",
      "Gene: SLC6A1\n",
      "Symbol: SLC6A1\n",
      "Name: solute carrier family 6 member 1\n",
      "Summary: The protein encoded by this gene is a gamma-aminobutyric acid (GABA) transporter that localizes to the plasma membrane. The encoded protein removes GABA from the synaptic cleft, restoring it to presynaptic terminals. [provided by RefSeq, Jan 2017].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - amino acid transport\n",
      "    - chemical synaptic transmission\n",
      "    - memory\n",
      "    - associative learning\n",
      "    - response to toxic substance\n",
      "    - response to sucrose\n",
      "    - response to lead ion\n",
      "    - positive regulation of gamma-aminobutyric acid secretion\n",
      "    - response to purine-containing compound\n",
      "    - negative regulation of synaptic transmission, GABAergic\n",
      "    - response to estradiol\n",
      "    - sodium ion transmembrane transport\n",
      "    - response to cocaine\n",
      "    - synapse organization\n",
      "    - response to calcium ion\n",
      "    - gamma-aminobutyric acid reuptake\n",
      "    - gamma-aminobutyric acid import\n",
      "    - gamma-aminobutyric acid import\n",
      "    - inorganic anion import across plasma membrane\n",
      "    - sodium ion import across plasma membrane\n",
      "    - transport across blood-brain barrier\n",
      "    - chloride transmembrane transport\n",
      "    - response to methamphetamine hydrochloride\n",
      "  CC:\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - cell surface\n",
      "    - membrane\n",
      "    - axon\n",
      "    - axon\n",
      "    - presynaptic membrane\n",
      "    - neuronal cell body\n",
      "    - postsynaptic membrane\n",
      "    - GABA-ergic synapse\n",
      "  MF:\n",
      "    - gamma-aminobutyric acid:sodium:chloride symporter activity\n",
      "    - gamma-aminobutyric acid:sodium:chloride symporter activity\n",
      "    - gamma-aminobutyric acid:sodium:chloride symporter activity\n",
      "    - gamma-aminobutyric acid:sodium:chloride symporter activity\n",
      "    - protein binding\n",
      "    - gamma-aminobutyric acid transmembrane transporter activity\n",
      "    - sodium:chloride symporter activity\n",
      "    - identical protein binding\n",
      "    - metal ion binding\n",
      "UniProt: {'Swiss-Prot': 'P30531', 'TrEMBL': ['A0A2R8Y4I3', 'B7Z3C5', 'A0A2R8Y495', 'A0A2R8YF74', 'C9J5P8', 'A0A2R8YCM3', 'A0A2R8YEF0', 'A0A2R8Y3X2', 'A0A2R8Y867', 'A0A2R8YFI0', 'A0A8V8TMZ9', 'A0A2R8YDD5']}\n",
      "\n",
      "Gene: S1PR2\n",
      "Symbol: S1PR2\n",
      "Name: sphingosine-1-phosphate receptor 2\n",
      "Summary: This gene encodes a member of the G protein-coupled receptors, as well as the EDG family of proteins. The encoded protein is a receptor for sphingosine 1-phosphate, which participates in cell proliferation, survival, and transcriptional activation. Defects in this gene have been associated with congenital profound deafness. [provided by RefSeq, Mar 2016].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - sphingosine-1-phosphate receptor signaling pathway\n",
      "    - sphingosine-1-phosphate receptor signaling pathway\n",
      "    - G protein-coupled receptor signaling pathway\n",
      "    - adenylate cyclase-activating G protein-coupled receptor signaling pathway\n",
      "    - positive regulation of cell population proliferation\n",
      "    - positive regulation of peptidyl-threonine phosphorylation\n",
      "    - regulation of metabolic process\n",
      "    - actin cytoskeleton organization\n",
      "    - filopodium assembly\n",
      "    - excitatory postsynaptic potential\n",
      "    - negative regulation of excitatory postsynaptic potential\n",
      "    - regulation of postsynapse assembly\n",
      "    - positive regulation of establishment of endothelial barrier\n",
      "  CC:\n",
      "    - cytoplasm\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - presynapse\n",
      "    - postsynapse\n",
      "    - glutamatergic synapse\n",
      "  MF:\n",
      "    - G protein-coupled receptor binding\n",
      "    - G protein-coupled receptor activity\n",
      "    - integrin binding\n",
      "    - protein binding\n",
      "    - lipid binding\n",
      "    - G protein-coupled peptide receptor activity\n",
      "    - sphingosine-1-phosphate receptor activity\n",
      "    - sphingosine-1-phosphate receptor activity\n",
      "    - sphingosine-1-phosphate receptor activity\n",
      "UniProt: {'Swiss-Prot': 'O95136'}\n",
      "\n",
      "Gene: CA12\n",
      "Symbol: CA12\n",
      "Name: carbonic anhydrase 12\n",
      "Summary: Carbonic anhydrases (CAs) are a large family of zinc metalloenzymes that catalyze the reversible hydration of carbon dioxide. They participate in a variety of biological processes, including respiration, calcification, acid-base balance, bone resorption, and the formation of aqueous humor, cerebrospinal fluid, saliva, and gastric acid. This gene product is a type I membrane protein that is highly expressed in normal tissues, such as kidney, colon and pancreas, and has been found to be overexpressed in 10% of clear cell renal carcinomas. Three transcript variants encoding different isoforms have been identified for this gene. [provided by RefSeq, Jun 2014].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - estrous cycle\n",
      "    - chloride ion homeostasis\n",
      "  CC:\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - membrane\n",
      "    - basolateral plasma membrane\n",
      "    - apical plasma membrane\n",
      "  MF:\n",
      "    - carbonate dehydratase activity\n",
      "    - zinc ion binding\n",
      "UniProt: {'Swiss-Prot': 'O43570', 'TrEMBL': 'B3KUB4'}\n",
      "\n",
      "Gene: SLC22A1\n",
      "Symbol: SLC22A1\n",
      "Name: solute carrier family 22 member 1\n",
      "Summary: Polyspecific organic cation transporters in the liver, kidney, intestine, and other organs are critical for elimination of many endogenous small organic cations as well as a wide array of drugs and environmental toxins. This gene is one of three similar cation transporter genes located in a cluster on chromosome 6. The encoded protein contains twelve putative transmembrane domains and is a plasma integral membrane protein. Two transcript variants encoding two different isoforms have been found for this gene, but only the longer variant encodes a functional transporter. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - xenobiotic metabolic process\n",
      "    - neurotransmitter transport\n",
      "    - serotonin transport\n",
      "    - establishment or maintenance of transmembrane electrochemical gradient\n",
      "    - organic cation transport\n",
      "    - organic cation transport\n",
      "    - organic cation transport\n",
      "    - quaternary ammonium group transport\n",
      "    - prostaglandin transport\n",
      "    - monoamine transport\n",
      "    - putrescine transport\n",
      "    - spermidine transport\n",
      "    - acetylcholine transport\n",
      "    - dopamine transport\n",
      "    - norepinephrine transport\n",
      "    - thiamine transport\n",
      "    - xenobiotic transport\n",
      "    - xenobiotic transport\n",
      "    - epinephrine transport\n",
      "    - serotonin uptake\n",
      "    - norepinephrine uptake\n",
      "    - thiamine transmembrane transport\n",
      "    - metanephric proximal tubule development\n",
      "    - purine-containing compound transmembrane transport\n",
      "    - dopamine uptake\n",
      "    - monoatomic cation transmembrane transport\n",
      "    - transport across blood-brain barrier\n",
      "    - (R)-carnitine transmembrane transport\n",
      "    - O-acyl-L-carnitine transmembrane transport\n",
      "    - spermidine transmembrane transport\n",
      "    - cellular detoxification\n",
      "    - cellular detoxification\n",
      "    - xenobiotic transport across blood-brain barrier\n",
      "  CC:\n",
      "    - plasma membrane\n",
      "    - basal plasma membrane\n",
      "    - membrane\n",
      "    - basolateral plasma membrane\n",
      "    - basolateral plasma membrane\n",
      "    - apical plasma membrane\n",
      "    - lateral plasma membrane\n",
      "    - presynapse\n",
      "  MF:\n",
      "    - acetylcholine transmembrane transporter activity\n",
      "    - neurotransmitter transmembrane transporter activity\n",
      "    - dopamine:sodium symporter activity\n",
      "    - norepinephrine:sodium symporter activity\n",
      "    - protein binding\n",
      "    - monoamine transmembrane transporter activity\n",
      "    - organic anion transmembrane transporter activity\n",
      "    - organic cation transmembrane transporter activity\n",
      "    - organic cation transmembrane transporter activity\n",
      "    - organic cation transmembrane transporter activity\n",
      "    - organic cation transmembrane transporter activity\n",
      "    - prostaglandin transmembrane transporter activity\n",
      "    - pyrimidine nucleoside transmembrane transporter activity\n",
      "    - thiamine transmembrane transporter activity\n",
      "    - putrescine transmembrane transporter activity\n",
      "    - spermidine transmembrane transporter activity\n",
      "    - quaternary ammonium group transmembrane transporter activity\n",
      "    - toxin transmembrane transporter activity\n",
      "    - toxin transmembrane transporter activity\n",
      "    - identical protein binding\n",
      "    - xenobiotic transmembrane transporter activity\n",
      "    - xenobiotic transmembrane transporter activity\n",
      "    - (R)-carnitine transmembrane transporter activity\n",
      "UniProt: {'Swiss-Prot': 'O15245', 'TrEMBL': ['F5H5P3', 'F5GY86']}\n",
      "\n",
      "Gene: FABP1\n",
      "Symbol: FABP1\n",
      "Name: fatty acid binding protein 1\n",
      "Summary: This gene encodes the fatty acid binding protein found in liver. Fatty acid binding proteins are a family of small, highly conserved, cytoplasmic proteins that bind long-chain fatty acids and other hydrophobic ligands. This protein and FABP6 (the ileal fatty acid binding protein) are also able to bind bile acids. It is thought that FABPs roles include fatty acid uptake, transport, and metabolism. [provided by RefSeq, Mar 2011].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - fatty acid transport\n",
      "    - long-chain fatty acid transport\n",
      "    - positive regulation of fatty acid beta-oxidation\n",
      "    - response to vitamin B3\n",
      "    - intestinal absorption\n",
      "    - cellular response to hydrogen peroxide\n",
      "    - cellular response to hypoxia\n",
      "    - cellular oxidant detoxification\n",
      "    - cellular detoxification\n",
      "  CC:\n",
      "    - nucleus\n",
      "    - nucleoplasm\n",
      "    - nucleoplasm\n",
      "    - peroxisomal matrix\n",
      "    - cytosol\n",
      "    - cytosol\n",
      "    - cytosol\n",
      "    - protein-containing complex\n",
      "    - apical cortex\n",
      "    - extracellular exosome\n",
      "  MF:\n",
      "    - chromatin binding\n",
      "    - long-chain fatty acid transmembrane transporter activity\n",
      "    - fatty acid binding\n",
      "    - protein binding\n",
      "    - phospholipid binding\n",
      "    - antioxidant activity\n",
      "    - bile acid binding\n",
      "    - oleic acid binding\n",
      "    - heterocyclic compound binding\n",
      "UniProt: {'Swiss-Prot': 'P07148', 'TrEMBL': ['Q05CP7', 'Q6FGL7', 'A8MW49']}\n",
      "\n",
      "Gene: HSD11B1\n",
      "Symbol: HSD11B1\n",
      "Name: hydroxysteroid 11-beta dehydrogenase 1\n",
      "Summary: The protein encoded by this gene is a microsomal enzyme that catalyzes the conversion of the stress hormone cortisol to the inactive metabolite cortisone. In addition, the encoded protein can catalyze the reverse reaction, the conversion of cortisone to cortisol. Too much cortisol can lead to central obesity, and a particular variation in this gene has been associated with obesity and insulin resistance in children. Mutations in this gene and H6PD (hexose-6-phosphate dehydrogenase (glucose 1-dehydrogenase)) are the cause of cortisone reductase deficiency. Alternate splicing results in multiple transcript variants encoding the same protein.[provided by RefSeq, May 2011].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - steroid catabolic process\n",
      "    - lung development\n",
      "  CC:\n",
      "    - endoplasmic reticulum membrane\n",
      "    - endoplasmic reticulum membrane\n",
      "    - endoplasmic reticulum membrane\n",
      "    - membrane\n",
      "    - intracellular membrane-bounded organelle\n",
      "  MF:\n",
      "    - steroid binding\n",
      "    - protein homodimerization activity\n",
      "    - 7-beta-hydroxysteroid dehydrogenase (NADP+) activity\n",
      "    - NADP binding\n",
      "    - 11-beta-hydroxysteroid dehydrogenase (NADP+) activity\n",
      "    - 11-beta-hydroxysteroid dehydrogenase (NADP+) activity\n",
      "    - cortisol dehydrogenase (NAD+) activity\n",
      "UniProt: {'Swiss-Prot': 'P28845', 'TrEMBL': ['A0A087WU76', 'X5D2L1', 'A0A0A0MQV1']}\n",
      "\n",
      "Gene: RNF43\n",
      "Symbol: RNF43\n",
      "Name: ring finger protein 43\n",
      "Summary: The protein encoded by this gene is a RING-type E3 ubiquitin ligase and is predicted to contain a transmembrane domain, a protease-associated domain, an ectodomain, and a cytoplasmic RING domain. This protein is thought to negatively regulate Wnt signaling, and expression of this gene results in an increase in ubiquitination of frizzled receptors, an alteration in their subcellular distribution, resulting in reduced surface levels of these receptors. Mutations in this gene have been reported in multiple tumor cells, including colorectal and endometrial cancers. Alternative splicing results in multiple transcript variants encoding different isoforms. [provided by RefSeq, Mar 2015].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - ubiquitin-dependent protein catabolic process\n",
      "    - ubiquitin-dependent protein catabolic process\n",
      "    - Wnt signaling pathway\n",
      "    - protein ubiquitination\n",
      "    - protein ubiquitination\n",
      "    - negative regulation of Wnt signaling pathway\n",
      "    - Wnt receptor catabolic process\n",
      "    - Wnt receptor catabolic process\n",
      "    - Wnt receptor catabolic process\n",
      "    - stem cell proliferation\n",
      "  CC:\n",
      "    - nuclear envelope\n",
      "    - endoplasmic reticulum membrane\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "  MF:\n",
      "    - ubiquitin-protein transferase activity\n",
      "    - frizzled binding\n",
      "    - frizzled binding\n",
      "    - protein binding\n",
      "    - zinc ion binding\n",
      "    - ubiquitin protein ligase activity\n",
      "    - ubiquitin protein ligase activity\n",
      "UniProt: {'Swiss-Prot': 'Q68DV7', 'TrEMBL': 'J3KSE3'}\n",
      "\n",
      "Gene: IMP4\n",
      "Symbol: IMP4\n",
      "Name: IMP U3 small nucleolar ribonucleoprotein 4\n",
      "Summary: The protein encoded by this gene, along with IMP3 and MPP10, is part of the 60-80S U3 small nucleolar ribonucleoprotein (U3 snoRNP) complex. This complex is necessary for the early cleavage steps of pre-18S ribosomal RNA processing. Several transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Feb 2016].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - rRNA processing\n",
      "    - rRNA processing\n",
      "    - maturation of SSU-rRNA\n",
      "    - ribosomal small subunit biogenesis\n",
      "  CC:\n",
      "    - fibrillar center\n",
      "    - nucleoplasm\n",
      "    - nucleolus\n",
      "    - nucleolus\n",
      "    - nucleolus\n",
      "    - preribosome\n",
      "    - small-subunit processome\n",
      "    - small-subunit processome\n",
      "    - Mpp10 complex\n",
      "    - Mpp10 complex\n",
      "    - Mpp10 complex\n",
      "  MF:\n",
      "    - protein binding\n",
      "    - snoRNA binding\n",
      "    - rRNA primary transcript binding\n",
      "UniProt: {'Swiss-Prot': 'Q96G21', 'TrEMBL': ['F8WCN0', 'E7ENR5', 'B8ZZ47', 'H0Y714', 'B9A008']}\n",
      "\n",
      "Gene: ZNF699\n",
      "Symbol: ZNF699\n",
      "Name: zinc finger protein 699\n",
      "Summary: Predicted to enable DNA-binding transcription factor activity, RNA polymerase II-specific and RNA polymerase II cis-regulatory region sequence-specific DNA binding activity. Predicted to be involved in regulation of transcription by RNA polymerase II. Predicted to be active in nucleus. [provided by Alliance of Genome Resources, Apr 2022]\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - regulation of transcription by RNA polymerase II\n",
      "  CC:\n",
      "    - nucleus\n",
      "  MF:\n",
      "    - RNA polymerase II cis-regulatory region sequence-specific DNA binding\n",
      "    - DNA-binding transcription factor activity, RNA polymerase II-specific\n",
      "    - protein binding\n",
      "    - zinc ion binding\n",
      "UniProt: {'Swiss-Prot': 'Q32M78'}\n",
      "\n",
      "Gene: GART\n",
      "Symbol: GART\n",
      "Name: phosphoribosylglycinamide formyltransferase, phosphoribosylglycinamide synthetase, phosphoribosylaminoimidazole synthetase\n",
      "Summary: The protein encoded by this gene is a trifunctional polypeptide. It has phosphoribosylglycinamide formyltransferase, phosphoribosylglycinamide synthetase, phosphoribosylaminoimidazole synthetase activity which is required for de novo purine biosynthesis. This enzyme is highly conserved in vertebrates. Alternative splicing of this gene results in two transcript variants encoding different isoforms. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - brainstem development\n",
      "    - purine nucleotide biosynthetic process\n",
      "    - GMP biosynthetic process\n",
      "    - 'de novo' IMP biosynthetic process\n",
      "    - 'de novo' IMP biosynthetic process\n",
      "    - purine ribonucleoside monophosphate biosynthetic process\n",
      "    - cerebellum development\n",
      "    - cerebral cortex development\n",
      "    - 'de novo' AMP biosynthetic process\n",
      "    - adenine biosynthetic process\n",
      "    - 'de novo' XMP biosynthetic process\n",
      "  CC:\n",
      "    - cytosol\n",
      "    - cytosol\n",
      "    - cytosol\n",
      "    - extracellular exosome\n",
      "  MF:\n",
      "    - phosphoribosylamine-glycine ligase activity\n",
      "    - phosphoribosylamine-glycine ligase activity\n",
      "    - phosphoribosylformylglycinamidine cyclo-ligase activity\n",
      "    - phosphoribosylformylglycinamidine cyclo-ligase activity\n",
      "    - phosphoribosylformylglycinamidine cyclo-ligase activity\n",
      "    - phosphoribosylglycinamide formyltransferase activity\n",
      "    - phosphoribosylglycinamide formyltransferase activity\n",
      "    - ATP binding\n",
      "    - metal ion binding\n",
      "UniProt: {'Swiss-Prot': 'P22102', 'TrEMBL': ['F8WD69', 'Q59HH3', 'C9JBJ1', 'C9JTV6', 'C9JKQ7', 'H7C366', 'A0A140TA71', 'C9JZG2']}\n",
      "\n",
      "Gene: SLC29A1\n",
      "Symbol: SLC29A1\n",
      "Name: solute carrier family 29 member 1 (Augustine blood group)\n",
      "Summary: This gene is a member of the equilibrative nucleoside transporter family. The gene encodes a transmembrane glycoprotein that localizes to the plasma and mitochondrial membranes and mediates the cellular uptake of nucleosides from the surrounding medium. The protein is categorized as an equilibrative (as opposed to concentrative) transporter that is sensitive to inhibition by nitrobenzylthioinosine (NBMPR). Nucleoside transporters are required for nucleotide synthesis in cells that lack de novo nucleoside synthesis pathways, and are also necessary for the uptake of cytotoxic nucleosides used for cancer and viral chemotherapies. Multiple alternatively spliced variants, encoding the same protein, have been found for this gene. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - neurotransmitter uptake\n",
      "    - nucleobase-containing compound metabolic process\n",
      "    - xenobiotic metabolic process\n",
      "    - neurotransmitter transport\n",
      "    - xenobiotic transmembrane transport\n",
      "    - lactation\n",
      "    - nucleobase transport\n",
      "    - adenine transport\n",
      "    - nucleoside transport\n",
      "    - purine nucleoside transmembrane transport\n",
      "    - purine nucleoside transmembrane transport\n",
      "    - cytidine transport\n",
      "    - uridine transmembrane transport\n",
      "    - uridine transmembrane transport\n",
      "    - uridine transmembrane transport\n",
      "    - adenosine transport\n",
      "    - adenosine transport\n",
      "    - inosine transport\n",
      "    - hypoxanthine transport\n",
      "    - thymine transport\n",
      "    - excitatory postsynaptic potential\n",
      "    - cellular response to glucose stimulus\n",
      "    - cellular response to hypoxia\n",
      "    - pyrimidine-containing compound transmembrane transport\n",
      "    - transport across blood-brain barrier\n",
      "    - nucleoside transmembrane transport\n",
      "    - nucleoside transmembrane transport\n",
      "    - guanine transmembrane transport\n",
      "    - uracil transmembrane transport\n",
      "    - pyrimidine nucleobase transmembrane transport\n",
      "    - purine nucleobase transmembrane transport\n",
      "  CC:\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - membrane\n",
      "    - basolateral plasma membrane\n",
      "    - apical plasma membrane\n",
      "    - presynapse\n",
      "    - postsynapse\n",
      "  MF:\n",
      "    - neurotransmitter transmembrane transporter activity\n",
      "    - neurotransmitter transmembrane transporter activity\n",
      "    - nucleoside transmembrane transporter activity\n",
      "    - nucleoside transmembrane transporter activity\n",
      "    - nucleoside transmembrane transporter activity\n",
      "    - adenine transmembrane transporter activity\n",
      "    - guanine transmembrane transporter activity\n",
      "    - uracil transmembrane transporter activity\n",
      "    - purine nucleoside transmembrane transporter activity\n",
      "    - purine nucleoside transmembrane transporter activity\n",
      "    - cytidine transmembrane transporter activity\n",
      "    - uridine transmembrane transporter activity\n",
      "    - pyrimidine- and adenosine-specific:sodium symporter activity\n",
      "UniProt: {'Swiss-Prot': 'Q99808', 'TrEMBL': ['A0A2U3TZM6', 'A0A2U3TZJ7', 'A0A2R8Y801', 'A0A2R8Y567', 'A0A2R8YFB9', 'A0A2R8Y854', 'A0A2R8Y6D1', 'A0A2U3TZM5', 'A0A2R8YFI6', 'A0A2R8YEH8']}\n",
      "\n",
      "Gene: PTGER2\n",
      "Symbol: PTGER2\n",
      "Name: prostaglandin E receptor 2\n",
      "Summary: This gene encodes a receptor for prostaglandin E2, a metabolite of arachidonic acid which has different biologic activities in a wide range of tissues. Mutations in this gene are associated with aspirin-induced susceptibility to asthma. [provided by RefSeq, Oct 2009].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - inflammatory response\n",
      "    - G protein-coupled receptor signaling pathway\n",
      "    - adenylate cyclase-activating G protein-coupled receptor signaling pathway\n",
      "    - adenylate cyclase-activating G protein-coupled receptor signaling pathway\n",
      "    - adenylate cyclase-activating G protein-coupled receptor signaling pathway\n",
      "    - positive regulation of cytosolic calcium ion concentration\n",
      "    - response to nematode\n",
      "    - response to lipopolysaccharide\n",
      "    - response to progesterone\n",
      "    - regulation of cell population proliferation\n",
      "    - cellular response to prostaglandin E stimulus\n",
      "    - cellular response to prostaglandin E stimulus\n",
      "  CC:\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "  MF:\n",
      "    - prostaglandin E receptor activity\n",
      "    - prostaglandin E receptor activity\n",
      "UniProt: {'Swiss-Prot': 'P43116'}\n",
      "\n",
      "Gene: CACNA1S\n",
      "Symbol: CACNA1S\n",
      "Name: calcium voltage-gated channel subunit alpha1 S\n",
      "Summary: This gene encodes one of the five subunits of the slowly inactivating L-type voltage-dependent calcium channel in skeletal muscle cells. Mutations in this gene have been associated with hypokalemic periodic paralysis, thyrotoxic periodic paralysis and malignant hyperthermia susceptibility. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - skeletal system development\n",
      "    - extraocular skeletal muscle development\n",
      "    - calcium ion transport\n",
      "    - muscle contraction\n",
      "    - striated muscle contraction\n",
      "    - endoplasmic reticulum organization\n",
      "    - myoblast fusion\n",
      "    - neuromuscular junction development\n",
      "    - skeletal muscle adaptation\n",
      "    - positive regulation of muscle contraction\n",
      "    - skeletal muscle fiber development\n",
      "    - release of sequestered calcium ion into cytosol\n",
      "    - calcium ion transmembrane transport\n",
      "    - cellular response to caffeine\n",
      "    - calcium ion import across plasma membrane\n",
      "  CC:\n",
      "    - cytoplasm\n",
      "    - plasma membrane\n",
      "    - voltage-gated calcium channel complex\n",
      "    - voltage-gated calcium channel complex\n",
      "    - sarcoplasmic reticulum\n",
      "    - T-tubule\n",
      "    - T-tubule\n",
      "    - T-tubule\n",
      "    - I band\n",
      "    - L-type voltage-gated calcium channel complex\n",
      "    - L-type voltage-gated calcium channel complex\n",
      "  MF:\n",
      "    - voltage-gated calcium channel activity\n",
      "    - voltage-gated calcium channel activity\n",
      "    - protein binding\n",
      "    - calmodulin binding\n",
      "    - high voltage-gated calcium channel activity\n",
      "    - high voltage-gated calcium channel activity\n",
      "    - small molecule binding\n",
      "    - metal ion binding\n",
      "    - molecular function activator activity\n",
      "UniProt: {'Swiss-Prot': 'Q13698', 'TrEMBL': ['A0A7P0TBR2', 'B1ALM3', 'A0AAQ5BGP9', 'A0A7P0T8M7', 'A0A7P0TB53', 'A0A7P0TBJ5', 'A0A7P0Z433']}\n",
      "\n",
      "Gene: SLCO3A1\n",
      "Symbol: SLCO3A1\n",
      "Name: solute carrier organic anion transporter family member 3A1\n",
      "Summary: Predicted to enable sodium-independent organic anion transmembrane transporter activity. Involved in positive regulation of NF-kappaB transcription factor activity; positive regulation of protein phosphorylation; and prostaglandin transport. Located in plasma membrane. [provided by Alliance of Genome Resources, Apr 2022]\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - monoatomic ion transport\n",
      "    - prostaglandin transport\n",
      "    - prostaglandin transport\n",
      "    - positive regulation of canonical NF-kappaB signal transduction\n",
      "    - sodium-independent organic anion transport\n",
      "    - sodium-independent organic anion transport\n",
      "    - positive regulation of MAPK cascade\n",
      "    - transmembrane transport\n",
      "    - transport across blood-brain barrier\n",
      "  CC:\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - basal plasma membrane\n",
      "    - basolateral plasma membrane\n",
      "    - apical plasma membrane\n",
      "  MF:\n",
      "    - protein binding\n",
      "    - organic anion transmembrane transporter activity\n",
      "    - prostaglandin transmembrane transporter activity\n",
      "    - sodium-independent organic anion transmembrane transporter activity\n",
      "    - sodium-independent organic anion transmembrane transporter activity\n",
      "UniProt: {'Swiss-Prot': 'Q9UIG8', 'TrEMBL': ['G3V2B7', 'G3V3K4', 'H0YJM0']}\n",
      "\n",
      "Gene: SULT1C2\n",
      "Symbol: SULT1C2\n",
      "Name: sulfotransferase family 1C member 2\n",
      "Summary: Sulfotransferase enzymes catalyze the sulfate conjugation of many hormones, neurotransmitters, drugs, and xenobiotic compounds. These cytosolic enzymes are different in their tissue distributions and substrate specificities. The gene structure (number and length of exons) is similar among family members. This gene encodes a protein that belongs to the SULT1 subfamily, responsible for transferring a sulfo moiety from PAPS to phenol-containing compounds. Two alternatively spliced transcript variants encoding different isoforms have been described for this gene. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - amine metabolic process\n",
      "    - sulfation\n",
      "    - sulfation\n",
      "    - sulfation\n",
      "  CC:\n",
      "    - cytoplasm\n",
      "    - mitochondrion\n",
      "    - lysosome\n",
      "    - cytosol\n",
      "  MF:\n",
      "    - aryl sulfotransferase activity\n",
      "    - aryl sulfotransferase activity\n",
      "    - protein binding\n",
      "    - sulfotransferase activity\n",
      "    - cholesterol sulfotransferase activity\n",
      "UniProt: {'Swiss-Prot': 'O00338', 'TrEMBL': ['H7C420', 'B4DLP0', 'B8ZZF7', 'C9JSR0', 'H7BZ17']}\n",
      "\n",
      "Gene: CHST3\n",
      "Symbol: CHST3\n",
      "Name: carbohydrate sulfotransferase 3\n",
      "Summary: This gene encodes an enzyme which catalyzes the sulfation of chondroitin, a proteoglycan found in the extracellular matrix and most cells which is involved in cell migration and differentiation. Mutations in this gene are associated with spondylepiphyseal dysplasia and humerospinal dysostosis. [provided by RefSeq, Mar 2009].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - carbohydrate metabolic process\n",
      "    - N-acetylglucosamine metabolic process\n",
      "    - sulfur compound metabolic process\n",
      "    - chondroitin sulfate biosynthetic process\n",
      "    - chondroitin sulfate biosynthetic process\n",
      "    - chondroitin sulfate biosynthetic process\n",
      "    - chondroitin sulfate biosynthetic process\n",
      "    - T cell homeostasis\n",
      "  CC:\n",
      "    - Golgi membrane\n",
      "  MF:\n",
      "    - N-acetylglucosamine 6-O-sulfotransferase activity\n",
      "    - chondroitin 6-sulfotransferase activity\n",
      "    - chondroitin 6-sulfotransferase activity\n",
      "    - chondroitin 6-sulfotransferase activity\n",
      "UniProt: {'Swiss-Prot': 'Q7LGC8'}\n",
      "\n",
      "Gene: DPF3\n",
      "Symbol: DPF3\n",
      "Name: double PHD fingers 3\n",
      "Summary: This gene encodes a member of the D4 protein family. The encoded protein is a transcription regulator that binds acetylated histones and is a component of the BAF chromatin remodeling complex. Alternate splicing results in multiple transcript variants encoding different isoforms. [provided by RefSeq, Jul 2013].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - chromatin remodeling\n",
      "    - regulation of transcription by RNA polymerase II\n",
      "    - nervous system development\n",
      "    - muscle organ development\n",
      "    - regulation of mitotic metaphase/anaphase transition\n",
      "    - positive regulation of cell differentiation\n",
      "    - positive regulation of myoblast differentiation\n",
      "    - regulation of G0 to G1 transition\n",
      "    - regulation of G1/S transition of mitotic cell cycle\n",
      "    - positive regulation of double-strand break repair\n",
      "    - regulation of nucleotide-excision repair\n",
      "  CC:\n",
      "    - chromatin\n",
      "    - nucleus\n",
      "    - nucleoplasm\n",
      "    - nucleoplasm\n",
      "    - SWI/SNF complex\n",
      "    - brahma complex\n",
      "    - nBAF complex\n",
      "    - nBAF complex\n",
      "  MF:\n",
      "    - protein binding\n",
      "    - zinc ion binding\n",
      "UniProt: {'Swiss-Prot': 'Q92784', 'TrEMBL': 'F8W7T1'}\n",
      "\n",
      "Gene: HNMT\n",
      "Symbol: HNMT\n",
      "Name: histamine N-methyltransferase\n",
      "Summary: In mammals, histamine is metabolized by two major pathways: N(tau)-methylation via histamine N-methyltransferase and oxidative deamination via diamine oxidase. This gene encodes the first enzyme which is found in the cytosol and uses S-adenosyl-L-methionine as the methyl donor. In the mammalian brain, the neurotransmitter activity of histamine is controlled by N(tau)-methylation as diamine oxidase is not found in the central nervous system. A common genetic polymorphism affects the activity levels of this gene product in red blood cells. Multiple alternatively spliced transcript variants that encode different proteins have been found for this gene. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - histamine metabolic process\n",
      "    - histamine catabolic process\n",
      "    - L-histidine catabolic process\n",
      "    - respiratory gaseous exchange by respiratory system\n",
      "    - methylation\n",
      "  CC:\n",
      "    - nucleoplasm\n",
      "    - cytoplasm\n",
      "    - centrosome\n",
      "    - cytosol\n",
      "    - cytosol\n",
      "    - cytosol\n",
      "    - extracellular exosome\n",
      "  MF:\n",
      "    - histamine N-methyltransferase activity\n",
      "    - histamine N-methyltransferase activity\n",
      "UniProt: {'Swiss-Prot': 'P50135', 'TrEMBL': 'B4DWC1'}\n",
      "\n",
      "Gene: CPS1\n",
      "Symbol: CPS1\n",
      "Name: carbamoyl-phosphate synthase 1\n",
      "Summary: The mitochondrial enzyme encoded by this gene catalyzes synthesis of carbamoyl phosphate from ammonia and bicarbonate. This reaction is the first committed step of the urea cycle, which is important in the removal of excess urea from cells. The encoded protein may also represent a core mitochondrial nucleoid protein. Three transcript variants encoding different isoforms have been found for this gene. The shortest isoform may not be localized to the mitochondrion. Mutations in this gene have been associated with carbamoyl phosphate synthetase deficiency, susceptibility to persistent pulmonary hypertension, and susceptibility to venoocclusive disease after bone marrow transplantation.[provided by RefSeq, May 2010].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - urea cycle\n",
      "    - urea cycle\n",
      "    - 'de novo' pyrimidine nucleobase biosynthetic process\n",
      "    - glutamine metabolic process\n",
      "    - midgut development\n",
      "    - response to xenobiotic stimulus\n",
      "    - response to toxic substance\n",
      "    - response to zinc ion\n",
      "    - response to amine\n",
      "    - citrulline biosynthetic process\n",
      "    - triglyceride catabolic process\n",
      "    - response to food\n",
      "    - response to lipopolysaccharide\n",
      "    - vasodilation\n",
      "    - response to starvation\n",
      "    - response to amino acid\n",
      "    - cellular response to fibroblast growth factor stimulus\n",
      "    - nitric oxide metabolic process\n",
      "    - homocysteine metabolic process\n",
      "    - monoatomic anion homeostasis\n",
      "    - response to growth hormone\n",
      "    - hepatocyte differentiation\n",
      "    - carbamoyl phosphate biosynthetic process\n",
      "    - cellular response to ammonium ion\n",
      "    - cellular response to cAMP\n",
      "    - cellular response to glucagon stimulus\n",
      "    - cellular response to oleic acid\n",
      "    - response to dexamethasone\n",
      "  CC:\n",
      "    - nucleolus\n",
      "    - cytoplasm\n",
      "    - mitochondrion\n",
      "    - mitochondrial inner membrane\n",
      "    - mitochondrial matrix\n",
      "    - plasma membrane\n",
      "    - protein-containing complex\n",
      "    - mitochondrial nucleoid\n",
      "  MF:\n",
      "    - carbamoyl-phosphate synthase (ammonia) activity\n",
      "    - carbamoyl-phosphate synthase (ammonia) activity\n",
      "    - carbamoyl-phosphate synthase (ammonia) activity\n",
      "    - carbamoyl-phosphate synthase (glutamine-hydrolyzing) activity\n",
      "    - endopeptidase activity\n",
      "    - calcium ion binding\n",
      "    - protein binding\n",
      "    - ATP binding\n",
      "    - phospholipid binding\n",
      "    - glutamate binding\n",
      "    - potassium ion binding\n",
      "    - small molecule binding\n",
      "    - protein-containing complex binding\n",
      "    - metal ion binding\n",
      "    - modified amino acid binding\n",
      "UniProt: {'Swiss-Prot': 'P31327', 'TrEMBL': ['A0A669KBF2', 'Q6PEK7']}\n",
      "\n",
      "Gene: GTF2H4\n",
      "Symbol: GTF2H4\n",
      "Name: general transcription factor IIH subunit 4\n",
      "Summary: Enables RNA polymerase II general transcription initiation factor activity. Involved in transcription by RNA polymerase II. Located in nuclear speck. Part of core TFIIH complex portion of holo TFIIH complex and transcription factor TFIID complex. [provided by Alliance of Genome Resources, Apr 2022]\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - DNA repair\n",
      "    - nucleotide-excision repair\n",
      "    - transcription by RNA polymerase II\n",
      "  CC:\n",
      "    - core TFIIH complex portion of holo TFIIH complex\n",
      "    - transcription factor TFIIH core complex\n",
      "    - nucleus\n",
      "    - nucleoplasm\n",
      "    - transcription factor TFIID complex\n",
      "    - transcription factor TFIIH holo complex\n",
      "    - transcription factor TFIIH holo complex\n",
      "    - nuclear speck\n",
      "  MF:\n",
      "    - ATPase activator activity\n",
      "    - double-stranded DNA binding\n",
      "    - protein binding\n",
      "    - RNA polymerase II general transcription initiation factor activity\n",
      "UniProt: {'Swiss-Prot': 'Q92759', 'TrEMBL': ['A0A0G2JHJ2', 'A0A1U9X7S4']}\n",
      "\n",
      "Gene: PNMT\n",
      "Symbol: PNMT\n",
      "Name: phenylethanolamine N-methyltransferase\n",
      "Summary: The product of this gene catalyzes the last step of the catecholamine biosynthesis pathway, which methylates norepinephrine to form epinephrine (adrenaline). The enzyme also has beta-carboline 2N-methyltransferase activity. This gene is thought to play a key step in regulating epinephrine production. Alternatively spliced transcript variants have been found for this gene. [provided by RefSeq, Nov 2012].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - methylation\n",
      "    - epinephrine biosynthetic process\n",
      "    - catecholamine biosynthetic process\n",
      "  CC:\n",
      "    - cytosol\n",
      "    - cytosol\n",
      "  MF:\n",
      "    - phenylethanolamine N-methyltransferase activity\n",
      "    - phenylethanolamine N-methyltransferase activity\n",
      "    - phenylethanolamine N-methyltransferase activity\n",
      "    - phenylethanolamine N-methyltransferase activity\n",
      "    - protein binding\n",
      "UniProt: {'Swiss-Prot': 'P11086', 'TrEMBL': ['J3QRI3', 'A8MT87']}\n",
      "\n",
      "Gene: MTTP\n",
      "Symbol: MTTP\n",
      "Name: microsomal triglyceride transfer protein\n",
      "Summary: MTP encodes the large subunit of the heterodimeric microsomal triglyceride transfer protein. Protein disulfide isomerase (PDI) completes the heterodimeric microsomal triglyceride transfer protein, which has been shown to play a central role in lipoprotein assembly. Mutations in MTP can cause abetalipoproteinemia. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - lipid metabolic process\n",
      "    - triglyceride metabolic process\n",
      "    - circadian rhythm\n",
      "    - protein secretion\n",
      "    - protein secretion\n",
      "    - phospholipid transport\n",
      "    - sterol transport\n",
      "    - triglyceride transport\n",
      "    - triglyceride transport\n",
      "    - low-density lipoprotein particle remodeling\n",
      "    - plasma lipoprotein particle assembly\n",
      "    - plasma lipoprotein particle assembly\n",
      "    - chylomicron assembly\n",
      "    - very-low-density lipoprotein particle assembly\n",
      "    - lipoprotein metabolic process\n",
      "    - cholesterol homeostasis\n",
      "    - lipoprotein transport\n",
      "    - response to calcium ion\n",
      "    - establishment of localization in cell\n",
      "    - intermembrane lipid transfer\n",
      "    - ceramide 1-phosphate transport\n",
      "  CC:\n",
      "    - endoplasmic reticulum\n",
      "    - endoplasmic reticulum\n",
      "    - endoplasmic reticulum lumen\n",
      "    - Golgi apparatus\n",
      "    - Golgi apparatus\n",
      "    - cytosol\n",
      "    - basolateral plasma membrane\n",
      "    - brush border membrane\n",
      "    - microvillus membrane\n",
      "    - vesicle\n",
      "    - receptor complex\n",
      "  MF:\n",
      "    - lipid transporter activity\n",
      "    - protein binding\n",
      "    - phospholipid transporter activity\n",
      "    - lipid binding\n",
      "    - apolipoprotein binding\n",
      "    - protein-containing complex binding\n",
      "    - protein heterodimerization activity\n",
      "    - phospholipid transfer activity\n",
      "    - phosphatidylcholine transfer activity\n",
      "    - phosphatidylcholine transfer activity\n",
      "    - cholesterol transfer activity\n",
      "    - cholesterol transfer activity\n",
      "    - triglyceride transfer activity\n",
      "    - triglyceride transfer activity\n",
      "    - ceramide 1-phosphate transfer activity\n",
      "    - phosphatidylethanolamine transfer activity\n",
      "UniProt: {'Swiss-Prot': 'P55157', 'TrEMBL': ['E9PBP6', 'B7Z7X3', 'D6RAB8', 'D6R915']}\n",
      "\n",
      "Gene: COX4I1\n",
      "Symbol: COX4I1\n",
      "Name: cytochrome c oxidase subunit 4I1\n",
      "Summary: Cytochrome c oxidase (COX) is the terminal enzyme of the mitochondrial respiratory chain. It is a multi-subunit enzyme complex that couples the transfer of electrons from cytochrome c to molecular oxygen and contributes to a proton electrochemical gradient across the inner mitochondrial membrane. The complex consists of 13 mitochondrial- and nuclear-encoded subunits. The mitochondrially-encoded subunits perform the electron transfer and proton pumping activities. The functions of the nuclear-encoded subunits are unknown but they may play a role in the regulation and assembly of the complex. This gene encodes the nuclear-encoded subunit IV isoform 1 of the human mitochondrial respiratory chain enzyme. It is located at the 3' of the NOC4 (neighbor of COX4) gene in a head-to-head orientation, and shares a promoter with it. Pseudogenes related to this gene are located on chromosomes 13 and 14. Alternative splicing results in multiple transcript variants encoding different isoforms. [provided by RefSeq, Jan 2016].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - generation of precursor metabolites and energy\n",
      "    - mitochondrial electron transport, cytochrome c to oxygen\n",
      "    - mitochondrial electron transport, cytochrome c to oxygen\n",
      "    - cellular respiration\n",
      "    - proton transmembrane transport\n",
      "  CC:\n",
      "    - nucleoplasm\n",
      "    - mitochondrion\n",
      "    - mitochondrion\n",
      "    - mitochondrion\n",
      "    - mitochondrial inner membrane\n",
      "    - mitochondrial inner membrane\n",
      "    - mitochondrial intermembrane space\n",
      "    - cytosol\n",
      "    - membrane\n",
      "    - mitochondrial membrane\n",
      "    - respiratory chain complex IV\n",
      "    - respiratory chain complex IV\n",
      "  MF:\n",
      "    - cytochrome-c oxidase activity\n",
      "    - protein binding\n",
      "UniProt: {'Swiss-Prot': 'P13073', 'TrEMBL': ['H3BNV9', 'Q86WV2', 'H3BN72', 'H3BPG0', 'H3BPR4', 'H3BNI5']}\n",
      "\n",
      "Gene: SCN3A\n",
      "Symbol: SCN3A\n",
      "Name: sodium voltage-gated channel alpha subunit 3\n",
      "Summary: Voltage-gated sodium channels are transmembrane glycoprotein complexes composed of a large alpha subunit with 24 transmembrane domains and one or more regulatory beta subunits. They are responsible for the generation and propagation of action potentials in neurons and muscle. This gene encodes one member of the sodium channel alpha subunit gene family, and is found in a cluster of five alpha subunit genes on chromosome 2. Multiple transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - sodium ion transport\n",
      "    - sodium ion transmembrane transport\n",
      "    - behavioral response to pain\n",
      "    - cardiac muscle cell action potential involved in contraction\n",
      "    - membrane depolarization during action potential\n",
      "  CC:\n",
      "    - voltage-gated sodium channel complex\n",
      "    - voltage-gated sodium channel complex\n",
      "    - plasma membrane\n",
      "    - basal plasma membrane\n",
      "    - sarcoplasm\n",
      "  MF:\n",
      "    - voltage-gated sodium channel activity\n",
      "    - voltage-gated sodium channel activity\n",
      "UniProt: {'Swiss-Prot': 'Q9NY46', 'TrEMBL': ['A0A590UJH3', 'Q9C007', 'A0A994J5P2', 'A0A1W2PSB2', 'C9JBM7', 'A0A1W2PRD1', 'E7EUE6', 'A0A1W2PQ58', 'A0A590UJW3']}\n",
      "\n",
      "Gene: LANCL1\n",
      "Symbol: LANCL1\n",
      "Name: LanC like glutathione S-transferase 1\n",
      "Summary: This gene encodes a loosely associated peripheral membrane protein related to the LanC family of bacterial membrane-associated proteins involved in the biosynthesis of antimicrobial peptides. This protein may play a role as a peptide-modifying enzyme component in eukaryotic cells. Previously considered a member of the G-protein-coupled receptor superfamily, this protein is now in the LanC family. Multiple alternatively spliced variants, encoding the same protein, have been identified. [provided by RefSeq, Nov 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - carbohydrate metabolic process\n",
      "    - G protein-coupled receptor signaling pathway\n",
      "    - peptide modification\n",
      "    - cellular detoxification\n",
      "  CC:\n",
      "    - cytoplasm\n",
      "    - plasma membrane\n",
      "  MF:\n",
      "    - glutathione transferase activity\n",
      "    - G protein-coupled receptor activity\n",
      "    - protein binding\n",
      "    - zinc ion binding\n",
      "    - SH3 domain binding\n",
      "    - glutathione binding\n",
      "    - low-density lipoprotein particle receptor binding\n",
      "UniProt: {'Swiss-Prot': 'O43813', 'TrEMBL': ['B2R602', 'Q53TN2', 'E9PHS0', 'F8WDS9', 'H7C2E3']}\n",
      "\n",
      "Gene: LAMA5\n",
      "Symbol: LAMA5\n",
      "Name: laminin subunit alpha 5\n",
      "Summary: This gene encodes one of the vertebrate laminin alpha chains. Laminins, a family of extracellular matrix glycoproteins, are the major noncollagenous constituent of basement membranes. They have been implicated in a wide variety of biological processes including cell adhesion, differentiation, migration, signaling, neurite outgrowth and metastasis. Laminins are composed of 3 non identical chains: laminin alpha, beta and gamma (formerly A, B1, and B2, respectively) and they form a cruciform structure consisting of 3 short arms, each formed by a different chain, and a long arm composed of all 3 chains. Each laminin chain is a multidomain protein encoded by a distinct gene. The protein encoded by this gene is the alpha-5 subunit of of laminin-10 (laminin-511), laminin-11 (laminin-521) and laminin-15 (laminin-523). [provided by RefSeq, Jun 2013].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - branching involved in ureteric bud morphogenesis\n",
      "    - morphogenesis of a polarized epithelium\n",
      "    - hair follicle development\n",
      "    - cell adhesion\n",
      "    - integrin-mediated signaling pathway\n",
      "    - muscle organ development\n",
      "    - morphogenesis of embryonic epithelium\n",
      "    - cell migration\n",
      "    - regulation of cell adhesion\n",
      "    - lung development\n",
      "    - regulation of cell migration\n",
      "    - substrate adhesion-dependent cell spreading\n",
      "    - trunk neural crest cell migration\n",
      "    - odontogenesis of dentin-containing tooth\n",
      "    - regulation of embryonic development\n",
      "    - skeletal system morphogenesis\n",
      "    - regulation of epithelial cell proliferation\n",
      "    - cilium assembly\n",
      "    - branching involved in salivary gland morphogenesis\n",
      "    - protein localization to plasma membrane\n",
      "    - postsynapse organization\n",
      "  CC:\n",
      "    - extracellular region\n",
      "    - basement membrane\n",
      "    - basement membrane\n",
      "    - laminin-5 complex\n",
      "    - extracellular space\n",
      "    - nucleus\n",
      "    - neuromuscular junction\n",
      "    - synaptic cleft\n",
      "    - laminin-10 complex\n",
      "    - laminin-10 complex\n",
      "    - laminin-11 complex\n",
      "    - collagen-containing extracellular matrix\n",
      "    - collagen-containing extracellular matrix\n",
      "    - extracellular exosome\n",
      "    - extracellular matrix of synaptic cleft\n",
      "    - glutamatergic synapse\n",
      "  MF:\n",
      "    - integrin binding\n",
      "    - extracellular matrix structural constituent\n",
      "    - receptor ligand activity\n",
      "    - cell adhesion mediator activity\n",
      "UniProt: {'Swiss-Prot': 'O15230', 'TrEMBL': ['A0A087WYH7', 'H7C5J6']}\n",
      "\n",
      "Gene: SLC6A2\n",
      "Symbol: SLC6A2\n",
      "Name: solute carrier family 6 member 2\n",
      "Summary: This gene encodes a member of the sodium:neurotransmitter symporter family. This member is a multi-pass membrane protein, which is responsible for reuptake of norepinephrine into presynaptic nerve terminals and is a regulator of norepinephrine homeostasis. Mutations in this gene cause orthostatic intolerance, a syndrome characterized by lightheadedness, fatigue, altered mentation and syncope. Alternatively spliced transcript variants encoding different isoforms have been identified in this gene.[provided by RefSeq, Feb 2010].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - neurotransmitter transport\n",
      "    - amino acid transport\n",
      "    - chemical synaptic transmission\n",
      "    - response to xenobiotic stimulus\n",
      "    - monoamine transport\n",
      "    - norepinephrine transport\n",
      "    - sodium ion transmembrane transport\n",
      "    - response to pain\n",
      "    - dopamine uptake involved in synaptic transmission\n",
      "    - norepinephrine uptake\n",
      "    - neuron cellular homeostasis\n",
      "    - chloride transmembrane transport\n",
      "  CC:\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - cell surface\n",
      "    - membrane\n",
      "    - axon\n",
      "    - synaptic vesicle membrane\n",
      "    - neuronal cell body membrane\n",
      "    - presynaptic membrane\n",
      "  MF:\n",
      "    - actin binding\n",
      "    - neurotransmitter transmembrane transporter activity\n",
      "    - neurotransmitter:sodium symporter activity\n",
      "    - dopamine:sodium symporter activity\n",
      "    - dopamine:sodium symporter activity\n",
      "    - norepinephrine:sodium symporter activity\n",
      "    - norepinephrine:sodium symporter activity\n",
      "    - norepinephrine:sodium symporter activity\n",
      "    - protein binding\n",
      "    - monoamine transmembrane transporter activity\n",
      "    - monoamine transmembrane transporter activity\n",
      "    - alpha-tubulin binding\n",
      "    - metal ion binding\n",
      "    - beta-tubulin binding\n",
      "UniProt: {'Swiss-Prot': 'P23975', 'TrEMBL': ['H3BM11', 'H3BRE9', 'H3BML6', 'H3BRS0', 'A0A804HLI4']}\n",
      "\n",
      "Gene: GPX4\n",
      "Symbol: GPX4\n",
      "Name: glutathione peroxidase 4\n",
      "Summary: The protein encoded by this gene belongs to the glutathione peroxidase family, members of which catalyze the reduction of hydrogen peroxide, organic hydroperoxides and lipid hydroperoxides, and thereby protect cells against oxidative damage. Several isozymes of this gene family exist in vertebrates, which vary in cellular location and substrate specificity. This isozyme has a high preference for lipid hydroperoxides and protects cells against membrane lipid peroxidation and cell death. It is also required for normal sperm development; thus, it has been identified as a 'moonlighting' protein because of its ability to serve dual functions as a peroxidase, as well as a structural protein in mature spermatozoa. Mutations in this gene are associated with Sedaghatian type of spondylometaphyseal dysplasia (SMDS). This isozyme is also a selenoprotein, containing the rare amino acid selenocysteine (Sec) at its active site. Sec is encoded by the UGA codon, which normally signals translation termination. The 3' UTRs of selenoprotein mRNAs contain a conserved stem-loop structure, designated the Sec insertion sequence (SECIS) element, that is necessary for the recognition of UGA as a Sec codon, rather than as a stop signal. Transcript variants resulting from alternative splicing or use of alternate promoters have been described to encode isoforms with different subcellular localization. [provided by RefSeq, Dec 2018].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - chromatin organization\n",
      "    - phospholipid metabolic process\n",
      "    - apoptotic process\n",
      "    - response to oxidative stress\n",
      "    - spermatogenesis\n",
      "    - dendrite development\n",
      "    - arachidonate metabolic process\n",
      "    - lipoxygenase pathway\n",
      "    - cerebellum development\n",
      "    - response to estradiol\n",
      "    - response to lipopolysaccharide\n",
      "    - multicellular organism growth\n",
      "    - long-chain fatty acid biosynthetic process\n",
      "    - protein polymerization\n",
      "    - cellular oxidant detoxification\n",
      "    - negative regulation of ferroptosis\n",
      "  CC:\n",
      "    - nucleus\n",
      "    - nucleus\n",
      "    - nuclear envelope\n",
      "    - mitochondrion\n",
      "    - mitochondrion\n",
      "    - cytosol\n",
      "    - cytosol\n",
      "    - protein-containing complex\n",
      "    - extracellular exosome\n",
      "  MF:\n",
      "    - glutathione peroxidase activity\n",
      "    - glutathione peroxidase activity\n",
      "    - glutathione peroxidase activity\n",
      "    - protein binding\n",
      "    - selenium binding\n",
      "    - identical protein binding\n",
      "    - phospholipid-hydroperoxide glutathione peroxidase activity\n",
      "    - phospholipid-hydroperoxide glutathione peroxidase activity\n",
      "    - phospholipid-hydroperoxide glutathione peroxidase activity\n",
      "    - phospholipid-hydroperoxide glutathione peroxidase activity\n",
      "UniProt: {'Swiss-Prot': 'P36969', 'TrEMBL': ['Q6PJX4', 'Q6PI42', 'K7EKX7', 'A0A0A0MTT1', 'A0A9L9PXS3', 'K7EJ20', 'A0A087X247', 'K7ENB4', 'K7ERP4', 'R4GNE4']}\n",
      "\n",
      "Gene: SI\n",
      "Symbol: SI\n",
      "Name: sucrase-isomaltase\n",
      "Summary: This gene encodes a sucrase-isomaltase enzyme that is expressed in the intestinal brush border. The encoded protein is synthesized as a precursor protein that is cleaved by pancreatic proteases into two enzymatic subunits sucrase and isomaltase. These two subunits heterodimerize to form the sucrose-isomaltase complex. This complex is essential for the digestion of dietary carbohydrates including starch, sucrose and isomaltose. Mutations in this gene are the cause of congenital sucrase-isomaltase deficiency.[provided by RefSeq, Apr 2010].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - sucrose catabolic process\n",
      "    - polysaccharide digestion\n",
      "  CC:\n",
      "    - Golgi apparatus\n",
      "    - plasma membrane\n",
      "    - brush border\n",
      "    - apical plasma membrane\n",
      "    - extracellular exosome\n",
      "  MF:\n",
      "    - alpha-1,4-glucosidase activity\n",
      "    - oligo-1,6-glucosidase activity\n",
      "    - sucrose alpha-glucosidase activity\n",
      "    - protein binding\n",
      "    - carbohydrate binding\n",
      "UniProt: {'Swiss-Prot': 'P14410', 'TrEMBL': 'F8WF21'}\n",
      "\n",
      "Gene: CA1\n",
      "Symbol: CA1\n",
      "Name: carbonic anhydrase 1\n",
      "Summary: Carbonic anhydrases (CAs) are a large family of zinc metalloenzymes that catalyze the reversible hydration of carbon dioxide. They participate in a variety of biological processes, including respiration, calcification, acid-base balance, bone resorption, and the formation of aqueous humor, cerebrospinal fluid, saliva and gastric acid. They show extensive diversity in tissue distribution and in their subcellular localization. This CA1 gene is closely linked to the CA2 and CA3 genes on chromosome 8. It encodes a cytosolic protein that is found at the highest level in erythrocytes. Allelic variants of this gene have been described in some populations. Alternative splicing and the use of alternative promoters results in multiple transcript variants. [provided by RefSeq, Nov 2016].\n",
      "GO Annotations:\n",
      "  CC:\n",
      "    - cytoplasm\n",
      "    - cytosol\n",
      "    - extracellular exosome\n",
      "  MF:\n",
      "    - arylesterase activity\n",
      "    - carbonate dehydratase activity\n",
      "    - carbonate dehydratase activity\n",
      "    - carbonate dehydratase activity\n",
      "    - protein binding\n",
      "    - zinc ion binding\n",
      "    - hydro-lyase activity\n",
      "    - cyanamide hydratase activity\n",
      "UniProt: {'Swiss-Prot': 'P00915', 'TrEMBL': ['E5RG81', 'V9HWE3', 'E5RG43', 'E5RJI8', 'E5RHP7', 'E5RFL2', 'E5RJF6', 'E5RGU8', 'E5RII2', 'E5RIF9', 'E5RFE7', 'E5RH81', 'E5RHS7', 'H0YBE2']}\n",
      "\n",
      "Gene: TNK1\n",
      "Symbol: TNK1\n",
      "Name: tyrosine kinase non receptor 1\n",
      "Summary: The protein encoded by this gene belongs to the tyrosine protein kinase family. Tyrosine protein kinases are important regulators of intracellular signal transduction pathways, mediating cellular proliferation, survival, and development. This gene is highly expressed in fetal tissues and at lower levels in few adult tissues, thus may function in signaling pathways utilized broadly during fetal development, and more selectively in adult tissues. It plays a negative regulatory role in the Ras-Raf1-MAPK pathway, and knockout mice have been shown to develop spontaneous tumors, suggesting a role as a tumor suppressor gene. Alternatively spliced transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Oct 2011].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - protein phosphorylation\n",
      "    - protein autophosphorylation\n",
      "  CC:\n",
      "    - cytoplasm\n",
      "    - plasma membrane\n",
      "    - membrane\n",
      "  MF:\n",
      "    - protein tyrosine kinase activity\n",
      "    - protein tyrosine kinase activity\n",
      "    - non-membrane spanning protein tyrosine kinase activity\n",
      "    - protein binding\n",
      "    - ATP binding\n",
      "UniProt: {'Swiss-Prot': 'Q13470', 'TrEMBL': ['U3KPR9', 'I3L334']}\n",
      "\n",
      "Gene: CACNA1F\n",
      "Symbol: CACNA1F\n",
      "Name: calcium voltage-gated channel subunit alpha1 F\n",
      "Summary: This gene encodes a multipass transmembrane protein that functions as an alpha-1 subunit of the voltage-dependent calcium channel, which mediates the influx of calcium ions into the cell. The encoded protein forms a complex of alpha-1, alpha-2/delta, beta, and gamma subunits in a 1:1:1:1 ratio. Mutations in this gene can cause X-linked eye disorders, including congenital stationary night blindness type 2A, cone-rod dystropy, and Aland Island eye disease. Alternatively spliced transcript variants encoding multiple isoforms have been observed. [provided by RefSeq, Aug 2013].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - visual perception\n",
      "    - detection of light stimulus involved in visual perception\n",
      "    - calcium ion transmembrane transport\n",
      "    - calcium ion import across plasma membrane\n",
      "    - negative regulation of voltage-gated calcium channel activity\n",
      "  CC:\n",
      "    - photoreceptor outer segment\n",
      "    - voltage-gated calcium channel complex\n",
      "    - voltage-gated calcium channel complex\n",
      "    - membrane\n",
      "    - perikaryon\n",
      "  MF:\n",
      "    - voltage-gated calcium channel activity\n",
      "    - voltage-gated calcium channel activity\n",
      "    - protein binding\n",
      "    - high voltage-gated calcium channel activity\n",
      "    - high voltage-gated calcium channel activity\n",
      "    - metal ion binding\n",
      "UniProt: {'Swiss-Prot': 'O60840', 'TrEMBL': 'H7C549'}\n",
      "\n",
      "Gene: ATP1A3\n",
      "Symbol: ATP1A3\n",
      "Name: ATPase Na+/K+ transporting subunit alpha 3\n",
      "Summary: The protein encoded by this gene belongs to the family of P-type cation transport ATPases, and to the subfamily of Na+/K+ -ATPases. Na+/K+ -ATPase is an integral membrane protein responsible for establishing and maintaining the electrochemical gradients of Na and K ions across the plasma membrane. These gradients are essential for osmoregulation, for sodium-coupled transport of a variety of organic and inorganic molecules, and for electrical excitability of nerve and muscle. This enzyme is composed of two subunits, a large catalytic subunit (alpha) and a smaller glycoprotein subunit (beta). The catalytic subunit of Na+/K+ -ATPase is encoded by multiple genes. This gene encodes an alpha 3 subunit. Alternatively spliced transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Jan 2012].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - intracellular sodium ion homeostasis\n",
      "    - intracellular sodium ion homeostasis\n",
      "    - establishment or maintenance of transmembrane electrochemical gradient\n",
      "    - intracellular potassium ion homeostasis\n",
      "    - intracellular potassium ion homeostasis\n",
      "    - sodium ion export across plasma membrane\n",
      "    - sodium ion export across plasma membrane\n",
      "    - regulation of resting membrane potential\n",
      "    - cellular response to steroid hormone stimulus\n",
      "    - cell communication by electrical coupling involved in cardiac conduction\n",
      "    - proton transmembrane transport\n",
      "    - response to glycoside\n",
      "    - cellular response to amyloid-beta\n",
      "    - neuron projection maintenance\n",
      "    - potassium ion import across plasma membrane\n",
      "    - potassium ion import across plasma membrane\n",
      "  CC:\n",
      "    - photoreceptor inner segment\n",
      "    - endoplasmic reticulum\n",
      "    - Golgi apparatus\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - sodium:potassium-exchanging ATPase complex\n",
      "    - sodium:potassium-exchanging ATPase complex\n",
      "    - sodium:potassium-exchanging ATPase complex\n",
      "    - membrane\n",
      "    - membrane\n",
      "    - axon\n",
      "    - organelle membrane\n",
      "    - neuronal cell body membrane\n",
      "    - neuronal cell body\n",
      "    - synapse\n",
      "    - photoreceptor inner segment membrane\n",
      "    - neuron to neuron synapse\n",
      "    - extracellular vesicle\n",
      "  MF:\n",
      "    - amyloid-beta binding\n",
      "    - amyloid-beta binding\n",
      "    - P-type sodium:potassium-exchanging transporter activity\n",
      "    - P-type sodium:potassium-exchanging transporter activity\n",
      "    - P-type sodium:potassium-exchanging transporter activity\n",
      "    - P-type sodium:potassium-exchanging transporter activity\n",
      "    - protein binding\n",
      "    - ATP binding\n",
      "    - ATP hydrolysis activity\n",
      "    - metal ion binding\n",
      "    - protein-folding chaperone binding\n",
      "    - steroid hormone binding\n",
      "UniProt: {'Swiss-Prot': 'P13637', 'TrEMBL': ['M0R116', 'Q53ES0', 'A0A0A0MT26', 'M0QXF2']}\n",
      "\n",
      "Gene: CD14\n",
      "Symbol: CD14\n",
      "Name: CD14 molecule\n",
      "Summary: The protein encoded by this gene is a surface antigen that is preferentially expressed on monocytes/macrophages. It cooperates with other proteins to mediate the innate immune response to bacterial lipopolysaccharide, and to viruses. This gene has been identified as a target candidate in the treatment of SARS-CoV-2-infected patients to potentially lessen or inhibit a severe inflammatory response. Alternative splicing results in multiple transcript variants encoding the same protein. [provided by RefSeq, Aug 2020].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - positive regulation of cytokine production\n",
      "    - cell surface pattern recognition receptor signaling pathway\n",
      "    - receptor-mediated endocytosis\n",
      "    - phagocytosis\n",
      "    - apoptotic process\n",
      "    - inflammatory response\n",
      "    - cell surface receptor signaling pathway\n",
      "    - lipopolysaccharide-mediated signaling pathway\n",
      "    - positive regulation of lipopolysaccharide-mediated signaling pathway\n",
      "    - response to magnesium ion\n",
      "    - positive regulation of type I interferon production\n",
      "    - positive regulation of type II interferon production\n",
      "    - positive regulation of interleukin-8 production\n",
      "    - positive regulation of tumor necrosis factor production\n",
      "    - positive regulation of tumor necrosis factor production\n",
      "    - toll-like receptor 4 signaling pathway\n",
      "    - toll-like receptor 4 signaling pathway\n",
      "    - positive regulation of toll-like receptor 4 signaling pathway\n",
      "    - response to tumor necrosis factor\n",
      "    - innate immune response\n",
      "    - response to ethanol\n",
      "    - positive regulation of endocytosis\n",
      "    - response to electrical stimulus\n",
      "    - cellular response to molecule of bacterial origin\n",
      "    - cellular response to lipopolysaccharide\n",
      "    - cellular response to lipopolysaccharide\n",
      "    - cellular response to lipopolysaccharide\n",
      "    - cellular response to lipoteichoic acid\n",
      "    - cellular response to diacyl bacterial lipopeptide\n",
      "    - cellular response to triacyl bacterial lipopeptide\n",
      "  CC:\n",
      "    - extracellular region\n",
      "    - extracellular space\n",
      "    - extracellular space\n",
      "    - Golgi apparatus\n",
      "    - plasma membrane\n",
      "    - external side of plasma membrane\n",
      "    - external side of plasma membrane\n",
      "    - endosome membrane\n",
      "    - secretory granule membrane\n",
      "    - membrane raft\n",
      "    - membrane raft\n",
      "    - lipopolysaccharide receptor complex\n",
      "    - lipopolysaccharide receptor complex\n",
      "    - extracellular exosome\n",
      "  MF:\n",
      "    - lipopolysaccharide binding\n",
      "    - lipopolysaccharide binding\n",
      "    - opsonin receptor activity\n",
      "    - lipopolysaccharide immune receptor activity\n",
      "    - protein binding\n",
      "    - peptidoglycan immune receptor activity\n",
      "    - lipoteichoic acid binding\n",
      "UniProt: {'Swiss-Prot': 'P08571'}\n",
      "\n",
      "Gene: CYP2E1\n",
      "Symbol: CYP2E1\n",
      "Name: cytochrome P450 family 2 subfamily E member 1\n",
      "Summary: This gene encodes a member of the cytochrome P450 superfamily of enzymes. The cytochrome P450 proteins are monooxygenases which catalyze many reactions involved in drug metabolism and synthesis of cholesterol, steroids and other lipids. This protein localizes to the endoplasmic reticulum and is induced by ethanol, the diabetic state, and starvation. The enzyme metabolizes both endogenous substrates, such as ethanol, acetone, and acetal, as well as exogenous substrates including benzene, carbon tetrachloride, ethylene glycol, and nitrosamines which are premutagens found in cigarette smoke. Due to its many substrates, this enzyme may be involved in such varied processes as gluconeogenesis, hepatic cirrhosis, diabetes, and cancer. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - long-chain fatty acid metabolic process\n",
      "    - lipid hydroxylation\n",
      "    - xenobiotic metabolic process\n",
      "    - xenobiotic metabolic process\n",
      "    - xenobiotic metabolic process\n",
      "    - xenobiotic metabolic process\n",
      "    - steroid metabolic process\n",
      "    - response to bacterium\n",
      "    - monoterpenoid metabolic process\n",
      "    - carbon tetrachloride metabolic process\n",
      "    - benzene metabolic process\n",
      "    - 4-nitrophenol metabolic process\n",
      "    - epoxygenase P450 pathway\n",
      "    - halogenated hydrocarbon metabolic process\n",
      "    - long-chain fatty acid biosynthetic process\n",
      "  CC:\n",
      "    - cytoplasm\n",
      "    - mitochondrial inner membrane\n",
      "    - endoplasmic reticulum membrane\n",
      "    - intracellular membrane-bounded organelle\n",
      "  MF:\n",
      "    - monooxygenase activity\n",
      "    - monooxygenase activity\n",
      "    - iron ion binding\n",
      "    - arachidonate epoxygenase activity\n",
      "    - oxidoreductase activity\n",
      "    - oxidoreductase activity, acting on paired donors, with incorporation or reduction of molecular oxygen, NAD(P)H as one donor, and incorporation of one atom of oxygen\n",
      "    - oxidoreductase activity, acting on paired donors, with incorporation or reduction of molecular oxygen, reduced flavin or flavoprotein as one donor, and incorporation of one atom of oxygen\n",
      "    - 4-nitrophenol 2-monooxygenase activity\n",
      "    - oxygen binding\n",
      "    - enzyme binding\n",
      "    - heme binding\n",
      "    - heme binding\n",
      "    - Hsp70 protein binding\n",
      "    - Hsp90 protein binding\n",
      "    - long-chain fatty acid omega-1 hydroxylase activity\n",
      "UniProt: {'Swiss-Prot': 'P05181', 'TrEMBL': ['H0Y7H4', 'H0YGV5', 'H0Y593', 'F5H694']}\n",
      "\n",
      "Gene: DDN\n",
      "Symbol: DDN\n",
      "Name: dendrin\n",
      "Summary: Predicted to enable DNA-binding transcription factor activity, RNA polymerase II-specific and RNA polymerase II cis-regulatory region sequence-specific DNA binding activity. Predicted to act upstream of or within positive regulation of transcription by RNA polymerase II. Located in cell projection and cytoplasm. [provided by Alliance of Genome Resources, Apr 2022]\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - positive regulation of transcription by RNA polymerase II\n",
      "  CC:\n",
      "    - nucleus\n",
      "    - cytoplasm\n",
      "    - endoplasmic reticulum membrane\n",
      "    - dendritic spine membrane\n",
      "    - cell projection\n",
      "    - perikaryon\n",
      "    - postsynaptic membrane\n",
      "    - presynapse\n",
      "  MF:\n",
      "    - RNA polymerase II cis-regulatory region sequence-specific DNA binding\n",
      "    - DNA-binding transcription factor activity, RNA polymerase II-specific\n",
      "    - protein binding\n",
      "UniProt: {'Swiss-Prot': 'O94850'}\n",
      "\n",
      "Gene: CYP17A1\n",
      "Symbol: CYP17A1\n",
      "Name: cytochrome P450 family 17 subfamily A member 1\n",
      "Summary: This gene encodes a member of the cytochrome P450 superfamily of enzymes. The cytochrome P450 proteins are monooxygenases which catalyze many reactions involved in drug metabolism and synthesis of cholesterol, steroids and other lipids. This protein localizes to the endoplasmic reticulum. It has both 17alpha-hydroxylase and 17,20-lyase activities and is a key enzyme in the steroidogenic pathway that produces progestins, mineralocorticoids, glucocorticoids, androgens, and estrogens. Mutations in this gene are associated with isolated steroid-17 alpha-hydroxylase deficiency, 17-alpha-hydroxylase/17,20-lyase deficiency, pseudohermaphroditism, and adrenal hyperplasia. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - steroid biosynthetic process\n",
      "    - androgen biosynthetic process\n",
      "    - glucocorticoid biosynthetic process\n",
      "    - glucocorticoid biosynthetic process\n",
      "    - sex differentiation\n",
      "    - steroid metabolic process\n",
      "    - hormone biosynthetic process\n",
      "    - hormone biosynthetic process\n",
      "    - progesterone metabolic process\n",
      "    - progesterone metabolic process\n",
      "  CC:\n",
      "    - endoplasmic reticulum\n",
      "    - endoplasmic reticulum membrane\n",
      "    - axon\n",
      "    - neuronal cell body\n",
      "  MF:\n",
      "    - steroid 17-alpha-monooxygenase activity\n",
      "    - steroid 17-alpha-monooxygenase activity\n",
      "    - steroid 17-alpha-monooxygenase activity\n",
      "    - steroid 17-alpha-monooxygenase activity\n",
      "    - iron ion binding\n",
      "    - lyase activity\n",
      "    - oxygen binding\n",
      "    - heme binding\n",
      "    - estrogen 16-alpha-hydroxylase activity\n",
      "UniProt: {'Swiss-Prot': 'P05093', 'TrEMBL': ['A0A1W2PRY0', 'Q1HB44', 'A0A1W2PQT5', 'A0A1W2PRK7', 'A0A1W2PQ28']}\n",
      "\n",
      "Gene: MAOA\n",
      "Symbol: MAOA\n",
      "Name: monoamine oxidase A\n",
      "Summary: This gene is one of two neighboring gene family members that encode mitochondrial enzymes which catalyze the oxidative deamination of amines, such as dopamine, norepinephrine, and serotonin. Mutation of this gene results in Brunner syndrome. This gene has also been associated with a variety of other psychiatric disorders, including antisocial behavior. Alternatively spliced transcript variants encoding multiple isoforms have been observed. [provided by RefSeq, Jul 2012].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - biogenic amine metabolic process\n",
      "    - positive regulation of signal transduction\n",
      "    - dopamine catabolic process\n",
      "  CC:\n",
      "    - mitochondrion\n",
      "    - mitochondrion\n",
      "    - mitochondrion\n",
      "    - mitochondrion\n",
      "    - mitochondrial outer membrane\n",
      "    - cytosol\n",
      "  MF:\n",
      "    - protein binding\n",
      "    - primary methylamine oxidase activity\n",
      "    - primary methylamine oxidase activity\n",
      "    - flavin adenine dinucleotide binding\n",
      "    - monoamine oxidase activity\n",
      "UniProt: {'Swiss-Prot': 'P21397', 'TrEMBL': ['Q49A63', 'Q53YE7', 'A0A8I5KP93', 'A0A8I5KUC3']}\n",
      "\n",
      "Gene: MIPEP\n",
      "Symbol: MIPEP\n",
      "Name: mitochondrial intermediate peptidase\n",
      "Summary: The product of this gene performs the final step in processing a specific class of nuclear-encoded proteins targeted to the mitochondrial matrix or inner membrane. This protein is primarily involved in the maturation of oxidative phosphorylation (OXPHOS)-related proteins. This gene may contribute to the functional effects of frataxin deficiency and the clinical manifestations of Friedreich ataxia. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - peptide metabolic process\n",
      "    - protein processing involved in protein targeting to mitochondrion\n",
      "  CC:\n",
      "    - mitochondrion\n",
      "    - mitochondrion\n",
      "    - mitochondrion\n",
      "    - mitochondrial matrix\n",
      "  MF:\n",
      "    - metalloendopeptidase activity\n",
      "    - protein binding\n",
      "    - metal ion binding\n",
      "UniProt: {'Swiss-Prot': 'Q99797'}\n",
      "\n",
      "Gene: GSTO1\n",
      "Symbol: GSTO1\n",
      "Name: glutathione S-transferase omega 1\n",
      "Summary: The protein encoded by this gene is an omega class glutathione S-transferase (GST) with glutathione-dependent thiol transferase and dehydroascorbate reductase activities. GSTs are involved in the metabolism of xenobiotics and carcinogens. The encoded protein acts as a homodimer and is found in the cytoplasm. Three transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Jul 2010].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - glutathione metabolic process\n",
      "    - regulation of release of sequestered calcium ion into cytosol by sarcoplasmic reticulum\n",
      "    - regulation of cardiac muscle contraction by regulation of the release of sequestered calcium ion\n",
      "    - positive regulation of skeletal muscle contraction by regulation of release of sequestered calcium ion\n",
      "    - L-ascorbic acid metabolic process\n",
      "    - L-ascorbic acid metabolic process\n",
      "    - xenobiotic catabolic process\n",
      "    - negative regulation of ryanodine-sensitive calcium-release channel activity\n",
      "    - positive regulation of ryanodine-sensitive calcium-release channel activity\n",
      "    - cellular response to arsenic-containing substance\n",
      "    - cellular oxidant detoxification\n",
      "  CC:\n",
      "    - cytoplasm\n",
      "    - cytoplasm\n",
      "    - cytosol\n",
      "    - extracellular exosome\n",
      "  MF:\n",
      "    - glutathione transferase activity\n",
      "    - glutathione transferase activity\n",
      "    - protein binding\n",
      "    - oxidoreductase activity\n",
      "    - glutathione dehydrogenase (ascorbate) activity\n",
      "    - glutathione dehydrogenase (ascorbate) activity\n",
      "    - methylarsonate reductase activity\n",
      "UniProt: {'Swiss-Prot': 'P78417', 'TrEMBL': ['Q5TA01', 'V9HWG9', 'Q5TA02']}\n",
      "\n",
      "Gene: SLCO4A1\n",
      "Symbol: SLCO4A1\n",
      "Name: solute carrier organic anion transporter family member 4A1\n",
      "Summary: Predicted to enable sodium-independent organic anion transmembrane transporter activity and thyroid hormone transmembrane transporter activity. Predicted to be involved in sodium-independent organic anion transport. Predicted to be located in plasma membrane. Predicted to be integral component of plasma membrane. [provided by Alliance of Genome Resources, Apr 2022]\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - monoatomic ion transport\n",
      "    - prostaglandin transport\n",
      "    - sodium-independent organic anion transport\n",
      "    - transmembrane transport\n",
      "    - thyroid hormone transport\n",
      "  CC:\n",
      "    - plasma membrane\n",
      "    - basolateral plasma membrane\n",
      "  MF:\n",
      "    - protein binding\n",
      "    - organic anion transmembrane transporter activity\n",
      "    - prostaglandin transmembrane transporter activity\n",
      "    - sodium-independent organic anion transmembrane transporter activity\n",
      "    - thyroid hormone transmembrane transporter activity\n",
      "UniProt: {'Swiss-Prot': 'Q96BD0', 'TrEMBL': ['A0A0A0MT51', 'E1P5H9']}\n",
      "\n",
      "Gene: ADH5\n",
      "Symbol: ADH5\n",
      "Name: alcohol dehydrogenase 5 (class III), chi polypeptide\n",
      "Summary: This gene encodes a member of the alcohol dehydrogenase family. Members of this family metabolize a wide variety of substrates, including ethanol, retinol, other aliphatic alcohols, hydroxysteroids, and lipid peroxidation products. The encoded protein forms a homodimer. It has virtually no activity for ethanol oxidation, but exhibits high activity for oxidation of long-chain primary alcohols and for oxidation of S-hydroxymethyl-glutathione, a spontaneous adduct between formaldehyde and glutathione. This enzyme is an important component of cellular metabolism for the elimination of formaldehyde, a potent irritant and sensitizing agent that causes lacrymation, rhinitis, pharyngitis, and contact dermatitis. The human genome contains several non-transcribed pseudogenes related to this gene. [provided by RefSeq, Oct 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - retinoid metabolic process\n",
      "    - respiratory system process\n",
      "    - fatty acid omega-oxidation\n",
      "    - response to lipopolysaccharide\n",
      "    - positive regulation of blood pressure\n",
      "    - formaldehyde catabolic process\n",
      "    - response to nitrosative stress\n",
      "    - response to redox state\n",
      "  CC:\n",
      "    - mitochondrion\n",
      "    - cytosol\n",
      "    - cytosol\n",
      "    - extracellular exosome\n",
      "  MF:\n",
      "    - alcohol dehydrogenase (NAD+) activity\n",
      "    - fatty acid binding\n",
      "    - zinc ion binding\n",
      "    - zinc ion binding\n",
      "    - electron transfer activity\n",
      "    - formaldehyde dehydrogenase (NAD+) activity\n",
      "    - identical protein binding\n",
      "    - S-(hydroxymethyl)glutathione dehydrogenase [NAD(P)+] activity\n",
      "    - S-(hydroxymethyl)glutathione dehydrogenase [NAD(P)+] activity\n",
      "    - S-nitrosoglutathione reductase (NADH) activity\n",
      "    - S-(hydroxymethyl)glutathione dehydrogenase (NADP+) activity\n",
      "    - S-(hydroxymethyl)glutathione dehydrogenase (NAD+) activity\n",
      "UniProt: {'Swiss-Prot': 'P11766', 'TrEMBL': ['D6RAY0', 'Q6IRT1', 'D6RFE4', 'H0YAG8', 'D6R9G2']}\n",
      "\n",
      "Gene: CLCN2\n",
      "Symbol: CLCN2\n",
      "Name: chloride voltage-gated channel 2\n",
      "Summary: This gene encodes a voltage-gated chloride channel. The encoded protein is a transmembrane protein that maintains chloride ion homeostasis in various cells. Defects in this gene may be a cause of certain epilepsies. Four transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Mar 2012].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - chloride transport\n",
      "    - phagocytosis, engulfment\n",
      "    - stabilization of membrane potential\n",
      "    - lung development\n",
      "    - regulation of aldosterone biosynthetic process\n",
      "    - positive regulation of oligodendrocyte differentiation\n",
      "    - retina development in camera-type eye\n",
      "    - regulation of resting membrane potential\n",
      "    - cell differentiation involved in salivary gland development\n",
      "    - cellular hypotonic response\n",
      "    - acinar cell differentiation\n",
      "    - regulation of membrane depolarization during action potential\n",
      "    - chloride transmembrane transport\n",
      "  CC:\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - basolateral plasma membrane\n",
      "    - dendritic spine membrane\n",
      "    - chloride channel complex\n",
      "    - axon initial segment\n",
      "    - perikaryon\n",
      "    - myelin sheath\n",
      "    - postsynaptic membrane\n",
      "    - astrocyte end-foot\n",
      "  MF:\n",
      "    - voltage-gated chloride channel activity\n",
      "    - voltage-gated chloride channel activity\n",
      "    - protein binding\n",
      "    - chloride channel regulator activity\n",
      "    - volume-sensitive chloride channel activity\n",
      "UniProt: {'Swiss-Prot': 'P51788', 'TrEMBL': ['A0A1B0GVL9', 'A0A1B0GV52', 'A0A1B0GVU4', 'A0A1B0GTY0', 'A0A1B0GUY6', 'A0A1B0GUZ8', 'A0A1B0GTJ3', 'A0A1B0GWC8', 'H7C0Q8', 'A0A1B0GVW7']}\n",
      "\n",
      "Gene: NME6\n",
      "Symbol: NME6\n",
      "Name: NME/NM23 nucleoside diphosphate kinase 6\n",
      "Summary: Nucleoside diphosphate (NDP) kinases (EC 2.7.4.6), such as NME6, are ubiquitous enzymes that catalyze transfer of gamma-phosphates, via a phosphohistidine intermediate, between nucleoside and dioxynucleoside tri- and diphosphates (Mehus et al., 1999 [PubMed 10453732]).[supplied by OMIM, Jul 2010].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - GTP biosynthetic process\n",
      "    - UTP biosynthetic process\n",
      "    - CTP biosynthetic process\n",
      "    - apoptotic process\n",
      "    - negative regulation of cell growth\n",
      "    - negative regulation of cell growth\n",
      "    - negative regulation of mitotic nuclear division\n",
      "    - negative regulation of mitotic nuclear division\n",
      "  CC:\n",
      "    - mitochondrion\n",
      "    - mitochondrion\n",
      "    - mitochondrion\n",
      "    - mitochondrial inner membrane\n",
      "    - mitochondrial matrix\n",
      "  MF:\n",
      "    - nucleoside diphosphate kinase activity\n",
      "    - nucleoside diphosphate kinase activity\n",
      "    - protein binding\n",
      "    - ATP binding\n",
      "    - metal ion binding\n",
      "UniProt: {'Swiss-Prot': 'O75414', 'TrEMBL': ['A0A2R8Y7V2', 'A0A0C4DG91', 'C9JQB1', 'A0A2R8Y547', 'F2Z2J2', 'C9J1J2', 'C9J867', 'C9J9V6']}\n",
      "\n",
      "Gene: COX8A\n",
      "Symbol: COX8A\n",
      "Name: cytochrome c oxidase subunit 8A\n",
      "Summary: The protein encoded by this gene is the terminal enzyme of the respiratory chain, coupling the transfer of electrons from cytochrome c to molecular oxygen, with the concomitant production of a proton electrochemical gradient across the inner mitochondrial membrane. In addition to 3 mitochondrially encoded subunits, which perform the catalytic function, the eukaryotic enzyme contains nuclear-encoded smaller subunits, ranging in number from 4 in some organisms to 10 in mammals. It has been proposed that nuclear-encoded subunits may be involved in the modulation of the catalytic function. This gene encodes one of the nuclear-encoded subunits. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - generation of precursor metabolites and energy\n",
      "    - mitochondrial electron transport, cytochrome c to oxygen\n",
      "    - cellular respiration\n",
      "    - proton transmembrane transport\n",
      "  CC:\n",
      "    - mitochondrion\n",
      "    - mitochondrion\n",
      "    - mitochondrial inner membrane\n",
      "    - mitochondrial membrane\n",
      "    - respiratory chain complex IV\n",
      "  MF:\n",
      "    - cytochrome-c oxidase activity\n",
      "    - protein binding\n",
      "UniProt: {'Swiss-Prot': 'P10176', 'TrEMBL': 'Q53XN1'}\n",
      "\n",
      "Gene: LTA4H\n",
      "Symbol: LTA4H\n",
      "Name: leukotriene A4 hydrolase\n",
      "Summary: The protein encoded by this gene is an enzyme that contains both hydrolase and aminopeptidase activities. The hydrolase activity is used in the final step of the biosynthesis of leukotriene B4, a proinflammatory mediator. The aminopeptidase activity has been shown to degrade proline-glycine-proline (PGP), a neutrophil chemoattractant and biomarker for chronic obstructive pulmonary disease (COPD). Several transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Sep 2015].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - proteolysis\n",
      "    - lipid metabolic process\n",
      "    - response to zinc ion\n",
      "    - leukotriene biosynthetic process\n",
      "    - leukotriene biosynthetic process\n",
      "    - leukotriene biosynthetic process\n",
      "    - protein metabolic process\n",
      "    - peptide catabolic process\n",
      "    - peptide catabolic process\n",
      "    - response to peptide hormone\n",
      "    - type I pneumocyte differentiation\n",
      "  CC:\n",
      "    - extracellular region\n",
      "    - nucleus\n",
      "    - nucleoplasm\n",
      "    - cytosol\n",
      "    - cytosol\n",
      "    - cytosol\n",
      "    - extracellular exosome\n",
      "    - tertiary granule lumen\n",
      "    - ficolin-1-rich granule lumen\n",
      "  MF:\n",
      "    - RNA binding\n",
      "    - aminopeptidase activity\n",
      "    - aminopeptidase activity\n",
      "    - epoxide hydrolase activity\n",
      "    - epoxide hydrolase activity\n",
      "    - epoxide hydrolase activity\n",
      "    - epoxide hydrolase activity\n",
      "    - leukotriene-A4 hydrolase activity\n",
      "    - leukotriene-A4 hydrolase activity\n",
      "    - protein binding\n",
      "    - peptidase activity\n",
      "    - zinc ion binding\n",
      "    - zinc ion binding\n",
      "    - tripeptide aminopeptidase activity\n",
      "    - metalloaminopeptidase activity\n",
      "UniProt: {'Swiss-Prot': 'P09960', 'TrEMBL': ['B4DEH5', 'A0A140VK27']}\n",
      "\n",
      "Gene: LMX1B\n",
      "Symbol: LMX1B\n",
      "Name: LIM homeobox transcription factor 1 beta\n",
      "Summary: This gene encodes a member of LIM-homeodomain family of proteins containing two N-terminal zinc-binding LIM domains, 1 homeodomain, and a C-terminal glutamine-rich domain. It functions as a transcription factor, and is essential for the normal development of dorsal limb structures, the glomerular basement membrane, the anterior segment of the eye, and dopaminergic and serotonergic neurons. Mutations in this gene are associated with nail-patella syndrome. Alternatively spliced transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Mar 2010].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - regulation of DNA-templated transcription\n",
      "    - regulation of transcription by RNA polymerase II\n",
      "    - dorsal/ventral pattern formation\n",
      "    - neuron differentiation\n",
      "    - neuron differentiation\n",
      "    - positive regulation of transcription by RNA polymerase II\n",
      "    - positive regulation of transcription by RNA polymerase II\n",
      "    - dopaminergic neuron differentiation\n",
      "  CC:\n",
      "    - chromatin\n",
      "    - nucleus\n",
      "    - nucleus\n",
      "  MF:\n",
      "    - RNA polymerase II transcription regulatory region sequence-specific DNA binding\n",
      "    - RNA polymerase II transcription regulatory region sequence-specific DNA binding\n",
      "    - DNA-binding transcription factor activity, RNA polymerase II-specific\n",
      "    - DNA-binding transcription factor activity, RNA polymerase II-specific\n",
      "    - DNA-binding transcription factor activity, RNA polymerase II-specific\n",
      "    - DNA-binding transcription factor activity\n",
      "    - protein binding\n",
      "    - metal ion binding\n",
      "    - sequence-specific double-stranded DNA binding\n",
      "UniProt: {'Swiss-Prot': 'O60663', 'TrEMBL': ['Q6ISE0', 'B7ZLH2']}\n",
      "\n",
      "Gene: LAMA4\n",
      "Symbol: LAMA4\n",
      "Name: laminin subunit alpha 4\n",
      "Summary: Laminins, a family of extracellular matrix glycoproteins, are the major noncollagenous constituent of basement membranes. They have been implicated in a wide variety of biological processes including cell adhesion, differentiation, migration, signaling, neurite outgrowth and metastasis. Laminins are composed of 3 non identical chains: laminin alpha, beta and gamma (formerly A, B1, and B2, respectively) and they form a cruciform structure consisting of 3 short arms, each formed by a different chain, and a long arm composed of all 3 chains. Each laminin chain is a multidomain protein encoded by a distinct gene. Several isoforms of each chain have been described. Different alpha, beta and gamma chain isomers combine to give rise to different heterotrimeric laminin isoforms which are designated by Arabic numerals in the order of their discovery, i.e. alpha1beta1gamma1 heterotrimer is laminin 1. The biological functions of the different chains and trimer molecules are largely unknown, but some of the chains have been shown to differ with respect to their tissue distribution, presumably reflecting diverse functions in vivo. This gene encodes the alpha chain isoform laminin, alpha 4. The domain structure of alpha 4 is similar to that of alpha 3, both of which resemble truncated versions of alpha 1 and alpha 2, in that approximately 1,200 residues at the N-terminus (domains IV, V and VI) have been lost. Laminin, alpha 4 contains the C-terminal G domain which distinguishes all alpha chains from the beta and gamma chains. The RNA analysis from adult and fetal tissues revealed developmental regulation of expression, however, the exact function of laminin, alpha 4 is not known. Tissue-specific utilization of alternative polyA-signal has been described in literature. Alternative splicing results in multiple transcript variants encoding distinct isoforms. [provided by RefSeq, Aug 2011].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - cell adhesion\n",
      "    - regulation of cell adhesion\n",
      "    - regulation of cell migration\n",
      "    - regulation of embryonic development\n",
      "    - negative regulation of cold-induced thermogenesis\n",
      "  CC:\n",
      "    - extracellular region\n",
      "    - basement membrane\n",
      "    - collagen-containing extracellular matrix\n",
      "    - collagen-containing extracellular matrix\n",
      "    - extracellular exosome\n",
      "  MF:\n",
      "    - signaling receptor binding\n",
      "    - extracellular matrix structural constituent\n",
      "    - extracellular matrix structural constituent\n",
      "    - protein binding\n",
      "UniProt: {'Swiss-Prot': 'Q16363', 'TrEMBL': ['Q5D044', 'A0A0A0MTC7', 'Q6LET9', 'H0YAP9', 'E5RFD7', 'H0YAQ5', 'A0A494C139', 'A0A494C1K8', 'E5RFQ2', 'E5RHF3', 'H0Y351', 'E5RK79']}\n",
      "\n",
      "Gene: ISOC2\n",
      "Symbol: ISOC2\n",
      "Name: isochorismatase domain containing 2\n",
      "Summary: Involved in protein destabilization. Located in cytoplasm and nucleus. [provided by Alliance of Genome Resources, Apr 2022]\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - protein destabilization\n",
      "  CC:\n",
      "    - nucleus\n",
      "    - cytoplasm\n",
      "    - cytoplasm\n",
      "    - mitochondrion\n",
      "  MF:\n",
      "    - protein binding\n",
      "UniProt: {'Swiss-Prot': 'Q96AB3', 'TrEMBL': ['K7ENV7', 'K7EKW4']}\n",
      "\n",
      "Gene: AKR1C1\n",
      "Symbol: AKR1C1\n",
      "Name: aldo-keto reductase family 1 member C1\n",
      "Summary: This gene encodes a member of the aldo/keto reductase superfamily, which consists of more than 40 known enzymes and proteins. These enzymes catalyze the conversion of aldehydes and ketones to their corresponding alcohols by utilizing NADH and/or NADPH as cofactors. The enzymes display overlapping but distinct substrate specificity. This enzyme catalyzes the reaction of progesterone to the inactive form 20-alpha-hydroxy-progesterone. This gene shares high sequence identity with three other gene members and is clustered with those three genes at chromosome 10p15-p14. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - retinoid metabolic process\n",
      "    - prostaglandin metabolic process\n",
      "    - xenobiotic metabolic process\n",
      "    - xenobiotic metabolic process\n",
      "    - digestion\n",
      "    - bile acid metabolic process\n",
      "    - bile acid and bile salt transport\n",
      "    - intestinal cholesterol absorption\n",
      "    - epithelial cell differentiation\n",
      "    - progesterone metabolic process\n",
      "    - progesterone metabolic process\n",
      "    - retinal metabolic process\n",
      "    - cholesterol homeostasis\n",
      "    - daunorubicin metabolic process\n",
      "    - daunorubicin metabolic process\n",
      "    - doxorubicin metabolic process\n",
      "    - doxorubicin metabolic process\n",
      "    - cellular response to jasmonic acid stimulus\n",
      "    - positive regulation of reactive oxygen species metabolic process\n",
      "  CC:\n",
      "    - cytosol\n",
      "    - cytosol\n",
      "    - cytosol\n",
      "    - extracellular exosome\n",
      "  MF:\n",
      "    - aldose reductase (NADPH) activity\n",
      "    - aldose reductase (NADPH) activity\n",
      "    - aldo-keto reductase (NADPH) activity\n",
      "    - estradiol 17-beta-dehydrogenase [NAD(P)+] activity\n",
      "    - protein binding\n",
      "    - oxidoreductase activity, acting on NAD(P)H, quinone or similar compound as acceptor\n",
      "    - carboxylic acid binding\n",
      "    - bile acid binding\n",
      "    - bile acid binding\n",
      "    - 3-beta-hydroxy-5-beta-steroid dehydrogenase (NADP+) activity\n",
      "    - steroid dehydrogenase activity, acting on the CH-OH group of donors, NAD or NADP as acceptor\n",
      "    - 17-alpha,20-alpha-dihydroxypregn-4-en-3-one dehydrogenase [NAD(P)+] activity\n",
      "    - androsterone dehydrogenase [NAD(P)+] activity\n",
      "    - 5-alpha-androstane-3-beta,17-beta-diol dehydrogenase (NADP+) activity\n",
      "    - androsterone dehydrogenase (B-specific) activity\n",
      "    - androstan-3-alpha,17-beta-diol dehydrogenase (NAD+) activity\n",
      "    - testosterone 17-beta-dehydrogenase (NADP+) activity\n",
      "    - ketosteroid monooxygenase activity\n",
      "    - ketosteroid monooxygenase activity\n",
      "    - trans-1,2-dihydrobenzene-1,2-diol dehydrogenase activity\n",
      "    - indanol dehydrogenase activity\n",
      "UniProt: {'Swiss-Prot': 'Q04828', 'TrEMBL': ['H0Y804', 'A6NHU4']}\n",
      "\n",
      "Gene: GPR21\n",
      "Symbol: GPR21\n",
      "Name: G protein-coupled receptor 21\n",
      "Summary: This gene encodes a member of the G-protein-coupled receptor 1 family. G-protein coupled receptors are membrane proteins which activate signaling cascades as a response to extracellular stress. The encoded protein activates a Gq signal transduction pathway which mobilizes calcium. [provided by RefSeq, Nov 2012].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - G protein-coupled receptor signaling pathway\n",
      "    - insulin receptor signaling pathway\n",
      "    - positive regulation of multicellular organism growth\n",
      "    - glucose homeostasis\n",
      "    - negative regulation of insulin receptor signaling pathway\n",
      "  CC:\n",
      "    - plasma membrane\n",
      "  MF:\n",
      "    - G protein-coupled receptor activity\n",
      "UniProt: {'Swiss-Prot': 'Q99679', 'TrEMBL': ['B4DSD1', 'H9NIL4']}\n",
      "\n",
      "Gene: SLC13A3\n",
      "Symbol: SLC13A3\n",
      "Name: solute carrier family 13 member 3\n",
      "Summary: Mammalian sodium-dicarboxylate cotransporters transport succinate and other Krebs cycle intermediates. They fall into 2 categories based on their substrate affinity: low affinity and high affinity. Both the low- and high-affinity transporters play an important role in the handling of citrate by the kidneys. The protein encoded by this gene represents the high-affinity form. Alternatively spliced transcript variants encoding different isoforms have been found for this gene, although the full-length nature of some of them have not been characterized yet. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - dicarboxylic acid transport\n",
      "    - dicarboxylic acid transport\n",
      "    - lipid transport\n",
      "    - alpha-ketoglutarate transport\n",
      "    - citrate transport\n",
      "    - citrate transport\n",
      "    - glutathione transmembrane transport\n",
      "    - sodium ion transmembrane transport\n",
      "    - succinate transmembrane transport\n",
      "    - succinate transmembrane transport\n",
      "    - succinate transmembrane transport\n",
      "    - transport across blood-brain barrier\n",
      "  CC:\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - basolateral plasma membrane\n",
      "    - extracellular exosome\n",
      "  MF:\n",
      "    - dicarboxylic acid transmembrane transporter activity\n",
      "    - dicarboxylic acid transmembrane transporter activity\n",
      "    - protein binding\n",
      "    - citrate transmembrane transporter activity\n",
      "    - citrate transmembrane transporter activity\n",
      "    - alpha-ketoglutarate transmembrane transporter activity\n",
      "    - succinate transmembrane transporter activity\n",
      "    - succinate transmembrane transporter activity\n",
      "    - succinate transmembrane transporter activity\n",
      "    - high-affinity sodium:dicarboxylate symporter activity\n",
      "    - sodium:dicarboxylate symporter activity\n",
      "    - sodium:dicarboxylate symporter activity\n",
      "    - sodium:dicarboxylate symporter activity\n",
      "    - glutathione transmembrane transporter activity\n",
      "UniProt: {'Swiss-Prot': 'Q8WWT9', 'TrEMBL': ['C9J4A3', 'B4DF27', 'H7C0C8', 'C9J7L4', 'A0A0A0MRQ6', 'X6RDV4']}\n",
      "\n",
      "Gene: ABCB11\n",
      "Symbol: ABCB11\n",
      "Name: ATP binding cassette subfamily B member 11\n",
      "Summary: The membrane-associated protein encoded by this gene is a member of the superfamily of ATP-binding cassette (ABC) transporters. ABC proteins transport various molecules across extra- and intra-cellular membranes. ABC genes are divided into seven distinct subfamilies (ABC1, MDR/TAP, MRP, ALD, OABP, GCN20, White). This protein is a member of the MDR/TAP subfamily. Members of the MDR/TAP subfamily are involved in multidrug resistance. The protein encoded by this gene is the major canalicular bile salt export pump in man. Mutations in this gene cause a form of progressive familial intrahepatic cholestases which are a group of inherited disorders with severe cholestatic liver disease from early infancy. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - fatty acid metabolic process\n",
      "    - bile acid biosynthetic process\n",
      "    - xenobiotic metabolic process\n",
      "    - xenobiotic metabolic process\n",
      "    - xenobiotic transmembrane transport\n",
      "    - xenobiotic transmembrane transport\n",
      "    - response to oxidative stress\n",
      "    - bile acid metabolic process\n",
      "    - bile acid metabolic process\n",
      "    - bile acid metabolic process\n",
      "    - bile acid and bile salt transport\n",
      "    - bile acid and bile salt transport\n",
      "    - bile acid and bile salt transport\n",
      "    - bile acid and bile salt transport\n",
      "    - canalicular bile acid transport\n",
      "    - canalicular bile acid transport\n",
      "    - canalicular bile acid transport\n",
      "    - canalicular bile acid transport\n",
      "    - protein ubiquitination\n",
      "    - protein ubiquitination\n",
      "    - regulation of fatty acid beta-oxidation\n",
      "    - carbohydrate transmembrane transport\n",
      "    - cholesterol homeostasis\n",
      "    - response to estrogen\n",
      "    - response to ethanol\n",
      "    - xenobiotic export from cell\n",
      "    - xenobiotic export from cell\n",
      "    - transmembrane transport\n",
      "    - lipid homeostasis\n",
      "    - phospholipid homeostasis\n",
      "    - positive regulation of bile acid secretion\n",
      "    - regulation of bile acid metabolic process\n",
      "  CC:\n",
      "    - Golgi membrane\n",
      "    - endosome\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - cell surface\n",
      "    - basolateral plasma membrane\n",
      "    - apical plasma membrane\n",
      "    - apical plasma membrane\n",
      "    - apical plasma membrane\n",
      "    - intercellular canaliculus\n",
      "    - intracellular canaliculus\n",
      "    - recycling endosome\n",
      "    - recycling endosome membrane\n",
      "    - extracellular exosome\n",
      "  MF:\n",
      "    - protein binding\n",
      "    - ATP binding\n",
      "    - ABC-type xenobiotic transporter activity\n",
      "    - ABC-type xenobiotic transporter activity\n",
      "    - bile acid transmembrane transporter activity\n",
      "    - bile acid transmembrane transporter activity\n",
      "    - canalicular bile acid transmembrane transporter activity\n",
      "    - canalicular bile acid transmembrane transporter activity\n",
      "    - canalicular bile acid transmembrane transporter activity\n",
      "    - canalicular bile acid transmembrane transporter activity\n",
      "    - carbohydrate transmembrane transporter activity\n",
      "    - ABC-type bile acid transporter activity\n",
      "    - ABC-type bile acid transporter activity\n",
      "    - ABC-type bile acid transporter activity\n",
      "    - ABC-type bile acid transporter activity\n",
      "    - ABC-type bile acid transporter activity\n",
      "    - ATP hydrolysis activity\n",
      "UniProt: {'Swiss-Prot': 'O95342', 'TrEMBL': ['H7C486', 'A0A3B3ISD4', 'A0A3B3IS78', 'A0A3B3ITV9']}\n",
      "\n",
      "Gene: MORC3\n",
      "Symbol: MORC3\n",
      "Name: MORC family CW-type zinc finger 3\n",
      "Summary: This gene encodes a protein that localizes to the nuclear matrix and forms nuclear bodies via an ATP-dependent mechanism. The protein is predicted to have coiled-coil and zinc finger domains and has RNA binding activity. Alternative splicing produces multiple transcript variants encoding distinct isoforms. [provided by RefSeq, Feb 2016].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - negative regulation of transcription by RNA polymerase II\n",
      "    - protein phosphorylation\n",
      "    - post-embryonic development\n",
      "    - peptidyl-serine phosphorylation\n",
      "    - negative regulation of interferon-beta production\n",
      "    - negative regulation of fibroblast proliferation\n",
      "    - protein stabilization\n",
      "    - maintenance of protein location in nucleus\n",
      "    - maintenance of protein location in nucleus\n",
      "    - antiviral innate immune response\n",
      "    - positive regulation of cellular senescence\n",
      "  CC:\n",
      "    - chromatin\n",
      "    - nucleus\n",
      "    - nucleoplasm\n",
      "    - nuclear matrix\n",
      "    - PML body\n",
      "    - PML body\n",
      "    - PML body\n",
      "  MF:\n",
      "    - DNA binding\n",
      "    - RNA binding\n",
      "    - protein binding\n",
      "    - zinc ion binding\n",
      "    - ATP hydrolysis activity\n",
      "    - protein-macromolecule adaptor activity\n",
      "    - methylated histone binding\n",
      "    - methylated histone binding\n",
      "UniProt: {'Swiss-Prot': 'Q14149', 'TrEMBL': ['Q4VBZ9', 'B4DHJ4', 'H0YHQ4', 'H0YIR5']}\n",
      "\n",
      "Gene: DPP7\n",
      "Symbol: DPP7\n",
      "Name: dipeptidyl peptidase 7\n",
      "Summary: The protein encoded by this gene is a post-proline cleaving aminopeptidase expressed in quiescent lymphocytes. The resting lymphocytes are maintained through suppression of apoptosis, a state which is disrupted by inhibition of this novel serine protease. The enzyme has strong sequence homology with prolylcarboxypeptidase and is active at both acidic and neutral pH. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - proteolysis\n",
      "    - lysosomal protein catabolic process\n",
      "  CC:\n",
      "    - extracellular region\n",
      "    - Golgi apparatus\n",
      "    - vesicle\n",
      "    - vesicle\n",
      "    - azurophil granule lumen\n",
      "    - intracellular membrane-bounded organelle\n",
      "    - extracellular exosome\n",
      "  MF:\n",
      "    - aminopeptidase activity\n",
      "    - serine-type peptidase activity\n",
      "    - dipeptidyl-peptidase activity\n",
      "    - serine-type exopeptidase activity\n",
      "UniProt: {'Swiss-Prot': 'Q9UHL4'}\n",
      "\n",
      "Gene: MAPK15\n",
      "Symbol: MAPK15\n",
      "Name: mitogen-activated protein kinase 15\n",
      "Summary: Enables MAP kinase activity and chromatin binding activity. Involved in several processes, including dopamine uptake; positive regulation of DNA metabolic process; and regulation of organelle organization. Located in several cellular components, including Golgi apparatus; autophagosome; and microtubule organizing center. Biomarker of breast cancer. [provided by Alliance of Genome Resources, Apr 2022]\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - regulation of COPII vesicle coating\n",
      "    - DNA damage response\n",
      "    - endoplasmic reticulum organization\n",
      "    - JNK cascade\n",
      "    - positive regulation of cell population proliferation\n",
      "    - positive regulation of cell population proliferation\n",
      "    - regulation of autophagy\n",
      "    - negative regulation of cell migration\n",
      "    - positive regulation of telomere maintenance\n",
      "    - intracellular signal transduction\n",
      "    - positive regulation of transcription by RNA polymerase II\n",
      "    - protein autophosphorylation\n",
      "    - dopamine uptake\n",
      "    - regulation of cilium assembly\n",
      "    - regulation of cilium assembly\n",
      "    - protein localization to ciliary transition zone\n",
      "    - positive regulation of metaphase/anaphase transition of meiosis I\n",
      "    - positive regulation of spindle assembly\n",
      "  CC:\n",
      "    - extracellular region\n",
      "    - nucleus\n",
      "    - nucleus\n",
      "    - cytoplasm\n",
      "    - cytoplasm\n",
      "    - autophagosome\n",
      "    - Golgi apparatus\n",
      "    - centriole\n",
      "    - cell-cell junction\n",
      "    - bicellular tight junction\n",
      "    - axoneme\n",
      "    - cytoplasmic vesicle\n",
      "    - ciliary basal body\n",
      "    - meiotic spindle\n",
      "  MF:\n",
      "    - chromatin binding\n",
      "    - protein kinase activity\n",
      "    - protein serine/threonine kinase activity\n",
      "    - JUN kinase activity\n",
      "    - MAP kinase activity\n",
      "    - protein binding\n",
      "    - ATP binding\n",
      "    - kinase activity\n",
      "    - SH3 domain binding\n",
      "    - protein serine kinase activity\n",
      "UniProt: {'Swiss-Prot': 'Q8TD08', 'TrEMBL': 'A0A1C7CYZ1'}\n",
      "\n",
      "Gene: DHRS9\n",
      "Symbol: DHRS9\n",
      "Name: dehydrogenase/reductase 9\n",
      "Summary: This gene encodes a member of the short-chain dehydrogenases/reductases (SDR) family. The encoded protein has been identified as a moonlighting protein based on its ability to perform mechanistically distinct functions. This protein demonstrates oxidoreductase activity toward hydroxysteroids and is able to convert 3-alpha-tetrahydroprogesterone to dihydroxyprogesterone and 3-alpha-androstanediol to dihydroxyprogesterone in the cytoplasm, and may additionally function as a transcriptional repressor in the nucleus. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Jan 2014].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - steroid metabolic process\n",
      "    - androgen metabolic process\n",
      "    - epithelial cell differentiation\n",
      "    - progesterone metabolic process\n",
      "    - retinol metabolic process\n",
      "    - 9-cis-retinoic acid biosynthetic process\n",
      "  CC:\n",
      "    - endoplasmic reticulum membrane\n",
      "    - endoplasmic reticulum membrane\n",
      "    - intracellular membrane-bounded organelle\n",
      "  MF:\n",
      "    - alcohol dehydrogenase (NAD+) activity\n",
      "    - all-trans-retinol dehydrogenase (NAD+) activity\n",
      "    - all-trans-retinol dehydrogenase (NAD+) activity\n",
      "    - protein binding\n",
      "    - oxidoreductase activity\n",
      "    - racemase and epimerase activity\n",
      "    - androsterone dehydrogenase [NAD(P)+] activity\n",
      "    - testosterone dehydrogenase (NAD+) activity\n",
      "    - androstan-3-alpha,17-beta-diol dehydrogenase (NAD+) activity\n",
      "UniProt: {'Swiss-Prot': 'Q9BPW9', 'TrEMBL': 'C9J430'}\n",
      "\n",
      "Gene: BCS1L\n",
      "Symbol: BCS1L\n",
      "Name: BCS1 homolog, ubiquinol-cytochrome c reductase complex chaperone\n",
      "Summary: This gene encodes a homolog of the S. cerevisiae bcs1 protein which is involved in the assembly of complex III of the mitochondrial respiratory chain. The encoded protein does not contain a mitochondrial targeting sequence but experimental studies confirm that it is imported into mitochondria. Mutations in this gene are associated with mitochondrial complex III deficiency and the GRACILE syndrome. Several alternatively spliced transcripts encoding two different isoforms have been described. [provided by RefSeq, Jan 2016].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - mitochondrion organization\n",
      "    - protein insertion into mitochondrial inner membrane from matrix\n",
      "    - mitochondrial respiratory chain complex I assembly\n",
      "    - mitochondrial cytochrome c oxidase assembly\n",
      "    - mitochondrial respiratory chain complex III assembly\n",
      "    - mitochondrial respiratory chain complex III assembly\n",
      "  CC:\n",
      "    - mitochondrion\n",
      "    - mitochondrion\n",
      "    - mitochondrial inner membrane\n",
      "    - respiratory chain complex III\n",
      "  MF:\n",
      "    - protein binding\n",
      "    - ATP binding\n",
      "    - ATP hydrolysis activity\n",
      "UniProt: {'Swiss-Prot': 'Q9Y276', 'TrEMBL': ['A0A024R445', 'A8JZZ8', 'C9J8G3', 'C9J4Q9', 'H7BZF6', 'H7C492', 'C9JAS4', 'A0A2R8Y7T3', 'C9J1S9']}\n",
      "\n",
      "Gene: DNAJA4\n",
      "Symbol: DNAJA4\n",
      "Name: DnaJ heat shock protein family (Hsp40) member A4\n",
      "Summary: Enables chaperone binding activity and unfolded protein binding activity. Involved in several processes, including negative regulation of endothelial cell migration; negative regulation of inclusion body assembly; and protein refolding. Located in cytosol and membrane. [provided by Alliance of Genome Resources, Apr 2022]\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - response to heat\n",
      "    - negative regulation of endothelial cell migration\n",
      "    - positive regulation of gene expression\n",
      "    - protein refolding\n",
      "    - protein refolding\n",
      "    - negative regulation of inclusion body assembly\n",
      "  CC:\n",
      "    - cytoplasm\n",
      "    - cytosol\n",
      "    - cytosol\n",
      "    - cytosol\n",
      "    - membrane\n",
      "  MF:\n",
      "    - protein binding\n",
      "    - ATP binding\n",
      "    - zinc ion binding\n",
      "    - Hsp70 protein binding\n",
      "    - unfolded protein binding\n",
      "    - protein-folding chaperone binding\n",
      "    - protein-folding chaperone binding\n",
      "UniProt: {'Swiss-Prot': 'Q8WW22', 'TrEMBL': ['F8WF76', 'Q69YX3', 'F5H170', 'F5H300', 'F5GWV1', 'C9JDE6']}\n",
      "\n",
      "Gene: SCN8A\n",
      "Symbol: SCN8A\n",
      "Name: sodium voltage-gated channel alpha subunit 8\n",
      "Summary: This gene encodes a member of the sodium channel alpha subunit gene family. The encoded protein forms the ion pore region of the voltage-gated sodium channel. This protein is essential for the rapid membrane depolarization that occurs during the formation of the action potential in excitable neurons. Mutations in this gene are associated with cognitive disability, pancerebellar atrophy and ataxia. Alternate splicing results in multiple transcript variants.[provided by RefSeq, May 2010].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - action potential\n",
      "    - sodium ion transport\n",
      "    - peripheral nervous system development\n",
      "    - neuronal action potential\n",
      "    - optic nerve development\n",
      "    - sodium ion transmembrane transport\n",
      "    - myelination\n",
      "    - cardiac muscle cell action potential involved in contraction\n",
      "  CC:\n",
      "    - voltage-gated sodium channel complex\n",
      "    - voltage-gated sodium channel complex\n",
      "    - voltage-gated sodium channel complex\n",
      "    - podosome\n",
      "    - plasma membrane\n",
      "    - membrane\n",
      "    - Z disc\n",
      "    - cell junction\n",
      "    - axon\n",
      "    - cytoplasmic vesicle\n",
      "    - node of Ranvier\n",
      "    - axon initial segment\n",
      "    - presynaptic active zone membrane\n",
      "    - anchoring junction\n",
      "    - parallel fiber to Purkinje cell synapse\n",
      "    - postsynaptic density membrane\n",
      "    - glutamatergic synapse\n",
      "  MF:\n",
      "    - voltage-gated sodium channel activity\n",
      "    - voltage-gated sodium channel activity\n",
      "    - voltage-gated sodium channel activity\n",
      "    - voltage-gated sodium channel activity\n",
      "    - protein binding\n",
      "    - sodium ion binding\n",
      "UniProt: {'Swiss-Prot': 'Q9UQD0', 'TrEMBL': ['A0A1B0GVD4', 'Q6B4S4', 'A0A590UK68', 'A0A1W2PQI5', 'A0A1B0GVM8', 'A0A590UK43', 'A0A590UJP2', 'F8W0Q0']}\n",
      "\n",
      "Gene: FBXO40\n",
      "Symbol: FBXO40\n",
      "Name: F-box protein 40\n",
      "Summary: Members of the F-box protein family, such as FBXO40, are characterized by an approximately 40-amino acid F-box motif. SCF complexes, formed by SKP1 (MIM 601434), cullin (see CUL1; MIM 603134), and F-box proteins, act as protein-ubiquitin ligases. F-box proteins interact with SKP1 through the F box, and they interact with ubiquitination targets through other protein interaction domains (Jin et al., 2004 [PubMed 15520277]).[supplied by OMIM, Mar 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - muscle cell differentiation\n",
      "  CC:\n",
      "    - cytoplasm\n",
      "    - cytoplasm\n",
      "    - cytosol\n",
      "  MF:\n",
      "    - zinc ion binding\n",
      "    - ubiquitin protein ligase activity\n",
      "UniProt: {'Swiss-Prot': 'Q9UH90'}\n",
      "\n",
      "Gene: ALDH1B1\n",
      "Symbol: ALDH1B1\n",
      "Name: aldehyde dehydrogenase 1 family member B1\n",
      "Summary: This protein belongs to the aldehyde dehydrogenases family of proteins. Aldehyde dehydrogenase is the second enzyme of the major oxidative pathway of alcohol metabolism. This gene does not contain introns in the coding sequence. The variation of this locus may affect the development of alcohol-related problems. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - carbohydrate metabolic process\n",
      "    - ethanol catabolic process\n",
      "  CC:\n",
      "    - nucleoplasm\n",
      "    - mitochondrion\n",
      "    - mitochondrion\n",
      "    - mitochondrial matrix\n",
      "    - cytosol\n",
      "  MF:\n",
      "    - aldehyde dehydrogenase (NAD+) activity\n",
      "    - NAD binding\n",
      "UniProt: {'Swiss-Prot': 'P30837', 'TrEMBL': ['A0A0U1RQK9', 'A0A384MTJ7']}\n",
      "\n",
      "Gene: MFSD2A\n",
      "Symbol: MFSD2A\n",
      "Name: MFSD2 lysolipid transporter A, lysophospholipid\n",
      "Summary: The protein encoded by this gene is a transmembrane protein and sodium-dependent lysophosphatidylcholine transporter. The encoded protein is involved in the establishment of the blood-brain barrier and is required for brain growth and function. Defects in this gene are a cause of a progressive microcephaly syndrome. [provided by RefSeq, Mar 2017].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - retinal pigment epithelium development\n",
      "    - phosphatidylcholine biosynthetic process\n",
      "    - brain development\n",
      "    - brain development\n",
      "    - photoreceptor cell morphogenesis\n",
      "    - carbohydrate transport\n",
      "    - cellular response to starvation\n",
      "    - positive regulation of triglyceride biosynthetic process\n",
      "    - fatty acid transport\n",
      "    - fatty acid transport\n",
      "    - fatty acid transport\n",
      "    - long-chain fatty acid transport\n",
      "    - hippocampus development\n",
      "    - positive regulation of cell growth\n",
      "    - negative regulation of fatty acid beta-oxidation\n",
      "    - very-low-density lipoprotein particle assembly\n",
      "    - maintenance of blood-brain barrier\n",
      "    - maintenance of blood-brain barrier\n",
      "    - photoreceptor cell outer segment organization\n",
      "    - regulation of multicellular organism growth\n",
      "    - transcytosis\n",
      "    - regulation of dendrite development\n",
      "    - cognition\n",
      "    - lysophospholipid transport\n",
      "    - lysophospholipid transport\n",
      "    - lysophospholipid transport\n",
      "    - transmembrane transport\n",
      "    - retina morphogenesis in camera-type eye\n",
      "    - establishment of blood-brain barrier\n",
      "    - establishment of blood-brain barrier\n",
      "    - motor behavior\n",
      "    - energy homeostasis\n",
      "    - lysophospholipid translocation\n",
      "    - lysophospholipid translocation\n",
      "    - regulation of neuron projection arborization\n",
      "    - transport across blood-brain barrier\n",
      "    - regulation of phosphatidylcholine metabolic process\n",
      "    - regulation of phosphatidylcholine metabolic process\n",
      "    - regulation of phosphatidylethanolamine metabolic process\n",
      "    - regulation of phosphatidylserine metabolic process\n",
      "    - lipid transport across blood-brain barrier\n",
      "    - lipid transport across blood-brain barrier\n",
      "    - lipid transport across blood-brain barrier\n",
      "    - embryonic brain development\n",
      "    - establishment of blood-retinal barrier\n",
      "  CC:\n",
      "    - endoplasmic reticulum membrane\n",
      "    - cytosol\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - membrane\n",
      "    - cytoplasmic ribonucleoprotein granule\n",
      "  MF:\n",
      "    - long-chain fatty acid transmembrane transporter activity\n",
      "    - long-chain fatty acid transmembrane transporter activity\n",
      "    - protein binding\n",
      "    - phospholipid transporter activity\n",
      "    - fatty acid transmembrane transporter activity\n",
      "    - fatty acid transmembrane transporter activity\n",
      "    - lysophospholipid:sodium symporter activity\n",
      "    - lysophospholipid:sodium symporter activity\n",
      "    - lysophosphatidylcholine flippase activity\n",
      "    - oleate transmembrane transporter activity\n",
      "UniProt: {'Swiss-Prot': 'Q8NA29', 'TrEMBL': ['E7EPI8', 'Q71RE4', 'B4DNN7', 'Q5SSK0']}\n",
      "\n",
      "Gene: IL10\n",
      "Symbol: IL10\n",
      "Name: interleukin 10\n",
      "Summary: The protein encoded by this gene is a cytokine produced primarily by monocytes and to a lesser extent by lymphocytes. This cytokine has pleiotropic effects in immunoregulation and inflammation. It down-regulates the expression of Th1 cytokines, MHC class II Ags, and costimulatory molecules on macrophages. It also enhances B cell survival, proliferation, and antibody production. This cytokine can block NF-kappa B activity, and is involved in the regulation of the JAK-STAT signaling pathway. Knockout studies in mice suggested the function of this cytokine as an essential immunoregulator in the intestinal tract. Mutations in this gene are associated with an increased susceptibility to HIV-1 infection and rheumatoid arthritis. [provided by RefSeq, May 2020].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - negative regulation of cytokine production\n",
      "    - positive regulation of cytokine production\n",
      "    - positive regulation of endothelial cell proliferation\n",
      "    - response to molecule of bacterial origin\n",
      "    - chronic inflammatory response to antigenic stimulus\n",
      "    - positive regulation of immunoglobulin production\n",
      "    - negative regulation of cytokine production involved in immune response\n",
      "    - negative regulation of chronic inflammatory response to antigenic stimulus\n",
      "    - positive regulation of B cell apoptotic process\n",
      "    - immune response\n",
      "    - negative regulation of cell population proliferation\n",
      "    - response to xenobiotic stimulus\n",
      "    - regulation of gene expression\n",
      "    - negative regulation of autophagy\n",
      "    - response to activity\n",
      "    - response to inactivity\n",
      "    - hemopoiesis\n",
      "    - B cell differentiation\n",
      "    - leukocyte chemotaxis\n",
      "    - negative regulation of myeloid dendritic cell activation\n",
      "    - negative regulation of B cell proliferation\n",
      "    - negative regulation of interferon-alpha production\n",
      "    - negative regulation of type II interferon production\n",
      "    - negative regulation of interleukin-1 production\n",
      "    - negative regulation of interleukin-12 production\n",
      "    - negative regulation of interleukin-18 production\n",
      "    - negative regulation of interleukin-6 production\n",
      "    - negative regulation of interleukin-6 production\n",
      "    - negative regulation of interleukin-8 production\n",
      "    - negative regulation of tumor necrosis factor production\n",
      "    - response to insulin\n",
      "    - negative regulation of heterotypic cell-cell adhesion\n",
      "    - positive regulation of heterotypic cell-cell adhesion\n",
      "    - response to carbon monoxide\n",
      "    - cellular response to hepatocyte growth factor stimulus\n",
      "    - type 2 immune response\n",
      "    - B cell proliferation\n",
      "    - negative regulation of T cell proliferation\n",
      "    - defense response to bacterium\n",
      "    - defense response to protozoan\n",
      "    - positive regulation of macrophage activation\n",
      "    - negative regulation of apoptotic process\n",
      "    - negative regulation of nitric oxide biosynthetic process\n",
      "    - regulation of isotype switching\n",
      "    - negative regulation of MHC class II biosynthetic process\n",
      "    - negative regulation of MHC class II biosynthetic process\n",
      "    - positive regulation of MHC class II biosynthetic process\n",
      "    - positive regulation of cell cycle\n",
      "    - positive regulation of DNA-templated transcription\n",
      "    - positive regulation of DNA-templated transcription\n",
      "    - negative regulation of mitotic cell cycle\n",
      "    - positive regulation of transcription by RNA polymerase II\n",
      "    - positive regulation of receptor signaling pathway via JAK-STAT\n",
      "    - positive regulation of receptor signaling pathway via JAK-STAT\n",
      "    - negative regulation of inflammatory response\n",
      "    - negative regulation of inflammatory response\n",
      "    - negative regulation of inflammatory response\n",
      "    - regulation of synapse organization\n",
      "    - negative regulation of membrane protein ectodomain proteolysis\n",
      "    - positive regulation of DNA-binding transcription factor activity\n",
      "    - response to glucocorticoid\n",
      "    - negative regulation of cytokine activity\n",
      "    - branching involved in labyrinthine layer morphogenesis\n",
      "    - cellular response to lipopolysaccharide\n",
      "    - cellular response to lipopolysaccharide\n",
      "    - cellular response to estradiol stimulus\n",
      "    - negative regulation of chemokine (C-C motif) ligand 5 production\n",
      "    - endothelial cell apoptotic process\n",
      "    - liver regeneration\n",
      "    - interleukin-10-mediated signaling pathway\n",
      "    - interleukin-10-mediated signaling pathway\n",
      "    - positive regulation of plasma cell differentiation\n",
      "    - positive regulation of miRNA transcription\n",
      "    - regulation of response to wounding\n",
      "    - negative regulation of oxidative stress-induced neuron intrinsic apoptotic signaling pathway\n",
      "    - regulation of complement-dependent cytotoxicity\n",
      "    - positive regulation of sprouting angiogenesis\n",
      "    - negative regulation of vascular associated smooth muscle cell proliferation\n",
      "    - positive regulation of vascular associated smooth muscle cell proliferation\n",
      "    - positive regulation of signaling receptor activity\n",
      "    - positive regulation of signaling receptor activity\n",
      "    - negative regulation of endothelial cell apoptotic process\n",
      "  CC:\n",
      "    - extracellular region\n",
      "    - extracellular space\n",
      "    - extracellular space\n",
      "    - extracellular space\n",
      "  MF:\n",
      "    - cytokine activity\n",
      "    - cytokine activity\n",
      "    - cytokine activity\n",
      "    - interleukin-10 receptor binding\n",
      "    - protein binding\n",
      "    - growth factor activity\n",
      "    - protein dimerization activity\n",
      "UniProt: {'Swiss-Prot': 'P22301', 'TrEMBL': ['A0A590UK12', 'Q6FGW4', 'A0A286YEX3']}\n",
      "\n",
      "Gene: MT1F\n",
      "Symbol: MT1F\n",
      "Name: metallothionein 1F\n",
      "Summary: Predicted to enable zinc ion binding activity. Involved in cellular response to cadmium ion. Located in cytoplasm and nucleus. [provided by Alliance of Genome Resources, Apr 2022]\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - intracellular zinc ion homeostasis\n",
      "    - detoxification of copper ion\n",
      "    - negative regulation of growth\n",
      "    - cellular response to cadmium ion\n",
      "    - cellular response to cadmium ion\n",
      "    - cellular response to copper ion\n",
      "    - cellular response to zinc ion\n",
      "    - cellular response to zinc ion\n",
      "  CC:\n",
      "    - nucleus\n",
      "    - nucleus\n",
      "    - cytoplasm\n",
      "    - cytoplasm\n",
      "  MF:\n",
      "    - protein binding\n",
      "    - zinc ion binding\n",
      "    - metal ion binding\n",
      "UniProt: {'Swiss-Prot': 'P04733', 'TrEMBL': 'H3BRY8'}\n",
      "\n",
      "Gene: CNR1\n",
      "Symbol: CNR1\n",
      "Name: cannabinoid receptor 1\n",
      "Summary: This gene encodes one of two cannabinoid receptors. The cannabinoids, principally delta-9-tetrahydrocannabinol and synthetic analogs, are psychoactive ingredients of marijuana. The cannabinoid receptors are members of the guanine-nucleotide-binding protein (G-protein) coupled receptor family, which inhibit adenylate cyclase activity in a dose-dependent, stereoselective and pertussis toxin-sensitive manner. The two receptors have been found to be involved in the cannabinoid-induced CNS effects (including alterations in mood and cognition) experienced by users of marijuana. Multiple transcript variants encoding two different protein isoforms have been described for this gene. [provided by RefSeq, May 2009].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - positive regulation of acute inflammatory response to antigenic stimulus\n",
      "    - G protein-coupled receptor signaling pathway, coupled to cyclic nucleotide second messenger\n",
      "    - adenylate cyclase-modulating G protein-coupled receptor signaling pathway\n",
      "    - adenylate cyclase-activating G protein-coupled receptor signaling pathway\n",
      "    - spermatogenesis\n",
      "    - axonal fasciculation\n",
      "    - response to nutrient\n",
      "    - memory\n",
      "    - positive regulation of neuron projection development\n",
      "    - negative regulation of serotonin secretion\n",
      "    - regulation of metabolic process\n",
      "    - positive regulation of fever generation\n",
      "    - negative regulation of fatty acid beta-oxidation\n",
      "    - regulation of synaptic transmission, GABAergic\n",
      "    - response to lipopolysaccharide\n",
      "    - negative regulation of mast cell activation\n",
      "    - negative regulation of dopamine secretion\n",
      "    - response to nicotine\n",
      "    - cannabinoid signaling pathway\n",
      "    - response to cocaine\n",
      "    - glucose homeostasis\n",
      "    - positive regulation of apoptotic process\n",
      "    - response to ethanol\n",
      "    - negative regulation of action potential\n",
      "    - negative regulation of blood pressure\n",
      "    - positive regulation of blood pressure\n",
      "    - regulation of insulin secretion\n",
      "    - regulation of synaptic transmission, glutamatergic\n",
      "    - maternal process involved in female pregnancy\n",
      "    - regulation of feeding behavior\n",
      "    - regulation of penile erection\n",
      "    - retrograde trans-synaptic signaling by endocannabinoid\n",
      "    - regulation of presynaptic cytosolic calcium ion concentration\n",
      "  CC:\n",
      "    - cytoplasm\n",
      "    - mitochondrial outer membrane\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - actin cytoskeleton\n",
      "    - growth cone\n",
      "    - presynaptic membrane\n",
      "    - membrane raft\n",
      "    - glutamatergic synapse\n",
      "    - GABA-ergic synapse\n",
      "  MF:\n",
      "    - G protein-coupled receptor activity\n",
      "    - cannabinoid receptor activity\n",
      "    - protein binding\n",
      "    - identical protein binding\n",
      "UniProt: {'Swiss-Prot': 'P21554', 'TrEMBL': ['V5KA96', 'S5TLS4', 'F8W908']}\n",
      "\n",
      "Gene: SCN2A\n",
      "Symbol: SCN2A\n",
      "Name: sodium voltage-gated channel alpha subunit 2\n",
      "Summary: Voltage-gated sodium channels are transmembrane glycoprotein complexes composed of a large alpha subunit with four repeat domains, each of which is composed of six membrane-spanning segments, and one or more regulatory beta subunits. Voltage-gated sodium channels function in the generation and propagation of action potentials in neurons and muscle. This gene encodes one member of the sodium channel alpha subunit gene family. Allelic variants of this gene are associated with seizure disorders and autism spectrum disorder. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Nov 2016].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - sodium ion transport\n",
      "    - nervous system development\n",
      "    - memory\n",
      "    - determination of adult lifespan\n",
      "    - intrinsic apoptotic signaling pathway in response to osmotic stress\n",
      "    - neuronal action potential\n",
      "    - neuronal action potential\n",
      "    - dentate gyrus development\n",
      "    - nerve development\n",
      "    - sodium ion transmembrane transport\n",
      "    - myelination\n",
      "    - neuron apoptotic process\n",
      "    - cellular response to hypoxia\n",
      "    - cardiac muscle cell action potential involved in contraction\n",
      "  CC:\n",
      "    - voltage-gated sodium channel complex\n",
      "    - voltage-gated sodium channel complex\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - intercalated disc\n",
      "    - membrane\n",
      "    - T-tubule\n",
      "    - axon\n",
      "    - node of Ranvier\n",
      "    - paranode region of axon\n",
      "    - presynaptic membrane\n",
      "    - glutamatergic synapse\n",
      "  MF:\n",
      "    - voltage-gated sodium channel activity\n",
      "    - voltage-gated sodium channel activity\n",
      "    - voltage-gated sodium channel activity\n",
      "    - voltage-gated sodium channel activity\n",
      "    - protein binding\n",
      "    - calmodulin binding\n",
      "UniProt: {'Swiss-Prot': 'Q99250', 'TrEMBL': ['A0A1B0GW67', 'A0A1B0GTX0', 'A0A669KBL5', 'F6U291', 'A0A1B0GTS6', 'A0A0D9SGG2', 'A0A1B0GW40', 'A0A669KAR0', 'A0A669KBM3', 'A0A1B0GWA6']}\n",
      "\n",
      "Gene: SRD5A3\n",
      "Symbol: SRD5A3\n",
      "Name: steroid 5 alpha-reductase 3\n",
      "Summary: The protein encoded by this gene belongs to the steroid 5-alpha reductase family, and polyprenol reductase subfamily. It is involved in the production of androgen 5-alpha-dihydrotestosterone (DHT) from testosterone, and maintenance of the androgen-androgen receptor activation pathway. This protein is also necessary for the conversion of polyprenol into dolichol, which is required for the synthesis of dolichol-linked monosaccharides and the oligosaccharide precursor used for N-linked glycosylation of proteins. Mutations in this gene are associated with congenital disorder of glycosylation type Iq. [provided by RefSeq, Mar 2011].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - dolichol-linked oligosaccharide biosynthetic process\n",
      "    - dolichol-linked oligosaccharide biosynthetic process\n",
      "    - dolichyl diphosphate biosynthetic process\n",
      "    - androgen biosynthetic process\n",
      "    - polyprenol catabolic process\n",
      "    - dolichol metabolic process\n",
      "    - dolichol biosynthetic process\n",
      "    - dolichol biosynthetic process\n",
      "  CC:\n",
      "    - endoplasmic reticulum\n",
      "    - endoplasmic reticulum\n",
      "    - endoplasmic reticulum membrane\n",
      "    - endoplasmic reticulum membrane\n",
      "  MF:\n",
      "    - 3-oxo-5-alpha-steroid 4-dehydrogenase activity\n",
      "    - oxidoreductase activity, acting on the CH-CH group of donors, NAD or NADP as acceptor\n",
      "    - oxidoreductase activity, acting on the CH-CH group of donors, NAD or NADP as acceptor\n",
      "    - 3-oxo-5-alpha-steroid 4-dehydrogenase (NADP+) activity\n",
      "    - polyprenol reductase activity\n",
      "    - polyprenal reductase activity\n",
      "UniProt: {'Swiss-Prot': 'Q9H8P0', 'TrEMBL': ['H0Y9P9', 'A0A7I2V5I5', 'A0A7P0TBH6']}\n",
      "\n",
      "Gene: DNAH12\n",
      "Symbol: DNAH12\n",
      "Name: dynein axonemal heavy chain 12\n",
      "Summary: Predicted to enable several functions, including ATP binding activity; dynein intermediate chain binding activity; and dynein light intermediate chain binding activity. Predicted to be involved in microtubule-based movement. Predicted to be located in cilium; cytoplasm; and microtubule. Predicted to be part of dynein complex. [provided by Alliance of Genome Resources, Apr 2022]\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - microtubule-based movement\n",
      "  CC:\n",
      "    - axonemal dynein complex\n",
      "    - microtubule\n",
      "  MF:\n",
      "    - ATP binding\n",
      "    - minus-end-directed microtubule motor activity\n",
      "    - ATP hydrolysis activity\n",
      "    - dynein intermediate chain binding\n",
      "    - dynein light intermediate chain binding\n",
      "UniProt: {'Swiss-Prot': 'Q6ZR08', 'TrEMBL': ['E9PG32', 'H7C5N3', 'H7C5S4']}\n",
      "\n",
      "Gene: COX15\n",
      "Symbol: COX15\n",
      "Name: cytochrome c oxidase assembly homolog COX15\n",
      "Summary: Cytochrome c oxidase (COX), the terminal component of the mitochondrial respiratory chain, catalyzes the electron transfer from reduced cytochrome c to oxygen. This component is a heteromeric complex consisting of 3 catalytic subunits encoded by mitochondrial genes and multiple structural subunits encoded by nuclear genes. The mitochondrially-encoded subunits function in electron transfer, and the nuclear-encoded subunits may function in the regulation and assembly of the complex. This nuclear gene encodes a protein which is not a structural subunit, but may be essential for the biogenesis of COX formation and may function in the hydroxylation of heme O, according to the yeast mutant studies. This protein is predicted to contain 5 transmembrane domains localized in the mitochondrial inner membrane. Alternative splicing of this gene generates two transcript variants diverging in the 3' region. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - heme biosynthetic process\n",
      "    - heme A biosynthetic process\n",
      "    - heme A biosynthetic process\n",
      "    - heme A biosynthetic process\n",
      "    - cytochrome complex assembly\n",
      "  CC:\n",
      "    - nucleoplasm\n",
      "    - mitochondrion\n",
      "    - mitochondrion\n",
      "    - mitochondrial inner membrane\n",
      "    - mitochondrial inner membrane\n",
      "    - cytochrome complex\n",
      "    - respiratory chain complex\n",
      "  MF:\n",
      "    - protein binding\n",
      "    - oxidoreductase activity, acting on the CH-CH group of donors\n",
      "    - oxidoreductase activity, acting on the CH-CH group of donors\n",
      "    - oxidoreductase activity, acting on NAD(P)H, heme protein as acceptor\n",
      "    - heme binding\n",
      "    - metal ion binding\n",
      "UniProt: {'Swiss-Prot': 'Q7KZN9', 'TrEMBL': 'B4DQM2'}\n",
      "\n",
      "Gene: GRM8\n",
      "Symbol: GRM8\n",
      "Name: glutamate metabotropic receptor 8\n",
      "Summary: L-glutamate is the major excitatory neurotransmitter in the central nervous system and activates both ionotropic and metabotropic glutamate receptors. Glutamatergic neurotransmission is involved in most aspects of normal brain function and can be perturbed in many neuropathologic conditions. The metabotropic glutamate receptors are a family of G protein-coupled receptors, that have been divided into 3 groups on the basis of sequence homology, putative signal transduction mechanisms, and pharmacologic properties. Group I includes GRM1 and GRM5 and these receptors have been shown to activate phospholipase C. Group II includes GRM2 and GRM3 while Group III includes GRM4, GRM6, GRM7 and GRM8. Group II and III receptors are linked to the inhibition of the cyclic AMP cascade but differ in their agonist selectivities. Alternatively spliced transcript variants encoding different isoforms have been described for this gene. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - adenylate cyclase-inhibiting G protein-coupled receptor signaling pathway\n",
      "    - adenylate cyclase-inhibiting G protein-coupled glutamate receptor signaling pathway\n",
      "    - G protein-coupled glutamate receptor signaling pathway\n",
      "    - visual perception\n",
      "    - regulation of synaptic transmission, glutamatergic\n",
      "  CC:\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "  MF:\n",
      "    - group III metabotropic glutamate receptor activity\n",
      "    - G protein-coupled receptor activity\n",
      "    - glutamate receptor activity\n",
      "UniProt: {'Swiss-Prot': 'O00222', 'TrEMBL': ['A0A9L9PYG5', 'E7ETK3', 'H7C4V5', 'C9J7I1']}\n",
      "\n",
      "Gene: SLC22A14\n",
      "Symbol: SLC22A14\n",
      "Name: solute carrier family 22 member 14\n",
      "Summary: This gene encodes a member of the organic-cation transporter family. It is located in a gene cluster with another member of the family, organic cation transporter like 3. The encoded protein is a transmembrane protein which is thought to transport small molecules and since this protein is conserved among several species, it is suggested to have a fundamental role in mammalian systems. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Feb 2016].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - flagellated sperm motility\n",
      "    - riboflavin transport\n",
      "    - sperm capacitation\n",
      "    - transmembrane transport\n",
      "  CC:\n",
      "    - mitochondrial inner membrane\n",
      "    - plasma membrane\n",
      "    - sperm principal piece\n",
      "  MF:\n",
      "    - protein binding\n",
      "    - riboflavin transmembrane transporter activity\n",
      "UniProt: {'Swiss-Prot': 'Q9Y267', 'TrEMBL': 'F5H7H1'}\n",
      "\n",
      "Gene: POLA1\n",
      "Symbol: POLA1\n",
      "Name: DNA polymerase alpha 1, catalytic subunit\n",
      "Summary: This gene encodes the catalytic subunit of DNA polymerase, which together with a regulatory and two primase subunits, forms the DNA polymerase alpha complex. The catalytic subunit plays an essential role in the initiation of DNA replication. [provided by RefSeq, Mar 2010].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - DNA synthesis involved in DNA repair\n",
      "    - DNA replication\n",
      "    - DNA replication, synthesis of primer\n",
      "    - DNA replication initiation\n",
      "    - DNA strand elongation involved in DNA replication\n",
      "    - leading strand elongation\n",
      "    - leading strand elongation\n",
      "    - lagging strand elongation\n",
      "    - lagging strand elongation\n",
      "    - DNA repair\n",
      "    - DNA repair\n",
      "    - nucleotide-excision repair\n",
      "    - double-strand break repair via nonhomologous end joining\n",
      "    - regulation of type I interferon production\n",
      "    - mitotic DNA replication initiation\n",
      "    - DNA synthesis involved in UV-damage excision repair\n",
      "  CC:\n",
      "    - chromatin\n",
      "    - nucleus\n",
      "    - nuclear envelope\n",
      "    - nucleoplasm\n",
      "    - nucleoplasm\n",
      "    - alpha DNA polymerase:primase complex\n",
      "    - alpha DNA polymerase:primase complex\n",
      "    - alpha DNA polymerase:primase complex\n",
      "    - alpha DNA polymerase:primase complex\n",
      "    - nucleolus\n",
      "    - cytosol\n",
      "    - nuclear matrix\n",
      "  MF:\n",
      "    - nucleotide binding\n",
      "    - DNA binding\n",
      "    - chromatin binding\n",
      "    - chromatin binding\n",
      "    - DNA replication origin binding\n",
      "    - single-stranded DNA binding\n",
      "    - DNA-directed DNA polymerase activity\n",
      "    - DNA-directed DNA polymerase activity\n",
      "    - DNA-directed DNA polymerase activity\n",
      "    - protein binding\n",
      "    - zinc ion binding\n",
      "    - protein kinase binding\n",
      "UniProt: {'Swiss-Prot': 'P09884', 'TrEMBL': ['A0A7I2V2V3', 'A6NMQ1', 'A0A5F9ZHW2']}\n",
      "\n",
      "Gene: GSTM2\n",
      "Symbol: GSTM2\n",
      "Name: glutathione S-transferase mu 2\n",
      "Summary: Cytosolic and membrane-bound forms of glutathione S-transferase are encoded by two distinct supergene families. At present, eight distinct classes of the soluble cytoplasmic mammalian glutathione S-transferases have been identified: alpha, kappa, mu, omega, pi, sigma, theta and zeta. This gene encodes a glutathione S-transferase that belongs to the mu class. The mu class of enzymes functions in the detoxification of electrophilic compounds, including carcinogens, therapeutic drugs, environmental toxins and products of oxidative stress, by conjugation with glutathione. The genes encoding the mu class of enzymes are organized in a gene cluster on chromosome 1p13.3 and are known to be highly polymorphic. These genetic variations can change an individual's susceptibility to carcinogens and toxins as well as affect the toxicity and efficacy of certain drugs. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - glutathione metabolic process\n",
      "    - glutathione metabolic process\n",
      "    - regulation of release of sequestered calcium ion into cytosol by sarcoplasmic reticulum\n",
      "    - regulation of cardiac muscle contraction by regulation of the release of sequestered calcium ion\n",
      "    - regulation of skeletal muscle contraction by regulation of release of sequestered calcium ion\n",
      "    - nitrobenzene metabolic process\n",
      "    - xenobiotic catabolic process\n",
      "    - linoleic acid metabolic process\n",
      "    - hepoxilin biosynthetic process\n",
      "    - relaxation of cardiac muscle\n",
      "    - negative regulation of ryanodine-sensitive calcium-release channel activity\n",
      "    - positive regulation of ryanodine-sensitive calcium-release channel activity\n",
      "    - cellular detoxification of nitrogen compound\n",
      "    - cellular response to caffeine\n",
      "    - cellular oxidant detoxification\n",
      "  CC:\n",
      "    - cytoplasm\n",
      "    - cytosol\n",
      "    - cytosol\n",
      "    - sarcoplasmic reticulum\n",
      "    - intercellular bridge\n",
      "    - extracellular exosome\n",
      "  MF:\n",
      "    - glutathione transferase activity\n",
      "    - glutathione transferase activity\n",
      "    - glutathione peroxidase activity\n",
      "    - signaling receptor binding\n",
      "    - fatty acid binding\n",
      "    - protein binding\n",
      "    - enzyme binding\n",
      "    - protein homodimerization activity\n",
      "    - glutathione binding\n",
      "UniProt: {'Swiss-Prot': 'P28161', 'TrEMBL': ['Q0D2I8', 'A0A384P5E9', 'E9PLF1', 'F6XZQ7', 'E9PGV1', 'E9PHN7', 'E9PHN6']}\n",
      "\n",
      "Gene: KLHL25\n",
      "Symbol: KLHL25\n",
      "Name: kelch like family member 25\n",
      "Summary: Involved in protein ubiquitination; regulation of translational initiation; and ubiquitin-dependent protein catabolic process. Located in cytoplasm. Part of Cul3-RING ubiquitin ligase complex. [provided by Alliance of Genome Resources, Apr 2022]\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - regulation of translational initiation\n",
      "    - ubiquitin-dependent protein catabolic process\n",
      "    - protein ubiquitination\n",
      "    - protein ubiquitination\n",
      "    - positive regulation of CD4-positive, CD25-positive, alpha-beta regulatory T cell differentiation\n",
      "    - negative regulation of fatty acid biosynthetic process\n",
      "    - positive regulation of fatty acid oxidation\n",
      "    - positive regulation of fatty acid oxidation\n",
      "  CC:\n",
      "    - cytoplasm\n",
      "    - cytosol\n",
      "    - Cul3-RING ubiquitin ligase complex\n",
      "  MF:\n",
      "    - ubiquitin-like ligase-substrate adaptor activity\n",
      "UniProt: {'Swiss-Prot': 'Q9H0H3'}\n",
      "\n",
      "Gene: PDE7B\n",
      "Symbol: PDE7B\n",
      "Name: phosphodiesterase 7B\n",
      "Summary: The 3',5'-cyclic nucleotides cAMP and cGMP function as second messengers in a wide variety of signal transduction pathways. 3',5'-cyclic nucleotide phosphodiesterases (PDEs) catalyze the hydrolysis of cAMP and cGMP to the corresponding 5'-monophosphates and provide a mechanism to downregulate cAMP and cGMP signaling. This gene encodes a cAMP-specific phosphodiesterase, a member of the cyclic nucleotide phosphodiesterase family.[provided by RefSeq, Apr 2009].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - cAMP catabolic process\n",
      "    - signal transduction\n",
      "    - chemical synaptic transmission\n",
      "    - cAMP-mediated signaling\n",
      "  CC:\n",
      "    - cytosol\n",
      "    - synapse\n",
      "  MF:\n",
      "    - 3',5'-cyclic-AMP phosphodiesterase activity\n",
      "    - 3',5'-cyclic-AMP phosphodiesterase activity\n",
      "    - metal ion binding\n",
      "    - 3',5'-cyclic-GMP phosphodiesterase activity\n",
      "UniProt: {'Swiss-Prot': 'Q9NP56', 'TrEMBL': ['H0Y689', 'A1E5M1']}\n",
      "\n",
      "Gene: SIGMAR1\n",
      "Symbol: SIGMAR1\n",
      "Name: sigma non-opioid intracellular receptor 1\n",
      "Summary: This gene encodes a receptor protein that interacts with a variety of psychotomimetic drugs, including cocaine and amphetamines. The receptor is believed to play an important role in the cellular functions of various tissues associated with the endocrine, immune, and nervous systems. As indicated by its previous name, opioid receptor sigma 1 (OPRS1), the product of this gene was erroneously thought to function as an opioid receptor; it is now thought to be a non-opioid receptor. Mutations in this gene has been associated with juvenile amyotrophic lateral sclerosis 16. Alternative splicing of this gene results in transcript variants encoding distinct isoforms. [provided by RefSeq, Aug 2013].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - lipid transport\n",
      "    - nervous system development\n",
      "    - G protein-coupled opioid receptor signaling pathway\n",
      "    - regulation of neuron apoptotic process\n",
      "    - protein homotrimerization\n",
      "    - regulation of postsynapse assembly\n",
      "  CC:\n",
      "    - nuclear envelope\n",
      "    - nuclear inner membrane\n",
      "    - nuclear outer membrane\n",
      "    - endoplasmic reticulum\n",
      "    - endoplasmic reticulum\n",
      "    - endoplasmic reticulum membrane\n",
      "    - lipid droplet\n",
      "    - cytosol\n",
      "    - postsynaptic density\n",
      "    - membrane\n",
      "    - growth cone\n",
      "    - cytoplasmic vesicle\n",
      "    - anchoring junction\n",
      "    - postsynaptic density membrane\n",
      "    - glutamatergic synapse\n",
      "  MF:\n",
      "    - G protein-coupled opioid receptor activity\n",
      "    - protein binding\n",
      "    - identical protein binding\n",
      "UniProt: {'Swiss-Prot': 'Q99720', 'TrEMBL': ['A2A3U5', 'B4DR71', 'A0A7P0Z4C2', 'A0A7P0T9D5', 'Q5T1J1', 'A0A7P0TA12']}\n",
      "\n",
      "Gene: ADCY5\n",
      "Symbol: ADCY5\n",
      "Name: adenylate cyclase 5\n",
      "Summary: This gene encodes a member of the membrane-bound adenylyl cyclase enzymes. Adenylyl cyclases mediate G protein-coupled receptor signaling through the synthesis of the second messenger cAMP. Activity of the encoded protein is stimulated by the Gs alpha subunit of G protein-coupled receptors and is inhibited by protein kinase A, calcium and Gi alpha subunits. Single nucleotide polymorphisms in this gene may be associated with low birth weight and type 2 diabetes. Alternatively spliced transcript variants that encode different isoforms have been observed for this gene. [provided by RefSeq, Dec 2010].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - G protein-coupled adenosine receptor signaling pathway\n",
      "    - cAMP biosynthetic process\n",
      "    - cAMP biosynthetic process\n",
      "    - adenylate cyclase-activating G protein-coupled receptor signaling pathway\n",
      "    - adenylate cyclase-activating G protein-coupled receptor signaling pathway\n",
      "    - adenylate cyclase-activating dopamine receptor signaling pathway\n",
      "    - adenylate cyclase-inhibiting dopamine receptor signaling pathway\n",
      "    - positive regulation of cytosolic calcium ion concentration\n",
      "    - locomotory behavior\n",
      "    - intracellular signal transduction\n",
      "    - neuromuscular process controlling balance\n",
      "    - regulation of insulin secretion involved in cellular response to glucose stimulus\n",
      "    - cellular response to forskolin\n",
      "  CC:\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - cilium\n",
      "    - membrane\n",
      "  MF:\n",
      "    - adenylate cyclase activity\n",
      "    - adenylate cyclase activity\n",
      "    - adenylate cyclase activity\n",
      "    - ATP binding\n",
      "    - adenylate cyclase binding\n",
      "    - metal ion binding\n",
      "    - scaffold protein binding\n",
      "UniProt: {'Swiss-Prot': 'O95622', 'TrEMBL': ['A0A384P5Q5', 'B3KWA8', 'B7Z2C7', 'C9JRT8', 'A0A8V8TNW3', 'A0A8V8TP58', 'F8WBM0']}\n",
      "\n",
      "Gene: SLC13A2\n",
      "Symbol: SLC13A2\n",
      "Name: solute carrier family 13 member 2\n",
      "Summary: The protein encoded by this gene is a sodium-coupled citrate transporter that is regulated by the chaperone activity of cyclophilin b. The encoded protein may play a role in the formation of kidney stones. [provided by RefSeq, Oct 2016].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - fumarate transport\n",
      "    - fumarate transport\n",
      "    - alpha-ketoglutarate transport\n",
      "    - alpha-ketoglutarate transport\n",
      "    - sodium ion transmembrane transport\n",
      "    - cellular response to lithium ion\n",
      "    - cellular response to lithium ion\n",
      "    - succinate transmembrane transport\n",
      "    - succinate transmembrane transport\n",
      "  CC:\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - membrane\n",
      "    - apical plasma membrane\n",
      "    - extracellular exosome\n",
      "  MF:\n",
      "    - protein binding\n",
      "    - fumarate transmembrane transporter activity\n",
      "    - fumarate transmembrane transporter activity\n",
      "    - alpha-ketoglutarate transmembrane transporter activity\n",
      "    - alpha-ketoglutarate transmembrane transporter activity\n",
      "    - succinate transmembrane transporter activity\n",
      "    - succinate transmembrane transporter activity\n",
      "    - low-affinity sodium:dicarboxylate symporter activity\n",
      "    - sodium:dicarboxylate symporter activity\n",
      "    - sodium:dicarboxylate symporter activity\n",
      "UniProt: {'Swiss-Prot': 'Q13183', 'TrEMBL': ['J3QR70', 'J3QL78']}\n",
      "\n",
      "Gene: MYH6\n",
      "Symbol: MYH6\n",
      "Name: myosin heavy chain 6\n",
      "Summary: Cardiac muscle myosin is a hexamer consisting of two heavy chain subunits, two light chain subunits, and two regulatory subunits. This gene encodes the alpha heavy chain subunit of cardiac myosin. The gene is located approximately 4kb downstream of the gene encoding the beta heavy chain subunit of cardiac myosin. Mutations in this gene cause familial hypertrophic cardiomyopathy and atrial septal defect 3. [provided by RefSeq, Feb 2017].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - in utero embryonic development\n",
      "    - regulation of the force of heart contraction\n",
      "    - regulation of the force of heart contraction\n",
      "    - regulation of heart rate\n",
      "    - muscle contraction\n",
      "    - striated muscle contraction\n",
      "    - striated muscle contraction\n",
      "    - adult heart development\n",
      "    - visceral muscle development\n",
      "    - regulation of heart contraction\n",
      "    - regulation of blood pressure\n",
      "    - cardiac muscle hypertrophy in response to stress\n",
      "    - muscle filament sliding\n",
      "    - muscle filament sliding\n",
      "    - myofibril assembly\n",
      "    - sarcomere organization\n",
      "    - ATP metabolic process\n",
      "    - atrial cardiac muscle tissue morphogenesis\n",
      "    - ventricular cardiac muscle tissue morphogenesis\n",
      "    - cardiac muscle cell development\n",
      "    - cardiac muscle contraction\n",
      "    - regulation of heart growth\n",
      "  CC:\n",
      "    - stress fiber\n",
      "    - cytoplasm\n",
      "    - cytosol\n",
      "    - muscle myosin complex\n",
      "    - myosin complex\n",
      "    - myosin II complex\n",
      "    - myofibril\n",
      "    - sarcomere\n",
      "    - Z disc\n",
      "    - myosin filament\n",
      "  MF:\n",
      "    - microfilament motor activity\n",
      "    - microfilament motor activity\n",
      "    - microfilament motor activity\n",
      "    - microfilament motor activity\n",
      "    - calmodulin binding\n",
      "    - ATP binding\n",
      "    - myosin phosphatase activity\n",
      "    - protein kinase binding\n",
      "    - actin filament binding\n",
      "UniProt: {'Swiss-Prot': 'P13533'}\n",
      "\n",
      "Gene: ABCB8\n",
      "Symbol: ABCB8\n",
      "Name: ATP binding cassette subfamily B member 8\n",
      "Summary: This nuclear gene encodes a multi-pass membrane protein that is targeted to the mitochondrial inner membrane. The encoded protein is an ATP-dependent transporter that may mediate the passage of organic and inorganic molecules out of the mitochondria. Loss of function of the related gene in mouse results in a disruption of iron homeostasis between the mitochondria and cytosol. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Aug 2013].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - cell volume homeostasis\n",
      "    - transmembrane transport\n",
      "    - potassium ion transmembrane transport\n",
      "    - mitochondrial potassium ion transmembrane transport\n",
      "  CC:\n",
      "    - nucleoplasm\n",
      "    - nucleolus\n",
      "    - mitochondrion\n",
      "    - mitochondrion\n",
      "    - mitochondrion\n",
      "    - mitochondrial inner membrane\n",
      "    - membrane\n",
      "    - mitochondrial membrane\n",
      "    - ATP-binding cassette (ABC) transporter complex\n",
      "    - mitochondrial ATP-gated potassium channel complex\n",
      "    - mitochondrial ATP-gated potassium channel complex\n",
      "  MF:\n",
      "    - protein binding\n",
      "    - ATP binding\n",
      "    - ATP hydrolysis activity\n",
      "    - ATPase-coupled transmembrane transporter activity\n",
      "    - ABC-type transporter activity\n",
      "UniProt: {'Swiss-Prot': 'Q9NUT2', 'TrEMBL': ['C9JTY4', 'F2Z2Z1', 'F2Z3G8', 'C9JA03', 'F8WC43', 'H7C5K9', 'C9JYI3']}\n",
      "\n",
      "Gene: ATP1A4\n",
      "Symbol: ATP1A4\n",
      "Name: ATPase Na+/K+ transporting subunit alpha 4\n",
      "Summary: The protein encoded by this gene belongs to the family of P-type cation transport ATPases, and to the subfamily of Na+/K+ -ATPases. Na+/K+ -ATPase is an integral membrane protein responsible for establishing and maintaining the electrochemical gradients of Na and K ions across the plasma membrane. These gradients are essential for osmoregulation, for sodium-coupled transport of a variety of organic and inorganic molecules, and for electrical excitability of nerve and muscle. This enzyme is composed of two subunits, a large catalytic subunit (alpha) and a smaller glycoprotein subunit (beta). The catalytic subunit of Na+/K+ -ATPase is encoded by multiple genes. This gene encodes an alpha 4 subunit. Alternatively spliced transcript variants encoding different isoforms have been identified. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - monoatomic ion transport\n",
      "    - potassium ion transport\n",
      "    - sodium ion transport\n",
      "    - intracellular sodium ion homeostasis\n",
      "    - cell surface receptor signaling pathway\n",
      "    - spermatogenesis\n",
      "    - fertilization\n",
      "    - establishment or maintenance of transmembrane electrochemical gradient\n",
      "    - intracellular potassium ion homeostasis\n",
      "    - flagellated sperm motility\n",
      "    - flagellated sperm motility\n",
      "    - regulation of cellular pH\n",
      "    - sodium ion transmembrane transport\n",
      "    - sodium ion transmembrane transport\n",
      "    - sodium ion export across plasma membrane\n",
      "    - regulation of membrane potential\n",
      "    - establishment of localization in cell\n",
      "    - potassium ion transmembrane transport\n",
      "    - potassium ion transmembrane transport\n",
      "    - transport across blood-brain barrier\n",
      "    - proton transmembrane transport\n",
      "    - proton transmembrane transport\n",
      "    - potassium ion import across plasma membrane\n",
      "  CC:\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - sodium:potassium-exchanging ATPase complex\n",
      "    - sodium:potassium-exchanging ATPase complex\n",
      "    - sodium:potassium-exchanging ATPase complex\n",
      "    - cell projection\n",
      "    - membrane raft\n",
      "    - sperm midpiece\n",
      "    - photoreceptor cell cilium\n",
      "    - rod photoreceptor outer segment\n",
      "  MF:\n",
      "    - P-type sodium:potassium-exchanging transporter activity\n",
      "    - P-type sodium:potassium-exchanging transporter activity\n",
      "    - P-type sodium:potassium-exchanging transporter activity\n",
      "    - P-type sodium:potassium-exchanging transporter activity\n",
      "    - protein binding\n",
      "    - ATP binding\n",
      "    - ATP hydrolysis activity\n",
      "    - ATPase-coupled monoatomic cation transmembrane transporter activity\n",
      "    - kinase binding\n",
      "    - metal ion binding\n",
      "UniProt: {'Swiss-Prot': 'Q13733', 'TrEMBL': ['A0A669K9X0', 'E9PRA5']}\n",
      "\n",
      "Gene: ATP1A2\n",
      "Symbol: ATP1A2\n",
      "Name: ATPase Na+/K+ transporting subunit alpha 2\n",
      "Summary: The protein encoded by this gene belongs to the family of P-type cation transport ATPases, and to the subfamily of Na+/K+ -ATPases. Na+/K+ -ATPase is an integral membrane protein responsible for establishing and maintaining the electrochemical gradients of Na and K ions across the plasma membrane. These gradients are essential for osmoregulation, for sodium-coupled transport of a variety of organic and inorganic molecules, and for electrical excitability of nerve and muscle. This enzyme is composed of two subunits, a large catalytic subunit (alpha) and a smaller glycoprotein subunit (beta). The catalytic subunit of Na+/K+ -ATPase is encoded by multiple genes. This gene encodes an alpha 2 subunit. Mutations in this gene result in familial basilar or hemiplegic migraines, and in a rare syndrome known as alternating hemiplegia of childhood. [provided by RefSeq, Oct 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - neurotransmitter uptake\n",
      "    - behavioral fear response\n",
      "    - regulation of the force of heart contraction\n",
      "    - regulation of respiratory gaseous exchange by nervous system process\n",
      "    - potassium ion transport\n",
      "    - sodium ion transport\n",
      "    - intracellular sodium ion homeostasis\n",
      "    - intracellular sodium ion homeostasis\n",
      "    - regulation of muscle contraction\n",
      "    - regulation of smooth muscle contraction\n",
      "    - regulation of striated muscle contraction\n",
      "    - regulation of blood pressure\n",
      "    - adult locomotory behavior\n",
      "    - visual learning\n",
      "    - establishment or maintenance of transmembrane electrochemical gradient\n",
      "    - regulation of cardiac muscle contraction by regulation of the release of sequestered calcium ion\n",
      "    - response to auditory stimulus\n",
      "    - neuronal action potential propagation\n",
      "    - regulation of vasoconstriction\n",
      "    - L-ascorbic acid metabolic process\n",
      "    - amygdala development\n",
      "    - olfactory cortex development\n",
      "    - intracellular potassium ion homeostasis\n",
      "    - intracellular potassium ion homeostasis\n",
      "    - intracellular potassium ion homeostasis\n",
      "    - response to nicotine\n",
      "    - locomotory exploration behavior\n",
      "    - sodium ion transmembrane transport\n",
      "    - response to potassium ion\n",
      "    - sodium ion export across plasma membrane\n",
      "    - sodium ion export across plasma membrane\n",
      "    - sodium ion export across plasma membrane\n",
      "    - sodium ion export across plasma membrane\n",
      "    - locomotion\n",
      "    - negative regulation of heart contraction\n",
      "    - positive regulation of heart contraction\n",
      "    - negative regulation of striated muscle contraction\n",
      "    - ATP metabolic process\n",
      "    - negative regulation of cytosolic calcium ion concentration\n",
      "    - regulation of glutamate uptake involved in transmission of nerve impulse\n",
      "    - regulation of synaptic transmission, glutamatergic\n",
      "    - relaxation of cardiac muscle\n",
      "    - cardiac muscle contraction\n",
      "    - cellular response to mechanical stimulus\n",
      "    - cellular response to steroid hormone stimulus\n",
      "    - potassium ion transmembrane transport\n",
      "    - regulation of cardiac muscle cell contraction\n",
      "    - membrane repolarization\n",
      "    - membrane depolarization during cardiac muscle cell action potential\n",
      "    - cell communication by electrical coupling involved in cardiac conduction\n",
      "    - monoatomic cation transmembrane transport\n",
      "    - transport across blood-brain barrier\n",
      "    - proton transmembrane transport\n",
      "    - negative regulation of calcium ion transmembrane transport\n",
      "    - response to glycoside\n",
      "    - response to glycoside\n",
      "    - potassium ion import across plasma membrane\n",
      "    - potassium ion import across plasma membrane\n",
      "    - potassium ion import across plasma membrane\n",
      "  CC:\n",
      "    - cytoplasm\n",
      "    - endosome\n",
      "    - endoplasmic reticulum\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - sodium:potassium-exchanging ATPase complex\n",
      "    - sodium:potassium-exchanging ATPase complex\n",
      "    - sodium:potassium-exchanging ATPase complex\n",
      "    - sodium:potassium-exchanging ATPase complex\n",
      "    - sodium:potassium-exchanging ATPase complex\n",
      "    - caveola\n",
      "    - cell surface\n",
      "    - intercalated disc\n",
      "    - membrane\n",
      "    - membrane\n",
      "    - T-tubule\n",
      "    - organelle membrane\n",
      "    - cell projection\n",
      "    - cell projection\n",
      "    - neuronal cell body\n",
      "    - dendritic spine\n",
      "    - extracellular vesicle\n",
      "  MF:\n",
      "    - P-type sodium:potassium-exchanging transporter activity\n",
      "    - P-type sodium:potassium-exchanging transporter activity\n",
      "    - P-type sodium:potassium-exchanging transporter activity\n",
      "    - P-type sodium:potassium-exchanging transporter activity\n",
      "    - P-type sodium:potassium-exchanging transporter activity\n",
      "    - steroid binding\n",
      "    - protein binding\n",
      "    - ATP binding\n",
      "    - phosphatase activity\n",
      "    - ATP hydrolysis activity\n",
      "    - ATPase-coupled monoatomic cation transmembrane transporter activity\n",
      "    - ATPase-coupled monoatomic cation transmembrane transporter activity\n",
      "    - potassium ion binding\n",
      "    - sodium ion binding\n",
      "    - protein heterodimerization activity\n",
      "    - protein-folding chaperone binding\n",
      "    - steroid hormone binding\n",
      "UniProt: {'Swiss-Prot': 'P50993', 'TrEMBL': ['H0Y7C1', 'A0A0S2Z3W6', 'B1AKY9']}\n",
      "\n",
      "Gene: ADCY1\n",
      "Symbol: ADCY1\n",
      "Name: adenylate cyclase 1\n",
      "Summary: This gene encodes a member of the of adenylate cyclase gene family that is primarily expressed in the brain. This protein is regulated by calcium/calmodulin concentration and may be involved in brain development. Alternate splicing results in multiple transcript variants. [provided by RefSeq, Aug 2013].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - cAMP biosynthetic process\n",
      "    - cAMP biosynthetic process\n",
      "    - adenylate cyclase-activating G protein-coupled receptor signaling pathway\n",
      "    - adenylate cyclase-activating G protein-coupled receptor signaling pathway\n",
      "    - axonogenesis\n",
      "    - long-term memory\n",
      "    - circadian rhythm\n",
      "    - response to xenobiotic stimulus\n",
      "    - positive regulation of CREB transcription factor activity\n",
      "    - intracellular signal transduction\n",
      "    - regulation of circadian rhythm\n",
      "    - cellular response to calcium ion\n",
      "    - presynaptic modulation of chemical synaptic transmission\n",
      "    - neuroinflammatory response\n",
      "    - positive regulation of long-term synaptic potentiation\n",
      "    - cellular response to forskolin\n",
      "  CC:\n",
      "    - nucleus\n",
      "    - cytoplasm\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - axon hillock\n",
      "    - membrane raft\n",
      "    - extracellular exosome\n",
      "    - Schaffer collateral - CA1 synapse\n",
      "    - hippocampal mossy fiber to CA3 synapse\n",
      "    - presynapse\n",
      "    - postsynaptic density membrane\n",
      "    - glutamatergic synapse\n",
      "  MF:\n",
      "    - adenylate cyclase activity\n",
      "    - calmodulin binding\n",
      "    - ATP binding\n",
      "    - calcium- and calmodulin-responsive adenylate cyclase activity\n",
      "    - calcium- and calmodulin-responsive adenylate cyclase activity\n",
      "    - metal ion binding\n",
      "UniProt: {'Swiss-Prot': 'Q08828', 'TrEMBL': 'C9J1J0'}\n",
      "\n",
      "Gene: ALDH5A1\n",
      "Symbol: ALDH5A1\n",
      "Name: aldehyde dehydrogenase 5 family member A1\n",
      "Summary: This protein belongs to the aldehyde dehydrogenase family of proteins. This gene encodes a mitochondrial NAD(+)-dependent succinic semialdehyde dehydrogenase. A deficiency of this enzyme, known as 4-hydroxybutyricaciduria, is a rare inborn error in the metabolism of the neurotransmitter 4-aminobutyric acid (GABA). In response to the defect, physiologic fluids from patients accumulate GHB, a compound with numerous neuromodulatory properties. Two transcript variants encoding distinct isoforms have been identified for this gene. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - succinate metabolic process\n",
      "    - glutamate metabolic process\n",
      "    - central nervous system development\n",
      "    - gamma-aminobutyric acid catabolic process\n",
      "    - gamma-aminobutyric acid catabolic process\n",
      "    - gamma-aminobutyric acid catabolic process\n",
      "    - gamma-aminobutyric acid catabolic process\n",
      "    - post-embryonic development\n",
      "    - synaptic transmission, GABAergic\n",
      "  CC:\n",
      "    - mitochondrion\n",
      "    - mitochondrion\n",
      "    - mitochondrion\n",
      "    - mitochondrion\n",
      "    - mitochondrion\n",
      "    - mitochondrial matrix\n",
      "    - synapse\n",
      "  MF:\n",
      "    - succinate-semialdehyde dehydrogenase (NAD+) activity\n",
      "    - succinate-semialdehyde dehydrogenase (NAD+) activity\n",
      "    - succinate-semialdehyde dehydrogenase (NAD+) activity\n",
      "    - identical protein binding\n",
      "UniProt: {'Swiss-Prot': 'P51649', 'TrEMBL': ['X5D299', 'X5DQN2', 'A0A5F9ZH98', 'A0A5F9ZI23', 'C9J8Q5', 'A0A5F9ZH05']}\n",
      "\n",
      "Gene: ETV4\n",
      "Symbol: ETV4\n",
      "Name: ETS variant transcription factor 4\n",
      "Summary: Enables DNA-binding transcription activator activity, RNA polymerase II-specific and RNA polymerase II cis-regulatory region sequence-specific DNA binding activity. Involved in positive regulation of keratinocyte differentiation and positive regulation of transcription by RNA polymerase II. Located in chromosome and nucleolus. [provided by Alliance of Genome Resources, Apr 2022]\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - regulation of transcription by RNA polymerase II\n",
      "    - cell differentiation\n",
      "    - positive regulation of keratinocyte differentiation\n",
      "    - positive regulation of transcription by RNA polymerase II\n",
      "  CC:\n",
      "    - chromatin\n",
      "    - nucleus\n",
      "    - nucleoplasm\n",
      "    - chromosome\n",
      "    - nucleolus\n",
      "  MF:\n",
      "    - RNA polymerase II cis-regulatory region sequence-specific DNA binding\n",
      "    - DNA-binding transcription factor activity, RNA polymerase II-specific\n",
      "    - DNA-binding transcription factor activity, RNA polymerase II-specific\n",
      "    - DNA-binding transcription activator activity, RNA polymerase II-specific\n",
      "    - protein binding\n",
      "    - sequence-specific double-stranded DNA binding\n",
      "UniProt: {'Swiss-Prot': 'P43268', 'TrEMBL': ['K7EMW0', 'B7Z5F4']}\n",
      "\n",
      "Gene: KIF2B\n",
      "Symbol: KIF2B\n",
      "Name: kinesin family member 2B\n",
      "Summary: Predicted to enable microtubule binding activity and microtubule motor activity. Involved in metaphase plate congression; microtubule depolymerization; and regulation of chromosome segregation. Located in intercellular bridge; mitotic spindle; and nucleolus. [provided by Alliance of Genome Resources, Apr 2022]\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - microtubule-based movement\n",
      "    - microtubule depolymerization\n",
      "    - microtubule depolymerization\n",
      "    - cell division\n",
      "    - metaphase chromosome alignment\n",
      "    - regulation of chromosome segregation\n",
      "  CC:\n",
      "    - kinetochore\n",
      "    - cytoplasm\n",
      "    - centrosome\n",
      "    - spindle\n",
      "    - cytosol\n",
      "    - kinesin complex\n",
      "    - microtubule\n",
      "  MF:\n",
      "    - microtubule motor activity\n",
      "    - ATP binding\n",
      "    - microtubule binding\n",
      "    - ATP hydrolysis activity\n",
      "UniProt: {'Swiss-Prot': 'Q8N4N8', 'TrEMBL': 'A0A140VKG5'}\n",
      "\n",
      "Gene: NQO1\n",
      "Symbol: NQO1\n",
      "Name: NAD(P)H quinone dehydrogenase 1\n",
      "Summary: This gene is a member of the NAD(P)H dehydrogenase (quinone) family and encodes a cytoplasmic 2-electron reductase. This FAD-binding protein forms homodimers and reduces quinones to hydroquinones. This protein's enzymatic activity prevents the one electron reduction of quinones that results in the production of radical species. Mutations in this gene have been associated with tardive dyskinesia (TD), an increased risk of hematotoxicity after exposure to benzene, and susceptibility to various forms of cancer. Altered expression of this protein has been seen in many tumors and is also associated with Alzheimer's disease (AD). Alternate transcriptional splice variants, encoding different isoforms, have been characterized. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - protein polyubiquitination\n",
      "    - response to ischemia\n",
      "    - NADH oxidation\n",
      "    - ubiquinone metabolic process\n",
      "    - xenobiotic metabolic process\n",
      "    - nitric oxide biosynthetic process\n",
      "    - response to oxidative stress\n",
      "    - synaptic transmission, cholinergic\n",
      "    - response to nutrient\n",
      "    - response to toxic substance\n",
      "    - response to hormone\n",
      "    - response to carbohydrate\n",
      "    - response to amine\n",
      "    - removal of superoxide radicals\n",
      "    - protein catabolic process\n",
      "    - response to estradiol\n",
      "    - response to lipopolysaccharide\n",
      "    - response to testosterone\n",
      "    - cellular response to oxidative stress\n",
      "    - negative regulation of protein catabolic process\n",
      "    - vitamin E metabolic process\n",
      "    - vitamin K metabolic process\n",
      "    - negative regulation of apoptotic process\n",
      "    - response to alkaloid\n",
      "    - positive regulation of neuron apoptotic process\n",
      "    - innate immune response\n",
      "    - cell redox homeostasis\n",
      "    - response to ethanol\n",
      "    - response to electrical stimulus\n",
      "    - cellular response to hydrogen peroxide\n",
      "    - NADPH oxidation\n",
      "    - cellular response to metal ion\n",
      "    - negative regulation of ferroptosis\n",
      "    - response to tetrachloromethane\n",
      "    - response to L-glutamine\n",
      "    - response to hydrogen sulfide\n",
      "    - response to flavonoid\n",
      "  CC:\n",
      "    - nucleus\n",
      "    - cytoplasm\n",
      "    - cytosol\n",
      "    - cytosol\n",
      "    - cytosol\n",
      "    - dendrite\n",
      "    - neuronal cell body\n",
      "    - synapse\n",
      "  MF:\n",
      "    - RNA binding\n",
      "    - NAD(P)H dehydrogenase (quinone) activity\n",
      "    - cytochrome-b5 reductase activity, acting on NAD(P)H\n",
      "    - superoxide dismutase activity\n",
      "    - protein binding\n",
      "    - NADPH dehydrogenase (quinone) activity\n",
      "    - identical protein binding\n",
      "    - NADH:ubiquinone reductase (non-electrogenic) activity\n",
      "UniProt: {'Swiss-Prot': 'P15559', 'TrEMBL': ['H3BNV2', 'B4DLR8', 'H3BRK3']}\n",
      "\n",
      "Gene: CYP2A6\n",
      "Symbol: CYP2A6\n",
      "Name: cytochrome P450 family 2 subfamily A member 6\n",
      "Summary: This gene, CYP2A6, encodes a member of the cytochrome P450 superfamily of enzymes. The cytochrome P450 proteins are monooxygenases which catalyze many reactions involved in drug metabolism and synthesis of cholesterol, steroids and other lipids. This protein localizes to the endoplasmic reticulum and its expression is induced by phenobarbital. The enzyme is known to hydroxylate coumarin, and also metabolizes nicotine, aflatoxin B1, nitrosamines, and some pharmaceuticals. Individuals with certain allelic variants are said to have a poor metabolizer phenotype, meaning they do not efficiently metabolize coumarin or nicotine. This gene is part of a large cluster of cytochrome P450 genes from the CYP2A, CYP2B and CYP2F subfamilies on chromosome 19q. The gene was formerly referred to as CYP2A3; however, it has been renamed CYP2A6. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - xenobiotic metabolic process\n",
      "    - xenobiotic metabolic process\n",
      "    - xenobiotic metabolic process\n",
      "    - steroid metabolic process\n",
      "    - coumarin metabolic process\n",
      "    - coumarin metabolic process\n",
      "    - coumarin metabolic process\n",
      "    - epoxygenase P450 pathway\n",
      "    - xenobiotic catabolic process\n",
      "    - coumarin catabolic process\n",
      "  CC:\n",
      "    - cytoplasm\n",
      "    - endoplasmic reticulum membrane\n",
      "    - cytoplasmic microtubule\n",
      "    - intracellular membrane-bounded organelle\n",
      "  MF:\n",
      "    - iron ion binding\n",
      "    - protein binding\n",
      "    - coumarin 7-hydroxylase activity\n",
      "    - coumarin 7-hydroxylase activity\n",
      "    - arachidonate epoxygenase activity\n",
      "    - oxidoreductase activity, acting on paired donors, with incorporation or reduction of molecular oxygen, reduced flavin or flavoprotein as one donor, and incorporation of one atom of oxygen\n",
      "    - enzyme binding\n",
      "    - heme binding\n",
      "    - heme binding\n",
      "UniProt: {'Swiss-Prot': 'P11509', 'TrEMBL': 'M0R2Z4'}\n",
      "\n",
      "Gene: GPX2\n",
      "Symbol: GPX2\n",
      "Name: glutathione peroxidase 2\n",
      "Summary: The protein encoded by this gene belongs to the glutathione peroxidase family, members of which catalyze the reduction of organic hydroperoxides and hydrogen peroxide (H2O2) by glutathione, and thereby protect cells against oxidative damage. Several isozymes of this gene family exist in vertebrates, which vary in cellular location and substrate specificity. This isozyme is predominantly expressed in the gastrointestinal tract (also in liver in human), is localized in the cytoplasm, and whose preferred substrate is hydrogen peroxide. Overexpression of this gene is associated with increased differentiation and proliferation in colorectal cancer. This isozyme is also a selenoprotein, containing the rare amino acid selenocysteine (Sec) at its active site. Sec is encoded by the UGA codon, which normally signals translation termination. The 3' UTRs of selenoprotein mRNAs contain a conserved stem-loop structure, designated the Sec insertion sequence (SECIS) element, that is necessary for the recognition of UGA as a Sec codon, rather than as a stop signal. Alternatively spliced transcript variants have been found for this gene. [provided by RefSeq, Jul 2016].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - response to oxidative stress\n",
      "    - cellular oxidant detoxification\n",
      "  CC:\n",
      "    - cytoplasm\n",
      "    - cytosol\n",
      "    - cytosol\n",
      "    - intercellular bridge\n",
      "    - mitotic spindle\n",
      "  MF:\n",
      "    - glutathione peroxidase activity\n",
      "    - glutathione peroxidase activity\n",
      "    - electron transfer activity\n",
      "    - phospholipid-hydroperoxide glutathione peroxidase activity\n",
      "UniProt: {'Swiss-Prot': 'P18283', 'TrEMBL': ['G3V323', 'H0YJK8', 'G3V4J6']}\n",
      "\n",
      "Gene: CHST4\n",
      "Symbol: CHST4\n",
      "Name: carbohydrate sulfotransferase 4\n",
      "Summary: This gene encodes an N-acetylglucosamine 6-O sulfotransferase. The encoded enzyme transfers sulfate from 3'phosphoadenosine 5'phospho-sulfate to the 6-hydroxyl group of N-acetylglucosamine on glycoproteins. This protein is localized to the Golgi and is involved in the modification of glycan structures on ligands of the lymphocyte homing receptor L-selectin. Alternate splicing in the 5' UTR results in multiple transcript variants that encode the same protein. [provided by RefSeq, Oct 2009].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - carbohydrate metabolic process\n",
      "    - N-acetylglucosamine metabolic process\n",
      "    - N-acetylglucosamine metabolic process\n",
      "    - protein sulfation\n",
      "    - sulfur compound metabolic process\n",
      "    - sulfur compound metabolic process\n",
      "    - inflammatory response\n",
      "    - immune response\n",
      "    - cell adhesion\n",
      "    - cell-cell signaling\n",
      "    - O-glycan processing\n",
      "    - positive regulation of leukocyte tethering or rolling\n",
      "  CC:\n",
      "    - Golgi membrane\n",
      "    - Golgi membrane\n",
      "    - trans-Golgi network\n",
      "    - trans-Golgi network\n",
      "  MF:\n",
      "    - N-acetylglucosamine 6-O-sulfotransferase activity\n",
      "    - N-acetylglucosamine 6-O-sulfotransferase activity\n",
      "    - N-acetylglucosamine 6-O-sulfotransferase activity\n",
      "    - sulfotransferase activity\n",
      "UniProt: {'Swiss-Prot': 'Q8NCG5'}\n",
      "\n",
      "Gene: AMD1\n",
      "Symbol: AMD1\n",
      "Name: adenosylmethionine decarboxylase 1\n",
      "Summary: This gene encodes an important intermediate enzyme in polyamine biosynthesis. The polyamines spermine, spermidine, and putrescine are low-molecular-weight aliphatic amines essential for cellular proliferation and tumor promotion. Multiple alternatively spliced transcript variants have been identified. Pseudogenes of this gene are found on chromosomes 5, 6, 10, X and Y. [provided by RefSeq, Dec 2013].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - polyamine metabolic process\n",
      "    - spermine biosynthetic process\n",
      "    - spermidine biosynthetic process\n",
      "  CC:\n",
      "    - cytosol\n",
      "    - cytosol\n",
      "  MF:\n",
      "    - adenosylmethionine decarboxylase activity\n",
      "    - adenosylmethionine decarboxylase activity\n",
      "    - protein binding\n",
      "    - putrescine binding\n",
      "    - identical protein binding\n",
      "UniProt: {'Swiss-Prot': 'P17707', 'TrEMBL': ['Q5VXN5', 'B4DZ60', 'A0A088AWN0', 'Q6N0B2', 'A0A5F9ZHD5', 'F6R5I9']}\n",
      "\n",
      "Gene: LAMA3\n",
      "Symbol: LAMA3\n",
      "Name: laminin subunit alpha 3\n",
      "Summary: The protein encoded by this gene belongs to the laminin family of secreted molecules. Laminins are heterotrimeric molecules that consist of alpha, beta, and gamma subunits that assemble through a coiled-coil domain. Laminins are essential for formation and function of the basement membrane and have additional functions in regulating cell migration and mechanical signal transduction. This gene encodes an alpha subunit and is responsive to several epithelial-mesenchymal regulators including keratinocyte growth factor, epidermal growth factor and insulin-like growth factor. Mutations in this gene have been identified as the cause of Herlitz type junctional epidermolysis bullosa and laryngoonychocutaneous syndrome. Alternative splicing and alternative promoter usage result in multiple transcript variants. [provided by RefSeq, Dec 2014].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - epidermis development\n",
      "    - regulation of cell adhesion\n",
      "    - regulation of cell migration\n",
      "    - hemidesmosome assembly\n",
      "    - endodermal cell differentiation\n",
      "    - regulation of embryonic development\n",
      "    - cell-cell adhesion\n",
      "  CC:\n",
      "    - extracellular region\n",
      "    - basement membrane\n",
      "    - laminin-3 complex\n",
      "    - laminin-5 complex\n",
      "    - endoplasmic reticulum\n",
      "    - adherens junction\n",
      "    - hemidesmosome\n",
      "    - collagen-containing extracellular matrix\n",
      "    - extracellular exosome\n",
      "  MF:\n",
      "    - signaling receptor binding\n",
      "    - structural molecule activity\n",
      "    - extracellular matrix structural constituent\n",
      "UniProt: {'Swiss-Prot': 'Q16787', 'TrEMBL': ['A0A0A0MTS5', 'A0A0A6YYF2', 'A0A0A0MSA0', 'A0A075B783', 'K7ERM0', 'A0A3B3ITG1', 'K7EIP4', 'K7EPP3', 'K7EQ42', 'K7EMU9']}\n",
      "\n",
      "Gene: PRAME\n",
      "Symbol: PRAME\n",
      "Name: PRAME nuclear receptor transcriptional regulator\n",
      "Summary: This gene encodes an antigen that is preferentially expressed in human melanomas and that is recognized by cytolytic T lymphocytes. It is not expressed in normal tissues, except testis. The encoded protein acts as a repressor of retinoic acid receptor, and likely confers a growth advantage to cancer cells via this function. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Apr 2014].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - protein polyubiquitination\n",
      "    - apoptotic process\n",
      "    - positive regulation of cell population proliferation\n",
      "    - cell differentiation\n",
      "    - negative regulation of apoptotic process\n",
      "    - proteasome-mediated ubiquitin-dependent protein catabolic process\n",
      "    - proteasome-mediated ubiquitin-dependent protein catabolic process\n",
      "    - negative regulation of cell differentiation\n",
      "    - negative regulation of DNA-templated transcription\n",
      "    - negative regulation of retinoic acid receptor signaling pathway\n",
      "  CC:\n",
      "    - chromatin\n",
      "    - nucleus\n",
      "    - nucleoplasm\n",
      "    - cytoplasm\n",
      "    - Golgi apparatus\n",
      "    - plasma membrane\n",
      "    - Cul2-RING ubiquitin ligase complex\n",
      "    - Cul2-RING ubiquitin ligase complex\n",
      "  MF:\n",
      "    - protein binding\n",
      "    - nuclear retinoic acid receptor binding\n",
      "    - ubiquitin-like ligase-substrate adaptor activity\n",
      "    - ubiquitin-like ligase-substrate adaptor activity\n",
      "UniProt: {'Swiss-Prot': 'P78395', 'TrEMBL': ['B7Z7K8', 'B7ZAT0', 'B7Z986', 'B5MD04', 'E7EW99', 'E7EMH2', 'H7C2P3', 'A0A0G2JRX8', 'A0A0G2JRX7', 'B5MCY4']}\n",
      "\n",
      "Gene: FMO4\n",
      "Symbol: FMO4\n",
      "Name: flavin containing dimethylaniline monoxygenase 4\n",
      "Summary: Metabolic N-oxidation of diet-derived amino-trimethylamine (TMA) is mediated by flavin-containing monooxygenase and is subject to an inherited FMO3 polymorphism in man. This results in a small subpopulation with reduced TMA N-oxidation capacity and causes fish odor syndrome (Trimethylaminuria). Three forms of the enzyme are encoded by genes clustered in the 1q23-q25 region. Flavin-containing monooxygenases are NADPH-dependent flavoenzymes that catalyzes the oxidation of soft nucleophilic heteroatom centers in drugs, pesticides, and xenobiotics. [provided by RefSeq, Jan 2015].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - xenobiotic catabolic process\n",
      "    - negative regulation of fatty acid oxidation\n",
      "    - energy homeostasis\n",
      "  CC:\n",
      "    - endoplasmic reticulum membrane\n",
      "  MF:\n",
      "    - N,N-dimethylaniline monooxygenase activity\n",
      "    - N,N-dimethylaniline monooxygenase activity\n",
      "    - protein binding\n",
      "    - hypotaurine monooxygenase activity\n",
      "    - flavin adenine dinucleotide binding\n",
      "    - NADP binding\n",
      "UniProt: {'Swiss-Prot': 'P31512'}\n",
      "\n",
      "Gene: PUM1\n",
      "Symbol: PUM1\n",
      "Name: pumilio RNA binding family member 1\n",
      "Summary: This gene encodes a member of the PUF family, evolutionarily conserved RNA-binding proteins related to the Pumilio proteins of Drosophila and the fem-3 mRNA binding factor proteins of C. elegans. The encoded protein contains a sequence-specific RNA binding domain comprised of eight repeats and N- and C-terminal flanking regions, and serves as a translational regulator of specific mRNAs by binding to their 3' untranslated regions. The evolutionarily conserved function of the encoded protein in invertebrates and lower vertebrates suggests that the human protein may be involved in translational regulation of embryogenesis, and cell development and differentiation. Alternatively spliced transcript variants encoding different isoforms have been described. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - regulation of translation\n",
      "    - spermatogenesis\n",
      "    - adult locomotory behavior\n",
      "    - post-transcriptional regulation of gene expression\n",
      "    - post-transcriptional regulation of gene expression\n",
      "    - post-transcriptional gene silencing\n",
      "    - miRNA processing\n",
      "    - miRNA processing\n",
      "    - regulation of mRNA stability\n",
      "    - regulation of mRNA stability\n",
      "    - stem cell differentiation\n",
      "    - regulation of cell cycle\n",
      "    - regulation of chromosome segregation\n",
      "    - regulation of miRNA-mediated gene silencing\n",
      "    - mRNA destabilization\n",
      "    - 3'-UTR-mediated mRNA destabilization\n",
      "    - positive regulation of RIG-I signaling pathway\n",
      "    - positive regulation of miRNA-mediated gene silencing\n",
      "  CC:\n",
      "    - P-body\n",
      "    - nucleoplasm\n",
      "    - cytoplasm\n",
      "    - cytosol\n",
      "    - cytosol\n",
      "    - cytosol\n",
      "    - cytoplasmic stress granule\n",
      "    - axon\n",
      "  MF:\n",
      "    - RNA binding\n",
      "    - RNA binding\n",
      "    - mRNA 3'-UTR binding\n",
      "    - mRNA 3'-UTR binding\n",
      "    - mRNA 3'-UTR binding\n",
      "    - protein binding\n",
      "    - miRNA binding\n",
      "UniProt: {'Swiss-Prot': 'Q14671', 'TrEMBL': ['Q5T1Z8', 'E9PM68', 'Q5T1Z4', 'H0YDC5', 'E9PR38', 'H0YDK8', 'H0YEH2', 'H0YC97', 'H0YCK1', 'H0YDQ6', 'H0YED4', 'E9PL65', 'E9PMX1']}\n",
      "\n",
      "Gene: ABCB4\n",
      "Symbol: ABCB4\n",
      "Name: ATP binding cassette subfamily B member 4\n",
      "Summary: The membrane-associated protein encoded by this gene is a member of the superfamily of ATP-binding cassette (ABC) transporters.  ABC proteins transport various molecules across extra- and intra-cellular membranes.  ABC genes are divided into seven distinct subfamilies (ABC1, MDR/TAP, MRP, ALD, OABP, GCN20, White).  This protein is a member of the MDR/TAP subfamily. Members of the MDR/TAP subfamily are involved in multidrug resistance as well as antigen presentation.  This gene encodes a full transporter and member of the p-glycoprotein family of membrane proteins with phosphatidylcholine as its substrate.  The function of this protein has not yet been determined; however, it may involve transport of phospholipids from liver hepatocytes into bile.  Alternative splicing of this gene results in several products of undetermined function. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - lipid metabolic process\n",
      "    - positive regulation of cholesterol transport\n",
      "    - positive regulation of cholesterol transport\n",
      "    - bile acid secretion\n",
      "    - phospholipid translocation\n",
      "    - phospholipid translocation\n",
      "    - phospholipid translocation\n",
      "    - transmembrane transport\n",
      "    - lipid homeostasis\n",
      "    - positive regulation of phospholipid translocation\n",
      "    - ceramide translocation\n",
      "    - response to fenofibrate\n",
      "    - cellular response to bile acid\n",
      "    - positive regulation of phospholipid transport\n",
      "    - positive regulation of phospholipid transport\n",
      "  CC:\n",
      "    - nucleoplasm\n",
      "    - cytoplasm\n",
      "    - cytosol\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - focal adhesion\n",
      "    - membrane\n",
      "    - apical plasma membrane\n",
      "    - apical plasma membrane\n",
      "    - clathrin-coated vesicle\n",
      "    - membrane raft\n",
      "    - intercellular canaliculus\n",
      "    - extracellular exosome\n",
      "  MF:\n",
      "    - protein binding\n",
      "    - ATP binding\n",
      "    - phospholipid transporter activity\n",
      "    - ATP hydrolysis activity\n",
      "    - ATPase-coupled transmembrane transporter activity\n",
      "    - ATPase-coupled transmembrane transporter activity\n",
      "    - ATPase-coupled transmembrane transporter activity\n",
      "    - phosphatidylcholine floppase activity\n",
      "    - phosphatidylcholine floppase activity\n",
      "    - phosphatidylcholine floppase activity\n",
      "    - ceramide floppase activity\n",
      "    - ABC-type transporter activity\n",
      "UniProt: {'Swiss-Prot': 'P21439', 'TrEMBL': ['A0A2R8Y8A1', 'E7EQI1', 'A0A2R8Y291', 'H7C0M2']}\n",
      "\n",
      "Gene: SLAMF7\n",
      "Symbol: SLAMF7\n",
      "Name: SLAM family member 7\n",
      "Summary: Enables identical protein binding activity. Predicted to be involved in adaptive immune response. Predicted to act upstream of or within regulation of natural killer cell activation. Located in endoplasmic reticulum. [provided by Alliance of Genome Resources, Apr 2022]\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - adaptive immune response\n",
      "    - immune response\n",
      "    - cell adhesion\n",
      "    - natural killer cell activation\n",
      "    - T cell activation\n",
      "    - natural killer cell mediated cytotoxicity\n",
      "  CC:\n",
      "    - plasma membrane\n",
      "    - external side of plasma membrane\n",
      "  MF:\n",
      "    - identical protein binding\n",
      "UniProt: {'Swiss-Prot': 'Q9NQ25', 'TrEMBL': ['B4DVL7', 'B4DW98', 'R4GND0']}\n",
      "\n",
      "Gene: DNA2\n",
      "Symbol: DNA2\n",
      "Name: DNA replication helicase/nuclease 2\n",
      "Summary: This gene encodes a member of the DNA2/NAM7 helicase family. The encoded protein is a conserved helicase/nuclease involved in the maintenance of mitochondrial and nuclear DNA stability. Mutations in this gene are associated with autosomal dominant progressive external ophthalmoplegia-6 (PEOA6) and Seckel syndrome 8. Alternatively spliced transcript variants have been found for this gene. [provided by RefSeq, Sep 2014].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - DNA replication checkpoint signaling\n",
      "    - telomere maintenance\n",
      "    - DNA double-strand break processing\n",
      "    - DNA replication\n",
      "    - mitochondrial DNA replication\n",
      "    - base-excision repair\n",
      "    - base-excision repair\n",
      "    - telomere maintenance via semi-conservative replication\n",
      "    - DNA geometric change\n",
      "    - DNA replication, Okazaki fragment processing\n",
      "    - DNA replication, removal of RNA primer\n",
      "    - mitochondrial DNA repair\n",
      "    - positive regulation of DNA replication\n",
      "    - replication fork reversal\n",
      "    - t-circle formation\n",
      "    - mitotic telomere maintenance via semi-conservative replication\n",
      "  CC:\n",
      "    - chromosome, telomeric region\n",
      "    - nucleus\n",
      "    - nucleoplasm\n",
      "    - nucleoplasm\n",
      "    - cytoplasm\n",
      "    - mitochondrion\n",
      "    - mitochondrion\n",
      "    - gamma DNA polymerase complex\n",
      "    - mitochondrial nucleoid\n",
      "  MF:\n",
      "    - DNA binding\n",
      "    - DNA helicase activity\n",
      "    - DNA helicase activity\n",
      "    - RNA binding\n",
      "    - helicase activity\n",
      "    - nuclease activity\n",
      "    - nuclease activity\n",
      "    - protein binding\n",
      "    - ATP binding\n",
      "    - ATP hydrolysis activity\n",
      "    - ATP hydrolysis activity\n",
      "    - site-specific endodeoxyribonuclease activity, specific for altered base\n",
      "    - 5'-flap endonuclease activity\n",
      "    - 5'-flap endonuclease activity\n",
      "    - 5'-flap endonuclease activity\n",
      "    - single-stranded DNA helicase activity\n",
      "    - 5'-3' DNA helicase activity\n",
      "    - metal ion binding\n",
      "    - 4 iron, 4 sulfur cluster binding\n",
      "UniProt: {'Swiss-Prot': 'P51530', 'TrEMBL': ['F8VR31', 'H0Y455', 'F8VPM6']}\n",
      "\n",
      "Gene: RIOK1\n",
      "Symbol: RIOK1\n",
      "Name: RIO kinase 1\n",
      "Summary: The protein encoded by this gene competes with pICln for inclusion in the protein arginine methyltransferase 5 complex. This complex targets substrates for dimethylation. The encoded protein is essential for the last steps in the maturation of 40S subunits. [provided by RefSeq, Jan 2017].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - chromatin remodeling\n",
      "    - maturation of SSU-rRNA\n",
      "    - maturation of SSU-rRNA\n",
      "    - ribosomal small subunit biogenesis\n",
      "    - positive regulation of rRNA processing\n",
      "  CC:\n",
      "    - nucleoplasm\n",
      "    - cytosol\n",
      "    - cytosol\n",
      "    - cytosol\n",
      "    - preribosome, small subunit precursor\n",
      "    - preribosome, small subunit precursor\n",
      "    - methyltransferase complex\n",
      "  MF:\n",
      "    - protein serine/threonine kinase activity\n",
      "    - 3-phosphoinositide-dependent protein kinase activity\n",
      "    - DNA-dependent protein kinase activity\n",
      "    - AMP-activated protein kinase activity\n",
      "    - eukaryotic translation initiation factor 2alpha kinase activity\n",
      "    - ribosomal protein S6 kinase activity\n",
      "    - protein binding\n",
      "    - ATP binding\n",
      "    - hydrolase activity\n",
      "    - histone H3S10 kinase activity\n",
      "    - histone H3T11 kinase activity\n",
      "    - histone H3T6 kinase activity\n",
      "    - histone H2AXS139 kinase activity\n",
      "    - histone H3S28 kinase activity\n",
      "    - histone H4S1 kinase activity\n",
      "    - histone H2AS1 kinase activity\n",
      "    - histone H2BS14 kinase activity\n",
      "    - metal ion binding\n",
      "    - histone H3T3 kinase activity\n",
      "    - histone H2AS121 kinase activity\n",
      "    - Rho-dependent protein serine/threonine kinase activity\n",
      "    - protein serine kinase activity\n",
      "    - histone H2BS36 kinase activity\n",
      "    - histone H3S57 kinase activity\n",
      "    - histone H3T45 kinase activity\n",
      "    - histone H2AT120 kinase activity\n",
      "UniProt: {'Swiss-Prot': 'Q9BRS2', 'TrEMBL': 'E9PFQ8'}\n",
      "\n",
      "Gene: CHST12\n",
      "Symbol: CHST12\n",
      "Name: carbohydrate sulfotransferase 12\n",
      "Summary: The protein encoded by this gene belongs to the sulfotransferase 2 family. It is localized to the golgi membrane, and catalyzes the transfer of sulfate to position 4 of the N-acetylgalactosamine (GalNAc) residue of chondroitin and desulfated dermatan sulfate. Chondroitin sulfate constitutes the predominant proteoglycan present in cartilage, and is distributed on the surfaces of many cells and extracellular matrices. Alternatively spliced transcript variants differing only in their 5' UTRs have been found for this gene. [provided by RefSeq, Aug 2011].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - carbohydrate biosynthetic process\n",
      "    - proteoglycan biosynthetic process\n",
      "    - chondroitin sulfate biosynthetic process\n",
      "    - chondroitin sulfate biosynthetic process\n",
      "    - dermatan sulfate biosynthetic process\n",
      "  CC:\n",
      "    - Golgi membrane\n",
      "    - Golgi membrane\n",
      "    - membrane\n",
      "  MF:\n",
      "    - sulfotransferase activity\n",
      "    - chondroitin 4-sulfotransferase activity\n",
      "    - chondroitin 4-sulfotransferase activity\n",
      "    - 3'-phosphoadenosine 5'-phosphosulfate binding\n",
      "UniProt: {'Swiss-Prot': 'Q9NRB3', 'TrEMBL': 'C9J991'}\n",
      "\n",
      "Gene: MMP15\n",
      "Symbol: MMP15\n",
      "Name: matrix metallopeptidase 15\n",
      "Summary: This gene encodes a member of the peptidase M10 family and membrane-type subfamily of matrix metalloproteinases (MMPs). Proteins in this family are involved in the breakdown of extracellular matrix in normal physiological processes, such as embryonic development, reproduction, and tissue remodeling, as well as in disease processes, such as arthritis and metastasis. Members of this subfamily contain a transmembrane domain suggesting that these proteins are expressed at the cell surface rather than secreted. The encoded preproprotein is proteolytically processed to generate the mature protease. This protein may play a role in cancer progression. [provided by RefSeq, Jan 2016].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - proteolysis\n",
      "    - extracellular matrix disassembly\n",
      "    - extracellular matrix organization\n",
      "    - collagen catabolic process\n",
      "    - collagen catabolic process\n",
      "    - response to estradiol\n",
      "    - endodermal cell differentiation\n",
      "    - protein modification process\n",
      "  CC:\n",
      "    - extracellular space\n",
      "    - plasma membrane\n",
      "    - extracellular matrix\n",
      "  MF:\n",
      "    - metalloendopeptidase activity\n",
      "    - metalloendopeptidase activity\n",
      "    - protein binding\n",
      "    - enzyme activator activity\n",
      "    - zinc ion binding\n",
      "    - metalloaminopeptidase activity\n",
      "UniProt: {'Swiss-Prot': 'P51511', 'TrEMBL': 'H3BT97'}\n",
      "\n",
      "Gene: CYP4A11\n",
      "Symbol: CYP4A11\n",
      "Name: cytochrome P450 family 4 subfamily A member 11\n",
      "Summary: This gene encodes a member of the cytochrome P450 superfamily of enzymes. The cytochrome P450 proteins are monooxygenases which catalyze many reactions involved in drug metabolism and synthesis of cholesterol, steroids and other lipids. This protein localizes to the endoplasmic reticulum and hydroxylates medium-chain fatty acids such as laurate and myristate. Multiple transcript variants have been found for this gene. [provided by RefSeq, Jan 2016].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - long-chain fatty acid metabolic process\n",
      "    - kidney development\n",
      "    - renal water homeostasis\n",
      "    - pressure natriuresis\n",
      "    - fatty acid metabolic process\n",
      "    - leukotriene metabolic process\n",
      "    - arachidonate metabolic process\n",
      "    - arachidonate metabolic process\n",
      "    - epoxygenase P450 pathway\n",
      "    - oxylipin biosynthetic process\n",
      "    - positive regulation of icosanoid secretion\n",
      "    - leukotriene B4 catabolic process\n",
      "    - linoleic acid metabolic process\n",
      "    - icosanoid biosynthetic process\n",
      "    - lauric acid metabolic process\n",
      "    - sodium ion homeostasis\n",
      "    - omega-hydroxylase P450 pathway\n",
      "  CC:\n",
      "    - cytoplasm\n",
      "    - endoplasmic reticulum membrane\n",
      "    - apical plasma membrane\n",
      "    - intracellular membrane-bounded organelle\n",
      "    - intracellular membrane-bounded organelle\n",
      "  MF:\n",
      "    - monooxygenase activity\n",
      "    - iron ion binding\n",
      "    - arachidonate monooxygenase activity\n",
      "    - arachidonate epoxygenase activity\n",
      "    - alkane 1-monooxygenase activity\n",
      "    - alkane 1-monooxygenase activity\n",
      "    - heme binding\n",
      "    - leukotriene-B4 20-monooxygenase activity\n",
      "    - arachidonate omega-hydroxylase activity\n",
      "    - long-chain fatty acid omega-hydroxylase activity\n",
      "UniProt: {'Swiss-Prot': 'Q02928', 'TrEMBL': ['V9GY41', 'V9GYQ2', 'V9GYC6', 'V9GYK4', 'A0A0C4DFV7', 'V9GZ77']}\n",
      "\n",
      "Gene: CYP39A1\n",
      "Symbol: CYP39A1\n",
      "Name: cytochrome P450 family 39 subfamily A member 1\n",
      "Summary: This gene encodes a member of the cytochrome P450 superfamily of enzymes. The cytochrome P450 proteins are monooxygenases which catalyze many reactions involved in drug metabolism and synthesis of cholesterol, steroids and other lipids. This endoplasmic reticulum protein is involved in the conversion of cholesterol to bile acids. Its substrates include the oxysterols 25-hydroxycholesterol, 27-hydroxycholesterol and 24-hydroxycholesterol. Alternate splicing results in multiple transcript variants. [provided by RefSeq, Jul 2013].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - steroid biosynthetic process\n",
      "    - bile acid biosynthetic process\n",
      "    - bile acid biosynthetic process\n",
      "    - bile acid biosynthetic process\n",
      "    - bile acid biosynthetic process\n",
      "    - cholesterol catabolic process\n",
      "    - digestion\n",
      "    - sterol metabolic process\n",
      "    - cholesterol homeostasis\n",
      "  CC:\n",
      "    - endoplasmic reticulum membrane\n",
      "    - intracellular membrane-bounded organelle\n",
      "  MF:\n",
      "    - iron ion binding\n",
      "    - protein binding\n",
      "    - oxysterol 7-alpha-hydroxylase activity\n",
      "    - oxysterol 7-alpha-hydroxylase activity\n",
      "    - heme binding\n",
      "    - 24S-hydroxycholesterol 7-alpha-hydroxylase activity\n",
      "UniProt: {'Swiss-Prot': 'Q9NYL5', 'TrEMBL': ['A0A087WTD2', 'B7Z786']}\n",
      "\n",
      "Gene: DECR2\n",
      "Symbol: DECR2\n",
      "Name: 2,4-dienoyl-CoA reductase 2\n",
      "Summary: Enables 2,4-dienoyl-CoA reductase (NADPH) activity and trans-2-enoyl-CoA reductase (NADPH) activity. Involved in unsaturated fatty acid biosynthetic process. Predicted to be located in peroxisome. [provided by Alliance of Genome Resources, Apr 2022]\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - fatty acid metabolic process\n",
      "    - unsaturated fatty acid biosynthetic process\n",
      "    - fatty acid beta-oxidation using acyl-CoA oxidase\n",
      "  CC:\n",
      "    - peroxisome\n",
      "    - peroxisomal membrane\n",
      "    - cytosol\n",
      "  MF:\n",
      "    - protein binding\n",
      "    - 2,4-dienoyl-CoA reductase (NADPH) activity\n",
      "    - 2,4-dienoyl-CoA reductase (NADPH) activity\n",
      "    - trans-2-enoyl-CoA reductase (NADPH) activity\n",
      "    - trans-2-enoyl-CoA reductase (NADPH) activity\n",
      "UniProt: {'Swiss-Prot': 'Q9NUI1', 'TrEMBL': ['A0A0J9YWI0', 'A0A0J9YXZ6', 'G3V0I9', 'H7C078', 'A0A0J9YY83', 'Q4VXZ8']}\n",
      "\n",
      "Gene: GRIK1\n",
      "Symbol: GRIK1\n",
      "Name: glutamate ionotropic receptor kainate type subunit 1\n",
      "Summary: Glutamate receptors are the predominant excitatory neurotransmitter receptors in the mammalian brain and are activated in a variety of normal neurophysiologic processes. This gene product belongs to the kainate family of glutamate receptors, which are composed of four subunits and function as ligand-activated ion channels. The subunit encoded by this gene is subject to RNA editing (CAG->CGG; Q->R) within the second transmembrane domain, which is thought to alter the properties of ion flow. Alternative splicing, resulting in transcript variants encoding different isoforms, has been noted for this gene. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - glutamate receptor signaling pathway\n",
      "    - chemical synaptic transmission\n",
      "    - nervous system development\n",
      "    - central nervous system development\n",
      "    - calcium-mediated signaling\n",
      "    - monoatomic ion transmembrane transport\n",
      "    - ionotropic glutamate receptor signaling pathway\n",
      "    - synaptic transmission, glutamatergic\n",
      "    - modulation of chemical synaptic transmission\n",
      "    - regulation of synaptic transmission, glutamatergic\n",
      "    - regulation of postsynaptic membrane potential\n",
      "  CC:\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - kainate selective glutamate receptor complex\n",
      "    - presynaptic membrane\n",
      "    - intracellular membrane-bounded organelle\n",
      "    - postsynaptic density membrane\n",
      "  MF:\n",
      "    - glutamate-gated receptor activity\n",
      "    - kainate selective glutamate receptor activity\n",
      "    - kainate selective glutamate receptor activity\n",
      "    - glutamate-gated calcium ion channel activity\n",
      "    - transmitter-gated monoatomic ion channel activity involved in regulation of postsynaptic membrane potential\n",
      "UniProt: {'Swiss-Prot': 'P39086', 'TrEMBL': ['E7ENK3', 'B7Z256', 'E9PD61', 'B7Z3V7', 'E7EPY9', 'E7EPZ0', 'H7BYG7']}\n",
      "\n",
      "Gene: CYP3A43\n",
      "Symbol: CYP3A43\n",
      "Name: cytochrome P450 family 3 subfamily A member 43\n",
      "Summary: This gene encodes a member of the cytochrome P450 superfamily of enzymes. The cytochrome P450 proteins are monooxygenases which catalyze many reactions involved in drug metabolism and synthesis of cholesterol, steroids and other lipids. The encoded protein has a low level of testosterone hydroxylase activity, and may play a role in aging mechanisms and cancer progression. This gene is part of a cluster of cytochrome P450 genes on chromosome 7q21.1. Alternate splicing results in multiple transcript variants encoding different isoforms. [provided by RefSeq, Jul 2013].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - steroid metabolic process\n",
      "    - oxidative demethylation\n",
      "    - cytochrome metabolic process\n",
      "  CC:\n",
      "    - endoplasmic reticulum membrane\n",
      "  MF:\n",
      "    - monooxygenase activity\n",
      "    - monooxygenase activity\n",
      "    - iron ion binding\n",
      "    - heme binding\n",
      "    - testosterone 6-beta-hydroxylase activity\n",
      "    - estrogen 16-alpha-hydroxylase activity\n",
      "UniProt: {'Swiss-Prot': 'Q9HB55', 'TrEMBL': ['E9PG90', 'Q9HB50', 'C9JA26', 'E7EMH4', 'E9PDL8']}\n",
      "\n",
      "Gene: SLCO1C1\n",
      "Symbol: SLCO1C1\n",
      "Name: solute carrier organic anion transporter family member 1C1\n",
      "Summary: This gene encodes a member of the organic anion transporter family. The encoded protein is a transmembrane receptor that mediates the sodium-independent uptake of thyroid hormones in brain tissues. This protein has particularly high affinity for the thyroid hormones thyroxine, tri-iodothyronine and reverse tri-iodothyronine. Polymorphisms in the gene encoding this protein may be associated with fatigue and depression in patients suffering from hyperthyroidism. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Mar 2009].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - monoatomic ion transport\n",
      "    - bile acid and bile salt transport\n",
      "    - sodium-independent organic anion transport\n",
      "    - transmembrane transport\n",
      "    - thyroid hormone transport\n",
      "    - transport across blood-brain barrier\n",
      "    - positive regulation of thyroid hormone generation\n",
      "  CC:\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - basolateral plasma membrane\n",
      "    - basolateral plasma membrane\n",
      "  MF:\n",
      "    - organic anion transmembrane transporter activity\n",
      "    - bile acid transmembrane transporter activity\n",
      "    - sodium-independent organic anion transmembrane transporter activity\n",
      "    - thyroid hormone transmembrane transporter activity\n",
      "UniProt: {'Swiss-Prot': 'Q9NYB5', 'TrEMBL': 'F5H6S4'}\n",
      "\n",
      "Gene: KRTAP13-4\n",
      "Symbol: KRTAP13-4\n",
      "Name: keratin associated protein 13-4\n",
      "Summary: Predicted to be located in cytosol. [provided by Alliance of Genome Resources, Apr 2022]\n",
      "GO Annotations:\n",
      "  CC:\n",
      "    - cytosol\n",
      "    - intermediate filament\n",
      "  MF:\n",
      "    - protein binding\n",
      "UniProt: {'Swiss-Prot': 'Q3LI77'}\n",
      "\n",
      "Gene: UGT2B15\n",
      "Symbol: UGT2B15\n",
      "Name: UDP glucuronosyltransferase family 2 member B15\n",
      "Summary: This gene encodes a glycosyltransferase that is invovled in the metabolism and elimination of toxic compounts, both endogenous and of xenobiotic origin. This gene plays a role in the regulation of estrogens and androgens. This locus is present in a cluster of similar genes and pseudogenes on chromosome 4. [provided by RefSeq, Aug 2016].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - xenobiotic metabolic process\n",
      "    - xenobiotic metabolic process\n",
      "    - steroid metabolic process\n",
      "    - estrogen metabolic process\n",
      "    - estrogen metabolic process\n",
      "  CC:\n",
      "    - endoplasmic reticulum membrane\n",
      "  MF:\n",
      "    - retinoic acid binding\n",
      "    - glucuronosyltransferase activity\n",
      "    - glucuronosyltransferase activity\n",
      "    - glucuronosyltransferase activity\n",
      "UniProt: {'Swiss-Prot': 'P54855'}\n",
      "\n",
      "Gene: GRIK5\n",
      "Symbol: GRIK5\n",
      "Name: glutamate ionotropic receptor kainate type subunit 5\n",
      "Summary: This gene encodes a protein that belongs to the glutamate-gated ionic channel family. Glutamate functions as the major excitatory neurotransmitter in the central nervous system through activation of ligand-gated ion channels and G protein-coupled membrane receptors. The protein encoded by this gene forms functional heteromeric kainate-preferring ionic channels with the subunits encoded by related gene family members. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Jul 2014].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - regulation of synaptic vesicle fusion to presynaptic active zone membrane\n",
      "    - monoatomic ion transmembrane transport\n",
      "    - ionotropic glutamate receptor signaling pathway\n",
      "    - synaptic transmission, glutamatergic\n",
      "    - modulation of chemical synaptic transmission\n",
      "    - excitatory postsynaptic potential\n",
      "    - regulation of presynaptic membrane potential\n",
      "  CC:\n",
      "    - nucleoplasm\n",
      "    - endoplasmic reticulum\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - kainate selective glutamate receptor complex\n",
      "    - presynaptic membrane\n",
      "    - cell projection\n",
      "    - hippocampal mossy fiber to CA3 synapse\n",
      "    - postsynaptic density membrane\n",
      "  MF:\n",
      "    - glutamate-gated receptor activity\n",
      "    - kainate selective glutamate receptor activity\n",
      "    - kainate selective glutamate receptor activity\n",
      "    - ligand-gated monoatomic ion channel activity involved in regulation of presynaptic membrane potential\n",
      "    - ligand-gated monoatomic ion channel activity involved in regulation of presynaptic membrane potential\n",
      "    - ligand-gated monoatomic ion channel activity involved in regulation of presynaptic membrane potential\n",
      "    - transmitter-gated monoatomic ion channel activity involved in regulation of postsynaptic membrane potential\n",
      "UniProt: {'Swiss-Prot': 'Q16478', 'TrEMBL': 'M0QXF6'}\n",
      "\n",
      "Gene: MFSD12\n",
      "Symbol: MFSD12\n",
      "Name: major facilitator superfamily domain containing 12\n",
      "Summary: Enables cysteine transmembrane transporter activity. Involved in cysteine transmembrane transport; pigment metabolic process involved in pigmentation; and regulation of melanin biosynthetic process. Located in lysosome and melanosome. Part of late endosome. [provided by Alliance of Genome Resources, Apr 2022]\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - carbohydrate transport\n",
      "    - melanin biosynthetic process\n",
      "    - pigment metabolic process involved in pigmentation\n",
      "    - pigment metabolic process involved in pigmentation\n",
      "    - regulation of melanin biosynthetic process\n",
      "    - regulation of melanin biosynthetic process\n",
      "    - negative regulation of melanin biosynthetic process\n",
      "    - cysteine transmembrane transport\n",
      "    - cysteine transmembrane transport\n",
      "  CC:\n",
      "    - lysosome\n",
      "    - lysosomal membrane\n",
      "    - late endosome\n",
      "    - plasma membrane\n",
      "    - melanosome membrane\n",
      "    - melanosome\n",
      "  MF:\n",
      "    - protein binding\n",
      "    - symporter activity\n",
      "    - cysteine transmembrane transporter activity\n",
      "    - cysteine transmembrane transporter activity\n",
      "UniProt: {'Swiss-Prot': 'Q6NUT3', 'TrEMBL': ['K7EQ22', 'K7ERJ8', 'K7EJS5', 'K7EKU1', 'K7ENE1', 'A0A087WU85', 'A0A0A0MS91']}\n",
      "\n",
      "Gene: CYP3A4\n",
      "Symbol: CYP3A4\n",
      "Name: cytochrome P450 family 3 subfamily A member 4\n",
      "Summary: This gene encodes a member of the cytochrome P450 superfamily of enzymes. The cytochrome P450 proteins are monooxygenases that catalyze many reactions involved in drug metabolism and synthesis of cholesterol, steroids and other lipids. This protein localizes to the endoplasmic reticulum and its expression is induced by glucocorticoids and some pharmacological agents. This enzyme is involved in the metabolism of approximately half the drugs in use today, including acetaminophen, codeine, cyclosporin A, diazepam, erythromycin, and chloroquine. The enzyme also metabolizes some steroids and carcinogens. This gene is part of a cluster of cytochrome P450 genes on chromosome 7q21.1. Previously another CYP3A gene, CYP3A3, was thought to exist; however, it is now thought that this sequence represents a transcript variant of CYP3A4. Alternatively spliced transcript variants encoding different isoforms have been identified. [provided by RefSeq, Aug 2020].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - lipid hydroxylation\n",
      "    - lipid metabolic process\n",
      "    - steroid catabolic process\n",
      "    - xenobiotic metabolic process\n",
      "    - xenobiotic metabolic process\n",
      "    - steroid metabolic process\n",
      "    - steroid metabolic process\n",
      "    - steroid metabolic process\n",
      "    - cholesterol metabolic process\n",
      "    - androgen metabolic process\n",
      "    - estrogen metabolic process\n",
      "    - alkaloid catabolic process\n",
      "    - monoterpenoid metabolic process\n",
      "    - calcitriol biosynthetic process from calciol\n",
      "    - xenobiotic catabolic process\n",
      "    - xenobiotic catabolic process\n",
      "    - vitamin D metabolic process\n",
      "    - vitamin D catabolic process\n",
      "    - retinol metabolic process\n",
      "    - retinol metabolic process\n",
      "    - retinoic acid metabolic process\n",
      "    - long-chain fatty acid biosynthetic process\n",
      "    - aflatoxin metabolic process\n",
      "    - oxidative demethylation\n",
      "    - oxidative demethylation\n",
      "  CC:\n",
      "    - cytoplasm\n",
      "    - endoplasmic reticulum membrane\n",
      "    - intracellular membrane-bounded organelle\n",
      "  MF:\n",
      "    - monooxygenase activity\n",
      "    - monooxygenase activity\n",
      "    - monooxygenase activity\n",
      "    - steroid binding\n",
      "    - iron ion binding\n",
      "    - protein binding\n",
      "    - steroid hydroxylase activity\n",
      "    - steroid hydroxylase activity\n",
      "    - retinoic acid 4-hydroxylase activity\n",
      "    - oxidoreductase activity\n",
      "    - oxygen binding\n",
      "    - enzyme binding\n",
      "    - heme binding\n",
      "    - vitamin D3 25-hydroxylase activity\n",
      "    - caffeine oxidase activity\n",
      "    - quinine 3-monooxygenase activity\n",
      "    - testosterone 6-beta-hydroxylase activity\n",
      "    - testosterone 6-beta-hydroxylase activity\n",
      "    - 1-alpha,25-dihydroxyvitamin D3 23-hydroxylase activity\n",
      "    - anandamide 8,9 epoxidase activity\n",
      "    - anandamide 11,12 epoxidase activity\n",
      "    - anandamide 14,15 epoxidase activity\n",
      "    - vitamin D 24-hydroxylase activity\n",
      "    - estrogen 16-alpha-hydroxylase activity\n",
      "    - estrogen 16-alpha-hydroxylase activity\n",
      "    - estrogen 2-hydroxylase activity\n",
      "    - 1,8-cineole 2-exo-monooxygenase activity\n",
      "UniProt: {'Swiss-Prot': 'P08684', 'TrEMBL': ['C9JBD2', 'Q6GRK0', 'A0A499FJM4', 'A0A494C0W7', 'E7EVM8']}\n",
      "\n",
      "Gene: NEUROG2\n",
      "Symbol: NEUROG2\n",
      "Name: neurogenin 2\n",
      "Summary: This gene encodes a neural-specific basic helix-loop-helix (bHLH) transcription factor that can specify a neuronal fate on ectodermal cells and is expressed in neural progenitor cells within the developing central and peripheral nervous systems. The protein product of this gene also plays a role in the differentiation and survival of midbrain dopaminergic neurons. [provided by RefSeq, Apr 2012].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - sensory organ development\n",
      "    - forebrain development\n",
      "    - positive regulation of transcription by RNA polymerase II\n",
      "    - positive regulation of DNA-binding transcription factor activity\n",
      "    - axon development\n",
      "  CC:\n",
      "    - chromatin\n",
      "    - nucleus\n",
      "  MF:\n",
      "    - DNA-binding transcription factor activity, RNA polymerase II-specific\n",
      "    - DNA-binding transcription factor activity, RNA polymerase II-specific\n",
      "    - protein binding\n",
      "    - protein dimerization activity\n",
      "    - E-box binding\n",
      "    - E-box binding\n",
      "    - sequence-specific double-stranded DNA binding\n",
      "UniProt: {'Swiss-Prot': 'Q9H2A3'}\n",
      "\n",
      "Gene: NUDT5\n",
      "Symbol: NUDT5\n",
      "Name: nudix hydrolase 5\n",
      "Summary: This gene belongs to the Nudix (nucleoside diphosphate linked moiety X) hydrolase superfamily. The encoded enzyme catalyzes the hydrolysis of modified nucleoside diphosphates, including ADP-ribose (ADPR) and 8-oxoGua-containing 8-oxo-dADP and 8-oxo-dGDP. Protein-bound ADP ribose can be hazardous to the cell because it can modify some amino acid residues, resulting in the inhibition of ATP-activated potassium channels. 8-oxoGua is an oxidized form of guanine that can potentially alter genetic information by pairing with adenine and cytosine in RNA. Presence of 8-oxoGua in RNA results in formation of abnormal proteins due to translational errors. [provided by RefSeq, Aug 2013].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - chromatin remodeling\n",
      "    - nucleoside phosphate metabolic process\n",
      "    - nucleotide metabolic process\n",
      "    - ribonucleoside diphosphate catabolic process\n",
      "    - D-ribose catabolic process\n",
      "    - D-ribose catabolic process\n",
      "    - ribose phosphate metabolic process\n",
      "    - nucleobase-containing small molecule metabolic process\n",
      "    - ATP generation from poly-ADP-D-ribose\n",
      "  CC:\n",
      "    - nucleus\n",
      "    - nucleus\n",
      "    - nucleus\n",
      "    - cytosol\n",
      "    - extracellular exosome\n",
      "  MF:\n",
      "    - magnesium ion binding\n",
      "    - protein binding\n",
      "    - nucleotidyltransferase activity\n",
      "    - ADP-sugar diphosphatase activity\n",
      "    - snoRNA binding\n",
      "    - identical protein binding\n",
      "    - protein homodimerization activity\n",
      "    - 8-oxo-dGDP phosphatase activity\n",
      "    - 8-oxo-GDP phosphatase activity\n",
      "    - ADP-ribose diphosphatase activity\n",
      "    - ADP-ribose diphosphatase activity\n",
      "    - ADP-ribose diphosphatase activity\n",
      "    - 5'-(N(7)-methylguanosine 5'-triphospho)-[mRNA] hydrolase activity\n",
      "UniProt: {'Swiss-Prot': 'Q9UKK9', 'TrEMBL': ['H0YEY4', 'A6NFX8', 'C9JYY9', 'A6NCQ0', 'A6NJU6']}\n",
      "\n",
      "Gene: TBXAS1\n",
      "Symbol: TBXAS1\n",
      "Name: thromboxane A synthase 1\n",
      "Summary: This gene encodes a member of the cytochrome P450 superfamily of enzymes. The cytochrome P450 proteins are monooxygenases which catalyze many reactions involved in drug metabolism and synthesis of cholesterol, steroids and other lipids. However, this protein is considered a member of the cytochrome P450 superfamily on the basis of sequence similarity rather than functional similarity. This endoplasmic reticulum membrane protein catalyzes the conversion of prostglandin H2 to thromboxane A2, a potent vasoconstrictor and inducer of platelet aggregation. The enzyme plays a role in several pathophysiological processes including hemostasis, cardiovascular disease, and stroke. Alternatively spliced transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Aug 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - prostaglandin biosynthetic process\n",
      "    - icosanoid metabolic process\n",
      "    - icosanoid metabolic process\n",
      "    - intracellular chloride ion homeostasis\n",
      "    - response to ethanol\n",
      "    - positive regulation of vasoconstriction\n",
      "    - prostanoid biosynthetic process\n",
      "    - response to fatty acid\n",
      "  CC:\n",
      "    - endoplasmic reticulum\n",
      "    - endoplasmic reticulum membrane\n",
      "    - endoplasmic reticulum membrane\n",
      "    - cytosol\n",
      "  MF:\n",
      "    - monooxygenase activity\n",
      "    - thromboxane-A synthase activity\n",
      "    - thromboxane-A synthase activity\n",
      "    - thromboxane-A synthase activity\n",
      "    - thromboxane-A synthase activity\n",
      "    - iron ion binding\n",
      "    - protein binding\n",
      "    - oxidoreductase activity, acting on paired donors, with incorporation or reduction of molecular oxygen\n",
      "    - heme binding\n",
      "    - 12-hydroxyheptadecatrienoic acid synthase activity\n",
      "    - hydroperoxy icosatetraenoate dehydratase activity\n",
      "UniProt: {'Swiss-Prot': 'P24557', 'TrEMBL': ['B4DVP1', 'Q53F23', 'F8WC80', 'C9JS68', 'A0A498U6I9', 'F8WD37']}\n",
      "\n",
      "Gene: CYP2J2\n",
      "Symbol: CYP2J2\n",
      "Name: cytochrome P450 family 2 subfamily J member 2\n",
      "Summary: This gene encodes a member of the cytochrome P450 superfamily of enzymes. The cytochrome P450 proteins are monooxygenases which catalyze many reactions involved in drug metabolism and synthesis of cholesterol, steroids and other lipids. This protein localizes to the endoplasmic reticulum and is thought to be the predominant enzyme responsible for epoxidation of endogenous arachidonic acid in cardiac tissue. Multiple transcript variants have been found for this gene. [provided by RefSeq, Jan 2016].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - organic acid metabolic process\n",
      "    - fatty acid metabolic process\n",
      "    - icosanoid metabolic process\n",
      "    - xenobiotic metabolic process\n",
      "    - xenobiotic metabolic process\n",
      "    - regulation of heart contraction\n",
      "    - epoxygenase P450 pathway\n",
      "    - epoxygenase P450 pathway\n",
      "    - linoleic acid metabolic process\n",
      "  CC:\n",
      "    - cytoplasm\n",
      "    - endoplasmic reticulum membrane\n",
      "    - intracellular membrane-bounded organelle\n",
      "    - extracellular exosome\n",
      "  MF:\n",
      "    - monooxygenase activity\n",
      "    - iron ion binding\n",
      "    - arachidonate epoxygenase activity\n",
      "    - arachidonate epoxygenase activity\n",
      "    - arachidonate 14,15-epoxygenase activity\n",
      "    - arachidonate 11,12-epoxygenase activity\n",
      "    - oxidoreductase activity, acting on paired donors, with incorporation or reduction of molecular oxygen, reduced flavin or flavoprotein as one donor, and incorporation of one atom of oxygen\n",
      "    - isomerase activity\n",
      "    - heme binding\n",
      "    - linoleic acid epoxygenase activity\n",
      "    - hydroperoxy icosatetraenoate isomerase activity\n",
      "    - arachidonate 5,6-epoxygenase activity\n",
      "UniProt: {'Swiss-Prot': 'P51589', 'TrEMBL': ['A0A3B3IU95', 'A0A3B3IT99', 'A0A3B3ITF2']}\n",
      "\n",
      "Gene: CYP26A1\n",
      "Symbol: CYP26A1\n",
      "Name: cytochrome P450 family 26 subfamily A member 1\n",
      "Summary: This gene encodes a member of the cytochrome P450 superfamily of enzymes. The cytochrome P450 proteins are monooxygenases which catalyze many reactions involved in drug metabolism and synthesis of cholesterol, steroids and other lipids. This endoplasmic reticulum protein acts on retinoids, including all-trans-retinoic acid (RA), with both 4-hydroxylation and 18-hydroxylation activities. This enzyme regulates the cellular level of retinoic acid which is involved in regulation of gene expression in both embryonic and adult tissues. Two alternatively spliced transcript variants of this gene, which encode the distinct isoforms, have been reported. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - kidney development\n",
      "    - vitamin metabolic process\n",
      "    - xenobiotic metabolic process\n",
      "    - central nervous system development\n",
      "    - response to retinoic acid\n",
      "    - response to vitamin A\n",
      "    - retinoic acid catabolic process\n",
      "    - retinoic acid catabolic process\n",
      "    - retinoic acid metabolic process\n",
      "    - negative regulation of retinoic acid receptor signaling pathway\n",
      "  CC:\n",
      "    - endoplasmic reticulum membrane\n",
      "  MF:\n",
      "    - retinoic acid binding\n",
      "    - monooxygenase activity\n",
      "    - iron ion binding\n",
      "    - retinoic acid 4-hydroxylase activity\n",
      "    - retinoic acid 4-hydroxylase activity\n",
      "    - oxidoreductase activity, acting on paired donors, with incorporation or reduction of molecular oxygen, NAD(P)H as one donor, and incorporation of one atom of oxygen\n",
      "    - oxygen binding\n",
      "    - heme binding\n",
      "    - all-trans retinoic acid 4-hydrolase activity\n",
      "    - all-trans retinoic acid 18-hydroxylase activity\n",
      "UniProt: {'Swiss-Prot': 'O43174', 'TrEMBL': 'A0A096LNN2'}\n",
      "\n",
      "Gene: ABHD12\n",
      "Symbol: ABHD12\n",
      "Name: abhydrolase domain containing 12, lysophospholipase\n",
      "Summary: This gene encodes an enzyme that catalyzes the hydrolysis of 2-arachidonoyl glycerol (2-AG), the main endocannabinoid lipid transmitter that acts on cannabinoid receptors, CB1 and CB2. The endocannabinoid system is involved in a wide range of physiological processes, including neurotransmission, mood, appetite, pain appreciation, addiction behavior, and inflammation. Mutations in this gene are associated with the neurodegenerative disease, PHARC (polyneuropathy, hearing loss, ataxia, retinitis pigmentosa, and cataract), resulting from an inborn error of endocannabinoid metabolism. Alternatively spliced transcript variants encoding different isoforms have been noted for this gene.[provided by RefSeq, Jan 2011].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - phosphatidylserine catabolic process\n",
      "    - phosphatidylserine catabolic process\n",
      "    - adult walking behavior\n",
      "    - phospholipid catabolic process\n",
      "    - response to auditory stimulus\n",
      "    - arachidonate metabolic process\n",
      "    - acylglycerol catabolic process\n",
      "    - glycerophospholipid catabolic process\n",
      "    - regulation of inflammatory response\n",
      "    - monoacylglycerol catabolic process\n",
      "    - monoacylglycerol catabolic process\n",
      "    - macromolecule depalmitoylation\n",
      "  CC:\n",
      "    - endoplasmic reticulum membrane\n",
      "    - endoplasmic reticulum membrane\n",
      "    - plasma membrane\n",
      "    - membrane\n",
      "    - AMPA glutamate receptor complex\n",
      "    - dendrite cytoplasm\n",
      "  MF:\n",
      "    - phospholipase activity\n",
      "    - lysophospholipase activity\n",
      "    - lysophospholipase activity\n",
      "    - palmitoyl-(protein) hydrolase activity\n",
      "    - monoacylglycerol lipase activity\n",
      "    - monoacylglycerol lipase activity\n",
      "    - monoacylglycerol lipase activity\n",
      "UniProt: {'Swiss-Prot': 'Q8N2K0', 'TrEMBL': ['I3L294', 'I3L440', 'A0A5F9ZH28', 'A0A5F9ZHH2', 'A0A5F9ZHG5', 'I3L206', 'A0A5F9ZGZ9', 'A0A5F9ZGW7', 'A0A5F9ZH49', 'A0A5F9ZHF0', 'A0A5F9ZH71', 'A0A5F9ZGT4', 'A0A5F9ZHB5', 'A0A5F9ZHM3', 'A0A5F9ZHQ0', 'Q5T712', 'A0A5F9ZHE8', 'I3L380', 'A0A5F9ZHQ8', 'I3L1V0']}\n",
      "\n",
      "Gene: HACL1\n",
      "Symbol: HACL1\n",
      "Name: 2-hydroxyacyl-CoA lyase 1\n",
      "Summary: Enables several functions, including 2-hydroxy-3-methylhexadecanoyl-CoA lyase activity; ATP binding activity; and cation binding activity. Involved in fatty acid alpha-oxidation; phytanic acid metabolic process; and protein targeting to peroxisome. Located in nucleoplasm and peroxisome. [provided by Alliance of Genome Resources, Apr 2022]\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - fatty acid alpha-oxidation\n",
      "    - fatty acid alpha-oxidation\n",
      "    - protein targeting to peroxisome\n",
      "    - fatty acid metabolic process\n",
      "    - methyl-branched fatty acid metabolic process\n",
      "    - phytanic acid metabolic process\n",
      "  CC:\n",
      "    - nucleoplasm\n",
      "    - peroxisome\n",
      "    - peroxisome\n",
      "    - peroxisomal matrix\n",
      "    - cytosol\n",
      "    - intracellular membrane-bounded organelle\n",
      "  MF:\n",
      "    - magnesium ion binding\n",
      "    - protein binding\n",
      "    - ATP binding\n",
      "    - carbon-carbon lyase activity\n",
      "    - carbon-carbon lyase activity\n",
      "    - thiamine pyrophosphate binding\n",
      "    - thiamine pyrophosphate binding\n",
      "    - thiamine pyrophosphate binding\n",
      "    - identical protein binding\n",
      "    - 2-hydroxyacyl-CoA lyase activity\n",
      "    - 2-hydroxyphytanoyl-CoA lyase activity\n",
      "UniProt: {'Swiss-Prot': 'Q9UJ83', 'TrEMBL': ['B3KPX4', 'B4DXR1', 'Q7Z773', 'C9J306', 'C9JJM7', 'F8WCH9']}\n",
      "\n",
      "Gene: DDO\n",
      "Symbol: DDO\n",
      "Name: D-aspartate oxidase\n",
      "Summary: The protein encoded by this gene is a peroxisomal flavoprotein that catalyzes the oxidative deamination of D-aspartate and N-methyl D-aspartate. Flavin adenine dinucleotide or 6-hydroxyflavin adenine dinucleotide can serve as the cofactor in this reaction. Several transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Jan 2019].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - aspartate metabolic process\n",
      "    - aspartate catabolic process\n",
      "    - insemination\n",
      "    - grooming behavior\n",
      "    - regulation of cell communication\n",
      "    - D-amino acid catabolic process\n",
      "    - D-amino acid catabolic process\n",
      "    - hormone metabolic process\n",
      "    - nervous system process\n",
      "    - nervous system process\n",
      "  CC:\n",
      "    - cytoplasm\n",
      "    - peroxisome\n",
      "    - peroxisome\n",
      "    - peroxisome\n",
      "    - peroxisomal matrix\n",
      "    - peroxisomal matrix\n",
      "    - cytosol\n",
      "    - cytosol\n",
      "  MF:\n",
      "    - protein binding\n",
      "    - D-aspartate oxidase activity\n",
      "    - D-aspartate oxidase activity\n",
      "    - D-glutamate oxidase activity\n",
      "    - FAD binding\n",
      "UniProt: {'Swiss-Prot': 'Q99489', 'TrEMBL': 'F2Z2E0'}\n",
      "\n",
      "Gene: SLK\n",
      "Symbol: SLK\n",
      "Name: STE20 like kinase\n",
      "Summary: Enables protein homodimerization activity and protein serine/threonine kinase activity. Involved in several processes, including cytoplasmic microtubule organization; protein autophosphorylation; and regulation of focal adhesion assembly. Located in perinuclear region of cytoplasm. [provided by Alliance of Genome Resources, Apr 2022]\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - chromatin remodeling\n",
      "    - apoptotic process\n",
      "    - regulation of cell migration\n",
      "    - regulation of cell migration\n",
      "    - regulation of cell migration\n",
      "    - cytoplasmic microtubule organization\n",
      "    - intracellular signal transduction\n",
      "    - regulation of apoptotic process\n",
      "    - regulation of apoptotic process\n",
      "    - protein autophosphorylation\n",
      "    - regulation of focal adhesion assembly\n",
      "  CC:\n",
      "    - cytoplasm\n",
      "    - cytoplasm\n",
      "    - cytosol\n",
      "    - cell leading edge\n",
      "    - perinuclear region of cytoplasm\n",
      "    - extracellular exosome\n",
      "  MF:\n",
      "    - protein serine/threonine kinase activity\n",
      "    - protein serine/threonine kinase activity\n",
      "    - 3-phosphoinositide-dependent protein kinase activity\n",
      "    - DNA-dependent protein kinase activity\n",
      "    - AMP-activated protein kinase activity\n",
      "    - eukaryotic translation initiation factor 2alpha kinase activity\n",
      "    - ribosomal protein S6 kinase activity\n",
      "    - protein binding\n",
      "    - ATP binding\n",
      "    - histone H3S10 kinase activity\n",
      "    - histone H3T11 kinase activity\n",
      "    - histone H3T6 kinase activity\n",
      "    - histone H2AXS139 kinase activity\n",
      "    - identical protein binding\n",
      "    - protein homodimerization activity\n",
      "    - histone H3S28 kinase activity\n",
      "    - histone H4S1 kinase activity\n",
      "    - histone H2AS1 kinase activity\n",
      "    - histone H2BS14 kinase activity\n",
      "    - cadherin binding\n",
      "    - histone H3T3 kinase activity\n",
      "    - histone H2AS121 kinase activity\n",
      "    - Rho-dependent protein serine/threonine kinase activity\n",
      "    - protein serine kinase activity\n",
      "    - histone H2BS36 kinase activity\n",
      "    - histone H3S57 kinase activity\n",
      "    - histone H3T45 kinase activity\n",
      "    - histone H2AT120 kinase activity\n",
      "UniProt: {'Swiss-Prot': 'Q9H2G2', 'TrEMBL': 'A0A5F9ZH68'}\n",
      "\n",
      "Gene: SLC6A4\n",
      "Symbol: SLC6A4\n",
      "Name: solute carrier family 6 member 4\n",
      "Summary: This gene encodes an integral membrane protein that transports the neurotransmitter serotonin from synaptic spaces into presynaptic neurons. The encoded protein terminates the action of serotonin and recycles it in a sodium-dependent manner. This protein is a target of psychomotor stimulants, such as amphetamines and cocaine, and is a member of the sodium:neurotransmitter symporter family. A repeat length polymorphism in the promoter of this gene has been shown to affect the rate of serotonin uptake. There have been conflicting results in the literature about the possible effect, if any, that this polymorphism may play in behavior and depression. [provided by RefSeq, May 2019].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - response to hypoxia\n",
      "    - neurotransmitter transport\n",
      "    - amino acid transport\n",
      "    - response to nutrient\n",
      "    - memory\n",
      "    - circadian rhythm\n",
      "    - response to xenobiotic stimulus\n",
      "    - response to toxic substance\n",
      "    - positive regulation of gene expression\n",
      "    - positive regulation of serotonin secretion\n",
      "    - monoamine transport\n",
      "    - negative regulation of cerebellar granule cell precursor proliferation\n",
      "    - negative regulation of synaptic transmission, dopaminergic\n",
      "    - response to estradiol\n",
      "    - sodium ion transmembrane transport\n",
      "    - vasoconstriction\n",
      "    - negative regulation of neuron differentiation\n",
      "    - positive regulation of cell cycle\n",
      "    - negative regulation of organ growth\n",
      "    - behavioral response to cocaine\n",
      "    - enteric nervous system development\n",
      "    - brain morphogenesis\n",
      "    - serotonin uptake\n",
      "    - serotonin uptake\n",
      "    - serotonin uptake\n",
      "    - serotonin uptake\n",
      "    - serotonin uptake\n",
      "    - membrane depolarization\n",
      "    - male mating behavior\n",
      "    - platelet aggregation\n",
      "    - cellular response to retinoic acid\n",
      "    - cellular response to cGMP\n",
      "    - regulation of thalamus size\n",
      "    - conditioned place preference\n",
      "  CC:\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - focal adhesion\n",
      "    - endosome membrane\n",
      "    - endomembrane system\n",
      "    - presynaptic membrane\n",
      "    - neuron projection\n",
      "    - membrane raft\n",
      "    - synapse\n",
      "    - synapse\n",
      "    - postsynaptic membrane\n",
      "    - serotonergic synapse\n",
      "  MF:\n",
      "    - integrin binding\n",
      "    - monoatomic cation channel activity\n",
      "    - neurotransmitter transmembrane transporter activity\n",
      "    - serotonin:sodium:chloride symporter activity\n",
      "    - serotonin:sodium:chloride symporter activity\n",
      "    - serotonin:sodium:chloride symporter activity\n",
      "    - serotonin:sodium:chloride symporter activity\n",
      "    - protein binding\n",
      "    - monoamine transmembrane transporter activity\n",
      "    - monoamine transmembrane transporter activity\n",
      "    - antiporter activity\n",
      "    - syntaxin-1 binding\n",
      "    - cocaine binding\n",
      "    - sodium ion binding\n",
      "    - identical protein binding\n",
      "    - nitric-oxide synthase binding\n",
      "    - actin filament binding\n",
      "    - serotonin binding\n",
      "    - serotonin binding\n",
      "UniProt: {'Swiss-Prot': 'P31645', 'TrEMBL': ['J3QKP3', 'B2R7Y7', 'J3KPR9']}\n",
      "\n",
      "Gene: CYP4B1\n",
      "Symbol: CYP4B1\n",
      "Name: cytochrome P450 family 4 subfamily B member 1\n",
      "Summary: This gene encodes a member of the cytochrome P450 superfamily of enzymes. The cytochrome P450 proteins are monooxygenases which catalyze many reactions involved in drug metabolism and synthesis of cholesterol, steroids and other lipids. This protein localizes to the endoplasmic reticulum. In rodents, the homologous protein has been shown to metabolize certain carcinogens; however, the specific function of the human protein has not been determined. Multiple transcript variants have been found for this gene. [provided by RefSeq, Jan 2016].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - fatty acid metabolic process\n",
      "    - biphenyl metabolic process\n",
      "  CC:\n",
      "    - endoplasmic reticulum membrane\n",
      "  MF:\n",
      "    - monooxygenase activity\n",
      "    - iron ion binding\n",
      "    - oxidoreductase activity, acting on paired donors, with incorporation or reduction of molecular oxygen, reduced flavin or flavoprotein as one donor, and incorporation of one atom of oxygen\n",
      "    - oxygen binding\n",
      "    - heme binding\n",
      "    - heterocyclic compound binding\n",
      "UniProt: {'Swiss-Prot': 'P13584', 'TrEMBL': ['B4DV41', 'Q8IZB0', 'F5H1Q8', 'E9PML0', 'E9PL36', 'F5H5Q6']}\n",
      "\n",
      "Gene: ZNF878\n",
      "Symbol: ZNF878\n",
      "Name: zinc finger protein 878\n",
      "Summary: Predicted to enable DNA-binding transcription factor activity, RNA polymerase II-specific and RNA polymerase II transcription regulatory region sequence-specific DNA binding activity. Predicted to be involved in regulation of transcription by RNA polymerase II. Predicted to be active in nucleus. [provided by Alliance of Genome Resources, Apr 2022]\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - regulation of transcription by RNA polymerase II\n",
      "  CC:\n",
      "    - nucleus\n",
      "  MF:\n",
      "    - RNA polymerase II transcription regulatory region sequence-specific DNA binding\n",
      "    - DNA-binding transcription factor activity, RNA polymerase II-specific\n",
      "    - zinc ion binding\n",
      "UniProt: {'Swiss-Prot': 'C9JN71'}\n",
      "\n",
      "Gene: CYP2C18\n",
      "Symbol: CYP2C18\n",
      "Name: cytochrome P450 family 2 subfamily C member 18\n",
      "Summary: This gene encodes a member of the cytochrome P450 superfamily of enzymes. The cytochrome P450 proteins are monooxygenases which catalyze many reactions involved in drug metabolism and synthesis of cholesterol, steroids and other lipids. This protein localizes to the endoplasmic reticulum but its specific substrate has not yet been determined. The gene is located within a cluster of cytochrome P450 genes on chromosome 10q24. An additional gene, CYP2C17, was once thought to exist; however, CYP2C17 is now considered an artefact based on a chimera of CYP2C18 and CYP2C19. Alternatively spliced transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - organic acid metabolic process\n",
      "    - xenobiotic metabolic process\n",
      "    - xenobiotic metabolic process\n",
      "    - arachidonate metabolic process\n",
      "    - retinol metabolic process\n",
      "    - retinoic acid metabolic process\n",
      "    - linoleic acid metabolic process\n",
      "  CC:\n",
      "    - cytoplasm\n",
      "    - endoplasmic reticulum membrane\n",
      "    - plasma membrane\n",
      "    - intracellular membrane-bounded organelle\n",
      "    - intracellular membrane-bounded organelle\n",
      "  MF:\n",
      "    - monooxygenase activity\n",
      "    - iron ion binding\n",
      "    - arachidonate epoxygenase activity\n",
      "    - retinoic acid 4-hydroxylase activity\n",
      "    - oxidoreductase activity, acting on paired donors, with incorporation or reduction of molecular oxygen, reduced flavin or flavoprotein as one donor, and incorporation of one atom of oxygen\n",
      "    - oxygen binding\n",
      "    - heme binding\n",
      "    - linoleic acid epoxygenase activity\n",
      "UniProt: {'Swiss-Prot': 'P33260', 'TrEMBL': 'Q7Z348'}\n",
      "\n",
      "Gene: EPHA1\n",
      "Symbol: EPHA1\n",
      "Name: EPH receptor A1\n",
      "Summary: This gene belongs to the ephrin receptor subfamily of the protein-tyrosine kinase family. EPH and EPH-related receptors have been implicated in mediating developmental events, particularly in the nervous system. Receptors in the EPH subfamily typically have a single kinase domain and an extracellular region containing a Cys-rich domain and 2 fibronectin type III repeats. The ephrin receptors are divided into 2 groups based on the similarity of their extracellular domain sequences and their affinities for binding ephrin-A and ephrin-B ligands. This gene is expressed in some human cancer cell lines and has been implicated in carcinogenesis. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - angiogenesis\n",
      "    - positive regulation of cell-matrix adhesion\n",
      "    - negative regulation of protein kinase activity\n",
      "    - cell surface receptor signaling pathway\n",
      "    - cell surface receptor protein tyrosine kinase signaling pathway\n",
      "    - epidermal growth factor receptor signaling pathway\n",
      "    - positive regulation of cell population proliferation\n",
      "    - insulin receptor signaling pathway\n",
      "    - fibroblast growth factor receptor signaling pathway\n",
      "    - peptidyl-tyrosine phosphorylation\n",
      "    - positive regulation of cell migration\n",
      "    - negative regulation of cell migration\n",
      "    - brain-derived neurotrophic factor receptor signaling pathway\n",
      "    - substrate adhesion-dependent cell spreading\n",
      "    - platelet-derived growth factor receptor-alpha signaling pathway\n",
      "    - platelet-derived growth factor receptor-beta signaling pathway\n",
      "    - vascular endothelial growth factor receptor-1 signaling pathway\n",
      "    - collagen-activated tyrosine kinase receptor signaling pathway\n",
      "    - vascular endothelial growth factor signaling pathway\n",
      "    - Kit signaling pathway\n",
      "    - macrophage colony-stimulating factor signaling pathway\n",
      "    - regulation of GTPase activity\n",
      "    - positive regulation of angiogenesis\n",
      "    - positive regulation of angiogenesis\n",
      "    - protein autophosphorylation\n",
      "    - insulin-like growth factor receptor signaling pathway\n",
      "    - hepatocyte growth factor receptor signaling pathway\n",
      "    - ephrin receptor signaling pathway\n",
      "    - positive regulation of stress fiber assembly\n",
      "    - activation of GTPase activity\n",
      "  CC:\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - receptor complex\n",
      "  MF:\n",
      "    - fibronectin binding\n",
      "    - protein kinase activity\n",
      "    - transmembrane receptor protein tyrosine kinase activity\n",
      "    - GPI-linked ephrin receptor activity\n",
      "    - transmembrane-ephrin receptor activity\n",
      "    - epidermal growth factor receptor activity\n",
      "    - fibroblast growth factor receptor activity\n",
      "    - hepatocyte growth factor receptor activity\n",
      "    - insulin receptor activity\n",
      "    - insulin-like growth factor receptor activity\n",
      "    - macrophage colony-stimulating factor receptor activity\n",
      "    - platelet-derived growth factor alpha-receptor activity\n",
      "    - platelet-derived growth factor beta-receptor activity\n",
      "    - stem cell factor receptor activity\n",
      "    - vascular endothelial growth factor receptor activity\n",
      "    - ATP binding\n",
      "    - boss receptor activity\n",
      "    - protein kinase binding\n",
      "    - placental growth factor receptor activity\n",
      "    - protein tyrosine kinase collagen receptor activity\n",
      "    - brain-derived neurotrophic factor receptor activity\n",
      "UniProt: {'Swiss-Prot': 'P21709'}\n",
      "\n",
      "Gene: CACNA1G\n",
      "Symbol: CACNA1G\n",
      "Name: calcium voltage-gated channel subunit alpha1 G\n",
      "Summary: Voltage-sensitive calcium channels mediate the entry of calcium ions into excitable cells, and are also involved in a variety of calcium-dependent processes, including muscle contraction, hormone or neurotransmitter release, gene expression, cell motility, cell division, and cell death. This gene encodes a T-type, low-voltage activated calcium channel. The T-type channels generate currents that are both transient, owing to fast inactivation, and tiny, owing to small conductance. T-type channels are thought to be involved in pacemaker activity, low-threshold calcium spikes, neuronal oscillations and resonance, and rebound burst firing. Many alternatively spliced transcript variants encoding different isoforms have been described for this gene. [provided by RefSeq, Sep 2011].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - sinoatrial node development\n",
      "    - chemical synaptic transmission\n",
      "    - response to nickel cation\n",
      "    - regulation of membrane potential\n",
      "    - regulation of atrial cardiac muscle cell membrane depolarization\n",
      "    - calcium ion import\n",
      "    - calcium ion transmembrane transport\n",
      "    - calcium ion transmembrane transport\n",
      "    - cardiac muscle cell action potential involved in contraction\n",
      "    - SA node cell action potential\n",
      "    - AV node cell action potential\n",
      "    - SA node cell to atrial cardiac muscle cell signaling\n",
      "    - AV node cell to bundle of His cell signaling\n",
      "    - membrane depolarization during AV node cell action potential\n",
      "    - membrane depolarization during SA node cell action potential\n",
      "    - regulation of heart rate by cardiac conduction\n",
      "    - calcium ion import across plasma membrane\n",
      "  CC:\n",
      "    - cytoplasm\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - voltage-gated calcium channel complex\n",
      "    - voltage-gated calcium channel complex\n",
      "    - synapse\n",
      "  MF:\n",
      "    - voltage-gated calcium channel activity\n",
      "    - voltage-gated calcium channel activity\n",
      "    - high voltage-gated calcium channel activity\n",
      "    - low voltage-gated calcium channel activity\n",
      "    - voltage-gated calcium channel activity involved in AV node cell action potential\n",
      "    - voltage-gated calcium channel activity involved SA node cell action potential\n",
      "    - scaffold protein binding\n",
      "UniProt: {'Swiss-Prot': 'O43497', 'TrEMBL': ['A0A0B4J1X2', 'I3L2W8', 'H0YAN0']}\n",
      "\n",
      "Gene: ETV6\n",
      "Symbol: ETV6\n",
      "Name: ETS variant transcription factor 6\n",
      "Summary: This gene encodes an ETS family transcription factor. The product of this gene contains two functional domains: a N-terminal pointed (PNT) domain that is involved in protein-protein interactions with itself and other proteins, and a C-terminal DNA-binding domain. Gene knockout studies in mice suggest that it is required for hematopoiesis and maintenance of the developing vascular network. This gene is known to be involved in a large number of chromosomal rearrangements associated with leukemia and congenital fibrosarcoma. [provided by RefSeq, Sep 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - negative regulation of transcription by RNA polymerase II\n",
      "    - negative regulation of transcription by RNA polymerase II\n",
      "    - regulation of transcription by RNA polymerase II\n",
      "    - vitellogenesis\n",
      "    - neurogenesis\n",
      "    - cell differentiation\n",
      "    - positive regulation of transcription by RNA polymerase II\n",
      "    - hematopoietic stem cell proliferation\n",
      "    - mesenchymal cell apoptotic process\n",
      "  CC:\n",
      "    - chromatin\n",
      "    - nucleus\n",
      "    - nucleolus\n",
      "    - cytosol\n",
      "    - cytosol\n",
      "    - plasma membrane\n",
      "  MF:\n",
      "    - RNA polymerase II transcription regulatory region sequence-specific DNA binding\n",
      "    - RNA polymerase II cis-regulatory region sequence-specific DNA binding\n",
      "    - DNA-binding transcription factor activity, RNA polymerase II-specific\n",
      "    - DNA-binding transcription factor activity, RNA polymerase II-specific\n",
      "    - DNA-binding transcription repressor activity, RNA polymerase II-specific\n",
      "    - DNA-binding transcription activator activity, RNA polymerase II-specific\n",
      "    - DNA-binding transcription factor activity\n",
      "    - protein binding\n",
      "    - protein domain specific binding\n",
      "UniProt: {'Swiss-Prot': 'P41212', 'TrEMBL': ['H0YG25', 'A0A0S2Z3C9', 'J3KN52']}\n",
      "\n",
      "Gene: IL9\n",
      "Symbol: IL9\n",
      "Name: interleukin 9\n",
      "Summary: The protein encoded by this gene is a cytokine that acts as a regulator of a variety of hematopoietic cells. This cytokine stimulates cell proliferation and prevents apoptosis. It functions through the interleukin 9 receptor (IL9R), which activates different signal transducer and activator (STAT) proteins and thus connects this cytokine to various biological processes. The gene encoding this cytokine has been identified as a candidate gene for asthma. Genetic studies on a mouse model of asthma demonstrated that this cytokine is a determining factor in the pathogenesis of bronchial hyperresponsiveness. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - inflammatory response\n",
      "    - positive regulation of cell population proliferation\n",
      "    - immunoglobulin mediated immune response\n",
      "    - B cell differentiation\n",
      "    - positive regulation of cell growth\n",
      "    - positive regulation of interleukin-5 production\n",
      "    - interleukin-9-mediated signaling pathway\n",
      "    - B cell proliferation\n",
      "    - regulation of receptor signaling pathway via JAK-STAT\n",
      "  CC:\n",
      "    - extracellular region\n",
      "    - extracellular space\n",
      "    - extracellular space\n",
      "  MF:\n",
      "    - cytokine activity\n",
      "    - cytokine activity\n",
      "    - interleukin-9 receptor binding\n",
      "    - growth factor activity\n",
      "UniProt: {'Swiss-Prot': 'P15248'}\n",
      "\n",
      "Gene: PTH2R\n",
      "Symbol: PTH2R\n",
      "Name: parathyroid hormone 2 receptor\n",
      "Summary: The protein encoded by this gene is a member of the G-protein coupled receptor 2 family. This protein is a receptor for parathyroid hormone (PTH). This receptor is more selective in ligand recognition and has a more specific tissue distribution compared to parathyroid hormone receptor 1 (PTHR1). It is activated only by PTH and not by parathyroid hormone-like hormone (PTHLH) and is particularly abundant in brain and pancreas. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Jan 2013].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - cell surface receptor signaling pathway\n",
      "    - G protein-coupled receptor signaling pathway\n",
      "    - adenylate cyclase-modulating G protein-coupled receptor signaling pathway\n",
      "    - positive regulation of cold-induced thermogenesis\n",
      "  CC:\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "  MF:\n",
      "    - parathyroid hormone receptor activity\n",
      "    - protein binding\n",
      "    - G protein-coupled peptide receptor activity\n",
      "    - peptide hormone binding\n",
      "UniProt: {'Swiss-Prot': 'P49190', 'TrEMBL': ['H7C0B0', 'B4DFN8']}\n",
      "\n",
      "Gene: LRAT\n",
      "Symbol: LRAT\n",
      "Name: lecithin retinol acyltransferase\n",
      "Summary: The protein encoded by this gene localizes to the endoplasmic reticulum, where it catalyzes the esterification of all-trans-retinol into all-trans-retinyl ester. This reaction is an important step in vitamin A metabolism in the visual system. Mutations in this gene have been associated with early-onset severe retinal dystrophy and Leber congenital amaurosis 14. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Aug 2014].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - retinoid metabolic process\n",
      "    - vitamin A metabolic process\n",
      "    - vitamin A metabolic process\n",
      "    - visual perception\n",
      "    - response to bacterium\n",
      "    - lipid storage\n",
      "    - positive regulation of lipid transport\n",
      "    - response to retinoic acid\n",
      "    - response to vitamin A\n",
      "    - retinol metabolic process\n",
      "    - retinol metabolic process\n",
      "    - retinol metabolic process\n",
      "    - maintenance of location in cell\n",
      "    - cellular response to leukemia inhibitory factor\n",
      "  CC:\n",
      "    - multivesicular body\n",
      "    - endoplasmic reticulum\n",
      "    - endoplasmic reticulum membrane\n",
      "    - rough endoplasmic reticulum\n",
      "    - rough endoplasmic reticulum\n",
      "    - perinuclear region of cytoplasm\n",
      "  MF:\n",
      "    - retinoic acid binding\n",
      "    - protein binding\n",
      "    - O-palmitoyltransferase activity\n",
      "    - acyltransferase activity\n",
      "    - retinol binding\n",
      "    - phosphatidylcholine-retinol O-acyltransferase activity\n",
      "    - phosphatidylcholine-retinol O-acyltransferase activity\n",
      "    - phosphatidylcholine-retinol O-acyltransferase activity\n",
      "UniProt: {'Swiss-Prot': 'O95237', 'TrEMBL': 'D6RC94'}\n",
      "\n",
      "Gene: CHAT\n",
      "Symbol: CHAT\n",
      "Name: choline O-acetyltransferase\n",
      "Summary: This gene encodes an enzyme which catalyzes the biosynthesis of the neurotransmitter acetylcholine. This gene product is a characteristic feature of cholinergic neurons, and changes in these neurons may explain some of the symptoms of Alzheimer's disease. Polymorphisms in this gene have been associated with Alzheimer's disease and mild cognitive impairment. Mutations in this gene are associated with congenital myasthenic syndrome associated with episodic apnea. Multiple transcript variants encoding different isoforms have been found for this gene, and some of these variants have been shown to encode more than one isoform. [provided by RefSeq, May 2010].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - phosphatidylcholine biosynthetic process\n",
      "    - neurotransmitter transport\n",
      "    - neuromuscular synaptic transmission\n",
      "    - acetylcholine biosynthetic process\n",
      "  CC:\n",
      "    - nucleus\n",
      "    - cytoplasm\n",
      "    - cytosol\n",
      "    - neuron projection\n",
      "    - synapse\n",
      "  MF:\n",
      "    - choline O-acetyltransferase activity\n",
      "    - choline O-acetyltransferase activity\n",
      "    - protein binding\n",
      "UniProt: {'Swiss-Prot': 'P28329', 'TrEMBL': ['Q6LEN6', 'Q6LEN5', 'A0A1W2PP46', 'R4GN13', 'A0A1W2PRG4']}\n",
      "\n",
      "Gene: MATN3\n",
      "Symbol: MATN3\n",
      "Name: matrilin 3\n",
      "Summary: This gene encodes a member of von Willebrand factor A domain containing protein family. This family of proteins is thought to be involved in the formation of filamentous networks in the extracellular matrices of various tissues. This protein contains two von Willebrand factor A domains; it is present in the cartilage extracellular matrix and has a role in the development and homeostasis of cartilage and bone. Mutations in this gene result in multiple epiphyseal dysplasia. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - skeletal system development\n",
      "    - extracellular matrix organization\n",
      "    - extracellular matrix organization\n",
      "    - cartilage development\n",
      "  CC:\n",
      "    - extracellular region\n",
      "    - endoplasmic reticulum lumen\n",
      "    - extracellular matrix\n",
      "    - collagen-containing extracellular matrix\n",
      "    - collagen-containing extracellular matrix\n",
      "    - collagen-containing extracellular matrix\n",
      "    - matrilin complex\n",
      "  MF:\n",
      "    - extracellular matrix structural constituent\n",
      "    - calcium ion binding\n",
      "    - protein binding\n",
      "UniProt: {'Swiss-Prot': 'O15232'}\n",
      "\n",
      "Gene: LAMA1\n",
      "Symbol: LAMA1\n",
      "Name: laminin subunit alpha 1\n",
      "Summary: This gene encodes one of the alpha 1 subunits of laminin. The laminins are a family of extracellular matrix glycoproteins that have a heterotrimeric structure consisting of an alpha, beta and gamma chain. These proteins make up a major component of the basement membrane and have been implicated in a wide variety of biological processes including cell adhesion, differentiation, migration, signaling, neurite outgrowth and metastasis. Mutations in this gene may be associated with Poretti-Boltshauser syndrome. [provided by RefSeq, Sep 2014].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - morphogenesis of an epithelial sheet\n",
      "    - cell adhesion\n",
      "    - cell surface receptor signaling pathway\n",
      "    - regulation of cell migration\n",
      "    - neuron projection development\n",
      "    - establishment of epithelial cell apical/basal polarity\n",
      "    - positive regulation of cell adhesion\n",
      "    - regulation of embryonic development\n",
      "    - positive regulation of muscle cell differentiation\n",
      "    - epithelial tube branching involved in lung morphogenesis\n",
      "    - branching involved in salivary gland morphogenesis\n",
      "    - retinal blood vessel morphogenesis\n",
      "    - regulation of basement membrane organization\n",
      "    - positive regulation of integrin-mediated signaling pathway\n",
      "  CC:\n",
      "    - extracellular region\n",
      "    - basement membrane\n",
      "    - basement membrane\n",
      "    - laminin-1 complex\n",
      "    - laminin-1 complex\n",
      "    - laminin-3 complex\n",
      "    - extracellular space\n",
      "    - cell-cell junction\n",
      "    - membrane\n",
      "    - extracellular matrix\n",
      "    - collagen-containing extracellular matrix\n",
      "    - collagen-containing extracellular matrix\n",
      "    - protein complex involved in cell-matrix adhesion\n",
      "  MF:\n",
      "    - signaling receptor binding\n",
      "    - extracellular matrix structural constituent\n",
      "    - protein binding\n",
      "    - glycosphingolipid binding\n",
      "UniProt: {'Swiss-Prot': 'P25391'}\n",
      "\n",
      "Gene: GRIN3A\n",
      "Symbol: GRIN3A\n",
      "Name: glutamate ionotropic receptor NMDA type subunit 3A\n",
      "Summary: This gene encodes a subunit of the N-methyl-D-aspartate (NMDA) receptors, which belong to the superfamily of glutamate-regulated ion channels, and function in physiological and pathological processes in the central nervous system. This subunit shows greater than 90% identity to the corresponding subunit in rat. Studies in the knockout mouse deficient in this subunit suggest that this gene may be involved in the development of synaptic elements by modulating NMDA receptor activity. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - calcium ion transport\n",
      "    - dendrite development\n",
      "    - ionotropic glutamate receptor signaling pathway\n",
      "    - synaptic transmission, glutamatergic\n",
      "    - response to ethanol\n",
      "    - regulation of synaptic plasticity\n",
      "    - rhythmic process\n",
      "    - modulation of chemical synaptic transmission\n",
      "    - regulation of postsynaptic membrane potential\n",
      "    - prepulse inhibition\n",
      "    - negative regulation of dendritic spine development\n",
      "    - calcium ion transmembrane transport\n",
      "    - monoatomic cation transmembrane transport\n",
      "    - presynaptic modulation of chemical synaptic transmission\n",
      "  CC:\n",
      "    - endoplasmic reticulum membrane\n",
      "    - plasma membrane\n",
      "    - membrane\n",
      "    - membrane\n",
      "    - NMDA selective glutamate receptor complex\n",
      "    - monoatomic ion channel complex\n",
      "    - neuron projection\n",
      "    - neuronal cell body\n",
      "    - synapse\n",
      "    - postsynaptic membrane\n",
      "    - presynapse\n",
      "    - postsynaptic density membrane\n",
      "    - neurotransmitter receptor complex\n",
      "    - glutamatergic synapse\n",
      "  MF:\n",
      "    - NMDA glutamate receptor activity\n",
      "    - NMDA glutamate receptor activity\n",
      "    - calcium channel activity\n",
      "    - protein binding\n",
      "    - glutamate receptor activity\n",
      "    - glycine binding\n",
      "    - identical protein binding\n",
      "    - protein phosphatase 2A binding\n",
      "    - serine binding\n",
      "    - glycine-gated cation channel activity\n",
      "    - transmitter-gated monoatomic ion channel activity involved in regulation of postsynaptic membrane potential\n",
      "UniProt: {'Swiss-Prot': 'Q8TCU5'}\n",
      "\n",
      "Gene: MFAP5\n",
      "Symbol: MFAP5\n",
      "Name: microfibril associated protein 5\n",
      "Summary: This gene encodes a 25-kD microfibril-associated glycoprotein which is a component of microfibrils of the extracellular matrix. The encoded protein promotes attachment of cells to microfibrils via alpha-V-beta-3 integrin. Deficiency of this gene in mice results in neutropenia. Alternate splicing results in multiple transcript variants encoding different isoforms. [provided by RefSeq, Jul 2014].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - embryonic eye morphogenesis\n",
      "    - definitive hemopoiesis\n",
      "    - supramolecular fiber organization\n",
      "  CC:\n",
      "    - microfibril\n",
      "    - microfibril\n",
      "    - extracellular region\n",
      "    - collagen-containing extracellular matrix\n",
      "  MF:\n",
      "    - extracellular matrix structural constituent\n",
      "UniProt: {'Swiss-Prot': 'Q13361', 'TrEMBL': ['F5H413', 'B3KW70', 'F5GYX4', 'H0YG03', 'F5H7Z2', 'F5H2W4', 'H0YGS3', 'F5H1C0']}\n",
      "\n",
      "Gene: LAMC2\n",
      "Symbol: LAMC2\n",
      "Name: laminin subunit gamma 2\n",
      "Summary: Laminins, a family of extracellular matrix glycoproteins, are the major noncollagenous constituent of basement membranes. They have been implicated in a wide variety of biological processes including cell adhesion, differentiation, migration, signaling, neurite outgrowth and metastasis. Laminins, composed of 3 non identical chains: laminin alpha, beta and gamma (formerly A, B1, and B2, respectively), have a cruciform structure consisting of 3 short arms, each formed by a different chain, and a long arm composed of all 3 chains. Each laminin chain is a multidomain protein encoded by a distinct gene. Several isoforms of each chain have been described. Different alpha, beta and gamma chain isomers combine to give rise to different heterotrimeric laminin isoforms which are designated by Arabic numerals in the order of their discovery, i.e. alpha1beta1gamma1 heterotrimer is laminin 1. The biological functions of the different chains and trimer molecules are largely unknown, but some of the chains have been shown to differ with respect to their tissue distribution, presumably reflecting diverse functions in vivo. This gene encodes the gamma chain isoform laminin, gamma 2. The gamma 2 chain, formerly thought to be a truncated version of beta chain (B2t), is highly homologous to the gamma 1 chain; however, it lacks domain VI, and domains V, IV and III are shorter. It is expressed in several fetal tissues but differently from gamma 1, and is specifically localized to epithelial cells in skin, lung and kidney. The gamma 2 chain together with alpha 3 and beta 3 chains constitute laminin 5 (earlier known as kalinin), which is an integral part of the anchoring filaments that connect epithelial cells to the underlying basement membrane. The epithelium-specific expression of the gamma 2 chain implied its role as an epithelium attachment molecule, and mutations in this gene have been associated with junctional epidermolysis bullosa, a skin disease characterized by blisters due to disruption of the epidermal-dermal junction. Two transcript variants resulting from alternative splicing of the 3' terminal exon, and encoding different isoforms of gamma 2 chain, have been described. The two variants are differentially expressed in embryonic tissues, however, the biological significance of the two forms is not known. Transcript variants utilizing alternative polyA_signal have also been noted in literature. [provided by RefSeq, Aug 2011].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - cell adhesion\n",
      "    - positive regulation of cell population proliferation\n",
      "    - epidermis development\n",
      "    - positive regulation of cell migration\n",
      "  CC:\n",
      "    - extracellular region\n",
      "    - laminin-2 complex\n",
      "    - extracellular space\n",
      "    - cell cortex\n",
      "    - perinuclear region of cytoplasm\n",
      "    - collagen-containing extracellular matrix\n",
      "  MF:\n",
      "    - extracellular matrix structural constituent\n",
      "    - heparin binding\n",
      "UniProt: {'Swiss-Prot': 'Q13753'}\n",
      "\n",
      "Gene: CPEB1\n",
      "Symbol: CPEB1\n",
      "Name: cytoplasmic polyadenylation element binding protein 1\n",
      "Summary: This gene encodes a member of the cytoplasmic polyadenylation element binding protein family. This highly conserved protein binds to a specific RNA sequence, called the cytoplasmic polyadenylation element, found in the 3' untranslated region of some mRNAs. The encoded protein functions in both the cytoplasm and the nucleus. It is involved in the regulation of mRNA translation, as well as processing of the 3' untranslated region, and may play a role in cell proliferation and tumorigenesis. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Jan 2014].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - mRNA processing\n",
      "    - translation\n",
      "    - regulation of mRNA 3'-end processing\n",
      "    - cellular response to insulin stimulus\n",
      "    - cellular response to amino acid stimulus\n",
      "    - cellular response to hypoxia\n",
      "    - negative regulation of cytoplasmic translation\n",
      "    - negative regulation of cytoplasmic translation\n",
      "  CC:\n",
      "    - P-body\n",
      "    - nucleus\n",
      "    - nucleus\n",
      "    - nucleoplasm\n",
      "    - cytoplasm\n",
      "    - cytoplasm\n",
      "    - cytoplasm\n",
      "    - cytosol\n",
      "    - postsynaptic density\n",
      "    - membrane\n",
      "    - dendrite\n",
      "    - neuron projection\n",
      "    - synapse\n",
      "    - ribonucleoprotein complex\n",
      "  MF:\n",
      "    - mRNA regulatory element binding translation repressor activity\n",
      "    - mRNA regulatory element binding translation repressor activity\n",
      "    - mRNA 3'-UTR binding\n",
      "    - protein binding\n",
      "    - translation factor activity, RNA binding\n",
      "    - mRNA 3'-UTR AU-rich region binding\n",
      "    - ribosome binding\n",
      "    - metal ion binding\n",
      "UniProt: {'Swiss-Prot': 'Q9BZB8', 'TrEMBL': ['A0A024R214', 'B4DWY2', 'H3BPD6', 'A0A087WXG7', 'H3BNX1', 'A0A087WVR7', 'H3BTE4', 'A0A087X171', 'A0A0J9YX15', 'A0A087WUQ9', 'A0A0J9YXP0']}\n",
      "\n",
      "Gene: SLC22A5\n",
      "Symbol: SLC22A5\n",
      "Name: solute carrier family 22 member 5\n",
      "Summary: Polyspecific organic cation transporters in the liver, kidney, intestine, and other organs are critical for elimination of many endogenous small organic cations as well as a wide array of drugs and environmental toxins. The encoded protein is a plasma integral membrane protein which functions both as an organic cation transporter and as a sodium-dependent high affinity carnitine transporter. The encoded protein is involved in the active cellular uptake of carnitine. Mutations in this gene are the cause of systemic primary carnitine deficiency (CDSP), an autosomal recessive disorder manifested early in life by hypoketotic hypoglycemia and acute metabolic decompensation, and later in life by skeletal myopathy or cardiomyopathy. Alternative splicing of this gene results in multiple transcript variants. [provided by RefSeq, Apr 2015].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - sodium ion transport\n",
      "    - response to symbiotic bacterium\n",
      "    - quaternary ammonium group transport\n",
      "    - quaternary ammonium group transport\n",
      "    - quaternary ammonium group transport\n",
      "    - carnitine transport\n",
      "    - carnitine transport\n",
      "    - response to type II interferon\n",
      "    - response to tumor necrosis factor\n",
      "    - positive regulation of intestinal epithelial structure maintenance\n",
      "    - sodium-dependent organic cation transport\n",
      "    - transport across blood-brain barrier\n",
      "    - transport across blood-brain barrier\n",
      "    - (R)-carnitine transport\n",
      "    - (R)-carnitine transport\n",
      "    - (R)-carnitine transmembrane transport\n",
      "    - (R)-carnitine transmembrane transport\n",
      "    - (R)-carnitine transmembrane transport\n",
      "    - carnitine transmembrane transport\n",
      "    - xenobiotic detoxification by transmembrane export across the plasma membrane\n",
      "  CC:\n",
      "    - cytoplasm\n",
      "    - endoplasmic reticulum\n",
      "    - endoplasmic reticulum\n",
      "    - cytosol\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - basal plasma membrane\n",
      "    - apical plasma membrane\n",
      "    - brush border membrane\n",
      "    - brush border membrane\n",
      "    - extracellular exosome\n",
      "  MF:\n",
      "    - protein binding\n",
      "    - ATP binding\n",
      "    - amino-acid betaine transmembrane transporter activity\n",
      "    - carnitine transmembrane transporter activity\n",
      "    - carnitine transmembrane transporter activity\n",
      "    - carnitine transmembrane transporter activity\n",
      "    - carnitine transmembrane transporter activity\n",
      "    - symporter activity\n",
      "    - quaternary ammonium group transmembrane transporter activity\n",
      "    - quaternary ammonium group transmembrane transporter activity\n",
      "    - quaternary ammonium group transmembrane transporter activity\n",
      "    - PDZ domain binding\n",
      "    - xenobiotic transmembrane transporter activity\n",
      "    - (R)-carnitine transmembrane transporter activity\n",
      "    - (R)-carnitine transmembrane transporter activity\n",
      "    - (R)-carnitine transmembrane transporter activity\n",
      "    - (R)-carnitine transmembrane transporter activity\n",
      "UniProt: {'Swiss-Prot': 'O76082', 'TrEMBL': ['H7BZC0', 'A0A8I5KTN6', 'A0A8I5KW68', 'A0A8I5KXW0', 'F8WCC9', 'H7C1R8', 'A0A8I5QL37', 'H7BZF0', 'A0A8I5KUR7', 'A0A8I5KUV4', 'A0A8I5KYG6']}\n",
      "\n",
      "Gene: SLC22A4\n",
      "Symbol: SLC22A4\n",
      "Name: solute carrier family 22 member 4\n",
      "Summary: Polyspecific organic cation transporters in the liver, kidney, intestine, and other organs are critical for elimination of many endogenous small organic cations as well as a wide array of drugs and environmental toxins. The encoded protein is an organic cation transporter and plasma integral membrane protein containing eleven putative transmembrane domains as well as a nucleotide-binding site motif. Transport by this protein is at least partially ATP-dependent. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - triglyceride metabolic process\n",
      "    - sodium ion transport\n",
      "    - neurotransmitter transport\n",
      "    - carnitine metabolic process\n",
      "    - quaternary ammonium group transport\n",
      "    - quaternary ammonium group transport\n",
      "    - carnitine transport\n",
      "    - xenobiotic transport\n",
      "    - amino acid import across plasma membrane\n",
      "    - acetate ester transport\n",
      "    - carnitine transmembrane transport\n",
      "  CC:\n",
      "    - mitochondrion\n",
      "    - mitochondrion\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - basal plasma membrane\n",
      "    - apical plasma membrane\n",
      "    - mitochondrial membrane\n",
      "  MF:\n",
      "    - acetylcholine transmembrane transporter activity\n",
      "    - protein binding\n",
      "    - ATP binding\n",
      "    - secondary active organic cation transmembrane transporter activity\n",
      "    - amino acid transmembrane transporter activity\n",
      "    - amino-acid betaine transmembrane transporter activity\n",
      "    - carnitine transmembrane transporter activity\n",
      "    - symporter activity\n",
      "    - quaternary ammonium group transmembrane transporter activity\n",
      "    - quaternary ammonium group transmembrane transporter activity\n",
      "    - PDZ domain binding\n",
      "UniProt: {'Swiss-Prot': 'Q9H015'}\n",
      "\n",
      "Gene: SLCO1A2\n",
      "Symbol: SLCO1A2\n",
      "Name: solute carrier organic anion transporter family member 1A2\n",
      "Summary: This gene encodes a sodium-independent transporter which mediates cellular uptake of organic ions in the liver. Its substrates include bile acids, bromosulphophthalein, and some steroidal compounds. The protein is a member of the SLC21A family of solute carriers. Alternative splicing of this gene results in multiple transcript variants. [provided by RefSeq, Dec 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - xenobiotic metabolic process\n",
      "    - monoatomic ion transport\n",
      "    - organic cation transport\n",
      "    - organic anion transport\n",
      "    - bile acid and bile salt transport\n",
      "    - bile acid and bile salt transport\n",
      "    - sodium-independent organic anion transport\n",
      "    - sodium-independent organic anion transport\n",
      "    - transmembrane transport\n",
      "  CC:\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - basal plasma membrane\n",
      "    - basolateral plasma membrane\n",
      "    - apical plasma membrane\n",
      "  MF:\n",
      "    - organic anion transmembrane transporter activity\n",
      "    - organic cation transmembrane transporter activity\n",
      "    - bile acid transmembrane transporter activity\n",
      "    - bile acid transmembrane transporter activity\n",
      "    - sodium-independent organic anion transmembrane transporter activity\n",
      "    - transmembrane transporter activity\n",
      "UniProt: {'Swiss-Prot': 'P46721', 'TrEMBL': ['F5H060', 'B4DJE6', 'C9JG34', 'C9JSB7', 'C9JHU9', 'F5GXY6', 'C9JTF6', 'C9K059', 'C9JCA7', 'C9JUW6', 'C9IYW8']}\n",
      "\n",
      "Gene: SLC12A3\n",
      "Symbol: SLC12A3\n",
      "Name: solute carrier family 12 member 3\n",
      "Summary: This gene encodes a renal thiazide-sensitive sodium-chloride cotransporter that is important for electrolyte homeostasis. This cotransporter mediates sodium and chloride reabsorption in the distal convoluted tubule. Mutations in this gene cause Gitelman syndrome, a disease similar to Bartter's syndrome, that is characterized by hypokalemic alkalosis combined with hypomagnesemia, low urinary calcium, and increased renin activity associated with normal blood pressure. This cotransporter is the target for thiazide diuretics that are used for treating high blood pressure. Multiple transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - response to dietary excess\n",
      "    - monoatomic ion transport\n",
      "    - sodium ion transport\n",
      "    - cell volume homeostasis\n",
      "    - signal transduction\n",
      "    - sodium ion transmembrane transport\n",
      "    - sodium ion transmembrane transport\n",
      "    - chloride ion homeostasis\n",
      "    - potassium ion homeostasis\n",
      "    - sodium ion homeostasis\n",
      "    - renal sodium ion absorption\n",
      "    - chloride transmembrane transport\n",
      "    - response to aldosterone\n",
      "    - potassium ion import across plasma membrane\n",
      "  CC:\n",
      "    - cytosol\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - membrane\n",
      "    - apical plasma membrane\n",
      "    - apical plasma membrane\n",
      "    - apical plasma membrane\n",
      "    - extracellular exosome\n",
      "    - extracellular exosome\n",
      "  MF:\n",
      "    - protein binding\n",
      "    - ATP binding\n",
      "    - sodium:potassium:chloride symporter activity\n",
      "    - sodium:chloride symporter activity\n",
      "    - sodium:chloride symporter activity\n",
      "UniProt: {'Swiss-Prot': 'P55017', 'TrEMBL': 'J3QSS1'}\n",
      "\n",
      "Gene: CTSC\n",
      "Symbol: CTSC\n",
      "Name: cathepsin C\n",
      "Summary: This gene encodes a member of the peptidase C1 family and lysosomal cysteine proteinase that appears to be a central coordinator for activation of many serine proteinases in cells of the immune system. Alternative splicing results in multiple transcript variants, at least one of which encodes a preproprotein that is proteolytically processed to generate heavy and light chains that form a disulfide-linked dimer. A portion of the propeptide acts as an intramolecular chaperone for the folding and stabilization of the mature enzyme. This enzyme requires chloride ions for activity and can degrade glucagon. Defects in the encoded protein have been shown to be a cause of Papillon-Lefevre syndrome, an autosomal recessive disorder characterized by palmoplantar keratosis and periodontitis. [provided by RefSeq, Nov 2015].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - T cell mediated cytotoxicity\n",
      "    - proteolysis\n",
      "    - apoptotic process\n",
      "    - immune response\n",
      "    - negative regulation of myelination\n",
      "    - proteolysis involved in protein catabolic process\n",
      "    - positive regulation of proteolysis involved in protein catabolic process\n",
      "    - positive regulation of microglial cell activation\n",
      "    - positive regulation of apoptotic signaling pathway\n",
      "  CC:\n",
      "    - extracellular region\n",
      "    - extracellular space\n",
      "    - extracellular space\n",
      "    - nucleoplasm\n",
      "    - lysosome\n",
      "    - lysosome\n",
      "    - lysosome\n",
      "    - endoplasmic reticulum lumen\n",
      "    - centrosome\n",
      "    - membrane\n",
      "    - COPII-coated ER to Golgi transport vesicle\n",
      "    - endoplasmic reticulum-Golgi intermediate compartment membrane\n",
      "    - azurophil granule lumen\n",
      "    - intracellular membrane-bounded organelle\n",
      "    - collagen-containing extracellular matrix\n",
      "    - extracellular exosome\n",
      "  MF:\n",
      "    - cysteine-type endopeptidase activity\n",
      "    - serine-type endopeptidase activity\n",
      "    - protein binding\n",
      "    - cysteine-type peptidase activity\n",
      "    - dipeptidyl-peptidase activity\n",
      "    - peptidase activator activity involved in apoptotic process\n",
      "    - phosphatase binding\n",
      "    - chloride ion binding\n",
      "    - identical protein binding\n",
      "    - protein-folding chaperone binding\n",
      "UniProt: {'Swiss-Prot': 'P53634', 'TrEMBL': ['A0A7I2V3A6', 'A0A7I2V3A2', 'A0A7I2V466', 'A0A7I2V5C5', 'H0YCY8', 'A0A7I2V2Q8', 'A0A7I2YQT5', 'H0YDA2', 'A0A7I2V5J2', 'A0A7I2V2F3', 'A0A7I2V592', 'A0A7I2V2V5', 'A0A7I2V5L5', 'A0A7I2V4Z3', 'A0A7I2V6E3', 'A0A7I2V3M6', 'A0A7I2YQ96']}\n",
      "\n",
      "Gene: CSTA\n",
      "Symbol: CSTA\n",
      "Name: cystatin A\n",
      "Summary: The cystatin superfamily encompasses proteins that contain multiple cystatin-like sequences. Some of the members are active cysteine protease inhibitors, while others have lost or perhaps never acquired this inhibitory activity. There are three inhibitory families in the superfamily, including the type 1 cystatins (stefins), type 2 cystatins, and kininogens. This gene encodes a stefin that functions as a cysteine protease inhibitor, forming tight complexes with papain and the cathepsins B, H, and L. The protein is one of the precursor proteins of cornified cell envelope in keratinocytes and plays a role in epidermal development and maintenance. Stefins have been proposed as prognostic and diagnostic tools for cancer. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - negative regulation of peptidase activity\n",
      "    - peptide cross-linking\n",
      "    - keratinocyte differentiation\n",
      "    - negative regulation of proteolysis\n",
      "    - cell-cell adhesion\n",
      "  CC:\n",
      "    - cornified envelope\n",
      "    - cornified envelope\n",
      "    - extracellular space\n",
      "    - nucleoplasm\n",
      "    - cytoplasm\n",
      "    - cytosol\n",
      "    - cytosol\n",
      "    - cytosol\n",
      "    - peptidase inhibitor complex\n",
      "  MF:\n",
      "    - protease binding\n",
      "    - cysteine-type endopeptidase inhibitor activity\n",
      "    - cysteine-type endopeptidase inhibitor activity\n",
      "    - protein binding\n",
      "UniProt: {'Swiss-Prot': 'P01040', 'TrEMBL': 'C9J0E4'}\n",
      "\n",
      "Gene: CYP1A1\n",
      "Symbol: CYP1A1\n",
      "Name: cytochrome P450 family 1 subfamily A member 1\n",
      "Summary: This gene, CYP1A1, encodes a member of the cytochrome P450 superfamily of enzymes. The cytochrome P450 proteins are monooxygenases which catalyze many reactions involved in drug metabolism and synthesis of cholesterol, steroids and other lipids. This protein localizes to the endoplasmic reticulum and its expression is induced by some polycyclic aromatic hydrocarbons (PAHs), some of which are found in cigarette smoke. The enzyme's endogenous substrate is unknown; however, it is able to metabolize some PAHs to carcinogenic intermediates. The gene has been associated with lung cancer risk. A related family member, CYP1A2, is located approximately 25 kb away from CYP1A1 on chromosome 15. Alternative splicing results in multiple transcript variants encoding distinct isoforms. [provided by RefSeq, Jan 2016].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - response to hypoxia\n",
      "    - long-chain fatty acid metabolic process\n",
      "    - lipid hydroxylation\n",
      "    - fatty acid metabolic process\n",
      "    - steroid biosynthetic process\n",
      "    - porphyrin-containing compound metabolic process\n",
      "    - xenobiotic metabolic process\n",
      "    - steroid metabolic process\n",
      "    - estrogen metabolic process\n",
      "    - amine metabolic process\n",
      "    - response to nematode\n",
      "    - response to herbicide\n",
      "    - ethylene metabolic process\n",
      "    - coumarin metabolic process\n",
      "    - flavonoid metabolic process\n",
      "    - response to iron(III) ion\n",
      "    - insecticide metabolic process\n",
      "    - phenol-containing compound metabolic process\n",
      "    - dibenzo-p-dioxin catabolic process\n",
      "    - epoxygenase P450 pathway\n",
      "    - response to food\n",
      "    - response to lipopolysaccharide\n",
      "    - response to vitamin A\n",
      "    - response to genistein\n",
      "    - response to immobilization stress\n",
      "    - vitamin D metabolic process\n",
      "    - retinol metabolic process\n",
      "    - retinol metabolic process\n",
      "    - long-chain fatty acid biosynthetic process\n",
      "    - 9-cis-retinoic acid biosynthetic process\n",
      "    - camera-type eye development\n",
      "    - nitric oxide metabolic process\n",
      "    - response to arsenic-containing substance\n",
      "    - digestive tract development\n",
      "    - tissue remodeling\n",
      "    - hydrogen peroxide biosynthetic process\n",
      "    - response to hyperoxia\n",
      "    - maternal process involved in parturition\n",
      "    - hepatocyte differentiation\n",
      "    - cellular response to copper ion\n",
      "    - omega-hydroxylase P450 pathway\n",
      "    - positive regulation of G1/S transition of mitotic cell cycle\n",
      "    - response to diphenyl ether\n",
      "    - response to Aroclor 1254\n",
      "    - response to 2,3,7,8-tetrachlorodibenzodioxine\n",
      "    - response to 3-methylcholanthrene\n",
      "  CC:\n",
      "    - mitochondrion\n",
      "    - mitochondrial inner membrane\n",
      "    - endoplasmic reticulum membrane\n",
      "    - intracellular membrane-bounded organelle\n",
      "  MF:\n",
      "    - monooxygenase activity\n",
      "    - monooxygenase activity\n",
      "    - iron ion binding\n",
      "    - protein binding\n",
      "    - arachidonate monooxygenase activity\n",
      "    - oxidoreductase activity\n",
      "    - oxidoreductase activity, acting on diphenols and related substances as donors\n",
      "    - flavonoid 3'-monooxygenase activity\n",
      "    - oxygen binding\n",
      "    - enzyme binding\n",
      "    - heme binding\n",
      "    - Hsp70 protein binding\n",
      "    - demethylase activity\n",
      "    - Hsp90 protein binding\n",
      "    - vitamin D 24-hydroxylase activity\n",
      "    - estrogen 16-alpha-hydroxylase activity\n",
      "    - estrogen 2-hydroxylase activity\n",
      "    - long-chain fatty acid omega-hydroxylase activity\n",
      "    - hydroperoxy icosatetraenoate dehydratase activity\n",
      "    - long-chain fatty acid omega-1 hydroxylase activity\n",
      "UniProt: {'Swiss-Prot': 'P04798', 'TrEMBL': ['A0N0X8', 'E7EMT5', 'A4F3V8']}\n",
      "\n",
      "Gene: HLA-DRB5\n",
      "Symbol: HLA-DRB5\n",
      "Name: major histocompatibility complex, class II, DR beta 5\n",
      "Summary: HLA-DRB5 belongs to the HLA class II beta chain paralogues. This class II molecule is a heterodimer consisting of an alpha (DRA) and a beta (DRB) chain, both anchored in the membrane. It plays a central role in the immune system by presenting peptides derived from extracellular proteins. Class II molecules are expressed in antigen presenting cells. The beta chain is approximately 26-28 kDa and its gene contains 6 exons. Exon one encodes the leader peptide, exons 2 and 3 encode the two extracellular domains, exon 4 encodes the transmembrane domain and exon 5 encodes the cytoplasmic tail. Within the DR molecule the beta chain contains all the polymorphisms specifying the peptide binding specificities. Typing for these polymorphisms is routinely done for bone marrow and kidney transplantation. There are multiple pseudogenes of this gene. [provided by RefSeq, Feb 2020].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - adaptive immune response\n",
      "    - peptide antigen assembly with MHC class II protein complex\n",
      "    - antigen processing and presentation of exogenous peptide antigen via MHC class II\n",
      "    - positive regulation of immune response\n",
      "    - positive regulation of T cell activation\n",
      "  CC:\n",
      "    - Golgi membrane\n",
      "    - lysosomal membrane\n",
      "    - lysosomal membrane\n",
      "    - lysosomal membrane\n",
      "    - plasma membrane\n",
      "    - ER to Golgi transport vesicle membrane\n",
      "    - transport vesicle membrane\n",
      "    - endocytic vesicle membrane\n",
      "    - clathrin-coated endocytic vesicle membrane\n",
      "    - late endosome membrane\n",
      "    - late endosome membrane\n",
      "    - trans-Golgi network membrane\n",
      "    - MHC class II protein complex\n",
      "    - MHC class II protein complex\n",
      "    - extracellular exosome\n",
      "    - lumenal side of endoplasmic reticulum membrane\n",
      "  MF:\n",
      "    - protein binding\n",
      "    - MHC class II protein complex binding\n",
      "    - peptide antigen binding\n",
      "    - peptide antigen binding\n",
      "UniProt: {'Swiss-Prot': 'Q30154', 'TrEMBL': ['A0AAQ5BI59', 'A0A2Z4LKS3']}\n",
      "\n",
      "Gene: PTHLH\n",
      "Symbol: PTHLH\n",
      "Name: parathyroid hormone like hormone\n",
      "Summary: The protein encoded by this gene is a member of the parathyroid hormone family. This hormone, via its receptor, PTHR1, regulates endochondral bone development and epithelial-mesenchymal interactions during the formation of the mammary glands and teeth. It is responsible for most cases of humoral hypercalcemia of malignancy, and mutations in this gene are associated with brachydactyly type E2 (BDE2). Alternatively spliced transcript variants have been found for this gene. There is also evidence for alternative translation initiation from non-AUG (CUG and GUG) start sites, downstream of the initiator AUG codon, resulting in nuclear forms of this hormone. [provided by RefSeq, Nov 2013].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - skeletal system development\n",
      "    - osteoblast development\n",
      "    - adenylate cyclase-activating G protein-coupled receptor signaling pathway\n",
      "    - adenylate cyclase-activating G protein-coupled receptor signaling pathway\n",
      "    - cell-cell signaling\n",
      "    - female pregnancy\n",
      "    - positive regulation of cell population proliferation\n",
      "    - negative regulation of cell population proliferation\n",
      "    - epidermis development\n",
      "    - regulation of gene expression\n",
      "    - bone mineralization\n",
      "    - regulation of chondrocyte differentiation\n",
      "    - negative regulation of chondrocyte differentiation\n",
      "    - cAMP metabolic process\n",
      "    - negative regulation of chondrocyte development\n",
      "  CC:\n",
      "    - extracellular region\n",
      "    - extracellular space\n",
      "    - nucleoplasm\n",
      "    - cytoplasm\n",
      "    - Golgi apparatus\n",
      "    - cytosol\n",
      "  MF:\n",
      "    - hormone activity\n",
      "    - protein binding\n",
      "    - peptide hormone receptor binding\n",
      "    - peptide hormone receptor binding\n",
      "UniProt: {'Swiss-Prot': 'P12272', 'TrEMBL': ['F5H485', 'Q53XY9', 'F5GZD9']}\n",
      "\n",
      "Gene: DRD5\n",
      "Symbol: DRD5\n",
      "Name: dopamine receptor D5\n",
      "Summary: This gene encodes the D5 subtype of the dopamine receptor. The D5 subtype is a G-protein coupled receptor which stimulates adenylyl cyclase. This receptor is expressed in neurons in the limbic regions of the brain. It has a 10-fold higher affinity for dopamine than the D1 subtype. Pseudogenes related to this gene reside on chromosomes 1 and 2. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - synaptic transmission, dopaminergic\n",
      "    - response to amphetamine\n",
      "    - regulation of systemic arterial blood pressure by vasopressin\n",
      "    - norepinephrine-epinephrine vasoconstriction involved in regulation of systemic arterial blood pressure\n",
      "    - intracellular calcium ion homeostasis\n",
      "    - adenylate cyclase-activating G protein-coupled receptor signaling pathway\n",
      "    - activation of adenylate cyclase activity\n",
      "    - adenylate cyclase-activating dopamine receptor signaling pathway\n",
      "    - G protein-coupled dopamine receptor signaling pathway\n",
      "    - chemical synaptic transmission\n",
      "    - associative learning\n",
      "    - transmission of nerve impulse\n",
      "    - negative regulation of NAD(P)H oxidase activity\n",
      "    - wound healing\n",
      "    - response to cocaine\n",
      "    - positive regulation of MAPK cascade\n",
      "    - positive regulation of adenylate cyclase activity\n",
      "    - negative regulation of blood pressure\n",
      "    - regulation of female receptivity\n",
      "    - sensitization\n",
      "    - phospholipase C-activating dopamine receptor signaling pathway\n",
      "    - long-term synaptic depression\n",
      "    - cellular response to catecholamine stimulus\n",
      "    - adenylate cyclase-activating adrenergic receptor signaling pathway\n",
      "    - reactive oxygen species metabolic process\n",
      "  CC:\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - cilium\n",
      "    - brush border membrane\n",
      "    - synapse\n",
      "    - ciliary membrane\n",
      "    - non-motile cilium\n",
      "  MF:\n",
      "    - dopamine neurotransmitter receptor activity, coupled via Gs\n",
      "    - dopamine neurotransmitter receptor activity, coupled via Gs\n",
      "    - G protein-coupled receptor activity\n",
      "    - dopamine neurotransmitter receptor activity\n",
      "    - protein binding\n",
      "    - dopamine binding\n",
      "UniProt: {'Swiss-Prot': 'P21918'}\n",
      "\n",
      "Gene: MADCAM1\n",
      "Symbol: MADCAM1\n",
      "Name: mucosal vascular addressin cell adhesion molecule 1\n",
      "Summary: The protein encoded by this gene is an endothelial cell adhesion molecule that interacts preferentially with the leukocyte beta7 integrin LPAM-1 (alpha4beta7), L-selectin, and VLA-4 (alpha4beta1) on myeloid cells to direct leukocytes into mucosal and inflamed tissues. It is a member of the immunoglobulin family and is similar to ICAM1 and VCAM1. At least seven alternatively spliced transcripts encoding different protein isoforms have been found for this gene, but the full-length nature of some variants has not been determined. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - positive regulation of leukocyte migration\n",
      "    - immune response\n",
      "    - cell adhesion\n",
      "    - cell-matrix adhesion\n",
      "    - signal transduction\n",
      "    - integrin-mediated signaling pathway\n",
      "    - integrin-mediated signaling pathway\n",
      "    - heterotypic cell-cell adhesion\n",
      "    - heterotypic cell-cell adhesion\n",
      "    - receptor clustering\n",
      "    - leukocyte tethering or rolling\n",
      "    - leukocyte tethering or rolling\n",
      "    - positive regulation of lymphocyte migration\n",
      "  CC:\n",
      "    - plasma membrane\n",
      "    - membrane\n",
      "  MF:\n",
      "    - integrin binding involved in cell-matrix adhesion\n",
      "    - integrin binding involved in cell-matrix adhesion\n",
      "UniProt: {'Swiss-Prot': 'Q13477', 'TrEMBL': 'U3KQ59'}\n",
      "\n",
      "Gene: TRPV1\n",
      "Symbol: TRPV1\n",
      "Name: transient receptor potential cation channel subfamily V member 1\n",
      "Summary: Capsaicin, the main pungent ingredient in hot chili peppers, elicits a sensation of burning pain by selectively activating sensory neurons that convey information about noxious stimuli to the central nervous system. The protein encoded by this gene is a receptor for capsaicin and is a non-selective cation channel that is structurally related to members of the TRP family of ion channels. This receptor is also activated by increases in temperature in the noxious range, suggesting that it functions as a transducer of painful thermal stimuli in vivo. Four transcript variants encoding the same protein, but with different 5' UTR sequence, have been described for this gene. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - negative regulation of transcription by RNA polymerase II\n",
      "    - fever generation\n",
      "    - microglial cell activation\n",
      "    - diet induced thermogenesis\n",
      "    - peptide secretion\n",
      "    - negative regulation of systemic arterial blood pressure\n",
      "    - lipid metabolic process\n",
      "    - cell surface receptor signaling pathway\n",
      "    - positive regulation of cytosolic calcium ion concentration\n",
      "    - chemosensory behavior\n",
      "    - negative regulation of heart rate\n",
      "    - negative regulation of mitochondrial membrane potential\n",
      "    - glutamate secretion\n",
      "    - calcium-mediated signaling\n",
      "    - cellular response to heat\n",
      "    - positive regulation of apoptotic process\n",
      "    - response to peptide hormone\n",
      "    - positive regulation of nitric oxide biosynthetic process\n",
      "    - behavioral response to pain\n",
      "    - sensory perception of taste\n",
      "    - sensory perception of mechanical stimulus\n",
      "    - thermoception\n",
      "    - detection of temperature stimulus involved in thermoception\n",
      "    - detection of temperature stimulus involved in sensory perception of pain\n",
      "    - detection of chemical stimulus involved in sensory perception of pain\n",
      "    - protein homotetramerization\n",
      "    - excitatory postsynaptic potential\n",
      "    - smooth muscle contraction involved in micturition\n",
      "    - calcium ion transmembrane transport\n",
      "    - calcium ion transmembrane transport\n",
      "    - cellular response to alkaloid\n",
      "    - cellular response to ATP\n",
      "    - cellular response to tumor necrosis factor\n",
      "    - cellular response to acidic pH\n",
      "    - cellular response to temperature stimulus\n",
      "    - negative regulation of establishment of blood-brain barrier\n",
      "    - calcium ion import across plasma membrane\n",
      "    - calcium ion import across plasma membrane\n",
      "    - response to capsazepine\n",
      "    - response to capsazepine\n",
      "    - cellular response to nerve growth factor stimulus\n",
      "  CC:\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - external side of plasma membrane\n",
      "    - membrane\n",
      "    - dendritic spine membrane\n",
      "    - neuronal cell body\n",
      "    - postsynaptic membrane\n",
      "    - GABA-ergic synapse\n",
      "  MF:\n",
      "    - transmembrane signaling receptor activity\n",
      "    - extracellular ligand-gated monoatomic ion channel activity\n",
      "    - excitatory extracellular ligand-gated monoatomic ion channel activity\n",
      "    - voltage-gated calcium channel activity\n",
      "    - calcium channel activity\n",
      "    - calcium channel activity\n",
      "    - protein binding\n",
      "    - calmodulin binding\n",
      "    - ATP binding\n",
      "    - intracellularly gated calcium channel activity\n",
      "    - chloride channel regulator activity\n",
      "    - phosphatidylinositol binding\n",
      "    - identical protein binding\n",
      "    - metal ion binding\n",
      "    - phosphoprotein binding\n",
      "    - temperature-gated ion channel activity\n",
      "UniProt: {'Swiss-Prot': 'Q8NER1', 'TrEMBL': ['I3L1R6', 'A0A3B3ISI9', 'E7ESJ2', 'E7EQ78']}\n",
      "\n",
      "Gene: CTSS\n",
      "Symbol: CTSS\n",
      "Name: cathepsin S\n",
      "Summary: The preproprotein encoded by this gene, a member of the peptidase C1 family, is a lysosomal cysteine proteinase that participates in the degradation of antigenic proteins to peptides for presentation on MHC class II molecules. The mature protein cleaves the invariant chain of MHC class II molecules in endolysosomal compartments and enables the formation of antigen-MHC class II complexes and the proper display of extracellular antigenic peptides by MHC-II. The mature protein also functions as an elastase over a broad pH range. When secreted from cells, this protein can remodel components of the extracellular matrix such as elastin, collagen, and fibronectin. This gene is implicated in the pathology of many inflammatory and autoimmune diseases and, given its elastase activity, plays a significant role in some pulmonary diseases. Alternatively spliced transcript variants encoding distinct isoforms have been found for this gene. [provided by RefSeq, May 2020].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - toll-like receptor signaling pathway\n",
      "    - adaptive immune response\n",
      "    - regulation of antigen processing and presentation\n",
      "    - proteolysis\n",
      "    - immune response\n",
      "    - response to acidic pH\n",
      "    - protein processing\n",
      "    - antigen processing and presentation\n",
      "    - antigen processing and presentation of exogenous peptide antigen via MHC class II\n",
      "    - antigen processing and presentation of exogenous peptide antigen via MHC class II\n",
      "    - extracellular matrix disassembly\n",
      "    - collagen catabolic process\n",
      "    - basement membrane disassembly\n",
      "    - antigen processing and presentation of peptide antigen\n",
      "    - proteolysis involved in protein catabolic process\n",
      "    - proteolysis involved in protein catabolic process\n",
      "    - cellular response to thyroid hormone stimulus\n",
      "    - positive regulation of cation channel activity\n",
      "  CC:\n",
      "    - extracellular region\n",
      "    - extracellular region\n",
      "    - extracellular space\n",
      "    - extracellular space\n",
      "    - extracellular space\n",
      "    - lysosome\n",
      "    - lysosome\n",
      "    - late endosome\n",
      "    - endolysosome lumen\n",
      "    - lysosomal lumen\n",
      "    - intracellular membrane-bounded organelle\n",
      "    - phagocytic vesicle\n",
      "    - collagen-containing extracellular matrix\n",
      "    - tertiary granule lumen\n",
      "    - ficolin-1-rich granule lumen\n",
      "  MF:\n",
      "    - fibronectin binding\n",
      "    - cysteine-type endopeptidase activity\n",
      "    - cysteine-type endopeptidase activity\n",
      "    - cysteine-type endopeptidase activity\n",
      "    - serine-type endopeptidase activity\n",
      "    - protein binding\n",
      "    - collagen binding\n",
      "    - cysteine-type peptidase activity\n",
      "    - laminin binding\n",
      "    - proteoglycan binding\n",
      "UniProt: {'Swiss-Prot': 'P25774', 'TrEMBL': ['U3KQE7', 'A0A7P0TB77', 'A0A7P0T8A1', 'A0A804DAZ0', 'A0A7P0T8H4', 'A0A7P0T844', 'A0A7P0T7Z6', 'A0A7P0TAQ0', 'A0A7P0T8U1', 'A0A7P0T904']}\n",
      "\n",
      "Gene: PTGIR\n",
      "Symbol: PTGIR\n",
      "Name: prostaglandin I2 receptor\n",
      "Summary: The protein encoded by this gene is a member of the G-protein coupled receptor family 1 and has been shown to be a receptor for prostacyclin. Prostacyclin, the major product of cyclooxygenase in macrovascular endothelium, elicits a potent vasodilation and inhibition of platelet aggregation through binding to this receptor. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - inflammatory response\n",
      "    - G protein-coupled receptor signaling pathway, coupled to cyclic nucleotide second messenger\n",
      "    - adenylate cyclase-activating G protein-coupled receptor signaling pathway\n",
      "    - adenylate cyclase-activating G protein-coupled receptor signaling pathway\n",
      "    - positive regulation of cytosolic calcium ion concentration\n",
      "    - cell-cell signaling\n",
      "    - negative regulation of platelet-derived growth factor receptor signaling pathway\n",
      "    - response to lipopolysaccharide\n",
      "    - negative regulation of smooth muscle cell proliferation\n",
      "    - negative regulation of smooth muscle cell proliferation\n",
      "  CC:\n",
      "    - cytosol\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "  MF:\n",
      "    - guanyl-nucleotide exchange factor activity\n",
      "    - prostacyclin receptor activity\n",
      "UniProt: {'Swiss-Prot': 'P43119'}\n",
      "\n",
      "Gene: CYP11B2\n",
      "Symbol: CYP11B2\n",
      "Name: cytochrome P450 family 11 subfamily B member 2\n",
      "Summary: This gene encodes a member of the cytochrome P450 superfamily of enzymes. The cytochrome P450 proteins are monooxygenases which catalyze many reactions involved in drug metabolism and synthesis of cholesterol, steroids and other lipids. This protein localizes to the mitochondrial inner membrane. The enzyme has steroid 18-hydroxylase activity to synthesize aldosterone and 18-oxocortisol as well as steroid 11 beta-hydroxylase activity. Mutations in this gene cause corticosterone methyl oxidase deficiency. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - regulation of blood volume by renal aldosterone\n",
      "    - renal water homeostasis\n",
      "    - C21-steroid hormone biosynthetic process\n",
      "    - glucocorticoid biosynthetic process\n",
      "    - mineralocorticoid biosynthetic process\n",
      "    - cholesterol metabolic process\n",
      "    - sterol metabolic process\n",
      "    - aldosterone biosynthetic process\n",
      "    - aldosterone biosynthetic process\n",
      "    - aldosterone biosynthetic process\n",
      "    - cellular response to hormone stimulus\n",
      "    - cortisol metabolic process\n",
      "    - cortisol biosynthetic process\n",
      "    - cortisol biosynthetic process\n",
      "    - cellular response to potassium ion\n",
      "    - potassium ion homeostasis\n",
      "    - sodium ion homeostasis\n",
      "    - cellular response to peptide hormone stimulus\n",
      "  CC:\n",
      "    - mitochondrion\n",
      "    - mitochondrion\n",
      "    - mitochondrial inner membrane\n",
      "    - mitochondrial inner membrane\n",
      "    - mitochondrial inner membrane\n",
      "  MF:\n",
      "    - steroid 11-beta-monooxygenase activity\n",
      "    - steroid 11-beta-monooxygenase activity\n",
      "    - steroid 11-beta-monooxygenase activity\n",
      "    - iron ion binding\n",
      "    - steroid hydroxylase activity\n",
      "    - heme binding\n",
      "    - heme binding\n",
      "    - corticosterone 18-monooxygenase activity\n",
      "UniProt: {'Swiss-Prot': 'P19099'}\n",
      "\n",
      "Gene: MFAP2\n",
      "Symbol: MFAP2\n",
      "Name: microfibril associated protein 2\n",
      "Summary: Microfibrillar-associated protein 2 is a major antigen of elastin-associated microfibrils and a candidate for involvement in the etiology of inherited connective tissue diseases. Four transcript variants encoding two different isoforms have been found for this gene. [provided by RefSeq, Sep 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - platelet formation\n",
      "    - embryonic eye morphogenesis\n",
      "    - embryonic eye morphogenesis\n",
      "    - post-embryonic eye morphogenesis\n",
      "    - positive regulation of cold-induced thermogenesis\n",
      "  CC:\n",
      "    - microfibril\n",
      "    - microfibril\n",
      "    - extracellular region\n",
      "    - collagen-containing extracellular matrix\n",
      "  MF:\n",
      "    - fibronectin binding\n",
      "    - extracellular matrix structural constituent\n",
      "    - protein binding\n",
      "    - fibrinogen binding\n",
      "UniProt: {'Swiss-Prot': 'P55001'}\n",
      "\n",
      "Gene: CHRM1\n",
      "Symbol: CHRM1\n",
      "Name: cholinergic receptor muscarinic 1\n",
      "Summary: The muscarinic cholinergic receptors belong to a larger family of G protein-coupled receptors. The functional diversity of these receptors is defined by the binding of acetylcholine and includes cellular responses such as adenylate cyclase inhibition, phosphoinositide degeneration, and potassium channel mediation. Muscarinic receptors influence many effects of acetylcholine in the central and peripheral nervous system. The muscarinic cholinergic receptor 1 is involved in mediation of vagally-induced bronchoconstriction and in the acid secretion of the gastrointestinal tract. The gene encoding this receptor is localized to 11q13. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - signal transduction\n",
      "    - G protein-coupled receptor signaling pathway\n",
      "    - G protein-coupled receptor signaling pathway, coupled to cyclic nucleotide second messenger\n",
      "    - adenylate cyclase-inhibiting G protein-coupled acetylcholine receptor signaling pathway\n",
      "    - phospholipase C-activating G protein-coupled receptor signaling pathway\n",
      "    - phospholipase C-activating G protein-coupled acetylcholine receptor signaling pathway\n",
      "    - G protein-coupled acetylcholine receptor signaling pathway\n",
      "    - chemical synaptic transmission\n",
      "    - neuromuscular synaptic transmission\n",
      "    - nervous system development\n",
      "    - regulation of locomotion\n",
      "    - positive regulation of monoatomic ion transport\n",
      "    - saliva secretion\n",
      "    - cognition\n",
      "    - regulation of postsynaptic membrane potential\n",
      "    - regulation of glial cell proliferation\n",
      "    - positive regulation of intracellular protein transport\n",
      "    - postsynaptic modulation of chemical synaptic transmission\n",
      "  CC:\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - membrane\n",
      "    - dendrite\n",
      "    - presynaptic membrane\n",
      "    - axon terminus\n",
      "    - synapse\n",
      "    - Schaffer collateral - CA1 synapse\n",
      "    - postsynaptic density membrane\n",
      "    - glutamatergic synapse\n",
      "    - cholinergic synapse\n",
      "  MF:\n",
      "    - phosphatidylinositol phospholipase C activity\n",
      "    - protein binding\n",
      "    - G protein-coupled acetylcholine receptor activity\n",
      "UniProt: {'Swiss-Prot': 'P11229', 'TrEMBL': ['F5GZF8', 'Q53XZ3']}\n",
      "\n",
      "Gene: MAG\n",
      "Symbol: MAG\n",
      "Name: myelin associated glycoprotein\n",
      "Summary: The protein encoded by this gene is a type I membrane protein and member of the immunoglobulin superfamily. It is thought to be involved in the process of myelination. It is a lectin that binds to sialylated glycoconjugates and mediates certain myelin-neuron cell-cell interactions. Three alternatively spliced transcripts encoding different isoforms have been described for this gene. [provided by RefSeq, Nov 2010].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - cell adhesion\n",
      "    - cell adhesion\n",
      "    - negative regulation of neuron projection development\n",
      "    - transmission of nerve impulse\n",
      "    - substantia nigra development\n",
      "    - negative regulation of axon extension\n",
      "    - axon regeneration\n",
      "    - positive regulation of myelination\n",
      "    - central nervous system myelin formation\n",
      "    - negative regulation of neuron apoptotic process\n",
      "    - negative regulation of neuron differentiation\n",
      "    - positive regulation of astrocyte differentiation\n",
      "    - cellular response to mechanical stimulus\n",
      "    - cell-cell adhesion via plasma-membrane adhesion molecules\n",
      "  CC:\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - paranode region of axon\n",
      "    - myelin sheath adaxonal region\n",
      "    - myelin sheath\n",
      "    - compact myelin\n",
      "    - Schmidt-Lanterman incisure\n",
      "    - membrane raft\n",
      "    - mesaxon\n",
      "  MF:\n",
      "    - signaling receptor binding\n",
      "    - protein kinase binding\n",
      "    - carbohydrate binding\n",
      "    - sialic acid binding\n",
      "    - sialic acid binding\n",
      "    - protein homodimerization activity\n",
      "    - ganglioside GT1b binding\n",
      "UniProt: {'Swiss-Prot': 'P20916', 'TrEMBL': ['M0QZU4', 'Q53ES7', 'M0R3B9', 'M0R3I4']}\n",
      "\n",
      "Gene: CFD\n",
      "Symbol: CFD\n",
      "Name: complement factor D\n",
      "Summary: This gene encodes a member of the S1, or chymotrypsin, family of serine peptidases. This protease catalyzes the cleavage of factor B, the rate-limiting step of the alternative pathway of complement activation. This protein also functions as an adipokine, a cell signaling protein secreted by adipocytes, which regulates insulin secretion in mice. Mutations in this gene underlie complement factor D deficiency, which is associated with recurrent bacterial meningitis infections in human patients. Alternative splicing of this gene results in multiple transcript variants. At least one of these variants encodes a preproprotein that is proteolytically processed to generate the mature protease. [provided by RefSeq, Nov 2015].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - proteolysis\n",
      "    - complement activation\n",
      "    - complement activation, alternative pathway\n",
      "    - response to bacterium\n",
      "    - protein maturation\n",
      "  CC:\n",
      "    - extracellular region\n",
      "    - extracellular space\n",
      "    - platelet alpha granule lumen\n",
      "    - secretory granule lumen\n",
      "    - extracellular exosome\n",
      "    - ficolin-1-rich granule lumen\n",
      "  MF:\n",
      "    - serine-type endopeptidase activity\n",
      "    - serine-type endopeptidase activity\n",
      "    - serine-type peptidase activity\n",
      "UniProt: {'Swiss-Prot': 'P00746', 'TrEMBL': ['A6XNE2', 'K7ERG9', 'A0A8Q3WKW0', 'A0A8Q3WKV6', 'A0A8Q3WLE6']}\n",
      "\n",
      "Gene: SCN10A\n",
      "Symbol: SCN10A\n",
      "Name: sodium voltage-gated channel alpha subunit 10\n",
      "Summary: The protein encoded by this gene is a tetrodotoxin-resistant voltage-gated sodium channel alpha subunit. The properties of the channel formed by the encoded transmembrane protein can be altered by interaction with different beta subunits. This protein may be involved in the onset of pain associated with peripheral neuropathy. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Jun 2014].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - regulation of heart rate\n",
      "    - sensory perception\n",
      "    - regulation of monoatomic ion transmembrane transport\n",
      "    - sodium ion transmembrane transport\n",
      "    - sodium ion transmembrane transport\n",
      "    - odontogenesis of dentin-containing tooth\n",
      "    - regulation of cardiac muscle contraction\n",
      "    - regulation of atrial cardiac muscle cell membrane depolarization\n",
      "    - cardiac muscle cell action potential involved in contraction\n",
      "    - membrane depolarization during action potential\n",
      "    - AV node cell action potential\n",
      "    - bundle of His cell action potential\n",
      "    - regulation of presynaptic membrane potential\n",
      "  CC:\n",
      "    - voltage-gated sodium channel complex\n",
      "    - voltage-gated sodium channel complex\n",
      "    - plasma membrane\n",
      "    - axon\n",
      "    - presynaptic membrane\n",
      "    - extracellular exosome\n",
      "    - clathrin complex\n",
      "    - glutamatergic synapse\n",
      "  MF:\n",
      "    - voltage-gated sodium channel activity\n",
      "    - voltage-gated sodium channel activity\n",
      "    - voltage-gated sodium channel activity\n",
      "    - transmembrane transporter binding\n",
      "    - voltage-gated monoatomic ion channel activity involved in regulation of presynaptic membrane potential\n",
      "UniProt: {'Swiss-Prot': 'Q9Y5Y9', 'TrEMBL': ['A0A2R8Y6J6', 'A0A590UJM0']}\n",
      "\n",
      "Gene: KLF7\n",
      "Symbol: KLF7\n",
      "Name: KLF transcription factor 7\n",
      "Summary: The protein encoded by this gene is a member of the Kruppel-like transcriptional regulator family. Members in this family regulate cell proliferation, differentiation and survival and contain three C2H2 zinc fingers at the C-terminus that mediate binding to GC-rich sites. This protein may contribute to the progression of type 2 diabetes by inhibiting insulin expression and secretion in pancreatic beta-cells and by deregulating adipocytokine secretion in adipocytes. A pseudogene of this gene is located on the long arm of chromosome 3. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Aug 2012].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - negative regulation of transcription by RNA polymerase II\n",
      "    - regulation of transcription by RNA polymerase II\n",
      "    - axonogenesis\n",
      "    - axon guidance\n",
      "    - glucose homeostasis\n",
      "    - regulation of epidermal cell differentiation\n",
      "    - positive regulation of transcription by RNA polymerase II\n",
      "    - dendrite morphogenesis\n",
      "    - negative regulation of insulin secretion involved in cellular response to glucose stimulus\n",
      "    - negative regulation of adipose tissue development\n",
      "  CC:\n",
      "    - chromatin\n",
      "    - nucleus\n",
      "    - nucleoplasm\n",
      "    - cytosol\n",
      "  MF:\n",
      "    - RNA polymerase II cis-regulatory region sequence-specific DNA binding\n",
      "    - RNA polymerase II cis-regulatory region sequence-specific DNA binding\n",
      "    - DNA-binding transcription factor activity, RNA polymerase II-specific\n",
      "    - DNA-binding transcription factor activity, RNA polymerase II-specific\n",
      "    - DNA-binding transcription factor activity, RNA polymerase II-specific\n",
      "    - DNA-binding transcription activator activity, RNA polymerase II-specific\n",
      "    - zinc ion binding\n",
      "UniProt: {'Swiss-Prot': 'O75840', 'TrEMBL': ['E7EQY2', 'A0A994J3U4', 'E7EUU0', 'C9JX20', 'A0A994J6Y4', 'A0A994J401', 'E9PHC9']}\n",
      "\n",
      "Gene: SLC6A9\n",
      "Symbol: SLC6A9\n",
      "Name: solute carrier family 6 member 9\n",
      "Summary: The amino acid glycine acts as an inhibitory neurotransmitter in the central nervous system. The protein encoded by this gene is one of two transporters that stop glycine signaling by removing it from the synaptic cleft. [provided by RefSeq, Jun 2016].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - neurotransmitter transport\n",
      "    - glycine transport\n",
      "    - glycine transport\n",
      "    - sodium ion transmembrane transport\n",
      "    - positive regulation of hemoglobin biosynthetic process\n",
      "    - regulation of synaptic transmission, glycinergic\n",
      "    - positive regulation of heme biosynthetic process\n",
      "    - transport across blood-brain barrier\n",
      "    - glycine import across plasma membrane\n",
      "    - glycine import across plasma membrane\n",
      "    - glycine import across plasma membrane\n",
      "  CC:\n",
      "    - endosome\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - basal plasma membrane\n",
      "    - postsynaptic density\n",
      "    - membrane\n",
      "    - basolateral plasma membrane\n",
      "    - apical plasma membrane\n",
      "    - lateral plasma membrane\n",
      "    - synaptic vesicle membrane\n",
      "    - dense core granule\n",
      "    - presynaptic membrane\n",
      "    - postsynaptic membrane\n",
      "    - hippocampal mossy fiber to CA3 synapse\n",
      "    - parallel fiber to Purkinje cell synapse\n",
      "  MF:\n",
      "    - amino acid:sodium symporter activity\n",
      "    - glycine transmembrane transporter activity\n",
      "    - glycine transmembrane transporter activity\n",
      "    - glycine:sodium symporter activity\n",
      "    - glycine:sodium symporter activity\n",
      "UniProt: {'Swiss-Prot': 'P48067', 'TrEMBL': ['B7Z3W8', 'B7Z589', 'B7Z3A9', 'E9PJ65', 'E9PLM5', 'J3KPA5']}\n",
      "\n",
      "Gene: KCNK18\n",
      "Symbol: KCNK18\n",
      "Name: potassium two pore domain channel subfamily K member 18\n",
      "Summary: Potassium channels play a role in many cellular processes including maintenance of the action potential, muscle contraction, hormone secretion, osmotic regulation, and ion flow. This gene encodes a member of the superfamily of potassium channel proteins containing two pore-forming P domains and the encoded protein functions as an outward rectifying potassium channel. A mutation in this gene has been found to be associated with migraine with aura.[provided by RefSeq, Jan 2011].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - potassium ion transport\n",
      "    - regulation of CD4-positive, CD25-positive, alpha-beta regulatory T cell differentiation\n",
      "    - cellular response to pH\n",
      "    - potassium ion transmembrane transport\n",
      "    - potassium ion export across plasma membrane\n",
      "  CC:\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "  MF:\n",
      "    - calcium-activated potassium channel activity\n",
      "    - outward rectifier potassium channel activity\n",
      "    - outward rectifier potassium channel activity\n",
      "    - potassium ion leak channel activity\n",
      "    - metal ion binding\n",
      "UniProt: {'Swiss-Prot': 'Q7Z418'}\n",
      "\n",
      "Gene: GSTM1\n",
      "Symbol: GSTM1\n",
      "Name: glutathione S-transferase mu 1\n",
      "Summary: Cytosolic and membrane-bound forms of glutathione S-transferase are encoded by two distinct supergene families. At present, eight distinct classes of the soluble cytoplasmic mammalian glutathione S-transferases have been identified: alpha, kappa, mu, omega, pi, sigma, theta and zeta. This gene encodes a glutathione S-transferase that belongs to the mu class. The mu class of enzymes functions in the detoxification of electrophilic compounds, including carcinogens, therapeutic drugs, environmental toxins and products of oxidative stress, by conjugation with glutathione. The genes encoding the mu class of enzymes are organized in a gene cluster on chromosome 1p13.3 and are known to be highly polymorphic. These genetic variations can change an individual's susceptibility to carcinogens and toxins as well as affect the toxicity and efficacy of certain drugs. Null mutations of this class mu gene have been linked with an increase in a number of cancers, likely due to an increased susceptibility to environmental toxins and carcinogens. Multiple protein isoforms are encoded by transcript variants of this gene. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - prostaglandin metabolic process\n",
      "    - glutathione metabolic process\n",
      "    - glutathione metabolic process\n",
      "    - nitrobenzene metabolic process\n",
      "    - xenobiotic catabolic process\n",
      "    - hepoxilin biosynthetic process\n",
      "    - cellular detoxification of nitrogen compound\n",
      "    - glutathione derivative biosynthetic process\n",
      "  CC:\n",
      "    - cytoplasm\n",
      "    - cytosol\n",
      "    - cytosol\n",
      "    - intercellular bridge\n",
      "  MF:\n",
      "    - glutathione transferase activity\n",
      "    - glutathione transferase activity\n",
      "    - enzyme binding\n",
      "    - protein homodimerization activity\n",
      "    - glutathione binding\n",
      "UniProt: {'Swiss-Prot': 'P09488', 'TrEMBL': ['X5D932', 'X5DR03', 'H3BQT3', 'B9ZVX7', 'E7EWW9', 'H3BRM6']}\n",
      "\n",
      "Gene: GRIK2\n",
      "Symbol: GRIK2\n",
      "Name: glutamate ionotropic receptor kainate type subunit 2\n",
      "Summary: Glutamate receptors are the predominant excitatory neurotransmitter receptors in the mammalian brain and are activated in a variety of normal neurophysiologic processes. This gene product belongs to the kainate family of glutamate receptors, which are composed of four subunits and function as ligand-activated ion channels. The subunit encoded by this gene is subject to RNA editing at multiple sites within the first and second transmembrane domains, which is thought to alter the structure and function of the receptor complex. Alternatively spliced transcript variants encoding different isoforms have also been described for this gene. Mutations in this gene have been associated with autosomal recessive cognitive disability. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - behavioral fear response\n",
      "    - intracellular calcium ion homeostasis\n",
      "    - glutamate receptor signaling pathway\n",
      "    - chemical synaptic transmission\n",
      "    - neuronal action potential\n",
      "    - calcium-mediated signaling\n",
      "    - monoatomic ion transmembrane transport\n",
      "    - ionotropic glutamate receptor signaling pathway\n",
      "    - synaptic transmission, glutamatergic\n",
      "    - negative regulation of neuron apoptotic process\n",
      "    - regulation of long-term neuronal synaptic plasticity\n",
      "    - regulation of short-term neuronal synaptic plasticity\n",
      "    - modulation of chemical synaptic transmission\n",
      "    - modulation of chemical synaptic transmission\n",
      "    - positive regulation of synaptic transmission\n",
      "    - neuron apoptotic process\n",
      "    - negative regulation of synaptic transmission, glutamatergic\n",
      "    - excitatory postsynaptic potential\n",
      "    - inhibitory postsynaptic potential\n",
      "    - presynaptic modulation of chemical synaptic transmission\n",
      "    - regulation of presynaptic membrane potential\n",
      "    - detection of cold stimulus involved in thermoception\n",
      "  CC:\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - kainate selective glutamate receptor complex\n",
      "    - presynaptic membrane\n",
      "    - neuronal cell body\n",
      "    - mossy fiber rosette\n",
      "    - hippocampal mossy fiber to CA3 synapse\n",
      "    - postsynaptic density membrane\n",
      "    - glutamatergic synapse\n",
      "  MF:\n",
      "    - glutamate-gated receptor activity\n",
      "    - glutamate-gated receptor activity\n",
      "    - extracellularly glutamate-gated ion channel activity\n",
      "    - kainate selective glutamate receptor activity\n",
      "    - kainate selective glutamate receptor activity\n",
      "    - glutamate-gated calcium ion channel activity\n",
      "    - ligand-gated monoatomic ion channel activity involved in regulation of presynaptic membrane potential\n",
      "    - transmitter-gated monoatomic ion channel activity involved in regulation of postsynaptic membrane potential\n",
      "UniProt: {'Swiss-Prot': 'Q13002', 'TrEMBL': ['A0A8D9PH75', 'Q8IY40', 'A8K0H7', 'G3XAD3', 'H7C2P5', 'A0A804HHV9', 'A0A804HI04', 'A0A0A0MRL4', 'A0A804HKU0', 'A0A804HL10', 'A0A804HKP5', 'H7BZX7', 'A0A804HIH6', 'A0A804HKS7']}\n",
      "\n",
      "Gene: PON1\n",
      "Symbol: PON1\n",
      "Name: paraoxonase 1\n",
      "Summary: This gene encodes a member of the paraoxonase family of enzymes and exhibits lactonase and ester hydrolase activity. Following synthesis in the kidney and liver, the enzyme is secreted into the circulation, where it binds to high density lipoprotein (HDL) particles and hydrolyzes thiolactones and xenobiotics, including paraoxon, a metabolite of the insecticide parathion. Polymorphisms in this gene may be associated with coronary artery disease and diabetic retinopathy. The gene is found in a cluster of three related paraoxonase genes on chromosome 7. [provided by RefSeq, Aug 2017].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - cholesterol metabolic process\n",
      "    - response to toxic substance\n",
      "    - response to toxic substance\n",
      "    - positive regulation of cholesterol efflux\n",
      "    - negative regulation of plasma lipoprotein oxidation\n",
      "    - carboxylic acid catabolic process\n",
      "    - organophosphate catabolic process\n",
      "    - phosphatidylcholine metabolic process\n",
      "    - lactone catabolic process\n",
      "  CC:\n",
      "    - extracellular region\n",
      "    - extracellular region\n",
      "    - extracellular space\n",
      "    - extracellular space\n",
      "    - endoplasmic reticulum membrane\n",
      "    - high-density lipoprotein particle\n",
      "    - high-density lipoprotein particle\n",
      "    - spherical high-density lipoprotein particle\n",
      "    - extracellular exosome\n",
      "    - blood microparticle\n",
      "  MF:\n",
      "    - aryldialkylphosphatase activity\n",
      "    - aryldialkylphosphatase activity\n",
      "    - arylesterase activity\n",
      "    - arylesterase activity\n",
      "    - arylesterase activity\n",
      "    - calcium ion binding\n",
      "    - phospholipid binding\n",
      "    - protein homodimerization activity\n",
      "    - acyl-L-homoserine-lactone lactonohydrolase activity\n",
      "UniProt: {'Swiss-Prot': 'P27169', 'TrEMBL': 'F8WF42'}\n",
      "\n",
      "Gene: KCNK3\n",
      "Symbol: KCNK3\n",
      "Name: potassium two pore domain channel subfamily K member 3\n",
      "Summary: This gene encodes a member of the superfamily of potassium channel proteins that contain two pore-forming P domains. The encoded protein is an outwardly rectifying channel that is sensitive to changes in extracellular pH and is inhibited by extracellular acidification. Also referred to as an acid-sensitive potassium channel, it is activated by the anesthetics halothane and isoflurane. Although three transcripts are detected in northern blots, there is currently no sequence available to confirm transcript variants for this gene. [provided by RefSeq, Aug 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - detection of hypoxic conditions in blood by carotid body chemoreceptor signaling\n",
      "    - potassium ion transport\n",
      "    - chemical synaptic transmission\n",
      "    - response to xenobiotic stimulus\n",
      "    - monoatomic ion transmembrane transport\n",
      "    - sodium ion transmembrane transport\n",
      "    - negative regulation of cytosolic calcium ion concentration\n",
      "    - regulation of resting membrane potential\n",
      "    - cellular response to zinc ion\n",
      "    - cellular response to hypoxia\n",
      "    - cellular response to acidic pH\n",
      "    - potassium ion transmembrane transport\n",
      "    - cochlea development\n",
      "    - regulation of action potential firing rate\n",
      "  CC:\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - synapse\n",
      "  MF:\n",
      "    - monoatomic ion channel activity\n",
      "    - open rectifier potassium channel activity\n",
      "    - potassium channel activity\n",
      "    - sodium channel activity\n",
      "    - protein binding\n",
      "    - outward rectifier potassium channel activity\n",
      "    - outward rectifier potassium channel activity\n",
      "    - potassium ion leak channel activity\n",
      "    - potassium ion leak channel activity\n",
      "    - S100 protein binding\n",
      "    - metal ion binding\n",
      "    - protein heterodimerization activity\n",
      "UniProt: {'Swiss-Prot': 'O14649'}\n",
      "\n",
      "Gene: CTF1\n",
      "Symbol: CTF1\n",
      "Name: cardiotrophin 1\n",
      "Summary: The protein encoded by this gene is a secreted cytokine that induces cardiac myocyte hypertrophy in vitro. It has been shown to bind and activate the ILST/gp130 receoptor. Two transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Dec 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - cell surface receptor signaling pathway via JAK-STAT\n",
      "    - cell-cell signaling\n",
      "    - nervous system development\n",
      "    - muscle organ development\n",
      "    - positive regulation of cell population proliferation\n",
      "    - neuron differentiation\n",
      "    - positive regulation of tyrosine phosphorylation of STAT protein\n",
      "    - neuron development\n",
      "  CC:\n",
      "    - extracellular region\n",
      "    - extracellular region\n",
      "    - extracellular space\n",
      "  MF:\n",
      "    - cytokine activity\n",
      "    - leukemia inhibitory factor receptor binding\n",
      "    - protein binding\n",
      "UniProt: {'Swiss-Prot': 'Q16619'}\n",
      "\n",
      "Gene: NUDT9\n",
      "Symbol: NUDT9\n",
      "Name: nudix hydrolase 9\n",
      "Summary: The protein encoded by this gene belongs to the Nudix hydrolase family. Nudix boxes are found in a family of diverse enzymes that catalyze the hydrolysis of nucleoside diphosphate derivatives. This enzyme is an ADP-ribose pyrophosphatase that catalyzes the hydrolysis of ADP-ribose to AMP and ribose-5-P. It requires divalent metal ions and an intact Nudix motif for enzymatic activity. Alternatively spliced transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Oct 2011].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - ADP catabolic process\n",
      "    - nucleobase-containing small molecule metabolic process\n",
      "  CC:\n",
      "    - mitochondrion\n",
      "    - mitochondrion\n",
      "    - mitochondrion\n",
      "    - mitochondrion\n",
      "    - mitochondrial matrix\n",
      "    - cytosol\n",
      "    - nuclear body\n",
      "    - cell junction\n",
      "    - nuclear membrane\n",
      "    - extracellular exosome\n",
      "  MF:\n",
      "    - protein binding\n",
      "    - ADP-sugar diphosphatase activity\n",
      "    - ADP-ribose diphosphatase activity\n",
      "    - ADP-ribose diphosphatase activity\n",
      "UniProt: {'Swiss-Prot': 'Q9BW91', 'TrEMBL': ['Q8NG26', 'Q96KB3', 'D6R8Z6', 'D6RAW2', 'H7C386']}\n",
      "\n",
      "Gene: CHST9\n",
      "Symbol: CHST9\n",
      "Name: carbohydrate sulfotransferase 9\n",
      "Summary: The protein encoded by this gene belongs to the sulfotransferase 2 family. It is localized to the golgi membrane, and catalyzes the transfer of sulfate to position 4 of non-reducing N-acetylgalactosamine (GalNAc) residues in both N-glycans and O-glycans. Sulfate groups on carbohydrates confer highly specific functions to glycoproteins, glycolipids, and proteoglycans, and are critical for cell-cell interaction, signal transduction, and embryonic development. Alternatively spliced transcript variants have been described for this gene. [provided by RefSeq, Aug 2011].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - sulfur compound metabolic process\n",
      "    - carbohydrate biosynthetic process\n",
      "    - proteoglycan biosynthetic process\n",
      "    - proteoglycan biosynthetic process\n",
      "    - glycosaminoglycan metabolic process\n",
      "    - chondroitin sulfate biosynthetic process\n",
      "    - hormone biosynthetic process\n",
      "  CC:\n",
      "    - Golgi membrane\n",
      "    - extracellular region\n",
      "    - membrane\n",
      "  MF:\n",
      "    - dermatan 4-sulfotransferase activity\n",
      "    - dermatan 4-sulfotransferase activity\n",
      "    - sulfotransferase activity\n",
      "    - chondroitin 4-sulfotransferase activity\n",
      "UniProt: {'Swiss-Prot': 'Q7L1S5'}\n",
      "\n",
      "Gene: TSLP\n",
      "Symbol: TSLP\n",
      "Name: thymic stromal lymphopoietin\n",
      "Summary: This gene encodes a hemopoietic cytokine proposed to signal through a heterodimeric receptor complex composed of the thymic stromal lymphopoietin receptor and the IL-7R alpha chain. It mainly impacts myeloid cells and induces the release of T cell-attracting chemokines from monocytes and enhances the maturation of CD11c(+) dendritic cells. The protein promotes T helper type 2 (TH2) cell responses that are associated with immunity in various inflammatory diseases, including asthma, allergic inflammation and chronic obstructive pulmonary disease. The protein is therefore considered a potential therapeutic target for the treatment of such diseases. In addition, the shorter (predominant) isoform is an antimicrobial protein, displaying antibacterial and antifungal activity against B. cereus, E. coli, E. faecalis, S. mitis, S. epidermidis, and C. albicans. Alternative splicing of this gene results in multiple transcript variants. [provided by RefSeq, Jul 2020].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - positive regulation of cytokine-mediated signaling pathway\n",
      "    - positive regulation of cytokine-mediated signaling pathway\n",
      "    - cell surface receptor signaling pathway via JAK-STAT\n",
      "    - positive regulation of cell population proliferation\n",
      "    - positive regulation of cell population proliferation\n",
      "    - cytokine-mediated signaling pathway\n",
      "    - positive regulation of chemokine production\n",
      "    - positive regulation of chemokine production\n",
      "    - positive regulation of interleukin-10 production\n",
      "    - positive regulation of interleukin-10 production\n",
      "    - positive regulation of interleukin-13 production\n",
      "    - positive regulation of interleukin-13 production\n",
      "    - positive regulation of interleukin-5 production\n",
      "    - positive regulation of interleukin-5 production\n",
      "    - positive regulation of interleukin-6 production\n",
      "    - positive regulation of interleukin-6 production\n",
      "    - positive regulation of mast cell activation\n",
      "    - interleukin-7-mediated signaling pathway\n",
      "    - negative regulation of apoptotic process\n",
      "    - positive regulation of inflammatory response\n",
      "    - positive regulation of inflammatory response\n",
      "    - defense response to Gram-negative bacterium\n",
      "    - defense response to fungus\n",
      "    - antimicrobial humoral immune response mediated by antimicrobial peptide\n",
      "    - positive regulation of chemokine (C-C motif) ligand 1 production\n",
      "    - positive regulation of granulocyte colony-stimulating factor production\n",
      "    - positive regulation of receptor signaling pathway via STAT\n",
      "    - positive regulation of receptor signaling pathway via STAT\n",
      "  CC:\n",
      "    - extracellular region\n",
      "    - extracellular region\n",
      "    - extracellular region\n",
      "    - extracellular region\n",
      "    - extracellular space\n",
      "  MF:\n",
      "    - cytokine activity\n",
      "    - cytokine activity\n",
      "    - cytokine activity\n",
      "    - interleukin-7 receptor binding\n",
      "    - interleukin-7 receptor binding\n",
      "UniProt: {'Swiss-Prot': 'Q969D9', 'TrEMBL': 'A0A0C4DG43'}\n",
      "\n",
      "Gene: SLC22A3\n",
      "Symbol: SLC22A3\n",
      "Name: solute carrier family 22 member 3\n",
      "Summary: Polyspecific organic cation transporters in the liver, kidney, intestine, and other organs are critical for elimination of many endogenous small organic cations as well as a wide array of drugs and environmental toxins. This gene is one of three similar cation transporter genes located in a cluster on chromosome 6. The encoded protein contains twelve putative transmembrane domains and is a plasma integral membrane protein. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - histamine metabolic process\n",
      "    - monoatomic ion transport\n",
      "    - neurotransmitter transport\n",
      "    - serotonin transport\n",
      "    - organic cation transport\n",
      "    - organic cation transport\n",
      "    - organic cation transport\n",
      "    - quaternary ammonium group transport\n",
      "    - quaternary ammonium group transport\n",
      "    - organic anion transport\n",
      "    - monocarboxylic acid transport\n",
      "    - monoamine transport\n",
      "    - spermidine transport\n",
      "    - dopamine transport\n",
      "    - norepinephrine transport\n",
      "    - regulation of appetite\n",
      "    - xenobiotic transport\n",
      "    - epinephrine transport\n",
      "    - histamine transport\n",
      "    - serotonin uptake\n",
      "    - histamine uptake\n",
      "    - histamine uptake\n",
      "    - norepinephrine uptake\n",
      "    - epinephrine uptake\n",
      "    - purine-containing compound transmembrane transport\n",
      "    - dopamine uptake\n",
      "    - transport across blood-brain barrier\n",
      "    - spermidine transmembrane transport\n",
      "    - cellular detoxification\n",
      "  CC:\n",
      "    - nuclear outer membrane\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - endomembrane system\n",
      "    - membrane\n",
      "    - basolateral plasma membrane\n",
      "    - apical plasma membrane\n",
      "    - mitochondrial membrane\n",
      "    - neuronal cell body\n",
      "    - presynapse\n",
      "  MF:\n",
      "    - neurotransmitter transmembrane transporter activity\n",
      "    - neurotransmitter transmembrane transporter activity\n",
      "    - protein binding\n",
      "    - monoamine transmembrane transporter activity\n",
      "    - monoamine transmembrane transporter activity\n",
      "    - organic anion transmembrane transporter activity\n",
      "    - organic cation transmembrane transporter activity\n",
      "    - organic cation transmembrane transporter activity\n",
      "    - organic cation transmembrane transporter activity\n",
      "    - organic cation transmembrane transporter activity\n",
      "    - spermidine transmembrane transporter activity\n",
      "    - quaternary ammonium group transmembrane transporter activity\n",
      "    - toxin transmembrane transporter activity\n",
      "UniProt: {'Swiss-Prot': 'O75751'}\n",
      "\n",
      "Gene: PDE4B\n",
      "Symbol: PDE4B\n",
      "Name: phosphodiesterase 4B\n",
      "Summary: This gene is a member of the type IV, cyclic AMP (cAMP)-specific, cyclic nucleotide phosphodiesterase (PDE) family. The encoded protein regulates the cellular concentrations of cyclic nucleotides and thereby play a role in signal transduction. Altered activity of this protein has been associated with schizophrenia and bipolar affective disorder. Alternative splicing and the use of alternative promoters results in multiple transcript variants encoding different isoforms. [provided by RefSeq, Jul 2014].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - neutrophil homeostasis\n",
      "    - cAMP catabolic process\n",
      "    - cAMP-mediated signaling\n",
      "    - neutrophil chemotaxis\n",
      "    - positive regulation of type II interferon production\n",
      "    - positive regulation of interleukin-2 production\n",
      "    - T cell receptor signaling pathway\n",
      "    - leukocyte migration\n",
      "    - cellular response to lipopolysaccharide\n",
      "    - cellular response to xenobiotic stimulus\n",
      "    - cellular response to epinephrine stimulus\n",
      "    - negative regulation of adenylate cyclase-activating adrenergic receptor signaling pathway\n",
      "    - regulation of cardiac muscle cell contraction\n",
      "    - negative regulation of relaxation of cardiac muscle\n",
      "    - regulation of calcium ion transmembrane transport via high voltage-gated calcium channel\n",
      "  CC:\n",
      "    - gamma-tubulin complex\n",
      "    - centrosome\n",
      "    - cytosol\n",
      "    - voltage-gated calcium channel complex\n",
      "    - synaptic vesicle\n",
      "    - postsynaptic density\n",
      "    - Z disc\n",
      "    - dendritic spine\n",
      "    - excitatory synapse\n",
      "  MF:\n",
      "    - 3',5'-cyclic-AMP phosphodiesterase activity\n",
      "    - 3',5'-cyclic-AMP phosphodiesterase activity\n",
      "    - 3',5'-cyclic-AMP phosphodiesterase activity\n",
      "    - 3',5'-cyclic-AMP phosphodiesterase activity\n",
      "    - 3',5'-cyclic-AMP phosphodiesterase activity\n",
      "    - calcium channel regulator activity\n",
      "    - protein binding\n",
      "    - cAMP binding\n",
      "    - gamma-tubulin binding\n",
      "    - transmembrane transporter binding\n",
      "    - metal ion binding\n",
      "    - 3',5'-cyclic-GMP phosphodiesterase activity\n",
      "UniProt: {'Swiss-Prot': 'Q07343', 'TrEMBL': ['Q59GM8', 'X5DR82', 'Q68CX5', 'X5DNX5', 'E9PJ03', 'E9PMG3', 'H0YCV7', 'E9PNB0', 'E9PR34']}\n",
      "\n",
      "Gene: KCNQ3\n",
      "Symbol: KCNQ3\n",
      "Name: potassium voltage-gated channel subfamily Q member 3\n",
      "Summary: This gene encodes a protein that functions in the regulation of neuronal excitability. The encoded protein forms an M-channel by associating with the products of the related KCNQ2 or KCNQ5 genes, which both encode integral membrane proteins. M-channel currents are inhibited by M1 muscarinic acetylcholine receptors and are activated by retigabine, a novel anti-convulsant drug. Defects in this gene are a cause of benign familial neonatal convulsions type 2 (BFNC2), also known as epilepsy, benign neonatal type 2 (EBN2). Alternative splicing of this gene results in multiple transcript variants. [provided by RefSeq, May 2014].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - protein targeting\n",
      "    - protein import into nucleus\n",
      "    - exocytosis\n",
      "    - endocytosis\n",
      "    - chemical synaptic transmission\n",
      "    - sensory perception of sound\n",
      "    - gene expression\n",
      "    - response to auditory stimulus\n",
      "    - neuron remodeling\n",
      "    - neuronal action potential\n",
      "    - nerve development\n",
      "    - psychomotor behavior\n",
      "    - regulation of synaptic plasticity\n",
      "    - neuron apoptotic process\n",
      "    - mitochondrial depolarization\n",
      "    - membrane hyperpolarization\n",
      "    - cellular response to calcium ion\n",
      "    - cellular response to xenobiotic stimulus\n",
      "    - potassium ion transmembrane transport\n",
      "    - potassium ion transmembrane transport\n",
      "    - apoptosome assembly\n",
      "    - excitatory chemical synaptic transmission\n",
      "    - inhibitory chemical synaptic transmission\n",
      "    - action potential initiation\n",
      "    - regulation of action potential firing threshold\n",
      "    - substantia propria of cornea development\n",
      "  CC:\n",
      "    - mitochondrion\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - voltage-gated potassium channel complex\n",
      "    - voltage-gated potassium channel complex\n",
      "    - cell surface\n",
      "    - node of Ranvier\n",
      "    - axon initial segment\n",
      "    - synapse\n",
      "  MF:\n",
      "    - voltage-gated potassium channel activity\n",
      "    - voltage-gated potassium channel activity\n",
      "    - protein binding\n",
      "    - calmodulin binding\n",
      "    - voltage-gated monoatomic cation channel activity\n",
      "UniProt: {'Swiss-Prot': 'O43525', 'TrEMBL': ['A0A1W2PRN8', 'E7ET42', 'A0A1W2PNZ2', 'A0A1W2PQ71']}\n",
      "\n",
      "Gene: FFAR1\n",
      "Symbol: FFAR1\n",
      "Name: free fatty acid receptor 1\n",
      "Summary: This gene encodes a member of the GP40 family of G protein-coupled receptors that are clustered together on chromosome 19. The encoded protein is a receptor for medium and long chain free fatty acids and may be involved in the metabolic regulation of insulin secretion. Polymorphisms in this gene may be associated with type 2 diabetes. [provided by RefSeq, Apr 2009].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - G protein-coupled receptor signaling pathway\n",
      "    - phospholipase C-activating G protein-coupled receptor signaling pathway\n",
      "    - positive regulation of cytosolic calcium ion concentration\n",
      "    - positive regulation of cytosolic calcium ion concentration\n",
      "    - insulin secretion\n",
      "    - positive regulation of insulin secretion\n",
      "    - negative regulation of interleukin-1 beta production\n",
      "    - glucose homeostasis\n",
      "    - positive regulation of calcium ion transport\n",
      "    - response to fatty acid\n",
      "    - response to fatty acid\n",
      "    - ion channel modulating, G protein-coupled receptor signaling pathway\n",
      "    - ligand-gated ion channel signaling pathway\n",
      "  CC:\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "  MF:\n",
      "    - G protein-coupled receptor activity\n",
      "    - lipid binding\n",
      "    - bioactive lipid receptor activity\n",
      "    - bioactive lipid receptor activity\n",
      "UniProt: {'Swiss-Prot': 'O14842'}\n",
      "\n",
      "Gene: CYP11B1\n",
      "Symbol: CYP11B1\n",
      "Name: cytochrome P450 family 11 subfamily B member 1\n",
      "Summary: This gene encodes a member of the cytochrome P450 superfamily of enzymes. The cytochrome P450 proteins are monooxygenases which catalyze many reactions involved in drug metabolism and synthesis of cholesterol, steroids and other lipids. This protein localizes to the mitochondrial inner membrane and is involved in the conversion of progesterone to cortisol in the adrenal cortex. Mutations in this gene cause congenital adrenal hyperplasia due to 11-beta-hydroxylase deficiency. Transcript variants encoding different isoforms have been noted for this gene. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - C21-steroid hormone biosynthetic process\n",
      "    - glucocorticoid biosynthetic process\n",
      "    - glucocorticoid biosynthetic process\n",
      "    - immune response\n",
      "    - cholesterol metabolic process\n",
      "    - regulation of blood pressure\n",
      "    - sterol metabolic process\n",
      "    - aldosterone biosynthetic process\n",
      "    - aldosterone biosynthetic process\n",
      "    - aldosterone biosynthetic process\n",
      "    - cellular response to hormone stimulus\n",
      "    - cortisol metabolic process\n",
      "    - cortisol biosynthetic process\n",
      "    - cortisol biosynthetic process\n",
      "    - cellular response to potassium ion\n",
      "    - glucose homeostasis\n",
      "    - cellular response to peptide hormone stimulus\n",
      "  CC:\n",
      "    - mitochondrion\n",
      "    - mitochondrion\n",
      "    - mitochondrial inner membrane\n",
      "    - mitochondrial inner membrane\n",
      "    - mitochondrial inner membrane\n",
      "  MF:\n",
      "    - steroid 11-beta-monooxygenase activity\n",
      "    - steroid 11-beta-monooxygenase activity\n",
      "    - steroid 11-beta-monooxygenase activity\n",
      "    - steroid 11-beta-monooxygenase activity\n",
      "    - iron ion binding\n",
      "    - heme binding\n",
      "    - corticosterone 18-monooxygenase activity\n",
      "UniProt: {'Swiss-Prot': 'P15538', 'TrEMBL': ['Q4VAR0', 'Q8TDD0', 'H0YBR4']}\n",
      "\n",
      "Gene: UGCG\n",
      "Symbol: UGCG\n",
      "Name: UDP-glucose ceramide glucosyltransferase\n",
      "Summary: This gene encodes an enzyme that catalyzes the first glycosylation step in the biosynthesis of glycosphingolipids, which are membrane components containing lipid and sugar moieties. The product of this reaction is glucosylceramide, which is the core structure of many glycosphingolipids. [provided by RefSeq, Dec 2014].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - protein lipidation\n",
      "    - glucosylceramide biosynthetic process\n",
      "    - glucosylceramide biosynthetic process\n",
      "    - glycosphingolipid biosynthetic process\n",
      "    - epidermis development\n",
      "    - regulation of signal transduction\n",
      "    - cell differentiation\n",
      "    - keratinocyte differentiation\n",
      "    - leptin-mediated signaling pathway\n",
      "    - neuron development\n",
      "    - establishment of skin barrier\n",
      "    - intestinal lipid absorption\n",
      "    - cornified envelope assembly\n",
      "  CC:\n",
      "    - Golgi membrane\n",
      "    - Golgi membrane\n",
      "    - membrane\n",
      "  MF:\n",
      "    - protein binding\n",
      "    - ceramide glucosyltransferase activity\n",
      "    - ceramide glucosyltransferase activity\n",
      "    - ceramide glucosyltransferase activity\n",
      "UniProt: {'Swiss-Prot': 'Q16739'}\n",
      "\n",
      "Gene: EPCAM\n",
      "Symbol: EPCAM\n",
      "Name: epithelial cell adhesion molecule\n",
      "Summary: This gene encodes a carcinoma-associated antigen and is a member of a family that includes at least two type I membrane proteins. This antigen is expressed on most normal epithelial cells and gastrointestinal carcinomas and functions as a homotypic calcium-independent cell adhesion molecule. The antigen is being used as a target for immunotherapy treatment of human carcinomas. Mutations in this gene result in congenital tufting enteropathy. [provided by RefSeq, Dec 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - ureteric bud development\n",
      "    - positive regulation of cell population proliferation\n",
      "    - signal transduction involved in regulation of gene expression\n",
      "    - positive regulation of transcription by RNA polymerase II\n",
      "    - stem cell differentiation\n",
      "    - cell-cell adhesion\n",
      "    - negative regulation of cell-cell adhesion mediated by cadherin\n",
      "    - positive regulation of stem cell proliferation\n",
      "  CC:\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - bicellular tight junction\n",
      "    - bicellular tight junction\n",
      "    - cell surface\n",
      "    - basolateral plasma membrane\n",
      "    - apical plasma membrane\n",
      "    - lateral plasma membrane\n",
      "    - extracellular exosome\n",
      "  MF:\n",
      "    - protein binding\n",
      "    - protein-containing complex binding\n",
      "    - cadherin binding involved in cell-cell adhesion\n",
      "UniProt: {'Swiss-Prot': 'P16422', 'TrEMBL': ['C9JKY3', 'B5MCA4']}\n",
      "\n",
      "Gene: IFNG\n",
      "Symbol: IFNG\n",
      "Name: interferon gamma\n",
      "Summary: This gene encodes a soluble cytokine that is a member of the type II interferon class. The encoded protein is secreted by cells of both the innate and adaptive immune systems. The active protein is a homodimer that binds to the interferon gamma receptor which triggers a cellular response to viral and microbial infections. Mutations in this gene are associated with an increased susceptibility to viral, bacterial and parasitic infections and to several autoimmune diseases. [provided by RefSeq, Dec 2015].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - negative regulation of transcription by RNA polymerase II\n",
      "    - microglial cell activation\n",
      "    - microglial cell activation\n",
      "    - positive regulation of cytokine production\n",
      "    - adaptive immune response\n",
      "    - macrophage activation involved in immune response\n",
      "    - apoptotic process\n",
      "    - humoral immune response\n",
      "    - cell surface receptor signaling pathway\n",
      "    - cell surface receptor signaling pathway via JAK-STAT\n",
      "    - positive regulation of cell population proliferation\n",
      "    - response to virus\n",
      "    - positive regulation of autophagy\n",
      "    - positive regulation of gene expression\n",
      "    - positive regulation of gene expression\n",
      "    - positive regulation of gene expression\n",
      "    - positive regulation of gene expression\n",
      "    - positive regulation of gene expression\n",
      "    - positive regulation of gene expression\n",
      "    - negative regulation of gene expression\n",
      "    - negative regulation of gene expression\n",
      "    - positive regulation of epithelial cell migration\n",
      "    - regulation of metabolic process\n",
      "    - macrophage differentiation\n",
      "    - negative regulation of epithelial cell differentiation\n",
      "    - positive regulation of protein-containing complex assembly\n",
      "    - negative regulation of interleukin-17 production\n",
      "    - positive regulation of chemokine production\n",
      "    - positive regulation of interleukin-1 beta production\n",
      "    - positive regulation of interleukin-12 production\n",
      "    - positive regulation of interleukin-23 production\n",
      "    - positive regulation of interleukin-6 production\n",
      "    - positive regulation of tumor necrosis factor production\n",
      "    - positive regulation of CD4-positive, CD25-positive, alpha-beta regulatory T cell differentiation involved in immune response\n",
      "    - positive regulation of peptidyl-serine phosphorylation of STAT protein\n",
      "    - positive regulation of smooth muscle cell apoptotic process\n",
      "    - Fc-gamma receptor signaling pathway involved in phagocytosis\n",
      "    - type III interferon-mediated signaling pathway\n",
      "    - positive regulation of protein import into nucleus\n",
      "    - positive regulation of tyrosine phosphorylation of STAT protein\n",
      "    - positive regulation of MHC class II biosynthetic process\n",
      "    - positive regulation of nitric oxide biosynthetic process\n",
      "    - positive regulation of osteoclast differentiation\n",
      "    - positive regulation of glycolytic process\n",
      "    - negative regulation of DNA-templated transcription\n",
      "    - positive regulation of transcription by RNA polymerase II\n",
      "    - astrocyte activation\n",
      "    - astrocyte activation\n",
      "    - negative regulation of smooth muscle cell proliferation\n",
      "    - positive regulation of inflammatory response\n",
      "    - positive regulation of phagocytosis\n",
      "    - positive regulation of neurogenesis\n",
      "    - regulation of insulin secretion\n",
      "    - positive regulation of membrane protein ectodomain proteolysis\n",
      "    - defense response to virus\n",
      "    - type II interferon-mediated signaling pathway\n",
      "    - type II interferon-mediated signaling pathway\n",
      "    - type II interferon-mediated signaling pathway\n",
      "    - positive regulation of fructose 1,6-bisphosphate 1-phosphatase activity\n",
      "    - positive regulation of fructose 1,6-bisphosphate metabolic process\n",
      "    - positive regulation of vitamin D biosynthetic process\n",
      "    - positive regulation of calcidiol 1-monooxygenase activity\n",
      "    - positive regulation of protein serine/threonine kinase activity\n",
      "    - positive regulation of protein deacetylation\n",
      "    - extrinsic apoptotic signaling pathway\n",
      "    - cell surface receptor signaling pathway via STAT\n",
      "    - cellular response to virus\n",
      "    - neuroinflammatory response\n",
      "    - negative regulation of amyloid-beta clearance\n",
      "    - positive regulation of cellular respiration\n",
      "    - positive regulation of amyloid-beta formation\n",
      "    - positive regulation of amyloid-beta formation\n",
      "    - positive regulation of protein localization to plasma membrane\n",
      "    - positive regulation of exosomal secretion\n",
      "    - positive regulation of iron ion import across plasma membrane\n",
      "    - positive regulation of NMDA glutamate receptor activity\n",
      "    - positive regulation of core promoter binding\n",
      "    - positive regulation of tumor necrosis factor (ligand) superfamily member 11 production\n",
      "  CC:\n",
      "    - extracellular region\n",
      "    - extracellular region\n",
      "    - extracellular space\n",
      "    - extracellular space\n",
      "  MF:\n",
      "    - cytokine activity\n",
      "    - cytokine activity\n",
      "    - cytokine activity\n",
      "    - type II interferon receptor binding\n",
      "    - protein binding\n",
      "UniProt: {'Swiss-Prot': 'P01579', 'TrEMBL': 'A0A7R8GUN6'}\n",
      "\n",
      "Gene: LAMB3\n",
      "Symbol: LAMB3\n",
      "Name: laminin subunit beta 3\n",
      "Summary: The product encoded by this gene is a laminin that belongs to a family of basement membrane proteins. This protein is a beta subunit laminin, which together with an alpha and a gamma subunit, forms laminin-5. Mutations in this gene cause epidermolysis bullosa junctional Herlitz type, and generalized atrophic benign epidermolysis bullosa, diseases that are characterized by blistering of the skin. Multiple alternatively spliced transcript variants that encode the same protein have been found for this gene. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - cell adhesion\n",
      "    - epidermis development\n",
      "    - endodermal cell differentiation\n",
      "    - brown fat cell differentiation\n",
      "  CC:\n",
      "    - extracellular region\n",
      "    - laminin-5 complex\n",
      "    - collagen-containing extracellular matrix\n",
      "  MF:\n",
      "    - structural molecule activity\n",
      "    - extracellular matrix structural constituent\n",
      "    - protein binding\n",
      "    - protein-containing complex binding\n",
      "UniProt: {'Swiss-Prot': 'Q13751', 'TrEMBL': ['Q5THA1', 'A0A0S2Z3R6', 'X1WI29']}\n",
      "\n",
      "Gene: MMP24\n",
      "Symbol: MMP24\n",
      "Name: matrix metallopeptidase 24\n",
      "Summary: This gene encodes a member of the peptidase M10 family of matrix metalloproteinases (MMPs). Proteins in this family are involved in the breakdown of extracellular matrix in normal physiological processes, such as embryonic development, reproduction, and tissue remodeling, as well as in disease processes, such as arthritis and metastasis. The encoded preproprotein is proteolytically processed to generate the mature protease. Unlike most MMPs, which are secreted, this protease is a member of the membrane-type MMP (MT-MMP) subfamily, contains a transmembrane domain and is expressed at the cell surface. Substrates of this protease include the proteins cadherin 2 and matrix metallopeptidase 2 (also known as 72 kDa type IV collagenase). The gene has previously been referred to as MMP25 but has been renamed matrix metallopeptidase 24 (MMP24). [provided by RefSeq, Oct 2019].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - proteolysis\n",
      "    - glial cell differentiation\n",
      "    - extracellular matrix organization\n",
      "    - collagen catabolic process\n",
      "    - cell-cell adhesion mediated by cadherin\n",
      "    - detection of temperature stimulus involved in sensory perception of pain\n",
      "    - neuronal stem cell population maintenance\n",
      "    - cell-cell adhesion via plasma-membrane adhesion molecules\n",
      "  CC:\n",
      "    - extracellular space\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - extracellular matrix\n",
      "    - trans-Golgi network membrane\n",
      "    - extracellular exosome\n",
      "  MF:\n",
      "    - metalloendopeptidase activity\n",
      "    - metalloendopeptidase activity\n",
      "    - enzyme activator activity\n",
      "    - zinc ion binding\n",
      "    - cadherin binding\n",
      "UniProt: {'Swiss-Prot': 'Q9Y5R2', 'TrEMBL': 'Q86VV6'}\n",
      "\n",
      "Gene: GPD2\n",
      "Symbol: GPD2\n",
      "Name: glycerol-3-phosphate dehydrogenase 2\n",
      "Summary: The protein encoded by this gene localizes to the inner mitochondrial membrane and catalyzes the conversion of glycerol-3-phosphate to dihydroxyacetone phosphate, using FAD as a cofactor. Along with GDP1, the encoded protein constitutes the glycerol phosphate shuttle, which reoxidizes NADH formed during glycolysis. Two transcript variants encoding the same protein have been found for this gene.[provided by RefSeq, Jan 2010].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - glycerol-3-phosphate metabolic process\n",
      "    - glycerol-3-phosphate shuttle\n",
      "    - glycerol-3-phosphate shuttle\n",
      "    - glycerol catabolic process\n",
      "  CC:\n",
      "    - mitochondrion\n",
      "    - mitochondrion\n",
      "    - mitochondrion\n",
      "    - mitochondrial inner membrane\n",
      "  MF:\n",
      "    - glycerol-3-phosphate dehydrogenase (quinone) activity\n",
      "    - glycerol-3-phosphate dehydrogenase (quinone) activity\n",
      "    - glycerol-3-phosphate dehydrogenase (quinone) activity\n",
      "    - calcium ion binding\n",
      "UniProt: {'Swiss-Prot': 'P43304', 'TrEMBL': ['E9PDK4', 'F5GYK7', 'E7EM56']}\n",
      "\n",
      "Gene: GRPEL2\n",
      "Symbol: GRPEL2\n",
      "Name: GrpE like 2, mitochondrial\n",
      "Summary: Predicted to enable adenyl-nucleotide exchange factor activity and unfolded protein binding activity. Predicted to be involved in protein import into mitochondrial matrix. Located in mitochondrion. [provided by Alliance of Genome Resources, Apr 2022]\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - protein folding\n",
      "    - intracellular protein transport\n",
      "    - protein import into mitochondrial matrix\n",
      "  CC:\n",
      "    - PAM complex, Tim23 associated import motor\n",
      "    - mitochondrion\n",
      "    - mitochondrion\n",
      "    - mitochondrial inner membrane\n",
      "    - TIM23 mitochondrial import inner membrane translocase complex\n",
      "  MF:\n",
      "    - adenyl-nucleotide exchange factor activity\n",
      "    - protein binding\n",
      "    - protein homodimerization activity\n",
      "    - unfolded protein binding\n",
      "    - protein-folding chaperone binding\n",
      "UniProt: {'Swiss-Prot': 'Q8TAA5', 'TrEMBL': 'D6RGI6'}\n",
      "\n",
      "Gene: CST5\n",
      "Symbol: CST5\n",
      "Name: cystatin D\n",
      "Summary: The cystatin superfamily encompasses proteins that contain multiple cystatin-like sequences. Some of the members are active cysteine protease inhibitors, while others have lost or perhaps never acquired this inhibitory activity. There are three inhibitory families in the superfamily, including the type 1 cystatins (stefins), type 2 cystatins and the kininogens. The type 2 cystatin proteins are a class of cysteine proteinase inhibitors found in a variety of human fluids and secretions. The cystatin locus on chromosome 20 contains the majority of the type 2 cystatin genes and pseudogenes. This gene is located in the cystatin locus and encodes a protein found in saliva and tears. The encoded protein may play a protective role against proteinases present in the oral cavity. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  CC:\n",
      "    - extracellular region\n",
      "    - extracellular space\n",
      "    - cytoplasm\n",
      "    - vesicle\n",
      "    - extracellular exosome\n",
      "  MF:\n",
      "    - cysteine-type endopeptidase inhibitor activity\n",
      "    - cysteine-type endopeptidase inhibitor activity\n",
      "    - protein binding\n",
      "UniProt: {'Swiss-Prot': 'P28325'}\n",
      "\n",
      "Gene: GRM4\n",
      "Symbol: GRM4\n",
      "Name: glutamate metabotropic receptor 4\n",
      "Summary: L-glutamate is the major excitatory neurotransmitter in the central nervous system and activates both ionotropic and metabotropic glutamate receptors. Glutamatergic neurotransmission is involved in most aspects of normal brain function and can be perturbed in many neuropathologic conditions. The metabotropic glutamate receptors are a family of G protein-coupled receptors, that have been divided into 3 groups on the basis of sequence homology, putative signal transduction mechanisms, and pharmacologic properties. Group I includes GRM1 and GRM5 and these receptors have been shown to activate phospholipase C. Group II includes GRM2 and GRM3 while Group III includes GRM4, GRM6, GRM7 and GRM8. Group II and III receptors are linked to the inhibition of the cyclic AMP cascade but differ in their agonist selectivities. Several transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Feb 2012].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - adenylate cyclase-inhibiting G protein-coupled glutamate receptor signaling pathway\n",
      "    - G protein-coupled glutamate receptor signaling pathway\n",
      "    - chemical synaptic transmission\n",
      "    - neurotransmitter secretion\n",
      "    - positive regulation of MAPK cascade\n",
      "    - regulation of neuron apoptotic process\n",
      "    - regulation of synaptic transmission, glutamatergic\n",
      "  CC:\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - cytoplasmic vesicle\n",
      "    - presynapse\n",
      "  MF:\n",
      "    - adenylate cyclase inhibiting G protein-coupled glutamate receptor activity\n",
      "    - G protein-coupled receptor activity\n",
      "    - glutamate receptor activity\n",
      "UniProt: {'Swiss-Prot': 'Q14833', 'TrEMBL': ['A1L4F9', 'A8K0J8', 'B7ZLU9', 'V9GYR3', 'V9GYB0']}\n",
      "\n",
      "Gene: GRK1\n",
      "Symbol: GRK1\n",
      "Name: G protein-coupled receptor kinase 1\n",
      "Summary: This gene encodes a member of the guanine nucleotide-binding protein (G protein)-coupled receptor kinase subfamily of the Ser/Thr protein kinase family. The protein phosphorylates rhodopsin and initiates its deactivation. Defects in GRK1 are known to cause Oguchi disease 2 (also known as stationary night blindness Oguchi type-2). [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - visual perception\n",
      "    - regulation of G protein-coupled receptor signaling pathway\n",
      "    - regulation of signal transduction\n",
      "    - G protein-coupled opsin signaling pathway\n",
      "    - regulation of opsin-mediated signaling pathway\n",
      "    - regulation of opsin-mediated signaling pathway\n",
      "    - protein autophosphorylation\n",
      "  CC:\n",
      "    - cytoplasm\n",
      "    - photoreceptor disc membrane\n",
      "  MF:\n",
      "    - protein kinase activity\n",
      "    - ATP binding\n",
      "    - rhodopsin kinase activity\n",
      "    - rhodopsin kinase activity\n",
      "    - rhodopsin kinase activity\n",
      "UniProt: {'Swiss-Prot': 'Q15835'}\n",
      "\n",
      "Gene: NEU3\n",
      "Symbol: NEU3\n",
      "Name: neuraminidase 3\n",
      "Summary: This gene product belongs to a family of glycohydrolytic enzymes which remove sialic acid residues from glycoproteins and glycolipids. It is localized in the plasma membrane, and its activity is specific for gangliosides. It may play a role in modulating the ganglioside content of the lipid bilayer. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - carbohydrate metabolic process\n",
      "    - ganglioside catabolic process\n",
      "    - ganglioside catabolic process\n",
      "    - oligosaccharide catabolic process\n",
      "    - oligosaccharide catabolic process\n",
      "    - positive regulation of epidermal growth factor receptor signaling pathway\n",
      "    - glycosphingolipid catabolic process\n",
      "    - negative regulation of clathrin-dependent endocytosis\n",
      "  CC:\n",
      "    - cytoplasm\n",
      "    - lysosome\n",
      "    - lysosomal membrane\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - caveola\n",
      "    - membrane\n",
      "    - early endosome membrane\n",
      "    - recycling endosome membrane\n",
      "  MF:\n",
      "    - exo-alpha-sialidase activity\n",
      "    - exo-alpha-sialidase activity\n",
      "    - exo-alpha-sialidase activity\n",
      "    - exo-alpha-sialidase activity\n",
      "    - protein binding\n",
      "    - alpha-sialidase activity\n",
      "UniProt: {'Swiss-Prot': 'Q9UQ49', 'TrEMBL': ['E9PNK1', 'E9PMZ3', 'E9PI40', 'E9PR25']}\n",
      "\n",
      "Gene: ABCB7\n",
      "Symbol: ABCB7\n",
      "Name: ATP binding cassette subfamily B member 7\n",
      "Summary: The membrane-associated protein encoded by this gene is a member of the superfamily of ATP-binding cassette (ABC) transporters. ABC proteins transport various molecules across extra- and intra-cellular membranes. ABC genes are divided into seven distinct subfamilies (ABC1, MDR/TAP, MRP, ALD, OABP, GCN20, White). This protein is a member of the MDR/TAP subfamily. Members of the MDR/TAP subfamily are involved in multidrug resistance as well as antigen presentation. This gene encodes a half-transporter involved in the transport of heme from the mitochondria to the cytosol. With iron/sulfur cluster precursors as its substrates, this protein may play a role in metal homeostasis. Mutations in this gene have been associated with mitochondrial iron accumulation and isodicentric (X)(q13) and sideroblastic anemia. Alternatively spliced transcript variants encoding multiple isoforms have been observed for this gene. [provided by RefSeq, Nov 2012].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - intracellular iron ion homeostasis\n",
      "    - intracellular iron ion homeostasis\n",
      "    - intracellular iron ion homeostasis\n",
      "    - heme transport\n",
      "    - iron-sulfur cluster assembly\n",
      "    - iron ion transmembrane transport\n",
      "    - transmembrane transport\n",
      "    - positive regulation of heme biosynthetic process\n",
      "    - positive regulation of heme biosynthetic process\n",
      "    - iron-sulfur cluster export from the mitochondrion\n",
      "    - positive regulation of iron-sulfur cluster assembly\n",
      "    - positive regulation of iron-sulfur cluster assembly\n",
      "    - negative regulation of reactive oxygen species biosynthetic process\n",
      "  CC:\n",
      "    - mitochondrion\n",
      "    - mitochondrion\n",
      "    - mitochondrial inner membrane\n",
      "    - mitochondrial inner membrane\n",
      "    - mitochondrial inner membrane\n",
      "  MF:\n",
      "    - protein binding\n",
      "    - ATP binding\n",
      "    - heme transmembrane transporter activity\n",
      "    - ATP hydrolysis activity\n",
      "    - ATPase-coupled transmembrane transporter activity\n",
      "    - identical protein binding\n",
      "    - protein homodimerization activity\n",
      "    - ABC-type iron-sulfur cluster transporter activity\n",
      "UniProt: {'Swiss-Prot': 'O75027', 'TrEMBL': ['A0A087WW65', 'A0A0S2Z2Z3', 'B4DGL8', 'A0A590UJS8', 'E9PNQ5', 'A0A2R8Y3N2', 'A0A5F9ZA98', 'A0A2R8YF35', 'A0A2R8Y473', 'E9PJR8']}\n",
      "\n",
      "Gene: GRIN3B\n",
      "Symbol: GRIN3B\n",
      "Name: glutamate ionotropic receptor NMDA type subunit 3B\n",
      "Summary: The protein encoded by this gene is a subunit of an N-methyl-D-aspartate (NMDA) receptor. The encoded protein is found primarily in motor neurons, where it forms a heterotetramer with GRIN1 to create an excitatory glycine receptor. Variations in this gene have been proposed to be linked to schizophrenia. [provided by RefSeq, Nov 2015].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - ionotropic glutamate receptor signaling pathway\n",
      "    - ionotropic glutamate receptor signaling pathway\n",
      "    - synaptic transmission, glutamatergic\n",
      "    - regulation of synaptic plasticity\n",
      "    - modulation of chemical synaptic transmission\n",
      "    - protein insertion into membrane\n",
      "    - regulation of calcium ion transport\n",
      "    - regulation of postsynaptic membrane potential\n",
      "    - calcium ion transmembrane transport\n",
      "    - monoatomic cation transmembrane transport\n",
      "    - regulation of presynaptic membrane potential\n",
      "  CC:\n",
      "    - plasma membrane\n",
      "    - NMDA selective glutamate receptor complex\n",
      "    - NMDA selective glutamate receptor complex\n",
      "    - monoatomic ion channel complex\n",
      "    - presynaptic membrane\n",
      "    - neuronal cell body\n",
      "    - postsynaptic density membrane\n",
      "    - neurotransmitter receptor complex\n",
      "  MF:\n",
      "    - NMDA glutamate receptor activity\n",
      "    - monoatomic cation channel activity\n",
      "    - calcium channel activity\n",
      "    - glutamate receptor activity\n",
      "    - glycine binding\n",
      "    - neurotransmitter receptor activity\n",
      "    - ligand-gated monoatomic ion channel activity involved in regulation of presynaptic membrane potential\n",
      "    - ligand-gated monoatomic ion channel activity involved in regulation of presynaptic membrane potential\n",
      "    - ligand-gated monoatomic ion channel activity involved in regulation of presynaptic membrane potential\n",
      "    - glycine-gated cation channel activity\n",
      "    - transmitter-gated monoatomic ion channel activity involved in regulation of postsynaptic membrane potential\n",
      "UniProt: {'Swiss-Prot': 'O60391', 'TrEMBL': 'Q5F0I5'}\n",
      "\n",
      "Gene: GRP\n",
      "Symbol: GRP\n",
      "Name: gastrin releasing peptide\n",
      "Summary: This gene encodes a member of the bombesin-like family of gastrin-releasing peptides. The encoded preproprotein is proteolytically processed to generate two peptides, gastrin-releasing peptide and neuromedin-C. These peptides regulate numerous functions of the gastrointestinal and central nervous systems, including release of gastrointestinal hormones, smooth muscle cell contraction, and epithelial cell proliferation. These peptides are also likely to play a role in human cancers of the lung, colon, stomach, pancreas, breast, and prostate. Alternative splicing results in multiple transcript variants, at least one of which encodes a preproprotein that is proteolytically processed. [provided by RefSeq, Jan 2016].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - ossification\n",
      "    - signal transduction\n",
      "    - neuropeptide signaling pathway\n",
      "    - neuropeptide signaling pathway\n",
      "    - regulation of gene expression\n",
      "    - social behavior\n",
      "    - psychomotor behavior\n",
      "    - response to external biotic stimulus\n",
      "    - mast cell degranulation\n",
      "    - positive regulation of peptide hormone secretion\n",
      "    - positive regulation of phospholipase C-activating G protein-coupled receptor signaling pathway\n",
      "    - negative regulation of voltage-gated potassium channel activity\n",
      "    - positive regulation of respiratory gaseous exchange\n",
      "    - cellular response to sodium phosphate\n",
      "    - negative regulation of voltage-gated sodium channel activity\n",
      "    - positive regulation of vascular associated smooth muscle cell apoptotic process\n",
      "    - positive regulation of behavioral fear response\n",
      "  CC:\n",
      "    - extracellular region\n",
      "    - extracellular space\n",
      "    - extracellular space\n",
      "    - secretory granule lumen\n",
      "    - neuron projection\n",
      "    - neuronal dense core vesicle\n",
      "  MF:\n",
      "    - signaling receptor binding\n",
      "    - neuropeptide hormone activity\n",
      "    - neuropeptide hormone activity\n",
      "    - protein binding\n",
      "UniProt: {'Swiss-Prot': 'P07492', 'TrEMBL': ['H0Y5C5', 'A0A140VJM7', 'H0YCH3']}\n",
      "\n",
      "Gene: KCNQ2\n",
      "Symbol: KCNQ2\n",
      "Name: potassium voltage-gated channel subfamily Q member 2\n",
      "Summary: The M channel is a slowly activating and deactivating potassium channel that plays a critical role in the regulation of neuronal excitability.  The M channel is formed by the association of the protein encoded by this gene and a related protein encoded by the KCNQ3 gene, both integral membrane proteins. M channel currents are inhibited by M1 muscarinic acetylcholine receptors and activated by retigabine, a novel anti-convulsant drug. Defects in this gene are a cause of benign familial neonatal convulsions type 1 (BFNC), also known as epilepsy, benign neonatal type 1 (EBN1). At least five transcript variants encoding five different isoforms have been found for this gene. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - action potential\n",
      "    - chemical synaptic transmission\n",
      "    - nervous system development\n",
      "    - potassium ion transmembrane transport\n",
      "    - potassium ion transmembrane transport\n",
      "  CC:\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - voltage-gated potassium channel complex\n",
      "    - voltage-gated potassium channel complex\n",
      "    - membrane\n",
      "    - node of Ranvier\n",
      "    - axon initial segment\n",
      "    - synapse\n",
      "  MF:\n",
      "    - voltage-gated potassium channel activity\n",
      "    - protein binding\n",
      "    - calmodulin binding\n",
      "    - voltage-gated monoatomic cation channel activity\n",
      "    - ankyrin binding\n",
      "UniProt: {'Swiss-Prot': 'O43526', 'TrEMBL': ['A0A1B0GUK1', 'Q53Y30', 'A0A0D9SGD4', 'A0A0D9SF10', 'A0AAQ5BGF6', 'A0A1B0GW14', 'A0A9L9PXL0', 'Q4VXP6', 'A0A0D9SG49', 'A0AAQ5BGI0', 'A0A0G2JH35', 'A0AAQ5BGE9', 'A0A0D9SGJ7', 'A0AAQ5BGD3', 'A0A0D9SEV1', 'A0A0D9SFE0', 'A0A0D9SGG3', 'A0AAQ5BGK9']}\n",
      "\n",
      "Gene: MSLN\n",
      "Symbol: MSLN\n",
      "Name: mesothelin\n",
      "Summary: This gene encodes a preproprotein that is proteolytically processed to generate two protein products, megakaryocyte potentiating factor and mesothelin. Megakaryocyte potentiating factor functions as a cytokine that can stimulate colony formation of bone marrow megakaryocytes. Mesothelin is a glycosylphosphatidylinositol-anchored cell-surface protein that may function as a cell adhesion protein. This protein is overexpressed in epithelial mesotheliomas, ovarian cancers and in specific squamous cell carcinomas. Alternative splicing results in multiple transcript variants, at least one of which encodes an isoform that is proteolytically processed. [provided by RefSeq, Feb 2016].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - cell adhesion\n",
      "    - cell-matrix adhesion\n",
      "  CC:\n",
      "    - extracellular region\n",
      "    - endoplasmic reticulum lumen\n",
      "    - Golgi apparatus\n",
      "    - plasma membrane\n",
      "    - cell surface\n",
      "    - membrane\n",
      "    - side of membrane\n",
      "  MF:\n",
      "    - protein binding\n",
      "UniProt: {'Swiss-Prot': 'Q13421', 'TrEMBL': ['H3BR90', 'H3BV92', 'H3BMA1', 'H3BUX1']}\n",
      "\n",
      "Gene: PDE10A\n",
      "Symbol: PDE10A\n",
      "Name: phosphodiesterase 10A\n",
      "Summary: The protein encoded by this gene belongs to the cyclic nucleotide phosphodiesterase family. It plays a role in signal transduction by regulating the intracellular concentration of cyclic nucleotides. This protein can hydrolyze both cAMP and cGMP to the corresponding nucleoside 5' monophosphate, but has higher affinity for cAMP, and is more efficient with cAMP as substrate. Alternatively spliced transcript variants have been described for this gene. [provided by RefSeq, Dec 2011].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - cAMP catabolic process\n",
      "    - negative regulation of cGMP-mediated signaling\n",
      "    - cAMP-mediated signaling\n",
      "    - cGMP catabolic process\n",
      "  CC:\n",
      "    - cytosol\n",
      "    - perikaryon\n",
      "    - glutamatergic synapse\n",
      "    - extrinsic component of postsynaptic density membrane\n",
      "  MF:\n",
      "    - 3',5'-cyclic-nucleotide phosphodiesterase activity\n",
      "    - 3',5'-cyclic-AMP phosphodiesterase activity\n",
      "    - 3',5'-cGMP-stimulated cyclic-nucleotide phosphodiesterase activity\n",
      "    - 3',5'-cGMP-stimulated cyclic-nucleotide phosphodiesterase activity\n",
      "    - 3',5'-cGMP-inhibited cyclic-nucleotide phosphodiesterase activity\n",
      "    - cAMP binding\n",
      "    - cAMP binding\n",
      "    - cGMP binding\n",
      "    - metal ion binding\n",
      "    - 3',5'-cyclic-GMP phosphodiesterase activity\n",
      "UniProt: {'Swiss-Prot': 'Q9Y233', 'TrEMBL': ['A0A3B3ISJ6', 'A0A1B1UZR0', 'A0A1B1UZQ1', 'A0A3B3IT18', 'A0A5F9ZI67', 'A0A7I2V301', 'A0A3B3IS50', 'A0A7I2YQV2', 'A0A3B3IRU7', 'A0A7I2V618', 'A0A3B3IRL3', 'A0A087WUD0', 'A0A5F9ZHF9', 'A0A7I2YQI4', 'A0AAA9X175', 'A0A3B3ITT8']}\n",
      "\n",
      "Gene: KCNQ5\n",
      "Symbol: KCNQ5\n",
      "Name: potassium voltage-gated channel subfamily Q member 5\n",
      "Summary: This gene is a member of the KCNQ potassium channel gene family that is differentially expressed in subregions of the brain and in skeletal muscle. The protein encoded by this gene yields currents that activate slowly with depolarization and can form heteromeric channels with the protein encoded by the KCNQ3 gene. Currents expressed from this protein have voltage dependences and inhibitor sensitivities in common with M-currents. They are also inhibited by M1 muscarinic receptor activation. Multiple transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, May 2009].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - potassium ion transmembrane transport\n",
      "    - potassium ion transmembrane transport\n",
      "    - potassium ion transmembrane transport\n",
      "  CC:\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - voltage-gated potassium channel complex\n",
      "    - voltage-gated potassium channel complex\n",
      "    - clathrin coat\n",
      "  MF:\n",
      "    - voltage-gated potassium channel activity\n",
      "    - voltage-gated potassium channel activity\n",
      "    - voltage-gated potassium channel activity\n",
      "    - protein binding\n",
      "UniProt: {'Swiss-Prot': 'Q9NR82', 'TrEMBL': ['H0Y3Z0', 'A6PVT7', 'F8WEA4', 'A6PVT8']}\n",
      "\n",
      "Gene: B4GALT1\n",
      "Symbol: B4GALT1\n",
      "Name: beta-1,4-galactosyltransferase 1\n",
      "Summary: This gene is one of seven beta-1,4-galactosyltransferase (beta4GalT) genes. They encode type II membrane-bound glycoproteins that appear to have exclusive specificity for the donor substrate UDP-galactose; all transfer galactose in a beta1,4 linkage to similar acceptor sugars: GlcNAc, Glc, and Xyl. Each beta4GalT has a distinct function in the biosynthesis of different glycoconjugates and saccharide structures. As type II membrane proteins, they have an N-terminal hydrophobic signal sequence that directs the protein to the Golgi apparatus and which then remains uncleaved to function as a transmembrane anchor. By sequence similarity, the beta4GalTs form four groups: beta4GalT1 and beta4GalT2, beta4GalT3 and beta4GalT4, beta4GalT5 and beta4GalT6, and beta4GalT7. This gene is unique among the beta4GalT genes because it encodes an enzyme that participates both in glycoconjugate and lactose biosynthesis. For the first activity, the enzyme adds galactose to N-acetylglucosamine residues that are either monosaccharides or the nonreducing ends of glycoprotein carbohydrate chains. The second activity is restricted to lactating mammary tissues where the enzyme forms a heterodimer with alpha-lactalbumin to catalyze UDP-galactose + D-glucose <=> UDP + lactose. The two enzymatic forms result from alternate transcription initiation sites and post-translational processing. Two transcripts, which differ only at the 5' end, with approximate lengths of 4.1 kb and 3.9 kb encode the same protein. The longer transcript encodes the type II membrane-bound, trans-Golgi resident protein involved in glycoconjugate biosynthesis. The shorter transcript encodes a protein which is cleaved to form the soluble lactose synthase. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - epithelial cell development\n",
      "    - acute inflammatory response\n",
      "    - lactose biosynthetic process\n",
      "    - galactose metabolic process\n",
      "    - protein N-linked glycosylation\n",
      "    - protein N-linked glycosylation\n",
      "    - lipid metabolic process\n",
      "    - cell adhesion\n",
      "    - binding of sperm to zona pellucida\n",
      "    - penetration of zona pellucida\n",
      "    - oligosaccharide biosynthetic process\n",
      "    - oligosaccharide biosynthetic process\n",
      "    - extracellular matrix organization\n",
      "    - positive regulation of apoptotic process\n",
      "    - development of secondary sexual characteristics\n",
      "    - epithelial cell proliferation\n",
      "    - negative regulation of epithelial cell proliferation\n",
      "    - regulation of acrosome reaction\n",
      "    - positive regulation of epithelial cell proliferation involved in wound healing\n",
      "    - angiogenesis involved in wound healing\n",
      "    - positive regulation of circulating fibrinogen levels\n",
      "    - macrophage migration\n",
      "  CC:\n",
      "    - Golgi trans cisterna\n",
      "    - Golgi membrane\n",
      "    - extracellular space\n",
      "    - Golgi apparatus\n",
      "    - Golgi apparatus\n",
      "    - plasma membrane\n",
      "    - external side of plasma membrane\n",
      "    - membrane\n",
      "    - basolateral plasma membrane\n",
      "    - desmosome\n",
      "    - filopodium\n",
      "    - secretory granule membrane\n",
      "    - brush border membrane\n",
      "    - Golgi cisterna membrane\n",
      "    - protein-containing complex\n",
      "    - azurophil granule membrane\n",
      "    - extracellular exosome\n",
      "  MF:\n",
      "    - beta-N-acetylglucosaminylglycopeptide beta-1,4-galactosyltransferase activity\n",
      "    - beta-N-acetylglucosaminylglycopeptide beta-1,4-galactosyltransferase activity\n",
      "    - beta-N-acetylglucosaminylglycopeptide beta-1,4-galactosyltransferase activity\n",
      "    - beta-N-acetylglucosaminylglycopeptide beta-1,4-galactosyltransferase activity\n",
      "    - N-acetyllactosamine synthase activity\n",
      "    - lactose synthase activity\n",
      "    - lactose synthase activity\n",
      "    - galactosyltransferase activity\n",
      "    - manganese ion binding\n",
      "    - UDP-galactosyltransferase activity\n",
      "    - UDP-galactosyltransferase activity\n",
      "    - identical protein binding\n",
      "    - alpha-tubulin binding\n",
      "    - beta-tubulin binding\n",
      "UniProt: {'Swiss-Prot': 'P15291'}\n",
      "\n",
      "Gene: KCNK9\n",
      "Symbol: KCNK9\n",
      "Name: potassium two pore domain channel subfamily K member 9\n",
      "Summary: This gene encodes a protein that contains multiple transmembrane regions and two pore-forming P domains and functions as a pH-dependent potassium channel. Amplification and overexpression of this gene have been observed in several types of human carcinomas. This gene is imprinted in the brain, with preferential expression from the maternal allele. A mutation in this gene was associated with Birk-Barel dysmorphism syndrome. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Jul 2017].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - potassium ion transport\n",
      "    - visual perception\n",
      "    - sodium ion transmembrane transport\n",
      "    - regulation of resting membrane potential\n",
      "    - cellular response to acidic pH\n",
      "    - regulation of action potential firing rate\n",
      "    - potassium ion import across plasma membrane\n",
      "    - potassium ion import across plasma membrane\n",
      "    - negative regulation of aldosterone secretion\n",
      "  CC:\n",
      "    - mitochondrial inner membrane\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - synaptic vesicle\n",
      "    - dendrite\n",
      "  MF:\n",
      "    - potassium channel activity\n",
      "    - sodium channel activity\n",
      "    - protein binding\n",
      "    - outward rectifier potassium channel activity\n",
      "    - outward rectifier potassium channel activity\n",
      "    - potassium ion leak channel activity\n",
      "    - identical protein binding\n",
      "    - metal ion binding\n",
      "    - protein heterodimerization activity\n",
      "UniProt: {'Swiss-Prot': 'Q9NPC2', 'TrEMBL': ['A0A3B3IU12', 'A0A3B3ITR0']}\n",
      "\n",
      "Gene: PTGER1\n",
      "Symbol: PTGER1\n",
      "Name: prostaglandin E receptor 1\n",
      "Summary: The protein encoded by this gene is a member of the G protein-coupled receptor family. This protein is one of four receptors identified for prostaglandin E2 (PGE2). Through a phosphatidylinositol-calcium second messenger system, G-Q proteins mediate this receptor's activity. Knockout studies in mice suggested a role of this receptor in mediating algesia and in regulation of blood pressure. Studies in mice also suggested that this gene may mediate adrenocorticotropic hormone response to bacterial endotoxin. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - inflammatory response\n",
      "    - G protein-coupled receptor signaling pathway\n",
      "    - adenylate cyclase-activating G protein-coupled receptor signaling pathway\n",
      "    - adenylate cyclase-activating dopamine receptor signaling pathway\n",
      "    - phospholipase C-activating G protein-coupled receptor signaling pathway\n",
      "    - positive regulation of cytosolic calcium ion concentration\n",
      "    - response to lipopolysaccharide\n",
      "    - response to prostaglandin E\n",
      "  CC:\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "  MF:\n",
      "    - prostaglandin E receptor activity\n",
      "    - prostaglandin E receptor activity\n",
      "    - D1 dopamine receptor binding\n",
      "UniProt: {'Swiss-Prot': 'P34995'}\n",
      "\n",
      "Gene: EPHB3\n",
      "Symbol: EPHB3\n",
      "Name: EPH receptor B3\n",
      "Summary: Ephrin receptors and their ligands, the ephrins, mediate numerous developmental processes, particularly in the nervous system. Based on their structures and sequence relationships, ephrins are divided into the ephrin-A (EFNA) class, which are anchored to the membrane by a glycosylphosphatidylinositol linkage, and the ephrin-B (EFNB) class, which are transmembrane proteins. The Eph family of receptors are divided into two groups based on the similarity of their extracellular domain sequences and their affinities for binding ephrin-A and ephrin-B ligands. Ephrin receptors make up the largest subgroup of the receptor tyrosine kinase (RTK) family. This gene encodes a receptor for ephrin-B family members. [provided by RefSeq, Mar 2010].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - angiogenesis\n",
      "    - urogenital system development\n",
      "    - epidermal growth factor receptor signaling pathway\n",
      "    - axon guidance\n",
      "    - axon guidance\n",
      "    - axonal fasciculation\n",
      "    - insulin receptor signaling pathway\n",
      "    - fibroblast growth factor receptor signaling pathway\n",
      "    - cell migration\n",
      "    - central nervous system projection neuron axonogenesis\n",
      "    - corpus callosum development\n",
      "    - regulation of cell-cell adhesion\n",
      "    - retinal ganglion cell axon guidance\n",
      "    - brain-derived neurotrophic factor receptor signaling pathway\n",
      "    - substrate adhesion-dependent cell spreading\n",
      "    - platelet-derived growth factor receptor-alpha signaling pathway\n",
      "    - platelet-derived growth factor receptor-beta signaling pathway\n",
      "    - vascular endothelial growth factor receptor-1 signaling pathway\n",
      "    - collagen-activated tyrosine kinase receptor signaling pathway\n",
      "    - vascular endothelial growth factor signaling pathway\n",
      "    - Kit signaling pathway\n",
      "    - macrophage colony-stimulating factor signaling pathway\n",
      "    - regulation of GTPase activity\n",
      "    - protein autophosphorylation\n",
      "    - insulin-like growth factor receptor signaling pathway\n",
      "    - hepatocyte growth factor receptor signaling pathway\n",
      "    - ephrin receptor signaling pathway\n",
      "    - ephrin receptor signaling pathway\n",
      "    - thymus development\n",
      "    - digestive tract morphogenesis\n",
      "    - regulation of axonogenesis\n",
      "    - positive regulation of synapse assembly\n",
      "    - roof of mouth development\n",
      "    - dendritic spine development\n",
      "    - dendritic spine morphogenesis\n",
      "  CC:\n",
      "    - extracellular region\n",
      "    - cytosol\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - dendrite\n",
      "  MF:\n",
      "    - ephrin receptor activity\n",
      "    - GPI-linked ephrin receptor activity\n",
      "    - transmembrane-ephrin receptor activity\n",
      "    - epidermal growth factor receptor activity\n",
      "    - fibroblast growth factor receptor activity\n",
      "    - hepatocyte growth factor receptor activity\n",
      "    - insulin receptor activity\n",
      "    - insulin-like growth factor receptor activity\n",
      "    - macrophage colony-stimulating factor receptor activity\n",
      "    - platelet-derived growth factor alpha-receptor activity\n",
      "    - platelet-derived growth factor beta-receptor activity\n",
      "    - stem cell factor receptor activity\n",
      "    - vascular endothelial growth factor receptor activity\n",
      "    - protein binding\n",
      "    - ATP binding\n",
      "    - axon guidance receptor activity\n",
      "    - boss receptor activity\n",
      "    - placental growth factor receptor activity\n",
      "    - protein tyrosine kinase collagen receptor activity\n",
      "    - brain-derived neurotrophic factor receptor activity\n",
      "UniProt: {'Swiss-Prot': 'P54753'}\n",
      "\n",
      "Gene: P2RY12\n",
      "Symbol: P2RY12\n",
      "Name: purinergic receptor P2Y12\n",
      "Summary: The product of this gene belongs to the family of G-protein coupled receptors. This family has several receptor subtypes with different pharmacological selectivity, which overlaps in some cases, for various adenosine and uridine nucleotides. This receptor is involved in platelet aggregation, and is a potential target for the treatment of thromboembolisms and other clotting disorders. Mutations in this gene are implicated in bleeding disorder, platelet type 8 (BDPLT8). Alternative splicing results in multiple transcript variants of this gene. [provided by RefSeq, Jul 2013].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - G protein-coupled adenosine receptor signaling pathway\n",
      "    - monoatomic ion transport\n",
      "    - substrate-dependent cell migration, cell extension\n",
      "    - G protein-coupled receptor signaling pathway\n",
      "    - G protein-coupled receptor signaling pathway\n",
      "    - adenylate cyclase-inhibiting G protein-coupled receptor signaling pathway\n",
      "    - phospholipase C-activating G protein-coupled receptor signaling pathway\n",
      "    - hemostasis\n",
      "    - calcium-mediated signaling\n",
      "    - cerebral cortex radial glia-guided migration\n",
      "    - cell projection organization\n",
      "    - lamellipodium assembly\n",
      "    - platelet activation\n",
      "    - positive regulation of integrin activation by cell surface receptor linked signal transduction\n",
      "    - positive regulation of cell adhesion mediated by integrin\n",
      "    - G protein-coupled purinergic nucleotide receptor signaling pathway\n",
      "    - positive regulation of monoatomic ion transport\n",
      "    - response to axon injury\n",
      "    - regulation of chemotaxis\n",
      "    - regulation of chemotaxis\n",
      "    - positive regulation of chemotaxis\n",
      "    - establishment of localization in cell\n",
      "    - positive regulation of phosphatidylinositol 3-kinase/protein kinase B signal transduction\n",
      "    - platelet aggregation\n",
      "    - cellular response to ATP\n",
      "    - visual system development\n",
      "    - positive regulation of ruffle assembly\n",
      "    - positive regulation of ruffle assembly\n",
      "    - regulation of microglial cell migration\n",
      "    - positive regulation of microglial cell migration\n",
      "    - positive regulation of microglial cell migration\n",
      "  CC:\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - cell surface\n",
      "    - membrane\n",
      "    - cell projection membrane\n",
      "    - cell body membrane\n",
      "  MF:\n",
      "    - G protein-coupled adenosine receptor activity\n",
      "    - G protein-coupled ADP receptor activity\n",
      "    - G protein-coupled ADP receptor activity\n",
      "    - guanyl-nucleotide exchange factor activity\n",
      "    - G protein-coupled purinergic nucleotide receptor activity\n",
      "UniProt: {'Swiss-Prot': 'Q9H244', 'TrEMBL': 'A8K7T1'}\n",
      "\n",
      "Gene: KL\n",
      "Symbol: KL\n",
      "Name: klotho\n",
      "Summary: This gene encodes a type-I membrane protein that is related to beta-glucosidases. Reduced production of this protein has been observed in patients with chronic renal failure (CRF), and this may be one of the factors underlying the degenerative processes (e.g., arteriosclerosis, osteoporosis, and skin atrophy) seen in CRF. Also, mutations within this protein have been associated with ageing and bone loss. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - negative regulation of systemic arterial blood pressure\n",
      "    - carbohydrate metabolic process\n",
      "    - energy reserve metabolic process\n",
      "    - determination of adult lifespan\n",
      "    - fibroblast growth factor receptor signaling pathway\n",
      "    - response to activity\n",
      "    - positive regulation of bone mineralization\n",
      "    - response to vitamin D\n",
      "    - norepinephrine biosynthetic process\n",
      "    - calcium ion homeostasis\n",
      "    - positive regulation of MAPKKK cascade by fibroblast growth factor receptor signaling pathway\n",
      "    - response to angiotensin\n",
      "  CC:\n",
      "    - extracellular region\n",
      "    - extracellular space\n",
      "    - plasma membrane\n",
      "    - membrane\n",
      "    - apical plasma membrane\n",
      "    - extracellular exosome\n",
      "  MF:\n",
      "    - beta-glucuronidase activity\n",
      "    - fibroblast growth factor receptor binding\n",
      "    - hormone activity\n",
      "    - vitamin D binding\n",
      "    - beta-glucosidase activity\n",
      "    - fibroblast growth factor binding\n",
      "    - fibroblast growth factor binding\n",
      "UniProt: {'Swiss-Prot': 'Q9UEF7'}\n",
      "\n",
      "Gene: PTH\n",
      "Symbol: PTH\n",
      "Name: parathyroid hormone\n",
      "Summary: This gene encodes a member of the parathyroid family of proteins. The encoded preproprotein is proteolytically processed to generate a protein that binds to the parathyroid hormone/parathyroid hormone-related peptide receptor and regulates blood calcium and phosphate levels. Excess production of the encoded protein, known as hyperparathyroidism, can result in hypercalcemia and kidney stones. On the other hand, defective processing of the encoded protein may lead to hypoparathyroidism, which can result in hypocalcemia and numbness. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Oct 2015].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - skeletal system development\n",
      "    - transcription by RNA polymerase II\n",
      "    - intracellular calcium ion homeostasis\n",
      "    - G protein-coupled receptor signaling pathway\n",
      "    - adenylate cyclase-activating G protein-coupled receptor signaling pathway\n",
      "    - adenylate cyclase-activating G protein-coupled receptor signaling pathway\n",
      "    - Rho protein signal transduction\n",
      "    - cell-cell signaling\n",
      "    - hormone-mediated apoptotic signaling pathway\n",
      "    - macromolecule biosynthetic process\n",
      "    - response to xenobiotic stimulus\n",
      "    - positive regulation of signal transduction\n",
      "    - response to lead ion\n",
      "    - regulation of gene expression\n",
      "    - positive regulation of gene expression\n",
      "    - negative regulation of gene expression\n",
      "    - magnesium ion homeostasis\n",
      "    - bone mineralization\n",
      "    - positive regulation of bone mineralization\n",
      "    - negative regulation of chondrocyte differentiation\n",
      "    - response to vitamin D\n",
      "    - bone resorption\n",
      "    - response to ethanol\n",
      "    - positive regulation of glycogen biosynthetic process\n",
      "    - positive regulation of transcription by RNA polymerase II\n",
      "    - cAMP metabolic process\n",
      "    - positive regulation of D-glucose import\n",
      "    - response to cadmium ion\n",
      "    - homeostasis of number of cells within a tissue\n",
      "    - phosphate ion homeostasis\n",
      "    - positive regulation of inositol phosphate biosynthetic process\n",
      "    - response to parathyroid hormone\n",
      "    - response to fibroblast growth factor\n",
      "    - positive regulation of cell proliferation in bone marrow\n",
      "    - negative regulation of apoptotic process in bone marrow cell\n",
      "    - positive regulation of osteoclast proliferation\n",
      "    - negative regulation of bone mineralization involved in bone maturation\n",
      "  CC:\n",
      "    - extracellular region\n",
      "    - extracellular region\n",
      "    - extracellular space\n",
      "  MF:\n",
      "    - hormone activity\n",
      "    - hormone activity\n",
      "    - protein binding\n",
      "    - protein binding\n",
      "    - parathyroid hormone receptor binding\n",
      "    - type 1 parathyroid hormone receptor binding\n",
      "    - receptor ligand activity\n",
      "    - peptide hormone receptor binding\n",
      "    - peptide hormone receptor binding\n",
      "UniProt: {'Swiss-Prot': 'P01270'}\n",
      "\n",
      "Gene: ALG2\n",
      "Symbol: ALG2\n",
      "Name: ALG2 alpha-1,3/1,6-mannosyltransferase\n",
      "Summary: This gene encodes a member of the glycosyltransferase 1 family. The encoded protein acts as an alpha 1,3 mannosyltransferase, mannosylating Man(2)GlcNAc(2)-dolichol diphosphate and Man(1)GlcNAc(2)-dolichol diphosphate to form Man(3)GlcNAc(2)-dolichol diphosphate. Defects in this gene have been associated with congenital disorder of glycosylation type Ih (CDG-Ii). Alternative splicing results in multiple transcript variants. [provided by RefSeq, Nov 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - protein glycosylation\n",
      "    - protein N-linked glycosylation\n",
      "    - dolichol-linked oligosaccharide biosynthetic process\n",
      "    - dolichol-linked oligosaccharide biosynthetic process\n",
      "    - dolichol-linked oligosaccharide biosynthetic process\n",
      "    - dolichol-linked oligosaccharide biosynthetic process\n",
      "  CC:\n",
      "    - endoplasmic reticulum membrane\n",
      "    - endomembrane system\n",
      "    - membrane\n",
      "    - cytoplasmic side of endoplasmic reticulum membrane\n",
      "  MF:\n",
      "    - alpha-1,3-mannosyltransferase activity\n",
      "    - alpha-1,3-mannosyltransferase activity\n",
      "    - alpha-1,3-mannosyltransferase activity\n",
      "    - GDP-Man:Man1GlcNAc2-PP-Dol alpha-1,3-mannosyltransferase activity\n",
      "    - protein binding\n",
      "    - GDP-Man:Man2GlcNAc2-PP-dolichol alpha-1,6-mannosyltransferase activity\n",
      "UniProt: {'Swiss-Prot': 'Q9H553', 'TrEMBL': 'A0A0A0MTE0'}\n",
      "\n",
      "Gene: SLC12A1\n",
      "Symbol: SLC12A1\n",
      "Name: solute carrier family 12 member 1\n",
      "Summary: This gene encodes a kidney-specific sodium-potassium-chloride cotransporter that is expressed on the luminal membrane of renal epithelial cells of the thick ascending limb of Henle's loop and the macula densa. It plays a key role in concentrating urine and accounts for most of the NaCl resorption. It is sensitive to such diuretics as furosemide and bumetanide. Some Bartter-like syndromes result from defects in this gene. Alternative splicing results in multiple transcript variants encoding distinct isoforms. Additional splice variants have been described but their biological validity in humans has not been experimentally proven.[provided by RefSeq, May 2010].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - monoatomic ion transport\n",
      "    - cell volume homeostasis\n",
      "    - cell volume homeostasis\n",
      "    - monoatomic ion transmembrane transport\n",
      "    - sodium ion transmembrane transport\n",
      "    - chloride ion homeostasis\n",
      "    - potassium ion homeostasis\n",
      "    - sodium ion homeostasis\n",
      "    - chloride transmembrane transport\n",
      "    - potassium ion import across plasma membrane\n",
      "  CC:\n",
      "    - plasma membrane\n",
      "    - membrane\n",
      "    - apical plasma membrane\n",
      "    - apical plasma membrane\n",
      "    - apical plasma membrane\n",
      "    - extracellular exosome\n",
      "    - extracellular exosome\n",
      "  MF:\n",
      "    - sodium:potassium:chloride symporter activity\n",
      "    - sodium:potassium:chloride symporter activity\n",
      "    - sodium:potassium:chloride symporter activity\n",
      "UniProt: {'Swiss-Prot': 'Q13621', 'TrEMBL': ['B4DPF4', 'Q8IUN5', 'A0A8I5KSK6', 'H0YNW0', 'H0YMG9', 'A0A2R8Y6V7', 'H0YLJ2']}\n",
      "\n",
      "Gene: HAMP\n",
      "Symbol: HAMP\n",
      "Name: hepcidin antimicrobial peptide\n",
      "Summary: The product encoded by this gene is involved in the maintenance of iron homeostasis, and it is necessary for the regulation of iron storage in macrophages, and for intestinal iron absorption. The preproprotein is post-translationally cleaved into mature peptides of 20, 22 and 25 amino acids, and these active peptides are rich in cysteines, which form intramolecular bonds that stabilize their beta-sheet structures. These peptides exhibit antimicrobial activity against bacteria and fungi. Mutations in this gene cause hemochromatosis type 2B, also known as juvenile hemochromatosis, a disease caused by severe iron overload that results in cardiomyopathy, cirrhosis, and endocrine failure. [provided by RefSeq, Oct 2014].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - negative regulation of transcription by RNA polymerase II\n",
      "    - positive regulation of receptor internalization\n",
      "    - myeloid cell homeostasis\n",
      "    - transcription by RNA polymerase II\n",
      "    - intracellular iron ion homeostasis\n",
      "    - intracellular iron ion homeostasis\n",
      "    - inflammatory response\n",
      "    - immune response\n",
      "    - signal transduction\n",
      "    - cell surface receptor signaling pathway via JAK-STAT\n",
      "    - response to iron ion\n",
      "    - protein catabolic process\n",
      "    - killing of cells of another organism\n",
      "    - killing of cells of another organism\n",
      "    - positive regulation of proteasomal ubiquitin-dependent protein catabolic process\n",
      "    - iron ion transmembrane transport\n",
      "    - negative regulation of iron ion transmembrane transport\n",
      "    - macrophage activation\n",
      "    - defense response to bacterium\n",
      "    - positive regulation of macrophage activation\n",
      "    - negative regulation of bone resorption\n",
      "    - positive regulation of transcription by RNA polymerase II\n",
      "    - negative regulation of inflammatory response\n",
      "    - defense response to Gram-negative bacterium\n",
      "    - defense response to Gram-positive bacterium\n",
      "    - defense response to fungus\n",
      "    - defense response to fungus\n",
      "    - establishment of localization in cell\n",
      "    - multicellular organismal-level iron ion homeostasis\n",
      "    - antimicrobial humoral immune response mediated by antimicrobial peptide\n",
      "    - positive regulation of protein polyubiquitination\n",
      "    - negative regulation of iron export across plasma membrane\n",
      "    - negative regulation of intestinal absorption\n",
      "    - positive regulation of receptor catabolic process\n",
      "  CC:\n",
      "    - extracellular region\n",
      "    - extracellular space\n",
      "    - extracellular space\n",
      "    - extracellular space\n",
      "    - nucleus\n",
      "  MF:\n",
      "    - signaling receptor binding\n",
      "    - hormone activity\n",
      "    - copper ion binding\n",
      "    - iron ion transmembrane transporter inhibitor activity\n",
      "    - transporter regulator activity\n",
      "UniProt: {'Swiss-Prot': 'P81172'}\n",
      "\n",
      "Gene: NMB\n",
      "Symbol: NMB\n",
      "Name: neuromedin B\n",
      "Summary: This gene encodes a member of the bombesin-like family of neuropeptides, which negatively regulate eating behavior. The encoded protein may regulate colonic smooth muscle contraction through binding to its cognate receptor, the neuromedin B receptor (NMBR). Polymorphisms of this gene may be associated with hunger, weight gain and obesity. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Jul 2015].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - signal transduction\n",
      "    - positive regulation of cytosolic calcium ion concentration\n",
      "    - neuropeptide signaling pathway\n",
      "    - cell-cell signaling\n",
      "    - negative regulation of interleukin-6 production\n",
      "    - positive regulation of interferon-alpha production\n",
      "    - positive regulation of hormone secretion\n",
      "    - negative regulation of hormone secretion\n",
      "    - arachidonate secretion\n",
      "    - positive regulation of osteoclast proliferation\n",
      "    - antiviral innate immune response\n",
      "    - sneeze reflex\n",
      "    - Leydig cell proliferation\n",
      "    - sensory perception of itch\n",
      "    - positive regulation of respiratory gaseous exchange\n",
      "    - positive regulation of testosterone secretion\n",
      "  CC:\n",
      "    - extracellular region\n",
      "    - extracellular space\n",
      "    - neuron projection\n",
      "    - neuron projection\n",
      "  MF:\n",
      "    - hormone activity\n",
      "    - neuropeptide hormone activity\n",
      "    - protein binding\n",
      "    - neuromedin B receptor binding\n",
      "UniProt: {'Swiss-Prot': 'P08949'}\n",
      "\n",
      "Gene: GRIK4\n",
      "Symbol: GRIK4\n",
      "Name: glutamate ionotropic receptor kainate type subunit 4\n",
      "Summary: This gene encodes a protein that belongs to the glutamate-gated ionic channel family. Glutamate functions as the major excitatory neurotransmitter in the central nervous system through activation of ligand-gated ion channels and G protein-coupled membrane receptors. The protein encoded by this gene forms functional heteromeric kainate-preferring ionic channels with the subunits encoded by related gene family members. Alternatively spliced transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Sep 2013].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - glutamate receptor signaling pathway\n",
      "    - chemical synaptic transmission\n",
      "    - monoatomic ion transmembrane transport\n",
      "    - ionotropic glutamate receptor signaling pathway\n",
      "    - synaptic transmission, glutamatergic\n",
      "    - modulation of chemical synaptic transmission\n",
      "    - regulation of postsynaptic membrane potential\n",
      "    - regulation of presynaptic membrane potential\n",
      "  CC:\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - kainate selective glutamate receptor complex\n",
      "    - presynaptic membrane\n",
      "    - cell projection\n",
      "    - hippocampal mossy fiber to CA3 synapse\n",
      "    - postsynaptic density membrane\n",
      "  MF:\n",
      "    - kainate selective glutamate receptor activity\n",
      "    - ligand-gated monoatomic ion channel activity involved in regulation of presynaptic membrane potential\n",
      "    - ligand-gated monoatomic ion channel activity involved in regulation of presynaptic membrane potential\n",
      "    - ligand-gated monoatomic ion channel activity involved in regulation of presynaptic membrane potential\n",
      "    - transmitter-gated monoatomic ion channel activity involved in regulation of postsynaptic membrane potential\n",
      "UniProt: {'Swiss-Prot': 'Q16099', 'TrEMBL': ['B2RAP6', 'A6H8K8', 'A0A8D9PH79', 'A0A8D9UJ88']}\n",
      "\n",
      "Gene: TYR\n",
      "Symbol: TYR\n",
      "Name: tyrosinase\n",
      "Summary: The enzyme encoded by this gene catalyzes the first 2 steps, and at least 1 subsequent step, in the conversion of tyrosine to melanin. The enzyme has both tyrosine hydroxylase and dopa oxidase catalytic activities, and requires copper for function. Mutations in this gene result in oculocutaneous albinism, and nonpathologic polymorphisms result in skin pigmentation variation. The human genome contains a pseudogene similar to the 3' half of this gene. [provided by RefSeq, Oct 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - melanin biosynthetic process from tyrosine\n",
      "    - eye pigment biosynthetic process\n",
      "    - visual perception\n",
      "    - cell population proliferation\n",
      "    - response to UV\n",
      "    - response to blue light\n",
      "    - response to vitamin D\n",
      "    - melanin biosynthetic process\n",
      "    - melanin biosynthetic process\n",
      "    - pigmentation\n",
      "    - thymus development\n",
      "    - response to cAMP\n",
      "  CC:\n",
      "    - cytoplasm\n",
      "    - lysosome\n",
      "    - Golgi-associated vesicle\n",
      "    - melanosome membrane\n",
      "    - melanosome\n",
      "    - intracellular membrane-bounded organelle\n",
      "    - perinuclear region of cytoplasm\n",
      "  MF:\n",
      "    - tyrosinase activity\n",
      "    - tyrosinase activity\n",
      "    - copper ion binding\n",
      "    - protein binding\n",
      "    - identical protein binding\n",
      "    - protein homodimerization activity\n",
      "UniProt: {'Swiss-Prot': 'P14679', 'TrEMBL': 'L8B082'}\n",
      "\n",
      "Gene: PHKG1\n",
      "Symbol: PHKG1\n",
      "Name: phosphorylase kinase catalytic subunit gamma 1\n",
      "Summary: This gene is a member of the Ser/Thr protein kinase family and encodes a protein with one protein kinase domain and two calmodulin-binding domains. This protein is the catalytic member of a 16 subunit protein kinase complex which contains equimolar ratios of 4 subunit types. The complex is a crucial glycogenolytic regulatory enzyme. This gene has two pseudogenes at chromosome 7q11.21 and one at chromosome 11p11.12. Multiple transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, May 2012].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - carbohydrate metabolic process\n",
      "    - glycogen metabolic process\n",
      "    - chromatin remodeling\n",
      "    - signal transduction\n",
      "  CC:\n",
      "    - cytoplasm\n",
      "    - cytosol\n",
      "    - phosphorylase kinase complex\n",
      "  MF:\n",
      "    - 3-phosphoinositide-dependent protein kinase activity\n",
      "    - DNA-dependent protein kinase activity\n",
      "    - AMP-activated protein kinase activity\n",
      "    - phosphorylase kinase activity\n",
      "    - eukaryotic translation initiation factor 2alpha kinase activity\n",
      "    - ribosomal protein S6 kinase activity\n",
      "    - calmodulin binding\n",
      "    - ATP binding\n",
      "    - enzyme binding\n",
      "    - histone H3S10 kinase activity\n",
      "    - histone H3T11 kinase activity\n",
      "    - histone H3T6 kinase activity\n",
      "    - histone H2AXS139 kinase activity\n",
      "    - histone H3S28 kinase activity\n",
      "    - histone H4S1 kinase activity\n",
      "    - histone H2AS1 kinase activity\n",
      "    - histone H2BS14 kinase activity\n",
      "    - histone H3T3 kinase activity\n",
      "    - histone H2AS121 kinase activity\n",
      "    - Rho-dependent protein serine/threonine kinase activity\n",
      "    - protein serine kinase activity\n",
      "    - histone H2BS36 kinase activity\n",
      "    - histone H3S57 kinase activity\n",
      "    - histone H3T45 kinase activity\n",
      "    - histone H2AT120 kinase activity\n",
      "UniProt: {'Swiss-Prot': 'Q16816', 'TrEMBL': ['F8WD25', 'B7Z6U2', 'C9J2Q6', 'C9JME8']}\n",
      "\n",
      "Gene: FNIP2\n",
      "Symbol: FNIP2\n",
      "Name: folliculin interacting protein 2\n",
      "Summary: This gene encodes a protein that binds to the tumor suppressor folliculin and to AMP-activated protein kinase (AMPK), and may play a role cellular metabolism and nutrient sensing by regulating the AMPK-mechanistic target of rapamycin signaling pathway. The encoded protein may also be involved in regulating the O6-methylguanine-induced apoptosis signaling pathway. This gene has a closely related paralog that encodes a protein with similar binding activities. Both related proteins also associate with the molecular chaperone heat shock protein-90 (Hsp90) and negatively regulate its ATPase activity and facilitate its association with folliculin. [provided by RefSeq, Jul 2017].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - negative regulation of transcription by RNA polymerase II\n",
      "    - regulation of protein phosphorylation\n",
      "    - protein phosphorylation\n",
      "    - intrinsic apoptotic signaling pathway in response to DNA damage\n",
      "    - positive regulation of protein-containing complex assembly\n",
      "    - positive regulation of peptidyl-serine phosphorylation\n",
      "    - positive regulation of TORC1 signaling\n",
      "  CC:\n",
      "    - cytoplasm\n",
      "    - cytoplasm\n",
      "    - lysosomal membrane\n",
      "    - cytosol\n",
      "    - centriolar satellite\n",
      "    - FNIP-folliculin RagC/D GAP\n",
      "  MF:\n",
      "    - protein binding\n",
      "    - ATPase inhibitor activity\n",
      "    - ATPase inhibitor activity\n",
      "    - protein-folding chaperone binding\n",
      "    - protein-folding chaperone binding\n",
      "UniProt: {'Swiss-Prot': 'Q9P278', 'TrEMBL': ['D6RFH5', 'H0Y8F3']}\n",
      "\n",
      "Gene: COX6B1\n",
      "Symbol: COX6B1\n",
      "Name: cytochrome c oxidase subunit 6B1\n",
      "Summary: Cytochrome c oxidase (COX), the terminal enzyme of the mitochondrial respiratory chain, catalyzes the electron transfer from reduced cytochrome c to oxygen. It is a heteromeric complex consisting of 3 catalytic subunits encoded by mitochondrial genes and multiple structural subunits encoded by nuclear genes. The mitochondrially-encoded subunits function in electron transfer, and the nuclear-encoded subunits may be involved in the regulation and assembly of the complex. This nuclear gene encodes subunit VIb. Mutations in this gene are associated with severe infantile encephalomyopathy. Three pseudogenes COX6BP-1, COX6BP-2 and COX6BP-3 have been found on chromosomes 7, 17 and 22q13.1-13.2, respectively. [provided by RefSeq, Jan 2010].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - mitochondrial electron transport, cytochrome c to oxygen\n",
      "    - substantia nigra development\n",
      "    - cellular respiration\n",
      "    - proton transmembrane transport\n",
      "  CC:\n",
      "    - mitochondrion\n",
      "    - mitochondrion\n",
      "    - mitochondrion\n",
      "    - mitochondrial inner membrane\n",
      "    - mitochondrial inner membrane\n",
      "    - mitochondrial membrane\n",
      "    - respiratory chain complex IV\n",
      "  MF:\n",
      "    - cytochrome-c oxidase activity\n",
      "UniProt: {'Swiss-Prot': 'P14854', 'TrEMBL': ['A0A494C160', 'K7EQD3']}\n",
      "\n",
      "Gene: SCN1A\n",
      "Symbol: SCN1A\n",
      "Name: sodium voltage-gated channel alpha subunit 1\n",
      "Summary: Voltage-dependent sodium channels are heteromeric complexes that regulate sodium exchange between intracellular and extracellular spaces and are essential for the generation and propagation of action potentials in muscle cells and neurons. Each sodium channel is composed of a large pore-forming, glycosylated alpha subunit and two smaller beta subunits. This gene encodes a sodium channel alpha subunit, which has four homologous domains, each of which contains six transmembrane regions. Allelic variants of this gene are associated with generalized epilepsy with febrile seizures and epileptic encephalopathy. Alternative splicing results in multiple transcript variants. The RefSeq Project has decided to create four representative RefSeq records. Three of the transcript variants are supported by experimental evidence and the fourth contains alternate 5' untranslated exons, the exact combination of which have not been experimentally confirmed for the full-length transcript. [provided by RefSeq, Oct 2015].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - sodium ion transport\n",
      "    - adult walking behavior\n",
      "    - determination of adult lifespan\n",
      "    - neuronal action potential propagation\n",
      "    - neuronal action potential\n",
      "    - nerve development\n",
      "    - sodium ion transmembrane transport\n",
      "    - neuromuscular process controlling posture\n",
      "    - detection of mechanical stimulus involved in sensory perception of pain\n",
      "    - establishment of localization in cell\n",
      "    - cardiac muscle cell action potential involved in contraction\n",
      "    - cardiac muscle cell action potential involved in contraction\n",
      "    - membrane depolarization during action potential\n",
      "    - membrane depolarization during action potential\n",
      "    - regulation of presynaptic membrane potential\n",
      "  CC:\n",
      "    - voltage-gated sodium channel complex\n",
      "    - nucleoplasm\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - intercalated disc\n",
      "    - nuclear body\n",
      "    - Z disc\n",
      "    - T-tubule\n",
      "    - node of Ranvier\n",
      "    - neuronal cell body\n",
      "    - axon initial segment\n",
      "  MF:\n",
      "    - voltage-gated sodium channel activity\n",
      "    - voltage-gated sodium channel activity\n",
      "    - voltage-gated sodium channel activity\n",
      "    - voltage-gated sodium channel activity\n",
      "    - voltage-gated monoatomic ion channel activity involved in regulation of presynaptic membrane potential\n",
      "    - voltage-gated monoatomic ion channel activity involved in regulation of presynaptic membrane potential\n",
      "    - voltage-gated monoatomic ion channel activity involved in regulation of presynaptic membrane potential\n",
      "UniProt: {'Swiss-Prot': 'P35498', 'TrEMBL': ['A0AAQ5BGN9', 'A0A1W2PPJ3', 'A0A1B0GVX7', 'A0A1B0GUN7', 'A0A286YEQ8', 'A0A286YFA8', 'A0A8I5KYC1', 'A0A1B0GWE6', 'A0A5F9ZHI6', 'A0A286YF26', 'A0A1B0GUX7']}\n",
      "\n",
      "Gene: PPIP5K1\n",
      "Symbol: PPIP5K1\n",
      "Name: diphosphoinositol pentakisphosphate kinase 1\n",
      "Summary: This gene encodes a dual functional inositol kinase. The encoded enzyme converts inositol hexakisphosphate to diphosphoinositol pentakisphosphate and diphosphoinositol pentakisphosphate to bis-diphosphoinositol tetrakisphosphate. This protein may be important for intracellular signaling pathways. Alternate splicing results in multiple transcript variants. A pseudogene of this gene is found on chromosome 15.[provided by RefSeq, Jun 2010].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - inositol metabolic process\n",
      "    - inositol metabolic process\n",
      "    - inositol phosphate biosynthetic process\n",
      "    - inositol phosphate metabolic process\n",
      "  CC:\n",
      "    - cytosol\n",
      "    - cytosol\n",
      "    - cytosol\n",
      "    - plasma membrane\n",
      "  MF:\n",
      "    - inositol-1,3,4,5,6-pentakisphosphate kinase activity\n",
      "    - inositol hexakisphosphate kinase activity\n",
      "    - inositol hexakisphosphate kinase activity\n",
      "    - inositol hexakisphosphate kinase activity\n",
      "    - diphosphoinositol pentakisphosphate kinase activity\n",
      "    - inositol hexakisphosphate 5-kinase activity\n",
      "    - ATP binding\n",
      "    - 5-diphosphoinositol pentakisphosphate 1-kinase activity\n",
      "    - 5-diphosphoinositol pentakisphosphate 1-kinase activity\n",
      "    - inositol hexakisphosphate 1-kinase activity\n",
      "UniProt: {'Swiss-Prot': 'Q6PFW1', 'TrEMBL': ['A0A9L9PYJ9', 'C9J5E6', 'C9J490', 'H7C436', 'A0A8J8ZH50', 'H7C398', 'F8W9A8', 'B7WPL9', 'H0Y3Y5', 'A0A2R8YGT1', 'C9JZX6']}\n",
      "\n",
      "Gene: ZNF92\n",
      "Symbol: ZNF92\n",
      "Name: zinc finger protein 92\n",
      "Summary: Predicted to enable DNA-binding transcription factor activity, RNA polymerase II-specific and RNA polymerase II cis-regulatory region sequence-specific DNA binding activity. Predicted to be involved in regulation of transcription, DNA-templated. Predicted to be located in nucleus. [provided by Alliance of Genome Resources, Apr 2022]\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - regulation of DNA-templated transcription\n",
      "    - regulation of transcription by RNA polymerase II\n",
      "  CC:\n",
      "    - nucleus\n",
      "  MF:\n",
      "    - RNA polymerase II cis-regulatory region sequence-specific DNA binding\n",
      "    - DNA-binding transcription factor activity, RNA polymerase II-specific\n",
      "    - DNA-binding transcription factor activity\n",
      "    - protein binding\n",
      "    - zinc ion binding\n",
      "UniProt: {'Swiss-Prot': 'Q03936', 'TrEMBL': ['C9IZS8', 'V9HVY7']}\n",
      "\n",
      "Gene: ZNF714\n",
      "Symbol: ZNF714\n",
      "Name: zinc finger protein 714\n",
      "Summary: Predicted to enable DNA-binding transcription factor activity, RNA polymerase II-specific and RNA polymerase II cis-regulatory region sequence-specific DNA binding activity. Predicted to be involved in regulation of transcription, DNA-templated. Predicted to be located in nucleus. [provided by Alliance of Genome Resources, Apr 2022]\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - regulation of DNA-templated transcription\n",
      "    - regulation of transcription by RNA polymerase II\n",
      "  CC:\n",
      "    - nucleus\n",
      "  MF:\n",
      "    - RNA polymerase II cis-regulatory region sequence-specific DNA binding\n",
      "    - DNA-binding transcription factor activity, RNA polymerase II-specific\n",
      "    - zinc ion binding\n",
      "UniProt: {'Swiss-Prot': 'Q96N38', 'TrEMBL': ['I3L2D0', 'A0A087WU35', 'A0A087X1X4', 'A0A087WX89', 'A0A087X1Z8', 'M0QY60', 'A0A087WV13']}\n",
      "\n",
      "Gene: HSD11B2\n",
      "Symbol: HSD11B2\n",
      "Name: hydroxysteroid 11-beta dehydrogenase 2\n",
      "Summary: There are at least two isozymes of the corticosteroid 11-beta-dehydrogenase, a microsomal enzyme complex responsible for the interconversion of cortisol and cortisone. The type I isozyme has both 11-beta-dehydrogenase (cortisol to cortisone) and 11-oxoreductase (cortisone to cortisol) activities. The type II isozyme, encoded by this gene, has only 11-beta-dehydrogenase activity. In aldosterone-selective epithelial tissues such as the kidney, the type II isozyme catalyzes the glucocorticoid cortisol to the inactive metabolite cortisone, thus preventing illicit activation of the mineralocorticoid receptor. In tissues that do not express the mineralocorticoid receptor, such as the placenta and testis, it protects cells from the growth-inhibiting and/or pro-apoptotic effects of cortisol, particularly during embryonic development. Mutations in this gene cause the syndrome of apparent mineralocorticoid excess and hypertension. [provided by RefSeq, Feb 2010].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - response to hypoxia\n",
      "    - regulation of blood volume by renal aldosterone\n",
      "    - female pregnancy\n",
      "    - glucocorticoid metabolic process\n",
      "    - response to xenobiotic stimulus\n",
      "    - response to food\n",
      "    - response to insulin\n",
      "    - cortisol metabolic process\n",
      "    - response to glucocorticoid\n",
      "  CC:\n",
      "    - endoplasmic reticulum membrane\n",
      "    - intracellular membrane-bounded organelle\n",
      "    - intracellular membrane-bounded organelle\n",
      "  MF:\n",
      "    - steroid binding\n",
      "    - NAD binding\n",
      "    - 11-beta-hydroxysteroid dehydrogenase (NAD+) activity\n",
      "    - 11-beta-hydroxysteroid dehydrogenase (NAD+) activity\n",
      "UniProt: {'Swiss-Prot': 'P80365', 'TrEMBL': 'R4GN04'}\n",
      "\n",
      "Gene: SLC27A1\n",
      "Symbol: SLC27A1\n",
      "Name: solute carrier family 27 member 1\n",
      "Summary: Enables biotin transmembrane transporter activity; efflux transmembrane transporter activity; and long-chain fatty acid transporter activity. Involved in several processes, including carboxylic acid transmembrane transport; glycerophospholipid biosynthetic process; and lipid transport across blood-brain barrier. Located in membrane. Part of plasma membrane. [provided by Alliance of Genome Resources, Apr 2022]\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - medium-chain fatty acid transport\n",
      "    - long-chain fatty acid metabolic process\n",
      "    - response to cold\n",
      "    - positive regulation of triglyceride biosynthetic process\n",
      "    - biotin transport\n",
      "    - long-chain fatty acid transport\n",
      "    - long-chain fatty acid transport\n",
      "    - long-chain fatty acid transport\n",
      "    - long-chain fatty acid import across plasma membrane\n",
      "    - long-chain fatty acid import across plasma membrane\n",
      "    - positive regulation of heat generation\n",
      "    - adiponectin-activated signaling pathway\n",
      "    - glucose import in response to insulin stimulus\n",
      "    - long-chain fatty acid import into cell\n",
      "    - long-chain fatty acid import into cell\n",
      "    - long-chain fatty acid import into cell\n",
      "    - export across plasma membrane\n",
      "    - export across plasma membrane\n",
      "    - transport across blood-brain barrier\n",
      "    - biotin import across plasma membrane\n",
      "    - lipid transport across blood-brain barrier\n",
      "  CC:\n",
      "    - endoplasmic reticulum\n",
      "    - endoplasmic reticulum membrane\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - basal plasma membrane\n",
      "    - membrane\n",
      "  MF:\n",
      "    - nucleotide binding\n",
      "    - long-chain fatty acid-CoA ligase activity\n",
      "    - long-chain fatty acid transmembrane transporter activity\n",
      "    - long-chain fatty acid transmembrane transporter activity\n",
      "    - long-chain fatty acid transmembrane transporter activity\n",
      "    - protein binding\n",
      "    - biotin transmembrane transporter activity\n",
      "    - fatty acid transmembrane transporter activity\n",
      "    - efflux transmembrane transporter activity\n",
      "    - efflux transmembrane transporter activity\n",
      "    - very long-chain fatty acid-CoA ligase activity\n",
      "    - identical protein binding\n",
      "    - protein serine/threonine kinase activator activity\n",
      "    - arachidonate-CoA ligase activity\n",
      "    - oleoyl-CoA ligase activity\n",
      "    - oleoyl-CoA ligase activity\n",
      "UniProt: {'Swiss-Prot': 'Q6PCB7', 'TrEMBL': ['M0QZC9', 'B7Z225', 'M0QXY2']}\n",
      "\n",
      "Gene: CD276\n",
      "Symbol: CD276\n",
      "Name: CD276 molecule\n",
      "Summary: The protein encoded by this gene belongs to the immunoglobulin superfamily, and thought to participate in the regulation of T-cell-mediated immune response. Studies show that while the transcript of this gene is ubiquitously expressed in normal tissues and solid tumors, the protein is preferentially expressed only in tumor tissues. Additionally, it was observed that the 3' UTR of this transcript contains a target site for miR29 microRNA, and there is an inverse correlation between the expression of this protein and miR29 levels, suggesting regulation of expression of this gene product by miR29. Alternatively spliced transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Sep 2011].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - regulation of cytokine production\n",
      "    - positive regulation of type II interferon production\n",
      "    - positive regulation of T cell proliferation\n",
      "    - T cell activation\n",
      "    - regulation of immune response\n",
      "    - T cell receptor signaling pathway\n",
      "  CC:\n",
      "    - external side of plasma membrane\n",
      "    - external side of plasma membrane\n",
      "    - membrane\n",
      "  MF:\n",
      "    - signaling receptor binding\n",
      "    - signaling receptor binding\n",
      "    - protein binding\n",
      "UniProt: {'Swiss-Prot': 'Q5ZPR3', 'TrEMBL': ['A0A0C4DGH0', 'A8K9J6', 'H0YLT3', 'H0YLT8', 'H0YKQ9', 'H0YN85', 'H0YK59', 'H0YL10', 'H3BM68', 'H0YN29', 'H0YK40']}\n",
      "\n",
      "Gene: GRK7\n",
      "Symbol: GRK7\n",
      "Name: G protein-coupled receptor kinase 7\n",
      "Summary: This gene encodes a member of the guanine nucleotide-binding protein (G protein)-coupled receptor kinase subfamily of the Ser/Thr protein kinase family. It is specifically expressed in the retina and the encoded protein has been shown to phosphorylate cone opsins and initiate their deactivation. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - signal transduction\n",
      "    - visual perception\n",
      "    - regulation of signal transduction\n",
      "    - regulation of opsin-mediated signaling pathway\n",
      "    - protein autophosphorylation\n",
      "  CC:\n",
      "    - cytoplasm\n",
      "    - photoreceptor disc membrane\n",
      "  MF:\n",
      "    - protein binding\n",
      "    - ATP binding\n",
      "    - rhodopsin kinase activity\n",
      "    - rhodopsin kinase activity\n",
      "    - rhodopsin kinase activity\n",
      "UniProt: {'Swiss-Prot': 'Q8WTQ7'}\n",
      "\n",
      "Gene: DHRS4\n",
      "Symbol: DHRS4\n",
      "Name: dehydrogenase/reductase 4\n",
      "Summary: Enables identical protein binding activity; oxidoreductase activity, acting on NAD(P)H, quinone or similar compound as acceptor; and oxidoreductase activity, acting on the CH-OH group of donors, NAD or NADP as acceptor. Involved in several processes, including cellular ketone metabolic process; positive regulation of reactive oxygen species metabolic process; and steroid metabolic process. Located in nucleus and peroxisomal membrane. [provided by Alliance of Genome Resources, Apr 2022]\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - alcohol metabolic process\n",
      "    - steroid metabolic process\n",
      "    - ketone metabolic process\n",
      "    - retinol metabolic process\n",
      "    - retinal metabolic process\n",
      "    - positive regulation of reactive oxygen species metabolic process\n",
      "  CC:\n",
      "    - nucleus\n",
      "    - nucleus\n",
      "    - mitochondrion\n",
      "    - mitochondrion\n",
      "    - peroxisome\n",
      "    - peroxisome\n",
      "    - peroxisomal membrane\n",
      "    - peroxisomal membrane\n",
      "    - peroxisomal matrix\n",
      "    - endoplasmic reticulum membrane\n",
      "    - cytosol\n",
      "  MF:\n",
      "    - 3-beta-hydroxysteroid 3-dehydrogenase (NADP+) activity\n",
      "    - carbonyl reductase (NADPH) activity\n",
      "    - carbonyl reductase (NADPH) activity\n",
      "    - oxidoreductase activity, acting on NAD(P)H, quinone or similar compound as acceptor\n",
      "    - alcohol dehydrogenase [NAD(P)+] activity\n",
      "    - 3-beta-hydroxy-5-beta-steroid dehydrogenase (NADP+) activity\n",
      "    - identical protein binding\n",
      "    - identical protein binding\n",
      "    - all-trans-retinol dehydrogenase (NADP+) activity\n",
      "UniProt: {'Swiss-Prot': 'Q9BTZ2', 'TrEMBL': ['H0YNP7', 'F5GWZ1']}\n",
      "\n",
      "Gene: SULF1\n",
      "Symbol: SULF1\n",
      "Name: sulfatase 1\n",
      "Summary: This gene encodes an extracellular heparan sulfate endosulfatase. The encoded enzyme selectively removes 6-O-sulfate groups from heparan sulfate chains of heparan sulfate proteoglycans (HSPGs). The enzyme is secreted through the Golgi and is subsequently localized to the cell surface. The expression of this gene may be down-regulated in several types of cancer, including hepatocellular (HCC), ovarian and breast cancers. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Aug 2013].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - kidney development\n",
      "    - negative regulation of endothelial cell proliferation\n",
      "    - chondrocyte development\n",
      "    - glomerular filtration\n",
      "    - apoptotic process\n",
      "    - positive regulation of vascular endothelial growth factor production\n",
      "    - positive regulation of vascular endothelial growth factor production\n",
      "    - esophagus smooth muscle contraction\n",
      "    - negative regulation of angiogenesis\n",
      "    - positive regulation of Wnt signaling pathway\n",
      "    - positive regulation of Wnt signaling pathway\n",
      "    - heparan sulfate proteoglycan metabolic process\n",
      "    - heparan sulfate proteoglycan metabolic process\n",
      "    - heparan sulfate proteoglycan metabolic process\n",
      "    - negative regulation of cell migration\n",
      "    - positive regulation of BMP signaling pathway\n",
      "    - glomerular basement membrane development\n",
      "    - glomerular basement membrane development\n",
      "    - glial cell-derived neurotrophic factor receptor signaling pathway\n",
      "    - regulation of fibroblast growth factor receptor signaling pathway\n",
      "    - negative regulation of fibroblast growth factor receptor signaling pathway\n",
      "    - negative regulation of fibroblast growth factor receptor signaling pathway\n",
      "    - negative regulation of fibroblast growth factor receptor signaling pathway\n",
      "    - vascular endothelial growth factor receptor signaling pathway\n",
      "    - embryonic skeletal system development\n",
      "    - cartilage development\n",
      "    - bone development\n",
      "    - innervation\n",
      "    - negative regulation of prostatic bud formation\n",
      "  CC:\n",
      "    - extracellular space\n",
      "    - extracellular space\n",
      "    - extracellular space\n",
      "    - endoplasmic reticulum\n",
      "    - endoplasmic reticulum\n",
      "    - Golgi apparatus\n",
      "    - Golgi stack\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - cell surface\n",
      "    - cell surface\n",
      "    - membrane raft\n",
      "    - collagen-containing extracellular matrix\n",
      "  MF:\n",
      "    - arylsulfatase activity\n",
      "    - arylsulfatase activity\n",
      "    - calcium ion binding\n",
      "    - glycosaminoglycan binding\n",
      "    - N-acetylglucosamine-6-sulfatase activity\n",
      "    - N-acetylglucosamine-6-sulfatase activity\n",
      "    - N-acetylglucosamine-6-sulfatase activity\n",
      "UniProt: {'Swiss-Prot': 'Q8IWU6', 'TrEMBL': ['E9PPQ3', 'E9PS14', 'E9PLS5', 'E9PJL8', 'E9PI06', 'H0YDR1', 'A0A087WWR8']}\n",
      "\n",
      "Gene: COX14\n",
      "Symbol: COX14\n",
      "Name: cytochrome c oxidase assembly factor COX14\n",
      "Summary: This gene encodes a small single-pass transmembrane protein that localizes to mitochondria. This protein may play a role in coordinating the early steps of cytochrome c oxidase (COX; also known as complex IV) subunit assembly and, in particular, the synthesis and assembly of the COX I subunit of the holoenzyme. Mutations in this gene have been associated with mitochondrial complex IV deficiency. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Nov 2012].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - mitochondrial cytochrome c oxidase assembly\n",
      "    - mitochondrial cytochrome c oxidase assembly\n",
      "    - mitochondrial cytochrome c oxidase assembly\n",
      "  CC:\n",
      "    - mitochondrion\n",
      "    - mitochondrion\n",
      "    - mitochondrion\n",
      "    - mitochondrial outer membrane\n",
      "    - mitochondrial inner membrane\n",
      "  MF:\n",
      "    - protein binding\n",
      "UniProt: {'Swiss-Prot': 'Q96I36'}\n",
      "\n",
      "Gene: DDX51\n",
      "Symbol: DDX51\n",
      "Name: DEAD-box helicase 51\n",
      "Summary: Enables RNA binding activity. Predicted to be involved in rRNA processing. Located in membrane. [provided by Alliance of Genome Resources, Apr 2022]\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - rRNA processing\n",
      "  CC:\n",
      "    - nucleus\n",
      "    - nucleolus\n",
      "    - membrane\n",
      "  MF:\n",
      "    - RNA binding\n",
      "    - RNA helicase activity\n",
      "    - ATP binding\n",
      "    - ATP hydrolysis activity\n",
      "UniProt: {'Swiss-Prot': 'Q8N8A6'}\n",
      "\n",
      "Gene: COX7A2\n",
      "Symbol: COX7A2\n",
      "Name: cytochrome c oxidase subunit 7A2\n",
      "Summary: Cytochrome c oxidase, the terminal component of the mitochondrial respiratory chain, catalyzes the electron transfer from reduced cytochrome c to oxygen. This component is a heteromeric complex consisting of three catalytic subunits encoded by mitochondrial genes, and multiple structural subunits encoded by nuclear genes. The mitochondrially-encoded subunits function in electron transfer, while the nuclear-encoded subunits may function in the regulation and assembly of the complex. This nuclear gene encodes polypeptide 2 (liver isoform) of subunit VIIa, with this polypeptide being present in both muscle and non-muscle tissues. In addition to polypeptide 2, subunit VIIa includes polypeptide 1 (muscle isoform), which is present only in muscle tissues, and a related protein, which is present in all tissues. Alternative splicing results in multiple transcript variants. Related pseudogenes have been identified on chromosomes 4 and 14. [provided by RefSeq, Oct 2009].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - mitochondrial electron transport, cytochrome c to oxygen\n",
      "    - cellular respiration\n",
      "    - mitochondrial respirasome assembly\n",
      "  CC:\n",
      "    - mitochondrion\n",
      "    - mitochondrial inner membrane\n",
      "    - mitochondrial membrane\n",
      "    - respiratory chain complex IV\n",
      "    - respiratory chain complex\n",
      "  MF:\n",
      "    - protein binding\n",
      "    - oxidoreductase activity\n",
      "UniProt: {'Swiss-Prot': 'P14406', 'TrEMBL': ['D6RIE3', 'H0UI06', 'D6R9C3', 'H0Y8S6', 'D6RGV5', 'D6RA35']}\n",
      "\n",
      "Gene: IDH1\n",
      "Symbol: IDH1\n",
      "Name: isocitrate dehydrogenase (NADP(+)) 1\n",
      "Summary: Isocitrate dehydrogenases catalyze the oxidative decarboxylation of isocitrate to 2-oxoglutarate. These enzymes belong to two distinct subclasses, one of which utilizes NAD(+) as the electron acceptor and the other NADP(+). Five isocitrate dehydrogenases have been reported: three NAD(+)-dependent isocitrate dehydrogenases, which localize to the mitochondrial matrix, and two NADP(+)-dependent isocitrate dehydrogenases, one of which is mitochondrial and the other predominantly cytosolic. Each NADP(+)-dependent isozyme is a homodimer. The protein encoded by this gene is the NADP(+)-dependent isocitrate dehydrogenase found in the cytoplasm and peroxisomes. It contains the PTS-1 peroxisomal targeting signal sequence. The presence of this enzyme in peroxisomes suggests roles in the regeneration of NADPH for intraperoxisomal reductions, such as the conversion of 2, 4-dienoyl-CoAs to 3-enoyl-CoAs, as well as in peroxisomal reactions that consume 2-oxoglutarate, namely the alpha-hydroxylation of phytanic acid. The cytoplasmic enzyme serves a significant role in cytoplasmic NADPH production. Alternatively spliced transcript variants encoding the same protein have been found for this gene. [provided by RefSeq, Sep 2013].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - glyoxylate cycle\n",
      "    - tricarboxylic acid cycle\n",
      "    - isocitrate metabolic process\n",
      "    - isocitrate metabolic process\n",
      "    - 2-oxoglutarate metabolic process\n",
      "    - NADP metabolic process\n",
      "    - glutathione metabolic process\n",
      "    - response to oxidative stress\n",
      "    - female gonad development\n",
      "    - response to steroid hormone\n",
      "    - regulation of phospholipid catabolic process\n",
      "    - regulation of phospholipid biosynthetic process\n",
      "  CC:\n",
      "    - extracellular region\n",
      "    - cytoplasm\n",
      "    - mitochondrion\n",
      "    - peroxisome\n",
      "    - peroxisome\n",
      "    - peroxisomal matrix\n",
      "    - cytosol\n",
      "    - cytosol\n",
      "    - cytosol\n",
      "    - secretory granule lumen\n",
      "    - extracellular exosome\n",
      "    - tertiary granule lumen\n",
      "    - ficolin-1-rich granule lumen\n",
      "  MF:\n",
      "    - magnesium ion binding\n",
      "    - isocitrate dehydrogenase (NADP+) activity\n",
      "    - isocitrate dehydrogenase (NADP+) activity\n",
      "    - protein binding\n",
      "    - identical protein binding\n",
      "    - protein homodimerization activity\n",
      "    - cadherin binding\n",
      "    - NADP binding\n",
      "    - NAD binding\n",
      "UniProt: {'Swiss-Prot': 'O75874', 'TrEMBL': ['C9JLU6', 'A0A024R3Y6', 'C9JJE5', 'C9J4N6']}\n",
      "\n",
      "Gene: FBLL1\n",
      "Symbol: FBLL1\n",
      "Name: fibrillarin like 1\n",
      "Summary: Predicted to enable RNA binding activity; histone-glutamine methyltransferase activity; and rRNA methyltransferase activity. Predicted to be involved in box C/D RNA 3'-end processing; histone glutamine methylation; and rRNA methylation. Predicted to act upstream of or within blastocyst hatching. Located in fibrillar center and nucleoplasm. [provided by Alliance of Genome Resources, Apr 2022]\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - box C/D sno(s)RNA 3'-end processing\n",
      "    - blastocyst hatching\n",
      "    - chromatin remodeling\n",
      "    - rRNA methylation\n",
      "  CC:\n",
      "    - fibrillar center\n",
      "    - nucleoplasm\n",
      "    - nucleolus\n",
      "    - Cajal body\n",
      "    - box C/D methylation guide snoRNP complex\n",
      "    - small-subunit processome\n",
      "  MF:\n",
      "    - RNA binding\n",
      "    - rRNA methyltransferase activity\n",
      "    - histone H2AQ104 methyltransferase activity\n",
      "\n",
      "Gene: PRCP\n",
      "Symbol: PRCP\n",
      "Name: prolylcarboxypeptidase\n",
      "Summary: This gene encodes a member of the peptidase S28 family of serine exopeptidases. The encoded preproprotein is proteolytically processed to generate the mature lysosomal prolylcarboxypeptidase. This enzyme cleaves C-terminal amino acids linked to proline in peptides such as angiotension II, III and des-Arg9-bradykinin. The cleavage occurs at acidic pH, but the enzyme activity is retained with some substrates at neutral pH. This enzyme has been shown to be an activator of the cell matrix-associated prekallikrein. The importance of angiotension II, one of the substrates of this enzyme, in regulating blood pressure and electrolyte balance suggests that this gene may be related to essential hypertension. A pseudogene of this gene has been identified on chromosome 2. Alternative splicing results in multiple transcript variants, at least one of which encodes an isoform that is proteolytically processed. [provided by RefSeq, Jan 2016].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - angiotensin maturation\n",
      "    - regulation of thyroid hormone receptor signaling pathway\n",
      "    - plasma kallikrein-kinin cascade\n",
      "    - negative regulation of systemic arterial blood pressure\n",
      "    - glucose homeostasis\n",
      "    - regulation of blood vessel endothelial cell migration\n",
      "    - angiogenesis involved in wound healing\n",
      "    - energy homeostasis\n",
      "    - regulation of reactive oxygen species metabolic process\n",
      "  CC:\n",
      "    - plasma membrane\n",
      "    - azurophil granule membrane\n",
      "    - intracellular membrane-bounded organelle\n",
      "    - basal part of cell\n",
      "    - extracellular exosome\n",
      "    - ficolin-1-rich granule membrane\n",
      "  MF:\n",
      "    - metallocarboxypeptidase activity\n",
      "    - serine-type carboxypeptidase activity\n",
      "    - protein binding\n",
      "    - dipeptidyl-peptidase activity\n",
      "UniProt: {'Swiss-Prot': 'P42785', 'TrEMBL': ['B3KR26', 'B7Z7Q6', 'A0A7P0TB23', 'E9PNJ1', 'A0A7P0Z451', 'E9PL85', 'E9PNF7', 'E9PQB5', 'E9PKN6', 'E9PLY4', 'E9PL49']}\n",
      "\n",
      "Gene: SLCO2B1\n",
      "Symbol: SLCO2B1\n",
      "Name: solute carrier organic anion transporter family member 2B1\n",
      "Summary: This locus encodes a member of the organic anion-transporting polypeptide family of membrane proteins. The protein encoded by this locus may function in regulation of placental uptake of sulfated steroids. Alternatively spliced transcript variants have been described. [provided by RefSeq, Nov 2010].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - xenobiotic metabolic process\n",
      "    - monoatomic ion transport\n",
      "    - organic anion transport\n",
      "    - bile acid and bile salt transport\n",
      "    - prostaglandin transport\n",
      "    - heme catabolic process\n",
      "    - sodium-independent organic anion transport\n",
      "    - sodium-independent organic anion transport\n",
      "    - transmembrane transport\n",
      "    - transmembrane transport\n",
      "    - transport across blood-brain barrier\n",
      "  CC:\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - basal plasma membrane\n",
      "    - basolateral plasma membrane\n",
      "    - apical plasma membrane\n",
      "    - apical plasma membrane\n",
      "    - apical plasma membrane\n",
      "  MF:\n",
      "    - organic anion transmembrane transporter activity\n",
      "    - organic anion transmembrane transporter activity\n",
      "    - bile acid transmembrane transporter activity\n",
      "    - prostaglandin transmembrane transporter activity\n",
      "    - sodium-independent organic anion transmembrane transporter activity\n",
      "    - sodium-independent organic anion transmembrane transporter activity\n",
      "    - sodium-independent organic anion transmembrane transporter activity\n",
      "    - transmembrane transporter activity\n",
      "UniProt: {'Swiss-Prot': 'O94956', 'TrEMBL': ['A0A1B0GX35', 'A0A0A0MTF1', 'E9PN87', 'E9PRW4', 'E9PIU9', 'H0YE11', 'E9PI53']}\n",
      "\n",
      "Gene: MPV17\n",
      "Symbol: MPV17\n",
      "Name: mitochondrial inner membrane protein MPV17\n",
      "Summary: This gene encodes a mitochondrial inner membrane protein that is implicated in the metabolism of reactive oxygen species. Mutations in this gene have been associated with the hepatocerebral form of mitochondrial DNA depletion syndrome (MDDS). [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - mitochondrial genome maintenance\n",
      "    - glomerular basement membrane development\n",
      "    - cellular response to reactive oxygen species\n",
      "    - homeostatic process\n",
      "    - inner ear development\n",
      "    - transmembrane transport\n",
      "    - regulation of mitochondrial DNA metabolic process\n",
      "    - regulation of mitochondrial DNA metabolic process\n",
      "    - regulation of reactive oxygen species metabolic process\n",
      "  CC:\n",
      "    - cytoplasm\n",
      "    - mitochondrion\n",
      "    - mitochondrion\n",
      "    - mitochondrion\n",
      "    - mitochondrial inner membrane\n",
      "    - peroxisome\n",
      "    - peroxisome\n",
      "    - peroxisomal membrane\n",
      "    - cytosol\n",
      "  MF:\n",
      "    - channel activity\n",
      "    - channel activity\n",
      "UniProt: {'Swiss-Prot': 'P39210', 'TrEMBL': ['B5MC53', 'A0A0S2Z3Z9', 'B5MC10', 'G5E9F5', 'B5MCF8', 'F8WEL3', 'A8MPV4', 'A8MTD3', 'H0Y6M5', 'C9J473', 'E7EX18']}\n",
      "\n",
      "Gene: GSTT1\n",
      "Symbol: GSTT1\n",
      "Name: glutathione S-transferase theta 1\n",
      "Summary: The protein encoded by this gene, glutathione S-transferase (GST) theta 1 (GSTT1), is a member of a superfamily of proteins that catalyze the conjugation of reduced glutathione to a variety of electrophilic and hydrophobic compounds. Human GSTs can be divided into five main classes: alpha, mu, pi, theta, and zeta. The theta class includes GSTT1, GSTT2, and GSTT2B. GSTT1 and GSTT2/GSTT2B share 55% amino acid sequence identity and may play a role in human carcinogenesis. The GSTT1 gene is haplotype-specific and is absent from 38% of the population. Alternative splicing of this gene results in multiple transcript variants. [provided by RefSeq, Sep 2015].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - glutathione metabolic process\n",
      "    - glutathione metabolic process\n",
      "    - cellular oxidant detoxification\n",
      "  CC:\n",
      "    - cytoplasm\n",
      "    - cytosol\n",
      "    - cytosol\n",
      "    - extracellular exosome\n",
      "  MF:\n",
      "    - glutathione transferase activity\n",
      "    - glutathione transferase activity\n",
      "    - glutathione peroxidase activity\n",
      "    - protein binding\n",
      "UniProt: {'Swiss-Prot': 'P30711', 'TrEMBL': ['Q4W251', 'A0A0G2JRN4', 'Q4GZS9', 'A0A0G2JRQ5', 'A0A0G2JRJ5', 'A0A0G2JMS2', 'A0A0G2JQM0', 'A0A0G2JQD8', 'A0A1B0GXB6', 'A0A0G2JQD2', 'A0A0G2JRG0']}\n",
      "\n",
      "Gene: DNAJB12\n",
      "Symbol: DNAJB12\n",
      "Name: DnaJ heat shock protein family (Hsp40) member B12\n",
      "Summary: DNAJB12 belongs to the evolutionarily conserved DNAJ/HSP40 family of proteins, which regulate molecular chaperone activity by stimulating ATPase activity. DNAJ proteins may have up to 3 distinct domains: a conserved 70-amino acid J domain, usually at the N terminus; a glycine/phenylalanine (G/F)-rich region; and a cysteine-rich domain containing 4 motifs resembling a zinc finger domain (Ohtsuka and Hata, 2000 [PubMed 11147971]).[supplied by OMIM, Mar 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - ERAD pathway\n",
      "    - chaperone cofactor-dependent protein refolding\n",
      "    - chaperone cofactor-dependent protein refolding\n",
      "    - protein-containing complex assembly\n",
      "    - cellular response to misfolded protein\n",
      "    - cellular response to misfolded protein\n",
      "  CC:\n",
      "    - endoplasmic reticulum\n",
      "    - endoplasmic reticulum membrane\n",
      "    - endoplasmic reticulum membrane\n",
      "    - membrane\n",
      "    - nuclear membrane\n",
      "  MF:\n",
      "    - Hsp70 protein binding\n",
      "    - Hsp70 protein binding\n",
      "UniProt: {'Swiss-Prot': 'Q9NXW2', 'TrEMBL': ['V9GYN7', 'J3KPS0', 'V9GY70']}\n",
      "\n",
      "Gene: CACNA2D1\n",
      "Symbol: CACNA2D1\n",
      "Name: calcium voltage-gated channel auxiliary subunit alpha2delta 1\n",
      "Summary: The preproprotein encoded by this gene is cleaved into multiple chains that comprise the alpha-2 and delta subunits of the voltage-dependent calcium channel complex. Calcium channels mediate the influx of calcium ions into the cell upon membrane polarization. Mutations in this gene can cause cardiac deficiencies, including Brugada syndrome and short QT syndrome. Alternate splicing results in multiple transcript variants, some of which may lack the delta subunit portion. [provided by RefSeq, Nov 2014].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - calcium ion transport\n",
      "    - regulation of calcium ion transport\n",
      "    - regulation of ventricular cardiac muscle cell membrane repolarization\n",
      "    - calcium ion transport into cytosol\n",
      "    - calcium ion transmembrane transport via high voltage-gated calcium channel\n",
      "    - cardiac muscle cell action potential involved in contraction\n",
      "    - membrane depolarization during bundle of His cell action potential\n",
      "    - regulation of heart rate by cardiac conduction\n",
      "    - calcium ion import across plasma membrane\n",
      "    - regulation of membrane repolarization during action potential\n",
      "    - regulation of calcium ion transmembrane transport via high voltage-gated calcium channel\n",
      "    - cellular response to amyloid-beta\n",
      "  CC:\n",
      "    - plasma membrane\n",
      "    - voltage-gated calcium channel complex\n",
      "    - sarcoplasmic reticulum\n",
      "    - presynaptic active zone membrane\n",
      "    - extracellular exosome\n",
      "    - GABA-ergic synapse\n",
      "    - neuronal dense core vesicle\n",
      "    - L-type voltage-gated calcium channel complex\n",
      "    - L-type voltage-gated calcium channel complex\n",
      "  MF:\n",
      "    - voltage-gated calcium channel activity\n",
      "    - voltage-gated calcium channel activity\n",
      "    - voltage-gated calcium channel activity\n",
      "    - voltage-gated calcium channel activity\n",
      "    - metal ion binding\n",
      "    - voltage-gated calcium channel activity involved in cardiac muscle cell action potential\n",
      "    - voltage-gated calcium channel activity involved in bundle of His cell action potential\n",
      "UniProt: {'Swiss-Prot': 'P54289', 'TrEMBL': ['Q9UIU0', 'E7ERK3', 'H0Y715', 'A0A994J5M8', 'A0A994J595']}\n",
      "\n",
      "Gene: MPI\n",
      "Symbol: MPI\n",
      "Name: mannose phosphate isomerase\n",
      "Summary: Phosphomannose isomerase catalyzes the interconversion of fructose-6-phosphate and mannose-6-phosphate and plays a critical role in maintaining the supply of D-mannose derivatives, which are required for most glycosylation reactions. Mutations in the MPI gene were found in patients with carbohydrate-deficient glycoprotein syndrome, type Ib. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Jan 2014].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - GDP-mannose biosynthetic process\n",
      "    - GDP-mannose biosynthetic process\n",
      "    - mannose to fructose-6-phosphate catabolic process\n",
      "    - mannose to fructose-6-phosphate catabolic process\n",
      "    - GDP-D-mannose biosynthetic process from fructose-6-phosphate\n",
      "  CC:\n",
      "    - cytosol\n",
      "    - cytosol\n",
      "    - extracellular exosome\n",
      "  MF:\n",
      "    - mannose-6-phosphate isomerase activity\n",
      "    - mannose-6-phosphate isomerase activity\n",
      "    - mannose-6-phosphate isomerase activity\n",
      "    - mannose-6-phosphate isomerase activity\n",
      "    - protein binding\n",
      "    - zinc ion binding\n",
      "UniProt: {'Swiss-Prot': 'P34949', 'TrEMBL': ['H3BPP3', 'H3BPB8', 'F5GX71', 'Q8NHZ6', 'B4DW50', 'B4DYB8', 'H3BM77', 'H3BN01', 'H3BMZ9', 'H3BPM5', 'H3BPU7', 'H3BP57', 'H3BT46', 'H3BT48', 'H3BU66', 'H3BUG1', 'H3BQX0', 'H3BNY8', 'H3BUZ9']}\n",
      "\n",
      "Gene: NUDT19\n",
      "Symbol: NUDT19\n",
      "Name: nudix hydrolase 19\n",
      "Summary: Predicted to enable magnesium ion binding activity. Predicted to be involved in purine nucleoside bisphosphate metabolic process. Predicted to be located in cytosol and peroxisomal matrix. [provided by Alliance of Genome Resources, Apr 2022]\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - fatty acid catabolic process\n",
      "    - coenzyme A catabolic process\n",
      "    - medium-chain fatty-acyl-CoA catabolic process\n",
      "    - butyryl-CoA catabolic process\n",
      "    - succinyl-CoA catabolic process\n",
      "    - propionyl-CoA metabolic process\n",
      "    - malonyl-CoA catabolic process\n",
      "  CC:\n",
      "    - mitochondrion\n",
      "    - peroxisomal matrix\n",
      "    - cytosol\n",
      "  MF:\n",
      "    - magnesium ion binding\n",
      "    - coenzyme A diphosphatase activity\n",
      "UniProt: {'Swiss-Prot': 'A8MXV4'}\n",
      "\n",
      "Gene: ZFP82\n",
      "Symbol: ZFP82\n",
      "Name: ZFP82 zinc finger protein\n",
      "Summary: Predicted to enable DNA-binding transcription factor activity, RNA polymerase II-specific and RNA polymerase II cis-regulatory region sequence-specific DNA binding activity. Predicted to be involved in regulation of transcription by RNA polymerase II. Predicted to be active in nucleus. [provided by Alliance of Genome Resources, Apr 2022]\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - regulation of transcription by RNA polymerase II\n",
      "  CC:\n",
      "    - nucleus\n",
      "  MF:\n",
      "    - RNA polymerase II cis-regulatory region sequence-specific DNA binding\n",
      "    - DNA-binding transcription factor activity, RNA polymerase II-specific\n",
      "    - zinc ion binding\n",
      "UniProt: {'Swiss-Prot': 'Q8N141', 'TrEMBL': ['K7ERD5', 'D3Y299']}\n",
      "\n",
      "Gene: PPAT\n",
      "Symbol: PPAT\n",
      "Name: phosphoribosyl pyrophosphate amidotransferase\n",
      "Summary: The protein encoded by this gene is a member of the purine/pyrimidine phosphoribosyltransferase family. It is a regulatory allosteric enzyme that catalyzes the first step of de novo purine nucleotide biosythetic pathway. This gene and PAICS/AIRC gene, a bifunctional enzyme catalyzing steps six and seven of this pathway, are located in close proximity on chromosome 4, and divergently transcribed from an intergenic region. [provided by RefSeq, Mar 2011].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - purine nucleotide biosynthetic process\n",
      "    - GMP biosynthetic process\n",
      "    - 'de novo' IMP biosynthetic process\n",
      "    - purine nucleobase biosynthetic process\n",
      "    - 'de novo' AMP biosynthetic process\n",
      "    - 'de novo' XMP biosynthetic process\n",
      "  CC:\n",
      "    - cytosol\n",
      "  MF:\n",
      "    - amidophosphoribosyltransferase activity\n",
      "    - protein binding\n",
      "    - metal ion binding\n",
      "    - 4 iron, 4 sulfur cluster binding\n",
      "UniProt: {'Swiss-Prot': 'Q06203', 'TrEMBL': ['Q59G63', 'A8K4H7', 'D6RE15', 'D6RCC8']}\n",
      "\n",
      "Gene: ALDH1A2\n",
      "Symbol: ALDH1A2\n",
      "Name: aldehyde dehydrogenase 1 family member A2\n",
      "Summary: This protein belongs to the aldehyde dehydrogenase family of proteins. The product of this gene is an enzyme that catalyzes the synthesis of retinoic acid (RA) from retinaldehyde. Retinoic acid, the active derivative of vitamin A (retinol), is a hormonal signaling molecule that functions in developing and adult tissues. The studies of a similar mouse gene suggest that this enzyme and the cytochrome CYP26A1, concurrently establish local embryonic retinoic acid levels which facilitate posterior organ development and prevent spina bifida. Four transcript variants encoding distinct isoforms have been identified for this gene. [provided by RefSeq, May 2011].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - blood vessel development\n",
      "    - kidney development\n",
      "    - liver development\n",
      "    - retinoic acid biosynthetic process\n",
      "    - retinoic acid biosynthetic process\n",
      "    - heart morphogenesis\n",
      "    - vitamin A metabolic process\n",
      "    - midgut development\n",
      "    - cell population proliferation\n",
      "    - positive regulation of cell population proliferation\n",
      "    - negative regulation of cell population proliferation\n",
      "    - determination of bilateral symmetry\n",
      "    - anterior/posterior pattern specification\n",
      "    - proximal/distal pattern formation\n",
      "    - positive regulation of gene expression\n",
      "    - neural crest cell development\n",
      "    - morphogenesis of embryonic epithelium\n",
      "    - neural tube development\n",
      "    - pituitary gland development\n",
      "    - neuron differentiation\n",
      "    - lung development\n",
      "    - hindbrain development\n",
      "    - pancreas development\n",
      "    - embryonic camera-type eye development\n",
      "    - response to estradiol\n",
      "    - response to retinoic acid\n",
      "    - response to vitamin A\n",
      "    - response to cytokine\n",
      "    - embryonic forelimb morphogenesis\n",
      "    - ureter maturation\n",
      "    - retinol metabolic process\n",
      "    - retinoic acid metabolic process\n",
      "    - retinoic acid metabolic process\n",
      "    - retinal metabolic process\n",
      "    - 9-cis-retinoic acid biosynthetic process\n",
      "    - positive regulation of apoptotic process\n",
      "    - retinoic acid receptor signaling pathway\n",
      "    - embryonic digestive tract development\n",
      "    - cardiac muscle tissue development\n",
      "    - protein homotetramerization\n",
      "    - face development\n",
      "    - cellular response to retinoic acid\n",
      "    - regulation of vascular endothelial cell proliferation\n",
      "  CC:\n",
      "    - cytoplasm\n",
      "    - cytosol\n",
      "    - perinuclear region of cytoplasm\n",
      "  MF:\n",
      "    - retinal dehydrogenase activity\n",
      "    - retinal dehydrogenase activity\n",
      "    - 3-chloroallyl aldehyde dehydrogenase activity\n",
      "    - aldehyde dehydrogenase (NAD+) activity\n",
      "    - retinal binding\n",
      "UniProt: {'Swiss-Prot': 'O94788', 'TrEMBL': ['H0YMG7', 'Q9UED3', 'A0A0G2JL97', 'H0YMT5', 'H0YKL3', 'E9PF31']}\n",
      "\n",
      "Gene: CYP2D6\n",
      "Symbol: LOC110740340\n",
      "Name: CYP2D6 promoter\n",
      "Summary: This region represents the promoter of the cytochrome P450 family 2 subfamily D member 6 (CYP2D6) gene. This promoter is induced by binding of hepatocyte nuclear factor 4 alpha. Activity of this sequence may be further stimulated by Kruppel like factor 9 and peroxisome proliferator activated receptor alpha, and is suppressed by nitric oxide and estrogen. Naturally-occurring polymorphic sites in this region, including rs1080985 in the distal promoter, are associated with expression of CYP2D6 and may have phenotypic consequences. [provided by RefSeq, Jul 2017]\n",
      "\n",
      "Gene: METTL23\n",
      "Symbol: METTL23\n",
      "Name: methyltransferase 23, arginine\n",
      "Summary: The protein encoded by this gene functions as a transcription factor regulator in the transcriptional pathway for human cognition. It is a partner of the alpha subunit of the GA-binding protein transcription factor. Mutations in this gene cause mild autosomal recessive intellectual disability. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Nov 2014].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - methylation\n",
      "    - epigenetic regulation of gene expression\n",
      "    - epigenetic programming in the zygotic pronuclei\n",
      "    - epigenetic programing of male pronucleus\n",
      "    - positive regulation of transcription by RNA polymerase II\n",
      "    - cognition\n",
      "  CC:\n",
      "    - female pronucleus\n",
      "    - male pronucleus\n",
      "    - nucleus\n",
      "    - nucleus\n",
      "    - cytoplasm\n",
      "    - cytoplasm\n",
      "    - protein-containing complex\n",
      "  MF:\n",
      "    - protein binding\n",
      "    - heat shock protein binding\n",
      "    - protein-arginine omega-N asymmetric methyltransferase activity\n",
      "    - histone H3R17 methyltransferase activity\n",
      "    - histone H3R17 methyltransferase activity\n",
      "    - DNA-binding transcription factor binding\n",
      "UniProt: {'Swiss-Prot': 'Q86XA0', 'TrEMBL': ['K7ERS2', 'Q8N712', 'K7EPR8', 'K7ENU9', 'K7EPT5', 'K7EJ00', 'K7ESG7', 'K7EL83', 'K7EMR3']}\n",
      "\n",
      "Gene: ALG1\n",
      "Symbol: ALG1\n",
      "Name: ALG1 chitobiosyldiphosphodolichol beta-mannosyltransferase\n",
      "Summary: The enzyme encoded by this gene catalyzes the first mannosylation step in the biosynthesis of lipid-linked oligosaccharides. This gene is mutated in congenital disorder of glycosylation type Ik. [provided by RefSeq, Dec 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - protein glycosylation\n",
      "    - protein N-linked glycosylation\n",
      "    - dolichol-linked oligosaccharide biosynthetic process\n",
      "    - dolichol-linked oligosaccharide biosynthetic process\n",
      "  CC:\n",
      "    - endoplasmic reticulum\n",
      "    - endoplasmic reticulum membrane\n",
      "    - membrane\n",
      "    - cytoplasmic side of endoplasmic reticulum membrane\n",
      "  MF:\n",
      "    - mannosyltransferase activity\n",
      "    - chitobiosyldiphosphodolichol beta-mannosyltransferase activity\n",
      "    - chitobiosyldiphosphodolichol beta-mannosyltransferase activity\n",
      "UniProt: {'Swiss-Prot': 'Q9BT22', 'TrEMBL': ['A0A804HLB7', 'A0A804HJL6', 'A0A804HKK4', 'K7EQK1', 'K7EID2', 'A0A804HJE3', 'K7EPU3', 'A0A804HIG6', 'A0A804HKJ4', 'A0A804HKX7', 'A0A804HKU4', 'A0A804HJQ1', 'A0A804HHW1']}\n",
      "\n",
      "Gene: STK10\n",
      "Symbol: STK10\n",
      "Name: serine/threonine kinase 10\n",
      "Summary: This gene encodes a member of the Ste20 family of serine/threonine protein kinases, and is similar to several known polo-like kinase kinases. The protein can associate with and phosphorylate polo-like kinase 1, and overexpression of a kinase-dead version of the protein interferes with normal cell cycle progression. The kinase can also negatively regulate interleukin 2 expression in T-cells via the mitogen activated protein kinase kinase 1 pathway. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - chromatin remodeling\n",
      "    - protein phosphorylation\n",
      "    - intracellular signal transduction\n",
      "    - protein autophosphorylation\n",
      "    - lymphocyte aggregation\n",
      "    - regulation of lymphocyte migration\n",
      "    - regulation of lymphocyte migration\n",
      "  CC:\n",
      "    - nucleoplasm\n",
      "    - cytoplasm\n",
      "    - cytosol\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - nuclear body\n",
      "    - specific granule membrane\n",
      "    - extracellular exosome\n",
      "  MF:\n",
      "    - protein serine/threonine kinase activity\n",
      "    - protein serine/threonine kinase activity\n",
      "    - protein serine/threonine kinase activity\n",
      "    - 3-phosphoinositide-dependent protein kinase activity\n",
      "    - DNA-dependent protein kinase activity\n",
      "    - AMP-activated protein kinase activity\n",
      "    - eukaryotic translation initiation factor 2alpha kinase activity\n",
      "    - ribosomal protein S6 kinase activity\n",
      "    - protein binding\n",
      "    - ATP binding\n",
      "    - histone H3S10 kinase activity\n",
      "    - histone H3T11 kinase activity\n",
      "    - histone H3T6 kinase activity\n",
      "    - histone H2AXS139 kinase activity\n",
      "    - identical protein binding\n",
      "    - protein homodimerization activity\n",
      "    - histone H3S28 kinase activity\n",
      "    - histone H4S1 kinase activity\n",
      "    - histone H2AS1 kinase activity\n",
      "    - histone H2BS14 kinase activity\n",
      "    - histone H3T3 kinase activity\n",
      "    - histone H2AS121 kinase activity\n",
      "    - Rho-dependent protein serine/threonine kinase activity\n",
      "    - protein serine kinase activity\n",
      "    - histone H2BS36 kinase activity\n",
      "    - histone H3S57 kinase activity\n",
      "    - histone H3T45 kinase activity\n",
      "    - histone H2AT120 kinase activity\n",
      "UniProt: {'Swiss-Prot': 'O94804', 'TrEMBL': 'H0YB71'}\n",
      "\n",
      "Gene: SERPINA7\n",
      "Symbol: SERPINA7\n",
      "Name: serpin family A member 7\n",
      "Summary: There are three proteins including thyroxine-binding globulin (TBG), transthyretin and albumin responsible for carrying the thyroid hormones thyroxine (T4) and 3,5,3'-triiodothyronine (T3) in the bloodstream. This gene encodes the major thyroid hormone transport protein, TBG, in serum. It belongs to the serpin family in genomics, but the protein has no inhibitory function like many other members of the serpin family. Mutations in this gene result in TGB deficiency, which has been classified as partial deficiency, complete deficiency, and excess, based on the level of serum TBG. Alternatively spliced transcript variants encoding different isoforms have been found, but the full-length nature of these variants has not been determined.[provided by RefSeq, Jun 2012].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - thyroid hormone transport\n",
      "  CC:\n",
      "    - extracellular region\n",
      "    - extracellular space\n",
      "    - extracellular exosome\n",
      "  MF:\n",
      "    - serine-type endopeptidase inhibitor activity\n",
      "UniProt: {'Swiss-Prot': 'P05543'}\n",
      "\n",
      "Gene: LAMC3\n",
      "Symbol: LAMC3\n",
      "Name: laminin subunit gamma 3\n",
      "Summary: Laminins, a family of extracellular matrix glycoproteins, are the major noncollagenous constituent of basement membranes. They have been implicated in a wide variety of biological processes including cell adhesion, differentiation, migration, signaling, neurite outgrowth and metastasis. Laminins are composed of 3 non identical chains: laminin alpha, beta and gamma (formerly A, B1, and B2, respectively) and they form a cruciform structure consisting of 3 short arms, each formed by a different chain, and a long arm composed of all 3 chains. Each laminin chain is a multidomain protein encoded by a distinct gene. Several isoforms of each chain have been described. Different alpha, beta and gamma chain isomers combine to give rise to different heterotrimeric laminin isoforms which are designated by Arabic numerals in the order of their discovery, i.e. alpha1beta1gamma1 heterotrimer is laminin 1. The biological functions of the different chains and trimer molecules are largely unknown, but some of the chains have been shown to differ with respect to their tissue distribution, presumably reflecting diverse functions in vivo. This gene encodes the gamma chain isoform laminin, gamma 3. The gamma 3 chain is most similar to the gamma 1 chain, and contains all the 6 domains expected of the gamma chain. It is a component of laminin 12. The gamma 3 chain is broadly expressed in skin, heart, lung, and the reproductive tracts. In skin, it is seen within the basement membrane of the dermal-epidermal junction at points of nerve penetration. Gamma 3 is also a prominent element of the apical surface of ciliated epithelial cells of lung, oviduct, epididymis, ductus deferens, and seminiferous tubules. The distribution of gamma 3-containing laminins along ciliated epithelial surfaces suggests that the apical laminins are important in the morphogenesis and structural stability of the ciliated processes of these cells. [provided by RefSeq, Aug 2011].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - cell morphogenesis\n",
      "    - cell adhesion\n",
      "    - visual perception\n",
      "    - astrocyte development\n",
      "    - retina development in camera-type eye\n",
      "  CC:\n",
      "    - extracellular region\n",
      "    - basement membrane\n",
      "    - membrane\n",
      "    - extracellular matrix\n",
      "  MF:\n",
      "    - structural molecule activity\n",
      "UniProt: {'Swiss-Prot': 'Q9Y6N6', 'TrEMBL': ['A0A7I2V3S4', 'Q8N2D6', 'H7BY04', 'A0A7I2YQP4']}\n",
      "\n",
      "Gene: ALDH4A1\n",
      "Symbol: ALDH4A1\n",
      "Name: aldehyde dehydrogenase 4 family member A1\n",
      "Summary: This protein belongs to the aldehyde dehydrogenase family of proteins. This enzyme is a mitochondrial matrix NAD-dependent dehydrogenase which catalyzes the second step of the proline degradation pathway, converting pyrroline-5-carboxylate to glutamate. Deficiency of this enzyme is associated with type II hyperprolinemia, an autosomal recessive disorder characterized by accumulation of delta-1-pyrroline-5-carboxylate (P5C) and proline. Alternatively spliced transcript variants encoding different isoforms have been identified for this gene. [provided by RefSeq, Jun 2009].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - proline metabolic process\n",
      "    - proline catabolic process\n",
      "    - proline catabolic process to glutamate\n",
      "    - 4-hydroxyproline catabolic process\n",
      "  CC:\n",
      "    - mitochondrion\n",
      "    - mitochondrion\n",
      "    - mitochondrial matrix\n",
      "    - cytosol\n",
      "  MF:\n",
      "    - 1-pyrroline-5-carboxylate dehydrogenase activity\n",
      "    - aldehyde dehydrogenase (NAD+) activity\n",
      "    - protein binding\n",
      "    - electron transfer activity\n",
      "    - identical protein binding\n",
      "UniProt: {'Swiss-Prot': 'P30038', 'TrEMBL': ['Q5TF55', 'A0A024RAC7']}\n",
      "\n",
      "Gene: ADH1A\n",
      "Symbol: ADH1A\n",
      "Name: alcohol dehydrogenase 1A (class I), alpha polypeptide\n",
      "Summary: This gene encodes a member of the alcohol dehydrogenase family. The encoded protein is the alpha subunit of class I alcohol dehydrogenase, which consists of several homo- and heterodimers of alpha, beta and gamma subunits. Alcohol dehydrogenases catalyze the oxidation of alcohols to aldehydes. This gene is active in the liver in early fetal life but only weakly active in adult liver. This gene is found in a cluster with six additional alcohol dehydrogenase genes, including those encoding the beta and gamma subunits, on the long arm of chromosome 4. Mutations in this gene may contribute to variation in certain personality traits and substance dependence. [provided by RefSeq, Nov 2010].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - alcohol metabolic process\n",
      "    - alcohol metabolic process\n",
      "    - retinol metabolic process\n",
      "    - retinoic acid metabolic process\n",
      "  CC:\n",
      "    - nucleoplasm\n",
      "    - cytosol\n",
      "    - cytosol\n",
      "    - cytosol\n",
      "    - plasma membrane\n",
      "    - cilium\n",
      "    - ciliary basal body\n",
      "  MF:\n",
      "    - alcohol dehydrogenase (NAD+) activity\n",
      "    - all-trans-retinol dehydrogenase (NAD+) activity\n",
      "    - protein binding\n",
      "    - zinc ion binding\n",
      "    - butanol dehydrogenase (NAD+) activity\n",
      "UniProt: {'Swiss-Prot': 'P07327'}\n",
      "\n",
      "Gene: ABCC5\n",
      "Symbol: ABCC5\n",
      "Name: ATP binding cassette subfamily C member 5\n",
      "Summary: The protein encoded by this gene is a member of the superfamily of ATP-binding cassette (ABC) transporters. ABC proteins transport various molecules across extra- and intra-cellular membranes. ABC genes are divided into seven distinct subfamilies (ABC1, MDR/TAP, MRP, ALD, OABP, GCN20, White). This protein is a member of the MRP subfamily which is involved in multi-drug resistance. This protein functions in the cellular export of its substrate, cyclic nucleotides. This export contributes to the degradation of phosphodiesterases and possibly an elimination pathway for cyclic nucleotides. Studies show that this protein provides resistance to thiopurine anticancer drugs, 6-mercatopurine and thioguanine, and the anti-HIV drug 9-(2-phosphonylmethoxyethyl)adenine. This protein may be involved in resistance to thiopurines in acute lymphoblastic leukemia and antiretroviral nucleoside analogs in HIV-infected patients. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Feb 2016].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - xenobiotic metabolic process\n",
      "    - xenobiotic transmembrane transport\n",
      "    - purine nucleotide transport\n",
      "    - hyaluronan biosynthetic process\n",
      "    - glutathione transmembrane transport\n",
      "    - heme transmembrane transport\n",
      "    - xenobiotic transport\n",
      "    - xenobiotic transport\n",
      "    - transmembrane transport\n",
      "    - transmembrane transport\n",
      "    - cAMP transport\n",
      "    - cGMP transport\n",
      "    - monoatomic anion transmembrane transport\n",
      "    - folate transmembrane transport\n",
      "    - export across plasma membrane\n",
      "    - transport across blood-brain barrier\n",
      "  CC:\n",
      "    - Golgi lumen\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - endosome membrane\n",
      "    - membrane\n",
      "    - membrane\n",
      "    - basolateral plasma membrane\n",
      "    - basolateral plasma membrane\n",
      "    - apical plasma membrane\n",
      "  MF:\n",
      "    - ATP binding\n",
      "    - organic anion transmembrane transporter activity\n",
      "    - ABC-type xenobiotic transporter activity\n",
      "    - purine nucleotide transmembrane transporter activity\n",
      "    - heme transmembrane transporter activity\n",
      "    - efflux transmembrane transporter activity\n",
      "    - ATP hydrolysis activity\n",
      "    - macromolecule transmembrane transporter activity\n",
      "    - glutathione transmembrane transporter activity\n",
      "    - ATPase-coupled transmembrane transporter activity\n",
      "    - ATPase-coupled transmembrane transporter activity\n",
      "    - xenobiotic transmembrane transporter activity\n",
      "    - ATPase-coupled inorganic anion transmembrane transporter activity\n",
      "    - carbohydrate derivative transmembrane transporter activity\n",
      "UniProt: {'Swiss-Prot': 'O15440', 'TrEMBL': ['H7C271', 'A5PKY6', 'F8WCY8', 'C9JZL5', 'E9PET4', 'F8WDH3']}\n",
      "\n",
      "Gene: ALDH6A1\n",
      "Symbol: ALDH6A1\n",
      "Name: aldehyde dehydrogenase 6 family member A1\n",
      "Summary: This gene encodes a member of the aldehyde dehydrogenase protein family. The encoded protein is a mitochondrial methylmalonate semialdehyde dehydrogenase that plays a role in the valine and pyrimidine catabolic pathways. This protein catalyzes the irreversible oxidative decarboxylation of malonate and methylmalonate semialdehydes to acetyl- and propionyl-CoA. Methylmalonate semialdehyde dehydrogenase deficiency is characterized by elevated beta-alanine, 3-hydroxypropionic acid, and both isomers of 3-amino and 3-hydroxyisobutyric acids in urine organic acids. Alternate splicing results in multiple transcript variants. [provided by RefSeq, Jun 2013].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - thymine catabolic process\n",
      "    - thymine catabolic process\n",
      "    - valine metabolic process\n",
      "    - valine catabolic process\n",
      "    - valine catabolic process\n",
      "    - branched-chain amino acid catabolic process\n",
      "    - thymine metabolic process\n",
      "    - brown fat cell differentiation\n",
      "  CC:\n",
      "    - nucleoplasm\n",
      "    - mitochondrion\n",
      "    - mitochondrion\n",
      "    - mitochondrion\n",
      "    - mitochondrion\n",
      "    - mitochondrial matrix\n",
      "  MF:\n",
      "    - RNA binding\n",
      "    - methylmalonate-semialdehyde dehydrogenase (acylating, NAD) activity\n",
      "    - methylmalonate-semialdehyde dehydrogenase (acylating, NAD) activity\n",
      "    - methylmalonate-semialdehyde dehydrogenase (acylating, NAD) activity\n",
      "    - methylmalonate-semialdehyde dehydrogenase (acylating, NAD) activity\n",
      "    - malonate-semialdehyde dehydrogenase (acetylating) activity\n",
      "UniProt: {'Swiss-Prot': 'Q02252', 'TrEMBL': ['G3V4Z4', 'A0A024R6G4']}\n",
      "\n",
      "Gene: CYP27A1\n",
      "Symbol: CYP27A1\n",
      "Name: cytochrome P450 family 27 subfamily A member 1\n",
      "Summary: This gene encodes a member of the cytochrome P450 superfamily of enzymes. The cytochrome P450 proteins are monooxygenases which catalyze many reactions involved in drug metabolism and synthesis of cholesterol, steroids and other lipids. This mitochondrial protein oxidizes cholesterol intermediates as part of the bile synthesis pathway. Since the conversion of cholesterol to bile acids is the major route for removing cholesterol from the body, this protein is important for overall cholesterol homeostasis. Mutations in this gene cause cerebrotendinous xanthomatosis, a rare autosomal recessive lipid storage disease. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - bile acid biosynthetic process\n",
      "    - bile acid biosynthetic process\n",
      "    - bile acid biosynthetic process\n",
      "    - cholesterol catabolic process\n",
      "    - cholesterol catabolic process\n",
      "    - cholesterol metabolic process\n",
      "    - sterol metabolic process\n",
      "    - calcitriol biosynthetic process from calciol\n",
      "    - calcitriol biosynthetic process from calciol\n",
      "  CC:\n",
      "    - mitochondrion\n",
      "    - mitochondrion\n",
      "    - mitochondrial inner membrane\n",
      "    - mitochondrial matrix\n",
      "  MF:\n",
      "    - iron ion binding\n",
      "    - steroid hydroxylase activity\n",
      "    - heme binding\n",
      "    - vitamin D3 25-hydroxylase activity\n",
      "    - vitamin D3 25-hydroxylase activity\n",
      "    - cholesterol 26-hydroxylase activity\n",
      "    - cholesterol 26-hydroxylase activity\n",
      "    - cholestanetetraol 26-dehydrogenase activity\n",
      "UniProt: {'Swiss-Prot': 'Q02318', 'TrEMBL': ['C9J1K5', 'F8WD90']}\n",
      "\n",
      "Gene: NNMT\n",
      "Symbol: NNMT\n",
      "Name: nicotinamide N-methyltransferase\n",
      "Summary: N-methylation is one method by which drug and other xenobiotic compounds are metabolized by the liver. This gene encodes the protein responsible for this enzymatic activity which uses S-adenosyl methionine as the methyl donor. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - nicotinamide metabolic process\n",
      "    - response to xenobiotic stimulus\n",
      "    - animal organ regeneration\n",
      "    - methylation\n",
      "    - NAD biosynthesis via nicotinamide riboside salvage pathway\n",
      "    - positive regulation of gluconeogenesis\n",
      "    - positive regulation of protein deacetylation\n",
      "  CC:\n",
      "    - cytosol\n",
      "    - cytosol\n",
      "  MF:\n",
      "    - nicotinamide N-methyltransferase activity\n",
      "    - nicotinamide N-methyltransferase activity\n",
      "    - nicotinamide N-methyltransferase activity\n",
      "    - nicotinamide N-methyltransferase activity\n",
      "    - pyridine N-methyltransferase activity\n",
      "UniProt: {'Swiss-Prot': 'P40261', 'TrEMBL': 'B0YJ53'}\n",
      "\n",
      "Gene: ITPKC\n",
      "Symbol: ITPKC\n",
      "Name: inositol-trisphosphate 3-kinase C\n",
      "Summary: This gene encodes a member of the inositol 1,4,5-trisphosphate [Ins(1,4,5)P(3)] 3-kinase family of enzymes that catalyze the phosphorylation of inositol 1,4,5-trisphosphate to 1,3,4,5-tetrakisphosphate. The encoded protein is localized to the nucleus and cytoplasm and has both nuclear import and nuclear export activity. Single nucleotide polymorphisms in this gene are associated with Kawasaki disease.[provided by RefSeq, Sep 2009].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - inositol phosphate biosynthetic process\n",
      "    - phosphatidylinositol phosphate biosynthetic process\n",
      "    - phosphatidylinositol phosphate biosynthetic process\n",
      "    - cellular response to calcium ion\n",
      "  CC:\n",
      "    - nucleus\n",
      "    - nucleus\n",
      "    - cytoplasm\n",
      "    - cytoplasm\n",
      "    - cytosol\n",
      "    - nuclear speck\n",
      "  MF:\n",
      "    - inositol hexakisphosphate kinase activity\n",
      "    - calmodulin binding\n",
      "    - ATP binding\n",
      "    - inositol-1,4,5-trisphosphate 3-kinase activity\n",
      "UniProt: {'Swiss-Prot': 'Q96DU7', 'TrEMBL': ['M0R2L7', 'A0A8V8TPP3', 'A0A8V8TNF9']}\n",
      "\n",
      "Gene: LAMB2\n",
      "Symbol: LAMB2\n",
      "Name: laminin subunit beta 2\n",
      "Summary: Laminins, a family of extracellular matrix glycoproteins, are the major noncollagenous constituent of basement membranes. They have been implicated in a wide variety of biological processes including cell adhesion, differentiation, migration, signaling, neurite outgrowth and metastasis. Laminins, composed of 3 non identical chains: laminin alpha, beta and gamma (formerly A, B1, and B2, respectively), form a cruciform structure consisting of 3 short arms, each formed by a different chain, and a long arm composed of all 3 chains. Each laminin chain is a multidomain protein encoded by a distinct gene. Several isoforms of each chain have been described. Different alpha, beta and gamma chain isomers combine to give rise to different heterotrimeric laminin isoforms which are designated by Arabic numerals in the order of their discovery, i.e. alpha1beta1gamma1 heterotrimer is laminin 1. The biological functions of the different chains and trimer molecules are largely unknown, but some of the chains have been shown to differ with respect to their tissue distribution, presumably reflecting diverse functions in vivo. This gene encodes the beta chain isoform laminin, beta 2. The beta 2 chain contains the 7 structural domains typical of beta chains of laminin, including the short alpha region. However, unlike beta 1 chain, beta 2 has a more restricted tissue distribution. It is enriched in the basement membrane of muscles at the neuromuscular junctions, kidney glomerulus and vascular smooth muscle. Transgenic mice in which the beta 2 chain gene was inactivated by homologous recombination, showed defects in the maturation of neuromuscular junctions and impairment of glomerular filtration. Alternative splicing involving a non consensus 5' splice site (gc) in the 5' UTR of this gene has been reported. It was suggested that inefficient splicing of this first intron, which does not change the protein sequence, results in a greater abundance of the unspliced form of the transcript than the spliced form. The full-length nature of the spliced transcript is not known. [provided by RefSeq, Aug 2011].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - cell adhesion\n",
      "    - axon guidance\n",
      "    - neuromuscular junction development\n",
      "    - visual perception\n",
      "    - astrocyte development\n",
      "    - Schwann cell development\n",
      "    - positive regulation of cell adhesion\n",
      "    - axon extension involved in regeneration\n",
      "    - positive regulation of muscle cell differentiation\n",
      "    - retina development in camera-type eye\n",
      "    - metanephric podocyte development\n",
      "    - metanephric glomerular basement membrane development\n",
      "    - regulation of basement membrane organization\n",
      "    - positive regulation of integrin-mediated signaling pathway\n",
      "  CC:\n",
      "    - extracellular region\n",
      "    - basement membrane\n",
      "    - basement membrane\n",
      "    - laminin-3 complex\n",
      "    - endoplasmic reticulum lumen\n",
      "    - neuromuscular junction\n",
      "    - synaptic cleft\n",
      "    - laminin-11 complex\n",
      "    - collagen-containing extracellular matrix\n",
      "    - collagen-containing extracellular matrix\n",
      "    - collagen-containing extracellular matrix\n",
      "    - extracellular exosome\n",
      "    - protein complex involved in cell-matrix adhesion\n",
      "  MF:\n",
      "    - integrin binding\n",
      "    - structural molecule activity\n",
      "    - extracellular matrix structural constituent\n",
      "    - extracellular matrix structural constituent\n",
      "    - protein binding\n",
      "    - structural constituent of synapse-associated extracellular matrix\n",
      "UniProt: {'Swiss-Prot': 'P55268', 'TrEMBL': 'F5H520'}\n",
      "\n",
      "Gene: RASAL1\n",
      "Symbol: RASAL1\n",
      "Name: RAS protein activator like 1\n",
      "Summary: The protein encoded by this gene is member of the GAP1 family of GTPase-activating proteins. These proteins stimulate the GTPase activity of normal RAS p21 but not its oncogenic counterpart. Acting as a suppressor of RAS function, the protein enhances the weak intrinsic GTPase activity of RAS proteins resulting in the inactive GDP-bound form of RAS, thereby allowing control of cellular proliferation and differentiation. This particular family member contains domains which are characteristic of the GAP1 subfamily of RasGAP proteins but, in contrast to the other GAP1 family members, this protein is strongly and selectively expressed in endocrine tissues. Alternatively spliced transcript variants that encode different isoforms have been described [provided by RefSeq, Jul 2010].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - signal transduction\n",
      "    - cell differentiation\n",
      "    - intracellular signal transduction\n",
      "    - negative regulation of Ras protein signal transduction\n",
      "    - cellular response to calcium ion\n",
      "    - positive regulation of dendrite extension\n",
      "  CC:\n",
      "    - cytosol\n",
      "    - cytosol\n",
      "    - plasma membrane\n",
      "  MF:\n",
      "    - GTPase activator activity\n",
      "    - phospholipid binding\n",
      "    - zinc ion binding\n",
      "UniProt: {'Swiss-Prot': 'O95294'}\n",
      "\n",
      "Gene: NPC1L1\n",
      "Symbol: NPC1L1\n",
      "Name: NPC1 like intracellular cholesterol transporter 1\n",
      "Summary: The protein encoded by this gene is a multi-pass membrane protein. It contains a conserved N-terminal Niemann-Pick C1 (NPC1) domain and a putative sterol-sensing domain (SSD) which includes a YQRL motif functioning as a plasma membrane to trans-Golgi network transport signal in other proteins. This protein takes up free cholesterol into cells through vesicular endocytosis and plays a critical role in the absorption of intestinal cholesterol. It also has the ability to transport alpha-tocopherol (vitamin E). The drug ezetimibe targets this protein and inhibits the absorption of intestinal cholesterol and alpha-tocopherol. In addition, this protein may play a critical role in regulating lipid metabolism. Polymorphic variations in this gene are associated with plasma total cholesterol and low-density lipoprotein cholesterol (LDL-C) levels and coronary heart disease (CHD) risk. Alternatively spliced transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Oct 2009].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - cholesterol biosynthetic process\n",
      "    - sterol transport\n",
      "    - intestinal cholesterol absorption\n",
      "    - intestinal cholesterol absorption\n",
      "    - cholesterol transport\n",
      "    - lipoprotein metabolic process\n",
      "    - vitamin E metabolic process\n",
      "    - cholesterol homeostasis\n",
      "    - vitamin transport\n",
      "    - cellular response to sterol depletion\n",
      "  CC:\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - apical plasma membrane\n",
      "    - cytoplasmic vesicle membrane\n",
      "  MF:\n",
      "    - protein binding\n",
      "    - vitamin E binding\n",
      "    - cholesterol binding\n",
      "    - cholesterol binding\n",
      "    - small GTPase binding\n",
      "    - myosin V binding\n",
      "    - protein homodimerization activity\n",
      "UniProt: {'Swiss-Prot': 'Q9UHC9', 'TrEMBL': ['A0A0C4DGG6', 'A0A0C4DFX6']}\n",
      "\n",
      "Gene: RAP2C\n",
      "Symbol: RAP2C\n",
      "Name: RAP2C, member of RAS oncogene family\n",
      "Summary: The protein encoded by this gene is a member of the Ras-related protein subfamily of the Ras GTPase superfamily. Members of this family are small GTPases that act as molecular switches to regulate cellular proliferation, differentiation, and apoptosis. This protein has been reported to activate in vitro transcriptional activity of the serum response element. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Sep 2012].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - microvillus assembly\n",
      "    - negative regulation of cell migration\n",
      "    - negative regulation of cell migration\n",
      "    - Rap protein signal transduction\n",
      "    - Rap protein signal transduction\n",
      "    - positive regulation of DNA-templated transcription\n",
      "    - regulation of protein tyrosine kinase activity\n",
      "    - establishment of endothelial intestinal barrier\n",
      "    - establishment of endothelial intestinal barrier\n",
      "  CC:\n",
      "    - cytoplasm\n",
      "    - cytosol\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - bicellular tight junction\n",
      "    - cell-cell contact zone\n",
      "    - recycling endosome membrane\n",
      "    - extracellular exosome\n",
      "    - tertiary granule membrane\n",
      "  MF:\n",
      "    - transcription coactivator activity\n",
      "    - GTPase activity\n",
      "    - G protein activity\n",
      "    - protein binding\n",
      "    - GTP binding\n",
      "    - GTP binding\n",
      "    - GDP binding\n",
      "    - GDP binding\n",
      "UniProt: {'Swiss-Prot': 'Q9Y3L5', 'TrEMBL': 'A0A087X2C3'}\n",
      "\n",
      "Gene: SLC22A18\n",
      "Symbol: SLC67A1-AS\n",
      "Name: SLC67A1 antisense RNA\n",
      "Summary: N/A\n",
      "\n",
      "Gene: PRRG2\n",
      "Symbol: PRRG2\n",
      "Name: proline rich and Gla domain 2\n",
      "Summary: The protein encoded by this gene is a single-pass transmembrane protein containing an N-terminal gamma-carboxyglutamic acid (Gla) domain and tandem Pro/Leu-Pro-Xaa-Tyr (PY) motifs at its C-terminal end. The Gla domain is exposed on the cell surface while the PY motifs are cytoplasmic. The PY motifs of the encoded protein have been shown to interact with YAP1, a WW domain-containing protein. Therefore, it is thought that the encoded protein may be part of a signal transduction pathway. Two transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Oct 2015].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - blood coagulation\n",
      "  CC:\n",
      "    - extracellular space\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "  MF:\n",
      "    - serine-type endopeptidase activity\n",
      "    - calcium ion binding\n",
      "    - protein binding\n",
      "UniProt: {'Swiss-Prot': 'O14669', 'TrEMBL': ['M0QZA1', 'M0QXL9']}\n",
      "\n",
      "Gene: COX16\n",
      "Symbol: COX16\n",
      "Name: cytochrome c oxidase assembly factor COX16\n",
      "Summary: Involved in mitochondrial cytochrome c oxidase assembly. Is integral component of mitochondrial inner membrane. [provided by Alliance of Genome Resources, Apr 2022]\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - mitochondrial cytochrome c oxidase assembly\n",
      "    - mitochondrial cytochrome c oxidase assembly\n",
      "    - mitochondrial cytochrome c oxidase assembly\n",
      "  CC:\n",
      "    - mitochondrion\n",
      "    - mitochondrial inner membrane\n",
      "    - mitochondrial inner membrane\n",
      "    - mitochondrial inner membrane\n",
      "  MF:\n",
      "    - molecular_function\n",
      "    - protein binding\n",
      "UniProt: {'Swiss-Prot': 'Q9P0S2', 'TrEMBL': 'A0A087WX56'}\n",
      "\n",
      "Gene: PRRG3\n",
      "Symbol: PRRG3\n",
      "Name: proline rich and Gla domain 3\n",
      "Summary: This gene encodes a protein which contains a vitamin K-dependent carboxylation/gamma-carboxyglutamic domain. The encoded protein is a member of a family of vitamin K-dependent transmembrane proteins which contain a glutamate-rich extracellular domain. [provided by RefSeq, Aug 2011].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - blood coagulation\n",
      "  CC:\n",
      "    - extracellular space\n",
      "    - membrane\n",
      "  MF:\n",
      "    - serine-type endopeptidase activity\n",
      "    - calcium ion binding\n",
      "UniProt: {'Swiss-Prot': 'Q9BZD7', 'TrEMBL': ['A6NDZ7', 'A0A6I8PL69', 'A0A494BZY9', 'H7C0C2', 'C9J7E6']}\n",
      "\n",
      "Gene: MFSD8\n",
      "Symbol: MFSD8\n",
      "Name: major facilitator superfamily domain containing 8\n",
      "Summary: This gene encodes a ubiquitous integral membrane protein that contains a transporter domain and a major facilitator superfamily (MFS) domain. Other members of the major facilitator superfamily transport small solutes through chemiosmotic ion gradients. The substrate transported by this protein is unknown. The protein likely localizes to lysosomal membranes. Mutations in this gene are correlated with a variant form of late infantile-onset neuronal ceroid lipofuscinoses (vLINCL). [provided by RefSeq, Oct 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - glycolytic process\n",
      "    - glycolipid metabolic process\n",
      "    - mitochondrion organization\n",
      "    - lysosome organization\n",
      "    - determination of adult lifespan\n",
      "    - glycoprotein metabolic process\n",
      "    - gene expression\n",
      "    - regulation of autophagy\n",
      "    - microglia differentiation\n",
      "    - iodide transport\n",
      "    - multicellular organism growth\n",
      "    - TORC1 signaling\n",
      "    - negative regulation of neuron apoptotic process\n",
      "    - cellular respiration\n",
      "    - neuron development\n",
      "    - astrocyte differentiation\n",
      "    - protein stabilization\n",
      "    - neuromuscular process\n",
      "    - maintenance of location\n",
      "    - neuron apoptotic process\n",
      "    - retina development in camera-type eye\n",
      "    - motor behavior\n",
      "    - inclusion body assembly\n",
      "    - reactive oxygen species metabolic process\n",
      "    - autophagosome maturation\n",
      "    - chloride transmembrane transport\n",
      "    - fluoride transmembrane transport\n",
      "    - lysosomal protein catabolic process\n",
      "    - regulation of lysosomal protein catabolic process\n",
      "  CC:\n",
      "    - mitochondrion\n",
      "    - lysosomal membrane\n",
      "    - lysosomal membrane\n",
      "    - lysosomal membrane\n",
      "    - endosome membrane\n",
      "    - chloride channel complex\n",
      "  MF:\n",
      "    - chloride channel activity\n",
      "    - chloride channel activity\n",
      "    - iodide transmembrane transporter activity\n",
      "    - iodide transmembrane transporter activity\n",
      "    - fluoride channel activity\n",
      "UniProt: {'Swiss-Prot': 'Q8NHS3', 'TrEMBL': ['A0A286YF73', 'A0A286YFM2', 'A0A286YFC6', 'B7Z2B2', 'A0A286YF72', 'A0A286YFH9', 'A0A286YFB5', 'A0A286YFI8', 'A0A286YEW8', 'A0A286YF93', 'E7ERQ4', 'A0A286YEV9', 'A0A286YEY8', 'A0A286YFH3', 'A0A286YEW2', 'A0A286YFF5', 'A0A286YEW7', 'A0A286YFF1', 'A0A286YF45', 'A0A286YEV4', 'A0A286YF00', 'A0A286YFI9', 'A0A286YFG7', 'A0A286YFM7', 'A0A286YFJ5', 'A0A286YFK4', 'A0A286YEX1', 'A0A286YER2', 'A0A286YEW6', 'A0A286YF82', 'A0A286YFE7', 'A0A286YF51', 'A0A286YFE6']}\n",
      "\n",
      "Gene: NETO2\n",
      "Symbol: NETO2\n",
      "Name: neuropilin and tolloid like 2\n",
      "Summary: This gene encodes a predicted transmembrane protein containing two extracellular CUB domains followed by a low-density lipoprotein class A (LDLa) domain. A similar gene in rats encodes a protein that modulates glutamate signaling in the brain by regulating kainate receptor function. Expression of this gene may be a biomarker for proliferating infantile hemangiomas. A pseudogene of this gene is located on the long arm of chromosome 8. Alternatively spliced transcript variants encoding multiple isoforms have been observed for this gene. [provided by RefSeq, Jan 2011].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - regulation of neurotransmitter receptor localization to postsynaptic specialization membrane\n",
      "    - neurotransmitter receptor localization to postsynaptic specialization membrane\n",
      "  CC:\n",
      "    - postsynaptic density\n",
      "    - postsynaptic density membrane\n",
      "    - glutamatergic synapse\n",
      "  MF:\n",
      "    - protein binding\n",
      "    - ionotropic glutamate receptor binding\n",
      "UniProt: {'Swiss-Prot': 'Q8NC67', 'TrEMBL': ['H3BSR0', 'Q32NC3', 'H3BM37', 'H3BTE1']}\n",
      "\n",
      "Gene: DAGLB\n",
      "Symbol: DAGLB\n",
      "Name: diacylglycerol lipase beta\n",
      "Summary: Enables lipase activity. Involved in arachidonic acid metabolic process. Located in nucleoplasm and plasma membrane. [provided by Alliance of Genome Resources, Apr 2022]\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - prostaglandin biosynthetic process\n",
      "    - monoacylglycerol biosynthetic process\n",
      "    - neuroblast proliferation\n",
      "    - positive regulation of triglyceride catabolic process\n",
      "    - arachidonate metabolic process\n",
      "    - arachidonate metabolic process\n",
      "    - arachidonate metabolic process\n",
      "    - neurogenesis\n",
      "    - diacylglycerol catabolic process\n",
      "    - regulation of inflammatory response\n",
      "    - cannabinoid biosynthetic process\n",
      "  CC:\n",
      "    - nucleoplasm\n",
      "    - lysosomal membrane\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "  MF:\n",
      "    - triacylglycerol lipase activity\n",
      "    - triacylglycerol lipase activity\n",
      "    - protein binding\n",
      "    - lipase activity\n",
      "    - metal ion binding\n",
      "    - monoacylglycerol lipase activity\n",
      "UniProt: {'Swiss-Prot': 'Q8NCG7', 'TrEMBL': ['F8WBN0', 'E7ET49', 'C9JA85']}\n",
      "\n",
      "Gene: NUDT13\n",
      "Symbol: NUDT13\n",
      "Name: nudix hydrolase 13\n",
      "Summary: Predicted to enable NADH pyrophosphatase activity. Predicted to be involved in NADH metabolic process and NADP catabolic process. Predicted to be located in mitochondrion. [provided by Alliance of Genome Resources, Apr 2022]\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - NADH metabolic process\n",
      "    - NADP catabolic process\n",
      "    - nucleobase-containing small molecule interconversion\n",
      "  CC:\n",
      "    - mitochondrion\n",
      "    - mitochondrion\n",
      "    - mitochondrial matrix\n",
      "  MF:\n",
      "    - NAD+ diphosphatase activity\n",
      "    - NADPH pyrophosphatase activity\n",
      "    - pyrophosphatase activity\n",
      "    - NADH pyrophosphatase activity\n",
      "    - metal ion binding\n",
      "UniProt: {'Swiss-Prot': 'Q86X67', 'TrEMBL': ['A0A087X1G4', 'B4E059']}\n",
      "\n",
      "Gene: CACNA2D2\n",
      "Symbol: CACNA2D2\n",
      "Name: calcium voltage-gated channel auxiliary subunit alpha2delta 2\n",
      "Summary: Calcium channels mediate the entry of calcium ions into the cell upon membrane polarization. This gene encodes the alpha-2/delta subunit of the voltage-dependent calcium channel complex. The complex consists of the main channel-forming subunit alpha-1, and auxiliary subunits alpha-2/delta, beta, and gamma. The auxiliary subunits function in the assembly and membrane localization of the complex, and modulate calcium currents and channel activation/inactivation kinetics. The subunit encoded by this gene undergoes post-translational cleavage to yield the extracellular alpha2 peptide and a membrane-anchored delta polypeptide. This subunit is a receptor for the antiepileptic drug, gabapentin. Mutations in this gene are associated with early infantile epileptic encephalopathy. Single nucleotide polymorphisms in this gene are correlated with increased sensitivity to opioid drugs. Alternative splicing results in multiple transcript variants encoding different isoforms. [provided by RefSeq, Mar 2014].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - neuromuscular junction development\n",
      "    - organ growth\n",
      "    - regulation of multicellular organism growth\n",
      "    - positive regulation of organ growth\n",
      "    - muscle cell development\n",
      "    - rhythmic synaptic transmission\n",
      "    - calcium ion transmembrane transport\n",
      "    - regulation of neurotransmitter receptor localization to postsynaptic specialization membrane\n",
      "  CC:\n",
      "    - plasma membrane\n",
      "    - voltage-gated calcium channel complex\n",
      "    - presynaptic active zone membrane\n",
      "    - GABA-ergic synapse\n",
      "  MF:\n",
      "    - voltage-gated calcium channel activity\n",
      "    - metal ion binding\n",
      "UniProt: {'Swiss-Prot': 'Q9NY47', 'TrEMBL': ['C9JVC9', 'C9JE82']}\n",
      "\n",
      "Gene: ZDHHC18\n",
      "Symbol: ZDHHC18\n",
      "Name: zDHHC palmitoyltransferase 18\n",
      "Summary: Enables palmitoyltransferase activity. Involved in peptidyl-L-cysteine S-palmitoylation. Located in Golgi apparatus. [provided by Alliance of Genome Resources, Apr 2022]\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - protein targeting to membrane\n",
      "    - peptidyl-L-cysteine S-palmitoylation\n",
      "    - protein palmitoylation\n",
      "    - innate immune response\n",
      "    - negative regulation of innate immune response\n",
      "    - negative regulation of cGAS/STING signaling pathway\n",
      "  CC:\n",
      "    - Golgi membrane\n",
      "    - endoplasmic reticulum\n",
      "    - Golgi apparatus\n",
      "    - Golgi apparatus\n",
      "    - Golgi apparatus\n",
      "    - cytosol\n",
      "  MF:\n",
      "    - palmitoyltransferase activity\n",
      "    - protein-cysteine S-palmitoyltransferase activity\n",
      "    - protein-cysteine S-palmitoyltransferase activity\n",
      "UniProt: {'Swiss-Prot': 'Q9NUE0', 'TrEMBL': ['H0YEN1', 'E9PJS3']}\n",
      "\n",
      "Gene: MAP1LC3B2\n",
      "Symbol: MAP1LC3B2\n",
      "Name: microtubule associated protein 1 light chain 3 beta 2\n",
      "Summary: Predicted to enable microtubule binding activity and ubiquitin protein ligase binding activity. Predicted to be involved in autophagy of mitochondrion; cellular response to nitrogen starvation; and macroautophagy. Located in intracellular membrane-bounded organelle. [provided by Alliance of Genome Resources, Apr 2022]\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - autophagosome assembly\n",
      "    - mitophagy\n",
      "    - cellular response to nitrogen starvation\n",
      "    - autophagosome maturation\n",
      "  CC:\n",
      "    - autophagosome membrane\n",
      "    - microtubule\n",
      "    - endomembrane system\n",
      "    - cytoplasmic vesicle\n",
      "  MF:\n",
      "    - microtubule binding\n",
      "    - phosphatidylethanolamine binding\n",
      "    - ubiquitin protein ligase binding\n",
      "UniProt: {'Swiss-Prot': 'A6NCE7'}\n",
      "\n",
      "Gene: LRRC1\n",
      "Symbol: LRRC1\n",
      "Name: leucine rich repeat containing 1\n",
      "Summary: Located in cytosol. [provided by Alliance of Genome Resources, Apr 2022]\n",
      "GO Annotations:\n",
      "  CC:\n",
      "    - cytoplasm\n",
      "    - cytosol\n",
      "    - membrane\n",
      "UniProt: {'Swiss-Prot': 'Q9BTT6', 'TrEMBL': 'Q5T0G3'}\n",
      "\n",
      "Gene: OGN\n",
      "Symbol: OGN\n",
      "Name: osteoglycin\n",
      "Summary: This gene encodes a member of the small leucine-rich proteoglycan (SLRP) family of proteins. The encoded protein induces ectopic bone formation in conjunction with transforming growth factor beta and may regulate osteoblast differentiation. High expression of the encoded protein may be associated with elevated heart left ventricular mass. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Jan 2016].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - signal transduction\n",
      "    - negative regulation of smooth muscle cell proliferation\n",
      "    - bone development\n",
      "    - articular cartilage development\n",
      "  CC:\n",
      "    - extracellular region\n",
      "    - extracellular region\n",
      "    - extracellular space\n",
      "    - Golgi lumen\n",
      "    - extracellular matrix\n",
      "    - lysosomal lumen\n",
      "    - collagen-containing extracellular matrix\n",
      "    - collagen-containing extracellular matrix\n",
      "    - extracellular exosome\n",
      "    - extracellular vesicle\n",
      "  MF:\n",
      "    - protein binding\n",
      "    - growth factor activity\n",
      "    - extracellular matrix structural constituent conferring compression resistance\n",
      "UniProt: {'Swiss-Prot': 'P20774', 'TrEMBL': ['Q7Z532', 'A8K0R3', 'B4DI63', 'Q5TBF5']}\n",
      "\n",
      "Gene: CHST7\n",
      "Symbol: CHST7\n",
      "Name: carbohydrate sulfotransferase 7\n",
      "Summary: This gene is a member of the Gal/GalNAc/GlcNAc (galactose/N-acetylgalactosamine/N-acetylglucosamine) 6-O-sulfotransferase (GST) family. Members of this family encode enzymes that catalyze the specific addition of sulfate groups to distinct hydroxyl and amino groups of carbohydrates. The encoded protein catalyzes the sulfation of 6-hydroxyl group of GalNAc in chondroitin. [provided by RefSeq, Aug 2013].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - polysaccharide metabolic process\n",
      "    - N-acetylglucosamine metabolic process\n",
      "    - N-acetylglucosamine metabolic process\n",
      "    - sulfur compound metabolic process\n",
      "    - chondroitin sulfate biosynthetic process\n",
      "    - chondroitin sulfate biosynthetic process\n",
      "    - chondroitin sulfate biosynthetic process\n",
      "  CC:\n",
      "    - Golgi membrane\n",
      "    - membrane\n",
      "    - membrane\n",
      "  MF:\n",
      "    - N-acetylglucosamine 6-O-sulfotransferase activity\n",
      "    - N-acetylglucosamine 6-O-sulfotransferase activity\n",
      "    - chondroitin 6-sulfotransferase activity\n",
      "    - chondroitin 6-sulfotransferase activity\n",
      "    - chondroitin 6-sulfotransferase activity\n",
      "UniProt: {'Swiss-Prot': 'Q9NS84'}\n",
      "\n",
      "Gene: BTN3A1\n",
      "Symbol: BTN3A1\n",
      "Name: butyrophilin subfamily 3 member A1\n",
      "Summary: The butyrophilin (BTN) genes are a group of major histocompatibility complex (MHC)-associated genes that encode type I membrane proteins with 2 extracellular immunoglobulin (Ig) domains and an intracellular B30.2 (PRYSPRY) domain. Three subfamilies of human BTN genes are located in the MHC class I region: the single-copy BTN1A1 gene (MIM 601610) and the BTN2 (e.g., BTN2A1; MIM 613590) and BTN3 (e.g., BNT3A1) genes, which have undergone tandem duplication, resulting in 3 copies of each (summary by Smith et al., 2010 [PubMed 20208008]).[supplied by OMIM, Nov 2010].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - regulation of cytokine production\n",
      "    - positive regulation of cytokine production\n",
      "    - adaptive immune response\n",
      "    - positive regulation of type II interferon production\n",
      "    - activated T cell proliferation\n",
      "    - T cell receptor signaling pathway\n",
      "    - T cell receptor signaling pathway\n",
      "  CC:\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - external side of plasma membrane\n",
      "  MF:\n",
      "    - signaling receptor binding\n",
      "    - protein binding\n",
      "UniProt: {'Swiss-Prot': 'O00481', 'TrEMBL': ['A6PVC0', 'A8K547', 'E9PFB8', 'E7EPR2']}\n",
      "\n",
      "Gene: CA14\n",
      "Symbol: CA14\n",
      "Name: carbonic anhydrase 14\n",
      "Summary: Carbonic anhydrases (CAs) are a large family of zinc metalloenzymes that catalyze the reversible hydration of carbon dioxide.  They participate in a variety of biological processes, including respiration, calcification, acid-base balance, bone resorption, and the formation of aqueous humor, cerebrospinal fluid, saliva, and gastric acid.  They show extensive diversity in tissue distribution and in their subcellular localization.  CA XIV is predicted to be a type I membrane protein and shares highest sequence similarity with the other transmembrane CA isoform, CA XII; however, they have different patterns of tissue-specific expression and thus may play different physiologic roles. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  CC:\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - membrane\n",
      "    - basolateral plasma membrane\n",
      "    - apical plasma membrane\n",
      "  MF:\n",
      "    - carbonate dehydratase activity\n",
      "    - zinc ion binding\n",
      "UniProt: {'Swiss-Prot': 'Q9ULX7', 'TrEMBL': ['U3KPU6', 'A8K3J4', 'U3KQH0']}\n",
      "\n",
      "Gene: ADH1C\n",
      "Symbol: ADH1C\n",
      "Name: alcohol dehydrogenase 1C (class I), gamma polypeptide\n",
      "Summary: This gene encodes class I alcohol dehydrogenase, gamma subunit, which is a member of the alcohol dehydrogenase family. Members of this enzyme family metabolize a wide variety of substrates, including ethanol, retinol, other aliphatic alcohols, hydroxysteroids, and lipid peroxidation products. Class I alcohol dehydrogenase, consisting of several homo- and heterodimers of alpha, beta, and gamma subunits, exhibits high activity for ethanol oxidation to acetaldehyde, thus playing a major role in ethanol catabolism. Three genes encoding alpha, beta and gamma subunits are tandemly organized in a genomic segment as a gene cluster. An association between ADH1C polymorphism and alcohol dependence has not been established. [provided by RefSeq, Sep 2019].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - retinol metabolic process\n",
      "    - retinoic acid metabolic process\n",
      "  CC:\n",
      "    - nucleoplasm\n",
      "    - cytosol\n",
      "    - cytosol\n",
      "    - cytosol\n",
      "    - plasma membrane\n",
      "    - cilium\n",
      "    - ciliary basal body\n",
      "  MF:\n",
      "    - alcohol dehydrogenase (NAD+) activity\n",
      "    - all-trans-retinol dehydrogenase (NAD+) activity\n",
      "    - protein binding\n",
      "    - zinc ion binding\n",
      "UniProt: {'Swiss-Prot': 'P00326', 'TrEMBL': ['A0A087WUC4', 'A0A087WU81']}\n",
      "\n",
      "Gene: ALOX15\n",
      "Symbol: ALOX15\n",
      "Name: arachidonate 15-lipoxygenase\n",
      "Summary: This gene encodes a member of the lipoxygenase family of proteins. The encoded enzyme acts on various polyunsaturated fatty acid substrates to generate various bioactive lipid mediators such as eicosanoids, hepoxilins, lipoxins, and other molecules. The encoded enzyme and its reaction products have been shown to regulate inflammation and immunity. Multiple pseudogenes of this gene have been identified in the human genome. [provided by RefSeq, Aug 2017].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - ossification\n",
      "    - negative regulation of adaptive immune response\n",
      "    - lipid metabolic process\n",
      "    - phosphatidylethanolamine biosynthetic process\n",
      "    - inflammatory response\n",
      "    - positive regulation of cell-substrate adhesion\n",
      "    - arachidonate metabolic process\n",
      "    - arachidonate metabolic process\n",
      "    - arachidonate metabolic process\n",
      "    - lipoxygenase pathway\n",
      "    - lipoxygenase pathway\n",
      "    - lipoxygenase pathway\n",
      "    - fatty acid oxidation\n",
      "    - bone mineralization\n",
      "    - positive regulation of actin filament polymerization\n",
      "    - lipid oxidation\n",
      "    - response to endoplasmic reticulum stress\n",
      "    - regulation of peroxisome proliferator activated receptor signaling pathway\n",
      "    - cellular response to interleukin-13\n",
      "    - wound healing\n",
      "    - long-chain fatty acid biosynthetic process\n",
      "    - apoptotic cell clearance\n",
      "    - linoleic acid metabolic process\n",
      "    - regulation of inflammatory response\n",
      "    - hepoxilin biosynthetic process\n",
      "    - positive regulation of ERK1 and ERK2 cascade\n",
      "    - cellular response to calcium ion\n",
      "    - regulation of engulfment of apoptotic cell\n",
      "    - lipoxin A4 biosynthetic process\n",
      "  CC:\n",
      "    - lipid droplet\n",
      "    - cytosol\n",
      "    - cytosol\n",
      "    - cytosol\n",
      "    - cytosol\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - cytoplasmic side of plasma membrane\n",
      "    - membrane\n",
      "  MF:\n",
      "    - arachidonate 12(S)-lipoxygenase activity\n",
      "    - arachidonate 12(S)-lipoxygenase activity\n",
      "    - iron ion binding\n",
      "    - protein binding\n",
      "    - phosphatidylinositol-4,5-bisphosphate binding\n",
      "    - linoleate 13S-lipoxygenase activity\n",
      "    - linoleate 13S-lipoxygenase activity\n",
      "    - arachidonate 15-lipoxygenase activity\n",
      "    - arachidonate 15-lipoxygenase activity\n",
      "    - arachidonate 15-lipoxygenase activity\n",
      "    - arachidonate 15-lipoxygenase activity\n",
      "UniProt: {'Swiss-Prot': 'P16050'}\n",
      "\n",
      "Gene: RARS2\n",
      "Symbol: RARS2\n",
      "Name: arginyl-tRNA synthetase 2, mitochondrial\n",
      "Summary: This nuclear gene encodes a protein that localizes to the mitochondria, where it catalyzes the transfer of L-arginine to its cognate tRNA, an important step in translation of mitochondrially-encoded proteins. Defects in this gene are a cause of pontocerebellar hypoplasia type 6 (PCH6). Alternative splicing results in multiple transcript variants. [provided by RefSeq, Jan 2016].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - tRNA aminoacylation for protein translation\n",
      "    - arginyl-tRNA aminoacylation\n",
      "    - mitochondrial translation\n",
      "  CC:\n",
      "    - mitochondrion\n",
      "    - mitochondrion\n",
      "    - mitochondrial matrix\n",
      "    - mitochondrial membrane\n",
      "  MF:\n",
      "    - RNA binding\n",
      "    - arginine-tRNA ligase activity\n",
      "    - arginine-tRNA ligase activity\n",
      "    - arginine-tRNA ligase activity\n",
      "    - protein binding\n",
      "    - ATP binding\n",
      "UniProt: {'Swiss-Prot': 'Q5T160', 'TrEMBL': ['A0A8I5KPZ0', 'H0UI22', 'A0A8I5KYL9', 'A0A8I5KPX4', 'A0A8I5KXM9', 'A0A8I5KQP4', 'A0A8I5KPZ4', 'H0Y450', 'A0A8I5KU19', 'A0A8I5QJN6', 'A0A8I5KQS8', 'A0A8I5KR81', 'A0A8I5KS32', 'A0A8I5QJX6', 'A0A8I5KR42', 'A0A8I5KRE6', 'A0A8I5KVK0', 'A0A8I5KP51', 'A0A8I5KWC6', 'A0A8I5KQK8', 'A0A8I5KQV7', 'A0A8I5KX97', 'A0A8I5KVZ5', 'A0A8I5KPR3', 'A0A8I5KS21', 'A0A8I5KWE8']}\n",
      "\n",
      "Gene: PSG3\n",
      "Symbol: PSG3\n",
      "Name: pregnancy specific beta-1-glycoprotein 3\n",
      "Summary: The human pregnancy-specific glycoproteins (PSGs) are a family of proteins that are synthesized in large amounts by placental trophoblasts and released into the maternal circulation during pregnancy. Molecular cloning and analysis of several PSG genes has indicated that the PSGs form a subgroup of the carcinoembryonic antigen (CEA) gene family, which belongs to the immunoglobulin superfamily of genes. Members of the CEA family consist of a single N domain, with structural similarity to the immunoglobulin variable domains, followed by a variable number of immunoglobulin constant-like A and/or B domains. Most PSGs have an arg-gly-asp (RGD) motif, which has been shown to function as an adhesion recognition signal for several integrins, in the N-terminal domain (summary by Teglund et al., 1994 [PubMed 7851896]). For additional general information about the PSG gene family, see PSG1 (MIM 176390).[supplied by OMIM, Oct 2009].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - defense response\n",
      "    - female pregnancy\n",
      "  CC:\n",
      "    - extracellular region\n",
      "UniProt: {'Swiss-Prot': 'Q16557', 'TrEMBL': ['M0QX68', 'M0QXP2']}\n",
      "\n",
      "Gene: CHST8\n",
      "Symbol: CHST8\n",
      "Name: carbohydrate sulfotransferase 8\n",
      "Summary: The protein encoded by this gene belongs to the sulfotransferase 2 family. It is predominantly expressed in the pituitary gland, and is localized to the golgi membrane. This protein catalyzes the transfer of sulfate to position 4 of non-reducing N-acetylgalactosamine (GalNAc) residues in both N-glycans and O-glycans. It is responsible for sulfation of GalNAc on luteinizing hormone (LH), which is required for production of the sex hormones. Mice lacking this enzyme, exhibit increased levels of circulating LH, and precocious sexual maturation of both male and female mice. Alternatively spliced transcript variants have been found for this gene. [provided by RefSeq, Aug 2011].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - sulfur compound metabolic process\n",
      "    - central nervous system development\n",
      "    - carbohydrate biosynthetic process\n",
      "    - proteoglycan biosynthetic process\n",
      "    - proteoglycan biosynthetic process\n",
      "    - hormone biosynthetic process\n",
      "  CC:\n",
      "    - Golgi membrane\n",
      "    - membrane\n",
      "  MF:\n",
      "    - dermatan 4-sulfotransferase activity\n",
      "    - dermatan 4-sulfotransferase activity\n",
      "    - protein binding\n",
      "    - sulfotransferase activity\n",
      "UniProt: {'Swiss-Prot': 'Q9H2A9', 'TrEMBL': 'K7ENM3'}\n",
      "\n",
      "Gene: MREG\n",
      "Symbol: MREG\n",
      "Name: melanoregulin\n",
      "Summary: Predicted to enable phosphatidylinositol binding activity. Predicted to be involved in melanocyte differentiation; melanosome transport; and phagosome maturation. Predicted to act upstream of or within developmental pigmentation. Predicted to be located in late endosome membrane and melanosome membrane. Predicted to be intrinsic component of organelle membrane. Predicted to be part of protein-containing complex. Predicted to be active in melanosome. [provided by Alliance of Genome Resources, Apr 2022]\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - melanocyte differentiation\n",
      "    - melanosome localization\n",
      "    - melanosome transport\n",
      "    - minus-end-directed organelle transport along microtubule\n",
      "    - phagosome maturation\n",
      "  CC:\n",
      "    - lysosomal membrane\n",
      "    - apical plasma membrane\n",
      "    - cytoplasmic vesicle membrane\n",
      "    - organelle membrane\n",
      "    - late endosome membrane\n",
      "    - protein-containing complex\n",
      "    - melanosome membrane\n",
      "    - melanosome\n",
      "  MF:\n",
      "    - protein binding\n",
      "    - phosphatidylinositol binding\n",
      "UniProt: {'Swiss-Prot': 'Q8N565', 'TrEMBL': ['C9JAG4', 'C9JYV9', 'C9JFU1']}\n",
      "\n",
      "Gene: SLC5A2\n",
      "Symbol: SLC5A2\n",
      "Name: solute carrier family 5 member 2\n",
      "Summary: This gene encodes a member of the sodium glucose cotransporter family which are sodium-dependent glucose transport proteins. The encoded protein is the major cotransporter involved in glucose reabsorption in the kidney. Mutations in this gene are associated with renal glucosuria. Two transcript variants, one protein-coding and one not, have been found for this gene. [provided by RefSeq, Feb 2015].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - alpha-glucoside transport\n",
      "    - carbohydrate metabolic process\n",
      "    - sodium ion transport\n",
      "    - hexose transmembrane transport\n",
      "    - renal D-glucose absorption\n",
      "    - D-glucose import across plasma membrane\n",
      "    - sodium ion import across plasma membrane\n",
      "  CC:\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - membrane\n",
      "    - apical plasma membrane\n",
      "    - extracellular exosome\n",
      "  MF:\n",
      "    - low-affinity D-glucose:sodium symporter activity\n",
      "    - D-glucose:sodium symporter activity\n",
      "    - D-glucose:sodium symporter activity\n",
      "    - D-glucose:sodium symporter activity\n",
      "    - D-glucose:sodium symporter activity\n",
      "    - protein binding\n",
      "    - alpha-glucoside transmembrane transporter activity\n",
      "    - metal ion binding\n",
      "    - D-glucose transmembrane transporter activity\n",
      "UniProt: {'Swiss-Prot': 'P31639', 'TrEMBL': ['H3BP44', 'Q8WY15']}\n",
      "\n",
      "Gene: PDE7A\n",
      "Symbol: PDE7A\n",
      "Name: phosphodiesterase 7A\n",
      "Summary: The protein encoded by this gene belongs to the cyclic nucleotide phosphodiesterase (PDE) family, and PDE7 subfamily. This PDE hydrolyzes the second messenger, cAMP, which is a regulator and mediator of a number of cellular responses to extracellular signals. Thus, by regulating the cellular concentration of cAMP, this protein plays a key role in many important physiological processes. Alternatively spliced transcript variants encoding different isoforms have been described for this gene. [provided by RefSeq, Jul 2011].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - cAMP catabolic process\n",
      "    - cAMP-mediated signaling\n",
      "  CC:\n",
      "    - cytosol\n",
      "  MF:\n",
      "    - 3',5'-cyclic-AMP phosphodiesterase activity\n",
      "    - 3',5'-cyclic-AMP phosphodiesterase activity\n",
      "    - metal ion binding\n",
      "    - 3',5'-cyclic-GMP phosphodiesterase activity\n",
      "UniProt: {'Swiss-Prot': 'Q13946', 'TrEMBL': 'E5RG23'}\n",
      "\n",
      "Gene: SLC19A1\n",
      "Symbol: SLC19A1\n",
      "Name: solute carrier family 19 member 1\n",
      "Summary: The membrane protein encoded by this gene is a transporter of folate and is involved in the regulation of intracellular concentrations of folate. Three transcript variants encoding different isoforms have been found for this gene.[provided by RefSeq, Mar 2011].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - xenobiotic transmembrane transport\n",
      "    - female pregnancy\n",
      "    - response to xenobiotic stimulus\n",
      "    - response to toxic substance\n",
      "    - organic anion transport\n",
      "    - folic acid transport\n",
      "    - folic acid transport\n",
      "    - folic acid metabolic process\n",
      "    - methotrexate transport\n",
      "    - monoatomic anion transmembrane transport\n",
      "    - folate transmembrane transport\n",
      "    - folate transmembrane transport\n",
      "    - cyclic-GMP-AMP transmembrane import across plasma membrane\n",
      "    - positive regulation of cGAS/STING signaling pathway\n",
      "    - transport across blood-brain barrier\n",
      "    - transport across blood-brain barrier\n",
      "    - folate import across plasma membrane\n",
      "  CC:\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - basolateral plasma membrane\n",
      "    - basolateral plasma membrane\n",
      "    - basolateral plasma membrane\n",
      "    - apical plasma membrane\n",
      "    - apical plasma membrane\n",
      "    - apical plasma membrane\n",
      "    - brush border membrane\n",
      "  MF:\n",
      "    - protein binding\n",
      "    - folic acid binding\n",
      "    - organic anion transmembrane transporter activity\n",
      "    - folic acid transmembrane transporter activity\n",
      "    - folic acid transmembrane transporter activity\n",
      "    - folic acid transmembrane transporter activity\n",
      "    - folate:monoatomic anion antiporter activity\n",
      "    - folate:monoatomic anion antiporter activity\n",
      "    - folate:monoatomic anion antiporter activity\n",
      "    - antiporter activity\n",
      "    - methotrexate transmembrane transporter activity\n",
      "    - xenobiotic transmembrane transporter activity\n",
      "    - 2',3'-cyclic GMP-AMP binding\n",
      "    - cyclic-GMP-AMP transmembrane transporter activity\n",
      "UniProt: {'Swiss-Prot': 'P41440', 'TrEMBL': ['E9PIL5', 'H0Y4T2', 'H3BTQ3', 'C9J8K6', 'C9JKP4']}\n",
      "\n",
      "Gene: GPR15\n",
      "Symbol: GPR15\n",
      "Name: G protein-coupled receptor 15\n",
      "Summary: This gene encodes a G protein-coupled receptor that acts as a chemokine receptor for human immunodeficiency virus type 1 and 2. The encoded protein localizes to the cell membrane. [provided by RefSeq, Nov 2012].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - angiogenesis\n",
      "    - G protein-coupled receptor signaling pathway\n",
      "    - G protein-coupled receptor signaling pathway\n",
      "    - symbiont entry into host cell\n",
      "    - T cell migration\n",
      "  CC:\n",
      "    - cytoplasm\n",
      "    - endosome\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "  MF:\n",
      "    - virus receptor activity\n",
      "    - G protein-coupled receptor activity\n",
      "    - G protein-coupled receptor activity\n",
      "    - protein binding\n",
      "    - coreceptor activity\n",
      "UniProt: {'Swiss-Prot': 'P49685', 'TrEMBL': ['B2R8H6', 'B6V9G9']}\n",
      "\n",
      "Gene: RAD21L1\n",
      "Symbol: RAD21L1\n",
      "Name: RAD21 cohesin complex component like 1\n",
      "Summary: Predicted to enable chromatin binding activity. Predicted to be involved in mitotic sister chromatid cohesion; replication-born double-strand break repair via sister chromatid exchange; and synaptonemal complex assembly. Predicted to act upstream of or within several processes, including double-strand break repair via homologous recombination; homologous chromosome segregation; and seminiferous tubule development. Predicted to be located in lateral element. Predicted to be part of nuclear meiotic cohesin complex and nuclear mitotic cohesin complex. Predicted to be active in synaptonemal complex. [provided by Alliance of Genome Resources, Apr 2022]\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - chromosome segregation\n",
      "    - sister chromatid cohesion\n",
      "    - meiotic cell cycle\n",
      "    - replication-born double-strand break repair via sister chromatid exchange\n",
      "  CC:\n",
      "    - nucleus\n",
      "    - chromosome\n",
      "    - meiotic cohesin complex\n",
      "    - meiotic cohesin complex\n",
      "  MF:\n",
      "    - chromatin binding\n",
      "UniProt: {'Swiss-Prot': 'Q9H4I0', 'TrEMBL': ['A0A140T8W2', 'A0A804HJ87']}\n",
      "\n",
      "Gene: MBOAT2\n",
      "Symbol: MBOAT2\n",
      "Name: membrane bound glycerophospholipid O-acyltransferase 2\n",
      "Summary: Enables 1-acylglycerophosphocholine O-acyltransferase activity; 1-acylglycerophosphoethanolamine O-acyltransferase activity; and 1-acylglycerophosphoserine O-acyltransferase activity. Involved in phosphatidylcholine acyl-chain remodeling; phosphatidylethanolamine acyl-chain remodeling; and phosphatidylserine acyl-chain remodeling. Predicted to be located in endoplasmic reticulum membrane. Predicted to be active in membrane. [provided by Alliance of Genome Resources, Apr 2022]\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - phospholipid biosynthetic process\n",
      "    - lipid modification\n",
      "    - regulation of chondrocyte differentiation\n",
      "    - phosphatidylserine acyl-chain remodeling\n",
      "    - phosphatidylcholine acyl-chain remodeling\n",
      "    - phosphatidylcholine acyl-chain remodeling\n",
      "    - phosphatidylcholine acyl-chain remodeling\n",
      "    - phosphatidylethanolamine acyl-chain remodeling\n",
      "    - phosphatidylethanolamine acyl-chain remodeling\n",
      "    - phosphatidylethanolamine acyl-chain remodeling\n",
      "  CC:\n",
      "    - endoplasmic reticulum membrane\n",
      "    - endoplasmic reticulum membrane\n",
      "    - membrane\n",
      "  MF:\n",
      "    - 1-acylglycerol-3-phosphate O-acyltransferase activity\n",
      "    - acyltransferase activity\n",
      "    - 2-acylglycerol-3-phosphate O-acyltransferase activity\n",
      "    - 1-acylglycerophosphocholine O-acyltransferase activity\n",
      "    - 1-acylglycerophosphoethanolamine O-acyltransferase activity\n",
      "    - 1-acylglycerophosphoserine O-acyltransferase activity\n",
      "UniProt: {'Swiss-Prot': 'Q6ZWT7', 'TrEMBL': ['B0AZU0', 'B7Z3I3', 'F8WDJ6', 'C9J2H7', 'H7C5D1']}\n",
      "\n",
      "Gene: SLCO4C1\n",
      "Symbol: SLCO4C1\n",
      "Name: solute carrier organic anion transporter family member 4C1\n",
      "Summary: SLCO4C1 belongs to the organic anion transporter (OATP) family. OATPs are involved in the membrane transport of bile acids, conjugated steroids, thyroid hormone, eicosanoids, peptides, and numerous drugs in many tissues (Mikkaichi et al., 2004 [PubMed 14993604]).[supplied by OMIM, Mar 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - monoatomic ion transport\n",
      "    - spermatogenesis\n",
      "    - cell differentiation\n",
      "    - sodium-independent organic anion transport\n",
      "    - sodium-independent organic anion transport\n",
      "    - transmembrane transport\n",
      "  CC:\n",
      "    - plasma membrane\n",
      "    - basolateral plasma membrane\n",
      "    - azurophil granule membrane\n",
      "    - specific granule membrane\n",
      "    - extracellular exosome\n",
      "  MF:\n",
      "    - organic anion transmembrane transporter activity\n",
      "    - sodium-independent organic anion transmembrane transporter activity\n",
      "    - sodium-independent organic anion transmembrane transporter activity\n",
      "UniProt: {'Swiss-Prot': 'Q6ZQN7'}\n",
      "\n",
      "Gene: KIAA0556\n",
      "Symbol: KATNIP\n",
      "Name: katanin interacting protein\n",
      "Summary: This gene encodes a novel, evolutionarily conserved, ciliary protein. In human hTERT-RPE1 cells, the protein is found at the base of cilia, decorating the ciliary axoneme, and enriched at the ciliary tip. The protein binds to microtubules in vitro and regulates their stability when it is overexpressed. A null mutation in this gene has been associated with Joubert syndrome, a recessive disorder that is characterized by a distinctive mid-hindbrain and cerebellar malformation and is also often associated with wider ciliopathy symptoms. Consistently, in a serum-starvation ciliogenesis assay, human fibroblast cells derived from patients with the mutation display a reduced number of ciliated cells with abnormally long cilia. [provided by RefSeq, Feb 2016].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - cerebrospinal fluid circulation\n",
      "  CC:\n",
      "    - extracellular space\n",
      "    - nucleoplasm\n",
      "    - cytoplasm\n",
      "    - plasma membrane\n",
      "    - cilium\n",
      "    - centriolar satellite\n",
      "    - ciliary basal body\n",
      "UniProt: {'Swiss-Prot': 'O60303', 'TrEMBL': ['H3BNJ6', 'A0A087X098', 'H3BPM2']}\n",
      "\n",
      "Gene: CYP24A1\n",
      "Symbol: CYP24A1\n",
      "Name: cytochrome P450 family 24 subfamily A member 1\n",
      "Summary: This gene encodes a member of the cytochrome P450 superfamily of enzymes. The cytochrome P450 proteins are monooxygenases which catalyze many reactions involved in drug metabolism and synthesis of cholesterol, steroids and other lipids. This mitochondrial protein initiates the degradation of 1,25-dihydroxyvitamin D3, the physiologically active form of vitamin D3, by hydroxylation of the side chain. In regulating the level of vitamin D3, this enzyme plays a role in calcium homeostasis and the vitamin D endocrine system. Alternatively spliced transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - osteoblast differentiation\n",
      "    - vitamin metabolic process\n",
      "    - response to vitamin D\n",
      "    - response to vitamin D\n",
      "    - calcitriol biosynthetic process from calciol\n",
      "    - vitamin D metabolic process\n",
      "    - vitamin D catabolic process\n",
      "    - vitamin D catabolic process\n",
      "    - vitamin D catabolic process\n",
      "    - vitamin D receptor signaling pathway\n",
      "  CC:\n",
      "    - mitochondrion\n",
      "    - mitochondrial inner membrane\n",
      "  MF:\n",
      "    - iron ion binding\n",
      "    - 25-hydroxycholecalciferol-24-hydroxylase activity\n",
      "    - 25-hydroxycholecalciferol-24-hydroxylase activity\n",
      "    - heme binding\n",
      "    - 1-alpha,25-dihydroxyvitamin D3 24-hydroxylase activity\n",
      "    - 1-alpha,25-dihydroxyvitamin D3 24-hydroxylase activity\n",
      "    - 1-alpha,25-dihydroxyvitamin D3 24-hydroxylase activity\n",
      "    - vitamin D3 25-hydroxylase activity\n",
      "    - 25-hydroxycholecalciferol-23-hydroxylase activity\n",
      "    - 1-alpha,25-dihydroxyvitamin D3 23-hydroxylase activity\n",
      "UniProt: {'Swiss-Prot': 'Q07973'}\n",
      "\n",
      "Gene: COX7C\n",
      "Symbol: COX7C\n",
      "Name: cytochrome c oxidase subunit 7C\n",
      "Summary: Cytochrome c oxidase (COX), the terminal component of the mitochondrial respiratory chain, catalyzes the electron transfer from reduced cytochrome c to oxygen. This component is a heteromeric complex consisting of 3 catalytic subunits encoded by mitochondrial genes and multiple structural subunits encoded by nuclear genes. The mitochondrially-encoded subunits function in electron transfer, and the nuclear-encoded subunits may function in the regulation and assembly of the complex. This nuclear gene encodes subunit VIIc, which shares 87% and 85% amino acid sequence identity with mouse and bovine COX VIIc, respectively, and is found in all tissues. A pseudogene COX7CP1 has been found on chromosome 13. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - generation of precursor metabolites and energy\n",
      "    - mitochondrial electron transport, cytochrome c to oxygen\n",
      "    - mitochondrial electron transport, cytochrome c to oxygen\n",
      "    - cellular respiration\n",
      "  CC:\n",
      "    - mitochondrion\n",
      "    - mitochondrion\n",
      "    - mitochondrial inner membrane\n",
      "    - mitochondrial membrane\n",
      "    - respiratory chain complex IV\n",
      "  MF:\n",
      "    - protein binding\n",
      "UniProt: {'Swiss-Prot': 'P15954', 'TrEMBL': 'D6R9Z7'}\n",
      "\n",
      "Gene: TRPM8\n",
      "Symbol: TRPM8\n",
      "Name: transient receptor potential cation channel subfamily M member 8\n",
      "Summary: Predicted to enable ligand-gated calcium channel activity. Predicted to be involved in calcium ion transmembrane transport and positive regulation of cold-induced thermogenesis. Predicted to act upstream of or within several processes, including cellular calcium ion homeostasis; response to cold; and thermoception. Located in plasma membrane. [provided by Alliance of Genome Resources, Apr 2022]\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - intracellular calcium ion homeostasis\n",
      "    - response to cold\n",
      "    - calcium-mediated signaling\n",
      "    - thermoception\n",
      "    - calcium ion transmembrane transport\n",
      "    - calcium ion transmembrane transport\n",
      "    - positive regulation of cold-induced thermogenesis\n",
      "  CC:\n",
      "    - endoplasmic reticulum membrane\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - external side of plasma membrane\n",
      "    - plasma membrane raft\n",
      "  MF:\n",
      "    - calcium channel activity\n",
      "    - protein binding\n",
      "    - identical protein binding\n",
      "    - metal ion binding\n",
      "    - ligand-gated calcium channel activity\n",
      "UniProt: {'Swiss-Prot': 'Q7Z2W7', 'TrEMBL': ['A0A0C4DFT0', 'W8DTH1', 'H7BZP4', 'H0Y7P0', 'F8WD55', 'A0A0A0MSV2']}\n",
      "\n",
      "Gene: CA5A\n",
      "Symbol: CA5A\n",
      "Name: carbonic anhydrase 5A\n",
      "Summary: Carbonic anhydrases (CAs) are a large family of zinc metalloenzymes that catalyze the reversible hydration of carbon dioxide.  They participate in a variety of biological processes, including respiration, calcification, acid-base balance, bone resorption, and the formation of aqueous humor, cerebrospinal fluid, saliva, and gastric acid.  They show extensive diversity in tissue distribution and in their subcellular localization.  CA VA is localized in the mitochondria and expressed primarily in the liver.  It may play an important role in ureagenesis and gluconeogenesis.  CA5A gene maps to chromosome 16q24.3 and an unprocessed pseudogene has been assigned to 16p12-p11.2. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  CC:\n",
      "    - cytoplasm\n",
      "    - mitochondrion\n",
      "    - mitochondrion\n",
      "    - mitochondrion\n",
      "    - mitochondrial matrix\n",
      "  MF:\n",
      "    - carbonate dehydratase activity\n",
      "    - carbonate dehydratase activity\n",
      "    - carbonate dehydratase activity\n",
      "    - zinc ion binding\n",
      "UniProt: {'Swiss-Prot': 'P35218', 'TrEMBL': ['A0A3B3IRX9', 'A0A3B3ITA6', 'A0A3B3ITU9']}\n",
      "\n",
      "Gene: GPX7\n",
      "Symbol: GPX7\n",
      "Name: glutathione peroxidase 7\n",
      "Summary: Enables catalase activity. Predicted to be involved in cellular response to oxidative stress. Located in endoplasmic reticulum. [provided by Alliance of Genome Resources, Apr 2022]\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - cellular response to oxidative stress\n",
      "    - cellular oxidant detoxification\n",
      "  CC:\n",
      "    - extracellular region\n",
      "    - endoplasmic reticulum\n",
      "    - endoplasmic reticulum lumen\n",
      "  MF:\n",
      "    - catalase activity\n",
      "    - peroxidase activity\n",
      "    - peroxidase activity\n",
      "    - glutathione peroxidase activity\n",
      "    - protein binding\n",
      "UniProt: {'Swiss-Prot': 'Q96SL4'}\n",
      "\n",
      "Gene: CYP2A7\n",
      "Symbol: CYP2A7\n",
      "Name: cytochrome P450 family 2 subfamily A member 7\n",
      "Summary: This gene encodes a member of the cytochrome P450 superfamily of enzymes. The cytochrome P450 proteins are monooxygenases which catalyze many reactions involved in drug metabolism and synthesis of cholesterol, steroids and other lipids. This protein localizes to the endoplasmic reticulum; its substrate has not yet been determined. This gene, which produces two transcript variants, is part of a large cluster of cytochrome P450 genes from the CYP2A, CYP2B and CYP2F subfamilies on chromosome 19q. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - xenobiotic metabolic process\n",
      "    - coumarin metabolic process\n",
      "    - epoxygenase P450 pathway\n",
      "  CC:\n",
      "    - cytoplasm\n",
      "    - endoplasmic reticulum membrane\n",
      "    - intracellular membrane-bounded organelle\n",
      "  MF:\n",
      "    - iron ion binding\n",
      "    - arachidonate epoxygenase activity\n",
      "    - oxidoreductase activity, acting on paired donors, with incorporation or reduction of molecular oxygen, reduced flavin or flavoprotein as one donor, and incorporation of one atom of oxygen\n",
      "    - oxygen binding\n",
      "    - heme binding\n",
      "UniProt: {'Swiss-Prot': 'P20853', 'TrEMBL': ['M0R020', 'F8W816']}\n",
      "\n",
      "Gene: NLGN4Y\n",
      "Symbol: NLGN4Y\n",
      "Name: neuroligin 4 Y-linked\n",
      "Summary: This gene encodes a type I membrane protein that belongs to the family of neuroligins, which are cell adhesion molecules present at the postsynaptic side of the synapse, and may be essential for the formation of functional synapses. Alternatively spliced transcript variants have been found for this gene.[provided by RefSeq, Mar 2011].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - neuron cell-cell adhesion\n",
      "    - learning\n",
      "    - social behavior\n",
      "    - vocalization behavior\n",
      "    - presynapse assembly\n",
      "  CC:\n",
      "    - plasma membrane\n",
      "    - membrane\n",
      "    - postsynaptic density membrane\n",
      "    - symmetric, GABA-ergic, inhibitory synapse\n",
      "    - asymmetric, glutamatergic, excitatory synapse\n",
      "    - postsynaptic specialization membrane\n",
      "  MF:\n",
      "    - protein binding\n",
      "    - neurexin family protein binding\n",
      "    - cell adhesion molecule binding\n",
      "    - scaffold protein binding\n",
      "UniProt: {'Swiss-Prot': 'Q8NFZ3', 'TrEMBL': ['A6NMU8', 'B4DHI3', 'H0Y7G6', 'A0A2R8YDT7']}\n",
      "\n",
      "Gene: CYP3A7\n",
      "Symbol: CYP3A7\n",
      "Name: cytochrome P450 family 3 subfamily A member 7\n",
      "Summary: This gene encodes a member of the cytochrome P450 superfamily of enzymes, which participate in drug metabolism and the synthesis of cholesterol, steroids and other lipids. This enzyme hydroxylates testosterone and dehydroepiandrosterone 3-sulphate, which is involved in the formation of estriol during pregnancy. This gene is part of a cluster of related genes on chromosome 7q21.1. Naturally-occurring readthrough transcription occurs between this gene and the downstream CYP3A51P pseudogene and is represented by GeneID:100861540. [provided by RefSeq, Jan 2015].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - lipid hydroxylation\n",
      "    - steroid biosynthetic process\n",
      "    - xenobiotic metabolic process\n",
      "    - steroid metabolic process\n",
      "    - steroid metabolic process\n",
      "    - estrogen metabolic process\n",
      "    - retinol metabolic process\n",
      "    - retinoic acid metabolic process\n",
      "    - oxidative demethylation\n",
      "  CC:\n",
      "    - endoplasmic reticulum membrane\n",
      "  MF:\n",
      "    - monooxygenase activity\n",
      "    - monooxygenase activity\n",
      "    - iron ion binding\n",
      "    - steroid hydroxylase activity\n",
      "    - retinoic acid 4-hydroxylase activity\n",
      "    - oxygen binding\n",
      "    - heme binding\n",
      "    - testosterone 6-beta-hydroxylase activity\n",
      "    - all-trans retinoic acid 18-hydroxylase activity\n",
      "    - estrogen 16-alpha-hydroxylase activity\n",
      "    - estrogen 16-alpha-hydroxylase activity\n",
      "    - estrogen 2-hydroxylase activity\n",
      "UniProt: {'Swiss-Prot': 'P24462'}\n",
      "\n",
      "Gene: CYP4F8\n",
      "Symbol: CYP4F8\n",
      "Name: cytochrome P450 family 4 subfamily F member 8\n",
      "Summary: This gene, CYP4F8, encodes a member of the cytochrome P450 superfamily of enzymes. The cytochrome P450 proteins are monooxygenases which catalyze many reactions involved in drug metabolism and synthesis of cholesterol, steroids and other lipids. This protein localizes to the endoplasmic reticulum and functions as a 19-hydroxylase of prostaglandins in seminal vesicles. This gene is part of a cluster of cytochrome P450 genes on chromosome 19. Another member of this family, CYP4F3, is approximately 18 kb away. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - icosanoid metabolic process\n",
      "    - prostaglandin metabolic process\n",
      "    - arachidonate metabolic process\n",
      "    - menaquinone catabolic process\n",
      "    - phylloquinone catabolic process\n",
      "  CC:\n",
      "    - endoplasmic reticulum membrane\n",
      "  MF:\n",
      "    - monooxygenase activity\n",
      "    - iron ion binding\n",
      "    - oxidoreductase activity, acting on paired donors, with incorporation or reduction of molecular oxygen, reduced flavin or flavoprotein as one donor, and incorporation of one atom of oxygen\n",
      "    - alkane 1-monooxygenase activity\n",
      "    - heme binding\n",
      "UniProt: {'Swiss-Prot': 'P98187', 'TrEMBL': ['A0A087WZJ1', 'A0A087WUC9', 'A0A087WVG9']}\n",
      "\n",
      "Gene: CTRB2\n",
      "Symbol: CTRB2\n",
      "Name: chymotrypsinogen B2\n",
      "Summary: This gene encodes a member of the serine protease family of enzymes and forms a principal precursor of the pancreatic proteolytic enzymes. The encoded preproprotein is synthesized in the acinar cells of the pancreas and secreted into the small intestine where it undergoes proteolytic activation to generate a functional enzyme. This CTRB2 gene is located head-to-head with the related CTRB1 gene. Some human populations have an alternate haplotype which inverts a 16.6 Kb region containing portions of intron 1, exon 1, and the upstream sequence of the CTRB1 and CTRB2 genes. In this inversion haplotype exon 1 and flanking sequence is swapped in CTRB1 and CTRB2. This inversion is associated with differential gene expression and increased risk for chronic pancreatitis. The GRCh38 assembly represents the minor allele for SNP rs8048956 of the CTRB1 gene. SNP rs8048956 is diagnostic for this inversion. [provided by RefSeq, Jan 2021].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - proteolysis\n",
      "    - digestion\n",
      "  CC:\n",
      "    - extracellular region\n",
      "  MF:\n",
      "    - serine-type endopeptidase activity\n",
      "UniProt: {'Swiss-Prot': 'Q6GPI1', 'TrEMBL': ['H3BMY1', 'H3BP92', 'H3BTQ4']}\n",
      "\n",
      "Gene: CYP2A13\n",
      "Symbol: CYP2A13\n",
      "Name: cytochrome P450 family 2 subfamily A member 13\n",
      "Summary: This gene encodes a member of the cytochrome P450 superfamily of enzymes. The cytochrome P450 proteins are monooxygenases which catalyze many reactions involved in drug metabolism and synthesis of cholesterol, steroids and other lipids. This protein localizes to the endoplasmic reticulum. Although its endogenous substrate has not been determined, it is known to metabolize 4-(methylnitrosamino)-1-(3-pyridyl)-1-butanone, a major nitrosamine specific to tobacco. This gene is part of a large cluster of cytochrome P450 genes from the CYP2A, CYP2B and CYP2F subfamilies on chromosome 19q. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - xenobiotic metabolic process\n",
      "    - coumarin metabolic process\n",
      "    - coumarin metabolic process\n",
      "    - epoxygenase P450 pathway\n",
      "    - aflatoxin metabolic process\n",
      "  CC:\n",
      "    - cytoplasm\n",
      "    - endoplasmic reticulum membrane\n",
      "    - intracellular membrane-bounded organelle\n",
      "  MF:\n",
      "    - monooxygenase activity\n",
      "    - iron ion binding\n",
      "    - coumarin 7-hydroxylase activity\n",
      "    - arachidonate epoxygenase activity\n",
      "    - oxidoreductase activity, acting on paired donors, with incorporation or reduction of molecular oxygen, reduced flavin or flavoprotein as one donor, and incorporation of one atom of oxygen\n",
      "    - heme binding\n",
      "    - heme binding\n",
      "UniProt: {'Swiss-Prot': 'Q16696'}\n",
      "\n",
      "Gene: GABRA5\n",
      "Symbol: GABRA5\n",
      "Name: gamma-aminobutyric acid type A receptor subunit alpha5\n",
      "Summary: GABA is the major inhibitory neurotransmitter in the mammalian brain where it acts at GABA-A receptors, which are ligand-gated chloride channels. Chloride conductance of these channels can be modulated by agents such as benzodiazepines that bind to the GABA-A receptor. At least 16 distinct subunits of GABA-A receptors have been identified. Transcript variants utilizing three different alternative non-coding first exons have been described. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - behavioral fear response\n",
      "    - signal transduction\n",
      "    - gamma-aminobutyric acid signaling pathway\n",
      "    - gamma-aminobutyric acid signaling pathway\n",
      "    - associative learning\n",
      "    - synaptic transmission, GABAergic\n",
      "    - regulation of postsynaptic membrane potential\n",
      "    - inner ear receptor cell development\n",
      "    - innervation\n",
      "    - cochlea development\n",
      "    - regulation of presynaptic membrane potential\n",
      "    - chloride transmembrane transport\n",
      "    - inhibitory synapse assembly\n",
      "  CC:\n",
      "    - nucleoplasm\n",
      "    - cytosol\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - dendrite membrane\n",
      "    - neuronal cell body membrane\n",
      "    - chloride channel complex\n",
      "    - presynaptic membrane\n",
      "    - postsynapse\n",
      "    - GABA-ergic synapse\n",
      "    - postsynaptic specialization membrane\n",
      "    - GABA-A receptor complex\n",
      "    - GABA-A receptor complex\n",
      "  MF:\n",
      "    - GABA-A receptor activity\n",
      "    - GABA-A receptor activity\n",
      "    - chloride channel activity\n",
      "    - benzodiazepine receptor activity\n",
      "    - GABA-gated chloride ion channel activity\n",
      "    - GABA-gated chloride ion channel activity\n",
      "    - signaling receptor activity\n",
      "    - GABA receptor binding\n",
      "    - ligand-gated monoatomic ion channel activity involved in regulation of presynaptic membrane potential\n",
      "    - ligand-gated monoatomic ion channel activity involved in regulation of presynaptic membrane potential\n",
      "    - ligand-gated monoatomic ion channel activity involved in regulation of presynaptic membrane potential\n",
      "    - transmitter-gated monoatomic ion channel activity involved in regulation of postsynaptic membrane potential\n",
      "UniProt: {'Swiss-Prot': 'P31644', 'TrEMBL': ['G3V2Q9', 'G3V2K2', 'G3V2G8', 'G3V2Y5', 'G3V408', 'G3V296']}\n",
      "\n",
      "Gene: SLC22A9\n",
      "Symbol: SLC22A9\n",
      "Name: solute carrier family 22 member 9\n",
      "Summary: Enables anion:anion antiporter activity; short-chain fatty acid transmembrane transporter activity; and sodium-independent organic anion transmembrane transporter activity. Involved in hormone transport; short-chain fatty acid import; and sodium-independent organic anion transport. Located in basolateral plasma membrane. [provided by Alliance of Genome Resources, Apr 2022]\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - hormone transport\n",
      "    - organic anion transport\n",
      "    - short-chain fatty acid transmembrane transport\n",
      "    - sodium-independent organic anion transport\n",
      "  CC:\n",
      "    - basolateral plasma membrane\n",
      "  MF:\n",
      "    - organic anion transmembrane transporter activity\n",
      "    - sodium-independent organic anion transmembrane transporter activity\n",
      "    - short-chain fatty acid transmembrane transporter activity\n",
      "UniProt: {'Swiss-Prot': 'Q8IVM8'}\n",
      "\n",
      "Gene: SLCO6A1\n",
      "Symbol: SLCO6A1\n",
      "Name: solute carrier organic anion transporter family member 6A1\n",
      "Summary: Predicted to enable sodium-independent organic anion transmembrane transporter activity. Predicted to be involved in sodium-independent organic anion transport. Predicted to be located in plasma membrane. Predicted to be integral component of membrane. Predicted to be integral component of plasma membrane. [provided by Alliance of Genome Resources, Apr 2022]\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - sodium-independent organic anion transport\n",
      "    - transmembrane transport\n",
      "  CC:\n",
      "    - basolateral plasma membrane\n",
      "  MF:\n",
      "    - protein binding\n",
      "    - sodium-independent organic anion transmembrane transporter activity\n",
      "UniProt: {'Swiss-Prot': 'Q86UG4', 'TrEMBL': ['A0A140VJU7', 'C9J020', 'H0Y8R6']}\n",
      "\n",
      "Gene: PMEL\n",
      "Symbol: PMEL\n",
      "Name: premelanosome protein\n",
      "Summary: This gene encodes a melanocyte-specific type I transmembrane glycoprotein. The encoded protein is enriched in melanosomes, which are the melanin-producing organelles in melanocytes, and plays an essential role in the structural organization of premelanosomes. This protein is involved in generating internal matrix fibers that define the transition from Stage I to Stage II melanosomes. This protein undergoes a complex pattern of prosttranslational processing and modification that is essential to the proper functioning of the protein. A secreted form of this protein that is released by proteolytic ectodomain shedding may be used as a melanoma-specific serum marker. Alternate splicing results in multiple transcript variants. [provided by RefSeq, Jan 2011].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - melanosome organization\n",
      "    - melanosome organization\n",
      "    - melanin biosynthetic process\n",
      "    - positive regulation of melanin biosynthetic process\n",
      "  CC:\n",
      "    - endoplasmic reticulum\n",
      "    - endoplasmic reticulum membrane\n",
      "    - Golgi apparatus\n",
      "    - plasma membrane\n",
      "    - multivesicular body membrane\n",
      "    - cis-Golgi network membrane\n",
      "    - melanosome membrane\n",
      "    - melanosome membrane\n",
      "    - melanosome\n",
      "    - melanosome\n",
      "    - extracellular exosome\n",
      "    - multivesicular body, internal vesicle\n",
      "  MF:\n",
      "    - protein binding\n",
      "    - identical protein binding\n",
      "UniProt: {'Swiss-Prot': 'P40967', 'TrEMBL': ['F8VYZ1', 'H0YIJ2', 'F8VZC6', 'F8VUB1', 'F8W1D1', 'F8VZV2', 'F8VXH8']}\n",
      "\n",
      "Gene: GJB7\n",
      "Symbol: GJB7\n",
      "Name: gap junction protein beta 7\n",
      "Summary: Connexins, such as GJB7, are involved in the formation of gap junctions, intercellular conduits that directly connect the cytoplasms of contacting cells. Each gap junction channel is formed by docking of 2 hemichannels, each of which contains 6 connexin subunits (Sohl et al., 2003 [PubMed 12881038]).[supplied by OMIM, Mar 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - cell-cell signaling\n",
      "    - transmembrane transport\n",
      "  CC:\n",
      "    - connexin complex\n",
      "  MF:\n",
      "    - gap junction channel activity\n",
      "    - gap junction channel activity\n",
      "UniProt: {'Swiss-Prot': 'Q6PEY0', 'TrEMBL': 'A0A0A0MRM5'}\n",
      "\n",
      "Gene: CYP8B1\n",
      "Symbol: CYP8B1\n",
      "Name: cytochrome P450 family 8 subfamily B member 1\n",
      "Summary: This gene encodes a member of the cytochrome P450 superfamily of enzymes. The cytochrome P450 proteins are monooxygenases which catalyze many reactions involved in drug metabolism and synthesis of cholesterol, steroids and other lipids. This endoplasmic reticulum membrane protein catalyzes the conversion of 7 alpha-hydroxy-4-cholesten-3-one into 7-alpha,12-alpha-dihydroxy-4-cholesten-3-one. The balance between these two steroids determines the relative amounts of cholic acid and chenodeoxycholic acid both of which are secreted in the bile and affect the solubility of cholesterol. This gene is unique among the cytochrome P450 genes in that it is intronless. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - steroid biosynthetic process\n",
      "    - bile acid biosynthetic process\n",
      "    - bile acid biosynthetic process\n",
      "    - bile acid biosynthetic process\n",
      "    - sterol metabolic process\n",
      "    - response to nutrient levels\n",
      "    - positive regulation of intestinal cholesterol absorption\n",
      "    - response to cholesterol\n",
      "  CC:\n",
      "    - endoplasmic reticulum membrane\n",
      "    - endoplasmic reticulum membrane\n",
      "  MF:\n",
      "    - iron ion binding\n",
      "    - sterol 12-alpha-hydroxylase activity\n",
      "    - sterol 12-alpha-hydroxylase activity\n",
      "    - sterol 12-alpha-hydroxylase activity\n",
      "    - sterol 12-alpha-hydroxylase activity\n",
      "    - oxygen binding\n",
      "    - heme binding\n",
      "UniProt: {'Swiss-Prot': 'Q9UNU6', 'TrEMBL': 'C9JFR9'}\n",
      "\n",
      "Gene: COX18\n",
      "Symbol: COX18\n",
      "Name: cytochrome c oxidase assembly factor COX18\n",
      "Summary: This gene encodes a cytochrome c oxidase assembly protein. The encoded protein is essential for integral membrane protein insertion into the mitochondrial inner membrane. It is also required for cytochrome c oxidase assembly and activity. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Jul 2014].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - respiratory chain complex IV assembly\n",
      "    - protein insertion into mitochondrial inner membrane from matrix\n",
      "    - protein insertion into mitochondrial inner membrane from matrix\n",
      "    - mitochondrial cytochrome c oxidase assembly\n",
      "    - mitochondrial cytochrome c oxidase assembly\n",
      "    - mitochondrial cytochrome c oxidase assembly\n",
      "    - protein insertion into mitochondrial membrane\n",
      "  CC:\n",
      "    - mitochondrion\n",
      "    - mitochondrial inner membrane\n",
      "    - mitochondrial inner membrane\n",
      "    - mitochondrial inner membrane\n",
      "  MF:\n",
      "    - protein binding\n",
      "    - membrane insertase activity\n",
      "    - membrane insertase activity\n",
      "    - membrane insertase activity\n",
      "    - membrane insertase activity\n",
      "UniProt: {'Swiss-Prot': 'Q8N8Q8', 'TrEMBL': 'B7ZL88'}\n",
      "\n",
      "Gene: CTBS\n",
      "Symbol: CTBS\n",
      "Name: chitobiase\n",
      "Summary: Chitobiase is a lysosomal glycosidase involved in degradation of asparagine-linked oligosaccharides on glycoproteins (Aronson and Kuranda, 1989 [PubMed 2531691]).[supplied by OMIM, Nov 2010].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - chitin catabolic process\n",
      "    - oligosaccharide catabolic process\n",
      "  CC:\n",
      "    - extracellular space\n",
      "    - lysosome\n",
      "  MF:\n",
      "    - chitinase activity\n",
      "    - chitin binding\n",
      "UniProt: {'Swiss-Prot': 'Q01459', 'TrEMBL': ['F6X5H7', 'Q8TC97']}\n",
      "\n",
      "Gene: CHST6\n",
      "Symbol: CHST6\n",
      "Name: carbohydrate sulfotransferase 6\n",
      "Summary: The protein encoded by this gene is an enzyme that catalyzes the transfer of a sulfate group to the GlcNAc residues of keratan. Keratan sulfate helps maintain corneal transparency. Defects in this gene are a cause of macular corneal dystrophy (MCD). [provided by RefSeq, Jan 2010].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - carbohydrate metabolic process\n",
      "    - N-acetylglucosamine metabolic process\n",
      "    - N-acetylglucosamine metabolic process\n",
      "    - N-acetylglucosamine metabolic process\n",
      "    - sulfur compound metabolic process\n",
      "    - keratan sulfate proteoglycan biosynthetic process\n",
      "    - keratan sulfate proteoglycan biosynthetic process\n",
      "    - keratan sulfate proteoglycan biosynthetic process\n",
      "  CC:\n",
      "    - Golgi membrane\n",
      "    - Golgi apparatus\n",
      "    - trans-Golgi network\n",
      "  MF:\n",
      "    - N-acetylglucosamine 6-O-sulfotransferase activity\n",
      "    - N-acetylglucosamine 6-O-sulfotransferase activity\n",
      "    - N-acetylglucosamine 6-O-sulfotransferase activity\n",
      "    - keratan sulfotransferase activity\n",
      "UniProt: {'Swiss-Prot': 'Q9GZX3'}\n",
      "\n",
      "Gene: TMCC2\n",
      "Symbol: TMCC2\n",
      "Name: transmembrane and coiled-coil domain family 2\n",
      "Summary: Involved in amyloid precursor protein metabolic process. Located in endoplasmic reticulum. [provided by Alliance of Genome Resources, Apr 2022]\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - amyloid precursor protein metabolic process\n",
      "    - amyloid precursor protein metabolic process\n",
      "  CC:\n",
      "    - endoplasmic reticulum\n",
      "    - endoplasmic reticulum membrane\n",
      "    - endomembrane system\n",
      "  MF:\n",
      "    - protein binding\n",
      "UniProt: {'Swiss-Prot': 'O75069', 'TrEMBL': ['Q8IW47', 'A0A0C4DFR1']}\n",
      "\n",
      "Gene: CYP19A1\n",
      "Symbol: CYP19A1\n",
      "Name: cytochrome P450 family 19 subfamily A member 1\n",
      "Summary: This gene encodes a member of the cytochrome P450 superfamily of enzymes. The cytochrome P450 proteins are monooxygenases which catalyze many reactions involved in drug metabolism and synthesis of cholesterol, steroids and other lipids. This protein localizes to the endoplasmic reticulum and catalyzes the last steps of estrogen biosynthesis. Mutations in this gene can result in either increased or decreased aromatase activity; the associated phenotypes suggest that estrogen functions both as a sex steroid hormone and in growth or differentiation. Alternative promoter use and alternative splicing results in multiple transcript variants that have different tissue specificities. [provided by RefSeq, Dec 2016].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - negative regulation of chronic inflammatory response\n",
      "    - steroid biosynthetic process\n",
      "    - estrogen biosynthetic process\n",
      "    - androgen catabolic process\n",
      "    - syncytium formation\n",
      "    - female gonad development\n",
      "    - negative regulation of macrophage chemotaxis\n",
      "    - sterol metabolic process\n",
      "    - female genitalia development\n",
      "    - mammary gland development\n",
      "    - response to estradiol\n",
      "    - uterus development\n",
      "    - prostate gland growth\n",
      "    - testosterone biosynthetic process\n",
      "    - positive regulation of estradiol secretion\n",
      "  CC:\n",
      "    - endoplasmic reticulum\n",
      "    - endoplasmic reticulum\n",
      "    - endoplasmic reticulum membrane\n",
      "    - membrane\n",
      "  MF:\n",
      "    - iron ion binding\n",
      "    - steroid hydroxylase activity\n",
      "    - electron transfer activity\n",
      "    - oxidoreductase activity, acting on paired donors, with incorporation or reduction of molecular oxygen, reduced flavin or flavoprotein as one donor, and incorporation of one atom of oxygen\n",
      "    - oxygen binding\n",
      "    - heme binding\n",
      "    - aromatase activity\n",
      "    - aromatase activity\n",
      "    - estrogen 2-hydroxylase activity\n",
      "UniProt: {'Swiss-Prot': 'P11511', 'TrEMBL': ['Q8IYG4', 'A8K6W3', 'Q05CU4', 'Q8TCA4', 'E7EQ08', 'E7EPL6', 'H0YKN1', 'H0YK57', 'E9PGZ6', 'H0YNJ7', 'H0YLS2', 'H0YLP1']}\n",
      "\n",
      "Gene: CYP2B6\n",
      "Symbol: CYP2B6\n",
      "Name: cytochrome P450 family 2 subfamily B member 6\n",
      "Summary: This gene, CYP2B6, encodes a member of the cytochrome P450 superfamily of enzymes. The cytochrome P450 proteins are monooxygenases which catalyze many reactions involved in drug metabolism and synthesis of cholesterol, steroids and other lipids. This protein localizes to the endoplasmic reticulum and its expression is induced by phenobarbital. The enzyme is known to metabolize some xenobiotics, such as the anti-cancer drugs cyclophosphamide and ifosphamide. Transcript variants for this gene have been described; however, it has not been resolved whether these transcripts are in fact produced by this gene or by a closely related pseudogene, CYP2B7. Both the gene and the pseudogene are located in the middle of a CYP2A pseudogene found in a large cluster of cytochrome P450 genes from the CYP2A, CYP2B and CYP2F subfamilies on chromosome 19q. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - xenobiotic metabolic process\n",
      "    - xenobiotic metabolic process\n",
      "    - xenobiotic metabolic process\n",
      "    - xenobiotic metabolic process\n",
      "    - steroid metabolic process\n",
      "    - epoxygenase P450 pathway\n",
      "    - xenobiotic catabolic process\n",
      "    - ketone metabolic process\n",
      "  CC:\n",
      "    - cytoplasm\n",
      "    - endoplasmic reticulum membrane\n",
      "    - intracellular membrane-bounded organelle\n",
      "  MF:\n",
      "    - monooxygenase activity\n",
      "    - monooxygenase activity\n",
      "    - iron ion binding\n",
      "    - testosterone 16-alpha-hydroxylase activity\n",
      "    - arachidonate epoxygenase activity\n",
      "    - oxidoreductase activity, acting on paired donors, with incorporation or reduction of molecular oxygen, reduced flavin or flavoprotein as one donor, and incorporation of one atom of oxygen\n",
      "    - heme binding\n",
      "    - heme binding\n",
      "    - testosterone 16-beta-hydroxylase activity\n",
      "    - anandamide 8,9 epoxidase activity\n",
      "    - anandamide 11,12 epoxidase activity\n",
      "    - anandamide 14,15 epoxidase activity\n",
      "    - estrogen 2-hydroxylase activity\n",
      "UniProt: {'Swiss-Prot': 'P20813', 'TrEMBL': ['M0QZJ2', 'A0A2R8YFA4']}\n",
      "\n",
      "Gene: PON3\n",
      "Symbol: PON3\n",
      "Name: paraoxonase 3\n",
      "Summary: This gene is a member of the paraoxonase family and lies in a cluster on chromosome 7 with the other two family members. The encoded protein is secreted into the bloodstream and associates with high-density lipoprotein (HDL). The protein also rapidly hydrolyzes lactones and can inhibit the oxidation of low-density lipoprotein (LDL), a function that is believed to slow the initiation and progression of atherosclerosis. Alternatively spliced variants which encode different protein isoforms have been described; however, only one has been fully characterized. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - renal sodium ion transport\n",
      "    - response to toxic substance\n",
      "    - negative regulation of superoxide anion generation\n",
      "    - carboxylic acid catabolic process\n",
      "    - establishment of localization in cell\n",
      "    - lactone catabolic process\n",
      "    - regulation of cellular response to drug\n",
      "  CC:\n",
      "    - extracellular region\n",
      "    - extracellular space\n",
      "    - endoplasmic reticulum membrane\n",
      "    - extracellular exosome\n",
      "  MF:\n",
      "    - aryldialkylphosphatase activity\n",
      "    - arylesterase activity\n",
      "    - arylesterase activity\n",
      "    - arylesterase activity\n",
      "    - protein homodimerization activity\n",
      "    - metal ion binding\n",
      "    - acyl-L-homoserine-lactone lactonohydrolase activity\n",
      "UniProt: {'Swiss-Prot': 'Q15166', 'TrEMBL': ['F8WBV4', 'F2Z2L3', 'F8WD41', 'C9JZ99']}\n",
      "\n",
      "Gene: ABCC10\n",
      "Symbol: ABCC10\n",
      "Name: ATP binding cassette subfamily C member 10\n",
      "Summary: The protein encoded by this gene is a member of the superfamily of ATP-binding cassette (ABC) transporters. ABC proteins transport various molecules across extra- and intra-cellular membranes. ABC genes are divided into seven distinct subfamilies (ABC1, MDR/TAP, MRP, ALD, OABP, GCN20, and White). This ABC full-transporter is a member of the MRP subfamily which is involved in multi-drug resistance. Multiple transcript variants encoding different isoforms have been found for this gene.[provided by RefSeq, Nov 2010].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - leukotriene metabolic process\n",
      "    - lipid transport\n",
      "    - xenobiotic transport\n",
      "    - transmembrane transport\n",
      "    - transmembrane transport\n",
      "    - leukotriene transport\n",
      "    - monoatomic anion transmembrane transport\n",
      "  CC:\n",
      "    - lysosomal membrane\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - basal plasma membrane\n",
      "    - membrane\n",
      "    - basolateral plasma membrane\n",
      "  MF:\n",
      "    - ATP binding\n",
      "    - ABC-type xenobiotic transporter activity\n",
      "    - ABC-type glutathione S-conjugate transporter activity\n",
      "    - ATP hydrolysis activity\n",
      "    - ATPase-coupled transmembrane transporter activity\n",
      "    - ATPase-coupled inorganic anion transmembrane transporter activity\n",
      "    - ABC-type transporter activity\n",
      "UniProt: {'Swiss-Prot': 'Q5T3U5'}\n",
      "\n",
      "Gene: CRYL1\n",
      "Symbol: CRYL1\n",
      "Name: crystallin lambda 1\n",
      "Summary: The uronate cycle functions as an alternative glucose metabolic pathway, accounting for about 5% of daily glucose catabolism. The product of this gene catalyzes the dehydrogenation of L-gulonate into dehydro-L-gulonate in the uronate cycle. The enzyme requires NAD(H) as a coenzyme, and is inhibited by inorganic phosphate. A similar gene in the rabbit is thought to serve a structural role in the lens of the eye. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - fatty acid metabolic process\n",
      "    - D-glucuronate catabolic process to D-xylulose 5-phosphate\n",
      "  CC:\n",
      "    - cytosol\n",
      "    - cytosol\n",
      "    - extracellular exosome\n",
      "  MF:\n",
      "    - protein homodimerization activity\n",
      "    - L-gulonate 3-dehydrogenase activity\n",
      "    - L-gulonate 3-dehydrogenase activity\n",
      "    - NAD+ binding\n",
      "UniProt: {'Swiss-Prot': 'Q9Y2S2', 'TrEMBL': ['A0A2R8Y4K2', 'V9HWG2', 'A0A087WW38', 'A0A2R8YFQ7', 'A0A2R8Y699', 'A0A2R8Y5S5', 'A0A2R8YCZ4', 'A0A2R8Y7B4']}\n",
      "\n",
      "Gene: SLC22A13\n",
      "Symbol: SLC22A13\n",
      "Name: solute carrier family 22 member 13\n",
      "Summary: This gene encodes a member of the organic-cation transporter family. It is located in a gene cluster with another member of the family, organic cation transporter like 4. The encoded protein is a transmembrane protein involved in the transport of small molecules. This protein can function to mediate urate uptake and is a high affinity nicotinate exchanger in the kidneys and the intestine. [provided by RefSeq, Sep 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - positive regulation of T cell mediated cytotoxicity directed against tumor cell target\n",
      "    - urate transport\n",
      "    - NAD biosynthesis via nicotinamide riboside salvage pathway\n",
      "    - negative regulation of fatty acid metabolic process\n",
      "    - transmembrane transport\n",
      "    - nicotinate transport\n",
      "  CC:\n",
      "    - endoplasmic reticulum\n",
      "    - Golgi apparatus\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - apical plasma membrane\n",
      "    - extracellular exosome\n",
      "  MF:\n",
      "    - urate transmembrane transporter activity\n",
      "    - nicotinate transmembrane transporter activity\n",
      "    - nicotinate transmembrane transporter activity\n",
      "UniProt: {'Swiss-Prot': 'Q9Y226', 'TrEMBL': ['A0A3B3ITC0', 'H0Y4Y1']}\n",
      "\n",
      "Gene: CYP7B1\n",
      "Symbol: CYP7B1\n",
      "Name: cytochrome P450 family 7 subfamily B member 1\n",
      "Summary: This gene encodes a member of the cytochrome P450 superfamily of enzymes. The cytochrome P450 proteins are monooxygenases which catalyze many reactions involved in drug metabolism and synthesis of cholesterol, steroids and other lipids. This endoplasmic reticulum membrane protein catalyzes the first reaction in the cholesterol catabolic pathway of extrahepatic tissues, which converts cholesterol to bile acids. This enzyme likely plays a minor role in total bile acid synthesis, but may also be involved in the development of atherosclerosis, neurosteroid metabolism and sex hormone synthesis. Mutations in this gene have been associated with hereditary spastic paraplegia (SPG5 or HSP), an autosomal recessive disorder. [provided by RefSeq, Apr 2016].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - steroid biosynthetic process\n",
      "    - bile acid biosynthetic process\n",
      "    - bile acid biosynthetic process\n",
      "    - bile acid biosynthetic process\n",
      "    - cholesterol metabolic process\n",
      "    - sterol metabolic process\n",
      "    - estrogen receptor signaling pathway\n",
      "    - negative regulation of intracellular estrogen receptor signaling pathway\n",
      "    - B cell chemotaxis\n",
      "    - cholesterol homeostasis\n",
      "    - epithelial cell proliferation\n",
      "    - positive regulation of epithelial cell proliferation\n",
      "    - prostate gland epithelium morphogenesis\n",
      "  CC:\n",
      "    - endoplasmic reticulum membrane\n",
      "  MF:\n",
      "    - iron ion binding\n",
      "    - protein binding\n",
      "    - oxysterol 7-alpha-hydroxylase activity\n",
      "    - oxysterol 7-alpha-hydroxylase activity\n",
      "    - heme binding\n",
      "    - 24S-hydroxycholesterol 7-alpha-hydroxylase activity\n",
      "    - 25-hydroxycholesterol 7-alpha-hydroxylase activity\n",
      "UniProt: {'Swiss-Prot': 'O75881', 'TrEMBL': 'Q05C57'}\n",
      "\n",
      "Gene: FMO5\n",
      "Symbol: FMO5\n",
      "Name: flavin containing dimethylaniline monoxygenase 5\n",
      "Summary: Metabolic N-oxidation of the diet-derived amino-trimethylamine (TMA) is mediated by flavin-containing monooxygenase and is subject to an inherited FMO3 polymorphism in man resulting in a small subpopulation with reduced TMA N-oxidation capacity resulting in fish odor syndrome Trimethylaminuria. Three forms of the enzyme, FMO1 found in fetal liver, FMO2 found in adult liver, and FMO3 are encoded by genes clustered in the 1q23-q25 region. Flavin-containing monooxygenases are NADPH-dependent flavoenzymes that catalyzes the oxidation of soft nucleophilic heteroatom centers in drugs, pesticides, and xenobiotics. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Jan 2009].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - lipid metabolic process\n",
      "    - xenobiotic metabolic process\n",
      "    - NADPH oxidation\n",
      "    - regulation of cholesterol metabolic process\n",
      "  CC:\n",
      "    - endoplasmic reticulum\n",
      "    - endoplasmic reticulum membrane\n",
      "    - cytosol\n",
      "  MF:\n",
      "    - aldehyde oxidase activity\n",
      "    - monooxygenase activity\n",
      "    - N,N-dimethylaniline monooxygenase activity\n",
      "    - hypotaurine monooxygenase activity\n",
      "    - flavin adenine dinucleotide binding\n",
      "    - NADP binding\n",
      "    - NADPH oxidase H202-forming activity\n",
      "UniProt: {'Swiss-Prot': 'P49326', 'TrEMBL': ['E9PJF3', 'E9PP51', 'E9PQ84']}\n",
      "\n",
      "Gene: NT5C\n",
      "Symbol: NT5C\n",
      "Name: 5', 3'-nucleotidase, cytosolic\n",
      "Summary: This gene encodes a nucleotidase that catalyzes the dephosphorylation of the 5' deoxyribonucleotides (dNTP) and 2'(3')-dNTP and ribonucleotides, but not 5' ribonucleotides. Of the different forms of nucleotidases characterized, this enzyme is unique in its preference for 5'-dNTP. It may be one of the enzymes involved in regulating the size of dNTP pools in cells. Alternatively spliced transcript variants have been found for this gene. [provided by RefSeq, Nov 2011].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - allantoin metabolic process\n",
      "    - IMP catabolic process\n",
      "    - dCMP catabolic process\n",
      "    - pyrimidine deoxyribonucleotide catabolic process\n",
      "    - pyrimidine deoxyribonucleotide catabolic process\n",
      "    - dephosphorylation\n",
      "    - amide catabolic process\n",
      "    - UMP catabolic process\n",
      "    - dGMP catabolic process\n",
      "    - dTMP catabolic process\n",
      "    - dUMP catabolic process\n",
      "  CC:\n",
      "    - nucleus\n",
      "    - cytoplasm\n",
      "    - mitochondrion\n",
      "    - cytosol\n",
      "    - cytosol\n",
      "    - extracellular exosome\n",
      "  MF:\n",
      "    - nucleotidase activity\n",
      "    - 5'-nucleotidase activity\n",
      "    - pyrimidine nucleotide binding\n",
      "    - identical protein binding\n",
      "    - metal ion binding\n",
      "    - IMP 5'-nucleotidase activity\n",
      "UniProt: {'Swiss-Prot': 'Q8TCD5', 'TrEMBL': ['J3KSF0', 'V9HWF3', 'J3KRC4', 'J3KSX6', 'J3QQX8', 'J3KSY6']}\n",
      "\n",
      "Gene: SLCO1B1\n",
      "Symbol: SLCO1B1\n",
      "Name: solute carrier organic anion transporter family member 1B1\n",
      "Summary: This gene encodes a liver-specific member of the organic anion transporter family. The encoded protein is a transmembrane receptor that mediates the sodium-independent uptake of numerous endogenous compounds including bilirubin, 17-beta-glucuronosyl estradiol and leukotriene C4. This protein is also involved in the removal of drug compounds such as statins, bromosulfophthalein and rifampin from the blood into the hepatocytes. Polymorphisms in the gene encoding this protein are associated with impaired transporter function. [provided by RefSeq, Mar 2009].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - xenobiotic metabolic process\n",
      "    - monoatomic ion transport\n",
      "    - organic anion transport\n",
      "    - bile acid and bile salt transport\n",
      "    - bile acid and bile salt transport\n",
      "    - prostaglandin transport\n",
      "    - heme catabolic process\n",
      "    - sodium-independent organic anion transport\n",
      "    - sodium-independent organic anion transport\n",
      "    - transmembrane transport\n",
      "    - thyroid hormone transport\n",
      "  CC:\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - basal plasma membrane\n",
      "    - membrane\n",
      "    - basolateral plasma membrane\n",
      "  MF:\n",
      "    - organic anion transmembrane transporter activity\n",
      "    - bile acid transmembrane transporter activity\n",
      "    - bile acid transmembrane transporter activity\n",
      "    - prostaglandin transmembrane transporter activity\n",
      "    - sodium-independent organic anion transmembrane transporter activity\n",
      "    - sodium-independent organic anion transmembrane transporter activity\n",
      "    - thyroid hormone transmembrane transporter activity\n",
      "UniProt: {'Swiss-Prot': 'Q9Y6L6', 'TrEMBL': 'Q05CV5'}\n",
      "\n",
      "Gene: ABCB5\n",
      "Symbol: ABCB5\n",
      "Name: ATP binding cassette subfamily B member 5\n",
      "Summary: ABCB5 belongs to the ATP-binding cassette (ABC) transporter superfamily of integral membrane proteins. These proteins participate in ATP-dependent transmembrane transport of structurally diverse molecules ranging from small ions, sugars, and peptides to more complex organic molecules (Chen et al., 2005 [PubMed 15760339]).[supplied by OMIM, Mar 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - eye development\n",
      "    - cell differentiation\n",
      "    - regulation of membrane potential\n",
      "    - xenobiotic transport\n",
      "    - transmembrane transport\n",
      "    - transmembrane transport\n",
      "    - transmembrane transport\n",
      "  CC:\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - apical plasma membrane\n",
      "  MF:\n",
      "    - ATP binding\n",
      "    - ABC-type xenobiotic transporter activity\n",
      "    - efflux transmembrane transporter activity\n",
      "    - ATP hydrolysis activity\n",
      "    - ATPase-coupled transmembrane transporter activity\n",
      "    - ATPase-coupled transmembrane transporter activity\n",
      "UniProt: {'Swiss-Prot': 'Q2M3G0', 'TrEMBL': 'H7C165'}\n",
      "\n",
      "Gene: KLK15\n",
      "Symbol: KLK15\n",
      "Name: kallikrein related peptidase 15\n",
      "Summary: Kallikreins are a subgroup of serine proteases having diverse physiological functions. Growing evidence suggests that many kallikreins are implicated in carcinogenesis and some have potential as novel cancer and other disease biomarkers. This gene is one of the fifteen kallikrein subfamily members located in a cluster on chromosome 19. In prostate cancer, this gene has increased expression, which indicates its possible use as a diagnostic or prognostic marker for prostate cancer. The gene contains multiple polyadenylation sites and alternative splicing results in multiple transcript variants encoding distinct isoforms. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - proteolysis\n",
      "    - protein maturation\n",
      "  CC:\n",
      "    - extracellular space\n",
      "    - secretory granule\n",
      "  MF:\n",
      "    - serine-type endopeptidase activity\n",
      "    - serine-type endopeptidase activity\n",
      "    - protein binding\n",
      "    - serine-type peptidase activity\n",
      "UniProt: {'Swiss-Prot': 'Q9H2R5', 'TrEMBL': ['A0A6B7HBY5', 'Q6UBM2', 'M0R0D7', 'M0R2F7', 'M0QX22']}\n",
      "\n",
      "Gene: SOST\n",
      "Symbol: SOST\n",
      "Name: sclerostin\n",
      "Summary: Sclerostin is a secreted glycoprotein with a C-terminal cysteine knot-like (CTCK) domain and sequence similarity to the DAN (differential screening-selected gene aberrative in neuroblastoma) family of bone morphogenetic protein (BMP) antagonists. Loss-of-function mutations in this gene are associated with an autosomal-recessive disorder, sclerosteosis, which causes progressive bone overgrowth. A deletion downstream of this gene, which causes reduced sclerostin expression, is associated with a milder form of the disorder called van Buchem disease. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - ossification\n",
      "    - response to mechanical stimulus\n",
      "    - negative regulation of Wnt signaling pathway\n",
      "    - negative regulation of ossification\n",
      "    - BMP signaling pathway\n",
      "    - negative regulation of BMP signaling pathway\n",
      "    - negative regulation of BMP signaling pathway\n",
      "    - negative regulation of protein-containing complex assembly\n",
      "    - positive regulation of DNA-templated transcription\n",
      "    - canonical Wnt signaling pathway\n",
      "    - cellular response to parathyroid hormone stimulus\n",
      "    - negative regulation of canonical Wnt signaling pathway\n",
      "  CC:\n",
      "    - extracellular region\n",
      "    - extracellular space\n",
      "    - Golgi apparatus\n",
      "    - protein-containing complex\n",
      "    - collagen-containing extracellular matrix\n",
      "  MF:\n",
      "    - protein binding\n",
      "    - heparin binding\n",
      "    - carbohydrate binding\n",
      "    - BMP binding\n",
      "    - DNA-binding transcription factor binding\n",
      "    - molecular function inhibitor activity\n",
      "    - molecular function inhibitor activity\n",
      "UniProt: {'Swiss-Prot': 'Q9BQB4'}\n",
      "\n",
      "Gene: CA5B\n",
      "Symbol: CA5B\n",
      "Name: carbonic anhydrase 5B\n",
      "Summary: Carbonic anhydrases (CAs) are a large family of zinc metalloenzymes that catalyze the reversible hydration of carbon dioxide. They participate in a variety of biological processes, including respiration, calcification, acid-base balance, bone resorption, and the formation of aqueous humor, cerebrospinal fluid, saliva, and gastric acid. They show extensive diversity in tissue distribution and in their subcellular localization. This gene encodes carbonic anhydrase 5B. CA5B, and the related CA5A gene, has its expression localized in the mitochondria though CA5B has a wider tissue distribution than CA5A, which is restricted to the liver, kidneys, and skeletal muscle. A carbonic anhydrase pseudogene (CA5BP1) is adjacent to the CA5B gene and these two loci produce CA5BP1-CA5B readthrough transcripts. [provided by RefSeq, Jan 2019].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - response to bacterium\n",
      "  CC:\n",
      "    - cytoplasm\n",
      "    - mitochondrion\n",
      "    - mitochondrion\n",
      "    - mitochondrion\n",
      "    - mitochondrial matrix\n",
      "  MF:\n",
      "    - carbonate dehydratase activity\n",
      "    - carbonate dehydratase activity\n",
      "    - protein binding\n",
      "    - zinc ion binding\n",
      "UniProt: {'Swiss-Prot': 'Q9Y2D0', 'TrEMBL': ['C9JA11', 'C9IZP3', 'H7C4G3']}\n",
      "\n",
      "Gene: ZNF684\n",
      "Symbol: ZNF684\n",
      "Name: zinc finger protein 684\n",
      "Summary: Enables sequence-specific double-stranded DNA binding activity. Predicted to be involved in negative regulation of transcription by RNA polymerase II. Predicted to act upstream of or within innate immune response and negative regulation of single stranded viral RNA replication via double stranded DNA intermediate. Predicted to be located in nucleus. [provided by Alliance of Genome Resources, Apr 2022]\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - regulation of transcription by RNA polymerase II\n",
      "  CC:\n",
      "    - nucleus\n",
      "  MF:\n",
      "    - RNA polymerase II transcription regulatory region sequence-specific DNA binding\n",
      "    - DNA-binding transcription factor activity, RNA polymerase II-specific\n",
      "    - zinc ion binding\n",
      "    - sequence-specific double-stranded DNA binding\n",
      "UniProt: {'Swiss-Prot': 'Q5T5D7', 'TrEMBL': ['Q5T5D9', 'B3KSP5', 'Q5T5D8']}\n",
      "\n",
      "Gene: FMO2\n",
      "Symbol: FMO2\n",
      "Name: flavin containing dimethylaniline monoxygenase 2\n",
      "Summary: This gene encodes a flavin-containing monooxygenase family member. It is an NADPH-dependent enzyme that catalyzes the N-oxidation of some primary alkylamines through an N-hydroxylamine intermediate. However, some human populations contain an allele (FMO2*2A) with a premature stop codon, resulting in a protein that is C-terminally-truncated, has no catalytic activity, and is likely degraded rapidly. This gene is found in a cluster with other related family members on chromosome 1. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Aug 2014].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - organic acid metabolic process\n",
      "    - NADP metabolic process\n",
      "    - xenobiotic metabolic process\n",
      "    - toxin metabolic process\n",
      "    - negative regulation of fatty acid oxidation\n",
      "    - NADPH oxidation\n",
      "    - oxygen metabolic process\n",
      "    - energy homeostasis\n",
      "  CC:\n",
      "    - endoplasmic reticulum membrane\n",
      "    - membrane\n",
      "  MF:\n",
      "    - N,N-dimethylaniline monooxygenase activity\n",
      "    - N,N-dimethylaniline monooxygenase activity\n",
      "    - hypotaurine monooxygenase activity\n",
      "    - flavin adenine dinucleotide binding\n",
      "    - NADP binding\n",
      "UniProt: {'Swiss-Prot': 'Q99518', 'TrEMBL': ['B4E2Q9', 'Q5JPC7', 'A0A0D9SFZ0', 'A0A0D9SG76']}\n",
      "\n",
      "Gene: ABCC11\n",
      "Symbol: ABCC11\n",
      "Name: ATP binding cassette subfamily C member 11\n",
      "Summary: The protein encoded by this gene is a member of the superfamily of ATP-binding cassette (ABC) transporters. ABC proteins transport various molecules across extra- and intra-cellular membranes. ABC genes are divided into seven distinct subfamilies (ABC1, MDR/TAP, MRP, ALD, OABP, GCN20, White). This ABC full transporter is a member of the MRP subfamily which is involved in multi-drug resistance. The product of this gene participates in physiological processes involving bile acids, conjugated steroids, and cyclic nucleotides. In addition, a SNP in this gene is responsible for determination of human earwax type. This gene and family member ABCC12 are determined to be derived by duplication and are both localized to chromosome 16q12.1. Multiple alternatively spliced transcript variants have been described for this gene. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - organic anion transport\n",
      "    - organic anion transport\n",
      "    - bile acid and bile salt transport\n",
      "    - purine nucleotide transport\n",
      "    - xenobiotic transport\n",
      "    - transmembrane transport\n",
      "    - transmembrane transport\n",
      "    - leukotriene transport\n",
      "    - monoatomic anion transmembrane transport\n",
      "  CC:\n",
      "    - vacuolar membrane\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - apical plasma membrane\n",
      "    - cytoplasmic vesicle membrane\n",
      "    - extracellular exosome\n",
      "  MF:\n",
      "    - ATP binding\n",
      "    - organic anion transmembrane transporter activity\n",
      "    - organic anion transmembrane transporter activity\n",
      "    - ABC-type xenobiotic transporter activity\n",
      "    - purine nucleotide transmembrane transporter activity\n",
      "    - purine nucleotide transmembrane transporter activity\n",
      "    - ABC-type glutathione S-conjugate transporter activity\n",
      "    - ABC-type bile acid transporter activity\n",
      "    - ATP hydrolysis activity\n",
      "    - ATPase-coupled transmembrane transporter activity\n",
      "    - ATPase-coupled inorganic anion transmembrane transporter activity\n",
      "UniProt: {'Swiss-Prot': 'Q96J66', 'TrEMBL': 'H3BRJ2'}\n",
      "\n",
      "Gene: ZNF582\n",
      "Symbol: ZNF582\n",
      "Name: zinc finger protein 582\n",
      "Summary: The protein encoded by this gene is a zing finger protein and putative transcription factor that is highly methylated in cervical cancers. Two transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Feb 2016].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - regulation of transcription by RNA polymerase II\n",
      "  CC:\n",
      "    - nucleus\n",
      "  MF:\n",
      "    - RNA polymerase II transcription regulatory region sequence-specific DNA binding\n",
      "    - DNA-binding transcription factor activity, RNA polymerase II-specific\n",
      "    - protein binding\n",
      "    - zinc ion binding\n",
      "UniProt: {'Swiss-Prot': 'Q96NG8', 'TrEMBL': ['K7ESE7', 'A0A075B771', 'A0A075B772', 'K7EKI7']}\n",
      "\n",
      "Gene: FAAH\n",
      "Symbol: FAAH\n",
      "Name: fatty acid amide hydrolase\n",
      "Summary: This gene encodes a protein that is responsible for the hydrolysis of a number of primary and secondary fatty acid amides, including the neuromodulatory compounds anandamide and oleamide. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - fatty acid catabolic process\n",
      "    - fatty acid catabolic process\n",
      "    - arachidonate metabolic process\n",
      "    - positive regulation of vasoconstriction\n",
      "    - monoacylglycerol catabolic process\n",
      "    - regulation of trans-synaptic signaling by endocannabinoid, modulating synaptic transmission\n",
      "  CC:\n",
      "    - endoplasmic reticulum membrane\n",
      "    - cytoskeleton\n",
      "    - organelle membrane\n",
      "    - presynapse\n",
      "    - postsynapse\n",
      "    - glutamatergic synapse\n",
      "  MF:\n",
      "    - amidase activity\n",
      "    - protein binding\n",
      "    - phospholipid binding\n",
      "    - fatty acid amide hydrolase activity\n",
      "    - fatty acid amide hydrolase activity\n",
      "    - fatty acid amide hydrolase activity\n",
      "    - fatty acid amide hydrolase activity\n",
      "    - identical protein binding\n",
      "    - monoacylglycerol lipase activity\n",
      "UniProt: {'Swiss-Prot': 'O00519', 'TrEMBL': ['A0A087WYA0', 'Q9UG55']}\n",
      "\n",
      "Gene: HLA-DRB1\n",
      "Symbol: HLA-DRB1\n",
      "Name: major histocompatibility complex, class II, DR beta 1\n",
      "Summary: HLA-DRB1 belongs to the HLA class II beta chain paralogs. The class II molecule is a heterodimer consisting of an alpha (DRA) and a beta chain (DRB), both anchored in the membrane. It plays a central role in the immune system by presenting peptides derived from extracellular proteins. Class II molecules are expressed in antigen presenting cells. The beta chain is approximately 26-28 kDa. It is encoded by 6 exons. Exon one encodes the leader peptide; exons 2 and 3 encode the two extracellular domains; exon 4 encodes the transmembrane domain; and exon 5 encodes the cytoplasmic tail. Within the DR molecule the beta chain contains all the polymorphisms specifying the peptide binding specificities. Hundreds of DRB1 alleles have been described and some alleles have increased frequencies associated with certain diseases or conditions. For example, DRB1*1302 has been related to acute and chronic hepatitis B virus persistence. There are multiple pseudogenes of this gene. [provided by RefSeq, Jul 2020].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - positive regulation of T cell mediated cytotoxicity\n",
      "    - positive regulation of protein phosphorylation\n",
      "    - inflammatory response to antigenic stimulus\n",
      "    - myeloid dendritic cell antigen processing and presentation\n",
      "    - antigen processing and presentation of endogenous peptide antigen via MHC class II\n",
      "    - peptide antigen assembly with MHC class II protein complex\n",
      "    - peptide antigen assembly with MHC class II protein complex\n",
      "    - positive regulation of T cell mediated immune response to tumor cell\n",
      "    - negative regulation of inflammatory response to antigenic stimulus\n",
      "    - immune response\n",
      "    - immune response\n",
      "    - immune response\n",
      "    - immune response\n",
      "    - humoral immune response\n",
      "    - cytoskeleton organization\n",
      "    - signal transduction\n",
      "    - signal transduction\n",
      "    - epidermis development\n",
      "    - detection of bacterium\n",
      "    - antigen processing and presentation of exogenous peptide antigen via MHC class II\n",
      "    - antigen processing and presentation of exogenous peptide antigen via MHC class II\n",
      "    - macrophage differentiation\n",
      "    - regulation of interleukin-10 production\n",
      "    - regulation of interleukin-4 production\n",
      "    - negative regulation of type II interferon production\n",
      "    - positive regulation of CD4-positive, CD25-positive, alpha-beta regulatory T cell differentiation\n",
      "    - positive regulation of kinase activity\n",
      "    - positive regulation of insulin secretion involved in cellular response to glucose stimulus\n",
      "    - T-helper 1 type immune response\n",
      "    - negative regulation of T cell proliferation\n",
      "    - positive regulation of canonical NF-kappaB signal transduction\n",
      "    - positive regulation of memory T cell differentiation\n",
      "    - positive regulation of MAPK cascade\n",
      "    - regulation of T-helper cell differentiation\n",
      "    - positive regulation of monocyte differentiation\n",
      "    - positive regulation of DNA-templated transcription\n",
      "    - positive regulation of viral entry into host cell\n",
      "    - positive regulation of immune response\n",
      "    - T cell receptor signaling pathway\n",
      "    - positive regulation of T cell activation\n",
      "    - protein tetramerization\n",
      "    - positive regulation of ERK1 and ERK2 cascade\n",
      "    - positive regulation of CD4-positive, alpha-beta T cell activation\n",
      "  CC:\n",
      "    - Golgi membrane\n",
      "    - immunological synapse\n",
      "    - extracellular space\n",
      "    - lysosomal membrane\n",
      "    - lysosomal membrane\n",
      "    - lysosomal membrane\n",
      "    - intermediate filament\n",
      "    - plasma membrane\n",
      "    - external side of plasma membrane\n",
      "    - cell surface\n",
      "    - ER to Golgi transport vesicle membrane\n",
      "    - membrane\n",
      "    - membrane\n",
      "    - membrane\n",
      "    - transport vesicle membrane\n",
      "    - endocytic vesicle membrane\n",
      "    - clathrin-coated endocytic vesicle membrane\n",
      "    - late endosome membrane\n",
      "    - late endosome membrane\n",
      "    - trans-Golgi network membrane\n",
      "    - MHC class II protein complex\n",
      "    - MHC class II protein complex\n",
      "    - MHC class II protein complex\n",
      "    - extracellular exosome\n",
      "    - lumenal side of endoplasmic reticulum membrane\n",
      "    - autolysosome membrane\n",
      "  MF:\n",
      "    - structural constituent of cytoskeleton\n",
      "    - protein binding\n",
      "    - MHC class II protein complex binding\n",
      "    - MHC class II protein complex binding\n",
      "    - polysaccharide binding\n",
      "    - MHC class II receptor activity\n",
      "    - MHC class II receptor activity\n",
      "    - MHC class II receptor activity\n",
      "    - peptide antigen binding\n",
      "    - peptide antigen binding\n",
      "    - T cell receptor binding\n",
      "    - CD4 receptor binding\n",
      "UniProt: {'Swiss-Prot': 'P01911', 'TrEMBL': ['Q5Y7D1', 'A0A224MM52', 'D7RIH8', 'X5DNQ0', 'A0A8Q3WLF4', 'A0A5S8K798', 'A0A5S8K7D6']}\n",
      "\n",
      "Gene: KIAA1549L\n",
      "Symbol: KIAA1549L\n",
      "Name: KIAA1549 like\n",
      "Summary: Predicted to be integral component of membrane. [provided by Alliance of Genome Resources, Apr 2022]\n",
      "GO Annotations:\n",
      "  CC:\n",
      "    - membrane\n",
      "UniProt: {'Swiss-Prot': 'Q6ZVL6', 'TrEMBL': ['A0A5F9UK30', 'Q12914', 'H0YDE5', 'A0A590UJI0']}\n"
     ]
    }
   ],
   "source": [
    "# -------------------------------\n",
    "# 1. Load and filter the predictions\n",
    "# -------------------------------\n",
    "# Assuming your CSV file is named \"predictions.csv\"\n",
    "df = pd.read_csv(\"../results/emb_extraction_model_pred.csv\")\n",
    "\n",
    "# Filter for genes with Label 0 but predicted as 1 (false positives)\n",
    "false_positives = df[(df['Label'] == 0) & (df['Pred'] == 1)]\n",
    "print(\"Genes with Label 0 but predicted as 1:\")\n",
    "print(false_positives)\n",
    "\n",
    "# -------------------------------\n",
    "# 2. Retrieve gene information using MyGene.info\n",
    "# -------------------------------\n",
    "mg = mygene.MyGeneInfo()\n",
    "gene_info = {}\n",
    "\n",
    "for gene in false_positives['Gene']:\n",
    "    # Query for gene details: symbol, name, summary, Gene Ontology terms, and UniProt links\n",
    "    result = mg.query(gene, species='human', fields='symbol,name,summary,go,uniprot', size=1)\n",
    "    if result['hits']:\n",
    "        gene_info[gene] = result['hits'][0]\n",
    "    else:\n",
    "        gene_info[gene] = \"No information found\"\n",
    "\n",
    "# Print out the gene information\n",
    "print(\"\\nBiological information for each false positive gene:\")\n",
    "for gene, info in gene_info.items():\n",
    "    print(f\"\\nGene: {gene}\")\n",
    "    if isinstance(info, dict):\n",
    "        print(\"Symbol:\", info.get('symbol', 'N/A'))\n",
    "        print(\"Name:\", info.get('name', 'N/A'))\n",
    "        print(\"Summary:\", info.get('summary', 'N/A'))\n",
    "        # Display GO annotations if available\n",
    "        if 'go' in info:\n",
    "            print(\"GO Annotations:\")\n",
    "            for category, terms in info['go'].items():\n",
    "                print(f\"  {category.upper()}:\")\n",
    "                # 'terms' may be a list or a single dict\n",
    "                if isinstance(terms, list):\n",
    "                    for term in terms:\n",
    "                        print(\"    -\", term.get('term', 'N/A'))\n",
    "                elif isinstance(terms, dict):\n",
    "                    print(\"    -\", terms.get('term', 'N/A'))\n",
    "        # Display UniProt info if available\n",
    "        if 'uniprot' in info:\n",
    "            print(\"UniProt:\", info['uniprot'])\n",
    "    else:\n",
    "        print(info)\n",
    "        "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "\n",
    "# -------------------------------\n",
    "# 3. Functional enrichment analysis using g:Profiler\n",
    "# -------------------------------\n",
    "gp = GProfiler(return_dataframe=True)\n",
    "genes_list = false_positives['Gene'].tolist()\n",
    "enrichment_results = gp.profile(organism='hsapiens', query=genes_list)\n",
    "\n",
    "print(\"\\nEnrichment Analysis Results:\")\n",
    "print(enrichment_results.head(10))  # display the top 10 enriched terms"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": ".dg_env",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.12.8"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
