{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 11,
   "id": "87f5fd37",
   "metadata": {},
   "outputs": [],
   "source": [
    "from pathlib import Path\n",
    "\n",
    "import pandas as pd\n",
    "import numpy as np\n",
    "import mygene\n"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "aad31f9b",
   "metadata": {},
   "source": [
    "### Results"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "id": "bdb787b8",
   "metadata": {},
   "outputs": [],
   "source": [
    "results_dir = \"../results/alzheimers/CPDB/seed_0/iter_1\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "id": "e6707ff6",
   "metadata": {},
   "outputs": [],
   "source": [
    "y_true = np.load(Path(results_dir, \"y_true.npy\"))\n",
    "y_pred = np.load(Path(results_dir, \"y_pred.npy\"))\n",
    "y_proba = np.load(Path(results_dir, \"y_proba.npy\"))\n",
    "gene_names = np.load(Path(results_dir, \"test_gene_names.npy\"))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "id": "185e567b",
   "metadata": {},
   "outputs": [],
   "source": [
    "predictions = pd.DataFrame({\n",
    "    \"gene_name\": gene_names,\n",
    "    \"y_true\": y_true,\n",
    "    \"y_pred\": y_pred,\n",
    "    \"y_proba\": y_proba\n",
    "})"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "id": "7b97bba9",
   "metadata": {},
   "outputs": [],
   "source": [
    "predictions.sort_values(by=\"y_proba\", ascending=False, inplace=True)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "39d97289",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "23        NTRK1\n",
       "35        PDE4D\n",
       "100        CHAT\n",
       "110     SLC27A1\n",
       "72      HLA-DRA\n",
       "105        MSLN\n",
       "116      SORCS1\n",
       "115        CTBS\n",
       "99       MRPL18\n",
       "79       LGALS4\n",
       "39     MAPKAPK3\n",
       "75        NAT8L\n",
       "19        HDAC1\n",
       "53         HAGH\n",
       "63        DGAT2\n",
       "106         RYK\n",
       "16         NRP1\n",
       "42        RRAS2\n",
       "25        ARRB1\n",
       "55         RPIA\n",
       "78        FBLN1\n",
       "18        BCL10\n",
       "2         APPL1\n",
       "71       ELOVL5\n",
       "83        REPS1\n",
       "Name: gene_name, dtype: object"
      ]
     },
     "execution_count": 14,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "predictions[predictions[\"y_true\"] == 0].head(25)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 15,
   "id": "4b791a7a",
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n",
      "Input sequence provided is already in string format. No operation performed\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Biological information for each gene:\n",
      "\n",
      "Gene: NTRK1\n",
      "Symbol: NTRK1\n",
      "Name: neurotrophic receptor tyrosine kinase 1\n",
      "Summary: This gene encodes a member of the neurotrophic tyrosine kinase receptor (NTKR) family. This kinase is a membrane-bound receptor that, upon neurotrophin binding, phosphorylates itself and members of the MAPK pathway. The presence of this kinase leads to cell differentiation and may play a role in specifying sensory neuron subtypes. Mutations in this gene have been associated with congenital insensitivity to pain, anhidrosis, self-mutilating behavior, cognitive disability and cancer. Alternate transcriptional splice variants of this gene have been found, but only three have been characterized to date. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - protein phosphorylation\n",
      "    - cell surface receptor protein tyrosine kinase signaling pathway\n",
      "    - cell surface receptor protein tyrosine kinase signaling pathway\n",
      "    - spermatogenesis\n",
      "    - nervous system development\n",
      "    - nervous system development\n",
      "    - axon guidance\n",
      "    - learning or memory\n",
      "    - circadian rhythm\n",
      "    - negative regulation of cell population proliferation\n",
      "    - negative regulation of cell population proliferation\n",
      "    - response to xenobiotic stimulus\n",
      "    - programmed cell death involved in cell development\n",
      "    - programmed cell death involved in cell development\n",
      "    - positive regulation of neuron projection development\n",
      "    - positive regulation of neuron projection development\n",
      "    - peptidyl-tyrosine phosphorylation\n",
      "    - olfactory nerve development\n",
      "    - cell differentiation\n",
      "    - B cell differentiation\n",
      "    - neuron projection development\n",
      "    - neuron projection development\n",
      "    - response to nutrient levels\n",
      "    - response to nicotine\n",
      "    - peptidyl-tyrosine autophosphorylation\n",
      "    - peptidyl-tyrosine autophosphorylation\n",
      "    - nerve growth factor signaling pathway\n",
      "    - nerve growth factor signaling pathway\n",
      "    - nerve growth factor signaling pathway\n",
      "    - nerve growth factor signaling pathway\n",
      "    - mechanoreceptor differentiation\n",
      "    - negative regulation of apoptotic process\n",
      "    - negative regulation of apoptotic process\n",
      "    - positive regulation of programmed cell death\n",
      "    - positive regulation of programmed cell death\n",
      "    - negative regulation of neuron apoptotic process\n",
      "    - negative regulation of neuron apoptotic process\n",
      "    - positive regulation of GTPase activity\n",
      "    - positive regulation of GTPase activity\n",
      "    - positive regulation of angiogenesis\n",
      "    - positive regulation of Ras protein signal transduction\n",
      "    - positive regulation of Ras protein signal transduction\n",
      "    - protein autophosphorylation\n",
      "    - neurotrophin TRK receptor signaling pathway\n",
      "    - ephrin receptor signaling pathway\n",
      "    - sympathetic nervous system development\n",
      "    - sympathetic nervous system development\n",
      "    - neuron development\n",
      "    - response to axon injury\n",
      "    - detection of temperature stimulus involved in sensory perception of pain\n",
      "    - detection of mechanical stimulus involved in sensory perception of pain\n",
      "    - positive regulation of NF-kappaB transcription factor activity\n",
      "    - neuron apoptotic process\n",
      "    - response to hydrostatic pressure\n",
      "    - response to electrical stimulus\n",
      "    - positive regulation of phosphatidylinositol 3-kinase/protein kinase B signal transduction\n",
      "    - positive regulation of phosphatidylinositol 3-kinase/protein kinase B signal transduction\n",
      "    - positive regulation of phosphatidylinositol 3-kinase/protein kinase B signal transduction\n",
      "    - positive regulation of synapse assembly\n",
      "    - positive regulation of synaptic transmission, glutamatergic\n",
      "    - Sertoli cell development\n",
      "    - innervation\n",
      "    - axonogenesis involved in innervation\n",
      "    - axonogenesis involved in innervation\n",
      "    - behavioral response to formalin induced pain\n",
      "    - positive regulation of ERK1 and ERK2 cascade\n",
      "    - positive regulation of ERK1 and ERK2 cascade\n",
      "    - cellular response to nicotine\n",
      "    - cellular response to growth factor stimulus\n",
      "    - cellular response to amyloid-beta\n",
      "    - cellular response to nerve growth factor stimulus\n",
      "    - cellular response to nerve growth factor stimulus\n",
      "    - cellular response to nerve growth factor stimulus\n",
      "    - cellular response to nerve growth factor stimulus\n",
      "    - cellular response to nerve growth factor stimulus\n",
      "  CC:\n",
      "    - cytoplasm\n",
      "    - mitochondrion\n",
      "    - endosome\n",
      "    - early endosome\n",
      "    - early endosome\n",
      "    - late endosome\n",
      "    - late endosome\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - cell surface\n",
      "    - endosome membrane\n",
      "    - membrane\n",
      "    - axon\n",
      "    - axon\n",
      "    - dendrite\n",
      "    - early endosome membrane\n",
      "    - late endosome membrane\n",
      "    - protein-containing complex\n",
      "    - protein-containing complex\n",
      "    - neuronal cell body\n",
      "    - receptor complex\n",
      "    - receptor complex\n",
      "    - recycling endosome\n",
      "    - recycling endosome membrane\n",
      "  MF:\n",
      "    - nucleotide binding\n",
      "    - protein kinase activity\n",
      "    - protein tyrosine kinase activity\n",
      "    - protein tyrosine kinase activity\n",
      "    - transmembrane receptor protein tyrosine kinase activity\n",
      "    - transmembrane receptor protein tyrosine kinase activity\n",
      "    - transmembrane receptor protein tyrosine kinase activity\n",
      "    - transmembrane receptor protein tyrosine kinase activity\n",
      "    - GPI-linked ephrin receptor activity\n",
      "    - neurotrophin receptor activity\n",
      "    - neurotrophin receptor activity\n",
      "    - neurotrophin p75 receptor binding\n",
      "    - protein binding\n",
      "    - ATP binding\n",
      "    - nerve growth factor receptor activity\n",
      "    - nerve growth factor receptor activity\n",
      "    - kinase activity\n",
      "    - transferase activity\n",
      "    - kinase binding\n",
      "    - identical protein binding\n",
      "    - protein homodimerization activity\n",
      "    - neurotrophin binding\n",
      "    - neurotrophin binding\n",
      "    - nerve growth factor binding\n",
      "    - nerve growth factor binding\n",
      "    - nerve growth factor binding\n",
      "UniProt: {'Swiss-Prot': 'P04629', 'TrEMBL': ['A0A6Q8PHG5', 'X5DR71', 'J3KP20', 'A0A6Q8PF65', 'E9PQG0', 'A0A6Q8PGU5']}\n",
      "\n",
      "Gene: PDE4D\n",
      "Symbol: PDE4D\n",
      "Name: phosphodiesterase 4D\n",
      "Summary: This gene encodes one of four mammalian counterparts to the fruit fly 'dunce' gene. The encoded protein has 3',5'-cyclic-AMP phosphodiesterase activity and degrades cAMP, which acts as a signal transduction molecule in multiple cell types. This gene uses different promoters to generate multiple alternatively spliced transcript variants that encode functional proteins.[provided by RefSeq, Sep 2009].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - regulation of heart rate\n",
      "    - cAMP catabolic process\n",
      "    - signal transduction\n",
      "    - positive regulation of heart rate\n",
      "    - regulation of release of sequestered calcium ion into cytosol by sarcoplasmic reticulum\n",
      "    - positive regulation of type II interferon production\n",
      "    - positive regulation of interleukin-2 production\n",
      "    - positive regulation of interleukin-5 production\n",
      "    - negative regulation of heart contraction\n",
      "    - T cell receptor signaling pathway\n",
      "    - establishment of endothelial barrier\n",
      "    - cellular response to cAMP\n",
      "    - cellular response to epinephrine stimulus\n",
      "    - adrenergic receptor signaling pathway\n",
      "    - regulation of cardiac muscle cell contraction\n",
      "    - negative regulation of adenylate cyclase-activating G protein-coupled receptor signaling pathway\n",
      "    - negative regulation of adenylate cyclase-activating G protein-coupled receptor signaling pathway\n",
      "    - negative regulation of cAMP/PKA signal transduction\n",
      "    - regulation of cell communication by electrical coupling involved in cardiac conduction\n",
      "    - negative regulation of relaxation of cardiac muscle\n",
      "    - regulation of calcium ion transmembrane transport via high voltage-gated calcium channel\n",
      "  CC:\n",
      "    - nucleoplasm\n",
      "    - cytoplasm\n",
      "    - centrosome\n",
      "    - cytosol\n",
      "    - cytosol\n",
      "    - cytoskeleton\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - voltage-gated calcium channel complex\n",
      "    - cilium\n",
      "    - membrane\n",
      "    - membrane\n",
      "    - membrane\n",
      "    - apical plasma membrane\n",
      "    - nuclear membrane\n",
      "    - calcium channel complex\n",
      "  MF:\n",
      "    - 3',5'-cyclic-nucleotide phosphodiesterase activity\n",
      "    - 3',5'-cyclic-nucleotide phosphodiesterase activity\n",
      "    - 3',5'-cyclic-AMP phosphodiesterase activity\n",
      "    - 3',5'-cyclic-AMP phosphodiesterase activity\n",
      "    - 3',5'-cyclic-AMP phosphodiesterase activity\n",
      "    - 3',5'-cyclic-AMP phosphodiesterase activity\n",
      "    - 3',5'-cyclic-AMP phosphodiesterase activity\n",
      "    - 3',5'-cyclic-AMP phosphodiesterase activity\n",
      "    - calcium channel regulator activity\n",
      "    - protein binding\n",
      "    - phosphoric diester hydrolase activity\n",
      "    - hydrolase activity\n",
      "    - enzyme binding\n",
      "    - signaling receptor regulator activity\n",
      "    - cAMP binding\n",
      "    - beta-2 adrenergic receptor binding\n",
      "    - transmembrane transporter binding\n",
      "    - metal ion binding\n",
      "    - 3',5'-cyclic-GMP phosphodiesterase activity\n",
      "    - ATPase binding\n",
      "    - scaffold protein binding\n",
      "    - heterocyclic compound binding\n",
      "UniProt: {'Swiss-Prot': 'Q08499', 'TrEMBL': ['A0A1B0GW84', 'A0A140VJR0', 'D6RAQ0', 'D6RIG1', 'D6R9L4', 'D6RHE0', 'D6RBB2']}\n",
      "\n",
      "Gene: CHAT\n",
      "Symbol: CHAT\n",
      "Name: choline O-acetyltransferase\n",
      "Summary: This gene encodes an enzyme which catalyzes the biosynthesis of the neurotransmitter acetylcholine. This gene product is a characteristic feature of cholinergic neurons, and changes in these neurons may explain some of the symptoms of Alzheimer's disease. Polymorphisms in this gene have been associated with Alzheimer's disease and mild cognitive impairment. Mutations in this gene are associated with congenital myasthenic syndrome associated with episodic apnea. Multiple transcript variants encoding different isoforms have been found for this gene, and some of these variants have been shown to encode more than one isoform. [provided by RefSeq, May 2010].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - phosphatidylcholine biosynthetic process\n",
      "    - neurotransmitter transport\n",
      "    - neuromuscular synaptic transmission\n",
      "    - acetylcholine biosynthetic process\n",
      "  CC:\n",
      "    - nucleus\n",
      "    - cytoplasm\n",
      "    - cytoplasm\n",
      "    - cytosol\n",
      "    - neuron projection\n",
      "    - synapse\n",
      "  MF:\n",
      "    - choline O-acetyltransferase activity\n",
      "    - choline O-acetyltransferase activity\n",
      "    - choline O-acetyltransferase activity\n",
      "    - protein binding\n",
      "    - transferase activity\n",
      "    - acyltransferase activity\n",
      "UniProt: {'Swiss-Prot': 'P28329', 'TrEMBL': ['Q6LEN6', 'Q6LEN5', 'R4GN13', 'A0A1W2PP46', 'A0A1W2PRG4']}\n",
      "\n",
      "Gene: SLC27A1\n",
      "Symbol: SLC27A1\n",
      "Name: solute carrier family 27 member 1\n",
      "Summary: Enables biotin transmembrane transporter activity; efflux transmembrane transporter activity; and long-chain fatty acid transporter activity. Involved in several processes, including carboxylic acid transmembrane transport; glycerophospholipid biosynthetic process; and lipid transport across blood-brain barrier. Located in membrane. Part of plasma membrane. [provided by Alliance of Genome Resources, Apr 2022]\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - medium-chain fatty acid transport\n",
      "    - medium-chain fatty acid transport\n",
      "    - long-chain fatty acid metabolic process\n",
      "    - lipid metabolic process\n",
      "    - fatty acid metabolic process\n",
      "    - lipid transport\n",
      "    - response to cold\n",
      "    - positive regulation of triglyceride biosynthetic process\n",
      "    - positive regulation of triglyceride biosynthetic process\n",
      "    - biotin transport\n",
      "    - fatty acid transport\n",
      "    - long-chain fatty acid transport\n",
      "    - long-chain fatty acid transport\n",
      "    - long-chain fatty acid transport\n",
      "    - long-chain fatty acid transport\n",
      "    - long-chain fatty acid import across plasma membrane\n",
      "    - long-chain fatty acid import across plasma membrane\n",
      "    - positive regulation of heat generation\n",
      "    - response to insulin\n",
      "    - adiponectin-activated signaling pathway\n",
      "    - glucose import in response to insulin stimulus\n",
      "    - glucose import in response to insulin stimulus\n",
      "    - long-chain fatty acid import into cell\n",
      "    - long-chain fatty acid import into cell\n",
      "    - long-chain fatty acid import into cell\n",
      "    - long-chain fatty acid import into cell\n",
      "    - export across plasma membrane\n",
      "    - export across plasma membrane\n",
      "    - transport across blood-brain barrier\n",
      "    - biotin import across plasma membrane\n",
      "    - lipid transport across blood-brain barrier\n",
      "    - lipid transport across blood-brain barrier\n",
      "  CC:\n",
      "    - cytoplasm\n",
      "    - endoplasmic reticulum\n",
      "    - endoplasmic reticulum\n",
      "    - endoplasmic reticulum membrane\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - basal plasma membrane\n",
      "    - basal plasma membrane\n",
      "    - endomembrane system\n",
      "    - membrane\n",
      "    - membrane\n",
      "  MF:\n",
      "    - nucleotide binding\n",
      "    - long-chain fatty acid-CoA ligase activity\n",
      "    - long-chain fatty acid-CoA ligase activity\n",
      "    - long-chain fatty acid transmembrane transporter activity\n",
      "    - long-chain fatty acid transmembrane transporter activity\n",
      "    - long-chain fatty acid transmembrane transporter activity\n",
      "    - long-chain fatty acid transmembrane transporter activity\n",
      "    - protein binding\n",
      "    - biotin transmembrane transporter activity\n",
      "    - fatty acid transmembrane transporter activity\n",
      "    - fatty acid transmembrane transporter activity\n",
      "    - efflux transmembrane transporter activity\n",
      "    - efflux transmembrane transporter activity\n",
      "    - ligase activity\n",
      "    - very long-chain fatty acid-CoA ligase activity\n",
      "    - identical protein binding\n",
      "    - protein serine/threonine kinase activator activity\n",
      "    - arachidonate-CoA ligase activity\n",
      "    - oleoyl-CoA ligase activity\n",
      "    - oleoyl-CoA ligase activity\n",
      "    - oleoyl-CoA ligase activity\n",
      "UniProt: {'Swiss-Prot': 'Q6PCB7', 'TrEMBL': ['M0QXY2', 'B7Z225', 'M0QZC9']}\n",
      "\n",
      "Gene: HLA-DRA\n",
      "Symbol: HLA-DRA\n",
      "Name: major histocompatibility complex, class II, DR alpha\n",
      "Summary: HLA-DRA is one of the HLA class II alpha chain paralogues. This class II molecule is a heterodimer consisting of an alpha and a beta chain, both anchored in the membrane. This molecule is expressed on the surface of various antigen presenting cells such as B lymphocytes, dendritic cells, and monocytes/macrophages, and plays a central role in the immune system and response by presenting peptides derived from extracellular proteins, in particular, pathogen-derived peptides to T cells. The alpha chain is approximately 33-35 kDa and its gene contains 5 exons. Exon 1 encodes the leader peptide, exons 2 and 3 encode the two extracellular domains, and exon 4 encodes the transmembrane domain and the cytoplasmic tail. DRA does not have polymorphisms in the peptide binding part and acts as the sole alpha chain for DRB1, DRB3, DRB4 and DRB5. [provided by RefSeq, Aug 2020].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - positive regulation of T cell mediated cytotoxicity\n",
      "    - adaptive immune response\n",
      "    - immune system process\n",
      "    - myeloid dendritic cell antigen processing and presentation\n",
      "    - antigen processing and presentation of endogenous peptide antigen via MHC class II\n",
      "    - peptide antigen assembly with MHC class II protein complex\n",
      "    - peptide antigen assembly with MHC class II protein complex\n",
      "    - antigen processing and presentation of peptide or polysaccharide antigen via MHC class II\n",
      "    - antigen processing and presentation of peptide or polysaccharide antigen via MHC class II\n",
      "    - immune response\n",
      "    - immune response\n",
      "    - antigen processing and presentation\n",
      "    - antigen processing and presentation of exogenous peptide antigen via MHC class II\n",
      "    - antigen processing and presentation of exogenous peptide antigen via MHC class II\n",
      "    - positive regulation of CD4-positive, CD25-positive, alpha-beta regulatory T cell differentiation\n",
      "    - positive regulation of memory T cell differentiation\n",
      "    - regulation of T-helper cell differentiation\n",
      "    - positive regulation of immune response\n",
      "    - positive regulation of T cell activation\n",
      "    - cognition\n",
      "    - positive regulation of CD4-positive, alpha-beta T cell activation\n",
      "  CC:\n",
      "    - Golgi membrane\n",
      "    - immunological synapse\n",
      "    - lysosome\n",
      "    - lysosome\n",
      "    - lysosomal membrane\n",
      "    - lysosomal membrane\n",
      "    - lysosomal membrane\n",
      "    - lysosomal membrane\n",
      "    - endosome\n",
      "    - endoplasmic reticulum\n",
      "    - endoplasmic reticulum membrane\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - cell surface\n",
      "    - ER to Golgi transport vesicle membrane\n",
      "    - membrane\n",
      "    - transport vesicle membrane\n",
      "    - endocytic vesicle membrane\n",
      "    - clathrin-coated endocytic vesicle membrane\n",
      "    - cytoplasmic vesicle\n",
      "    - early endosome membrane\n",
      "    - late endosome membrane\n",
      "    - late endosome membrane\n",
      "    - late endosome membrane\n",
      "    - trans-Golgi network membrane\n",
      "    - MHC class II protein complex\n",
      "    - MHC class II protein complex\n",
      "    - MHC class II protein complex\n",
      "    - MHC class II protein complex\n",
      "    - extracellular exosome\n",
      "    - lumenal side of endoplasmic reticulum membrane\n",
      "    - autolysosome membrane\n",
      "  MF:\n",
      "    - protein binding\n",
      "    - MHC class II protein complex binding\n",
      "    - MHC class II protein complex binding\n",
      "    - polysaccharide binding\n",
      "    - MHC class II receptor activity\n",
      "    - peptide antigen binding\n",
      "    - peptide antigen binding\n",
      "    - T cell receptor binding\n",
      "UniProt: {'Swiss-Prot': 'P01903', 'TrEMBL': ['A0A0G2JH46', 'A0A0G2JMH6', 'Q30118']}\n",
      "\n",
      "Gene: MSLN\n",
      "Symbol: MSLN\n",
      "Name: mesothelin\n",
      "Summary: This gene encodes a preproprotein that is proteolytically processed to generate two protein products, megakaryocyte potentiating factor and mesothelin. Megakaryocyte potentiating factor functions as a cytokine that can stimulate colony formation of bone marrow megakaryocytes. Mesothelin is a glycosylphosphatidylinositol-anchored cell-surface protein that may function as a cell adhesion protein. This protein is overexpressed in epithelial mesotheliomas, ovarian cancers and in specific squamous cell carcinomas. Alternative splicing results in multiple transcript variants, at least one of which encodes an isoform that is proteolytically processed. [provided by RefSeq, Feb 2016].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - cell adhesion\n",
      "    - cell adhesion\n",
      "    - cell-matrix adhesion\n",
      "  CC:\n",
      "    - extracellular region\n",
      "    - extracellular region\n",
      "    - endoplasmic reticulum lumen\n",
      "    - Golgi apparatus\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - cell surface\n",
      "    - membrane\n",
      "    - membrane\n",
      "    - side of membrane\n",
      "  MF:\n",
      "    - protein binding\n",
      "UniProt: {'Swiss-Prot': 'Q13421', 'TrEMBL': ['H3BUX1', 'H3BMA1', 'H3BV92', 'H3BR90']}\n",
      "\n",
      "Gene: SORCS1\n",
      "Symbol: SORCS1\n",
      "Name: sortilin related VPS10 domain containing receptor 1\n",
      "Summary: This gene encodes one family member of vacuolar protein sorting 10 (VPS10) domain-containing receptor proteins. The VPS10 domain name comes from the yeast carboxypeptidase Y sorting receptor Vps10 protein. Members of this gene family are large with many exons but the CDS lengths are usually less than 3700 nt. Very large introns typically separate the exons encoding the VPS10 domain; the remaining exons are separated by much smaller-sized introns. These genes are strongly expressed in the central nervous system. Two of the five family members (sortilin and sortilin-related receptor) are synthesized as preproproteins; it is not yet known if this encoded protein is also a preproprotein. Alternatively spliced transcript variants encoding different isoforms have been identified. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - post-Golgi vesicle-mediated transport\n",
      "    - neuropeptide signaling pathway\n",
      "  CC:\n",
      "    - Golgi apparatus\n",
      "    - membrane\n",
      "    - membrane\n",
      "    - membrane\n",
      "  MF:\n",
      "    - protein binding\n",
      "    - neuropeptide receptor activity\n",
      "UniProt: {'Swiss-Prot': 'Q8WY21', 'TrEMBL': ['E6Y3F9', 'A8K182', 'E6Y3G0', 'B3KWN9', 'B3KVZ0', 'X6R7D6', 'H7C2U3']}\n",
      "\n",
      "Gene: CTBS\n",
      "Symbol: CTBS\n",
      "Name: chitobiase\n",
      "Summary: Chitobiase is a lysosomal glycosidase involved in degradation of asparagine-linked oligosaccharides on glycoproteins (Aronson and Kuranda, 1989 [PubMed 2531691]).[supplied by OMIM, Nov 2010].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - carbohydrate metabolic process\n",
      "    - chitin catabolic process\n",
      "    - chitin catabolic process\n",
      "    - oligosaccharide catabolic process\n",
      "    - oligosaccharide catabolic process\n",
      "  CC:\n",
      "    - extracellular space\n",
      "    - lysosome\n",
      "  MF:\n",
      "    - hydrolase activity, hydrolyzing O-glycosyl compounds\n",
      "    - chitinase activity\n",
      "    - chitinase activity\n",
      "    - chitin binding\n",
      "    - hydrolase activity\n",
      "    - hydrolase activity, acting on glycosyl bonds\n",
      "UniProt: {'Swiss-Prot': 'Q01459', 'TrEMBL': ['F6X5H7', 'Q8TC97']}\n",
      "\n",
      "Gene: MRPL18\n",
      "Symbol: MRPL18\n",
      "Name: mitochondrial ribosomal protein L18\n",
      "Summary: This nuclear gene encodes a protein component of the larger 39S subunit of mitochondrial ribosome. This protein may also aid in the import of nuclear-encoded 5S rRNA into mitochondria. Alternative splicing results in multiple transcript variants, most of which are not predicted to encode a protein. A pseudogene of this gene is found on chromosome 16. [provided by RefSeq, Jan 2016].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - translation\n",
      "    - translation\n",
      "    - mitochondrial translation\n",
      "    - rRNA import into mitochondrion\n",
      "  CC:\n",
      "    - extracellular space\n",
      "    - mitochondrion\n",
      "    - mitochondrion\n",
      "    - mitochondrion\n",
      "    - mitochondrion\n",
      "    - mitochondrial inner membrane\n",
      "    - mitochondrial inner membrane\n",
      "    - mitochondrial inner membrane\n",
      "    - mitochondrial ribosome\n",
      "    - mitochondrial large ribosomal subunit\n",
      "    - mitochondrial large ribosomal subunit\n",
      "    - ribosome\n",
      "    - ribonucleoprotein complex\n",
      "  MF:\n",
      "    - RNA binding\n",
      "    - structural constituent of ribosome\n",
      "    - structural constituent of ribosome\n",
      "    - protein binding\n",
      "    - 5S rRNA binding\n",
      "    - 5S rRNA binding\n",
      "UniProt: {'Swiss-Prot': 'Q9H0U6'}\n",
      "\n",
      "Gene: LGALS4\n",
      "Symbol: LGALS4\n",
      "Name: galectin 4\n",
      "Summary: The galectins are a family of beta-galactoside-binding proteins implicated in modulating cell-cell and cell-matrix interactions. The expression of this gene is restricted to small intestine, colon, and rectum, and it is underexpressed in colorectal cancer. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - antibacterial peptide biosynthetic process\n",
      "    - cell adhesion\n",
      "    - defense response to bacterium\n",
      "  CC:\n",
      "    - extracellular space\n",
      "    - cytosol\n",
      "    - plasma membrane\n",
      "    - extracellular matrix\n",
      "  MF:\n",
      "    - protein binding\n",
      "    - galactoside binding\n",
      "    - carbohydrate binding\n",
      "    - carbohydrate binding\n",
      "    - carbohydrate binding\n",
      "UniProt: {'Swiss-Prot': 'P56470', 'TrEMBL': ['M0R349', 'Q6FHZ4', 'M0QZ93', 'M0R1B2']}\n",
      "\n",
      "Gene: MAPKAPK3\n",
      "Symbol: MAPKAPK3\n",
      "Name: MAPK activated protein kinase 3\n",
      "Summary: This gene encodes a member of the Ser/Thr protein kinase family. This kinase functions as a mitogen-activated protein kinase (MAP kinase)- activated protein kinase. MAP kinases are also known as extracellular signal-regulated kinases (ERKs), act as an integration point for multiple biochemical signals. This kinase was shown to be activated by growth inducers and stress stimulation of cells. In vitro studies demonstrated that ERK, p38 MAP kinase and Jun N-terminal kinase were all able to phosphorylate and activate this kinase, which suggested the role of this kinase as an integrative element of signaling in both mitogen and stress responses. This kinase was reported to interact with, phosphorylate and repress the activity of E47, which is a basic helix-loop-helix transcription factor known to be involved in the regulation of tissue-specific gene expression and cell differentiation. Alternate splicing results in multiple transcript variants that encode the same protein. [provided by RefSeq, Sep 2011].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - MAPK cascade\n",
      "    - MAPK cascade\n",
      "    - toll-like receptor signaling pathway\n",
      "    - toll-like receptor signaling pathway\n",
      "    - toll-like receptor signaling pathway\n",
      "    - signal transduction\n",
      "    - response to lipopolysaccharide\n",
      "    - response to lipopolysaccharide\n",
      "    - response to cytokine\n",
      "    - response to cytokine\n",
      "    - intracellular signal transduction\n",
      "    - macropinocytosis\n",
      "    - macropinocytosis\n",
      "    - vascular endothelial growth factor receptor signaling pathway\n",
      "    - vascular endothelial growth factor receptor signaling pathway\n",
      "  CC:\n",
      "    - nucleus\n",
      "    - nucleus\n",
      "    - nucleus\n",
      "    - nucleoplasm\n",
      "    - nucleoplasm\n",
      "    - cytoplasm\n",
      "    - cytoplasm\n",
      "    - cytosol\n",
      "  MF:\n",
      "    - nucleotide binding\n",
      "    - protein kinase activity\n",
      "    - protein serine/threonine kinase activity\n",
      "    - protein serine/threonine kinase activity\n",
      "    - protein serine/threonine kinase activity\n",
      "    - calcium/calmodulin-dependent protein kinase activity\n",
      "    - MAP kinase kinase activity\n",
      "    - protein binding\n",
      "    - calmodulin binding\n",
      "    - ATP binding\n",
      "    - calcium-dependent protein serine/threonine kinase activity\n",
      "    - kinase activity\n",
      "    - transferase activity\n",
      "    - mitogen-activated protein kinase binding\n",
      "    - protein serine kinase activity\n",
      "UniProt: {'Swiss-Prot': 'Q16644', 'TrEMBL': ['H7C0G6', 'C9J8E1', 'C9JPW3']}\n",
      "\n",
      "Gene: NAT8L\n",
      "Symbol: NAT8L\n",
      "Name: N-acetyltransferase 8 like\n",
      "Summary: This gene encodes a single-pass membrane protein, which contains a conserved sequence of the GCN5 or NAT superfamily of N-acetyltransferases and is a member of the N-acyltransferase (NAT) superfamily. This protein is a neuron-specific protein and is the N-acetylaspartate (NAA) biosynthetic enzyme, catalyzing the NAA synthesis from L-aspartate and acetyl-CoA. NAA is a major storage and transport form of acetyl coenzyme A specific to the nervous system. The gene mutation results in primary NAA deficiency (hypoacetylaspartia). [provided by RefSeq, Dec 2010].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - aspartate family amino acid metabolic process\n",
      "  CC:\n",
      "    - cytoplasm\n",
      "    - cytoplasm\n",
      "    - mitochondrion\n",
      "    - mitochondrion\n",
      "    - mitochondrial matrix\n",
      "    - endoplasmic reticulum\n",
      "    - endoplasmic reticulum membrane\n",
      "    - membrane\n",
      "    - mitochondrial membrane\n",
      "    - mitochondrial membrane\n",
      "    - mitochondrial membrane\n",
      "    - intracellular membrane-bounded organelle\n",
      "  MF:\n",
      "    - protein binding\n",
      "    - N-acetyltransferase activity\n",
      "    - transferase activity\n",
      "    - acyltransferase activity\n",
      "    - acyltransferase activity, transferring groups other than amino-acyl groups\n",
      "    - L-aspartate N-acetyltransferase activity\n",
      "    - L-aspartate N-acetyltransferase activity\n",
      "    - L-aspartate N-acetyltransferase activity\n",
      "    - L-aspartate N-acetyltransferase activity\n",
      "    - L-aspartate N-acetyltransferase activity\n",
      "UniProt: {'Swiss-Prot': 'Q8N9F0'}\n",
      "\n",
      "Gene: HDAC1\n",
      "Symbol: HDAC1\n",
      "Name: histone deacetylase 1\n",
      "Summary: Histone acetylation and deacetylation, catalyzed by multisubunit complexes, play a key role in the regulation of eukaryotic gene expression. The protein encoded by this gene belongs to the histone deacetylase/acuc/apha family and is a component of the histone deacetylase complex. It also interacts with retinoblastoma tumor-suppressor protein and this complex is a key element in the control of cell proliferation and differentiation. Together with metastasis-associated protein-2, it deacetylates p53 and modulates its effect on cell growth and apoptosis. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - negative regulation of transcription by RNA polymerase II\n",
      "    - negative regulation of transcription by RNA polymerase II\n",
      "    - negative regulation of transcription by RNA polymerase II\n",
      "    - negative regulation of transcription by RNA polymerase II\n",
      "    - negative regulation of transcription by RNA polymerase II\n",
      "    - chromatin organization\n",
      "    - chromatin organization\n",
      "    - chromatin remodeling\n",
      "    - chromatin remodeling\n",
      "    - chromatin remodeling\n",
      "    - DNA methylation-dependent constitutive heterochromatin formation\n",
      "    - regulation of transcription by RNA polymerase II\n",
      "    - protein deacetylation\n",
      "    - endoderm development\n",
      "    - circadian rhythm\n",
      "    - positive regulation of cell population proliferation\n",
      "    - positive regulation of cell population proliferation\n",
      "    - epidermal cell differentiation\n",
      "    - epidermal cell differentiation\n",
      "    - positive regulation of gene expression\n",
      "    - negative regulation of gene expression\n",
      "    - negative regulation of myotube differentiation\n",
      "    - hippocampus development\n",
      "    - neuron differentiation\n",
      "    - negative regulation of cell migration\n",
      "    - negative regulation of transforming growth factor beta receptor signaling pathway\n",
      "    - heterochromatin formation\n",
      "    - heterochromatin formation\n",
      "    - circadian regulation of gene expression\n",
      "    - circadian regulation of gene expression\n",
      "    - positive regulation of intracellular estrogen receptor signaling pathway\n",
      "    - positive regulation of intracellular estrogen receptor signaling pathway\n",
      "    - cellular response to platelet-derived growth factor stimulus\n",
      "    - odontogenesis of dentin-containing tooth\n",
      "    - odontogenesis of dentin-containing tooth\n",
      "    - regulation of cell fate specification\n",
      "    - embryonic digit morphogenesis\n",
      "    - embryonic digit morphogenesis\n",
      "    - negative regulation of apoptotic process\n",
      "    - negative regulation of apoptotic process\n",
      "    - negative regulation of apoptotic process\n",
      "    - negative regulation of canonical NF-kappaB signal transduction\n",
      "    - negative regulation by host of viral transcription\n",
      "    - negative regulation of gene expression, epigenetic\n",
      "    - negative regulation of DNA-templated transcription\n",
      "    - negative regulation of DNA-templated transcription\n",
      "    - negative regulation of DNA-templated transcription\n",
      "    - negative regulation of DNA-templated transcription\n",
      "    - positive regulation of DNA-templated transcription\n",
      "    - positive regulation of DNA-templated transcription\n",
      "    - positive regulation of transcription by RNA polymerase II\n",
      "    - rhythmic process\n",
      "    - positive regulation of smooth muscle cell proliferation\n",
      "    - oligodendrocyte differentiation\n",
      "    - positive regulation of oligodendrocyte differentiation\n",
      "    - negative regulation of androgen receptor signaling pathway\n",
      "    - hair follicle placode formation\n",
      "    - hair follicle placode formation\n",
      "    - eyelid development in camera-type eye\n",
      "    - eyelid development in camera-type eye\n",
      "    - fungiform papilla formation\n",
      "    - fungiform papilla formation\n",
      "    - negative regulation of canonical Wnt signaling pathway\n",
      "    - negative regulation of canonical Wnt signaling pathway\n",
      "    - negative regulation of stem cell population maintenance\n",
      "    - positive regulation of stem cell population maintenance\n",
      "    - regulation of stem cell differentiation\n",
      "    - negative regulation of intrinsic apoptotic signaling pathway\n",
      "  CC:\n",
      "    - histone deacetylase complex\n",
      "    - histone deacetylase complex\n",
      "    - histone deacetylase complex\n",
      "    - chromatin\n",
      "    - chromatin\n",
      "    - chromatin\n",
      "    - heterochromatin\n",
      "    - nucleus\n",
      "    - nucleus\n",
      "    - nucleus\n",
      "    - nucleoplasm\n",
      "    - nucleoplasm\n",
      "    - transcription regulator complex\n",
      "    - cytoplasm\n",
      "    - cytoplasm\n",
      "    - cytosol\n",
      "    - NuRD complex\n",
      "    - NuRD complex\n",
      "    - NuRD complex\n",
      "    - NuRD complex\n",
      "    - transcription repressor complex\n",
      "    - protein-containing complex\n",
      "    - protein-containing complex\n",
      "    - protein-containing complex\n",
      "    - neuronal cell body\n",
      "    - Sin3-type complex\n",
      "    - Sin3-type complex\n",
      "    - Sin3-type complex\n",
      "  MF:\n",
      "    - transcription cis-regulatory region binding\n",
      "    - RNA polymerase II cis-regulatory region sequence-specific DNA binding\n",
      "    - RNA polymerase II core promoter sequence-specific DNA binding\n",
      "    - core promoter sequence-specific DNA binding\n",
      "    - transcription corepressor binding\n",
      "    - p53 binding\n",
      "    - DNA binding\n",
      "    - chromatin binding\n",
      "    - transcription corepressor activity\n",
      "    - transcription corepressor activity\n",
      "    - histone deacetylase activity\n",
      "    - histone deacetylase activity\n",
      "    - histone deacetylase activity\n",
      "    - histone deacetylase activity\n",
      "    - histone deacetylase activity\n",
      "    - protein binding\n",
      "    - protein binding\n",
      "    - hydrolase activity\n",
      "    - hydrolase activity, acting on carbon-nitrogen (but not peptide) bonds, in linear amides\n",
      "    - enzyme binding\n",
      "    - nucleosomal DNA binding\n",
      "    - protein lysine deacetylase activity\n",
      "    - protein lysine deacetylase activity\n",
      "    - protein lysine deacetylase activity\n",
      "    - protein lysine deacetylase activity\n",
      "    - Krueppel-associated box domain binding\n",
      "    - histone deacetylase binding\n",
      "    - metal ion binding\n",
      "    - NF-kappaB binding\n",
      "    - NF-kappaB binding\n",
      "    - RNA polymerase II-specific DNA-binding transcription factor binding\n",
      "    - RNA polymerase II-specific DNA-binding transcription factor binding\n",
      "    - E-box binding\n",
      "    - DNA-binding transcription factor binding\n",
      "    - DNA-binding transcription factor binding\n",
      "    - DNA-binding transcription factor binding\n",
      "    - histone deacetylase activity, hydrolytic mechanism\n",
      "    - protein decrotonylase activity\n",
      "    - histone decrotonylase activity\n",
      "    - histone decrotonylase activity\n",
      "    - protein lysine delactylase activity\n",
      "    - protein lysine delactylase activity\n",
      "    - promoter-specific chromatin binding\n",
      "UniProt: {'Swiss-Prot': 'Q13547', 'TrEMBL': ['A0ABB0MVB4', 'Q6IT96', 'A0ABB0MVA0', 'Q5TEE2']}\n",
      "\n",
      "Gene: HAGH\n",
      "Symbol: HAGH\n",
      "Name: hydroxyacylglutathione hydrolase\n",
      "Summary: The enzyme encoded by this gene is classified as a thiolesterase and is responsible for the hydrolysis of S-lactoyl-glutathione to reduced glutathione and D-lactate. Three transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Oct 2013].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - glutathione metabolic process\n",
      "    - glutathione metabolic process\n",
      "    - glutathione biosynthetic process\n",
      "    - methylglyoxal catabolic process to D-lactate via S-lactoyl-glutathione\n",
      "  CC:\n",
      "    - cytoplasm\n",
      "    - cytoplasm\n",
      "    - mitochondrion\n",
      "    - mitochondrion\n",
      "    - mitochondrion\n",
      "    - mitochondrial matrix\n",
      "    - mitochondrial matrix\n",
      "    - cytosol\n",
      "  MF:\n",
      "    - hydroxyacylglutathione hydrolase activity\n",
      "    - hydroxyacylglutathione hydrolase activity\n",
      "    - hydroxyacylglutathione hydrolase activity\n",
      "    - protein binding\n",
      "    - hydrolase activity\n",
      "    - metal ion binding\n",
      "UniProt: {'Swiss-Prot': 'Q16775', 'TrEMBL': ['H3BQW8', 'H3BPQ4', 'H3BV79', 'H3BPK3']}\n",
      "\n",
      "Gene: DGAT2\n",
      "Symbol: DGAT2\n",
      "Name: diacylglycerol O-acyltransferase 2\n",
      "Summary: This gene encodes one of two enzymes which catalyzes the final reaction in the synthesis of triglycerides in which diacylglycerol is covalently bound to long chain fatty acyl-CoAs. The encoded protein catalyzes this reaction at low concentrations of magnesium chloride while the other enzyme has high activity at high concentrations of magnesium chloride. Multiple transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Dec 2011].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - glycerol metabolic process\n",
      "    - lipid metabolic process\n",
      "    - monoacylglycerol biosynthetic process\n",
      "    - diacylglycerol biosynthetic process\n",
      "    - diacylglycerol biosynthetic process\n",
      "    - response to nutrient\n",
      "    - positive regulation of triglyceride biosynthetic process\n",
      "    - triglyceride biosynthetic process\n",
      "    - triglyceride biosynthetic process\n",
      "    - triglyceride biosynthetic process\n",
      "    - triglyceride biosynthetic process\n",
      "    - triglyceride biosynthetic process\n",
      "    - lipid storage\n",
      "    - lipid storage\n",
      "    - low-density lipoprotein particle clearance\n",
      "    - low-density lipoprotein particle clearance\n",
      "    - long-chain fatty-acyl-CoA metabolic process\n",
      "    - long-chain fatty-acyl-CoA metabolic process\n",
      "    - intracellular triglyceride homeostasis\n",
      "    - intracellular triglyceride homeostasis\n",
      "    - retinol metabolic process\n",
      "    - cholesterol homeostasis\n",
      "    - cholesterol homeostasis\n",
      "    - positive regulation of gluconeogenesis\n",
      "    - negative regulation of fatty acid oxidation\n",
      "    - diacylglycerol metabolic process\n",
      "    - diacylglycerol metabolic process\n",
      "    - diacylglycerol metabolic process\n",
      "    - regulation of lipoprotein metabolic process\n",
      "    - fatty acid homeostasis\n",
      "    - fatty acid homeostasis\n",
      "    - fat pad development\n",
      "    - fat pad development\n",
      "    - cellular response to oleic acid\n",
      "    - cellular response to oleic acid\n",
      "    - regulation of cholesterol metabolic process\n",
      "    - regulation of plasma lipoprotein particle levels\n",
      "    - regulation of plasma lipoprotein particle levels\n",
      "  CC:\n",
      "    - cytoplasm\n",
      "    - mitochondrion\n",
      "    - mitochondrion\n",
      "    - endoplasmic reticulum\n",
      "    - endoplasmic reticulum\n",
      "    - endoplasmic reticulum membrane\n",
      "    - endoplasmic reticulum membrane\n",
      "    - endoplasmic reticulum membrane\n",
      "    - endoplasmic reticulum membrane\n",
      "    - endoplasmic reticulum membrane\n",
      "    - endoplasmic reticulum membrane\n",
      "    - lipid droplet\n",
      "    - lipid droplet\n",
      "    - lipid droplet\n",
      "    - lipid droplet\n",
      "    - cytosol\n",
      "    - membrane\n",
      "    - membrane\n",
      "    - perinuclear region of cytoplasm\n",
      "    - perinuclear region of cytoplasm\n",
      "    - perinuclear endoplasmic reticulum membrane\n",
      "  MF:\n",
      "    - 2-acylglycerol O-acyltransferase activity\n",
      "    - diacylglycerol O-acyltransferase activity\n",
      "    - diacylglycerol O-acyltransferase activity\n",
      "    - diacylglycerol O-acyltransferase activity\n",
      "    - diacylglycerol O-acyltransferase activity\n",
      "    - protein binding\n",
      "    - O-acyltransferase activity\n",
      "    - transferase activity\n",
      "    - acyltransferase activity\n",
      "    - protein homodimerization activity\n",
      "    - protein homodimerization activity\n",
      "    - retinol O-fatty-acyltransferase activity\n",
      "UniProt: {'Swiss-Prot': 'Q96PD7', 'TrEMBL': ['S4R3Z3', 'S4R383', 'S4R3S3', 'S4R449']}\n",
      "\n",
      "Gene: RYK\n",
      "Symbol: RYK\n",
      "Name: receptor like tyrosine kinase\n",
      "Summary: The protein encoded by this gene is an atypical member of the family of growth factor receptor protein tyrosine kinases, differing from other members at a number of conserved residues in the activation and nucleotide binding domains. This gene product belongs to a subfamily whose members do not appear to be regulated by phosphorylation in the activation segment. It has been suggested that mediation of biological activity by recruitment of a signaling-competent auxiliary protein may occur through an as yet uncharacterized mechanism. The encoded protein has a leucine-rich extracellular domain with a WIF-type Wnt binding region, a single transmembrane domain, and an intracellular tyrosine kinase domain. This protein is involved in stimulating Wnt signaling pathways such as the regulation of axon pathfinding. Alternative splicing results in multiple transcript variants encoding distinct isoforms. [provided by RefSeq, Feb 2012].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - protein phosphorylation\n",
      "    - signal transduction\n",
      "    - cell surface receptor protein tyrosine kinase signaling pathway\n",
      "    - axonogenesis\n",
      "    - axonogenesis\n",
      "    - axonogenesis\n",
      "    - axonogenesis\n",
      "    - axon guidance\n",
      "    - axon guidance\n",
      "    - axon guidance\n",
      "    - synapse assembly\n",
      "    - positive regulation of neuron projection development\n",
      "    - Wnt signaling pathway\n",
      "    - neurogenesis\n",
      "    - corpus callosum development\n",
      "    - corpus callosum development\n",
      "    - neuron differentiation\n",
      "    - neuron differentiation\n",
      "    - neuron projection development\n",
      "    - neuron projection development\n",
      "    - non-canonical Wnt signaling pathway\n",
      "    - chemorepulsion of dopaminergic neuron axon\n",
      "    - chemorepulsion of dopaminergic neuron axon\n",
      "    - positive regulation of MAPK cascade\n",
      "    - skeletal system morphogenesis\n",
      "    - negative regulation of axon extension involved in axon guidance\n",
      "    - axon extension involved in axon guidance\n",
      "    - negative chemotaxis\n",
      "    - positive regulation of phosphatidylinositol 3-kinase/protein kinase B signal transduction\n",
      "    - canonical Wnt signaling pathway\n",
      "    - Wnt signaling pathway, planar cell polarity pathway\n",
      "    - Wnt signaling pathway, planar cell polarity pathway\n",
      "    - commissural neuron axon guidance\n",
      "    - commissural neuron axon guidance\n",
      "    - positive regulation of cell proliferation in midbrain\n",
      "    - midbrain dopaminergic neuron differentiation\n",
      "    - midbrain dopaminergic neuron differentiation\n",
      "  CC:\n",
      "    - nucleus\n",
      "    - nucleus\n",
      "    - nucleus\n",
      "    - cytoplasm\n",
      "    - cytoplasm\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - membrane\n",
      "    - membrane\n",
      "    - membrane\n",
      "    - receptor complex\n",
      "    - presynaptic active zone membrane\n",
      "    - postsynaptic density membrane\n",
      "    - glutamatergic synapse\n",
      "  MF:\n",
      "    - nucleotide binding\n",
      "    - protein kinase activity\n",
      "    - protein tyrosine kinase activity\n",
      "    - transmembrane receptor protein tyrosine kinase activity\n",
      "    - transmembrane signaling receptor activity\n",
      "    - transmembrane signaling receptor activity\n",
      "    - frizzled binding\n",
      "    - frizzled binding\n",
      "    - frizzled binding\n",
      "    - protein binding\n",
      "    - ATP binding\n",
      "    - coreceptor activity\n",
      "    - kinase activity\n",
      "    - transferase activity\n",
      "    - Wnt-protein binding\n",
      "    - Wnt-protein binding\n",
      "    - Wnt-protein binding\n",
      "    - Wnt receptor activity\n",
      "    - Wnt receptor activity\n",
      "UniProt: {'Swiss-Prot': 'P34925', 'TrEMBL': ['Q8WTZ8', 'Q59FQ5', 'A0A0D9SFZ4', 'H7C4P6']}\n",
      "\n",
      "Gene: NRP1\n",
      "Symbol: NRP1\n",
      "Name: neuropilin 1\n",
      "Summary: This gene encodes one of two neuropilins, which contain specific protein domains which allow them to participate in several different types of signaling pathways that control cell migration. Neuropilins contain a large N-terminal extracellular domain, made up of complement-binding, coagulation factor V/VIII, and meprin domains. These proteins also contains a short membrane-spanning domain and a small cytoplasmic domain. Neuropilins bind many ligands and various types of co-receptors; they affect cell survival, migration, and attraction. Some of the ligands and co-receptors bound by neuropilins are vascular endothelial growth factor (VEGF) and semaphorin family members. This protein has also been determined to act as a co-receptor for SARS-CoV-2 (which causes COVID-19) to infect host cells. [provided by RefSeq, Nov 2020].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - angiogenesis\n",
      "    - angiogenesis\n",
      "    - angiogenesis\n",
      "    - angiogenesis\n",
      "    - branching involved in blood vessel morphogenesis\n",
      "    - branching involved in blood vessel morphogenesis\n",
      "    - vasculogenesis\n",
      "    - neural crest cell migration\n",
      "    - neuron migration\n",
      "    - neuron migration\n",
      "    - positive regulation of endothelial cell proliferation\n",
      "    - positive regulation of endothelial cell proliferation\n",
      "    - sprouting angiogenesis\n",
      "    - sprouting angiogenesis\n",
      "    - sprouting angiogenesis\n",
      "    - cell migration involved in sprouting angiogenesis\n",
      "    - cell migration involved in sprouting angiogenesis\n",
      "    - outflow tract septum morphogenesis\n",
      "    - outflow tract septum morphogenesis\n",
      "    - substrate-dependent cell migration, cell extension\n",
      "    - signal transduction\n",
      "    - integrin-mediated signaling pathway\n",
      "    - integrin-mediated signaling pathway\n",
      "    - cell-cell signaling\n",
      "    - nervous system development\n",
      "    - axon guidance\n",
      "    - axon guidance\n",
      "    - axon guidance\n",
      "    - axon guidance\n",
      "    - axon guidance\n",
      "    - axonal fasciculation\n",
      "    - heart development\n",
      "    - motor neuron axon guidance\n",
      "    - motor neuron axon guidance\n",
      "    - response to wounding\n",
      "    - animal organ morphogenesis\n",
      "    - positive regulation of endothelial cell migration\n",
      "    - positive regulation of endothelial cell migration\n",
      "    - positive regulation of endothelial cell migration\n",
      "    - positive regulation of endothelial cell migration\n",
      "    - positive regulation of smooth muscle cell migration\n",
      "    - dendrite development\n",
      "    - facial nerve structural organization\n",
      "    - facial nerve structural organization\n",
      "    - trigeminal nerve morphogenesis\n",
      "    - trigeminal nerve structural organization\n",
      "    - trigeminal nerve structural organization\n",
      "    - vestibulocochlear nerve structural organization\n",
      "    - branchiomotor neuron axon guidance\n",
      "    - branchiomotor neuron axon guidance\n",
      "    - gonadotrophin-releasing hormone neuronal migration to the hypothalamus\n",
      "    - cell differentiation\n",
      "    - negative regulation of axon extension\n",
      "    - regulation of vascular endothelial growth factor receptor signaling pathway\n",
      "    - retinal ganglion cell axon guidance\n",
      "    - retinal ganglion cell axon guidance\n",
      "    - regulation of Cdc42 protein signal transduction\n",
      "    - regulation of Cdc42 protein signal transduction\n",
      "    - cellular response to hepatocyte growth factor stimulus\n",
      "    - endothelial cell chemotaxis\n",
      "    - cellular response to vascular endothelial growth factor stimulus\n",
      "    - cellular response to vascular endothelial growth factor stimulus\n",
      "    - ventral trunk neural crest cell migration\n",
      "    - vascular endothelial growth factor signaling pathway\n",
      "    - neuropilin signaling pathway\n",
      "    - neuropilin signaling pathway\n",
      "    - VEGF-activated neuropilin signaling pathway\n",
      "    - VEGF-activated neuropilin signaling pathway\n",
      "    - VEGF-activated neuropilin signaling pathway\n",
      "    - positive regulation of phosphorylation\n",
      "    - negative regulation of neuron apoptotic process\n",
      "    - blood vessel endothelial cell migration\n",
      "    - endothelial cell migration\n",
      "    - positive regulation of angiogenesis\n",
      "    - symbiont entry into host cell\n",
      "    - platelet-derived growth factor receptor signaling pathway\n",
      "    - vascular endothelial growth factor receptor signaling pathway\n",
      "    - vascular endothelial growth factor receptor signaling pathway\n",
      "    - vascular endothelial growth factor receptor signaling pathway\n",
      "    - hepatocyte growth factor receptor signaling pathway\n",
      "    - sympathetic nervous system development\n",
      "    - neuron development\n",
      "    - axon extension\n",
      "    - regulation of axon extension involved in axon guidance\n",
      "    - positive regulation of axon extension involved in axon guidance\n",
      "    - positive regulation of axon extension involved in axon guidance\n",
      "    - negative regulation of axon extension involved in axon guidance\n",
      "    - artery morphogenesis\n",
      "    - artery morphogenesis\n",
      "    - axon extension involved in axon guidance\n",
      "    - axon extension involved in axon guidance\n",
      "    - positive chemotaxis\n",
      "    - positive chemotaxis\n",
      "    - positive regulation of filopodium assembly\n",
      "    - positive regulation of filopodium assembly\n",
      "    - positive regulation of filopodium assembly\n",
      "    - positive regulation of stress fiber assembly\n",
      "    - positive regulation of focal adhesion assembly\n",
      "    - axonogenesis involved in innervation\n",
      "    - axonogenesis involved in innervation\n",
      "    - regulation of vesicle-mediated transport\n",
      "    - dichotomous subdivision of terminal units involved in salivary gland branching\n",
      "    - angiogenesis involved in coronary vascular morphogenesis\n",
      "    - angiogenesis involved in coronary vascular morphogenesis\n",
      "    - coronary artery morphogenesis\n",
      "    - retina vasculature development in camera-type eye\n",
      "    - retina vasculature morphogenesis in camera-type eye\n",
      "    - retina vasculature morphogenesis in camera-type eye\n",
      "    - renal artery morphogenesis\n",
      "    - sympathetic ganglion development\n",
      "    - sympathetic ganglion development\n",
      "    - trigeminal ganglion development\n",
      "    - positive regulation of ERK1 and ERK2 cascade\n",
      "    - positive regulation of ERK1 and ERK2 cascade\n",
      "    - semaphorin-plexin signaling pathway\n",
      "    - semaphorin-plexin signaling pathway\n",
      "    - semaphorin-plexin signaling pathway\n",
      "    - commissural neuron axon guidance\n",
      "    - commissural neuron axon guidance\n",
      "    - positive regulation of cell migration involved in sprouting angiogenesis\n",
      "    - endothelial tip cell fate specification\n",
      "    - sensory neuron axon guidance\n",
      "    - motor neuron migration\n",
      "    - motor neuron migration\n",
      "    - sympathetic neuron projection extension\n",
      "    - sympathetic neuron projection extension\n",
      "    - sympathetic neuron projection guidance\n",
      "    - sympathetic neuron projection guidance\n",
      "    - postsynapse organization\n",
      "    - regulation of plasma membrane bounded cell projection organization\n",
      "    - basal dendrite development\n",
      "    - basal dendrite development\n",
      "    - basal dendrite arborization\n",
      "    - basal dendrite arborization\n",
      "    - positive regulation of substrate adhesion-dependent cell spreading\n",
      "    - neural crest cell migration involved in autonomic nervous system development\n",
      "    - neural crest cell migration involved in autonomic nervous system development\n",
      "    - protein localization to early endosome\n",
      "    - protein localization to early endosome\n",
      "    - facioacoustic ganglion development\n",
      "    - dorsal root ganglion morphogenesis\n",
      "    - otic placode development\n",
      "    - negative regulation of extrinsic apoptotic signaling pathway\n",
      "  CC:\n",
      "    - semaphorin receptor complex\n",
      "    - extracellular region\n",
      "    - extracellular space\n",
      "    - cytoplasm\n",
      "    - mitochondrion\n",
      "    - early endosome\n",
      "    - early endosome\n",
      "    - cytosol\n",
      "    - neurofilament\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - focal adhesion\n",
      "    - focal adhesion\n",
      "    - focal adhesion\n",
      "    - focal adhesion\n",
      "    - membrane\n",
      "    - axon\n",
      "    - axon\n",
      "    - axon\n",
      "    - cytoplasmic vesicle\n",
      "    - mitochondrial membrane\n",
      "    - neuron projection\n",
      "    - neuron projection\n",
      "    - receptor complex\n",
      "    - postsynaptic membrane\n",
      "    - sorting endosome\n",
      "    - sorting endosome\n",
      "    - glutamatergic synapse\n",
      "  MF:\n",
      "    - vascular endothelial growth factor receptor activity\n",
      "    - vascular endothelial growth factor receptor activity\n",
      "    - vascular endothelial growth factor receptor activity\n",
      "    - vascular endothelial growth factor receptor activity\n",
      "    - vascular endothelial growth factor receptor activity\n",
      "    - GTPase activator activity\n",
      "    - protein binding\n",
      "    - heparin binding\n",
      "    - coreceptor activity\n",
      "    - semaphorin receptor activity\n",
      "    - semaphorin receptor activity\n",
      "    - semaphorin receptor activity\n",
      "    - growth factor binding\n",
      "    - growth factor binding\n",
      "    - growth factor binding\n",
      "    - protein kinase binding\n",
      "    - cytokine binding\n",
      "    - vascular endothelial growth factor binding\n",
      "    - vascular endothelial growth factor binding\n",
      "    - vascular endothelial growth factor binding\n",
      "    - metal ion binding\n",
      "UniProt: {'Swiss-Prot': 'O14786', 'TrEMBL': ['A8K9V7', 'Q6X907', 'E9PEP6', 'Q59F20', 'H0Y4N6', 'Q5JWQ4', 'E7EX60', 'Q5T7F0', 'Q5JWQ2', 'H0Y7Z2', 'Q5JWQ6', 'V9GYL7', 'H0Y4A0']}\n",
      "\n",
      "Gene: RRAS2\n",
      "Symbol: RRAS2\n",
      "Name: RAS related 2\n",
      "Summary: This gene encodes a member of the R-Ras subfamily of Ras-like small GTPases. The encoded protein associates with the plasma membrane and may function as a signal transducer. This protein may play an important role in activating signal transduction pathways that control cell proliferation. Mutations in this gene are associated with the growth of certain tumors. Pseudogenes of this gene are found on chromosomes 1 and 2. Alternate splicing results in multiple transcript variants. [provided by RefSeq, Apr 2010].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - osteoblast differentiation\n",
      "    - signal transduction\n",
      "    - Ras protein signal transduction\n",
      "    - Ras protein signal transduction\n",
      "    - Ras protein signal transduction\n",
      "    - cell migration\n",
      "    - Schwann cell migration\n",
      "    - positive regulation of Schwann cell migration\n",
      "  CC:\n",
      "    - Golgi membrane\n",
      "    - endoplasmic reticulum\n",
      "    - Golgi apparatus\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - focal adhesion\n",
      "    - membrane\n",
      "    - membrane\n",
      "    - extracellular exosome\n",
      "  MF:\n",
      "    - nucleotide binding\n",
      "    - GTPase activity\n",
      "    - GTPase activity\n",
      "    - GTPase activity\n",
      "    - GTPase activity\n",
      "    - protein binding\n",
      "    - GTP binding\n",
      "    - GTP binding\n",
      "    - hydrolase activity\n",
      "    - GDP binding\n",
      "UniProt: {'Swiss-Prot': 'P62070', 'TrEMBL': ['E9PQK5', 'E9PQ87', 'E9PK85', 'E9PQC5']}\n",
      "\n",
      "Gene: ARRB1\n",
      "Symbol: ARRB1\n",
      "Name: arrestin beta 1\n",
      "Summary: Members of arrestin/beta-arrestin protein family are thought to participate in agonist-mediated desensitization of G-protein-coupled receptors and cause specific dampening of cellular responses to stimuli such as hormones, neurotransmitters, or sensory signals. Arrestin beta 1 is a cytosolic protein and acts as a cofactor in the beta-adrenergic receptor kinase (BARK) mediated desensitization of beta-adrenergic receptors. Besides the central nervous system, it is expressed at high levels in peripheral blood leukocytes, and thus the BARK/beta-arrestin system is believed to play a major role in regulating receptor-mediated immune functions. Alternatively spliced transcripts encoding different isoforms of arrestin beta 1 have been described. [provided by RefSeq, Jan 2011].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - positive regulation of protein phosphorylation\n",
      "    - desensitization of G protein-coupled receptor signaling pathway\n",
      "    - G protein-coupled receptor internalization\n",
      "    - G protein-coupled receptor internalization\n",
      "    - G protein-coupled receptor internalization\n",
      "    - positive regulation of receptor internalization\n",
      "    - positive regulation of receptor internalization\n",
      "    - regulation of transcription by RNA polymerase II\n",
      "    - ubiquitin-dependent protein catabolic process\n",
      "    - signal transduction\n",
      "    - sensory perception\n",
      "    - negative regulation of signal transduction\n",
      "    - positive regulation of cardiac muscle hypertrophy\n",
      "    - positive regulation of cardiac muscle hypertrophy\n",
      "    - protein transport\n",
      "    - protein ubiquitination\n",
      "    - negative regulation of protein ubiquitination\n",
      "    - negative regulation of protein ubiquitination\n",
      "    - negative regulation of NF-kappaB transcription factor activity\n",
      "    - negative regulation of interleukin-6 production\n",
      "    - negative regulation of interleukin-8 production\n",
      "    - positive regulation of Rho protein signal transduction\n",
      "    - stress fiber assembly\n",
      "    - proteasome-mediated ubiquitin-dependent protein catabolic process\n",
      "    - proteasome-mediated ubiquitin-dependent protein catabolic process\n",
      "    - negative regulation of Notch signaling pathway\n",
      "    - negative regulation of Notch signaling pathway\n",
      "    - positive regulation of transcription by RNA polymerase II\n",
      "    - positive regulation of ERK1 and ERK2 cascade\n",
      "    - positive regulation of ERK1 and ERK2 cascade\n",
      "    - positive regulation of intracellular signal transduction\n",
      "  CC:\n",
      "    - Golgi membrane\n",
      "    - chromatin\n",
      "    - nucleus\n",
      "    - nucleus\n",
      "    - nucleus\n",
      "    - nucleoplasm\n",
      "    - cytoplasm\n",
      "    - cytoplasm\n",
      "    - cytoplasm\n",
      "    - lysosomal membrane\n",
      "    - cytosol\n",
      "    - cytosol\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - clathrin-coated pit\n",
      "    - membrane\n",
      "    - nuclear body\n",
      "    - cytoplasmic vesicle membrane\n",
      "    - endocytic vesicle membrane\n",
      "    - pseudopodium\n",
      "    - cytoplasmic vesicle\n",
      "    - cytoplasmic vesicle\n",
      "    - cytoplasmic vesicle\n",
      "    - cell projection\n",
      "  MF:\n",
      "    - G protein-coupled receptor binding\n",
      "    - G protein-coupled receptor binding\n",
      "    - transcription coactivator activity\n",
      "    - histone acetyltransferase activity\n",
      "    - enzyme inhibitor activity\n",
      "    - GTPase activator activity\n",
      "    - insulin-like growth factor receptor binding\n",
      "    - protein binding\n",
      "    - enzyme binding\n",
      "    - ubiquitin protein ligase binding\n",
      "    - angiotensin receptor binding\n",
      "    - angiotensin receptor binding\n",
      "    - molecular adaptor activity\n",
      "    - molecular adaptor activity\n",
      "    - molecular adaptor activity\n",
      "    - arrestin family protein binding\n",
      "UniProt: {'Swiss-Prot': 'P49407', 'TrEMBL': ['H0YD81', 'B7Z1Q3', 'H0YDS1', 'H0YEQ2', 'E9PM35', 'H0YET9']}\n",
      "\n",
      "Gene: RPIA\n",
      "Symbol: RPIA\n",
      "Name: ribose 5-phosphate isomerase A\n",
      "Summary: The protein encoded by this gene is an enzyme, which catalyzes the reversible conversion between ribose-5-phosphate and ribulose-5-phosphate in the pentose-phosphate pathway. This gene is highly conserved in most organisms. The enzyme plays an essential role in the carbohydrate metabolism. Mutations in this gene cause ribose 5-phosphate isomerase deficiency. A pseudogene is found on chromosome 18. [provided by RefSeq, Mar 2010].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - D-ribose metabolic process\n",
      "    - pentose-phosphate shunt\n",
      "    - pentose-phosphate shunt\n",
      "    - pentose-phosphate shunt, non-oxidative branch\n",
      "    - pentose-phosphate shunt, non-oxidative branch\n",
      "  CC:\n",
      "    - cytoplasm\n",
      "    - mitochondrion\n",
      "    - cytosol\n",
      "    - cytosol\n",
      "    - cytosol\n",
      "  MF:\n",
      "    - ribose-5-phosphate isomerase activity\n",
      "    - ribose-5-phosphate isomerase activity\n",
      "    - ribose-5-phosphate isomerase activity\n",
      "    - ribose-5-phosphate isomerase activity\n",
      "    - ribose-5-phosphate isomerase activity\n",
      "    - protein binding\n",
      "    - isomerase activity\n",
      "    - carbohydrate binding\n",
      "    - identical protein binding\n",
      "    - monosaccharide binding\n",
      "UniProt: {'Swiss-Prot': 'P49247'}\n",
      "\n",
      "Gene: FBLN1\n",
      "Symbol: FBLN1\n",
      "Name: fibulin 1\n",
      "Summary: Fibulin 1 is a secreted glycoprotein that becomes incorporated into a fibrillar extracellular matrix. Calcium-binding is apparently required to mediate its binding to laminin and nidogen. It mediates platelet adhesion via binding fibrinogen. Four splice variants which differ in the 3' end have been identified. Each variant encodes a different isoform, but no functional distinctions have been identified among the four variants. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - negative regulation of protein phosphorylation\n",
      "    - negative regulation of cell adhesion\n",
      "    - integrin-mediated signaling pathway\n",
      "    - embryo implantation\n",
      "    - positive regulation of gene expression\n",
      "    - extracellular matrix organization\n",
      "    - extracellular matrix organization\n",
      "    - positive regulation of fibroblast proliferation\n",
      "    - negative regulation of ERK1 and ERK2 cascade\n",
      "    - negative regulation of transforming growth factor beta production\n",
      "    - blood coagulation, fibrin clot formation\n",
      "    - negative regulation of substrate adhesion-dependent cell spreading\n",
      "    - negative regulation of transformation of host cell by virus\n",
      "    - positive regulation of substrate-dependent cell migration, cell attachment to substrate\n",
      "    - negative regulation of cell motility\n",
      "    - negative regulation of stem cell proliferation\n",
      "  CC:\n",
      "    - extracellular region\n",
      "    - extracellular region\n",
      "    - extracellular region\n",
      "    - extracellular region\n",
      "    - basement membrane\n",
      "    - extracellular space\n",
      "    - extracellular space\n",
      "    - extracellular matrix\n",
      "    - extracellular matrix\n",
      "    - extracellular matrix\n",
      "    - extracellular matrix\n",
      "    - extracellular exosome\n",
      "    - elastic fiber\n",
      "  MF:\n",
      "    - fibronectin binding\n",
      "    - integrin binding\n",
      "    - integrin binding\n",
      "    - extracellular matrix structural constituent\n",
      "    - extracellular matrix structural constituent\n",
      "    - calcium ion binding\n",
      "    - calcium ion binding\n",
      "    - protein binding\n",
      "    - peptidase activator activity\n",
      "    - identical protein binding\n",
      "    - identical protein binding\n",
      "    - protein-containing complex binding\n",
      "    - fibrinogen binding\n",
      "UniProt: {'Swiss-Prot': 'P23142', 'TrEMBL': ['A0A8S0LWY1', 'Q8NBH6', 'B1AHM9', 'B1AHL2', 'A0A0U1RRA4', 'B1AHM6', 'H7C1M6', 'B1AHM5', 'B1AHM7', 'B1AHN3', 'B1AHM8']}\n",
      "\n",
      "Gene: BCL10\n",
      "Symbol: BCL10\n",
      "Name: BCL10 immune signaling adaptor\n",
      "Summary: This gene was identified by its translocation in a case of mucosa-associated lymphoid tissue (MALT) lymphoma. The protein encoded by this gene contains a caspase recruitment domain (CARD), and has been shown to induce apoptosis and to activate NF-kappaB. This protein is reported to interact with other CARD domain containing proteins including CARD9, 10, 11 and 14, which are thought to function as upstream regulators in NF-kappaB signaling. This protein is found to form a complex with MALT1, a protein encoded by another gene known to be translocated in MALT lymphoma. MALT1 and this protein are thought to synergize in the activation of NF-kappaB, and the deregulation of either of them may contribute to the same pathogenetic process that leads to the malignancy. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Mar 2016].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - B cell apoptotic process\n",
      "    - positive regulation of cytokine production\n",
      "    - neural tube closure\n",
      "    - neural tube closure\n",
      "    - toll-like receptor signaling pathway\n",
      "    - adaptive immune response\n",
      "    - adaptive immune response\n",
      "    - adaptive immune response\n",
      "    - adaptive immune response\n",
      "    - immune system process\n",
      "    - negative regulation of mature B cell apoptotic process\n",
      "    - apoptotic process\n",
      "    - cellular defense response\n",
      "    - response to fungus\n",
      "    - programmed cell death\n",
      "    - immunoglobulin mediated immune response\n",
      "    - positive regulation of protein ubiquitination\n",
      "    - lipopolysaccharide-mediated signaling pathway\n",
      "    - response to food\n",
      "    - response to lipopolysaccharide\n",
      "    - positive regulation of interleukin-6 production\n",
      "    - positive regulation of interleukin-8 production\n",
      "    - positive regulation of lymphotoxin A production\n",
      "    - positive regulation of mast cell cytokine production\n",
      "    - non-canonical NF-kappaB signal transduction\n",
      "    - positive regulation of phosphorylation\n",
      "    - regulation of apoptotic process\n",
      "    - positive regulation of apoptotic process\n",
      "    - positive regulation of apoptotic process\n",
      "    - positive regulation of canonical NF-kappaB signal transduction\n",
      "    - positive regulation of canonical NF-kappaB signal transduction\n",
      "    - positive regulation of canonical NF-kappaB signal transduction\n",
      "    - innate immune response\n",
      "    - innate immune response\n",
      "    - positive regulation of DNA-templated transcription\n",
      "    - T cell receptor signaling pathway\n",
      "    - regulation of T cell receptor signaling pathway\n",
      "    - positive regulation of T cell receptor signaling pathway\n",
      "    - positive regulation of T cell activation\n",
      "    - positive regulation of NF-kappaB transcription factor activity\n",
      "    - protein homooligomerization\n",
      "    - antifungal innate immune response\n",
      "    - antifungal innate immune response\n",
      "    - T cell apoptotic process\n",
      "    - cellular response to lipopolysaccharide\n",
      "    - cellular response to mechanical stimulus\n",
      "    - apoptotic signaling pathway\n",
      "    - positive regulation of extrinsic apoptotic signaling pathway\n",
      "    - positive regulation of extrinsic apoptotic signaling pathway\n",
      "    - positive regulation of extrinsic apoptotic signaling pathway\n",
      "  CC:\n",
      "    - immunological synapse\n",
      "    - polkadots\n",
      "    - nucleus\n",
      "    - nucleoplasm\n",
      "    - cytoplasm\n",
      "    - cytoplasm\n",
      "    - lysosome\n",
      "    - cytosol\n",
      "    - cytosol\n",
      "    - cytosol\n",
      "    - cytosol\n",
      "    - cytoplasmic microtubule\n",
      "    - membrane\n",
      "    - CBM complex\n",
      "    - CBM complex\n",
      "    - CBM complex\n",
      "    - CBM complex\n",
      "    - protein-containing complex\n",
      "    - protein-containing complex\n",
      "    - membrane raft\n",
      "    - perinuclear region of cytoplasm\n",
      "    - perinuclear region of cytoplasm\n",
      "  MF:\n",
      "    - protease binding\n",
      "    - transcription coactivator activity\n",
      "    - transcription coactivator activity\n",
      "    - protein binding\n",
      "    - kinase activator activity\n",
      "    - kinase activator activity\n",
      "    - protein-macromolecule adaptor activity\n",
      "    - ubiquitin protein ligase binding\n",
      "    - signaling adaptor activity\n",
      "    - identical protein binding\n",
      "    - identical protein binding\n",
      "    - identical protein binding\n",
      "    - protein kinase B binding\n",
      "    - protein kinase B binding\n",
      "    - protein-containing complex binding\n",
      "    - CARD domain binding\n",
      "    - NF-kappaB binding\n",
      "    - NF-kappaB binding\n",
      "    - general transcription initiation factor binding\n",
      "UniProt: {'Swiss-Prot': 'O95999', 'TrEMBL': ['A2TDT2', 'A0A087WWW9', 'A0A3B3ISX2']}\n",
      "\n",
      "Gene: APPL1\n",
      "Symbol: APPL1\n",
      "Name: adaptor protein, phosphotyrosine interacting with PH domain and leucine zipper 1\n",
      "Summary: The protein encoded by this gene has been shown to be involved in the regulation of cell proliferation, and in the crosstalk between the adiponectin signalling and insulin signalling pathways. The encoded protein binds many other proteins, including RAB5A, DCC, AKT2, PIK3CA, adiponectin receptors, and proteins of the NuRD/MeCP1 complex. This protein is found associated with endosomal membranes, but can be released by EGF and translocated to the nucleus. [provided by RefSeq, Jul 2008].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - protein import into nucleus\n",
      "    - protein import into nucleus\n",
      "    - signal transduction\n",
      "    - transforming growth factor beta receptor signaling pathway\n",
      "    - transforming growth factor beta receptor signaling pathway\n",
      "    - insulin receptor signaling pathway\n",
      "    - insulin receptor signaling pathway\n",
      "    - insulin receptor signaling pathway\n",
      "    - positive regulation of biosynthetic process\n",
      "    - regulation of fibroblast migration\n",
      "    - regulation of fibroblast migration\n",
      "    - signaling\n",
      "    - adiponectin-activated signaling pathway\n",
      "    - regulation of toll-like receptor 4 signaling pathway\n",
      "    - regulation of toll-like receptor 4 signaling pathway\n",
      "    - cellular response to hepatocyte growth factor stimulus\n",
      "    - cellular response to hepatocyte growth factor stimulus\n",
      "    - regulation of innate immune response\n",
      "    - regulation of innate immune response\n",
      "    - regulation of D-glucose import\n",
      "    - positive regulation of D-glucose import\n",
      "    - positive regulation of melanin biosynthetic process\n",
      "    - positive regulation of transport\n",
      "    - maintenance of synapse structure\n",
      "    - maintenance of synapse structure\n",
      "    - positive regulation of cytokine production involved in inflammatory response\n",
      "    - positive regulation of cytokine production involved in inflammatory response\n",
      "    - regulation of protein localization to plasma membrane\n",
      "    - positive regulation of macropinocytosis\n",
      "    - positive regulation of macropinocytosis\n",
      "    - negative regulation of Fc-gamma receptor signaling pathway involved in phagocytosis\n",
      "    - negative regulation of Fc-gamma receptor signaling pathway involved in phagocytosis\n",
      "    - regulation of G1/S transition of mitotic cell cycle\n",
      "    - regulation of G1/S transition of mitotic cell cycle\n",
      "  CC:\n",
      "    - ruffle\n",
      "    - ruffle\n",
      "    - nucleus\n",
      "    - nucleus\n",
      "    - cytoplasm\n",
      "    - cytoplasm\n",
      "    - endosome\n",
      "    - endosome\n",
      "    - early endosome\n",
      "    - early endosome\n",
      "    - early endosome\n",
      "    - cytosol\n",
      "    - plasma membrane\n",
      "    - endosome membrane\n",
      "    - endosome membrane\n",
      "    - endosome membrane\n",
      "    - vesicle membrane\n",
      "    - actin cytoskeleton\n",
      "    - membrane\n",
      "    - membrane\n",
      "    - cytoplasmic vesicle\n",
      "    - cytoplasmic vesicle\n",
      "    - early endosome membrane\n",
      "    - early phagosome\n",
      "    - early phagosome\n",
      "    - cell projection\n",
      "    - macropinosome\n",
      "    - macropinosome\n",
      "    - phagocytic vesicle\n",
      "    - extracellular exosome\n",
      "    - intracellular vesicle\n",
      "    - glutamatergic synapse\n",
      "    - glutamatergic synapse\n",
      "  MF:\n",
      "    - phosphatidylserine binding\n",
      "    - phosphatidylserine binding\n",
      "    - protein binding\n",
      "    - phosphatidylinositol binding\n",
      "    - identical protein binding\n",
      "    - identical protein binding\n",
      "    - protein homodimerization activity\n",
      "    - protein kinase B binding\n",
      "    - protein kinase B binding\n",
      "    - protein-containing complex binding\n",
      "    - beta-tubulin binding\n",
      "UniProt: {'Swiss-Prot': 'Q9UKG1', 'TrEMBL': ['C9JAB0', 'C9K0C4']}\n",
      "\n",
      "Gene: ELOVL5\n",
      "Symbol: ELOVL5\n",
      "Name: ELOVL fatty acid elongase 5\n",
      "Summary: This gene belongs to the ELO family. It is highly expressed in the adrenal gland and testis, and encodes a multi-pass membrane protein that is localized in the endoplasmic reticulum. This protein is involved in the elongation of long-chain polyunsaturated fatty acids. Mutations in this gene have been associated with spinocerebellar ataxia-38 (SCA38). Alternatively spliced transcript variants have been found for this gene. [provided by RefSeq, Sep 2014].\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - lipid metabolic process\n",
      "    - fatty acid metabolic process\n",
      "    - fatty acid biosynthetic process\n",
      "    - unsaturated fatty acid biosynthetic process\n",
      "    - fatty acid elongation, saturated fatty acid\n",
      "    - fatty acid elongation, saturated fatty acid\n",
      "    - sphingolipid biosynthetic process\n",
      "    - fatty acid elongation\n",
      "    - fatty acid elongation, monounsaturated fatty acid\n",
      "    - fatty acid elongation, monounsaturated fatty acid\n",
      "    - fatty acid elongation, monounsaturated fatty acid\n",
      "    - fatty acid elongation, polyunsaturated fatty acid\n",
      "    - fatty acid elongation, polyunsaturated fatty acid\n",
      "    - fatty acid elongation, polyunsaturated fatty acid\n",
      "    - long-chain fatty-acyl-CoA biosynthetic process\n",
      "    - long-chain fatty-acyl-CoA biosynthetic process\n",
      "    - alpha-linolenic acid metabolic process\n",
      "    - alpha-linolenic acid metabolic process\n",
      "    - long-chain fatty acid biosynthetic process\n",
      "    - very long-chain fatty acid biosynthetic process\n",
      "    - very long-chain fatty acid biosynthetic process\n",
      "    - very long-chain fatty acid biosynthetic process\n",
      "    - linoleic acid metabolic process\n",
      "    - linoleic acid metabolic process\n",
      "    - positive regulation of fatty acid biosynthetic process\n",
      "    - fatty acid derivative biosynthetic process\n",
      "  CC:\n",
      "    - endoplasmic reticulum\n",
      "    - endoplasmic reticulum\n",
      "    - endoplasmic reticulum membrane\n",
      "    - endoplasmic reticulum membrane\n",
      "    - endoplasmic reticulum membrane\n",
      "    - endoplasmic reticulum membrane\n",
      "    - membrane\n",
      "    - membrane\n",
      "    - dendrite\n",
      "    - cell projection\n",
      "    - neuronal cell body\n",
      "    - dendritic tree\n",
      "  MF:\n",
      "    - protein binding\n",
      "    - fatty acid elongase activity\n",
      "    - fatty acid elongase activity\n",
      "    - fatty acid elongase activity\n",
      "    - fatty acid elongase activity\n",
      "    - fatty acid elongase activity\n",
      "    - transferase activity\n",
      "UniProt: {'Swiss-Prot': 'Q9NYP7', 'TrEMBL': ['B3KWH9', 'A0A0A0MTI6']}\n",
      "\n",
      "Gene: REPS1\n",
      "Symbol: REPS1\n",
      "Name: RALBP1 associated Eps domain containing 1\n",
      "Summary: This gene encodes a signaling adaptor protein with two EH domains that interacts with proteins that participate in signaling, endocytosis and cytoskeletal changes. The encoded protein has been found in association with intersectin 1 and Src homology 3-domain growth factor receptor-bound 2-like (endophilin) interacting protein 1 when intersectin 1 was isolated from clathrin-coated pits. The encoded protein has also been shown to interact with amphiphysin, a cytoplasmic protein at the surface of synaptic vesicles. Alternative splicing results in multiple transcript variants encoding different isoforms. [provided by RefSeq, Mar 2014]\n",
      "GO Annotations:\n",
      "  BP:\n",
      "    - endocytosis\n",
      "    - endosomal transport\n",
      "  CC:\n",
      "    - nucleoplasm\n",
      "    - cytoplasm\n",
      "    - cytosol\n",
      "    - cytosol\n",
      "    - cytosol\n",
      "    - plasma membrane\n",
      "    - plasma membrane\n",
      "    - clathrin-coated pit\n",
      "    - clathrin-coated pit\n",
      "    - membrane\n",
      "  MF:\n",
      "    - calcium ion binding\n",
      "    - protein binding\n",
      "    - SH3 domain binding\n",
      "    - protein-macromolecule adaptor activity\n",
      "    - metal ion binding\n",
      "UniProt: {'Swiss-Prot': 'Q96D71', 'TrEMBL': ['F2Z3L2', 'B2R7D3', 'H0YE89', 'H0YDT0', 'H7C1V2', 'H7C225', 'H0YCF0', 'E9PMG1', 'H0YE73', 'H0YCR2']}\n"
     ]
    }
   ],
   "source": [
    "mg = mygene.MyGeneInfo()\n",
    "gene_info = {}\n",
    "\n",
    "for gene in predictions[predictions[\"y_true\"] == 0].head(25)[\"gene_name\"]:\n",
    "    result = mg.query(\n",
    "        gene, species=\"human\", fields=\"symbol,name,summary,go,uniprot\", size=1\n",
    "    )\n",
    "    if result[\"hits\"]:\n",
    "        gene_info[gene] = result[\"hits\"][0]\n",
    "    else:\n",
    "        gene_info[gene] = \"No information found\"\n",
    "\n",
    "print(\"\\nBiological information for each gene:\")\n",
    "for gene, info in gene_info.items():\n",
    "    print(f\"\\nGene: {gene}\")\n",
    "    if isinstance(info, dict):\n",
    "        print(\"Symbol:\", info.get(\"symbol\", \"N/A\"))\n",
    "        print(\"Name:\", info.get(\"name\", \"N/A\"))\n",
    "        print(\"Summary:\", info.get(\"summary\", \"N/A\"))\n",
    "        if \"go\" in info:\n",
    "            print(\"GO Annotations:\")\n",
    "            for category, terms in info[\"go\"].items():\n",
    "                print(f\"  {category.upper()}:\")\n",
    "                # 'terms' may be a list or a single dict\n",
    "                if isinstance(terms, list):\n",
    "                    for term in terms:\n",
    "                        print(\"    -\", term.get(\"term\", \"N/A\"))\n",
    "                elif isinstance(terms, dict):\n",
    "                    print(\"    -\", terms.get(\"term\", \"N/A\"))\n",
    "        if \"uniprot\" in info:\n",
    "            print(\"UniProt:\", info[\"uniprot\"])\n",
    "    else:\n",
    "        print(info)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "847b3d86",
   "metadata": {},
   "outputs": [],
   "source": [
    "CHAT https://www.sciencedirect.com/science/article/pii/S0306452217308151 & https://www.nature.com/articles/s41598-024-78751-2\n",
    "\n"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": ".dg_env (3.12.8)",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.12.8"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 5
}
